{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "env: STUDY=Apprill_transmissionMucus2019\n",
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus\n"
     ]
    }
   ],
   "source": [
    "%env STUDY Apprill_transmissionMucus2019\n",
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "for i in {I..M}\n",
    "do\n",
    "mkdir \"Lib${i}_folder\"\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibI_folder\n",
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibJ_folder\n",
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibK_folder\n",
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibL_folder\n",
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibM_folder\n"
     ]
    }
   ],
   "source": [
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibI_folder\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_LibI_$STUDY.txt\n",
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibJ_folder\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_LibJ_$STUDY.txt\n",
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibK_folder\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_LibK_$STUDY.txt\n",
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibL_folder\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_LibL_$STUDY.txt\n",
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibM_folder\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_LibM_$STUDY.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as ps\n",
    "import numpy as np\n",
    "\n",
    "dir_path='/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibI_folder/'\n",
    "\n",
    "\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_LibI_Apprill_transmissionMucus2019.txt',\n",
    "                                    names=header_list)\n",
    "\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_L001\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "\n",
    "\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpMucus_LibI.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "dir_path='/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibJ_folder/'\n",
    "\n",
    "\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_LibJ_Apprill_transmissionMucus2019.txt',\n",
    "                                    names=header_list)\n",
    "\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_L001\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "\n",
    "\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpMucus_LibJ.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "dir_path='/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibK_folder/'\n",
    "\n",
    "\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_LibK_Apprill_transmissionMucus2019.txt',\n",
    "                                    names=header_list)\n",
    "\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_L001\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "\n",
    "\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpMucus_LibK.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [],
   "source": [
    "dir_path='/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibL_folder/'\n",
    "\n",
    "\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_LibL_Apprill_transmissionMucus2019.txt',\n",
    "                                    names=header_list)\n",
    "\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_L001\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "\n",
    "\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpMucus_LibL.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "dir_path='/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIApril2019TransmissionExpMucus/LibM_folder/'\n",
    "\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_LibM_Apprill_transmissionMucus2019.txt',\n",
    "                                    names=header_list)\n",
    "\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_L001\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "\n",
    "\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpMucus_LibM.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/projects/2020_SCTLD_meta-analysis\n"
     ]
    }
   ],
   "source": [
    "%cd /space/home/rosales/projects/2020_SCTLD_meta-analysis/\n",
    "#!mkdir Folder_Apprill_transmissionMucus2019"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Imported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpMucus_LibI.csv as PairedEndFastqManifestPhred33V2 to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_LibI\n",
      "Imported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpMucus_LibJ.csv as PairedEndFastqManifestPhred33V2 to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_LibJ\n",
      "Imported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpMucus_LibK.csv as PairedEndFastqManifestPhred33V2 to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_LibK\n",
      "Imported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpMucus_LibL.csv as PairedEndFastqManifestPhred33V2 to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_LibL\n",
      "Imported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpMucus_LibM.csv as PairedEndFastqManifestPhred33V2 to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_LibM\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for f1 in /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpMucus_Lib*\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.csv};\n",
    "\n",
    "qiime tools import \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_transmissionMucus2019/Import_$fn \\\n",
    "--type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "--input-path $f1 \\\n",
    "--input-format PairedEndFastqManifestPhred33V2\n",
    "\n",
    "done\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_LibI.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_LibJ.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_LibK.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_LibL.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_LibM.qzv\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for f1 in /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_Lib*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data $f1 \\\n",
    "--o-visualization Folder_Apprill_transmissionMucus2019/$fn.qzv\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/13-DNA-cont_S3_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/13-DNA-cont_S3_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/13-DNA-cont_S3_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/13-DNA-cont_S3_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.75 s (54 us/read; 1.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,846\n",
      "  Read 1 with adapter:                  13,789 (99.6%)\n",
      "  Read 2 with adapter:                  13,755 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        13,846 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,161,443 bp\n",
      "  Read 1:     2,081,614 bp\n",
      "  Read 2:     2,079,829 bp\n",
      "Total written (filtered):      3,624,519 bp (87.1%)\n",
      "  Read 1:     1,819,685 bp\n",
      "  Read 2:     1,804,834 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13789 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t93\t0.0\t1\t9 84\n",
      "19\t13657\t0.0\t1\t12739 918\n",
      "20\t37\t0.0\t1\t9 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13755 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t54.1\t0\t1\n",
      "19\t121\t0.0\t1\t18 103\n",
      "20\t13601\t0.0\t2\t13203 398\n",
      "21\t32\t0.0\t2\t11 21\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1304G-TF-H-PA_S29_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1304G-TF-H-PA_S29_38_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1304G-TF-H-PA_S29_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1304G-TF-H-PA_S29_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.61 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,553\n",
      "  Read 1 with adapter:                 115,258 (98.0%)\n",
      "  Read 2 with adapter:                 116,857 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       117,553 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,484,436 bp\n",
      "  Read 1:    17,749,980 bp\n",
      "  Read 2:    17,734,456 bp\n",
      "Total written (filtered):     30,958,883 bp (87.2%)\n",
      "  Read 1:    15,560,420 bp\n",
      "  Read 2:    15,398,463 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 115258 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t9\t0.0\t1\t3 6\n",
      "18\t566\t0.0\t1\t47 519\n",
      "19\t114398\t0.0\t1\t100368 14030\n",
      "20\t278\t0.0\t1\t64 214\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 116857 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t24\t459.2\t0\t24\n",
      "11\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t9\t0.0\t1\t8 1\n",
      "19\t999\t0.0\t1\t136 863\n",
      "20\t115542\t0.0\t2\t112047 3495\n",
      "21\t275\t0.0\t2\t64 211\n",
      "32\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1305C-T1-H-OA-M_S19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1305C-T1-H-OA-M_S19_39_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1305C-T1-H-OA-M_S19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1305C-T1-H-OA-M_S19_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.55 s (21 us/read; 2.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            165,613\n",
      "  Read 1 with adapter:                 164,967 (99.6%)\n",
      "  Read 2 with adapter:                 164,796 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       165,613 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,692,505 bp\n",
      "  Read 1:    24,857,806 bp\n",
      "  Read 2:    24,834,699 bp\n",
      "Total written (filtered):     43,263,965 bp (87.1%)\n",
      "  Read 1:    21,723,937 bp\n",
      "  Read 2:    21,540,028 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 164967 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t4\t0.0\t1\t2 2\n",
      "14\t4\t0.0\t1\t3 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t11\t0.0\t1\t8 3\n",
      "18\t788\t0.0\t1\t104 684\n",
      "19\t163699\t0.0\t1\t154127 9572\n",
      "20\t444\t0.0\t1\t82 362\n",
      "26\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 164796 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t18\t646.9\t0\t18\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t2\t0.0\t1\t2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t14\t0.0\t1\t7 7\n",
      "19\t1277\t0.0\t1\t204 1073\n",
      "20\t163068\t0.0\t2\t158503 4565\n",
      "21\t406\t0.0\t2\t84 322\n",
      "23\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1307-T1-D-DL-M_S17_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1307-T1-D-DL-M_S17_40_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1307-T1-D-DL-M_S17_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1307-T1-D-DL-M_S17_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.48 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            156,228\n",
      "  Read 1 with adapter:                 155,789 (99.7%)\n",
      "  Read 2 with adapter:                 155,505 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       156,228 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,900,298 bp\n",
      "  Read 1:    23,459,953 bp\n",
      "  Read 2:    23,440,345 bp\n",
      "Total written (filtered):     40,831,600 bp (87.1%)\n",
      "  Read 1:    20,500,359 bp\n",
      "  Read 2:    20,331,241 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 155789 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.6\t0\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t3\t0.0\t1\t0 3\n",
      "16\t8\t0.0\t1\t4 4\n",
      "17\t11\t0.0\t1\t6 5\n",
      "18\t660\t0.0\t1\t82 578\n",
      "19\t154698\t0.0\t1\t145950 8748\n",
      "20\t398\t0.0\t1\t80 318\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 155505 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t14\t610.3\t0\t14\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t17\t0.0\t1\t8 9\n",
      "19\t1031\t0.0\t1\t180 851\n",
      "20\t154086\t0.0\t2\t150136 3950\n",
      "21\t345\t0.0\t2\t75 270\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1312-T1-H-DL-M_S12_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1312-T1-H-DL-M_S12_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1312-T1-H-DL-M_S12_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1312-T1-H-DL-M_S12_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.67 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,672\n",
      "  Read 1 with adapter:                 117,232 (99.6%)\n",
      "  Read 2 with adapter:                 117,076 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       117,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,410,411 bp\n",
      "  Read 1:    17,722,718 bp\n",
      "  Read 2:    17,687,693 bp\n",
      "Total written (filtered):     30,842,776 bp (87.1%)\n",
      "  Read 1:    15,495,683 bp\n",
      "  Read 2:    15,347,093 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 117232 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t4\t0.4\t0\t4\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t589\t0.0\t1\t51 538\n",
      "19\t116297\t0.0\t1\t109314 6983\n",
      "20\t326\t0.0\t1\t61 265\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 117076 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1838.6\t0\t1\n",
      "4\t17\t459.7\t0\t17\n",
      "10\t3\t0.1\t1\t3\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t10\t0.0\t1\t8 2\n",
      "19\t840\t0.0\t1\t153 687\n",
      "20\t115892\t0.0\t2\t113180 2712\n",
      "21\t302\t0.0\t2\t61 241\n",
      "23\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1320E-T1-H-PS-M_LibI_S26_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1320E-T1-H-PS-M_LibI_S26_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1320E-T1-H-PS-M_LibI_S26_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1320E-T1-H-PS-M_LibI_S26_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.19 s (22 us/read; 2.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,063\n",
      "  Read 1 with adapter:                 141,450 (99.6%)\n",
      "  Read 2 with adapter:                 141,221 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       142,063 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,810,045 bp\n",
      "  Read 1:    21,414,775 bp\n",
      "  Read 2:    21,395,270 bp\n",
      "Total written (filtered):     37,299,874 bp (87.1%)\n",
      "  Read 1:    18,727,774 bp\n",
      "  Read 2:    18,572,100 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 141450 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.5\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t5\t0.0\t1\t4 1\n",
      "13\t7\t0.0\t1\t6 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t5\t0.0\t1\t4 1\n",
      "17\t5\t0.0\t1\t5\n",
      "18\t789\t0.0\t1\t86 703\n",
      "19\t140234\t0.0\t1\t131001 9233\n",
      "20\t397\t0.0\t1\t66 331\n",
      "23\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 141221 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2219.7\t0\t1\n",
      "4\t22\t554.9\t0\t22\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t4\t0.0\t1\t1 3\n",
      "18\t17\t0.0\t1\t9 8\n",
      "19\t1137\t0.0\t1\t184 953\n",
      "20\t139646\t0.0\t2\t135661 3985\n",
      "21\t383\t0.0\t2\t81 302\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1326D-TF-D-PS-M_S34_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1326D-TF-D-PS-M_S34_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1326D-TF-D-PS-M_S34_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1326D-TF-D-PS-M_S34_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.37 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            158,786\n",
      "  Read 1 with adapter:                 158,001 (99.5%)\n",
      "  Read 2 with adapter:                 157,893 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       158,786 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,935,236 bp\n",
      "  Read 1:    23,973,213 bp\n",
      "  Read 2:    23,962,023 bp\n",
      "Total written (filtered):     41,777,483 bp (87.2%)\n",
      "  Read 1:    20,971,908 bp\n",
      "  Read 2:    20,805,575 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 158001 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t2.4\t0\t1\n",
      "9\t4\t0.6\t0\t4\n",
      "10\t2\t0.2\t1\t2\n",
      "12\t9\t0.0\t1\t9\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t4\t0.0\t1\t1 3\n",
      "16\t7\t0.0\t1\t2 5\n",
      "17\t9\t0.0\t1\t3 6\n",
      "18\t943\t0.0\t1\t91 852\n",
      "19\t156574\t0.0\t1\t145109 11465\n",
      "20\t443\t0.0\t1\t80 363\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 157893 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2481.0\t0\t1\n",
      "4\t20\t620.3\t0\t20\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t20\t0.0\t1\t11 9\n",
      "19\t1350\t0.0\t1\t207 1143\n",
      "20\t156077\t0.0\t2\t151199 4878\n",
      "21\t409\t0.0\t2\t78 331\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1350D-T1-H-SS-M_S18_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1350D-T1-H-SS-M_S18_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1350D-T1-H-SS-M_S18_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1350D-T1-H-SS-M_S18_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.38 s (21 us/read; 2.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            158,192\n",
      "  Read 1 with adapter:                 157,667 (99.7%)\n",
      "  Read 2 with adapter:                 157,202 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       158,192 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,161,237 bp\n",
      "  Read 1:    23,589,278 bp\n",
      "  Read 2:    23,571,959 bp\n",
      "Total written (filtered):     41,023,409 bp (87.0%)\n",
      "  Read 1:    20,594,167 bp\n",
      "  Read 2:    20,429,242 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 157667 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t2\t0.2\t1\t1 1\n",
      "11\t5\t0.0\t1\t2 3\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t13\t0.0\t1\t8 5\n",
      "18\t914\t0.0\t1\t85 829\n",
      "19\t156272\t0.0\t1\t146458 9814\n",
      "20\t450\t0.0\t1\t102 348\n",
      "31\t1\t0.0\t1\t1\n",
      "39\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 157202 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2471.8\t0\t1\n",
      "4\t17\t617.9\t0\t17\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t7\t0.0\t1\t2 5\n",
      "17\t7\t0.0\t1\t5 2\n",
      "18\t25\t0.0\t1\t9 16\n",
      "19\t1301\t0.0\t1\t180 1121\n",
      "20\t155443\t0.0\t2\t150892 4551\n",
      "21\t394\t0.0\t2\t93 301\n",
      "22\t1\t0.0\t2\t0 1\n",
      "25\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1350G-TF-D-SS-M_S10_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1350G-TF-D-SS-M_S10_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1350G-TF-D-SS-M_S10_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1350G-TF-D-SS-M_S10_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.89 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,706\n",
      "  Read 1 with adapter:                 129,062 (99.5%)\n",
      "  Read 2 with adapter:                 128,911 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       129,706 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,157,792 bp\n",
      "  Read 1:    19,583,370 bp\n",
      "  Read 2:    19,574,422 bp\n",
      "Total written (filtered):     34,129,065 bp (87.2%)\n",
      "  Read 1:    17,131,665 bp\n",
      "  Read 2:    16,997,400 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 129062 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t6\t0.0\t1\t2 4\n",
      "17\t5\t0.0\t1\t1 4\n",
      "18\t756\t0.0\t1\t70 686\n",
      "19\t127917\t0.0\t1\t119313 8604\n",
      "20\t367\t0.0\t1\t51 316\n",
      "21\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 128911 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t13\t506.7\t0\t13\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t3\t0.0\t1\t2 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t17\t0.0\t1\t10 7\n",
      "19\t1234\t0.0\t1\t177 1057\n",
      "20\t127282\t0.0\t2\t123680 3602\n",
      "21\t352\t0.0\t2\t66 286\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1356C-T1-H-MC-M_S21_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1356C-T1-H-MC-M_S21_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1356C-T1-H-MC-M_S21_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1356C-T1-H-MC-M_S21_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.66 s (19 us/read; 3.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            197,595\n",
      "  Read 1 with adapter:                 196,902 (99.6%)\n",
      "  Read 2 with adapter:                 196,539 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       197,595 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,621,887 bp\n",
      "  Read 1:    29,825,687 bp\n",
      "  Read 2:    29,796,200 bp\n",
      "Total written (filtered):     51,952,096 bp (87.1%)\n",
      "  Read 1:    26,085,138 bp\n",
      "  Read 2:    25,866,958 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 196902 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.8\t0\t1\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t8\t0.0\t1\t8\n",
      "13\t3\t0.0\t1\t1 2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t6\t0.0\t1\t3 3\n",
      "17\t8\t0.0\t1\t3 5\n",
      "18\t985\t0.0\t1\t122 863\n",
      "19\t195329\t0.0\t1\t182954 12375\n",
      "20\t555\t0.0\t1\t97 458\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 196539 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t3087.4\t0\t2\n",
      "4\t23\t771.9\t0\t23\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t3 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t21\t0.0\t1\t14 7\n",
      "19\t1520\t0.0\t1\t246 1274\n",
      "20\t194468\t0.0\t2\t188492 5976\n",
      "21\t491\t0.0\t2\t115 376\n",
      "29\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1360C-TF-H-MC-M_S31_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1360C-TF-H-MC-M_S31_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1360C-TF-H-MC-M_S31_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1360C-TF-H-MC-M_S31_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.51 s (20 us/read; 2.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            123,440\n",
      "  Read 1 with adapter:                 122,937 (99.6%)\n",
      "  Read 2 with adapter:                 122,746 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       123,440 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,210,061 bp\n",
      "  Read 1:    18,611,131 bp\n",
      "  Read 2:    18,598,930 bp\n",
      "Total written (filtered):     32,420,752 bp (87.1%)\n",
      "  Read 1:    16,275,671 bp\n",
      "  Read 2:    16,145,081 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 122937 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t2 2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t11\t0.0\t1\t5 6\n",
      "18\t580\t0.0\t1\t74 506\n",
      "19\t122012\t0.0\t1\t114292 7720\n",
      "20\t323\t0.0\t1\t66 257\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 122746 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t17\t482.2\t0\t17\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t10\t0.0\t1\t5 5\n",
      "19\t1042\t0.0\t1\t160 882\n",
      "20\t121365\t0.0\t2\t118277 3088\n",
      "21\t304\t0.0\t2\t64 240\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1363A-TF-H-MC-M_S11_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1363A-TF-H-MC-M_S11_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1363A-TF-H-MC-M_S11_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1363A-TF-H-MC-M_S11_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.88 s (19 us/read; 3.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            148,961\n",
      "  Read 1 with adapter:                 148,388 (99.6%)\n",
      "  Read 2 with adapter:                 148,188 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       148,961 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,169,340 bp\n",
      "  Read 1:    22,093,197 bp\n",
      "  Read 2:    22,076,143 bp\n",
      "Total written (filtered):     38,387,707 bp (86.9%)\n",
      "  Read 1:    19,274,272 bp\n",
      "  Read 2:    19,113,435 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 148388 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t4\t0.0\t1\t1 3\n",
      "12\t9\t0.0\t1\t9\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t7\t0.0\t1\t4 3\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t791\t0.0\t1\t93 698\n",
      "19\t147082\t0.0\t1\t138523 8559\n",
      "20\t477\t0.0\t1\t77 400\n",
      "21\t2\t0.0\t1\t2\n",
      "35\t1\t0.0\t1\t1\n",
      "42\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 148188 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2327.5\t0\t2\n",
      "4\t13\t581.9\t0\t13\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t17\t0.0\t1\t10 7\n",
      "19\t1076\t0.0\t1\t160 916\n",
      "20\t146744\t0.0\t2\t143118 3626\n",
      "21\t331\t0.0\t2\t68 263\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364A-TF-H-OA-M_S9_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364A-TF-H-OA-M_S9_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364A-TF-H-OA-M_S9_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364A-TF-H-OA-M_S9_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.52 s (19 us/read; 3.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            135,697\n",
      "  Read 1 with adapter:                 135,259 (99.7%)\n",
      "  Read 2 with adapter:                 134,979 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       135,697 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,181,743 bp\n",
      "  Read 1:    20,097,231 bp\n",
      "  Read 2:    20,084,512 bp\n",
      "Total written (filtered):     34,913,588 bp (86.9%)\n",
      "  Read 1:    17,527,712 bp\n",
      "  Read 2:    17,385,876 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 135259 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t4\t0.0\t1\t1 3\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t9\t0.0\t1\t3 6\n",
      "18\t670\t0.0\t1\t63 607\n",
      "19\t134168\t0.0\t1\t126452 7716\n",
      "20\t393\t0.0\t1\t72 321\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 134979 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t8\t530.1\t0\t8\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t14\t0.0\t1\t7 7\n",
      "19\t1062\t0.0\t1\t170 892\n",
      "20\t133552\t0.0\t2\t130093 3459\n",
      "21\t332\t0.0\t2\t82 250\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/13-DNA-cont_S3_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/13-DNA-cont_S3_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/13-DNA-cont_S3_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/13-DNA-cont_S3_54_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1304G-TF-H-PA_S29_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1304G-TF-H-PA_S29_38_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1304G-TF-H-PA_S29_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1304G-TF-H-PA_S29_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1305C-T1-H-OA-M_S19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1305C-T1-H-OA-M_S19_39_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1305C-T1-H-OA-M_S19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1305C-T1-H-OA-M_S19_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1307-T1-D-DL-M_S17_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1307-T1-D-DL-M_S17_40_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1307-T1-D-DL-M_S17_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1307-T1-D-DL-M_S17_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1312-T1-H-DL-M_S12_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1312-T1-H-DL-M_S12_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1312-T1-H-DL-M_S12_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1312-T1-H-DL-M_S12_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1320E-T1-H-PS-M_LibI_S26_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1320E-T1-H-PS-M_LibI_S26_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1320E-T1-H-PS-M_LibI_S26_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1320E-T1-H-PS-M_LibI_S26_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1326D-TF-D-PS-M_S34_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1326D-TF-D-PS-M_S34_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1326D-TF-D-PS-M_S34_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1326D-TF-D-PS-M_S34_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1350D-T1-H-SS-M_S18_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1350D-T1-H-SS-M_S18_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1350D-T1-H-SS-M_S18_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1350D-T1-H-SS-M_S18_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1350G-TF-D-SS-M_S10_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1350G-TF-D-SS-M_S10_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1350G-TF-D-SS-M_S10_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1350G-TF-D-SS-M_S10_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1356C-T1-H-MC-M_S21_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1356C-T1-H-MC-M_S21_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1356C-T1-H-MC-M_S21_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1356C-T1-H-MC-M_S21_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1360C-TF-H-MC-M_S31_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1360C-TF-H-MC-M_S31_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1360C-TF-H-MC-M_S31_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1360C-TF-H-MC-M_S31_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1363A-TF-H-MC-M_S11_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1363A-TF-H-MC-M_S11_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1363A-TF-H-MC-M_S11_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1363A-TF-H-MC-M_S11_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364A-TF-H-OA-M_S9_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364A-TF-H-OA-M_S9_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364A-TF-H-OA-M_S9_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364A-TF-H-OA-M_S9_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364B-TF-D-OA-M_S28_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364B-TF-D-OA-M_S28_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364B-TF-D-OA-M_S28_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364B-TF-D-OA-M_S28_50_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364B-TF-D-OA-M_S28_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364B-TF-D-OA-M_S28_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364B-TF-D-OA-M_S28_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364B-TF-D-OA-M_S28_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.46 s (21 us/read; 2.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            162,997\n",
      "  Read 1 with adapter:                 162,482 (99.7%)\n",
      "  Read 2 with adapter:                 162,288 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       162,997 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,062,459 bp\n",
      "  Read 1:    24,542,190 bp\n",
      "  Read 2:    24,520,269 bp\n",
      "Total written (filtered):     42,731,001 bp (87.1%)\n",
      "  Read 1:    21,455,454 bp\n",
      "  Read 2:    21,275,547 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 162482 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.6\t0\t2\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t6\t0.0\t1\t4 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t11\t0.0\t1\t3 8\n",
      "18\t749\t0.0\t1\t100 649\n",
      "19\t161260\t0.0\t1\t151987 9273\n",
      "20\t444\t0.0\t1\t76 368\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 162288 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t21\t636.7\t0\t21\n",
      "13\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t1112\t0.0\t1\t175 937\n",
      "20\t160729\t0.0\t2\t156762 3967\n",
      "21\t404\t0.0\t2\t87 317\n",
      "22\t4\t0.0\t2\t2 2\n",
      "31\t1\t0.0\t2\t1\n",
      "33\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364E-T1-H-OA-M_S20_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364E-T1-H-OA-M_S20_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364E-T1-H-OA-M_S20_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364E-T1-H-OA-M_S20_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.93 s (18 us/read; 3.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            163,788\n",
      "  Read 1 with adapter:                 163,327 (99.7%)\n",
      "  Read 2 with adapter:                 162,969 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       163,788 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,332,925 bp\n",
      "  Read 1:    24,178,737 bp\n",
      "  Read 2:    24,154,188 bp\n",
      "Total written (filtered):     41,972,046 bp (86.8%)\n",
      "  Read 1:    21,076,086 bp\n",
      "  Read 2:    20,895,960 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 163327 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t4\t0.6\t0\t4\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t7\t0.0\t1\t4 3\n",
      "13\t5\t0.0\t1\t3 2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t14\t0.0\t1\t7 7\n",
      "18\t793\t0.0\t1\t82 711\n",
      "19\t162084\t0.0\t1\t152636 9448\n",
      "20\t412\t0.0\t1\t72 340\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 162969 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2559.2\t0\t1\n",
      "4\t20\t639.8\t0\t20\n",
      "10\t1\t0.2\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t11\t0.0\t1\t7 4\n",
      "19\t1140\t0.0\t1\t181 959\n",
      "20\t161401\t0.0\t2\t157129 4272\n",
      "21\t384\t0.0\t2\t88 296\n",
      "22\t1\t0.0\t2\t0 1\n",
      "23\t1\t0.0\t2\t1\n",
      "27\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1368E-TF-D-MC-M_S38_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1368E-TF-D-MC-M_S38_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1368E-TF-D-MC-M_S38_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1368E-TF-D-MC-M_S38_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.31 s (21 us/read; 2.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            159,997\n",
      "  Read 1 with adapter:                 159,238 (99.5%)\n",
      "  Read 2 with adapter:                 159,030 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       159,997 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,295,741 bp\n",
      "  Read 1:    24,154,159 bp\n",
      "  Read 2:    24,141,582 bp\n",
      "Total written (filtered):     42,091,817 bp (87.2%)\n",
      "  Read 1:    21,129,430 bp\n",
      "  Read 2:    20,962,387 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 159238 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t2\t0.2\t1\t2\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t4\t0.0\t1\t0 4\n",
      "16\t6\t0.0\t1\t2 4\n",
      "17\t7\t0.0\t1\t2 5\n",
      "18\t1089\t0.0\t1\t108 981\n",
      "19\t157676\t0.0\t1\t146421 11255\n",
      "20\t440\t0.0\t1\t72 368\n",
      "31\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 159030 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2500.0\t0\t1\n",
      "4\t23\t625.0\t0\t23\n",
      "9\t1\t0.6\t0\t1\n",
      "13\t6\t0.0\t1\t2 4\n",
      "14\t2\t0.0\t1\t1 1\n",
      "17\t5\t0.0\t1\t1 4\n",
      "18\t17\t0.0\t1\t11 6\n",
      "19\t1348\t0.0\t1\t190 1158\n",
      "20\t157194\t0.0\t2\t152032 5162\n",
      "21\t431\t0.0\t2\t74 357\n",
      "22\t1\t0.0\t2\t0 1\n",
      "29\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1379E-TF-H-MC-M_S24_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1379E-TF-H-MC-M_S24_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1379E-TF-H-MC-M_S24_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1379E-TF-H-MC-M_S24_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.94 s (20 us/read; 2.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            195,539\n",
      "  Read 1 with adapter:                 194,855 (99.7%)\n",
      "  Read 2 with adapter:                 194,500 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       195,539 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,031,293 bp\n",
      "  Read 1:    29,030,924 bp\n",
      "  Read 2:    29,000,369 bp\n",
      "Total written (filtered):     50,441,160 bp (86.9%)\n",
      "  Read 1:    25,329,279 bp\n",
      "  Read 2:    25,111,881 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 194855 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.7\t0\t1\n",
      "10\t2\t0.2\t1\t1 1\n",
      "12\t9\t0.0\t1\t6 3\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t3\t0.0\t1\t3\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t6\t0.0\t1\t4 2\n",
      "17\t16\t0.0\t1\t10 6\n",
      "18\t1086\t0.0\t1\t120 966\n",
      "19\t193057\t0.0\t1\t181373 11684\n",
      "20\t670\t0.0\t1\t90 580\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 194500 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t20\t763.8\t0\t20\n",
      "9\t2\t0.7\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t4\t0.0\t1\t1 3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t20\t0.0\t1\t13 7\n",
      "19\t1581\t0.0\t1\t249 1332\n",
      "20\t192353\t0.0\t2\t186498 5855\n",
      "21\t510\t0.0\t2\t100 410\n",
      "22\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/14-DNA-cont_S4_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/14-DNA-cont_S4_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/14-DNA-cont_S4_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/14-DNA-cont_S4_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.99 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,680\n",
      "  Read 1 with adapter:                  29,575 (99.6%)\n",
      "  Read 2 with adapter:                  29,498 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,680 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,860,729 bp\n",
      "  Read 1:     4,431,760 bp\n",
      "  Read 2:     4,428,969 bp\n",
      "Total written (filtered):      7,709,179 bp (87.0%)\n",
      "  Read 1:     3,869,960 bp\n",
      "  Read 2:     3,839,219 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29575 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t155\t0.0\t1\t19 136\n",
      "19\t29366\t0.0\t1\t27549 1817\n",
      "20\t50\t0.0\t1\t9 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 29498 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t115.9\t0\t3\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t2\t0.0\t1\t1 1\n",
      "19\t217\t0.0\t1\t29 188\n",
      "20\t29209\t0.0\t2\t28359 850\n",
      "21\t65\t0.0\t2\t11 54\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1610E-TF-H-OA-M_S23_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1610E-TF-H-OA-M_S23_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1610E-TF-H-OA-M_S23_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1610E-TF-H-OA-M_S23_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.76 s (19 us/read; 3.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            148,273\n",
      "  Read 1 with adapter:                 147,838 (99.7%)\n",
      "  Read 2 with adapter:                 147,579 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       148,273 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,203,572 bp\n",
      "  Read 1:    22,119,119 bp\n",
      "  Read 2:    22,084,453 bp\n",
      "Total written (filtered):     38,444,484 bp (87.0%)\n",
      "  Read 1:    19,310,569 bp\n",
      "  Read 2:    19,133,915 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 147838 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t4 1\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t692\t0.0\t1\t67 625\n",
      "19\t146741\t0.0\t1\t138266 8475\n",
      "20\t386\t0.0\t1\t70 316\n",
      "25\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 147579 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t17\t579.2\t0\t17\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t9\t0.0\t1\t2 7\n",
      "19\t1054\t0.0\t1\t170 884\n",
      "20\t146153\t0.0\t2\t142301 3852\n",
      "21\t338\t0.0\t2\t72 266\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1655A-TF-D-CN-M_S7_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1655A-TF-D-CN-M_S7_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1655A-TF-D-CN-M_S7_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1655A-TF-D-CN-M_S7_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.91 s (22 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            135,228\n",
      "  Read 1 with adapter:                 134,624 (99.6%)\n",
      "  Read 2 with adapter:                 134,537 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       135,228 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,513,014 bp\n",
      "  Read 1:    20,261,413 bp\n",
      "  Read 2:    20,251,601 bp\n",
      "Total written (filtered):     35,266,009 bp (87.0%)\n",
      "  Read 1:    17,703,969 bp\n",
      "  Read 2:    17,562,040 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 134624 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t2.1\t0\t1\n",
      "9\t1\t0.5\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t13\t0.0\t1\t9 4\n",
      "18\t682\t0.0\t1\t65 617\n",
      "19\t133529\t0.0\t1\t124873 8656\n",
      "20\t380\t0.0\t1\t84 296\n",
      "21\t1\t0.0\t1\t1\n",
      "43\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 134537 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2112.9\t0\t1\n",
      "4\t23\t528.2\t0\t23\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t16\t0.0\t1\t12 4\n",
      "19\t1084\t0.0\t1\t193 891\n",
      "20\t133065\t0.0\t2\t129352 3713\n",
      "21\t340\t0.0\t2\t69 271\n",
      "24\t1\t0.0\t2\t1\n",
      "32\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1659D-TF-H-SS-M_S8_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1659D-TF-H-SS-M_S8_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1659D-TF-H-SS-M_S8_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1659D-TF-H-SS-M_S8_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.14 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            141,342\n",
      "  Read 1 with adapter:                 140,738 (99.6%)\n",
      "  Read 2 with adapter:                 140,430 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       141,342 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,427,283 bp\n",
      "  Read 1:    21,223,515 bp\n",
      "  Read 2:    21,203,768 bp\n",
      "Total written (filtered):     36,946,471 bp (87.1%)\n",
      "  Read 1:    18,550,008 bp\n",
      "  Read 2:    18,396,463 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 140738 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t7\t0.0\t1\t6 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t0 3\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t8\t0.0\t1\t2 6\n",
      "18\t827\t0.0\t1\t89 738\n",
      "19\t139448\t0.0\t1\t130822 8626\n",
      "20\t436\t0.0\t1\t78 358\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 140430 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t19\t552.1\t0\t19\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t14\t0.0\t1\t5 9\n",
      "19\t1245\t0.0\t1\t195 1050\n",
      "20\t138802\t0.0\t2\t134942 3860\n",
      "21\t335\t0.0\t2\t71 264\n",
      "31\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1662G-T1-H-MC-M_S37_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1662G-T1-H-MC-M_S37_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1662G-T1-H-MC-M_S37_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1662G-T1-H-MC-M_S37_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.75 s (22 us/read; 2.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            168,120\n",
      "  Read 1 with adapter:                 167,467 (99.6%)\n",
      "  Read 2 with adapter:                 167,247 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       168,120 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,724,176 bp\n",
      "  Read 1:    25,371,256 bp\n",
      "  Read 2:    25,352,920 bp\n",
      "Total written (filtered):     44,199,375 bp (87.1%)\n",
      "  Read 1:    22,189,915 bp\n",
      "  Read 2:    22,009,460 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 167467 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t2\t0.2\t1\t2\n",
      "12\t4\t0.0\t1\t2 2\n",
      "13\t7\t0.0\t1\t4 3\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t6\t0.0\t1\t4 2\n",
      "17\t15\t0.0\t1\t8 7\n",
      "18\t804\t0.0\t1\t94 710\n",
      "19\t166187\t0.0\t1\t154874 11313\n",
      "20\t435\t0.0\t1\t85 350\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 167247 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t2626.9\t0\t3\n",
      "4\t28\t656.7\t0\t28\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t16\t0.0\t1\t8 8\n",
      "19\t1317\t0.0\t1\t198 1119\n",
      "20\t165472\t0.0\t2\t160214 5258\n",
      "21\t402\t0.0\t2\t78 324\n",
      "22\t2\t0.0\t2\t0 2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1669A-TF-D-MC-M_S22_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1669A-TF-D-MC-M_S22_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1669A-TF-D-MC-M_S22_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1669A-TF-D-MC-M_S22_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.24 s (20 us/read; 3.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            162,364\n",
      "  Read 1 with adapter:                 161,764 (99.6%)\n",
      "  Read 2 with adapter:                 161,534 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       162,364 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,675,029 bp\n",
      "  Read 1:    24,343,516 bp\n",
      "  Read 2:    24,331,513 bp\n",
      "Total written (filtered):     42,372,679 bp (87.1%)\n",
      "  Read 1:    21,270,497 bp\n",
      "  Read 2:    21,102,182 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 161764 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "12\t7\t0.0\t1\t6 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t9\t0.0\t1\t5 4\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t822\t0.0\t1\t98 724\n",
      "19\t160470\t0.0\t1\t150627 9843\n",
      "20\t445\t0.0\t1\t91 354\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 161534 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2536.9\t0\t2\n",
      "4\t18\t634.2\t0\t18\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t18\t0.0\t1\t5 13\n",
      "19\t1331\t0.0\t1\t206 1125\n",
      "20\t159797\t0.0\t2\t155354 4443\n",
      "21\t359\t0.0\t2\t87 272\n",
      "35\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1669I-T1-H-MC-M_S30_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1669I-T1-H-MC-M_S30_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1669I-T1-H-MC-M_S30_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1669I-T1-H-MC-M_S30_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.44 s (21 us/read; 2.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            166,742\n",
      "  Read 1 with adapter:                 166,038 (99.6%)\n",
      "  Read 2 with adapter:                 165,870 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       166,742 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,257,002 bp\n",
      "  Read 1:    25,136,341 bp\n",
      "  Read 2:    25,120,661 bp\n",
      "Total written (filtered):     43,786,724 bp (87.1%)\n",
      "  Read 1:    21,982,209 bp\n",
      "  Read 2:    21,804,515 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 166038 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t2\t0.2\t1\t1 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t6\t0.0\t1\t5 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t7\t0.0\t1\t2 5\n",
      "17\t16\t0.0\t1\t8 8\n",
      "18\t878\t0.0\t1\t92 786\n",
      "19\t164693\t0.0\t1\t153924 10769\n",
      "20\t427\t0.0\t1\t71 356\n",
      "23\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 165870 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2605.3\t0\t1\n",
      "4\t15\t651.3\t0\t15\n",
      "10\t1\t0.2\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t19\t0.0\t1\t13 6\n",
      "19\t1317\t0.0\t1\t217 1100\n",
      "20\t164093\t0.0\t2\t159676 4417\n",
      "21\t413\t0.0\t2\t84 329\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1672B-T1-H-PS-M_S33_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1672B-T1-H-PS-M_S33_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1672B-T1-H-PS-M_S33_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1672B-T1-H-PS-M_S33_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.41 s (21 us/read; 2.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            159,739\n",
      "  Read 1 with adapter:                 159,232 (99.7%)\n",
      "  Read 2 with adapter:                 158,949 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       159,739 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,198,860 bp\n",
      "  Read 1:    24,107,922 bp\n",
      "  Read 2:    24,090,938 bp\n",
      "Total written (filtered):     41,996,049 bp (87.1%)\n",
      "  Read 1:    21,083,026 bp\n",
      "  Read 2:    20,913,023 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 159232 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.6\t0\t3\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t8\t0.0\t1\t4 4\n",
      "13\t6\t0.0\t1\t5 1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t6\t0.0\t1\t2 4\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t692\t0.0\t1\t76 616\n",
      "19\t158134\t0.0\t1\t149059 9075\n",
      "20\t368\t0.0\t1\t71 297\n",
      "21\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 158949 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2495.9\t0\t2\n",
      "4\t12\t624.0\t0\t12\n",
      "11\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t6\t0.0\t1\t3 3\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t15\t0.0\t1\t6 9\n",
      "19\t1127\t0.0\t1\t198 929\n",
      "20\t157395\t0.0\t2\t153410 3985\n",
      "21\t384\t0.0\t2\t88 296\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1681E-T1-H-PA-M_S32_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1681E-T1-H-PA-M_S32_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1681E-T1-H-PA-M_S32_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1681E-T1-H-PA-M_S32_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.08 s (22 us/read; 2.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,620\n",
      "  Read 1 with adapter:                 142,169 (99.7%)\n",
      "  Read 2 with adapter:                 141,830 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       142,620 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,043,971 bp\n",
      "  Read 1:    21,531,405 bp\n",
      "  Read 2:    21,512,566 bp\n",
      "Total written (filtered):     37,507,830 bp (87.1%)\n",
      "  Read 1:    18,830,588 bp\n",
      "  Read 2:    18,677,242 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 142169 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t705\t0.0\t1\t76 629\n",
      "19\t141067\t0.0\t1\t132496 8571\n",
      "20\t378\t0.0\t1\t76 302\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 141830 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t16\t557.1\t0\t16\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t13\t0.0\t1\t8 5\n",
      "19\t1279\t0.0\t1\t160 1119\n",
      "20\t140153\t0.0\t2\t136319 3834\n",
      "21\t354\t0.0\t2\t65 289\n",
      "24\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1692C-T1-H-CN-M_S6_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1692C-T1-H-CN-M_S6_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1692C-T1-H-CN-M_S6_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1692C-T1-H-CN-M_S6_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.61 s (20 us/read; 2.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,114\n",
      "  Read 1 with adapter:                 127,996 (99.1%)\n",
      "  Read 2 with adapter:                 128,368 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       129,114 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,912,749 bp\n",
      "  Read 1:    18,943,422 bp\n",
      "  Read 2:    18,969,327 bp\n",
      "Total written (filtered):     32,914,875 bp (86.8%)\n",
      "  Read 1:    16,511,920 bp\n",
      "  Read 2:    16,402,955 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 127996 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t9\t0.0\t1\t1 8\n",
      "18\t669\t0.0\t1\t88 581\n",
      "19\t126943\t0.0\t1\t116355 10588\n",
      "20\t359\t0.0\t1\t71 288\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 128368 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2017.4\t0\t1\n",
      "4\t12\t504.4\t0\t12\n",
      "11\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t13\t0.0\t1\t7 6\n",
      "19\t999\t0.0\t1\t165 834\n",
      "20\t127033\t0.0\t2\t123928 3105\n",
      "21\t298\t0.0\t2\t71 227\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1694-T1-D-DL-M_S13_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1694-T1-D-DL-M_S13_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1694-T1-D-DL-M_S13_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1694-T1-D-DL-M_S13_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.10 s (22 us/read; 2.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            141,128\n",
      "  Read 1 with adapter:                 140,570 (99.6%)\n",
      "  Read 2 with adapter:                 140,452 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       141,128 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,165,214 bp\n",
      "  Read 1:    21,092,184 bp\n",
      "  Read 2:    21,073,030 bp\n",
      "Total written (filtered):     36,686,765 bp (87.0%)\n",
      "  Read 1:    18,421,768 bp\n",
      "  Read 2:    18,264,997 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 140570 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t551.3\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t10\t0.0\t1\t5 5\n",
      "18\t659\t0.0\t1\t78 581\n",
      "19\t139552\t0.0\t1\t130690 8862\n",
      "20\t335\t0.0\t1\t66 269\n",
      "23\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 140452 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2205.1\t0\t1\n",
      "4\t18\t551.3\t0\t18\n",
      "9\t1\t0.5\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t6\t0.0\t1\t3 3\n",
      "19\t1000\t0.0\t1\t161 839\n",
      "20\t139051\t0.0\t2\t135560 3491\n",
      "21\t361\t0.0\t2\t77 284\n",
      "22\t1\t0.0\t2\t1\n",
      "33\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364E-T1-H-OA-M_S20_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1364E-T1-H-OA-M_S20_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364E-T1-H-OA-M_S20_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1364E-T1-H-OA-M_S20_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1368E-TF-D-MC-M_S38_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1368E-TF-D-MC-M_S38_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1368E-TF-D-MC-M_S38_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1368E-TF-D-MC-M_S38_52_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1379E-TF-H-MC-M_S24_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1379E-TF-H-MC-M_S24_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1379E-TF-H-MC-M_S24_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1379E-TF-H-MC-M_S24_53_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/14-DNA-cont_S4_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/14-DNA-cont_S4_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/14-DNA-cont_S4_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/14-DNA-cont_S4_55_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1610E-TF-H-OA-M_S23_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1610E-TF-H-OA-M_S23_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1610E-TF-H-OA-M_S23_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1610E-TF-H-OA-M_S23_56_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1655A-TF-D-CN-M_S7_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1655A-TF-D-CN-M_S7_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1655A-TF-D-CN-M_S7_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1655A-TF-D-CN-M_S7_57_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1659D-TF-H-SS-M_S8_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1659D-TF-H-SS-M_S8_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1659D-TF-H-SS-M_S8_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1659D-TF-H-SS-M_S8_58_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1662G-T1-H-MC-M_S37_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1662G-T1-H-MC-M_S37_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1662G-T1-H-MC-M_S37_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1662G-T1-H-MC-M_S37_59_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1669A-TF-D-MC-M_S22_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1669A-TF-D-MC-M_S22_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1669A-TF-D-MC-M_S22_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1669A-TF-D-MC-M_S22_60_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1669I-T1-H-MC-M_S30_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1669I-T1-H-MC-M_S30_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1669I-T1-H-MC-M_S30_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1669I-T1-H-MC-M_S30_61_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1672B-T1-H-PS-M_S33_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1672B-T1-H-PS-M_S33_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1672B-T1-H-PS-M_S33_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1672B-T1-H-PS-M_S33_62_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1681E-T1-H-PA-M_S32_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1681E-T1-H-PA-M_S32_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1681E-T1-H-PA-M_S32_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1681E-T1-H-PA-M_S32_63_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1692C-T1-H-CN-M_S6_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1692C-T1-H-CN-M_S6_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1692C-T1-H-CN-M_S6_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1692C-T1-H-CN-M_S6_64_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1694-T1-D-DL-M_S13_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1694-T1-D-DL-M_S13_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1694-T1-D-DL-M_S13_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1694-T1-D-DL-M_S13_65_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1695-TA-H-DL-M_S35_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1695-TA-H-DL-M_S35_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1695-TA-H-DL-M_S35_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1695-TA-H-DL-M_S35_66_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1695-TA-H-DL-M_S35_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1695-TA-H-DL-M_S35_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1695-TA-H-DL-M_S35_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1695-TA-H-DL-M_S35_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.49 s (21 us/read; 2.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            166,137\n",
      "  Read 1 with adapter:                 165,489 (99.6%)\n",
      "  Read 2 with adapter:                 165,314 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       166,137 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,127,962 bp\n",
      "  Read 1:    25,083,618 bp\n",
      "  Read 2:    25,044,344 bp\n",
      "Total written (filtered):     43,678,879 bp (87.1%)\n",
      "  Read 1:    21,939,747 bp\n",
      "  Read 2:    21,739,132 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 165489 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t649.0\t0\t1\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t1\t0.2\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t3\t0.0\t1\t0 3\n",
      "16\t7\t0.0\t1\t3 4\n",
      "17\t10\t0.0\t1\t7 3\n",
      "18\t673\t0.0\t1\t70 603\n",
      "19\t164393\t0.0\t1\t153711 10682\n",
      "20\t388\t0.0\t1\t71 317\n",
      "41\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 165314 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t16\t649.0\t0\t16\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t16\t0.0\t1\t9 7\n",
      "19\t1147\t0.0\t1\t183 964\n",
      "20\t163745\t0.0\t2\t159182 4563\n",
      "21\t386\t0.0\t2\t87 299\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1833A-TF-H-CN-M_S5_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1833A-TF-H-CN-M_S5_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1833A-TF-H-CN-M_S5_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1833A-TF-H-CN-M_S5_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.56 s (18 us/read; 3.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            140,824\n",
      "  Read 1 with adapter:                 140,298 (99.6%)\n",
      "  Read 2 with adapter:                 140,098 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       140,824 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,499,690 bp\n",
      "  Read 1:    21,252,095 bp\n",
      "  Read 2:    21,247,595 bp\n",
      "Total written (filtered):     37,033,449 bp (87.1%)\n",
      "  Read 1:    18,586,832 bp\n",
      "  Read 2:    18,446,617 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 140298 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t5\t0.5\t0\t5\n",
      "12\t3\t0.0\t1\t2 1\n",
      "14\t6\t0.0\t1\t5 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t6\t0.0\t1\t1 5\n",
      "17\t9\t0.0\t1\t7 2\n",
      "18\t642\t0.0\t1\t82 560\n",
      "19\t139249\t0.0\t1\t130172 9077\n",
      "20\t376\t0.0\t1\t60 316\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 140098 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2200.4\t0\t1\n",
      "4\t13\t550.1\t0\t13\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t5 1\n",
      "18\t13\t0.0\t1\t8 5\n",
      "19\t1045\t0.0\t1\t167 878\n",
      "20\t138659\t0.0\t2\t134897 3762\n",
      "21\t354\t0.0\t2\t83 271\n",
      "22\t1\t0.0\t2\t1\n",
      "26\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1833E-T1-H-CN-M_S15_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1833E-T1-H-CN-M_S15_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1833E-T1-H-CN-M_S15_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1833E-T1-H-CN-M_S15_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.50 s (19 us/read; 3.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            128,484\n",
      "  Read 1 with adapter:                 127,892 (99.5%)\n",
      "  Read 2 with adapter:                 127,913 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       128,484 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,640,729 bp\n",
      "  Read 1:    19,324,857 bp\n",
      "  Read 2:    19,315,872 bp\n",
      "Total written (filtered):     33,653,824 bp (87.1%)\n",
      "  Read 1:    16,895,326 bp\n",
      "  Read 2:    16,758,498 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 127892 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t501.9\t0\t1\n",
      "9\t1\t0.5\t0\t1\n",
      "11\t4\t0.0\t1\t2 2\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t4\t0.0\t1\t0 4\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t594\t0.0\t1\t55 539\n",
      "19\t126977\t0.0\t1\t118955 8022\n",
      "20\t300\t0.0\t1\t46 254\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 127913 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t7\t501.9\t0\t7\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t7\t0.0\t1\t2 5\n",
      "18\t15\t0.0\t1\t9 6\n",
      "19\t970\t0.0\t1\t149 821\n",
      "20\t126605\t0.0\t2\t123516 3089\n",
      "21\t301\t0.0\t2\t68 233\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835A-TF-D-OA-M_S27_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835A-TF-D-OA-M_S27_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835A-TF-D-OA-M_S27_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835A-TF-D-OA-M_S27_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.89 s (22 us/read; 2.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            134,186\n",
      "  Read 1 with adapter:                 133,088 (99.2%)\n",
      "  Read 2 with adapter:                 133,378 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       134,186 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,503,787 bp\n",
      "  Read 1:    20,257,932 bp\n",
      "  Read 2:    20,245,855 bp\n",
      "Total written (filtered):     35,309,312 bp (87.2%)\n",
      "  Read 1:    17,729,705 bp\n",
      "  Read 2:    17,579,607 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 133088 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t2.0\t0\t2\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t5\t0.0\t1\t2 3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t8\t0.0\t1\t5 3\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t661\t0.0\t1\t76 585\n",
      "19\t132025\t0.0\t1\t121555 10470\n",
      "20\t368\t0.0\t1\t75 293\n",
      "30\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 133378 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2096.7\t0\t1\n",
      "4\t28\t524.2\t0\t28\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t3\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t15\t0.0\t1\t8 7\n",
      "19\t1115\t0.0\t1\t177 938\n",
      "20\t131875\t0.0\t2\t128662 3213\n",
      "21\t335\t0.0\t2\t73 262\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835B-TF-H-OA-M_S1_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835B-TF-H-OA-M_S1_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835B-TF-H-OA-M_S1_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835B-TF-H-OA-M_S1_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.28 s (21 us/read; 2.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,994\n",
      "  Read 1 with adapter:                 109,629 (99.7%)\n",
      "  Read 2 with adapter:                 109,413 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,994 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,194,519 bp\n",
      "  Read 1:    16,600,058 bp\n",
      "  Read 2:    16,594,461 bp\n",
      "Total written (filtered):     28,924,520 bp (87.1%)\n",
      "  Read 1:    14,517,442 bp\n",
      "  Read 2:    14,407,078 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109629 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t557\t0.0\t1\t75 482\n",
      "19\t108763\t0.0\t1\t102454 6309\n",
      "20\t295\t0.0\t1\t60 235\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 109413 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t11\t429.7\t0\t11\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t915\t0.0\t1\t143 772\n",
      "20\t108214\t0.0\t2\t105517 2697\n",
      "21\t258\t0.0\t2\t64 194\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835D-T1-H-OA-M_S36_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835D-T1-H-OA-M_S36_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835D-T1-H-OA-M_S36_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835D-T1-H-OA-M_S36_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.65 s (22 us/read; 2.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            166,063\n",
      "  Read 1 with adapter:                 165,558 (99.7%)\n",
      "  Read 2 with adapter:                 165,346 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       166,063 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,090,302 bp\n",
      "  Read 1:    25,066,454 bp\n",
      "  Read 2:    25,023,848 bp\n",
      "Total written (filtered):     43,639,253 bp (87.1%)\n",
      "  Read 1:    21,921,203 bp\n",
      "  Read 2:    21,718,050 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 165558 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t9\t0.0\t1\t4 5\n",
      "18\t704\t0.0\t1\t107 597\n",
      "19\t164414\t0.0\t1\t154377 10037\n",
      "20\t421\t0.0\t1\t81 340\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 165346 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2594.7\t0\t1\n",
      "4\t18\t648.7\t0\t18\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t11\t0.0\t1\t5 6\n",
      "19\t1098\t0.0\t1\t210 888\n",
      "20\t163858\t0.0\t2\t159608 4250\n",
      "21\t349\t0.0\t2\t83 266\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1839-T1-D-DL-M_S14_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1839-T1-D-DL-M_S14_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1839-T1-D-DL-M_S14_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1839-T1-D-DL-M_S14_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.71 s (22 us/read; 2.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,705\n",
      "  Read 1 with adapter:                 120,117 (99.5%)\n",
      "  Read 2 with adapter:                 120,103 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       120,705 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,225,015 bp\n",
      "  Read 1:    18,129,689 bp\n",
      "  Read 2:    18,095,326 bp\n",
      "Total written (filtered):     31,541,874 bp (87.1%)\n",
      "  Read 1:    15,847,787 bp\n",
      "  Read 2:    15,694,087 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 120117 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t537\t0.0\t1\t58 479\n",
      "19\t119282\t0.0\t1\t111400 7882\n",
      "20\t280\t0.0\t1\t74 206\n",
      "31\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 120103 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1886.0\t0\t1\n",
      "4\t15\t471.5\t0\t15\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t7\t0.0\t1\t5 2\n",
      "19\t778\t0.0\t1\t124 654\n",
      "20\t119046\t0.0\t2\t116286 2760\n",
      "21\t251\t0.0\t2\t46 205\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1848B-TD-H-CN-M_S16_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1848B-TD-H-CN-M_S16_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1848B-TD-H-CN-M_S16_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1848B-TD-H-CN-M_S16_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.93 s (20 us/read; 2.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            143,625\n",
      "  Read 1 with adapter:                 143,173 (99.7%)\n",
      "  Read 2 with adapter:                 142,916 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       143,625 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,100,349 bp\n",
      "  Read 1:    21,547,392 bp\n",
      "  Read 2:    21,552,957 bp\n",
      "Total written (filtered):     37,523,179 bp (87.1%)\n",
      "  Read 1:    18,827,511 bp\n",
      "  Read 2:    18,695,668 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 143173 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t561.0\t0\t1\n",
      "9\t1\t0.5\t0\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t5\t0.0\t1\t3 2\n",
      "16\t5\t0.0\t1\t5\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t649\t0.0\t1\t89 560\n",
      "19\t142146\t0.0\t1\t134302 7844\n",
      "20\t355\t0.0\t1\t71 284\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 142916 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t20\t561.0\t0\t20\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t9\t0.0\t1\t7 2\n",
      "19\t971\t0.0\t1\t159 812\n",
      "20\t141611\t0.0\t2\t138236 3375\n",
      "21\t299\t0.0\t2\t65 234\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1870C-T1-H-PS-M_S25_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1870C-T1-H-PS-M_S25_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1870C-T1-H-PS-M_S25_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1870C-T1-H-PS-M_S25_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.95 s (20 us/read; 2.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            146,760\n",
      "  Read 1 with adapter:                 146,258 (99.7%)\n",
      "  Read 2 with adapter:                 145,939 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       146,760 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,553,374 bp\n",
      "  Read 1:    21,776,997 bp\n",
      "  Read 2:    21,776,377 bp\n",
      "Total written (filtered):     37,857,274 bp (86.9%)\n",
      "  Read 1:    18,998,692 bp\n",
      "  Read 2:    18,858,582 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 146258 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "12\t12\t0.0\t1\t12\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t3\t0.0\t1\t3\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t7\t0.0\t1\t5 2\n",
      "17\t9\t0.0\t1\t5 4\n",
      "18\t821\t0.0\t1\t78 743\n",
      "19\t145017\t0.0\t1\t136846 8171\n",
      "20\t384\t0.0\t1\t81 303\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 145939 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t8\t573.3\t0\t8\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t15\t0.0\t1\t10 5\n",
      "19\t1135\t0.0\t1\t205 930\n",
      "20\t144406\t0.0\t2\t140693 3713\n",
      "21\t363\t0.0\t2\t95 268\n",
      "36\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/mock-1_LibI_S2_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/mock-1_LibI_S2_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/mock-1_LibI_S2_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/mock-1_LibI_S2_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.47 s (21 us/read; 2.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            118,482\n",
      "  Read 1 with adapter:                 117,981 (99.6%)\n",
      "  Read 2 with adapter:                 117,768 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       118,482 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,745,388 bp\n",
      "  Read 1:    17,889,816 bp\n",
      "  Read 2:    17,855,572 bp\n",
      "Total written (filtered):     31,150,156 bp (87.1%)\n",
      "  Read 1:    15,648,711 bp\n",
      "  Read 2:    15,501,445 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 117981 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t9\t0.0\t1\t3 6\n",
      "18\t738\t0.0\t1\t77 661\n",
      "19\t116936\t0.0\t1\t108854 8082\n",
      "20\t288\t0.0\t1\t49 239\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 117768 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1851.3\t0\t1\n",
      "4\t14\t462.8\t0\t14\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t5\t0.0\t1\t2 3\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t12\t0.0\t1\t8 4\n",
      "19\t1137\t0.0\t1\t155 982\n",
      "20\t116305\t0.0\t2\t112697 3608\n",
      "21\t279\t0.0\t2\t64 215\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1833A-TF-H-CN-M_S5_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1833A-TF-H-CN-M_S5_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1833A-TF-H-CN-M_S5_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1833A-TF-H-CN-M_S5_67_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1833E-T1-H-CN-M_S15_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1833E-T1-H-CN-M_S15_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1833E-T1-H-CN-M_S15_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1833E-T1-H-CN-M_S15_68_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835A-TF-D-OA-M_S27_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835A-TF-D-OA-M_S27_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835A-TF-D-OA-M_S27_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835A-TF-D-OA-M_S27_69_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835B-TF-H-OA-M_S1_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835B-TF-H-OA-M_S1_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835B-TF-H-OA-M_S1_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835B-TF-H-OA-M_S1_70_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835D-T1-H-OA-M_S36_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1835D-T1-H-OA-M_S36_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835D-T1-H-OA-M_S36_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1835D-T1-H-OA-M_S36_71_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1839-T1-D-DL-M_S14_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1839-T1-D-DL-M_S14_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1839-T1-D-DL-M_S14_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1839-T1-D-DL-M_S14_72_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1848B-TD-H-CN-M_S16_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1848B-TD-H-CN-M_S16_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1848B-TD-H-CN-M_S16_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1848B-TD-H-CN-M_S16_73_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1870C-T1-H-PS-M_S25_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/1870C-T1-H-PS-M_S25_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1870C-T1-H-PS-M_S25_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/1870C-T1-H-PS-M_S25_74_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/mock-1_LibI_S2_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-_waaj4j8/mock-1_LibI_S2_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/mock-1_LibI_S2_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-7e27_a0z/e7062277-eb8f-4ad8-8a7c-e979d22c6ab7/data/mock-1_LibI_S2_75_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_LibI.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1226B-TF-H-MC-M_S3_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1226B-TF-H-MC-M_S3_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1226B-TF-H-MC-M_S3_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1226B-TF-H-MC-M_S3_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.63 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            118,565\n",
      "  Read 1 with adapter:                 117,698 (99.3%)\n",
      "  Read 2 with adapter:                 118,109 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       118,565 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,697,259 bp\n",
      "  Read 1:    17,858,521 bp\n",
      "  Read 2:    17,838,738 bp\n",
      "Total written (filtered):     31,099,877 bp (87.1%)\n",
      "  Read 1:    15,622,564 bp\n",
      "  Read 2:    15,477,313 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 117698 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t7\t0.0\t1\t6 1\n",
      "13\t4\t0.0\t1\t4\n",
      "16\t7\t0.0\t1\t1 6\n",
      "17\t9\t0.0\t1\t6 3\n",
      "18\t526\t0.0\t1\t73 453\n",
      "19\t116832\t0.0\t1\t105706 11126\n",
      "20\t311\t0.0\t1\t61 250\n",
      "24\t1\t0.0\t1\t1\n",
      "36\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 118109 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t463.1\t0\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t4\t0.0\t1\t1 3\n",
      "19\t991\t0.0\t1\t167 824\n",
      "20\t116827\t0.0\t2\t114859 1968\n",
      "21\t276\t0.0\t2\t68 208\n",
      "22\t1\t0.0\t2\t0 1\n",
      "41\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1295D-TF-D-PA-M_S34_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1295D-TF-D-PA-M_S34_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1295D-TF-D-PA-M_S34_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1295D-TF-D-PA-M_S34_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.88 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,108\n",
      "  Read 1 with adapter:                 129,229 (99.3%)\n",
      "  Read 2 with adapter:                 129,485 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       130,108 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,277,997 bp\n",
      "  Read 1:    19,645,611 bp\n",
      "  Read 2:    19,632,386 bp\n",
      "Total written (filtered):     34,234,390 bp (87.2%)\n",
      "  Read 1:    17,190,826 bp\n",
      "  Read 2:    17,043,564 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 129229 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t7\t0.0\t1\t5 2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t10\t0.0\t1\t5 5\n",
      "18\t831\t0.0\t1\t100 731\n",
      "19\t127984\t0.0\t1\t116693 11291\n",
      "20\t385\t0.0\t1\t77 308\n",
      "32\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 129485 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t4\t508.2\t0\t4\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t8\t0.0\t1\t4 4\n",
      "19\t1087\t0.0\t1\t173 914\n",
      "20\t128069\t0.0\t2\t125674 2395\n",
      "21\t311\t0.0\t2\t57 254\n",
      "29\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1312-TF-H-DL-M_S9_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1312-TF-H-DL-M_S9_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1312-TF-H-DL-M_S9_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1312-TF-H-DL-M_S9_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.90 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            136,742\n",
      "  Read 1 with adapter:                 135,981 (99.4%)\n",
      "  Read 2 with adapter:                 136,148 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       136,742 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,281,770 bp\n",
      "  Read 1:    20,644,387 bp\n",
      "  Read 2:    20,637,383 bp\n",
      "Total written (filtered):     35,976,447 bp (87.1%)\n",
      "  Read 1:    18,061,216 bp\n",
      "  Read 2:    17,915,231 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 135981 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t11\t0.0\t1\t6 5\n",
      "17\t5\t0.0\t1\t0 5\n",
      "18\t707\t0.0\t1\t80 627\n",
      "19\t134876\t0.0\t1\t123888 10988\n",
      "20\t370\t0.0\t1\t82 288\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 136148 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t534.1\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t12\t0.0\t1\t6 6\n",
      "19\t1007\t0.0\t1\t138 869\n",
      "20\t134816\t0.0\t2\t132489 2327\n",
      "21\t303\t0.0\t2\t67 236\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1313-T1-H-DL-M_S27_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1313-T1-H-DL-M_S27_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1313-T1-H-DL-M_S27_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1313-T1-H-DL-M_S27_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.97 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            131,552\n",
      "  Read 1 with adapter:                 130,853 (99.5%)\n",
      "  Read 2 with adapter:                 131,025 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       131,552 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,687,382 bp\n",
      "  Read 1:    19,863,534 bp\n",
      "  Read 2:    19,823,848 bp\n",
      "Total written (filtered):     34,581,720 bp (87.1%)\n",
      "  Read 1:    17,377,670 bp\n",
      "  Read 2:    17,204,050 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 130853 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t2 2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t650\t0.0\t1\t66 584\n",
      "19\t129809\t0.0\t1\t119637 10172\n",
      "20\t378\t0.0\t1\t67 311\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 131025 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t513.9\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t12\t0.0\t1\t9 3\n",
      "19\t922\t0.0\t1\t151 771\n",
      "20\t129776\t0.0\t2\t127559 2217\n",
      "21\t306\t0.0\t2\t68 238\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1319A-TF-H-SS-M_S22_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1319A-TF-H-SS-M_S22_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1319A-TF-H-SS-M_S22_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1319A-TF-H-SS-M_S22_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.50 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,719\n",
      "  Read 1 with adapter:                 100,008 (99.3%)\n",
      "  Read 2 with adapter:                 100,321 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,719 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,382,491 bp\n",
      "  Read 1:    15,197,166 bp\n",
      "  Read 2:    15,185,325 bp\n",
      "Total written (filtered):     26,476,977 bp (87.1%)\n",
      "  Read 1:    13,297,390 bp\n",
      "  Read 2:    13,179,587 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 100008 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t5\t0.0\t1\t1 4\n",
      "18\t583\t0.0\t1\t79 504\n",
      "19\t99087\t0.0\t1\t90097 8990\n",
      "20\t314\t0.0\t1\t53 261\n",
      "21\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 100321 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t393.4\t0\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t13\t0.0\t1\t7 6\n",
      "19\t816\t0.0\t1\t127 689\n",
      "20\t99233\t0.0\t2\t97708 1525\n",
      "21\t245\t0.0\t2\t48 197\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1319H-TF-D-SS-M_S16_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1319H-TF-D-SS-M_S16_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1319H-TF-D-SS-M_S16_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1319H-TF-D-SS-M_S16_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.89 s (22 us/read; 2.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            131,075\n",
      "  Read 1 with adapter:                 130,393 (99.5%)\n",
      "  Read 2 with adapter:                 130,385 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       131,075 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,578,120 bp\n",
      "  Read 1:    19,791,664 bp\n",
      "  Read 2:    19,786,456 bp\n",
      "Total written (filtered):     34,494,427 bp (87.2%)\n",
      "  Read 1:    17,314,715 bp\n",
      "  Read 2:    17,179,712 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 130393 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t4\t0.0\t1\t2 2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t5\t0.0\t1\t4 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t819\t0.0\t1\t91 728\n",
      "19\t129134\t0.0\t1\t119676 9458\n",
      "20\t414\t0.0\t1\t68 346\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 130385 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t512.0\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t17\t0.0\t1\t11 6\n",
      "19\t1199\t0.0\t1\t203 996\n",
      "20\t128809\t0.0\t2\t126597 2212\n",
      "21\t344\t0.0\t2\t71 273\n",
      "22\t1\t0.0\t2\t0 1\n",
      "23\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1320E-T1-H-PS-M_S26_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1320E-T1-H-PS-M_S26_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1320E-T1-H-PS-M_S26_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1320E-T1-H-PS-M_S26_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.03 s (21 us/read; 2.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            144,308\n",
      "  Read 1 with adapter:                 143,066 (99.1%)\n",
      "  Read 2 with adapter:                 143,762 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       144,308 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,544,637 bp\n",
      "  Read 1:    21,783,418 bp\n",
      "  Read 2:    21,761,219 bp\n",
      "Total written (filtered):     37,952,335 bp (87.2%)\n",
      "  Read 1:    19,065,562 bp\n",
      "  Read 2:    18,886,773 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 143066 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "12\t3\t0.0\t1\t1 2\n",
      "13\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t6\t0.0\t1\t4 2\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t626\t0.0\t1\t80 546\n",
      "19\t142119\t0.0\t1\t126841 15278\n",
      "20\t303\t0.0\t1\t66 237\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 143762 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t6\t563.7\t0\t6\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "15\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t14\t0.0\t1\t4 10\n",
      "19\t953\t0.0\t1\t162 791\n",
      "20\t142466\t0.0\t2\t140263 2203\n",
      "21\t310\t0.0\t2\t68 242\n",
      "22\t1\t0.0\t2\t1\n",
      "24\t1\t0.0\t2\t1\n",
      "38\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1327A-TF-D-OA-M_S37_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1327A-TF-D-OA-M_S37_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1327A-TF-D-OA-M_S37_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1327A-TF-D-OA-M_S37_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.96 s (20 us/read; 2.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            146,448\n",
      "  Read 1 with adapter:                 145,641 (99.4%)\n",
      "  Read 2 with adapter:                 145,950 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       146,448 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,205,067 bp\n",
      "  Read 1:    22,108,681 bp\n",
      "  Read 2:    22,096,386 bp\n",
      "Total written (filtered):     38,520,239 bp (87.1%)\n",
      "  Read 1:    19,341,909 bp\n",
      "  Read 2:    19,178,330 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 145641 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.6\t0\t2\n",
      "12\t8\t0.0\t1\t7 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t9\t0.0\t1\t2 7\n",
      "18\t695\t0.0\t1\t87 608\n",
      "19\t144515\t0.0\t1\t133425 11090\n",
      "20\t405\t0.0\t1\t83 322\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 145950 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2288.2\t0\t1\n",
      "4\t7\t572.1\t0\t7\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t1096\t0.0\t1\t206 890\n",
      "20\t144495\t0.0\t2\t142435 2060\n",
      "21\t335\t0.0\t2\t87 248\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1335-TF-H-DL-M_S10_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1335-TF-H-DL-M_S10_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1335-TF-H-DL-M_S10_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1335-TF-H-DL-M_S10_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.53 s (22 us/read; 2.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,457\n",
      "  Read 1 with adapter:                 113,214 (98.9%)\n",
      "  Read 2 with adapter:                 113,998 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,529,316 bp\n",
      "  Read 1:    17,254,964 bp\n",
      "  Read 2:    17,274,352 bp\n",
      "Total written (filtered):     30,099,334 bp (87.2%)\n",
      "  Read 1:    15,104,172 bp\n",
      "  Read 2:    14,995,162 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113214 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t6\t0.0\t1\t6\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t0 4\n",
      "17\t8\t0.0\t1\t6 2\n",
      "18\t512\t0.0\t1\t55 457\n",
      "19\t112371\t0.0\t1\t97814 14557\n",
      "20\t312\t0.0\t1\t50 262\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 113998 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t5\t447.1\t0\t5\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t9\t0.0\t1\t6 3\n",
      "19\t915\t0.0\t1\t146 769\n",
      "20\t112795\t0.0\t2\t110939 1856\n",
      "21\t264\t0.0\t2\t56 208\n",
      "28\t1\t0.0\t2\t1\n",
      "34\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1338I-T1-H-OA-M_S35_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1338I-T1-H-OA-M_S35_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1338I-T1-H-OA-M_S35_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1338I-T1-H-OA-M_S35_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.53 s (23 us/read; 2.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,389\n",
      "  Read 1 with adapter:                 109,884 (99.5%)\n",
      "  Read 2 with adapter:                 110,003 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       110,389 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,684,751 bp\n",
      "  Read 1:    16,353,900 bp\n",
      "  Read 2:    16,330,851 bp\n",
      "Total written (filtered):     28,397,846 bp (86.9%)\n",
      "  Read 1:    14,266,425 bp\n",
      "  Read 2:    14,131,421 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109884 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t5\t0.0\t1\t1 4\n",
      "17\t7\t0.0\t1\t5 2\n",
      "18\t530\t0.0\t1\t71 459\n",
      "19\t109026\t0.0\t1\t101431 7595\n",
      "20\t303\t0.0\t1\t65 238\n",
      "21\t2\t0.0\t1\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110003 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t5\t431.2\t0\t5\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t9\t0.0\t1\t2 7\n",
      "19\t741\t0.0\t1\t140 601\n",
      "20\t108977\t0.0\t2\t107403 1574\n",
      "21\t260\t0.0\t2\t55 205\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1339B-T1-H-SS-M_S25_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1339B-T1-H-SS-M_S25_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1339B-T1-H-SS-M_S25_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1339B-T1-H-SS-M_S25_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.41 s (23 us/read; 2.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            149,906\n",
      "  Read 1 with adapter:                 149,239 (99.6%)\n",
      "  Read 2 with adapter:                 149,240 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       149,906 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,244,101 bp\n",
      "  Read 1:    22,624,397 bp\n",
      "  Read 2:    22,619,704 bp\n",
      "Total written (filtered):     39,425,196 bp (87.1%)\n",
      "  Read 1:    19,789,313 bp\n",
      "  Read 2:    19,635,883 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 149239 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t4\t0.0\t1\t1 3\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t754\t0.0\t1\t77 677\n",
      "19\t148083\t0.0\t1\t137851 10232\n",
      "20\t382\t0.0\t1\t80 302\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 149240 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t585.6\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t0 2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t12\t0.0\t1\t9 3\n",
      "19\t1195\t0.0\t1\t189 1006\n",
      "20\t147673\t0.0\t2\t145148 2525\n",
      "21\t345\t0.0\t2\t79 266\n",
      "26\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1345C-T1-H-CN-M_S11_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1345C-T1-H-CN-M_S11_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1345C-T1-H-CN-M_S11_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1345C-T1-H-CN-M_S11_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.07 s (21 us/read; 2.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,359\n",
      "  Read 1 with adapter:                  99,742 (99.4%)\n",
      "  Read 2 with adapter:                 100,000 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,292,527 bp\n",
      "  Read 1:    15,146,516 bp\n",
      "  Read 2:    15,146,011 bp\n",
      "Total written (filtered):     26,398,333 bp (87.1%)\n",
      "  Read 1:    13,251,680 bp\n",
      "  Read 2:    13,146,653 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99742 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t8\t0.0\t1\t3 5\n",
      "18\t431\t0.0\t1\t62 369\n",
      "19\t99020\t0.0\t1\t90462 8558\n",
      "20\t266\t0.0\t1\t53 213\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 100000 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t392.0\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "18\t5\t0.0\t1\t1 4\n",
      "19\t752\t0.0\t1\t107 645\n",
      "20\t99018\t0.0\t2\t97673 1345\n",
      "21\t215\t0.0\t2\t51 164\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1350G-TF-D-SS-M_S15_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1350G-TF-D-SS-M_S15_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1350G-TF-D-SS-M_S15_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1350G-TF-D-SS-M_S15_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.79 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            123,636\n",
      "  Read 1 with adapter:                 122,952 (99.4%)\n",
      "  Read 2 with adapter:                 123,069 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       123,636 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,331,804 bp\n",
      "  Read 1:    18,668,226 bp\n",
      "  Read 2:    18,663,578 bp\n",
      "Total written (filtered):     32,535,839 bp (87.2%)\n",
      "  Read 1:    16,332,596 bp\n",
      "  Read 2:    16,203,243 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 122952 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t4\t0.0\t1\t3 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t6\t0.0\t1\t5 1\n",
      "18\t756\t0.0\t1\t82 674\n",
      "19\t121796\t0.0\t1\t111934 9862\n",
      "20\t374\t0.0\t1\t82 292\n",
      "21\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t1\n",
      "34\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 123069 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t4\t483.0\t0\t4\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t5\t0.0\t1\t5\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t18\t0.0\t1\t5 13\n",
      "19\t1130\t0.0\t1\t147 983\n",
      "20\t121624\t0.0\t2\t119048 2576\n",
      "21\t276\t0.0\t2\t57 219\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1226B-TF-H-MC-M_S3_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1226B-TF-H-MC-M_S3_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1226B-TF-H-MC-M_S3_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1226B-TF-H-MC-M_S3_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1295D-TF-D-PA-M_S34_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1295D-TF-D-PA-M_S34_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1295D-TF-D-PA-M_S34_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1295D-TF-D-PA-M_S34_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1312-TF-H-DL-M_S9_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1312-TF-H-DL-M_S9_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1312-TF-H-DL-M_S9_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1312-TF-H-DL-M_S9_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1313-T1-H-DL-M_S27_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1313-T1-H-DL-M_S27_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1313-T1-H-DL-M_S27_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1313-T1-H-DL-M_S27_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1319A-TF-H-SS-M_S22_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1319A-TF-H-SS-M_S22_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1319A-TF-H-SS-M_S22_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1319A-TF-H-SS-M_S22_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1319H-TF-D-SS-M_S16_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1319H-TF-D-SS-M_S16_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1319H-TF-D-SS-M_S16_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1319H-TF-D-SS-M_S16_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1320E-T1-H-PS-M_S26_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1320E-T1-H-PS-M_S26_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1320E-T1-H-PS-M_S26_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1320E-T1-H-PS-M_S26_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1327A-TF-D-OA-M_S37_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1327A-TF-D-OA-M_S37_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1327A-TF-D-OA-M_S37_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1327A-TF-D-OA-M_S37_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1335-TF-H-DL-M_S10_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1335-TF-H-DL-M_S10_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1335-TF-H-DL-M_S10_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1335-TF-H-DL-M_S10_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1338I-T1-H-OA-M_S35_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1338I-T1-H-OA-M_S35_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1338I-T1-H-OA-M_S35_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1338I-T1-H-OA-M_S35_50_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1339B-T1-H-SS-M_S25_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1339B-T1-H-SS-M_S25_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1339B-T1-H-SS-M_S25_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1339B-T1-H-SS-M_S25_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1345C-T1-H-CN-M_S11_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1345C-T1-H-CN-M_S11_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1345C-T1-H-CN-M_S11_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1345C-T1-H-CN-M_S11_52_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1350G-TF-D-SS-M_S15_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1350G-TF-D-SS-M_S15_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1350G-TF-D-SS-M_S15_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1350G-TF-D-SS-M_S15_53_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1356C-T1-H-MC-M_S17_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1356C-T1-H-MC-M_S17_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1356C-T1-H-MC-M_S17_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1356C-T1-H-MC-M_S17_54_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1356C-T1-H-MC-M_S17_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1356C-T1-H-MC-M_S17_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1356C-T1-H-MC-M_S17_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1356C-T1-H-MC-M_S17_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.40 s (20 us/read; 3.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,590\n",
      "  Read 1 with adapter:                 119,916 (99.4%)\n",
      "  Read 2 with adapter:                 120,168 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       120,590 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,403,588 bp\n",
      "  Read 1:    18,207,385 bp\n",
      "  Read 2:    18,196,203 bp\n",
      "Total written (filtered):     31,722,727 bp (87.1%)\n",
      "  Read 1:    15,929,240 bp\n",
      "  Read 2:    15,793,487 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 119916 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t523\t0.0\t1\t71 452\n",
      "19\t119054\t0.0\t1\t108526 10528\n",
      "20\t322\t0.0\t1\t53 269\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 120168 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1884.2\t0\t1\n",
      "4\t5\t471.1\t0\t5\n",
      "11\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t8\t0.0\t1\t1 7\n",
      "19\t794\t0.0\t1\t133 661\n",
      "20\t119064\t0.0\t2\t116847 2217\n",
      "21\t292\t0.0\t2\t68 224\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1360D-TF-D-MC-M_S36_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1360D-TF-D-MC-M_S36_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1360D-TF-D-MC-M_S36_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1360D-TF-D-MC-M_S36_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.81 s (22 us/read; 2.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            127,824\n",
      "  Read 1 with adapter:                 127,187 (99.5%)\n",
      "  Read 2 with adapter:                 127,381 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       127,824 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,587,864 bp\n",
      "  Read 1:    19,300,732 bp\n",
      "  Read 2:    19,287,132 bp\n",
      "Total written (filtered):     33,624,711 bp (87.1%)\n",
      "  Read 1:    16,884,512 bp\n",
      "  Read 2:    16,740,199 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 127187 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t5\t0.0\t1\t1 4\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t561\t0.0\t1\t54 507\n",
      "19\t126271\t0.0\t1\t117446 8825\n",
      "20\t333\t0.0\t1\t71 262\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 127381 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t499.3\t0\t3\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t8\t0.0\t1\t4 4\n",
      "19\t869\t0.0\t1\t143 726\n",
      "20\t126208\t0.0\t2\t124453 1755\n",
      "21\t285\t0.0\t2\t63 222\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1365-TF-D-DL-M_S31_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1365-TF-D-DL-M_S31_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1365-TF-D-DL-M_S31_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1365-TF-D-DL-M_S31_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.59 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,768\n",
      "  Read 1 with adapter:                 115,237 (99.5%)\n",
      "  Read 2 with adapter:                 115,307 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,768 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,890,685 bp\n",
      "  Read 1:    17,449,573 bp\n",
      "  Read 2:    17,441,112 bp\n",
      "Total written (filtered):     30,396,117 bp (87.1%)\n",
      "  Read 1:    15,260,437 bp\n",
      "  Read 2:    15,135,680 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 115237 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t0 1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t6\t0.0\t1\t6\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t11\t0.0\t1\t7 4\n",
      "18\t637\t0.0\t1\t53 584\n",
      "19\t114228\t0.0\t1\t106126 8102\n",
      "20\t345\t0.0\t1\t58 287\n",
      "24\t1\t0.0\t1\t1\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 115307 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t452.2\t0\t3\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t4\t0.0\t1\t1 3\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t17\t0.0\t1\t10 7\n",
      "19\t870\t0.0\t1\t148 722\n",
      "20\t114135\t0.0\t2\t112202 1933\n",
      "21\t270\t0.0\t2\t57 213\n",
      "33\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1369-TF-H-DL-M_S30_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1369-TF-H-DL-M_S30_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1369-TF-H-DL-M_S30_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1369-TF-H-DL-M_S30_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.30 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            148,857\n",
      "  Read 1 with adapter:                 148,065 (99.5%)\n",
      "  Read 2 with adapter:                 148,332 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       148,857 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,922,793 bp\n",
      "  Read 1:    22,466,506 bp\n",
      "  Read 2:    22,456,287 bp\n",
      "Total written (filtered):     39,144,191 bp (87.1%)\n",
      "  Read 1:    19,653,657 bp\n",
      "  Read 2:    19,490,534 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 148065 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t9\t0.0\t1\t2 7\n",
      "17\t14\t0.0\t1\t6 8\n",
      "18\t698\t0.0\t1\t74 624\n",
      "19\t146904\t0.0\t1\t135590 11314\n",
      "20\t430\t0.0\t1\t74 356\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 148332 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t7\t581.5\t0\t7\n",
      "8\t1\t2.3\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t16\t0.0\t1\t7 9\n",
      "19\t1060\t0.0\t1\t166 894\n",
      "20\t146868\t0.0\t2\t144471 2397\n",
      "21\t371\t0.0\t2\t65 306\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1370A-TF-D-CN-M_S13_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1370A-TF-D-CN-M_S13_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1370A-TF-D-CN-M_S13_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1370A-TF-D-CN-M_S13_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.82 s (22 us/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,142\n",
      "  Read 1 with adapter:                 129,482 (99.5%)\n",
      "  Read 2 with adapter:                 129,631 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       130,142 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,276,599 bp\n",
      "  Read 1:    19,640,416 bp\n",
      "  Read 2:    19,636,183 bp\n",
      "Total written (filtered):     34,224,905 bp (87.1%)\n",
      "  Read 1:    17,180,540 bp\n",
      "  Read 2:    17,044,365 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 129482 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t11\t0.0\t1\t5 6\n",
      "18\t560\t0.0\t1\t62 498\n",
      "19\t128537\t0.0\t1\t119253 9284\n",
      "20\t363\t0.0\t1\t70 293\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 129631 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t508.4\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t8\t0.0\t1\t2 6\n",
      "19\t1029\t0.0\t1\t177 852\n",
      "20\t128262\t0.0\t2\t125894 2368\n",
      "21\t320\t0.0\t2\t79 241\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/15-DNA-Control_S39_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/15-DNA-Control_S39_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/15-DNA-Control_S39_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/15-DNA-Control_S39_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.64 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,815\n",
      "  Read 1 with adapter:                   8,770 (99.5%)\n",
      "  Read 2 with adapter:                   8,773 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         8,815 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,645,447 bp\n",
      "  Read 1:     1,323,115 bp\n",
      "  Read 2:     1,322,332 bp\n",
      "Total written (filtered):      2,303,433 bp (87.1%)\n",
      "  Read 1:     1,156,516 bp\n",
      "  Read 2:     1,146,917 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8770 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "18\t57\t0.0\t1\t6 51\n",
      "19\t8679\t0.0\t1\t8056 623\n",
      "20\t33\t0.0\t1\t3 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8773 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t70\t0.0\t1\t11 59\n",
      "20\t8678\t0.0\t2\t8500 178\n",
      "21\t25\t0.0\t2\t4 21\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/16-DNA-Control_S40_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/16-DNA-Control_S40_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/16-DNA-Control_S40_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/16-DNA-Control_S40_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.74 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,301\n",
      "  Read 1 with adapter:                  11,249 (99.5%)\n",
      "  Read 2 with adapter:                  11,240 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,301 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,296,627 bp\n",
      "  Read 1:     1,649,000 bp\n",
      "  Read 2:     1,647,627 bp\n",
      "Total written (filtered):      2,858,232 bp (86.7%)\n",
      "  Read 1:     1,435,312 bp\n",
      "  Read 2:     1,422,920 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11249 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t80\t0.0\t1\t7 73\n",
      "19\t11132\t0.0\t1\t10412 720\n",
      "20\t37\t0.0\t1\t4 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11240 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t120\t0.0\t1\t18 102\n",
      "20\t11090\t0.0\t2\t10873 217\n",
      "21\t29\t0.0\t2\t9 20\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1610D-TF-D-OA-M_S19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1610D-TF-D-OA-M_S19_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1610D-TF-D-OA-M_S19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1610D-TF-D-OA-M_S19_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.32 s (20 us/read; 3.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            166,773\n",
      "  Read 1 with adapter:                 165,739 (99.4%)\n",
      "  Read 2 with adapter:                 165,944 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       166,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,349,026 bp\n",
      "  Read 1:    25,177,176 bp\n",
      "  Read 2:    25,171,850 bp\n",
      "Total written (filtered):     43,883,271 bp (87.2%)\n",
      "  Read 1:    22,028,873 bp\n",
      "  Read 2:    21,854,398 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 165739 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t9\t0.0\t1\t9\n",
      "13\t5\t0.0\t1\t4 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t10\t0.0\t1\t4 6\n",
      "17\t8\t0.0\t1\t3 5\n",
      "18\t1049\t0.0\t1\t88 961\n",
      "19\t164174\t0.0\t1\t150719 13455\n",
      "20\t479\t0.0\t1\t100 379\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 165944 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t8\t651.5\t0\t8\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t22\t0.0\t1\t12 10\n",
      "19\t1576\t0.0\t1\t231 1345\n",
      "20\t163911\t0.0\t2\t160037 3874\n",
      "21\t410\t0.0\t2\t81 329\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1633A-TF-D-PA-M_S28_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1633A-TF-D-PA-M_S28_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1633A-TF-D-PA-M_S28_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1633A-TF-D-PA-M_S28_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.64 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            119,410\n",
      "  Read 1 with adapter:                 118,641 (99.4%)\n",
      "  Read 2 with adapter:                 118,840 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       119,410 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,056,462 bp\n",
      "  Read 1:    18,030,585 bp\n",
      "  Read 2:    18,025,877 bp\n",
      "Total written (filtered):     31,426,800 bp (87.2%)\n",
      "  Read 1:    15,776,855 bp\n",
      "  Read 2:    15,649,945 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 118641 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t4\t0.0\t1\t0 4\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t7\t0.0\t1\t4 3\n",
      "18\t693\t0.0\t1\t69 624\n",
      "19\t117589\t0.0\t1\t107345 10244\n",
      "20\t335\t0.0\t1\t46 289\n",
      "27\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 118840 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1865.8\t0\t1\n",
      "4\t3\t466.4\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t14\t0.0\t1\t11 3\n",
      "19\t1028\t0.0\t1\t150 878\n",
      "20\t117474\t0.0\t2\t115450 2024\n",
      "21\t310\t0.0\t2\t73 237\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1661A-TF-D-CN-M_S12_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1661A-TF-D-CN-M_S12_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1661A-TF-D-CN-M_S12_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1661A-TF-D-CN-M_S12_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.68 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,683\n",
      "  Read 1 with adapter:                 120,064 (99.5%)\n",
      "  Read 2 with adapter:                 120,134 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       120,683 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,441,501 bp\n",
      "  Read 1:    18,222,594 bp\n",
      "  Read 2:    18,218,907 bp\n",
      "Total written (filtered):     31,758,859 bp (87.2%)\n",
      "  Read 1:    15,941,822 bp\n",
      "  Read 2:    15,817,037 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 120064 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.8\t0\t1\n",
      "9\t4\t0.5\t0\t4\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t11\t0.0\t1\t5 6\n",
      "18\t660\t0.0\t1\t78 582\n",
      "19\t118994\t0.0\t1\t110552 8442\n",
      "20\t373\t0.0\t1\t68 305\n",
      "23\t1\t0.0\t1\t1\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 120134 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t471.4\t0\t2\n",
      "12\t3\t0.0\t1\t1 2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t12\t0.0\t1\t9 3\n",
      "19\t994\t0.0\t1\t173 821\n",
      "20\t118813\t0.0\t2\t117007 1806\n",
      "21\t300\t0.0\t2\t65 235\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1672D-TF-D-PS-M_S6_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1672D-TF-D-PS-M_S6_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1672D-TF-D-PS-M_S6_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1672D-TF-D-PS-M_S6_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.29 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            148,032\n",
      "  Read 1 with adapter:                 147,181 (99.4%)\n",
      "  Read 2 with adapter:                 147,374 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       148,032 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,684,404 bp\n",
      "  Read 1:    22,352,711 bp\n",
      "  Read 2:    22,331,693 bp\n",
      "Total written (filtered):     38,942,014 bp (87.1%)\n",
      "  Read 1:    19,556,872 bp\n",
      "  Read 2:    19,385,142 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 147181 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t4\t0.6\t0\t4\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t9\t0.0\t1\t8 1\n",
      "13\t4\t0.0\t1\t3 1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t6\t0.0\t1\t1 5\n",
      "16\t8\t0.0\t1\t2 6\n",
      "17\t11\t0.0\t1\t6 5\n",
      "18\t776\t0.0\t1\t93 683\n",
      "19\t145966\t0.0\t1\t134234 11732\n",
      "20\t392\t0.0\t1\t71 321\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 147374 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2313.0\t0\t1\n",
      "4\t2\t578.2\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t16\t0.0\t1\t8 8\n",
      "19\t1137\t0.0\t1\t195 942\n",
      "20\t145848\t0.0\t2\t143317 2531\n",
      "21\t353\t0.0\t2\t76 277\n",
      "22\t3\t0.0\t2\t1 2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1681A-TF-D-PA-M_S5_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1681A-TF-D-PA-M_S5_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1681A-TF-D-PA-M_S5_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1681A-TF-D-PA-M_S5_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.29 s (24 us/read; 2.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,610\n",
      "  Read 1 with adapter:                  95,968 (99.3%)\n",
      "  Read 2 with adapter:                  96,095 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        96,610 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,170,933 bp\n",
      "  Read 1:    14,587,303 bp\n",
      "  Read 2:    14,583,630 bp\n",
      "Total written (filtered):     25,426,787 bp (87.2%)\n",
      "  Read 1:    12,764,324 bp\n",
      "  Read 2:    12,662,463 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 95968 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t6\t0.0\t1\t5 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t566\t0.0\t1\t62 504\n",
      "19\t95141\t0.0\t1\t86999 8142\n",
      "20\t239\t0.0\t1\t45 194\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 96095 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t377.4\t0\t3\n",
      "8\t1\t1.5\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t5\t0.0\t1\t1 4\n",
      "19\t870\t0.0\t1\t127 743\n",
      "20\t94963\t0.0\t2\t93060 1903\n",
      "21\t245\t0.0\t2\t49 196\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/17-DNA-Control_S41_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/17-DNA-Control_S41_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/17-DNA-Control_S41_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/17-DNA-Control_S41_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.30 s (501 us/read; 0.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                600\n",
      "  Read 1 with adapter:                     598 (99.7%)\n",
      "  Read 2 with adapter:                     597 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):           600 (100.0%)\n",
      "\n",
      "Total basepairs processed:       174,744 bp\n",
      "  Read 1:        87,388 bp\n",
      "  Read 2:        87,356 bp\n",
      "Total written (filtered):        151,449 bp (86.7%)\n",
      "  Read 1:        76,029 bp\n",
      "  Read 2:        75,420 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 598 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t1 2\n",
      "19\t595\t0.0\t1\t545 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 597 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t5\t0.0\t1\t1 4\n",
      "20\t591\t0.0\t2\t571 20\n",
      "21\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1827D-T1-H-PA-M_S18_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1827D-T1-H-PA-M_S18_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1827D-T1-H-PA-M_S18_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1827D-T1-H-PA-M_S18_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.17 s (17 us/read; 3.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,024\n",
      "  Read 1 with adapter:                 129,149 (99.3%)\n",
      "  Read 2 with adapter:                 129,380 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       130,024 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,260,545 bp\n",
      "  Read 1:    19,633,441 bp\n",
      "  Read 2:    19,627,104 bp\n",
      "Total written (filtered):     34,220,634 bp (87.2%)\n",
      "  Read 1:    17,180,005 bp\n",
      "  Read 2:    17,040,629 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 129149 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.5\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t700\t0.0\t1\t70 630\n",
      "19\t128068\t0.0\t1\t116338 11730\n",
      "20\t363\t0.0\t1\t58 305\n",
      "21\t2\t0.0\t1\t2\n",
      "37\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 129380 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t8\t507.9\t0\t8\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t17\t0.0\t1\t11 6\n",
      "19\t1260\t0.0\t1\t188 1072\n",
      "20\t127740\t0.0\t2\t125024 2716\n",
      "21\t347\t0.0\t2\t71 276\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1827E-TF-D-PA-M_S4_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1827E-TF-D-PA-M_S4_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1827E-TF-D-PA-M_S4_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1827E-TF-D-PA-M_S4_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.13 s (20 us/read; 2.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,613\n",
      "  Read 1 with adapter:                 104,039 (99.5%)\n",
      "  Read 2 with adapter:                 104,107 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       104,613 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,579,791 bp\n",
      "  Read 1:    15,795,095 bp\n",
      "  Read 2:    15,784,696 bp\n",
      "Total written (filtered):     27,522,266 bp (87.2%)\n",
      "  Read 1:    13,818,747 bp\n",
      "  Read 2:    13,703,519 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 104039 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "7\t1\t6.4\t0\t1\n",
      "12\t5\t0.0\t1\t2 3\n",
      "13\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t614\t0.0\t1\t64 550\n",
      "19\t103115\t0.0\t1\t95191 7924\n",
      "20\t294\t0.0\t1\t52 242\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 104107 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t408.6\t0\t3\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t4\t0.0\t1\t2 2\n",
      "15\t5\t0.0\t1\t3 2\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t11\t0.0\t1\t6 5\n",
      "19\t1031\t0.0\t1\t137 894\n",
      "20\t102794\t0.0\t2\t100678 2116\n",
      "21\t246\t0.0\t2\t53 193\n",
      "40\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1360D-TF-D-MC-M_S36_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1360D-TF-D-MC-M_S36_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1360D-TF-D-MC-M_S36_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1360D-TF-D-MC-M_S36_55_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1365-TF-D-DL-M_S31_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1365-TF-D-DL-M_S31_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1365-TF-D-DL-M_S31_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1365-TF-D-DL-M_S31_56_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1369-TF-H-DL-M_S30_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1369-TF-H-DL-M_S30_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1369-TF-H-DL-M_S30_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1369-TF-H-DL-M_S30_57_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1370A-TF-D-CN-M_S13_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1370A-TF-D-CN-M_S13_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1370A-TF-D-CN-M_S13_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1370A-TF-D-CN-M_S13_58_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/15-DNA-Control_S39_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/15-DNA-Control_S39_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/15-DNA-Control_S39_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/15-DNA-Control_S39_59_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/16-DNA-Control_S40_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/16-DNA-Control_S40_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/16-DNA-Control_S40_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/16-DNA-Control_S40_65_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1610D-TF-D-OA-M_S19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1610D-TF-D-OA-M_S19_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1610D-TF-D-OA-M_S19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1610D-TF-D-OA-M_S19_60_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1633A-TF-D-PA-M_S28_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1633A-TF-D-PA-M_S28_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1633A-TF-D-PA-M_S28_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1633A-TF-D-PA-M_S28_61_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1661A-TF-D-CN-M_S12_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1661A-TF-D-CN-M_S12_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1661A-TF-D-CN-M_S12_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1661A-TF-D-CN-M_S12_62_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1672D-TF-D-PS-M_S6_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1672D-TF-D-PS-M_S6_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1672D-TF-D-PS-M_S6_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1672D-TF-D-PS-M_S6_63_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1681A-TF-D-PA-M_S5_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1681A-TF-D-PA-M_S5_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1681A-TF-D-PA-M_S5_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1681A-TF-D-PA-M_S5_64_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/17-DNA-Control_S41_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/17-DNA-Control_S41_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/17-DNA-Control_S41_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/17-DNA-Control_S41_66_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1827D-T1-H-PA-M_S18_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1827D-T1-H-PA-M_S18_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1827D-T1-H-PA-M_S18_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1827D-T1-H-PA-M_S18_67_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1827E-TF-D-PA-M_S4_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1827E-TF-D-PA-M_S4_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1827E-TF-D-PA-M_S4_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1827E-TF-D-PA-M_S4_68_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1833B-TF-D-CN-M_S33_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1833B-TF-D-CN-M_S33_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1833B-TF-D-CN-M_S33_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1833B-TF-D-CN-M_S33_69_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1833B-TF-D-CN-M_S33_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1833B-TF-D-CN-M_S33_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1833B-TF-D-CN-M_S33_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1833B-TF-D-CN-M_S33_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.42 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,195\n",
      "  Read 1 with adapter:                 113,648 (99.5%)\n",
      "  Read 2 with adapter:                 113,733 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,195 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,480,786 bp\n",
      "  Read 1:    17,243,058 bp\n",
      "  Read 2:    17,237,728 bp\n",
      "Total written (filtered):     30,047,825 bp (87.1%)\n",
      "  Read 1:    15,084,086 bp\n",
      "  Read 2:    14,963,739 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113648 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t561\t0.0\t1\t71 490\n",
      "19\t112748\t0.0\t1\t104593 8155\n",
      "20\t315\t0.0\t1\t47 268\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 113733 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t446.1\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t8\t0.0\t1\t2 6\n",
      "19\t795\t0.0\t1\t116 679\n",
      "20\t112657\t0.0\t2\t110674 1983\n",
      "21\t259\t0.0\t2\t71 188\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1835B-TF-H-OA-M-LibraryJ_S1_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1835B-TF-H-OA-M-LibraryJ_S1_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1835B-TF-H-OA-M-LibraryJ_S1_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1835B-TF-H-OA-M-LibraryJ_S1_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.82 s (20 us/read; 2.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,412\n",
      "  Read 1 with adapter:                  89,980 (99.5%)\n",
      "  Read 2 with adapter:                  90,029 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        90,412 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,291,868 bp\n",
      "  Read 1:    13,647,566 bp\n",
      "  Read 2:    13,644,302 bp\n",
      "Total written (filtered):     23,782,451 bp (87.1%)\n",
      "  Read 1:    11,938,183 bp\n",
      "  Read 2:    11,844,268 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 89980 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t431\t0.0\t1\t64 367\n",
      "19\t89271\t0.0\t1\t82889 6382\n",
      "20\t259\t0.0\t1\t43 216\n",
      "42\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 90029 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t353.2\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t10\t0.0\t1\t4 6\n",
      "19\t700\t0.0\t1\t100 600\n",
      "20\t89082\t0.0\t2\t87668 1414\n",
      "21\t231\t0.0\t2\t54 177\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1840B-TF-H-PS-M_S8_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1840B-TF-H-PS-M_S8_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1840B-TF-H-PS-M_S8_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1840B-TF-H-PS-M_S8_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.43 s (20 us/read; 2.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            121,226\n",
      "  Read 1 with adapter:                 120,546 (99.4%)\n",
      "  Read 2 with adapter:                 120,795 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       121,226 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,606,601 bp\n",
      "  Read 1:    18,303,200 bp\n",
      "  Read 2:    18,303,401 bp\n",
      "Total written (filtered):     31,901,307 bp (87.1%)\n",
      "  Read 1:    16,013,081 bp\n",
      "  Read 2:    15,888,226 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 120546 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t510\t0.0\t1\t44 466\n",
      "19\t119680\t0.0\t1\t109921 9759\n",
      "20\t335\t0.0\t1\t46 289\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 120795 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1894.2\t0\t1\n",
      "4\t2\t473.5\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t5 1\n",
      "18\t9\t0.0\t1\t6 3\n",
      "19\t880\t0.0\t1\t146 734\n",
      "20\t119622\t0.0\t2\t117449 2173\n",
      "21\t270\t0.0\t2\t54 216\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1842A-TF-D-PS-M_S29_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1842A-TF-D-PS-M_S29_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1842A-TF-D-PS-M_S29_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1842A-TF-D-PS-M_S29_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.51 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,179\n",
      "  Read 1 with adapter:                 110,220 (99.1%)\n",
      "  Read 2 with adapter:                 110,704 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,179 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,564,768 bp\n",
      "  Read 1:    16,783,952 bp\n",
      "  Read 2:    16,780,816 bp\n",
      "Total written (filtered):     29,257,555 bp (87.2%)\n",
      "  Read 1:    14,690,155 bp\n",
      "  Read 2:    14,567,400 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110220 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t10\t0.0\t1\t9 1\n",
      "13\t4\t0.0\t1\t3 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t8\t0.0\t1\t4 4\n",
      "17\t9\t0.0\t1\t5 4\n",
      "18\t515\t0.0\t1\t65 450\n",
      "19\t109376\t0.0\t1\t98108 11268\n",
      "20\t288\t0.0\t1\t57 231\n",
      "23\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "32\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110704 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t434.3\t0\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t5\t0.0\t1\t2 3\n",
      "19\t842\t0.0\t1\t146 696\n",
      "20\t109615\t0.0\t2\t107792 1823\n",
      "21\t237\t0.0\t2\t59 178\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1858A-TF-H-PA-M_S24_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1858A-TF-H-PA-M_S24_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1858A-TF-H-PA-M_S24_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1858A-TF-H-PA-M_S24_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.52 s (19 us/read; 3.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,384\n",
      "  Read 1 with adapter:                 128,711 (99.5%)\n",
      "  Read 2 with adapter:                 128,806 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       129,384 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,067,271 bp\n",
      "  Read 1:    19,536,339 bp\n",
      "  Read 2:    19,530,932 bp\n",
      "Total written (filtered):     34,047,054 bp (87.1%)\n",
      "  Read 1:    17,091,378 bp\n",
      "  Read 2:    16,955,676 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 128711 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t4\t0.5\t0\t4\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t770\t0.0\t1\t76 694\n",
      "19\t127537\t0.0\t1\t117771 9766\n",
      "20\t377\t0.0\t1\t73 304\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 128806 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2021.6\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t15\t0.0\t1\t11 4\n",
      "19\t1132\t0.0\t1\t168 964\n",
      "20\t127323\t0.0\t2\t124970 2353\n",
      "21\t329\t0.0\t2\t65 264\n",
      "32\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1863A-TF-H-CN-M_S20_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1863A-TF-H-CN-M_S20_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1863A-TF-H-CN-M_S20_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1863A-TF-H-CN-M_S20_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.38 s (18 us/read; 3.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            135,361\n",
      "  Read 1 with adapter:                 134,716 (99.5%)\n",
      "  Read 2 with adapter:                 134,839 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       135,361 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,864,013 bp\n",
      "  Read 1:    20,434,430 bp\n",
      "  Read 2:    20,429,583 bp\n",
      "Total written (filtered):     35,608,929 bp (87.1%)\n",
      "  Read 1:    17,875,197 bp\n",
      "  Read 2:    17,733,732 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 134716 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t2 2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t10\t0.0\t1\t3 7\n",
      "18\t657\t0.0\t1\t73 584\n",
      "19\t133650\t0.0\t1\t124198 9452\n",
      "20\t384\t0.0\t1\t63 321\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 134839 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2115.0\t0\t1\n",
      "4\t2\t528.8\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t3\t0.0\t1\t1 2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t16\t0.0\t1\t3 13\n",
      "19\t1079\t0.0\t1\t186 893\n",
      "20\t133404\t0.0\t2\t131376 2028\n",
      "21\t324\t0.0\t2\t69 255\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1863I-T1-H-CN-M_S32_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1863I-T1-H-CN-M_S32_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1863I-T1-H-CN-M_S32_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1863I-T1-H-CN-M_S32_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.61 s (21 us/read; 2.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            126,849\n",
      "  Read 1 with adapter:                 126,256 (99.5%)\n",
      "  Read 2 with adapter:                 126,347 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       126,849 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,303,835 bp\n",
      "  Read 1:    19,153,576 bp\n",
      "  Read 2:    19,150,259 bp\n",
      "Total written (filtered):     33,379,207 bp (87.1%)\n",
      "  Read 1:    16,755,070 bp\n",
      "  Read 2:    16,624,137 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 126256 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t5\t0.0\t1\t3 2\n",
      "16\t4\t0.0\t1\t0 4\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t604\t0.0\t1\t58 546\n",
      "19\t125298\t0.0\t1\t116182 9116\n",
      "20\t336\t0.0\t1\t67 269\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 126347 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t5\t495.5\t0\t5\n",
      "9\t2\t0.5\t0\t2\n",
      "11\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t8\t0.0\t1\t3 5\n",
      "19\t924\t0.0\t1\t156 768\n",
      "20\t125121\t0.0\t2\t122853 2268\n",
      "21\t278\t0.0\t2\t61 217\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1864B-TF-D-PS-M_S7_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1864B-TF-D-PS-M_S7_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1864B-TF-D-PS-M_S7_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1864B-TF-D-PS-M_S7_76_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.40 s (22 us/read; 2.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,517\n",
      "  Read 1 with adapter:                 109,025 (99.6%)\n",
      "  Read 2 with adapter:                 108,992 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,517 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,070,672 bp\n",
      "  Read 1:    16,536,907 bp\n",
      "  Read 2:    16,533,765 bp\n",
      "Total written (filtered):     28,820,368 bp (87.1%)\n",
      "  Read 1:    14,465,807 bp\n",
      "  Read 2:    14,354,561 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109025 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t5\t0.0\t1\t1 4\n",
      "16\t4\t0.0\t1\t4\n",
      "17\t8\t0.0\t1\t2 6\n",
      "18\t553\t0.0\t1\t66 487\n",
      "19\t108162\t0.0\t1\t100406 7756\n",
      "20\t283\t0.0\t1\t59 224\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108992 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t427.8\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t8\t0.0\t1\t3 5\n",
      "19\t806\t0.0\t1\t138 668\n",
      "20\t107888\t0.0\t2\t105999 1889\n",
      "21\t273\t0.0\t2\t65 208\n",
      "22\t1\t0.0\t2\t0 1\n",
      "40\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2186A-TF-D-PA-M_S21_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2186A-TF-D-PA-M_S21_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2186A-TF-D-PA-M_S21_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2186A-TF-D-PA-M_S21_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.30 s (19 us/read; 3.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,951\n",
      "  Read 1 with adapter:                 116,690 (98.9%)\n",
      "  Read 2 with adapter:                 117,337 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       117,951 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,617,173 bp\n",
      "  Read 1:    17,810,511 bp\n",
      "  Read 2:    17,806,662 bp\n",
      "Total written (filtered):     31,054,579 bp (87.2%)\n",
      "  Read 1:    15,593,757 bp\n",
      "  Read 2:    15,460,822 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 116690 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t7\t0.0\t1\t1 6\n",
      "18\t634\t0.0\t1\t78 556\n",
      "19\t115731\t0.0\t1\t102143 13588\n",
      "20\t310\t0.0\t1\t54 256\n",
      "21\t1\t0.0\t1\t0 1\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 117337 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1843.0\t0\t1\n",
      "4\t1\t460.7\t0\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t4\t0.0\t1\t4\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t18\t0.0\t1\t13 5\n",
      "19\t1094\t0.0\t1\t144 950\n",
      "20\t115915\t0.0\t2\t113692 2223\n",
      "21\t291\t0.0\t2\t62 229\n",
      "23\t1\t0.0\t2\t1\n",
      "35\t1\t0.0\t2\t1\n",
      "41\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2199D-TF-D-SS-M_S14_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2199D-TF-D-SS-M_S14_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2199D-TF-D-SS-M_S14_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2199D-TF-D-SS-M_S14_78_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.69 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            118,987\n",
      "  Read 1 with adapter:                 118,261 (99.4%)\n",
      "  Read 2 with adapter:                 118,370 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       118,987 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,929,242 bp\n",
      "  Read 1:    17,966,732 bp\n",
      "  Read 2:    17,962,510 bp\n",
      "Total written (filtered):     31,316,290 bp (87.2%)\n",
      "  Read 1:    15,720,247 bp\n",
      "  Read 2:    15,596,043 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 118261 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t702\t0.0\t1\t67 635\n",
      "19\t117198\t0.0\t1\t107217 9981\n",
      "20\t338\t0.0\t1\t59 279\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 118370 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1859.2\t0\t1\n",
      "4\t3\t464.8\t0\t3\n",
      "5\t1\t116.2\t0\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t15\t0.0\t1\t10 5\n",
      "19\t1073\t0.0\t1\t180 893\n",
      "20\t116963\t0.0\t2\t114694 2269\n",
      "21\t303\t0.0\t2\t66 237\n",
      "27\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2199E-TF-H-SS-M_S23_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2199E-TF-H-SS-M_S23_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2199E-TF-H-SS-M_S23_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2199E-TF-H-SS-M_S23_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.90 s (22 us/read; 2.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,987\n",
      "  Read 1 with adapter:                 129,365 (99.5%)\n",
      "  Read 2 with adapter:                 129,373 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       129,987 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,193,681 bp\n",
      "  Read 1:    19,604,713 bp\n",
      "  Read 2:    19,588,968 bp\n",
      "Total written (filtered):     34,149,645 bp (87.1%)\n",
      "  Read 1:    17,147,348 bp\n",
      "  Read 2:    17,002,297 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 129365 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t6\t0.0\t1\t6\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t12\t0.0\t1\t6 6\n",
      "18\t791\t0.0\t1\t88 703\n",
      "19\t128178\t0.0\t1\t119045 9133\n",
      "20\t362\t0.0\t1\t66 296\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 129373 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2031.0\t0\t1\n",
      "4\t2\t507.8\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "18\t10\t0.0\t1\t5 5\n",
      "19\t1011\t0.0\t1\t156 855\n",
      "20\t128028\t0.0\t2\t126088 1940\n",
      "21\t315\t0.0\t2\t72 243\n",
      "23\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/PCR-Control_S38_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/PCR-Control_S38_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/PCR-Control_S38_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/PCR-Control_S38_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.28 s (1208 us/read; 0.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                230\n",
      "  Read 1 with adapter:                     226 (98.3%)\n",
      "  Read 2 with adapter:                     229 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):           230 (100.0%)\n",
      "\n",
      "Total basepairs processed:        67,292 bp\n",
      "  Read 1:        33,659 bp\n",
      "  Read 2:        33,633 bp\n",
      "Total written (filtered):         58,423 bp (86.8%)\n",
      "  Read 1:        29,366 bp\n",
      "  Read 2:        29,057 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 226 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t1 1\n",
      "19\t223\t0.0\t1\t180 43\n",
      "20\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 229 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t4\t0.0\t1\t0 4\n",
      "20\t225\t0.0\t2\t221 4\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/mock-1_LibJ_S2_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/mock-1_LibJ_S2_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/mock-1_LibJ_S2_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/mock-1_LibJ_S2_80_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.00 s (22 us/read; 2.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,241\n",
      "  Read 1 with adapter:                  90,635 (99.3%)\n",
      "  Read 2 with adapter:                  90,792 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        91,241 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,532,132 bp\n",
      "  Read 1:    13,777,237 bp\n",
      "  Read 2:    13,754,895 bp\n",
      "Total written (filtered):     23,995,311 bp (87.2%)\n",
      "  Read 1:    12,055,537 bp\n",
      "  Read 2:    11,939,774 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90635 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.3\t0\t2\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t512\t0.0\t1\t49 463\n",
      "19\t89868\t0.0\t1\t82377 7491\n",
      "20\t236\t0.0\t1\t54 182\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 90792 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t356.4\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t845\t0.0\t1\t137 708\n",
      "20\t89700\t0.0\t2\t87941 1759\n",
      "21\t227\t0.0\t2\t46 181\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1835B-TF-H-OA-M-LibraryJ_S1_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1835B-TF-H-OA-M-LibraryJ_S1_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1835B-TF-H-OA-M-LibraryJ_S1_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1835B-TF-H-OA-M-LibraryJ_S1_70_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1840B-TF-H-PS-M_S8_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1840B-TF-H-PS-M_S8_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1840B-TF-H-PS-M_S8_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1840B-TF-H-PS-M_S8_71_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1842A-TF-D-PS-M_S29_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1842A-TF-D-PS-M_S29_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1842A-TF-D-PS-M_S29_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1842A-TF-D-PS-M_S29_72_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1858A-TF-H-PA-M_S24_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1858A-TF-H-PA-M_S24_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1858A-TF-H-PA-M_S24_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1858A-TF-H-PA-M_S24_73_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1863A-TF-H-CN-M_S20_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1863A-TF-H-CN-M_S20_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1863A-TF-H-CN-M_S20_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1863A-TF-H-CN-M_S20_74_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1863I-T1-H-CN-M_S32_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1863I-T1-H-CN-M_S32_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1863I-T1-H-CN-M_S32_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1863I-T1-H-CN-M_S32_75_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1864B-TF-D-PS-M_S7_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/1864B-TF-D-PS-M_S7_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1864B-TF-D-PS-M_S7_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/1864B-TF-D-PS-M_S7_76_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2186A-TF-D-PA-M_S21_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2186A-TF-D-PA-M_S21_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2186A-TF-D-PA-M_S21_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2186A-TF-D-PA-M_S21_77_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2199D-TF-D-SS-M_S14_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2199D-TF-D-SS-M_S14_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2199D-TF-D-SS-M_S14_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2199D-TF-D-SS-M_S14_78_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2199E-TF-H-SS-M_S23_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/2199E-TF-H-SS-M_S23_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2199E-TF-H-SS-M_S23_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/2199E-TF-H-SS-M_S23_79_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/PCR-Control_S38_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/PCR-Control_S38_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/PCR-Control_S38_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/PCR-Control_S38_81_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/mock-1_LibJ_S2_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kehidiye/mock-1_LibJ_S2_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/mock-1_LibJ_S2_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-k9y0fyfa/9126b4e5-d01c-4b8f-b7b2-ccffdcda04de/data/mock-1_LibJ_S2_80_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_LibJ.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1295A-T1-H-PA-M_S18_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1295A-T1-H-PA-M_S18_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1295A-T1-H-PA-M_S18_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1295A-T1-H-PA-M_S18_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.16 s (25 us/read; 2.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,686\n",
      "  Read 1 with adapter:                  45,192 (98.9%)\n",
      "  Read 2 with adapter:                  45,360 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,686 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,793,054 bp\n",
      "  Read 1:     6,897,279 bp\n",
      "  Read 2:     6,895,775 bp\n",
      "Total written (filtered):     12,027,613 bp (87.2%)\n",
      "  Read 1:     6,038,748 bp\n",
      "  Read 2:     5,988,865 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 45192 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t218\t0.0\t1\t21 197\n",
      "19\t44853\t0.0\t1\t39032 5821\n",
      "20\t118\t0.0\t1\t23 95\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 45360 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t713.8\t0\t1\n",
      "4\t1\t178.5\t0\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t6\t0.0\t1\t3 3\n",
      "19\t377\t0.0\t1\t65 312\n",
      "20\t44833\t0.0\t2\t44094 739\n",
      "21\t140\t0.0\t2\t33 107\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1301-T1-H-DL-M_S28_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1301-T1-H-DL-M_S28_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1301-T1-H-DL-M_S28_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1301-T1-H-DL-M_S28_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.52 s (21 us/read; 2.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,537\n",
      "  Read 1 with adapter:                 119,585 (99.2%)\n",
      "  Read 2 with adapter:                 119,935 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       120,537 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,386,619 bp\n",
      "  Read 1:    18,200,657 bp\n",
      "  Read 2:    18,185,962 bp\n",
      "Total written (filtered):     31,716,967 bp (87.2%)\n",
      "  Read 1:    15,928,894 bp\n",
      "  Read 2:    15,788,073 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 119585 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "13\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t10\t0.0\t1\t5 5\n",
      "18\t595\t0.0\t1\t62 533\n",
      "19\t118666\t0.0\t1\t100823 17843\n",
      "20\t305\t0.0\t1\t60 245\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 119935 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1883.4\t0\t1\n",
      "4\t3\t470.8\t0\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t993\t0.0\t1\t152 841\n",
      "20\t118641\t0.0\t2\t116409 2232\n",
      "21\t278\t0.0\t2\t58 220\n",
      "29\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1304A-T1-H-PA-M_S17_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1304A-T1-H-PA-M_S17_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1304A-T1-H-PA-M_S17_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1304A-T1-H-PA-M_S17_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.03 s (22 us/read; 2.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,712\n",
      "  Read 1 with adapter:                  91,851 (99.1%)\n",
      "  Read 2 with adapter:                  92,121 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        92,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,994,304 bp\n",
      "  Read 1:    13,998,595 bp\n",
      "  Read 2:    13,995,709 bp\n",
      "Total written (filtered):     24,407,585 bp (87.2%)\n",
      "  Read 1:    12,253,722 bp\n",
      "  Read 2:    12,153,863 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91851 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t14\t0.0\t1\t7 7\n",
      "18\t440\t0.0\t1\t63 377\n",
      "19\t91180\t0.0\t1\t79223 11957\n",
      "20\t207\t0.0\t1\t41 166\n",
      "37\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 92121 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1448.6\t0\t1\n",
      "4\t5\t362.2\t0\t5\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t5\t0.0\t1\t4 1\n",
      "19\t681\t0.0\t1\t92 589\n",
      "20\t91200\t0.0\t2\t89655 1545\n",
      "21\t226\t0.0\t2\t62 164\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1305B-TF-D-OA-M_S13_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1305B-TF-D-OA-M_S13_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1305B-TF-D-OA-M_S13_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1305B-TF-D-OA-M_S13_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.09 s (24 us/read; 2.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             88,549\n",
      "  Read 1 with adapter:                  87,424 (98.7%)\n",
      "  Read 2 with adapter:                  88,040 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        88,549 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,737,161 bp\n",
      "  Read 1:    13,370,204 bp\n",
      "  Read 2:    13,366,957 bp\n",
      "Total written (filtered):     23,316,512 bp (87.2%)\n",
      "  Read 1:    11,709,601 bp\n",
      "  Read 2:    11,606,911 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 87424 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.4\t0\t1\n",
      "9\t4\t0.3\t0\t4\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t3\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t587\t0.0\t1\t54 533\n",
      "19\t86596\t0.0\t1\t70270 16326\n",
      "20\t223\t0.0\t1\t36 187\n",
      "36\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 88040 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t345.9\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t5\t0.0\t1\t1 4\n",
      "19\t889\t0.0\t1\t139 750\n",
      "20\t86927\t0.0\t2\t84565 2362\n",
      "21\t207\t0.0\t2\t58 149\n",
      "31\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1319G-T1-H-SS-M_S6_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1319G-T1-H-SS-M_S6_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1319G-T1-H-SS-M_S6_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1319G-T1-H-SS-M_S6_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.47 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            157,133\n",
      "  Read 1 with adapter:                 153,759 (97.9%)\n",
      "  Read 2 with adapter:                 156,473 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       157,133 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,442,819 bp\n",
      "  Read 1:    23,725,107 bp\n",
      "  Read 2:    23,717,712 bp\n",
      "Total written (filtered):     41,393,251 bp (87.2%)\n",
      "  Read 1:    20,804,060 bp\n",
      "  Read 2:    20,589,191 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 153759 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t5\t0.0\t1\t3 2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t606\t0.0\t1\t62 544\n",
      "19\t152840\t0.0\t1\t96126 56714\n",
      "20\t299\t0.0\t1\t49 250\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 156473 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2455.2\t0\t1\n",
      "4\t6\t613.8\t0\t6\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t11\t0.0\t1\t4 7\n",
      "19\t1146\t0.0\t1\t172 974\n",
      "20\t154941\t0.0\t2\t152004 2937\n",
      "21\t362\t0.0\t2\t88 274\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1322-T1-D-DL-M_S45_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1322-T1-D-DL-M_S45_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1322-T1-D-DL-M_S45_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1322-T1-D-DL-M_S45_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.68 s (26 us/read; 2.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            102,132\n",
      "  Read 1 with adapter:                  99,131 (97.1%)\n",
      "  Read 2 with adapter:                  99,890 (97.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       102,132 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,604,889 bp\n",
      "  Read 1:    15,341,275 bp\n",
      "  Read 2:    15,263,614 bp\n",
      "Total written (filtered):     26,724,754 bp (87.3%)\n",
      "  Read 1:    13,458,075 bp\n",
      "  Read 2:    13,266,679 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99131 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t0 2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t6\t0.0\t1\t2 4\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t503\t0.0\t1\t52 451\n",
      "19\t98332\t0.0\t1\t75688 22644\n",
      "20\t279\t0.0\t1\t52 227\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 99890 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1595.8\t0\t1\n",
      "4\t4\t399.0\t0\t4\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t11\t0.0\t1\t6 5\n",
      "19\t969\t0.0\t1\t145 824\n",
      "20\t98641\t0.0\t2\t96817 1824\n",
      "21\t254\t0.0\t2\t48 206\n",
      "24\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1327-T1-H-OA-M_S11_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1327-T1-H-OA-M_S11_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1327-T1-H-OA-M_S11_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1327-T1-H-OA-M_S11_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.56 s (25 us/read; 2.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,364\n",
      "  Read 1 with adapter:                  60,760 (99.0%)\n",
      "  Read 2 with adapter:                  61,078 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        61,364 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,141,147 bp\n",
      "  Read 1:     9,073,055 bp\n",
      "  Read 2:     9,068,092 bp\n",
      "Total written (filtered):     15,765,728 bp (86.9%)\n",
      "  Read 1:     7,918,706 bp\n",
      "  Read 2:     7,847,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 60760 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t320\t0.0\t1\t46 274\n",
      "19\t60269\t0.0\t1\t49544 10725\n",
      "20\t163\t0.0\t1\t44 119\n",
      "110\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 61078 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t958.8\t0\t1\n",
      "4\t2\t239.7\t0\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t9\t0.0\t1\t8 1\n",
      "19\t530\t0.0\t1\t81 449\n",
      "20\t60392\t0.0\t2\t59330 1062\n",
      "21\t137\t0.0\t2\t33 104\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1327B-TF-H-OA-M_S37_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1327B-TF-H-OA-M_S37_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1327B-TF-H-OA-M_S37_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1327B-TF-H-OA-M_S37_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.77 s (21 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,329\n",
      "  Read 1 with adapter:                  81,518 (99.0%)\n",
      "  Read 2 with adapter:                  81,957 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        82,329 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,857,706 bp\n",
      "  Read 1:    12,430,275 bp\n",
      "  Read 2:    12,427,431 bp\n",
      "Total written (filtered):     21,670,443 bp (87.2%)\n",
      "  Read 1:    10,881,563 bp\n",
      "  Read 2:    10,788,880 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 81518 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t346\t0.0\t1\t50 296\n",
      "19\t80916\t0.0\t1\t65926 14990\n",
      "20\t242\t0.0\t1\t59 183\n",
      "22\t1\t0.0\t1\t1\n",
      "32\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 81957 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t7\t321.6\t0\t7\n",
      "11\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t4\t0.0\t1\t2 2\n",
      "19\t621\t0.0\t1\t97 524\n",
      "20\t81128\t0.0\t2\t79799 1329\n",
      "21\t189\t0.0\t2\t35 154\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1338A-TF-H-OA-M_S33_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1338A-TF-H-OA-M_S33_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1338A-TF-H-OA-M_S33_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1338A-TF-H-OA-M_S33_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.13 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,102\n",
      "  Read 1 with adapter:                  93,264 (99.1%)\n",
      "  Read 2 with adapter:                  93,734 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        94,102 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,413,882 bp\n",
      "  Read 1:    14,208,069 bp\n",
      "  Read 2:    14,205,813 bp\n",
      "Total written (filtered):     24,767,932 bp (87.2%)\n",
      "  Read 1:    12,436,247 bp\n",
      "  Read 2:    12,331,685 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 93264 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.4\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t7\t0.0\t1\t2 5\n",
      "18\t406\t0.0\t1\t65 341\n",
      "19\t92591\t0.0\t1\t78378 14213\n",
      "20\t249\t0.0\t1\t53 196\n",
      "21\t1\t0.0\t1\t1\n",
      "24\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 93734 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1470.3\t0\t1\n",
      "4\t3\t367.6\t0\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t668\t0.0\t1\t106 562\n",
      "20\t92831\t0.0\t2\t91038 1793\n",
      "21\t218\t0.0\t2\t41 177\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1339A-TF-D-SS-M_S8_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1339A-TF-D-SS-M_S8_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1339A-TF-D-SS-M_S8_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1339A-TF-D-SS-M_S8_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.29 s (22 us/read; 2.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            147,492\n",
      "  Read 1 with adapter:                 146,049 (99.0%)\n",
      "  Read 2 with adapter:                 146,855 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       147,492 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,521,763 bp\n",
      "  Read 1:    22,264,953 bp\n",
      "  Read 2:    22,256,810 bp\n",
      "Total written (filtered):     38,811,339 bp (87.2%)\n",
      "  Read 1:    19,490,481 bp\n",
      "  Read 2:    19,320,858 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 146049 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t9\t0.0\t1\t3 6\n",
      "18\t773\t0.0\t1\t69 704\n",
      "19\t144837\t0.0\t1\t120650 24187\n",
      "20\t412\t0.0\t1\t85 327\n",
      "21\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 146855 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t7\t576.1\t0\t7\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t1 3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t1334\t0.0\t1\t206 1128\n",
      "20\t145115\t0.0\t2\t142399 2716\n",
      "21\t377\t0.0\t2\t88 289\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1339C-TF-H-SS-M_S35_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1339C-TF-H-SS-M_S35_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1339C-TF-H-SS-M_S35_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1339C-TF-H-SS-M_S35_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.55 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,418\n",
      "  Read 1 with adapter:                 108,454 (99.1%)\n",
      "  Read 2 with adapter:                 108,796 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,418 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,021,580 bp\n",
      "  Read 1:    16,520,299 bp\n",
      "  Read 2:    16,501,281 bp\n",
      "Total written (filtered):     28,786,232 bp (87.2%)\n",
      "  Read 1:    14,460,109 bp\n",
      "  Read 2:    14,326,123 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108454 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t6\t0.0\t1\t3 3\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t7\t0.0\t1\t6 1\n",
      "17\t13\t0.0\t1\t6 7\n",
      "18\t614\t0.0\t1\t63 551\n",
      "19\t107517\t0.0\t1\t90975 16542\n",
      "20\t292\t0.0\t1\t50 242\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108796 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1709.7\t0\t2\n",
      "4\t2\t427.4\t0\t2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t18\t0.0\t1\t5 13\n",
      "19\t937\t0.0\t1\t157 780\n",
      "20\t107568\t0.0\t2\t105301 2267\n",
      "21\t265\t0.0\t2\t59 206\n",
      "22\t2\t0.0\t2\t2\n",
      "31\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1342D-TF-H-OA-M_S27_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1342D-TF-H-OA-M_S27_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1342D-TF-H-OA-M_S27_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1342D-TF-H-OA-M_S27_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.92 s (22 us/read; 2.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,352\n",
      "  Read 1 with adapter:                  85,646 (99.2%)\n",
      "  Read 2 with adapter:                  85,965 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        86,352 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,068,312 bp\n",
      "  Read 1:    13,037,756 bp\n",
      "  Read 2:    13,030,556 bp\n",
      "Total written (filtered):     22,722,542 bp (87.2%)\n",
      "  Read 1:    11,410,700 bp\n",
      "  Read 2:    11,311,842 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 85646 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t1 2\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t387\t0.0\t1\t55 332\n",
      "19\t85036\t0.0\t1\t72208 12828\n",
      "20\t206\t0.0\t1\t32 174\n",
      "28\t1\t0.0\t1\t1\n",
      "36\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 85965 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1349.2\t0\t1\n",
      "4\t5\t337.3\t0\t5\n",
      "13\t2\t0.0\t1\t2\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t4\t0.0\t1\t3 1\n",
      "19\t632\t0.0\t1\t123 509\n",
      "20\t85142\t0.0\t2\t83755 1387\n",
      "21\t174\t0.0\t2\t40 134\n",
      "26\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1345A-TF-H-CN-M_S5_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1345A-TF-H-CN-M_S5_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1345A-TF-H-CN-M_S5_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1345A-TF-H-CN-M_S5_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.92 s (19 us/read; 3.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            204,503\n",
      "  Read 1 with adapter:                 202,379 (99.0%)\n",
      "  Read 2 with adapter:                 203,522 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       204,503 (100.0%)\n",
      "\n",
      "Total basepairs processed:    61,725,538 bp\n",
      "  Read 1:    30,856,718 bp\n",
      "  Read 2:    30,868,820 bp\n",
      "Total written (filtered):     53,811,806 bp (87.2%)\n",
      "  Read 1:    27,012,054 bp\n",
      "  Read 2:    26,799,752 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 202379 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t3.1\t0\t1\n",
      "9\t1\t0.8\t0\t1\n",
      "10\t2\t0.2\t1\t2\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t6\t0.0\t1\t5 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t10\t0.0\t1\t6 4\n",
      "18\t870\t0.0\t1\t68 802\n",
      "19\t201001\t0.0\t1\t164515 36486\n",
      "20\t474\t0.0\t1\t92 382\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 203522 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t3195.4\t0\t3\n",
      "4\t9\t798.8\t0\t9\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t18\t0.0\t1\t7 11\n",
      "19\t1567\t0.0\t1\t246 1321\n",
      "20\t201450\t0.0\t2\t196498 4952\n",
      "21\t468\t0.0\t2\t97 371\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1295A-T1-H-PA-M_S18_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1295A-T1-H-PA-M_S18_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1295A-T1-H-PA-M_S18_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1295A-T1-H-PA-M_S18_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1301-T1-H-DL-M_S28_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1301-T1-H-DL-M_S28_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1301-T1-H-DL-M_S28_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1301-T1-H-DL-M_S28_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1304A-T1-H-PA-M_S17_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1304A-T1-H-PA-M_S17_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1304A-T1-H-PA-M_S17_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1304A-T1-H-PA-M_S17_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1305B-TF-D-OA-M_S13_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1305B-TF-D-OA-M_S13_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1305B-TF-D-OA-M_S13_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1305B-TF-D-OA-M_S13_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1319G-T1-H-SS-M_S6_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1319G-T1-H-SS-M_S6_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1319G-T1-H-SS-M_S6_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1319G-T1-H-SS-M_S6_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1322-T1-D-DL-M_S45_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1322-T1-D-DL-M_S45_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1322-T1-D-DL-M_S45_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1322-T1-D-DL-M_S45_50_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1327-T1-H-OA-M_S11_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1327-T1-H-OA-M_S11_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1327-T1-H-OA-M_S11_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1327-T1-H-OA-M_S11_52_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1327B-TF-H-OA-M_S37_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1327B-TF-H-OA-M_S37_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1327B-TF-H-OA-M_S37_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1327B-TF-H-OA-M_S37_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1338A-TF-H-OA-M_S33_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1338A-TF-H-OA-M_S33_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1338A-TF-H-OA-M_S33_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1338A-TF-H-OA-M_S33_53_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1339A-TF-D-SS-M_S8_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1339A-TF-D-SS-M_S8_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1339A-TF-D-SS-M_S8_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1339A-TF-D-SS-M_S8_54_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1339C-TF-H-SS-M_S35_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1339C-TF-H-SS-M_S35_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1339C-TF-H-SS-M_S35_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1339C-TF-H-SS-M_S35_55_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1342D-TF-H-OA-M_S27_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1342D-TF-H-OA-M_S27_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1342D-TF-H-OA-M_S27_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1342D-TF-H-OA-M_S27_56_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1345A-TF-H-CN-M_S5_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1345A-TF-H-CN-M_S5_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1345A-TF-H-CN-M_S5_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1345A-TF-H-CN-M_S5_57_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1345B-TF-D-CN-M_S3_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1345B-TF-D-CN-M_S3_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1345B-TF-D-CN-M_S3_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1345B-TF-D-CN-M_S3_58_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1345B-TF-D-CN-M_S3_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1345B-TF-D-CN-M_S3_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1345B-TF-D-CN-M_S3_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1345B-TF-D-CN-M_S3_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.34 s (20 us/read; 2.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,773\n",
      "  Read 1 with adapter:                 113,604 (99.0%)\n",
      "  Read 2 with adapter:                 114,223 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,643,868 bp\n",
      "  Read 1:    17,323,433 bp\n",
      "  Read 2:    17,320,435 bp\n",
      "Total written (filtered):     30,201,922 bp (87.2%)\n",
      "  Read 1:    15,165,207 bp\n",
      "  Read 2:    15,036,715 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113604 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t3\t0.0\t1\t1 2\n",
      "12\t2\t0.0\t1\t2\n",
      "15\t2\t0.0\t1\t0 2\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t515\t0.0\t1\t63 452\n",
      "19\t112757\t0.0\t1\t93140 19617\n",
      "20\t319\t0.0\t1\t62 257\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 114223 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1793.3\t0\t1\n",
      "4\t1\t448.3\t0\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t959\t0.0\t1\t121 838\n",
      "20\t112955\t0.0\t2\t110671 2284\n",
      "21\t293\t0.0\t2\t58 235\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1347C-T1-H-PS-M_S23_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1347C-T1-H-PS-M_S23_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1347C-T1-H-PS-M_S23_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1347C-T1-H-PS-M_S23_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.02 s (27 us/read; 2.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,475\n",
      "  Read 1 with adapter:                  74,042 (98.1%)\n",
      "  Read 2 with adapter:                  75,107 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,475 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,611,549 bp\n",
      "  Read 1:    11,307,593 bp\n",
      "  Read 2:    11,303,956 bp\n",
      "Total written (filtered):     19,703,174 bp (87.1%)\n",
      "  Read 1:     9,900,916 bp\n",
      "  Read 2:     9,802,258 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 74042 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "7\t1\t4.6\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t253\t0.0\t1\t43 210\n",
      "19\t73609\t0.0\t1\t49035 24574\n",
      "20\t172\t0.0\t1\t31 141\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 75107 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1179.3\t0\t1\n",
      "4\t3\t294.8\t0\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t532\t0.0\t1\t105 427\n",
      "20\t74393\t0.0\t2\t73236 1157\n",
      "21\t173\t0.0\t2\t36 137\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1356E-TF-D-MC-M_S15_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1356E-TF-D-MC-M_S15_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1356E-TF-D-MC-M_S15_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1356E-TF-D-MC-M_S15_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.09 s (26 us/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,497\n",
      "  Read 1 with adapter:                  79,396 (98.6%)\n",
      "  Read 2 with adapter:                  80,032 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        80,497 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,300,405 bp\n",
      "  Read 1:    12,151,962 bp\n",
      "  Read 2:    12,148,443 bp\n",
      "Total written (filtered):     21,192,257 bp (87.2%)\n",
      "  Read 1:    10,643,797 bp\n",
      "  Read 2:    10,548,460 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 79396 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t19.7\t0\t1\n",
      "9\t1\t0.3\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t496\t0.0\t1\t43 453\n",
      "19\t78696\t0.0\t1\t61616 17080\n",
      "20\t193\t0.0\t1\t40 153\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 80032 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t314.4\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t14\t0.0\t1\t11 3\n",
      "19\t755\t0.0\t1\t113 642\n",
      "20\t79043\t0.0\t2\t76893 2150\n",
      "21\t208\t0.0\t2\t57 151\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1356d-tf-h-mc-m_S40_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1356d-tf-h-mc-m_S40_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1356d-tf-h-mc-m_S40_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1356d-tf-h-mc-m_S40_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.26 s (262696 us/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                  1\n",
      "  Read 1 with adapter:                       1 (100.0%)\n",
      "  Read 2 with adapter:                       1 (100.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):             1 (100.0%)\n",
      "\n",
      "Total basepairs processed:           302 bp\n",
      "  Read 1:           151 bp\n",
      "  Read 2:           151 bp\n",
      "Total written (filtered):            263 bp (87.1%)\n",
      "  Read 1:           132 bp\n",
      "  Read 2:           131 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 1 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1360A-T1-H-MC-M_S38_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1360A-T1-H-MC-M_S38_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1360A-T1-H-MC-M_S38_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1360A-T1-H-MC-M_S38_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.34 s (22 us/read; 2.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            154,760\n",
      "  Read 1 with adapter:                 153,487 (99.2%)\n",
      "  Read 2 with adapter:                 154,062 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       154,760 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,681,297 bp\n",
      "  Read 1:    23,346,417 bp\n",
      "  Read 2:    23,334,880 bp\n",
      "Total written (filtered):     40,685,184 bp (87.2%)\n",
      "  Read 1:    20,430,589 bp\n",
      "  Read 2:    20,254,595 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 153487 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.6\t0\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t3\t0.0\t1\t0 3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t6\t0.0\t1\t4 2\n",
      "17\t10\t0.0\t1\t0 10\n",
      "18\t685\t0.0\t1\t92 593\n",
      "19\t152397\t0.0\t1\t129197 23200\n",
      "20\t376\t0.0\t1\t77 299\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 154062 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t604.5\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t2\t0.0\t1\t0 2\n",
      "18\t13\t0.0\t1\t6 7\n",
      "19\t1209\t0.0\t1\t204 1005\n",
      "20\t152449\t0.0\t2\t149715 2734\n",
      "21\t379\t0.0\t2\t85 294\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1363B-TF-D-MC-M_S21_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1363B-TF-D-MC-M_S21_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1363B-TF-D-MC-M_S21_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1363B-TF-D-MC-M_S21_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.00 s (23 us/read; 2.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             87,311\n",
      "  Read 1 with adapter:                  86,473 (99.0%)\n",
      "  Read 2 with adapter:                  86,947 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        87,311 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,364,421 bp\n",
      "  Read 1:    13,183,811 bp\n",
      "  Read 2:    13,180,610 bp\n",
      "Total written (filtered):     22,983,181 bp (87.2%)\n",
      "  Read 1:    11,541,003 bp\n",
      "  Read 2:    11,442,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 86473 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t4\t0.0\t1\t4\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t8\t0.0\t1\t2 6\n",
      "18\t332\t0.0\t1\t39 293\n",
      "19\t85876\t0.0\t1\t69153 16723\n",
      "20\t243\t0.0\t1\t47 196\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 86947 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t5\t341.1\t0\t5\n",
      "11\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t8\t0.0\t1\t6 2\n",
      "19\t603\t0.0\t1\t91 512\n",
      "20\t86120\t0.0\t2\t84896 1224\n",
      "21\t207\t0.0\t2\t35 172\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1363b-t1-h-mc-m_S41_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1363b-t1-h-mc-m_S41_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1363b-t1-h-mc-m_S41_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1363b-t1-h-mc-m_S41_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.18 s (24 us/read; 2.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,193\n",
      "  Read 1 with adapter:                  87,871 (97.4%)\n",
      "  Read 2 with adapter:                  88,261 (97.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        90,193 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,203,019 bp\n",
      "  Read 1:    13,609,835 bp\n",
      "  Read 2:    13,593,184 bp\n",
      "Total written (filtered):     23,769,240 bp (87.4%)\n",
      "  Read 1:    11,940,547 bp\n",
      "  Read 2:    11,828,693 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 87871 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.3\t0\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t8\t0.0\t1\t6 2\n",
      "18\t430\t0.0\t1\t26 404\n",
      "19\t87194\t0.0\t1\t71724 15470\n",
      "20\t231\t0.0\t1\t39 192\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 88261 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1409.3\t0\t3\n",
      "4\t3\t352.3\t0\t3\n",
      "8\t1\t1.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t7\t0.0\t1\t2 5\n",
      "19\t814\t0.0\t1\t120 694\n",
      "20\t87205\t0.0\t2\t85782 1423\n",
      "21\t222\t0.0\t2\t36 186\n",
      "32\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1365-T1-H-DL-M_S24_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1365-T1-H-DL-M_S24_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1365-T1-H-DL-M_S24_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1365-T1-H-DL-M_S24_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.59 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,858\n",
      "  Read 1 with adapter:                 114,976 (99.2%)\n",
      "  Read 2 with adapter:                 115,369 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,858 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,885,323 bp\n",
      "  Read 1:    17,483,687 bp\n",
      "  Read 2:    17,401,636 bp\n",
      "Total written (filtered):     30,394,420 bp (87.1%)\n",
      "  Read 1:    15,299,457 bp\n",
      "  Read 2:    15,094,963 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 114976 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "10\t2\t0.1\t1\t1 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t11\t0.0\t1\t6 5\n",
      "18\t505\t0.0\t1\t60 445\n",
      "19\t114150\t0.0\t1\t98021 16129\n",
      "20\t296\t0.0\t1\t53 243\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 115369 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t452.6\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t9\t0.0\t1\t5 4\n",
      "19\t870\t0.0\t1\t148 722\n",
      "20\t114193\t0.0\t2\t112475 1718\n",
      "21\t281\t0.0\t2\t67 214\n",
      "22\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1370B-TF-H-CN-M_S32_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1370B-TF-H-CN-M_S32_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1370B-TF-H-CN-M_S32_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1370B-TF-H-CN-M_S32_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.04 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,421\n",
      "  Read 1 with adapter:                  89,713 (99.2%)\n",
      "  Read 2 with adapter:                  90,039 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        90,421 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,279,178 bp\n",
      "  Read 1:    13,648,168 bp\n",
      "  Read 2:    13,631,010 bp\n",
      "Total written (filtered):     23,774,635 bp (87.2%)\n",
      "  Read 1:    11,943,892 bp\n",
      "  Read 2:    11,830,743 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 89713 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.3\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t385\t0.0\t1\t49 336\n",
      "19\t89095\t0.0\t1\t76229 12866\n",
      "20\t214\t0.0\t1\t41 173\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 90039 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t353.2\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t4\t0.0\t1\t3 1\n",
      "19\t686\t0.0\t1\t127 559\n",
      "20\t89129\t0.0\t2\t87677 1452\n",
      "21\t215\t0.0\t2\t46 169\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1370E-T1-H-CN-M_S30_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1370E-T1-H-CN-M_S30_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1370E-T1-H-CN-M_S30_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1370E-T1-H-CN-M_S30_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.92 s (23 us/read; 2.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,445\n",
      "  Read 1 with adapter:                  83,532 (98.9%)\n",
      "  Read 2 with adapter:                  84,035 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        84,445 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,485,357 bp\n",
      "  Read 1:    12,748,860 bp\n",
      "  Read 2:    12,736,497 bp\n",
      "Total written (filtered):     22,218,309 bp (87.2%)\n",
      "  Read 1:    11,161,997 bp\n",
      "  Read 2:    11,056,312 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 83532 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.3\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t7\t0.0\t1\t7\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t0 3\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t377\t0.0\t1\t55 322\n",
      "19\t82911\t0.0\t1\t66433 16478\n",
      "20\t226\t0.0\t1\t33 193\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 84035 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1319.5\t0\t1\n",
      "4\t2\t329.9\t0\t2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t649\t0.0\t1\t125 524\n",
      "20\t83163\t0.0\t2\t81699 1464\n",
      "21\t212\t0.0\t2\t52 160\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1379B-T1-H-MC-M_S44_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1379B-T1-H-MC-M_S44_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1379B-T1-H-MC-M_S44_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1379B-T1-H-MC-M_S44_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.60 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,874\n",
      "  Read 1 with adapter:                  58,452 (97.6%)\n",
      "  Read 2 with adapter:                  58,662 (98.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        59,874 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,890,876 bp\n",
      "  Read 1:     8,946,080 bp\n",
      "  Read 2:     8,944,796 bp\n",
      "Total written (filtered):     15,607,637 bp (87.2%)\n",
      "  Read 1:     7,835,693 bp\n",
      "  Read 2:     7,771,944 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 58452 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t6\t0.0\t1\t5 1\n",
      "18\t322\t0.0\t1\t29 293\n",
      "19\t57941\t0.0\t1\t47956 9985\n",
      "20\t173\t0.0\t1\t13 160\n",
      "25\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 58662 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t233.9\t0\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t4\t0.0\t1\t3 1\n",
      "19\t475\t0.0\t1\t64 411\n",
      "20\t58043\t0.0\t2\t56826 1217\n",
      "21\t136\t0.0\t2\t29 107\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1379C-TF-D-MC-M_S36_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1379C-TF-D-MC-M_S36_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1379C-TF-D-MC-M_S36_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1379C-TF-D-MC-M_S36_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.20 s (24 us/read; 2.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,771\n",
      "  Read 1 with adapter:                  91,809 (99.0%)\n",
      "  Read 2 with adapter:                  92,199 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        92,771 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,004,391 bp\n",
      "  Read 1:    14,006,672 bp\n",
      "  Read 2:    13,997,719 bp\n",
      "Total written (filtered):     24,417,056 bp (87.2%)\n",
      "  Read 1:    12,262,568 bp\n",
      "  Read 2:    12,154,488 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91809 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t6\t0.0\t1\t1 5\n",
      "18\t485\t0.0\t1\t50 435\n",
      "19\t91044\t0.0\t1\t75015 16029\n",
      "20\t266\t0.0\t1\t48 218\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 92199 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t362.4\t0\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t17\t0.0\t1\t8 9\n",
      "19\t876\t0.0\t1\t145 731\n",
      "20\t91069\t0.0\t2\t88960 2109\n",
      "21\t228\t0.0\t2\t44 184\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1610C-T1-H-OA-M_S26_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1610C-T1-H-OA-M_S26_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1610C-T1-H-OA-M_S26_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1610C-T1-H-OA-M_S26_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.26 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,665\n",
      "  Read 1 with adapter:                 105,903 (99.3%)\n",
      "  Read 2 with adapter:                 106,186 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,665 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,170,524 bp\n",
      "  Read 1:    16,088,151 bp\n",
      "  Read 2:    16,082,373 bp\n",
      "Total written (filtered):     28,035,542 bp (87.1%)\n",
      "  Read 1:    14,076,265 bp\n",
      "  Read 2:    13,959,277 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 105903 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t4\t0.0\t1\t3 1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t7\t0.0\t1\t2 5\n",
      "18\t456\t0.0\t1\t73 383\n",
      "19\t105175\t0.0\t1\t90661 14514\n",
      "20\t254\t0.0\t1\t42 212\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 106186 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t4\t416.7\t0\t4\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t8\t0.0\t1\t2 6\n",
      "19\t759\t0.0\t1\t124 635\n",
      "20\t105165\t0.0\t2\t103493 1672\n",
      "21\t244\t0.0\t2\t58 186\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1613B-TF-H-SS-M_S10_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1613B-TF-H-SS-M_S10_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1613B-TF-H-SS-M_S10_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1613B-TF-H-SS-M_S10_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.29 s (20 us/read; 3.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            167,813\n",
      "  Read 1 with adapter:                 165,656 (98.7%)\n",
      "  Read 2 with adapter:                 166,924 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       167,813 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,663,945 bp\n",
      "  Read 1:    25,337,084 bp\n",
      "  Read 2:    25,326,861 bp\n",
      "Total written (filtered):     44,180,118 bp (87.2%)\n",
      "  Read 1:    22,190,336 bp\n",
      "  Read 2:    21,989,782 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 165656 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.6\t0\t3\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t8\t0.0\t1\t7 1\n",
      "13\t5\t0.0\t1\t3 2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t6\t0.0\t1\t2 4\n",
      "17\t14\t0.0\t1\t3 11\n",
      "18\t1013\t0.0\t1\t87 926\n",
      "19\t164139\t0.0\t1\t132081 32058\n",
      "20\t463\t0.0\t1\t96 367\n",
      "24\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 166924 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2622.1\t0\t2\n",
      "4\t7\t655.5\t0\t7\n",
      "11\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t6\t0.0\t1\t1 5\n",
      "18\t18\t0.0\t1\t7 11\n",
      "19\t1627\t0.0\t1\t234 1393\n",
      "20\t164795\t0.0\t2\t160224 4571\n",
      "21\t463\t0.0\t2\t97 366\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1654B-TF-H-SS-M_S25_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1654B-TF-H-SS-M_S25_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1654B-TF-H-SS-M_S25_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1654B-TF-H-SS-M_S25_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.14 s (26 us/read; 2.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,206\n",
      "  Read 1 with adapter:                  80,501 (99.1%)\n",
      "  Read 2 with adapter:                  80,851 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        81,206 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,448,684 bp\n",
      "  Read 1:    12,230,573 bp\n",
      "  Read 2:    12,218,111 bp\n",
      "Total written (filtered):     21,302,792 bp (87.1%)\n",
      "  Read 1:    10,701,296 bp\n",
      "  Read 2:    10,601,496 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 80501 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t5\t0.0\t1\t4 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t395\t0.0\t1\t47 348\n",
      "19\t79894\t0.0\t1\t66830 13064\n",
      "20\t198\t0.0\t1\t41 157\n",
      "42\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 80851 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t597\t0.0\t1\t90 507\n",
      "20\t80039\t0.0\t2\t78752 1287\n",
      "21\t208\t0.0\t2\t43 165\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1347C-T1-H-PS-M_S23_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1347C-T1-H-PS-M_S23_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1347C-T1-H-PS-M_S23_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1347C-T1-H-PS-M_S23_59_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1356E-TF-D-MC-M_S15_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1356E-TF-D-MC-M_S15_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1356E-TF-D-MC-M_S15_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1356E-TF-D-MC-M_S15_61_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1356d-tf-h-mc-m_S40_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1356d-tf-h-mc-m_S40_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1356d-tf-h-mc-m_S40_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1356d-tf-h-mc-m_S40_60_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1360A-T1-H-MC-M_S38_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1360A-T1-H-MC-M_S38_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1360A-T1-H-MC-M_S38_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1360A-T1-H-MC-M_S38_62_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1363B-TF-D-MC-M_S21_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1363B-TF-D-MC-M_S21_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1363B-TF-D-MC-M_S21_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1363B-TF-D-MC-M_S21_64_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1363b-t1-h-mc-m_S41_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1363b-t1-h-mc-m_S41_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1363b-t1-h-mc-m_S41_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1363b-t1-h-mc-m_S41_63_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1365-T1-H-DL-M_S24_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1365-T1-H-DL-M_S24_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1365-T1-H-DL-M_S24_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1365-T1-H-DL-M_S24_65_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1370B-TF-H-CN-M_S32_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1370B-TF-H-CN-M_S32_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1370B-TF-H-CN-M_S32_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1370B-TF-H-CN-M_S32_66_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1370E-T1-H-CN-M_S30_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1370E-T1-H-CN-M_S30_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1370E-T1-H-CN-M_S30_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1370E-T1-H-CN-M_S30_67_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1379B-T1-H-MC-M_S44_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1379B-T1-H-MC-M_S44_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1379B-T1-H-MC-M_S44_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1379B-T1-H-MC-M_S44_68_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1379C-TF-D-MC-M_S36_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1379C-TF-D-MC-M_S36_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1379C-TF-D-MC-M_S36_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1379C-TF-D-MC-M_S36_69_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1610C-T1-H-OA-M_S26_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1610C-T1-H-OA-M_S26_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1610C-T1-H-OA-M_S26_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1610C-T1-H-OA-M_S26_70_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1613B-TF-H-SS-M_S10_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1613B-TF-H-SS-M_S10_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1613B-TF-H-SS-M_S10_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1613B-TF-H-SS-M_S10_71_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1654B-TF-H-SS-M_S25_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1654B-TF-H-SS-M_S25_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1654B-TF-H-SS-M_S25_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1654B-TF-H-SS-M_S25_72_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1655C-T1-H-CN-M_S29_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1655C-T1-H-CN-M_S29_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1655C-T1-H-CN-M_S29_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1655C-T1-H-CN-M_S29_73_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1655C-T1-H-CN-M_S29_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1655C-T1-H-CN-M_S29_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1655C-T1-H-CN-M_S29_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1655C-T1-H-CN-M_S29_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.34 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            147,446\n",
      "  Read 1 with adapter:                 145,787 (98.9%)\n",
      "  Read 2 with adapter:                 146,676 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       147,446 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,410,103 bp\n",
      "  Read 1:    22,201,092 bp\n",
      "  Read 2:    22,209,011 bp\n",
      "Total written (filtered):     38,708,052 bp (87.2%)\n",
      "  Read 1:    19,431,575 bp\n",
      "  Read 2:    19,276,477 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 145787 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.6\t0\t3\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t7\t0.0\t1\t6 1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t0 3\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t8\t0.0\t1\t3 5\n",
      "18\t653\t0.0\t1\t90 563\n",
      "19\t144752\t0.0\t1\t114543 30209\n",
      "20\t355\t0.0\t1\t80 275\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 146676 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t2303.8\t0\t4\n",
      "4\t2\t576.0\t0\t2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t25\t0.0\t1\t13 12\n",
      "19\t1132\t0.0\t1\t193 939\n",
      "20\t145168\t0.0\t2\t142726 2442\n",
      "21\t337\t0.0\t2\t69 268\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1659E-TF-D-SS-M_S9_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1659E-TF-D-SS-M_S9_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1659E-TF-D-SS-M_S9_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1659E-TF-D-SS-M_S9_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.73 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,501\n",
      "  Read 1 with adapter:                 119,254 (99.0%)\n",
      "  Read 2 with adapter:                 119,855 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       120,501 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,369,874 bp\n",
      "  Read 1:    18,187,348 bp\n",
      "  Read 2:    18,182,526 bp\n",
      "Total written (filtered):     31,708,502 bp (87.2%)\n",
      "  Read 1:    15,921,967 bp\n",
      "  Read 2:    15,786,535 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 119254 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "11\t3\t0.0\t1\t0 3\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t638\t0.0\t1\t52 586\n",
      "19\t118285\t0.0\t1\t98282 20003\n",
      "20\t307\t0.0\t1\t58 249\n",
      "21\t2\t0.0\t1\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 119855 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1882.8\t0\t2\n",
      "4\t5\t470.7\t0\t5\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t0 2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t13\t0.0\t1\t8 5\n",
      "19\t1198\t0.0\t1\t171 1027\n",
      "20\t118316\t0.0\t2\t115346 2970\n",
      "21\t306\t0.0\t2\t52 254\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1661B-TF-H-CN-M_S31_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1661B-TF-H-CN-M_S31_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1661B-TF-H-CN-M_S31_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1661B-TF-H-CN-M_S31_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.01 s (22 us/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,690\n",
      "  Read 1 with adapter:                  91,823 (99.1%)\n",
      "  Read 2 with adapter:                  92,283 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        92,690 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,954,136 bp\n",
      "  Read 1:    13,978,234 bp\n",
      "  Read 2:    13,975,902 bp\n",
      "Total written (filtered):     24,364,584 bp (87.2%)\n",
      "  Read 1:    12,233,809 bp\n",
      "  Read 2:    12,130,775 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91823 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t11\t0.0\t1\t5 6\n",
      "18\t348\t0.0\t1\t43 305\n",
      "19\t91228\t0.0\t1\t75991 15237\n",
      "20\t224\t0.0\t1\t37 187\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 92283 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t362.1\t0\t1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t11\t0.0\t1\t8 3\n",
      "19\t703\t0.0\t1\t143 560\n",
      "20\t91344\t0.0\t2\t89755 1589\n",
      "21\t220\t0.0\t2\t49 171\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1661C-T1-H-CN-M_S2_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1661C-T1-H-CN-M_S2_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1661C-T1-H-CN-M_S2_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1661C-T1-H-CN-M_S2_76_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.69 s (23 us/read; 2.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,337\n",
      "  Read 1 with adapter:                  72,698 (99.1%)\n",
      "  Read 2 with adapter:                  73,033 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        73,337 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,831,482 bp\n",
      "  Read 1:    10,916,995 bp\n",
      "  Read 2:    10,914,487 bp\n",
      "Total written (filtered):     18,990,182 bp (87.0%)\n",
      "  Read 1:     9,535,866 bp\n",
      "  Read 2:     9,454,316 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 72698 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t289\t0.0\t1\t29 260\n",
      "19\t72237\t0.0\t1\t59567 12670\n",
      "20\t166\t0.0\t1\t33 133\n",
      "34\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 73033 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1145.9\t0\t1\n",
      "4\t4\t286.5\t0\t4\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t5\t0.0\t1\t4 1\n",
      "19\t560\t0.0\t1\t82 478\n",
      "20\t72268\t0.0\t2\t70979 1289\n",
      "21\t190\t0.0\t2\t44 146\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1669C-TF-H-OA-M_S22_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1669C-TF-H-OA-M_S22_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1669C-TF-H-OA-M_S22_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1669C-TF-H-OA-M_S22_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.98 s (22 us/read; 2.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            137,171\n",
      "  Read 1 with adapter:                 135,906 (99.1%)\n",
      "  Read 2 with adapter:                 136,655 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       137,171 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,402,361 bp\n",
      "  Read 1:    20,711,090 bp\n",
      "  Read 2:    20,691,271 bp\n",
      "Total written (filtered):     36,087,969 bp (87.2%)\n",
      "  Read 1:    18,129,138 bp\n",
      "  Read 2:    17,958,831 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 135906 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t0 4\n",
      "17\t8\t0.0\t1\t1 7\n",
      "18\t500\t0.0\t1\t74 426\n",
      "19\t135067\t0.0\t1\t109060 26007\n",
      "20\t317\t0.0\t1\t68 249\n",
      "22\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 136655 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2143.3\t0\t1\n",
      "4\t6\t535.8\t0\t6\n",
      "13\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t8\t0.0\t1\t4 4\n",
      "19\t792\t0.0\t1\t124 668\n",
      "20\t135558\t0.0\t2\t133523 2035\n",
      "21\t285\t0.0\t2\t55 230\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/18-DNA-Control_S19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/18-DNA-Control_S19_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/18-DNA-Control_S19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/18-DNA-Control_S19_82_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.30 s (730 us/read; 0.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                409\n",
      "  Read 1 with adapter:                     402 (98.3%)\n",
      "  Read 2 with adapter:                     407 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):           409 (100.0%)\n",
      "\n",
      "Total basepairs processed:       122,426 bp\n",
      "  Read 1:        61,220 bp\n",
      "  Read 2:        61,206 bp\n",
      "Total written (filtered):        106,651 bp (87.1%)\n",
      "  Read 1:        53,584 bp\n",
      "  Read 2:        53,067 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 402 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 2\n",
      "19\t400\t0.0\t1\t325 75\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 407 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t1\n",
      "20\t406\t0.0\t2\t390 16\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1831A-TF-D-OA-M_S12_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1831A-TF-D-OA-M_S12_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1831A-TF-D-OA-M_S12_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1831A-TF-D-OA-M_S12_78_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.50 s (22 us/read; 2.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,921\n",
      "  Read 1 with adapter:                 112,747 (99.0%)\n",
      "  Read 2 with adapter:                 113,451 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,921 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,325,254 bp\n",
      "  Read 1:    17,164,787 bp\n",
      "  Read 2:    17,160,467 bp\n",
      "Total written (filtered):     29,915,156 bp (87.2%)\n",
      "  Read 1:    15,022,934 bp\n",
      "  Read 2:    14,892,222 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 112747 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t3\t0.1\t1\t1 2\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t8\t0.0\t1\t1 7\n",
      "18\t526\t0.0\t1\t60 466\n",
      "19\t111891\t0.0\t1\t91114 20777\n",
      "20\t303\t0.0\t1\t64 239\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 113451 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1780.0\t0\t3\n",
      "4\t5\t445.0\t0\t5\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t6\t0.0\t1\t1 5\n",
      "19\t909\t0.0\t1\t150 759\n",
      "20\t112223\t0.0\t2\t110408 1815\n",
      "21\t299\t0.0\t2\t67 232\n",
      "24\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1835B-TF-H-OA-M-Repeat-LibK_S1_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1835B-TF-H-OA-M-Repeat-LibK_S1_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1835B-TF-H-OA-M-Repeat-LibK_S1_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1835B-TF-H-OA-M-Repeat-LibK_S1_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.40 s (24 us/read; 2.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,173\n",
      "  Read 1 with adapter:                  57,695 (99.2%)\n",
      "  Read 2 with adapter:                  57,890 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        58,173 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,557,249 bp\n",
      "  Read 1:     8,779,620 bp\n",
      "  Read 2:     8,777,629 bp\n",
      "Total written (filtered):     15,303,938 bp (87.2%)\n",
      "  Read 1:     7,683,636 bp\n",
      "  Read 2:     7,620,302 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 57695 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t0.9\t0\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t0 4\n",
      "18\t297\t0.0\t1\t50 247\n",
      "19\t57244\t0.0\t1\t48405 8839\n",
      "20\t141\t0.0\t1\t29 112\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 57890 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t909.0\t0\t1\n",
      "4\t3\t227.2\t0\t3\n",
      "10\t1\t0.1\t1\t0 1\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t518\t0.0\t1\t79 439\n",
      "20\t57220\t0.0\t2\t56290 930\n",
      "21\t138\t0.0\t2\t35 103\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1863-TF-D-CN-M_S4_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1863-TF-D-CN-M_S4_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1863-TF-D-CN-M_S4_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1863-TF-D-CN-M_S4_80_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.17 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,101\n",
      "  Read 1 with adapter:                  34,784 (99.1%)\n",
      "  Read 2 with adapter:                  34,952 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,101 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,595,121 bp\n",
      "  Read 1:     5,298,695 bp\n",
      "  Read 2:     5,296,426 bp\n",
      "Total written (filtered):      9,235,529 bp (87.2%)\n",
      "  Read 1:     4,637,912 bp\n",
      "  Read 2:     4,597,617 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 34784 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t175\t0.0\t1\t19 156\n",
      "19\t34519\t0.0\t1\t28799 5720\n",
      "20\t83\t0.0\t1\t25 58\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 34952 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t137.1\t0\t1\n",
      "18\t4\t0.0\t1\t0 4\n",
      "19\t292\t0.0\t1\t47 245\n",
      "20\t34570\t0.0\t2\t34019 551\n",
      "21\t85\t0.0\t2\t17 68\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1870D-TF-D-PS-M_S34_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1870D-TF-D-PS-M_S34_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1870D-TF-D-PS-M_S34_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1870D-TF-D-PS-M_S34_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.14 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,071\n",
      "  Read 1 with adapter:                  91,173 (99.0%)\n",
      "  Read 2 with adapter:                  91,514 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        92,071 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,799,540 bp\n",
      "  Read 1:    13,902,512 bp\n",
      "  Read 2:    13,897,028 bp\n",
      "Total written (filtered):     24,238,093 bp (87.2%)\n",
      "  Read 1:    12,170,553 bp\n",
      "  Read 2:    12,067,540 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91173 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t4\t0.0\t1\t2 2\n",
      "16\t6\t0.0\t1\t4 2\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t492\t0.0\t1\t51 441\n",
      "19\t90405\t0.0\t1\t76328 14077\n",
      "20\t256\t0.0\t1\t36 220\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 91514 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1438.6\t0\t3\n",
      "4\t2\t359.7\t0\t2\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t6\t0.0\t1\t1 5\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t9\t0.0\t1\t7 2\n",
      "19\t813\t0.0\t1\t123 690\n",
      "20\t90446\t0.0\t2\t88546 1900\n",
      "21\t221\t0.0\t2\t37 184\n",
      "37\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/19-DNA-Control_S20_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/19-DNA-Control_S20_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/19-DNA-Control_S20_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/19-DNA-Control_S20_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.60 s (29 us/read; 2.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,117\n",
      "  Read 1 with adapter:                  53,573 (99.0%)\n",
      "  Read 2 with adapter:                  53,754 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,117 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,309,177 bp\n",
      "  Read 1:     8,157,468 bp\n",
      "  Read 2:     8,151,709 bp\n",
      "Total written (filtered):     14,216,732 bp (87.2%)\n",
      "  Read 1:     7,139,768 bp\n",
      "  Read 2:     7,076,964 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 53573 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t279\t0.0\t1\t19 260\n",
      "19\t53141\t0.0\t1\t45917 7224\n",
      "20\t142\t0.0\t1\t29 113\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 53754 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t845.6\t0\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "18\t4\t0.0\t1\t2 2\n",
      "19\t432\t0.0\t1\t59 373\n",
      "20\t53189\t0.0\t2\t51950 1239\n",
      "21\t126\t0.0\t2\t25 101\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1Flat-DD-OA-M_S16_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1Flat-DD-OA-M_S16_84_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1Flat-DD-OA-M_S16_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1Flat-DD-OA-M_S16_84_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.16 s (34 us/read; 1.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,369\n",
      "  Read 1 with adapter:                  34,038 (99.0%)\n",
      "  Read 2 with adapter:                  34,238 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,369 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,371,486 bp\n",
      "  Read 1:     5,187,137 bp\n",
      "  Read 2:     5,184,349 bp\n",
      "Total written (filtered):      9,040,342 bp (87.2%)\n",
      "  Read 1:     4,540,498 bp\n",
      "  Read 2:     4,499,844 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 34038 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t153\t0.0\t1\t19 134\n",
      "19\t33804\t0.0\t1\t28391 5413\n",
      "20\t76\t0.0\t1\t12 64\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 34238 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t134.3\t0\t3\n",
      "18\t3\t0.0\t1\t2 1\n",
      "19\t280\t0.0\t1\t51 229\n",
      "20\t33873\t0.0\t2\t33218 655\n",
      "21\t79\t0.0\t2\t22 57\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1Flat-DD-PS-M_S14_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1Flat-DD-PS-M_S14_85_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1Flat-DD-PS-M_S14_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1Flat-DD-PS-M_S14_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.57 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,173\n",
      "  Read 1 with adapter:                 108,510 (98.5%)\n",
      "  Read 2 with adapter:                 109,576 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       110,173 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,220,984 bp\n",
      "  Read 1:    16,612,603 bp\n",
      "  Read 2:    16,608,381 bp\n",
      "Total written (filtered):     28,969,077 bp (87.2%)\n",
      "  Read 1:    14,551,263 bp\n",
      "  Read 2:    14,417,814 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108510 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t107.6\t0\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t2\t0.0\t1\t2\n",
      "16\t7\t0.0\t1\t1 6\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t514\t0.0\t1\t54 460\n",
      "19\t107719\t0.0\t1\t81323 26396\n",
      "20\t259\t0.0\t1\t43 216\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 109576 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1721.5\t0\t1\n",
      "4\t2\t430.4\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t20\t0.0\t1\t9 11\n",
      "19\t1062\t0.0\t1\t149 913\n",
      "20\t108221\t0.0\t2\t105518 2703\n",
      "21\t259\t0.0\t2\t46 213\n",
      "23\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/2199B-T1-H-SS-M_S7_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/2199B-T1-H-SS-M_S7_86_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/2199B-T1-H-SS-M_S7_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/2199B-T1-H-SS-M_S7_86_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.31 s (25 us/read; 2.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,530\n",
      "  Read 1 with adapter:                  92,684 (99.1%)\n",
      "  Read 2 with adapter:                  93,087 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        93,530 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,122,930 bp\n",
      "  Read 1:    14,066,842 bp\n",
      "  Read 2:    14,056,088 bp\n",
      "Total written (filtered):     24,501,067 bp (87.1%)\n",
      "  Read 1:    12,306,096 bp\n",
      "  Read 2:    12,194,971 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 92684 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t7\t0.0\t1\t5 2\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t6\t0.0\t1\t2 4\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t399\t0.0\t1\t39 360\n",
      "19\t92023\t0.0\t1\t75805 16218\n",
      "20\t239\t0.0\t1\t47 192\n",
      "23\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 93087 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1461.4\t0\t3\n",
      "4\t1\t365.4\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t3\t0.0\t1\t1 2\n",
      "19\t770\t0.0\t1\t122 648\n",
      "20\t92052\t0.0\t2\t90360 1692\n",
      "21\t250\t0.0\t2\t56 194\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/7Flat-DD-PS-M_S42_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/7Flat-DD-PS-M_S42_87_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/7Flat-DD-PS-M_S42_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/7Flat-DD-PS-M_S42_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.19 s (26 us/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,327\n",
      "  Read 1 with adapter:                  82,249 (97.5%)\n",
      "  Read 2 with adapter:                  82,590 (97.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        84,327 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,417,853 bp\n",
      "  Read 1:    12,709,675 bp\n",
      "  Read 2:    12,708,178 bp\n",
      "Total written (filtered):     22,204,059 bp (87.4%)\n",
      "  Read 1:    11,147,135 bp\n",
      "  Read 2:    11,056,924 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 82249 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t328\t0.0\t1\t37 291\n",
      "19\t81678\t0.0\t1\t68928 12750\n",
      "20\t223\t0.0\t1\t48 175\n",
      "21\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 82590 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t4\t329.4\t0\t4\n",
      "14\t4\t0.0\t1\t0 4\n",
      "18\t5\t0.0\t1\t2 3\n",
      "19\t644\t0.0\t1\t98 546\n",
      "20\t81737\t0.0\t2\t80254 1483\n",
      "21\t196\t0.0\t2\t36 160\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1659E-TF-D-SS-M_S9_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1659E-TF-D-SS-M_S9_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1659E-TF-D-SS-M_S9_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1659E-TF-D-SS-M_S9_74_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1661B-TF-H-CN-M_S31_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1661B-TF-H-CN-M_S31_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1661B-TF-H-CN-M_S31_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1661B-TF-H-CN-M_S31_75_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1661C-T1-H-CN-M_S2_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1661C-T1-H-CN-M_S2_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1661C-T1-H-CN-M_S2_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1661C-T1-H-CN-M_S2_76_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1669C-TF-H-OA-M_S22_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1669C-TF-H-OA-M_S22_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1669C-TF-H-OA-M_S22_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1669C-TF-H-OA-M_S22_77_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/18-DNA-Control_S19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/18-DNA-Control_S19_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/18-DNA-Control_S19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/18-DNA-Control_S19_82_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1831A-TF-D-OA-M_S12_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1831A-TF-D-OA-M_S12_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1831A-TF-D-OA-M_S12_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1831A-TF-D-OA-M_S12_78_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1835B-TF-H-OA-M-Repeat-LibK_S1_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1835B-TF-H-OA-M-Repeat-LibK_S1_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1835B-TF-H-OA-M-Repeat-LibK_S1_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1835B-TF-H-OA-M-Repeat-LibK_S1_79_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1863-TF-D-CN-M_S4_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1863-TF-D-CN-M_S4_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1863-TF-D-CN-M_S4_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1863-TF-D-CN-M_S4_80_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1870D-TF-D-PS-M_S34_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1870D-TF-D-PS-M_S34_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1870D-TF-D-PS-M_S34_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1870D-TF-D-PS-M_S34_81_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/19-DNA-Control_S20_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/19-DNA-Control_S20_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/19-DNA-Control_S20_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/19-DNA-Control_S20_83_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1Flat-DD-OA-M_S16_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1Flat-DD-OA-M_S16_84_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1Flat-DD-OA-M_S16_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1Flat-DD-OA-M_S16_84_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1Flat-DD-PS-M_S14_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/1Flat-DD-PS-M_S14_85_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1Flat-DD-PS-M_S14_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/1Flat-DD-PS-M_S14_85_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/2199B-T1-H-SS-M_S7_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/2199B-T1-H-SS-M_S7_86_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/2199B-T1-H-SS-M_S7_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/2199B-T1-H-SS-M_S7_86_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/7Flat-DD-PS-M_S42_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/7Flat-DD-PS-M_S42_87_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/7Flat-DD-PS-M_S42_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/7Flat-DD-PS-M_S42_87_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/Blank-Field_S43_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/Blank-Field_S43_88_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/Blank-Field_S43_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/Blank-Field_S43_88_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/Blank-Field_S43_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/Blank-Field_S43_88_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/Blank-Field_S43_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/Blank-Field_S43_88_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.75 s (49 us/read; 1.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,173\n",
      "  Read 1 with adapter:                  14,854 (97.9%)\n",
      "  Read 2 with adapter:                  14,872 (98.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        15,173 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,556,272 bp\n",
      "  Read 1:     2,278,142 bp\n",
      "  Read 2:     2,278,130 bp\n",
      "Total written (filtered):      3,976,751 bp (87.3%)\n",
      "  Read 1:     1,995,976 bp\n",
      "  Read 2:     1,980,775 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14854 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "18\t85\t0.0\t1\t7 78\n",
      "19\t14735\t0.0\t1\t12602 2133\n",
      "20\t33\t0.0\t1\t8 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14872 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t4\t0.0\t1\t1 3\n",
      "19\t121\t0.0\t1\t19 102\n",
      "20\t14703\t0.0\t2\t14386 317\n",
      "21\t44\t0.0\t2\t5 39\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/mock-2-LibK_S39_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/mock-2-LibK_S39_89_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/mock-2-LibK_S39_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/mock-2-LibK_S39_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.33 s (23 us/read; 2.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,524\n",
      "  Read 1 with adapter:                  99,601 (99.1%)\n",
      "  Read 2 with adapter:                  99,901 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,524 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,333,038 bp\n",
      "  Read 1:    15,178,512 bp\n",
      "  Read 2:    15,154,526 bp\n",
      "Total written (filtered):     26,443,919 bp (87.2%)\n",
      "  Read 1:    13,286,534 bp\n",
      "  Read 2:    13,157,385 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99601 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.5\t0\t1\n",
      "9\t3\t0.4\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t10\t0.0\t1\t4 6\n",
      "18\t586\t0.0\t1\t51 535\n",
      "19\t98714\t0.0\t1\t83430 15284\n",
      "20\t272\t0.0\t1\t49 223\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 99901 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1570.7\t0\t2\n",
      "4\t1\t392.7\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t2\t0.0\t1\t2\n",
      "17\t6\t0.0\t1\t5 1\n",
      "18\t25\t0.0\t1\t15 10\n",
      "19\t999\t0.0\t1\t137 862\n",
      "20\t98609\t0.0\t2\t96550 2059\n",
      "21\t252\t0.0\t2\t56 196\n",
      "42\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/mock-2-LibK_S39_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-pyudqy96/mock-2-LibK_S39_89_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/mock-2-LibK_S39_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-bomzli9l/629dd3b6-53bc-4979-bbb5-1b77237cc3fd/data/mock-2-LibK_S39_89_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_LibK.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1226C-T1-H-MC-M_S13_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1226C-T1-H-MC-M_S13_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1226C-T1-H-MC-M_S13_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1226C-T1-H-MC-M_S13_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.78 s (29 us/read; 2.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,853\n",
      "  Read 1 with adapter:                  56,873 (91.9%)\n",
      "  Read 2 with adapter:                  56,427 (91.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        61,853 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,651,351 bp\n",
      "  Read 1:     9,328,218 bp\n",
      "  Read 2:     9,323,133 bp\n",
      "Total written (filtered):     16,443,486 bp (88.2%)\n",
      "  Read 1:     8,248,252 bp\n",
      "  Read 2:     8,195,234 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 56873 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t760\t0.0\t1\t29 731\n",
      "19\t55937\t0.0\t1\t45998 9939\n",
      "20\t170\t0.0\t1\t33 137\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 56427 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t966.5\t0\t19\n",
      "4\t1\t241.6\t0\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t1 1\n",
      "19\t404\t0.0\t1\t56 348\n",
      "20\t55873\t0.0\t2\t54298 1575\n",
      "21\t125\t0.0\t2\t25 100\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1295E-TF-H-PA-M_S20_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1295E-TF-H-PA-M_S20_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1295E-TF-H-PA-M_S20_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1295E-TF-H-PA-M_S20_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.74 s (23 us/read; 2.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,917\n",
      "  Read 1 with adapter:                  69,762 (91.9%)\n",
      "  Read 2 with adapter:                  69,073 (91.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,917 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,867,907 bp\n",
      "  Read 1:    11,434,680 bp\n",
      "  Read 2:    11,433,227 bp\n",
      "Total written (filtered):     20,161,875 bp (88.2%)\n",
      "  Read 1:    10,109,429 bp\n",
      "  Read 2:    10,052,446 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 69762 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t296.6\t0\t1\n",
      "9\t1\t0.3\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t294\t0.0\t1\t31 263\n",
      "19\t69303\t0.0\t1\t57433 11870\n",
      "20\t151\t0.0\t1\t42 109\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 69073 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t1186.2\t0\t17\n",
      "4\t1\t296.6\t0\t1\n",
      "18\t3\t0.0\t1\t2 1\n",
      "19\t519\t0.0\t1\t53 466\n",
      "20\t68382\t0.0\t2\t66786 1596\n",
      "21\t151\t0.0\t2\t32 119\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1301-TF-D-DL-M_S35_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1301-TF-D-DL-M_S35_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1301-TF-D-DL-M_S35_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1301-TF-D-DL-M_S35_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.33 s (27 us/read; 2.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,939\n",
      "  Read 1 with adapter:                  45,724 (91.6%)\n",
      "  Read 2 with adapter:                  45,360 (90.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,939 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,071,647 bp\n",
      "  Read 1:     7,535,456 bp\n",
      "  Read 2:     7,536,191 bp\n",
      "Total written (filtered):     13,296,387 bp (88.2%)\n",
      "  Read 1:     6,666,901 bp\n",
      "  Read 2:     6,629,486 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 45724 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t264\t0.0\t1\t26 238\n",
      "19\t45325\t0.0\t1\t35662 9663\n",
      "20\t122\t0.0\t1\t25 97\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 45360 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t780.3\t0\t12\n",
      "4\t1\t195.1\t0\t1\n",
      "18\t3\t0.0\t1\t1 2\n",
      "19\t393\t0.0\t1\t43 350\n",
      "20\t44827\t0.0\t2\t43570 1257\n",
      "21\t124\t0.0\t2\t30 94\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1301-TF-H-DL-M_S31_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1301-TF-H-DL-M_S31_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1301-TF-H-DL-M_S31_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1301-TF-H-DL-M_S31_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.14 s (21 us/read; 2.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,084\n",
      "  Read 1 with adapter:                  92,558 (92.5%)\n",
      "  Read 2 with adapter:                  91,715 (91.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,084 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,170,576 bp\n",
      "  Read 1:    15,086,167 bp\n",
      "  Read 2:    15,084,409 bp\n",
      "Total written (filtered):     26,579,581 bp (88.1%)\n",
      "  Read 1:    13,328,500 bp\n",
      "  Read 2:    13,251,081 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 92558 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t391.0\t0\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t1105\t0.0\t1\t50 1055\n",
      "19\t91213\t0.0\t1\t77462 13751\n",
      "20\t229\t0.0\t1\t45 184\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 91715 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t1563.8\t0\t22\n",
      "4\t3\t391.0\t0\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t6\t0.0\t1\t2 4\n",
      "19\t707\t0.0\t1\t94 613\n",
      "20\t90777\t0.0\t2\t88498 2279\n",
      "21\t192\t0.0\t2\t51 141\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1302-T1-D-DL-M_S36_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1302-T1-D-DL-M_S36_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1302-T1-D-DL-M_S36_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1302-T1-D-DL-M_S36_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.74 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,698\n",
      "  Read 1 with adapter:                 104,692 (91.3%)\n",
      "  Read 2 with adapter:                 104,611 (91.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,698 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,564,608 bp\n",
      "  Read 1:    17,304,919 bp\n",
      "  Read 2:    17,259,689 bp\n",
      "Total written (filtered):     30,485,586 bp (88.2%)\n",
      "  Read 1:    15,316,783 bp\n",
      "  Read 2:    15,168,803 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 104692 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t448.0\t0\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t1241\t0.0\t1\t49 1192\n",
      "19\t103153\t0.0\t1\t79469 23684\n",
      "20\t285\t0.0\t1\t41 244\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 104611 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t1792.2\t0\t33\n",
      "4\t5\t448.0\t0\t5\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "17\t7\t0.0\t1\t5 2\n",
      "18\t16\t0.0\t1\t7 9\n",
      "19\t826\t0.0\t1\t101 725\n",
      "20\t103498\t0.0\t2\t100950 2548\n",
      "21\t221\t0.0\t2\t61 160\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1304E-TF-D-PA-M_S18_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1304E-TF-D-PA-M_S18_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1304E-TF-D-PA-M_S18_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1304E-TF-D-PA-M_S18_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 4.29 s (21 us/read; 2.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            207,251\n",
      "  Read 1 with adapter:                 191,603 (92.4%)\n",
      "  Read 2 with adapter:                 189,209 (91.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       207,251 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,537,118 bp\n",
      "  Read 1:    31,270,902 bp\n",
      "  Read 2:    31,266,216 bp\n",
      "Total written (filtered):     55,115,523 bp (88.1%)\n",
      "  Read 1:    27,631,354 bp\n",
      "  Read 2:    27,484,169 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 191603 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t809.6\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t3\t0.0\t1\t1 2\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t10\t0.0\t1\t7 3\n",
      "18\t1260\t0.0\t1\t107 1153\n",
      "19\t189862\t0.0\t1\t159735 30127\n",
      "20\t459\t0.0\t1\t100 359\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 189209 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t3238.3\t0\t32\n",
      "4\t3\t809.6\t0\t3\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t4\t0.0\t1\t4\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t3\t0.0\t1\t3\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t19\t0.0\t1\t7 12\n",
      "19\t1880\t0.0\t1\t227 1653\n",
      "20\t186783\t0.0\t2\t180113 6670\n",
      "21\t474\t0.0\t2\t77 397\n",
      "32\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1313-TF-D-DL-M_S34_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1313-TF-D-DL-M_S34_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1313-TF-D-DL-M_S34_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1313-TF-D-DL-M_S34_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.24 s (23 us/read; 2.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             97,941\n",
      "  Read 1 with adapter:                  90,332 (92.2%)\n",
      "  Read 2 with adapter:                  89,343 (91.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        97,941 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,555,470 bp\n",
      "  Read 1:    14,778,240 bp\n",
      "  Read 2:    14,777,230 bp\n",
      "Total written (filtered):     26,053,867 bp (88.2%)\n",
      "  Read 1:    13,062,309 bp\n",
      "  Read 2:    12,991,558 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90332 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t382.6\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t0 4\n",
      "18\t547\t0.0\t1\t54 493\n",
      "19\t89546\t0.0\t1\t73772 15774\n",
      "20\t227\t0.0\t1\t40 187\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 89343 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1530.3\t0\t29\n",
      "4\t2\t382.6\t0\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t8\t0.0\t1\t6 2\n",
      "19\t837\t0.0\t1\t106 731\n",
      "20\t88260\t0.0\t2\t85457 2803\n",
      "21\t202\t0.0\t2\t48 154\n",
      "23\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1313-TF-H-DL-M_S30_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1313-TF-H-DL-M_S30_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1313-TF-H-DL-M_S30_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1313-TF-H-DL-M_S30_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.50 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            105,990\n",
      "  Read 1 with adapter:                  98,155 (92.6%)\n",
      "  Read 2 with adapter:                  97,244 (91.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       105,990 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,961,107 bp\n",
      "  Read 1:    15,986,668 bp\n",
      "  Read 2:    15,974,439 bp\n",
      "Total written (filtered):     28,153,106 bp (88.1%)\n",
      "  Read 1:    14,122,831 bp\n",
      "  Read 2:    14,030,275 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 98155 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t414.0\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t1303\t0.0\t1\t45 1258\n",
      "19\t96605\t0.0\t1\t82403 14202\n",
      "20\t239\t0.0\t1\t52 187\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 97244 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t1656.1\t0\t14\n",
      "4\t3\t414.0\t0\t3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t3\t0.0\t1\t3\n",
      "19\t599\t0.0\t1\t70 529\n",
      "20\t96433\t0.0\t2\t94473 1960\n",
      "21\t188\t0.0\t2\t41 147\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1320B-TF-D-PS-M_S23_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1320B-TF-D-PS-M_S23_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1320B-TF-D-PS-M_S23_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1320B-TF-D-PS-M_S23_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.22 s (30 us/read; 2.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,345\n",
      "  Read 1 with adapter:                  38,041 (92.0%)\n",
      "  Read 2 with adapter:                  37,644 (91.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,345 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,476,325 bp\n",
      "  Read 1:     6,237,801 bp\n",
      "  Read 2:     6,238,524 bp\n",
      "Total written (filtered):     11,001,059 bp (88.2%)\n",
      "  Read 1:     5,515,124 bp\n",
      "  Read 2:     5,485,935 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 38041 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t189\t0.0\t1\t22 167\n",
      "19\t37744\t0.0\t1\t30829 6915\n",
      "20\t104\t0.0\t1\t30 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 37644 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t646.0\t0\t5\n",
      "4\t1\t161.5\t0\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t1\t0.0\t1\t0 1\n",
      "19\t267\t0.0\t1\t37 230\n",
      "20\t37274\t0.0\t2\t36477 797\n",
      "21\t92\t0.0\t2\t17 75\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1323-T1-H-DL-M_S28_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1323-T1-H-DL-M_S28_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1323-T1-H-DL-M_S28_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1323-T1-H-DL-M_S28_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.15 s (22 us/read; 2.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             99,036\n",
      "  Read 1 with adapter:                  88,881 (89.7%)\n",
      "  Read 2 with adapter:                  89,062 (89.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        99,036 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,538,042 bp\n",
      "  Read 1:    14,283,706 bp\n",
      "  Read 2:    14,254,336 bp\n",
      "Total written (filtered):     25,069,560 bp (87.8%)\n",
      "  Read 1:    12,595,520 bp\n",
      "  Read 2:    12,474,040 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 88881 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t4\t0.0\t1\t4\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t727\t0.0\t1\t39 688\n",
      "19\t87927\t0.0\t1\t62519 25408\n",
      "20\t212\t0.0\t1\t47 165\n",
      "36\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 89062 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t1547.4\t0\t23\n",
      "4\t5\t386.9\t0\t5\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "18\t10\t0.0\t1\t5 5\n",
      "19\t625\t0.0\t1\t88 537\n",
      "20\t88210\t0.0\t2\t86293 1917\n",
      "21\t187\t0.0\t2\t39 148\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1326C-T1-H-PS-M_S21_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1326C-T1-H-PS-M_S21_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1326C-T1-H-PS-M_S21_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1326C-T1-H-PS-M_S21_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.52 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,589\n",
      "  Read 1 with adapter:                  96,029 (90.1%)\n",
      "  Read 2 with adapter:                  95,909 (90.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,589 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,139,405 bp\n",
      "  Read 1:    16,065,988 bp\n",
      "  Read 2:    16,073,417 bp\n",
      "Total written (filtered):     28,398,415 bp (88.4%)\n",
      "  Read 1:    14,242,116 bp\n",
      "  Read 2:    14,156,299 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 96029 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t416.4\t0\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t8\t0.0\t1\t4 4\n",
      "18\t844\t0.0\t1\t60 784\n",
      "19\t94939\t0.0\t1\t70109 24830\n",
      "20\t228\t0.0\t1\t48 180\n",
      "32\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 95909 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t1665.5\t0\t31\n",
      "4\t4\t416.4\t0\t4\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t7\t0.0\t1\t4 3\n",
      "19\t659\t0.0\t1\t95 564\n",
      "20\t94965\t0.0\t2\t93012 1953\n",
      "21\t237\t0.0\t2\t58 179\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1326E-TF-H-PS-M_S26_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1326E-TF-H-PS-M_S26_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1326E-TF-H-PS-M_S26_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1326E-TF-H-PS-M_S26_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.23 s (32 us/read; 1.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,977\n",
      "  Read 1 with adapter:                  35,875 (92.0%)\n",
      "  Read 2 with adapter:                  35,489 (91.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        38,977 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,757,853 bp\n",
      "  Read 1:     5,878,724 bp\n",
      "  Read 2:     5,879,129 bp\n",
      "Total written (filtered):     10,366,901 bp (88.2%)\n",
      "  Read 1:     5,197,281 bp\n",
      "  Read 2:     5,169,620 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 35875 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t260\t0.0\t1\t17 243\n",
      "19\t35526\t0.0\t1\t29062 6464\n",
      "20\t87\t0.0\t1\t19 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 35489 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t609.0\t0\t5\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t4\t0.0\t1\t2 2\n",
      "19\t242\t0.0\t1\t33 209\n",
      "20\t35165\t0.0\t2\t34257 908\n",
      "21\t71\t0.0\t2\t17 54\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1338C-TF-D-OA-M_S11_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1338C-TF-D-OA-M_S11_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1338C-TF-D-OA-M_S11_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1338C-TF-D-OA-M_S11_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.09 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,699\n",
      "  Read 1 with adapter:                  67,501 (86.9%)\n",
      "  Read 2 with adapter:                  69,021 (88.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        77,699 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,449,950 bp\n",
      "  Read 1:    11,723,403 bp\n",
      "  Read 2:    11,726,547 bp\n",
      "Total written (filtered):     20,787,790 bp (88.6%)\n",
      "  Read 1:    10,440,955 bp\n",
      "  Read 2:    10,346,835 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 67501 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t187\t0.0\t1\t26 161\n",
      "19\t67190\t0.0\t1\t33202 33988\n",
      "20\t118\t0.0\t1\t24 94\n",
      "32\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 69021 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t1214.0\t0\t17\n",
      "4\t4\t303.5\t0\t4\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t6\t0.0\t1\t3 3\n",
      "19\t443\t0.0\t1\t59 384\n",
      "20\t68426\t0.0\t2\t67107 1319\n",
      "21\t121\t0.0\t2\t26 95\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1226C-T1-H-MC-M_S13_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1226C-T1-H-MC-M_S13_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1226C-T1-H-MC-M_S13_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1226C-T1-H-MC-M_S13_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1295E-TF-H-PA-M_S20_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1295E-TF-H-PA-M_S20_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1295E-TF-H-PA-M_S20_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1295E-TF-H-PA-M_S20_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1301-TF-D-DL-M_S35_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1301-TF-D-DL-M_S35_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1301-TF-D-DL-M_S35_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1301-TF-D-DL-M_S35_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1301-TF-H-DL-M_S31_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1301-TF-H-DL-M_S31_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1301-TF-H-DL-M_S31_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1301-TF-H-DL-M_S31_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1302-T1-D-DL-M_S36_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1302-T1-D-DL-M_S36_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1302-T1-D-DL-M_S36_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1302-T1-D-DL-M_S36_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1304E-TF-D-PA-M_S18_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1304E-TF-D-PA-M_S18_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1304E-TF-D-PA-M_S18_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1304E-TF-D-PA-M_S18_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1313-TF-D-DL-M_S34_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1313-TF-D-DL-M_S34_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1313-TF-D-DL-M_S34_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1313-TF-D-DL-M_S34_50_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1313-TF-H-DL-M_S30_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1313-TF-H-DL-M_S30_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1313-TF-H-DL-M_S30_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1313-TF-H-DL-M_S30_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1320B-TF-D-PS-M_S23_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1320B-TF-D-PS-M_S23_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1320B-TF-D-PS-M_S23_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1320B-TF-D-PS-M_S23_52_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1323-T1-H-DL-M_S28_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1323-T1-H-DL-M_S28_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1323-T1-H-DL-M_S28_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1323-T1-H-DL-M_S28_53_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1326C-T1-H-PS-M_S21_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1326C-T1-H-PS-M_S21_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1326C-T1-H-PS-M_S21_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1326C-T1-H-PS-M_S21_54_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1326E-TF-H-PS-M_S26_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1326E-TF-H-PS-M_S26_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1326E-TF-H-PS-M_S26_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1326E-TF-H-PS-M_S26_55_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1338C-TF-D-OA-M_S11_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1338C-TF-D-OA-M_S11_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1338C-TF-D-OA-M_S11_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1338C-TF-D-OA-M_S11_56_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1347A-TF-D-PS-M_S24_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1347A-TF-D-PS-M_S24_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1347A-TF-D-PS-M_S24_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1347A-TF-D-PS-M_S24_57_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1347A-TF-D-PS-M_S24_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1347A-TF-D-PS-M_S24_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1347A-TF-D-PS-M_S24_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1347A-TF-D-PS-M_S24_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.00 s (21 us/read; 2.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             97,646\n",
      "  Read 1 with adapter:                  89,790 (92.0%)\n",
      "  Read 2 with adapter:                  88,748 (90.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        97,646 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,467,554 bp\n",
      "  Read 1:    14,732,941 bp\n",
      "  Read 2:    14,734,613 bp\n",
      "Total written (filtered):     25,987,825 bp (88.2%)\n",
      "  Read 1:    13,027,176 bp\n",
      "  Read 2:    12,960,649 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 89790 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t381.4\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t411\t0.0\t1\t32 379\n",
      "19\t89143\t0.0\t1\t72800 16343\n",
      "20\t223\t0.0\t1\t41 182\n",
      "33\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 88748 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t1525.7\t0\t26\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t731\t0.0\t1\t101 630\n",
      "20\t87764\t0.0\t2\t85620 2144\n",
      "21\t215\t0.0\t2\t60 155\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1350F-TF-H-SS-M_S9_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1350F-TF-H-SS-M_S9_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1350F-TF-H-SS-M_S9_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1350F-TF-H-SS-M_S9_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.35 s (34 us/read; 1.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,456\n",
      "  Read 1 with adapter:                  35,106 (89.0%)\n",
      "  Read 2 with adapter:                  35,365 (89.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        39,456 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,905,577 bp\n",
      "  Read 1:     5,952,778 bp\n",
      "  Read 2:     5,952,799 bp\n",
      "Total written (filtered):     10,532,007 bp (88.5%)\n",
      "  Read 1:     5,285,996 bp\n",
      "  Read 2:     5,246,011 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 35106 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t284\t0.0\t1\t15 269\n",
      "19\t34739\t0.0\t1\t23355 11384\n",
      "20\t79\t0.0\t1\t13 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 35365 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t616.5\t0\t16\n",
      "4\t2\t154.1\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "18\t3\t0.0\t1\t3\n",
      "19\t262\t0.0\t1\t47 215\n",
      "20\t34997\t0.0\t2\t34263 734\n",
      "21\t82\t0.0\t2\t22 60\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1368B-T1-H-MC-M_S14_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1368B-T1-H-MC-M_S14_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1368B-T1-H-MC-M_S14_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1368B-T1-H-MC-M_S14_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.40 s (29 us/read; 2.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,667\n",
      "  Read 1 with adapter:                  44,929 (92.3%)\n",
      "  Read 2 with adapter:                  44,488 (91.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,667 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,684,450 bp\n",
      "  Read 1:     7,343,419 bp\n",
      "  Read 2:     7,341,031 bp\n",
      "Total written (filtered):     12,941,613 bp (88.1%)\n",
      "  Read 1:     6,489,917 bp\n",
      "  Read 2:     6,451,696 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 44929 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t286\t0.0\t1\t30 256\n",
      "19\t44516\t0.0\t1\t37197 7319\n",
      "20\t120\t0.0\t1\t24 96\n",
      "52\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 44488 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t760.4\t0\t8\n",
      "4\t1\t190.1\t0\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t339\t0.0\t1\t36 303\n",
      "20\t44049\t0.0\t2\t42920 1129\n",
      "21\t86\t0.0\t2\t21 65\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1369-T1-H-DL-M_S27_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1369-T1-H-DL-M_S27_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1369-T1-H-DL-M_S27_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1369-T1-H-DL-M_S27_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.86 s (24 us/read; 2.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,069\n",
      "  Read 1 with adapter:                 110,561 (92.1%)\n",
      "  Read 2 with adapter:                 109,653 (91.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       120,069 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,151,612 bp\n",
      "  Read 1:    18,091,747 bp\n",
      "  Read 2:    18,059,865 bp\n",
      "Total written (filtered):     31,860,054 bp (88.1%)\n",
      "  Read 1:    15,992,301 bp\n",
      "  Read 2:    15,867,753 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110561 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t0 2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t5\t0.0\t1\t4 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t1 3\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t6\t0.0\t1\t2 4\n",
      "17\t9\t0.0\t1\t5 4\n",
      "18\t1355\t0.0\t1\t57 1298\n",
      "19\t108908\t0.0\t1\t90021 18887\n",
      "20\t269\t0.0\t1\t37 232\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 109653 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t1876.1\t0\t20\n",
      "4\t2\t469.0\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t9\t0.0\t1\t6 3\n",
      "19\t800\t0.0\t1\t109 691\n",
      "20\t108548\t0.0\t2\t105943 2605\n",
      "21\t268\t0.0\t2\t56 212\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1390-T1-D-DL-M_S32_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1390-T1-D-DL-M_S32_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1390-T1-D-DL-M_S32_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1390-T1-D-DL-M_S32_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.67 s (30 us/read; 2.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,315\n",
      "  Read 1 with adapter:                  52,116 (92.5%)\n",
      "  Read 2 with adapter:                  51,491 (91.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,315 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,997,507 bp\n",
      "  Read 1:     8,499,265 bp\n",
      "  Read 2:     8,498,242 bp\n",
      "Total written (filtered):     14,978,258 bp (88.1%)\n",
      "  Read 1:     7,509,220 bp\n",
      "  Read 2:     7,469,038 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 52116 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t263\t0.0\t1\t26 237\n",
      "19\t51714\t0.0\t1\t43062 8652\n",
      "20\t131\t0.0\t1\t28 103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 51491 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t879.9\t0\t21\n",
      "11\t1\t0.0\t1\t0 1\n",
      "18\t3\t0.0\t1\t2 1\n",
      "19\t374\t0.0\t1\t53 321\n",
      "20\t50962\t0.0\t2\t49745 1217\n",
      "21\t130\t0.0\t2\t24 106\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1613A-T1-H-SS-M_S7_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1613A-T1-H-SS-M_S7_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1613A-T1-H-SS-M_S7_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1613A-T1-H-SS-M_S7_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.31 s (25 us/read; 2.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,033\n",
      "  Read 1 with adapter:                  48,235 (92.7%)\n",
      "  Read 2 with adapter:                  47,836 (91.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,033 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,695,840 bp\n",
      "  Read 1:     7,851,204 bp\n",
      "  Read 2:     7,844,636 bp\n",
      "Total written (filtered):     13,823,608 bp (88.1%)\n",
      "  Read 1:     6,935,224 bp\n",
      "  Read 2:     6,888,384 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 48235 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.2\t0\t3\n",
      "10\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t528\t0.0\t1\t18 510\n",
      "19\t47589\t0.0\t1\t40828 6761\n",
      "20\t108\t0.0\t1\t20 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 47836 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t813.0\t0\t13\n",
      "13\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t1 1\n",
      "19\t354\t0.0\t1\t53 301\n",
      "20\t47348\t0.0\t2\t46159 1189\n",
      "21\t118\t0.0\t2\t26 92\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1613A-TF-D-SS-M_S8_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1613A-TF-D-SS-M_S8_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1613A-TF-D-SS-M_S8_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1613A-TF-D-SS-M_S8_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.11 s (37 us/read; 1.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,970\n",
      "  Read 1 with adapter:                  27,671 (92.3%)\n",
      "  Read 2 with adapter:                  27,419 (91.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,970 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,021,062 bp\n",
      "  Read 1:     4,510,343 bp\n",
      "  Read 2:     4,510,719 bp\n",
      "Total written (filtered):      7,947,295 bp (88.1%)\n",
      "  Read 1:     3,984,731 bp\n",
      "  Read 2:     3,962,564 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 27671 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t117.1\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t163\t0.0\t1\t21 142\n",
      "19\t27436\t0.0\t1\t22504 4932\n",
      "20\t67\t0.0\t1\t17 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 27419 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t468.3\t0\t3\n",
      "4\t2\t117.1\t0\t2\n",
      "18\t2\t0.0\t1\t1 1\n",
      "19\t203\t0.0\t1\t32 171\n",
      "20\t27144\t0.0\t2\t26426 718\n",
      "21\t65\t0.0\t2\t10 55\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1633B-TF-H-PA-M_S37_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1633B-TF-H-PA-M_S37_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1633B-TF-H-PA-M_S37_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1633B-TF-H-PA-M_S37_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.15 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             95,447\n",
      "  Read 1 with adapter:                  88,450 (92.7%)\n",
      "  Read 2 with adapter:                  87,185 (91.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        95,447 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,790,467 bp\n",
      "  Read 1:    14,396,579 bp\n",
      "  Read 2:    14,393,888 bp\n",
      "Total written (filtered):     25,367,417 bp (88.1%)\n",
      "  Read 1:    12,716,323 bp\n",
      "  Read 2:    12,651,094 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 88450 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t372.8\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t5\t0.0\t1\t4 1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t0 3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t412\t0.0\t1\t47 365\n",
      "19\t87823\t0.0\t1\t74583 13240\n",
      "20\t200\t0.0\t1\t33 167\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 87185 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1491.4\t0\t16\n",
      "4\t2\t372.8\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "18\t9\t0.0\t1\t6 3\n",
      "19\t766\t0.0\t1\t95 671\n",
      "20\t86195\t0.0\t2\t83710 2485\n",
      "21\t195\t0.0\t2\t40 155\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1633D-T1-H-PA-M_S44_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1633D-T1-H-PA-M_S44_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1633D-T1-H-PA-M_S44_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1633D-T1-H-PA-M_S44_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.67 s (22 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            123,854\n",
      "  Read 1 with adapter:                 114,718 (92.6%)\n",
      "  Read 2 with adapter:                 113,152 (91.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       123,854 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,353,978 bp\n",
      "  Read 1:    18,678,580 bp\n",
      "  Read 2:    18,675,398 bp\n",
      "Total written (filtered):     32,912,746 bp (88.1%)\n",
      "  Read 1:    16,499,260 bp\n",
      "  Read 2:    16,413,486 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 114718 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t483.8\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t10\t0.0\t1\t9 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t0 4\n",
      "18\t520\t0.0\t1\t51 469\n",
      "19\t113880\t0.0\t1\t95712 18168\n",
      "20\t300\t0.0\t1\t59 241\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 113152 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t1935.2\t0\t18\n",
      "4\t3\t483.8\t0\t3\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t14\t0.0\t1\t7 7\n",
      "19\t972\t0.0\t1\t103 869\n",
      "20\t111864\t0.0\t2\t109485 2379\n",
      "21\t272\t0.0\t2\t61 211\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1659B-T1-H-SS-M_S6_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1659B-T1-H-SS-M_S6_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1659B-T1-H-SS-M_S6_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1659B-T1-H-SS-M_S6_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 4.02 s (22 us/read; 2.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            186,481\n",
      "  Read 1 with adapter:                 173,624 (93.1%)\n",
      "  Read 2 with adapter:                 171,459 (91.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       186,481 (100.0%)\n",
      "\n",
      "Total basepairs processed:    56,214,450 bp\n",
      "  Read 1:    28,129,012 bp\n",
      "  Read 2:    28,085,438 bp\n",
      "Total written (filtered):     49,489,493 bp (88.0%)\n",
      "  Read 1:    24,831,713 bp\n",
      "  Read 2:    24,657,780 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 173624 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.2\t1\t2\n",
      "12\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t9\t0.0\t1\t6 3\n",
      "18\t1967\t0.0\t1\t76 1891\n",
      "19\t171211\t0.0\t1\t148345 22866\n",
      "20\t421\t0.0\t1\t82 339\n",
      "25\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "42\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 171459 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t2913.8\t0\t30\n",
      "4\t4\t728.4\t0\t4\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t3 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t13\t0.0\t1\t4 9\n",
      "19\t1276\t0.0\t1\t187 1089\n",
      "20\t169738\t0.0\t2\t165267 4471\n",
      "21\t387\t0.0\t2\t88 299\n",
      "42\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1662D-TF-H-MC-M_S15_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1662D-TF-H-MC-M_S15_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1662D-TF-H-MC-M_S15_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1662D-TF-H-MC-M_S15_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.27 s (31 us/read; 1.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,286\n",
      "  Read 1 with adapter:                  38,557 (93.4%)\n",
      "  Read 2 with adapter:                  38,159 (92.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,286 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,292,549 bp\n",
      "  Read 1:     6,147,163 bp\n",
      "  Read 2:     6,145,386 bp\n",
      "Total written (filtered):     10,797,174 bp (87.8%)\n",
      "  Read 1:     5,414,683 bp\n",
      "  Read 2:     5,382,491 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 38557 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t209\t0.0\t1\t23 186\n",
      "19\t38214\t0.0\t1\t33183 5031\n",
      "20\t129\t0.0\t1\t20 109\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 38159 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t645.1\t0\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "18\t4\t0.0\t1\t2 2\n",
      "19\t300\t0.0\t1\t40 260\n",
      "20\t37771\t0.0\t2\t36797 974\n",
      "21\t80\t0.0\t2\t15 65\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1666E-T1-H-PA-M_S17_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1666E-T1-H-PA-M_S17_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1666E-T1-H-PA-M_S17_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1666E-T1-H-PA-M_S17_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.55 s (24 us/read; 2.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,182\n",
      "  Read 1 with adapter:                  59,165 (92.2%)\n",
      "  Read 2 with adapter:                  58,416 (91.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,182 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,366,278 bp\n",
      "  Read 1:     9,683,816 bp\n",
      "  Read 2:     9,682,462 bp\n",
      "Total written (filtered):     17,074,588 bp (88.2%)\n",
      "  Read 1:     8,559,852 bp\n",
      "  Read 2:     8,514,736 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 59165 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t257\t0.0\t1\t29 228\n",
      "19\t58761\t0.0\t1\t49070 9691\n",
      "20\t136\t0.0\t1\t31 105\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 58416 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t1002.8\t0\t14\n",
      "4\t1\t250.7\t0\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t3\t0.0\t1\t3\n",
      "19\t479\t0.0\t1\t66 413\n",
      "20\t57776\t0.0\t2\t56654 1122\n",
      "21\t140\t0.0\t2\t31 109\n",
      "31\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1681B-TF-H-PA-M_S19_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1681B-TF-H-PA-M_S19_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1681B-TF-H-PA-M_S19_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1681B-TF-H-PA-M_S19_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.86 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            128,639\n",
      "  Read 1 with adapter:                 119,513 (92.9%)\n",
      "  Read 2 with adapter:                 118,034 (91.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       128,639 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,760,187 bp\n",
      "  Read 1:    19,382,082 bp\n",
      "  Read 2:    19,378,105 bp\n",
      "Total written (filtered):     34,130,502 bp (88.1%)\n",
      "  Read 1:    17,111,763 bp\n",
      "  Read 2:    17,018,739 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 119513 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t502.5\t0\t1\n",
      "9\t2\t0.5\t0\t2\n",
      "10\t3\t0.1\t1\t3\n",
      "12\t4\t0.0\t1\t4\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t647\t0.0\t1\t55 592\n",
      "19\t118515\t0.0\t1\t100703 17812\n",
      "20\t332\t0.0\t1\t57 275\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 118034 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t2010.0\t0\t21\n",
      "4\t6\t502.5\t0\t6\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t0 3\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t10\t0.0\t1\t3 7\n",
      "19\t1046\t0.0\t1\t132 914\n",
      "20\t116670\t0.0\t2\t113329 3341\n",
      "21\t270\t0.0\t2\t65 205\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1692A-TF-D-CN-M_S4_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1692A-TF-D-CN-M_S4_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1692A-TF-D-CN-M_S4_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1692A-TF-D-CN-M_S4_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.51 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,186\n",
      "  Read 1 with adapter:                  51,121 (92.6%)\n",
      "  Read 2 with adapter:                  50,416 (91.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,186 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,645,019 bp\n",
      "  Read 1:     8,321,918 bp\n",
      "  Read 2:     8,323,101 bp\n",
      "Total written (filtered):     14,665,963 bp (88.1%)\n",
      "  Read 1:     7,350,740 bp\n",
      "  Read 2:     7,315,223 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 51121 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t215.6\t0\t2\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t208\t0.0\t1\t27 181\n",
      "19\t50773\t0.0\t1\t42576 8197\n",
      "20\t132\t0.0\t1\t29 103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 50416 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t862.3\t0\t10\n",
      "4\t2\t215.6\t0\t2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t4\t0.0\t1\t2 2\n",
      "19\t330\t0.0\t1\t36 294\n",
      "20\t49964\t0.0\t2\t48898 1066\n",
      "21\t104\t0.0\t2\t22 82\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1692B-TF-H-CN-M_S5_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1692B-TF-H-CN-M_S5_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1692B-TF-H-CN-M_S5_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1692B-TF-H-CN-M_S5_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.55 s (28 us/read; 2.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,825\n",
      "  Read 1 with adapter:                  51,113 (93.2%)\n",
      "  Read 2 with adapter:                  50,495 (92.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,825 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,473,470 bp\n",
      "  Read 1:     8,236,976 bp\n",
      "  Read 2:     8,236,494 bp\n",
      "Total written (filtered):     14,492,819 bp (88.0%)\n",
      "  Read 1:     7,265,925 bp\n",
      "  Read 2:     7,226,894 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 51113 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "18\t217\t0.0\t1\t24 193\n",
      "19\t50762\t0.0\t1\t43601 7161\n",
      "20\t132\t0.0\t1\t24 108\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 50495 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t856.6\t0\t3\n",
      "4\t3\t214.2\t0\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t325\t0.0\t1\t40 285\n",
      "20\t50028\t0.0\t2\t48843 1185\n",
      "21\t134\t0.0\t2\t28 106\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1350F-TF-H-SS-M_S9_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1350F-TF-H-SS-M_S9_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1350F-TF-H-SS-M_S9_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1350F-TF-H-SS-M_S9_58_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1368B-T1-H-MC-M_S14_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1368B-T1-H-MC-M_S14_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1368B-T1-H-MC-M_S14_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1368B-T1-H-MC-M_S14_59_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1369-T1-H-DL-M_S27_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1369-T1-H-DL-M_S27_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1369-T1-H-DL-M_S27_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1369-T1-H-DL-M_S27_60_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1390-T1-D-DL-M_S32_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1390-T1-D-DL-M_S32_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1390-T1-D-DL-M_S32_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1390-T1-D-DL-M_S32_61_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1613A-T1-H-SS-M_S7_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1613A-T1-H-SS-M_S7_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1613A-T1-H-SS-M_S7_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1613A-T1-H-SS-M_S7_62_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1613A-TF-D-SS-M_S8_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1613A-TF-D-SS-M_S8_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1613A-TF-D-SS-M_S8_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1613A-TF-D-SS-M_S8_63_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1633B-TF-H-PA-M_S37_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1633B-TF-H-PA-M_S37_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1633B-TF-H-PA-M_S37_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1633B-TF-H-PA-M_S37_64_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1633D-T1-H-PA-M_S44_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1633D-T1-H-PA-M_S44_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1633D-T1-H-PA-M_S44_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1633D-T1-H-PA-M_S44_65_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1659B-T1-H-SS-M_S6_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1659B-T1-H-SS-M_S6_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1659B-T1-H-SS-M_S6_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1659B-T1-H-SS-M_S6_66_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1662D-TF-H-MC-M_S15_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1662D-TF-H-MC-M_S15_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1662D-TF-H-MC-M_S15_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1662D-TF-H-MC-M_S15_67_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1666E-T1-H-PA-M_S17_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1666E-T1-H-PA-M_S17_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1666E-T1-H-PA-M_S17_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1666E-T1-H-PA-M_S17_68_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1681B-TF-H-PA-M_S19_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1681B-TF-H-PA-M_S19_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1681B-TF-H-PA-M_S19_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1681B-TF-H-PA-M_S19_69_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1692A-TF-D-CN-M_S4_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1692A-TF-D-CN-M_S4_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1692A-TF-D-CN-M_S4_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1692A-TF-D-CN-M_S4_70_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1692B-TF-H-CN-M_S5_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1692B-TF-H-CN-M_S5_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1692B-TF-H-CN-M_S5_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1692B-TF-H-CN-M_S5_71_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1695-T1-H-DL-M_S29_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1695-T1-H-DL-M_S29_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1695-T1-H-DL-M_S29_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1695-T1-H-DL-M_S29_72_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1695-T1-H-DL-M_S29_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1695-T1-H-DL-M_S29_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1695-T1-H-DL-M_S29_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1695-T1-H-DL-M_S29_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.45 s (28 us/read; 2.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,816\n",
      "  Read 1 with adapter:                  47,505 (91.7%)\n",
      "  Read 2 with adapter:                  47,023 (90.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,816 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,744,848 bp\n",
      "  Read 1:     7,374,912 bp\n",
      "  Read 2:     7,369,936 bp\n",
      "Total written (filtered):     12,902,670 bp (87.5%)\n",
      "  Read 1:     6,472,634 bp\n",
      "  Read 2:     6,430,036 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 47505 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t202.4\t0\t1\n",
      "9\t2\t0.2\t0\t2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t378\t0.0\t1\t29 349\n",
      "19\t47014\t0.0\t1\t37606 9408\n",
      "20\t106\t0.0\t1\t17 89\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 47023 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t809.6\t0\t15\n",
      "4\t3\t202.4\t0\t3\n",
      "16\t2\t0.0\t1\t1 1\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t347\t0.0\t1\t61 286\n",
      "20\t46543\t0.0\t2\t45360 1183\n",
      "21\t108\t0.0\t2\t27 81\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1826D-T1-H-SS-M_S42_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1826D-T1-H-SS-M_S42_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1826D-T1-H-SS-M_S42_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1826D-T1-H-SS-M_S42_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.56 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,116\n",
      "  Read 1 with adapter:                 108,120 (93.9%)\n",
      "  Read 2 with adapter:                 106,756 (92.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,116 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,536,470 bp\n",
      "  Read 1:    17,270,708 bp\n",
      "  Read 2:    17,265,762 bp\n",
      "Total written (filtered):     30,349,140 bp (87.9%)\n",
      "  Read 1:    15,217,491 bp\n",
      "  Read 2:    15,131,649 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108120 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t10\t0.0\t1\t8 2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t1259\t0.0\t1\t67 1192\n",
      "19\t106542\t0.0\t1\t92254 14288\n",
      "20\t298\t0.0\t1\t59 239\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 106756 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1798.7\t0\t10\n",
      "4\t3\t449.7\t0\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t13\t0.0\t1\t6 7\n",
      "19\t961\t0.0\t1\t125 836\n",
      "20\t105514\t0.0\t2\t102807 2707\n",
      "21\t241\t0.0\t2\t44 197\n",
      "22\t1\t0.0\t2\t1\n",
      "23\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1831B-TF-H-OA-M_S12_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1831B-TF-H-OA-M_S12_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1831B-TF-H-OA-M_S12_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1831B-TF-H-OA-M_S12_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.46 s (19 us/read; 3.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            132,443\n",
      "  Read 1 with adapter:                 120,138 (90.7%)\n",
      "  Read 2 with adapter:                 119,146 (90.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       132,443 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,927,014 bp\n",
      "  Read 1:    19,962,914 bp\n",
      "  Read 2:    19,964,100 bp\n",
      "Total written (filtered):     35,263,071 bp (88.3%)\n",
      "  Read 1:    17,680,623 bp\n",
      "  Read 2:    17,582,448 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 120138 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t0 4\n",
      "18\t557\t0.0\t1\t71 486\n",
      "19\t119309\t0.0\t1\t95453 23856\n",
      "20\t260\t0.0\t1\t54 206\n",
      "21\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 119146 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t2069.4\t0\t33\n",
      "4\t3\t517.4\t0\t3\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t8\t0.0\t1\t4 4\n",
      "19\t930\t0.0\t1\t125 805\n",
      "20\t117858\t0.0\t2\t114887 2971\n",
      "21\t306\t0.0\t2\t69 237\n",
      "22\t1\t0.0\t2\t1\n",
      "32\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1831D-T1-H-OA-M_S10_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1831D-T1-H-OA-M_S10_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1831D-T1-H-OA-M_S10_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1831D-T1-H-OA-M_S10_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.30 s (18 us/read; 3.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            186,325\n",
      "  Read 1 with adapter:                 171,665 (92.1%)\n",
      "  Read 2 with adapter:                 170,123 (91.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       186,325 (100.0%)\n",
      "\n",
      "Total basepairs processed:    56,186,936 bp\n",
      "  Read 1:    28,104,887 bp\n",
      "  Read 2:    28,082,049 bp\n",
      "Total written (filtered):     49,526,769 bp (88.1%)\n",
      "  Read 1:    24,845,654 bp\n",
      "  Read 2:    24,681,115 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 171665 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t727.8\t0\t1\n",
      "9\t3\t0.7\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t9\t0.0\t1\t6 3\n",
      "18\t2725\t0.0\t1\t82 2643\n",
      "19\t168502\t0.0\t1\t143767 24735\n",
      "20\t418\t0.0\t1\t73 345\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 170123 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t38\t2911.3\t0\t38\n",
      "4\t7\t727.8\t0\t7\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t10\t0.0\t1\t6 4\n",
      "19\t1060\t0.0\t1\t154 906\n",
      "20\t168661\t0.0\t2\t164774 3887\n",
      "21\t341\t0.0\t2\t68 273\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1835B-TF-H-OA-M-Repeat-LibL_S1_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1835B-TF-H-OA-M-Repeat-LibL_S1_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1835B-TF-H-OA-M-Repeat-LibL_S1_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1835B-TF-H-OA-M-Repeat-LibL_S1_76_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.76 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,291\n",
      "  Read 1 with adapter:                  72,716 (91.7%)\n",
      "  Read 2 with adapter:                  71,854 (90.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        79,291 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,924,234 bp\n",
      "  Read 1:    11,961,592 bp\n",
      "  Read 2:    11,962,642 bp\n",
      "Total written (filtered):     21,106,753 bp (88.2%)\n",
      "  Read 1:    10,580,285 bp\n",
      "  Read 2:    10,526,468 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 72716 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.3\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t419\t0.0\t1\t39 380\n",
      "19\t72101\t0.0\t1\t60807 11294\n",
      "20\t182\t0.0\t1\t27 155\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 71854 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t1238.9\t0\t23\n",
      "4\t1\t309.7\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t6\t0.0\t1\t1 5\n",
      "19\t611\t0.0\t1\t95 516\n",
      "20\t71072\t0.0\t2\t69544 1528\n",
      "21\t138\t0.0\t2\t27 111\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1842B-TF-H-PS-M_S38_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1842B-TF-H-PS-M_S38_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1842B-TF-H-PS-M_S38_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1842B-TF-H-PS-M_S38_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.67 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,558\n",
      "  Read 1 with adapter:                 106,096 (92.6%)\n",
      "  Read 2 with adapter:                 104,915 (91.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,558 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,152,835 bp\n",
      "  Read 1:    17,083,533 bp\n",
      "  Read 2:    17,069,302 bp\n",
      "Total written (filtered):     30,040,404 bp (88.0%)\n",
      "  Read 1:    15,068,219 bp\n",
      "  Read 2:    14,972,185 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 106096 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t7\t0.0\t1\t4 3\n",
      "18\t739\t0.0\t1\t57 682\n",
      "19\t105079\t0.0\t1\t88932 16147\n",
      "20\t264\t0.0\t1\t65 199\n",
      "21\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 104915 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t1790.0\t0\t34\n",
      "4\t1\t447.5\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t9\t0.0\t1\t3 6\n",
      "19\t778\t0.0\t1\t103 675\n",
      "20\t103835\t0.0\t2\t101094 2741\n",
      "21\t251\t0.0\t2\t67 184\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1842D-T1-H-PS-M_S22_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1842D-T1-H-PS-M_S22_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1842D-T1-H-PS-M_S22_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1842D-T1-H-PS-M_S22_78_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.25 s (23 us/read; 2.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             95,877\n",
      "  Read 1 with adapter:                  88,172 (92.0%)\n",
      "  Read 2 with adapter:                  87,414 (91.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        95,877 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,915,810 bp\n",
      "  Read 1:    14,458,447 bp\n",
      "  Read 2:    14,457,363 bp\n",
      "Total written (filtered):     25,493,613 bp (88.2%)\n",
      "  Read 1:    12,783,996 bp\n",
      "  Read 2:    12,709,617 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 88172 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t10\t0.0\t1\t6 4\n",
      "18\t935\t0.0\t1\t44 891\n",
      "19\t86995\t0.0\t1\t72527 14468\n",
      "20\t217\t0.0\t1\t46 171\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 87414 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1498.1\t0\t10\n",
      "4\t2\t374.5\t0\t2\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t4\t0.0\t1\t2 2\n",
      "19\t504\t0.0\t1\t74 430\n",
      "20\t86706\t0.0\t2\t85024 1682\n",
      "21\t186\t0.0\t2\t38 148\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1848C-T1-H-CN-M_S3_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1848C-T1-H-CN-M_S3_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1848C-T1-H-CN-M_S3_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1848C-T1-H-CN-M_S3_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.03 s (36 us/read; 1.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,604\n",
      "  Read 1 with adapter:                  26,712 (93.4%)\n",
      "  Read 2 with adapter:                  26,382 (92.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,604 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,622,303 bp\n",
      "  Read 1:     4,312,560 bp\n",
      "  Read 2:     4,309,743 bp\n",
      "Total written (filtered):      7,587,429 bp (88.0%)\n",
      "  Read 1:     3,805,116 bp\n",
      "  Read 2:     3,782,313 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 26712 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t148\t0.0\t1\t5 143\n",
      "19\t26494\t0.0\t1\t22929 3565\n",
      "20\t67\t0.0\t1\t13 54\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 26382 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t446.9\t0\t5\n",
      "19\t178\t0.0\t1\t18 160\n",
      "20\t26146\t0.0\t2\t25385 761\n",
      "21\t53\t0.0\t2\t5 48\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1858D-T1-H-PA-M_S16_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1858D-T1-H-PA-M_S16_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1858D-T1-H-PA-M_S16_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1858D-T1-H-PA-M_S16_80_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.85 s (22 us/read; 2.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,516\n",
      "  Read 1 with adapter:                 118,842 (91.8%)\n",
      "  Read 2 with adapter:                 117,681 (90.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       129,516 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,058,542 bp\n",
      "  Read 1:    19,529,511 bp\n",
      "  Read 2:    19,529,031 bp\n",
      "Total written (filtered):     34,448,728 bp (88.2%)\n",
      "  Read 1:    17,271,988 bp\n",
      "  Read 2:    17,176,740 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 118842 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t0 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t693\t0.0\t1\t66 627\n",
      "19\t117836\t0.0\t1\t95434 22402\n",
      "20\t295\t0.0\t1\t62 233\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 117681 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t2023.7\t0\t26\n",
      "4\t2\t505.9\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t12\t0.0\t1\t7 5\n",
      "19\t1074\t0.0\t1\t146 928\n",
      "20\t116275\t0.0\t2\t113444 2831\n",
      "21\t284\t0.0\t2\t59 225\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1870A-TF-H-PS-M_S25_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1870A-TF-H-PS-M_S25_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1870A-TF-H-PS-M_S25_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1870A-TF-H-PS-M_S25_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.52 s (24 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,042\n",
      "  Read 1 with adapter:                  57,079 (92.0%)\n",
      "  Read 2 with adapter:                  56,461 (91.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        62,042 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,709,375 bp\n",
      "  Read 1:     9,347,564 bp\n",
      "  Read 2:     9,361,811 bp\n",
      "Total written (filtered):     16,496,370 bp (88.2%)\n",
      "  Read 1:     8,263,289 bp\n",
      "  Read 2:     8,233,081 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 57079 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t326\t0.0\t1\t38 288\n",
      "19\t56616\t0.0\t1\t46757 9859\n",
      "20\t130\t0.0\t1\t32 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 56461 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t969.4\t0\t12\n",
      "4\t3\t242.4\t0\t3\n",
      "12\t1\t0.0\t1\t0 1\n",
      "18\t3\t0.0\t1\t2 1\n",
      "19\t363\t0.0\t1\t51 312\n",
      "20\t55970\t0.0\t2\t54547 1423\n",
      "21\t108\t0.0\t2\t19 89\n",
      "51\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/20-DNA-Control_S39_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/20-DNA-Control_S39_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/20-DNA-Control_S39_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/20-DNA-Control_S39_82_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.65 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,207\n",
      "  Read 1 with adapter:                   7,469 (91.0%)\n",
      "  Read 2 with adapter:                   7,448 (90.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         8,207 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,473,051 bp\n",
      "  Read 1:     1,236,517 bp\n",
      "  Read 2:     1,236,534 bp\n",
      "Total written (filtered):      2,182,286 bp (88.2%)\n",
      "  Read 1:     1,094,639 bp\n",
      "  Read 2:     1,087,647 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7469 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t50\t0.0\t1\t4 46\n",
      "19\t7402\t0.0\t1\t5331 2071\n",
      "20\t17\t0.0\t1\t5 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7448 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t128.2\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "19\t67\t0.0\t1\t9 58\n",
      "20\t7361\t0.0\t2\t7188 173\n",
      "21\t18\t0.0\t2\t5 13\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/21-DNA-Control_S40_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/21-DNA-Control_S40_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/21-DNA-Control_S40_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/21-DNA-Control_S40_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.56 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              5,615\n",
      "  Read 1 with adapter:                   5,139 (91.5%)\n",
      "  Read 2 with adapter:                   5,088 (90.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         5,615 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,690,130 bp\n",
      "  Read 1:       844,961 bp\n",
      "  Read 2:       845,169 bp\n",
      "Total written (filtered):      1,490,838 bp (88.2%)\n",
      "  Read 1:       747,329 bp\n",
      "  Read 2:       743,509 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5139 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t2 21\n",
      "19\t5102\t0.0\t1\t3970 1132\n",
      "20\t14\t0.0\t1\t3 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5088 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t87.7\t0\t3\n",
      "4\t1\t21.9\t0\t1\n",
      "19\t49\t0.0\t1\t8 41\n",
      "20\t5019\t0.0\t2\t4877 142\n",
      "21\t16\t0.0\t2\t1 15\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/24-DNA-Control_S43_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/24-DNA-Control_S43_84_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/24-DNA-Control_S43_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/24-DNA-Control_S43_84_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.27 s (21 us/read; 2.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,233\n",
      "  Read 1 with adapter:                 101,082 (95.2%)\n",
      "  Read 2 with adapter:                  99,557 (93.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,233 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,388,709 bp\n",
      "  Read 1:    12,708,840 bp\n",
      "  Read 2:    12,679,869 bp\n",
      "Total written (filtered):     21,478,728 bp (84.6%)\n",
      "  Read 1:    10,789,034 bp\n",
      "  Read 2:    10,689,694 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101082 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "11\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t6\t0.0\t1\t6\n",
      "18\t1017\t0.0\t1\t76 941\n",
      "19\t99730\t0.0\t1\t89719 10011\n",
      "20\t322\t0.0\t1\t66 256\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 99557 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t1659.9\t0\t18\n",
      "4\t1\t415.0\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t13\t0.0\t1\t3 10\n",
      "19\t853\t0.0\t1\t120 733\n",
      "20\t98414\t0.0\t2\t95588 2826\n",
      "21\t255\t0.0\t2\t39 216\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/2Flat-DD-OA-M_S33_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/2Flat-DD-OA-M_S33_85_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/2Flat-DD-OA-M_S33_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/2Flat-DD-OA-M_S33_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.14 s (26 us/read; 2.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             83,930\n",
      "  Read 1 with adapter:                  77,451 (92.3%)\n",
      "  Read 2 with adapter:                  76,603 (91.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        83,930 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,304,945 bp\n",
      "  Read 1:    12,661,509 bp\n",
      "  Read 2:    12,643,436 bp\n",
      "Total written (filtered):     22,302,534 bp (88.1%)\n",
      "  Read 1:    11,190,258 bp\n",
      "  Read 2:    11,112,276 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 77451 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t327.9\t0\t1\n",
      "9\t1\t0.3\t0\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t459\t0.0\t1\t38 421\n",
      "19\t76776\t0.0\t1\t62954 13822\n",
      "20\t202\t0.0\t1\t43 159\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 76603 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t1311.4\t0\t25\n",
      "4\t3\t327.9\t0\t3\n",
      "12\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t1\t0.0\t1\t0 1\n",
      "19\t543\t0.0\t1\t71 472\n",
      "20\t75876\t0.0\t2\t73611 2265\n",
      "21\t149\t0.0\t2\t25 124\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/4Flat-DD-PS-M_S41_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/4Flat-DD-PS-M_S41_86_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/4Flat-DD-PS-M_S41_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/4Flat-DD-PS-M_S41_86_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.58 s (24 us/read; 2.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,334\n",
      "  Read 1 with adapter:                  98,080 (92.2%)\n",
      "  Read 2 with adapter:                  97,337 (91.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,334 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,879,546 bp\n",
      "  Read 1:    15,955,607 bp\n",
      "  Read 2:    15,923,939 bp\n",
      "Total written (filtered):     28,071,134 bp (88.1%)\n",
      "  Read 1:    14,092,746 bp\n",
      "  Read 2:    13,978,388 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 98080 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t415.4\t0\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t7\t0.0\t1\t1 6\n",
      "18\t842\t0.0\t1\t54 788\n",
      "19\t96975\t0.0\t1\t77551 19424\n",
      "20\t248\t0.0\t1\t61 187\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 97337 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t1661.5\t0\t30\n",
      "4\t8\t415.4\t0\t8\n",
      "11\t3\t0.0\t1\t3\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t6\t0.0\t1\t5 1\n",
      "19\t717\t0.0\t1\t120 597\n",
      "20\t96358\t0.0\t2\t94229 2129\n",
      "21\t213\t0.0\t2\t44 169\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1826D-T1-H-SS-M_S42_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1826D-T1-H-SS-M_S42_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1826D-T1-H-SS-M_S42_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1826D-T1-H-SS-M_S42_73_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1831B-TF-H-OA-M_S12_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1831B-TF-H-OA-M_S12_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1831B-TF-H-OA-M_S12_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1831B-TF-H-OA-M_S12_74_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1831D-T1-H-OA-M_S10_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1831D-T1-H-OA-M_S10_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1831D-T1-H-OA-M_S10_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1831D-T1-H-OA-M_S10_75_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1835B-TF-H-OA-M-Repeat-LibL_S1_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1835B-TF-H-OA-M-Repeat-LibL_S1_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1835B-TF-H-OA-M-Repeat-LibL_S1_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1835B-TF-H-OA-M-Repeat-LibL_S1_76_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1842B-TF-H-PS-M_S38_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1842B-TF-H-PS-M_S38_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1842B-TF-H-PS-M_S38_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1842B-TF-H-PS-M_S38_77_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1842D-T1-H-PS-M_S22_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1842D-T1-H-PS-M_S22_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1842D-T1-H-PS-M_S22_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1842D-T1-H-PS-M_S22_78_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1848C-T1-H-CN-M_S3_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1848C-T1-H-CN-M_S3_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1848C-T1-H-CN-M_S3_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1848C-T1-H-CN-M_S3_79_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1858D-T1-H-PA-M_S16_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1858D-T1-H-PA-M_S16_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1858D-T1-H-PA-M_S16_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1858D-T1-H-PA-M_S16_80_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1870A-TF-H-PS-M_S25_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/1870A-TF-H-PS-M_S25_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1870A-TF-H-PS-M_S25_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/1870A-TF-H-PS-M_S25_81_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/20-DNA-Control_S39_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/20-DNA-Control_S39_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/20-DNA-Control_S39_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/20-DNA-Control_S39_82_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/21-DNA-Control_S40_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/21-DNA-Control_S40_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/21-DNA-Control_S40_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/21-DNA-Control_S40_83_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/24-DNA-Control_S43_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/24-DNA-Control_S43_84_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/24-DNA-Control_S43_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/24-DNA-Control_S43_84_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/2Flat-DD-OA-M_S33_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/2Flat-DD-OA-M_S33_85_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/2Flat-DD-OA-M_S33_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/2Flat-DD-OA-M_S33_85_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/4Flat-DD-PS-M_S41_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/4Flat-DD-PS-M_S41_86_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/4Flat-DD-PS-M_S41_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/4Flat-DD-PS-M_S41_86_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/mock-1-LibL_S2_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/mock-1-LibL_S2_87_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/mock-1-LibL_S2_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/mock-1-LibL_S2_87_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/mock-1-LibL_S2_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xxdkayf9/mock-1-LibL_S2_87_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/mock-1-LibL_S2_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-hlwdn8ee/672efd0a-987c-4a47-87dc-f2fcc7e29933/data/mock-1-LibL_S2_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.78 s (21 us/read; 2.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,637\n",
      "  Read 1 with adapter:                  79,954 (92.3%)\n",
      "  Read 2 with adapter:                  78,978 (91.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        86,637 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,128,250 bp\n",
      "  Read 1:    13,073,496 bp\n",
      "  Read 2:    13,054,754 bp\n",
      "Total written (filtered):     23,030,788 bp (88.1%)\n",
      "  Read 1:    11,554,673 bp\n",
      "  Read 2:    11,476,115 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 79954 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t428\t0.0\t1\t46 382\n",
      "19\t79324\t0.0\t1\t65219 14105\n",
      "20\t190\t0.0\t1\t46 144\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 78978 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1353.7\t0\t15\n",
      "4\t2\t338.4\t0\t2\n",
      "9\t1\t0.3\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t4\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t756\t0.0\t1\t97 659\n",
      "20\t78009\t0.0\t2\t75924 2085\n",
      "21\t180\t0.0\t2\t30 150\n",
      "\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_LibL.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1226A-TF-D-MC-M_S14_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1226A-TF-D-MC-M_S14_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1226A-TF-D-MC-M_S14_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1226A-TF-D-MC-M_S14_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.18 s (17 us/read; 3.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            184,256\n",
      "  Read 1 with adapter:                 178,779 (97.0%)\n",
      "  Read 2 with adapter:                 179,156 (97.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       184,256 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,577,489 bp\n",
      "  Read 1:    27,796,054 bp\n",
      "  Read 2:    27,781,435 bp\n",
      "Total written (filtered):     48,598,905 bp (87.4%)\n",
      "  Read 1:    24,399,767 bp\n",
      "  Read 2:    24,199,138 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 178779 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t4\t0.0\t1\t3 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t886\t0.0\t1\t92 794\n",
      "19\t177432\t0.0\t1\t167662 9770\n",
      "20\t442\t0.0\t1\t87 355\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 179156 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t2879.0\t0\t3\n",
      "4\t1\t719.8\t0\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t6\t0.0\t1\t3 3\n",
      "19\t1110\t0.0\t1\t112 998\n",
      "20\t177639\t0.0\t2\t174184 3455\n",
      "21\t391\t0.0\t2\t83 308\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1305A-TF-H-OA-M_S12_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1305A-TF-H-OA-M_S12_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1305A-TF-H-OA-M_S12_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1305A-TF-H-OA-M_S12_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.84 s (20 us/read; 2.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            139,972\n",
      "  Read 1 with adapter:                 135,891 (97.1%)\n",
      "  Read 2 with adapter:                 135,942 (97.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       139,972 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,252,687 bp\n",
      "  Read 1:    21,129,153 bp\n",
      "  Read 2:    21,123,534 bp\n",
      "Total written (filtered):     36,953,195 bp (87.5%)\n",
      "  Read 1:    18,547,644 bp\n",
      "  Read 2:    18,405,551 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 135891 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t546.8\t0\t1\n",
      "7\t1\t8.5\t0\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t8\t0.0\t1\t2 6\n",
      "18\t661\t0.0\t1\t78 583\n",
      "19\t134876\t0.0\t1\t126256 8620\n",
      "20\t335\t0.0\t1\t57 278\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 135942 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2187.1\t0\t1\n",
      "4\t1\t546.8\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t7\t0.0\t1\t5 2\n",
      "19\t1130\t0.0\t1\t159 971\n",
      "20\t134452\t0.0\t2\t130445 4007\n",
      "21\t346\t0.0\t2\t70 276\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1314-T1-D-DL-M_S33_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1314-T1-D-DL-M_S33_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1314-T1-D-DL-M_S33_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1314-T1-D-DL-M_S33_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.39 s (25 us/read; 2.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,288\n",
      "  Read 1 with adapter:                  91,633 (97.2%)\n",
      "  Read 2 with adapter:                  91,635 (97.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        94,288 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,411,103 bp\n",
      "  Read 1:    14,217,429 bp\n",
      "  Read 2:    14,193,674 bp\n",
      "Total written (filtered):     24,838,290 bp (87.4%)\n",
      "  Read 1:    12,476,717 bp\n",
      "  Read 2:    12,361,573 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91633 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t2\t0.0\t1\t2\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t496\t0.0\t1\t42 454\n",
      "19\t90902\t0.0\t1\t86496 4406\n",
      "20\t220\t0.0\t1\t48 172\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 91635 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1473.2\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t746\t0.0\t1\t121 625\n",
      "20\t90645\t0.0\t2\t88494 2151\n",
      "21\t226\t0.0\t2\t52 174\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1320A-TF-H-PS-M_S27_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1320A-TF-H-PS-M_S27_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1320A-TF-H-PS-M_S27_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1320A-TF-H-PS-M_S27_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 4.57 s (21 us/read; 2.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            220,630\n",
      "  Read 1 with adapter:                 214,768 (97.3%)\n",
      "  Read 2 with adapter:                 214,799 (97.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       220,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,618,703 bp\n",
      "  Read 1:    33,313,409 bp\n",
      "  Read 2:    33,305,294 bp\n",
      "Total written (filtered):     58,243,671 bp (87.4%)\n",
      "  Read 1:    29,233,397 bp\n",
      "  Read 2:    29,010,274 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 214768 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t215.5\t0\t1\n",
      "11\t1\t0.1\t1\t1\n",
      "12\t5\t0.0\t1\t1 4\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t14\t0.0\t1\t6 8\n",
      "18\t940\t0.0\t1\t111 829\n",
      "19\t213322\t0.0\t1\t202087 11235\n",
      "20\t478\t0.0\t1\t88 390\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 214799 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t3447.3\t0\t1\n",
      "4\t1\t861.8\t0\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t19\t0.0\t1\t11 8\n",
      "19\t1345\t0.0\t1\t203 1142\n",
      "20\t212956\t0.0\t2\t208224 4732\n",
      "21\t468\t0.0\t2\t103 365\n",
      "38\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1323-TF-H-DL-M_S31_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1323-TF-H-DL-M_S31_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1323-TF-H-DL-M_S31_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1323-TF-H-DL-M_S31_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.87 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,327\n",
      "  Read 1 with adapter:                  70,729 (97.8%)\n",
      "  Read 2 with adapter:                  70,717 (97.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        72,327 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,103,580 bp\n",
      "  Read 1:     9,581,688 bp\n",
      "  Read 2:     9,521,892 bp\n",
      "Total written (filtered):     16,346,188 bp (85.6%)\n",
      "  Read 1:     8,238,126 bp\n",
      "  Read 2:     8,108,062 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 70729 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.3\t0\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t4\t0.0\t1\t3 1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t438\t0.0\t1\t49 389\n",
      "19\t70068\t0.0\t1\t65957 4111\n",
      "20\t211\t0.0\t1\t38 173\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 70717 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t282.5\t0\t1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t6\t0.0\t1\t3 3\n",
      "19\t600\t0.0\t1\t103 497\n",
      "20\t69941\t0.0\t2\t68082 1859\n",
      "21\t158\t0.0\t2\t39 119\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1335-T1-H-DL-M_S29_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1335-T1-H-DL-M_S29_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1335-T1-H-DL-M_S29_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1335-T1-H-DL-M_S29_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.76 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,467\n",
      "  Read 1 with adapter:                  62,854 (97.5%)\n",
      "  Read 2 with adapter:                  62,850 (97.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,467 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,354,782 bp\n",
      "  Read 1:     9,706,686 bp\n",
      "  Read 2:     9,648,096 bp\n",
      "Total written (filtered):     16,904,062 bp (87.3%)\n",
      "  Read 1:     8,512,632 bp\n",
      "  Read 2:     8,391,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 62854 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t5\t0.0\t1\t2 3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t4\t0.0\t1\t0 4\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t305\t0.0\t1\t32 273\n",
      "19\t62344\t0.0\t1\t58963 3381\n",
      "20\t188\t0.0\t1\t29 159\n",
      "30\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 62850 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1007.3\t0\t1\n",
      "4\t1\t251.8\t0\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t4\t0.0\t1\t2 2\n",
      "19\t433\t0.0\t1\t75 358\n",
      "20\t62247\t0.0\t2\t60695 1552\n",
      "21\t160\t0.0\t2\t33 127\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1342B-T1-H-OA-M_S10_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1342B-T1-H-OA-M_S10_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1342B-T1-H-OA-M_S10_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1342B-T1-H-OA-M_S10_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.89 s (18 us/read; 3.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            157,821\n",
      "  Read 1 with adapter:                 153,580 (97.3%)\n",
      "  Read 2 with adapter:                 153,544 (97.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       157,821 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,275,236 bp\n",
      "  Read 1:    22,656,601 bp\n",
      "  Read 2:    22,618,635 bp\n",
      "Total written (filtered):     39,287,735 bp (86.8%)\n",
      "  Read 1:    19,739,044 bp\n",
      "  Read 2:    19,548,691 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 153580 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t7\t0.0\t1\t6 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t8\t0.0\t1\t6 2\n",
      "18\t766\t0.0\t1\t68 698\n",
      "19\t152382\t0.0\t1\t143769 8613\n",
      "20\t404\t0.0\t1\t79 325\n",
      "26\t1\t0.0\t1\t0 1\n",
      "36\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 153544 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2466.0\t0\t1\n",
      "4\t3\t616.5\t0\t3\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t18\t0.0\t1\t11 7\n",
      "19\t1196\t0.0\t1\t183 1013\n",
      "20\t151908\t0.0\t2\t148240 3668\n",
      "21\t407\t0.0\t2\t116 291\n",
      "22\t1\t0.0\t2\t0 1\n",
      "23\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1347B-TF-H-PS-M_S28_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1347B-TF-H-PS-M_S28_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1347B-TF-H-PS-M_S28_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1347B-TF-H-PS-M_S28_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.43 s (24 us/read; 2.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            102,531\n",
      "  Read 1 with adapter:                  99,727 (97.3%)\n",
      "  Read 2 with adapter:                  99,738 (97.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       102,531 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,950,698 bp\n",
      "  Read 1:    15,480,292 bp\n",
      "  Read 2:    15,470,406 bp\n",
      "Total written (filtered):     27,062,028 bp (87.4%)\n",
      "  Read 1:    13,585,763 bp\n",
      "  Read 2:    13,476,265 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99727 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t100.1\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t501\t0.0\t1\t39 462\n",
      "19\t98938\t0.0\t1\t93197 5741\n",
      "20\t277\t0.0\t1\t43 234\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 99738 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1602.0\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t6\t0.0\t1\t4 2\n",
      "19\t778\t0.0\t1\t104 674\n",
      "20\t98737\t0.0\t2\t96238 2499\n",
      "21\t212\t0.0\t2\t56 156\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1365-TF-H-DL-M_S30_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1365-TF-H-DL-M_S30_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1365-TF-H-DL-M_S30_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1365-TF-H-DL-M_S30_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.12 s (24 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,625\n",
      "  Read 1 with adapter:                  84,157 (97.2%)\n",
      "  Read 2 with adapter:                  84,366 (97.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        86,625 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,135,800 bp\n",
      "  Read 1:    13,071,081 bp\n",
      "  Read 2:    13,064,719 bp\n",
      "Total written (filtered):     22,850,376 bp (87.4%)\n",
      "  Read 1:    11,472,423 bp\n",
      "  Read 2:    11,377,953 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 84157 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.3\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t7\t0.0\t1\t5 2\n",
      "18\t472\t0.0\t1\t38 434\n",
      "19\t83487\t0.0\t1\t78355 5132\n",
      "20\t183\t0.0\t1\t35 148\n",
      "21\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 84366 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1353.5\t0\t1\n",
      "4\t1\t338.4\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t6\t0.0\t1\t6\n",
      "19\t663\t0.0\t1\t91 572\n",
      "20\t83495\t0.0\t2\t81503 1992\n",
      "21\t190\t0.0\t2\t42 148\n",
      "28\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1368A-TF-H-MC-M_S16_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1368A-TF-H-MC-M_S16_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1368A-TF-H-MC-M_S16_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1368A-TF-H-MC-M_S16_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 5.63 s (20 us/read; 3.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            281,828\n",
      "  Read 1 with adapter:                 274,607 (97.4%)\n",
      "  Read 2 with adapter:                 274,614 (97.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       281,828 (100.0%)\n",
      "\n",
      "Total basepairs processed:    82,995,592 bp\n",
      "  Read 1:    41,521,796 bp\n",
      "  Read 2:    41,473,796 bp\n",
      "Total written (filtered):     72,288,501 bp (87.1%)\n",
      "  Read 1:    36,305,186 bp\n",
      "  Read 2:    35,983,315 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 274607 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t1100.9\t0\t1\n",
      "8\t2\t4.3\t0\t2\n",
      "9\t3\t1.1\t0\t3\n",
      "10\t2\t0.3\t1\t1 1\n",
      "11\t2\t0.1\t1\t2\n",
      "12\t12\t0.0\t1\t9 3\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t1533\t0.0\t1\t126 1407\n",
      "19\t272243\t0.0\t1\t256931 15312\n",
      "20\t789\t0.0\t1\t139 650\n",
      "21\t3\t0.0\t1\t0 3\n",
      "22\t1\t0.0\t1\t0 1\n",
      "26\t1\t0.0\t1\t1\n",
      "50\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 274614 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t4403.6\t0\t1\n",
      "9\t1\t1.1\t0\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t32\t0.0\t1\t20 12\n",
      "19\t2359\t0.0\t1\t328 2031\n",
      "20\t271495\t0.0\t2\t264172 7323\n",
      "21\t712\t0.0\t2\t143 569\n",
      "29\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1650-T1-D-DL-M_S34_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1650-T1-D-DL-M_S34_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1650-T1-D-DL-M_S34_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1650-T1-D-DL-M_S34_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 4.31 s (21 us/read; 2.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            204,213\n",
      "  Read 1 with adapter:                 198,272 (97.1%)\n",
      "  Read 2 with adapter:                 198,382 (97.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       204,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:    61,568,347 bp\n",
      "  Read 1:    30,818,151 bp\n",
      "  Read 2:    30,750,196 bp\n",
      "Total written (filtered):     53,835,363 bp (87.4%)\n",
      "  Read 1:    27,051,608 bp\n",
      "  Read 2:    26,783,755 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 198272 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.8\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t7\t0.0\t1\t2 5\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t5\t0.0\t1\t1 4\n",
      "17\t11\t0.0\t1\t3 8\n",
      "18\t1030\t0.0\t1\t113 917\n",
      "19\t196722\t0.0\t1\t185934 10788\n",
      "20\t487\t0.0\t1\t94 393\n",
      "23\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "34\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 198382 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t3190.8\t0\t2\n",
      "4\t4\t797.7\t0\t4\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t18\t0.0\t1\t10 8\n",
      "19\t1508\t0.0\t1\t231 1277\n",
      "20\t196373\t0.0\t2\t191565 4808\n",
      "21\t472\t0.0\t2\t88 384\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654A-TF-D-SS-M_S7_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654A-TF-D-SS-M_S7_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654A-TF-D-SS-M_S7_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654A-TF-D-SS-M_S7_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.59 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,766\n",
      "  Read 1 with adapter:                  62,877 (97.1%)\n",
      "  Read 2 with adapter:                  62,964 (97.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,766 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,547,536 bp\n",
      "  Read 1:     9,777,824 bp\n",
      "  Read 2:     9,769,712 bp\n",
      "Total written (filtered):     17,094,117 bp (87.4%)\n",
      "  Read 1:     8,583,354 bp\n",
      "  Read 2:     8,510,763 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 62877 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t319\t0.0\t1\t29 290\n",
      "19\t62365\t0.0\t1\t58912 3453\n",
      "20\t180\t0.0\t1\t32 148\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 62964 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t253.0\t0\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t3\t0.0\t1\t1 2\n",
      "19\t438\t0.0\t1\t63 375\n",
      "20\t62363\t0.0\t2\t61041 1322\n",
      "21\t156\t0.0\t2\t25 131\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654C-T1-H-SS-M1_S5_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654C-T1-H-SS-M1_S5_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654C-T1-H-SS-M1_S5_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654C-T1-H-SS-M1_S5_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.80 s (25 us/read; 2.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,116\n",
      "  Read 1 with adapter:                  71,083 (97.2%)\n",
      "  Read 2 with adapter:                  71,093 (97.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        73,116 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,905,095 bp\n",
      "  Read 1:    10,958,696 bp\n",
      "  Read 2:    10,946,399 bp\n",
      "Total written (filtered):     19,133,248 bp (87.3%)\n",
      "  Read 1:     9,608,313 bp\n",
      "  Read 2:     9,524,935 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 71083 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t385\t0.0\t1\t40 345\n",
      "19\t70503\t0.0\t1\t67028 3475\n",
      "20\t186\t0.0\t1\t35 151\n",
      "25\t1\t0.0\t1\t0 1\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 71093 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1142.4\t0\t1\n",
      "4\t1\t285.6\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t4\t0.0\t1\t2 2\n",
      "19\t526\t0.0\t1\t69 457\n",
      "20\t70379\t0.0\t2\t68789 1590\n",
      "21\t178\t0.0\t2\t35 143\n",
      "31\t1\t0.0\t2\t0 1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1226A-TF-D-MC-M_S14_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1226A-TF-D-MC-M_S14_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1226A-TF-D-MC-M_S14_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1226A-TF-D-MC-M_S14_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1305A-TF-H-OA-M_S12_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1305A-TF-H-OA-M_S12_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1305A-TF-H-OA-M_S12_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1305A-TF-H-OA-M_S12_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1314-T1-D-DL-M_S33_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1314-T1-D-DL-M_S33_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1314-T1-D-DL-M_S33_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1314-T1-D-DL-M_S33_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1320A-TF-H-PS-M_S27_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1320A-TF-H-PS-M_S27_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1320A-TF-H-PS-M_S27_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1320A-TF-H-PS-M_S27_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1323-TF-H-DL-M_S31_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1323-TF-H-DL-M_S31_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1323-TF-H-DL-M_S31_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1323-TF-H-DL-M_S31_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1335-T1-H-DL-M_S29_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1335-T1-H-DL-M_S29_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1335-T1-H-DL-M_S29_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1335-T1-H-DL-M_S29_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1342B-T1-H-OA-M_S10_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1342B-T1-H-OA-M_S10_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1342B-T1-H-OA-M_S10_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1342B-T1-H-OA-M_S10_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1347B-TF-H-PS-M_S28_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1347B-TF-H-PS-M_S28_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1347B-TF-H-PS-M_S28_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1347B-TF-H-PS-M_S28_50_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1365-TF-H-DL-M_S30_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1365-TF-H-DL-M_S30_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1365-TF-H-DL-M_S30_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1365-TF-H-DL-M_S30_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1368A-TF-H-MC-M_S16_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1368A-TF-H-MC-M_S16_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1368A-TF-H-MC-M_S16_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1368A-TF-H-MC-M_S16_52_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1650-T1-D-DL-M_S34_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1650-T1-D-DL-M_S34_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1650-T1-D-DL-M_S34_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1650-T1-D-DL-M_S34_53_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654A-TF-D-SS-M_S7_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654A-TF-D-SS-M_S7_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654A-TF-D-SS-M_S7_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654A-TF-D-SS-M_S7_54_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654C-T1-H-SS-M1_S5_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654C-T1-H-SS-M1_S5_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654C-T1-H-SS-M1_S5_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654C-T1-H-SS-M1_S5_55_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654C-T1-H-SS-M2_S42_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654C-T1-H-SS-M2_S42_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654C-T1-H-SS-M2_S42_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654C-T1-H-SS-M2_S42_56_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654C-T1-H-SS-M2_S42_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1654C-T1-H-SS-M2_S42_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654C-T1-H-SS-M2_S42_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1654C-T1-H-SS-M2_S42_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.82 s (21 us/read; 2.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            178,146\n",
      "  Read 1 with adapter:                 173,915 (97.6%)\n",
      "  Read 2 with adapter:                 173,741 (97.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       178,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,210,038 bp\n",
      "  Read 1:    26,621,797 bp\n",
      "  Read 2:    26,588,241 bp\n",
      "Total written (filtered):     46,432,868 bp (87.3%)\n",
      "  Read 1:    23,318,083 bp\n",
      "  Read 2:    23,114,785 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 173915 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t2.7\t0\t1\n",
      "9\t2\t0.7\t0\t2\n",
      "10\t1\t0.2\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t1087\t0.0\t1\t84 1003\n",
      "19\t172303\t0.0\t1\t163047 9256\n",
      "20\t508\t0.0\t1\t82 426\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 173741 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t2783.5\t0\t3\n",
      "4\t1\t695.9\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t1 2\n",
      "14\t9\t0.0\t1\t5 4\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t8\t0.0\t1\t4 4\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t17\t0.0\t1\t8 9\n",
      "19\t1547\t0.0\t1\t226 1321\n",
      "20\t171731\t0.0\t2\t167531 4200\n",
      "21\t414\t0.0\t2\t89 325\n",
      "22\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1655D-TF-H-CN-M1_S4_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1655D-TF-H-CN-M1_S4_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1655D-TF-H-CN-M1_S4_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1655D-TF-H-CN-M1_S4_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.07 s (22 us/read; 2.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,977\n",
      "  Read 1 with adapter:                  89,560 (97.4%)\n",
      "  Read 2 with adapter:                  89,498 (97.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        91,977 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,276,448 bp\n",
      "  Read 1:    13,140,736 bp\n",
      "  Read 2:    13,135,712 bp\n",
      "Total written (filtered):     22,785,761 bp (86.7%)\n",
      "  Read 1:    11,439,410 bp\n",
      "  Read 2:    11,346,351 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 89560 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t5\t0.0\t1\t2 3\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t7\t0.0\t1\t5 2\n",
      "18\t485\t0.0\t1\t53 432\n",
      "19\t88830\t0.0\t1\t84302 4528\n",
      "20\t225\t0.0\t1\t33 192\n",
      "42\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 89498 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1437.1\t0\t2\n",
      "4\t2\t359.3\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t2\n",
      "18\t7\t0.0\t1\t1 6\n",
      "19\t704\t0.0\t1\t110 594\n",
      "20\t88558\t0.0\t2\t86428 2130\n",
      "21\t218\t0.0\t2\t38 180\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1655D-TF-H-CN-M2_S43_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1655D-TF-H-CN-M2_S43_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1655D-TF-H-CN-M2_S43_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1655D-TF-H-CN-M2_S43_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.13 s (19 us/read; 3.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            164,772\n",
      "  Read 1 with adapter:                 160,742 (97.6%)\n",
      "  Read 2 with adapter:                 160,578 (97.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       164,772 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,590,038 bp\n",
      "  Read 1:    24,787,566 bp\n",
      "  Read 2:    24,802,472 bp\n",
      "Total written (filtered):     43,325,811 bp (87.4%)\n",
      "  Read 1:    21,733,915 bp\n",
      "  Read 2:    21,591,896 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 160742 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.2\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t12\t0.0\t1\t8 4\n",
      "18\t804\t0.0\t1\t80 724\n",
      "19\t159465\t0.0\t1\t151476 7989\n",
      "20\t447\t0.0\t1\t72 375\n",
      "26\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 160578 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t643.6\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t25\t0.0\t1\t15 10\n",
      "19\t1265\t0.0\t1\t200 1065\n",
      "20\t158880\t0.0\t2\t155313 3567\n",
      "21\t399\t0.0\t2\t102 297\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1662H-TF-D-MC-M_S15_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1662H-TF-D-MC-M_S15_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1662H-TF-D-MC-M_S15_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1662H-TF-D-MC-M_S15_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.56 s (19 us/read; 3.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            183,578\n",
      "  Read 1 with adapter:                 178,558 (97.3%)\n",
      "  Read 2 with adapter:                 178,696 (97.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       183,578 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,383,353 bp\n",
      "  Read 1:    27,712,706 bp\n",
      "  Read 2:    27,670,647 bp\n",
      "Total written (filtered):     48,418,244 bp (87.4%)\n",
      "  Read 1:    24,320,564 bp\n",
      "  Read 2:    24,097,680 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 178558 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.7\t0\t2\n",
      "10\t2\t0.2\t1\t2\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t5\t0.0\t1\t4 1\n",
      "17\t8\t0.0\t1\t7 1\n",
      "18\t851\t0.0\t1\t77 774\n",
      "19\t177206\t0.0\t1\t167980 9226\n",
      "20\t474\t0.0\t1\t87 387\n",
      "21\t1\t0.0\t1\t0 1\n",
      "33\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 178696 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2868.4\t0\t1\n",
      "4\t3\t717.1\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t7\t0.0\t1\t4 3\n",
      "19\t1206\t0.0\t1\t187 1019\n",
      "20\t177115\t0.0\t2\t173585 3530\n",
      "21\t359\t0.0\t2\t93 266\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1666B-TF-H-PA-M_S21_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1666B-TF-H-PA-M_S21_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1666B-TF-H-PA-M_S21_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1666B-TF-H-PA-M_S21_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.80 s (17 us/read; 3.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,191\n",
      "  Read 1 with adapter:                 156,893 (97.3%)\n",
      "  Read 2 with adapter:                 156,811 (97.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       161,191 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,642,052 bp\n",
      "  Read 1:    24,328,864 bp\n",
      "  Read 2:    24,313,188 bp\n",
      "Total written (filtered):     42,526,519 bp (87.4%)\n",
      "  Read 1:    21,348,469 bp\n",
      "  Read 2:    21,178,050 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 156893 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t3\t0.2\t1\t2 1\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t2\t0.0\t1\t0 2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t7\t0.0\t1\t4 3\n",
      "18\t915\t0.0\t1\t79 836\n",
      "19\t155500\t0.0\t1\t146913 8587\n",
      "20\t455\t0.0\t1\t90 365\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 156811 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2518.6\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t15\t0.0\t1\t10 5\n",
      "19\t1345\t0.0\t1\t181 1164\n",
      "20\t155082\t0.0\t2\t151414 3668\n",
      "21\t356\t0.0\t2\t68 288\n",
      "25\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1666C-TF-D-PA-M_S20_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1666C-TF-D-PA-M_S20_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1666C-TF-D-PA-M_S20_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1666C-TF-D-PA-M_S20_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.71 s (16 us/read; 3.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            165,326\n",
      "  Read 1 with adapter:                 160,964 (97.4%)\n",
      "  Read 2 with adapter:                 160,998 (97.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       165,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,908,849 bp\n",
      "  Read 1:    24,961,352 bp\n",
      "  Read 2:    24,947,497 bp\n",
      "Total written (filtered):     43,631,955 bp (87.4%)\n",
      "  Read 1:    21,903,503 bp\n",
      "  Read 2:    21,728,452 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 160964 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t10\t0.0\t1\t5 5\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t803\t0.0\t1\t87 716\n",
      "19\t159688\t0.0\t1\t151129 8559\n",
      "20\t450\t0.0\t1\t77 373\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 160998 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2583.2\t0\t1\n",
      "4\t4\t645.8\t0\t4\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t14\t0.0\t1\t7 7\n",
      "19\t1148\t0.0\t1\t184 964\n",
      "20\t159456\t0.0\t2\t155897 3559\n",
      "21\t367\t0.0\t2\t88 279\n",
      "23\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1672E-TF-H-PS-M_S26_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1672E-TF-H-PS-M_S26_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1672E-TF-H-PS-M_S26_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1672E-TF-H-PS-M_S26_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.78 s (22 us/read; 2.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            127,787\n",
      "  Read 1 with adapter:                 124,216 (97.2%)\n",
      "  Read 2 with adapter:                 124,285 (97.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       127,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,558,967 bp\n",
      "  Read 1:    19,285,100 bp\n",
      "  Read 2:    19,273,867 bp\n",
      "Total written (filtered):     33,714,175 bp (87.4%)\n",
      "  Read 1:    16,925,334 bp\n",
      "  Read 2:    16,788,841 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 124216 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t9\t0.0\t1\t5 4\n",
      "18\t582\t0.0\t1\t51 531\n",
      "19\t123329\t0.0\t1\t116086 7243\n",
      "20\t286\t0.0\t1\t45 241\n",
      "51\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 124285 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1996.7\t0\t1\n",
      "4\t1\t499.2\t0\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t861\t0.0\t1\t105 756\n",
      "20\t123149\t0.0\t2\t120057 3092\n",
      "21\t262\t0.0\t2\t45 217\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1695-TF-D-DL-M_S35_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1695-TF-D-DL-M_S35_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1695-TF-D-DL-M_S35_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1695-TF-D-DL-M_S35_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.18 s (20 us/read; 3.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,527\n",
      "  Read 1 with adapter:                 106,333 (97.1%)\n",
      "  Read 2 with adapter:                 106,329 (97.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,527 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,064,755 bp\n",
      "  Read 1:    16,536,397 bp\n",
      "  Read 2:    16,528,358 bp\n",
      "Total written (filtered):     28,919,196 bp (87.5%)\n",
      "  Read 1:    14,516,539 bp\n",
      "  Read 2:    14,402,657 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 106333 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t702\t0.0\t1\t56 646\n",
      "19\t105310\t0.0\t1\t99235 6075\n",
      "20\t304\t0.0\t1\t61 243\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 106329 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1711.4\t0\t1\n",
      "4\t3\t427.8\t0\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t10\t0.0\t1\t6 4\n",
      "19\t1041\t0.0\t1\t121 920\n",
      "20\t104994\t0.0\t2\t101970 3024\n",
      "21\t273\t0.0\t2\t62 211\n",
      "28\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1695-TF-H-DL-M_S32_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1695-TF-H-DL-M_S32_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1695-TF-H-DL-M_S32_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1695-TF-H-DL-M_S32_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.76 s (27 us/read; 2.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,216\n",
      "  Read 1 with adapter:                  63,502 (97.4%)\n",
      "  Read 2 with adapter:                  63,517 (97.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        65,216 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,657,528 bp\n",
      "  Read 1:     9,831,012 bp\n",
      "  Read 2:     9,826,516 bp\n",
      "Total written (filtered):     17,181,196 bp (87.4%)\n",
      "  Read 1:     8,624,674 bp\n",
      "  Read 2:     8,556,522 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 63502 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t5\t0.0\t1\t4 1\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t279\t0.0\t1\t29 250\n",
      "19\t63049\t0.0\t1\t59864 3185\n",
      "20\t154\t0.0\t1\t26 128\n",
      "21\t1\t0.0\t1\t1\n",
      "32\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 63517 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1019.0\t0\t1\n",
      "4\t1\t254.8\t0\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t417\t0.0\t1\t51 366\n",
      "20\t62968\t0.0\t2\t61810 1158\n",
      "21\t123\t0.0\t2\t23 100\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826C-TF-D-SS-M_S8_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826C-TF-D-SS-M_S8_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826C-TF-D-SS-M_S8_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826C-TF-D-SS-M_S8_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.79 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,993\n",
      "  Read 1 with adapter:                  65,018 (97.1%)\n",
      "  Read 2 with adapter:                  65,058 (97.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        66,993 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,189,175 bp\n",
      "  Read 1:    10,097,130 bp\n",
      "  Read 2:    10,092,045 bp\n",
      "Total written (filtered):     17,653,216 bp (87.4%)\n",
      "  Read 1:     8,861,976 bp\n",
      "  Read 2:     8,791,240 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 65018 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t359\t0.0\t1\t22 337\n",
      "19\t64470\t0.0\t1\t60992 3478\n",
      "20\t185\t0.0\t1\t34 151\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 65058 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t10\t0.0\t1\t4 6\n",
      "19\t494\t0.0\t1\t77 417\n",
      "20\t64382\t0.0\t2\t62952 1430\n",
      "21\t170\t0.0\t2\t36 134\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826D-T1-H-SS-M_S6_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826D-T1-H-SS-M_S6_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826D-T1-H-SS-M_S6_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826D-T1-H-SS-M_S6_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.93 s (24 us/read; 2.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            121,140\n",
      "  Read 1 with adapter:                 117,189 (96.7%)\n",
      "  Read 2 with adapter:                 117,607 (97.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       121,140 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,438,941 bp\n",
      "  Read 1:    18,226,259 bp\n",
      "  Read 2:    18,212,682 bp\n",
      "Total written (filtered):     31,861,456 bp (87.4%)\n",
      "  Read 1:    16,000,156 bp\n",
      "  Read 2:    15,861,300 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 117189 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t8\t0.0\t1\t5 3\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t2 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t739\t0.0\t1\t58 681\n",
      "19\t116081\t0.0\t1\t108237 7844\n",
      "20\t349\t0.0\t1\t59 290\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 117607 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1892.8\t0\t2\n",
      "4\t1\t473.2\t0\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t15\t0.0\t1\t7 8\n",
      "19\t923\t0.0\t1\t130 793\n",
      "20\t116345\t0.0\t2\t113310 3035\n",
      "21\t307\t0.0\t2\t78 229\n",
      "24\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826E-TF-H-SS-M_S9_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826E-TF-H-SS-M_S9_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826E-TF-H-SS-M_S9_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826E-TF-H-SS-M_S9_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.52 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,796\n",
      "  Read 1 with adapter:                 110,422 (97.0%)\n",
      "  Read 2 with adapter:                 110,658 (97.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,796 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,202,143 bp\n",
      "  Read 1:    17,104,916 bp\n",
      "  Read 2:    17,097,227 bp\n",
      "Total written (filtered):     29,892,056 bp (87.4%)\n",
      "  Read 1:    15,007,201 bp\n",
      "  Read 2:    14,884,855 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110422 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t4\t0.0\t1\t1 3\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t536\t0.0\t1\t59 477\n",
      "19\t109550\t0.0\t1\t102089 7461\n",
      "20\t319\t0.0\t1\t65 254\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110658 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t444.5\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t0 4\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t12\t0.0\t1\t8 4\n",
      "19\t942\t0.0\t1\t138 804\n",
      "20\t109446\t0.0\t2\t106696 2750\n",
      "21\t247\t0.0\t2\t60 187\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1827C-TF-H-PA-M_S36_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1827C-TF-H-PA-M_S36_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1827C-TF-H-PA-M_S36_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1827C-TF-H-PA-M_S36_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.49 s (22 us/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,201\n",
      "  Read 1 with adapter:                 156,705 (97.2%)\n",
      "  Read 2 with adapter:                 156,859 (97.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       161,201 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,601,648 bp\n",
      "  Read 1:    24,310,488 bp\n",
      "  Read 2:    24,291,160 bp\n",
      "Total written (filtered):     42,488,369 bp (87.4%)\n",
      "  Read 1:    21,333,452 bp\n",
      "  Read 2:    21,154,917 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 156705 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t6\t0.0\t1\t3 3\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t721\t0.0\t1\t66 655\n",
      "19\t155534\t0.0\t1\t147390 8144\n",
      "20\t431\t0.0\t1\t74 357\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 156859 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2518.8\t0\t2\n",
      "4\t3\t629.7\t0\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t7\t0.0\t1\t1 6\n",
      "19\t1177\t0.0\t1\t138 1039\n",
      "20\t155309\t0.0\t2\t151655 3654\n",
      "21\t357\t0.0\t2\t69 288\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1835B-TF-H-OA-M-Repeat-LibM_S1_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1835B-TF-H-OA-M-Repeat-LibM_S1_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1835B-TF-H-OA-M-Repeat-LibM_S1_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1835B-TF-H-OA-M-Repeat-LibM_S1_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.18 s (20 us/read; 3.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,544\n",
      "  Read 1 with adapter:                 107,382 (97.1%)\n",
      "  Read 2 with adapter:                 107,319 (97.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       110,544 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,355,216 bp\n",
      "  Read 1:    16,679,660 bp\n",
      "  Read 2:    16,675,556 bp\n",
      "Total written (filtered):     29,169,602 bp (87.5%)\n",
      "  Read 1:    14,639,729 bp\n",
      "  Read 2:    14,529,873 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 107382 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.7\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t11\t0.0\t1\t3 8\n",
      "18\t546\t0.0\t1\t69 477\n",
      "19\t106550\t0.0\t1\t100662 5888\n",
      "20\t267\t0.0\t1\t48 219\n",
      "21\t1\t0.0\t1\t1\n",
      "30\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 107319 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1727.2\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t7\t0.0\t1\t4 3\n",
      "19\t862\t0.0\t1\t110 752\n",
      "20\t106192\t0.0\t2\t103864 2328\n",
      "21\t243\t0.0\t2\t51 192\n",
      "22\t2\t0.0\t2\t2\n",
      "29\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1840A-TF-D-PS-M_S25_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1840A-TF-D-PS-M_S25_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1840A-TF-D-PS-M_S25_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1840A-TF-D-PS-M_S25_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.47 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,946\n",
      "  Read 1 with adapter:                 108,065 (97.4%)\n",
      "  Read 2 with adapter:                 108,040 (97.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       110,946 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,461,858 bp\n",
      "  Read 1:    16,734,217 bp\n",
      "  Read 2:    16,727,641 bp\n",
      "Total written (filtered):     29,248,636 bp (87.4%)\n",
      "  Read 1:    14,681,289 bp\n",
      "  Read 2:    14,567,347 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108065 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t505\t0.0\t1\t61 444\n",
      "19\t107262\t0.0\t1\t101402 5860\n",
      "20\t279\t0.0\t1\t46 233\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108040 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1733.5\t0\t1\n",
      "4\t1\t433.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t689\t0.0\t1\t99 590\n",
      "20\t107111\t0.0\t2\t104786 2325\n",
      "21\t234\t0.0\t2\t54 180\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1655D-TF-H-CN-M1_S4_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1655D-TF-H-CN-M1_S4_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1655D-TF-H-CN-M1_S4_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1655D-TF-H-CN-M1_S4_57_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1655D-TF-H-CN-M2_S43_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1655D-TF-H-CN-M2_S43_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1655D-TF-H-CN-M2_S43_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1655D-TF-H-CN-M2_S43_58_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1662H-TF-D-MC-M_S15_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1662H-TF-D-MC-M_S15_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1662H-TF-D-MC-M_S15_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1662H-TF-D-MC-M_S15_59_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1666B-TF-H-PA-M_S21_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1666B-TF-H-PA-M_S21_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1666B-TF-H-PA-M_S21_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1666B-TF-H-PA-M_S21_60_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1666C-TF-D-PA-M_S20_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1666C-TF-D-PA-M_S20_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1666C-TF-D-PA-M_S20_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1666C-TF-D-PA-M_S20_61_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1672E-TF-H-PS-M_S26_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1672E-TF-H-PS-M_S26_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1672E-TF-H-PS-M_S26_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1672E-TF-H-PS-M_S26_62_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1695-TF-D-DL-M_S35_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1695-TF-D-DL-M_S35_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1695-TF-D-DL-M_S35_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1695-TF-D-DL-M_S35_63_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1695-TF-H-DL-M_S32_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1695-TF-H-DL-M_S32_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1695-TF-H-DL-M_S32_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1695-TF-H-DL-M_S32_64_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826C-TF-D-SS-M_S8_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826C-TF-D-SS-M_S8_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826C-TF-D-SS-M_S8_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826C-TF-D-SS-M_S8_65_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826D-T1-H-SS-M_S6_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826D-T1-H-SS-M_S6_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826D-T1-H-SS-M_S6_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826D-T1-H-SS-M_S6_66_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826E-TF-H-SS-M_S9_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1826E-TF-H-SS-M_S9_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826E-TF-H-SS-M_S9_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1826E-TF-H-SS-M_S9_67_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1827C-TF-H-PA-M_S36_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1827C-TF-H-PA-M_S36_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1827C-TF-H-PA-M_S36_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1827C-TF-H-PA-M_S36_68_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1835B-TF-H-OA-M-Repeat-LibM_S1_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1835B-TF-H-OA-M-Repeat-LibM_S1_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1835B-TF-H-OA-M-Repeat-LibM_S1_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1835B-TF-H-OA-M-Repeat-LibM_S1_69_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1840A-TF-D-PS-M_S25_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1840A-TF-D-PS-M_S25_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1840A-TF-D-PS-M_S25_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1840A-TF-D-PS-M_S25_70_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1840D-T1-H-PS-M_S24_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1840D-T1-H-PS-M_S24_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1840D-T1-H-PS-M_S24_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1840D-T1-H-PS-M_S24_71_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1840D-T1-H-PS-M_S24_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1840D-T1-H-PS-M_S24_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1840D-T1-H-PS-M_S24_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1840D-T1-H-PS-M_S24_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.32 s (23 us/read; 2.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            143,790\n",
      "  Read 1 with adapter:                 139,187 (96.8%)\n",
      "  Read 2 with adapter:                 140,070 (97.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       143,790 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,361,886 bp\n",
      "  Read 1:    21,687,068 bp\n",
      "  Read 2:    21,674,818 bp\n",
      "Total written (filtered):     37,917,151 bp (87.4%)\n",
      "  Read 1:    19,042,928 bp\n",
      "  Read 2:    18,874,223 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 139187 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t5\t0.0\t1\t4 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t662\t0.0\t1\t70 592\n",
      "19\t138178\t0.0\t1\t128532 9646\n",
      "20\t330\t0.0\t1\t58 272\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 140070 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t561.7\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t4\t0.0\t1\t4\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "18\t12\t0.0\t1\t10 2\n",
      "19\t1019\t0.0\t1\t156 863\n",
      "20\t138723\t0.0\t2\t135936 2787\n",
      "21\t305\t0.0\t2\t69 236\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1848A-TF-D-CN-M_S3_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1848A-TF-D-CN-M_S3_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1848A-TF-D-CN-M_S3_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1848A-TF-D-CN-M_S3_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.65 s (29 us/read; 2.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,666\n",
      "  Read 1 with adapter:                  55,110 (97.3%)\n",
      "  Read 2 with adapter:                  55,136 (97.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,666 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,100,133 bp\n",
      "  Read 1:     8,551,695 bp\n",
      "  Read 2:     8,548,438 bp\n",
      "Total written (filtered):     14,950,824 bp (87.4%)\n",
      "  Read 1:     7,504,810 bp\n",
      "  Read 2:     7,446,014 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55110 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t295\t0.0\t1\t29 266\n",
      "19\t54652\t0.0\t1\t51995 2657\n",
      "20\t149\t0.0\t1\t29 120\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 55136 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t2\t0.0\t1\t2\n",
      "18\t7\t0.0\t1\t3 4\n",
      "19\t416\t0.0\t1\t59 357\n",
      "20\t54567\t0.0\t2\t53423 1144\n",
      "21\t144\t0.0\t2\t28 116\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1858C-TF-D-PA-M_S19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1858C-TF-D-PA-M_S19_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1858C-TF-D-PA-M_S19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1858C-TF-D-PA-M_S19_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.42 s (22 us/read; 2.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            154,627\n",
      "  Read 1 with adapter:                 150,311 (97.2%)\n",
      "  Read 2 with adapter:                 150,271 (97.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       154,627 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,671,743 bp\n",
      "  Read 1:    23,343,977 bp\n",
      "  Read 2:    23,327,766 bp\n",
      "Total written (filtered):     40,812,232 bp (87.4%)\n",
      "  Read 1:    20,488,728 bp\n",
      "  Read 2:    20,323,504 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 150311 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t604.0\t0\t1\n",
      "9\t1\t0.6\t0\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t5\t0.0\t1\t2 3\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t5\t0.0\t1\t1 4\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t931\t0.0\t1\t79 852\n",
      "19\t148964\t0.0\t1\t139935 9029\n",
      "20\t392\t0.0\t1\t70 322\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 150271 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2416.0\t0\t1\n",
      "4\t2\t604.0\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t11\t0.0\t1\t3 8\n",
      "19\t1389\t0.0\t1\t206 1183\n",
      "20\t148498\t0.0\t2\t144017 4481\n",
      "21\t355\t0.0\t2\t81 274\n",
      "25\t1\t0.0\t2\t1\n",
      "28\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1864A-TF-H-PS-M_S37_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1864A-TF-H-PS-M_S37_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1864A-TF-H-PS-M_S37_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1864A-TF-H-PS-M_S37_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.09 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,209\n",
      "  Read 1 with adapter:                  82,743 (97.1%)\n",
      "  Read 2 with adapter:                  82,855 (97.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        85,209 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,677,012 bp\n",
      "  Read 1:    12,822,704 bp\n",
      "  Read 2:    12,854,308 bp\n",
      "Total written (filtered):     22,448,500 bp (87.4%)\n",
      "  Read 1:    11,250,803 bp\n",
      "  Read 2:    11,197,697 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 82743 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t357\t0.0\t1\t34 323\n",
      "19\t82214\t0.0\t1\t78013 4201\n",
      "20\t167\t0.0\t1\t34 133\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 82855 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1331.4\t0\t1\n",
      "4\t2\t332.8\t0\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t0 2\n",
      "19\t566\t0.0\t1\t66 500\n",
      "20\t82142\t0.0\t2\t80509 1633\n",
      "21\t140\t0.0\t2\t26 114\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1864C-T1-H-PS-M_S23_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1864C-T1-H-PS-M_S23_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1864C-T1-H-PS-M_S23_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1864C-T1-H-PS-M_S23_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.53 s (24 us/read; 2.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,799\n",
      "  Read 1 with adapter:                 103,856 (97.2%)\n",
      "  Read 2 with adapter:                 104,023 (97.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,799 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,036,285 bp\n",
      "  Read 1:    15,518,175 bp\n",
      "  Read 2:    15,518,110 bp\n",
      "Total written (filtered):     26,983,516 bp (86.9%)\n",
      "  Read 1:    13,545,287 bp\n",
      "  Read 2:    13,438,229 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103856 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t540\t0.0\t1\t54 486\n",
      "19\t103046\t0.0\t1\t96554 6492\n",
      "20\t252\t0.0\t1\t54 198\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 104023 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1668.7\t0\t1\n",
      "4\t2\t417.2\t0\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t4\t0.0\t1\t1 3\n",
      "19\t747\t0.0\t1\t120 627\n",
      "20\t103027\t0.0\t2\t100616 2411\n",
      "21\t237\t0.0\t2\t60 177\n",
      "25\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/2186B-TF-H-PA-M_S22_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/2186B-TF-H-PA-M_S22_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/2186B-TF-H-PA-M_S22_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/2186B-TF-H-PA-M_S22_76_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.68 s (23 us/read; 2.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,327\n",
      "  Read 1 with adapter:                  71,307 (97.2%)\n",
      "  Read 2 with adapter:                  71,472 (97.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        73,327 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,111,814 bp\n",
      "  Read 1:    11,059,474 bp\n",
      "  Read 2:    11,052,340 bp\n",
      "Total written (filtered):     19,328,132 bp (87.4%)\n",
      "  Read 1:     9,704,835 bp\n",
      "  Read 2:     9,623,297 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 71307 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t349\t0.0\t1\t40 309\n",
      "19\t70763\t0.0\t1\t66341 4422\n",
      "20\t187\t0.0\t1\t32 155\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 71472 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t552\t0.0\t1\t78 474\n",
      "20\t70755\t0.0\t2\t69085 1670\n",
      "21\t162\t0.0\t2\t34 128\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/2186E-T1-H-PA-M_S17_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/2186E-T1-H-PA-M_S17_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/2186E-T1-H-PA-M_S17_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/2186E-T1-H-PA-M_S17_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.95 s (20 us/read; 2.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            195,038\n",
      "  Read 1 with adapter:                 189,716 (97.3%)\n",
      "  Read 2 with adapter:                 189,765 (97.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       195,038 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,866,660 bp\n",
      "  Read 1:    29,441,988 bp\n",
      "  Read 2:    29,424,672 bp\n",
      "Total written (filtered):     51,468,775 bp (87.4%)\n",
      "  Read 1:    25,838,042 bp\n",
      "  Read 2:    25,630,733 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 189716 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.7\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t8\t0.0\t1\t7 1\n",
      "18\t1101\t0.0\t1\t82 1019\n",
      "19\t188079\t0.0\t1\t177249 10830\n",
      "20\t514\t0.0\t1\t97 417\n",
      "21\t1\t0.0\t1\t0 1\n",
      "39\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 189765 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t3047.5\t0\t1\n",
      "4\t3\t761.9\t0\t3\n",
      "12\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t0 3\n",
      "18\t14\t0.0\t1\t9 5\n",
      "19\t1704\t0.0\t1\t258 1446\n",
      "20\t187562\t0.0\t2\t183242 4320\n",
      "21\t471\t0.0\t2\t110 361\n",
      "23\t1\t0.0\t2\t1\n",
      "26\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/22-DNA-Control_S39_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/22-DNA-Control_S39_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/22-DNA-Control_S39_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/22-DNA-Control_S39_78_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.89 s (37 us/read; 1.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,844\n",
      "  Read 1 with adapter:                  23,142 (97.1%)\n",
      "  Read 2 with adapter:                  23,138 (97.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,844 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,170,423 bp\n",
      "  Read 1:     3,587,351 bp\n",
      "  Read 2:     3,583,072 bp\n",
      "Total written (filtered):      6,268,279 bp (87.4%)\n",
      "  Read 1:     3,147,784 bp\n",
      "  Read 2:     3,120,495 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23142 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t164\t0.0\t1\t14 150\n",
      "19\t22906\t0.0\t1\t21674 1232\n",
      "20\t66\t0.0\t1\t18 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 23138 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "19\t224\t0.0\t1\t29 195\n",
      "20\t22869\t0.0\t2\t22189 680\n",
      "21\t44\t0.0\t2\t10 34\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/23-DNA-Control_S40_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/23-DNA-Control_S40_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/23-DNA-Control_S40_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/23-DNA-Control_S40_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.59 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              6,872\n",
      "  Read 1 with adapter:                   6,676 (97.1%)\n",
      "  Read 2 with adapter:                   6,676 (97.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         6,872 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,069,781 bp\n",
      "  Read 1:     1,036,464 bp\n",
      "  Read 2:     1,033,317 bp\n",
      "Total written (filtered):      1,809,506 bp (87.4%)\n",
      "  Read 1:       909,657 bp\n",
      "  Read 2:       899,849 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6676 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "18\t45\t0.0\t1\t4 41\n",
      "19\t6603\t0.0\t1\t6236 367\n",
      "20\t25\t0.0\t1\t4 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6676 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t0 1\n",
      "18\t2\t0.0\t1\t1 1\n",
      "19\t53\t0.0\t1\t11 42\n",
      "20\t6607\t0.0\t2\t6400 207\n",
      "21\t13\t0.0\t2\t3 10\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/3-FLAT-DD-OA-M_S18_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/3-FLAT-DD-OA-M_S18_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/3-FLAT-DD-OA-M_S18_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/3-FLAT-DD-OA-M_S18_80_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.90 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,911\n",
      "  Read 1 with adapter:                  70,827 (97.1%)\n",
      "  Read 2 with adapter:                  70,944 (97.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        72,911 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,993,730 bp\n",
      "  Read 1:    11,002,283 bp\n",
      "  Read 2:    10,991,447 bp\n",
      "Total written (filtered):     19,229,645 bp (87.4%)\n",
      "  Read 1:     9,656,760 bp\n",
      "  Read 2:     9,572,885 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 70827 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t326\t0.0\t1\t39 287\n",
      "19\t70325\t0.0\t1\t65545 4780\n",
      "20\t169\t0.0\t1\t27 142\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 70944 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "18\t3\t0.0\t1\t1 2\n",
      "19\t469\t0.0\t1\t72 397\n",
      "20\t70302\t0.0\t2\t68436 1866\n",
      "21\t168\t0.0\t2\t32 136\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/5Flat-DD-PS-M_S11_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/5Flat-DD-PS-M_S11_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/5Flat-DD-PS-M_S11_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/5Flat-DD-PS-M_S11_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.32 s (27 us/read; 2.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,731\n",
      "  Read 1 with adapter:                  47,352 (97.2%)\n",
      "  Read 2 with adapter:                  47,361 (97.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,731 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,687,977 bp\n",
      "  Read 1:     7,345,881 bp\n",
      "  Read 2:     7,342,096 bp\n",
      "Total written (filtered):     12,841,554 bp (87.4%)\n",
      "  Read 1:     6,446,381 bp\n",
      "  Read 2:     6,395,173 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 47352 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t254\t0.0\t1\t22 232\n",
      "19\t46969\t0.0\t1\t44207 2762\n",
      "20\t117\t0.0\t1\t19 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 47361 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t190.4\t0\t2\n",
      "13\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "18\t5\t0.0\t1\t0 5\n",
      "19\t353\t0.0\t1\t51 302\n",
      "20\t46881\t0.0\t2\t45400 1481\n",
      "21\t117\t0.0\t2\t28 89\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/6Flat-DD-OA-M_S41_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/6Flat-DD-OA-M_S41_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/6Flat-DD-OA-M_S41_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/6Flat-DD-OA-M_S41_82_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.30 s (21 us/read; 2.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            154,486\n",
      "  Read 1 with adapter:                 150,058 (97.1%)\n",
      "  Read 2 with adapter:                 150,003 (97.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       154,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,458,629 bp\n",
      "  Read 1:    23,265,014 bp\n",
      "  Read 2:    23,193,615 bp\n",
      "Total written (filtered):     40,608,818 bp (87.4%)\n",
      "  Read 1:    20,414,352 bp\n",
      "  Read 2:    20,194,466 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 150058 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t10\t0.0\t1\t9 1\n",
      "13\t5\t0.0\t1\t3 2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t9\t0.0\t1\t8 1\n",
      "18\t700\t0.0\t1\t100 600\n",
      "19\t148941\t0.0\t1\t141893 7048\n",
      "20\t384\t0.0\t1\t57 327\n",
      "25\t1\t0.0\t1\t0 1\n",
      "50\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 150003 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t603.5\t0\t2\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t5\t0.0\t1\t1 4\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t6\t0.0\t1\t5 1\n",
      "18\t11\t0.0\t1\t6 5\n",
      "19\t1085\t0.0\t1\t194 891\n",
      "20\t148536\t0.0\t2\t145562 2974\n",
      "21\t347\t0.0\t2\t75 272\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/Flat-Swab-Control_S13_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/Flat-Swab-Control_S13_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/Flat-Swab-Control_S13_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/Flat-Swab-Control_S13_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.44 s (22 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            159,933\n",
      "  Read 1 with adapter:                 155,472 (97.2%)\n",
      "  Read 2 with adapter:                 155,504 (97.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       159,933 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,143,109 bp\n",
      "  Read 1:    24,078,041 bp\n",
      "  Read 2:    24,065,068 bp\n",
      "Total written (filtered):     42,080,859 bp (87.4%)\n",
      "  Read 1:    21,124,733 bp\n",
      "  Read 2:    20,956,126 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 155472 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "12\t4\t0.0\t1\t2 2\n",
      "13\t4\t0.0\t1\t2 2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t979\t0.0\t1\t60 919\n",
      "19\t154091\t0.0\t1\t144542 9549\n",
      "20\t378\t0.0\t1\t71 307\n",
      "29\t1\t0.0\t1\t0 1\n",
      "32\t2\t0.0\t1\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 155504 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2499.0\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t15\t0.0\t1\t7 8\n",
      "19\t1423\t0.0\t1\t184 1239\n",
      "20\t153685\t0.0\t2\t149003 4682\n",
      "21\t370\t0.0\t2\t87 283\n",
      "25\t1\t0.0\t2\t1\n",
      "32\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/PCR-Control-2_S38_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/PCR-Control-2_S38_85_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/PCR-Control-2_S38_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/PCR-Control-2_S38_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.31 s (662 us/read; 0.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                474\n",
      "  Read 1 with adapter:                     460 (97.0%)\n",
      "  Read 2 with adapter:                     456 (96.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):           474 (100.0%)\n",
      "\n",
      "Total basepairs processed:       143,082 bp\n",
      "  Read 1:        71,560 bp\n",
      "  Read 2:        71,522 bp\n",
      "Total written (filtered):        125,226 bp (87.5%)\n",
      "  Read 1:        62,822 bp\n",
      "  Read 2:        62,404 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 460 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 2\n",
      "19\t458\t0.0\t1\t401 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 456 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t2\t0.0\t1\t1 1\n",
      "20\t454\t0.0\t2\t432 22\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/mock-1_LibM_S2_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/mock-1_LibM_S2_84_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/mock-1_LibM_S2_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/mock-1_LibM_S2_84_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.39 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,946\n",
      "  Read 1 with adapter:                  98,061 (97.1%)\n",
      "  Read 2 with adapter:                  97,999 (97.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,946 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,455,791 bp\n",
      "  Read 1:    15,240,917 bp\n",
      "  Read 2:    15,214,874 bp\n",
      "Total written (filtered):     26,633,843 bp (87.5%)\n",
      "  Read 1:    13,378,193 bp\n",
      "  Read 2:    13,255,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 98061 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.4\t0\t3\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t4\t0.0\t1\t4\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t590\t0.0\t1\t51 539\n",
      "19\t97205\t0.0\t1\t91070 6135\n",
      "20\t248\t0.0\t1\t43 205\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 97999 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t10\t0.0\t1\t6 4\n",
      "19\t928\t0.0\t1\t124 804\n",
      "20\t96814\t0.0\t2\t94204 2610\n",
      "21\t238\t0.0\t2\t58 180\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1848A-TF-D-CN-M_S3_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1848A-TF-D-CN-M_S3_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1848A-TF-D-CN-M_S3_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1848A-TF-D-CN-M_S3_72_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1858C-TF-D-PA-M_S19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1858C-TF-D-PA-M_S19_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1858C-TF-D-PA-M_S19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1858C-TF-D-PA-M_S19_73_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1864A-TF-H-PS-M_S37_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1864A-TF-H-PS-M_S37_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1864A-TF-H-PS-M_S37_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1864A-TF-H-PS-M_S37_74_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1864C-T1-H-PS-M_S23_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/1864C-T1-H-PS-M_S23_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1864C-T1-H-PS-M_S23_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/1864C-T1-H-PS-M_S23_75_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/2186B-TF-H-PA-M_S22_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/2186B-TF-H-PA-M_S22_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/2186B-TF-H-PA-M_S22_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/2186B-TF-H-PA-M_S22_76_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/2186E-T1-H-PA-M_S17_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/2186E-T1-H-PA-M_S17_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/2186E-T1-H-PA-M_S17_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/2186E-T1-H-PA-M_S17_77_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/22-DNA-Control_S39_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/22-DNA-Control_S39_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/22-DNA-Control_S39_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/22-DNA-Control_S39_78_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/23-DNA-Control_S40_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/23-DNA-Control_S40_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/23-DNA-Control_S40_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/23-DNA-Control_S40_79_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/3-FLAT-DD-OA-M_S18_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/3-FLAT-DD-OA-M_S18_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/3-FLAT-DD-OA-M_S18_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/3-FLAT-DD-OA-M_S18_80_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/5Flat-DD-PS-M_S11_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/5Flat-DD-PS-M_S11_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/5Flat-DD-PS-M_S11_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/5Flat-DD-PS-M_S11_81_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/6Flat-DD-OA-M_S41_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/6Flat-DD-OA-M_S41_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/6Flat-DD-OA-M_S41_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/6Flat-DD-OA-M_S41_82_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/Flat-Swab-Control_S13_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/Flat-Swab-Control_S13_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/Flat-Swab-Control_S13_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/Flat-Swab-Control_S13_83_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/PCR-Control-2_S38_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/PCR-Control-2_S38_85_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/PCR-Control-2_S38_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/PCR-Control-2_S38_85_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/mock-1_LibM_S2_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xjc48isn/mock-1_LibM_S2_84_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/mock-1_LibM_S2_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-3tj1tqez/3f982224-5b9c-45de-95fc-0695f8e84599/data/mock-1_LibM_S2_84_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_LibM.qza\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "#trim PCR primers\n",
    "for f1 in /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_transmissionMucus2019/Import_files_Apprill_2019TransmissionExpMucus_Lib*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%*.csv};\n",
    "\n",
    "qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences $f1 \\\n",
    "--p-cores 2 \\\n",
    "--p-front-f GTGYCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACNVGGGTWTCTAAT \\\n",
    "--o-trimmed-sequences Folder_Apprill_transmissionMucus2019/trim_$fn \\\n",
    "--verbose\n",
    "\n",
    "done\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_LibI.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_LibJ.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_LibK.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_LibL.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_LibM.qzv\n"
     ]
    }
   ],
   "source": [
    "#visualize\n",
    "%%bash\n",
    "\n",
    "for f1 in Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_Lib*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime demux summarize  --i-data $f1  --o-visualization Folder_Apprill_transmissionMucus2019/$fn.qzv\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibI.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus_LibI.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Apprill_transmissionMucus2019/dada_trim_Import_files_Apprill_2019TransmissionExpMucus_LibI.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibJ.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus_LibJ.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Apprill_transmissionMucus2019/dada_trim_Import_files_Apprill_2019TransmissionExpMucus_LibJ.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibK.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus_LibK.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Apprill_transmissionMucus2019/dada_trim_Import_files_Apprill_2019TransmissionExpMucus_LibK.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibL.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus_LibL.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Apprill_transmissionMucus2019/dada_trim_Import_files_Apprill_2019TransmissionExpMucus_LibL.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibM.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus_LibM.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Apprill_transmissionMucus2019/dada_trim_Import_files_Apprill_2019TransmissionExpMucus_LibM.qza\n"
     ]
    }
   ],
   "source": [
    "#run DADA2\n",
    "%%bash\n",
    "\n",
    "for f1 in Folder_Apprill_transmissionMucus2019/trim_Import_files_Apprill_2019TransmissionExpMucus_Lib*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime dada2 denoise-single --i-demultiplexed-seqs $f1 --p-max-ee 3 --p-trunc-len 0 --p-n-threads 10 --o-representative-sequences Folder_Apprill_transmissionMucus2019/rep_$fn.qza --o-table Folder_Apprill_transmissionMucus2019/table_$fn.qza --o-denoising-stats Folder_Apprill_transmissionMucus2019/dada_$fn.qza \n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#Merge seqs\n",
    "!qiime feature-table merge-seqs \\\n",
    "--i-data  Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus_LibI.qza \\\n",
    "--i-data  Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus_LibJ.qza \\\n",
    "--i-data  Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus_LibK.qza \\\n",
    "--i-data  Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus_LibL.qza \\\n",
    "--i-data  Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus_LibM.qza \\\n",
    "--o-merged-data Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_Apprill_transmissionMucus2019/table_Apprill_2019TransmissionExpMucus.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#Merge tables\n",
    "!qiime feature-table merge \\\n",
    " --i-tables Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibI.qza \\\n",
    " --i-tables Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibJ.qza \\\n",
    " --i-tables Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibK.qza \\\n",
    " --i-tables Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibL.qza \\\n",
    " --i-tables Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibM.qza \\\n",
    " --o-merged-table Folder_Apprill_transmissionMucus2019/table_Apprill_2019TransmissionExpMucus.qza \\\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/dada_trim_Import_files_Apprill_2019TransmissionExpMucus_LibI.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/dada_trim_Import_files_Apprill_2019TransmissionExpMucus_LibJ.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/dada_trim_Import_files_Apprill_2019TransmissionExpMucus_LibK.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/dada_trim_Import_files_Apprill_2019TransmissionExpMucus_LibL.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/dada_trim_Import_files_Apprill_2019TransmissionExpMucus_LibM.qzv\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for f1 in Folder_Apprill_transmissionMucus2019/dada_trim_Import_files_Apprill_2019TransmissionExpMucus_Lib*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime metadata tabulate \\\n",
    "  --m-input-file $f1 \\\n",
    "  --o-visualization Folder_Apprill_transmissionMucus2019/$fn\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibI.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibJ.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibK.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibL.qzv\n",
      "Saved Visualization to: Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_LibM.qzv\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for f1 in Folder_Apprill_transmissionMucus2019/table_trim_Import_files_Apprill_2019TransmissionExpMucus_Lib*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime metadata tabulate \\\n",
    "  --m-input-file $f1 \\\n",
    "  --o-visualization Folder_Apprill_transmissionMucus2019/$fn\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_Apprill_transmissionMucus2019/table_Apprill_2019TransmissionExpMucus.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file Folder_Apprill_transmissionMucus2019/table_Apprill_2019TransmissionExpMucus.qza \\\n",
    "  --o-visualization Folder_Apprill_transmissionMucus2019/table_Apprill_2019TransmissionExpMucus.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Taxonomy] to: Folder_Apprill_transmissionMucus2019/taxonomy_Apprill_2019TransmissionExpMucus.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier /space/home/rosales/databases/SILVA_DBs/silva-138-99-nb-classifier.qza \\\n",
    "  --i-reads Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus.qza \\\n",
    "  --p-n-jobs -7 \\\n",
    "  --o-classification Folder_Apprill_transmissionMucus2019/taxonomy_Apprill_2019TransmissionExpMucus.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_Apprill_transmissionMucus2019/taxonomy__Apprill_2019TransmissionExpMucus.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file Folder_Apprill_transmissionMucus2019/taxonomy_Apprill_2019TransmissionExpMucus.qza \\\n",
    "  --o-visualization Folder_Apprill_transmissionMucus2019/taxonomy__Apprill_2019TransmissionExpMucus.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_Apprill_transmissionMucus2019/table-BacArc_Apprill_2019TransmissionExpMucus.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime taxa filter-table \\\n",
    "--i-table Folder_Apprill_transmissionMucus2019/table_Apprill_2019TransmissionExpMucus.qza \\\n",
    "--i-taxonomy Folder_Apprill_transmissionMucus2019/taxonomy_Apprill_2019TransmissionExpMucus.qza \\\n",
    "--p-exclude mitochondria,chloroplast \\\n",
    "--p-include d__Bacteria,d__Archaea \\\n",
    "--o-filtered-table Folder_Apprill_transmissionMucus2019/table-BacArc_Apprill_2019TransmissionExpMucus.qza\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_Apprill_transmissionMucus2019/rep-BacArc_Apprill_2019TransmissionExpMucus.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime taxa filter-seqs \\\n",
    "--i-sequences Folder_Apprill_transmissionMucus2019/rep_trim_Import_files_Apprill_2019TransmissionExpMucus.qza \\\n",
    "--i-taxonomy Folder_Apprill_transmissionMucus2019/taxonomy_Apprill_2019TransmissionExpMucus.qza \\\n",
    "--p-exclude mitochondria,chloroplast \\\n",
    "--p-include d__Bacteria,d__Archaea \\\n",
    "--o-filtered-sequences Folder_Apprill_transmissionMucus2019/rep-BacArc_Apprill_2019TransmissionExpMucus.qza\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
