{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/projects/2020_SCTLD_meta-analysis\n",
      "/space/home/rosales/anaconda2/envs/qiime2-2020.2/bin/qiime\r\n"
     ]
    }
   ],
   "source": [
    "%cd /space/home/rosales/projects/2020_SCTLD_meta-analysis\n",
    "!which qiime"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam145-192 as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam145-192.qza\u001b[0m\n",
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam1-48/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam1-48.qza\u001b[0m\n",
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam193-240/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam193-240.qza\u001b[0m\n",
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam241-288/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam241-288.qza\u001b[0m\n",
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam289-336/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam289-336.qza\u001b[0m\n",
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam337-387/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam337-387.qza\u001b[0m\n",
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam49-96/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam49-96.qza\u001b[0m\n",
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam97-144/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam97-144.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam145-192 \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam145-192.qza\n",
    "\n",
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam1-48/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam1-48.qza\n",
    "\n",
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam193-240/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam193-240.qza\n",
    "\n",
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam241-288/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam241-288.qza\n",
    "\n",
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam289-336/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam289-336.qza\n",
    "\n",
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam337-387/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam337-387.qza\n",
    "\n",
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam49-96/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam49-96.qza\n",
    "\n",
    "\n",
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sam97-144/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_sam97-144.qza\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGmicro2018_145-192.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGmicro2018_145-192.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGmicrobiome2018_sam145-192.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sample-metadata-Sam145-192.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGmicro2018_145-192.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGmicro2018_145-192"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGmicro2018_1-48.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGmicro2018_1-48.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGmicrobiome2018_sam1-48.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sample-metadata-Sam1-48.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGmicro2018_1-48.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGmicro2018_1-48"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGmicro2018_193-240.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGmicro2018_193-240.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGmicrobiome2018_sam193-240.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sample-metadata-Sam193-240.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGmicro2018_193-240.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGmicro2018_193-240"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGmicro2018_241-288.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGmicro2018_241-288.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGmicrobiome2018_sam241-288.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sample-metadata-Sam241-288.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGmicro2018_241-288.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGmicro2018_241-288"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGmicro2018_289-336.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGmicro2018_289-336.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGmicrobiome2018_sam289-336.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sample-metadata-Sam289-336.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGmicro2018_289-336.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGmicro2018_289-336\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGmicro2018_337-38.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGmicro2018_337-38.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGmicrobiome2018_sam337-387.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sample-metadata-Sam337-387.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGmicro2018_337-38.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGmicro2018_337-38"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGmicro2018_49-96.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGmicro2018_49-96.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGmicrobiome2018_sam49-96.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sample-metadata-Sam49-96.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGmicro2018_49-96.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGmicro2018_49-96\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGmicro2018_97-144.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGmicro2018_97-144.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGmicrobiome2018_sam97-144.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGmicrobiome2018_fastqfiles/sample-metadata-Sam97-144.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGmicro2018_97-144.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGmicro2018_97-144\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "for f1 in Folder_SWGmicrobiome2018/demux_Muller_SWGmicro2018_*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data $f1 \\\n",
    "--o-visualization Folder_SWGmicrobiome2018/$fn\n",
    "\n",
    "done\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/391.D_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/391.D_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/391.D_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/391.D_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.64 s (20 us/read; 3.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            131,855\n",
      "  Read 1 with adapter:                      14 (0.0%)\n",
      "  Read 2 with adapter:                   4,562 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       131,855 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,779,110 bp\n",
      "  Read 1:    29,415,805 bp\n",
      "  Read 2:    30,363,305 bp\n",
      "Total written (filtered):     59,722,847 bp (99.9%)\n",
      "  Read 1:    29,415,549 bp\n",
      "  Read 2:    30,307,298 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t2060.2\t0\t13\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4562 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t2060.2\t0\t29\n",
      "4\t11\t515.1\t0\t11\n",
      "5\t18\t128.8\t0\t18\n",
      "6\t29\t32.2\t0\t29\n",
      "7\t140\t8.0\t0\t140\n",
      "8\t211\t2.0\t0\t211\n",
      "9\t865\t0.5\t0\t863 2\n",
      "10\t1604\t0.1\t1\t1596 8\n",
      "11\t101\t0.0\t1\t101\n",
      "12\t43\t0.0\t1\t43\n",
      "13\t51\t0.0\t1\t51\n",
      "14\t49\t0.0\t1\t49\n",
      "15\t101\t0.0\t1\t101\n",
      "16\t113\t0.0\t1\t111 2\n",
      "17\t231\t0.0\t1\t227 4\n",
      "18\t290\t0.0\t1\t280 8 2\n",
      "19\t578\t0.0\t1\t409 168 1\n",
      "20\t15\t0.0\t2\t3 0 12\n",
      "21\t75\t0.0\t2\t0 75\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "60\t1\t0.0\t2\t1\n",
      "102\t1\t0.0\t2\t0 1\n",
      "183\t1\t0.0\t2\t1\n",
      "193\t1\t0.0\t2\t1\n",
      "212\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/392.D_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/392.D_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/392.D_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/392.D_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.00 s (44 us/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,802\n",
      "  Read 1 with adapter:                      15 (0.1%)\n",
      "  Read 2 with adapter:                     209 (0.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,802 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,350,117 bp\n",
      "  Read 1:     5,094,801 bp\n",
      "  Read 2:     5,255,316 bp\n",
      "Total written (filtered):     10,346,614 bp (100.0%)\n",
      "  Read 1:     5,094,756 bp\n",
      "  Read 2:     5,251,858 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 15 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t356.3\t0\t15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 209 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t356.3\t0\t1\n",
      "8\t2\t0.3\t0\t2\n",
      "9\t5\t0.1\t0\t5\n",
      "10\t22\t0.0\t1\t22\n",
      "11\t3\t0.0\t1\t2 1\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t5\t0.0\t1\t5\n",
      "14\t4\t0.0\t1\t4\n",
      "15\t10\t0.0\t1\t10\n",
      "16\t14\t0.0\t1\t13 1\n",
      "17\t14\t0.0\t1\t12 2\n",
      "18\t34\t0.0\t1\t34\n",
      "19\t67\t0.0\t1\t48 19\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t19\t0.0\t2\t0 19\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/392.U_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/392.U_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/392.U_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/392.U_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.32 s (21 us/read; 2.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,749\n",
      "  Read 1 with adapter:                      43 (0.0%)\n",
      "  Read 2 with adapter:                   1,511 (1.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,749 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,333,162 bp\n",
      "  Read 1:    24,287,574 bp\n",
      "  Read 2:    25,045,588 bp\n",
      "Total written (filtered):     49,309,544 bp (100.0%)\n",
      "  Read 1:    24,286,919 bp\n",
      "  Read 2:    25,022,625 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 43 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t1699.2\t0\t40\n",
      "113\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t0 1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1511 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1699.2\t0\t8\n",
      "5\t5\t106.2\t0\t5\n",
      "6\t7\t26.6\t0\t7\n",
      "7\t25\t6.6\t0\t25\n",
      "8\t31\t1.7\t0\t31\n",
      "9\t109\t0.4\t0\t109\n",
      "10\t283\t0.1\t1\t282 1\n",
      "11\t18\t0.0\t1\t18\n",
      "12\t20\t0.0\t1\t20\n",
      "13\t14\t0.0\t1\t14\n",
      "14\t28\t0.0\t1\t28\n",
      "15\t43\t0.0\t1\t43\n",
      "16\t62\t0.0\t1\t60 2\n",
      "17\t143\t0.0\t1\t140 3\n",
      "18\t198\t0.0\t1\t187 8 3\n",
      "19\t425\t0.0\t1\t279 146\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t79\t0.0\t2\t0 78 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/393.H_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/393.H_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/393.H_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/393.H_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.73 s (34 us/read; 1.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,082\n",
      "  Read 1 with adapter:                      24 (0.0%)\n",
      "  Read 2 with adapter:                     762 (1.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,082 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,120,281 bp\n",
      "  Read 1:    11,369,418 bp\n",
      "  Read 2:    11,750,863 bp\n",
      "Total written (filtered):     23,107,698 bp (99.9%)\n",
      "  Read 1:    11,369,062 bp\n",
      "  Read 2:    11,738,636 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 24 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t798.2\t0\t21\n",
      "38\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 762 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t798.2\t0\t1\n",
      "4\t1\t199.5\t0\t1\n",
      "6\t2\t12.5\t0\t2\n",
      "7\t8\t3.1\t0\t8\n",
      "8\t6\t0.8\t0\t6\n",
      "9\t44\t0.2\t0\t44\n",
      "10\t80\t0.0\t1\t80\n",
      "11\t10\t0.0\t1\t10\n",
      "12\t23\t0.0\t1\t23\n",
      "13\t27\t0.0\t1\t27\n",
      "14\t33\t0.0\t1\t30 3\n",
      "15\t41\t0.0\t1\t40 1\n",
      "16\t39\t0.0\t1\t37 2\n",
      "17\t102\t0.0\t1\t100 2\n",
      "18\t89\t0.0\t1\t81 5 3\n",
      "19\t215\t0.0\t1\t148 67\n",
      "20\t6\t0.0\t2\t0 1 5\n",
      "21\t34\t0.0\t2\t0 34\n",
      "215\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/394.H_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/394.H_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/394.H_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/394.H_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.25 s (35 us/read; 1.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,732\n",
      "  Read 1 with adapter:                      21 (0.1%)\n",
      "  Read 2 with adapter:                   1,074 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,732 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,066,771 bp\n",
      "  Read 1:     7,894,630 bp\n",
      "  Read 2:     8,172,141 bp\n",
      "Total written (filtered):     16,052,028 bp (99.9%)\n",
      "  Read 1:     7,894,184 bp\n",
      "  Read 2:     8,157,844 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t558.3\t0\t19\n",
      "172\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1074 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t558.3\t0\t6\n",
      "4\t3\t139.6\t0\t3\n",
      "5\t2\t34.9\t0\t2\n",
      "6\t6\t8.7\t0\t6\n",
      "7\t24\t2.2\t0\t24\n",
      "8\t55\t0.5\t0\t55\n",
      "9\t168\t0.1\t0\t166 2\n",
      "10\t264\t0.0\t1\t264\n",
      "11\t22\t0.0\t1\t21 1\n",
      "12\t16\t0.0\t1\t16\n",
      "13\t19\t0.0\t1\t19\n",
      "14\t35\t0.0\t1\t35\n",
      "15\t34\t0.0\t1\t34\n",
      "16\t41\t0.0\t1\t41\n",
      "17\t90\t0.0\t1\t87 3\n",
      "18\t73\t0.0\t1\t68 3 2\n",
      "19\t182\t0.0\t1\t110 71 1\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t28\t0.0\t2\t0 28\n",
      "134\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/395.H_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/395.H_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/395.H_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/395.H_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.64 s (25 us/read; 2.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,193\n",
      "  Read 1 with adapter:                      31 (0.0%)\n",
      "  Read 2 with adapter:                   2,988 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       107,193 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,612,666 bp\n",
      "  Read 1:    23,896,504 bp\n",
      "  Read 2:    24,716,162 bp\n",
      "Total written (filtered):     48,572,804 bp (99.9%)\n",
      "  Read 1:    23,896,393 bp\n",
      "  Read 2:    24,676,411 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 31 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t1674.9\t0\t30\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2988 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t1674.9\t0\t25\n",
      "4\t5\t418.7\t0\t5\n",
      "5\t9\t104.7\t0\t9\n",
      "6\t12\t26.2\t0\t12\n",
      "7\t92\t6.5\t0\t92\n",
      "8\t133\t1.6\t0\t133\n",
      "9\t469\t0.4\t0\t468 1\n",
      "10\t728\t0.1\t1\t720 8\n",
      "11\t45\t0.0\t1\t44 1\n",
      "12\t52\t0.0\t1\t52\n",
      "13\t42\t0.0\t1\t41 1\n",
      "14\t63\t0.0\t1\t61 2\n",
      "15\t114\t0.0\t1\t114\n",
      "16\t132\t0.0\t1\t126 6\n",
      "17\t220\t0.0\t1\t213 7\n",
      "18\t249\t0.0\t1\t238 9 2\n",
      "19\t502\t0.0\t1\t312 190\n",
      "20\t14\t0.0\t2\t0 0 14\n",
      "21\t80\t0.0\t2\t0 79 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/396.D_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/396.D_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/396.D_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/396.D_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.82 s (28 us/read; 2.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,621\n",
      "  Read 1 with adapter:                      56 (0.1%)\n",
      "  Read 2 with adapter:                   1,389 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,621 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,314,059 bp\n",
      "  Read 1:    14,412,115 bp\n",
      "  Read 2:    14,901,944 bp\n",
      "Total written (filtered):     29,294,486 bp (99.9%)\n",
      "  Read 1:    14,411,947 bp\n",
      "  Read 2:    14,882,539 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 56 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t1009.7\t0\t56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1389 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1009.7\t0\t3\n",
      "4\t1\t252.4\t0\t1\n",
      "5\t3\t63.1\t0\t3\n",
      "6\t6\t15.8\t0\t6\n",
      "7\t26\t3.9\t0\t26\n",
      "8\t38\t1.0\t0\t38\n",
      "9\t148\t0.2\t0\t148\n",
      "10\t359\t0.1\t1\t359\n",
      "11\t34\t0.0\t1\t34\n",
      "12\t27\t0.0\t1\t27\n",
      "13\t18\t0.0\t1\t18\n",
      "14\t17\t0.0\t1\t17\n",
      "15\t53\t0.0\t1\t50 3\n",
      "16\t75\t0.0\t1\t70 5\n",
      "17\t128\t0.0\t1\t124 4\n",
      "18\t124\t0.0\t1\t120 4\n",
      "19\t268\t0.0\t1\t186 82\n",
      "20\t14\t0.0\t2\t0 0 14\n",
      "21\t46\t0.0\t2\t0 46\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/396.U_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/396.U_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/396.U_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/396.U_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.35 s (31 us/read; 1.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,158\n",
      "  Read 1 with adapter:                      17 (0.0%)\n",
      "  Read 2 with adapter:                     699 (1.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,158 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,961,038 bp\n",
      "  Read 1:     9,814,795 bp\n",
      "  Read 2:    10,146,243 bp\n",
      "Total written (filtered):     19,949,839 bp (99.9%)\n",
      "  Read 1:     9,814,035 bp\n",
      "  Read 2:    10,135,804 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 17 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t690.0\t0\t11\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 699 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t690.0\t0\t3\n",
      "4\t1\t172.5\t0\t1\n",
      "6\t5\t10.8\t0\t5\n",
      "7\t9\t2.7\t0\t9\n",
      "8\t14\t0.7\t0\t14\n",
      "9\t42\t0.2\t0\t42\n",
      "10\t134\t0.0\t1\t134\n",
      "11\t34\t0.0\t1\t34\n",
      "12\t25\t0.0\t1\t25\n",
      "13\t17\t0.0\t1\t17\n",
      "14\t18\t0.0\t1\t18\n",
      "15\t31\t0.0\t1\t30 1\n",
      "16\t35\t0.0\t1\t34 1\n",
      "17\t64\t0.0\t1\t62 2\n",
      "18\t88\t0.0\t1\t85 2 1\n",
      "19\t145\t0.0\t1\t106 39\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t28\t0.0\t2\t0 28\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/397.D_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/397.D_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/397.D_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/397.D_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.31 s (21 us/read; 2.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,553\n",
      "  Read 1 with adapter:                      53 (0.0%)\n",
      "  Read 2 with adapter:                   1,629 (1.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,553 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,398,291 bp\n",
      "  Read 1:    24,299,498 bp\n",
      "  Read 2:    25,098,793 bp\n",
      "Total written (filtered):     49,373,562 bp (99.9%)\n",
      "  Read 1:    24,299,316 bp\n",
      "  Read 2:    25,074,246 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 53 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t52\t1696.1\t0\t52\n",
      "26\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1629 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1696.1\t0\t9\n",
      "4\t2\t424.0\t0\t2\n",
      "5\t4\t106.0\t0\t4\n",
      "6\t2\t26.5\t0\t2\n",
      "7\t27\t6.6\t0\t27\n",
      "8\t32\t1.7\t0\t32\n",
      "9\t126\t0.4\t0\t126\n",
      "10\t309\t0.1\t1\t309\n",
      "11\t20\t0.0\t1\t20\n",
      "12\t24\t0.0\t1\t24\n",
      "13\t21\t0.0\t1\t21\n",
      "14\t27\t0.0\t1\t27\n",
      "15\t64\t0.0\t1\t62 2\n",
      "16\t83\t0.0\t1\t82 1\n",
      "17\t163\t0.0\t1\t157 6\n",
      "18\t190\t0.0\t1\t181 8 1\n",
      "19\t429\t0.0\t1\t283 145 1\n",
      "20\t16\t0.0\t2\t0 0 16\n",
      "21\t80\t0.0\t2\t0 80\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/397.U_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/397.U_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/397.U_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/397.U_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.16 s (34 us/read; 1.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,282\n",
      "  Read 1 with adapter:                      21 (0.1%)\n",
      "  Read 2 with adapter:                     870 (2.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,282 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,390,060 bp\n",
      "  Read 1:     7,559,422 bp\n",
      "  Read 2:     7,830,638 bp\n",
      "Total written (filtered):     15,376,940 bp (99.9%)\n",
      "  Read 1:     7,558,835 bp\n",
      "  Read 2:     7,818,105 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t535.7\t0\t18\n",
      "101\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 870 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t535.7\t0\t2\n",
      "5\t1\t33.5\t0\t1\n",
      "6\t3\t8.4\t0\t3\n",
      "7\t15\t2.1\t0\t15\n",
      "8\t21\t0.5\t0\t21\n",
      "9\t72\t0.1\t0\t72\n",
      "10\t182\t0.0\t1\t182\n",
      "11\t12\t0.0\t1\t12\n",
      "12\t19\t0.0\t1\t19\n",
      "13\t30\t0.0\t1\t30\n",
      "14\t37\t0.0\t1\t36 1\n",
      "15\t48\t0.0\t1\t47 1\n",
      "16\t37\t0.0\t1\t37\n",
      "17\t92\t0.0\t1\t87 5\n",
      "18\t105\t0.0\t1\t100 2 3\n",
      "19\t171\t0.0\t1\t100 70 1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t20\t0.0\t2\t0 20\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/398.D_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/398.D_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/398.D_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/398.D_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.00 s (26 us/read; 2.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             76,134\n",
      "  Read 1 with adapter:                      49 (0.1%)\n",
      "  Read 2 with adapter:                   1,214 (1.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        76,134 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,565,628 bp\n",
      "  Read 1:    17,002,052 bp\n",
      "  Read 2:    17,563,576 bp\n",
      "Total written (filtered):     34,546,831 bp (99.9%)\n",
      "  Read 1:    17,001,905 bp\n",
      "  Read 2:    17,544,926 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 49 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t1189.6\t0\t49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1214 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1189.6\t0\t5\n",
      "4\t1\t297.4\t0\t1\n",
      "5\t4\t74.3\t0\t4\n",
      "6\t2\t18.6\t0\t2\n",
      "7\t19\t4.6\t0\t19\n",
      "8\t16\t1.2\t0\t16\n",
      "9\t69\t0.3\t0\t68 1\n",
      "10\t179\t0.1\t1\t179\n",
      "11\t37\t0.0\t1\t37\n",
      "12\t33\t0.0\t1\t31 2\n",
      "13\t28\t0.0\t1\t28\n",
      "14\t31\t0.0\t1\t31\n",
      "15\t69\t0.0\t1\t69\n",
      "16\t73\t0.0\t1\t70 3\n",
      "17\t99\t0.0\t1\t95 4\n",
      "18\t184\t0.0\t1\t177 7\n",
      "19\t298\t0.0\t1\t202 95 1\n",
      "20\t7\t0.0\t2\t0 0 7\n",
      "21\t58\t0.0\t2\t0 58\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/398.U_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/398.U_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/398.U_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/398.U_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.25 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,704\n",
      "  Read 1 with adapter:                      43 (0.0%)\n",
      "  Read 2 with adapter:                   1,122 (1.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        96,704 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,981,813 bp\n",
      "  Read 1:    21,638,477 bp\n",
      "  Read 2:    22,343,336 bp\n",
      "Total written (filtered):     43,963,907 bp (100.0%)\n",
      "  Read 1:    21,638,348 bp\n",
      "  Read 2:    22,325,559 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 43 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t1511.0\t0\t43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1122 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1511.0\t0\t9\n",
      "4\t1\t377.8\t0\t1\n",
      "6\t4\t23.6\t0\t4\n",
      "7\t9\t5.9\t0\t9\n",
      "8\t11\t1.5\t0\t11\n",
      "9\t52\t0.4\t0\t52\n",
      "10\t135\t0.1\t1\t134 1\n",
      "11\t20\t0.0\t1\t20\n",
      "12\t32\t0.0\t1\t32\n",
      "13\t25\t0.0\t1\t25\n",
      "14\t30\t0.0\t1\t30\n",
      "15\t46\t0.0\t1\t45 1\n",
      "16\t58\t0.0\t1\t58\n",
      "17\t138\t0.0\t1\t136 2\n",
      "18\t167\t0.0\t1\t160 4 3\n",
      "19\t320\t0.0\t1\t233 86 1\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t54\t0.0\t2\t0 53 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/399.D_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/399.D_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/399.D_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/399.D_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.20 s (25 us/read; 2.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             87,071\n",
      "  Read 1 with adapter:                      42 (0.0%)\n",
      "  Read 2 with adapter:                     877 (1.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        87,071 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,624,033 bp\n",
      "  Read 1:    19,497,648 bp\n",
      "  Read 2:    20,126,385 bp\n",
      "Total written (filtered):     39,609,718 bp (100.0%)\n",
      "  Read 1:    19,497,522 bp\n",
      "  Read 2:    20,112,196 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 42 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t1360.5\t0\t42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 877 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1360.5\t0\t3\n",
      "4\t1\t340.1\t0\t1\n",
      "6\t2\t21.3\t0\t2\n",
      "7\t9\t5.3\t0\t9\n",
      "8\t5\t1.3\t0\t5\n",
      "9\t44\t0.3\t0\t43 1\n",
      "10\t76\t0.1\t1\t76\n",
      "11\t16\t0.0\t1\t16\n",
      "12\t24\t0.0\t1\t24\n",
      "13\t25\t0.0\t1\t25\n",
      "14\t23\t0.0\t1\t23\n",
      "15\t47\t0.0\t1\t47\n",
      "16\t41\t0.0\t1\t40 1\n",
      "17\t88\t0.0\t1\t87 1\n",
      "18\t146\t0.0\t1\t141 3 2\n",
      "19\t274\t0.0\t1\t203 70 1\n",
      "20\t10\t0.0\t2\t0 0 10\n",
      "21\t43\t0.0\t2\t0 42 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/399.U_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/399.U_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/399.U_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/399.U_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.22 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,998\n",
      "  Read 1 with adapter:                      57 (0.1%)\n",
      "  Read 2 with adapter:                   1,431 (1.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        90,998 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,409,121 bp\n",
      "  Read 1:    20,370,284 bp\n",
      "  Read 2:    21,038,837 bp\n",
      "Total written (filtered):     41,386,299 bp (99.9%)\n",
      "  Read 1:    20,369,389 bp\n",
      "  Read 2:    21,016,910 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 57 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t1421.8\t0\t50\n",
      "25\t1\t0.0\t1\t1\n",
      "120\t6\t0.0\t1\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1431 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1421.8\t0\t11\n",
      "4\t3\t355.5\t0\t3\n",
      "5\t2\t88.9\t0\t2\n",
      "6\t4\t22.2\t0\t4\n",
      "7\t10\t5.6\t0\t10\n",
      "8\t14\t1.4\t0\t14\n",
      "9\t75\t0.3\t0\t75\n",
      "10\t207\t0.1\t1\t206 1\n",
      "11\t51\t0.0\t1\t48 3\n",
      "12\t46\t0.0\t1\t46\n",
      "13\t27\t0.0\t1\t27\n",
      "14\t32\t0.0\t1\t32\n",
      "15\t86\t0.0\t1\t84 2\n",
      "16\t106\t0.0\t1\t104 2\n",
      "17\t143\t0.0\t1\t140 3\n",
      "18\t201\t0.0\t1\t199 2\n",
      "19\t341\t0.0\t1\t242 99\n",
      "20\t18\t0.0\t2\t0 0 18\n",
      "21\t53\t0.0\t2\t0 53\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/400.D_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/400.D_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/400.D_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/400.D_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.09 s (21 us/read; 2.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            147,960\n",
      "  Read 1 with adapter:                      64 (0.0%)\n",
      "  Read 2 with adapter:                   1,781 (1.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       147,960 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,236,667 bp\n",
      "  Read 1:    33,078,070 bp\n",
      "  Read 2:    34,158,597 bp\n",
      "Total written (filtered):     67,207,943 bp (100.0%)\n",
      "  Read 1:    33,077,816 bp\n",
      "  Read 2:    34,130,127 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 64 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t2311.9\t0\t61\n",
      "18\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1781 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t2311.9\t0\t6\n",
      "4\t1\t578.0\t0\t1\n",
      "5\t2\t144.5\t0\t2\n",
      "6\t3\t36.1\t0\t3\n",
      "7\t15\t9.0\t0\t15\n",
      "8\t15\t2.3\t0\t15\n",
      "9\t78\t0.6\t0\t78\n",
      "10\t242\t0.1\t1\t242\n",
      "11\t28\t0.0\t1\t28\n",
      "12\t34\t0.0\t1\t33 1\n",
      "13\t31\t0.0\t1\t31\n",
      "14\t35\t0.0\t1\t35\n",
      "15\t99\t0.0\t1\t97 2\n",
      "16\t92\t0.0\t1\t92\n",
      "17\t179\t0.0\t1\t174 5\n",
      "18\t272\t0.0\t1\t258 12 2\n",
      "19\t523\t0.0\t1\t338 185\n",
      "20\t17\t0.0\t2\t0 0 17\n",
      "21\t107\t0.0\t2\t0 106 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/391.D_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/391.D_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/391.D_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/391.D_7_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/392.D_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/392.D_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/392.D_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/392.D_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/392.U_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/392.U_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/392.U_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/392.U_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/393.H_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/393.H_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/393.H_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/393.H_18_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/394.H_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/394.H_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/394.H_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/394.H_29_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/395.H_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/395.H_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/395.H_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/395.H_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/396.D_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/396.D_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/396.D_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/396.D_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/396.U_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/396.U_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/396.U_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/396.U_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/397.D_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/397.D_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/397.D_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/397.D_12_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/397.U_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/397.U_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/397.U_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/397.U_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/398.D_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/398.D_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/398.D_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/398.D_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/398.U_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/398.U_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/398.U_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/398.U_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/399.D_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/399.D_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/399.D_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/399.D_2_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/399.U_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/399.U_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/399.U_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/399.U_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/400.D_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/400.D_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/400.D_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/400.D_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/400.U_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/400.U_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/400.U_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/400.U_45_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/400.U_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/400.U_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/400.U_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/400.U_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.42 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,675\n",
      "  Read 1 with adapter:                      26 (0.0%)\n",
      "  Read 2 with adapter:                     714 (1.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,675 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,920,245 bp\n",
      "  Read 1:    11,766,820 bp\n",
      "  Read 2:    12,153,425 bp\n",
      "Total written (filtered):     23,909,037 bp (100.0%)\n",
      "  Read 1:    11,766,720 bp\n",
      "  Read 2:    12,142,317 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 26 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t823.0\t0\t25\n",
      "25\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 714 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t823.0\t0\t5\n",
      "7\t6\t3.2\t0\t6\n",
      "8\t6\t0.8\t0\t6\n",
      "9\t30\t0.2\t0\t30\n",
      "10\t83\t0.1\t1\t83\n",
      "11\t33\t0.0\t1\t33\n",
      "12\t27\t0.0\t1\t27\n",
      "13\t19\t0.0\t1\t19\n",
      "14\t12\t0.0\t1\t12\n",
      "15\t46\t0.0\t1\t46\n",
      "16\t51\t0.0\t1\t48 3\n",
      "17\t90\t0.0\t1\t86 4\n",
      "18\t110\t0.0\t1\t108 2\n",
      "19\t164\t0.0\t1\t120 43 1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t29\t0.0\t2\t0 29\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/401.H_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/401.H_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/401.H_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/401.H_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.20 s (35 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,832\n",
      "  Read 1 with adapter:                      16 (0.0%)\n",
      "  Read 2 with adapter:                     770 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,832 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,203,456 bp\n",
      "  Read 1:     7,468,886 bp\n",
      "  Read 2:     7,734,570 bp\n",
      "Total written (filtered):     15,192,378 bp (99.9%)\n",
      "  Read 1:     7,468,790 bp\n",
      "  Read 2:     7,723,588 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t528.6\t0\t14\n",
      "27\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 770 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t528.6\t0\t2\n",
      "5\t1\t33.0\t0\t1\n",
      "6\t1\t8.3\t0\t1\n",
      "7\t14\t2.1\t0\t14\n",
      "8\t17\t0.5\t0\t17\n",
      "9\t83\t0.1\t0\t83\n",
      "10\t196\t0.0\t1\t196\n",
      "11\t14\t0.0\t1\t14\n",
      "12\t14\t0.0\t1\t14\n",
      "13\t10\t0.0\t1\t10\n",
      "14\t11\t0.0\t1\t11\n",
      "15\t32\t0.0\t1\t32\n",
      "16\t44\t0.0\t1\t43 1\n",
      "17\t67\t0.0\t1\t64 3\n",
      "18\t72\t0.0\t1\t67 2 3\n",
      "19\t163\t0.0\t1\t108 54 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t27\t0.0\t2\t0 27\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/402.H_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/402.H_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/402.H_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/402.H_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.20 s (39 us/read; 1.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,923\n",
      "  Read 1 with adapter:                      12 (0.0%)\n",
      "  Read 2 with adapter:                     634 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,923 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,943,441 bp\n",
      "  Read 1:     6,852,719 bp\n",
      "  Read 2:     7,090,722 bp\n",
      "Total written (filtered):     13,934,255 bp (99.9%)\n",
      "  Read 1:     6,852,683 bp\n",
      "  Read 2:     7,081,572 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t483.2\t0\t12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 634 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t483.2\t0\t3\n",
      "4\t2\t120.8\t0\t2\n",
      "6\t3\t7.5\t0\t3\n",
      "7\t8\t1.9\t0\t8\n",
      "8\t14\t0.5\t0\t14\n",
      "9\t57\t0.1\t0\t57\n",
      "10\t130\t0.0\t1\t130\n",
      "11\t18\t0.0\t1\t18\n",
      "12\t12\t0.0\t1\t12\n",
      "13\t19\t0.0\t1\t19\n",
      "14\t14\t0.0\t1\t14\n",
      "15\t21\t0.0\t1\t20 1\n",
      "16\t32\t0.0\t1\t32\n",
      "17\t75\t0.0\t1\t71 4\n",
      "18\t79\t0.0\t1\t78 1\n",
      "19\t128\t0.0\t1\t85 43\n",
      "20\t6\t0.0\t2\t0 1 5\n",
      "21\t13\t0.0\t2\t0 12 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/403.H_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/403.H_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/403.H_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/403.H_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.95 s (45 us/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,137\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                     362 (1.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        21,137 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,465,867 bp\n",
      "  Read 1:     4,650,153 bp\n",
      "  Read 2:     4,815,714 bp\n",
      "Total written (filtered):      9,460,605 bp (99.9%)\n",
      "  Read 1:     4,650,129 bp\n",
      "  Read 2:     4,810,476 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t330.3\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 362 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t330.3\t0\t2\n",
      "7\t4\t1.3\t0\t4\n",
      "8\t6\t0.3\t0\t6\n",
      "9\t36\t0.1\t0\t36\n",
      "10\t76\t0.0\t1\t76\n",
      "11\t15\t0.0\t1\t15\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t9\t0.0\t1\t9\n",
      "14\t6\t0.0\t1\t6\n",
      "15\t19\t0.0\t1\t19\n",
      "16\t24\t0.0\t1\t24\n",
      "17\t22\t0.0\t1\t22\n",
      "18\t45\t0.0\t1\t44 0 1\n",
      "19\t80\t0.0\t1\t58 22\n",
      "21\t13\t0.0\t2\t0 12 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/404.D_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/404.D_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/404.D_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/404.D_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.89 s (20 us/read; 2.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            238,467\n",
      "  Read 1 with adapter:                     125 (0.1%)\n",
      "  Read 2 with adapter:                   9,656 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       238,467 (100.0%)\n",
      "\n",
      "Total basepairs processed:   108,655,102 bp\n",
      "  Read 1:    53,387,290 bp\n",
      "  Read 2:    55,267,812 bp\n",
      "Total written (filtered):    108,535,459 bp (99.9%)\n",
      "  Read 1:    53,386,440 bp\n",
      "  Read 2:    55,149,019 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 125 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t119\t3726.0\t0\t119\n",
      "27\t3\t0.0\t1\t3\n",
      "119\t2\t0.0\t1\t2\n",
      "174\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9656 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t55\t3726.0\t0\t55\n",
      "4\t8\t931.5\t0\t8\n",
      "5\t28\t232.9\t0\t28\n",
      "6\t78\t58.2\t0\t78\n",
      "7\t383\t14.6\t0\t383\n",
      "8\t515\t3.6\t0\t515\n",
      "9\t2051\t0.9\t0\t2050 1\n",
      "10\t2796\t0.2\t1\t2781 15\n",
      "11\t131\t0.1\t1\t129 2\n",
      "12\t133\t0.0\t1\t133\n",
      "13\t142\t0.0\t1\t140 2\n",
      "14\t196\t0.0\t1\t195 1\n",
      "15\t299\t0.0\t1\t289 10\n",
      "16\t271\t0.0\t1\t259 12\n",
      "17\t550\t0.0\t1\t535 15\n",
      "18\t575\t0.0\t1\t546 22 7\n",
      "19\t1209\t0.0\t1\t772 431 6\n",
      "20\t37\t0.0\t2\t0 3 34\n",
      "21\t189\t0.0\t2\t0 188 1\n",
      "22\t3\t0.0\t2\t0 1 2\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t1\n",
      "80\t1\t0.0\t2\t1\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t1\n",
      "194\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/404.U_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/404.U_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/404.U_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/404.U_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.33 s (19 us/read; 3.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            229,301\n",
      "  Read 1 with adapter:                     146 (0.1%)\n",
      "  Read 2 with adapter:                   7,977 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       229,301 (100.0%)\n",
      "\n",
      "Total basepairs processed:   104,473,382 bp\n",
      "  Read 1:    51,349,942 bp\n",
      "  Read 2:    53,123,440 bp\n",
      "Total written (filtered):    104,371,301 bp (99.9%)\n",
      "  Read 1:    51,348,964 bp\n",
      "  Read 2:    53,022,337 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 146 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t138\t3582.8\t0\t138\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t2\t0.0\t1\t2\n",
      "101\t2\t0.0\t1\t2\n",
      "106\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7977 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t3582.8\t0\t44\n",
      "4\t9\t895.7\t0\t9\n",
      "5\t19\t223.9\t0\t19\n",
      "6\t58\t56.0\t0\t58\n",
      "7\t303\t14.0\t0\t303\n",
      "8\t402\t3.5\t0\t402\n",
      "9\t1493\t0.9\t0\t1491 2\n",
      "10\t2133\t0.2\t1\t2125 8\n",
      "11\t98\t0.1\t1\t98\n",
      "12\t153\t0.0\t1\t153\n",
      "13\t142\t0.0\t1\t142\n",
      "14\t183\t0.0\t1\t173 10\n",
      "15\t266\t0.0\t1\t260 6\n",
      "16\t281\t0.0\t1\t262 19\n",
      "17\t539\t0.0\t1\t517 22\n",
      "18\t585\t0.0\t1\t541 27 17\n",
      "19\t1084\t0.0\t1\t709 370 5\n",
      "20\t24\t0.0\t2\t0 0 24\n",
      "21\t156\t0.0\t2\t0 154 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/405.D_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/405.D_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/405.D_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/405.D_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.60 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            158,994\n",
      "  Read 1 with adapter:                     107 (0.1%)\n",
      "  Read 2 with adapter:                   5,931 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       158,994 (100.0%)\n",
      "\n",
      "Total basepairs processed:    72,447,990 bp\n",
      "  Read 1:    35,604,612 bp\n",
      "  Read 2:    36,843,378 bp\n",
      "Total written (filtered):     72,373,338 bp (99.9%)\n",
      "  Read 1:    35,604,027 bp\n",
      "  Read 2:    36,769,311 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 107 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t100\t2484.3\t0\t100\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "38\t2\t0.0\t1\t2\n",
      "106\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5931 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t2484.3\t0\t39\n",
      "4\t7\t621.1\t0\t7\n",
      "5\t18\t155.3\t0\t18\n",
      "6\t46\t38.8\t0\t46\n",
      "7\t207\t9.7\t0\t207\n",
      "8\t326\t2.4\t0\t326\n",
      "9\t1164\t0.6\t0\t1163 1\n",
      "10\t1625\t0.2\t1\t1618 7\n",
      "11\t92\t0.0\t1\t92\n",
      "12\t81\t0.0\t1\t79 2\n",
      "13\t87\t0.0\t1\t84 3\n",
      "14\t155\t0.0\t1\t150 5\n",
      "15\t192\t0.0\t1\t189 3\n",
      "16\t212\t0.0\t1\t208 4\n",
      "17\t399\t0.0\t1\t392 7\n",
      "18\t419\t0.0\t1\t401 11 7\n",
      "19\t775\t0.0\t1\t461 311 3\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t71\t0.0\t2\t0 70 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/405.U_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/405.U_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/405.U_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/405.U_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.44 s (18 us/read; 3.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            186,472\n",
      "  Read 1 with adapter:                     134 (0.1%)\n",
      "  Read 2 with adapter:                   5,067 (2.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       186,472 (100.0%)\n",
      "\n",
      "Total basepairs processed:    84,920,301 bp\n",
      "  Read 1:    41,750,318 bp\n",
      "  Read 2:    43,169,983 bp\n",
      "Total written (filtered):     84,851,789 bp (99.9%)\n",
      "  Read 1:    41,749,748 bp\n",
      "  Read 2:    43,102,041 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 134 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t127\t2913.6\t0\t127\n",
      "27\t7\t0.0\t1\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5067 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t2913.6\t0\t18\n",
      "4\t2\t728.4\t0\t2\n",
      "5\t9\t182.1\t0\t9\n",
      "6\t23\t45.5\t0\t23\n",
      "7\t138\t11.4\t0\t138\n",
      "8\t202\t2.8\t0\t202\n",
      "9\t784\t0.7\t0\t782 2\n",
      "10\t1246\t0.2\t1\t1238 8\n",
      "11\t69\t0.0\t1\t67 2\n",
      "12\t105\t0.0\t1\t101 4\n",
      "13\t113\t0.0\t1\t112 1\n",
      "14\t148\t0.0\t1\t145 3\n",
      "15\t186\t0.0\t1\t182 4\n",
      "16\t192\t0.0\t1\t190 2\n",
      "17\t395\t0.0\t1\t384 11\n",
      "18\t433\t0.0\t1\t413 16 4\n",
      "19\t845\t0.0\t1\t559 282 4\n",
      "20\t29\t0.0\t2\t0 0 29\n",
      "21\t127\t0.0\t2\t0 125 2\n",
      "102\t1\t0.0\t2\t0 1\n",
      "132\t2\t0.0\t2\t0 2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/406.D_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/406.D_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/406.D_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/406.D_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.01 s (19 us/read; 3.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            206,237\n",
      "  Read 1 with adapter:                     118 (0.1%)\n",
      "  Read 2 with adapter:                  10,351 (5.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       206,237 (100.0%)\n",
      "\n",
      "Total basepairs processed:    94,024,827 bp\n",
      "  Read 1:    46,182,265 bp\n",
      "  Read 2:    47,842,562 bp\n",
      "Total written (filtered):     93,903,182 bp (99.9%)\n",
      "  Read 1:    46,181,160 bp\n",
      "  Read 2:    47,722,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 118 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t106\t3222.5\t0\t106\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t3\t0.0\t1\t3\n",
      "39\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "135\t1\t0.0\t1\t0 1\n",
      "168\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10351 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t3222.5\t0\t61\n",
      "4\t14\t805.6\t0\t14\n",
      "5\t50\t201.4\t0\t50\n",
      "6\t100\t50.4\t0\t100\n",
      "7\t492\t12.6\t0\t492\n",
      "8\t743\t3.1\t0\t743\n",
      "9\t2561\t0.8\t0\t2558 3\n",
      "10\t2986\t0.2\t1\t2973 13\n",
      "11\t111\t0.0\t1\t110 1\n",
      "12\t114\t0.0\t1\t113 1\n",
      "13\t100\t0.0\t1\t99 1\n",
      "14\t220\t0.0\t1\t215 5\n",
      "15\t262\t0.0\t1\t253 9\n",
      "16\t265\t0.0\t1\t256 9\n",
      "17\t526\t0.0\t1\t512 14\n",
      "18\t560\t0.0\t1\t517 35 8\n",
      "19\t1025\t0.0\t1\t633 392\n",
      "20\t30\t0.0\t2\t0 0 30\n",
      "21\t129\t0.0\t2\t0 128 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/406.U_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/406.U_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/406.U_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/406.U_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.04 s (20 us/read; 3.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            152,605\n",
      "  Read 1 with adapter:                      95 (0.1%)\n",
      "  Read 2 with adapter:                   4,477 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       152,605 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,506,035 bp\n",
      "  Read 1:    34,169,830 bp\n",
      "  Read 2:    35,336,205 bp\n",
      "Total written (filtered):     69,446,446 bp (99.9%)\n",
      "  Read 1:    34,168,976 bp\n",
      "  Read 2:    35,277,470 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 95 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t2384.5\t0\t88\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t0 1\n",
      "125\t2\t0.0\t1\t2\n",
      "132\t2\t0.0\t1\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4477 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t2384.5\t0\t35\n",
      "4\t7\t596.1\t0\t7\n",
      "5\t13\t149.0\t0\t13\n",
      "6\t28\t37.3\t0\t28\n",
      "7\t123\t9.3\t0\t123\n",
      "8\t193\t2.3\t0\t193\n",
      "9\t728\t0.6\t0\t727 1\n",
      "10\t1125\t0.1\t1\t1122 3\n",
      "11\t80\t0.0\t1\t79 1\n",
      "12\t97\t0.0\t1\t94 3\n",
      "13\t99\t0.0\t1\t99\n",
      "14\t123\t0.0\t1\t122 1\n",
      "15\t166\t0.0\t1\t163 3\n",
      "16\t140\t0.0\t1\t134 6\n",
      "17\t331\t0.0\t1\t322 9\n",
      "18\t351\t0.0\t1\t333 11 7\n",
      "19\t719\t0.0\t1\t474 245\n",
      "20\t18\t0.0\t2\t0 0 18\n",
      "21\t98\t0.0\t2\t0 98\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/407.D_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/407.D_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/407.D_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/407.D_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.12 s (24 us/read; 2.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            131,487\n",
      "  Read 1 with adapter:                      95 (0.1%)\n",
      "  Read 2 with adapter:                   6,403 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       131,487 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,931,567 bp\n",
      "  Read 1:    29,437,237 bp\n",
      "  Read 2:    30,494,330 bp\n",
      "Total written (filtered):     59,853,954 bp (99.9%)\n",
      "  Read 1:    29,436,760 bp\n",
      "  Read 2:    30,417,194 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 95 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t89\t2054.5\t0\t89\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "84\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6403 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t2054.5\t0\t44\n",
      "4\t5\t513.6\t0\t5\n",
      "5\t14\t128.4\t0\t14\n",
      "6\t67\t32.1\t0\t67\n",
      "7\t312\t8.0\t0\t312\n",
      "8\t399\t2.0\t0\t399\n",
      "9\t1415\t0.5\t0\t1414 1\n",
      "10\t1677\t0.1\t1\t1670 7\n",
      "11\t93\t0.0\t1\t91 2\n",
      "12\t109\t0.0\t1\t108 1\n",
      "13\t101\t0.0\t1\t99 2\n",
      "14\t187\t0.0\t1\t183 4\n",
      "15\t191\t0.0\t1\t186 5\n",
      "16\t220\t0.0\t1\t209 11\n",
      "17\t408\t0.0\t1\t393 15\n",
      "18\t360\t0.0\t1\t334 14 12\n",
      "19\t682\t0.0\t1\t427 255\n",
      "20\t18\t0.0\t2\t0 1 17\n",
      "21\t99\t0.0\t2\t0 97 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/407.U_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/407.U_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/407.U_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/407.U_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.78 s (19 us/read; 3.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            197,339\n",
      "  Read 1 with adapter:                     131 (0.1%)\n",
      "  Read 2 with adapter:                   6,470 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       197,339 (100.0%)\n",
      "\n",
      "Total basepairs processed:    89,884,665 bp\n",
      "  Read 1:    44,175,438 bp\n",
      "  Read 2:    45,709,227 bp\n",
      "Total written (filtered):     89,800,140 bp (99.9%)\n",
      "  Read 1:    44,174,184 bp\n",
      "  Read 2:    45,625,956 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 131 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t115\t3083.4\t0\t115\n",
      "22\t1\t0.0\t1\t1\n",
      "23\t2\t0.0\t1\t2\n",
      "25\t2\t0.0\t1\t1 1\n",
      "27\t7\t0.0\t1\t7\n",
      "120\t2\t0.0\t1\t2\n",
      "181\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6470 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t3083.4\t0\t35\n",
      "4\t7\t770.9\t0\t7\n",
      "5\t11\t192.7\t0\t11\n",
      "6\t45\t48.2\t0\t45\n",
      "7\t241\t12.0\t0\t241\n",
      "8\t342\t3.0\t0\t342\n",
      "9\t1225\t0.8\t0\t1224 1\n",
      "10\t1737\t0.2\t1\t1731 6\n",
      "11\t94\t0.0\t1\t94\n",
      "12\t100\t0.0\t1\t100\n",
      "13\t99\t0.0\t1\t98 1\n",
      "14\t139\t0.0\t1\t138 1\n",
      "15\t222\t0.0\t1\t214 8\n",
      "16\t208\t0.0\t1\t202 6\n",
      "17\t426\t0.0\t1\t414 12\n",
      "18\t470\t0.0\t1\t443 20 7\n",
      "19\t920\t0.0\t1\t606 311 3\n",
      "20\t17\t0.0\t2\t0 1 16\n",
      "21\t122\t0.0\t2\t0 122\n",
      "109\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "167\t1\t0.0\t2\t1\n",
      "168\t1\t0.0\t2\t1\n",
      "205\t1\t0.0\t2\t0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "211\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "231\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/408.D_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/408.D_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/408.D_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/408.D_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.33 s (19 us/read; 3.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            226,491\n",
      "  Read 1 with adapter:                     180 (0.1%)\n",
      "  Read 2 with adapter:                  10,758 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       226,491 (100.0%)\n",
      "\n",
      "Total basepairs processed:   103,244,171 bp\n",
      "  Read 1:    50,716,967 bp\n",
      "  Read 2:    52,527,204 bp\n",
      "Total written (filtered):    103,115,967 bp (99.9%)\n",
      "  Read 1:    50,716,254 bp\n",
      "  Read 2:    52,399,713 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 180 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t178\t3538.9\t0\t178\n",
      "27\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10758 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t3538.9\t0\t85\n",
      "4\t15\t884.7\t0\t15\n",
      "5\t27\t221.2\t0\t27\n",
      "6\t76\t55.3\t0\t76\n",
      "7\t475\t13.8\t0\t475\n",
      "8\t636\t3.5\t0\t636\n",
      "9\t2481\t0.9\t0\t2477 4\n",
      "10\t3325\t0.2\t1\t3309 16\n",
      "11\t124\t0.1\t1\t123 1\n",
      "12\t105\t0.0\t1\t104 1\n",
      "13\t132\t0.0\t1\t130 2\n",
      "14\t193\t0.0\t1\t189 4\n",
      "15\t271\t0.0\t1\t268 3\n",
      "16\t304\t0.0\t1\t298 6\n",
      "17\t560\t0.0\t1\t542 18\n",
      "18\t601\t0.0\t1\t552 25 24\n",
      "19\t1148\t0.0\t1\t730 415 3\n",
      "20\t35\t0.0\t2\t0 1 34\n",
      "21\t162\t0.0\t2\t0 162\n",
      "100\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/408.U_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/408.U_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/408.U_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/408.U_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.32 s (20 us/read; 2.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            211,477\n",
      "  Read 1 with adapter:                     117 (0.1%)\n",
      "  Read 2 with adapter:                   7,094 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       211,477 (100.0%)\n",
      "\n",
      "Total basepairs processed:    96,345,340 bp\n",
      "  Read 1:    47,357,099 bp\n",
      "  Read 2:    48,988,241 bp\n",
      "Total written (filtered):     96,255,401 bp (99.9%)\n",
      "  Read 1:    47,356,403 bp\n",
      "  Read 2:    48,898,998 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 117 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t111\t3304.3\t0\t111\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7094 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t3304.3\t0\t43\n",
      "4\t6\t826.1\t0\t6\n",
      "5\t23\t206.5\t0\t23\n",
      "6\t52\t51.6\t0\t52\n",
      "7\t308\t12.9\t0\t308\n",
      "8\t347\t3.2\t0\t347\n",
      "9\t1409\t0.8\t0\t1408 1\n",
      "10\t1929\t0.2\t1\t1923 6\n",
      "11\t99\t0.1\t1\t97 2\n",
      "12\t93\t0.0\t1\t92 1\n",
      "13\t96\t0.0\t1\t95 1\n",
      "14\t139\t0.0\t1\t137 2\n",
      "15\t208\t0.0\t1\t204 4\n",
      "16\t216\t0.0\t1\t205 11\n",
      "17\t435\t0.0\t1\t427 8\n",
      "18\t527\t0.0\t1\t498 17 12\n",
      "19\t994\t0.0\t1\t649 344 1\n",
      "20\t25\t0.0\t2\t0 1 24\n",
      "21\t136\t0.0\t2\t0 135 1\n",
      "22\t5\t0.0\t2\t0 0 5\n",
      "69\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/409.H_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/409.H_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/409.H_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/409.H_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.99 s (19 us/read; 3.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            269,585\n",
      "  Read 1 with adapter:                     138 (0.1%)\n",
      "  Read 2 with adapter:                   9,255 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       269,585 (100.0%)\n",
      "\n",
      "Total basepairs processed:   122,818,944 bp\n",
      "  Read 1:    60,365,863 bp\n",
      "  Read 2:    62,453,081 bp\n",
      "Total written (filtered):    122,699,834 bp (99.9%)\n",
      "  Read 1:    60,364,942 bp\n",
      "  Read 2:    62,334,892 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 138 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t132\t4212.3\t0\t132\n",
      "20\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "211\t1\t0.0\t1\t0 1\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9255 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t4212.3\t0\t49\n",
      "4\t9\t1053.1\t0\t9\n",
      "5\t23\t263.3\t0\t23\n",
      "6\t61\t65.8\t0\t61\n",
      "7\t300\t16.5\t0\t300\n",
      "8\t404\t4.1\t0\t404\n",
      "9\t1699\t1.0\t0\t1699\n",
      "10\t2473\t0.3\t1\t2462 11\n",
      "11\t152\t0.1\t1\t152\n",
      "12\t171\t0.0\t1\t165 6\n",
      "13\t163\t0.0\t1\t163\n",
      "14\t248\t0.0\t1\t241 7\n",
      "15\t326\t0.0\t1\t316 10\n",
      "16\t345\t0.0\t1\t332 13\n",
      "17\t634\t0.0\t1\t612 22\n",
      "18\t648\t0.0\t1\t595 32 21\n",
      "19\t1346\t0.0\t1\t828 516 2\n",
      "20\t25\t0.0\t2\t0 1 24\n",
      "21\t178\t0.0\t2\t0 177 1\n",
      "193\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/410.H_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/410.H_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/410.H_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/410.H_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.60 s (19 us/read; 3.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            238,999\n",
      "  Read 1 with adapter:                     179 (0.1%)\n",
      "  Read 2 with adapter:                   7,417 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       238,999 (100.0%)\n",
      "\n",
      "Total basepairs processed:   108,856,396 bp\n",
      "  Read 1:    53,513,240 bp\n",
      "  Read 2:    55,343,156 bp\n",
      "Total written (filtered):    108,758,395 bp (99.9%)\n",
      "  Read 1:    53,511,958 bp\n",
      "  Read 2:    55,246,437 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 179 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t168\t3734.4\t0\t168\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t4\t0.0\t1\t4\n",
      "38\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "121\t1\t0.0\t1\t1\n",
      "131\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7417 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t3734.4\t0\t40\n",
      "4\t3\t933.6\t0\t3\n",
      "5\t16\t233.4\t0\t16\n",
      "6\t38\t58.3\t0\t38\n",
      "7\t219\t14.6\t0\t219\n",
      "8\t334\t3.6\t0\t334\n",
      "9\t1332\t0.9\t0\t1332\n",
      "10\t1934\t0.2\t1\t1922 12\n",
      "11\t118\t0.1\t1\t118\n",
      "12\t144\t0.0\t1\t142 2\n",
      "13\t131\t0.0\t1\t131\n",
      "14\t213\t0.0\t1\t208 5\n",
      "15\t238\t0.0\t1\t231 7\n",
      "16\t269\t0.0\t1\t261 8\n",
      "17\t495\t0.0\t1\t484 11\n",
      "18\t563\t0.0\t1\t522 26 15\n",
      "19\t1143\t0.0\t1\t719 422 2\n",
      "20\t31\t0.0\t2\t1 1 29\n",
      "21\t147\t0.0\t2\t0 145 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "153\t3\t0.0\t2\t0 3\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "231\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/401.H_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/401.H_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/401.H_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/401.H_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/402.H_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/402.H_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/402.H_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/402.H_26_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/403.H_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/403.H_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/403.H_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/403.H_35_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/404.D_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/404.D_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/404.D_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/404.D_20_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/404.U_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/404.U_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/404.U_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/404.U_3_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/405.D_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/405.D_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/405.D_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/405.D_21_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/405.U_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/405.U_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/405.U_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/405.U_8_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/406.D_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/406.D_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/406.D_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/406.D_16_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/406.U_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/406.U_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/406.U_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/406.U_1_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/407.D_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/407.D_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/407.D_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/407.D_27_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/407.U_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/407.U_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/407.U_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/407.U_25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/408.D_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/408.D_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/408.D_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/408.D_24_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/408.U_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/408.U_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/408.U_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/408.U_11_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/409.H_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/409.H_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/409.H_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/409.H_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/410.H_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/410.H_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/410.H_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/410.H_6_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/411.H_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/411.H_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/411.H_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/411.H_15_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/411.H_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/411.H_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/411.H_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/411.H_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.10 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            271,703\n",
      "  Read 1 with adapter:                     167 (0.1%)\n",
      "  Read 2 with adapter:                   9,237 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       271,703 (100.0%)\n",
      "\n",
      "Total basepairs processed:   123,787,274 bp\n",
      "  Read 1:    60,845,993 bp\n",
      "  Read 2:    62,941,281 bp\n",
      "Total written (filtered):    123,666,076 bp (99.9%)\n",
      "  Read 1:    60,844,787 bp\n",
      "  Read 2:    62,821,289 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 167 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t158\t4245.4\t0\t158\n",
      "10\t1\t0.3\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "38\t1\t0.0\t1\t1\n",
      "106\t1\t0.0\t1\t1\n",
      "125\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t0 1\n",
      "219\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9237 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t55\t4245.4\t0\t55\n",
      "4\t8\t1061.3\t0\t8\n",
      "5\t25\t265.3\t0\t25\n",
      "6\t52\t66.3\t0\t52\n",
      "7\t283\t16.6\t0\t283\n",
      "8\t431\t4.1\t0\t431\n",
      "9\t1668\t1.0\t0\t1668\n",
      "10\t2355\t0.3\t1\t2349 6\n",
      "11\t150\t0.1\t1\t147 3\n",
      "12\t157\t0.0\t1\t155 2\n",
      "13\t172\t0.0\t1\t171 1\n",
      "14\t254\t0.0\t1\t245 9\n",
      "15\t335\t0.0\t1\t327 8\n",
      "16\t323\t0.0\t1\t314 9\n",
      "17\t611\t0.0\t1\t586 25\n",
      "18\t690\t0.0\t1\t632 39 19\n",
      "19\t1431\t0.0\t1\t912 519\n",
      "20\t31\t0.0\t2\t1 4 26\n",
      "21\t199\t0.0\t2\t0 199\n",
      "34\t1\t0.0\t2\t1\n",
      "102\t2\t0.0\t2\t1 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "207\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/412.D_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/412.D_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/412.D_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/412.D_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.33 s (21 us/read; 2.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,709\n",
      "  Read 1 with adapter:                      58 (0.1%)\n",
      "  Read 2 with adapter:                   1,732 (1.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       112,709 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,307,401 bp\n",
      "  Read 1:    25,239,442 bp\n",
      "  Read 2:    26,067,959 bp\n",
      "Total written (filtered):     51,281,936 bp (100.0%)\n",
      "  Read 1:    25,239,268 bp\n",
      "  Read 2:    26,042,668 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 58 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t1761.1\t0\t58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1732 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1761.1\t0\t9\n",
      "4\t1\t440.3\t0\t1\n",
      "5\t2\t110.1\t0\t2\n",
      "6\t10\t27.5\t0\t10\n",
      "7\t25\t6.9\t0\t25\n",
      "8\t33\t1.7\t0\t33\n",
      "9\t142\t0.4\t0\t142\n",
      "10\t413\t0.1\t1\t412 1\n",
      "11\t23\t0.0\t1\t23\n",
      "12\t22\t0.0\t1\t22\n",
      "13\t28\t0.0\t1\t28\n",
      "14\t41\t0.0\t1\t40 1\n",
      "15\t52\t0.0\t1\t50 2\n",
      "16\t80\t0.0\t1\t78 2\n",
      "17\t181\t0.0\t1\t179 2\n",
      "18\t160\t0.0\t1\t155 3 2\n",
      "19\t414\t0.0\t1\t280 134\n",
      "20\t16\t0.0\t2\t0 0 16\n",
      "21\t76\t0.0\t2\t0 75 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/412.U_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/412.U_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/412.U_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/412.U_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.55 s (20 us/read; 2.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            124,717\n",
      "  Read 1 with adapter:                      91 (0.1%)\n",
      "  Read 2 with adapter:                   3,643 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       124,717 (100.0%)\n",
      "\n",
      "Total basepairs processed:    56,576,055 bp\n",
      "  Read 1:    27,805,781 bp\n",
      "  Read 2:    28,770,274 bp\n",
      "Total written (filtered):     56,524,583 bp (99.9%)\n",
      "  Read 1:    27,803,005 bp\n",
      "  Read 2:    28,721,578 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t78\t1948.7\t0\t78\n",
      "147\t1\t0.0\t1\t1\n",
      "191\t5\t0.0\t1\t5\n",
      "192\t3\t0.0\t1\t3\n",
      "216\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3643 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t1948.7\t0\t23\n",
      "4\t2\t487.2\t0\t2\n",
      "5\t8\t121.8\t0\t8\n",
      "6\t15\t30.4\t0\t15\n",
      "7\t98\t7.6\t0\t98\n",
      "8\t160\t1.9\t0\t160\n",
      "9\t583\t0.5\t0\t583\n",
      "10\t879\t0.1\t1\t877 2\n",
      "11\t54\t0.0\t1\t54\n",
      "12\t63\t0.0\t1\t63\n",
      "13\t72\t0.0\t1\t72\n",
      "14\t87\t0.0\t1\t85 2\n",
      "15\t131\t0.0\t1\t129 2\n",
      "16\t130\t0.0\t1\t126 4\n",
      "17\t286\t0.0\t1\t279 7\n",
      "18\t312\t0.0\t1\t294 11 7\n",
      "19\t632\t0.0\t1\t390 241 1\n",
      "20\t14\t0.0\t2\t0 0 14\n",
      "21\t92\t0.0\t2\t0 91 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/413.D_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/413.D_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/413.D_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/413.D_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.12 s (18 us/read; 3.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,590\n",
      "  Read 1 with adapter:                      80 (0.1%)\n",
      "  Read 2 with adapter:                   1,599 (1.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,590 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,617,928 bp\n",
      "  Read 1:    25,886,599 bp\n",
      "  Read 2:    26,731,329 bp\n",
      "Total written (filtered):     52,593,567 bp (100.0%)\n",
      "  Read 1:    25,886,056 bp\n",
      "  Read 2:    26,707,511 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 80 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t1806.1\t0\t77\n",
      "27\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1599 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1806.1\t0\t5\n",
      "4\t1\t451.5\t0\t1\n",
      "5\t1\t112.9\t0\t1\n",
      "6\t2\t28.2\t0\t2\n",
      "7\t15\t7.1\t0\t15\n",
      "8\t36\t1.8\t0\t36\n",
      "9\t131\t0.4\t0\t131\n",
      "10\t349\t0.1\t1\t347 2\n",
      "11\t20\t0.0\t1\t20\n",
      "12\t24\t0.0\t1\t24\n",
      "13\t23\t0.0\t1\t23\n",
      "14\t38\t0.0\t1\t38\n",
      "15\t49\t0.0\t1\t47 2\n",
      "16\t76\t0.0\t1\t72 4\n",
      "17\t169\t0.0\t1\t165 4\n",
      "18\t168\t0.0\t1\t157 9 2\n",
      "19\t395\t0.0\t1\t278 117\n",
      "20\t9\t0.0\t2\t0 0 9\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/413.U_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/413.U_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/413.U_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/413.U_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.39 s (30 us/read; 1.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,842\n",
      "  Read 1 with adapter:                      32 (0.1%)\n",
      "  Read 2 with adapter:                   1,428 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,842 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,606,614 bp\n",
      "  Read 1:    10,114,522 bp\n",
      "  Read 2:    10,492,092 bp\n",
      "Total written (filtered):     20,586,656 bp (99.9%)\n",
      "  Read 1:    10,113,598 bp\n",
      "  Read 2:    10,473,058 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 32 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t716.3\t0\t27\n",
      "27\t1\t0.0\t1\t0 1\n",
      "192\t2\t0.0\t1\t2\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1428 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t716.3\t0\t7\n",
      "4\t3\t179.1\t0\t3\n",
      "5\t4\t44.8\t0\t4\n",
      "6\t7\t11.2\t0\t7\n",
      "7\t35\t2.8\t0\t35\n",
      "8\t64\t0.7\t0\t64\n",
      "9\t212\t0.2\t0\t212\n",
      "10\t334\t0.0\t1\t334\n",
      "11\t18\t0.0\t1\t18\n",
      "12\t29\t0.0\t1\t29\n",
      "13\t34\t0.0\t1\t33 1\n",
      "14\t59\t0.0\t1\t57 2\n",
      "15\t49\t0.0\t1\t48 1\n",
      "16\t73\t0.0\t1\t72 1\n",
      "17\t112\t0.0\t1\t111 1\n",
      "18\t121\t0.0\t1\t119 2\n",
      "19\t234\t0.0\t1\t134 98 2\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t29\t0.0\t2\t0 29\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/414.D_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/414.D_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/414.D_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/414.D_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.58 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,751\n",
      "  Read 1 with adapter:                      35 (0.1%)\n",
      "  Read 2 with adapter:                     978 (1.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        67,751 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,782,529 bp\n",
      "  Read 1:    15,140,872 bp\n",
      "  Read 2:    15,641,657 bp\n",
      "Total written (filtered):     30,767,114 bp (99.9%)\n",
      "  Read 1:    15,140,276 bp\n",
      "  Read 2:    15,626,838 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 35 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t1058.6\t0\t30\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 978 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1058.6\t0\t5\n",
      "4\t2\t264.7\t0\t2\n",
      "6\t2\t16.5\t0\t2\n",
      "7\t13\t4.1\t0\t13\n",
      "8\t16\t1.0\t0\t16\n",
      "9\t55\t0.3\t0\t55\n",
      "10\t162\t0.1\t1\t162\n",
      "11\t32\t0.0\t1\t32\n",
      "12\t19\t0.0\t1\t19\n",
      "13\t16\t0.0\t1\t16\n",
      "14\t30\t0.0\t1\t29 1\n",
      "15\t55\t0.0\t1\t55\n",
      "16\t72\t0.0\t1\t71 1\n",
      "17\t96\t0.0\t1\t93 3\n",
      "18\t118\t0.0\t1\t111 5 2\n",
      "19\t235\t0.0\t1\t164 71\n",
      "20\t6\t0.0\t2\t0 1 5\n",
      "21\t44\t0.0\t2\t0 43 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/414.U_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/414.U_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/414.U_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/414.U_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.74 s (29 us/read; 2.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,296\n",
      "  Read 1 with adapter:                      25 (0.0%)\n",
      "  Read 2 with adapter:                   1,090 (1.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        59,296 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,822,647 bp\n",
      "  Read 1:    13,187,760 bp\n",
      "  Read 2:    13,634,887 bp\n",
      "Total written (filtered):     26,806,394 bp (99.9%)\n",
      "  Read 1:    13,187,685 bp\n",
      "  Read 2:    13,618,709 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 25 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t926.5\t0\t25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1090 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t926.5\t0\t6\n",
      "5\t1\t57.9\t0\t1\n",
      "6\t4\t14.5\t0\t4\n",
      "7\t18\t3.6\t0\t18\n",
      "8\t16\t0.9\t0\t16\n",
      "9\t98\t0.2\t0\t98\n",
      "10\t183\t0.1\t1\t183\n",
      "11\t20\t0.0\t1\t20\n",
      "12\t27\t0.0\t1\t27\n",
      "13\t20\t0.0\t1\t20\n",
      "14\t29\t0.0\t1\t27 2\n",
      "15\t64\t0.0\t1\t62 2\n",
      "16\t57\t0.0\t1\t56 1\n",
      "17\t142\t0.0\t1\t140 2\n",
      "18\t119\t0.0\t1\t113 4 2\n",
      "19\t236\t0.0\t1\t170 65 1\n",
      "20\t7\t0.0\t2\t0 0 7\n",
      "21\t43\t0.0\t2\t0 43\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/415.D_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/415.D_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/415.D_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/415.D_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.34 s (20 us/read; 2.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            164,005\n",
      "  Read 1 with adapter:                     107 (0.1%)\n",
      "  Read 2 with adapter:                   3,515 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       164,005 (100.0%)\n",
      "\n",
      "Total basepairs processed:    74,658,310 bp\n",
      "  Read 1:    36,713,139 bp\n",
      "  Read 2:    37,945,171 bp\n",
      "Total written (filtered):     74,607,982 bp (99.9%)\n",
      "  Read 1:    36,712,514 bp\n",
      "  Read 2:    37,895,468 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 107 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t103\t2562.6\t0\t103\n",
      "26\t1\t0.0\t1\t0 1\n",
      "38\t2\t0.0\t1\t2\n",
      "214\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3515 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t2562.6\t0\t19\n",
      "4\t5\t640.6\t0\t5\n",
      "5\t6\t160.2\t0\t6\n",
      "6\t9\t40.0\t0\t9\n",
      "7\t66\t10.0\t0\t66\n",
      "8\t86\t2.5\t0\t86\n",
      "9\t391\t0.6\t0\t391\n",
      "10\t795\t0.2\t1\t792 3\n",
      "11\t51\t0.0\t1\t51\n",
      "12\t75\t0.0\t1\t73 2\n",
      "13\t68\t0.0\t1\t67 1\n",
      "14\t117\t0.0\t1\t111 6\n",
      "15\t104\t0.0\t1\t99 5\n",
      "16\t185\t0.0\t1\t184 1\n",
      "17\t357\t0.0\t1\t351 6\n",
      "18\t332\t0.0\t1\t308 20 4\n",
      "19\t714\t0.0\t1\t475 239\n",
      "20\t27\t0.0\t2\t0 1 26\n",
      "21\t107\t0.0\t2\t0 107\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/415.U_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/415.U_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/415.U_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/415.U_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.70 s (19 us/read; 3.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            143,648\n",
      "  Read 1 with adapter:                      88 (0.1%)\n",
      "  Read 2 with adapter:                   4,243 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       143,648 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,280,604 bp\n",
      "  Read 1:    32,051,120 bp\n",
      "  Read 2:    33,229,484 bp\n",
      "Total written (filtered):     65,225,255 bp (99.9%)\n",
      "  Read 1:    32,050,644 bp\n",
      "  Read 2:    33,174,611 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 88 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t87\t2244.5\t0\t87\n",
      "215\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4243 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t2244.5\t0\t27\n",
      "4\t1\t561.1\t0\t1\n",
      "5\t13\t140.3\t0\t13\n",
      "6\t29\t35.1\t0\t29\n",
      "7\t138\t8.8\t0\t138\n",
      "8\t209\t2.2\t0\t209\n",
      "9\t757\t0.5\t0\t756 1\n",
      "10\t1166\t0.1\t1\t1162 4\n",
      "11\t50\t0.0\t1\t50\n",
      "12\t45\t0.0\t1\t45\n",
      "13\t53\t0.0\t1\t53\n",
      "14\t92\t0.0\t1\t90 2\n",
      "15\t117\t0.0\t1\t113 4\n",
      "16\t166\t0.0\t1\t162 4\n",
      "17\t308\t0.0\t1\t301 7\n",
      "18\t280\t0.0\t1\t270 8 2\n",
      "19\t655\t0.0\t1\t430 224 1\n",
      "20\t22\t0.0\t2\t0 0 22\n",
      "21\t112\t0.0\t2\t0 111 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/416.D_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/416.D_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/416.D_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/416.D_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.40 s (22 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            157,724\n",
      "  Read 1 with adapter:                      74 (0.0%)\n",
      "  Read 2 with adapter:                   3,201 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       157,724 (100.0%)\n",
      "\n",
      "Total basepairs processed:    71,722,972 bp\n",
      "  Read 1:    35,267,403 bp\n",
      "  Read 2:    36,455,569 bp\n",
      "Total written (filtered):     71,676,256 bp (99.9%)\n",
      "  Read 1:    35,266,304 bp\n",
      "  Read 2:    36,409,952 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 74 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t69\t2464.4\t0\t69\n",
      "27\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t3\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3201 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t2464.4\t0\t19\n",
      "4\t5\t616.1\t0\t5\n",
      "5\t5\t154.0\t0\t5\n",
      "6\t14\t38.5\t0\t14\n",
      "7\t82\t9.6\t0\t82\n",
      "8\t100\t2.4\t0\t100\n",
      "9\t349\t0.6\t0\t349\n",
      "10\t692\t0.2\t1\t684 8\n",
      "11\t46\t0.0\t1\t46\n",
      "12\t70\t0.0\t1\t69 1\n",
      "13\t78\t0.0\t1\t77 1\n",
      "14\t74\t0.0\t1\t70 4\n",
      "15\t129\t0.0\t1\t126 3\n",
      "16\t146\t0.0\t1\t145 1\n",
      "17\t285\t0.0\t1\t276 9\n",
      "18\t333\t0.0\t1\t307 17 9\n",
      "19\t639\t0.0\t1\t435 203 1\n",
      "20\t25\t0.0\t2\t0 0 25\n",
      "21\t105\t0.0\t2\t0 104 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/416.U_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/416.U_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/416.U_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/416.U_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.82 s (19 us/read; 3.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            205,303\n",
      "  Read 1 with adapter:                     115 (0.1%)\n",
      "  Read 2 with adapter:                   8,156 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       205,303 (100.0%)\n",
      "\n",
      "Total basepairs processed:    93,499,882 bp\n",
      "  Read 1:    45,941,567 bp\n",
      "  Read 2:    47,558,315 bp\n",
      "Total written (filtered):     93,398,760 bp (99.9%)\n",
      "  Read 1:    45,940,612 bp\n",
      "  Read 2:    47,458,148 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 115 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t102\t3207.9\t0\t102\n",
      "24\t6\t0.0\t1\t6\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t2\t0.0\t1\t2\n",
      "92\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8156 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t3207.9\t0\t45\n",
      "4\t13\t802.0\t0\t13\n",
      "5\t28\t200.5\t0\t28\n",
      "6\t62\t50.1\t0\t62\n",
      "7\t338\t12.5\t0\t338\n",
      "8\t455\t3.1\t0\t455\n",
      "9\t1671\t0.8\t0\t1670 1\n",
      "10\t2346\t0.2\t1\t2340 6\n",
      "11\t107\t0.0\t1\t105 2\n",
      "12\t109\t0.0\t1\t109\n",
      "13\t111\t0.0\t1\t111\n",
      "14\t162\t0.0\t1\t160 2\n",
      "15\t249\t0.0\t1\t240 9\n",
      "16\t266\t0.0\t1\t260 6\n",
      "17\t503\t0.0\t1\t492 11\n",
      "18\t530\t0.0\t1\t499 18 13\n",
      "19\t1016\t0.0\t1\t670 346\n",
      "20\t13\t0.0\t2\t0 2 11\n",
      "21\t129\t0.0\t2\t0 127 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/417.H_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/417.H_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/417.H_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/417.H_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.37 s (20 us/read; 3.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            172,659\n",
      "  Read 1 with adapter:                     136 (0.1%)\n",
      "  Read 2 with adapter:                   3,318 (1.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       172,659 (100.0%)\n",
      "\n",
      "Total basepairs processed:    78,495,827 bp\n",
      "  Read 1:    38,604,688 bp\n",
      "  Read 2:    39,891,139 bp\n",
      "Total written (filtered):     78,446,757 bp (99.9%)\n",
      "  Read 1:    38,603,689 bp\n",
      "  Read 2:    39,843,068 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 136 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t133\t2697.8\t0\t133\n",
      "192\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3318 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t2697.8\t0\t13\n",
      "4\t2\t674.4\t0\t2\n",
      "5\t7\t168.6\t0\t7\n",
      "6\t8\t42.2\t0\t8\n",
      "7\t52\t10.5\t0\t52\n",
      "8\t71\t2.6\t0\t71\n",
      "9\t375\t0.7\t0\t375\n",
      "10\t711\t0.2\t1\t708 3\n",
      "11\t48\t0.0\t1\t46 2\n",
      "12\t82\t0.0\t1\t82\n",
      "13\t81\t0.0\t1\t80 1\n",
      "14\t94\t0.0\t1\t92 2\n",
      "15\t133\t0.0\t1\t129 4\n",
      "16\t155\t0.0\t1\t149 6\n",
      "17\t275\t0.0\t1\t265 10\n",
      "18\t369\t0.0\t1\t349 15 5\n",
      "19\t695\t0.0\t1\t452 243\n",
      "20\t21\t0.0\t2\t0 0 21\n",
      "21\t122\t0.0\t2\t0 122\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/418.H_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/418.H_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/418.H_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/418.H_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.51 s (32 us/read; 1.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,485\n",
      "  Read 1 with adapter:                      27 (0.1%)\n",
      "  Read 2 with adapter:                     854 (1.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        47,485 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,515,700 bp\n",
      "  Read 1:    10,580,339 bp\n",
      "  Read 2:    10,935,361 bp\n",
      "Total written (filtered):     21,502,498 bp (99.9%)\n",
      "  Read 1:    10,580,225 bp\n",
      "  Read 2:    10,922,273 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 27 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t742.0\t0\t25\n",
      "12\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 854 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t742.0\t0\t7\n",
      "4\t4\t185.5\t0\t4\n",
      "6\t3\t11.6\t0\t3\n",
      "7\t15\t2.9\t0\t15\n",
      "8\t20\t0.7\t0\t20\n",
      "9\t87\t0.2\t0\t87\n",
      "10\t124\t0.0\t1\t124\n",
      "11\t7\t0.0\t1\t7\n",
      "12\t27\t0.0\t1\t27\n",
      "13\t12\t0.0\t1\t12\n",
      "14\t23\t0.0\t1\t23\n",
      "15\t51\t0.0\t1\t51\n",
      "16\t46\t0.0\t1\t45 1\n",
      "17\t107\t0.0\t1\t104 3\n",
      "18\t102\t0.0\t1\t96 5 1\n",
      "19\t184\t0.0\t1\t124 60\n",
      "20\t3\t0.0\t2\t0 1 2\n",
      "21\t29\t0.0\t2\t0 28 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "215\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/419.H_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/419.H_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/419.H_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/419.H_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.09 s (28 us/read; 2.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,645\n",
      "  Read 1 with adapter:                      39 (0.1%)\n",
      "  Read 2 with adapter:                   2,475 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        74,645 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,744,505 bp\n",
      "  Read 1:    16,575,394 bp\n",
      "  Read 2:    17,169,111 bp\n",
      "Total written (filtered):     33,711,380 bp (99.9%)\n",
      "  Read 1:    16,574,944 bp\n",
      "  Read 2:    17,136,436 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 39 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t1166.3\t0\t36\n",
      "27\t1\t0.0\t1\t1\n",
      "99\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2475 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1166.3\t0\t12\n",
      "4\t3\t291.6\t0\t3\n",
      "5\t6\t72.9\t0\t6\n",
      "6\t10\t18.2\t0\t10\n",
      "7\t70\t4.6\t0\t70\n",
      "8\t111\t1.1\t0\t111\n",
      "9\t398\t0.3\t0\t397 1\n",
      "10\t623\t0.1\t1\t616 7\n",
      "11\t43\t0.0\t1\t43\n",
      "12\t29\t0.0\t1\t28 1\n",
      "13\t42\t0.0\t1\t42\n",
      "14\t57\t0.0\t1\t55 2\n",
      "15\t89\t0.0\t1\t86 3\n",
      "16\t95\t0.0\t1\t92 3\n",
      "17\t213\t0.0\t1\t206 7\n",
      "18\t207\t0.0\t1\t192 8 7\n",
      "19\t411\t0.0\t1\t281 130\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t49\t0.0\t2\t0 49\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/420.D_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/420.D_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/420.D_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/420.D_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.01 s (38 us/read; 1.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,724\n",
      "  Read 1 with adapter:                      19 (0.1%)\n",
      "  Read 2 with adapter:                     762 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        26,724 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,984,408 bp\n",
      "  Read 1:     5,883,528 bp\n",
      "  Read 2:     6,100,880 bp\n",
      "Total written (filtered):     11,973,916 bp (99.9%)\n",
      "  Read 1:     5,882,881 bp\n",
      "  Read 2:     6,091,035 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t417.6\t0\t16\n",
      "191\t1\t0.0\t1\t0 1\n",
      "192\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 762 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t417.6\t0\t3\n",
      "4\t2\t104.4\t0\t2\n",
      "5\t1\t26.1\t0\t1\n",
      "6\t3\t6.5\t0\t3\n",
      "7\t21\t1.6\t0\t21\n",
      "8\t25\t0.4\t0\t25\n",
      "9\t136\t0.1\t0\t136\n",
      "10\t216\t0.0\t1\t216\n",
      "11\t9\t0.0\t1\t9\n",
      "12\t17\t0.0\t1\t17\n",
      "13\t10\t0.0\t1\t10\n",
      "14\t15\t0.0\t1\t15\n",
      "15\t26\t0.0\t1\t26\n",
      "16\t34\t0.0\t1\t33 1\n",
      "17\t59\t0.0\t1\t58 1\n",
      "18\t53\t0.0\t1\t51 2\n",
      "19\t115\t0.0\t1\t81 34\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t15\t0.0\t2\t0 15\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/420.U_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/420.U_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/420.U_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/420.U_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.08 s (40 us/read; 1.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,975\n",
      "  Read 1 with adapter:                      31 (0.1%)\n",
      "  Read 2 with adapter:                     797 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        26,975 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,076,498 bp\n",
      "  Read 1:     5,929,187 bp\n",
      "  Read 2:     6,147,311 bp\n",
      "Total written (filtered):     12,065,079 bp (99.9%)\n",
      "  Read 1:     5,928,391 bp\n",
      "  Read 2:     6,136,688 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 31 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t421.5\t0\t24\n",
      "20\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t1 1\n",
      "192\t1\t0.0\t1\t0 1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 797 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t421.5\t0\t3\n",
      "5\t1\t26.3\t0\t1\n",
      "6\t4\t6.6\t0\t4\n",
      "7\t18\t1.6\t0\t18\n",
      "8\t42\t0.4\t0\t42\n",
      "9\t126\t0.1\t0\t126\n",
      "10\t200\t0.0\t1\t200\n",
      "11\t18\t0.0\t1\t18\n",
      "12\t10\t0.0\t1\t9 1\n",
      "13\t5\t0.0\t1\t5\n",
      "14\t14\t0.0\t1\t14\n",
      "15\t37\t0.0\t1\t37\n",
      "16\t35\t0.0\t1\t34 1\n",
      "17\t53\t0.0\t1\t52 1\n",
      "18\t68\t0.0\t1\t64 2 2\n",
      "19\t145\t0.0\t1\t101 44\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t14\t0.0\t2\t0 14\n",
      "61\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/412.D_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/412.D_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/412.D_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/412.D_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/412.U_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/412.U_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/412.U_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/412.U_4_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/413.D_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/413.D_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/413.D_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/413.D_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/413.U_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/413.U_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/413.U_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/413.U_32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/414.D_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/414.D_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/414.D_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/414.D_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/414.U_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/414.U_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/414.U_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/414.U_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/415.D_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/415.D_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/415.D_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/415.D_33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/415.U_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/415.U_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/415.U_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/415.U_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/416.D_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/416.D_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/416.D_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/416.D_9_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/416.U_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/416.U_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/416.U_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/416.U_31_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/417.H_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/417.H_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/417.H_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/417.H_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/418.H_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/418.H_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/418.H_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/418.H_14_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/419.H_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/419.H_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/419.H_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/419.H_17_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/420.D_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/420.D_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/420.D_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/420.D_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/420.U_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/420.U_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/420.U_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/420.U_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/421.U_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/421.U_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/421.U_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/421.U_39_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/421.U_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-cdsvlq8r/421.U_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/421.U_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-grt50tfs/d0440f4f-8a92-4332-83aa-b2fca7923724/data/421.U_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.73 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,185\n",
      "  Read 1 with adapter:                      29 (0.0%)\n",
      "  Read 2 with adapter:                     669 (1.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        67,185 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,534,752 bp\n",
      "  Read 1:    15,030,331 bp\n",
      "  Read 2:    15,504,421 bp\n",
      "Total written (filtered):     30,523,770 bp (100.0%)\n",
      "  Read 1:    15,030,220 bp\n",
      "  Read 2:    15,493,550 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t1049.8\t0\t28\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 669 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1049.8\t0\t2\n",
      "7\t4\t4.1\t0\t4\n",
      "8\t6\t1.0\t0\t6\n",
      "9\t24\t0.3\t0\t24\n",
      "10\t70\t0.1\t1\t70\n",
      "11\t8\t0.0\t1\t8\n",
      "12\t18\t0.0\t1\t18\n",
      "13\t18\t0.0\t1\t18\n",
      "14\t13\t0.0\t1\t13\n",
      "15\t37\t0.0\t1\t37\n",
      "16\t39\t0.0\t1\t39\n",
      "17\t79\t0.0\t1\t78 1\n",
      "18\t117\t0.0\t1\t111 6\n",
      "19\t190\t0.0\t1\t140 50\n",
      "20\t9\t0.0\t2\t0 0 9\n",
      "21\t35\t0.0\t2\t0 35\n",
      "\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGmicro2018_145-192.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/300.D_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/300.D_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/300.D_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/300.D_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.84 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,659\n",
      "  Read 1 with adapter:                      28 (0.0%)\n",
      "  Read 2 with adapter:                     647 (0.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,659 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,417,287 bp\n",
      "  Read 1:    16,935,339 bp\n",
      "  Read 2:    17,481,948 bp\n",
      "Total written (filtered):     34,405,959 bp (100.0%)\n",
      "  Read 1:    16,935,255 bp\n",
      "  Read 2:    17,470,704 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 28 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t1182.2\t0\t28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 647 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1182.2\t0\t2\n",
      "5\t2\t73.9\t0\t2\n",
      "7\t1\t4.6\t0\t1\n",
      "8\t3\t1.2\t0\t3\n",
      "9\t20\t0.3\t0\t20\n",
      "10\t26\t0.1\t1\t26\n",
      "11\t5\t0.0\t1\t5\n",
      "12\t9\t0.0\t1\t9\n",
      "13\t8\t0.0\t1\t8\n",
      "14\t13\t0.0\t1\t13\n",
      "15\t31\t0.0\t1\t31\n",
      "16\t25\t0.0\t1\t25\n",
      "17\t79\t0.0\t1\t77 2\n",
      "18\t74\t0.0\t1\t72 2\n",
      "19\t278\t0.0\t1\t211 67\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t66\t0.0\t2\t0 66\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/300.U_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/300.U_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/300.U_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/300.U_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.89 s (18 us/read; 3.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            221,673\n",
      "  Read 1 with adapter:                     111 (0.1%)\n",
      "  Read 2 with adapter:                   3,896 (1.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       221,673 (100.0%)\n",
      "\n",
      "Total basepairs processed:   100,778,639 bp\n",
      "  Read 1:    49,570,016 bp\n",
      "  Read 2:    51,208,623 bp\n",
      "Total written (filtered):    100,719,305 bp (99.9%)\n",
      "  Read 1:    49,568,993 bp\n",
      "  Read 2:    51,150,312 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 111 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t102\t3463.6\t0\t102\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t3\t0.0\t1\t3\n",
      "77\t2\t0.0\t1\t2\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3896 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t3463.6\t0\t14\n",
      "4\t3\t865.9\t0\t3\n",
      "5\t2\t216.5\t0\t2\n",
      "6\t12\t54.1\t0\t12\n",
      "7\t51\t13.5\t0\t51\n",
      "8\t87\t3.4\t0\t87\n",
      "9\t323\t0.8\t0\t323\n",
      "10\t802\t0.2\t1\t800 2\n",
      "11\t48\t0.1\t1\t48\n",
      "12\t73\t0.0\t1\t72 1\n",
      "13\t54\t0.0\t1\t54\n",
      "14\t100\t0.0\t1\t100\n",
      "15\t139\t0.0\t1\t136 3\n",
      "16\t172\t0.0\t1\t167 5\n",
      "17\t366\t0.0\t1\t360 6\n",
      "18\t443\t0.0\t1\t420 17 6\n",
      "19\t1021\t0.0\t1\t733 286 2\n",
      "20\t40\t0.0\t2\t0 2 38\n",
      "21\t142\t0.0\t2\t0 139 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/301.D_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/301.D_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/301.D_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/301.D_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.67 s (28 us/read; 2.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,193\n",
      "  Read 1 with adapter:                      28 (0.0%)\n",
      "  Read 2 with adapter:                   2,434 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        60,193 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,971,578 bp\n",
      "  Read 1:    13,235,888 bp\n",
      "  Read 2:    13,735,690 bp\n",
      "Total written (filtered):     26,940,340 bp (99.9%)\n",
      "  Read 1:    13,235,226 bp\n",
      "  Read 2:    13,705,114 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 28 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t940.5\t0\t22\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "87\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2434 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t940.5\t0\t14\n",
      "4\t1\t235.1\t0\t1\n",
      "5\t9\t58.8\t0\t9\n",
      "6\t27\t14.7\t0\t27\n",
      "7\t113\t3.7\t0\t113\n",
      "8\t163\t0.9\t0\t163\n",
      "9\t536\t0.2\t0\t536\n",
      "10\t512\t0.1\t1\t511 1\n",
      "11\t21\t0.0\t1\t21\n",
      "12\t23\t0.0\t1\t23\n",
      "13\t23\t0.0\t1\t22 1\n",
      "14\t70\t0.0\t1\t68 2\n",
      "15\t64\t0.0\t1\t62 2\n",
      "16\t110\t0.0\t1\t107 3\n",
      "17\t175\t0.0\t1\t168 7\n",
      "18\t180\t0.0\t1\t172 4 4\n",
      "19\t339\t0.0\t1\t214 123 2\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t40\t0.0\t2\t0 39 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/301.U_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/301.U_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/301.U_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/301.U_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.59 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             71,457\n",
      "  Read 1 with adapter:                      46 (0.1%)\n",
      "  Read 2 with adapter:                     681 (1.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        71,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,482,602 bp\n",
      "  Read 1:    15,982,570 bp\n",
      "  Read 2:    16,500,032 bp\n",
      "Total written (filtered):     32,470,660 bp (100.0%)\n",
      "  Read 1:    15,982,049 bp\n",
      "  Read 2:    16,488,611 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 46 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t1116.5\t0\t43\n",
      "88\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 681 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1116.5\t0\t3\n",
      "4\t1\t279.1\t0\t1\n",
      "5\t1\t69.8\t0\t1\n",
      "7\t5\t4.4\t0\t5\n",
      "8\t5\t1.1\t0\t5\n",
      "9\t19\t0.3\t0\t19\n",
      "10\t55\t0.1\t1\t55\n",
      "11\t8\t0.0\t1\t8\n",
      "12\t16\t0.0\t1\t15 1\n",
      "13\t8\t0.0\t1\t8\n",
      "14\t11\t0.0\t1\t11\n",
      "15\t24\t0.0\t1\t24\n",
      "16\t36\t0.0\t1\t35 1\n",
      "17\t68\t0.0\t1\t67 1\n",
      "18\t113\t0.0\t1\t110 3\n",
      "19\t258\t0.0\t1\t200 58\n",
      "20\t9\t0.0\t2\t0 0 9\n",
      "21\t41\t0.0\t2\t0 41\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/302.D_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/302.D_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/302.D_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/302.D_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.21 s (22 us/read; 2.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,397\n",
      "  Read 1 with adapter:                      58 (0.1%)\n",
      "  Read 2 with adapter:                   2,333 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,397 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,497,664 bp\n",
      "  Read 1:    22,369,882 bp\n",
      "  Read 2:    23,127,782 bp\n",
      "Total written (filtered):     45,464,777 bp (99.9%)\n",
      "  Read 1:    22,369,546 bp\n",
      "  Read 2:    23,095,231 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 58 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t55\t1568.7\t0\t55\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2333 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1568.7\t0\t12\n",
      "4\t4\t392.2\t0\t4\n",
      "5\t10\t98.0\t0\t10\n",
      "6\t17\t24.5\t0\t17\n",
      "7\t74\t6.1\t0\t74\n",
      "8\t109\t1.5\t0\t109\n",
      "9\t369\t0.4\t0\t369\n",
      "10\t363\t0.1\t1\t363\n",
      "11\t40\t0.0\t1\t40\n",
      "12\t41\t0.0\t1\t41\n",
      "13\t20\t0.0\t1\t19 1\n",
      "14\t48\t0.0\t1\t48\n",
      "15\t84\t0.0\t1\t83 1\n",
      "16\t92\t0.0\t1\t88 4\n",
      "17\t213\t0.0\t1\t212 1\n",
      "18\t261\t0.0\t1\t244 9 8\n",
      "19\t494\t0.0\t1\t354 138 2\n",
      "20\t18\t0.0\t2\t0 1 17\n",
      "21\t63\t0.0\t2\t0 63\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/302.U_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/302.U_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/302.U_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/302.U_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.08 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,991\n",
      "  Read 1 with adapter:                      22 (0.1%)\n",
      "  Read 2 with adapter:                     602 (1.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,991 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,844,084 bp\n",
      "  Read 1:     7,298,068 bp\n",
      "  Read 2:     7,546,016 bp\n",
      "Total written (filtered):     14,835,169 bp (99.9%)\n",
      "  Read 1:     7,298,002 bp\n",
      "  Read 2:     7,537,167 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t515.5\t0\t22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 602 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t128.9\t0\t1\n",
      "5\t2\t32.2\t0\t2\n",
      "6\t3\t8.1\t0\t3\n",
      "7\t15\t2.0\t0\t15\n",
      "8\t16\t0.5\t0\t16\n",
      "9\t67\t0.1\t0\t67\n",
      "10\t92\t0.0\t1\t92\n",
      "11\t11\t0.0\t1\t11\n",
      "12\t9\t0.0\t1\t9\n",
      "13\t5\t0.0\t1\t4 1\n",
      "14\t16\t0.0\t1\t16\n",
      "15\t29\t0.0\t1\t28 1\n",
      "16\t45\t0.0\t1\t43 2\n",
      "17\t53\t0.0\t1\t51 2\n",
      "18\t84\t0.0\t1\t83 1\n",
      "19\t129\t0.0\t1\t79 49 1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t20\t0.0\t2\t0 20\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/303.D_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/303.D_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/303.D_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/303.D_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.68 s (20 us/read; 3.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            134,269\n",
      "  Read 1 with adapter:                      92 (0.1%)\n",
      "  Read 2 with adapter:                   3,292 (2.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       134,269 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,937,081 bp\n",
      "  Read 1:    29,959,825 bp\n",
      "  Read 2:    30,977,256 bp\n",
      "Total written (filtered):     60,890,981 bp (99.9%)\n",
      "  Read 1:    29,959,169 bp\n",
      "  Read 2:    30,931,812 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 92 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t2098.0\t0\t88\n",
      "25\t1\t0.0\t1\t1\n",
      "37\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3292 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t2098.0\t0\t21\n",
      "4\t4\t524.5\t0\t4\n",
      "5\t2\t131.1\t0\t2\n",
      "6\t20\t32.8\t0\t20\n",
      "7\t69\t8.2\t0\t69\n",
      "8\t119\t2.0\t0\t119\n",
      "9\t465\t0.5\t0\t465\n",
      "10\t818\t0.1\t1\t816 2\n",
      "11\t37\t0.0\t1\t37\n",
      "12\t60\t0.0\t1\t58 2\n",
      "13\t42\t0.0\t1\t42\n",
      "14\t74\t0.0\t1\t72 2\n",
      "15\t91\t0.0\t1\t86 5\n",
      "16\t144\t0.0\t1\t143 1\n",
      "17\t259\t0.0\t1\t258 1\n",
      "18\t264\t0.0\t1\t249 8 7\n",
      "19\t693\t0.0\t1\t482 210 1\n",
      "20\t18\t0.0\t2\t0 4 14\n",
      "21\t89\t0.0\t2\t0 89\n",
      "97\t1\t0.0\t2\t1\n",
      "193\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/303.U_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/303.U_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/303.U_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/303.U_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.92 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,947\n",
      "  Read 1 with adapter:                      44 (0.1%)\n",
      "  Read 2 with adapter:                   3,090 (3.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        81,947 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,843,043 bp\n",
      "  Read 1:    18,088,395 bp\n",
      "  Read 2:    18,754,648 bp\n",
      "Total written (filtered):     36,799,921 bp (99.9%)\n",
      "  Read 1:    18,085,675 bp\n",
      "  Read 2:    18,714,246 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 44 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t1280.4\t0\t28\n",
      "23\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "106\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t7\t0.0\t1\t4 3\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3090 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1280.4\t0\t15\n",
      "5\t4\t80.0\t0\t4\n",
      "6\t13\t20.0\t0\t13\n",
      "7\t122\t5.0\t0\t122\n",
      "8\t172\t1.3\t0\t172\n",
      "9\t617\t0.3\t0\t617\n",
      "10\t633\t0.1\t1\t631 2\n",
      "11\t47\t0.0\t1\t46 1\n",
      "12\t49\t0.0\t1\t49\n",
      "13\t25\t0.0\t1\t25\n",
      "14\t71\t0.0\t1\t69 2\n",
      "15\t116\t0.0\t1\t113 3\n",
      "16\t101\t0.0\t1\t94 7\n",
      "17\t238\t0.0\t1\t231 7\n",
      "18\t278\t0.0\t1\t262 7 9\n",
      "19\t523\t0.0\t1\t341 180 2\n",
      "20\t9\t0.0\t2\t0 0 9\n",
      "21\t55\t0.0\t2\t0 55\n",
      "22\t2\t0.0\t2\t0 1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/304.D_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/304.D_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/304.D_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/304.D_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.52 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,500\n",
      "  Read 1 with adapter:                      33 (0.1%)\n",
      "  Read 2 with adapter:                   2,259 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        61,500 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,582,534 bp\n",
      "  Read 1:    13,548,282 bp\n",
      "  Read 2:    14,034,252 bp\n",
      "Total written (filtered):     27,550,587 bp (99.9%)\n",
      "  Read 1:    13,546,560 bp\n",
      "  Read 2:    14,004,027 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 33 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t960.9\t0\t22\n",
      "13\t1\t0.0\t1\t1\n",
      "24\t2\t0.0\t1\t2\n",
      "100\t1\t0.0\t1\t1\n",
      "206\t2\t0.0\t1\t2\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t3\t0.0\t1\t2 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2259 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t960.9\t0\t15\n",
      "4\t1\t240.2\t0\t1\n",
      "5\t10\t60.1\t0\t10\n",
      "6\t17\t15.0\t0\t17\n",
      "7\t63\t3.8\t0\t63\n",
      "8\t109\t0.9\t0\t109\n",
      "9\t387\t0.2\t0\t386 1\n",
      "10\t567\t0.1\t1\t565 2\n",
      "11\t32\t0.0\t1\t32\n",
      "12\t23\t0.0\t1\t23\n",
      "13\t23\t0.0\t1\t23\n",
      "14\t59\t0.0\t1\t58 1\n",
      "15\t62\t0.0\t1\t61 1\n",
      "16\t91\t0.0\t1\t87 4\n",
      "17\t169\t0.0\t1\t164 5\n",
      "18\t169\t0.0\t1\t153 11 5\n",
      "19\t406\t0.0\t1\t252 151 3\n",
      "20\t5\t0.0\t2\t0 1 4\n",
      "21\t48\t0.0\t2\t0 48\n",
      "218\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t0 1\n",
      "238\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/304.U_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/304.U_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/304.U_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/304.U_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.69 s (24 us/read; 2.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,143\n",
      "  Read 1 with adapter:                      48 (0.1%)\n",
      "  Read 2 with adapter:                   2,202 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        70,143 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,544,701 bp\n",
      "  Read 1:    15,490,384 bp\n",
      "  Read 2:    16,054,317 bp\n",
      "Total written (filtered):     31,512,109 bp (99.9%)\n",
      "  Read 1:    15,489,067 bp\n",
      "  Read 2:    16,023,042 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 48 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t41\t1096.0\t0\t41\n",
      "27\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "216\t3\t0.0\t1\t2 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2202 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1096.0\t0\t2\n",
      "4\t2\t274.0\t0\t2\n",
      "5\t5\t68.5\t0\t5\n",
      "6\t4\t17.1\t0\t4\n",
      "7\t67\t4.3\t0\t67\n",
      "8\t77\t1.1\t0\t77\n",
      "9\t352\t0.3\t0\t351 1\n",
      "10\t384\t0.1\t1\t380 4\n",
      "11\t27\t0.0\t1\t27\n",
      "12\t41\t0.0\t1\t39 2\n",
      "13\t27\t0.0\t1\t26 1\n",
      "14\t68\t0.0\t1\t66 2\n",
      "15\t98\t0.0\t1\t98\n",
      "16\t96\t0.0\t1\t93 3\n",
      "17\t187\t0.0\t1\t180 7\n",
      "18\t229\t0.0\t1\t205 16 8\n",
      "19\t464\t0.0\t1\t281 182 1\n",
      "20\t11\t0.0\t2\t0 1 10\n",
      "21\t59\t0.0\t2\t0 59\n",
      "218\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/305.H_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/305.H_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/305.H_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/305.H_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.49 s (20 us/read; 2.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            121,520\n",
      "  Read 1 with adapter:                      75 (0.1%)\n",
      "  Read 2 with adapter:                   2,975 (2.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       121,520 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,080,493 bp\n",
      "  Read 1:    27,079,135 bp\n",
      "  Read 2:    28,001,358 bp\n",
      "Total written (filtered):     55,038,398 bp (99.9%)\n",
      "  Read 1:    27,078,415 bp\n",
      "  Read 2:    27,959,983 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 75 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t70\t1898.8\t0\t70\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "215\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2975 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1898.8\t0\t8\n",
      "4\t3\t474.7\t0\t3\n",
      "5\t6\t118.7\t0\t6\n",
      "6\t13\t29.7\t0\t13\n",
      "7\t136\t7.4\t0\t136\n",
      "8\t134\t1.9\t0\t134\n",
      "9\t421\t0.5\t0\t421\n",
      "10\t559\t0.1\t1\t555 4\n",
      "11\t55\t0.0\t1\t55\n",
      "12\t56\t0.0\t1\t56\n",
      "13\t19\t0.0\t1\t19\n",
      "14\t51\t0.0\t1\t51\n",
      "15\t114\t0.0\t1\t110 4\n",
      "16\t107\t0.0\t1\t104 3\n",
      "17\t244\t0.0\t1\t238 6\n",
      "18\t313\t0.0\t1\t295 13 5\n",
      "19\t622\t0.0\t1\t421 199 2\n",
      "20\t23\t0.0\t2\t0 1 22\n",
      "21\t89\t0.0\t2\t0 89\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/306.H_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/306.H_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/306.H_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/306.H_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.20 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,795\n",
      "  Read 1 with adapter:                      50 (0.0%)\n",
      "  Read 2 with adapter:                   2,999 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       103,795 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,944,069 bp\n",
      "  Read 1:    23,090,982 bp\n",
      "  Read 2:    23,853,087 bp\n",
      "Total written (filtered):     46,903,350 bp (99.9%)\n",
      "  Read 1:    23,089,509 bp\n",
      "  Read 2:    23,813,841 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 50 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t1621.8\t0\t40\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "106\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2999 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1621.8\t0\t12\n",
      "4\t10\t405.4\t0\t10\n",
      "5\t11\t101.4\t0\t11\n",
      "6\t30\t25.3\t0\t30\n",
      "7\t146\t6.3\t0\t146\n",
      "8\t183\t1.6\t0\t183\n",
      "9\t577\t0.4\t0\t577\n",
      "10\t592\t0.1\t1\t590 2\n",
      "11\t45\t0.0\t1\t45\n",
      "12\t43\t0.0\t1\t43\n",
      "13\t23\t0.0\t1\t22 1\n",
      "14\t41\t0.0\t1\t38 3\n",
      "15\t80\t0.0\t1\t77 3\n",
      "16\t95\t0.0\t1\t94 1\n",
      "17\t225\t0.0\t1\t218 7\n",
      "18\t271\t0.0\t1\t250 15 6\n",
      "19\t519\t0.0\t1\t354 165\n",
      "20\t18\t0.0\t2\t0 1 17\n",
      "21\t75\t0.0\t2\t0 75\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/307.H_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/307.H_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/307.H_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/307.H_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.86 s (21 us/read; 2.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            138,101\n",
      "  Read 1 with adapter:                      37 (0.0%)\n",
      "  Read 2 with adapter:                   2,207 (1.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       138,101 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,665,638 bp\n",
      "  Read 1:    30,860,346 bp\n",
      "  Read 2:    31,805,292 bp\n",
      "Total written (filtered):     62,631,682 bp (99.9%)\n",
      "  Read 1:    30,859,892 bp\n",
      "  Read 2:    31,771,790 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 37 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t2157.8\t0\t34\n",
      "25\t1\t0.0\t1\t1\n",
      "112\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2207 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t2157.8\t0\t13\n",
      "4\t3\t539.5\t0\t3\n",
      "5\t7\t134.9\t0\t7\n",
      "6\t8\t33.7\t0\t8\n",
      "7\t29\t8.4\t0\t29\n",
      "8\t47\t2.1\t0\t47\n",
      "9\t173\t0.5\t0\t173\n",
      "10\t388\t0.1\t1\t386 2\n",
      "11\t62\t0.0\t1\t62\n",
      "12\t46\t0.0\t1\t46\n",
      "13\t25\t0.0\t1\t24 1\n",
      "14\t48\t0.0\t1\t46 2\n",
      "15\t93\t0.0\t1\t93\n",
      "16\t126\t0.0\t1\t120 6\n",
      "17\t204\t0.0\t1\t199 5\n",
      "18\t253\t0.0\t1\t244 8 1\n",
      "19\t580\t0.0\t1\t440 139 1\n",
      "20\t10\t0.0\t2\t0 1 9\n",
      "21\t89\t0.0\t2\t0 88 1\n",
      "148\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/308.D_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/308.D_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/308.D_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/308.D_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.27 s (15 us/read; 4.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            293,250\n",
      "  Read 1 with adapter:                     216 (0.1%)\n",
      "  Read 2 with adapter:                  12,352 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       293,250 (100.0%)\n",
      "\n",
      "Total basepairs processed:   133,485,721 bp\n",
      "  Read 1:    65,587,808 bp\n",
      "  Read 2:    67,897,913 bp\n",
      "Total written (filtered):    133,338,463 bp (99.9%)\n",
      "  Read 1:    65,586,293 bp\n",
      "  Read 2:    67,752,170 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 216 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t197\t4582.0\t0\t197\n",
      "25\t7\t0.0\t1\t7\n",
      "27\t8\t0.0\t1\t8\n",
      "28\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "167\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12352 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t4582.0\t0\t91\n",
      "4\t17\t1145.5\t0\t17\n",
      "5\t34\t286.4\t0\t34\n",
      "6\t135\t71.6\t0\t135\n",
      "7\t623\t17.9\t0\t623\n",
      "8\t837\t4.5\t0\t837\n",
      "9\t3012\t1.1\t0\t3010 2\n",
      "10\t3743\t0.3\t1\t3731 12\n",
      "11\t151\t0.1\t1\t150 1\n",
      "12\t94\t0.0\t1\t94\n",
      "13\t95\t0.0\t1\t94 1\n",
      "14\t129\t0.0\t1\t124 5\n",
      "15\t221\t0.0\t1\t218 3\n",
      "16\t239\t0.0\t1\t230 9\n",
      "17\t555\t0.0\t1\t542 13\n",
      "18\t647\t0.0\t1\t611 25 11\n",
      "19\t1476\t0.0\t1\t1047 423 6\n",
      "20\t31\t0.0\t2\t0 1 30\n",
      "21\t211\t0.0\t2\t0 209 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t2\t0.0\t2\t2\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "216\t4\t0.0\t2\t4\n",
      "217\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/308.U_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/308.U_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/308.U_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/308.U_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.80 s (22 us/read; 2.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             83,405\n",
      "  Read 1 with adapter:                      54 (0.1%)\n",
      "  Read 2 with adapter:                   3,085 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        83,405 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,409,091 bp\n",
      "  Read 1:    18,367,330 bp\n",
      "  Read 2:    19,041,761 bp\n",
      "Total written (filtered):     37,366,615 bp (99.9%)\n",
      "  Read 1:    18,365,006 bp\n",
      "  Read 2:    19,001,609 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 54 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t1303.2\t0\t43\n",
      "27\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t2 1\n",
      "217\t6\t0.0\t1\t5 1\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3085 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t1303.2\t0\t19\n",
      "4\t6\t325.8\t0\t6\n",
      "5\t10\t81.5\t0\t10\n",
      "6\t11\t20.4\t0\t11\n",
      "7\t98\t5.1\t0\t98\n",
      "8\t142\t1.3\t0\t142\n",
      "9\t615\t0.3\t0\t613 2\n",
      "10\t824\t0.1\t1\t816 8\n",
      "11\t45\t0.0\t1\t45\n",
      "12\t38\t0.0\t1\t38\n",
      "13\t55\t0.0\t1\t54 1\n",
      "14\t71\t0.0\t1\t67 4\n",
      "15\t87\t0.0\t1\t85 2\n",
      "16\t92\t0.0\t1\t92\n",
      "17\t171\t0.0\t1\t161 10\n",
      "18\t220\t0.0\t1\t211 5 4\n",
      "19\t496\t0.0\t1\t341 153 2\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t65\t0.0\t2\t0 65\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "115\t3\t0.0\t2\t3\n",
      "211\t1\t0.0\t2\t1\n",
      "213\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/300.D_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/300.D_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/300.D_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/300.D_27_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/300.U_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/300.U_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/300.U_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/300.U_6_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/301.D_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/301.D_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/301.D_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/301.D_1_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/301.U_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/301.U_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/301.U_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/301.U_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/302.D_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/302.D_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/302.D_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/302.D_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/302.U_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/302.U_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/302.U_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/302.U_26_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/303.D_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/303.D_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/303.D_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/303.D_35_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/303.U_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/303.U_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/303.U_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/303.U_20_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/304.D_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/304.D_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/304.D_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/304.D_15_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/304.U_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/304.U_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/304.U_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/304.U_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/305.H_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/305.H_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/305.H_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/305.H_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/306.H_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/306.H_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/306.H_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/306.H_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/307.H_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/307.H_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/307.H_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/307.H_7_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/308.D_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/308.D_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/308.D_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/308.D_16_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/308.U_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/308.U_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/308.U_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/308.U_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/310.D_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/310.D_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/310.D_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/310.D_2_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/310.D_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/310.D_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/310.D_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/310.D_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.70 s (17 us/read; 3.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            215,429\n",
      "  Read 1 with adapter:                     171 (0.1%)\n",
      "  Read 2 with adapter:                  11,225 (5.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       215,429 (100.0%)\n",
      "\n",
      "Total basepairs processed:    97,984,811 bp\n",
      "  Read 1:    48,115,229 bp\n",
      "  Read 2:    49,869,582 bp\n",
      "Total written (filtered):     97,851,789 bp (99.9%)\n",
      "  Read 1:    48,113,640 bp\n",
      "  Read 2:    49,738,149 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 171 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t157\t3366.1\t0\t157\n",
      "20\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t2\t0.0\t1\t2\n",
      "77\t4\t0.0\t1\t4\n",
      "100\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11225 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t65\t3366.1\t0\t65\n",
      "4\t8\t841.5\t0\t8\n",
      "5\t33\t210.4\t0\t33\n",
      "6\t123\t52.6\t0\t123\n",
      "7\t568\t13.1\t0\t568\n",
      "8\t783\t3.3\t0\t783\n",
      "9\t2767\t0.8\t0\t2762 5\n",
      "10\t3273\t0.2\t1\t3255 18\n",
      "11\t141\t0.1\t1\t141\n",
      "12\t147\t0.0\t1\t146 1\n",
      "13\t123\t0.0\t1\t122 1\n",
      "14\t190\t0.0\t1\t177 13\n",
      "15\t245\t0.0\t1\t240 5\n",
      "16\t276\t0.0\t1\t260 16\n",
      "17\t527\t0.0\t1\t515 12\n",
      "18\t583\t0.0\t1\t545 23 15\n",
      "19\t1208\t0.0\t1\t800 406 2\n",
      "20\t24\t0.0\t2\t0 2 22\n",
      "21\t133\t0.0\t2\t0 133\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t0 1\n",
      "174\t3\t0.0\t2\t0 0 3\n",
      "214\t1\t0.0\t2\t0 0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/310.U_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/310.U_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/310.U_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/310.U_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.29 s (30 us/read; 1.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,461\n",
      "  Read 1 with adapter:                      22 (0.1%)\n",
      "  Read 2 with adapter:                   1,723 (4.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,461 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,692,467 bp\n",
      "  Read 1:     9,158,897 bp\n",
      "  Read 2:     9,533,570 bp\n",
      "Total written (filtered):     18,668,682 bp (99.9%)\n",
      "  Read 1:     9,157,390 bp\n",
      "  Read 2:     9,511,292 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t663.5\t0\t12\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "41\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1723 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t663.5\t0\t8\n",
      "5\t3\t41.5\t0\t3\n",
      "6\t15\t10.4\t0\t15\n",
      "7\t68\t2.6\t0\t68\n",
      "8\t77\t0.6\t0\t77\n",
      "9\t331\t0.2\t0\t331\n",
      "10\t419\t0.0\t1\t418 1\n",
      "11\t22\t0.0\t1\t22\n",
      "12\t36\t0.0\t1\t36\n",
      "13\t38\t0.0\t1\t38\n",
      "14\t55\t0.0\t1\t54 1\n",
      "15\t64\t0.0\t1\t64\n",
      "16\t54\t0.0\t1\t51 3\n",
      "17\t113\t0.0\t1\t109 4\n",
      "18\t130\t0.0\t1\t124 2 4\n",
      "19\t256\t0.0\t1\t166 89 1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t28\t0.0\t2\t0 28\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/312.D_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/312.D_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/312.D_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/312.D_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.83 s (52 us/read; 1.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,866\n",
      "  Read 1 with adapter:                      16 (0.1%)\n",
      "  Read 2 with adapter:                   1,018 (6.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        15,866 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,487,760 bp\n",
      "  Read 1:     3,151,049 bp\n",
      "  Read 2:     3,336,711 bp\n",
      "Total written (filtered):      6,472,556 bp (99.8%)\n",
      "  Read 1:     3,148,206 bp\n",
      "  Read 2:     3,324,350 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t247.9\t0\t1\n",
      "27\t2\t0.0\t1\t1 1\n",
      "204\t1\t0.0\t1\t0 1\n",
      "209\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "214\t2\t0.0\t1\t2\n",
      "216\t2\t0.0\t1\t1 1\n",
      "217\t5\t0.0\t1\t5\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1018 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t247.9\t0\t13\n",
      "5\t2\t15.5\t0\t2\n",
      "6\t9\t3.9\t0\t9\n",
      "7\t57\t1.0\t0\t57\n",
      "8\t58\t0.2\t0\t58\n",
      "9\t192\t0.1\t0\t192\n",
      "10\t280\t0.0\t1\t278 2\n",
      "11\t31\t0.0\t1\t30 1\n",
      "12\t21\t0.0\t1\t21\n",
      "13\t18\t0.0\t1\t18\n",
      "14\t38\t0.0\t1\t34 4\n",
      "15\t27\t0.0\t1\t27\n",
      "16\t37\t0.0\t1\t37\n",
      "17\t52\t0.0\t1\t45 7\n",
      "18\t62\t0.0\t1\t57 4 1\n",
      "19\t108\t0.0\t1\t69 39\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t9\t0.0\t2\t0 9\n",
      "115\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/312.U_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/312.U_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/312.U_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/312.U_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.86 s (49 us/read; 1.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,399\n",
      "  Read 1 with adapter:                       9 (0.1%)\n",
      "  Read 2 with adapter:                     935 (5.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        17,399 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,368,675 bp\n",
      "  Read 1:     3,592,680 bp\n",
      "  Read 2:     3,775,995 bp\n",
      "Total written (filtered):      7,355,873 bp (99.8%)\n",
      "  Read 1:     3,591,776 bp\n",
      "  Read 2:     3,764,097 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t271.9\t0\t4\n",
      "27\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 935 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t271.9\t0\t6\n",
      "5\t5\t17.0\t0\t5\n",
      "6\t8\t4.2\t0\t8\n",
      "7\t35\t1.1\t0\t35\n",
      "8\t55\t0.3\t0\t55\n",
      "9\t191\t0.1\t0\t191\n",
      "10\t237\t0.0\t1\t237\n",
      "11\t15\t0.0\t1\t15\n",
      "12\t14\t0.0\t1\t14\n",
      "13\t9\t0.0\t1\t9\n",
      "14\t25\t0.0\t1\t25\n",
      "15\t39\t0.0\t1\t38 1\n",
      "16\t24\t0.0\t1\t23 1\n",
      "17\t54\t0.0\t1\t53 1\n",
      "18\t59\t0.0\t1\t55 3 1\n",
      "19\t132\t0.0\t1\t83 49\n",
      "21\t26\t0.0\t2\t0 26\n",
      "230\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/313.H_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/313.H_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/313.H_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/313.H_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.26 s (20 us/read; 3.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            167,406\n",
      "  Read 1 with adapter:                      42 (0.0%)\n",
      "  Read 2 with adapter:                   5,999 (3.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       167,406 (100.0%)\n",
      "\n",
      "Total basepairs processed:    75,701,070 bp\n",
      "  Read 1:    37,245,353 bp\n",
      "  Read 2:    38,455,717 bp\n",
      "Total written (filtered):     75,625,305 bp (99.9%)\n",
      "  Read 1:    37,243,199 bp\n",
      "  Read 2:    38,382,106 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 42 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t2615.7\t0\t31\n",
      "27\t1\t0.0\t1\t1\n",
      "112\t1\t0.0\t1\t1\n",
      "197\t1\t0.0\t1\t1\n",
      "212\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t5\t0.0\t1\t3 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5999 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t2615.7\t0\t40\n",
      "4\t10\t653.9\t0\t10\n",
      "5\t37\t163.5\t0\t37\n",
      "6\t58\t40.9\t0\t58\n",
      "7\t203\t10.2\t0\t203\n",
      "8\t315\t2.6\t0\t315\n",
      "9\t1323\t0.6\t0\t1323\n",
      "10\t1738\t0.2\t1\t1735 3\n",
      "11\t91\t0.0\t1\t90 1\n",
      "12\t88\t0.0\t1\t88\n",
      "13\t62\t0.0\t1\t62\n",
      "14\t94\t0.0\t1\t92 2\n",
      "15\t136\t0.0\t1\t132 4\n",
      "16\t139\t0.0\t1\t134 5\n",
      "17\t332\t0.0\t1\t322 10\n",
      "18\t330\t0.0\t1\t311 14 5\n",
      "19\t852\t0.0\t1\t589 261 2\n",
      "20\t25\t0.0\t2\t0 0 25\n",
      "21\t122\t0.0\t2\t0 122\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/314.H_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/314.H_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/314.H_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/314.H_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.78 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,041\n",
      "  Read 1 with adapter:                     190 (0.3%)\n",
      "  Read 2 with adapter:                   2,178 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        69,041 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,854,479 bp\n",
      "  Read 1:    15,148,343 bp\n",
      "  Read 2:    15,706,136 bp\n",
      "Total written (filtered):     30,816,502 bp (99.9%)\n",
      "  Read 1:    15,139,614 bp\n",
      "  Read 2:    15,676,888 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 190 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t1078.8\t0\t22\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t3\t0.0\t1\t3\n",
      "35\t1\t0.0\t1\t1\n",
      "36\t4\t0.0\t1\t0 4\n",
      "37\t142\t0.0\t1\t136 6\n",
      "38\t2\t0.0\t1\t1 1\n",
      "204\t1\t0.0\t1\t0 1\n",
      "216\t3\t0.0\t1\t2 1\n",
      "217\t9\t0.0\t1\t9\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2178 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t1078.8\t0\t18\n",
      "4\t2\t269.7\t0\t2\n",
      "5\t4\t67.4\t0\t4\n",
      "6\t8\t16.9\t0\t8\n",
      "7\t50\t4.2\t0\t50\n",
      "8\t93\t1.1\t0\t93\n",
      "9\t334\t0.3\t0\t333 1\n",
      "10\t554\t0.1\t1\t550 4\n",
      "11\t28\t0.0\t1\t28\n",
      "12\t29\t0.0\t1\t28 1\n",
      "13\t40\t0.0\t1\t40\n",
      "14\t65\t0.0\t1\t63 2\n",
      "15\t98\t0.0\t1\t98\n",
      "16\t66\t0.0\t1\t65 1\n",
      "17\t155\t0.0\t1\t150 5\n",
      "18\t169\t0.0\t1\t160 6 3\n",
      "19\t404\t0.0\t1\t273 131\n",
      "20\t12\t0.0\t2\t4 0 8\n",
      "21\t46\t0.0\t2\t0 46\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/315.H_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/315.H_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/315.H_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/315.H_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.13 s (17 us/read; 3.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            182,851\n",
      "  Read 1 with adapter:                      70 (0.0%)\n",
      "  Read 2 with adapter:                   4,723 (2.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       182,851 (100.0%)\n",
      "\n",
      "Total basepairs processed:    82,961,009 bp\n",
      "  Read 1:    40,810,013 bp\n",
      "  Read 2:    42,150,996 bp\n",
      "Total written (filtered):     82,895,154 bp (99.9%)\n",
      "  Read 1:    40,809,551 bp\n",
      "  Read 2:    42,085,603 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 70 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t67\t2857.0\t0\t67\n",
      "20\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4723 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t2857.0\t0\t39\n",
      "4\t8\t714.3\t0\t8\n",
      "5\t10\t178.6\t0\t10\n",
      "6\t32\t44.6\t0\t32\n",
      "7\t79\t11.2\t0\t79\n",
      "8\t166\t2.8\t0\t166\n",
      "9\t684\t0.7\t0\t684\n",
      "10\t1132\t0.2\t1\t1125 7\n",
      "11\t59\t0.0\t1\t59\n",
      "12\t70\t0.0\t1\t69 1\n",
      "13\t69\t0.0\t1\t68 1\n",
      "14\t121\t0.0\t1\t114 7\n",
      "15\t131\t0.0\t1\t126 5\n",
      "16\t149\t0.0\t1\t145 4\n",
      "17\t378\t0.0\t1\t370 8\n",
      "18\t400\t0.0\t1\t375 16 9\n",
      "19\t1025\t0.0\t1\t713 312\n",
      "20\t17\t0.0\t2\t0 0 17\n",
      "21\t150\t0.0\t2\t0 149 1\n",
      "78\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/316.D_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/316.D_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/316.D_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/316.D_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.16 s (34 us/read; 1.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,909\n",
      "  Read 1 with adapter:                      15 (0.0%)\n",
      "  Read 2 with adapter:                     594 (1.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,909 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,378,889 bp\n",
      "  Read 1:     7,557,186 bp\n",
      "  Read 2:     7,821,703 bp\n",
      "Total written (filtered):     15,370,001 bp (99.9%)\n",
      "  Read 1:     7,557,141 bp\n",
      "  Read 2:     7,812,860 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 15 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t529.8\t0\t15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 594 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t529.8\t0\t3\n",
      "4\t1\t132.5\t0\t1\n",
      "5\t2\t33.1\t0\t2\n",
      "6\t1\t8.3\t0\t1\n",
      "7\t13\t2.1\t0\t13\n",
      "8\t17\t0.5\t0\t17\n",
      "9\t40\t0.1\t0\t40\n",
      "10\t92\t0.0\t1\t92\n",
      "11\t12\t0.0\t1\t12\n",
      "12\t19\t0.0\t1\t18 1\n",
      "13\t13\t0.0\t1\t13\n",
      "14\t17\t0.0\t1\t17\n",
      "15\t30\t0.0\t1\t28 2\n",
      "16\t36\t0.0\t1\t36\n",
      "17\t60\t0.0\t1\t59 1\n",
      "18\t74\t0.0\t1\t70 3 1\n",
      "19\t139\t0.0\t1\t98 41\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t21\t0.0\t2\t0 21\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/316.U_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/316.U_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/316.U_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/316.U_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.86 s (47 us/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,083\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                     225 (1.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        18,083 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,222,156 bp\n",
      "  Read 1:     4,044,887 bp\n",
      "  Read 2:     4,177,269 bp\n",
      "Total written (filtered):      8,218,536 bp (100.0%)\n",
      "  Read 1:     4,044,860 bp\n",
      "  Read 2:     4,173,676 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t282.5\t0\t9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 225 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t282.5\t0\t2\n",
      "5\t2\t17.7\t0\t2\n",
      "6\t1\t4.4\t0\t1\n",
      "7\t6\t1.1\t0\t6\n",
      "8\t5\t0.3\t0\t5\n",
      "9\t4\t0.1\t0\t4\n",
      "10\t19\t0.0\t1\t19\n",
      "11\t5\t0.0\t1\t5\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t5\t0.0\t1\t5\n",
      "15\t13\t0.0\t1\t13\n",
      "16\t16\t0.0\t1\t16\n",
      "17\t30\t0.0\t1\t27 3\n",
      "18\t36\t0.0\t1\t32 4\n",
      "19\t61\t0.0\t1\t34 27\n",
      "21\t13\t0.0\t2\t0 13\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/317.D_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/317.D_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/317.D_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/317.D_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.68 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,007\n",
      "  Read 1 with adapter:                      29 (0.0%)\n",
      "  Read 2 with adapter:                   1,246 (1.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        65,007 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,574,316 bp\n",
      "  Read 1:    14,545,623 bp\n",
      "  Read 2:    15,028,693 bp\n",
      "Total written (filtered):     29,555,739 bp (99.9%)\n",
      "  Read 1:    14,545,536 bp\n",
      "  Read 2:    15,010,203 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1015.7\t0\t29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1246 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1015.7\t0\t6\n",
      "4\t2\t253.9\t0\t2\n",
      "5\t3\t63.5\t0\t3\n",
      "6\t7\t15.9\t0\t7\n",
      "7\t36\t4.0\t0\t36\n",
      "8\t27\t1.0\t0\t27\n",
      "9\t78\t0.2\t0\t78\n",
      "10\t194\t0.1\t1\t193 1\n",
      "11\t41\t0.0\t1\t41\n",
      "12\t33\t0.0\t1\t33\n",
      "13\t14\t0.0\t1\t14\n",
      "14\t31\t0.0\t1\t31\n",
      "15\t77\t0.0\t1\t76 1\n",
      "16\t98\t0.0\t1\t95 3\n",
      "17\t105\t0.0\t1\t103 2\n",
      "18\t177\t0.0\t1\t173 3 1\n",
      "19\t256\t0.0\t1\t190 66\n",
      "20\t12\t0.0\t2\t0 1 11\n",
      "21\t48\t0.0\t2\t0 47 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/317.U_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/317.U_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/317.U_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/317.U_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.11 s (33 us/read; 1.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,796\n",
      "  Read 1 with adapter:                      17 (0.1%)\n",
      "  Read 2 with adapter:                     604 (1.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,796 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,299,820 bp\n",
      "  Read 1:     7,524,725 bp\n",
      "  Read 2:     7,775,095 bp\n",
      "Total written (filtered):     15,290,419 bp (99.9%)\n",
      "  Read 1:     7,524,464 bp\n",
      "  Read 2:     7,765,955 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 17 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t528.1\t0\t16\n",
      "213\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 604 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t528.1\t0\t4\n",
      "4\t1\t132.0\t0\t1\n",
      "5\t3\t33.0\t0\t3\n",
      "6\t5\t8.3\t0\t5\n",
      "7\t21\t2.1\t0\t21\n",
      "8\t16\t0.5\t0\t16\n",
      "9\t41\t0.1\t0\t41\n",
      "10\t64\t0.0\t1\t64\n",
      "11\t15\t0.0\t1\t15\n",
      "12\t10\t0.0\t1\t10\n",
      "13\t10\t0.0\t1\t10\n",
      "14\t15\t0.0\t1\t15\n",
      "15\t35\t0.0\t1\t35\n",
      "16\t42\t0.0\t1\t40 2\n",
      "17\t50\t0.0\t1\t49 1\n",
      "18\t98\t0.0\t1\t93 4 1\n",
      "19\t146\t0.0\t1\t102 44\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t25\t0.0\t2\t0 25\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/318.D_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/318.D_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/318.D_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/318.D_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.56 s (16 us/read; 3.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            158,838\n",
      "  Read 1 with adapter:                      80 (0.1%)\n",
      "  Read 2 with adapter:                   3,228 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       158,838 (100.0%)\n",
      "\n",
      "Total basepairs processed:    72,298,152 bp\n",
      "  Read 1:    35,558,173 bp\n",
      "  Read 2:    36,739,979 bp\n",
      "Total written (filtered):     72,253,309 bp (99.9%)\n",
      "  Read 1:    35,557,720 bp\n",
      "  Read 2:    36,695,589 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 80 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t74\t2481.8\t0\t74\n",
      "24\t1\t0.0\t1\t0 1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "101\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3228 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t2481.8\t0\t20\n",
      "4\t5\t620.5\t0\t5\n",
      "5\t10\t155.1\t0\t10\n",
      "6\t24\t38.8\t0\t24\n",
      "7\t164\t9.7\t0\t164\n",
      "8\t148\t2.4\t0\t148\n",
      "9\t455\t0.6\t0\t455\n",
      "10\t587\t0.2\t1\t584 3\n",
      "11\t70\t0.0\t1\t69 1\n",
      "12\t44\t0.0\t1\t44\n",
      "13\t20\t0.0\t1\t20\n",
      "14\t51\t0.0\t1\t51\n",
      "15\t113\t0.0\t1\t111 2\n",
      "16\t159\t0.0\t1\t154 5\n",
      "17\t227\t0.0\t1\t227\n",
      "18\t337\t0.0\t1\t326 7 4\n",
      "19\t649\t0.0\t1\t464 183 2\n",
      "20\t20\t0.0\t2\t0 0 20\n",
      "21\t122\t0.0\t2\t0 122\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/318.U_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/318.U_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/318.U_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/318.U_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.07 s (26 us/read; 2.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,946\n",
      "  Read 1 with adapter:                      51 (0.1%)\n",
      "  Read 2 with adapter:                   1,494 (1.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        79,946 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,294,568 bp\n",
      "  Read 1:    17,840,447 bp\n",
      "  Read 2:    18,454,121 bp\n",
      "Total written (filtered):     36,271,639 bp (99.9%)\n",
      "  Read 1:    17,839,937 bp\n",
      "  Read 2:    18,431,702 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 51 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t1249.2\t0\t46\n",
      "4\t1\t312.3\t0\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "61\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "181\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1494 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1249.2\t0\t7\n",
      "4\t1\t312.3\t0\t1\n",
      "5\t5\t78.1\t0\t5\n",
      "6\t5\t19.5\t0\t5\n",
      "7\t48\t4.9\t0\t48\n",
      "8\t46\t1.2\t0\t46\n",
      "9\t116\t0.3\t0\t116\n",
      "10\t200\t0.1\t1\t200\n",
      "11\t35\t0.0\t1\t33 2\n",
      "12\t36\t0.0\t1\t35 1\n",
      "13\t36\t0.0\t1\t36\n",
      "14\t18\t0.0\t1\t17 1\n",
      "15\t64\t0.0\t1\t62 2\n",
      "16\t82\t0.0\t1\t79 3\n",
      "17\t155\t0.0\t1\t154 1\n",
      "18\t198\t0.0\t1\t195 2 1\n",
      "19\t376\t0.0\t1\t258 118\n",
      "20\t11\t0.0\t2\t0 1 10\n",
      "21\t54\t0.0\t2\t0 54\n",
      "85\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/319.D_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/319.D_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/319.D_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/319.D_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.56 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,473\n",
      "  Read 1 with adapter:                      44 (0.0%)\n",
      "  Read 2 with adapter:                   1,842 (1.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,473 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,508,279 bp\n",
      "  Read 1:    24,344,811 bp\n",
      "  Read 2:    25,163,468 bp\n",
      "Total written (filtered):     49,478,847 bp (99.9%)\n",
      "  Read 1:    24,343,828 bp\n",
      "  Read 2:    25,135,019 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 44 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t1710.5\t0\t40\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1842 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1710.5\t0\t8\n",
      "4\t1\t427.6\t0\t1\n",
      "5\t3\t106.9\t0\t3\n",
      "6\t3\t26.7\t0\t3\n",
      "7\t23\t6.7\t0\t23\n",
      "8\t27\t1.7\t0\t27\n",
      "9\t123\t0.4\t0\t122 1\n",
      "10\t284\t0.1\t1\t283 1\n",
      "11\t29\t0.0\t1\t29\n",
      "12\t55\t0.0\t1\t55\n",
      "13\t46\t0.0\t1\t46\n",
      "14\t63\t0.0\t1\t63\n",
      "15\t81\t0.0\t1\t80 1\n",
      "16\t83\t0.0\t1\t79 4\n",
      "17\t188\t0.0\t1\t181 7\n",
      "18\t228\t0.0\t1\t218 8 2\n",
      "19\t496\t0.0\t1\t347 148 1\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t87\t0.0\t2\t0 86 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/319.U_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/319.U_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/319.U_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/319.U_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.05 s (44 us/read; 1.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,782\n",
      "  Read 1 with adapter:                      12 (0.1%)\n",
      "  Read 2 with adapter:                     690 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,782 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,346,048 bp\n",
      "  Read 1:     5,069,721 bp\n",
      "  Read 2:     5,276,327 bp\n",
      "Total written (filtered):     10,336,065 bp (99.9%)\n",
      "  Read 1:     5,069,382 bp\n",
      "  Read 2:     5,266,683 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t371.6\t0\t10\n",
      "92\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 690 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t371.6\t0\t1\n",
      "4\t1\t92.9\t0\t1\n",
      "5\t2\t23.2\t0\t2\n",
      "7\t15\t1.5\t0\t15\n",
      "8\t23\t0.4\t0\t23\n",
      "9\t89\t0.1\t0\t89\n",
      "10\t160\t0.0\t1\t157 3\n",
      "11\t19\t0.0\t1\t19\n",
      "12\t11\t0.0\t1\t11\n",
      "13\t12\t0.0\t1\t12\n",
      "14\t25\t0.0\t1\t22 3\n",
      "15\t22\t0.0\t1\t22\n",
      "16\t24\t0.0\t1\t23 1\n",
      "17\t61\t0.0\t1\t58 3\n",
      "18\t64\t0.0\t1\t59 5\n",
      "19\t144\t0.0\t1\t90 54\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t13\t0.0\t2\t0 12 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/320.D_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/320.D_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/320.D_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/320.D_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.68 s (21 us/read; 2.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            128,217\n",
      "  Read 1 with adapter:                     100 (0.1%)\n",
      "  Read 2 with adapter:                   2,638 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       128,217 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,075,276 bp\n",
      "  Read 1:    28,549,823 bp\n",
      "  Read 2:    29,525,453 bp\n",
      "Total written (filtered):     58,036,263 bp (99.9%)\n",
      "  Read 1:    28,548,389 bp\n",
      "  Read 2:    29,487,874 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 100 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t2003.4\t0\t91\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "75\t1\t0.0\t1\t1\n",
      "144\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2638 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t2003.4\t0\t17\n",
      "4\t4\t500.8\t0\t4\n",
      "5\t4\t125.2\t0\t4\n",
      "6\t7\t31.3\t0\t7\n",
      "7\t43\t7.8\t0\t43\n",
      "8\t75\t2.0\t0\t75\n",
      "9\t302\t0.5\t0\t302\n",
      "10\t570\t0.1\t1\t568 2\n",
      "11\t47\t0.0\t1\t46 1\n",
      "12\t61\t0.0\t1\t61\n",
      "13\t42\t0.0\t1\t42\n",
      "14\t52\t0.0\t1\t50 2\n",
      "15\t103\t0.0\t1\t100 3\n",
      "16\t128\t0.0\t1\t125 3\n",
      "17\t237\t0.0\t1\t232 5\n",
      "18\t260\t0.0\t1\t243 12 5\n",
      "19\t572\t0.0\t1\t390 182\n",
      "20\t13\t0.0\t2\t0 1 12\n",
      "21\t99\t0.0\t2\t0 98 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/310.U_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/310.U_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/310.U_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/310.U_4_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/312.D_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/312.D_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/312.D_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/312.D_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/312.U_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/312.U_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/312.U_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/312.U_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/313.H_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/313.H_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/313.H_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/313.H_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/314.H_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/314.H_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/314.H_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/314.H_25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/315.H_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/315.H_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/315.H_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/315.H_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/316.D_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/316.D_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/316.D_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/316.D_33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/316.U_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/316.U_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/316.U_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/316.U_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/317.D_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/317.D_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/317.D_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/317.D_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/317.U_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/317.U_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/317.U_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/317.U_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/318.D_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/318.D_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/318.D_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/318.D_17_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/318.U_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/318.U_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/318.U_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/318.U_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/319.D_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/319.D_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/319.D_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/319.D_32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/319.U_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/319.U_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/319.U_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/319.U_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/320.D_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/320.D_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/320.D_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/320.D_24_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/320.U_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/320.U_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/320.U_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/320.U_45_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/320.U_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/320.U_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/320.U_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/320.U_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.39 s (31 us/read; 1.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,810\n",
      "  Read 1 with adapter:                      22 (0.0%)\n",
      "  Read 2 with adapter:                     560 (1.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,810 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,209,612 bp\n",
      "  Read 1:     9,939,552 bp\n",
      "  Read 2:    10,270,060 bp\n",
      "Total written (filtered):     20,199,175 bp (99.9%)\n",
      "  Read 1:     9,938,421 bp\n",
      "  Read 2:    10,260,754 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t700.2\t0\t17\n",
      "216\t5\t0.0\t1\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 560 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t700.2\t0\t1\n",
      "5\t1\t43.8\t0\t1\n",
      "6\t2\t10.9\t0\t2\n",
      "7\t4\t2.7\t0\t4\n",
      "8\t2\t0.7\t0\t2\n",
      "9\t16\t0.2\t0\t16\n",
      "10\t58\t0.0\t1\t57 1\n",
      "11\t11\t0.0\t1\t11\n",
      "12\t18\t0.0\t1\t18\n",
      "13\t14\t0.0\t1\t14\n",
      "14\t17\t0.0\t1\t17\n",
      "15\t22\t0.0\t1\t22\n",
      "16\t32\t0.0\t1\t31 1\n",
      "17\t56\t0.0\t1\t55 1\n",
      "18\t92\t0.0\t1\t91 1\n",
      "19\t186\t0.0\t1\t131 55\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t21\t0.0\t2\t0 19 2\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/321.H_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/321.H_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/321.H_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/321.H_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.80 s (22 us/read; 2.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,844\n",
      "  Read 1 with adapter:                      54 (0.1%)\n",
      "  Read 2 with adapter:                   1,598 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        81,844 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,064,562 bp\n",
      "  Read 1:    18,222,193 bp\n",
      "  Read 2:    18,842,369 bp\n",
      "Total written (filtered):     37,040,149 bp (99.9%)\n",
      "  Read 1:    18,221,617 bp\n",
      "  Read 2:    18,818,532 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 54 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t51\t1278.8\t0\t51\n",
      "27\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t0 1\n",
      "204\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1598 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1278.8\t0\t12\n",
      "4\t3\t319.7\t0\t3\n",
      "5\t5\t79.9\t0\t5\n",
      "6\t4\t20.0\t0\t4\n",
      "7\t45\t5.0\t0\t45\n",
      "8\t46\t1.2\t0\t46\n",
      "9\t159\t0.3\t0\t159\n",
      "10\t218\t0.1\t1\t217 1\n",
      "11\t39\t0.0\t1\t38 1\n",
      "12\t38\t0.0\t1\t37 1\n",
      "13\t17\t0.0\t1\t17\n",
      "14\t25\t0.0\t1\t24 1\n",
      "15\t85\t0.0\t1\t85\n",
      "16\t75\t0.0\t1\t74 1\n",
      "17\t149\t0.0\t1\t146 3\n",
      "18\t203\t0.0\t1\t191 9 3\n",
      "19\t416\t0.0\t1\t284 132\n",
      "20\t11\t0.0\t2\t1 0 10\n",
      "21\t43\t0.0\t2\t0 41 2\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "213\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/322.H_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/322.H_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/322.H_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/322.H_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.94 s (50 us/read; 1.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,876\n",
      "  Read 1 with adapter:                      13 (0.1%)\n",
      "  Read 2 with adapter:                     415 (2.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        18,876 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,319,118 bp\n",
      "  Read 1:     4,084,630 bp\n",
      "  Read 2:     4,234,488 bp\n",
      "Total written (filtered):      8,312,382 bp (99.9%)\n",
      "  Read 1:     4,083,718 bp\n",
      "  Read 2:     4,228,664 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t294.9\t0\t8\n",
      "25\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "216\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 415 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t294.9\t0\t2\n",
      "5\t1\t18.4\t0\t1\n",
      "6\t3\t4.6\t0\t3\n",
      "7\t9\t1.2\t0\t9\n",
      "8\t11\t0.3\t0\t11\n",
      "9\t60\t0.1\t0\t60\n",
      "10\t88\t0.0\t1\t88\n",
      "11\t8\t0.0\t1\t8\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t5\t0.0\t1\t5\n",
      "14\t12\t0.0\t1\t11 1\n",
      "15\t10\t0.0\t1\t10\n",
      "16\t12\t0.0\t1\t12\n",
      "17\t38\t0.0\t1\t38\n",
      "18\t33\t0.0\t1\t32 1\n",
      "19\t98\t0.0\t1\t61 37\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t15\t0.0\t2\t0 15\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/323.H_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/323.H_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/323.H_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/323.H_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.73 s (26 us/read; 2.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,710\n",
      "  Read 1 with adapter:                      36 (0.1%)\n",
      "  Read 2 with adapter:                     860 (1.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        66,710 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,180,038 bp\n",
      "  Read 1:    14,843,906 bp\n",
      "  Read 2:    15,336,132 bp\n",
      "Total written (filtered):     30,165,849 bp (100.0%)\n",
      "  Read 1:    14,843,584 bp\n",
      "  Read 2:    15,322,265 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 36 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t1042.3\t0\t35\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 860 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1042.3\t0\t3\n",
      "5\t2\t65.1\t0\t2\n",
      "6\t2\t16.3\t0\t2\n",
      "7\t9\t4.1\t0\t9\n",
      "8\t8\t1.0\t0\t8\n",
      "9\t29\t0.3\t0\t29\n",
      "10\t87\t0.1\t1\t87\n",
      "11\t22\t0.0\t1\t22\n",
      "12\t26\t0.0\t1\t26\n",
      "13\t18\t0.0\t1\t18\n",
      "14\t25\t0.0\t1\t25\n",
      "15\t49\t0.0\t1\t48 1\n",
      "16\t48\t0.0\t1\t48\n",
      "17\t92\t0.0\t1\t92\n",
      "18\t107\t0.0\t1\t102 2 3\n",
      "19\t271\t0.0\t1\t207 63 1\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t56\t0.0\t2\t0 56\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/324.D_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/324.D_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/324.D_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/324.D_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.89 s (18 us/read; 3.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            269,723\n",
      "  Read 1 with adapter:                     161 (0.1%)\n",
      "  Read 2 with adapter:                   9,716 (3.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       269,723 (100.0%)\n",
      "\n",
      "Total basepairs processed:   122,874,968 bp\n",
      "  Read 1:    60,391,188 bp\n",
      "  Read 2:    62,483,780 bp\n",
      "Total written (filtered):    122,750,856 bp (99.9%)\n",
      "  Read 1:    60,388,868 bp\n",
      "  Read 2:    62,361,988 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 161 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t143\t4214.4\t0\t143\n",
      "9\t1\t1.0\t0\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t0 1\n",
      "101\t9\t0.0\t1\t8 1\n",
      "120\t3\t0.0\t1\t3\n",
      "141\t1\t0.0\t1\t1\n",
      "170\t1\t0.0\t1\t1\n",
      "175\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9716 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t67\t4214.4\t0\t67\n",
      "4\t12\t1053.6\t0\t12\n",
      "5\t27\t263.4\t0\t27\n",
      "6\t88\t65.9\t0\t88\n",
      "7\t322\t16.5\t0\t322\n",
      "8\t513\t4.1\t0\t513\n",
      "9\t1927\t1.0\t0\t1922 5\n",
      "10\t2856\t0.3\t1\t2846 10\n",
      "11\t131\t0.1\t1\t129 2\n",
      "12\t112\t0.0\t1\t112\n",
      "13\t99\t0.0\t1\t97 2\n",
      "14\t178\t0.0\t1\t175 3\n",
      "15\t250\t0.0\t1\t244 6\n",
      "16\t281\t0.0\t1\t272 9\n",
      "17\t545\t0.0\t1\t524 21\n",
      "18\t614\t0.0\t1\t574 29 11\n",
      "19\t1484\t0.0\t1\t1045 437 2\n",
      "20\t36\t0.0\t2\t0 1 35\n",
      "21\t164\t0.0\t2\t0 161 3\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t0 1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "171\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/324.U_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/324.U_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/324.U_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/324.U_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.75 s (19 us/read; 3.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            253,779\n",
      "  Read 1 with adapter:                     147 (0.1%)\n",
      "  Read 2 with adapter:                   6,533 (2.6%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       253,778 (100.0%)\n",
      "\n",
      "Total basepairs processed:   115,573,050 bp\n",
      "  Read 1:    56,830,952 bp\n",
      "  Read 2:    58,742,098 bp\n",
      "Total written (filtered):    115,482,578 bp (99.9%)\n",
      "  Read 1:    56,830,108 bp\n",
      "  Read 2:    58,652,470 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 147 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t143\t3965.3\t0\t143\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "112\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6533 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t3965.3\t0\t35\n",
      "4\t6\t991.3\t0\t6\n",
      "5\t6\t247.8\t0\t6\n",
      "6\t33\t62.0\t0\t33\n",
      "7\t149\t15.5\t0\t149\n",
      "8\t252\t3.9\t0\t252\n",
      "9\t975\t1.0\t0\t975\n",
      "10\t1527\t0.2\t1\t1520 7\n",
      "11\t98\t0.1\t1\t98\n",
      "12\t115\t0.0\t1\t113 2\n",
      "13\t100\t0.0\t1\t100\n",
      "14\t160\t0.0\t1\t156 4\n",
      "15\t207\t0.0\t1\t199 8\n",
      "16\t272\t0.0\t1\t265 7\n",
      "17\t569\t0.0\t1\t555 14\n",
      "18\t584\t0.0\t1\t552 21 11\n",
      "19\t1226\t0.0\t1\t836 389 1\n",
      "20\t35\t0.0\t2\t0 0 35\n",
      "21\t176\t0.0\t2\t0 174 2\n",
      "22\t4\t0.0\t2\t0 1 3\n",
      "86\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "231\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/325.D_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/325.D_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/325.D_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/325.D_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.73 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            223,358\n",
      "  Read 1 with adapter:                     235 (0.1%)\n",
      "  Read 2 with adapter:                   7,553 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       223,358 (100.0%)\n",
      "\n",
      "Total basepairs processed:   101,735,182 bp\n",
      "  Read 1:    50,005,631 bp\n",
      "  Read 2:    51,729,551 bp\n",
      "Total written (filtered):    101,622,499 bp (99.9%)\n",
      "  Read 1:    49,992,386 bp\n",
      "  Read 2:    51,630,113 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 235 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t110\t3490.0\t0\t110\n",
      "9\t1\t0.9\t0\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "100\t9\t0.0\t1\t9\n",
      "101\t75\t0.0\t1\t74 1\n",
      "102\t3\t0.0\t1\t3\n",
      "118\t1\t0.0\t1\t1\n",
      "119\t2\t0.0\t1\t2\n",
      "120\t22\t0.0\t1\t22\n",
      "125\t1\t0.0\t1\t1\n",
      "127\t4\t0.0\t1\t4\n",
      "178\t1\t0.0\t1\t1\n",
      "188\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7553 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t3490.0\t0\t32\n",
      "4\t8\t872.5\t0\t8\n",
      "5\t12\t218.1\t0\t12\n",
      "6\t53\t54.5\t0\t53\n",
      "7\t235\t13.6\t0\t235\n",
      "8\t343\t3.4\t0\t343\n",
      "9\t1245\t0.9\t0\t1245\n",
      "10\t1869\t0.2\t1\t1860 9\n",
      "11\t102\t0.1\t1\t102\n",
      "12\t170\t0.0\t1\t169 1\n",
      "13\t148\t0.0\t1\t147 1\n",
      "14\t256\t0.0\t1\t252 4\n",
      "15\t252\t0.0\t1\t244 8\n",
      "16\t311\t0.0\t1\t302 9\n",
      "17\t546\t0.0\t1\t531 15\n",
      "18\t543\t0.0\t1\t509 18 16\n",
      "19\t1250\t0.0\t1\t891 358 1\n",
      "20\t23\t0.0\t2\t0 0 23\n",
      "21\t146\t0.0\t2\t0 145 1\n",
      "22\t3\t0.0\t2\t0 1 2\n",
      "58\t2\t0.0\t2\t1 1\n",
      "132\t1\t0.0\t2\t0 1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "205\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/325.U_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/325.U_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/325.U_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/325.U_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.69 s (19 us/read; 3.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            353,385\n",
      "  Read 1 with adapter:                     279 (0.1%)\n",
      "  Read 2 with adapter:                  10,424 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       353,385 (100.0%)\n",
      "\n",
      "Total basepairs processed:   160,921,612 bp\n",
      "  Read 1:    79,117,114 bp\n",
      "  Read 2:    81,804,498 bp\n",
      "Total written (filtered):    160,773,674 bp (99.9%)\n",
      "  Read 1:    79,110,467 bp\n",
      "  Read 2:    81,663,207 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 279 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t227\t5521.6\t0\t227\n",
      "25\t2\t0.0\t1\t2\n",
      "37\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t1\n",
      "90\t1\t0.0\t1\t1\n",
      "119\t43\t0.0\t1\t40 3\n",
      "120\t1\t0.0\t1\t1\n",
      "144\t1\t0.0\t1\t0 1\n",
      "153\t1\t0.0\t1\t1\n",
      "167\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10424 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t5521.6\t0\t48\n",
      "4\t8\t1380.4\t0\t8\n",
      "5\t27\t345.1\t0\t27\n",
      "6\t52\t86.3\t0\t52\n",
      "7\t304\t21.6\t0\t304\n",
      "8\t399\t5.4\t0\t399\n",
      "9\t1593\t1.3\t0\t1592 1\n",
      "10\t2610\t0.3\t1\t2597 13\n",
      "11\t176\t0.1\t1\t176\n",
      "12\t202\t0.0\t1\t202\n",
      "13\t197\t0.0\t1\t193 4\n",
      "14\t314\t0.0\t1\t303 11\n",
      "15\t338\t0.0\t1\t335 3\n",
      "16\t358\t0.0\t1\t345 13\n",
      "17\t690\t0.0\t1\t671 19\n",
      "18\t843\t0.0\t1\t787 38 18\n",
      "19\t1964\t0.0\t1\t1357 602 5\n",
      "20\t53\t0.0\t2\t0 2 51\n",
      "21\t231\t0.0\t2\t0 230 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t1\n",
      "132\t3\t0.0\t2\t0 3\n",
      "153\t1\t0.0\t2\t0 1\n",
      "154\t2\t0.0\t2\t0 2\n",
      "166\t1\t0.0\t2\t1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "189\t1\t0.0\t2\t1\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "216\t3\t0.0\t2\t3\n",
      "217\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/326.D_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/326.D_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/326.D_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/326.D_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.09 s (19 us/read; 3.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            380,162\n",
      "  Read 1 with adapter:                     211 (0.1%)\n",
      "  Read 2 with adapter:                  11,506 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       380,162 (100.0%)\n",
      "\n",
      "Total basepairs processed:   173,119,174 bp\n",
      "  Read 1:    85,104,022 bp\n",
      "  Read 2:    88,015,152 bp\n",
      "Total written (filtered):    172,961,364 bp (99.9%)\n",
      "  Read 1:    85,102,592 bp\n",
      "  Read 2:    87,858,772 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 211 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t188\t5940.0\t0\t188\n",
      "21\t2\t0.0\t1\t1 1\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t17\t0.0\t1\t16 1\n",
      "101\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11506 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t5940.0\t0\t61\n",
      "4\t11\t1485.0\t0\t11\n",
      "5\t19\t371.3\t0\t19\n",
      "6\t51\t92.8\t0\t51\n",
      "7\t298\t23.2\t0\t298\n",
      "8\t399\t5.8\t0\t399\n",
      "9\t1665\t1.5\t0\t1663 2\n",
      "10\t2745\t0.4\t1\t2737 8\n",
      "11\t196\t0.1\t1\t194 2\n",
      "12\t244\t0.0\t1\t244\n",
      "13\t244\t0.0\t1\t241 3\n",
      "14\t355\t0.0\t1\t341 14\n",
      "15\t401\t0.0\t1\t397 4\n",
      "16\t431\t0.0\t1\t416 15\n",
      "17\t868\t0.0\t1\t841 27\n",
      "18\t1009\t0.0\t1\t949 38 22\n",
      "19\t2193\t0.0\t1\t1500 692 1\n",
      "20\t43\t0.0\t2\t0 3 40\n",
      "21\t266\t0.0\t2\t0 265 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "154\t1\t0.0\t2\t0 1\n",
      "217\t3\t0.0\t2\t3\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/326.U_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/326.U_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/326.U_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/326.U_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.39 s (18 us/read; 3.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            246,106\n",
      "  Read 1 with adapter:                     174 (0.1%)\n",
      "  Read 2 with adapter:                   6,980 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       246,106 (100.0%)\n",
      "\n",
      "Total basepairs processed:   112,005,434 bp\n",
      "  Read 1:    55,045,674 bp\n",
      "  Read 2:    56,959,760 bp\n",
      "Total written (filtered):    111,910,590 bp (99.9%)\n",
      "  Read 1:    55,044,844 bp\n",
      "  Read 2:    56,865,746 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 174 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t169\t3845.4\t0\t169\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "223\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6980 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t3845.4\t0\t46\n",
      "4\t5\t961.4\t0\t5\n",
      "5\t8\t240.3\t0\t8\n",
      "6\t38\t60.1\t0\t38\n",
      "7\t178\t15.0\t0\t178\n",
      "8\t289\t3.8\t0\t289\n",
      "9\t1141\t0.9\t0\t1141\n",
      "10\t1727\t0.2\t1\t1722 5\n",
      "11\t115\t0.1\t1\t115\n",
      "12\t124\t0.0\t1\t123 1\n",
      "13\t113\t0.0\t1\t113\n",
      "14\t163\t0.0\t1\t160 3\n",
      "15\t215\t0.0\t1\t210 5\n",
      "16\t261\t0.0\t1\t254 7\n",
      "17\t521\t0.0\t1\t507 14\n",
      "18\t555\t0.0\t1\t524 22 9\n",
      "19\t1260\t0.0\t1\t854 404 2\n",
      "20\t30\t0.0\t2\t1 0 29\n",
      "21\t178\t0.0\t2\t0 174 4\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "53\t2\t0.0\t2\t1 1\n",
      "132\t7\t0.0\t2\t1 6\n",
      "157\t1\t0.0\t2\t1\n",
      "230\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/327.D_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/327.D_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/327.D_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/327.D_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.68 s (18 us/read; 3.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            380,440\n",
      "  Read 1 with adapter:                     267 (0.1%)\n",
      "  Read 2 with adapter:                  11,790 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       380,440 (100.0%)\n",
      "\n",
      "Total basepairs processed:   173,288,319 bp\n",
      "  Read 1:    85,194,133 bp\n",
      "  Read 2:    88,094,186 bp\n",
      "Total written (filtered):    173,131,976 bp (99.9%)\n",
      "  Read 1:    85,192,724 bp\n",
      "  Read 2:    87,939,252 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 267 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t259\t5944.4\t0\t259\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "120\t1\t0.0\t1\t1\n",
      "127\t2\t0.0\t1\t2\n",
      "153\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11790 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t5944.4\t0\t61\n",
      "4\t11\t1486.1\t0\t11\n",
      "5\t25\t371.5\t0\t25\n",
      "6\t90\t92.9\t0\t90\n",
      "7\t335\t23.2\t0\t335\n",
      "8\t554\t5.8\t0\t554\n",
      "9\t2134\t1.5\t0\t2134\n",
      "10\t2896\t0.4\t1\t2874 22\n",
      "11\t195\t0.1\t1\t193 2\n",
      "12\t224\t0.0\t1\t218 6\n",
      "13\t234\t0.0\t1\t234\n",
      "14\t279\t0.0\t1\t276 3\n",
      "15\t338\t0.0\t1\t331 7\n",
      "16\t371\t0.0\t1\t356 15\n",
      "17\t773\t0.0\t1\t754 19\n",
      "18\t862\t0.0\t1\t812 35 15\n",
      "19\t2102\t0.0\t1\t1413 685 4\n",
      "20\t47\t0.0\t2\t0 1 46\n",
      "21\t245\t0.0\t2\t0 244 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t1 1\n",
      "146\t1\t0.0\t2\t1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "202\t1\t0.0\t2\t1\n",
      "205\t1\t0.0\t2\t1\n",
      "206\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/327.U_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/327.U_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/327.U_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/327.U_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.10 s (18 us/read; 3.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            331,500\n",
      "  Read 1 with adapter:                     189 (0.1%)\n",
      "  Read 2 with adapter:                   7,737 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       331,500 (100.0%)\n",
      "\n",
      "Total basepairs processed:   150,952,304 bp\n",
      "  Read 1:    74,236,081 bp\n",
      "  Read 2:    76,716,223 bp\n",
      "Total written (filtered):    150,843,960 bp (99.9%)\n",
      "  Read 1:    74,235,126 bp\n",
      "  Read 2:    76,608,834 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 189 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t181\t5179.7\t0\t181\n",
      "12\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t0 1\n",
      "27\t3\t0.0\t1\t2 1\n",
      "55\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7737 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t5179.7\t0\t45\n",
      "4\t10\t1294.9\t0\t10\n",
      "5\t14\t323.7\t0\t14\n",
      "6\t39\t80.9\t0\t39\n",
      "7\t180\t20.2\t0\t180\n",
      "8\t263\t5.1\t0\t263\n",
      "9\t1074\t1.3\t0\t1072 2\n",
      "10\t1729\t0.3\t1\t1722 7\n",
      "11\t145\t0.1\t1\t142 3\n",
      "12\t164\t0.0\t1\t160 4\n",
      "13\t156\t0.0\t1\t155 1\n",
      "14\t212\t0.0\t1\t205 7\n",
      "15\t275\t0.0\t1\t268 7\n",
      "16\t272\t0.0\t1\t268 4\n",
      "17\t533\t0.0\t1\t522 11\n",
      "18\t732\t0.0\t1\t697 26 9\n",
      "19\t1599\t0.0\t1\t1113 483 3\n",
      "20\t45\t0.0\t2\t2 0 43\n",
      "21\t244\t0.0\t2\t0 244\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "213\t1\t0.0\t2\t0 0 1\n",
      "214\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/328.D_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/328.D_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/328.D_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/328.D_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.79 s (20 us/read; 3.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            291,878\n",
      "  Read 1 with adapter:                     150 (0.1%)\n",
      "  Read 2 with adapter:                   9,834 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       291,878 (100.0%)\n",
      "\n",
      "Total basepairs processed:   132,935,644 bp\n",
      "  Read 1:    65,343,599 bp\n",
      "  Read 2:    67,592,045 bp\n",
      "Total written (filtered):    132,804,402 bp (99.9%)\n",
      "  Read 1:    65,342,131 bp\n",
      "  Read 2:    67,462,271 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 150 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t131\t4560.6\t0\t131\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t1\t0.0\t1\t0 1\n",
      "27\t9\t0.0\t1\t9\n",
      "81\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "119\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "199\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9834 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t63\t4560.6\t0\t63\n",
      "4\t9\t1140.1\t0\t9\n",
      "5\t23\t285.0\t0\t23\n",
      "6\t64\t71.3\t0\t64\n",
      "7\t324\t17.8\t0\t324\n",
      "8\t428\t4.5\t0\t428\n",
      "9\t1686\t1.1\t0\t1684 2\n",
      "10\t2454\t0.3\t1\t2449 5\n",
      "11\t130\t0.1\t1\t130\n",
      "12\t198\t0.0\t1\t196 2\n",
      "13\t180\t0.0\t1\t179 1\n",
      "14\t268\t0.0\t1\t255 13\n",
      "15\t335\t0.0\t1\t325 10\n",
      "16\t343\t0.0\t1\t329 14\n",
      "17\t666\t0.0\t1\t651 15\n",
      "18\t731\t0.0\t1\t678 34 19\n",
      "19\t1686\t0.0\t1\t1138 547 1\n",
      "20\t42\t0.0\t2\t0 1 41\n",
      "21\t190\t0.0\t2\t0 189 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "132\t2\t0.0\t2\t0 2\n",
      "147\t1\t0.0\t2\t1\n",
      "153\t5\t0.0\t2\t1 3 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/328.U_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/328.U_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/328.U_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/328.U_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.26 s (20 us/read; 3.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            214,455\n",
      "  Read 1 with adapter:                     148 (0.1%)\n",
      "  Read 2 with adapter:                   5,964 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       214,455 (100.0%)\n",
      "\n",
      "Total basepairs processed:    97,635,649 bp\n",
      "  Read 1:    48,003,476 bp\n",
      "  Read 2:    49,632,173 bp\n",
      "Total written (filtered):     97,554,624 bp (99.9%)\n",
      "  Read 1:    48,002,896 bp\n",
      "  Read 2:    49,551,728 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 148 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t145\t3350.9\t0\t145\n",
      "27\t2\t0.0\t1\t2\n",
      "91\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5964 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t3350.9\t0\t26\n",
      "4\t3\t837.7\t0\t3\n",
      "5\t10\t209.4\t0\t10\n",
      "6\t34\t52.4\t0\t34\n",
      "7\t172\t13.1\t0\t172\n",
      "8\t247\t3.3\t0\t247\n",
      "9\t965\t0.8\t0\t965\n",
      "10\t1355\t0.2\t1\t1346 9\n",
      "11\t92\t0.1\t1\t92\n",
      "12\t130\t0.0\t1\t128 2\n",
      "13\t125\t0.0\t1\t125\n",
      "14\t176\t0.0\t1\t172 4\n",
      "15\t206\t0.0\t1\t200 6\n",
      "16\t234\t0.0\t1\t229 5\n",
      "17\t459\t0.0\t1\t447 12\n",
      "18\t479\t0.0\t1\t446 18 15\n",
      "19\t1065\t0.0\t1\t705 360\n",
      "20\t34\t0.0\t2\t0 2 32\n",
      "21\t146\t0.0\t2\t0 146\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "153\t2\t0.0\t2\t1 1\n",
      "216\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/329.H_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/329.H_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/329.H_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/329.H_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.16 s (20 us/read; 2.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            251,551\n",
      "  Read 1 with adapter:                     182 (0.1%)\n",
      "  Read 2 with adapter:                   5,211 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       251,551 (100.0%)\n",
      "\n",
      "Total basepairs processed:   114,523,856 bp\n",
      "  Read 1:    56,328,704 bp\n",
      "  Read 2:    58,195,152 bp\n",
      "Total written (filtered):    114,446,251 bp (99.9%)\n",
      "  Read 1:    56,327,545 bp\n",
      "  Read 2:    58,118,706 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 182 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t173\t3930.5\t0\t173\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "63\t1\t0.0\t1\t1\n",
      "120\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5211 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t3930.5\t0\t16\n",
      "4\t5\t982.6\t0\t5\n",
      "5\t6\t245.7\t0\t6\n",
      "6\t15\t61.4\t0\t15\n",
      "7\t75\t15.4\t0\t75\n",
      "8\t117\t3.8\t0\t117\n",
      "9\t532\t1.0\t0\t532\n",
      "10\t1022\t0.2\t1\t1018 4\n",
      "11\t94\t0.1\t1\t94\n",
      "12\t137\t0.0\t1\t135 2\n",
      "13\t128\t0.0\t1\t126 2\n",
      "14\t183\t0.0\t1\t176 7\n",
      "15\t257\t0.0\t1\t252 5\n",
      "16\t263\t0.0\t1\t256 7\n",
      "17\t485\t0.0\t1\t477 8\n",
      "18\t518\t0.0\t1\t499 15 4\n",
      "19\t1132\t0.0\t1\t809 322 1\n",
      "20\t38\t0.0\t2\t0 1 37\n",
      "21\t180\t0.0\t2\t0 179 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t1\n",
      "201\t1\t0.0\t2\t1\n",
      "206\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "224\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/330.H_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/330.H_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/330.H_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/330.H_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.51 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,061\n",
      "  Read 1 with adapter:                      31 (0.1%)\n",
      "  Read 2 with adapter:                     933 (1.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,061 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,517,655 bp\n",
      "  Read 1:    12,551,497 bp\n",
      "  Read 2:    12,966,158 bp\n",
      "Total written (filtered):     25,502,436 bp (99.9%)\n",
      "  Read 1:    12,551,301 bp\n",
      "  Read 2:    12,951,135 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 31 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t876.0\t0\t29\n",
      "25\t1\t0.0\t1\t1\n",
      "84\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 933 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t876.0\t0\t2\n",
      "4\t3\t219.0\t0\t3\n",
      "5\t2\t54.7\t0\t2\n",
      "6\t2\t13.7\t0\t2\n",
      "7\t18\t3.4\t0\t18\n",
      "8\t16\t0.9\t0\t16\n",
      "9\t41\t0.2\t0\t41\n",
      "10\t130\t0.1\t1\t130\n",
      "11\t26\t0.0\t1\t26\n",
      "12\t34\t0.0\t1\t34\n",
      "13\t19\t0.0\t1\t19\n",
      "14\t32\t0.0\t1\t30 2\n",
      "15\t54\t0.0\t1\t54\n",
      "16\t62\t0.0\t1\t60 2\n",
      "17\t118\t0.0\t1\t116 2\n",
      "18\t113\t0.0\t1\t109 4\n",
      "19\t207\t0.0\t1\t149 58\n",
      "20\t9\t0.0\t2\t0 0 9\n",
      "21\t41\t0.0\t2\t0 40 1\n",
      "209\t1\t0.0\t2\t1\n",
      "214\t2\t0.0\t2\t1 1\n",
      "218\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/321.H_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/321.H_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/321.H_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/321.H_21_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/322.H_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/322.H_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/322.H_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/322.H_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/323.H_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/323.H_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/323.H_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/323.H_18_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/324.D_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/324.D_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/324.D_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/324.D_29_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/324.U_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/324.U_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/324.U_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/324.U_12_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/325.D_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/325.D_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/325.D_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/325.D_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/325.U_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/325.U_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/325.U_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/325.U_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/326.D_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/326.D_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/326.D_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/326.D_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/326.U_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/326.U_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/326.U_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/326.U_31_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/327.D_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/327.D_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/327.D_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/327.D_9_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/327.U_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/327.U_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/327.U_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/327.U_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/328.D_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/328.D_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/328.D_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/328.D_8_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/328.U_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/328.U_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/328.U_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/328.U_14_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/329.H_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/329.H_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/329.H_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/329.H_11_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/330.H_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/330.H_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/330.H_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/330.H_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/331.H_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/331.H_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/331.H_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/331.H_3_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/331.H_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-3we7mpfr/331.H_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/331.H_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-7c1ebxkv/a1fe7ad6-17f3-41b3-8b71-ecf04aeea4f2/data/331.H_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.40 s (19 us/read; 3.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            331,030\n",
      "  Read 1 with adapter:                     160 (0.0%)\n",
      "  Read 2 with adapter:                   9,227 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       331,030 (100.0%)\n",
      "\n",
      "Total basepairs processed:   150,744,553 bp\n",
      "  Read 1:    74,118,694 bp\n",
      "  Read 2:    76,625,859 bp\n",
      "Total written (filtered):    150,618,668 bp (99.9%)\n",
      "  Read 1:    74,118,025 bp\n",
      "  Read 2:    76,500,643 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 160 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t157\t5172.3\t0\t157\n",
      "27\t1\t0.0\t1\t1\n",
      "51\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9227 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t5172.3\t0\t49\n",
      "4\t4\t1293.1\t0\t4\n",
      "5\t16\t323.3\t0\t16\n",
      "6\t53\t80.8\t0\t53\n",
      "7\t248\t20.2\t0\t248\n",
      "8\t326\t5.1\t0\t326\n",
      "9\t1423\t1.3\t0\t1423\n",
      "10\t2255\t0.3\t1\t2244 11\n",
      "11\t168\t0.1\t1\t168\n",
      "12\t181\t0.0\t1\t179 2\n",
      "13\t161\t0.0\t1\t158 3\n",
      "14\t216\t0.0\t1\t209 7\n",
      "15\t306\t0.0\t1\t296 10\n",
      "16\t353\t0.0\t1\t342 11\n",
      "17\t674\t0.0\t1\t658 16\n",
      "18\t749\t0.0\t1\t703 31 15\n",
      "19\t1750\t0.0\t1\t1186 561 3\n",
      "20\t48\t0.0\t2\t0 1 47\n",
      "21\t234\t0.0\t2\t0 232 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t1\n",
      "126\t1\t0.0\t2\t1\n",
      "132\t4\t0.0\t2\t2 2\n",
      "153\t1\t0.0\t2\t0 1\n",
      "168\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGmicro2018_1-48.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/421.D_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/421.D_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/421.D_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/421.D_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.48 s (22 us/read; 2.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,968\n",
      "  Read 1 with adapter:                      56 (0.1%)\n",
      "  Read 2 with adapter:                   3,547 (3.2%)\n",
      "Pairs that were too short:                   2 (0.0%)\n",
      "Pairs written (passing filters):       110,966 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,439,296 bp\n",
      "  Read 1:    24,799,328 bp\n",
      "  Read 2:    25,639,968 bp\n",
      "Total written (filtered):     50,391,955 bp (99.9%)\n",
      "  Read 1:    24,797,522 bp\n",
      "  Read 2:    25,594,433 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 56 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t1733.9\t0\t43\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t4\t0.0\t1\t4\n",
      "210\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t3\n",
      "223\t2\t0.0\t1\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3547 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1733.9\t0\t29\n",
      "4\t7\t433.5\t0\t7\n",
      "5\t9\t108.4\t0\t9\n",
      "6\t18\t27.1\t0\t18\n",
      "7\t104\t6.8\t0\t104\n",
      "8\t166\t1.7\t0\t166\n",
      "9\t588\t0.4\t0\t588\n",
      "10\t1168\t0.1\t1\t1162 6\n",
      "11\t51\t0.0\t1\t51\n",
      "12\t57\t0.0\t1\t57\n",
      "13\t48\t0.0\t1\t47 1\n",
      "14\t58\t0.0\t1\t56 2\n",
      "15\t70\t0.0\t1\t69 1\n",
      "16\t92\t0.0\t1\t88 4\n",
      "17\t214\t0.0\t1\t209 5\n",
      "18\t252\t0.0\t1\t240 9 3\n",
      "19\t503\t0.0\t1\t342 160 1\n",
      "20\t18\t0.0\t2\t0 1 17\n",
      "21\t91\t0.0\t2\t0 91\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t1 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/422.D_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/422.D_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/422.D_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/422.D_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.70 s (16 us/read; 3.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            172,003\n",
      "  Read 1 with adapter:                      99 (0.1%)\n",
      "  Read 2 with adapter:                   2,368 (1.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       172,003 (100.0%)\n",
      "\n",
      "Total basepairs processed:    78,295,147 bp\n",
      "  Read 1:    38,518,926 bp\n",
      "  Read 2:    39,776,221 bp\n",
      "Total written (filtered):     78,257,842 bp (100.0%)\n",
      "  Read 1:    38,518,088 bp\n",
      "  Read 2:    39,739,754 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t94\t2687.5\t0\t94\n",
      "38\t1\t0.0\t1\t1\n",
      "75\t2\t0.0\t1\t2\n",
      "171\t1\t0.0\t1\t1\n",
      "197\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2368 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t2687.5\t0\t13\n",
      "4\t1\t671.9\t0\t1\n",
      "5\t3\t168.0\t0\t3\n",
      "6\t8\t42.0\t0\t8\n",
      "7\t34\t10.5\t0\t34\n",
      "8\t44\t2.6\t0\t44\n",
      "9\t176\t0.7\t0\t176\n",
      "10\t454\t0.2\t1\t450 4\n",
      "11\t25\t0.0\t1\t25\n",
      "12\t35\t0.0\t1\t35\n",
      "13\t46\t0.0\t1\t46\n",
      "14\t79\t0.0\t1\t79\n",
      "15\t76\t0.0\t1\t75 1\n",
      "16\t85\t0.0\t1\t83 2\n",
      "17\t219\t0.0\t1\t211 8\n",
      "18\t263\t0.0\t1\t250 7 6\n",
      "19\t654\t0.0\t1\t431 222 1\n",
      "20\t22\t0.0\t2\t0 3 19\n",
      "21\t125\t0.0\t2\t0 121 4\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "86\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/422.U_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/422.U_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/422.U_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/422.U_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.73 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            167,961\n",
      "  Read 1 with adapter:                      92 (0.1%)\n",
      "  Read 2 with adapter:                   3,779 (2.2%)\n",
      "Pairs that were too short:                   2 (0.0%)\n",
      "Pairs written (passing filters):       167,959 (100.0%)\n",
      "\n",
      "Total basepairs processed:    76,448,593 bp\n",
      "  Read 1:    37,591,957 bp\n",
      "  Read 2:    38,856,636 bp\n",
      "Total written (filtered):     76,394,312 bp (99.9%)\n",
      "  Read 1:    37,590,800 bp\n",
      "  Read 2:    38,803,512 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 92 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t89\t2624.4\t0\t89\n",
      "25\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3779 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t2624.4\t0\t13\n",
      "4\t1\t656.1\t0\t1\n",
      "5\t11\t164.0\t0\t11\n",
      "6\t18\t41.0\t0\t18\n",
      "7\t82\t10.3\t0\t82\n",
      "8\t146\t2.6\t0\t146\n",
      "9\t550\t0.6\t0\t548 2\n",
      "10\t903\t0.2\t1\t900 3\n",
      "11\t50\t0.0\t1\t50\n",
      "12\t75\t0.0\t1\t75\n",
      "13\t74\t0.0\t1\t74\n",
      "14\t84\t0.0\t1\t83 1\n",
      "15\t110\t0.0\t1\t107 3\n",
      "16\t111\t0.0\t1\t108 3\n",
      "17\t296\t0.0\t1\t287 9\n",
      "18\t343\t0.0\t1\t323 13 7\n",
      "19\t755\t0.0\t1\t495 259 1\n",
      "20\t22\t0.0\t2\t0 2 20\n",
      "21\t124\t0.0\t2\t0 123 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t2\t0.0\t2\t2\n",
      "230\t1\t0.0\t2\t0 0 1\n",
      "231\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/423.D_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/423.D_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/423.D_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/423.D_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.18 s (34 us/read; 1.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,450\n",
      "  Read 1 with adapter:                      21 (0.1%)\n",
      "  Read 2 with adapter:                     777 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,450 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,564,248 bp\n",
      "  Read 1:     7,651,501 bp\n",
      "  Read 2:     7,912,747 bp\n",
      "Total written (filtered):     15,553,329 bp (99.9%)\n",
      "  Read 1:     7,651,438 bp\n",
      "  Read 2:     7,901,891 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t538.3\t0\t21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 777 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t538.3\t0\t3\n",
      "4\t1\t134.6\t0\t1\n",
      "6\t1\t8.4\t0\t1\n",
      "7\t15\t2.1\t0\t15\n",
      "8\t24\t0.5\t0\t24\n",
      "9\t117\t0.1\t0\t117\n",
      "10\t198\t0.0\t1\t196 2\n",
      "11\t10\t0.0\t1\t10\n",
      "12\t10\t0.0\t1\t10\n",
      "13\t12\t0.0\t1\t12\n",
      "14\t16\t0.0\t1\t16\n",
      "15\t24\t0.0\t1\t23 1\n",
      "16\t42\t0.0\t1\t41 1\n",
      "17\t73\t0.0\t1\t72 1\n",
      "18\t70\t0.0\t1\t67 2 1\n",
      "19\t132\t0.0\t1\t84 48\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t21\t0.0\t2\t0 21\n",
      "77\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/423.U_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/423.U_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/423.U_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/423.U_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.48 s (29 us/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,871\n",
      "  Read 1 with adapter:                      21 (0.0%)\n",
      "  Read 2 with adapter:                     445 (0.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,871 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,562,905 bp\n",
      "  Read 1:    11,592,663 bp\n",
      "  Read 2:    11,970,242 bp\n",
      "Total written (filtered):     23,555,173 bp (100.0%)\n",
      "  Read 1:    11,592,487 bp\n",
      "  Read 2:    11,962,686 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t810.5\t0\t19\n",
      "27\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 445 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t810.5\t0\t1\n",
      "8\t1\t0.8\t0\t1\n",
      "9\t23\t0.2\t0\t23\n",
      "10\t37\t0.0\t1\t37\n",
      "11\t7\t0.0\t1\t7\n",
      "12\t13\t0.0\t1\t12 1\n",
      "13\t8\t0.0\t1\t8\n",
      "14\t13\t0.0\t1\t13\n",
      "15\t18\t0.0\t1\t18\n",
      "16\t32\t0.0\t1\t31 1\n",
      "17\t47\t0.0\t1\t46 1\n",
      "18\t70\t0.0\t1\t69 1\n",
      "19\t130\t0.0\t1\t95 35\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t39\t0.0\t2\t0 39\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/424.D_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/424.D_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/424.D_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/424.D_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.23 s (15 us/read; 3.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            213,921\n",
      "  Read 1 with adapter:                      89 (0.0%)\n",
      "  Read 2 with adapter:                   3,342 (1.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       213,921 (100.0%)\n",
      "\n",
      "Total basepairs processed:    97,379,549 bp\n",
      "  Read 1:    47,903,018 bp\n",
      "  Read 2:    49,476,531 bp\n",
      "Total written (filtered):     97,328,710 bp (99.9%)\n",
      "  Read 1:    47,902,383 bp\n",
      "  Read 2:    49,426,327 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 89 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t87\t3342.5\t0\t87\n",
      "156\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3342 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t3342.5\t0\t21\n",
      "4\t2\t835.6\t0\t2\n",
      "5\t4\t208.9\t0\t4\n",
      "6\t8\t52.2\t0\t8\n",
      "7\t57\t13.1\t0\t57\n",
      "8\t76\t3.3\t0\t76\n",
      "9\t314\t0.8\t0\t313 1\n",
      "10\t709\t0.2\t1\t704 5\n",
      "11\t42\t0.1\t1\t41 1\n",
      "12\t64\t0.0\t1\t63 1\n",
      "13\t53\t0.0\t1\t53\n",
      "14\t100\t0.0\t1\t96 4\n",
      "15\t99\t0.0\t1\t97 2\n",
      "16\t110\t0.0\t1\t106 4\n",
      "17\t309\t0.0\t1\t306 3\n",
      "18\t321\t0.0\t1\t304 12 5\n",
      "19\t875\t0.0\t1\t589 284 2\n",
      "20\t25\t0.0\t2\t0 0 25\n",
      "21\t143\t0.0\t2\t0 143\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 1\n",
      "92\t3\t0.0\t2\t3\n",
      "215\t2\t0.0\t2\t1 1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/424.U_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/424.U_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/424.U_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/424.U_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.98 s (22 us/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            137,736\n",
      "  Read 1 with adapter:                      65 (0.0%)\n",
      "  Read 2 with adapter:                   2,660 (1.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       137,736 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,621,004 bp\n",
      "  Read 1:    30,798,980 bp\n",
      "  Read 2:    31,822,024 bp\n",
      "Total written (filtered):     62,581,329 bp (99.9%)\n",
      "  Read 1:    30,798,128 bp\n",
      "  Read 2:    31,783,201 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 65 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t2152.1\t0\t61\n",
      "20\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2660 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t2152.1\t0\t10\n",
      "4\t5\t538.0\t0\t5\n",
      "5\t8\t134.5\t0\t8\n",
      "6\t7\t33.6\t0\t7\n",
      "7\t41\t8.4\t0\t41\n",
      "8\t77\t2.1\t0\t77\n",
      "9\t295\t0.5\t0\t294 1\n",
      "10\t536\t0.1\t1\t534 2\n",
      "11\t33\t0.0\t1\t33\n",
      "12\t61\t0.0\t1\t61\n",
      "13\t60\t0.0\t1\t59 1\n",
      "14\t88\t0.0\t1\t84 4\n",
      "15\t116\t0.0\t1\t114 2\n",
      "16\t104\t0.0\t1\t101 3\n",
      "17\t207\t0.0\t1\t201 6\n",
      "18\t308\t0.0\t1\t291 12 5\n",
      "19\t583\t0.0\t1\t384 199\n",
      "20\t19\t0.0\t2\t0 1 18\n",
      "21\t94\t0.0\t2\t0 93 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "214\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/425.H_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/425.H_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/425.H_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/425.H_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.91 s (55 us/read; 1.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,606\n",
      "  Read 1 with adapter:                      12 (0.1%)\n",
      "  Read 2 with adapter:                     609 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        16,606 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,357,066 bp\n",
      "  Read 1:     3,606,491 bp\n",
      "  Read 2:     3,750,575 bp\n",
      "Total written (filtered):      7,348,577 bp (99.9%)\n",
      "  Read 1:     3,606,033 bp\n",
      "  Read 2:     3,742,544 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t259.5\t0\t10\n",
      "212\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 609 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t259.5\t0\t2\n",
      "4\t1\t64.9\t0\t1\n",
      "5\t2\t16.2\t0\t2\n",
      "6\t2\t4.1\t0\t2\n",
      "7\t14\t1.0\t0\t14\n",
      "8\t27\t0.3\t0\t27\n",
      "9\t91\t0.1\t0\t91\n",
      "10\t179\t0.0\t1\t179\n",
      "11\t10\t0.0\t1\t10\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t8\t0.0\t1\t8\n",
      "14\t21\t0.0\t1\t19 2\n",
      "15\t22\t0.0\t1\t21 1\n",
      "16\t17\t0.0\t1\t17\n",
      "17\t49\t0.0\t1\t49\n",
      "18\t55\t0.0\t1\t52 2 1\n",
      "19\t92\t0.0\t1\t56 35 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t8\t0.0\t2\t0 8\n",
      "115\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/426.H_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/426.H_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/426.H_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/426.H_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.28 s (34 us/read; 1.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,647\n",
      "  Read 1 with adapter:                      23 (0.1%)\n",
      "  Read 2 with adapter:                   1,024 (2.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        37,647 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,935,490 bp\n",
      "  Read 1:     8,318,681 bp\n",
      "  Read 2:     8,616,809 bp\n",
      "Total written (filtered):     16,920,017 bp (99.9%)\n",
      "  Read 1:     8,317,335 bp\n",
      "  Read 2:     8,602,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t588.2\t0\t16\n",
      "25\t1\t0.0\t1\t1\n",
      "203\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t0 1\n",
      "216\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1024 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t588.2\t0\t4\n",
      "5\t2\t36.8\t0\t2\n",
      "7\t26\t2.3\t0\t26\n",
      "8\t40\t0.6\t0\t40\n",
      "9\t128\t0.1\t0\t128\n",
      "10\t234\t0.0\t1\t233 1\n",
      "11\t21\t0.0\t1\t21\n",
      "12\t20\t0.0\t1\t19 1\n",
      "13\t30\t0.0\t1\t30\n",
      "14\t30\t0.0\t1\t28 2\n",
      "15\t32\t0.0\t1\t32\n",
      "16\t55\t0.0\t1\t54 1\n",
      "17\t89\t0.0\t1\t85 4\n",
      "18\t83\t0.0\t1\t82 0 1\n",
      "19\t200\t0.0\t1\t139 61\n",
      "20\t7\t0.0\t2\t0 0 7\n",
      "21\t22\t0.0\t2\t0 22\n",
      "61\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/427.H_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/427.H_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/427.H_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/427.H_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.11 s (39 us/read; 1.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,526\n",
      "  Read 1 with adapter:                      13 (0.0%)\n",
      "  Read 2 with adapter:                     715 (2.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,526 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,844,563 bp\n",
      "  Read 1:     6,310,261 bp\n",
      "  Read 2:     6,534,302 bp\n",
      "Total written (filtered):     12,834,294 bp (99.9%)\n",
      "  Read 1:     6,309,582 bp\n",
      "  Read 2:     6,524,712 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t445.7\t0\t10\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 715 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t445.7\t0\t4\n",
      "4\t2\t111.4\t0\t2\n",
      "5\t1\t27.9\t0\t1\n",
      "6\t4\t7.0\t0\t4\n",
      "7\t19\t1.7\t0\t19\n",
      "8\t21\t0.4\t0\t21\n",
      "9\t91\t0.1\t0\t91\n",
      "10\t177\t0.0\t1\t177\n",
      "11\t16\t0.0\t1\t16\n",
      "12\t18\t0.0\t1\t18\n",
      "13\t20\t0.0\t1\t20\n",
      "14\t31\t0.0\t1\t30 1\n",
      "15\t25\t0.0\t1\t25\n",
      "16\t36\t0.0\t1\t34 2\n",
      "17\t55\t0.0\t1\t52 3\n",
      "18\t58\t0.0\t1\t57 1\n",
      "19\t120\t0.0\t1\t77 43\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t15\t0.0\t2\t0 15\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/428.D_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/428.D_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/428.D_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/428.D_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.42 s (17 us/read; 3.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            255,855\n",
      "  Read 1 with adapter:                     118 (0.0%)\n",
      "  Read 2 with adapter:                   6,371 (2.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       255,855 (100.0%)\n",
      "\n",
      "Total basepairs processed:   116,053,008 bp\n",
      "  Read 1:    57,061,508 bp\n",
      "  Read 2:    58,991,500 bp\n",
      "Total written (filtered):    115,962,860 bp (99.9%)\n",
      "  Read 1:    57,058,422 bp\n",
      "  Read 2:    58,904,438 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 118 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t99\t3997.7\t0\t99\n",
      "21\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t2\t0.0\t1\t0 2\n",
      "111\t1\t0.0\t1\t1\n",
      "160\t1\t0.0\t1\t1\n",
      "208\t1\t0.0\t1\t0 1\n",
      "214\t2\t0.0\t1\t2\n",
      "215\t2\t0.0\t1\t2\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6371 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t3997.7\t0\t28\n",
      "4\t9\t999.4\t0\t9\n",
      "5\t19\t249.9\t0\t19\n",
      "6\t62\t62.5\t0\t62\n",
      "7\t157\t15.6\t0\t157\n",
      "8\t291\t3.9\t0\t291\n",
      "9\t1038\t1.0\t0\t1038\n",
      "10\t1515\t0.2\t1\t1509 6\n",
      "11\t87\t0.1\t1\t86 1\n",
      "12\t76\t0.0\t1\t75 1\n",
      "13\t71\t0.0\t1\t71\n",
      "14\t138\t0.0\t1\t133 5\n",
      "15\t176\t0.0\t1\t171 5\n",
      "16\t227\t0.0\t1\t220 7\n",
      "17\t528\t0.0\t1\t513 15\n",
      "18\t527\t0.0\t1\t491 27 9\n",
      "19\t1214\t0.0\t1\t812 401 1\n",
      "20\t33\t0.0\t2\t0 0 33\n",
      "21\t164\t0.0\t2\t0 162 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "204\t1\t0.0\t2\t0 0 1\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "216\t4\t0.0\t2\t4\n",
      "220\t1\t0.0\t2\t1\n",
      "229\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/428.U_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/428.U_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/428.U_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/428.U_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.41 s (19 us/read; 3.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            180,475\n",
      "  Read 1 with adapter:                      42 (0.0%)\n",
      "  Read 2 with adapter:                   5,173 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       180,475 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,928,471 bp\n",
      "  Read 1:    40,328,359 bp\n",
      "  Read 2:    41,600,112 bp\n",
      "Total written (filtered):     81,861,586 bp (99.9%)\n",
      "  Read 1:    40,327,665 bp\n",
      "  Read 2:    41,533,921 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 42 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t2819.9\t0\t37\n",
      "23\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5173 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t2819.9\t0\t36\n",
      "4\t10\t705.0\t0\t10\n",
      "5\t17\t176.2\t0\t17\n",
      "6\t49\t44.1\t0\t49\n",
      "7\t200\t11.0\t0\t200\n",
      "8\t271\t2.8\t0\t271\n",
      "9\t1014\t0.7\t0\t1014\n",
      "10\t1518\t0.2\t1\t1515 3\n",
      "11\t82\t0.0\t1\t82\n",
      "12\t72\t0.0\t1\t71 1\n",
      "13\t42\t0.0\t1\t42\n",
      "14\t54\t0.0\t1\t49 5\n",
      "15\t95\t0.0\t1\t94 1\n",
      "16\t152\t0.0\t1\t146 6\n",
      "17\t282\t0.0\t1\t273 9\n",
      "18\t353\t0.0\t1\t338 13 2\n",
      "19\t784\t0.0\t1\t521 261 2\n",
      "20\t19\t0.0\t2\t0 2 17\n",
      "21\t112\t0.0\t2\t0 111 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t1\n",
      "172\t1\t0.0\t2\t1\n",
      "194\t1\t0.0\t2\t1\n",
      "217\t4\t0.0\t2\t3 1\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t1\n",
      "231\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/429.D_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/429.D_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/429.D_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/429.D_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.80 s (20 us/read; 3.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            194,209\n",
      "  Read 1 with adapter:                      75 (0.0%)\n",
      "  Read 2 with adapter:                   6,810 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       194,209 (100.0%)\n",
      "\n",
      "Total basepairs processed:    88,318,342 bp\n",
      "  Read 1:    43,411,338 bp\n",
      "  Read 2:    44,907,004 bp\n",
      "Total written (filtered):     88,232,112 bp (99.9%)\n",
      "  Read 1:    43,410,052 bp\n",
      "  Read 2:    44,822,060 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 75 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t67\t3034.5\t0\t67\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "92\t2\t0.0\t1\t2\n",
      "197\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6810 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t3034.5\t0\t43\n",
      "4\t12\t758.6\t0\t12\n",
      "5\t22\t189.7\t0\t22\n",
      "6\t57\t47.4\t0\t57\n",
      "7\t282\t11.9\t0\t282\n",
      "8\t395\t3.0\t0\t395\n",
      "9\t1427\t0.7\t0\t1426 1\n",
      "10\t1870\t0.2\t1\t1856 14\n",
      "11\t78\t0.0\t1\t77 1\n",
      "12\t76\t0.0\t1\t75 1\n",
      "13\t104\t0.0\t1\t104\n",
      "14\t137\t0.0\t1\t135 2\n",
      "15\t160\t0.0\t1\t159 1\n",
      "16\t216\t0.0\t1\t208 8\n",
      "17\t376\t0.0\t1\t370 6\n",
      "18\t471\t0.0\t1\t436 24 11\n",
      "19\t919\t0.0\t1\t574 342 3\n",
      "20\t26\t0.0\t2\t0 0 26\n",
      "21\t129\t0.0\t2\t0 129\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t1\n",
      "115\t2\t0.0\t2\t2\n",
      "199\t1\t0.0\t2\t0 0 1\n",
      "216\t3\t0.0\t2\t3\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/429.U_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/429.U_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/429.U_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/429.U_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.46 s (20 us/read; 3.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            175,638\n",
      "  Read 1 with adapter:                      42 (0.0%)\n",
      "  Read 2 with adapter:                   5,700 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       175,638 (100.0%)\n",
      "\n",
      "Total basepairs processed:    79,814,707 bp\n",
      "  Read 1:    39,272,012 bp\n",
      "  Read 2:    40,542,695 bp\n",
      "Total written (filtered):     79,743,302 bp (99.9%)\n",
      "  Read 1:    39,271,228 bp\n",
      "  Read 2:    40,472,074 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 42 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t38\t2744.3\t0\t38\n",
      "25\t1\t0.0\t1\t1\n",
      "211\t1\t0.0\t1\t0 1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5700 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t2744.3\t0\t43\n",
      "4\t4\t686.1\t0\t4\n",
      "5\t23\t171.5\t0\t23\n",
      "6\t69\t42.9\t0\t69\n",
      "7\t198\t10.7\t0\t198\n",
      "8\t325\t2.7\t0\t325\n",
      "9\t1255\t0.7\t0\t1253 2\n",
      "10\t1603\t0.2\t1\t1594 9\n",
      "11\t81\t0.0\t1\t81\n",
      "12\t69\t0.0\t1\t69\n",
      "13\t58\t0.0\t1\t58\n",
      "14\t112\t0.0\t1\t109 3\n",
      "15\t123\t0.0\t1\t121 2\n",
      "16\t159\t0.0\t1\t152 7\n",
      "17\t319\t0.0\t1\t312 7\n",
      "18\t350\t0.0\t1\t333 14 3\n",
      "19\t776\t0.0\t1\t538 237 1\n",
      "20\t18\t0.0\t2\t0 0 18\n",
      "21\t110\t0.0\t2\t0 110\n",
      "205\t1\t0.0\t2\t0 0 1\n",
      "216\t4\t0.0\t2\t4\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/430.D_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/430.D_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/430.D_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/430.D_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.70 s (21 us/read; 2.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,742\n",
      "  Read 1 with adapter:                      35 (0.0%)\n",
      "  Read 2 with adapter:                   5,060 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       130,742 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,234,627 bp\n",
      "  Read 1:    29,136,818 bp\n",
      "  Read 2:    30,097,809 bp\n",
      "Total written (filtered):     59,170,050 bp (99.9%)\n",
      "  Read 1:    29,134,432 bp\n",
      "  Read 2:    30,035,618 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 35 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t2042.8\t0\t24\n",
      "171\t1\t0.0\t1\t1\n",
      "207\t2\t0.0\t1\t2\n",
      "210\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t5\t0.0\t1\t5\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5060 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t2042.8\t0\t48\n",
      "4\t12\t510.7\t0\t12\n",
      "5\t24\t127.7\t0\t24\n",
      "6\t61\t31.9\t0\t61\n",
      "7\t198\t8.0\t0\t198\n",
      "8\t299\t2.0\t0\t299\n",
      "9\t1186\t0.5\t0\t1186\n",
      "10\t1369\t0.1\t1\t1364 5\n",
      "11\t77\t0.0\t1\t76 1\n",
      "12\t69\t0.0\t1\t67 2\n",
      "13\t59\t0.0\t1\t59\n",
      "14\t77\t0.0\t1\t75 2\n",
      "15\t94\t0.0\t1\t94\n",
      "16\t136\t0.0\t1\t129 7\n",
      "17\t257\t0.0\t1\t252 5\n",
      "18\t284\t0.0\t1\t267 8 9\n",
      "19\t683\t0.0\t1\t437 242 4\n",
      "20\t24\t0.0\t2\t0 3 21\n",
      "21\t96\t0.0\t2\t0 96\n",
      "115\t2\t0.0\t2\t2\n",
      "215\t2\t0.0\t2\t1 1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "219\t1\t0.0\t2\t1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/421.D_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/421.D_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/421.D_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/421.D_17_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/422.D_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/422.D_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/422.D_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/422.D_11_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/422.U_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/422.U_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/422.U_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/422.U_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/423.D_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/423.D_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/423.D_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/423.D_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/423.U_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/423.U_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/423.U_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/423.U_26_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/424.D_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/424.D_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/424.D_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/424.D_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/424.U_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/424.U_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/424.U_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/424.U_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/425.H_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/425.H_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/425.H_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/425.H_1_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/426.H_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/426.H_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/426.H_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/426.H_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/427.H_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/427.H_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/427.H_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/427.H_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/428.D_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/428.D_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/428.D_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/428.D_21_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/428.U_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/428.U_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/428.U_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/428.U_3_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/429.D_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/429.D_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/429.D_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/429.D_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/429.U_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/429.U_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/429.U_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/429.U_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/430.D_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/430.D_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/430.D_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/430.D_16_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/430.U_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/430.U_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/430.U_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/430.U_38_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/430.U_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/430.U_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/430.U_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/430.U_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.84 s (21 us/read; 2.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            132,650\n",
      "  Read 1 with adapter:                      37 (0.0%)\n",
      "  Read 2 with adapter:                   4,475 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       132,650 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,193,531 bp\n",
      "  Read 1:    29,611,573 bp\n",
      "  Read 2:    30,581,958 bp\n",
      "Total written (filtered):     60,138,123 bp (99.9%)\n",
      "  Read 1:    29,610,794 bp\n",
      "  Read 2:    30,527,329 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 37 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t2072.7\t0\t33\n",
      "27\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4475 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t2072.7\t0\t40\n",
      "4\t7\t518.2\t0\t7\n",
      "5\t32\t129.5\t0\t32\n",
      "6\t50\t32.4\t0\t50\n",
      "7\t190\t8.1\t0\t190\n",
      "8\t282\t2.0\t0\t282\n",
      "9\t1010\t0.5\t0\t1010\n",
      "10\t1163\t0.1\t1\t1160 3\n",
      "11\t59\t0.0\t1\t58 1\n",
      "12\t64\t0.0\t1\t64\n",
      "13\t46\t0.0\t1\t45 1\n",
      "14\t60\t0.0\t1\t60\n",
      "15\t93\t0.0\t1\t91 2\n",
      "16\t98\t0.0\t1\t94 4\n",
      "17\t240\t0.0\t1\t231 9\n",
      "18\t281\t0.0\t1\t269 9 3\n",
      "19\t667\t0.0\t1\t436 231\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t79\t0.0\t2\t0 79\n",
      "167\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/431.D_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/431.D_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/431.D_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/431.D_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.98 s (19 us/read; 3.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            155,648\n",
      "  Read 1 with adapter:                      61 (0.0%)\n",
      "  Read 2 with adapter:                   5,443 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       155,648 (100.0%)\n",
      "\n",
      "Total basepairs processed:    70,790,722 bp\n",
      "  Read 1:    34,806,182 bp\n",
      "  Read 2:    35,984,540 bp\n",
      "Total written (filtered):     70,721,047 bp (99.9%)\n",
      "  Read 1:    34,805,122 bp\n",
      "  Read 2:    35,915,925 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 61 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t2432.0\t0\t56\n",
      "27\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t0 2\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5443 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t2432.0\t0\t34\n",
      "4\t8\t608.0\t0\t8\n",
      "5\t14\t152.0\t0\t14\n",
      "6\t49\t38.0\t0\t49\n",
      "7\t218\t9.5\t0\t218\n",
      "8\t294\t2.4\t0\t294\n",
      "9\t1163\t0.6\t0\t1163\n",
      "10\t1520\t0.1\t1\t1514 6\n",
      "11\t57\t0.0\t1\t57\n",
      "12\t68\t0.0\t1\t67 1\n",
      "13\t70\t0.0\t1\t69 1\n",
      "14\t114\t0.0\t1\t111 3\n",
      "15\t131\t0.0\t1\t128 3\n",
      "16\t145\t0.0\t1\t141 4\n",
      "17\t331\t0.0\t1\t328 3\n",
      "18\t346\t0.0\t1\t325 17 4\n",
      "19\t758\t0.0\t1\t484 273 1\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t100\t0.0\t2\t0 100\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t0 1\n",
      "234\t1\t0.0\t2\t1\n",
      "238\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/431.U_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/431.U_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/431.U_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/431.U_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.01 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,454\n",
      "  Read 1 with adapter:                      28 (0.0%)\n",
      "  Read 2 with adapter:                   2,533 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        82,454 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,290,208 bp\n",
      "  Read 1:    18,337,446 bp\n",
      "  Read 2:    18,952,762 bp\n",
      "Total written (filtered):     37,255,999 bp (99.9%)\n",
      "  Read 1:    18,336,435 bp\n",
      "  Read 2:    18,919,564 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 28 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t1288.3\t0\t22\n",
      "12\t1\t0.0\t1\t1\n",
      "73\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2533 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1288.3\t0\t29\n",
      "4\t2\t322.1\t0\t2\n",
      "5\t8\t80.5\t0\t8\n",
      "6\t12\t20.1\t0\t12\n",
      "7\t71\t5.0\t0\t71\n",
      "8\t111\t1.3\t0\t111\n",
      "9\t481\t0.3\t0\t481\n",
      "10\t638\t0.1\t1\t634 4\n",
      "11\t37\t0.0\t1\t37\n",
      "12\t32\t0.0\t1\t32\n",
      "13\t44\t0.0\t1\t44\n",
      "14\t68\t0.0\t1\t67 1\n",
      "15\t64\t0.0\t1\t62 2\n",
      "16\t78\t0.0\t1\t75 3\n",
      "17\t197\t0.0\t1\t192 5\n",
      "18\t200\t0.0\t1\t190 4 6\n",
      "19\t388\t0.0\t1\t247 141\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t64\t0.0\t2\t0 64\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "219\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/432.D_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/432.D_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/432.D_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/432.D_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.94 s (19 us/read; 3.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            153,745\n",
      "  Read 1 with adapter:                      64 (0.0%)\n",
      "  Read 2 with adapter:                   5,994 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       153,745 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,729,630 bp\n",
      "  Read 1:    34,294,932 bp\n",
      "  Read 2:    35,434,698 bp\n",
      "Total written (filtered):     69,654,497 bp (99.9%)\n",
      "  Read 1:    34,291,358 bp\n",
      "  Read 2:    35,363,139 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 64 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t2402.3\t0\t45\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "101\t2\t0.0\t1\t2\n",
      "154\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t7\t0.0\t1\t7\n",
      "218\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5994 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t2402.3\t0\t61\n",
      "4\t9\t600.6\t0\t9\n",
      "5\t33\t150.1\t0\t33\n",
      "6\t71\t37.5\t0\t71\n",
      "7\t285\t9.4\t0\t285\n",
      "8\t395\t2.3\t0\t395\n",
      "9\t1504\t0.6\t0\t1502 2\n",
      "10\t1596\t0.1\t1\t1592 4\n",
      "11\t78\t0.0\t1\t77 1\n",
      "12\t62\t0.0\t1\t61 1\n",
      "13\t60\t0.0\t1\t59 1\n",
      "14\t96\t0.0\t1\t93 3\n",
      "15\t112\t0.0\t1\t111 1\n",
      "16\t130\t0.0\t1\t123 7\n",
      "17\t286\t0.0\t1\t281 5\n",
      "18\t366\t0.0\t1\t354 10 2\n",
      "19\t732\t0.0\t1\t463 267 2\n",
      "20\t16\t0.0\t2\t1 1 14\n",
      "21\t96\t0.0\t2\t0 95 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "227\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/432.U_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/432.U_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/432.U_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/432.U_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.77 s (21 us/read; 2.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            132,608\n",
      "  Read 1 with adapter:                      25 (0.0%)\n",
      "  Read 2 with adapter:                   3,612 (2.7%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       132,607 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,042,068 bp\n",
      "  Read 1:    29,546,371 bp\n",
      "  Read 2:    30,495,697 bp\n",
      "Total written (filtered):     59,991,427 bp (99.9%)\n",
      "  Read 1:    29,544,665 bp\n",
      "  Read 2:    30,446,762 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 25 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t2072.0\t0\t17\n",
      "25\t1\t0.0\t1\t1\n",
      "108\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "216\t2\t0.0\t1\t1 1\n",
      "217\t3\t0.0\t1\t1 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3612 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t2072.0\t0\t31\n",
      "4\t3\t518.0\t0\t3\n",
      "5\t14\t129.5\t0\t14\n",
      "6\t27\t32.4\t0\t27\n",
      "7\t106\t8.1\t0\t106\n",
      "8\t139\t2.0\t0\t139\n",
      "9\t657\t0.5\t0\t655 2\n",
      "10\t896\t0.1\t1\t894 2\n",
      "11\t67\t0.0\t1\t67\n",
      "12\t65\t0.0\t1\t64 1\n",
      "13\t62\t0.0\t1\t61 1\n",
      "14\t100\t0.0\t1\t96 4\n",
      "15\t96\t0.0\t1\t93 3\n",
      "16\t103\t0.0\t1\t100 3\n",
      "17\t268\t0.0\t1\t262 6\n",
      "18\t277\t0.0\t1\t256 15 6\n",
      "19\t597\t0.0\t1\t361 234 2\n",
      "20\t12\t0.0\t2\t0 1 11\n",
      "21\t79\t0.0\t2\t0 79\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "216\t6\t0.0\t2\t6\n",
      "217\t2\t0.0\t2\t2\n",
      "218\t1\t0.0\t2\t1\n",
      "231\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/433.H_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/433.H_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/433.H_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/433.H_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.36 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,493\n",
      "  Read 1 with adapter:                      34 (0.0%)\n",
      "  Read 2 with adapter:                   1,256 (1.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       104,493 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,434,965 bp\n",
      "  Read 1:    23,353,437 bp\n",
      "  Read 2:    24,081,528 bp\n",
      "Total written (filtered):     47,414,679 bp (100.0%)\n",
      "  Read 1:    23,352,697 bp\n",
      "  Read 2:    24,061,982 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 34 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t1632.7\t0\t31\n",
      "214\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1256 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1632.7\t0\t10\n",
      "4\t2\t408.2\t0\t2\n",
      "6\t8\t25.5\t0\t8\n",
      "7\t20\t6.4\t0\t20\n",
      "8\t25\t1.6\t0\t25\n",
      "9\t85\t0.4\t0\t85\n",
      "10\t178\t0.1\t1\t178\n",
      "11\t21\t0.0\t1\t21\n",
      "12\t33\t0.0\t1\t33\n",
      "13\t18\t0.0\t1\t18\n",
      "14\t22\t0.0\t1\t21 1\n",
      "15\t51\t0.0\t1\t50 1\n",
      "16\t45\t0.0\t1\t44 1\n",
      "17\t128\t0.0\t1\t124 4\n",
      "18\t176\t0.0\t1\t172 3 1\n",
      "19\t360\t0.0\t1\t246 114\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t62\t0.0\t2\t0 62\n",
      "215\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/434.H_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/434.H_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/434.H_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/434.H_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.70 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,096\n",
      "  Read 1 with adapter:                      10 (0.0%)\n",
      "  Read 2 with adapter:                   2,276 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        73,096 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,044,492 bp\n",
      "  Read 1:    16,260,616 bp\n",
      "  Read 2:    16,783,876 bp\n",
      "Total written (filtered):     33,015,947 bp (99.9%)\n",
      "  Read 1:    16,259,945 bp\n",
      "  Read 2:    16,756,002 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 10 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1142.1\t0\t7\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2276 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t1142.1\t0\t21\n",
      "5\t7\t71.4\t0\t7\n",
      "6\t13\t17.8\t0\t13\n",
      "7\t84\t4.5\t0\t84\n",
      "8\t138\t1.1\t0\t138\n",
      "9\t435\t0.3\t0\t435\n",
      "10\t702\t0.1\t1\t700 2\n",
      "11\t55\t0.0\t1\t55\n",
      "12\t38\t0.0\t1\t38\n",
      "13\t27\t0.0\t1\t27\n",
      "14\t45\t0.0\t1\t44 1\n",
      "15\t50\t0.0\t1\t48 2\n",
      "16\t61\t0.0\t1\t60 1\n",
      "17\t131\t0.0\t1\t128 3\n",
      "18\t135\t0.0\t1\t130 3 2\n",
      "19\t299\t0.0\t1\t200 99\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t28\t0.0\t2\t0 28\n",
      "61\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/435.H_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/435.H_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/435.H_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/435.H_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.54 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,031\n",
      "  Read 1 with adapter:                      13 (0.0%)\n",
      "  Read 2 with adapter:                   2,325 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        63,031 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,474,720 bp\n",
      "  Read 1:    14,011,355 bp\n",
      "  Read 2:    14,463,365 bp\n",
      "Total written (filtered):     28,445,128 bp (99.9%)\n",
      "  Read 1:    14,009,336 bp\n",
      "  Read 2:    14,435,792 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t984.9\t0\t3\n",
      "77\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t1\n",
      "214\t3\t0.0\t1\t3\n",
      "215\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "221\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2325 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t984.9\t0\t25\n",
      "4\t3\t246.2\t0\t3\n",
      "5\t12\t61.6\t0\t12\n",
      "6\t22\t15.4\t0\t22\n",
      "7\t90\t3.8\t0\t90\n",
      "8\t131\t1.0\t0\t131\n",
      "9\t548\t0.2\t0\t547 1\n",
      "10\t751\t0.1\t1\t749 2\n",
      "11\t46\t0.0\t1\t46\n",
      "12\t29\t0.0\t1\t28 1\n",
      "13\t14\t0.0\t1\t14\n",
      "14\t17\t0.0\t1\t17\n",
      "15\t36\t0.0\t1\t36\n",
      "16\t55\t0.0\t1\t55\n",
      "17\t133\t0.0\t1\t133\n",
      "18\t99\t0.0\t1\t95 3 1\n",
      "19\t272\t0.0\t1\t173 98 1\n",
      "20\t4\t0.0\t2\t0 1 3\n",
      "21\t35\t0.0\t2\t0 35\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/436.D_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/436.D_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/436.D_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/436.D_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.08 s (36 us/read; 1.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,503\n",
      "  Read 1 with adapter:                      15 (0.1%)\n",
      "  Read 2 with adapter:                     451 (1.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,503 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,398,712 bp\n",
      "  Read 1:     6,590,752 bp\n",
      "  Read 2:     6,807,960 bp\n",
      "Total written (filtered):     13,391,809 bp (99.9%)\n",
      "  Read 1:     6,590,707 bp\n",
      "  Read 2:     6,801,102 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 15 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t461.0\t0\t15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 451 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t461.0\t0\t2\n",
      "4\t1\t115.2\t0\t1\n",
      "6\t1\t7.2\t0\t1\n",
      "7\t8\t1.8\t0\t8\n",
      "8\t4\t0.5\t0\t4\n",
      "9\t18\t0.1\t0\t18\n",
      "10\t83\t0.0\t1\t83\n",
      "11\t19\t0.0\t1\t19\n",
      "12\t19\t0.0\t1\t18 1\n",
      "13\t6\t0.0\t1\t6\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t30\t0.0\t1\t29 1\n",
      "16\t23\t0.0\t1\t23\n",
      "17\t43\t0.0\t1\t43\n",
      "18\t76\t0.0\t1\t73 3\n",
      "19\t89\t0.0\t1\t63 26\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t22\t0.0\t2\t0 22\n",
      "69\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/436.U_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/436.U_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/436.U_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/436.U_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.87 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,396\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                     196 (1.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        14,396 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,503,063 bp\n",
      "  Read 1:     3,197,032 bp\n",
      "  Read 2:     3,306,031 bp\n",
      "Total written (filtered):      6,500,077 bp (100.0%)\n",
      "  Read 1:     3,197,023 bp\n",
      "  Read 2:     3,303,054 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t224.9\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 196 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "7\t7\t0.9\t0\t7\n",
      "8\t1\t0.2\t0\t1\n",
      "9\t9\t0.1\t0\t9\n",
      "10\t23\t0.0\t1\t23\n",
      "11\t7\t0.0\t1\t7\n",
      "12\t12\t0.0\t1\t12\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t11\t0.0\t1\t11\n",
      "15\t12\t0.0\t1\t10 2\n",
      "16\t12\t0.0\t1\t11 1\n",
      "17\t20\t0.0\t1\t19 1\n",
      "18\t28\t0.0\t1\t27 1\n",
      "19\t42\t0.0\t1\t29 13\n",
      "21\t8\t0.0\t2\t0 8\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/437.D_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/437.D_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/437.D_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/437.D_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.69 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,086\n",
      "  Read 1 with adapter:                      30 (0.0%)\n",
      "  Read 2 with adapter:                     933 (1.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        62,086 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,235,193 bp\n",
      "  Read 1:    13,887,378 bp\n",
      "  Read 2:    14,347,815 bp\n",
      "Total written (filtered):     28,220,667 bp (99.9%)\n",
      "  Read 1:    13,887,248 bp\n",
      "  Read 2:    14,333,419 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 30 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t970.1\t0\t28\n",
      "19\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 933 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t970.1\t0\t1\n",
      "4\t1\t242.5\t0\t1\n",
      "5\t2\t60.6\t0\t2\n",
      "6\t1\t15.2\t0\t1\n",
      "7\t8\t3.8\t0\t8\n",
      "8\t12\t0.9\t0\t12\n",
      "9\t47\t0.2\t0\t47\n",
      "10\t171\t0.1\t1\t171\n",
      "11\t29\t0.0\t1\t29\n",
      "12\t22\t0.0\t1\t21 1\n",
      "13\t18\t0.0\t1\t17 1\n",
      "14\t25\t0.0\t1\t24 1\n",
      "15\t40\t0.0\t1\t38 2\n",
      "16\t66\t0.0\t1\t65 1\n",
      "17\t106\t0.0\t1\t106\n",
      "18\t121\t0.0\t1\t115 5 1\n",
      "19\t213\t0.0\t1\t152 61\n",
      "20\t8\t0.0\t2\t0 0 8\n",
      "21\t41\t0.0\t2\t0 41\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/437.U_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/437.U_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/437.U_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/437.U_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.20 s (32 us/read; 1.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,020\n",
      "  Read 1 with adapter:                      21 (0.1%)\n",
      "  Read 2 with adapter:                     562 (1.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        38,020 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,291,923 bp\n",
      "  Read 1:     8,506,916 bp\n",
      "  Read 2:     8,785,007 bp\n",
      "Total written (filtered):     17,282,840 bp (99.9%)\n",
      "  Read 1:     8,506,480 bp\n",
      "  Read 2:     8,776,360 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t594.1\t0\t17\n",
      "25\t1\t0.0\t1\t1\n",
      "120\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 562 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t594.1\t0\t2\n",
      "4\t2\t148.5\t0\t2\n",
      "5\t1\t37.1\t0\t1\n",
      "6\t2\t9.3\t0\t2\n",
      "7\t3\t2.3\t0\t3\n",
      "8\t4\t0.6\t0\t4\n",
      "9\t35\t0.1\t0\t35\n",
      "10\t78\t0.0\t1\t78\n",
      "11\t23\t0.0\t1\t23\n",
      "12\t19\t0.0\t1\t19\n",
      "13\t10\t0.0\t1\t10\n",
      "14\t9\t0.0\t1\t9\n",
      "15\t31\t0.0\t1\t31\n",
      "16\t30\t0.0\t1\t28 2\n",
      "17\t68\t0.0\t1\t67 1\n",
      "18\t96\t0.0\t1\t94 2\n",
      "19\t113\t0.0\t1\t81 32\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t33\t0.0\t2\t0 33\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/438.D_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/438.D_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/438.D_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/438.D_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.44 s (23 us/read; 2.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,009\n",
      "  Read 1 with adapter:                      67 (0.1%)\n",
      "  Read 2 with adapter:                   1,794 (1.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,009 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,215,563 bp\n",
      "  Read 1:    23,716,070 bp\n",
      "  Read 2:    24,499,493 bp\n",
      "Total written (filtered):     48,189,322 bp (99.9%)\n",
      "  Read 1:    23,715,111 bp\n",
      "  Read 2:    24,474,211 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 67 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t1656.4\t0\t59\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "101\t1\t0.0\t1\t1\n",
      "144\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1794 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1656.4\t0\t9\n",
      "4\t2\t414.1\t0\t2\n",
      "5\t3\t103.5\t0\t3\n",
      "6\t4\t25.9\t0\t4\n",
      "7\t24\t6.5\t0\t24\n",
      "8\t50\t1.6\t0\t50\n",
      "9\t161\t0.4\t0\t161\n",
      "10\t494\t0.1\t1\t493 1\n",
      "11\t56\t0.0\t1\t56\n",
      "12\t44\t0.0\t1\t44\n",
      "13\t19\t0.0\t1\t19\n",
      "14\t25\t0.0\t1\t25\n",
      "15\t56\t0.0\t1\t56\n",
      "16\t71\t0.0\t1\t70 1\n",
      "17\t123\t0.0\t1\t122 1\n",
      "18\t214\t0.0\t1\t206 7 1\n",
      "19\t365\t0.0\t1\t246 119\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t61\t0.0\t2\t0 61\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/438.U_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/438.U_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/438.U_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/438.U_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.40 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             97,300\n",
      "  Read 1 with adapter:                      38 (0.0%)\n",
      "  Read 2 with adapter:                   1,308 (1.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        97,300 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,273,270 bp\n",
      "  Read 1:    21,781,500 bp\n",
      "  Read 2:    22,491,770 bp\n",
      "Total written (filtered):     44,252,786 bp (100.0%)\n",
      "  Read 1:    21,781,149 bp\n",
      "  Read 2:    22,471,637 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 38 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t1520.3\t0\t36\n",
      "27\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1308 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1520.3\t0\t5\n",
      "4\t1\t380.1\t0\t1\n",
      "5\t5\t95.0\t0\t5\n",
      "6\t5\t23.8\t0\t5\n",
      "7\t9\t5.9\t0\t9\n",
      "8\t18\t1.5\t0\t18\n",
      "9\t63\t0.4\t0\t63\n",
      "10\t225\t0.1\t1\t224 1\n",
      "11\t34\t0.0\t1\t34\n",
      "12\t33\t0.0\t1\t33\n",
      "13\t15\t0.0\t1\t15\n",
      "14\t19\t0.0\t1\t19\n",
      "15\t69\t0.0\t1\t69\n",
      "16\t86\t0.0\t1\t84 2\n",
      "17\t118\t0.0\t1\t115 3\n",
      "18\t199\t0.0\t1\t193 5 1\n",
      "19\t334\t0.0\t1\t229 104 1\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t59\t0.0\t2\t0 59\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/439.D_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/439.D_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/439.D_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/439.D_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.99 s (20 us/read; 3.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            149,703\n",
      "  Read 1 with adapter:                      65 (0.0%)\n",
      "  Read 2 with adapter:                   3,334 (2.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       149,703 (100.0%)\n",
      "\n",
      "Total basepairs processed:    68,123,529 bp\n",
      "  Read 1:    33,500,443 bp\n",
      "  Read 2:    34,623,086 bp\n",
      "Total written (filtered):     68,078,668 bp (99.9%)\n",
      "  Read 1:    33,499,692 bp\n",
      "  Read 2:    34,578,976 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 65 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t57\t2339.1\t0\t57\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t4\t0.0\t1\t4\n",
      "210\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3334 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t2339.1\t0\t20\n",
      "4\t3\t584.8\t0\t3\n",
      "5\t6\t146.2\t0\t6\n",
      "6\t17\t36.5\t0\t17\n",
      "7\t71\t9.1\t0\t71\n",
      "8\t103\t2.3\t0\t103\n",
      "9\t512\t0.6\t0\t509 3\n",
      "10\t1011\t0.1\t1\t1003 8\n",
      "11\t74\t0.0\t1\t73 1\n",
      "12\t64\t0.0\t1\t63 1\n",
      "13\t52\t0.0\t1\t51 1\n",
      "14\t61\t0.0\t1\t58 3\n",
      "15\t92\t0.0\t1\t89 3\n",
      "16\t113\t0.0\t1\t108 5\n",
      "17\t199\t0.0\t1\t195 4\n",
      "18\t298\t0.0\t1\t289 7 2\n",
      "19\t524\t0.0\t1\t359 162 3\n",
      "20\t19\t0.0\t2\t0 1 18\n",
      "21\t90\t0.0\t2\t0 90\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t1\n",
      "148\t2\t0.0\t2\t0 2\n",
      "218\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/439.U_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/439.U_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/439.U_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/439.U_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.38 s (28 us/read; 2.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,773\n",
      "  Read 1 with adapter:                      19 (0.0%)\n",
      "  Read 2 with adapter:                     641 (1.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,589,431 bp\n",
      "  Read 1:    11,111,858 bp\n",
      "  Read 2:    11,477,573 bp\n",
      "Total written (filtered):     22,579,210 bp (100.0%)\n",
      "  Read 1:    11,111,702 bp\n",
      "  Read 2:    11,467,508 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t777.7\t0\t18\n",
      "102\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 641 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t777.7\t0\t3\n",
      "5\t1\t48.6\t0\t1\n",
      "7\t8\t3.0\t0\t8\n",
      "8\t11\t0.8\t0\t11\n",
      "9\t36\t0.2\t0\t36\n",
      "10\t88\t0.0\t1\t88\n",
      "11\t24\t0.0\t1\t24\n",
      "12\t26\t0.0\t1\t26\n",
      "13\t18\t0.0\t1\t18\n",
      "14\t10\t0.0\t1\t10\n",
      "15\t25\t0.0\t1\t25\n",
      "16\t30\t0.0\t1\t30\n",
      "17\t54\t0.0\t1\t54\n",
      "18\t108\t0.0\t1\t107 1\n",
      "19\t166\t0.0\t1\t120 46\n",
      "20\t7\t0.0\t2\t0 0 7\n",
      "21\t25\t0.0\t2\t0 25\n",
      "217\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/431.D_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/431.D_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/431.D_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/431.D_18_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/431.U_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/431.U_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/431.U_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/431.U_27_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/432.D_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/432.D_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/432.D_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/432.D_4_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/432.U_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/432.U_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/432.U_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/432.U_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/433.H_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/433.H_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/433.H_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/433.H_2_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/434.H_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/434.H_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/434.H_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/434.H_35_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/435.H_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/435.H_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/435.H_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/435.H_20_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/436.D_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/436.D_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/436.D_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/436.D_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/436.U_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/436.U_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/436.U_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/436.U_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/437.D_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/437.D_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/437.D_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/437.D_25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/437.U_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/437.U_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/437.U_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/437.U_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/438.D_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/438.D_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/438.D_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/438.D_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/438.U_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/438.U_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/438.U_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/438.U_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/439.D_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/439.D_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/439.D_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/439.D_15_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/439.U_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/439.U_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/439.U_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/439.U_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/440.D_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/440.D_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/440.D_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/440.D_42_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/440.D_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/440.D_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/440.D_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/440.D_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.43 s (29 us/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,867\n",
      "  Read 1 with adapter:                      20 (0.0%)\n",
      "  Read 2 with adapter:                     400 (0.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,867 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,690,647 bp\n",
      "  Read 1:    11,165,630 bp\n",
      "  Read 2:    11,525,017 bp\n",
      "Total written (filtered):     22,683,430 bp (100.0%)\n",
      "  Read 1:    11,165,570 bp\n",
      "  Read 2:    11,517,860 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t779.2\t0\t20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 400 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t779.2\t0\t1\n",
      "5\t1\t48.7\t0\t1\n",
      "7\t2\t3.0\t0\t2\n",
      "8\t4\t0.8\t0\t4\n",
      "9\t12\t0.2\t0\t12\n",
      "10\t33\t0.0\t1\t33\n",
      "11\t4\t0.0\t1\t4\n",
      "12\t14\t0.0\t1\t14\n",
      "13\t7\t0.0\t1\t7\n",
      "14\t5\t0.0\t1\t5\n",
      "15\t11\t0.0\t1\t11\n",
      "16\t13\t0.0\t1\t13\n",
      "17\t38\t0.0\t1\t36 2\n",
      "18\t58\t0.0\t1\t56 2\n",
      "19\t156\t0.0\t1\t111 45\n",
      "20\t5\t0.0\t2\t0 1 4\n",
      "21\t34\t0.0\t2\t0 34\n",
      "224\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/440.U_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/440.U_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/440.U_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/440.U_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.22 s (31 us/read; 1.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,448\n",
      "  Read 1 with adapter:                      17 (0.0%)\n",
      "  Read 2 with adapter:                     550 (1.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        39,448 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,870,060 bp\n",
      "  Read 1:     8,788,982 bp\n",
      "  Read 2:     9,081,078 bp\n",
      "Total written (filtered):     17,861,630 bp (100.0%)\n",
      "  Read 1:     8,788,885 bp\n",
      "  Read 2:     9,072,745 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 17 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t616.4\t0\t15\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 550 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t616.4\t0\t3\n",
      "5\t2\t38.5\t0\t2\n",
      "6\t2\t9.6\t0\t2\n",
      "7\t8\t2.4\t0\t8\n",
      "8\t2\t0.6\t0\t2\n",
      "9\t36\t0.2\t0\t36\n",
      "10\t87\t0.0\t1\t87\n",
      "11\t18\t0.0\t1\t17 1\n",
      "12\t12\t0.0\t1\t12\n",
      "13\t12\t0.0\t1\t12\n",
      "14\t18\t0.0\t1\t17 1\n",
      "15\t35\t0.0\t1\t35\n",
      "16\t31\t0.0\t1\t29 2\n",
      "17\t57\t0.0\t1\t55 2\n",
      "18\t73\t0.0\t1\t72 1\n",
      "19\t124\t0.0\t1\t75 49\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t26\t0.0\t2\t0 26\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/441.H_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/441.H_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/441.H_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/441.H_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.78 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              9,231\n",
      "  Read 1 with adapter:                       7 (0.1%)\n",
      "  Read 2 with adapter:                     140 (1.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         9,231 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,120,110 bp\n",
      "  Read 1:     2,024,489 bp\n",
      "  Read 2:     2,095,621 bp\n",
      "Total written (filtered):      4,117,435 bp (99.9%)\n",
      "  Read 1:     2,023,805 bp\n",
      "  Read 2:     2,093,630 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t144.2\t0\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 140 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "7\t2\t0.6\t0\t2\n",
      "8\t7\t0.1\t0\t7\n",
      "9\t12\t0.0\t0\t12\n",
      "10\t28\t0.0\t1\t28\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t9\t0.0\t1\t9\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t9\t0.0\t1\t9\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t17\t0.0\t1\t17\n",
      "18\t17\t0.0\t1\t17\n",
      "19\t25\t0.0\t1\t18 7\n",
      "21\t4\t0.0\t2\t0 4\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/442.H_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/442.H_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/442.H_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/442.H_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.11 s (34 us/read; 1.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,167\n",
      "  Read 1 with adapter:                      25 (0.1%)\n",
      "  Read 2 with adapter:                     516 (1.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,167 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,493,164 bp\n",
      "  Read 1:     7,125,792 bp\n",
      "  Read 2:     7,367,372 bp\n",
      "Total written (filtered):     14,484,619 bp (99.9%)\n",
      "  Read 1:     7,125,272 bp\n",
      "  Read 2:     7,359,347 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 25 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t502.6\t0\t22\n",
      "27\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 516 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t502.6\t0\t2\n",
      "6\t3\t7.9\t0\t3\n",
      "7\t7\t2.0\t0\t7\n",
      "8\t6\t0.5\t0\t6\n",
      "9\t35\t0.1\t0\t35\n",
      "10\t88\t0.0\t1\t87 1\n",
      "11\t9\t0.0\t1\t9\n",
      "12\t12\t0.0\t1\t12\n",
      "13\t9\t0.0\t1\t9\n",
      "14\t22\t0.0\t1\t22\n",
      "15\t29\t0.0\t1\t29\n",
      "16\t26\t0.0\t1\t23 3\n",
      "17\t48\t0.0\t1\t48\n",
      "18\t65\t0.0\t1\t64 1\n",
      "19\t120\t0.0\t1\t83 37\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t28\t0.0\t2\t0 28\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/443.H_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/443.H_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/443.H_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/443.H_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.78 s (23 us/read; 2.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,236\n",
      "  Read 1 with adapter:                      29 (0.0%)\n",
      "  Read 2 with adapter:                     865 (1.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        78,236 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,607,135 bp\n",
      "  Read 1:    17,520,456 bp\n",
      "  Read 2:    18,086,679 bp\n",
      "Total written (filtered):     35,593,207 bp (100.0%)\n",
      "  Read 1:    17,520,369 bp\n",
      "  Read 2:    18,072,838 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1222.4\t0\t29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 865 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1222.4\t0\t3\n",
      "4\t4\t305.6\t0\t4\n",
      "5\t2\t76.4\t0\t2\n",
      "6\t1\t19.1\t0\t1\n",
      "7\t8\t4.8\t0\t8\n",
      "8\t12\t1.2\t0\t12\n",
      "9\t46\t0.3\t0\t46\n",
      "10\t97\t0.1\t1\t97\n",
      "11\t14\t0.0\t1\t14\n",
      "12\t18\t0.0\t1\t18\n",
      "13\t14\t0.0\t1\t13 1\n",
      "14\t21\t0.0\t1\t21\n",
      "15\t33\t0.0\t1\t33\n",
      "16\t43\t0.0\t1\t43\n",
      "17\t101\t0.0\t1\t96 5\n",
      "18\t120\t0.0\t1\t117 3\n",
      "19\t259\t0.0\t1\t182 77\n",
      "20\t12\t0.0\t2\t0 0 12\n",
      "21\t57\t0.0\t2\t0 57\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/444.D_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/444.D_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/444.D_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/444.D_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.75 s (21 us/read; 2.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            223,255\n",
      "  Read 1 with adapter:                     118 (0.1%)\n",
      "  Read 2 with adapter:                   9,260 (4.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       223,255 (100.0%)\n",
      "\n",
      "Total basepairs processed:   101,736,382 bp\n",
      "  Read 1:    49,988,379 bp\n",
      "  Read 2:    51,748,003 bp\n",
      "Total written (filtered):    101,620,492 bp (99.9%)\n",
      "  Read 1:    49,987,048 bp\n",
      "  Read 2:    51,633,444 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 118 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t105\t3488.4\t0\t105\n",
      "10\t1\t0.2\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "92\t4\t0.0\t1\t4\n",
      "95\t1\t0.0\t1\t1\n",
      "119\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "200\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9260 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t63\t3488.4\t0\t63\n",
      "4\t17\t872.1\t0\t17\n",
      "5\t29\t218.0\t0\t29\n",
      "6\t78\t54.5\t0\t78\n",
      "7\t358\t13.6\t0\t358\n",
      "8\t523\t3.4\t0\t523\n",
      "9\t2048\t0.9\t0\t2046 2\n",
      "10\t2559\t0.2\t1\t2547 12\n",
      "11\t140\t0.1\t1\t139 1\n",
      "12\t157\t0.0\t1\t156 1\n",
      "13\t143\t0.0\t1\t141 2\n",
      "14\t224\t0.0\t1\t215 9\n",
      "15\t230\t0.0\t1\t219 11\n",
      "16\t265\t0.0\t1\t257 8\n",
      "17\t523\t0.0\t1\t503 20\n",
      "18\t577\t0.0\t1\t539 23 15\n",
      "19\t1113\t0.0\t1\t667 444 2\n",
      "20\t33\t0.0\t2\t0 0 33\n",
      "21\t165\t0.0\t2\t0 161 4\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 1\n",
      "86\t1\t0.0\t2\t1\n",
      "144\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "160\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t1\n",
      "216\t6\t0.0\t2\t4 2\n",
      "217\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/444.U_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/444.U_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/444.U_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/444.U_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.92 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            124,995\n",
      "  Read 1 with adapter:                      67 (0.1%)\n",
      "  Read 2 with adapter:                   3,955 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       124,995 (100.0%)\n",
      "\n",
      "Total basepairs processed:    56,938,103 bp\n",
      "  Read 1:    27,988,182 bp\n",
      "  Read 2:    28,949,921 bp\n",
      "Total written (filtered):     56,886,613 bp (99.9%)\n",
      "  Read 1:    27,986,816 bp\n",
      "  Read 2:    28,899,797 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 67 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t55\t1953.0\t0\t55\n",
      "27\t3\t0.0\t1\t3\n",
      "38\t1\t0.0\t1\t1\n",
      "120\t4\t0.0\t1\t4\n",
      "127\t2\t0.0\t1\t2\n",
      "132\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3955 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t1953.0\t0\t27\n",
      "4\t5\t488.3\t0\t5\n",
      "5\t11\t122.1\t0\t11\n",
      "6\t22\t30.5\t0\t22\n",
      "7\t142\t7.6\t0\t142\n",
      "8\t251\t1.9\t0\t251\n",
      "9\t809\t0.5\t0\t808 1\n",
      "10\t988\t0.1\t1\t985 3\n",
      "11\t50\t0.0\t1\t49 1\n",
      "12\t68\t0.0\t1\t68\n",
      "13\t46\t0.0\t1\t46\n",
      "14\t89\t0.0\t1\t87 2\n",
      "15\t108\t0.0\t1\t107 1\n",
      "16\t142\t0.0\t1\t138 4\n",
      "17\t250\t0.0\t1\t245 5\n",
      "18\t297\t0.0\t1\t283 8 6\n",
      "19\t553\t0.0\t1\t345 208\n",
      "20\t15\t0.0\t2\t0 1 14\n",
      "21\t77\t0.0\t2\t0 77\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t2\t0.0\t2\t2\n",
      "153\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/445.D_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/445.D_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/445.D_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/445.D_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.38 s (21 us/read; 2.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            157,854\n",
      "  Read 1 with adapter:                     101 (0.1%)\n",
      "  Read 2 with adapter:                   7,939 (5.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       157,854 (100.0%)\n",
      "\n",
      "Total basepairs processed:    71,948,824 bp\n",
      "  Read 1:    35,339,203 bp\n",
      "  Read 2:    36,609,621 bp\n",
      "Total written (filtered):     71,854,630 bp (99.9%)\n",
      "  Read 1:    35,338,232 bp\n",
      "  Read 2:    36,516,398 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t96\t2466.5\t0\t96\n",
      "111\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "135\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "184\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7939 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t2466.5\t0\t54\n",
      "4\t10\t616.6\t0\t10\n",
      "5\t40\t154.2\t0\t40\n",
      "6\t97\t38.5\t0\t97\n",
      "7\t443\t9.6\t0\t443\n",
      "8\t549\t2.4\t0\t549\n",
      "9\t1957\t0.6\t0\t1957\n",
      "10\t2131\t0.2\t1\t2124 7\n",
      "11\t88\t0.0\t1\t88\n",
      "12\t102\t0.0\t1\t102\n",
      "13\t97\t0.0\t1\t96 1\n",
      "14\t176\t0.0\t1\t167 9\n",
      "15\t168\t0.0\t1\t164 4\n",
      "16\t238\t0.0\t1\t220 18\n",
      "17\t403\t0.0\t1\t391 12\n",
      "18\t434\t0.0\t1\t411 16 7\n",
      "19\t813\t0.0\t1\t486 325 2\n",
      "20\t25\t0.0\t2\t0 2 23\n",
      "21\t110\t0.0\t2\t0 109 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "202\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/445.U_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/445.U_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/445.U_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/445.U_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.70 s (21 us/read; 2.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            178,506\n",
      "  Read 1 with adapter:                      69 (0.0%)\n",
      "  Read 2 with adapter:                   6,655 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       178,506 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,267,231 bp\n",
      "  Read 1:    39,967,923 bp\n",
      "  Read 2:    41,299,308 bp\n",
      "Total written (filtered):     81,185,096 bp (99.9%)\n",
      "  Read 1:    39,966,193 bp\n",
      "  Read 2:    41,218,903 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 69 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t2789.2\t0\t53\n",
      "38\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "101\t13\t0.0\t1\t13\n",
      "120\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6655 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t41\t2789.2\t0\t41\n",
      "4\t10\t697.3\t0\t10\n",
      "5\t36\t174.3\t0\t36\n",
      "6\t81\t43.6\t0\t81\n",
      "7\t265\t10.9\t0\t265\n",
      "8\t460\t2.7\t0\t460\n",
      "9\t1511\t0.7\t0\t1511\n",
      "10\t1832\t0.2\t1\t1820 12\n",
      "11\t89\t0.0\t1\t89\n",
      "12\t72\t0.0\t1\t70 2\n",
      "13\t90\t0.0\t1\t90\n",
      "14\t134\t0.0\t1\t130 4\n",
      "15\t123\t0.0\t1\t122 1\n",
      "16\t195\t0.0\t1\t192 3\n",
      "17\t392\t0.0\t1\t385 7\n",
      "18\t395\t0.0\t1\t373 11 11\n",
      "19\t787\t0.0\t1\t512 275\n",
      "20\t22\t0.0\t2\t0 0 22\n",
      "21\t116\t0.0\t2\t0 116\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/446.D_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/446.D_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/446.D_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/446.D_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.46 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            200,946\n",
      "  Read 1 with adapter:                     101 (0.1%)\n",
      "  Read 2 with adapter:                   6,259 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       200,946 (100.0%)\n",
      "\n",
      "Total basepairs processed:    91,531,859 bp\n",
      "  Read 1:    44,995,119 bp\n",
      "  Read 2:    46,536,740 bp\n",
      "Total written (filtered):     91,450,478 bp (99.9%)\n",
      "  Read 1:    44,994,135 bp\n",
      "  Read 2:    46,456,343 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t93\t3139.8\t0\t93\n",
      "25\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t1\n",
      "49\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "219\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6259 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t60\t3139.8\t0\t60\n",
      "4\t10\t784.9\t0\t10\n",
      "5\t20\t196.2\t0\t20\n",
      "6\t47\t49.1\t0\t47\n",
      "7\t192\t12.3\t0\t192\n",
      "8\t278\t3.1\t0\t278\n",
      "9\t1217\t0.8\t0\t1215 2\n",
      "10\t1671\t0.2\t1\t1661 10\n",
      "11\t74\t0.0\t1\t74\n",
      "12\t132\t0.0\t1\t132\n",
      "13\t113\t0.0\t1\t113\n",
      "14\t171\t0.0\t1\t162 9\n",
      "15\t153\t0.0\t1\t148 5\n",
      "16\t196\t0.0\t1\t187 9\n",
      "17\t393\t0.0\t1\t380 13\n",
      "18\t469\t0.0\t1\t446 17 6\n",
      "19\t904\t0.0\t1\t564 339 1\n",
      "20\t33\t0.0\t2\t0 2 31\n",
      "21\t117\t0.0\t2\t0 117\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "155\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t4\t0.0\t2\t4\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/446.U_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/446.U_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/446.U_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/446.U_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.27 s (20 us/read; 2.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            208,918\n",
      "  Read 1 with adapter:                     101 (0.0%)\n",
      "  Read 2 with adapter:                   6,827 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       208,918 (100.0%)\n",
      "\n",
      "Total basepairs processed:    95,169,781 bp\n",
      "  Read 1:    46,779,558 bp\n",
      "  Read 2:    48,390,223 bp\n",
      "Total written (filtered):     95,079,593 bp (99.9%)\n",
      "  Read 1:    46,778,394 bp\n",
      "  Read 2:    48,301,199 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t92\t3264.3\t0\t92\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "92\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "154\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t0 1\n",
      "213\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6827 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t3264.3\t0\t59\n",
      "4\t3\t816.1\t0\t3\n",
      "5\t21\t204.0\t0\t21\n",
      "6\t48\t51.0\t0\t48\n",
      "7\t201\t12.8\t0\t201\n",
      "8\t320\t3.2\t0\t320\n",
      "9\t1204\t0.8\t0\t1202 2\n",
      "10\t1813\t0.2\t1\t1805 8\n",
      "11\t82\t0.0\t1\t82\n",
      "12\t122\t0.0\t1\t122\n",
      "13\t137\t0.0\t1\t137\n",
      "14\t166\t0.0\t1\t161 5\n",
      "15\t199\t0.0\t1\t197 2\n",
      "16\t187\t0.0\t1\t185 2\n",
      "17\t423\t0.0\t1\t410 13\n",
      "18\t583\t0.0\t1\t544 22 17\n",
      "19\t1095\t0.0\t1\t673 420 2\n",
      "20\t25\t0.0\t2\t0 3 22\n",
      "21\t129\t0.0\t2\t0 128 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "119\t1\t0.0\t2\t1\n",
      "165\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/447.D_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/447.D_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/447.D_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/447.D_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.52 s (25 us/read; 2.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,394\n",
      "  Read 1 with adapter:                     108 (0.1%)\n",
      "  Read 2 with adapter:                   5,230 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       142,394 (100.0%)\n",
      "\n",
      "Total basepairs processed:    64,876,080 bp\n",
      "  Read 1:    31,885,143 bp\n",
      "  Read 2:    32,990,937 bp\n",
      "Total written (filtered):     64,809,740 bp (99.9%)\n",
      "  Read 1:    31,884,251 bp\n",
      "  Read 2:    32,925,489 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t103\t2224.9\t0\t103\n",
      "27\t1\t0.0\t1\t1\n",
      "88\t2\t0.0\t1\t2\n",
      "184\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5230 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t2224.9\t0\t39\n",
      "4\t8\t556.2\t0\t8\n",
      "5\t16\t139.1\t0\t16\n",
      "6\t37\t34.8\t0\t37\n",
      "7\t196\t8.7\t0\t196\n",
      "8\t314\t2.2\t0\t314\n",
      "9\t1036\t0.5\t0\t1036\n",
      "10\t1350\t0.1\t1\t1347 3\n",
      "11\t52\t0.0\t1\t52\n",
      "12\t85\t0.0\t1\t84 1\n",
      "13\t104\t0.0\t1\t102 2\n",
      "14\t150\t0.0\t1\t145 5\n",
      "15\t148\t0.0\t1\t146 2\n",
      "16\t180\t0.0\t1\t172 8\n",
      "17\t382\t0.0\t1\t376 6\n",
      "18\t328\t0.0\t1\t305 16 7\n",
      "19\t693\t0.0\t1\t410 282 1\n",
      "20\t17\t0.0\t2\t0 0 17\n",
      "21\t91\t0.0\t2\t0 91\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t1\n",
      "168\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/447.U_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/447.U_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/447.U_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/447.U_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.38 s (19 us/read; 3.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            226,940\n",
      "  Read 1 with adapter:                     119 (0.1%)\n",
      "  Read 2 with adapter:                   7,575 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       226,940 (100.0%)\n",
      "\n",
      "Total basepairs processed:   103,390,071 bp\n",
      "  Read 1:    50,822,777 bp\n",
      "  Read 2:    52,567,294 bp\n",
      "Total written (filtered):    103,292,403 bp (99.9%)\n",
      "  Read 1:    50,821,821 bp\n",
      "  Read 2:    52,470,582 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 119 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t113\t3545.9\t0\t113\n",
      "27\t2\t0.0\t1\t1 1\n",
      "53\t1\t0.0\t1\t1\n",
      "103\t1\t0.0\t1\t1\n",
      "194\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7575 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t3545.9\t0\t61\n",
      "4\t3\t886.5\t0\t3\n",
      "5\t20\t221.6\t0\t20\n",
      "6\t62\t55.4\t0\t62\n",
      "7\t308\t13.9\t0\t308\n",
      "8\t391\t3.5\t0\t391\n",
      "9\t1490\t0.9\t0\t1489 1\n",
      "10\t1984\t0.2\t1\t1979 5\n",
      "11\t87\t0.1\t1\t87\n",
      "12\t122\t0.0\t1\t121 1\n",
      "13\t121\t0.0\t1\t120 1\n",
      "14\t170\t0.0\t1\t161 9\n",
      "15\t205\t0.0\t1\t201 4\n",
      "16\t237\t0.0\t1\t226 11\n",
      "17\t473\t0.0\t1\t466 7\n",
      "18\t570\t0.0\t1\t539 16 15\n",
      "19\t1095\t0.0\t1\t679 415 1\n",
      "20\t29\t0.0\t2\t0 4 25\n",
      "21\t137\t0.0\t2\t0 135 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 1\n",
      "126\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "190\t1\t0.0\t2\t1\n",
      "216\t4\t0.0\t2\t4\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/448.D_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/448.D_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/448.D_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/448.D_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.64 s (19 us/read; 3.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            187,624\n",
      "  Read 1 with adapter:                     147 (0.1%)\n",
      "  Read 2 with adapter:                   6,270 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       187,624 (100.0%)\n",
      "\n",
      "Total basepairs processed:    85,476,828 bp\n",
      "  Read 1:    42,016,084 bp\n",
      "  Read 2:    43,460,744 bp\n",
      "Total written (filtered):     85,394,440 bp (99.9%)\n",
      "  Read 1:    42,014,159 bp\n",
      "  Read 2:    43,380,281 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 147 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t126\t2931.6\t0\t126\n",
      "22\t9\t0.0\t1\t9\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "80\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "153\t1\t0.0\t1\t1\n",
      "171\t1\t0.0\t1\t1\n",
      "182\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6270 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t2931.6\t0\t33\n",
      "4\t7\t732.9\t0\t7\n",
      "5\t20\t183.2\t0\t20\n",
      "6\t37\t45.8\t0\t37\n",
      "7\t211\t11.5\t0\t211\n",
      "8\t325\t2.9\t0\t325\n",
      "9\t1157\t0.7\t0\t1157\n",
      "10\t1590\t0.2\t1\t1578 12\n",
      "11\t81\t0.0\t1\t80 1\n",
      "12\t145\t0.0\t1\t143 2\n",
      "13\t153\t0.0\t1\t150 3\n",
      "14\t199\t0.0\t1\t193 6\n",
      "15\t188\t0.0\t1\t185 3\n",
      "16\t226\t0.0\t1\t217 9\n",
      "17\t424\t0.0\t1\t412 12\n",
      "18\t451\t0.0\t1\t426 13 12\n",
      "19\t891\t0.0\t1\t556 332 3\n",
      "20\t28\t0.0\t2\t0 1 27\n",
      "21\t99\t0.0\t2\t0 99\n",
      "144\t2\t0.0\t2\t2\n",
      "167\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/448.U_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/448.U_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/448.U_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/448.U_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.69 s (19 us/read; 3.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            195,664\n",
      "  Read 1 with adapter:                     105 (0.1%)\n",
      "  Read 2 with adapter:                   6,556 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       195,664 (100.0%)\n",
      "\n",
      "Total basepairs processed:    89,128,757 bp\n",
      "  Read 1:    43,807,033 bp\n",
      "  Read 2:    45,321,724 bp\n",
      "Total written (filtered):     89,043,565 bp (99.9%)\n",
      "  Read 1:    43,806,326 bp\n",
      "  Read 2:    45,237,239 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 105 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t94\t3057.2\t0\t94\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t8\t0.0\t1\t7 1\n",
      "92\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6556 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t3057.2\t0\t43\n",
      "4\t9\t764.3\t0\t9\n",
      "5\t26\t191.1\t0\t26\n",
      "6\t49\t47.8\t0\t49\n",
      "7\t223\t11.9\t0\t223\n",
      "8\t340\t3.0\t0\t340\n",
      "9\t1244\t0.7\t0\t1243 1\n",
      "10\t1668\t0.2\t1\t1661 7\n",
      "11\t71\t0.0\t1\t71\n",
      "12\t116\t0.0\t1\t113 3\n",
      "13\t123\t0.0\t1\t123\n",
      "14\t194\t0.0\t1\t187 7\n",
      "15\t192\t0.0\t1\t187 5\n",
      "16\t200\t0.0\t1\t191 9\n",
      "17\t495\t0.0\t1\t481 14\n",
      "18\t486\t0.0\t1\t462 11 13\n",
      "19\t926\t0.0\t1\t583 341 2\n",
      "20\t19\t0.0\t2\t0 2 17\n",
      "21\t122\t0.0\t2\t0 122\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t1 2\n",
      "154\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "218\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/449.H_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/449.H_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/449.H_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/449.H_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.56 s (20 us/read; 3.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            177,712\n",
      "  Read 1 with adapter:                     110 (0.1%)\n",
      "  Read 2 with adapter:                   6,236 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       177,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    80,967,310 bp\n",
      "  Read 1:    39,794,592 bp\n",
      "  Read 2:    41,172,718 bp\n",
      "Total written (filtered):     80,887,751 bp (99.9%)\n",
      "  Read 1:    39,793,826 bp\n",
      "  Read 2:    41,093,925 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t102\t2776.8\t0\t102\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "53\t2\t0.0\t1\t2\n",
      "78\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "106\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6236 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t38\t2776.8\t0\t38\n",
      "4\t2\t694.2\t0\t2\n",
      "5\t16\t173.5\t0\t16\n",
      "6\t61\t43.4\t0\t61\n",
      "7\t261\t10.8\t0\t261\n",
      "8\t338\t2.7\t0\t338\n",
      "9\t1252\t0.7\t0\t1251 1\n",
      "10\t1594\t0.2\t1\t1589 5\n",
      "11\t83\t0.0\t1\t82 1\n",
      "12\t102\t0.0\t1\t102\n",
      "13\t126\t0.0\t1\t126\n",
      "14\t169\t0.0\t1\t159 10\n",
      "15\t148\t0.0\t1\t147 1\n",
      "16\t170\t0.0\t1\t163 7\n",
      "17\t393\t0.0\t1\t384 9\n",
      "18\t430\t0.0\t1\t397 17 16\n",
      "19\t912\t0.0\t1\t563 345 4\n",
      "20\t27\t0.0\t2\t0 0 27\n",
      "21\t108\t0.0\t2\t0 108\n",
      "61\t1\t0.0\t2\t1\n",
      "92\t1\t0.0\t2\t1\n",
      "132\t2\t0.0\t2\t1 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/440.U_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/440.U_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/440.U_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/440.U_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/441.H_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/441.H_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/441.H_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/441.H_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/442.H_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/442.H_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/442.H_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/442.H_32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/443.H_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/443.H_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/443.H_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/443.H_33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/444.D_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/444.D_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/444.D_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/444.D_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/444.U_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/444.U_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/444.U_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/444.U_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/445.D_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/445.D_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/445.D_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/445.D_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/445.U_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/445.U_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/445.U_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/445.U_14_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/446.D_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/446.D_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/446.D_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/446.D_31_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/446.U_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/446.U_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/446.U_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/446.U_12_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/447.D_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/447.D_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/447.D_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/447.D_9_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/447.U_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/447.U_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/447.U_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/447.U_7_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/448.D_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/448.D_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/448.D_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/448.D_24_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/448.U_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/448.U_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/448.U_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/448.U_6_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/449.H_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/449.H_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/449.H_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/449.H_8_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/450.H_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/450.H_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/450.H_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/450.H_29_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/450.H_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-blpahi_4/450.H_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/450.H_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-fj71qpmn/ce79e534-b909-4a0a-bd58-af9a7d3b4e5c/data/450.H_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.89 s (18 us/read; 3.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            211,051\n",
      "  Read 1 with adapter:                     112 (0.1%)\n",
      "  Read 2 with adapter:                   6,006 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       211,051 (100.0%)\n",
      "\n",
      "Total basepairs processed:    96,124,124 bp\n",
      "  Read 1:    47,260,620 bp\n",
      "  Read 2:    48,863,504 bp\n",
      "Total written (filtered):     96,043,130 bp (99.9%)\n",
      "  Read 1:    47,259,831 bp\n",
      "  Read 2:    48,783,299 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 112 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t104\t3297.7\t0\t104\n",
      "9\t1\t0.8\t0\t1\n",
      "27\t4\t0.0\t1\t4\n",
      "120\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6006 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t3297.7\t0\t35\n",
      "4\t7\t824.4\t0\t7\n",
      "5\t13\t206.1\t0\t13\n",
      "6\t30\t51.5\t0\t30\n",
      "7\t166\t12.9\t0\t166\n",
      "8\t230\t3.2\t0\t230\n",
      "9\t1008\t0.8\t0\t1006 2\n",
      "10\t1495\t0.2\t1\t1490 5\n",
      "11\t93\t0.1\t1\t93\n",
      "12\t110\t0.0\t1\t110\n",
      "13\t111\t0.0\t1\t109 2\n",
      "14\t169\t0.0\t1\t165 4\n",
      "15\t189\t0.0\t1\t184 5\n",
      "16\t220\t0.0\t1\t212 8\n",
      "17\t457\t0.0\t1\t448 9\n",
      "18\t504\t0.0\t1\t475 16 13\n",
      "19\t978\t0.0\t1\t645 330 3\n",
      "20\t36\t0.0\t2\t0 0 36\n",
      "21\t145\t0.0\t2\t0 143 2\n",
      "22\t5\t0.0\t2\t0 0 5\n",
      "147\t1\t0.0\t2\t1\n",
      "200\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "220\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGmicro2018_193-240.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/451.H_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/451.H_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/451.H_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/451.H_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.12 s (19 us/read; 3.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            274,306\n",
      "  Read 1 with adapter:                     153 (0.1%)\n",
      "  Read 2 with adapter:                  10,340 (3.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       274,306 (100.0%)\n",
      "\n",
      "Total basepairs processed:   124,990,820 bp\n",
      "  Read 1:    61,429,247 bp\n",
      "  Read 2:    63,561,573 bp\n",
      "Total written (filtered):    124,861,913 bp (99.9%)\n",
      "  Read 1:    61,428,321 bp\n",
      "  Read 2:    63,433,592 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 153 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t142\t4286.0\t0\t142\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t4\t0.0\t1\t4\n",
      "82\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10340 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t4286.0\t0\t48\n",
      "4\t10\t1071.5\t0\t10\n",
      "5\t27\t267.9\t0\t27\n",
      "6\t93\t67.0\t0\t93\n",
      "7\t454\t16.7\t0\t454\n",
      "8\t579\t4.2\t0\t579\n",
      "9\t2295\t1.0\t0\t2294 1\n",
      "10\t2632\t0.3\t1\t2619 13\n",
      "11\t113\t0.1\t1\t112 1\n",
      "12\t172\t0.0\t1\t171 1\n",
      "13\t176\t0.0\t1\t175 1\n",
      "14\t241\t0.0\t1\t237 4\n",
      "15\t311\t0.0\t1\t304 7\n",
      "16\t334\t0.0\t1\t319 15\n",
      "17\t596\t0.0\t1\t582 14\n",
      "18\t669\t0.0\t1\t629 24 16\n",
      "19\t1379\t0.0\t1\t855 522 2\n",
      "20\t36\t0.0\t2\t0 0 36\n",
      "21\t170\t0.0\t2\t0 170\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "190\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/452.D_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/452.D_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/452.D_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/452.D_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.80 s (14 us/read; 4.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            262,929\n",
      "  Read 1 with adapter:                     192 (0.1%)\n",
      "  Read 2 with adapter:                  11,617 (4.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       262,929 (100.0%)\n",
      "\n",
      "Total basepairs processed:   119,830,420 bp\n",
      "  Read 1:    58,872,383 bp\n",
      "  Read 2:    60,958,037 bp\n",
      "Total written (filtered):    119,692,897 bp (99.9%)\n",
      "  Read 1:    58,871,317 bp\n",
      "  Read 2:    60,821,580 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 192 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t189\t4108.3\t0\t189\n",
      "100\t1\t0.0\t1\t1\n",
      "191\t1\t0.0\t1\t1\n",
      "208\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11617 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t4108.3\t0\t79\n",
      "4\t14\t1027.1\t0\t14\n",
      "5\t35\t256.8\t0\t35\n",
      "6\t111\t64.2\t0\t111\n",
      "7\t540\t16.0\t0\t540\n",
      "8\t754\t4.0\t0\t754\n",
      "9\t2965\t1.0\t0\t2958 7\n",
      "10\t3463\t0.3\t1\t3455 8\n",
      "11\t115\t0.1\t1\t113 2\n",
      "12\t108\t0.0\t1\t108\n",
      "13\t97\t0.0\t1\t95 2\n",
      "14\t172\t0.0\t1\t167 5\n",
      "15\t275\t0.0\t1\t268 7\n",
      "16\t286\t0.0\t1\t276 10\n",
      "17\t551\t0.0\t1\t531 20\n",
      "18\t599\t0.0\t1\t561 31 7\n",
      "19\t1237\t0.0\t1\t795 439 3\n",
      "20\t27\t0.0\t2\t0 1 26\n",
      "21\t181\t0.0\t2\t0 179 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t0 1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "234\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/452.U_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/452.U_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/452.U_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/452.U_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.10 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            175,552\n",
      "  Read 1 with adapter:                     106 (0.1%)\n",
      "  Read 2 with adapter:                   3,506 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       175,552 (100.0%)\n",
      "\n",
      "Total basepairs processed:    79,765,973 bp\n",
      "  Read 1:    39,224,882 bp\n",
      "  Read 2:    40,541,091 bp\n",
      "Total written (filtered):     79,715,245 bp (99.9%)\n",
      "  Read 1:    39,224,265 bp\n",
      "  Read 2:    40,490,980 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 106 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t100\t2743.0\t0\t100\n",
      "9\t2\t0.7\t0\t2\n",
      "27\t3\t0.0\t1\t3\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3506 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t2743.0\t0\t12\n",
      "4\t2\t685.8\t0\t2\n",
      "5\t3\t171.4\t0\t3\n",
      "6\t16\t42.9\t0\t16\n",
      "7\t78\t10.7\t0\t78\n",
      "8\t99\t2.7\t0\t99\n",
      "9\t456\t0.7\t0\t455 1\n",
      "10\t727\t0.2\t1\t722 5\n",
      "11\t56\t0.0\t1\t56\n",
      "12\t66\t0.0\t1\t65 1\n",
      "13\t73\t0.0\t1\t73\n",
      "14\t96\t0.0\t1\t92 4\n",
      "15\t139\t0.0\t1\t136 3\n",
      "16\t146\t0.0\t1\t144 2\n",
      "17\t314\t0.0\t1\t305 9\n",
      "18\t338\t0.0\t1\t299 27 12\n",
      "19\t760\t0.0\t1\t464 294 2\n",
      "20\t20\t0.0\t2\t0 0 20\n",
      "21\t101\t0.0\t2\t0 101\n",
      "77\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "230\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/453.D_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/453.D_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/453.D_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/453.D_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.77 s (17 us/read; 3.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            228,086\n",
      "  Read 1 with adapter:                     182 (0.1%)\n",
      "  Read 2 with adapter:                   8,833 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       228,086 (100.0%)\n",
      "\n",
      "Total basepairs processed:   103,908,142 bp\n",
      "  Read 1:    51,065,978 bp\n",
      "  Read 2:    52,842,164 bp\n",
      "Total written (filtered):    103,800,184 bp (99.9%)\n",
      "  Read 1:    51,064,764 bp\n",
      "  Read 2:    52,735,420 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 182 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t154\t3563.8\t0\t154\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t24\t0.0\t1\t24\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8833 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t3563.8\t0\t49\n",
      "4\t15\t891.0\t0\t15\n",
      "5\t20\t222.7\t0\t20\n",
      "6\t79\t55.7\t0\t79\n",
      "7\t363\t13.9\t0\t363\n",
      "8\t523\t3.5\t0\t523\n",
      "9\t2035\t0.9\t0\t2034 1\n",
      "10\t2524\t0.2\t1\t2511 13\n",
      "11\t91\t0.1\t1\t91\n",
      "12\t107\t0.0\t1\t106 1\n",
      "13\t108\t0.0\t1\t108\n",
      "14\t181\t0.0\t1\t178 3\n",
      "15\t194\t0.0\t1\t190 4\n",
      "16\t295\t0.0\t1\t288 7\n",
      "17\t461\t0.0\t1\t445 16\n",
      "18\t535\t0.0\t1\t501 22 12\n",
      "19\t1070\t0.0\t1\t698 368 4\n",
      "20\t25\t0.0\t2\t0 0 25\n",
      "21\t153\t0.0\t2\t0 152 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t1\n",
      "144\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/453.U_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/453.U_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/453.U_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/453.U_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.96 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,794\n",
      "  Read 1 with adapter:                      32 (0.0%)\n",
      "  Read 2 with adapter:                   2,309 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        72,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,813,113 bp\n",
      "  Read 1:    16,114,977 bp\n",
      "  Read 2:    16,698,136 bp\n",
      "Total written (filtered):     32,781,785 bp (99.9%)\n",
      "  Read 1:    16,114,389 bp\n",
      "  Read 2:    16,667,396 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 32 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t1137.4\t0\t28\n",
      "25\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "136\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2309 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1137.4\t0\t8\n",
      "4\t5\t284.4\t0\t5\n",
      "5\t6\t71.1\t0\t6\n",
      "6\t26\t17.8\t0\t26\n",
      "7\t51\t4.4\t0\t51\n",
      "8\t96\t1.1\t0\t96\n",
      "9\t410\t0.3\t0\t410\n",
      "10\t520\t0.1\t1\t518 2\n",
      "11\t28\t0.0\t1\t28\n",
      "12\t40\t0.0\t1\t40\n",
      "13\t49\t0.0\t1\t49\n",
      "14\t76\t0.0\t1\t71 5\n",
      "15\t98\t0.0\t1\t97 1\n",
      "16\t93\t0.0\t1\t91 2\n",
      "17\t165\t0.0\t1\t153 12\n",
      "18\t193\t0.0\t1\t182 6 5\n",
      "19\t379\t0.0\t1\t246 133\n",
      "20\t15\t0.0\t2\t0 1 14\n",
      "21\t49\t0.0\t2\t0 49\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/454.D_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/454.D_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/454.D_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/454.D_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.30 s (13 us/read; 4.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            332,201\n",
      "  Read 1 with adapter:                     160 (0.0%)\n",
      "  Read 2 with adapter:                  13,208 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       332,201 (100.0%)\n",
      "\n",
      "Total basepairs processed:   151,384,806 bp\n",
      "  Read 1:    74,393,016 bp\n",
      "  Read 2:    76,991,790 bp\n",
      "Total written (filtered):    151,224,745 bp (99.9%)\n",
      "  Read 1:    74,392,315 bp\n",
      "  Read 2:    76,832,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 160 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t157\t5190.6\t0\t157\n",
      "23\t2\t0.0\t1\t2\n",
      "184\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13208 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t62\t5190.6\t0\t62\n",
      "4\t18\t1297.7\t0\t18\n",
      "5\t29\t324.4\t0\t29\n",
      "6\t107\t81.1\t0\t107\n",
      "7\t541\t20.3\t0\t541\n",
      "8\t739\t5.1\t0\t739\n",
      "9\t3085\t1.3\t0\t3081 4\n",
      "10\t4066\t0.3\t1\t4053 13\n",
      "11\t155\t0.1\t1\t154 1\n",
      "12\t160\t0.0\t1\t159 1\n",
      "13\t157\t0.0\t1\t157\n",
      "14\t230\t0.0\t1\t224 6\n",
      "15\t297\t0.0\t1\t286 11\n",
      "16\t315\t0.0\t1\t305 10\n",
      "17\t672\t0.0\t1\t651 21\n",
      "18\t757\t0.0\t1\t725 24 8\n",
      "19\t1562\t0.0\t1\t1043 516 3\n",
      "20\t43\t0.0\t2\t0 2 41\n",
      "21\t202\t0.0\t2\t0 202\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t1\n",
      "130\t1\t0.0\t2\t1\n",
      "212\t1\t0.0\t2\t0 1\n",
      "214\t3\t0.0\t2\t2 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "221\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/454.U_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/454.U_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/454.U_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/454.U_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.32 s (18 us/read; 3.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            301,290\n",
      "  Read 1 with adapter:                     176 (0.1%)\n",
      "  Read 2 with adapter:                   7,997 (2.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       301,290 (100.0%)\n",
      "\n",
      "Total basepairs processed:   136,971,268 bp\n",
      "  Read 1:    67,262,097 bp\n",
      "  Read 2:    69,709,171 bp\n",
      "Total written (filtered):    136,862,758 bp (99.9%)\n",
      "  Read 1:    67,260,267 bp\n",
      "  Read 2:    69,602,491 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 176 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t166\t4707.7\t0\t166\n",
      "27\t3\t0.0\t1\t3\n",
      "73\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t1 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7997 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t51\t4707.7\t0\t51\n",
      "4\t6\t1176.9\t0\t6\n",
      "5\t18\t294.2\t0\t18\n",
      "6\t49\t73.6\t0\t49\n",
      "7\t254\t18.4\t0\t254\n",
      "8\t361\t4.6\t0\t361\n",
      "9\t1344\t1.1\t0\t1341 3\n",
      "10\t2120\t0.3\t1\t2111 9\n",
      "11\t102\t0.1\t1\t101 1\n",
      "12\t115\t0.0\t1\t114 1\n",
      "13\t96\t0.0\t1\t95 1\n",
      "14\t142\t0.0\t1\t142\n",
      "15\t260\t0.0\t1\t254 6\n",
      "16\t274\t0.0\t1\t262 12\n",
      "17\t524\t0.0\t1\t512 12\n",
      "18\t697\t0.0\t1\t669 24 4\n",
      "19\t1333\t0.0\t1\t885 446 2\n",
      "20\t29\t0.0\t2\t0 2 27\n",
      "21\t206\t0.0\t2\t0 205 1\n",
      "22\t4\t0.0\t2\t0 1 3\n",
      "61\t1\t0.0\t2\t1\n",
      "171\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t4\t0.0\t2\t4\n",
      "217\t3\t0.0\t2\t3\n",
      "224\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/455.D_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/455.D_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/455.D_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/455.D_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.93 s (14 us/read; 4.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            271,136\n",
      "  Read 1 with adapter:                     203 (0.1%)\n",
      "  Read 2 with adapter:                  12,640 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       271,136 (100.0%)\n",
      "\n",
      "Total basepairs processed:   123,546,163 bp\n",
      "  Read 1:    60,683,910 bp\n",
      "  Read 2:    62,862,253 bp\n",
      "Total written (filtered):    123,394,182 bp (99.9%)\n",
      "  Read 1:    60,680,376 bp\n",
      "  Read 2:    62,713,806 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 203 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t166\t4236.5\t0\t166\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t9\t0.0\t1\t9\n",
      "69\t1\t0.0\t1\t1\n",
      "100\t13\t0.0\t1\t11 2\n",
      "101\t3\t0.0\t1\t3\n",
      "102\t4\t0.0\t1\t4\n",
      "104\t2\t0.0\t1\t2\n",
      "120\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12640 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t89\t4236.5\t0\t89\n",
      "4\t15\t1059.1\t0\t15\n",
      "5\t29\t264.8\t0\t29\n",
      "6\t120\t66.2\t0\t120\n",
      "7\t591\t16.5\t0\t591\n",
      "8\t807\t4.1\t0\t807\n",
      "9\t3087\t1.0\t0\t3084 3\n",
      "10\t3646\t0.3\t1\t3625 21\n",
      "11\t139\t0.1\t1\t137 2\n",
      "12\t183\t0.0\t1\t180 3\n",
      "13\t214\t0.0\t1\t211 3\n",
      "14\t250\t0.0\t1\t243 7\n",
      "15\t366\t0.0\t1\t354 12\n",
      "16\t358\t0.0\t1\t347 11\n",
      "17\t576\t0.0\t1\t554 22\n",
      "18\t692\t0.0\t1\t652 28 12\n",
      "19\t1265\t0.0\t1\t824 440 1\n",
      "20\t30\t0.0\t2\t0 3 27\n",
      "21\t175\t0.0\t2\t0 173 2\n",
      "22\t5\t0.0\t2\t0 1 4\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/455.U_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/455.U_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/455.U_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/455.U_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.09 s (38 us/read; 1.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,277\n",
      "  Read 1 with adapter:                      14 (0.0%)\n",
      "  Read 2 with adapter:                   1,025 (3.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,277 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,567,910 bp\n",
      "  Read 1:     6,162,502 bp\n",
      "  Read 2:     6,405,408 bp\n",
      "Total written (filtered):     12,554,246 bp (99.9%)\n",
      "  Read 1:     6,162,227 bp\n",
      "  Read 2:     6,392,019 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t441.8\t0\t12\n",
      "27\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1025 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t441.8\t0\t4\n",
      "4\t1\t110.5\t0\t1\n",
      "5\t2\t27.6\t0\t2\n",
      "6\t8\t6.9\t0\t8\n",
      "7\t35\t1.7\t0\t35\n",
      "8\t36\t0.4\t0\t36\n",
      "9\t198\t0.1\t0\t198\n",
      "10\t235\t0.0\t1\t235\n",
      "11\t10\t0.0\t1\t10\n",
      "12\t22\t0.0\t1\t21 1\n",
      "13\t22\t0.0\t1\t22\n",
      "14\t35\t0.0\t1\t35\n",
      "15\t43\t0.0\t1\t42 1\n",
      "16\t45\t0.0\t1\t44 1\n",
      "17\t86\t0.0\t1\t84 2\n",
      "18\t73\t0.0\t1\t69 3 1\n",
      "19\t151\t0.0\t1\t92 58 1\n",
      "20\t3\t0.0\t2\t0 1 2\n",
      "21\t15\t0.0\t2\t0 15\n",
      "115\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/456.D_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/456.D_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/456.D_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/456.D_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.74 s (20 us/read; 3.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            188,392\n",
      "  Read 1 with adapter:                     112 (0.1%)\n",
      "  Read 2 with adapter:                   7,922 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       188,392 (100.0%)\n",
      "\n",
      "Total basepairs processed:    85,723,893 bp\n",
      "  Read 1:    42,115,779 bp\n",
      "  Read 2:    43,608,114 bp\n",
      "Total written (filtered):     85,628,393 bp (99.9%)\n",
      "  Read 1:    42,115,353 bp\n",
      "  Read 2:    43,513,040 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 112 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t108\t2943.6\t0\t108\n",
      "21\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7922 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t2943.6\t0\t37\n",
      "4\t13\t735.9\t0\t13\n",
      "5\t25\t184.0\t0\t25\n",
      "6\t81\t46.0\t0\t81\n",
      "7\t346\t11.5\t0\t346\n",
      "8\t528\t2.9\t0\t528\n",
      "9\t1843\t0.7\t0\t1842 1\n",
      "10\t2260\t0.2\t1\t2254 6\n",
      "11\t76\t0.0\t1\t75 1\n",
      "12\t76\t0.0\t1\t76\n",
      "13\t105\t0.0\t1\t105\n",
      "14\t134\t0.0\t1\t131 3\n",
      "15\t200\t0.0\t1\t192 8\n",
      "16\t223\t0.0\t1\t214 9\n",
      "17\t416\t0.0\t1\t405 11\n",
      "18\t491\t0.0\t1\t469 17 5\n",
      "19\t901\t0.0\t1\t573 328\n",
      "20\t21\t0.0\t2\t0 1 20\n",
      "21\t141\t0.0\t2\t0 139 2\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "217\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/456.U_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/456.U_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/456.U_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/456.U_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.64 s (31 us/read; 1.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,928\n",
      "  Read 1 with adapter:                      31 (0.1%)\n",
      "  Read 2 with adapter:                   2,120 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,928 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,818,743 bp\n",
      "  Read 1:    11,690,642 bp\n",
      "  Read 2:    12,128,101 bp\n",
      "Total written (filtered):     23,789,744 bp (99.9%)\n",
      "  Read 1:    11,689,285 bp\n",
      "  Read 2:    12,100,459 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 31 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t827.0\t0\t25\n",
      "201\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t4\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2120 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t827.0\t0\t8\n",
      "4\t2\t206.8\t0\t2\n",
      "5\t3\t51.7\t0\t3\n",
      "6\t14\t12.9\t0\t14\n",
      "7\t61\t3.2\t0\t61\n",
      "8\t114\t0.8\t0\t114\n",
      "9\t371\t0.2\t0\t371\n",
      "10\t476\t0.1\t1\t476\n",
      "11\t47\t0.0\t1\t46 1\n",
      "12\t31\t0.0\t1\t30 1\n",
      "13\t53\t0.0\t1\t53\n",
      "14\t88\t0.0\t1\t84 4\n",
      "15\t82\t0.0\t1\t80 2\n",
      "16\t94\t0.0\t1\t89 5\n",
      "17\t144\t0.0\t1\t141 3\n",
      "18\t184\t0.0\t1\t172 6 6\n",
      "19\t314\t0.0\t1\t175 138 1\n",
      "20\t5\t0.0\t2\t0 1 4\n",
      "21\t28\t0.0\t2\t0 28\n",
      "115\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/457.H_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/457.H_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/457.H_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/457.H_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.62 s (20 us/read; 3.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            131,029\n",
      "  Read 1 with adapter:                      98 (0.1%)\n",
      "  Read 2 with adapter:                   4,096 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       131,029 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,372,233 bp\n",
      "  Read 1:    29,177,118 bp\n",
      "  Read 2:    30,195,115 bp\n",
      "Total written (filtered):     59,315,786 bp (99.9%)\n",
      "  Read 1:    29,174,976 bp\n",
      "  Read 2:    30,140,810 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 98 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t86\t2047.3\t0\t86\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "105\t1\t0.0\t1\t0 1\n",
      "192\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t0 1\n",
      "216\t5\t0.0\t1\t5\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4096 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t2047.3\t0\t26\n",
      "4\t5\t511.8\t0\t5\n",
      "5\t12\t128.0\t0\t12\n",
      "6\t22\t32.0\t0\t22\n",
      "7\t133\t8.0\t0\t133\n",
      "8\t176\t2.0\t0\t176\n",
      "9\t741\t0.5\t0\t741\n",
      "10\t971\t0.1\t1\t966 5\n",
      "11\t50\t0.0\t1\t49 1\n",
      "12\t51\t0.0\t1\t51\n",
      "13\t49\t0.0\t1\t49\n",
      "14\t92\t0.0\t1\t89 3\n",
      "15\t145\t0.0\t1\t141 4\n",
      "16\t167\t0.0\t1\t161 6\n",
      "17\t282\t0.0\t1\t271 11\n",
      "18\t367\t0.0\t1\t347 13 7\n",
      "19\t703\t0.0\t1\t446 255 2\n",
      "20\t11\t0.0\t2\t0 1 10\n",
      "21\t88\t0.0\t2\t0 88\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/458.H_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/458.H_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/458.H_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/458.H_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.65 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            119,263\n",
      "  Read 1 with adapter:                      65 (0.1%)\n",
      "  Read 2 with adapter:                   5,067 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       119,263 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,052,912 bp\n",
      "  Read 1:    26,545,717 bp\n",
      "  Read 2:    27,507,195 bp\n",
      "Total written (filtered):     53,990,970 bp (99.9%)\n",
      "  Read 1:    26,545,085 bp\n",
      "  Read 2:    27,445,885 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 65 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t1863.5\t0\t59\n",
      "21\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "126\t1\t0.0\t1\t1\n",
      "127\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5067 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1863.5\t0\t29\n",
      "4\t8\t465.9\t0\t8\n",
      "5\t14\t116.5\t0\t14\n",
      "6\t41\t29.1\t0\t41\n",
      "7\t252\t7.3\t0\t252\n",
      "8\t302\t1.8\t0\t302\n",
      "9\t1181\t0.5\t0\t1179 2\n",
      "10\t1349\t0.1\t1\t1345 4\n",
      "11\t35\t0.0\t1\t35\n",
      "12\t67\t0.0\t1\t66 1\n",
      "13\t53\t0.0\t1\t53\n",
      "14\t110\t0.0\t1\t106 4\n",
      "15\t131\t0.0\t1\t128 3\n",
      "16\t165\t0.0\t1\t161 4\n",
      "17\t304\t0.0\t1\t295 9\n",
      "18\t302\t0.0\t1\t277 18 7\n",
      "19\t620\t0.0\t1\t372 246 2\n",
      "20\t17\t0.0\t2\t0 2 15\n",
      "21\t85\t0.0\t2\t0 82 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/459.H_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/459.H_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/459.H_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/459.H_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.98 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,116\n",
      "  Read 1 with adapter:                      36 (0.0%)\n",
      "  Read 2 with adapter:                   2,506 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        81,116 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,522,515 bp\n",
      "  Read 1:    17,940,550 bp\n",
      "  Read 2:    18,581,965 bp\n",
      "Total written (filtered):     36,487,772 bp (99.9%)\n",
      "  Read 1:    17,939,840 bp\n",
      "  Read 2:    18,547,932 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 36 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t1267.4\t0\t32\n",
      "27\t1\t0.0\t1\t1\n",
      "154\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2506 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1267.4\t0\t5\n",
      "4\t1\t316.9\t0\t1\n",
      "5\t4\t79.2\t0\t4\n",
      "6\t14\t19.8\t0\t14\n",
      "7\t54\t5.0\t0\t54\n",
      "8\t93\t1.2\t0\t93\n",
      "9\t418\t0.3\t0\t417 1\n",
      "10\t528\t0.1\t1\t527 1\n",
      "11\t43\t0.0\t1\t43\n",
      "12\t45\t0.0\t1\t45\n",
      "13\t52\t0.0\t1\t52\n",
      "14\t92\t0.0\t1\t89 3\n",
      "15\t104\t0.0\t1\t101 3\n",
      "16\t100\t0.0\t1\t95 5\n",
      "17\t187\t0.0\t1\t182 5\n",
      "18\t245\t0.0\t1\t228 13 4\n",
      "19\t455\t0.0\t1\t280 173 2\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t55\t0.0\t2\t0 55\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/460.D_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/460.D_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/460.D_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/460.D_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.13 s (14 us/read; 4.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            231,009\n",
      "  Read 1 with adapter:                     111 (0.0%)\n",
      "  Read 2 with adapter:                   3,095 (1.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       231,009 (100.0%)\n",
      "\n",
      "Total basepairs processed:   105,142,398 bp\n",
      "  Read 1:    51,724,258 bp\n",
      "  Read 2:    53,418,140 bp\n",
      "Total written (filtered):    105,094,731 bp (100.0%)\n",
      "  Read 1:    51,723,439 bp\n",
      "  Read 2:    53,371,292 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 111 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t106\t3609.5\t0\t106\n",
      "27\t1\t0.0\t1\t1\n",
      "92\t3\t0.0\t1\t3\n",
      "198\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3095 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t3609.5\t0\t7\n",
      "4\t5\t902.4\t0\t5\n",
      "5\t2\t225.6\t0\t2\n",
      "6\t7\t56.4\t0\t7\n",
      "7\t59\t14.1\t0\t59\n",
      "8\t60\t3.5\t0\t60\n",
      "9\t233\t0.9\t0\t233\n",
      "10\t623\t0.2\t1\t621 2\n",
      "11\t41\t0.1\t1\t41\n",
      "12\t50\t0.0\t1\t50\n",
      "13\t54\t0.0\t1\t54\n",
      "14\t69\t0.0\t1\t69\n",
      "15\t122\t0.0\t1\t119 3\n",
      "16\t114\t0.0\t1\t113 1\n",
      "17\t235\t0.0\t1\t231 4\n",
      "18\t373\t0.0\t1\t354 10 9\n",
      "19\t863\t0.0\t1\t585 277 1\n",
      "20\t31\t0.0\t2\t0 0 31\n",
      "21\t145\t0.0\t2\t0 144 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/451.H_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/451.H_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/451.H_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/451.H_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/452.D_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/452.D_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/452.D_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/452.D_4_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/452.U_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/452.U_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/452.U_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/452.U_8_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/453.D_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/453.D_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/453.D_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/453.D_20_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/453.U_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/453.U_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/453.U_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/453.U_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/454.D_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/454.D_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/454.D_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/454.D_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/454.U_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/454.U_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/454.U_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/454.U_14_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/455.D_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/455.D_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/455.D_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/455.D_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/455.U_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/455.U_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/455.U_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/455.U_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/456.D_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/456.D_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/456.D_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/456.D_6_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/456.U_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/456.U_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/456.U_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/456.U_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/457.H_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/457.H_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/457.H_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/457.H_31_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/458.H_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/458.H_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/458.H_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/458.H_12_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/459.H_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/459.H_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/459.H_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/459.H_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/460.D_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/460.D_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/460.D_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/460.D_33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/460.U_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/460.U_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/460.U_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/460.U_7_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/460.U_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/460.U_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/460.U_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/460.U_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.64 s (28 us/read; 2.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,643\n",
      "  Read 1 with adapter:                      42 (0.1%)\n",
      "  Read 2 with adapter:                   1,857 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        57,643 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,873,813 bp\n",
      "  Read 1:    12,704,660 bp\n",
      "  Read 2:    13,169,153 bp\n",
      "Total written (filtered):     25,849,761 bp (99.9%)\n",
      "  Read 1:    12,704,486 bp\n",
      "  Read 2:    13,145,275 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 42 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t900.7\t0\t40\n",
      "27\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1857 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t900.7\t0\t6\n",
      "4\t1\t225.2\t0\t1\n",
      "5\t8\t56.3\t0\t8\n",
      "6\t14\t14.1\t0\t14\n",
      "7\t73\t3.5\t0\t73\n",
      "8\t101\t0.9\t0\t101\n",
      "9\t395\t0.2\t0\t394 1\n",
      "10\t408\t0.1\t1\t408\n",
      "11\t19\t0.0\t1\t19\n",
      "12\t24\t0.0\t1\t24\n",
      "13\t24\t0.0\t1\t23 1\n",
      "14\t39\t0.0\t1\t38 1\n",
      "15\t67\t0.0\t1\t66 1\n",
      "16\t61\t0.0\t1\t58 3\n",
      "17\t123\t0.0\t1\t120 3\n",
      "18\t158\t0.0\t1\t142 10 6\n",
      "19\t292\t0.0\t1\t188 104\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t38\t0.0\t2\t0 38\n",
      "61\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/461.D_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/461.D_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/461.D_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/461.D_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.96 s (18 us/read; 3.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            164,454\n",
      "  Read 1 with adapter:                     105 (0.1%)\n",
      "  Read 2 with adapter:                   6,067 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       164,454 (100.0%)\n",
      "\n",
      "Total basepairs processed:    74,843,480 bp\n",
      "  Read 1:    36,779,214 bp\n",
      "  Read 2:    38,064,266 bp\n",
      "Total written (filtered):     74,770,208 bp (99.9%)\n",
      "  Read 1:    36,778,153 bp\n",
      "  Read 2:    37,992,055 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 105 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t93\t2569.6\t0\t93\n",
      "27\t5\t0.0\t1\t4 1\n",
      "37\t1\t0.0\t1\t1\n",
      "101\t2\t0.0\t1\t1 1\n",
      "102\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6067 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t2569.6\t0\t34\n",
      "4\t5\t642.4\t0\t5\n",
      "5\t17\t160.6\t0\t17\n",
      "6\t47\t40.1\t0\t47\n",
      "7\t237\t10.0\t0\t237\n",
      "8\t324\t2.5\t0\t324\n",
      "9\t1518\t0.6\t0\t1517 1\n",
      "10\t1842\t0.2\t1\t1837 5\n",
      "11\t80\t0.0\t1\t78 2\n",
      "12\t52\t0.0\t1\t51 1\n",
      "13\t51\t0.0\t1\t51\n",
      "14\t81\t0.0\t1\t78 3\n",
      "15\t132\t0.0\t1\t131 1\n",
      "16\t182\t0.0\t1\t178 4\n",
      "17\t307\t0.0\t1\t296 11\n",
      "18\t367\t0.0\t1\t355 8 4\n",
      "19\t663\t0.0\t1\t454 208 1\n",
      "20\t19\t0.0\t2\t0 1 18\n",
      "21\t108\t0.0\t2\t0 108\n",
      "126\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/461.U_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/461.U_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/461.U_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/461.U_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.45 s (27 us/read; 2.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,486\n",
      "  Read 1 with adapter:                      36 (0.1%)\n",
      "  Read 2 with adapter:                   1,785 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,977,233 bp\n",
      "  Read 1:    11,765,406 bp\n",
      "  Read 2:    12,211,827 bp\n",
      "Total written (filtered):     23,953,395 bp (99.9%)\n",
      "  Read 1:    11,764,755 bp\n",
      "  Read 2:    12,188,640 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 36 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t835.7\t0\t31\n",
      "21\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1785 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t835.7\t0\t8\n",
      "4\t3\t208.9\t0\t3\n",
      "5\t3\t52.2\t0\t3\n",
      "6\t10\t13.1\t0\t10\n",
      "7\t51\t3.3\t0\t51\n",
      "8\t85\t0.8\t0\t85\n",
      "9\t320\t0.2\t0\t320\n",
      "10\t443\t0.1\t1\t440 3\n",
      "11\t32\t0.0\t1\t32\n",
      "12\t27\t0.0\t1\t27\n",
      "13\t31\t0.0\t1\t31\n",
      "14\t52\t0.0\t1\t50 2\n",
      "15\t62\t0.0\t1\t59 3\n",
      "16\t76\t0.0\t1\t76\n",
      "17\t120\t0.0\t1\t120\n",
      "18\t126\t0.0\t1\t119 5 2\n",
      "19\t296\t0.0\t1\t174 122\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t34\t0.0\t2\t0 33 1\n",
      "30\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/462.D_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/462.D_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/462.D_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/462.D_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.55 s (21 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,159\n",
      "  Read 1 with adapter:                      46 (0.1%)\n",
      "  Read 2 with adapter:                     645 (0.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        72,159 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,838,098 bp\n",
      "  Read 1:    16,159,732 bp\n",
      "  Read 2:    16,678,366 bp\n",
      "Total written (filtered):     32,826,954 bp (100.0%)\n",
      "  Read 1:    16,159,594 bp\n",
      "  Read 2:    16,667,360 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 46 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t1127.5\t0\t46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 645 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1127.5\t0\t1\n",
      "4\t1\t281.9\t0\t1\n",
      "5\t1\t70.5\t0\t1\n",
      "6\t1\t17.6\t0\t1\n",
      "7\t6\t4.4\t0\t6\n",
      "8\t7\t1.1\t0\t7\n",
      "9\t17\t0.3\t0\t17\n",
      "10\t64\t0.1\t1\t64\n",
      "11\t10\t0.0\t1\t10\n",
      "12\t8\t0.0\t1\t8\n",
      "13\t8\t0.0\t1\t8\n",
      "14\t13\t0.0\t1\t13\n",
      "15\t37\t0.0\t1\t37\n",
      "16\t36\t0.0\t1\t34 2\n",
      "17\t66\t0.0\t1\t63 3\n",
      "18\t108\t0.0\t1\t105 1 2\n",
      "19\t208\t0.0\t1\t147 60 1\n",
      "20\t4\t0.0\t2\t0 1 3\n",
      "21\t47\t0.0\t2\t0 47\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/462.U_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/462.U_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/462.U_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/462.U_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.10 s (30 us/read; 1.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,237\n",
      "  Read 1 with adapter:                      25 (0.1%)\n",
      "  Read 2 with adapter:                     506 (1.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        36,237 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,336,299 bp\n",
      "  Read 1:     8,028,202 bp\n",
      "  Read 2:     8,308,097 bp\n",
      "Total written (filtered):     16,327,294 bp (99.9%)\n",
      "  Read 1:     8,026,857 bp\n",
      "  Read 2:     8,300,437 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 25 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t566.2\t0\t17\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "192\t2\t0.0\t1\t2\n",
      "209\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t0 2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 506 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t566.2\t0\t1\n",
      "4\t2\t141.6\t0\t2\n",
      "5\t2\t35.4\t0\t2\n",
      "6\t2\t8.8\t0\t2\n",
      "7\t19\t2.2\t0\t19\n",
      "8\t10\t0.6\t0\t10\n",
      "9\t27\t0.1\t0\t27\n",
      "10\t58\t0.0\t1\t58\n",
      "11\t15\t0.0\t1\t15\n",
      "12\t23\t0.0\t1\t23\n",
      "13\t9\t0.0\t1\t9\n",
      "14\t10\t0.0\t1\t10\n",
      "15\t33\t0.0\t1\t32 1\n",
      "16\t23\t0.0\t1\t23\n",
      "17\t46\t0.0\t1\t44 2\n",
      "18\t83\t0.0\t1\t80 3\n",
      "19\t122\t0.0\t1\t91 30 1\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t16\t0.0\t2\t0 16\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/463.D_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/463.D_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/463.D_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/463.D_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.56 s (17 us/read; 3.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            152,071\n",
      "  Read 1 with adapter:                     123 (0.1%)\n",
      "  Read 2 with adapter:                   7,791 (5.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       152,071 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,118,702 bp\n",
      "  Read 1:    33,936,824 bp\n",
      "  Read 2:    35,181,878 bp\n",
      "Total written (filtered):     69,026,672 bp (99.9%)\n",
      "  Read 1:    33,934,944 bp\n",
      "  Read 2:    35,091,728 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 123 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t108\t2376.1\t0\t108\n",
      "22\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "100\t2\t0.0\t1\t2\n",
      "101\t1\t0.0\t1\t1\n",
      "102\t2\t0.0\t1\t2\n",
      "103\t3\t0.0\t1\t3\n",
      "104\t1\t0.0\t1\t1\n",
      "161\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7791 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t41\t2376.1\t0\t41\n",
      "4\t12\t594.0\t0\t12\n",
      "5\t23\t148.5\t0\t23\n",
      "6\t93\t37.1\t0\t93\n",
      "7\t428\t9.3\t0\t428\n",
      "8\t537\t2.3\t0\t537\n",
      "9\t2049\t0.6\t0\t2047 2\n",
      "10\t2215\t0.1\t1\t2201 14\n",
      "11\t85\t0.0\t1\t85\n",
      "12\t88\t0.0\t1\t88\n",
      "13\t72\t0.0\t1\t71 1\n",
      "14\t135\t0.0\t1\t132 3\n",
      "15\t179\t0.0\t1\t176 3\n",
      "16\t207\t0.0\t1\t195 12\n",
      "17\t356\t0.0\t1\t340 16\n",
      "18\t402\t0.0\t1\t377 19 6\n",
      "19\t741\t0.0\t1\t478 262 1\n",
      "20\t27\t0.0\t2\t0 3 24\n",
      "21\t96\t0.0\t2\t0 96\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "240\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/463.U_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/463.U_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/463.U_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/463.U_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.26 s (18 us/read; 3.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            178,947\n",
      "  Read 1 with adapter:                     115 (0.1%)\n",
      "  Read 2 with adapter:                   2,999 (1.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       178,947 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,321,678 bp\n",
      "  Read 1:    39,997,726 bp\n",
      "  Read 2:    41,323,952 bp\n",
      "Total written (filtered):     81,275,658 bp (99.9%)\n",
      "  Read 1:    39,996,207 bp\n",
      "  Read 2:    41,279,451 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 115 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t105\t2796.0\t0\t105\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "37\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t3\n",
      "217\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2999 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t2796.0\t0\t10\n",
      "4\t3\t699.0\t0\t3\n",
      "5\t13\t174.8\t0\t13\n",
      "6\t14\t43.7\t0\t14\n",
      "7\t70\t10.9\t0\t70\n",
      "8\t62\t2.7\t0\t62\n",
      "9\t271\t0.7\t0\t271\n",
      "10\t516\t0.2\t1\t515 1\n",
      "11\t50\t0.0\t1\t50\n",
      "12\t57\t0.0\t1\t57\n",
      "13\t67\t0.0\t1\t67\n",
      "14\t89\t0.0\t1\t84 5\n",
      "15\t138\t0.0\t1\t137 1\n",
      "16\t149\t0.0\t1\t142 7\n",
      "17\t253\t0.0\t1\t246 7\n",
      "18\t327\t0.0\t1\t306 14 7\n",
      "19\t771\t0.0\t1\t521 250\n",
      "20\t20\t0.0\t2\t0 1 19\n",
      "21\t116\t0.0\t2\t0 115 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "204\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/464.D_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/464.D_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/464.D_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/464.D_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.97 s (13 us/read; 4.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            226,135\n",
      "  Read 1 with adapter:                     143 (0.1%)\n",
      "  Read 2 with adapter:                   8,854 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       226,135 (100.0%)\n",
      "\n",
      "Total basepairs processed:   103,037,965 bp\n",
      "  Read 1:    50,633,206 bp\n",
      "  Read 2:    52,404,759 bp\n",
      "Total written (filtered):    102,927,760 bp (99.9%)\n",
      "  Read 1:    50,632,103 bp\n",
      "  Read 2:    52,295,657 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 143 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t132\t3533.4\t0\t132\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t3\t0.0\t1\t3\n",
      "31\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8854 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t60\t3533.4\t0\t60\n",
      "4\t9\t883.3\t0\t9\n",
      "5\t31\t220.8\t0\t31\n",
      "6\t69\t55.2\t0\t69\n",
      "7\t392\t13.8\t0\t392\n",
      "8\t523\t3.5\t0\t523\n",
      "9\t1973\t0.9\t0\t1973\n",
      "10\t2486\t0.2\t1\t2471 15\n",
      "11\t104\t0.1\t1\t103 1\n",
      "12\t111\t0.0\t1\t108 3\n",
      "13\t134\t0.0\t1\t131 3\n",
      "14\t161\t0.0\t1\t154 7\n",
      "15\t212\t0.0\t1\t208 4\n",
      "16\t266\t0.0\t1\t257 9\n",
      "17\t486\t0.0\t1\t470 16\n",
      "18\t549\t0.0\t1\t509 22 18\n",
      "19\t1100\t0.0\t1\t674 425 1\n",
      "20\t20\t0.0\t2\t0 1 19\n",
      "21\t157\t0.0\t2\t0 156 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "174\t2\t0.0\t2\t0 0 2\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t4\t0.0\t2\t3 0 1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "240\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/464.U_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/464.U_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/464.U_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/464.U_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.07 s (34 us/read; 1.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,867\n",
      "  Read 1 with adapter:                      25 (0.1%)\n",
      "  Read 2 with adapter:                     504 (1.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        31,867 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,453,897 bp\n",
      "  Read 1:     7,109,205 bp\n",
      "  Read 2:     7,344,692 bp\n",
      "Total written (filtered):     14,446,174 bp (99.9%)\n",
      "  Read 1:     7,109,082 bp\n",
      "  Read 2:     7,337,092 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 25 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t497.9\t0\t23\n",
      "27\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 504 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t497.9\t0\t1\n",
      "4\t1\t124.5\t0\t1\n",
      "5\t2\t31.1\t0\t2\n",
      "6\t1\t7.8\t0\t1\n",
      "7\t13\t1.9\t0\t13\n",
      "8\t7\t0.5\t0\t7\n",
      "9\t27\t0.1\t0\t27\n",
      "10\t76\t0.0\t1\t76\n",
      "11\t20\t0.0\t1\t19 1\n",
      "12\t10\t0.0\t1\t10\n",
      "13\t8\t0.0\t1\t8\n",
      "14\t13\t0.0\t1\t13\n",
      "15\t27\t0.0\t1\t27\n",
      "16\t29\t0.0\t1\t27 2\n",
      "17\t59\t0.0\t1\t57 2\n",
      "18\t90\t0.0\t1\t90\n",
      "19\t105\t0.0\t1\t77 27 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t14\t0.0\t2\t0 14\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/465.H_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/465.H_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/465.H_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/465.H_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.75 s (52 us/read; 1.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,380\n",
      "  Read 1 with adapter:                       9 (0.1%)\n",
      "  Read 2 with adapter:                     286 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        14,380 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,463,553 bp\n",
      "  Read 1:     3,177,789 bp\n",
      "  Read 2:     3,285,764 bp\n",
      "Total written (filtered):      6,459,300 bp (99.9%)\n",
      "  Read 1:     3,177,738 bp\n",
      "  Read 2:     3,281,562 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t224.7\t0\t8\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 286 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t224.7\t0\t1\n",
      "4\t1\t56.2\t0\t1\n",
      "5\t1\t14.0\t0\t1\n",
      "7\t6\t0.9\t0\t6\n",
      "8\t2\t0.2\t0\t2\n",
      "9\t15\t0.1\t0\t15\n",
      "10\t48\t0.0\t1\t48\n",
      "11\t10\t0.0\t1\t10\n",
      "12\t12\t0.0\t1\t12\n",
      "13\t12\t0.0\t1\t12\n",
      "14\t11\t0.0\t1\t11\n",
      "15\t23\t0.0\t1\t23\n",
      "16\t16\t0.0\t1\t16\n",
      "17\t26\t0.0\t1\t26\n",
      "18\t46\t0.0\t1\t45 1\n",
      "19\t46\t0.0\t1\t34 12\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t8\t0.0\t2\t0 8\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/466.H_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/466.H_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/466.H_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/466.H_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.15 s (35 us/read; 1.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,781\n",
      "  Read 1 with adapter:                      17 (0.1%)\n",
      "  Read 2 with adapter:                     612 (1.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,781 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,710,172 bp\n",
      "  Read 1:     7,229,598 bp\n",
      "  Read 2:     7,480,574 bp\n",
      "Total written (filtered):     14,701,094 bp (99.9%)\n",
      "  Read 1:     7,229,333 bp\n",
      "  Read 2:     7,471,761 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 17 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t512.2\t0\t16\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 612 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t128.1\t0\t2\n",
      "5\t2\t32.0\t0\t2\n",
      "6\t2\t8.0\t0\t2\n",
      "7\t14\t2.0\t0\t14\n",
      "8\t17\t0.5\t0\t17\n",
      "9\t69\t0.1\t0\t69\n",
      "10\t132\t0.0\t1\t132\n",
      "11\t14\t0.0\t1\t14\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t11\t0.0\t1\t11\n",
      "14\t14\t0.0\t1\t14\n",
      "15\t30\t0.0\t1\t30\n",
      "16\t22\t0.0\t1\t22\n",
      "17\t42\t0.0\t1\t41 1\n",
      "18\t82\t0.0\t1\t77 5\n",
      "19\t127\t0.0\t1\t78 48 1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t22\t0.0\t2\t0 22\n",
      "116\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/467.H_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/467.H_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/467.H_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/467.H_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.98 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,479\n",
      "  Read 1 with adapter:                      47 (0.1%)\n",
      "  Read 2 with adapter:                   1,595 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        80,479 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,516,068 bp\n",
      "  Read 1:    17,952,867 bp\n",
      "  Read 2:    18,563,201 bp\n",
      "Total written (filtered):     36,492,348 bp (99.9%)\n",
      "  Read 1:    17,952,513 bp\n",
      "  Read 2:    18,539,835 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 47 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t1257.5\t0\t45\n",
      "27\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1595 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1257.5\t0\t4\n",
      "4\t1\t314.4\t0\t1\n",
      "5\t3\t78.6\t0\t3\n",
      "6\t6\t19.6\t0\t6\n",
      "7\t28\t4.9\t0\t28\n",
      "8\t50\t1.2\t0\t50\n",
      "9\t164\t0.3\t0\t164\n",
      "10\t280\t0.1\t1\t278 2\n",
      "11\t23\t0.0\t1\t23\n",
      "12\t31\t0.0\t1\t31\n",
      "13\t43\t0.0\t1\t43\n",
      "14\t44\t0.0\t1\t42 2\n",
      "15\t86\t0.0\t1\t83 3\n",
      "16\t87\t0.0\t1\t84 3\n",
      "17\t167\t0.0\t1\t162 5\n",
      "18\t188\t0.0\t1\t174 12 2\n",
      "19\t327\t0.0\t1\t218 108 1\n",
      "20\t9\t0.0\t2\t0 0 9\n",
      "21\t53\t0.0\t2\t0 53\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/468.D_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/468.D_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/468.D_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/468.D_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.51 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,157\n",
      "  Read 1 with adapter:                      23 (0.0%)\n",
      "  Read 2 with adapter:                   2,838 (5.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,157 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,052,876 bp\n",
      "  Read 1:    12,294,431 bp\n",
      "  Read 2:    12,758,445 bp\n",
      "Total written (filtered):     25,016,455 bp (99.9%)\n",
      "  Read 1:    12,292,769 bp\n",
      "  Read 2:    12,723,686 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t877.5\t0\t13\n",
      "27\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "110\t1\t0.0\t1\t1\n",
      "125\t1\t0.0\t1\t1\n",
      "203\t1\t0.0\t1\t0 1\n",
      "212\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2838 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t877.5\t0\t26\n",
      "4\t4\t219.4\t0\t4\n",
      "5\t22\t54.8\t0\t22\n",
      "6\t40\t13.7\t0\t40\n",
      "7\t131\t3.4\t0\t131\n",
      "8\t189\t0.9\t0\t189\n",
      "9\t660\t0.2\t0\t660\n",
      "10\t754\t0.1\t1\t752 2\n",
      "11\t39\t0.0\t1\t39\n",
      "12\t33\t0.0\t1\t33\n",
      "13\t34\t0.0\t1\t34\n",
      "14\t56\t0.0\t1\t56\n",
      "15\t69\t0.0\t1\t68 1\n",
      "16\t92\t0.0\t1\t87 5\n",
      "17\t174\t0.0\t1\t168 6\n",
      "18\t148\t0.0\t1\t140 5 3\n",
      "19\t316\t0.0\t1\t202 114\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t38\t0.0\t2\t0 37 1\n",
      "111\t1\t0.0\t2\t1\n",
      "115\t3\t0.0\t2\t3\n",
      "217\t4\t0.0\t2\t3 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/468.U_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/468.U_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/468.U_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/468.U_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.48 s (21 us/read; 2.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            266,116\n",
      "  Read 1 with adapter:                     109 (0.0%)\n",
      "  Read 2 with adapter:                   8,919 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       266,116 (100.0%)\n",
      "\n",
      "Total basepairs processed:   120,872,183 bp\n",
      "  Read 1:    59,452,164 bp\n",
      "  Read 2:    61,420,019 bp\n",
      "Total written (filtered):    120,755,140 bp (99.9%)\n",
      "  Read 1:    59,450,901 bp\n",
      "  Read 2:    61,304,239 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t101\t4158.1\t0\t101\n",
      "27\t4\t0.0\t1\t4\n",
      "208\t1\t0.0\t1\t0 1\n",
      "211\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8919 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t4158.1\t0\t53\n",
      "4\t10\t1039.5\t0\t10\n",
      "5\t20\t259.9\t0\t20\n",
      "6\t76\t65.0\t0\t76\n",
      "7\t274\t16.2\t0\t274\n",
      "8\t454\t4.1\t0\t454\n",
      "9\t1723\t1.0\t0\t1721 2\n",
      "10\t2256\t0.3\t1\t2242 14\n",
      "11\t125\t0.1\t1\t123 2\n",
      "12\t166\t0.0\t1\t166\n",
      "13\t155\t0.0\t1\t154 1\n",
      "14\t201\t0.0\t1\t192 9\n",
      "15\t319\t0.0\t1\t315 4\n",
      "16\t329\t0.0\t1\t321 8\n",
      "17\t600\t0.0\t1\t584 16\n",
      "18\t655\t0.0\t1\t604 30 21\n",
      "19\t1277\t0.0\t1\t794 481 2\n",
      "20\t37\t0.0\t2\t0 0 37\n",
      "21\t172\t0.0\t2\t0 172\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "47\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t0 1\n",
      "137\t1\t0.0\t2\t1\n",
      "165\t1\t0.0\t2\t1\n",
      "215\t2\t0.0\t2\t2\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t3\t0.0\t2\t3\n",
      "229\t1\t0.0\t2\t1\n",
      "231\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/469.D_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/469.D_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/469.D_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/469.D_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.07 s (21 us/read; 2.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,486\n",
      "  Read 1 with adapter:                      53 (0.1%)\n",
      "  Read 2 with adapter:                   4,722 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        96,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,429,652 bp\n",
      "  Read 1:    21,309,008 bp\n",
      "  Read 2:    22,120,644 bp\n",
      "Total written (filtered):     43,372,476 bp (99.9%)\n",
      "  Read 1:    21,307,565 bp\n",
      "  Read 2:    22,064,911 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 53 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t1507.6\t0\t45\n",
      "12\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4722 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t1507.6\t0\t31\n",
      "4\t5\t376.9\t0\t5\n",
      "5\t19\t94.2\t0\t19\n",
      "6\t75\t23.6\t0\t75\n",
      "7\t237\t5.9\t0\t237\n",
      "8\t327\t1.5\t0\t327\n",
      "9\t1178\t0.4\t0\t1178\n",
      "10\t1229\t0.1\t1\t1219 10\n",
      "11\t56\t0.0\t1\t55 1\n",
      "12\t51\t0.0\t1\t51\n",
      "13\t36\t0.0\t1\t36\n",
      "14\t85\t0.0\t1\t85\n",
      "15\t125\t0.0\t1\t123 2\n",
      "16\t143\t0.0\t1\t137 6\n",
      "17\t279\t0.0\t1\t268 11\n",
      "18\t295\t0.0\t1\t277 12 6\n",
      "19\t472\t0.0\t1\t286 185 1\n",
      "20\t18\t0.0\t2\t0 1 17\n",
      "21\t59\t0.0\t2\t0 59\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/469.U_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/469.U_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/469.U_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/469.U_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.00 s (18 us/read; 3.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            218,244\n",
      "  Read 1 with adapter:                      31 (0.0%)\n",
      "  Read 2 with adapter:                   8,688 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       218,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,783,675 bp\n",
      "  Read 1:    48,601,325 bp\n",
      "  Read 2:    50,182,350 bp\n",
      "Total written (filtered):     98,677,015 bp (99.9%)\n",
      "  Read 1:    48,599,337 bp\n",
      "  Read 2:    50,077,678 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 31 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t3410.1\t0\t20\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "193\t2\t0.0\t1\t2\n",
      "201\t1\t0.0\t1\t1\n",
      "212\t2\t0.0\t1\t2\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8688 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t3410.1\t0\t73\n",
      "4\t8\t852.5\t0\t8\n",
      "5\t55\t213.1\t0\t55\n",
      "6\t101\t53.3\t0\t101\n",
      "7\t339\t13.3\t0\t339\n",
      "8\t593\t3.3\t0\t593\n",
      "9\t2083\t0.8\t0\t2080 3\n",
      "10\t2430\t0.2\t1\t2426 4\n",
      "11\t103\t0.1\t1\t102 1\n",
      "12\t87\t0.0\t1\t87\n",
      "13\t85\t0.0\t1\t85\n",
      "14\t133\t0.0\t1\t130 3\n",
      "15\t190\t0.0\t1\t186 4\n",
      "16\t240\t0.0\t1\t232 8\n",
      "17\t417\t0.0\t1\t400 17\n",
      "18\t500\t0.0\t1\t469 26 5\n",
      "19\t1088\t0.0\t1\t712 372 4\n",
      "20\t15\t0.0\t2\t0 1 14\n",
      "21\t140\t0.0\t2\t0 140\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "207\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "218\t2\t0.0\t2\t2\n",
      "224\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/461.D_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/461.D_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/461.D_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/461.D_15_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/461.U_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/461.U_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/461.U_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/461.U_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/462.D_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/462.D_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/462.D_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/462.D_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/462.U_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/462.U_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/462.U_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/462.U_1_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/463.D_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/463.D_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/463.D_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/463.D_27_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/463.U_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/463.U_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/463.U_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/463.U_18_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/464.D_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/464.D_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/464.D_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/464.D_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/464.U_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/464.U_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/464.U_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/464.U_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/465.H_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/465.H_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/465.H_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/465.H_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/466.H_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/466.H_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/466.H_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/466.H_2_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/467.H_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/467.H_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/467.H_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/467.H_24_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/468.D_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/468.D_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/468.D_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/468.D_25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/468.U_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/468.U_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/468.U_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/468.U_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/469.D_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/469.D_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/469.D_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/469.D_21_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/469.U_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/469.U_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/469.U_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/469.U_29_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/470.D_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/470.D_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/470.D_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/470.D_16_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/470.D_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/470.D_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/470.D_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/470.D_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.87 s (18 us/read; 3.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            215,956\n",
      "  Read 1 with adapter:                      79 (0.0%)\n",
      "  Read 2 with adapter:                   8,741 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       215,956 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,015,374 bp\n",
      "  Read 1:    48,179,496 bp\n",
      "  Read 2:    49,835,878 bp\n",
      "Total written (filtered):     97,902,193 bp (99.9%)\n",
      "  Read 1:    48,174,706 bp\n",
      "  Read 2:    49,727,487 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 79 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t3374.3\t0\t53\n",
      "25\t3\t0.0\t1\t3\n",
      "192\t2\t0.0\t1\t0 2\n",
      "193\t16\t0.0\t1\t14 2\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8741 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t3374.3\t0\t50\n",
      "4\t9\t843.6\t0\t9\n",
      "5\t34\t210.9\t0\t34\n",
      "6\t87\t52.7\t0\t87\n",
      "7\t328\t13.2\t0\t328\n",
      "8\t496\t3.3\t0\t496\n",
      "9\t1972\t0.8\t0\t1970 2\n",
      "10\t2321\t0.2\t1\t2306 15\n",
      "11\t95\t0.1\t1\t95\n",
      "12\t114\t0.0\t1\t111 3\n",
      "13\t108\t0.0\t1\t105 3\n",
      "14\t186\t0.0\t1\t182 4\n",
      "15\t248\t0.0\t1\t245 3\n",
      "16\t273\t0.0\t1\t262 11\n",
      "17\t490\t0.0\t1\t474 16\n",
      "18\t549\t0.0\t1\t506 31 12\n",
      "19\t1199\t0.0\t1\t761 434 4\n",
      "20\t22\t0.0\t2\t0 3 19\n",
      "21\t154\t0.0\t2\t0 154\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t1\n",
      "228\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/470.U_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/470.U_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/470.U_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/470.U_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.92 s (19 us/read; 3.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            150,172\n",
      "  Read 1 with adapter:                      39 (0.0%)\n",
      "  Read 2 with adapter:                   5,892 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       150,172 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,917,560 bp\n",
      "  Read 1:    33,406,339 bp\n",
      "  Read 2:    34,511,221 bp\n",
      "Total written (filtered):     67,845,928 bp (99.9%)\n",
      "  Read 1:    33,404,935 bp\n",
      "  Read 2:    34,440,993 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 39 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t2346.4\t0\t31\n",
      "25\t2\t0.0\t1\t2\n",
      "193\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5892 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t2346.4\t0\t46\n",
      "4\t7\t586.6\t0\t7\n",
      "5\t40\t146.7\t0\t40\n",
      "6\t77\t36.7\t0\t77\n",
      "7\t231\t9.2\t0\t231\n",
      "8\t394\t2.3\t0\t394\n",
      "9\t1561\t0.6\t0\t1561\n",
      "10\t1555\t0.1\t1\t1553 2\n",
      "11\t62\t0.0\t1\t62\n",
      "12\t63\t0.0\t1\t63\n",
      "13\t54\t0.0\t1\t54\n",
      "14\t72\t0.0\t1\t68 4\n",
      "15\t150\t0.0\t1\t148 2\n",
      "16\t147\t0.0\t1\t142 5\n",
      "17\t288\t0.0\t1\t281 7\n",
      "18\t370\t0.0\t1\t352 14 4\n",
      "19\t658\t0.0\t1\t449 208 1\n",
      "20\t13\t0.0\t2\t0 1 12\n",
      "21\t99\t0.0\t2\t0 99\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "213\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/471.D_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/471.D_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/471.D_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/471.D_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.95 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,579\n",
      "  Read 1 with adapter:                      41 (0.1%)\n",
      "  Read 2 with adapter:                   3,936 (5.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        78,579 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,274,238 bp\n",
      "  Read 1:    17,312,899 bp\n",
      "  Read 2:    17,961,339 bp\n",
      "Total written (filtered):     35,224,657 bp (99.9%)\n",
      "  Read 1:    17,310,415 bp\n",
      "  Read 2:    17,914,242 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 41 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t1227.8\t0\t28\n",
      "25\t1\t0.0\t1\t1\n",
      "82\t1\t0.0\t1\t1\n",
      "124\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t1 1\n",
      "217\t7\t0.0\t1\t7\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3936 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t1227.8\t0\t19\n",
      "4\t7\t306.9\t0\t7\n",
      "5\t20\t76.7\t0\t20\n",
      "6\t40\t19.2\t0\t40\n",
      "7\t179\t4.8\t0\t179\n",
      "8\t276\t1.2\t0\t276\n",
      "9\t964\t0.3\t0\t962 2\n",
      "10\t1006\t0.1\t1\t1000 6\n",
      "11\t45\t0.0\t1\t45\n",
      "12\t42\t0.0\t1\t40 2\n",
      "13\t48\t0.0\t1\t48\n",
      "14\t67\t0.0\t1\t65 2\n",
      "15\t144\t0.0\t1\t140 4\n",
      "16\t116\t0.0\t1\t112 4\n",
      "17\t241\t0.0\t1\t237 4\n",
      "18\t220\t0.0\t1\t202 11 7\n",
      "19\t442\t0.0\t1\t264 177 1\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t46\t0.0\t2\t0 46\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/471.U_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/471.U_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/471.U_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/471.U_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.31 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,071\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                   2,242 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,071 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,555,561 bp\n",
      "  Read 1:    10,590,664 bp\n",
      "  Read 2:    10,964,897 bp\n",
      "Total written (filtered):     21,528,626 bp (99.9%)\n",
      "  Read 1:    10,590,429 bp\n",
      "  Read 2:    10,938,197 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t751.1\t0\t6\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2242 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t751.1\t0\t14\n",
      "4\t4\t187.8\t0\t4\n",
      "5\t17\t46.9\t0\t17\n",
      "6\t38\t11.7\t0\t38\n",
      "7\t104\t2.9\t0\t104\n",
      "8\t149\t0.7\t0\t149\n",
      "9\t534\t0.2\t0\t532 2\n",
      "10\t596\t0.0\t1\t596\n",
      "11\t23\t0.0\t1\t23\n",
      "12\t15\t0.0\t1\t15\n",
      "13\t16\t0.0\t1\t16\n",
      "14\t39\t0.0\t1\t38 1\n",
      "15\t42\t0.0\t1\t42\n",
      "16\t62\t0.0\t1\t59 3\n",
      "17\t156\t0.0\t1\t154 2\n",
      "18\t135\t0.0\t1\t120 7 8\n",
      "19\t263\t0.0\t1\t169 94\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t30\t0.0\t2\t0 30\n",
      "45\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/472.D_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/472.D_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/472.D_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/472.D_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.39 s (19 us/read; 3.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            181,141\n",
      "  Read 1 with adapter:                      94 (0.1%)\n",
      "  Read 2 with adapter:                   5,472 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       181,141 (100.0%)\n",
      "\n",
      "Total basepairs processed:    82,372,880 bp\n",
      "  Read 1:    40,494,090 bp\n",
      "  Read 2:    41,878,790 bp\n",
      "Total written (filtered):     82,301,430 bp (99.9%)\n",
      "  Read 1:    40,493,474 bp\n",
      "  Read 2:    41,807,956 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 94 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t92\t2830.3\t0\t92\n",
      "127\t1\t0.0\t1\t0 1\n",
      "213\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5472 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t2830.3\t0\t28\n",
      "4\t4\t707.6\t0\t4\n",
      "5\t10\t176.9\t0\t10\n",
      "6\t37\t44.2\t0\t37\n",
      "7\t152\t11.1\t0\t152\n",
      "8\t278\t2.8\t0\t278\n",
      "9\t1043\t0.7\t0\t1042 1\n",
      "10\t1435\t0.2\t1\t1431 4\n",
      "11\t63\t0.0\t1\t62 1\n",
      "12\t92\t0.0\t1\t91 1\n",
      "13\t67\t0.0\t1\t66 1\n",
      "14\t126\t0.0\t1\t121 5\n",
      "15\t161\t0.0\t1\t155 6\n",
      "16\t190\t0.0\t1\t183 7\n",
      "17\t362\t0.0\t1\t350 12\n",
      "18\t423\t0.0\t1\t400 17 6\n",
      "19\t857\t0.0\t1\t572 284 1\n",
      "20\t14\t0.0\t2\t0 1 13\n",
      "21\t124\t0.0\t2\t0 124\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "174\t2\t0.0\t2\t0 0 2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/472.U_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/472.U_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/472.U_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/472.U_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.51 s (24 us/read; 2.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            105,493\n",
      "  Read 1 with adapter:                      50 (0.0%)\n",
      "  Read 2 with adapter:                   2,372 (2.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       105,493 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,846,166 bp\n",
      "  Read 1:    23,527,841 bp\n",
      "  Read 2:    24,318,325 bp\n",
      "Total written (filtered):     47,811,573 bp (99.9%)\n",
      "  Read 1:    23,527,691 bp\n",
      "  Read 2:    24,283,882 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 50 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t1648.3\t0\t50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2372 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1648.3\t0\t9\n",
      "4\t4\t412.1\t0\t4\n",
      "5\t4\t103.0\t0\t4\n",
      "6\t7\t25.8\t0\t7\n",
      "7\t40\t6.4\t0\t40\n",
      "8\t77\t1.6\t0\t77\n",
      "9\t246\t0.4\t0\t246\n",
      "10\t441\t0.1\t1\t438 3\n",
      "11\t33\t0.0\t1\t33\n",
      "12\t58\t0.0\t1\t55 3\n",
      "13\t56\t0.0\t1\t56\n",
      "14\t89\t0.0\t1\t85 4\n",
      "15\t102\t0.0\t1\t100 2\n",
      "16\t133\t0.0\t1\t126 7\n",
      "17\t275\t0.0\t1\t263 12\n",
      "18\t213\t0.0\t1\t201 5 7\n",
      "19\t499\t0.0\t1\t329 170\n",
      "20\t12\t0.0\t2\t1 0 11\n",
      "21\t72\t0.0\t2\t0 72\n",
      "155\t1\t0.0\t2\t0 1\n",
      "230\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/473.H_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/473.H_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/473.H_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/473.H_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.10 s (19 us/read; 3.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            163,373\n",
      "  Read 1 with adapter:                      24 (0.0%)\n",
      "  Read 2 with adapter:                   6,784 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       163,373 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,841,019 bp\n",
      "  Read 1:    36,318,094 bp\n",
      "  Read 2:    37,522,925 bp\n",
      "Total written (filtered):     73,756,988 bp (99.9%)\n",
      "  Read 1:    36,316,338 bp\n",
      "  Read 2:    37,440,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 24 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t2552.7\t0\t13\n",
      "27\t2\t0.0\t1\t2\n",
      "42\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "208\t1\t0.0\t1\t1\n",
      "211\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t1 2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6784 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t38\t2552.7\t0\t38\n",
      "4\t9\t638.2\t0\t9\n",
      "5\t38\t159.5\t0\t38\n",
      "6\t80\t39.9\t0\t80\n",
      "7\t257\t10.0\t0\t257\n",
      "8\t393\t2.5\t0\t393\n",
      "9\t1658\t0.6\t0\t1656 2\n",
      "10\t2012\t0.2\t1\t2002 10\n",
      "11\t101\t0.0\t1\t101\n",
      "12\t56\t0.0\t1\t56\n",
      "13\t58\t0.0\t1\t58\n",
      "14\t102\t0.0\t1\t97 5\n",
      "15\t122\t0.0\t1\t119 3\n",
      "16\t184\t0.0\t1\t178 6\n",
      "17\t335\t0.0\t1\t319 16\n",
      "18\t382\t0.0\t1\t364 14 4\n",
      "19\t821\t0.0\t1\t521 299 1\n",
      "20\t18\t0.0\t2\t0 1 17\n",
      "21\t110\t0.0\t2\t0 110\n",
      "114\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "217\t6\t0.0\t2\t6\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/474.H_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/474.H_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/474.H_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/474.H_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.56 s (21 us/read; 2.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            121,494\n",
      "  Read 1 with adapter:                      30 (0.0%)\n",
      "  Read 2 with adapter:                   4,154 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       121,494 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,160,045 bp\n",
      "  Read 1:    27,141,736 bp\n",
      "  Read 2:    28,018,309 bp\n",
      "Total written (filtered):     55,107,499 bp (99.9%)\n",
      "  Read 1:    27,140,926 bp\n",
      "  Read 2:    27,966,573 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 30 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t1898.3\t0\t25\n",
      "24\t1\t0.0\t1\t1\n",
      "118\t1\t0.0\t1\t1\n",
      "157\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4154 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t1898.3\t0\t23\n",
      "4\t6\t474.6\t0\t6\n",
      "5\t18\t118.6\t0\t18\n",
      "6\t38\t29.7\t0\t38\n",
      "7\t147\t7.4\t0\t147\n",
      "8\t241\t1.9\t0\t241\n",
      "9\t820\t0.5\t0\t819 1\n",
      "10\t1174\t0.1\t1\t1170 4\n",
      "11\t70\t0.0\t1\t70\n",
      "12\t53\t0.0\t1\t53\n",
      "13\t55\t0.0\t1\t55\n",
      "14\t74\t0.0\t1\t73 1\n",
      "15\t134\t0.0\t1\t131 3\n",
      "16\t125\t0.0\t1\t122 3\n",
      "17\t277\t0.0\t1\t268 9\n",
      "18\t265\t0.0\t1\t244 16 5\n",
      "19\t536\t0.0\t1\t367 168 1\n",
      "20\t12\t0.0\t2\t0 0 12\n",
      "21\t84\t0.0\t2\t0 84\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "230\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/475.H_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/475.H_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/475.H_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/475.H_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.72 s (25 us/read; 2.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,914\n",
      "  Read 1 with adapter:                      15 (0.0%)\n",
      "  Read 2 with adapter:                   2,142 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        67,914 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,608,847 bp\n",
      "  Read 1:    15,054,887 bp\n",
      "  Read 2:    15,553,960 bp\n",
      "Total written (filtered):     30,579,644 bp (99.9%)\n",
      "  Read 1:    15,054,201 bp\n",
      "  Read 2:    15,525,443 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 15 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1061.2\t0\t11\n",
      "24\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2142 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1061.2\t0\t11\n",
      "4\t2\t265.3\t0\t2\n",
      "5\t9\t66.3\t0\t9\n",
      "6\t20\t16.6\t0\t20\n",
      "7\t55\t4.1\t0\t55\n",
      "8\t92\t1.0\t0\t92\n",
      "9\t383\t0.3\t0\t383\n",
      "10\t519\t0.1\t1\t519\n",
      "11\t27\t0.0\t1\t27\n",
      "12\t39\t0.0\t1\t37 2\n",
      "13\t26\t0.0\t1\t26\n",
      "14\t58\t0.0\t1\t56 2\n",
      "15\t67\t0.0\t1\t65 2\n",
      "16\t79\t0.0\t1\t76 3\n",
      "17\t163\t0.0\t1\t158 5\n",
      "18\t154\t0.0\t1\t138 12 4\n",
      "19\t378\t0.0\t1\t233 144 1\n",
      "20\t10\t0.0\t2\t0 1 9\n",
      "21\t46\t0.0\t2\t0 46\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "217\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/476.D_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/476.D_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/476.D_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/476.D_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.34 s (20 us/read; 2.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,715\n",
      "  Read 1 with adapter:                      46 (0.0%)\n",
      "  Read 2 with adapter:                   1,337 (1.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,715 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,206,154 bp\n",
      "  Read 1:    25,688,312 bp\n",
      "  Read 2:    26,517,842 bp\n",
      "Total written (filtered):     52,184,756 bp (100.0%)\n",
      "  Read 1:    25,688,150 bp\n",
      "  Read 2:    26,496,606 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 46 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t1792.4\t0\t45\n",
      "27\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1337 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1792.4\t0\t6\n",
      "4\t1\t448.1\t0\t1\n",
      "5\t2\t112.0\t0\t2\n",
      "6\t3\t28.0\t0\t3\n",
      "7\t13\t7.0\t0\t13\n",
      "8\t19\t1.8\t0\t19\n",
      "9\t58\t0.4\t0\t58\n",
      "10\t185\t0.1\t1\t185\n",
      "11\t38\t0.0\t1\t38\n",
      "12\t48\t0.0\t1\t48\n",
      "13\t29\t0.0\t1\t29\n",
      "14\t27\t0.0\t1\t27\n",
      "15\t64\t0.0\t1\t64\n",
      "16\t92\t0.0\t1\t91 1\n",
      "17\t121\t0.0\t1\t117 4\n",
      "18\t200\t0.0\t1\t192 7 1\n",
      "19\t348\t0.0\t1\t256 92\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t68\t0.0\t2\t0 68\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/476.U_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/476.U_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/476.U_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/476.U_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.01 s (35 us/read; 1.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,794\n",
      "  Read 1 with adapter:                      13 (0.0%)\n",
      "  Read 2 with adapter:                     394 (1.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,971,381 bp\n",
      "  Read 1:     6,377,211 bp\n",
      "  Read 2:     6,594,170 bp\n",
      "Total written (filtered):     12,965,208 bp (100.0%)\n",
      "  Read 1:     6,377,172 bp\n",
      "  Read 2:     6,588,036 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t449.9\t0\t13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 394 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "7\t2\t1.8\t0\t2\n",
      "8\t7\t0.4\t0\t7\n",
      "9\t15\t0.1\t0\t15\n",
      "10\t55\t0.0\t1\t55\n",
      "11\t9\t0.0\t1\t9\n",
      "12\t15\t0.0\t1\t14 1\n",
      "13\t10\t0.0\t1\t10\n",
      "14\t10\t0.0\t1\t10\n",
      "15\t22\t0.0\t1\t22\n",
      "16\t30\t0.0\t1\t30\n",
      "17\t45\t0.0\t1\t42 3\n",
      "18\t78\t0.0\t1\t75 3\n",
      "19\t82\t0.0\t1\t48 33 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t13\t0.0\t2\t0 13\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/477.D_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/477.D_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/477.D_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/477.D_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.36 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,292\n",
      "  Read 1 with adapter:                      73 (0.1%)\n",
      "  Read 2 with adapter:                   1,268 (1.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        94,292 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,871,566 bp\n",
      "  Read 1:    21,091,395 bp\n",
      "  Read 2:    21,780,171 bp\n",
      "Total written (filtered):     42,851,167 bp (100.0%)\n",
      "  Read 1:    21,091,069 bp\n",
      "  Read 2:    21,760,098 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 73 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t69\t1473.3\t0\t69\n",
      "27\t3\t0.0\t1\t3\n",
      "38\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1268 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1473.3\t0\t6\n",
      "4\t3\t368.3\t0\t3\n",
      "5\t1\t92.1\t0\t1\n",
      "6\t2\t23.0\t0\t2\n",
      "7\t11\t5.8\t0\t11\n",
      "8\t23\t1.4\t0\t23\n",
      "9\t67\t0.4\t0\t66 1\n",
      "10\t177\t0.1\t1\t176 1\n",
      "11\t28\t0.0\t1\t28\n",
      "12\t23\t0.0\t1\t23\n",
      "13\t16\t0.0\t1\t16\n",
      "14\t33\t0.0\t1\t33\n",
      "15\t63\t0.0\t1\t62 1\n",
      "16\t88\t0.0\t1\t86 2\n",
      "17\t152\t0.0\t1\t148 4\n",
      "18\t190\t0.0\t1\t185 5\n",
      "19\t312\t0.0\t1\t216 96\n",
      "20\t7\t0.0\t2\t0 0 7\n",
      "21\t64\t0.0\t2\t0 64\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/477.U_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/477.U_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/477.U_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/477.U_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.42 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,326\n",
      "  Read 1 with adapter:                      41 (0.1%)\n",
      "  Read 2 with adapter:                     666 (1.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,674,569 bp\n",
      "  Read 1:    12,139,325 bp\n",
      "  Read 2:    12,535,244 bp\n",
      "Total written (filtered):     24,663,851 bp (100.0%)\n",
      "  Read 1:    12,139,202 bp\n",
      "  Read 2:    12,524,649 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 41 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t41\t848.8\t0\t41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 666 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t848.8\t0\t2\n",
      "4\t2\t212.2\t0\t2\n",
      "7\t2\t3.3\t0\t2\n",
      "8\t4\t0.8\t0\t4\n",
      "9\t21\t0.2\t0\t21\n",
      "10\t80\t0.1\t1\t79 1\n",
      "11\t25\t0.0\t1\t25\n",
      "12\t20\t0.0\t1\t20\n",
      "13\t19\t0.0\t1\t19\n",
      "14\t12\t0.0\t1\t12\n",
      "15\t30\t0.0\t1\t30\n",
      "16\t52\t0.0\t1\t52\n",
      "17\t82\t0.0\t1\t80 2\n",
      "18\t111\t0.0\t1\t102 7 2\n",
      "19\t168\t0.0\t1\t118 50\n",
      "20\t7\t0.0\t2\t0 0 7\n",
      "21\t28\t0.0\t2\t0 27 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/478.D_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/478.D_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/478.D_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/478.D_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.26 s (22 us/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,216\n",
      "  Read 1 with adapter:                      85 (0.1%)\n",
      "  Read 2 with adapter:                   1,393 (1.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       104,216 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,384,057 bp\n",
      "  Read 1:    23,311,584 bp\n",
      "  Read 2:    24,072,473 bp\n",
      "Total written (filtered):     47,362,654 bp (100.0%)\n",
      "  Read 1:    23,311,308 bp\n",
      "  Read 2:    24,051,346 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 85 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t84\t1628.4\t0\t84\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1393 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1628.4\t0\t6\n",
      "4\t3\t407.1\t0\t3\n",
      "5\t4\t101.8\t0\t4\n",
      "6\t2\t25.4\t0\t2\n",
      "7\t13\t6.4\t0\t13\n",
      "8\t18\t1.6\t0\t18\n",
      "9\t80\t0.4\t0\t80\n",
      "10\t230\t0.1\t1\t230\n",
      "11\t45\t0.0\t1\t44 1\n",
      "12\t45\t0.0\t1\t45\n",
      "13\t35\t0.0\t1\t35\n",
      "14\t22\t0.0\t1\t19 3\n",
      "15\t63\t0.0\t1\t63\n",
      "16\t101\t0.0\t1\t96 5\n",
      "17\t140\t0.0\t1\t139 1\n",
      "18\t185\t0.0\t1\t183 2\n",
      "19\t335\t0.0\t1\t231 104\n",
      "20\t11\t0.0\t2\t0 1 10\n",
      "21\t54\t0.0\t2\t0 54\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/478.U_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/478.U_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/478.U_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/478.U_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.90 s (39 us/read; 1.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,965\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                     271 (1.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,965 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,345,187 bp\n",
      "  Read 1:     5,086,340 bp\n",
      "  Read 2:     5,258,847 bp\n",
      "Total written (filtered):     10,340,540 bp (100.0%)\n",
      "  Read 1:     5,086,103 bp\n",
      "  Read 2:     5,254,437 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t358.8\t0\t7\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 271 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t358.8\t0\t2\n",
      "7\t3\t1.4\t0\t3\n",
      "8\t3\t0.4\t0\t3\n",
      "9\t7\t0.1\t0\t7\n",
      "10\t39\t0.0\t1\t39\n",
      "11\t13\t0.0\t1\t13\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t8\t0.0\t1\t8\n",
      "14\t7\t0.0\t1\t7\n",
      "15\t14\t0.0\t1\t14\n",
      "16\t20\t0.0\t1\t20\n",
      "17\t33\t0.0\t1\t32 1\n",
      "18\t43\t0.0\t1\t43\n",
      "19\t61\t0.0\t1\t47 14\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t11\t0.0\t2\t0 10 1\n",
      "226\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/479.D_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/479.D_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/479.D_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/479.D_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.63 s (20 us/read; 3.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            134,064\n",
      "  Read 1 with adapter:                      55 (0.0%)\n",
      "  Read 2 with adapter:                   1,871 (1.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       134,064 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,976,089 bp\n",
      "  Read 1:    29,999,627 bp\n",
      "  Read 2:    30,976,462 bp\n",
      "Total written (filtered):     60,947,044 bp (100.0%)\n",
      "  Read 1:    29,999,337 bp\n",
      "  Read 2:    30,947,707 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t2094.8\t0\t53\n",
      "27\t1\t0.0\t1\t1\n",
      "104\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1871 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t2094.8\t0\t10\n",
      "4\t2\t523.7\t0\t2\n",
      "5\t3\t130.9\t0\t3\n",
      "6\t4\t32.7\t0\t4\n",
      "7\t23\t8.2\t0\t23\n",
      "8\t32\t2.0\t0\t32\n",
      "9\t98\t0.5\t0\t98\n",
      "10\t302\t0.1\t1\t300 2\n",
      "11\t48\t0.0\t1\t48\n",
      "12\t66\t0.0\t1\t65 1\n",
      "13\t48\t0.0\t1\t48\n",
      "14\t45\t0.0\t1\t44 1\n",
      "15\t103\t0.0\t1\t103\n",
      "16\t126\t0.0\t1\t125 1\n",
      "17\t174\t0.0\t1\t172 2\n",
      "18\t273\t0.0\t1\t262 9 2\n",
      "19\t408\t0.0\t1\t281 125 2\n",
      "20\t16\t0.0\t2\t0 0 16\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "216\t2\t0.0\t2\t2\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/470.U_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/470.U_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/470.U_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/470.U_11_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/471.D_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/471.D_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/471.D_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/471.D_35_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/471.U_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/471.U_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/471.U_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/471.U_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/472.D_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/472.D_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/472.D_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/472.D_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/472.U_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/472.U_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/472.U_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/472.U_17_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/473.H_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/473.H_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/473.H_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/473.H_32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/474.H_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/474.H_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/474.H_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/474.H_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/475.H_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/475.H_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/475.H_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/475.H_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/476.D_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/476.D_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/476.D_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/476.D_9_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/476.U_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/476.U_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/476.U_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/476.U_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/477.D_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/477.D_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/477.D_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/477.D_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/477.U_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/477.U_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/477.U_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/477.U_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/478.D_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/478.D_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/478.D_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/478.D_3_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/478.U_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/478.U_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/478.U_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/478.U_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/479.D_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/479.D_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/479.D_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/479.D_26_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/479.U_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/479.U_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/479.U_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/479.U_46_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/479.U_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1p7u0da4/479.U_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/479.U_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-3du1tkf5/23b4c3fa-b8a4-4537-84cb-41b464ec199f/data/479.U_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.16 s (28 us/read; 2.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,230\n",
      "  Read 1 with adapter:                      23 (0.1%)\n",
      "  Read 2 with adapter:                     763 (1.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,230 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,698,721 bp\n",
      "  Read 1:     9,194,324 bp\n",
      "  Read 2:     9,504,397 bp\n",
      "Total written (filtered):     18,687,195 bp (99.9%)\n",
      "  Read 1:     9,194,255 bp\n",
      "  Read 2:     9,492,940 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t644.2\t0\t23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 763 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t644.2\t0\t6\n",
      "5\t1\t40.3\t0\t1\n",
      "7\t9\t2.5\t0\t9\n",
      "8\t6\t0.6\t0\t6\n",
      "9\t36\t0.2\t0\t36\n",
      "10\t112\t0.0\t1\t111 1\n",
      "11\t38\t0.0\t1\t38\n",
      "12\t37\t0.0\t1\t37\n",
      "13\t25\t0.0\t1\t25\n",
      "14\t11\t0.0\t1\t11\n",
      "15\t49\t0.0\t1\t49\n",
      "16\t66\t0.0\t1\t65 1\n",
      "17\t82\t0.0\t1\t81 1\n",
      "18\t120\t0.0\t1\t119 1\n",
      "19\t133\t0.0\t1\t99 34\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t26\t0.0\t2\t0 26\n",
      "\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGmicro2018_241-288.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/480.D_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/480.D_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/480.D_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/480.D_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.17 s (15 us/read; 4.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            355,828\n",
      "  Read 1 with adapter:                     172 (0.0%)\n",
      "  Read 2 with adapter:                   3,744 (1.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       355,828 (100.0%)\n",
      "\n",
      "Total basepairs processed:   161,945,813 bp\n",
      "  Read 1:    79,684,584 bp\n",
      "  Read 2:    82,261,229 bp\n",
      "Total written (filtered):    161,883,408 bp (100.0%)\n",
      "  Read 1:    79,683,246 bp\n",
      "  Read 2:    82,200,162 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 172 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t168\t5559.8\t0\t168\n",
      "201\t2\t0.0\t1\t2\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3744 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t5559.8\t0\t13\n",
      "4\t3\t1390.0\t0\t3\n",
      "5\t7\t347.5\t0\t7\n",
      "6\t10\t86.9\t0\t10\n",
      "7\t52\t21.7\t0\t52\n",
      "8\t47\t5.4\t0\t47\n",
      "9\t202\t1.4\t0\t201 1\n",
      "10\t501\t0.3\t1\t500 1\n",
      "11\t64\t0.1\t1\t62 2\n",
      "12\t68\t0.0\t1\t68\n",
      "13\t44\t0.0\t1\t44\n",
      "14\t53\t0.0\t1\t50 3\n",
      "15\t112\t0.0\t1\t110 2\n",
      "16\t146\t0.0\t1\t144 2\n",
      "17\t389\t0.0\t1\t382 7\n",
      "18\t525\t0.0\t1\t503 21 1\n",
      "19\t1218\t0.0\t1\t860 356 2\n",
      "20\t28\t0.0\t2\t0 2 26\n",
      "21\t253\t0.0\t2\t0 252 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "216\t5\t0.0\t2\t5\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/480.U_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/480.U_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/480.U_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/480.U_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.64 s (29 us/read; 2.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,846\n",
      "  Read 1 with adapter:                      30 (0.1%)\n",
      "  Read 2 with adapter:                   1,107 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,846 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,259,641 bp\n",
      "  Read 1:    12,419,223 bp\n",
      "  Read 2:    12,840,418 bp\n",
      "Total written (filtered):     25,242,575 bp (99.9%)\n",
      "  Read 1:    12,417,977 bp\n",
      "  Read 2:    12,824,598 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 30 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t872.6\t0\t21\n",
      "27\t4\t0.0\t1\t3 1\n",
      "213\t1\t0.0\t1\t0 1\n",
      "215\t2\t0.0\t1\t1 1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1107 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t872.6\t0\t4\n",
      "4\t2\t218.1\t0\t2\n",
      "5\t3\t54.5\t0\t3\n",
      "6\t2\t13.6\t0\t2\n",
      "7\t23\t3.4\t0\t23\n",
      "8\t31\t0.9\t0\t31\n",
      "9\t120\t0.2\t0\t120\n",
      "10\t268\t0.1\t1\t268\n",
      "11\t19\t0.0\t1\t19\n",
      "12\t24\t0.0\t1\t23 1\n",
      "13\t16\t0.0\t1\t16\n",
      "14\t19\t0.0\t1\t19\n",
      "15\t31\t0.0\t1\t31\n",
      "16\t45\t0.0\t1\t43 2\n",
      "17\t91\t0.0\t1\t87 4\n",
      "18\t139\t0.0\t1\t136 3\n",
      "19\t229\t0.0\t1\t167 62\n",
      "20\t9\t0.0\t2\t0 2 7\n",
      "21\t30\t0.0\t2\t0 30\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/481.H_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/481.H_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/481.H_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/481.H_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.13 s (35 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,783\n",
      "  Read 1 with adapter:                      22 (0.1%)\n",
      "  Read 2 with adapter:                     629 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        31,783 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,311,398 bp\n",
      "  Read 1:     7,033,470 bp\n",
      "  Read 2:     7,277,928 bp\n",
      "Total written (filtered):     14,301,278 bp (99.9%)\n",
      "  Read 1:     7,032,761 bp\n",
      "  Read 2:     7,268,517 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t496.6\t0\t14\n",
      "27\t5\t0.0\t1\t5\n",
      "100\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 629 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t496.6\t0\t1\n",
      "7\t11\t1.9\t0\t11\n",
      "8\t14\t0.5\t0\t14\n",
      "9\t55\t0.1\t0\t55\n",
      "10\t138\t0.0\t1\t138\n",
      "11\t13\t0.0\t1\t13\n",
      "12\t12\t0.0\t1\t12\n",
      "13\t10\t0.0\t1\t10\n",
      "14\t6\t0.0\t1\t6\n",
      "15\t19\t0.0\t1\t19\n",
      "16\t35\t0.0\t1\t35\n",
      "17\t65\t0.0\t1\t63 2\n",
      "18\t98\t0.0\t1\t98\n",
      "19\t127\t0.0\t1\t89 38\n",
      "20\t4\t0.0\t2\t0 1 3\n",
      "21\t20\t0.0\t2\t0 20\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/482.H_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/482.H_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/482.H_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/482.H_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.09 s (30 us/read; 2.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,616\n",
      "  Read 1 with adapter:                      20 (0.1%)\n",
      "  Read 2 with adapter:                     624 (1.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        36,616 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,548,778 bp\n",
      "  Read 1:     8,136,146 bp\n",
      "  Read 2:     8,412,632 bp\n",
      "Total written (filtered):     16,539,115 bp (99.9%)\n",
      "  Read 1:     8,135,825 bp\n",
      "  Read 2:     8,403,290 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t572.1\t0\t17\n",
      "27\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 624 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t572.1\t0\t2\n",
      "5\t1\t35.8\t0\t1\n",
      "6\t1\t8.9\t0\t1\n",
      "7\t12\t2.2\t0\t12\n",
      "8\t14\t0.6\t0\t14\n",
      "9\t46\t0.1\t0\t46\n",
      "10\t112\t0.0\t1\t112\n",
      "11\t15\t0.0\t1\t15\n",
      "12\t15\t0.0\t1\t14 1\n",
      "13\t7\t0.0\t1\t7\n",
      "14\t13\t0.0\t1\t12 1\n",
      "15\t23\t0.0\t1\t23\n",
      "16\t32\t0.0\t1\t32\n",
      "17\t61\t0.0\t1\t61\n",
      "18\t101\t0.0\t1\t98 2 1\n",
      "19\t140\t0.0\t1\t99 41\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t25\t0.0\t2\t0 25\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/483.H_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/483.H_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/483.H_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/483.H_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.86 s (43 us/read; 1.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,019\n",
      "  Read 1 with adapter:                      15 (0.1%)\n",
      "  Read 2 with adapter:                     325 (1.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        20,019 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,035,306 bp\n",
      "  Read 1:     4,443,101 bp\n",
      "  Read 2:     4,592,205 bp\n",
      "Total written (filtered):      9,030,287 bp (99.9%)\n",
      "  Read 1:     4,443,056 bp\n",
      "  Read 2:     4,587,231 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 15 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t312.8\t0\t15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 325 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t312.8\t0\t2\n",
      "7\t5\t1.2\t0\t5\n",
      "8\t4\t0.3\t0\t4\n",
      "9\t26\t0.1\t0\t26\n",
      "10\t43\t0.0\t1\t43\n",
      "11\t7\t0.0\t1\t7\n",
      "12\t9\t0.0\t1\t9\n",
      "13\t7\t0.0\t1\t7\n",
      "14\t7\t0.0\t1\t7\n",
      "15\t14\t0.0\t1\t14\n",
      "16\t16\t0.0\t1\t16\n",
      "17\t39\t0.0\t1\t39\n",
      "18\t65\t0.0\t1\t64 1\n",
      "19\t67\t0.0\t1\t59 8\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t11\t0.0\t2\t0 11\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/484.D_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/484.D_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/484.D_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/484.D_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.50 s (18 us/read; 3.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            361,969\n",
      "  Read 1 with adapter:                     148 (0.0%)\n",
      "  Read 2 with adapter:                  12,281 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       361,969 (100.0%)\n",
      "\n",
      "Total basepairs processed:   164,891,871 bp\n",
      "  Read 1:    81,042,217 bp\n",
      "  Read 2:    83,849,654 bp\n",
      "Total written (filtered):    164,731,897 bp (99.9%)\n",
      "  Read 1:    81,041,463 bp\n",
      "  Read 2:    83,690,434 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 148 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t144\t5655.8\t0\t144\n",
      "27\t2\t0.0\t1\t2\n",
      "52\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12281 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t5655.8\t0\t88\n",
      "4\t11\t1413.9\t0\t11\n",
      "5\t45\t353.5\t0\t45\n",
      "6\t84\t88.4\t0\t84\n",
      "7\t368\t22.1\t0\t368\n",
      "8\t490\t5.5\t0\t490\n",
      "9\t2292\t1.4\t0\t2289 3\n",
      "10\t3485\t0.3\t1\t3469 16\n",
      "11\t204\t0.1\t1\t201 3\n",
      "12\t221\t0.0\t1\t221\n",
      "13\t172\t0.0\t1\t172\n",
      "14\t277\t0.0\t1\t269 8\n",
      "15\t306\t0.0\t1\t302 4\n",
      "16\t346\t0.0\t1\t326 20\n",
      "17\t793\t0.0\t1\t759 34\n",
      "18\t990\t0.0\t1\t935 26 29\n",
      "19\t1794\t0.0\t1\t1197 595 2\n",
      "20\t42\t0.0\t2\t0 1 41\n",
      "21\t251\t0.0\t2\t0 251\n",
      "22\t5\t0.0\t2\t0 1 4\n",
      "30\t1\t0.0\t2\t1\n",
      "156\t1\t0.0\t2\t1\n",
      "215\t2\t0.0\t2\t1 1\n",
      "216\t7\t0.0\t2\t7\n",
      "217\t2\t0.0\t2\t2\n",
      "225\t1\t0.0\t2\t1\n",
      "226\t3\t0.0\t2\t3\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/484.U_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/484.U_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/484.U_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/484.U_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.05 s (18 us/read; 3.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            337,993\n",
      "  Read 1 with adapter:                     155 (0.0%)\n",
      "  Read 2 with adapter:                  10,721 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       337,993 (100.0%)\n",
      "\n",
      "Total basepairs processed:   153,961,712 bp\n",
      "  Read 1:    75,683,566 bp\n",
      "  Read 2:    78,278,146 bp\n",
      "Total written (filtered):    153,822,585 bp (99.9%)\n",
      "  Read 1:    75,682,346 bp\n",
      "  Read 2:    78,140,239 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 155 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t148\t5281.1\t0\t148\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "54\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10721 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t90\t5281.1\t0\t90\n",
      "4\t8\t1320.3\t0\t8\n",
      "5\t36\t330.1\t0\t36\n",
      "6\t61\t82.5\t0\t61\n",
      "7\t380\t20.6\t0\t380\n",
      "8\t460\t5.2\t0\t460\n",
      "9\t2037\t1.3\t0\t2035 2\n",
      "10\t3088\t0.3\t1\t3076 12\n",
      "11\t169\t0.1\t1\t167 2\n",
      "12\t140\t0.0\t1\t137 3\n",
      "13\t124\t0.0\t1\t123 1\n",
      "14\t194\t0.0\t1\t193 1\n",
      "15\t252\t0.0\t1\t244 8\n",
      "16\t307\t0.0\t1\t300 7\n",
      "17\t639\t0.0\t1\t629 10\n",
      "18\t859\t0.0\t1\t803 37 19\n",
      "19\t1612\t0.0\t1\t1094 514 4\n",
      "20\t33\t0.0\t2\t0 1 32\n",
      "21\t215\t0.0\t2\t0 213 2\n",
      "22\t3\t0.0\t2\t0 1 2\n",
      "81\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t1\n",
      "215\t2\t0.0\t2\t2\n",
      "216\t6\t0.0\t2\t6\n",
      "217\t3\t0.0\t2\t3\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/485.D_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/485.D_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/485.D_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/485.D_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.33 s (22 us/read; 2.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            287,354\n",
      "  Read 1 with adapter:                     163 (0.1%)\n",
      "  Read 2 with adapter:                  13,263 (4.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       287,354 (100.0%)\n",
      "\n",
      "Total basepairs processed:   130,961,739 bp\n",
      "  Read 1:    64,327,357 bp\n",
      "  Read 2:    66,634,382 bp\n",
      "Total written (filtered):    130,796,345 bp (99.9%)\n",
      "  Read 1:    64,325,208 bp\n",
      "  Read 2:    66,471,137 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 163 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t146\t4489.9\t0\t146\n",
      "19\t1\t0.0\t1\t1\n",
      "27\t5\t0.0\t1\t5\n",
      "101\t5\t0.0\t1\t5\n",
      "121\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t1\n",
      "162\t1\t0.0\t1\t0 1\n",
      "210\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13263 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t113\t4489.9\t0\t113\n",
      "4\t13\t1122.5\t0\t13\n",
      "5\t37\t280.6\t0\t37\n",
      "6\t80\t70.2\t0\t80\n",
      "7\t534\t17.5\t0\t534\n",
      "8\t612\t4.4\t0\t612\n",
      "9\t2885\t1.1\t0\t2878 7\n",
      "10\t4023\t0.3\t1\t4007 16\n",
      "11\t168\t0.1\t1\t167 1\n",
      "12\t239\t0.0\t1\t234 5\n",
      "13\t200\t0.0\t1\t198 2\n",
      "14\t277\t0.0\t1\t271 6\n",
      "15\t330\t0.0\t1\t324 6\n",
      "16\t330\t0.0\t1\t322 8\n",
      "17\t711\t0.0\t1\t681 30\n",
      "18\t821\t0.0\t1\t769 30 22\n",
      "19\t1626\t0.0\t1\t1067 554 5\n",
      "20\t39\t0.0\t2\t0 1 38\n",
      "21\t208\t0.0\t2\t0 208\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t1\n",
      "184\t2\t0.0\t2\t1 1\n",
      "211\t1\t0.0\t2\t1\n",
      "215\t2\t0.0\t2\t2\n",
      "216\t8\t0.0\t2\t8\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/485.U_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/485.U_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/485.U_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/485.U_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.94 s (21 us/read; 2.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            234,954\n",
      "  Read 1 with adapter:                     171 (0.1%)\n",
      "  Read 2 with adapter:                  10,311 (4.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       234,954 (100.0%)\n",
      "\n",
      "Total basepairs processed:   107,083,077 bp\n",
      "  Read 1:    52,608,183 bp\n",
      "  Read 2:    54,474,894 bp\n",
      "Total written (filtered):    106,953,414 bp (99.9%)\n",
      "  Read 1:    52,606,652 bp\n",
      "  Read 2:    54,346,762 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 171 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t162\t3671.2\t0\t162\n",
      "27\t1\t0.0\t1\t1\n",
      "120\t7\t0.0\t1\t7\n",
      "178\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10311 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t3671.2\t0\t58\n",
      "4\t18\t917.8\t0\t18\n",
      "5\t21\t229.4\t0\t21\n",
      "6\t74\t57.4\t0\t74\n",
      "7\t394\t14.3\t0\t394\n",
      "8\t556\t3.6\t0\t556\n",
      "9\t2060\t0.9\t0\t2059 1\n",
      "10\t2995\t0.2\t1\t2984 11\n",
      "11\t131\t0.1\t1\t129 2\n",
      "12\t161\t0.0\t1\t161\n",
      "13\t158\t0.0\t1\t155 3\n",
      "14\t228\t0.0\t1\t219 9\n",
      "15\t269\t0.0\t1\t260 9\n",
      "16\t361\t0.0\t1\t343 18\n",
      "17\t676\t0.0\t1\t660 16\n",
      "18\t684\t0.0\t1\t655 14 15\n",
      "19\t1255\t0.0\t1\t805 449 1\n",
      "20\t30\t0.0\t2\t0 0 30\n",
      "21\t172\t0.0\t2\t0 169 3\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t1\n",
      "177\t1\t0.0\t2\t1\n",
      "210\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "223\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/486.D_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/486.D_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/486.D_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/486.D_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.93 s (19 us/read; 3.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            355,519\n",
      "  Read 1 with adapter:                     196 (0.1%)\n",
      "  Read 2 with adapter:                  11,669 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       355,519 (100.0%)\n",
      "\n",
      "Total basepairs processed:   161,954,775 bp\n",
      "  Read 1:    79,607,137 bp\n",
      "  Read 2:    82,347,638 bp\n",
      "Total written (filtered):    161,804,461 bp (99.9%)\n",
      "  Read 1:    79,605,490 bp\n",
      "  Read 2:    82,198,971 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 196 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t175\t5555.0\t0\t175\n",
      "21\t2\t0.0\t1\t2\n",
      "24\t3\t0.0\t1\t3\n",
      "25\t8\t0.0\t1\t8\n",
      "27\t4\t0.0\t1\t4\n",
      "92\t1\t0.0\t1\t1\n",
      "189\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11669 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t74\t5555.0\t0\t74\n",
      "4\t13\t1388.7\t0\t13\n",
      "5\t29\t347.2\t0\t29\n",
      "6\t58\t86.8\t0\t58\n",
      "7\t390\t21.7\t0\t390\n",
      "8\t542\t5.4\t0\t542\n",
      "9\t2227\t1.4\t0\t2224 3\n",
      "10\t3413\t0.3\t1\t3399 14\n",
      "11\t165\t0.1\t1\t162 3\n",
      "12\t180\t0.0\t1\t180\n",
      "13\t99\t0.0\t1\t97 2\n",
      "14\t245\t0.0\t1\t237 8\n",
      "15\t280\t0.0\t1\t275 5\n",
      "16\t347\t0.0\t1\t333 14\n",
      "17\t761\t0.0\t1\t748 13\n",
      "18\t876\t0.0\t1\t834 29 13\n",
      "19\t1697\t0.0\t1\t1144 551 2\n",
      "20\t46\t0.0\t2\t0 2 44\n",
      "21\t208\t0.0\t2\t0 207 1\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "114\t1\t0.0\t2\t1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 1\n",
      "154\t1\t0.0\t2\t1\n",
      "170\t1\t0.0\t2\t0 1\n",
      "214\t1\t0.0\t2\t0 1\n",
      "216\t3\t0.0\t2\t1 1 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/486.U_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/486.U_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/486.U_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/486.U_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.66 s (20 us/read; 2.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            230,403\n",
      "  Read 1 with adapter:                     142 (0.1%)\n",
      "  Read 2 with adapter:                   9,690 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       230,403 (100.0%)\n",
      "\n",
      "Total basepairs processed:   104,991,509 bp\n",
      "  Read 1:    51,583,671 bp\n",
      "  Read 2:    53,407,838 bp\n",
      "Total written (filtered):    104,869,654 bp (99.9%)\n",
      "  Read 1:    51,582,490 bp\n",
      "  Read 2:    53,287,164 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 142 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t127\t3600.0\t0\t127\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t5\t0.0\t1\t5\n",
      "38\t2\t0.0\t1\t2\n",
      "100\t1\t0.0\t1\t1\n",
      "101\t2\t0.0\t1\t2\n",
      "106\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9690 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t3600.0\t0\t61\n",
      "4\t9\t900.0\t0\t9\n",
      "5\t35\t225.0\t0\t35\n",
      "6\t74\t56.3\t0\t74\n",
      "7\t293\t14.1\t0\t293\n",
      "8\t500\t3.5\t0\t500\n",
      "9\t2115\t0.9\t0\t2113 2\n",
      "10\t2604\t0.2\t1\t2594 10\n",
      "11\t149\t0.1\t1\t146 3\n",
      "12\t154\t0.0\t1\t152 2\n",
      "13\t169\t0.0\t1\t167 2\n",
      "14\t226\t0.0\t1\t218 8\n",
      "15\t229\t0.0\t1\t227 2\n",
      "16\t353\t0.0\t1\t341 12\n",
      "17\t635\t0.0\t1\t615 20\n",
      "18\t678\t0.0\t1\t635 23 20\n",
      "19\t1233\t0.0\t1\t806 424 3\n",
      "20\t32\t0.0\t2\t0 5 27\n",
      "21\t135\t0.0\t2\t0 134 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "186\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t3\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/487.D_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/487.D_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/487.D_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/487.D_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.93 s (18 us/read; 3.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            277,120\n",
      "  Read 1 with adapter:                     160 (0.1%)\n",
      "  Read 2 with adapter:                  12,819 (4.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       277,120 (100.0%)\n",
      "\n",
      "Total basepairs processed:   126,323,696 bp\n",
      "  Read 1:    62,057,301 bp\n",
      "  Read 2:    64,266,395 bp\n",
      "Total written (filtered):    126,171,181 bp (99.9%)\n",
      "  Read 1:    62,054,949 bp\n",
      "  Read 2:    64,116,232 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 160 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t136\t4330.0\t0\t136\n",
      "25\t5\t0.0\t1\t5\n",
      "27\t3\t0.0\t1\t3\n",
      "31\t1\t0.0\t1\t1\n",
      "101\t9\t0.0\t1\t9\n",
      "103\t1\t0.0\t1\t1\n",
      "120\t4\t0.0\t1\t4\n",
      "215\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12819 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t92\t4330.0\t0\t92\n",
      "4\t19\t1082.5\t0\t19\n",
      "5\t48\t270.6\t0\t48\n",
      "6\t127\t67.7\t0\t127\n",
      "7\t589\t16.9\t0\t589\n",
      "8\t829\t4.2\t0\t829\n",
      "9\t3123\t1.1\t0\t3118 5\n",
      "10\t3915\t0.3\t1\t3899 16\n",
      "11\t180\t0.1\t1\t174 6\n",
      "12\t129\t0.0\t1\t129\n",
      "13\t98\t0.0\t1\t96 2\n",
      "14\t182\t0.0\t1\t171 11\n",
      "15\t232\t0.0\t1\t226 6\n",
      "16\t272\t0.0\t1\t261 11\n",
      "17\t670\t0.0\t1\t643 27\n",
      "18\t750\t0.0\t1\t699 38 13\n",
      "19\t1332\t0.0\t1\t858 472 2\n",
      "20\t40\t0.0\t2\t0 3 37\n",
      "21\t188\t0.0\t2\t0 188\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "214\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/487.U_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/487.U_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/487.U_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/487.U_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.14 s (18 us/read; 3.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            345,098\n",
      "  Read 1 with adapter:                     240 (0.1%)\n",
      "  Read 2 with adapter:                  10,575 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       345,098 (100.0%)\n",
      "\n",
      "Total basepairs processed:   157,202,404 bp\n",
      "  Read 1:    77,280,174 bp\n",
      "  Read 2:    79,922,230 bp\n",
      "Total written (filtered):    157,063,466 bp (99.9%)\n",
      "  Read 1:    77,278,534 bp\n",
      "  Read 2:    79,784,932 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 240 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t225\t5392.2\t0\t225\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t8\t0.0\t1\t7 1\n",
      "38\t1\t0.0\t1\t1\n",
      "101\t2\t0.0\t1\t2\n",
      "120\t1\t0.0\t1\t1\n",
      "157\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10575 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t5392.2\t0\t54\n",
      "4\t20\t1348.0\t0\t20\n",
      "5\t31\t337.0\t0\t31\n",
      "6\t72\t84.3\t0\t72\n",
      "7\t348\t21.1\t0\t348\n",
      "8\t433\t5.3\t0\t433\n",
      "9\t1927\t1.3\t0\t1922 5\n",
      "10\t2980\t0.3\t1\t2965 15\n",
      "11\t196\t0.1\t1\t196\n",
      "12\t161\t0.0\t1\t160 1\n",
      "13\t119\t0.0\t1\t117 2\n",
      "14\t209\t0.0\t1\t202 7\n",
      "15\t237\t0.0\t1\t234 3\n",
      "16\t328\t0.0\t1\t313 15\n",
      "17\t689\t0.0\t1\t664 25\n",
      "18\t844\t0.0\t1\t797 34 13\n",
      "19\t1646\t0.0\t1\t1141 502 3\n",
      "20\t51\t0.0\t2\t2 0 49\n",
      "21\t217\t0.0\t2\t0 216 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t1\n",
      "199\t1\t0.0\t2\t1\n",
      "215\t2\t0.0\t2\t2\n",
      "216\t4\t0.0\t2\t4\n",
      "217\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/488.D_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/488.D_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/488.D_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/488.D_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.62 s (18 us/read; 3.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            312,332\n",
      "  Read 1 with adapter:                     315 (0.1%)\n",
      "  Read 2 with adapter:                  12,388 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       312,332 (100.0%)\n",
      "\n",
      "Total basepairs processed:   142,334,261 bp\n",
      "  Read 1:    69,944,619 bp\n",
      "  Read 2:    72,389,642 bp\n",
      "Total written (filtered):    142,182,398 bp (99.9%)\n",
      "  Read 1:    69,942,443 bp\n",
      "  Read 2:    72,239,955 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 315 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t300\t4880.2\t0\t300\n",
      "25\t5\t0.0\t1\t5\n",
      "27\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t1\n",
      "92\t2\t0.0\t1\t2\n",
      "95\t1\t0.0\t1\t0 1\n",
      "122\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "139\t1\t0.0\t1\t1\n",
      "186\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12388 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t86\t4880.2\t0\t86\n",
      "4\t19\t1220.0\t0\t19\n",
      "5\t38\t305.0\t0\t38\n",
      "6\t90\t76.3\t0\t90\n",
      "7\t561\t19.1\t0\t561\n",
      "8\t738\t4.8\t0\t738\n",
      "9\t2838\t1.2\t0\t2831 7\n",
      "10\t3674\t0.3\t1\t3655 19\n",
      "11\t154\t0.1\t1\t152 2\n",
      "12\t133\t0.0\t1\t132 1\n",
      "13\t96\t0.0\t1\t95 1\n",
      "14\t157\t0.0\t1\t153 4\n",
      "15\t227\t0.0\t1\t221 6\n",
      "16\t301\t0.0\t1\t290 11\n",
      "17\t728\t0.0\t1\t710 18\n",
      "18\t777\t0.0\t1\t743 21 13\n",
      "19\t1539\t0.0\t1\t1064 471 4\n",
      "20\t40\t0.0\t2\t0 2 38\n",
      "21\t183\t0.0\t2\t0 183\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "90\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t2 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/488.U_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/488.U_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/488.U_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/488.U_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.83 s (18 us/read; 3.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            388,560\n",
      "  Read 1 with adapter:                     206 (0.1%)\n",
      "  Read 2 with adapter:                  18,391 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       388,560 (100.0%)\n",
      "\n",
      "Total basepairs processed:   177,128,024 bp\n",
      "  Read 1:    87,012,510 bp\n",
      "  Read 2:    90,115,514 bp\n",
      "Total written (filtered):    176,904,974 bp (99.9%)\n",
      "  Read 1:    87,010,684 bp\n",
      "  Read 2:    89,894,290 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 206 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t194\t6071.2\t0\t194\n",
      "27\t4\t0.0\t1\t4\n",
      "55\t2\t0.0\t1\t2\n",
      "111\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "159\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18391 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t164\t6071.2\t0\t164\n",
      "4\t14\t1517.8\t0\t14\n",
      "5\t64\t379.5\t0\t64\n",
      "6\t143\t94.9\t0\t143\n",
      "7\t818\t23.7\t0\t818\n",
      "8\t949\t5.9\t0\t949\n",
      "9\t4340\t1.5\t0\t4333 7\n",
      "10\t5678\t0.4\t1\t5651 27\n",
      "11\t263\t0.1\t1\t260 3\n",
      "12\t266\t0.0\t1\t265 1\n",
      "13\t213\t0.0\t1\t210 3\n",
      "14\t306\t0.0\t1\t299 7\n",
      "15\t377\t0.0\t1\t365 12\n",
      "16\t402\t0.0\t1\t384 18\n",
      "17\t910\t0.0\t1\t871 39\n",
      "18\t1111\t0.0\t1\t1066 28 17\n",
      "19\t2075\t0.0\t1\t1385 683 7\n",
      "20\t32\t0.0\t2\t0 2 30\n",
      "21\t242\t0.0\t2\t0 242\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 1\n",
      "147\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "214\t3\t0.0\t2\t3\n",
      "215\t5\t0.0\t2\t4 1\n",
      "216\t6\t0.0\t2\t5 1\n",
      "217\t3\t0.0\t2\t3\n",
      "236\t1\t0.0\t2\t1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/480.D_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/480.D_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/480.D_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/480.D_14_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/480.U_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/480.U_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/480.U_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/480.U_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/481.H_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/481.H_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/481.H_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/481.H_16_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/482.H_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/482.H_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/482.H_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/482.H_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/483.H_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/483.H_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/483.H_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/483.H_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/484.D_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/484.D_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/484.D_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/484.D_6_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/484.U_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/484.U_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/484.U_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/484.U_2_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/485.D_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/485.D_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/485.D_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/485.D_32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/485.U_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/485.U_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/485.U_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/485.U_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/486.D_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/486.D_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/486.D_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/486.D_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/486.U_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/486.U_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/486.U_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/486.U_1_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/487.D_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/487.D_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/487.D_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/487.D_12_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/487.U_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/487.U_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/487.U_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/487.U_21_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/488.D_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/488.D_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/488.D_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/488.D_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/488.U_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/488.U_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/488.U_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/488.U_18_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/489.H_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/489.H_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/489.H_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/489.H_29_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/489.H_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/489.H_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/489.H_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/489.H_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.59 s (19 us/read; 3.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            342,659\n",
      "  Read 1 with adapter:                     211 (0.1%)\n",
      "  Read 2 with adapter:                  13,056 (3.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       342,659 (100.0%)\n",
      "\n",
      "Total basepairs processed:   156,121,467 bp\n",
      "  Read 1:    76,719,959 bp\n",
      "  Read 2:    79,401,508 bp\n",
      "Total written (filtered):    155,955,617 bp (99.9%)\n",
      "  Read 1:    76,718,323 bp\n",
      "  Read 2:    79,237,294 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 211 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t194\t5354.0\t0\t194\n",
      "13\t1\t0.0\t1\t1\n",
      "19\t4\t0.0\t1\t4\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t3\t0.0\t1\t3\n",
      "120\t4\t0.0\t1\t4\n",
      "150\t1\t0.0\t1\t1\n",
      "179\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13056 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t95\t5354.0\t0\t95\n",
      "4\t9\t1338.5\t0\t9\n",
      "5\t46\t334.6\t0\t46\n",
      "6\t68\t83.7\t0\t68\n",
      "7\t452\t20.9\t0\t452\n",
      "8\t565\t5.2\t0\t565\n",
      "9\t2651\t1.3\t0\t2650 1\n",
      "10\t3860\t0.3\t1\t3840 20\n",
      "11\t222\t0.1\t1\t222\n",
      "12\t236\t0.0\t1\t230 6\n",
      "13\t235\t0.0\t1\t234 1\n",
      "14\t332\t0.0\t1\t328 4\n",
      "15\t318\t0.0\t1\t312 6\n",
      "16\t344\t0.0\t1\t332 12\n",
      "17\t782\t0.0\t1\t755 27\n",
      "18\t899\t0.0\t1\t844 37 18\n",
      "19\t1667\t0.0\t1\t1100 564 3\n",
      "20\t33\t0.0\t2\t0 0 33\n",
      "21\t223\t0.0\t2\t0 220 3\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "115\t2\t0.0\t2\t1 0 1\n",
      "126\t1\t0.0\t2\t1\n",
      "143\t1\t0.0\t2\t0 1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "205\t1\t0.0\t2\t1\n",
      "216\t9\t0.0\t2\t9\n",
      "217\t1\t0.0\t2\t1\n",
      "232\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/490.H_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/490.H_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/490.H_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/490.H_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.23 s (17 us/read; 3.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            311,474\n",
      "  Read 1 with adapter:                     217 (0.1%)\n",
      "  Read 2 with adapter:                  14,943 (4.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       311,474 (100.0%)\n",
      "\n",
      "Total basepairs processed:   141,988,148 bp\n",
      "  Read 1:    69,746,235 bp\n",
      "  Read 2:    72,241,913 bp\n",
      "Total written (filtered):    141,808,657 bp (99.9%)\n",
      "  Read 1:    69,744,100 bp\n",
      "  Read 2:    72,064,557 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 217 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t199\t4866.8\t0\t199\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t8\t0.0\t1\t8\n",
      "127\t5\t0.0\t1\t5\n",
      "206\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14943 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t140\t4866.8\t0\t140\n",
      "4\t28\t1216.7\t0\t28\n",
      "5\t43\t304.2\t0\t43\n",
      "6\t130\t76.0\t0\t130\n",
      "7\t613\t19.0\t0\t613\n",
      "8\t899\t4.8\t0\t899\n",
      "9\t3515\t1.2\t0\t3514 1\n",
      "10\t4783\t0.3\t1\t4759 24\n",
      "11\t160\t0.1\t1\t159 1\n",
      "12\t150\t0.0\t1\t147 3\n",
      "13\t124\t0.0\t1\t123 1\n",
      "14\t250\t0.0\t1\t244 6\n",
      "15\t296\t0.0\t1\t289 7\n",
      "16\t336\t0.0\t1\t323 13\n",
      "17\t832\t0.0\t1\t796 36\n",
      "18\t845\t0.0\t1\t796 27 22\n",
      "19\t1579\t0.0\t1\t1031 545 3\n",
      "20\t35\t0.0\t2\t1 1 33\n",
      "21\t170\t0.0\t2\t0 169 1\n",
      "61\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t1\n",
      "153\t3\t0.0\t2\t1 2\n",
      "162\t1\t0.0\t2\t1\n",
      "216\t7\t0.0\t2\t5 2\n",
      "228\t1\t0.0\t2\t0 0 1\n",
      "229\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/491.H_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/491.H_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/491.H_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/491.H_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.99 s (18 us/read; 3.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            272,807\n",
      "  Read 1 with adapter:                     168 (0.1%)\n",
      "  Read 2 with adapter:                  10,536 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       272,807 (100.0%)\n",
      "\n",
      "Total basepairs processed:   124,310,026 bp\n",
      "  Read 1:    61,089,026 bp\n",
      "  Read 2:    63,221,000 bp\n",
      "Total written (filtered):    124,176,866 bp (99.9%)\n",
      "  Read 1:    61,088,294 bp\n",
      "  Read 2:    63,088,572 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 168 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t161\t4262.6\t0\t161\n",
      "10\t1\t0.3\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t3\t0.0\t1\t3\n",
      "108\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10536 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t65\t4262.6\t0\t65\n",
      "4\t13\t1065.7\t0\t13\n",
      "5\t32\t266.4\t0\t32\n",
      "6\t97\t66.6\t0\t97\n",
      "7\t348\t16.7\t0\t348\n",
      "8\t528\t4.2\t0\t528\n",
      "9\t2186\t1.0\t0\t2183 3\n",
      "10\t3124\t0.3\t1\t3111 13\n",
      "11\t170\t0.1\t1\t166 4\n",
      "12\t162\t0.0\t1\t159 3\n",
      "13\t127\t0.0\t1\t125 2\n",
      "14\t200\t0.0\t1\t193 7\n",
      "15\t221\t0.0\t1\t217 4\n",
      "16\t302\t0.0\t1\t288 14\n",
      "17\t700\t0.0\t1\t676 24\n",
      "18\t674\t0.0\t1\t639 24 11\n",
      "19\t1381\t0.0\t1\t938 441 2\n",
      "20\t29\t0.0\t2\t0 1 28\n",
      "21\t157\t0.0\t2\t0 156 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 1\n",
      "184\t1\t0.0\t2\t1\n",
      "211\t1\t0.0\t2\t0 1\n",
      "215\t3\t0.0\t2\t3\n",
      "216\t9\t0.0\t2\t7 2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/492.D_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/492.D_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/492.D_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/492.D_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.50 s (16 us/read; 3.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            215,897\n",
      "  Read 1 with adapter:                     210 (0.1%)\n",
      "  Read 2 with adapter:                  10,138 (4.7%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       215,896 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,272,519 bp\n",
      "  Read 1:    48,257,590 bp\n",
      "  Read 2:    50,014,929 bp\n",
      "Total written (filtered):     98,142,981 bp (99.9%)\n",
      "  Read 1:    48,247,268 bp\n",
      "  Read 2:    49,895,713 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 210 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t117\t3373.4\t0\t117\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "49\t1\t0.0\t1\t1\n",
      "94\t4\t0.0\t1\t4\n",
      "99\t1\t0.0\t1\t0 1\n",
      "100\t57\t0.0\t1\t51 6\n",
      "101\t3\t0.0\t1\t3\n",
      "102\t8\t0.0\t1\t8\n",
      "103\t4\t0.0\t1\t4\n",
      "120\t1\t0.0\t1\t1\n",
      "126\t3\t0.0\t1\t1 2\n",
      "127\t1\t0.0\t1\t1\n",
      "214\t2\t0.0\t1\t0 2\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t3\n",
      "223\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10138 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t82\t3373.4\t0\t82\n",
      "4\t12\t843.3\t0\t12\n",
      "5\t39\t210.8\t0\t39\n",
      "6\t92\t52.7\t0\t92\n",
      "7\t488\t13.2\t0\t488\n",
      "8\t585\t3.3\t0\t585\n",
      "9\t2422\t0.8\t0\t2420 2\n",
      "10\t3173\t0.2\t1\t3160 13\n",
      "11\t119\t0.1\t1\t118 1\n",
      "12\t105\t0.0\t1\t104 1\n",
      "13\t94\t0.0\t1\t91 3\n",
      "14\t160\t0.0\t1\t158 2\n",
      "15\t189\t0.0\t1\t186 3\n",
      "16\t252\t0.0\t1\t250 2\n",
      "17\t536\t0.0\t1\t515 21\n",
      "18\t561\t0.0\t1\t532 19 10\n",
      "19\t1037\t0.0\t1\t707 327 3\n",
      "20\t26\t0.0\t2\t0 0 26\n",
      "21\t160\t0.0\t2\t0 160\n",
      "22\t1\t0.0\t2\t0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "28\t1\t0.0\t2\t0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "184\t1\t0.0\t2\t1\n",
      "196\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/492.U_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/492.U_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/492.U_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/492.U_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.84 s (21 us/read; 2.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             89,296\n",
      "  Read 1 with adapter:                      53 (0.1%)\n",
      "  Read 2 with adapter:                   3,710 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        89,296 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,306,890 bp\n",
      "  Read 1:    19,780,845 bp\n",
      "  Read 2:    20,526,045 bp\n",
      "Total written (filtered):     40,256,308 bp (99.9%)\n",
      "  Read 1:    19,776,369 bp\n",
      "  Read 2:    20,479,939 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 53 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1395.2\t0\t29\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "196\t1\t0.0\t1\t0 1\n",
      "208\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t1\n",
      "215\t2\t0.0\t1\t2\n",
      "216\t15\t0.0\t1\t15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3710 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t1395.2\t0\t39\n",
      "4\t4\t348.8\t0\t4\n",
      "5\t11\t87.2\t0\t11\n",
      "6\t32\t21.8\t0\t32\n",
      "7\t133\t5.5\t0\t133\n",
      "8\t197\t1.4\t0\t197\n",
      "9\t782\t0.3\t0\t781 1\n",
      "10\t1094\t0.1\t1\t1092 2\n",
      "11\t56\t0.0\t1\t56\n",
      "12\t44\t0.0\t1\t43 1\n",
      "13\t42\t0.0\t1\t41 1\n",
      "14\t63\t0.0\t1\t61 2\n",
      "15\t59\t0.0\t1\t58 1\n",
      "16\t76\t0.0\t1\t75 1\n",
      "17\t232\t0.0\t1\t221 11\n",
      "18\t277\t0.0\t1\t259 14 4\n",
      "19\t492\t0.0\t1\t313 179\n",
      "20\t16\t0.0\t2\t0 0 16\n",
      "21\t55\t0.0\t2\t0 53 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "136\t1\t0.0\t2\t1\n",
      "203\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/493.D_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/493.D_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/493.D_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/493.D_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.68 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,014\n",
      "  Read 1 with adapter:                      46 (0.1%)\n",
      "  Read 2 with adapter:                   2,931 (4.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        65,014 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,359,656 bp\n",
      "  Read 1:    14,409,736 bp\n",
      "  Read 2:    14,949,920 bp\n",
      "Total written (filtered):     29,322,074 bp (99.9%)\n",
      "  Read 1:    14,408,048 bp\n",
      "  Read 2:    14,914,026 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 46 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t1015.8\t0\t36\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t1 1\n",
      "209\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "215\t2\t0.0\t1\t2\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2931 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t1015.8\t0\t24\n",
      "4\t2\t254.0\t0\t2\n",
      "5\t6\t63.5\t0\t6\n",
      "6\t19\t15.9\t0\t19\n",
      "7\t104\t4.0\t0\t104\n",
      "8\t180\t1.0\t0\t180\n",
      "9\t640\t0.2\t0\t640\n",
      "10\t859\t0.1\t1\t858 1\n",
      "11\t34\t0.0\t1\t34\n",
      "12\t43\t0.0\t1\t41 2\n",
      "13\t41\t0.0\t1\t41\n",
      "14\t71\t0.0\t1\t68 3\n",
      "15\t60\t0.0\t1\t58 2\n",
      "16\t89\t0.0\t1\t87 2\n",
      "17\t168\t0.0\t1\t157 11\n",
      "18\t176\t0.0\t1\t163 8 5\n",
      "19\t353\t0.0\t1\t232 118 3\n",
      "20\t5\t0.0\t2\t1 0 4\n",
      "21\t55\t0.0\t2\t0 54 1\n",
      "209\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/493.U_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/493.U_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/493.U_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/493.U_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.54 s (22 us/read; 2.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,929\n",
      "  Read 1 with adapter:                      99 (0.1%)\n",
      "  Read 2 with adapter:                   4,851 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,929 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,540,875 bp\n",
      "  Read 1:    25,802,296 bp\n",
      "  Read 2:    26,738,579 bp\n",
      "Total written (filtered):     52,475,953 bp (99.9%)\n",
      "  Read 1:    25,797,370 bp\n",
      "  Read 2:    26,678,583 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t1811.4\t0\t73\n",
      "13\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "36\t1\t0.0\t1\t1\n",
      "95\t1\t0.0\t1\t0 1\n",
      "196\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "215\t4\t0.0\t1\t4\n",
      "216\t15\t0.0\t1\t13 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4851 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t1811.4\t0\t34\n",
      "4\t6\t452.8\t0\t6\n",
      "5\t10\t113.2\t0\t10\n",
      "6\t34\t28.3\t0\t34\n",
      "7\t198\t7.1\t0\t198\n",
      "8\t242\t1.8\t0\t242\n",
      "9\t1116\t0.4\t0\t1114 2\n",
      "10\t1433\t0.1\t1\t1425 8\n",
      "11\t71\t0.0\t1\t70 1\n",
      "12\t68\t0.0\t1\t67 1\n",
      "13\t30\t0.0\t1\t29 1\n",
      "14\t74\t0.0\t1\t74\n",
      "15\t90\t0.0\t1\t88 2\n",
      "16\t133\t0.0\t1\t130 3\n",
      "17\t267\t0.0\t1\t263 4\n",
      "18\t306\t0.0\t1\t288 11 7\n",
      "19\t649\t0.0\t1\t444 205\n",
      "20\t19\t0.0\t2\t0 0 19\n",
      "21\t62\t0.0\t2\t0 62\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "129\t1\t0.0\t2\t1\n",
      "146\t1\t0.0\t2\t1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "216\t3\t0.0\t2\t2 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/494.D_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/494.D_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/494.D_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/494.D_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.34 s (26 us/read; 2.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             88,478\n",
      "  Read 1 with adapter:                      54 (0.1%)\n",
      "  Read 2 with adapter:                   4,255 (4.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        88,478 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,199,312 bp\n",
      "  Read 1:    19,736,682 bp\n",
      "  Read 2:    20,462,630 bp\n",
      "Total written (filtered):     40,145,166 bp (99.9%)\n",
      "  Read 1:    19,735,354 bp\n",
      "  Read 2:    20,409,812 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 54 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t1382.5\t0\t44\n",
      "20\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t1\t0.0\t1\t0 1\n",
      "212\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4255 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1382.5\t0\t29\n",
      "4\t3\t345.6\t0\t3\n",
      "5\t18\t86.4\t0\t18\n",
      "6\t48\t21.6\t0\t48\n",
      "7\t199\t5.4\t0\t199\n",
      "8\t255\t1.4\t0\t255\n",
      "9\t966\t0.3\t0\t965 1\n",
      "10\t1208\t0.1\t1\t1200 8\n",
      "11\t45\t0.0\t1\t45\n",
      "12\t50\t0.0\t1\t49 1\n",
      "13\t55\t0.0\t1\t54 1\n",
      "14\t92\t0.0\t1\t90 2\n",
      "15\t108\t0.0\t1\t105 3\n",
      "16\t105\t0.0\t1\t98 7\n",
      "17\t256\t0.0\t1\t243 13\n",
      "18\t262\t0.0\t1\t245 9 8\n",
      "19\t482\t0.0\t1\t292 189 1\n",
      "20\t9\t0.0\t2\t1 1 7\n",
      "21\t53\t0.0\t2\t0 52 1\n",
      "61\t2\t0.0\t2\t2\n",
      "168\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "195\t1\t0.0\t2\t1\n",
      "207\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "217\t2\t0.0\t2\t2\n",
      "241\t1\t0.0\t2\t1\n",
      "244\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/494.U_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/494.U_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/494.U_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/494.U_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.67 s (26 us/read; 2.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,213\n",
      "  Read 1 with adapter:                      52 (0.1%)\n",
      "  Read 2 with adapter:                   2,689 (4.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        63,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,579,086 bp\n",
      "  Read 1:    14,031,354 bp\n",
      "  Read 2:    14,547,732 bp\n",
      "Total written (filtered):     28,543,310 bp (99.9%)\n",
      "  Read 1:    14,029,147 bp\n",
      "  Read 2:    14,514,163 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 52 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t41\t987.7\t0\t41\n",
      "27\t1\t0.0\t1\t1\n",
      "117\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "216\t7\t0.0\t1\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2689 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t987.7\t0\t18\n",
      "4\t1\t246.9\t0\t1\n",
      "5\t9\t61.7\t0\t9\n",
      "6\t30\t15.4\t0\t30\n",
      "7\t138\t3.9\t0\t138\n",
      "8\t142\t1.0\t0\t142\n",
      "9\t564\t0.2\t0\t563 1\n",
      "10\t785\t0.1\t1\t783 2\n",
      "11\t29\t0.0\t1\t27 2\n",
      "12\t34\t0.0\t1\t33 1\n",
      "13\t27\t0.0\t1\t27\n",
      "14\t58\t0.0\t1\t55 3\n",
      "15\t61\t0.0\t1\t61\n",
      "16\t78\t0.0\t1\t77 1\n",
      "17\t155\t0.0\t1\t152 3\n",
      "18\t162\t0.0\t1\t152 6 4\n",
      "19\t336\t0.0\t1\t229 106 1\n",
      "20\t8\t0.0\t2\t0 1 7\n",
      "21\t49\t0.0\t2\t0 49\n",
      "216\t4\t0.0\t2\t4\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/495.D_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/495.D_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/495.D_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/495.D_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.08 s (16 us/read; 3.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            261,247\n",
      "  Read 1 with adapter:                     170 (0.1%)\n",
      "  Read 2 with adapter:                  16,717 (6.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       261,247 (100.0%)\n",
      "\n",
      "Total basepairs processed:   119,162,923 bp\n",
      "  Read 1:    58,488,984 bp\n",
      "  Read 2:    60,673,939 bp\n",
      "Total written (filtered):    118,978,109 bp (99.8%)\n",
      "  Read 1:    58,485,923 bp\n",
      "  Read 2:    60,492,186 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 170 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t139\t4082.0\t0\t139\n",
      "22\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t6\t0.0\t1\t6\n",
      "32\t1\t0.0\t1\t1\n",
      "100\t8\t0.0\t1\t7 1\n",
      "101\t9\t0.0\t1\t7 2\n",
      "104\t1\t0.0\t1\t1\n",
      "164\t1\t0.0\t1\t1\n",
      "184\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 16717 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t149\t4082.0\t0\t149\n",
      "4\t24\t1020.5\t0\t24\n",
      "5\t75\t255.1\t0\t75\n",
      "6\t229\t63.8\t0\t229\n",
      "7\t1125\t15.9\t0\t1125\n",
      "8\t1346\t4.0\t0\t1346\n",
      "9\t4515\t1.0\t0\t4508 7\n",
      "10\t5565\t0.2\t1\t5543 22\n",
      "11\t140\t0.1\t1\t139 1\n",
      "12\t110\t0.0\t1\t108 2\n",
      "13\t72\t0.0\t1\t72\n",
      "14\t144\t0.0\t1\t140 4\n",
      "15\t216\t0.0\t1\t215 1\n",
      "16\t261\t0.0\t1\t252 9\n",
      "17\t631\t0.0\t1\t612 19\n",
      "18\t684\t0.0\t1\t652 24 8\n",
      "19\t1247\t0.0\t1\t853 392 2\n",
      "20\t35\t0.0\t2\t0 3 32\n",
      "21\t144\t0.0\t2\t0 143 1\n",
      "83\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t1\n",
      "203\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/495.U_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/495.U_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/495.U_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/495.U_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.66 s (20 us/read; 3.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            135,341\n",
      "  Read 1 with adapter:                      45 (0.0%)\n",
      "  Read 2 with adapter:                   3,073 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       135,341 (100.0%)\n",
      "\n",
      "Total basepairs processed:    61,521,687 bp\n",
      "  Read 1:    30,249,483 bp\n",
      "  Read 2:    31,272,204 bp\n",
      "Total written (filtered):     61,478,793 bp (99.9%)\n",
      "  Read 1:    30,249,111 bp\n",
      "  Read 2:    31,229,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 45 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t2114.7\t0\t43\n",
      "27\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3073 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t2114.7\t0\t15\n",
      "4\t4\t528.7\t0\t4\n",
      "5\t8\t132.2\t0\t8\n",
      "6\t14\t33.0\t0\t14\n",
      "7\t56\t8.3\t0\t56\n",
      "8\t104\t2.1\t0\t104\n",
      "9\t424\t0.5\t0\t424\n",
      "10\t729\t0.1\t1\t726 3\n",
      "11\t73\t0.0\t1\t70 3\n",
      "12\t52\t0.0\t1\t52\n",
      "13\t41\t0.0\t1\t41\n",
      "14\t66\t0.0\t1\t64 2\n",
      "15\t88\t0.0\t1\t88\n",
      "16\t120\t0.0\t1\t116 4\n",
      "17\t271\t0.0\t1\t265 6\n",
      "18\t291\t0.0\t1\t281 8 2\n",
      "19\t601\t0.0\t1\t420 180 1\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t101\t0.0\t2\t0 99 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/496.D_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/496.D_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/496.D_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/496.D_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.51 s (14 us/read; 4.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            317,806\n",
      "  Read 1 with adapter:                     210 (0.1%)\n",
      "  Read 2 with adapter:                  17,888 (5.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       317,806 (100.0%)\n",
      "\n",
      "Total basepairs processed:   144,895,503 bp\n",
      "  Read 1:    71,149,329 bp\n",
      "  Read 2:    73,746,174 bp\n",
      "Total written (filtered):    144,690,939 bp (99.9%)\n",
      "  Read 1:    71,147,659 bp\n",
      "  Read 2:    73,543,280 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 210 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t194\t4965.7\t0\t194\n",
      "6\t1\t77.6\t0\t1\n",
      "21\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t6\t0.0\t1\t6\n",
      "32\t1\t0.0\t1\t1\n",
      "46\t1\t0.0\t1\t1\n",
      "92\t2\t0.0\t1\t2\n",
      "185\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 17888 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t137\t4965.7\t0\t137\n",
      "4\t23\t1241.4\t0\t23\n",
      "5\t75\t310.4\t0\t75\n",
      "6\t181\t77.6\t0\t181\n",
      "7\t1039\t19.4\t0\t1039\n",
      "8\t1167\t4.8\t0\t1167\n",
      "9\t4609\t1.2\t0\t4600 9\n",
      "10\t5886\t0.3\t1\t5862 24\n",
      "11\t190\t0.1\t1\t190\n",
      "12\t151\t0.0\t1\t146 5\n",
      "13\t118\t0.0\t1\t117 1\n",
      "14\t193\t0.0\t1\t192 1\n",
      "15\t279\t0.0\t1\t268 11\n",
      "16\t311\t0.0\t1\t304 7\n",
      "17\t757\t0.0\t1\t735 22\n",
      "18\t864\t0.0\t1\t809 36 19\n",
      "19\t1648\t0.0\t1\t1081 564 3\n",
      "20\t39\t0.0\t2\t0 1 38\n",
      "21\t208\t0.0\t2\t0 207 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "77\t1\t0.0\t2\t1\n",
      "146\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "216\t3\t0.0\t2\t3\n",
      "218\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "245\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/496.U_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/496.U_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/496.U_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/496.U_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.25 s (31 us/read; 1.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,384\n",
      "  Read 1 with adapter:                      19 (0.0%)\n",
      "  Read 2 with adapter:                   1,796 (4.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        40,384 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,172,924 bp\n",
      "  Read 1:     8,917,208 bp\n",
      "  Read 2:     9,255,716 bp\n",
      "Total written (filtered):     18,148,765 bp (99.9%)\n",
      "  Read 1:     8,915,659 bp\n",
      "  Read 2:     9,233,106 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t631.0\t0\t12\n",
      "216\t6\t0.0\t1\t6\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1796 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t631.0\t0\t10\n",
      "5\t5\t39.4\t0\t5\n",
      "6\t9\t9.9\t0\t9\n",
      "7\t55\t2.5\t0\t55\n",
      "8\t77\t0.6\t0\t77\n",
      "9\t382\t0.2\t0\t382\n",
      "10\t477\t0.0\t1\t475 2\n",
      "11\t22\t0.0\t1\t22\n",
      "12\t20\t0.0\t1\t19 1\n",
      "13\t31\t0.0\t1\t30 1\n",
      "14\t55\t0.0\t1\t55\n",
      "15\t58\t0.0\t1\t54 4\n",
      "16\t62\t0.0\t1\t60 2\n",
      "17\t138\t0.0\t1\t137 1\n",
      "18\t134\t0.0\t1\t126 3 5\n",
      "19\t232\t0.0\t1\t142 86 4\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t27\t0.0\t2\t0 27\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/497.H_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/497.H_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/497.H_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/497.H_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.24 s (34 us/read; 1.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,602\n",
      "  Read 1 with adapter:                      37 (0.1%)\n",
      "  Read 2 with adapter:                   1,778 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        36,602 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,312,133 bp\n",
      "  Read 1:     7,993,010 bp\n",
      "  Read 2:     8,319,123 bp\n",
      "Total written (filtered):     16,286,842 bp (99.8%)\n",
      "  Read 1:     7,989,541 bp\n",
      "  Read 2:     8,297,301 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 37 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t571.9\t0\t19\n",
      "27\t2\t0.0\t1\t2\n",
      "144\t1\t0.0\t1\t1\n",
      "206\t2\t0.0\t1\t2\n",
      "211\t1\t0.0\t1\t1\n",
      "215\t2\t0.0\t1\t1 1\n",
      "216\t9\t0.0\t1\t9\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1778 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t571.9\t0\t18\n",
      "4\t3\t143.0\t0\t3\n",
      "5\t5\t35.7\t0\t5\n",
      "6\t13\t8.9\t0\t13\n",
      "7\t55\t2.2\t0\t55\n",
      "8\t92\t0.6\t0\t92\n",
      "9\t389\t0.1\t0\t389\n",
      "10\t534\t0.0\t1\t531 3\n",
      "11\t19\t0.0\t1\t17 2\n",
      "12\t22\t0.0\t1\t22\n",
      "13\t18\t0.0\t1\t18\n",
      "14\t33\t0.0\t1\t32 1\n",
      "15\t41\t0.0\t1\t41\n",
      "16\t45\t0.0\t1\t44 1\n",
      "17\t111\t0.0\t1\t109 2\n",
      "18\t113\t0.0\t1\t110 1 2\n",
      "19\t236\t0.0\t1\t158 76 2\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t26\t0.0\t2\t0 26\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/498.H_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/498.H_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/498.H_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/498.H_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.76 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,921\n",
      "  Read 1 with adapter:                      47 (0.1%)\n",
      "  Read 2 with adapter:                   2,350 (3.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        65,921 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,659,436 bp\n",
      "  Read 1:    14,563,250 bp\n",
      "  Read 2:    15,096,186 bp\n",
      "Total written (filtered):     29,624,551 bp (99.9%)\n",
      "  Read 1:    14,559,172 bp\n",
      "  Read 2:    15,065,379 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 47 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t1030.0\t0\t26\n",
      "27\t2\t0.0\t1\t2\n",
      "63\t1\t0.0\t1\t1\n",
      "211\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t0 1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "216\t12\t0.0\t1\t12\n",
      "217\t1\t0.0\t1\t1\n",
      "221\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2350 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t1030.0\t0\t13\n",
      "4\t1\t257.5\t0\t1\n",
      "5\t9\t64.4\t0\t9\n",
      "6\t13\t16.1\t0\t13\n",
      "7\t68\t4.0\t0\t68\n",
      "8\t91\t1.0\t0\t91\n",
      "9\t391\t0.3\t0\t391\n",
      "10\t595\t0.1\t1\t590 5\n",
      "11\t37\t0.0\t1\t36 1\n",
      "12\t62\t0.0\t1\t61 1\n",
      "13\t51\t0.0\t1\t51\n",
      "14\t92\t0.0\t1\t90 2\n",
      "15\t57\t0.0\t1\t54 3\n",
      "16\t67\t0.0\t1\t66 1\n",
      "17\t163\t0.0\t1\t155 8\n",
      "18\t194\t0.0\t1\t190 3 1\n",
      "19\t390\t0.0\t1\t266 122 2\n",
      "20\t8\t0.0\t2\t0 0 8\n",
      "21\t47\t0.0\t2\t0 46 1\n",
      "119\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/499.H_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/499.H_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/499.H_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/499.H_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.03 s (38 us/read; 1.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,001\n",
      "  Read 1 with adapter:                      29 (0.1%)\n",
      "  Read 2 with adapter:                   1,220 (4.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,001 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,928,714 bp\n",
      "  Read 1:     5,842,004 bp\n",
      "  Read 2:     6,086,710 bp\n",
      "Total written (filtered):     11,910,375 bp (99.8%)\n",
      "  Read 1:     5,838,270 bp\n",
      "  Read 2:     6,072,105 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t421.9\t0\t10\n",
      "27\t2\t0.0\t1\t2\n",
      "201\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t0 1\n",
      "215\t2\t0.0\t1\t1 1\n",
      "216\t9\t0.0\t1\t9\n",
      "217\t2\t0.0\t1\t2\n",
      "221\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1220 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t421.9\t0\t7\n",
      "5\t2\t26.4\t0\t2\n",
      "6\t12\t6.6\t0\t12\n",
      "7\t70\t1.6\t0\t70\n",
      "8\t70\t0.4\t0\t70\n",
      "9\t290\t0.1\t0\t289 1\n",
      "10\t340\t0.0\t1\t338 2\n",
      "11\t20\t0.0\t1\t20\n",
      "12\t15\t0.0\t1\t15\n",
      "13\t13\t0.0\t1\t13\n",
      "14\t17\t0.0\t1\t16 1\n",
      "15\t29\t0.0\t1\t28 1\n",
      "16\t25\t0.0\t1\t23 2\n",
      "17\t77\t0.0\t1\t75 2\n",
      "18\t75\t0.0\t1\t72 2 1\n",
      "19\t133\t0.0\t1\t87 45 1\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t19\t0.0\t2\t0 19\n",
      "115\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/490.H_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/490.H_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/490.H_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/490.H_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/491.H_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/491.H_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/491.H_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/491.H_4_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/492.D_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/492.D_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/492.D_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/492.D_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/492.U_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/492.U_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/492.U_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/492.U_3_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/493.D_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/493.D_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/493.D_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/493.D_27_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/493.U_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/493.U_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/493.U_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/493.U_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/494.D_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/494.D_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/494.D_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/494.D_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/494.U_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/494.U_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/494.U_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/494.U_17_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/495.D_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/495.D_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/495.D_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/495.D_11_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/495.U_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/495.U_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/495.U_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/495.U_31_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/496.D_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/496.D_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/496.D_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/496.D_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/496.U_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/496.U_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/496.U_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/496.U_33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/497.H_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/497.H_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/497.H_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/497.H_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/498.H_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/498.H_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/498.H_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/498.H_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/499.H_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/499.H_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/499.H_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/499.H_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/500.H_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/500.H_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/500.H_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/500.H_24_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/500.H_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/500.H_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/500.H_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/500.H_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.85 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,217\n",
      "  Read 1 with adapter:                      33 (0.0%)\n",
      "  Read 2 with adapter:                   2,375 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        82,217 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,122,362 bp\n",
      "  Read 1:    18,258,991 bp\n",
      "  Read 2:    18,863,371 bp\n",
      "Total written (filtered):     37,088,948 bp (99.9%)\n",
      "  Read 1:    18,256,919 bp\n",
      "  Read 2:    18,832,029 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 33 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t1284.6\t0\t22\n",
      "25\t1\t0.0\t1\t1\n",
      "37\t1\t0.0\t1\t1\n",
      "216\t9\t0.0\t1\t7 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2375 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t1284.6\t0\t13\n",
      "4\t5\t321.2\t0\t5\n",
      "5\t6\t80.3\t0\t6\n",
      "6\t21\t20.1\t0\t21\n",
      "7\t54\t5.0\t0\t54\n",
      "8\t81\t1.3\t0\t81\n",
      "9\t407\t0.3\t0\t405 2\n",
      "10\t667\t0.1\t1\t665 2\n",
      "11\t54\t0.0\t1\t54\n",
      "12\t31\t0.0\t1\t30 1\n",
      "13\t22\t0.0\t1\t22\n",
      "14\t31\t0.0\t1\t31\n",
      "15\t59\t0.0\t1\t59\n",
      "16\t59\t0.0\t1\t58 1\n",
      "17\t180\t0.0\t1\t176 4\n",
      "18\t237\t0.0\t1\t227 10\n",
      "19\t365\t0.0\t1\t218 146 1\n",
      "20\t14\t0.0\t2\t0 0 14\n",
      "21\t67\t0.0\t2\t0 67\n",
      "90\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/501.H_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/501.H_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/501.H_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/501.H_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.85 s (18 us/read; 3.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            213,980\n",
      "  Read 1 with adapter:                     115 (0.1%)\n",
      "  Read 2 with adapter:                   2,558 (1.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       213,980 (100.0%)\n",
      "\n",
      "Total basepairs processed:    97,272,516 bp\n",
      "  Read 1:    47,853,841 bp\n",
      "  Read 2:    49,418,675 bp\n",
      "Total written (filtered):     97,229,520 bp (100.0%)\n",
      "  Read 1:    47,852,699 bp\n",
      "  Read 2:    49,376,821 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 115 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t3343.4\t0\t109\n",
      "27\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "110\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2558 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t3343.4\t0\t10\n",
      "5\t4\t209.0\t0\t4\n",
      "6\t4\t52.2\t0\t4\n",
      "7\t35\t13.1\t0\t35\n",
      "8\t29\t3.3\t0\t29\n",
      "9\t121\t0.8\t0\t121\n",
      "10\t275\t0.2\t1\t275\n",
      "11\t40\t0.1\t1\t40\n",
      "12\t63\t0.0\t1\t63\n",
      "13\t47\t0.0\t1\t47\n",
      "14\t45\t0.0\t1\t45\n",
      "15\t102\t0.0\t1\t101 1\n",
      "16\t115\t0.0\t1\t114 1\n",
      "17\t312\t0.0\t1\t309 3\n",
      "18\t397\t0.0\t1\t380 12 5\n",
      "19\t749\t0.0\t1\t531 217 1\n",
      "20\t31\t0.0\t2\t0 1 30\n",
      "21\t174\t0.0\t2\t0 172 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "221\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/502.H_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/502.H_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/502.H_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/502.H_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.88 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            153,550\n",
      "  Read 1 with adapter:                      60 (0.0%)\n",
      "  Read 2 with adapter:                   1,568 (1.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       153,550 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,859,679 bp\n",
      "  Read 1:    34,375,920 bp\n",
      "  Read 2:    35,483,759 bp\n",
      "Total written (filtered):     69,833,403 bp (100.0%)\n",
      "  Read 1:    34,375,716 bp\n",
      "  Read 2:    35,457,687 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 60 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t2399.2\t0\t59\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1568 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t2399.2\t0\t7\n",
      "4\t1\t599.8\t0\t1\n",
      "5\t3\t150.0\t0\t3\n",
      "6\t4\t37.5\t0\t4\n",
      "7\t16\t9.4\t0\t16\n",
      "8\t20\t2.3\t0\t20\n",
      "9\t54\t0.6\t0\t54\n",
      "10\t164\t0.1\t1\t163 1\n",
      "11\t22\t0.0\t1\t21 1\n",
      "12\t28\t0.0\t1\t28\n",
      "13\t11\t0.0\t1\t11\n",
      "14\t37\t0.0\t1\t36 1\n",
      "15\t46\t0.0\t1\t46\n",
      "16\t71\t0.0\t1\t70 1\n",
      "17\t205\t0.0\t1\t202 3\n",
      "18\t244\t0.0\t1\t233 11\n",
      "19\t522\t0.0\t1\t381 139 2\n",
      "20\t14\t0.0\t2\t0 0 14\n",
      "21\t97\t0.0\t2\t0 97\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/503.H_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/503.H_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/503.H_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/503.H_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.23 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            136,814\n",
      "  Read 1 with adapter:                      54 (0.0%)\n",
      "  Read 2 with adapter:                   2,957 (2.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       136,814 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,106,558 bp\n",
      "  Read 1:    30,547,961 bp\n",
      "  Read 2:    31,558,597 bp\n",
      "Total written (filtered):     62,062,121 bp (99.9%)\n",
      "  Read 1:    30,546,948 bp\n",
      "  Read 2:    31,515,173 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 54 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t2137.7\t0\t50\n",
      "213\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2957 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t2137.7\t0\t11\n",
      "5\t1\t133.6\t0\t1\n",
      "6\t4\t33.4\t0\t4\n",
      "7\t45\t8.4\t0\t45\n",
      "8\t68\t2.1\t0\t68\n",
      "9\t294\t0.5\t0\t293 1\n",
      "10\t624\t0.1\t1\t618 6\n",
      "11\t62\t0.0\t1\t62\n",
      "12\t67\t0.0\t1\t66 1\n",
      "13\t66\t0.0\t1\t66\n",
      "14\t90\t0.0\t1\t90\n",
      "15\t96\t0.0\t1\t95 1\n",
      "16\t120\t0.0\t1\t113 7\n",
      "17\t287\t0.0\t1\t280 7\n",
      "18\t321\t0.0\t1\t307 10 4\n",
      "19\t697\t0.0\t1\t488 208 1\n",
      "20\t16\t0.0\t2\t0 0 16\n",
      "21\t85\t0.0\t2\t0 85\n",
      "214\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/504.H_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/504.H_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/504.H_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/504.H_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.21 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            224,995\n",
      "  Read 1 with adapter:                      93 (0.0%)\n",
      "  Read 2 with adapter:                   6,808 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       224,995 (100.0%)\n",
      "\n",
      "Total basepairs processed:   102,345,114 bp\n",
      "  Read 1:    50,346,837 bp\n",
      "  Read 2:    51,998,277 bp\n",
      "Total written (filtered):    102,257,174 bp (99.9%)\n",
      "  Read 1:    50,345,717 bp\n",
      "  Read 2:    51,911,457 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 93 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t3515.5\t0\t88\n",
      "27\t1\t0.0\t1\t1\n",
      "181\t1\t0.0\t1\t0 1\n",
      "216\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6808 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t71\t3515.5\t0\t71\n",
      "4\t7\t878.9\t0\t7\n",
      "5\t24\t219.7\t0\t24\n",
      "6\t60\t54.9\t0\t60\n",
      "7\t187\t13.7\t0\t187\n",
      "8\t317\t3.4\t0\t317\n",
      "9\t1245\t0.9\t0\t1244 1\n",
      "10\t2004\t0.2\t1\t1998 6\n",
      "11\t97\t0.1\t1\t96 1\n",
      "12\t78\t0.0\t1\t78\n",
      "13\t62\t0.0\t1\t60 2\n",
      "14\t102\t0.0\t1\t99 3\n",
      "15\t142\t0.0\t1\t137 5\n",
      "16\t205\t0.0\t1\t202 3\n",
      "17\t447\t0.0\t1\t438 9\n",
      "18\t503\t0.0\t1\t471 20 12\n",
      "19\t1084\t0.0\t1\t741 339 4\n",
      "20\t20\t0.0\t2\t0 0 20\n",
      "21\t147\t0.0\t2\t0 147\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/505.H_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/505.H_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/505.H_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/505.H_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.49 s (20 us/read; 3.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            178,733\n",
      "  Read 1 with adapter:                      72 (0.0%)\n",
      "  Read 2 with adapter:                   5,009 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       178,733 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,235,824 bp\n",
      "  Read 1:    39,956,234 bp\n",
      "  Read 2:    41,279,590 bp\n",
      "Total written (filtered):     81,168,884 bp (99.9%)\n",
      "  Read 1:    39,954,663 bp\n",
      "  Read 2:    41,214,221 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 72 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t63\t2792.7\t0\t63\n",
      "27\t3\t0.0\t1\t3\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t5\t0.0\t1\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5009 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t2792.7\t0\t32\n",
      "4\t5\t698.2\t0\t5\n",
      "5\t12\t174.5\t0\t12\n",
      "6\t34\t43.6\t0\t34\n",
      "7\t118\t10.9\t0\t118\n",
      "8\t202\t2.7\t0\t202\n",
      "9\t894\t0.7\t0\t892 2\n",
      "10\t1487\t0.2\t1\t1480 7\n",
      "11\t79\t0.0\t1\t78 1\n",
      "12\t65\t0.0\t1\t65\n",
      "13\t52\t0.0\t1\t52\n",
      "14\t80\t0.0\t1\t76 4\n",
      "15\t101\t0.0\t1\t99 2\n",
      "16\t136\t0.0\t1\t133 3\n",
      "17\t348\t0.0\t1\t337 11\n",
      "18\t372\t0.0\t1\t355 13 4\n",
      "19\t848\t0.0\t1\t585 262 1\n",
      "20\t15\t0.0\t2\t1 0 14\n",
      "21\t123\t0.0\t2\t0 123\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/506.H_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/506.H_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/506.H_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/506.H_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.72 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,546\n",
      "  Read 1 with adapter:                      31 (0.0%)\n",
      "  Read 2 with adapter:                   1,470 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,546 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,145,481 bp\n",
      "  Read 1:    14,324,129 bp\n",
      "  Read 2:    14,821,352 bp\n",
      "Total written (filtered):     29,123,132 bp (99.9%)\n",
      "  Read 1:    14,322,301 bp\n",
      "  Read 2:    14,800,831 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 31 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t1008.5\t0\t19\n",
      "27\t3\t0.0\t1\t3\n",
      "89\t2\t0.0\t1\t2\n",
      "216\t7\t0.0\t1\t6 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1470 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1008.5\t0\t6\n",
      "4\t1\t252.1\t0\t1\n",
      "5\t4\t63.0\t0\t4\n",
      "6\t6\t15.8\t0\t6\n",
      "7\t27\t3.9\t0\t27\n",
      "8\t27\t1.0\t0\t27\n",
      "9\t204\t0.2\t0\t204\n",
      "10\t366\t0.1\t1\t365 1\n",
      "11\t38\t0.0\t1\t38\n",
      "12\t30\t0.0\t1\t30\n",
      "13\t16\t0.0\t1\t15 1\n",
      "14\t32\t0.0\t1\t31 1\n",
      "15\t45\t0.0\t1\t45\n",
      "16\t49\t0.0\t1\t46 3\n",
      "17\t100\t0.0\t1\t97 3\n",
      "18\t191\t0.0\t1\t186 5\n",
      "19\t281\t0.0\t1\t201 80\n",
      "20\t10\t0.0\t2\t0 0 10\n",
      "21\t36\t0.0\t2\t0 36\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/507.H_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/507.H_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/507.H_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/507.H_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.19 s (32 us/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,932\n",
      "  Read 1 with adapter:                      29 (0.1%)\n",
      "  Read 2 with adapter:                   1,021 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        36,932 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,649,390 bp\n",
      "  Read 1:     8,180,271 bp\n",
      "  Read 2:     8,469,119 bp\n",
      "Total written (filtered):     16,633,618 bp (99.9%)\n",
      "  Read 1:     8,178,154 bp\n",
      "  Read 2:     8,455,464 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t577.1\t0\t14\n",
      "27\t2\t0.0\t1\t2\n",
      "100\t1\t0.0\t1\t1\n",
      "120\t5\t0.0\t1\t5\n",
      "127\t2\t0.0\t1\t2\n",
      "201\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1021 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t577.1\t0\t4\n",
      "6\t6\t9.0\t0\t6\n",
      "7\t36\t2.3\t0\t36\n",
      "8\t31\t0.6\t0\t31\n",
      "9\t148\t0.1\t0\t148\n",
      "10\t275\t0.0\t1\t271 4\n",
      "11\t20\t0.0\t1\t20\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t12\t0.0\t1\t12\n",
      "14\t23\t0.0\t1\t21 2\n",
      "15\t33\t0.0\t1\t32 1\n",
      "16\t27\t0.0\t1\t26 1\n",
      "17\t107\t0.0\t1\t104 3\n",
      "18\t102\t0.0\t1\t99 1 2\n",
      "19\t163\t0.0\t1\t118 44 1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t24\t0.0\t2\t0 24\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/508.H_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/508.H_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/508.H_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/508.H_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.52 s (30 us/read; 2.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,592\n",
      "  Read 1 with adapter:                      31 (0.1%)\n",
      "  Read 2 with adapter:                   1,302 (2.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,592 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,876,019 bp\n",
      "  Read 1:    11,242,696 bp\n",
      "  Read 2:    11,633,323 bp\n",
      "Total written (filtered):     22,856,439 bp (99.9%)\n",
      "  Read 1:    11,241,431 bp\n",
      "  Read 2:    11,615,008 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 31 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t790.5\t0\t21\n",
      "27\t5\t0.0\t1\t5\n",
      "201\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1302 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t790.5\t0\t6\n",
      "6\t2\t12.4\t0\t2\n",
      "7\t31\t3.1\t0\t31\n",
      "8\t37\t0.8\t0\t37\n",
      "9\t168\t0.2\t0\t168\n",
      "10\t325\t0.0\t1\t322 3\n",
      "11\t24\t0.0\t1\t24\n",
      "12\t11\t0.0\t1\t11\n",
      "13\t16\t0.0\t1\t16\n",
      "14\t24\t0.0\t1\t24\n",
      "15\t53\t0.0\t1\t53\n",
      "16\t56\t0.0\t1\t53 3\n",
      "17\t116\t0.0\t1\t113 3\n",
      "18\t131\t0.0\t1\t129 2\n",
      "19\t245\t0.0\t1\t160 85\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t50\t0.0\t2\t0 50\n",
      "224\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/509.H_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/509.H_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/509.H_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/509.H_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.71 s (17 us/read; 3.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            327,519\n",
      "  Read 1 with adapter:                     192 (0.1%)\n",
      "  Read 2 with adapter:                  12,745 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       327,519 (100.0%)\n",
      "\n",
      "Total basepairs processed:   149,239,956 bp\n",
      "  Read 1:    73,339,100 bp\n",
      "  Read 2:    75,900,856 bp\n",
      "Total written (filtered):    149,078,727 bp (99.9%)\n",
      "  Read 1:    73,338,034 bp\n",
      "  Read 2:    75,740,693 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 192 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t179\t5117.5\t0\t179\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t8\t0.0\t1\t8\n",
      "81\t1\t0.0\t1\t1\n",
      "158\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12745 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t100\t5117.5\t0\t100\n",
      "4\t8\t1279.4\t0\t8\n",
      "5\t35\t319.8\t0\t35\n",
      "6\t72\t80.0\t0\t72\n",
      "7\t433\t20.0\t0\t433\n",
      "8\t542\t5.0\t0\t542\n",
      "9\t2521\t1.2\t0\t2520 1\n",
      "10\t3900\t0.3\t1\t3883 17\n",
      "11\t227\t0.1\t1\t227\n",
      "12\t220\t0.0\t1\t219 1\n",
      "13\t146\t0.0\t1\t145 1\n",
      "14\t271\t0.0\t1\t267 4\n",
      "15\t324\t0.0\t1\t318 6\n",
      "16\t349\t0.0\t1\t335 14\n",
      "17\t719\t0.0\t1\t705 14\n",
      "18\t857\t0.0\t1\t811 30 16\n",
      "19\t1738\t0.0\t1\t1160 572 6\n",
      "20\t40\t0.0\t2\t0 2 38\n",
      "21\t227\t0.0\t2\t0 227\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "28\t1\t0.0\t2\t0 1\n",
      "132\t1\t0.0\t2\t1\n",
      "153\t2\t0.0\t2\t0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "199\t1\t0.0\t2\t0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t4\t0.0\t2\t2 1 1\n",
      "230\t1\t0.0\t2\t1\n",
      "242\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/510.H_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/510.H_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/510.H_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/510.H_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.48 s (20 us/read; 2.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            316,285\n",
      "  Read 1 with adapter:                     142 (0.0%)\n",
      "  Read 2 with adapter:                  10,783 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       316,285 (100.0%)\n",
      "\n",
      "Total basepairs processed:   144,080,307 bp\n",
      "  Read 1:    70,818,572 bp\n",
      "  Read 2:    73,261,735 bp\n",
      "Total written (filtered):    143,939,871 bp (99.9%)\n",
      "  Read 1:    70,816,615 bp\n",
      "  Read 2:    73,123,256 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 142 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t129\t4942.0\t0\t129\n",
      "4\t1\t1235.5\t0\t1\n",
      "9\t1\t1.2\t0\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t1\n",
      "106\t2\t0.0\t1\t2\n",
      "147\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "221\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10783 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t68\t4942.0\t0\t68\n",
      "4\t5\t1235.5\t0\t5\n",
      "5\t38\t308.9\t0\t38\n",
      "6\t53\t77.2\t0\t53\n",
      "7\t386\t19.3\t0\t386\n",
      "8\t429\t4.8\t0\t429\n",
      "9\t2066\t1.2\t0\t2061 5\n",
      "10\t3113\t0.3\t1\t3094 19\n",
      "11\t160\t0.1\t1\t160\n",
      "12\t159\t0.0\t1\t156 3\n",
      "13\t165\t0.0\t1\t164 1\n",
      "14\t234\t0.0\t1\t229 5\n",
      "15\t256\t0.0\t1\t253 3\n",
      "16\t303\t0.0\t1\t292 11\n",
      "17\t693\t0.0\t1\t673 20\n",
      "18\t800\t0.0\t1\t771 15 14\n",
      "19\t1610\t0.0\t1\t1086 520 4\n",
      "20\t51\t0.0\t2\t2 3 46\n",
      "21\t179\t0.0\t2\t0 176 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t1\n",
      "120\t1\t0.0\t2\t1\n",
      "152\t1\t0.0\t2\t1\n",
      "208\t1\t0.0\t2\t1\n",
      "216\t8\t0.0\t2\t7 0 1\n",
      "225\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/511.H_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/511.H_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/511.H_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/511.H_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.27 s (17 us/read; 3.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            375,547\n",
      "  Read 1 with adapter:                     236 (0.1%)\n",
      "  Read 2 with adapter:                  13,933 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       375,547 (100.0%)\n",
      "\n",
      "Total basepairs processed:   171,120,438 bp\n",
      "  Read 1:    84,100,819 bp\n",
      "  Read 2:    87,019,619 bp\n",
      "Total written (filtered):    170,946,438 bp (99.9%)\n",
      "  Read 1:    84,099,409 bp\n",
      "  Read 2:    86,847,029 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 236 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t228\t5867.9\t0\t228\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t2\t0.0\t1\t2\n",
      "94\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "188\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13933 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t86\t5867.9\t0\t86\n",
      "4\t14\t1467.0\t0\t14\n",
      "5\t43\t366.7\t0\t43\n",
      "6\t117\t91.7\t0\t117\n",
      "7\t532\t22.9\t0\t532\n",
      "8\t650\t5.7\t0\t650\n",
      "9\t2773\t1.4\t0\t2772 1\n",
      "10\t4373\t0.4\t1\t4355 18\n",
      "11\t186\t0.1\t1\t185 1\n",
      "12\t198\t0.0\t1\t196 2\n",
      "13\t194\t0.0\t1\t192 2\n",
      "14\t238\t0.0\t1\t230 8\n",
      "15\t323\t0.0\t1\t311 12\n",
      "16\t333\t0.0\t1\t324 9\n",
      "17\t736\t0.0\t1\t711 25\n",
      "18\t980\t0.0\t1\t930 35 15\n",
      "19\t1844\t0.0\t1\t1233 610 1\n",
      "20\t48\t0.0\t2\t0 2 46\n",
      "21\t252\t0.0\t2\t0 251 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "73\t1\t0.0\t2\t1\n",
      "132\t2\t0.0\t2\t2\n",
      "153\t1\t0.0\t2\t1\n",
      "216\t5\t0.0\t2\t3 2\n",
      "223\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/512.H_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/512.H_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/512.H_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/512.H_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.79 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            118,992\n",
      "  Read 1 with adapter:                      55 (0.0%)\n",
      "  Read 2 with adapter:                   2,748 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       118,992 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,030,656 bp\n",
      "  Read 1:    26,582,926 bp\n",
      "  Read 2:    27,447,730 bp\n",
      "Total written (filtered):     53,991,142 bp (99.9%)\n",
      "  Read 1:    26,582,502 bp\n",
      "  Read 2:    27,408,640 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t52\t1859.2\t0\t52\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2748 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1859.2\t0\t12\n",
      "5\t4\t116.2\t0\t4\n",
      "6\t18\t29.1\t0\t18\n",
      "7\t71\t7.3\t0\t71\n",
      "8\t85\t1.8\t0\t85\n",
      "9\t381\t0.5\t0\t381\n",
      "10\t602\t0.1\t1\t599 3\n",
      "11\t42\t0.0\t1\t42\n",
      "12\t45\t0.0\t1\t45\n",
      "13\t33\t0.0\t1\t33\n",
      "14\t60\t0.0\t1\t60\n",
      "15\t84\t0.0\t1\t83 1\n",
      "16\t121\t0.0\t1\t115 6\n",
      "17\t246\t0.0\t1\t239 7\n",
      "18\t257\t0.0\t1\t243 9 5\n",
      "19\t587\t0.0\t1\t384 203\n",
      "20\t13\t0.0\t2\t0 1 12\n",
      "21\t82\t0.0\t2\t0 82\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "217\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/513.H_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/513.H_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/513.H_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/513.H_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.19 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,748\n",
      "  Read 1 with adapter:                      62 (0.1%)\n",
      "  Read 2 with adapter:                   3,397 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        91,748 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,527,107 bp\n",
      "  Read 1:    20,397,021 bp\n",
      "  Read 2:    21,130,086 bp\n",
      "Total written (filtered):     41,480,543 bp (99.9%)\n",
      "  Read 1:    20,393,485 bp\n",
      "  Read 2:    21,087,058 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 62 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t1433.6\t0\t42\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "90\t1\t0.0\t1\t1\n",
      "112\t1\t0.0\t1\t1\n",
      "162\t1\t0.0\t1\t1\n",
      "186\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t3\t0.0\t1\t1 2\n",
      "216\t5\t0.0\t1\t5\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3397 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t1433.6\t0\t14\n",
      "4\t1\t358.4\t0\t1\n",
      "5\t9\t89.6\t0\t9\n",
      "6\t23\t22.4\t0\t23\n",
      "7\t141\t5.6\t0\t141\n",
      "8\t177\t1.4\t0\t177\n",
      "9\t604\t0.3\t0\t604\n",
      "10\t1007\t0.1\t1\t1002 5\n",
      "11\t54\t0.0\t1\t54\n",
      "12\t35\t0.0\t1\t35\n",
      "13\t29\t0.0\t1\t29\n",
      "14\t53\t0.0\t1\t53\n",
      "15\t84\t0.0\t1\t83 1\n",
      "16\t101\t0.0\t1\t97 4\n",
      "17\t218\t0.0\t1\t209 9\n",
      "18\t248\t0.0\t1\t231 13 4\n",
      "19\t537\t0.0\t1\t353 184\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t54\t0.0\t2\t0 54\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "230\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/514.H_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/514.H_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/514.H_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/514.H_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.18 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            169,336\n",
      "  Read 1 with adapter:                     125 (0.1%)\n",
      "  Read 2 with adapter:                   4,974 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       169,336 (100.0%)\n",
      "\n",
      "Total basepairs processed:    77,012,000 bp\n",
      "  Read 1:    37,857,306 bp\n",
      "  Read 2:    39,154,694 bp\n",
      "Total written (filtered):     76,944,931 bp (99.9%)\n",
      "  Read 1:    37,855,619 bp\n",
      "  Read 2:    39,089,312 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 125 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t118\t2645.9\t0\t118\n",
      "38\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "216\t5\t0.0\t1\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4974 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t2645.9\t0\t28\n",
      "4\t4\t661.5\t0\t4\n",
      "5\t7\t165.4\t0\t7\n",
      "6\t18\t41.3\t0\t18\n",
      "7\t151\t10.3\t0\t151\n",
      "8\t185\t2.6\t0\t185\n",
      "9\t835\t0.6\t0\t835\n",
      "10\t1349\t0.2\t1\t1342 7\n",
      "11\t89\t0.0\t1\t89\n",
      "12\t97\t0.0\t1\t97\n",
      "13\t88\t0.0\t1\t86 2\n",
      "14\t124\t0.0\t1\t120 4\n",
      "15\t115\t0.0\t1\t111 4\n",
      "16\t149\t0.0\t1\t146 3\n",
      "17\t379\t0.0\t1\t369 10\n",
      "18\t396\t0.0\t1\t364 24 8\n",
      "19\t822\t0.0\t1\t558 263 1\n",
      "20\t27\t0.0\t2\t0 1 26\n",
      "21\t107\t0.0\t2\t0 107\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "217\t1\t0.0\t2\t1\n",
      "228\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/515.H_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/515.H_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/515.H_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/515.H_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.18 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,045\n",
      "  Read 1 with adapter:                      61 (0.1%)\n",
      "  Read 2 with adapter:                   1,054 (1.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       103,045 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,872,440 bp\n",
      "  Read 1:    23,062,784 bp\n",
      "  Read 2:    23,809,656 bp\n",
      "Total written (filtered):     46,854,751 bp (100.0%)\n",
      "  Read 1:    23,062,484 bp\n",
      "  Read 2:    23,792,267 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 61 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t60\t1610.1\t0\t60\n",
      "120\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1054 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1610.1\t0\t5\n",
      "5\t1\t100.6\t0\t1\n",
      "6\t1\t25.2\t0\t1\n",
      "7\t13\t6.3\t0\t13\n",
      "8\t13\t1.6\t0\t13\n",
      "9\t49\t0.4\t0\t49\n",
      "10\t103\t0.1\t1\t100 3\n",
      "11\t19\t0.0\t1\t19\n",
      "12\t24\t0.0\t1\t24\n",
      "13\t9\t0.0\t1\t9\n",
      "14\t16\t0.0\t1\t16\n",
      "15\t33\t0.0\t1\t33\n",
      "16\t50\t0.0\t1\t46 4\n",
      "17\t127\t0.0\t1\t123 4\n",
      "18\t161\t0.0\t1\t157 3 1\n",
      "19\t354\t0.0\t1\t256 98\n",
      "20\t10\t0.0\t2\t0 1 9\n",
      "21\t65\t0.0\t2\t0 65\n",
      "216\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/501.H_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/501.H_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/501.H_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/501.H_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/502.H_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/502.H_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/502.H_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/502.H_35_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/503.H_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/503.H_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/503.H_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/503.H_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/504.H_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/504.H_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/504.H_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/504.H_15_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/505.H_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/505.H_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/505.H_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/505.H_26_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/506.H_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/506.H_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/506.H_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/506.H_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/507.H_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/507.H_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/507.H_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/507.H_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/508.H_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/508.H_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/508.H_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/508.H_7_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/509.H_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/509.H_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/509.H_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/509.H_20_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/510.H_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/510.H_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/510.H_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/510.H_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/511.H_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/511.H_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/511.H_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/511.H_9_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/512.H_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/512.H_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/512.H_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/512.H_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/513.H_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/513.H_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/513.H_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/513.H_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/514.H_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/514.H_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/514.H_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/514.H_25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/515.H_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/515.H_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/515.H_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/515.H_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/516.H_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/516.H_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/516.H_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/516.H_8_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/516.H_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-dkp80xf4/516.H_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/516.H_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-0lt850d7/bd1087af-96e7-4948-9e39-00fe469bb3e3/data/516.H_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.98 s (21 us/read; 2.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,948\n",
      "  Read 1 with adapter:                      47 (0.1%)\n",
      "  Read 2 with adapter:                   2,481 (2.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        93,948 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,606,870 bp\n",
      "  Read 1:    20,944,525 bp\n",
      "  Read 2:    21,662,345 bp\n",
      "Total written (filtered):     42,573,284 bp (99.9%)\n",
      "  Read 1:    20,943,745 bp\n",
      "  Read 2:    21,629,539 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 47 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t1467.9\t0\t44\n",
      "216\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2481 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1467.9\t0\t9\n",
      "4\t2\t367.0\t0\t2\n",
      "5\t5\t91.7\t0\t5\n",
      "6\t9\t22.9\t0\t9\n",
      "7\t77\t5.7\t0\t77\n",
      "8\t102\t1.4\t0\t102\n",
      "9\t424\t0.4\t0\t424\n",
      "10\t671\t0.1\t1\t667 4\n",
      "11\t38\t0.0\t1\t38\n",
      "12\t33\t0.0\t1\t33\n",
      "13\t28\t0.0\t1\t28\n",
      "14\t52\t0.0\t1\t50 2\n",
      "15\t68\t0.0\t1\t67 1\n",
      "16\t71\t0.0\t1\t69 2\n",
      "17\t193\t0.0\t1\t190 3\n",
      "18\t207\t0.0\t1\t192 14 1\n",
      "19\t410\t0.0\t1\t282 127 1\n",
      "20\t7\t0.0\t2\t0 0 7\n",
      "21\t71\t0.0\t2\t0 70 1\n",
      "22\t2\t0.0\t2\t0 1 1\n",
      "61\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGmicro2018_289-336.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/517.H_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/517.H_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/517.H_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/517.H_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.13 s (35 us/read; 1.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,597\n",
      "  Read 1 with adapter:                      23 (0.1%)\n",
      "  Read 2 with adapter:                   1,338 (4.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,597 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,633,340 bp\n",
      "  Read 1:     7,180,865 bp\n",
      "  Read 2:     7,452,475 bp\n",
      "Total written (filtered):     14,615,517 bp (99.9%)\n",
      "  Read 1:     7,179,876 bp\n",
      "  Read 2:     7,435,641 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t509.3\t0\t16\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "215\t2\t0.0\t1\t2\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1338 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t509.3\t0\t5\n",
      "5\t2\t31.8\t0\t2\n",
      "6\t7\t8.0\t0\t7\n",
      "7\t55\t2.0\t0\t55\n",
      "8\t54\t0.5\t0\t54\n",
      "9\t283\t0.1\t0\t283\n",
      "10\t430\t0.0\t1\t428 2\n",
      "11\t14\t0.0\t1\t14\n",
      "12\t12\t0.0\t1\t12\n",
      "13\t18\t0.0\t1\t18\n",
      "14\t12\t0.0\t1\t11 1\n",
      "15\t36\t0.0\t1\t34 2\n",
      "16\t30\t0.0\t1\t30\n",
      "17\t77\t0.0\t1\t77\n",
      "18\t104\t0.0\t1\t101 3\n",
      "19\t167\t0.0\t1\t110 57\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t25\t0.0\t2\t0 25\n",
      "199\t1\t0.0\t2\t1\n",
      "205\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/518.H_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/518.H_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/518.H_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/518.H_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.82 s (26 us/read; 2.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             71,225\n",
      "  Read 1 with adapter:                      45 (0.1%)\n",
      "  Read 2 with adapter:                   2,980 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        71,225 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,233,231 bp\n",
      "  Read 1:    15,825,751 bp\n",
      "  Read 2:    16,407,480 bp\n",
      "Total written (filtered):     32,194,919 bp (99.9%)\n",
      "  Read 1:    15,824,248 bp\n",
      "  Read 2:    16,370,671 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 45 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t1112.9\t0\t36\n",
      "27\t2\t0.0\t1\t2\n",
      "55\t1\t0.0\t1\t0 1\n",
      "198\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2980 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t1112.9\t0\t19\n",
      "4\t4\t278.2\t0\t4\n",
      "5\t4\t69.6\t0\t4\n",
      "6\t18\t17.4\t0\t18\n",
      "7\t127\t4.3\t0\t127\n",
      "8\t157\t1.1\t0\t157\n",
      "9\t635\t0.3\t0\t634 1\n",
      "10\t909\t0.1\t1\t907 2\n",
      "11\t33\t0.0\t1\t33\n",
      "12\t38\t0.0\t1\t38\n",
      "13\t34\t0.0\t1\t33 1\n",
      "14\t54\t0.0\t1\t54\n",
      "15\t65\t0.0\t1\t61 4\n",
      "16\t68\t0.0\t1\t65 3\n",
      "17\t157\t0.0\t1\t151 6\n",
      "18\t206\t0.0\t1\t190 11 5\n",
      "19\t401\t0.0\t1\t265 136\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t44\t0.0\t2\t0 44\n",
      "145\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/519.H_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/519.H_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/519.H_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/519.H_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.37 s (30 us/read; 2.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,096\n",
      "  Read 1 with adapter:                      41 (0.1%)\n",
      "  Read 2 with adapter:                   1,932 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        46,096 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,747,536 bp\n",
      "  Read 1:    10,182,795 bp\n",
      "  Read 2:    10,564,741 bp\n",
      "Total written (filtered):     20,721,467 bp (99.9%)\n",
      "  Read 1:    10,181,030 bp\n",
      "  Read 2:    10,540,437 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 41 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t720.2\t0\t32\n",
      "26\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "218\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1932 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t720.2\t0\t16\n",
      "4\t1\t180.1\t0\t1\n",
      "5\t5\t45.0\t0\t5\n",
      "6\t19\t11.3\t0\t19\n",
      "7\t79\t2.8\t0\t79\n",
      "8\t109\t0.7\t0\t109\n",
      "9\t388\t0.2\t0\t388\n",
      "10\t540\t0.0\t1\t539 1\n",
      "11\t21\t0.0\t1\t21\n",
      "12\t20\t0.0\t1\t18 2\n",
      "13\t17\t0.0\t1\t17\n",
      "14\t41\t0.0\t1\t40 1\n",
      "15\t50\t0.0\t1\t47 3\n",
      "16\t57\t0.0\t1\t54 3\n",
      "17\t140\t0.0\t1\t132 8\n",
      "18\t135\t0.0\t1\t126 8 1\n",
      "19\t252\t0.0\t1\t161 91\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t35\t0.0\t2\t0 34 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/520.H_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/520.H_49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/520.H_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/520.H_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.36 s (32 us/read; 1.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,765\n",
      "  Read 1 with adapter:                      38 (0.1%)\n",
      "  Read 2 with adapter:                   1,621 (3.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,765 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,217,062 bp\n",
      "  Read 1:     9,432,536 bp\n",
      "  Read 2:     9,784,526 bp\n",
      "Total written (filtered):     19,193,218 bp (99.9%)\n",
      "  Read 1:     9,430,316 bp\n",
      "  Read 2:     9,762,902 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 38 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t668.2\t0\t28\n",
      "193\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "214\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t3 1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1621 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t668.2\t0\t14\n",
      "4\t2\t167.1\t0\t2\n",
      "5\t3\t41.8\t0\t3\n",
      "6\t8\t10.4\t0\t8\n",
      "7\t58\t2.6\t0\t58\n",
      "8\t73\t0.7\t0\t73\n",
      "9\t289\t0.2\t0\t289\n",
      "10\t456\t0.0\t1\t456\n",
      "11\t23\t0.0\t1\t23\n",
      "12\t20\t0.0\t1\t19 1\n",
      "13\t26\t0.0\t1\t25 1\n",
      "14\t30\t0.0\t1\t29 1\n",
      "15\t52\t0.0\t1\t51 1\n",
      "16\t50\t0.0\t1\t50\n",
      "17\t106\t0.0\t1\t102 4\n",
      "18\t127\t0.0\t1\t121 4 2\n",
      "19\t232\t0.0\t1\t148 83 1\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t35\t0.0\t2\t0 35\n",
      "115\t2\t0.0\t2\t0 2\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "217\t3\t0.0\t2\t2 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/521.H_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/521.H_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/521.H_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/521.H_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.36 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,676\n",
      "  Read 1 with adapter:                      30 (0.1%)\n",
      "  Read 2 with adapter:                   1,988 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,676 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,974,441 bp\n",
      "  Read 1:    11,287,599 bp\n",
      "  Read 2:    11,686,842 bp\n",
      "Total written (filtered):     22,950,573 bp (99.9%)\n",
      "  Read 1:    11,287,441 bp\n",
      "  Read 2:    11,663,132 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 30 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t791.8\t0\t27\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1988 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t791.8\t0\t10\n",
      "5\t4\t49.5\t0\t4\n",
      "6\t9\t12.4\t0\t9\n",
      "7\t97\t3.1\t0\t97\n",
      "8\t101\t0.8\t0\t101\n",
      "9\t401\t0.2\t0\t400 1\n",
      "10\t711\t0.0\t1\t709 2\n",
      "11\t26\t0.0\t1\t26\n",
      "12\t20\t0.0\t1\t20\n",
      "13\t12\t0.0\t1\t12\n",
      "14\t15\t0.0\t1\t14 1\n",
      "15\t42\t0.0\t1\t42\n",
      "16\t51\t0.0\t1\t50 1\n",
      "17\t94\t0.0\t1\t91 3\n",
      "18\t119\t0.0\t1\t113 4 2\n",
      "19\t235\t0.0\t1\t165 70\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t37\t0.0\t2\t0 37\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/522.H_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/522.H_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/522.H_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/522.H_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.72 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            116,713\n",
      "  Read 1 with adapter:                      62 (0.1%)\n",
      "  Read 2 with adapter:                   3,152 (2.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       116,713 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,061,484 bp\n",
      "  Read 1:    26,087,345 bp\n",
      "  Read 2:    26,974,139 bp\n",
      "Total written (filtered):     53,018,301 bp (99.9%)\n",
      "  Read 1:    26,086,674 bp\n",
      "  Read 2:    26,931,627 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 62 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t1823.6\t0\t58\n",
      "27\t1\t0.0\t1\t1\n",
      "127\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3152 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1823.6\t0\t10\n",
      "5\t9\t114.0\t0\t9\n",
      "6\t13\t28.5\t0\t13\n",
      "7\t130\t7.1\t0\t130\n",
      "8\t108\t1.8\t0\t108\n",
      "9\t489\t0.4\t0\t487 2\n",
      "10\t831\t0.1\t1\t824 7\n",
      "11\t61\t0.0\t1\t61\n",
      "12\t41\t0.0\t1\t40 1\n",
      "13\t33\t0.0\t1\t33\n",
      "14\t58\t0.0\t1\t57 1\n",
      "15\t102\t0.0\t1\t100 2\n",
      "16\t120\t0.0\t1\t116 4\n",
      "17\t229\t0.0\t1\t225 4\n",
      "18\t271\t0.0\t1\t258 7 6\n",
      "19\t556\t0.0\t1\t372 184\n",
      "20\t10\t0.0\t2\t0 0 10\n",
      "21\t73\t0.0\t2\t0 73\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t2 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/523.H_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/523.H_51_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/523.H_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/523.H_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.80 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,423\n",
      "  Read 1 with adapter:                      25 (0.2%)\n",
      "  Read 2 with adapter:                     491 (4.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,423 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,903,959 bp\n",
      "  Read 1:     2,395,787 bp\n",
      "  Read 2:     2,508,172 bp\n",
      "Total written (filtered):      4,894,138 bp (99.8%)\n",
      "  Read 1:     2,391,874 bp\n",
      "  Read 2:     2,502,264 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 25 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t178.5\t0\t5\n",
      "27\t2\t0.0\t1\t2\n",
      "192\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "215\t2\t0.0\t1\t2\n",
      "216\t12\t0.0\t1\t12\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 491 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t178.5\t0\t4\n",
      "6\t4\t2.8\t0\t4\n",
      "7\t23\t0.7\t0\t23\n",
      "8\t11\t0.2\t0\t11\n",
      "9\t101\t0.0\t0\t101\n",
      "10\t176\t0.0\t1\t175 1\n",
      "11\t3\t0.0\t1\t2 1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t9\t0.0\t1\t9\n",
      "15\t17\t0.0\t1\t17\n",
      "16\t18\t0.0\t1\t15 3\n",
      "17\t38\t0.0\t1\t38\n",
      "18\t24\t0.0\t1\t22 2\n",
      "19\t48\t0.0\t1\t34 14\n",
      "20\t2\t0.0\t2\t1 0 1\n",
      "21\t7\t0.0\t2\t0 7\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/524.H_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/524.H_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/524.H_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/524.H_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.15 s (21 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            146,579\n",
      "  Read 1 with adapter:                     123 (0.1%)\n",
      "  Read 2 with adapter:                   5,748 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       146,579 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,785,868 bp\n",
      "  Read 1:    32,818,649 bp\n",
      "  Read 2:    33,967,219 bp\n",
      "Total written (filtered):     66,713,726 bp (99.9%)\n",
      "  Read 1:    32,817,823 bp\n",
      "  Read 2:    33,895,903 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 123 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t117\t2290.3\t0\t117\n",
      "24\t1\t0.0\t1\t0 1\n",
      "26\t1\t0.0\t1\t1\n",
      "84\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5748 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t2290.3\t0\t40\n",
      "4\t9\t572.6\t0\t9\n",
      "5\t17\t143.1\t0\t17\n",
      "6\t50\t35.8\t0\t50\n",
      "7\t244\t8.9\t0\t244\n",
      "8\t270\t2.2\t0\t270\n",
      "9\t1156\t0.6\t0\t1154 2\n",
      "10\t1653\t0.1\t1\t1642 11\n",
      "11\t90\t0.0\t1\t90\n",
      "12\t78\t0.0\t1\t78\n",
      "13\t60\t0.0\t1\t60\n",
      "14\t102\t0.0\t1\t99 3\n",
      "15\t139\t0.0\t1\t133 6\n",
      "16\t207\t0.0\t1\t201 6\n",
      "17\t403\t0.0\t1\t386 17\n",
      "18\t349\t0.0\t1\t330 10 9\n",
      "19\t740\t0.0\t1\t455 285\n",
      "20\t17\t0.0\t2\t0 0 17\n",
      "21\t122\t0.0\t2\t0 122\n",
      "154\t1\t0.0\t2\t0 1\n",
      "193\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/525.H_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/525.H_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/525.H_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/525.H_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.13 s (21 us/read; 2.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            198,419\n",
      "  Read 1 with adapter:                     138 (0.1%)\n",
      "  Read 2 with adapter:                   7,772 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       198,419 (100.0%)\n",
      "\n",
      "Total basepairs processed:    90,413,377 bp\n",
      "  Read 1:    44,428,546 bp\n",
      "  Read 2:    45,984,831 bp\n",
      "Total written (filtered):     90,316,217 bp (99.9%)\n",
      "  Read 1:    44,427,871 bp\n",
      "  Read 2:    45,888,346 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 138 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t133\t3100.3\t0\t133\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "97\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7772 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t3100.3\t0\t48\n",
      "4\t11\t775.1\t0\t11\n",
      "5\t24\t193.8\t0\t24\n",
      "6\t48\t48.4\t0\t48\n",
      "7\t317\t12.1\t0\t317\n",
      "8\t345\t3.0\t0\t345\n",
      "9\t1578\t0.8\t0\t1574 4\n",
      "10\t2238\t0.2\t1\t2228 10\n",
      "11\t122\t0.0\t1\t122\n",
      "12\t134\t0.0\t1\t130 4\n",
      "13\t126\t0.0\t1\t125 1\n",
      "14\t162\t0.0\t1\t156 6\n",
      "15\t209\t0.0\t1\t202 7\n",
      "16\t247\t0.0\t1\t236 11\n",
      "17\t430\t0.0\t1\t417 13\n",
      "18\t555\t0.0\t1\t526 22 7\n",
      "19\t1027\t0.0\t1\t657 370\n",
      "20\t22\t0.0\t2\t0 0 22\n",
      "21\t124\t0.0\t2\t0 124\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "196\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/526.H_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/526.H_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/526.H_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/526.H_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.39 s (22 us/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            156,641\n",
      "  Read 1 with adapter:                      86 (0.1%)\n",
      "  Read 2 with adapter:                   5,743 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       156,641 (100.0%)\n",
      "\n",
      "Total basepairs processed:    71,363,680 bp\n",
      "  Read 1:    35,073,603 bp\n",
      "  Read 2:    36,290,077 bp\n",
      "Total written (filtered):     71,291,174 bp (99.9%)\n",
      "  Read 1:    35,072,926 bp\n",
      "  Read 2:    36,218,248 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 86 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t76\t2447.5\t0\t76\n",
      "25\t4\t0.0\t1\t4\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5743 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t2447.5\t0\t28\n",
      "4\t7\t611.9\t0\t7\n",
      "5\t18\t153.0\t0\t18\n",
      "6\t38\t38.2\t0\t38\n",
      "7\t185\t9.6\t0\t185\n",
      "8\t285\t2.4\t0\t285\n",
      "9\t1146\t0.6\t0\t1145 1\n",
      "10\t1652\t0.1\t1\t1646 6\n",
      "11\t76\t0.0\t1\t73 3\n",
      "12\t96\t0.0\t1\t89 7\n",
      "13\t71\t0.0\t1\t71\n",
      "14\t118\t0.0\t1\t116 2\n",
      "15\t131\t0.0\t1\t127 4\n",
      "16\t205\t0.0\t1\t198 7\n",
      "17\t423\t0.0\t1\t416 7\n",
      "18\t397\t0.0\t1\t375 15 7\n",
      "19\t738\t0.0\t1\t471 266 1\n",
      "20\t22\t0.0\t2\t0 1 21\n",
      "21\t104\t0.0\t2\t0 103 1\n",
      "22\t2\t0.0\t2\t0 1 1\n",
      "155\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/527.H_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/527.H_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/527.H_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/527.H_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.47 s (23 us/read; 2.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,679\n",
      "  Read 1 with adapter:                      82 (0.1%)\n",
      "  Read 2 with adapter:                   3,218 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,416,664 bp\n",
      "  Read 1:    24,291,673 bp\n",
      "  Read 2:    25,124,991 bp\n",
      "Total written (filtered):     49,373,181 bp (99.9%)\n",
      "  Read 1:    24,290,733 bp\n",
      "  Read 2:    25,082,448 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 82 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t1698.1\t0\t77\n",
      "38\t2\t0.0\t1\t2\n",
      "201\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3218 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1698.1\t0\t11\n",
      "4\t2\t424.5\t0\t2\n",
      "5\t9\t106.1\t0\t9\n",
      "6\t18\t26.5\t0\t18\n",
      "7\t101\t6.6\t0\t101\n",
      "8\t131\t1.7\t0\t131\n",
      "9\t566\t0.4\t0\t566\n",
      "10\t894\t0.1\t1\t890 4\n",
      "11\t39\t0.0\t1\t39\n",
      "12\t34\t0.0\t1\t34\n",
      "13\t36\t0.0\t1\t35 1\n",
      "14\t46\t0.0\t1\t46\n",
      "15\t81\t0.0\t1\t80 1\n",
      "16\t119\t0.0\t1\t116 3\n",
      "17\t229\t0.0\t1\t221 8\n",
      "18\t275\t0.0\t1\t263 7 5\n",
      "19\t536\t0.0\t1\t346 188 2\n",
      "20\t9\t0.0\t2\t0 0 9\n",
      "21\t75\t0.0\t2\t0 75\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "198\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/528.H_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/528.H_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/528.H_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/528.H_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.39 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            144,985\n",
      "  Read 1 with adapter:                      90 (0.1%)\n",
      "  Read 2 with adapter:                   4,651 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       144,985 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,963,976 bp\n",
      "  Read 1:    32,423,769 bp\n",
      "  Read 2:    33,540,207 bp\n",
      "Total written (filtered):     65,902,349 bp (99.9%)\n",
      "  Read 1:    32,422,082 bp\n",
      "  Read 2:    33,480,267 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t78\t2265.4\t0\t78\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "45\t1\t0.0\t1\t1\n",
      "66\t1\t0.0\t1\t1\n",
      "191\t1\t0.0\t1\t0 1\n",
      "192\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4651 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t2265.4\t0\t19\n",
      "4\t3\t566.3\t0\t3\n",
      "5\t5\t141.6\t0\t5\n",
      "6\t37\t35.4\t0\t37\n",
      "7\t172\t8.8\t0\t172\n",
      "8\t196\t2.2\t0\t196\n",
      "9\t821\t0.6\t0\t819 2\n",
      "10\t1304\t0.1\t1\t1297 7\n",
      "11\t52\t0.0\t1\t52\n",
      "12\t72\t0.0\t1\t72\n",
      "13\t77\t0.0\t1\t75 2\n",
      "14\t107\t0.0\t1\t104 3\n",
      "15\t126\t0.0\t1\t124 2\n",
      "16\t144\t0.0\t1\t135 9\n",
      "17\t340\t0.0\t1\t330 10\n",
      "18\t387\t0.0\t1\t371 8 8\n",
      "19\t662\t0.0\t1\t432 226 4\n",
      "20\t28\t0.0\t2\t0 0 28\n",
      "21\t97\t0.0\t2\t0 94 3\n",
      "158\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/529.H_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/529.H_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/529.H_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/529.H_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.67 s (22 us/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            123,622\n",
      "  Read 1 with adapter:                      85 (0.1%)\n",
      "  Read 2 with adapter:                   4,787 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       123,622 (100.0%)\n",
      "\n",
      "Total basepairs processed:    56,240,951 bp\n",
      "  Read 1:    27,635,298 bp\n",
      "  Read 2:    28,605,653 bp\n",
      "Total written (filtered):     56,180,829 bp (99.9%)\n",
      "  Read 1:    27,633,997 bp\n",
      "  Read 2:    28,546,832 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 85 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t74\t1931.6\t0\t74\n",
      "27\t4\t0.0\t1\t4\n",
      "100\t3\t0.0\t1\t3\n",
      "120\t2\t0.0\t1\t2\n",
      "215\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4787 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t1931.6\t0\t35\n",
      "4\t1\t482.9\t0\t1\n",
      "5\t14\t120.7\t0\t14\n",
      "6\t37\t30.2\t0\t37\n",
      "7\t184\t7.5\t0\t184\n",
      "8\t236\t1.9\t0\t236\n",
      "9\t995\t0.5\t0\t992 3\n",
      "10\t1404\t0.1\t1\t1398 6\n",
      "11\t66\t0.0\t1\t65 1\n",
      "12\t71\t0.0\t1\t71\n",
      "13\t46\t0.0\t1\t46\n",
      "14\t99\t0.0\t1\t98 1\n",
      "15\t112\t0.0\t1\t108 4\n",
      "16\t130\t0.0\t1\t124 6\n",
      "17\t320\t0.0\t1\t311 9\n",
      "18\t359\t0.0\t1\t347 7 5\n",
      "19\t600\t0.0\t1\t369 229 2\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t65\t0.0\t2\t0 65\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/530.H_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/530.H_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/530.H_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/530.H_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.88 s (41 us/read; 1.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,244\n",
      "  Read 1 with adapter:                      36 (0.2%)\n",
      "  Read 2 with adapter:                     966 (4.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        21,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,388,281 bp\n",
      "  Read 1:     4,600,941 bp\n",
      "  Read 2:     4,787,340 bp\n",
      "Total written (filtered):      9,369,179 bp (99.8%)\n",
      "  Read 1:     4,596,352 bp\n",
      "  Read 2:     4,772,827 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 36 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t331.9\t0\t12\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "41\t1\t0.0\t1\t1\n",
      "192\t2\t0.0\t1\t2\n",
      "202\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t3\t0.0\t1\t1 2\n",
      "216\t13\t0.0\t1\t12 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 966 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t331.9\t0\t5\n",
      "5\t3\t20.7\t0\t3\n",
      "6\t7\t5.2\t0\t7\n",
      "7\t48\t1.3\t0\t48\n",
      "8\t38\t0.3\t0\t38\n",
      "9\t190\t0.1\t0\t190\n",
      "10\t309\t0.0\t1\t309\n",
      "11\t17\t0.0\t1\t17\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t7\t0.0\t1\t7\n",
      "14\t12\t0.0\t1\t12\n",
      "15\t14\t0.0\t1\t14\n",
      "16\t20\t0.0\t1\t20\n",
      "17\t55\t0.0\t1\t55\n",
      "18\t84\t0.0\t1\t82 2\n",
      "19\t104\t0.0\t1\t67 37\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t18\t0.0\t2\t0 18\n",
      "114\t1\t0.0\t2\t1\n",
      "115\t26\t0.0\t2\t22 4\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/531.H_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/531.H_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/531.H_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/531.H_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.85 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,225\n",
      "  Read 1 with adapter:                      16 (0.1%)\n",
      "  Read 2 with adapter:                     584 (4.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        14,225 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,234,854 bp\n",
      "  Read 1:     3,052,808 bp\n",
      "  Read 2:     3,182,046 bp\n",
      "Total written (filtered):      6,225,927 bp (99.9%)\n",
      "  Read 1:     3,051,270 bp\n",
      "  Read 2:     3,174,657 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t222.3\t0\t9\n",
      "215\t2\t0.0\t1\t2\n",
      "216\t4\t0.0\t1\t3 1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 584 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t222.3\t0\t5\n",
      "5\t5\t13.9\t0\t5\n",
      "6\t3\t3.5\t0\t3\n",
      "7\t17\t0.9\t0\t17\n",
      "8\t32\t0.2\t0\t32\n",
      "9\t129\t0.1\t0\t129\n",
      "10\t178\t0.0\t1\t177 1\n",
      "11\t13\t0.0\t1\t13\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t5\t0.0\t1\t5\n",
      "15\t16\t0.0\t1\t16\n",
      "16\t9\t0.0\t1\t9\n",
      "17\t27\t0.0\t1\t25 2\n",
      "18\t43\t0.0\t1\t39 3 1\n",
      "19\t75\t0.0\t1\t46 29\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t11\t0.0\t2\t0 11\n",
      "115\t3\t0.0\t2\t1 2\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/517.H_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/517.H_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/517.H_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/517.H_31_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/518.H_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/518.H_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/518.H_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/518.H_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/519.H_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/519.H_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/519.H_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/519.H_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/520.H_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/520.H_49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/520.H_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/520.H_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/521.H_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/521.H_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/521.H_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/521.H_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/522.H_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/522.H_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/522.H_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/522.H_32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/523.H_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/523.H_51_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/523.H_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/523.H_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/524.H_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/524.H_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/524.H_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/524.H_6_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/525.H_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/525.H_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/525.H_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/525.H_12_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/526.H_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/526.H_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/526.H_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/526.H_24_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/527.H_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/527.H_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/527.H_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/527.H_26_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/528.H_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/528.H_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/528.H_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/528.H_4_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/529.H_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/529.H_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/529.H_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/529.H_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/530.H_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/530.H_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/530.H_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/530.H_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/531.H_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/531.H_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/531.H_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/531.H_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/532.H_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/532.H_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/532.H_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/532.H_18_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/532.H_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/532.H_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/532.H_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/532.H_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.84 s (21 us/read; 2.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,608\n",
      "  Read 1 with adapter:                      40 (0.0%)\n",
      "  Read 2 with adapter:                   1,115 (1.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        85,608 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,886,419 bp\n",
      "  Read 1:    19,131,968 bp\n",
      "  Read 2:    19,754,451 bp\n",
      "Total written (filtered):     38,867,426 bp (100.0%)\n",
      "  Read 1:    19,130,547 bp\n",
      "  Read 2:    19,736,879 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 40 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t1337.6\t0\t33\n",
      "27\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t5\t0.0\t1\t4 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1115 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1337.6\t0\t5\n",
      "4\t1\t334.4\t0\t1\n",
      "5\t3\t83.6\t0\t3\n",
      "6\t4\t20.9\t0\t4\n",
      "7\t31\t5.2\t0\t31\n",
      "8\t21\t1.3\t0\t21\n",
      "9\t76\t0.3\t0\t76\n",
      "10\t176\t0.1\t1\t176\n",
      "11\t17\t0.0\t1\t17\n",
      "12\t13\t0.0\t1\t13\n",
      "13\t13\t0.0\t1\t13\n",
      "14\t19\t0.0\t1\t19\n",
      "15\t28\t0.0\t1\t28\n",
      "16\t51\t0.0\t1\t48 3\n",
      "17\t88\t0.0\t1\t85 3\n",
      "18\t163\t0.0\t1\t155 6 2\n",
      "19\t336\t0.0\t1\t224 111 1\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t62\t0.0\t2\t0 62\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/533.H_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/533.H_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/533.H_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/533.H_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.09 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,067\n",
      "  Read 1 with adapter:                      54 (0.1%)\n",
      "  Read 2 with adapter:                   3,637 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        86,067 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,974,919 bp\n",
      "  Read 1:    19,153,512 bp\n",
      "  Read 2:    19,821,407 bp\n",
      "Total written (filtered):     38,929,101 bp (99.9%)\n",
      "  Read 1:    19,152,238 bp\n",
      "  Read 2:    19,776,863 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 54 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t1344.8\t0\t46\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3637 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1344.8\t0\t29\n",
      "4\t3\t336.2\t0\t3\n",
      "5\t21\t84.0\t0\t21\n",
      "6\t33\t21.0\t0\t33\n",
      "7\t161\t5.3\t0\t161\n",
      "8\t189\t1.3\t0\t189\n",
      "9\t825\t0.3\t0\t823 2\n",
      "10\t1036\t0.1\t1\t1031 5\n",
      "11\t37\t0.0\t1\t37\n",
      "12\t49\t0.0\t1\t47 2\n",
      "13\t27\t0.0\t1\t26 1\n",
      "14\t60\t0.0\t1\t58 2\n",
      "15\t68\t0.0\t1\t66 2\n",
      "16\t88\t0.0\t1\t87 1\n",
      "17\t221\t0.0\t1\t210 11\n",
      "18\t236\t0.0\t1\t221 9 6\n",
      "19\t486\t0.0\t1\t312 174\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t53\t0.0\t2\t0 52 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/534.H_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/534.H_50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/534.H_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/534.H_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.01 s (21 us/read; 2.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             95,880\n",
      "  Read 1 with adapter:                      37 (0.0%)\n",
      "  Read 2 with adapter:                   1,971 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        95,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,522,129 bp\n",
      "  Read 1:    21,400,405 bp\n",
      "  Read 2:    22,121,724 bp\n",
      "Total written (filtered):     43,493,910 bp (99.9%)\n",
      "  Read 1:    21,399,750 bp\n",
      "  Read 2:    22,094,160 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 37 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t1498.1\t0\t34\n",
      "120\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1971 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1498.1\t0\t12\n",
      "4\t2\t374.5\t0\t2\n",
      "5\t5\t93.6\t0\t5\n",
      "6\t12\t23.4\t0\t12\n",
      "7\t34\t5.9\t0\t34\n",
      "8\t57\t1.5\t0\t57\n",
      "9\t237\t0.4\t0\t237\n",
      "10\t522\t0.1\t1\t514 8\n",
      "11\t35\t0.0\t1\t35\n",
      "12\t20\t0.0\t1\t20\n",
      "13\t18\t0.0\t1\t18\n",
      "14\t47\t0.0\t1\t44 3\n",
      "15\t46\t0.0\t1\t45 1\n",
      "16\t66\t0.0\t1\t63 3\n",
      "17\t173\t0.0\t1\t172 1\n",
      "18\t190\t0.0\t1\t179 9 2\n",
      "19\t412\t0.0\t1\t270 142\n",
      "20\t18\t0.0\t2\t0 0 18\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/535.H_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/535.H_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/535.H_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/535.H_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.47 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,799\n",
      "  Read 1 with adapter:                      46 (0.1%)\n",
      "  Read 2 with adapter:                   2,319 (4.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,799 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,174,575 bp\n",
      "  Read 1:    11,858,096 bp\n",
      "  Read 2:    12,316,479 bp\n",
      "Total written (filtered):     24,142,814 bp (99.9%)\n",
      "  Read 1:    11,854,797 bp\n",
      "  Read 2:    12,288,017 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 46 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t840.6\t0\t28\n",
      "27\t3\t0.0\t1\t3\n",
      "120\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t0 1\n",
      "210\t1\t0.0\t1\t0 1\n",
      "213\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t8\t0.0\t1\t8\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2319 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t840.6\t0\t10\n",
      "4\t3\t210.2\t0\t3\n",
      "5\t8\t52.5\t0\t8\n",
      "6\t9\t13.1\t0\t9\n",
      "7\t84\t3.3\t0\t84\n",
      "8\t137\t0.8\t0\t137\n",
      "9\t489\t0.2\t0\t489\n",
      "10\t743\t0.1\t1\t736 7\n",
      "11\t28\t0.0\t1\t28\n",
      "12\t28\t0.0\t1\t28\n",
      "13\t22\t0.0\t1\t22\n",
      "14\t43\t0.0\t1\t42 1\n",
      "15\t53\t0.0\t1\t53\n",
      "16\t52\t0.0\t1\t52\n",
      "17\t122\t0.0\t1\t116 6\n",
      "18\t147\t0.0\t1\t133 12 2\n",
      "19\t305\t0.0\t1\t197 107 1\n",
      "20\t9\t0.0\t2\t0 0 9\n",
      "21\t25\t0.0\t2\t0 25\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/536.H_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/536.H_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/536.H_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/536.H_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.22 s (28 us/read; 2.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,977\n",
      "  Read 1 with adapter:                      89 (0.1%)\n",
      "  Read 2 with adapter:                   1,699 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        79,977 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,329,215 bp\n",
      "  Read 1:    17,865,372 bp\n",
      "  Read 2:    18,463,843 bp\n",
      "Total written (filtered):     36,301,285 bp (99.9%)\n",
      "  Read 1:    17,860,774 bp\n",
      "  Read 2:    18,440,511 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 89 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t1249.6\t0\t47\n",
      "18\t1\t0.0\t1\t1\n",
      "100\t14\t0.0\t1\t14\n",
      "101\t8\t0.0\t1\t8\n",
      "103\t8\t0.0\t1\t7 1\n",
      "119\t9\t0.0\t1\t9\n",
      "120\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1699 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1249.6\t0\t6\n",
      "4\t3\t312.4\t0\t3\n",
      "5\t2\t78.1\t0\t2\n",
      "6\t12\t19.5\t0\t12\n",
      "7\t38\t4.9\t0\t38\n",
      "8\t49\t1.2\t0\t49\n",
      "9\t190\t0.3\t0\t189 1\n",
      "10\t457\t0.1\t1\t457\n",
      "11\t39\t0.0\t1\t38 1\n",
      "12\t32\t0.0\t1\t32\n",
      "13\t23\t0.0\t1\t23\n",
      "14\t32\t0.0\t1\t31 1\n",
      "15\t67\t0.0\t1\t65 2\n",
      "16\t64\t0.0\t1\t64\n",
      "17\t138\t0.0\t1\t132 6\n",
      "18\t201\t0.0\t1\t194 7\n",
      "19\t284\t0.0\t1\t196 87 1\n",
      "20\t7\t0.0\t2\t0 0 7\n",
      "21\t54\t0.0\t2\t0 54\n",
      "59\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/537.H_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/537.H_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/537.H_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/537.H_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.99 s (26 us/read; 2.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,791\n",
      "  Read 1 with adapter:                      48 (0.1%)\n",
      "  Read 2 with adapter:                   1,884 (2.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,791 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,382,324 bp\n",
      "  Read 1:    16,902,308 bp\n",
      "  Read 2:    17,480,016 bp\n",
      "Total written (filtered):     34,355,752 bp (99.9%)\n",
      "  Read 1:    16,901,121 bp\n",
      "  Read 2:    17,454,631 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 48 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t1184.2\t0\t39\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t1\n",
      "179\t1\t0.0\t1\t1\n",
      "202\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1884 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1184.2\t0\t11\n",
      "5\t5\t74.0\t0\t5\n",
      "6\t10\t18.5\t0\t10\n",
      "7\t35\t4.6\t0\t35\n",
      "8\t51\t1.2\t0\t51\n",
      "9\t264\t0.3\t0\t264\n",
      "10\t576\t0.1\t1\t570 6\n",
      "11\t39\t0.0\t1\t37 2\n",
      "12\t28\t0.0\t1\t27 1\n",
      "13\t21\t0.0\t1\t21\n",
      "14\t20\t0.0\t1\t20\n",
      "15\t48\t0.0\t1\t46 2\n",
      "16\t58\t0.0\t1\t55 3\n",
      "17\t133\t0.0\t1\t129 4\n",
      "18\t215\t0.0\t1\t207 6 2\n",
      "19\t295\t0.0\t1\t213 82\n",
      "20\t12\t0.0\t2\t0 1 11\n",
      "21\t58\t0.0\t2\t0 58\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "98\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/538.H_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/538.H_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/538.H_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/538.H_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.29 s (30 us/read; 2.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,135\n",
      "  Read 1 with adapter:                      19 (0.0%)\n",
      "  Read 2 with adapter:                     822 (1.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        43,135 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,559,476 bp\n",
      "  Read 1:     9,617,023 bp\n",
      "  Read 2:     9,942,453 bp\n",
      "Total written (filtered):     19,547,974 bp (99.9%)\n",
      "  Read 1:     9,616,966 bp\n",
      "  Read 2:     9,931,008 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t674.0\t0\t19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 822 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t674.0\t0\t2\n",
      "5\t2\t42.1\t0\t2\n",
      "6\t8\t10.5\t0\t8\n",
      "7\t21\t2.6\t0\t21\n",
      "8\t23\t0.7\t0\t23\n",
      "9\t83\t0.2\t0\t83\n",
      "10\t212\t0.0\t1\t210 2\n",
      "11\t19\t0.0\t1\t19\n",
      "12\t14\t0.0\t1\t14\n",
      "13\t11\t0.0\t1\t11\n",
      "14\t8\t0.0\t1\t8\n",
      "15\t19\t0.0\t1\t19\n",
      "16\t33\t0.0\t1\t33\n",
      "17\t83\t0.0\t1\t81 2\n",
      "18\t118\t0.0\t1\t116 2\n",
      "19\t140\t0.0\t1\t89 49 2\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t21\t0.0\t2\t0 21\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/539.H_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/539.H_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/539.H_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/539.H_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.77 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            177,643\n",
      "  Read 1 with adapter:                      93 (0.1%)\n",
      "  Read 2 with adapter:                   6,883 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       177,643 (100.0%)\n",
      "\n",
      "Total basepairs processed:    80,905,461 bp\n",
      "  Read 1:    39,780,844 bp\n",
      "  Read 2:    41,124,617 bp\n",
      "Total written (filtered):     80,820,633 bp (99.9%)\n",
      "  Read 1:    39,779,636 bp\n",
      "  Read 2:    41,040,997 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 93 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t84\t2775.7\t0\t84\n",
      "24\t2\t0.0\t1\t2\n",
      "27\t2\t0.0\t1\t2\n",
      "92\t1\t0.0\t1\t1\n",
      "122\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "222\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6883 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t2775.7\t0\t50\n",
      "4\t10\t693.9\t0\t10\n",
      "5\t24\t173.5\t0\t24\n",
      "6\t85\t43.4\t0\t85\n",
      "7\t302\t10.8\t0\t302\n",
      "8\t372\t2.7\t0\t372\n",
      "9\t1448\t0.7\t0\t1445 3\n",
      "10\t2089\t0.2\t1\t2076 13\n",
      "11\t84\t0.0\t1\t84\n",
      "12\t69\t0.0\t1\t67 2\n",
      "13\t55\t0.0\t1\t55\n",
      "14\t120\t0.0\t1\t118 2\n",
      "15\t152\t0.0\t1\t149 3\n",
      "16\t186\t0.0\t1\t175 11\n",
      "17\t426\t0.0\t1\t414 12\n",
      "18\t428\t0.0\t1\t403 21 4\n",
      "19\t848\t0.0\t1\t579 267 2\n",
      "20\t19\t0.0\t2\t0 1 18\n",
      "21\t113\t0.0\t2\t0 112 1\n",
      "147\t1\t0.0\t2\t1\n",
      "211\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/540.H_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/540.H_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/540.H_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/540.H_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.84 s (21 us/read; 2.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            232,341\n",
      "  Read 1 with adapter:                      82 (0.0%)\n",
      "  Read 2 with adapter:                   8,550 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       232,341 (100.0%)\n",
      "\n",
      "Total basepairs processed:   105,782,221 bp\n",
      "  Read 1:    52,026,042 bp\n",
      "  Read 2:    53,756,179 bp\n",
      "Total written (filtered):    105,675,879 bp (99.9%)\n",
      "  Read 1:    52,025,491 bp\n",
      "  Read 2:    53,650,388 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 82 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t3630.3\t0\t77\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "118\t1\t0.0\t1\t1\n",
      "126\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8550 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t3630.3\t0\t58\n",
      "4\t16\t907.6\t0\t16\n",
      "5\t41\t226.9\t0\t41\n",
      "6\t65\t56.7\t0\t65\n",
      "7\t326\t14.2\t0\t326\n",
      "8\t443\t3.5\t0\t443\n",
      "9\t1825\t0.9\t0\t1820 5\n",
      "10\t2504\t0.2\t1\t2498 6\n",
      "11\t146\t0.1\t1\t146\n",
      "12\t115\t0.0\t1\t114 1\n",
      "13\t106\t0.0\t1\t106\n",
      "14\t157\t0.0\t1\t155 2\n",
      "15\t161\t0.0\t1\t160 1\n",
      "16\t219\t0.0\t1\t213 6\n",
      "17\t520\t0.0\t1\t505 15\n",
      "18\t587\t0.0\t1\t550 28 9\n",
      "19\t1085\t0.0\t1\t706 377 2\n",
      "20\t21\t0.0\t2\t0 1 20\n",
      "21\t146\t0.0\t2\t0 145 1\n",
      "95\t1\t0.0\t2\t1\n",
      "155\t1\t0.0\t2\t0 1\n",
      "160\t1\t0.0\t2\t1\n",
      "172\t1\t0.0\t2\t1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "230\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/541.H_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/541.H_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/541.H_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/541.H_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.51 s (19 us/read; 3.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            232,297\n",
      "  Read 1 with adapter:                      95 (0.0%)\n",
      "  Read 2 with adapter:                   8,537 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       232,297 (100.0%)\n",
      "\n",
      "Total basepairs processed:   105,771,273 bp\n",
      "  Read 1:    52,019,272 bp\n",
      "  Read 2:    53,752,001 bp\n",
      "Total written (filtered):    105,667,657 bp (99.9%)\n",
      "  Read 1:    52,018,226 bp\n",
      "  Read 2:    53,649,431 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 95 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t3629.6\t0\t85\n",
      "6\t1\t56.7\t0\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t4\t0.0\t1\t4\n",
      "193\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "222\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8537 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t70\t3629.6\t0\t70\n",
      "4\t6\t907.4\t0\t6\n",
      "5\t31\t226.9\t0\t31\n",
      "6\t94\t56.7\t0\t94\n",
      "7\t317\t14.2\t0\t317\n",
      "8\t503\t3.5\t0\t503\n",
      "9\t1844\t0.9\t0\t1842 2\n",
      "10\t2716\t0.2\t1\t2699 17\n",
      "11\t121\t0.1\t1\t121\n",
      "12\t76\t0.0\t1\t76\n",
      "13\t66\t0.0\t1\t65 1\n",
      "14\t114\t0.0\t1\t112 2\n",
      "15\t158\t0.0\t1\t154 4\n",
      "16\t202\t0.0\t1\t197 5\n",
      "17\t475\t0.0\t1\t462 13\n",
      "18\t559\t0.0\t1\t525 19 15\n",
      "19\t1004\t0.0\t1\t650 354\n",
      "20\t15\t0.0\t2\t0 0 15\n",
      "21\t162\t0.0\t2\t0 161 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t1\n",
      "147\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/542.H_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/542.H_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/542.H_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/542.H_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.47 s (27 us/read; 2.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,848\n",
      "  Read 1 with adapter:                      60 (0.1%)\n",
      "  Read 2 with adapter:                   3,580 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        92,848 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,217,339 bp\n",
      "  Read 1:    20,742,234 bp\n",
      "  Read 2:    21,475,105 bp\n",
      "Total written (filtered):     42,172,735 bp (99.9%)\n",
      "  Read 1:    20,740,871 bp\n",
      "  Read 2:    21,431,864 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 60 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t1450.8\t0\t54\n",
      "120\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3580 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t1450.8\t0\t20\n",
      "4\t7\t362.7\t0\t7\n",
      "5\t12\t90.7\t0\t12\n",
      "6\t31\t22.7\t0\t31\n",
      "7\t164\t5.7\t0\t164\n",
      "8\t202\t1.4\t0\t202\n",
      "9\t764\t0.4\t0\t762 2\n",
      "10\t1105\t0.1\t1\t1103 2\n",
      "11\t37\t0.0\t1\t36 1\n",
      "12\t35\t0.0\t1\t34 1\n",
      "13\t23\t0.0\t1\t23\n",
      "14\t64\t0.0\t1\t64\n",
      "15\t91\t0.0\t1\t88 3\n",
      "16\t116\t0.0\t1\t112 4\n",
      "17\t214\t0.0\t1\t209 5\n",
      "18\t232\t0.0\t1\t220 6 6\n",
      "19\t392\t0.0\t1\t242 149 1\n",
      "20\t14\t0.0\t2\t0 1 13\n",
      "21\t55\t0.0\t2\t0 55\n",
      "61\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/544.H_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/544.H_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/544.H_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/544.H_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.56 s (28 us/read; 2.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,268\n",
      "  Read 1 with adapter:                      47 (0.1%)\n",
      "  Read 2 with adapter:                   1,977 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,268 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,522,757 bp\n",
      "  Read 1:    12,539,157 bp\n",
      "  Read 2:    12,983,600 bp\n",
      "Total written (filtered):     25,496,222 bp (99.9%)\n",
      "  Read 1:    12,538,408 bp\n",
      "  Read 2:    12,957,814 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 47 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t879.2\t0\t44\n",
      "185\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1977 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t879.2\t0\t5\n",
      "4\t1\t219.8\t0\t1\n",
      "5\t7\t54.9\t0\t7\n",
      "6\t18\t13.7\t0\t18\n",
      "7\t73\t3.4\t0\t73\n",
      "8\t99\t0.9\t0\t99\n",
      "9\t358\t0.2\t0\t358\n",
      "10\t534\t0.1\t1\t532 2\n",
      "11\t35\t0.0\t1\t35\n",
      "12\t21\t0.0\t1\t21\n",
      "13\t28\t0.0\t1\t28\n",
      "14\t43\t0.0\t1\t43\n",
      "15\t49\t0.0\t1\t45 4\n",
      "16\t64\t0.0\t1\t64\n",
      "17\t141\t0.0\t1\t135 6\n",
      "18\t132\t0.0\t1\t119 11 2\n",
      "19\t332\t0.0\t1\t209 122 1\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t27\t0.0\t2\t0 27\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A1_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A1_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A1_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A1_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.31 s (18 us/read; 3.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            239,591\n",
      "  Read 1 with adapter:                     155 (0.1%)\n",
      "  Read 2 with adapter:                  13,566 (5.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       239,591 (100.0%)\n",
      "\n",
      "Total basepairs processed:   109,261,691 bp\n",
      "  Read 1:    53,651,957 bp\n",
      "  Read 2:    55,609,734 bp\n",
      "Total written (filtered):    109,106,461 bp (99.9%)\n",
      "  Read 1:    53,650,879 bp\n",
      "  Read 2:    55,455,582 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 155 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t148\t3743.6\t0\t148\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t0 1\n",
      "202\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13566 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t3743.6\t0\t77\n",
      "4\t20\t935.9\t0\t20\n",
      "5\t66\t234.0\t0\t66\n",
      "6\t142\t58.5\t0\t142\n",
      "7\t834\t14.6\t0\t834\n",
      "8\t982\t3.7\t0\t982\n",
      "9\t3563\t0.9\t0\t3559 4\n",
      "10\t4099\t0.2\t1\t4078 21\n",
      "11\t105\t0.1\t1\t105\n",
      "12\t120\t0.0\t1\t118 2\n",
      "13\t92\t0.0\t1\t92\n",
      "14\t159\t0.0\t1\t158 1\n",
      "15\t231\t0.0\t1\t221 10\n",
      "16\t293\t0.0\t1\t283 10\n",
      "17\t592\t0.0\t1\t571 21\n",
      "18\t741\t0.0\t1\t680 41 20\n",
      "19\t1247\t0.0\t1\t779 464 4\n",
      "20\t27\t0.0\t2\t0 0 27\n",
      "21\t173\t0.0\t2\t0 170 3\n",
      "61\t1\t0.0\t2\t1\n",
      "192\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A2_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A2_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A2_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A2_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.76 s (17 us/read; 3.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            215,401\n",
      "  Read 1 with adapter:                     157 (0.1%)\n",
      "  Read 2 with adapter:                  12,102 (5.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       215,401 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,225,533 bp\n",
      "  Read 1:    48,234,456 bp\n",
      "  Read 2:    49,991,077 bp\n",
      "Total written (filtered):     98,086,315 bp (99.9%)\n",
      "  Read 1:    48,233,460 bp\n",
      "  Read 2:    49,852,855 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 157 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t151\t3365.6\t0\t151\n",
      "20\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "173\t1\t0.0\t1\t1\n",
      "180\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12102 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t3365.6\t0\t72\n",
      "4\t16\t841.4\t0\t16\n",
      "5\t54\t210.4\t0\t54\n",
      "6\t129\t52.6\t0\t129\n",
      "7\t743\t13.1\t0\t743\n",
      "8\t799\t3.3\t0\t799\n",
      "9\t2993\t0.8\t0\t2992 1\n",
      "10\t3876\t0.2\t1\t3860 16\n",
      "11\t120\t0.1\t1\t119 1\n",
      "12\t103\t0.0\t1\t103\n",
      "13\t99\t0.0\t1\t99\n",
      "14\t168\t0.0\t1\t166 2\n",
      "15\t216\t0.0\t1\t212 4\n",
      "16\t255\t0.0\t1\t250 5\n",
      "17\t565\t0.0\t1\t554 11\n",
      "18\t649\t0.0\t1\t611 23 15\n",
      "19\t1080\t0.0\t1\t731 347 2\n",
      "20\t26\t0.0\t2\t0 1 25\n",
      "21\t128\t0.0\t2\t0 128\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "180\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "223\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A3_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A3_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A3_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A3_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.35 s (17 us/read; 3.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            253,059\n",
      "  Read 1 with adapter:                     129 (0.1%)\n",
      "  Read 2 with adapter:                  15,070 (6.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       253,059 (100.0%)\n",
      "\n",
      "Total basepairs processed:   115,414,700 bp\n",
      "  Read 1:    56,664,416 bp\n",
      "  Read 2:    58,750,284 bp\n",
      "Total written (filtered):    115,242,194 bp (99.9%)\n",
      "  Read 1:    56,663,523 bp\n",
      "  Read 2:    58,578,671 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 129 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t121\t3954.0\t0\t121\n",
      "21\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "81\t1\t0.0\t1\t0 1\n",
      "92\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "157\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15070 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t96\t3954.0\t0\t96\n",
      "4\t24\t988.5\t0\t24\n",
      "5\t61\t247.1\t0\t61\n",
      "6\t158\t61.8\t0\t158\n",
      "7\t922\t15.4\t0\t922\n",
      "8\t1024\t3.9\t0\t1024\n",
      "9\t3958\t1.0\t0\t3954 4\n",
      "10\t4575\t0.2\t1\t4563 12\n",
      "11\t170\t0.1\t1\t168 2\n",
      "12\t148\t0.0\t1\t145 3\n",
      "13\t142\t0.0\t1\t142\n",
      "14\t233\t0.0\t1\t223 10\n",
      "15\t281\t0.0\t1\t271 10\n",
      "16\t302\t0.0\t1\t291 11\n",
      "17\t659\t0.0\t1\t639 20\n",
      "18\t776\t0.0\t1\t722 34 20\n",
      "19\t1336\t0.0\t1\t820 511 5\n",
      "20\t31\t0.0\t2\t1 0 30\n",
      "21\t164\t0.0\t2\t0 164\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "159\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t0 1\n",
      "221\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B1_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B1_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B1_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B1_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.01 s (18 us/read; 3.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            217,041\n",
      "  Read 1 with adapter:                     170 (0.1%)\n",
      "  Read 2 with adapter:                  11,882 (5.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       217,041 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,969,868 bp\n",
      "  Read 1:    48,600,518 bp\n",
      "  Read 2:    50,369,350 bp\n",
      "Total written (filtered):     98,832,126 bp (99.9%)\n",
      "  Read 1:    48,599,758 bp\n",
      "  Read 2:    50,232,368 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 170 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t167\t3391.3\t0\t167\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11882 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t82\t3391.3\t0\t82\n",
      "4\t10\t847.8\t0\t10\n",
      "5\t62\t212.0\t0\t62\n",
      "6\t124\t53.0\t0\t124\n",
      "7\t713\t13.2\t0\t713\n",
      "8\t765\t3.3\t0\t765\n",
      "9\t2932\t0.8\t0\t2930 2\n",
      "10\t3762\t0.2\t1\t3746 16\n",
      "11\t129\t0.1\t1\t128 1\n",
      "12\t114\t0.0\t1\t113 1\n",
      "13\t110\t0.0\t1\t107 3\n",
      "14\t163\t0.0\t1\t158 5\n",
      "15\t209\t0.0\t1\t202 7\n",
      "16\t251\t0.0\t1\t242 9\n",
      "17\t524\t0.0\t1\t508 16\n",
      "18\t669\t0.0\t1\t640 21 8\n",
      "19\t1088\t0.0\t1\t714 373 1\n",
      "20\t25\t0.0\t2\t0 2 23\n",
      "21\t140\t0.0\t2\t0 138 2\n",
      "83\t1\t0.0\t2\t1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t1\n",
      "171\t1\t0.0\t2\t1\n",
      "206\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t3\n",
      "217\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/533.H_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/533.H_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/533.H_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/533.H_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/534.H_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/534.H_50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/534.H_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/534.H_50_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/535.H_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/535.H_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/535.H_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/535.H_29_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/536.H_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/536.H_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/536.H_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/536.H_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/537.H_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/537.H_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/537.H_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/537.H_35_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/538.H_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/538.H_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/538.H_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/538.H_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/539.H_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/539.H_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/539.H_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/539.H_33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/540.H_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/540.H_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/540.H_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/540.H_27_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/541.H_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/541.H_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/541.H_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/541.H_20_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/542.H_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/542.H_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/542.H_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/542.H_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/544.H_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/544.H_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/544.H_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/544.H_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A1_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A1_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A1_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A1_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A2_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A2_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A2_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A2_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A3_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/A3_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A3_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/A3_7_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B1_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B1_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B1_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B1_3_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B2_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B2_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B2_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B2_42_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B2_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B2_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B2_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B2_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.14 s (18 us/read; 3.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            235,514\n",
      "  Read 1 with adapter:                     174 (0.1%)\n",
      "  Read 2 with adapter:                  12,625 (5.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       235,514 (100.0%)\n",
      "\n",
      "Total basepairs processed:   107,389,001 bp\n",
      "  Read 1:    52,739,291 bp\n",
      "  Read 2:    54,649,710 bp\n",
      "Total written (filtered):    107,241,623 bp (99.9%)\n",
      "  Read 1:    52,738,189 bp\n",
      "  Read 2:    54,503,434 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 174 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t169\t3679.9\t0\t169\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "186\t1\t0.0\t1\t0 1\n",
      "201\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12625 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t78\t3679.9\t0\t78\n",
      "4\t16\t920.0\t0\t16\n",
      "5\t60\t230.0\t0\t60\n",
      "6\t120\t57.5\t0\t120\n",
      "7\t732\t14.4\t0\t732\n",
      "8\t809\t3.6\t0\t809\n",
      "9\t3168\t0.9\t0\t3166 2\n",
      "10\t3809\t0.2\t1\t3790 19\n",
      "11\t143\t0.1\t1\t141 2\n",
      "12\t119\t0.0\t1\t117 2\n",
      "13\t124\t0.0\t1\t122 2\n",
      "14\t209\t0.0\t1\t206 3\n",
      "15\t235\t0.0\t1\t232 3\n",
      "16\t275\t0.0\t1\t268 7\n",
      "17\t619\t0.0\t1\t592 27\n",
      "18\t734\t0.0\t1\t694 26 14\n",
      "19\t1179\t0.0\t1\t733 444 2\n",
      "20\t26\t0.0\t2\t0 0 26\n",
      "21\t156\t0.0\t2\t0 155 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "35\t2\t0.0\t2\t2\n",
      "43\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t1\n",
      "157\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t2 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B3_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B3_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B3_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B3_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.23 s (18 us/read; 3.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            229,650\n",
      "  Read 1 with adapter:                     152 (0.1%)\n",
      "  Read 2 with adapter:                  13,745 (6.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       229,650 (100.0%)\n",
      "\n",
      "Total basepairs processed:   104,738,493 bp\n",
      "  Read 1:    51,420,950 bp\n",
      "  Read 2:    53,317,543 bp\n",
      "Total written (filtered):    104,583,021 bp (99.9%)\n",
      "  Read 1:    51,420,284 bp\n",
      "  Read 2:    53,162,737 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 152 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t145\t3588.3\t0\t145\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t2\t0.0\t1\t2\n",
      "75\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13745 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t3588.3\t0\t85\n",
      "4\t19\t897.1\t0\t19\n",
      "5\t68\t224.3\t0\t68\n",
      "6\t143\t56.1\t0\t143\n",
      "7\t928\t14.0\t0\t928\n",
      "8\t1066\t3.5\t0\t1066\n",
      "9\t3592\t0.9\t0\t3589 3\n",
      "10\t4122\t0.2\t1\t4100 22\n",
      "11\t130\t0.1\t1\t130\n",
      "12\t104\t0.0\t1\t103 1\n",
      "13\t111\t0.0\t1\t111\n",
      "14\t197\t0.0\t1\t191 6\n",
      "15\t231\t0.0\t1\t227 4\n",
      "16\t270\t0.0\t1\t262 8\n",
      "17\t626\t0.0\t1\t607 19\n",
      "18\t704\t0.0\t1\t671 25 8\n",
      "19\t1183\t0.0\t1\t748 433 2\n",
      "20\t28\t0.0\t2\t0 2 26\n",
      "21\t131\t0.0\t2\t0 131\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t1\n",
      "223\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C1_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C1_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C1_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C1_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.74 s (21 us/read; 2.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            128,449\n",
      "  Read 1 with adapter:                      80 (0.1%)\n",
      "  Read 2 with adapter:                   7,148 (5.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       128,449 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,572,033 bp\n",
      "  Read 1:    28,760,107 bp\n",
      "  Read 2:    29,811,926 bp\n",
      "Total written (filtered):     58,489,564 bp (99.9%)\n",
      "  Read 1:    28,759,722 bp\n",
      "  Read 2:    29,729,842 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 80 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t78\t2007.0\t0\t78\n",
      "27\t1\t0.0\t1\t1\n",
      "124\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7148 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t2007.0\t0\t39\n",
      "4\t7\t501.8\t0\t7\n",
      "5\t29\t125.4\t0\t29\n",
      "6\t86\t31.4\t0\t86\n",
      "7\t441\t7.8\t0\t441\n",
      "8\t521\t2.0\t0\t521\n",
      "9\t1802\t0.5\t0\t1800 2\n",
      "10\t2098\t0.1\t1\t2084 14\n",
      "11\t78\t0.0\t1\t78\n",
      "12\t58\t0.0\t1\t58\n",
      "13\t52\t0.0\t1\t51 1\n",
      "14\t119\t0.0\t1\t114 5\n",
      "15\t131\t0.0\t1\t128 3\n",
      "16\t178\t0.0\t1\t170 8\n",
      "17\t367\t0.0\t1\t358 9\n",
      "18\t381\t0.0\t1\t360 13 8\n",
      "19\t638\t0.0\t1\t400 236 2\n",
      "20\t19\t0.0\t2\t0 0 19\n",
      "21\t98\t0.0\t2\t0 98\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C2_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C2_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C2_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C2_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.12 s (19 us/read; 3.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            216,808\n",
      "  Read 1 with adapter:                     154 (0.1%)\n",
      "  Read 2 with adapter:                  12,761 (5.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       216,808 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,875,551 bp\n",
      "  Read 1:    48,544,794 bp\n",
      "  Read 2:    50,330,757 bp\n",
      "Total written (filtered):     98,727,944 bp (99.9%)\n",
      "  Read 1:    48,544,102 bp\n",
      "  Read 2:    50,183,842 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 154 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t150\t3387.6\t0\t150\n",
      "50\t2\t0.0\t1\t2\n",
      "55\t1\t0.0\t1\t1\n",
      "87\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12761 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t3387.6\t0\t72\n",
      "4\t21\t846.9\t0\t21\n",
      "5\t71\t211.7\t0\t71\n",
      "6\t107\t52.9\t0\t107\n",
      "7\t769\t13.2\t0\t769\n",
      "8\t858\t3.3\t0\t858\n",
      "9\t3212\t0.8\t0\t3209 3\n",
      "10\t3804\t0.2\t1\t3788 16\n",
      "11\t150\t0.1\t1\t148 2\n",
      "12\t148\t0.0\t1\t146 2\n",
      "13\t140\t0.0\t1\t138 2\n",
      "14\t232\t0.0\t1\t226 6\n",
      "15\t246\t0.0\t1\t239 7\n",
      "16\t267\t0.0\t1\t253 14\n",
      "17\t673\t0.0\t1\t649 24\n",
      "18\t715\t0.0\t1\t672 29 14\n",
      "19\t1111\t0.0\t1\t695 416\n",
      "20\t23\t0.0\t2\t0 1 22\n",
      "21\t131\t0.0\t2\t0 131\n",
      "22\t4\t0.0\t2\t0 1 3\n",
      "115\t2\t0.0\t2\t0 1 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t3\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C3_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C3_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C3_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C3_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.99 s (20 us/read; 3.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            150,392\n",
      "  Read 1 with adapter:                      88 (0.1%)\n",
      "  Read 2 with adapter:                   8,391 (5.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       150,392 (100.0%)\n",
      "\n",
      "Total basepairs processed:    68,580,697 bp\n",
      "  Read 1:    33,674,975 bp\n",
      "  Read 2:    34,905,722 bp\n",
      "Total written (filtered):     68,484,850 bp (99.9%)\n",
      "  Read 1:    33,674,585 bp\n",
      "  Read 2:    34,810,265 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 88 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t84\t2349.9\t0\t84\n",
      "27\t3\t0.0\t1\t3\n",
      "57\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8391 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t2349.9\t0\t50\n",
      "4\t12\t587.5\t0\t12\n",
      "5\t36\t146.9\t0\t36\n",
      "6\t109\t36.7\t0\t109\n",
      "7\t530\t9.2\t0\t530\n",
      "8\t596\t2.3\t0\t596\n",
      "9\t2083\t0.6\t0\t2079 4\n",
      "10\t2542\t0.1\t1\t2529 13\n",
      "11\t78\t0.0\t1\t77 1\n",
      "12\t81\t0.0\t1\t80 1\n",
      "13\t72\t0.0\t1\t71 1\n",
      "14\t160\t0.0\t1\t156 4\n",
      "15\t149\t0.0\t1\t145 4\n",
      "16\t222\t0.0\t1\t217 5\n",
      "17\t366\t0.0\t1\t357 9\n",
      "18\t461\t0.0\t1\t424 23 14\n",
      "19\t741\t0.0\t1\t467 274\n",
      "20\t21\t0.0\t2\t0 1 20\n",
      "21\t77\t0.0\t2\t0 77\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "30\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D1_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D1_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D1_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D1_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.12 s (18 us/read; 3.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            171,790\n",
      "  Read 1 with adapter:                     101 (0.1%)\n",
      "  Read 2 with adapter:                  10,298 (6.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       171,790 (100.0%)\n",
      "\n",
      "Total basepairs processed:    78,347,590 bp\n",
      "  Read 1:    38,462,783 bp\n",
      "  Read 2:    39,884,807 bp\n",
      "Total written (filtered):     78,230,562 bp (99.9%)\n",
      "  Read 1:    38,462,112 bp\n",
      "  Read 2:    39,768,450 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t99\t2684.2\t0\t99\n",
      "178\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10298 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t64\t2684.2\t0\t64\n",
      "4\t12\t671.1\t0\t12\n",
      "5\t51\t167.8\t0\t51\n",
      "6\t121\t41.9\t0\t121\n",
      "7\t660\t10.5\t0\t660\n",
      "8\t712\t2.6\t0\t712\n",
      "9\t2640\t0.7\t0\t2637 3\n",
      "10\t3175\t0.2\t1\t3164 11\n",
      "11\t88\t0.0\t1\t88\n",
      "12\t84\t0.0\t1\t84\n",
      "13\t82\t0.0\t1\t82\n",
      "14\t159\t0.0\t1\t155 4\n",
      "15\t180\t0.0\t1\t178 2\n",
      "16\t262\t0.0\t1\t258 4\n",
      "17\t506\t0.0\t1\t492 14\n",
      "18\t500\t0.0\t1\t474 17 9\n",
      "19\t868\t0.0\t1\t513 352 3\n",
      "20\t18\t0.0\t2\t0 0 18\n",
      "21\t114\t0.0\t2\t0 112 2\n",
      "147\t1\t0.0\t2\t1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D2_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D2_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D2_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D2_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.31 s (17 us/read; 3.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            250,484\n",
      "  Read 1 with adapter:                     154 (0.1%)\n",
      "  Read 2 with adapter:                  15,609 (6.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       250,484 (100.0%)\n",
      "\n",
      "Total basepairs processed:   114,251,196 bp\n",
      "  Read 1:    56,086,871 bp\n",
      "  Read 2:    58,164,325 bp\n",
      "Total written (filtered):    114,071,666 bp (99.8%)\n",
      "  Read 1:    56,086,099 bp\n",
      "  Read 2:    57,985,567 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 154 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t150\t3913.8\t0\t150\n",
      "27\t2\t0.0\t1\t2\n",
      "92\t1\t0.0\t1\t1\n",
      "176\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15609 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t3913.8\t0\t109\n",
      "4\t20\t978.5\t0\t20\n",
      "5\t68\t244.6\t0\t68\n",
      "6\t158\t61.2\t0\t158\n",
      "7\t975\t15.3\t0\t975\n",
      "8\t1002\t3.8\t0\t1002\n",
      "9\t3894\t1.0\t0\t3890 4\n",
      "10\t4940\t0.2\t1\t4918 22\n",
      "11\t154\t0.1\t1\t152 2\n",
      "12\t147\t0.0\t1\t143 4\n",
      "13\t142\t0.0\t1\t141 1\n",
      "14\t202\t0.0\t1\t192 10\n",
      "15\t321\t0.0\t1\t312 9\n",
      "16\t356\t0.0\t1\t345 11\n",
      "17\t744\t0.0\t1\t720 24\n",
      "18\t802\t0.0\t1\t744 39 19\n",
      "19\t1404\t0.0\t1\t859 543 2\n",
      "20\t23\t0.0\t2\t0 0 23\n",
      "21\t135\t0.0\t2\t0 135\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "223\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D3_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D3_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D3_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D3_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.00 s (20 us/read; 2.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            148,706\n",
      "  Read 1 with adapter:                      93 (0.1%)\n",
      "  Read 2 with adapter:                   8,972 (6.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       148,706 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,820,776 bp\n",
      "  Read 1:    33,294,395 bp\n",
      "  Read 2:    34,526,381 bp\n",
      "Total written (filtered):     67,717,868 bp (99.8%)\n",
      "  Read 1:    33,293,570 bp\n",
      "  Read 2:    34,424,298 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 93 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t87\t2323.5\t0\t87\n",
      "27\t1\t0.0\t1\t1\n",
      "101\t3\t0.0\t1\t2 1\n",
      "113\t1\t0.0\t1\t1\n",
      "121\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8972 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t2323.5\t0\t47\n",
      "4\t8\t580.9\t0\t8\n",
      "5\t45\t145.2\t0\t45\n",
      "6\t105\t36.3\t0\t105\n",
      "7\t570\t9.1\t0\t570\n",
      "8\t604\t2.3\t0\t604\n",
      "9\t2305\t0.6\t0\t2302 3\n",
      "10\t2737\t0.1\t1\t2724 13\n",
      "11\t98\t0.0\t1\t97 1\n",
      "12\t83\t0.0\t1\t81 2\n",
      "13\t66\t0.0\t1\t66\n",
      "14\t142\t0.0\t1\t138 4\n",
      "15\t146\t0.0\t1\t145 1\n",
      "16\t228\t0.0\t1\t219 9\n",
      "17\t438\t0.0\t1\t424 14\n",
      "18\t425\t0.0\t1\t401 21 3\n",
      "19\t790\t0.0\t1\t496 293 1\n",
      "20\t25\t0.0\t2\t0 2 23\n",
      "21\t106\t0.0\t2\t0 106\n",
      "22\t1\t0.0\t2\t0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E1_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E1_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E1_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E1_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.81 s (17 us/read; 3.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            339,477\n",
      "  Read 1 with adapter:                     206 (0.1%)\n",
      "  Read 2 with adapter:                  14,954 (4.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       339,477 (100.0%)\n",
      "\n",
      "Total basepairs processed:   154,723,525 bp\n",
      "  Read 1:    76,018,058 bp\n",
      "  Read 2:    78,705,467 bp\n",
      "Total written (filtered):    154,532,204 bp (99.9%)\n",
      "  Read 1:    76,016,793 bp\n",
      "  Read 2:    78,515,411 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 206 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t197\t5304.3\t0\t197\n",
      "24\t3\t0.0\t1\t3\n",
      "25\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "50\t1\t0.0\t1\t0 1\n",
      "76\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t0 1\n",
      "214\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14954 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t5304.3\t0\t47\n",
      "4\t13\t1326.1\t0\t13\n",
      "5\t54\t331.5\t0\t54\n",
      "6\t90\t82.9\t0\t90\n",
      "7\t493\t20.7\t0\t493\n",
      "8\t664\t5.2\t0\t664\n",
      "9\t2756\t1.3\t0\t2755 1\n",
      "10\t4361\t0.3\t1\t4343 18\n",
      "11\t184\t0.1\t1\t179 5\n",
      "12\t194\t0.0\t1\t188 6\n",
      "13\t200\t0.0\t1\t198 2\n",
      "14\t394\t0.0\t1\t383 11\n",
      "15\t426\t0.0\t1\t411 15\n",
      "16\t602\t0.0\t1\t584 18\n",
      "17\t1081\t0.0\t1\t1047 34\n",
      "18\t1130\t0.0\t1\t1054 50 26\n",
      "19\t1981\t0.0\t1\t1165 815 1\n",
      "20\t41\t0.0\t2\t1 1 39\n",
      "21\t230\t0.0\t2\t0 229 1\n",
      "22\t5\t0.0\t2\t0 1 4\n",
      "43\t1\t0.0\t2\t0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "92\t1\t0.0\t2\t1\n",
      "211\t1\t0.0\t2\t0 1\n",
      "213\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E2_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E2_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E2_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E2_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.05 s (17 us/read; 3.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            234,975\n",
      "  Read 1 with adapter:                     133 (0.1%)\n",
      "  Read 2 with adapter:                  14,143 (6.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       234,975 (100.0%)\n",
      "\n",
      "Total basepairs processed:   107,169,820 bp\n",
      "  Read 1:    52,614,272 bp\n",
      "  Read 2:    54,555,548 bp\n",
      "Total written (filtered):    107,007,618 bp (99.8%)\n",
      "  Read 1:    52,613,036 bp\n",
      "  Read 2:    54,394,582 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 133 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t123\t3671.5\t0\t123\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t2\t0.0\t1\t2\n",
      "55\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "127\t2\t0.0\t1\t2\n",
      "214\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14143 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t116\t3671.5\t0\t116\n",
      "4\t17\t917.9\t0\t17\n",
      "5\t56\t229.5\t0\t56\n",
      "6\t136\t57.4\t0\t136\n",
      "7\t862\t14.3\t0\t862\n",
      "8\t995\t3.6\t0\t995\n",
      "9\t3475\t0.9\t0\t3472 3\n",
      "10\t4568\t0.2\t1\t4542 26\n",
      "11\t154\t0.1\t1\t154\n",
      "12\t127\t0.0\t1\t126 1\n",
      "13\t110\t0.0\t1\t110\n",
      "14\t201\t0.0\t1\t195 6\n",
      "15\t240\t0.0\t1\t235 5\n",
      "16\t298\t0.0\t1\t282 16\n",
      "17\t597\t0.0\t1\t587 10\n",
      "18\t742\t0.0\t1\t706 27 9\n",
      "19\t1273\t0.0\t1\t809 463 1\n",
      "20\t15\t0.0\t2\t0 1 14\n",
      "21\t153\t0.0\t2\t0 151 2\n",
      "92\t1\t0.0\t2\t1\n",
      "118\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "215\t2\t0.0\t2\t2\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "224\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E3_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E3_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E3_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E3_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.97 s (18 us/read; 3.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            221,357\n",
      "  Read 1 with adapter:                     149 (0.1%)\n",
      "  Read 2 with adapter:                  13,168 (5.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       221,357 (100.0%)\n",
      "\n",
      "Total basepairs processed:   100,955,136 bp\n",
      "  Read 1:    49,564,581 bp\n",
      "  Read 2:    51,390,555 bp\n",
      "Total written (filtered):    100,805,596 bp (99.9%)\n",
      "  Read 1:    49,563,086 bp\n",
      "  Read 2:    51,242,510 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 149 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t141\t3458.7\t0\t141\n",
      "23\t1\t0.0\t1\t1\n",
      "81\t1\t0.0\t1\t1\n",
      "87\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "154\t1\t0.0\t1\t0 1\n",
      "193\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13168 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t107\t3458.7\t0\t107\n",
      "4\t15\t864.7\t0\t15\n",
      "5\t67\t216.2\t0\t67\n",
      "6\t153\t54.0\t0\t153\n",
      "7\t937\t13.5\t0\t937\n",
      "8\t894\t3.4\t0\t894\n",
      "9\t3489\t0.8\t0\t3488 1\n",
      "10\t4030\t0.2\t1\t4018 12\n",
      "11\t123\t0.1\t1\t122 1\n",
      "12\t145\t0.0\t1\t144 1\n",
      "13\t88\t0.0\t1\t87 1\n",
      "14\t173\t0.0\t1\t168 5\n",
      "15\t201\t0.0\t1\t196 5\n",
      "16\t259\t0.0\t1\t252 7\n",
      "17\t533\t0.0\t1\t511 22\n",
      "18\t631\t0.0\t1\t602 22 7\n",
      "19\t1134\t0.0\t1\t743 390 1\n",
      "20\t27\t0.0\t2\t0 0 27\n",
      "21\t152\t0.0\t2\t0 152\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "205\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t2 1\n",
      "240\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F1_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F1_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F1_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F1_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.26 s (18 us/read; 3.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            239,359\n",
      "  Read 1 with adapter:                     149 (0.1%)\n",
      "  Read 2 with adapter:                  14,092 (5.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       239,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:   109,160,178 bp\n",
      "  Read 1:    53,595,337 bp\n",
      "  Read 2:    55,564,841 bp\n",
      "Total written (filtered):    108,998,440 bp (99.9%)\n",
      "  Read 1:    53,594,285 bp\n",
      "  Read 2:    55,404,155 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 149 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t143\t3740.0\t0\t143\n",
      "27\t3\t0.0\t1\t3\n",
      "124\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14092 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t3740.0\t0\t80\n",
      "4\t15\t935.0\t0\t15\n",
      "5\t77\t233.7\t0\t77\n",
      "6\t157\t58.4\t0\t157\n",
      "7\t911\t14.6\t0\t911\n",
      "8\t935\t3.7\t0\t935\n",
      "9\t3573\t0.9\t0\t3571 2\n",
      "10\t4389\t0.2\t1\t4368 21\n",
      "11\t140\t0.1\t1\t137 3\n",
      "12\t134\t0.0\t1\t133 1\n",
      "13\t125\t0.0\t1\t124 1\n",
      "14\t184\t0.0\t1\t178 6\n",
      "15\t227\t0.0\t1\t225 2\n",
      "16\t281\t0.0\t1\t266 15\n",
      "17\t611\t0.0\t1\t594 17\n",
      "18\t754\t0.0\t1\t717 25 12\n",
      "19\t1278\t0.0\t1\t808 468 2\n",
      "20\t27\t0.0\t2\t0 0 27\n",
      "21\t185\t0.0\t2\t0 184 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F2_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F2_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F2_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F2_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.00 s (20 us/read; 3.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            153,038\n",
      "  Read 1 with adapter:                     113 (0.1%)\n",
      "  Read 2 with adapter:                   9,024 (5.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       153,038 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,795,053 bp\n",
      "  Read 1:    34,268,069 bp\n",
      "  Read 2:    35,526,984 bp\n",
      "Total written (filtered):     69,691,064 bp (99.9%)\n",
      "  Read 1:    34,267,595 bp\n",
      "  Read 2:    35,423,469 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t2391.2\t0\t109\n",
      "27\t2\t0.0\t1\t2\n",
      "38\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9024 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t2391.2\t0\t53\n",
      "4\t12\t597.8\t0\t12\n",
      "5\t59\t149.5\t0\t59\n",
      "6\t123\t37.4\t0\t123\n",
      "7\t585\t9.3\t0\t585\n",
      "8\t734\t2.3\t0\t734\n",
      "9\t2229\t0.6\t0\t2228 1\n",
      "10\t2708\t0.1\t1\t2692 16\n",
      "11\t87\t0.0\t1\t87\n",
      "12\t62\t0.0\t1\t61 1\n",
      "13\t79\t0.0\t1\t77 2\n",
      "14\t99\t0.0\t1\t94 5\n",
      "15\t147\t0.0\t1\t147\n",
      "16\t226\t0.0\t1\t217 9\n",
      "17\t436\t0.0\t1\t424 12\n",
      "18\t449\t0.0\t1\t425 17 7\n",
      "19\t798\t0.0\t1\t520 278\n",
      "20\t18\t0.0\t2\t0 1 17\n",
      "21\t107\t0.0\t2\t0 106 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "115\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t1\n",
      "167\t1\t0.0\t2\t0 1\n",
      "174\t1\t0.0\t2\t0 1\n",
      "213\t1\t0.0\t2\t0 0 1\n",
      "214\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F3_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F3_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F3_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F3_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.10 s (18 us/read; 3.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            223,395\n",
      "  Read 1 with adapter:                     140 (0.1%)\n",
      "  Read 2 with adapter:                  13,104 (5.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       223,395 (100.0%)\n",
      "\n",
      "Total basepairs processed:   101,877,284 bp\n",
      "  Read 1:    50,018,947 bp\n",
      "  Read 2:    51,858,337 bp\n",
      "Total written (filtered):    101,725,360 bp (99.9%)\n",
      "  Read 1:    50,017,963 bp\n",
      "  Read 2:    51,707,397 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 140 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t133\t3490.5\t0\t133\n",
      "27\t2\t0.0\t1\t2\n",
      "55\t1\t0.0\t1\t1\n",
      "92\t2\t0.0\t1\t2\n",
      "101\t1\t0.0\t1\t1\n",
      "191\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13104 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t117\t3490.5\t0\t117\n",
      "4\t22\t872.6\t0\t22\n",
      "5\t57\t218.2\t0\t57\n",
      "6\t136\t54.5\t0\t136\n",
      "7\t744\t13.6\t0\t744\n",
      "8\t820\t3.4\t0\t820\n",
      "9\t3119\t0.9\t0\t3115 4\n",
      "10\t4276\t0.2\t1\t4260 16\n",
      "11\t124\t0.1\t1\t123 1\n",
      "12\t121\t0.0\t1\t119 2\n",
      "13\t124\t0.0\t1\t124\n",
      "14\t196\t0.0\t1\t184 12\n",
      "15\t251\t0.0\t1\t247 4\n",
      "16\t292\t0.0\t1\t283 9\n",
      "17\t616\t0.0\t1\t601 15\n",
      "18\t688\t0.0\t1\t645 32 11\n",
      "19\t1217\t0.0\t1\t776 439 2\n",
      "20\t19\t0.0\t2\t0 0 19\n",
      "21\t153\t0.0\t2\t0 153\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t1\n",
      "99\t2\t0.0\t2\t2\n",
      "103\t1\t0.0\t2\t1\n",
      "124\t1\t0.0\t2\t0 1\n",
      "196\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G1_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G1_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G1_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G1_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.04 s (18 us/read; 3.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            285,517\n",
      "  Read 1 with adapter:                     188 (0.1%)\n",
      "  Read 2 with adapter:                  16,692 (5.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       285,517 (100.0%)\n",
      "\n",
      "Total basepairs processed:   130,209,605 bp\n",
      "  Read 1:    63,931,776 bp\n",
      "  Read 2:    66,277,829 bp\n",
      "Total written (filtered):    130,020,839 bp (99.9%)\n",
      "  Read 1:    63,930,259 bp\n",
      "  Read 2:    66,090,580 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 188 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t176\t4461.2\t0\t176\n",
      "16\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "55\t5\t0.0\t1\t4 1\n",
      "119\t1\t0.0\t1\t1\n",
      "145\t1\t0.0\t1\t0 1\n",
      "179\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 16692 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t106\t4461.2\t0\t106\n",
      "4\t31\t1115.3\t0\t31\n",
      "5\t104\t278.8\t0\t104\n",
      "6\t222\t69.7\t0\t222\n",
      "7\t1128\t17.4\t0\t1128\n",
      "8\t1220\t4.4\t0\t1220\n",
      "9\t4398\t1.1\t0\t4393 5\n",
      "10\t5100\t0.3\t1\t5078 22\n",
      "11\t160\t0.1\t1\t157 3\n",
      "12\t131\t0.0\t1\t129 2\n",
      "13\t109\t0.0\t1\t107 2\n",
      "14\t165\t0.0\t1\t159 6\n",
      "15\t270\t0.0\t1\t264 6\n",
      "16\t304\t0.0\t1\t292 12\n",
      "17\t678\t0.0\t1\t655 23\n",
      "18\t864\t0.0\t1\t814 41 9\n",
      "19\t1452\t0.0\t1\t953 494 5\n",
      "20\t35\t0.0\t2\t1 3 31\n",
      "21\t205\t0.0\t2\t0 204 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "58\t1\t0.0\t2\t1\n",
      "118\t1\t0.0\t2\t1\n",
      "159\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G2_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G2_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G2_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G2_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.78 s (17 us/read; 3.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            288,191\n",
      "  Read 1 with adapter:                     193 (0.1%)\n",
      "  Read 2 with adapter:                  18,607 (6.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       288,191 (100.0%)\n",
      "\n",
      "Total basepairs processed:   131,458,541 bp\n",
      "  Read 1:    64,525,361 bp\n",
      "  Read 2:    66,933,180 bp\n",
      "Total written (filtered):    131,247,965 bp (99.8%)\n",
      "  Read 1:    64,524,245 bp\n",
      "  Read 2:    66,723,720 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 193 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t186\t4503.0\t0\t186\n",
      "6\t1\t70.4\t0\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "96\t1\t0.0\t1\t1\n",
      "168\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18607 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t114\t4503.0\t0\t114\n",
      "4\t25\t1125.7\t0\t25\n",
      "5\t105\t281.4\t0\t105\n",
      "6\t208\t70.4\t0\t208\n",
      "7\t1297\t17.6\t0\t1297\n",
      "8\t1328\t4.4\t0\t1328\n",
      "9\t4874\t1.1\t0\t4874\n",
      "10\t5635\t0.3\t1\t5607 28\n",
      "11\t177\t0.1\t1\t174 3\n",
      "12\t185\t0.0\t1\t181 4\n",
      "13\t185\t0.0\t1\t181 4\n",
      "14\t240\t0.0\t1\t231 9\n",
      "15\t347\t0.0\t1\t339 8\n",
      "16\t346\t0.0\t1\t334 12\n",
      "17\t735\t0.0\t1\t707 28\n",
      "18\t931\t0.0\t1\t883 33 15\n",
      "19\t1655\t0.0\t1\t1063 587 5\n",
      "20\t37\t0.0\t2\t0 2 35\n",
      "21\t170\t0.0\t2\t0 169 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "98\t1\t0.0\t2\t1\n",
      "99\t1\t0.0\t2\t1\n",
      "126\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t2 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G3_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G3_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G3_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G3_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.33 s (19 us/read; 3.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            179,568\n",
      "  Read 1 with adapter:                     123 (0.1%)\n",
      "  Read 2 with adapter:                  11,219 (6.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       179,568 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,907,766 bp\n",
      "  Read 1:    40,208,906 bp\n",
      "  Read 2:    41,698,860 bp\n",
      "Total written (filtered):     81,780,480 bp (99.8%)\n",
      "  Read 1:    40,208,436 bp\n",
      "  Read 2:    41,572,044 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 123 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t121\t2805.8\t0\t121\n",
      "27\t1\t0.0\t1\t1\n",
      "80\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11219 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t65\t2805.8\t0\t65\n",
      "4\t13\t701.4\t0\t13\n",
      "5\t63\t175.4\t0\t63\n",
      "6\t126\t43.8\t0\t126\n",
      "7\t752\t11.0\t0\t752\n",
      "8\t797\t2.7\t0\t797\n",
      "9\t2794\t0.7\t0\t2791 3\n",
      "10\t3412\t0.2\t1\t3400 12\n",
      "11\t134\t0.0\t1\t132 2\n",
      "12\t99\t0.0\t1\t99\n",
      "13\t127\t0.0\t1\t127\n",
      "14\t197\t0.0\t1\t193 4\n",
      "15\t194\t0.0\t1\t187 7\n",
      "16\t272\t0.0\t1\t261 11\n",
      "17\t533\t0.0\t1\t515 18\n",
      "18\t541\t0.0\t1\t507 18 16\n",
      "19\t969\t0.0\t1\t591 378\n",
      "20\t23\t0.0\t2\t1 1 21\n",
      "21\t103\t0.0\t2\t0 102 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "126\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B3_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/B3_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B3_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/B3_14_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C1_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C1_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C1_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C1_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C2_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C2_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C2_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C2_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C3_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/C3_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C3_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/C3_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D1_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D1_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D1_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D1_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D2_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D2_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D2_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D2_11_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D3_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/D3_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D3_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/D3_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E1_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E1_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E1_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E1_8_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E2_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E2_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E2_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E2_15_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E3_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/E3_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E3_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/E3_1_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F1_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F1_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F1_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F1_2_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F2_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F2_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F2_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F2_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F3_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/F3_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F3_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/F3_25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G1_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G1_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G1_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G1_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G2_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G2_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G2_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G2_21_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G3_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/G3_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G3_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/G3_16_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H1_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H1_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H1_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H1_9_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H1_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H1_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H1_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H1_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.50 s (19 us/read; 3.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            184,208\n",
      "  Read 1 with adapter:                     146 (0.1%)\n",
      "  Read 2 with adapter:                  10,389 (5.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       184,208 (100.0%)\n",
      "\n",
      "Total basepairs processed:    84,002,649 bp\n",
      "  Read 1:    41,248,066 bp\n",
      "  Read 2:    42,754,583 bp\n",
      "Total written (filtered):     83,882,601 bp (99.9%)\n",
      "  Read 1:    41,247,387 bp\n",
      "  Read 2:    42,635,214 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 146 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t141\t2878.2\t0\t141\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t1\n",
      "73\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10389 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t57\t2878.2\t0\t57\n",
      "4\t20\t719.6\t0\t20\n",
      "5\t57\t179.9\t0\t57\n",
      "6\t118\t45.0\t0\t118\n",
      "7\t594\t11.2\t0\t594\n",
      "8\t699\t2.8\t0\t699\n",
      "9\t2550\t0.7\t0\t2548 2\n",
      "10\t3214\t0.2\t1\t3207 7\n",
      "11\t131\t0.0\t1\t131\n",
      "12\t97\t0.0\t1\t95 2\n",
      "13\t117\t0.0\t1\t116 1\n",
      "14\t143\t0.0\t1\t137 6\n",
      "15\t207\t0.0\t1\t205 2\n",
      "16\t251\t0.0\t1\t242 9\n",
      "17\t516\t0.0\t1\t493 23\n",
      "18\t506\t0.0\t1\t472 24 10\n",
      "19\t948\t0.0\t1\t588 359 1\n",
      "20\t24\t0.0\t2\t0 1 23\n",
      "21\t130\t0.0\t2\t0 127 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "61\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "104\t1\t0.0\t2\t0 1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H2_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H2_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H2_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H2_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.23 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            225,562\n",
      "  Read 1 with adapter:                     123 (0.1%)\n",
      "  Read 2 with adapter:                  13,702 (6.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       225,562 (100.0%)\n",
      "\n",
      "Total basepairs processed:   102,876,433 bp\n",
      "  Read 1:    50,504,139 bp\n",
      "  Read 2:    52,372,294 bp\n",
      "Total written (filtered):    102,719,136 bp (99.8%)\n",
      "  Read 1:    50,503,178 bp\n",
      "  Read 2:    52,215,958 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 123 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t118\t3524.4\t0\t118\n",
      "24\t1\t0.0\t1\t1\n",
      "122\t2\t0.0\t1\t2\n",
      "123\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13702 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t68\t3524.4\t0\t68\n",
      "4\t17\t881.1\t0\t17\n",
      "5\t78\t220.3\t0\t78\n",
      "6\t154\t55.1\t0\t154\n",
      "7\t837\t13.8\t0\t837\n",
      "8\t942\t3.4\t0\t942\n",
      "9\t3469\t0.9\t0\t3469\n",
      "10\t4247\t0.2\t1\t4230 17\n",
      "11\t134\t0.1\t1\t132 2\n",
      "12\t136\t0.0\t1\t133 3\n",
      "13\t110\t0.0\t1\t109 1\n",
      "14\t204\t0.0\t1\t201 3\n",
      "15\t266\t0.0\t1\t259 7\n",
      "16\t323\t0.0\t1\t317 6\n",
      "17\t661\t0.0\t1\t641 20\n",
      "18\t687\t0.0\t1\t648 25 14\n",
      "19\t1189\t0.0\t1\t733 451 5\n",
      "20\t27\t0.0\t2\t0 2 25\n",
      "21\t144\t0.0\t2\t0 144\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "199\t1\t0.0\t2\t0 1\n",
      "200\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H3_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H3_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H3_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H3_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.74 s (19 us/read; 3.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            253,391\n",
      "  Read 1 with adapter:                     136 (0.1%)\n",
      "  Read 2 with adapter:                  13,433 (5.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       253,391 (100.0%)\n",
      "\n",
      "Total basepairs processed:   115,527,741 bp\n",
      "  Read 1:    56,735,690 bp\n",
      "  Read 2:    58,792,051 bp\n",
      "Total written (filtered):    115,371,009 bp (99.9%)\n",
      "  Read 1:    56,734,394 bp\n",
      "  Read 2:    58,636,615 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 136 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t129\t3959.2\t0\t129\n",
      "92\t1\t0.0\t1\t1\n",
      "106\t1\t0.0\t1\t1\n",
      "116\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "143\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13433 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t3959.2\t0\t91\n",
      "4\t18\t989.8\t0\t18\n",
      "5\t73\t247.5\t0\t73\n",
      "6\t138\t61.9\t0\t138\n",
      "7\t770\t15.5\t0\t770\n",
      "8\t810\t3.9\t0\t810\n",
      "9\t3161\t1.0\t0\t3154 7\n",
      "10\t4411\t0.2\t1\t4393 18\n",
      "11\t139\t0.1\t1\t138 1\n",
      "12\t148\t0.0\t1\t147 1\n",
      "13\t104\t0.0\t1\t103 1\n",
      "14\t198\t0.0\t1\t191 7\n",
      "15\t270\t0.0\t1\t261 9\n",
      "16\t289\t0.0\t1\t277 12\n",
      "17\t632\t0.0\t1\t619 13\n",
      "18\t746\t0.0\t1\t707 28 11\n",
      "19\t1218\t0.0\t1\t771 444 3\n",
      "20\t21\t0.0\t2\t0 1 20\n",
      "21\t185\t0.0\t2\t0 185\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "137\t1\t0.0\t2\t1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "215\t3\t0.0\t2\t2 1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H2_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H2_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H2_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H2_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H3_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xpojr54y/H3_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H3_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-0hj801fs/37d5646e-c4db-48d7-9ce9-445c73849a46/data/H3_17_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGmicro2018_337-38.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/332.D_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/332.D_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/332.D_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/332.D_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.64 s (16 us/read; 3.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            229,104\n",
      "  Read 1 with adapter:                     139 (0.1%)\n",
      "  Read 2 with adapter:                  15,418 (6.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       229,104 (100.0%)\n",
      "\n",
      "Total basepairs processed:   104,515,373 bp\n",
      "  Read 1:    51,299,051 bp\n",
      "  Read 2:    53,216,322 bp\n",
      "Total written (filtered):    104,345,091 bp (99.8%)\n",
      "  Read 1:    51,298,101 bp\n",
      "  Read 2:    53,046,990 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 139 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t134\t3579.8\t0\t134\n",
      "92\t1\t0.0\t1\t1\n",
      "99\t1\t0.0\t1\t0 1\n",
      "100\t2\t0.0\t1\t2\n",
      "157\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15418 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t112\t3579.8\t0\t112\n",
      "4\t19\t894.9\t0\t19\n",
      "5\t68\t223.7\t0\t68\n",
      "6\t189\t55.9\t0\t189\n",
      "7\t972\t14.0\t0\t972\n",
      "8\t1144\t3.5\t0\t1144\n",
      "9\t4474\t0.9\t0\t4473 1\n",
      "10\t4765\t0.2\t1\t4749 16\n",
      "11\t141\t0.1\t1\t141\n",
      "12\t141\t0.0\t1\t139 2\n",
      "13\t108\t0.0\t1\t107 1\n",
      "14\t181\t0.0\t1\t173 8\n",
      "15\t236\t0.0\t1\t234 2\n",
      "16\t263\t0.0\t1\t259 4\n",
      "17\t516\t0.0\t1\t506 10\n",
      "18\t597\t0.0\t1\t566 23 8\n",
      "19\t1346\t0.0\t1\t856 489 1\n",
      "20\t23\t0.0\t2\t1 2 20\n",
      "21\t120\t0.0\t2\t0 120\n",
      "139\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/332.U_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/332.U_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/332.U_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/332.U_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.50 s (27 us/read; 2.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,468\n",
      "  Read 1 with adapter:                      51 (0.1%)\n",
      "  Read 2 with adapter:                   2,041 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,468 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,958,945 bp\n",
      "  Read 1:    12,255,238 bp\n",
      "  Read 2:    12,703,707 bp\n",
      "Total written (filtered):     24,929,921 bp (99.9%)\n",
      "  Read 1:    12,253,301 bp\n",
      "  Read 2:    12,676,620 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 51 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t866.7\t0\t40\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t7\t0.0\t1\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2041 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t866.7\t0\t5\n",
      "4\t1\t216.7\t0\t1\n",
      "5\t3\t54.2\t0\t3\n",
      "6\t14\t13.5\t0\t14\n",
      "7\t66\t3.4\t0\t66\n",
      "8\t85\t0.8\t0\t85\n",
      "9\t393\t0.2\t0\t393\n",
      "10\t547\t0.1\t1\t544 3\n",
      "11\t22\t0.0\t1\t21 1\n",
      "12\t33\t0.0\t1\t33\n",
      "13\t34\t0.0\t1\t33 1\n",
      "14\t51\t0.0\t1\t49 2\n",
      "15\t61\t0.0\t1\t59 2\n",
      "16\t80\t0.0\t1\t76 4\n",
      "17\t134\t0.0\t1\t130 4\n",
      "18\t141\t0.0\t1\t131 8 2\n",
      "19\t311\t0.0\t1\t206 104 1\n",
      "20\t15\t0.0\t2\t0 0 15\n",
      "21\t38\t0.0\t2\t0 38\n",
      "115\t5\t0.0\t2\t5\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/333.D_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/333.D_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/333.D_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/333.D_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.44 s (20 us/read; 2.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            170,023\n",
      "  Read 1 with adapter:                      98 (0.1%)\n",
      "  Read 2 with adapter:                  10,278 (6.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       170,023 (100.0%)\n",
      "\n",
      "Total basepairs processed:    77,420,660 bp\n",
      "  Read 1:    38,005,737 bp\n",
      "  Read 2:    39,414,923 bp\n",
      "Total written (filtered):     77,300,611 bp (99.8%)\n",
      "  Read 1:    38,004,641 bp\n",
      "  Read 2:    39,295,970 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 98 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t2656.6\t0\t91\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "120\t3\t0.0\t1\t3\n",
      "195\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10278 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t2656.6\t0\t47\n",
      "4\t22\t664.2\t0\t22\n",
      "5\t47\t166.0\t0\t47\n",
      "6\t98\t41.5\t0\t98\n",
      "7\t510\t10.4\t0\t510\n",
      "8\t693\t2.6\t0\t693\n",
      "9\t2635\t0.6\t0\t2633 2\n",
      "10\t2997\t0.2\t1\t2980 17\n",
      "11\t141\t0.0\t1\t141\n",
      "12\t152\t0.0\t1\t150 2\n",
      "13\t145\t0.0\t1\t141 4\n",
      "14\t227\t0.0\t1\t220 7\n",
      "15\t227\t0.0\t1\t219 8\n",
      "16\t276\t0.0\t1\t269 7\n",
      "17\t461\t0.0\t1\t445 16\n",
      "18\t460\t0.0\t1\t431 20 9\n",
      "19\t987\t0.0\t1\t644 342 1\n",
      "20\t16\t0.0\t2\t0 0 16\n",
      "21\t129\t0.0\t2\t0 128 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "126\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "211\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/333.U_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/333.U_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/333.U_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/333.U_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.58 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,349\n",
      "  Read 1 with adapter:                      31 (0.1%)\n",
      "  Read 2 with adapter:                   2,236 (3.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        59,349 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,738,754 bp\n",
      "  Read 1:    13,129,384 bp\n",
      "  Read 2:    13,609,370 bp\n",
      "Total written (filtered):     26,708,136 bp (99.9%)\n",
      "  Read 1:    13,128,021 bp\n",
      "  Read 2:    13,580,115 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 31 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t927.3\t0\t24\n",
      "25\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2236 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t927.3\t0\t18\n",
      "4\t1\t231.8\t0\t1\n",
      "5\t5\t58.0\t0\t5\n",
      "6\t17\t14.5\t0\t17\n",
      "7\t74\t3.6\t0\t74\n",
      "8\t90\t0.9\t0\t90\n",
      "9\t399\t0.2\t0\t398 1\n",
      "10\t573\t0.1\t1\t572 1\n",
      "11\t28\t0.0\t1\t28\n",
      "12\t35\t0.0\t1\t34 1\n",
      "13\t43\t0.0\t1\t43\n",
      "14\t71\t0.0\t1\t66 5\n",
      "15\t67\t0.0\t1\t64 3\n",
      "16\t92\t0.0\t1\t86 6\n",
      "17\t145\t0.0\t1\t141 4\n",
      "18\t158\t0.0\t1\t141 9 8\n",
      "19\t377\t0.0\t1\t246 130 1\n",
      "20\t8\t0.0\t2\t0 0 8\n",
      "21\t33\t0.0\t2\t0 33\n",
      "212\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/334.D_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/334.D_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/334.D_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/334.D_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.96 s (17 us/read; 3.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            238,634\n",
      "  Read 1 with adapter:                     167 (0.1%)\n",
      "  Read 2 with adapter:                  14,246 (6.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       238,634 (100.0%)\n",
      "\n",
      "Total basepairs processed:   108,825,172 bp\n",
      "  Read 1:    53,436,358 bp\n",
      "  Read 2:    55,388,814 bp\n",
      "Total written (filtered):    108,657,333 bp (99.8%)\n",
      "  Read 1:    53,432,081 bp\n",
      "  Read 2:    55,225,252 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 167 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t129\t3728.7\t0\t129\n",
      "92\t1\t0.0\t1\t1\n",
      "100\t29\t0.0\t1\t27 2\n",
      "101\t3\t0.0\t1\t3\n",
      "102\t3\t0.0\t1\t3\n",
      "103\t1\t0.0\t1\t1\n",
      "186\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14246 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t112\t3728.7\t0\t112\n",
      "4\t15\t932.2\t0\t15\n",
      "5\t65\t233.0\t0\t65\n",
      "6\t184\t58.3\t0\t184\n",
      "7\t732\t14.6\t0\t732\n",
      "8\t975\t3.6\t0\t975\n",
      "9\t3777\t0.9\t0\t3773 4\n",
      "10\t4275\t0.2\t1\t4255 20\n",
      "11\t158\t0.1\t1\t158\n",
      "12\t148\t0.0\t1\t146 2\n",
      "13\t160\t0.0\t1\t159 1\n",
      "14\t216\t0.0\t1\t212 4\n",
      "15\t303\t0.0\t1\t290 13\n",
      "16\t327\t0.0\t1\t311 16\n",
      "17\t577\t0.0\t1\t553 24\n",
      "18\t643\t0.0\t1\t604 27 12\n",
      "19\t1402\t0.0\t1\t973 427 2\n",
      "20\t28\t0.0\t2\t0 1 27\n",
      "21\t132\t0.0\t2\t0 130 2\n",
      "22\t6\t0.0\t2\t0 1 5\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "190\t1\t0.0\t2\t1\n",
      "204\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "217\t3\t0.0\t2\t3\n",
      "226\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/334.U_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/334.U_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/334.U_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/334.U_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.44 s (29 us/read; 2.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,557\n",
      "  Read 1 with adapter:                      33 (0.1%)\n",
      "  Read 2 with adapter:                   1,567 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,557 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,344,648 bp\n",
      "  Read 1:    10,977,071 bp\n",
      "  Read 2:    11,367,577 bp\n",
      "Total written (filtered):     22,323,536 bp (99.9%)\n",
      "  Read 1:    10,976,452 bp\n",
      "  Read 2:    11,347,084 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 33 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t774.3\t0\t28\n",
      "27\t1\t0.0\t1\t1\n",
      "38\t2\t0.0\t1\t2\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1567 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t774.3\t0\t9\n",
      "5\t2\t48.4\t0\t2\n",
      "6\t7\t12.1\t0\t7\n",
      "7\t48\t3.0\t0\t48\n",
      "8\t60\t0.8\t0\t60\n",
      "9\t292\t0.2\t0\t292\n",
      "10\t382\t0.0\t1\t380 2\n",
      "11\t41\t0.0\t1\t40 1\n",
      "12\t25\t0.0\t1\t25\n",
      "13\t28\t0.0\t1\t28\n",
      "14\t53\t0.0\t1\t52 1\n",
      "15\t50\t0.0\t1\t48 2\n",
      "16\t59\t0.0\t1\t58 1\n",
      "17\t129\t0.0\t1\t125 4\n",
      "18\t99\t0.0\t1\t84 12 3\n",
      "19\t254\t0.0\t1\t175 79\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t23\t0.0\t2\t0 23\n",
      "215\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/335.D_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/335.D_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/335.D_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/335.D_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.14 s (18 us/read; 3.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            119,829\n",
      "  Read 1 with adapter:                      92 (0.1%)\n",
      "  Read 2 with adapter:                   6,621 (5.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       119,829 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,508,530 bp\n",
      "  Read 1:    26,763,127 bp\n",
      "  Read 2:    27,745,403 bp\n",
      "Total written (filtered):     54,431,004 bp (99.9%)\n",
      "  Read 1:    26,762,487 bp\n",
      "  Read 2:    27,668,517 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 92 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t84\t1872.3\t0\t84\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t4\t0.0\t1\t4\n",
      "101\t1\t0.0\t1\t1\n",
      "130\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6621 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t1872.3\t0\t54\n",
      "4\t8\t468.1\t0\t8\n",
      "5\t25\t117.0\t0\t25\n",
      "6\t64\t29.3\t0\t64\n",
      "7\t360\t7.3\t0\t360\n",
      "8\t458\t1.8\t0\t458\n",
      "9\t1670\t0.5\t0\t1669 1\n",
      "10\t1930\t0.1\t1\t1919 11\n",
      "11\t81\t0.0\t1\t81\n",
      "12\t80\t0.0\t1\t80\n",
      "13\t75\t0.0\t1\t74 1\n",
      "14\t103\t0.0\t1\t100 3\n",
      "15\t150\t0.0\t1\t147 3\n",
      "16\t199\t0.0\t1\t194 5\n",
      "17\t299\t0.0\t1\t287 12\n",
      "18\t292\t0.0\t1\t277 9 6\n",
      "19\t674\t0.0\t1\t436 235 3\n",
      "20\t14\t0.0\t2\t0 1 13\n",
      "21\t80\t0.0\t2\t0 80\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "160\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/335.U_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/335.U_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/335.U_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/335.U_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.00 s (24 us/read; 2.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             83,348\n",
      "  Read 1 with adapter:                      38 (0.0%)\n",
      "  Read 2 with adapter:                   1,797 (2.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        83,348 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,696,462 bp\n",
      "  Read 1:    18,532,698 bp\n",
      "  Read 2:    19,163,764 bp\n",
      "Total written (filtered):     37,668,872 bp (99.9%)\n",
      "  Read 1:    18,531,541 bp\n",
      "  Read 2:    19,137,331 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 38 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t1302.3\t0\t32\n",
      "37\t1\t0.0\t1\t1\n",
      "192\t2\t0.0\t1\t1 1\n",
      "211\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1797 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1302.3\t0\t4\n",
      "4\t2\t325.6\t0\t2\n",
      "5\t3\t81.4\t0\t3\n",
      "6\t5\t20.3\t0\t5\n",
      "7\t29\t5.1\t0\t29\n",
      "8\t28\t1.3\t0\t28\n",
      "9\t159\t0.3\t0\t159\n",
      "10\t358\t0.1\t1\t357 1\n",
      "11\t27\t0.0\t1\t27\n",
      "12\t66\t0.0\t1\t66\n",
      "13\t42\t0.0\t1\t41 1\n",
      "14\t71\t0.0\t1\t70 1\n",
      "15\t78\t0.0\t1\t77 1\n",
      "16\t85\t0.0\t1\t83 2\n",
      "17\t155\t0.0\t1\t149 6\n",
      "18\t172\t0.0\t1\t164 5 3\n",
      "19\t458\t0.0\t1\t298 160\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t41\t0.0\t2\t0 40 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/336.D_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/336.D_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/336.D_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/336.D_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.72 s (17 us/read; 3.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            158,477\n",
      "  Read 1 with adapter:                     134 (0.1%)\n",
      "  Read 2 with adapter:                   9,769 (6.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       158,477 (100.0%)\n",
      "\n",
      "Total basepairs processed:    72,185,154 bp\n",
      "  Read 1:    35,436,337 bp\n",
      "  Read 2:    36,748,817 bp\n",
      "Total written (filtered):     72,071,690 bp (99.8%)\n",
      "  Read 1:    35,434,929 bp\n",
      "  Read 2:    36,636,761 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 134 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t119\t2476.2\t0\t119\n",
      "24\t4\t0.0\t1\t4\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "37\t1\t0.0\t1\t1\n",
      "100\t3\t0.0\t1\t3\n",
      "101\t1\t0.0\t1\t1\n",
      "103\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9769 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t2476.2\t0\t50\n",
      "4\t11\t619.1\t0\t11\n",
      "5\t29\t154.8\t0\t29\n",
      "6\t123\t38.7\t0\t123\n",
      "7\t524\t9.7\t0\t524\n",
      "8\t666\t2.4\t0\t666\n",
      "9\t2566\t0.6\t0\t2564 2\n",
      "10\t2871\t0.2\t1\t2854 17\n",
      "11\t101\t0.0\t1\t101\n",
      "12\t134\t0.0\t1\t133 1\n",
      "13\t130\t0.0\t1\t128 2\n",
      "14\t180\t0.0\t1\t175 5\n",
      "15\t202\t0.0\t1\t200 2\n",
      "16\t266\t0.0\t1\t256 10\n",
      "17\t408\t0.0\t1\t393 15\n",
      "18\t436\t0.0\t1\t398 30 8\n",
      "19\t964\t0.0\t1\t625 337 2\n",
      "20\t18\t0.0\t2\t0 0 18\n",
      "21\t83\t0.0\t2\t0 83\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "45\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/336U_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/336U_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/336U_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/336U_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.45 s (29 us/read; 2.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,446\n",
      "  Read 1 with adapter:                      40 (0.1%)\n",
      "  Read 2 with adapter:                   2,111 (4.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,446 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,136,257 bp\n",
      "  Read 1:    10,858,764 bp\n",
      "  Read 2:    11,277,493 bp\n",
      "Total written (filtered):     22,106,607 bp (99.9%)\n",
      "  Read 1:    10,857,302 bp\n",
      "  Read 2:    11,249,305 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 40 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t772.6\t0\t30\n",
      "23\t2\t0.0\t1\t2\n",
      "27\t2\t0.0\t1\t2\n",
      "192\t1\t0.0\t1\t1\n",
      "216\t5\t0.0\t1\t3 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2111 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t772.6\t0\t13\n",
      "4\t1\t193.1\t0\t1\n",
      "5\t3\t48.3\t0\t3\n",
      "6\t14\t12.1\t0\t14\n",
      "7\t61\t3.0\t0\t61\n",
      "8\t93\t0.8\t0\t93\n",
      "9\t328\t0.2\t0\t328\n",
      "10\t502\t0.0\t1\t499 3\n",
      "11\t29\t0.0\t1\t29\n",
      "12\t42\t0.0\t1\t42\n",
      "13\t56\t0.0\t1\t56\n",
      "14\t81\t0.0\t1\t80 1\n",
      "15\t96\t0.0\t1\t93 3\n",
      "16\t78\t0.0\t1\t72 6\n",
      "17\t156\t0.0\t1\t151 5\n",
      "18\t170\t0.0\t1\t155 10 5\n",
      "19\t328\t0.0\t1\t229 99\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t45\t0.0\t2\t0 45\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/337.H_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/337.H_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/337.H_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/337.H_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.61 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,619\n",
      "  Read 1 with adapter:                      44 (0.1%)\n",
      "  Read 2 with adapter:                   1,721 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        59,619 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,892,994 bp\n",
      "  Read 1:    13,214,228 bp\n",
      "  Read 2:    13,678,766 bp\n",
      "Total written (filtered):     26,867,986 bp (99.9%)\n",
      "  Read 1:    13,212,959 bp\n",
      "  Read 2:    13,655,027 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 44 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t931.5\t0\t37\n",
      "24\t1\t0.0\t1\t1\n",
      "124\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "213\t2\t0.0\t1\t2\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1721 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t931.5\t0\t5\n",
      "5\t3\t58.2\t0\t3\n",
      "6\t7\t14.6\t0\t7\n",
      "7\t35\t3.6\t0\t35\n",
      "8\t56\t0.9\t0\t56\n",
      "9\t240\t0.2\t0\t240\n",
      "10\t431\t0.1\t1\t429 2\n",
      "11\t38\t0.0\t1\t38\n",
      "12\t31\t0.0\t1\t31\n",
      "13\t30\t0.0\t1\t29 1\n",
      "14\t47\t0.0\t1\t45 2\n",
      "15\t66\t0.0\t1\t66\n",
      "16\t62\t0.0\t1\t61 1\n",
      "17\t146\t0.0\t1\t141 5\n",
      "18\t132\t0.0\t1\t127 4 1\n",
      "19\t345\t0.0\t1\t247 98\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t40\t0.0\t2\t0 40\n",
      "115\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/338.H_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/338.H_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/338.H_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/338.H_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.00 s (22 us/read; 2.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,590\n",
      "  Read 1 with adapter:                      65 (0.1%)\n",
      "  Read 2 with adapter:                   2,305 (2.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        91,590 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,449,887 bp\n",
      "  Read 1:    20,364,500 bp\n",
      "  Read 2:    21,085,387 bp\n",
      "Total written (filtered):     41,415,685 bp (99.9%)\n",
      "  Read 1:    20,363,151 bp\n",
      "  Read 2:    21,052,534 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 65 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t1431.1\t0\t56\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t2\t0.0\t1\t2\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2305 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t1431.1\t0\t14\n",
      "4\t2\t357.8\t0\t2\n",
      "5\t10\t89.4\t0\t10\n",
      "6\t6\t22.4\t0\t6\n",
      "7\t44\t5.6\t0\t44\n",
      "8\t89\t1.4\t0\t89\n",
      "9\t293\t0.3\t0\t292 1\n",
      "10\t531\t0.1\t1\t530 1\n",
      "11\t39\t0.0\t1\t39\n",
      "12\t38\t0.0\t1\t38\n",
      "13\t40\t0.0\t1\t39 1\n",
      "14\t59\t0.0\t1\t59\n",
      "15\t79\t0.0\t1\t75 4\n",
      "16\t85\t0.0\t1\t81 4\n",
      "17\t206\t0.0\t1\t197 9\n",
      "18\t195\t0.0\t1\t182 10 3\n",
      "19\t492\t0.0\t1\t353 137 2\n",
      "20\t8\t0.0\t2\t0 0 8\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t3\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/339.H_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/339.H_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/339.H_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/339.H_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.35 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            136,961\n",
      "  Read 1 with adapter:                      77 (0.1%)\n",
      "  Read 2 with adapter:                   2,471 (1.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       136,961 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,243,346 bp\n",
      "  Read 1:    30,608,912 bp\n",
      "  Read 2:    31,634,434 bp\n",
      "Total written (filtered):     62,205,392 bp (99.9%)\n",
      "  Read 1:    30,607,744 bp\n",
      "  Read 2:    31,597,648 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 77 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t68\t2140.0\t0\t68\n",
      "27\t1\t0.0\t1\t1\n",
      "103\t7\t0.0\t1\t6 1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2471 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t2140.0\t0\t7\n",
      "4\t3\t535.0\t0\t3\n",
      "5\t4\t133.8\t0\t4\n",
      "6\t12\t33.4\t0\t12\n",
      "7\t32\t8.4\t0\t32\n",
      "8\t44\t2.1\t0\t44\n",
      "9\t231\t0.5\t0\t231\n",
      "10\t477\t0.1\t1\t476 1\n",
      "11\t50\t0.0\t1\t50\n",
      "12\t52\t0.0\t1\t51 1\n",
      "13\t48\t0.0\t1\t48\n",
      "14\t45\t0.0\t1\t43 2\n",
      "15\t83\t0.0\t1\t83\n",
      "16\t101\t0.0\t1\t100 1\n",
      "17\t244\t0.0\t1\t239 5\n",
      "18\t297\t0.0\t1\t275 17 5\n",
      "19\t647\t0.0\t1\t456 190 1\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t81\t0.0\t2\t0 81\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/340.D_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/340.D_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/340.D_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/340.D_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.31 s (16 us/read; 3.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            205,938\n",
      "  Read 1 with adapter:                     148 (0.1%)\n",
      "  Read 2 with adapter:                   7,203 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       205,938 (100.0%)\n",
      "\n",
      "Total basepairs processed:    93,760,833 bp\n",
      "  Read 1:    46,084,961 bp\n",
      "  Read 2:    47,675,872 bp\n",
      "Total written (filtered):     93,670,355 bp (99.9%)\n",
      "  Read 1:    46,084,180 bp\n",
      "  Read 2:    47,586,175 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 148 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t142\t3217.8\t0\t142\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "37\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7203 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t3217.8\t0\t42\n",
      "4\t11\t804.4\t0\t11\n",
      "5\t12\t201.1\t0\t12\n",
      "6\t56\t50.3\t0\t56\n",
      "7\t231\t12.6\t0\t231\n",
      "8\t361\t3.1\t0\t361\n",
      "9\t1479\t0.8\t0\t1479\n",
      "10\t2247\t0.2\t1\t2233 14\n",
      "11\t101\t0.0\t1\t101\n",
      "12\t113\t0.0\t1\t111 2\n",
      "13\t74\t0.0\t1\t73 1\n",
      "14\t117\t0.0\t1\t115 2\n",
      "15\t134\t0.0\t1\t132 2\n",
      "16\t204\t0.0\t1\t198 6\n",
      "17\t369\t0.0\t1\t361 8\n",
      "18\t462\t0.0\t1\t428 28 6\n",
      "19\t1016\t0.0\t1\t728 288\n",
      "20\t20\t0.0\t2\t0 2 18\n",
      "21\t147\t0.0\t2\t0 146 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "216\t5\t0.0\t2\t5\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/340.U_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/340.U_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/340.U_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/340.U_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.49 s (21 us/read; 2.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            116,751\n",
      "  Read 1 with adapter:                      57 (0.0%)\n",
      "  Read 2 with adapter:                   1,396 (1.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       116,751 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,064,798 bp\n",
      "  Read 1:    26,109,099 bp\n",
      "  Read 2:    26,955,699 bp\n",
      "Total written (filtered):     53,041,257 bp (100.0%)\n",
      "  Read 1:    26,108,918 bp\n",
      "  Read 2:    26,932,339 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 57 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t1824.2\t0\t56\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1396 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1824.2\t0\t5\n",
      "4\t2\t456.1\t0\t2\n",
      "6\t2\t28.5\t0\t2\n",
      "7\t8\t7.1\t0\t8\n",
      "8\t13\t1.8\t0\t13\n",
      "9\t60\t0.4\t0\t60\n",
      "10\t124\t0.1\t1\t124\n",
      "11\t18\t0.0\t1\t18\n",
      "12\t33\t0.0\t1\t33\n",
      "13\t22\t0.0\t1\t22\n",
      "14\t34\t0.0\t1\t32 2\n",
      "15\t62\t0.0\t1\t62\n",
      "16\t86\t0.0\t1\t82 4\n",
      "17\t140\t0.0\t1\t136 4\n",
      "18\t195\t0.0\t1\t187 7 1\n",
      "19\t487\t0.0\t1\t354 132 1\n",
      "20\t16\t0.0\t2\t0 1 15\n",
      "21\t87\t0.0\t2\t0 86 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/332.D_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/332.D_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/332.D_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/332.D_17_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/332.U_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/332.U_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/332.U_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/332.U_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/333.D_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/333.D_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/333.D_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/333.D_8_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/333.U_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/333.U_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/333.U_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/333.U_33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/334.D_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/334.D_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/334.D_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/334.D_32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/334.U_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/334.U_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/334.U_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/334.U_11_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/335.D_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/335.D_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/335.D_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/335.D_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/335.U_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/335.U_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/335.U_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/335.U_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/336.D_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/336.D_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/336.D_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/336.D_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/336U_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/336U_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/336U_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/336U_6_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/337.H_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/337.H_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/337.H_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/337.H_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/338.H_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/338.H_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/338.H_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/338.H_29_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/339.H_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/339.H_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/339.H_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/339.H_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/340.D_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/340.D_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/340.D_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/340.D_35_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/340.U_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/340.U_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/340.U_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/340.U_7_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/341.D_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/341.D_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/341.D_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/341.D_1_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/341.D_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/341.D_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/341.D_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/341.D_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.01 s (43 us/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,750\n",
      "  Read 1 with adapter:                      13 (0.1%)\n",
      "  Read 2 with adapter:                   1,222 (5.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,750 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,480,535 bp\n",
      "  Read 1:     5,132,572 bp\n",
      "  Read 2:     5,347,963 bp\n",
      "Total written (filtered):     10,464,919 bp (99.9%)\n",
      "  Read 1:     5,132,014 bp\n",
      "  Read 2:     5,332,905 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t371.1\t0\t9\n",
      "25\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t0 1\n",
      "213\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1222 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t371.1\t0\t7\n",
      "5\t3\t23.2\t0\t3\n",
      "6\t14\t5.8\t0\t14\n",
      "7\t67\t1.4\t0\t67\n",
      "8\t95\t0.4\t0\t95\n",
      "9\t283\t0.1\t0\t282 1\n",
      "10\t278\t0.0\t1\t277 1\n",
      "11\t12\t0.0\t1\t11 1\n",
      "12\t12\t0.0\t1\t12\n",
      "13\t11\t0.0\t1\t11\n",
      "14\t16\t0.0\t1\t15 1\n",
      "15\t23\t0.0\t1\t21 2\n",
      "16\t47\t0.0\t1\t46 1\n",
      "17\t75\t0.0\t1\t73 2\n",
      "18\t76\t0.0\t1\t73 3\n",
      "19\t177\t0.0\t1\t110 67\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t21\t0.0\t2\t0 21\n",
      "115\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/341.U_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/341.U_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/341.U_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/341.U_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.08 s (38 us/read; 1.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,135\n",
      "  Read 1 with adapter:                      16 (0.1%)\n",
      "  Read 2 with adapter:                     926 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,135 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,432,047 bp\n",
      "  Read 1:     6,097,873 bp\n",
      "  Read 2:     6,334,174 bp\n",
      "Total written (filtered):     12,417,578 bp (99.9%)\n",
      "  Read 1:     6,096,146 bp\n",
      "  Read 2:     6,321,432 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t439.6\t0\t8\n",
      "204\t1\t0.0\t1\t0 1\n",
      "209\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "216\t5\t0.0\t1\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 926 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t439.6\t0\t5\n",
      "6\t7\t6.9\t0\t7\n",
      "7\t23\t1.7\t0\t23\n",
      "8\t24\t0.4\t0\t24\n",
      "9\t129\t0.1\t0\t129\n",
      "10\t220\t0.0\t1\t219 1\n",
      "11\t18\t0.0\t1\t18\n",
      "12\t32\t0.0\t1\t31 1\n",
      "13\t22\t0.0\t1\t22\n",
      "14\t38\t0.0\t1\t37 1\n",
      "15\t37\t0.0\t1\t36 1\n",
      "16\t23\t0.0\t1\t23\n",
      "17\t61\t0.0\t1\t59 2\n",
      "18\t82\t0.0\t1\t77 1 4\n",
      "19\t181\t0.0\t1\t117 64\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t21\t0.0\t2\t0 21\n",
      "215\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/342.D_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/342.D_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/342.D_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/342.D_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.17 s (32 us/read; 1.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,520\n",
      "  Read 1 with adapter:                      23 (0.1%)\n",
      "  Read 2 with adapter:                   1,715 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        36,520 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,262,212 bp\n",
      "  Read 1:     7,972,303 bp\n",
      "  Read 2:     8,289,909 bp\n",
      "Total written (filtered):     16,238,377 bp (99.9%)\n",
      "  Read 1:     7,970,722 bp\n",
      "  Read 2:     8,267,655 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t570.6\t0\t14\n",
      "13\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "213\t2\t0.0\t1\t1 1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1715 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t570.6\t0\t4\n",
      "4\t2\t142.7\t0\t2\n",
      "5\t3\t35.7\t0\t3\n",
      "6\t8\t8.9\t0\t8\n",
      "7\t82\t2.2\t0\t82\n",
      "8\t91\t0.6\t0\t91\n",
      "9\t358\t0.1\t0\t358\n",
      "10\t369\t0.0\t1\t367 2\n",
      "11\t23\t0.0\t1\t22 1\n",
      "12\t31\t0.0\t1\t30 1\n",
      "13\t30\t0.0\t1\t30\n",
      "14\t49\t0.0\t1\t48 1\n",
      "15\t60\t0.0\t1\t58 2\n",
      "16\t56\t0.0\t1\t55 1\n",
      "17\t104\t0.0\t1\t104\n",
      "18\t121\t0.0\t1\t108 9 4\n",
      "19\t288\t0.0\t1\t187 101\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t27\t0.0\t2\t0 27\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/342.U_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/342.U_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/342.U_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/342.U_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.38 s (31 us/read; 1.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,148\n",
      "  Read 1 with adapter:                      35 (0.1%)\n",
      "  Read 2 with adapter:                   1,625 (3.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,148 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,261,321 bp\n",
      "  Read 1:     9,950,261 bp\n",
      "  Read 2:    10,311,060 bp\n",
      "Total written (filtered):     20,238,651 bp (99.9%)\n",
      "  Read 1:     9,948,903 bp\n",
      "  Read 2:    10,289,748 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 35 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t705.4\t0\t29\n",
      "192\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1625 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t705.4\t0\t10\n",
      "4\t3\t176.4\t0\t3\n",
      "5\t4\t44.1\t0\t4\n",
      "6\t7\t11.0\t0\t7\n",
      "7\t58\t2.8\t0\t58\n",
      "8\t72\t0.7\t0\t72\n",
      "9\t276\t0.2\t0\t276\n",
      "10\t373\t0.0\t1\t372 1\n",
      "11\t29\t0.0\t1\t28 1\n",
      "12\t37\t0.0\t1\t37\n",
      "13\t27\t0.0\t1\t27\n",
      "14\t58\t0.0\t1\t56 2\n",
      "15\t42\t0.0\t1\t41 1\n",
      "16\t61\t0.0\t1\t58 3\n",
      "17\t129\t0.0\t1\t125 4\n",
      "18\t111\t0.0\t1\t104 5 2\n",
      "19\t297\t0.0\t1\t179 118\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t26\t0.0\t2\t0 26\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/343.D_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/343.D_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/343.D_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/343.D_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.17 s (19 us/read; 3.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            167,538\n",
      "  Read 1 with adapter:                     103 (0.1%)\n",
      "  Read 2 with adapter:                   5,445 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       167,538 (100.0%)\n",
      "\n",
      "Total basepairs processed:    76,129,493 bp\n",
      "  Read 1:    37,418,278 bp\n",
      "  Read 2:    38,711,215 bp\n",
      "Total written (filtered):     76,057,152 bp (99.9%)\n",
      "  Read 1:    37,416,446 bp\n",
      "  Read 2:    38,640,706 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t2617.8\t0\t91\n",
      "24\t1\t0.0\t1\t0 1\n",
      "25\t2\t0.0\t1\t2\n",
      "38\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "123\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "216\t5\t0.0\t1\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5445 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t2617.8\t0\t20\n",
      "4\t3\t654.4\t0\t3\n",
      "5\t12\t163.6\t0\t12\n",
      "6\t22\t40.9\t0\t22\n",
      "7\t180\t10.2\t0\t180\n",
      "8\t198\t2.6\t0\t198\n",
      "9\t1011\t0.6\t0\t1008 3\n",
      "10\t1587\t0.2\t1\t1583 4\n",
      "11\t91\t0.0\t1\t90 1\n",
      "12\t87\t0.0\t1\t84 3\n",
      "13\t67\t0.0\t1\t67\n",
      "14\t118\t0.0\t1\t117 1\n",
      "15\t144\t0.0\t1\t142 2\n",
      "16\t173\t0.0\t1\t170 3\n",
      "17\t305\t0.0\t1\t294 11\n",
      "18\t397\t0.0\t1\t374 17 6\n",
      "19\t901\t0.0\t1\t620 281\n",
      "20\t18\t0.0\t2\t0 2 16\n",
      "21\t104\t0.0\t2\t0 103 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/343.U_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/343.U_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/343.U_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/343.U_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.48 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,420\n",
      "  Read 1 with adapter:                      27 (0.1%)\n",
      "  Read 2 with adapter:                     404 (0.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,420 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,206,817 bp\n",
      "  Read 1:     9,944,034 bp\n",
      "  Read 2:    10,262,783 bp\n",
      "Total written (filtered):     20,199,941 bp (100.0%)\n",
      "  Read 1:     9,943,953 bp\n",
      "  Read 2:    10,255,988 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 27 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t694.1\t0\t27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 404 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t694.1\t0\t1\n",
      "6\t2\t10.8\t0\t2\n",
      "7\t7\t2.7\t0\t7\n",
      "8\t2\t0.7\t0\t2\n",
      "9\t12\t0.2\t0\t12\n",
      "10\t19\t0.0\t1\t19\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t11\t0.0\t1\t11\n",
      "13\t5\t0.0\t1\t5\n",
      "14\t6\t0.0\t1\t6\n",
      "15\t21\t0.0\t1\t21\n",
      "16\t24\t0.0\t1\t24\n",
      "17\t52\t0.0\t1\t50 2\n",
      "18\t60\t0.0\t1\t58 2\n",
      "19\t158\t0.0\t1\t116 41 1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t18\t0.0\t2\t0 18\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/344.D_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/344.D_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/344.D_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/344.D_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.43 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,619\n",
      "  Read 1 with adapter:                      67 (0.1%)\n",
      "  Read 2 with adapter:                   1,907 (1.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,619 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,100,560 bp\n",
      "  Read 1:    25,618,475 bp\n",
      "  Read 2:    26,482,085 bp\n",
      "Total written (filtered):     52,070,988 bp (99.9%)\n",
      "  Read 1:    25,618,108 bp\n",
      "  Read 2:    26,452,880 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 67 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t64\t1790.9\t0\t64\n",
      "27\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "110\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1907 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1790.9\t0\t5\n",
      "4\t1\t447.7\t0\t1\n",
      "5\t2\t111.9\t0\t2\n",
      "6\t3\t28.0\t0\t3\n",
      "7\t17\t7.0\t0\t17\n",
      "8\t36\t1.7\t0\t36\n",
      "9\t127\t0.4\t0\t127\n",
      "10\t305\t0.1\t1\t305\n",
      "11\t47\t0.0\t1\t47\n",
      "12\t43\t0.0\t1\t43\n",
      "13\t39\t0.0\t1\t39\n",
      "14\t48\t0.0\t1\t47 1\n",
      "15\t89\t0.0\t1\t87 2\n",
      "16\t104\t0.0\t1\t104\n",
      "17\t192\t0.0\t1\t187 5\n",
      "18\t221\t0.0\t1\t215 5 1\n",
      "19\t536\t0.0\t1\t379 156 1\n",
      "20\t12\t0.0\t2\t0 0 12\n",
      "21\t79\t0.0\t2\t0 77 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/344.U_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/344.U_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/344.U_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/344.U_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.46 s (31 us/read; 1.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,458\n",
      "  Read 1 with adapter:                      38 (0.1%)\n",
      "  Read 2 with adapter:                   1,439 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        47,458 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,364,540 bp\n",
      "  Read 1:    10,493,075 bp\n",
      "  Read 2:    10,871,465 bp\n",
      "Total written (filtered):     21,343,006 bp (99.9%)\n",
      "  Read 1:    10,491,670 bp\n",
      "  Read 2:    10,851,336 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 38 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t741.5\t0\t30\n",
      "13\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "198\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1439 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t741.5\t0\t10\n",
      "4\t2\t185.4\t0\t2\n",
      "5\t4\t46.3\t0\t4\n",
      "6\t4\t11.6\t0\t4\n",
      "7\t44\t2.9\t0\t44\n",
      "8\t52\t0.7\t0\t52\n",
      "9\t186\t0.2\t0\t186\n",
      "10\t304\t0.0\t1\t299 5\n",
      "11\t28\t0.0\t1\t28\n",
      "12\t27\t0.0\t1\t27\n",
      "13\t38\t0.0\t1\t38\n",
      "14\t66\t0.0\t1\t65 1\n",
      "15\t52\t0.0\t1\t52\n",
      "16\t74\t0.0\t1\t68 6\n",
      "17\t104\t0.0\t1\t99 5\n",
      "18\t126\t0.0\t1\t114 10 2\n",
      "19\t265\t0.0\t1\t180 84 1\n",
      "20\t11\t0.0\t2\t0 1 10\n",
      "21\t39\t0.0\t2\t0 38 1\n",
      "61\t1\t0.0\t2\t1\n",
      "207\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/345.H_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/345.H_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/345.H_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/345.H_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.84 s (43 us/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,381\n",
      "  Read 1 with adapter:                      16 (0.1%)\n",
      "  Read 2 with adapter:                     941 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        19,381 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,648,330 bp\n",
      "  Read 1:     4,240,277 bp\n",
      "  Read 2:     4,408,053 bp\n",
      "Total written (filtered):      8,636,292 bp (99.9%)\n",
      "  Read 1:     4,239,997 bp\n",
      "  Read 2:     4,396,295 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t302.8\t0\t14\n",
      "25\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 941 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t302.8\t0\t6\n",
      "4\t1\t75.7\t0\t1\n",
      "6\t9\t4.7\t0\t9\n",
      "7\t49\t1.2\t0\t49\n",
      "8\t70\t0.3\t0\t70\n",
      "9\t196\t0.1\t0\t196\n",
      "10\t184\t0.0\t1\t184\n",
      "11\t19\t0.0\t1\t19\n",
      "12\t14\t0.0\t1\t14\n",
      "13\t10\t0.0\t1\t10\n",
      "14\t34\t0.0\t1\t34\n",
      "15\t31\t0.0\t1\t31\n",
      "16\t35\t0.0\t1\t34 1\n",
      "17\t73\t0.0\t1\t71 2\n",
      "18\t83\t0.0\t1\t78 4 1\n",
      "19\t117\t0.0\t1\t73 44\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t9\t0.0\t2\t0 9\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/346.H_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/346.H_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/346.H_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/346.H_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.12 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,597\n",
      "  Read 1 with adapter:                      48 (0.1%)\n",
      "  Read 2 with adapter:                   1,039 (1.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        90,597 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,110,256 bp\n",
      "  Read 1:    20,216,364 bp\n",
      "  Read 2:    20,893,892 bp\n",
      "Total written (filtered):     41,092,798 bp (100.0%)\n",
      "  Read 1:    20,216,070 bp\n",
      "  Read 2:    20,876,728 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 48 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t1415.6\t0\t46\n",
      "25\t1\t0.0\t1\t1\n",
      "131\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1039 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1415.6\t0\t1\n",
      "7\t7\t5.5\t0\t7\n",
      "8\t13\t1.4\t0\t13\n",
      "9\t46\t0.3\t0\t46\n",
      "10\t111\t0.1\t1\t110 1\n",
      "11\t14\t0.0\t1\t14\n",
      "12\t20\t0.0\t1\t19 1\n",
      "13\t11\t0.0\t1\t11\n",
      "14\t29\t0.0\t1\t28 1\n",
      "15\t49\t0.0\t1\t46 3\n",
      "16\t46\t0.0\t1\t46\n",
      "17\t128\t0.0\t1\t124 4\n",
      "18\t152\t0.0\t1\t148 2 2\n",
      "19\t341\t0.0\t1\t252 89\n",
      "20\t9\t0.0\t2\t0 0 9\n",
      "21\t61\t0.0\t2\t0 60 1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/347.H_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/347.H_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/347.H_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/347.H_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.20 s (22 us/read; 2.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,369\n",
      "  Read 1 with adapter:                      63 (0.1%)\n",
      "  Read 2 with adapter:                   1,796 (1.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,369 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,595,435 bp\n",
      "  Read 1:    22,427,215 bp\n",
      "  Read 2:    23,168,220 bp\n",
      "Total written (filtered):     45,568,180 bp (99.9%)\n",
      "  Read 1:    22,426,743 bp\n",
      "  Read 2:    23,141,437 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 63 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t1568.3\t0\t59\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1796 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1568.3\t0\t5\n",
      "4\t1\t392.1\t0\t1\n",
      "5\t3\t98.0\t0\t3\n",
      "6\t4\t24.5\t0\t4\n",
      "7\t38\t6.1\t0\t38\n",
      "8\t33\t1.5\t0\t33\n",
      "9\t117\t0.4\t0\t117\n",
      "10\t361\t0.1\t1\t359 2\n",
      "11\t40\t0.0\t1\t40\n",
      "12\t50\t0.0\t1\t50\n",
      "13\t30\t0.0\t1\t30\n",
      "14\t51\t0.0\t1\t50 1\n",
      "15\t75\t0.0\t1\t73 2\n",
      "16\t95\t0.0\t1\t93 2\n",
      "17\t185\t0.0\t1\t180 5\n",
      "18\t184\t0.0\t1\t181 2 1\n",
      "19\t457\t0.0\t1\t330 127\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t51\t0.0\t2\t0 51\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "30\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/348.D_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/348.D_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/348.D_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/348.D_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.92 s (19 us/read; 3.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            210,821\n",
      "  Read 1 with adapter:                      89 (0.0%)\n",
      "  Read 2 with adapter:                   8,606 (4.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       210,821 (100.0%)\n",
      "\n",
      "Total basepairs processed:    95,734,791 bp\n",
      "  Read 1:    47,076,616 bp\n",
      "  Read 2:    48,658,175 bp\n",
      "Total written (filtered):     95,627,297 bp (99.9%)\n",
      "  Read 1:    47,074,459 bp\n",
      "  Read 2:    48,552,838 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 89 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t76\t3294.1\t0\t76\n",
      "25\t1\t0.0\t1\t1\n",
      "30\t1\t0.0\t1\t1\n",
      "62\t1\t0.0\t1\t1\n",
      "101\t3\t0.0\t1\t3\n",
      "212\t1\t0.0\t1\t0 1\n",
      "213\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t2 1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8606 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t62\t3294.1\t0\t62\n",
      "4\t7\t823.5\t0\t7\n",
      "5\t37\t205.9\t0\t37\n",
      "6\t85\t51.5\t0\t85\n",
      "7\t306\t12.9\t0\t306\n",
      "8\t488\t3.2\t0\t488\n",
      "9\t1764\t0.8\t0\t1764\n",
      "10\t2589\t0.2\t1\t2579 10\n",
      "11\t126\t0.1\t1\t125 1\n",
      "12\t103\t0.0\t1\t102 1\n",
      "13\t101\t0.0\t1\t100 1\n",
      "14\t125\t0.0\t1\t120 5\n",
      "15\t186\t0.0\t1\t183 3\n",
      "16\t253\t0.0\t1\t243 10\n",
      "17\t498\t0.0\t1\t486 12\n",
      "18\t524\t0.0\t1\t484 29 11\n",
      "19\t1193\t0.0\t1\t805 388\n",
      "20\t24\t0.0\t2\t0 1 23\n",
      "21\t131\t0.0\t2\t0 130 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/348.U_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/348.U_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/348.U_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/348.U_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.27 s (30 us/read; 1.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,044\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                   2,277 (5.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,044 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,851,520 bp\n",
      "  Read 1:     9,257,535 bp\n",
      "  Read 2:     9,593,985 bp\n",
      "Total written (filtered):     18,823,293 bp (99.9%)\n",
      "  Read 1:     9,256,890 bp\n",
      "  Read 2:     9,566,403 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t656.9\t0\t6\n",
      "192\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2277 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t656.9\t0\t16\n",
      "4\t3\t164.2\t0\t3\n",
      "5\t13\t41.1\t0\t13\n",
      "6\t24\t10.3\t0\t24\n",
      "7\t89\t2.6\t0\t89\n",
      "8\t141\t0.6\t0\t141\n",
      "9\t487\t0.2\t0\t487\n",
      "10\t630\t0.0\t1\t628 2\n",
      "11\t36\t0.0\t1\t36\n",
      "12\t46\t0.0\t1\t45 1\n",
      "13\t34\t0.0\t1\t33 1\n",
      "14\t49\t0.0\t1\t46 3\n",
      "15\t69\t0.0\t1\t68 1\n",
      "16\t85\t0.0\t1\t84 1\n",
      "17\t131\t0.0\t1\t128 3\n",
      "18\t121\t0.0\t1\t111 5 5\n",
      "19\t276\t0.0\t1\t183 93\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t22\t0.0\t2\t0 22\n",
      "217\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/349.D_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/349.D_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/349.D_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/349.D_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.09 s (19 us/read; 3.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            159,386\n",
      "  Read 1 with adapter:                      40 (0.0%)\n",
      "  Read 2 with adapter:                   6,096 (3.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       159,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:    72,190,465 bp\n",
      "  Read 1:    35,506,522 bp\n",
      "  Read 2:    36,683,943 bp\n",
      "Total written (filtered):     72,112,915 bp (99.9%)\n",
      "  Read 1:    35,505,874 bp\n",
      "  Read 2:    36,607,041 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 40 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t2490.4\t0\t34\n",
      "27\t3\t0.0\t1\t3\n",
      "55\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6096 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t2490.4\t0\t37\n",
      "4\t4\t622.6\t0\t4\n",
      "5\t20\t155.7\t0\t20\n",
      "6\t67\t38.9\t0\t67\n",
      "7\t200\t9.7\t0\t200\n",
      "8\t301\t2.4\t0\t301\n",
      "9\t1200\t0.6\t0\t1198 2\n",
      "10\t1771\t0.2\t1\t1762 9\n",
      "11\t116\t0.0\t1\t116\n",
      "12\t86\t0.0\t1\t85 1\n",
      "13\t80\t0.0\t1\t80\n",
      "14\t129\t0.0\t1\t127 2\n",
      "15\t162\t0.0\t1\t155 7\n",
      "16\t195\t0.0\t1\t189 6\n",
      "17\t336\t0.0\t1\t326 10\n",
      "18\t379\t0.0\t1\t354 17 8\n",
      "19\t898\t0.0\t1\t608 290\n",
      "20\t21\t0.0\t2\t0 1 20\n",
      "21\t87\t0.0\t2\t0 86 1\n",
      "152\t1\t0.0\t2\t1\n",
      "175\t1\t0.0\t2\t1\n",
      "216\t4\t0.0\t2\t4\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/349.U_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/349.U_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/349.U_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/349.U_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.93 s (29 us/read; 2.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,762\n",
      "  Read 1 with adapter:                      29 (0.0%)\n",
      "  Read 2 with adapter:                   2,922 (4.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        66,762 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,046,070 bp\n",
      "  Read 1:    14,764,047 bp\n",
      "  Read 2:    15,282,023 bp\n",
      "Total written (filtered):     30,008,315 bp (99.9%)\n",
      "  Read 1:    14,763,080 bp\n",
      "  Read 2:    15,245,235 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t1043.2\t0\t23\n",
      "27\t2\t0.0\t1\t2\n",
      "193\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2922 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1043.2\t0\t16\n",
      "4\t3\t260.8\t0\t3\n",
      "5\t7\t65.2\t0\t7\n",
      "6\t28\t16.3\t0\t28\n",
      "7\t90\t4.1\t0\t90\n",
      "8\t138\t1.0\t0\t138\n",
      "9\t596\t0.3\t0\t592 4\n",
      "10\t864\t0.1\t1\t858 6\n",
      "11\t43\t0.0\t1\t43\n",
      "12\t45\t0.0\t1\t45\n",
      "13\t30\t0.0\t1\t29 1\n",
      "14\t63\t0.0\t1\t61 2\n",
      "15\t90\t0.0\t1\t90\n",
      "16\t105\t0.0\t1\t104 1\n",
      "17\t161\t0.0\t1\t155 6\n",
      "18\t180\t0.0\t1\t167 10 3\n",
      "19\t404\t0.0\t1\t290 114\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t43\t0.0\t2\t0 43\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/350.D_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/350.D_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/350.D_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/350.D_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.96 s (19 us/read; 3.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            211,882\n",
      "  Read 1 with adapter:                      90 (0.0%)\n",
      "  Read 2 with adapter:                   6,991 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       211,882 (100.0%)\n",
      "\n",
      "Total basepairs processed:    96,168,979 bp\n",
      "  Read 1:    47,312,603 bp\n",
      "  Read 2:    48,856,376 bp\n",
      "Total written (filtered):     96,075,664 bp (99.9%)\n",
      "  Read 1:    47,309,876 bp\n",
      "  Read 2:    48,765,788 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t75\t3310.7\t0\t75\n",
      "25\t3\t0.0\t1\t3\n",
      "124\t1\t0.0\t1\t1\n",
      "193\t2\t0.0\t1\t2\n",
      "201\t1\t0.0\t1\t1\n",
      "211\t3\t0.0\t1\t3\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t3\t0.0\t1\t2 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6991 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t3310.7\t0\t47\n",
      "4\t5\t827.7\t0\t5\n",
      "5\t24\t206.9\t0\t24\n",
      "6\t61\t51.7\t0\t61\n",
      "7\t187\t12.9\t0\t187\n",
      "8\t334\t3.2\t0\t334\n",
      "9\t1300\t0.8\t0\t1300\n",
      "10\t1973\t0.2\t1\t1961 12\n",
      "11\t146\t0.1\t1\t146\n",
      "12\t134\t0.0\t1\t133 1\n",
      "13\t82\t0.0\t1\t79 3\n",
      "14\t141\t0.0\t1\t135 6\n",
      "15\t195\t0.0\t1\t193 2\n",
      "16\t184\t0.0\t1\t175 9\n",
      "17\t423\t0.0\t1\t417 6\n",
      "18\t436\t0.0\t1\t412 18 6\n",
      "19\t1149\t0.0\t1\t807 342\n",
      "20\t18\t0.0\t2\t0 0 18\n",
      "21\t135\t0.0\t2\t0 135\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t1\n",
      "199\t1\t0.0\t2\t1\n",
      "213\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t1 2\n",
      "217\t3\t0.0\t2\t2 0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/341.U_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/341.U_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/341.U_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/341.U_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/342.D_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/342.D_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/342.D_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/342.D_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/342.U_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/342.U_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/342.U_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/342.U_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/343.D_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/343.D_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/343.D_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/343.D_31_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/343.U_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/343.U_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/343.U_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/343.U_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/344.D_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/344.D_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/344.D_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/344.D_12_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/344.U_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/344.U_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/344.U_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/344.U_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/345.H_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/345.H_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/345.H_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/345.H_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/346.H_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/346.H_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/346.H_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/346.H_25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/347.H_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/347.H_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/347.H_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/347.H_16_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/348.D_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/348.D_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/348.D_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/348.D_4_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/348.U_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/348.U_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/348.U_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/348.U_27_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/349.D_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/349.D_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/349.D_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/349.D_2_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/349.U_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/349.U_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/349.U_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/349.U_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/350.D_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/350.D_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/350.D_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/350.D_9_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/350.U_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/350.U_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/350.U_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/350.U_20_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/350.U_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/350.U_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/350.U_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/350.U_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.91 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,853\n",
      "  Read 1 with adapter:                      19 (0.0%)\n",
      "  Read 2 with adapter:                   3,162 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        80,853 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,421,735 bp\n",
      "  Read 1:    17,908,003 bp\n",
      "  Read 2:    18,513,732 bp\n",
      "Total written (filtered):     36,379,675 bp (99.9%)\n",
      "  Read 1:    17,906,499 bp\n",
      "  Read 2:    18,473,176 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1263.3\t0\t12\n",
      "192\t1\t0.0\t1\t1\n",
      "203\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t3 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3162 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1263.3\t0\t15\n",
      "4\t1\t315.8\t0\t1\n",
      "5\t12\t79.0\t0\t12\n",
      "6\t27\t19.7\t0\t27\n",
      "7\t95\t4.9\t0\t95\n",
      "8\t161\t1.2\t0\t161\n",
      "9\t617\t0.3\t0\t617\n",
      "10\t895\t0.1\t1\t888 7\n",
      "11\t50\t0.0\t1\t50\n",
      "12\t48\t0.0\t1\t47 1\n",
      "13\t44\t0.0\t1\t43 1\n",
      "14\t49\t0.0\t1\t49\n",
      "15\t104\t0.0\t1\t100 4\n",
      "16\t96\t0.0\t1\t92 4\n",
      "17\t185\t0.0\t1\t181 4\n",
      "18\t212\t0.0\t1\t197 12 3\n",
      "19\t483\t0.0\t1\t333 150\n",
      "20\t8\t0.0\t2\t0 0 8\n",
      "21\t55\t0.0\t2\t0 55\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "216\t3\t0.0\t2\t3\n",
      "226\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/351.D_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/351.D_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/351.D_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/351.D_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.50 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,052\n",
      "  Read 1 with adapter:                      32 (0.0%)\n",
      "  Read 2 with adapter:                   4,372 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,052 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,354,668 bp\n",
      "  Read 1:    25,250,619 bp\n",
      "  Read 2:    26,104,049 bp\n",
      "Total written (filtered):     51,299,448 bp (99.9%)\n",
      "  Read 1:    25,250,523 bp\n",
      "  Read 2:    26,048,925 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 32 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t1766.4\t0\t32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4372 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t1766.4\t0\t30\n",
      "4\t3\t441.6\t0\t3\n",
      "5\t15\t110.4\t0\t15\n",
      "6\t42\t27.6\t0\t42\n",
      "7\t178\t6.9\t0\t178\n",
      "8\t210\t1.7\t0\t210\n",
      "9\t842\t0.4\t0\t842\n",
      "10\t1244\t0.1\t1\t1236 8\n",
      "11\t65\t0.0\t1\t65\n",
      "12\t57\t0.0\t1\t57\n",
      "13\t37\t0.0\t1\t36 1\n",
      "14\t91\t0.0\t1\t86 5\n",
      "15\t119\t0.0\t1\t117 2\n",
      "16\t149\t0.0\t1\t147 2\n",
      "17\t282\t0.0\t1\t269 13\n",
      "18\t295\t0.0\t1\t279 11 5\n",
      "19\t617\t0.0\t1\t423 193 1\n",
      "20\t14\t0.0\t2\t0 0 14\n",
      "21\t78\t0.0\t2\t0 77 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "229\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/351.U_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/351.U_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/351.U_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/351.U_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.56 s (21 us/read; 2.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            124,636\n",
      "  Read 1 with adapter:                      63 (0.1%)\n",
      "  Read 2 with adapter:                   6,289 (5.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       124,636 (100.0%)\n",
      "\n",
      "Total basepairs processed:    56,319,286 bp\n",
      "  Read 1:    27,668,677 bp\n",
      "  Read 2:    28,650,609 bp\n",
      "Total written (filtered):     56,236,655 bp (99.9%)\n",
      "  Read 1:    27,664,288 bp\n",
      "  Read 2:    28,572,367 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 63 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t1947.4\t0\t39\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "192\t2\t0.0\t1\t1 1\n",
      "193\t12\t0.0\t1\t9 3\n",
      "194\t1\t0.0\t1\t0 1\n",
      "213\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t4\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6289 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t1947.4\t0\t47\n",
      "4\t5\t486.9\t0\t5\n",
      "5\t20\t121.7\t0\t20\n",
      "6\t47\t30.4\t0\t47\n",
      "7\t233\t7.6\t0\t233\n",
      "8\t357\t1.9\t0\t357\n",
      "9\t1451\t0.5\t0\t1448 3\n",
      "10\t1790\t0.1\t1\t1782 8\n",
      "11\t96\t0.0\t1\t96\n",
      "12\t90\t0.0\t1\t89 1\n",
      "13\t74\t0.0\t1\t72 2\n",
      "14\t105\t0.0\t1\t105\n",
      "15\t175\t0.0\t1\t173 2\n",
      "16\t181\t0.0\t1\t174 7\n",
      "17\t356\t0.0\t1\t349 7\n",
      "18\t347\t0.0\t1\t325 17 5\n",
      "19\t791\t0.0\t1\t542 249\n",
      "20\t15\t0.0\t2\t0 0 15\n",
      "21\t94\t0.0\t2\t0 92 2\n",
      "95\t1\t0.0\t2\t1\n",
      "115\t5\t0.0\t2\t5\n",
      "151\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t2\t0.0\t2\t1 1\n",
      "218\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/352.D_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/352.D_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/352.D_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/352.D_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.37 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            144,757\n",
      "  Read 1 with adapter:                      64 (0.0%)\n",
      "  Read 2 with adapter:                   5,580 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       144,757 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,794,564 bp\n",
      "  Read 1:    32,356,178 bp\n",
      "  Read 2:    33,438,386 bp\n",
      "Total written (filtered):     65,724,830 bp (99.9%)\n",
      "  Read 1:    32,355,371 bp\n",
      "  Read 2:    33,369,459 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 64 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t2261.8\t0\t58\n",
      "10\t1\t0.1\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "138\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5580 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t2261.8\t0\t37\n",
      "4\t7\t565.5\t0\t7\n",
      "5\t32\t141.4\t0\t32\n",
      "6\t53\t35.3\t0\t53\n",
      "7\t193\t8.8\t0\t193\n",
      "8\t261\t2.2\t0\t261\n",
      "9\t1092\t0.6\t0\t1092\n",
      "10\t1675\t0.1\t1\t1669 6\n",
      "11\t103\t0.0\t1\t103\n",
      "12\t105\t0.0\t1\t104 1\n",
      "13\t64\t0.0\t1\t64\n",
      "14\t98\t0.0\t1\t96 2\n",
      "15\t133\t0.0\t1\t129 4\n",
      "16\t184\t0.0\t1\t180 4\n",
      "17\t318\t0.0\t1\t311 7\n",
      "18\t337\t0.0\t1\t322 10 5\n",
      "19\t767\t0.0\t1\t508 259\n",
      "20\t24\t0.0\t2\t0 0 24\n",
      "21\t96\t0.0\t2\t0 96\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/352.U_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/352.U_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/352.U_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/352.U_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.02 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,462\n",
      "  Read 1 with adapter:                      22 (0.0%)\n",
      "  Read 2 with adapter:                   3,955 (4.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        90,462 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,782,673 bp\n",
      "  Read 1:    20,047,987 bp\n",
      "  Read 2:    20,734,686 bp\n",
      "Total written (filtered):     40,732,153 bp (99.9%)\n",
      "  Read 1:    20,046,425 bp\n",
      "  Read 2:    20,685,728 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1413.5\t0\t15\n",
      "216\t3\t0.0\t1\t3\n",
      "217\t3\t0.0\t1\t3\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3955 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t1413.5\t0\t42\n",
      "4\t6\t353.4\t0\t6\n",
      "5\t13\t88.3\t0\t13\n",
      "6\t35\t22.1\t0\t35\n",
      "7\t124\t5.5\t0\t124\n",
      "8\t215\t1.4\t0\t215\n",
      "9\t866\t0.3\t0\t865 1\n",
      "10\t1152\t0.1\t1\t1148 4\n",
      "11\t70\t0.0\t1\t70\n",
      "12\t59\t0.0\t1\t58 1\n",
      "13\t53\t0.0\t1\t52 1\n",
      "14\t57\t0.0\t1\t56 1\n",
      "15\t84\t0.0\t1\t84\n",
      "16\t122\t0.0\t1\t120 2\n",
      "17\t218\t0.0\t1\t212 6\n",
      "18\t219\t0.0\t1\t206 9 4\n",
      "19\t549\t0.0\t1\t395 154\n",
      "20\t16\t0.0\t2\t0 3 13\n",
      "21\t50\t0.0\t2\t0 49 1\n",
      "146\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/353.H_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/353.H_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/353.H_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/353.H_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.72 s (19 us/read; 3.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,138\n",
      "  Read 1 with adapter:                      24 (0.0%)\n",
      "  Read 2 with adapter:                   4,872 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       142,138 (100.0%)\n",
      "\n",
      "Total basepairs processed:    64,113,676 bp\n",
      "  Read 1:    31,546,458 bp\n",
      "  Read 2:    32,567,218 bp\n",
      "Total written (filtered):     64,047,410 bp (99.9%)\n",
      "  Read 1:    31,545,107 bp\n",
      "  Read 2:    32,502,303 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 24 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t2220.9\t0\t18\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4872 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t2220.9\t0\t18\n",
      "4\t4\t555.2\t0\t4\n",
      "5\t11\t138.8\t0\t11\n",
      "6\t33\t34.7\t0\t33\n",
      "7\t97\t8.7\t0\t97\n",
      "8\t176\t2.2\t0\t176\n",
      "9\t825\t0.5\t0\t824 1\n",
      "10\t1439\t0.1\t1\t1428 11\n",
      "11\t75\t0.0\t1\t73 2\n",
      "12\t61\t0.0\t1\t60 1\n",
      "13\t35\t0.0\t1\t35\n",
      "14\t77\t0.0\t1\t74 3\n",
      "15\t142\t0.0\t1\t138 4\n",
      "16\t203\t0.0\t1\t195 8\n",
      "17\t372\t0.0\t1\t362 10\n",
      "18\t355\t0.0\t1\t337 13 5\n",
      "19\t837\t0.0\t1\t562 275\n",
      "20\t16\t0.0\t2\t3 2 11\n",
      "21\t88\t0.0\t2\t0 88\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t3\n",
      "217\t2\t0.0\t2\t2\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/354.H_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/354.H_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/354.H_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/354.H_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.85 s (21 us/read; 2.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             89,337\n",
      "  Read 1 with adapter:                      21 (0.0%)\n",
      "  Read 2 with adapter:                   4,345 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        89,337 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,153,724 bp\n",
      "  Read 1:    19,735,653 bp\n",
      "  Read 2:    20,418,071 bp\n",
      "Total written (filtered):     40,098,848 bp (99.9%)\n",
      "  Read 1:    19,733,435 bp\n",
      "  Read 2:    20,365,413 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1395.9\t0\t10\n",
      "26\t1\t0.0\t1\t0 1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t2\t0.0\t1\t1 1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t6\t0.0\t1\t5 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4345 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t1395.9\t0\t33\n",
      "4\t5\t349.0\t0\t5\n",
      "5\t24\t87.2\t0\t24\n",
      "6\t45\t21.8\t0\t45\n",
      "7\t163\t5.5\t0\t163\n",
      "8\t229\t1.4\t0\t229\n",
      "9\t1021\t0.3\t0\t1021\n",
      "10\t1384\t0.1\t1\t1376 8\n",
      "11\t59\t0.0\t1\t59\n",
      "12\t59\t0.0\t1\t56 3\n",
      "13\t28\t0.0\t1\t27 1\n",
      "14\t55\t0.0\t1\t53 2\n",
      "15\t92\t0.0\t1\t90 2\n",
      "16\t85\t0.0\t1\t81 4\n",
      "17\t198\t0.0\t1\t189 9\n",
      "18\t230\t0.0\t1\t220 8 2\n",
      "19\t544\t0.0\t1\t362 181 1\n",
      "20\t10\t0.0\t2\t0 0 10\n",
      "21\t74\t0.0\t2\t0 74\n",
      "77\t1\t0.0\t2\t1\n",
      "148\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t3\t0.0\t2\t2 1\n",
      "229\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/355.H_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/355.H_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/355.H_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/355.H_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.76 s (23 us/read; 2.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,883\n",
      "  Read 1 with adapter:                      11 (0.0%)\n",
      "  Read 2 with adapter:                   3,285 (4.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,883 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,225,793 bp\n",
      "  Read 1:    16,829,971 bp\n",
      "  Read 2:    17,395,822 bp\n",
      "Total written (filtered):     34,184,784 bp (99.9%)\n",
      "  Read 1:    16,829,244 bp\n",
      "  Read 2:    17,355,540 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1185.7\t0\t6\n",
      "27\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "221\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3285 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t1185.7\t0\t21\n",
      "4\t9\t296.4\t0\t9\n",
      "5\t19\t74.1\t0\t19\n",
      "6\t40\t18.5\t0\t40\n",
      "7\t109\t4.6\t0\t109\n",
      "8\t175\t1.2\t0\t175\n",
      "9\t645\t0.3\t0\t644 1\n",
      "10\t1020\t0.1\t1\t1018 2\n",
      "11\t55\t0.0\t1\t54 1\n",
      "12\t47\t0.0\t1\t46 1\n",
      "13\t26\t0.0\t1\t25 1\n",
      "14\t54\t0.0\t1\t53 1\n",
      "15\t75\t0.0\t1\t74 1\n",
      "16\t96\t0.0\t1\t94 2\n",
      "17\t173\t0.0\t1\t163 10\n",
      "18\t199\t0.0\t1\t191 7 1\n",
      "19\t458\t0.0\t1\t305 153\n",
      "20\t9\t0.0\t2\t1 2 6\n",
      "21\t54\t0.0\t2\t0 54\n",
      "217\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/356.D_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/356.D_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/356.D_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/356.D_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.58 s (15 us/read; 3.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            496,195\n",
      "  Read 1 with adapter:                     224 (0.0%)\n",
      "  Read 2 with adapter:                  10,373 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       496,195 (100.0%)\n",
      "\n",
      "Total basepairs processed:   225,867,538 bp\n",
      "  Read 1:   111,096,306 bp\n",
      "  Read 2:   114,771,232 bp\n",
      "Total written (filtered):    225,710,899 bp (99.9%)\n",
      "  Read 1:   111,095,049 bp\n",
      "  Read 2:   114,615,850 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 224 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t213\t7753.0\t0\t213\n",
      "20\t1\t0.0\t1\t1\n",
      "21\t2\t0.0\t1\t2\n",
      "25\t4\t0.0\t1\t4\n",
      "27\t2\t0.0\t1\t2\n",
      "195\t1\t0.0\t1\t0 1\n",
      "207\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10373 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t7753.0\t0\t23\n",
      "4\t10\t1938.3\t0\t10\n",
      "5\t10\t484.6\t0\t10\n",
      "6\t31\t121.1\t0\t31\n",
      "7\t174\t30.3\t0\t174\n",
      "8\t192\t7.6\t0\t192\n",
      "9\t954\t1.9\t0\t952 2\n",
      "10\t2135\t0.5\t1\t2119 16\n",
      "11\t130\t0.1\t1\t129 1\n",
      "12\t201\t0.0\t1\t199 2\n",
      "13\t193\t0.0\t1\t192 1\n",
      "14\t300\t0.0\t1\t292 8\n",
      "15\t396\t0.0\t1\t387 9\n",
      "16\t444\t0.0\t1\t431 13\n",
      "17\t971\t0.0\t1\t935 36\n",
      "18\t1068\t0.0\t1\t1005 42 21\n",
      "19\t2705\t0.0\t1\t1877 825 3\n",
      "20\t67\t0.0\t2\t3 4 60\n",
      "21\t354\t0.0\t2\t0 352 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t1\n",
      "193\t1\t0.0\t2\t1\n",
      "205\t1\t0.0\t2\t1\n",
      "213\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t2 0 1\n",
      "217\t4\t0.0\t2\t4\n",
      "221\t1\t0.0\t2\t1\n",
      "224\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/356.U_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/356.U_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/356.U_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/356.U_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.39 s (29 us/read; 2.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,225\n",
      "  Read 1 with adapter:                      21 (0.0%)\n",
      "  Read 2 with adapter:                   1,099 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,225 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,735,340 bp\n",
      "  Read 1:    10,682,202 bp\n",
      "  Read 2:    11,053,138 bp\n",
      "Total written (filtered):     21,718,553 bp (99.9%)\n",
      "  Read 1:    10,681,553 bp\n",
      "  Read 2:    11,037,000 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t753.5\t0\t17\n",
      "27\t1\t0.0\t1\t1\n",
      "138\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1099 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t753.5\t0\t5\n",
      "4\t1\t188.4\t0\t1\n",
      "5\t2\t47.1\t0\t2\n",
      "6\t4\t11.8\t0\t4\n",
      "7\t13\t2.9\t0\t13\n",
      "8\t17\t0.7\t0\t17\n",
      "9\t105\t0.2\t0\t105\n",
      "10\t206\t0.0\t1\t206\n",
      "11\t19\t0.0\t1\t18 1\n",
      "12\t29\t0.0\t1\t27 2\n",
      "13\t22\t0.0\t1\t22\n",
      "14\t58\t0.0\t1\t57 1\n",
      "15\t39\t0.0\t1\t38 1\n",
      "16\t67\t0.0\t1\t63 4\n",
      "17\t102\t0.0\t1\t99 3\n",
      "18\t124\t0.0\t1\t112 9 3\n",
      "19\t251\t0.0\t1\t165 85 1\n",
      "20\t8\t0.0\t2\t0 0 8\n",
      "21\t25\t0.0\t2\t0 25\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/357.D_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/357.D_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/357.D_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/357.D_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.46 s (21 us/read; 2.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            167,062\n",
      "  Read 1 with adapter:                     112 (0.1%)\n",
      "  Read 2 with adapter:                   2,793 (1.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       167,062 (100.0%)\n",
      "\n",
      "Total basepairs processed:    76,032,515 bp\n",
      "  Read 1:    37,403,053 bp\n",
      "  Read 2:    38,629,462 bp\n",
      "Total written (filtered):     75,990,093 bp (99.9%)\n",
      "  Read 1:    37,402,502 bp\n",
      "  Read 2:    38,587,591 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 112 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t2610.3\t0\t109\n",
      "24\t1\t0.0\t1\t1\n",
      "100\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2793 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t2610.3\t0\t10\n",
      "4\t1\t652.6\t0\t1\n",
      "5\t9\t163.1\t0\t9\n",
      "6\t13\t40.8\t0\t13\n",
      "7\t64\t10.2\t0\t64\n",
      "8\t77\t2.5\t0\t77\n",
      "9\t234\t0.6\t0\t232 2\n",
      "10\t413\t0.2\t1\t412 1\n",
      "11\t93\t0.0\t1\t92 1\n",
      "12\t75\t0.0\t1\t74 1\n",
      "13\t39\t0.0\t1\t39\n",
      "14\t49\t0.0\t1\t49\n",
      "15\t125\t0.0\t1\t124 1\n",
      "16\t146\t0.0\t1\t140 6\n",
      "17\t278\t0.0\t1\t273 5\n",
      "18\t382\t0.0\t1\t365 12 5\n",
      "19\t635\t0.0\t1\t472 163\n",
      "20\t24\t0.0\t2\t0 0 24\n",
      "21\t121\t0.0\t2\t0 121\n",
      "22\t2\t0.0\t2\t0 1 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/357.U_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/357.U_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/357.U_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/357.U_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.91 s (24 us/read; 2.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,240\n",
      "  Read 1 with adapter:                      37 (0.0%)\n",
      "  Read 2 with adapter:                   1,360 (1.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        80,240 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,507,806 bp\n",
      "  Read 1:    17,959,657 bp\n",
      "  Read 2:    18,548,149 bp\n",
      "Total written (filtered):     36,486,404 bp (99.9%)\n",
      "  Read 1:    17,959,546 bp\n",
      "  Read 2:    18,526,858 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 37 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t1253.8\t0\t37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1360 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1253.8\t0\t10\n",
      "4\t3\t313.4\t0\t3\n",
      "5\t2\t78.4\t0\t2\n",
      "6\t6\t19.6\t0\t6\n",
      "7\t31\t4.9\t0\t31\n",
      "8\t26\t1.2\t0\t26\n",
      "9\t83\t0.3\t0\t83\n",
      "10\t197\t0.1\t1\t196 1\n",
      "11\t45\t0.0\t1\t45\n",
      "12\t29\t0.0\t1\t28 1\n",
      "13\t15\t0.0\t1\t15\n",
      "14\t28\t0.0\t1\t26 2\n",
      "15\t62\t0.0\t1\t61 1\n",
      "16\t68\t0.0\t1\t67 1\n",
      "17\t112\t0.0\t1\t112\n",
      "18\t241\t0.0\t1\t234 4 3\n",
      "19\t324\t0.0\t1\t229 94 1\n",
      "20\t13\t0.0\t2\t0 0 13\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "216\t2\t0.0\t2\t2\n",
      "221\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/358.D_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/358.D_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/358.D_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/358.D_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.06 s (24 us/read; 2.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,754\n",
      "  Read 1 with adapter:                      43 (0.1%)\n",
      "  Read 2 with adapter:                   1,755 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        85,754 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,914,914 bp\n",
      "  Read 1:    19,133,583 bp\n",
      "  Read 2:    19,781,331 bp\n",
      "Total written (filtered):     38,888,795 bp (99.9%)\n",
      "  Read 1:    19,133,356 bp\n",
      "  Read 2:    19,755,439 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 43 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t1339.9\t0\t42\n",
      "101\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1755 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1339.9\t0\t6\n",
      "4\t4\t335.0\t0\t4\n",
      "5\t4\t83.7\t0\t4\n",
      "6\t10\t20.9\t0\t10\n",
      "7\t41\t5.2\t0\t41\n",
      "8\t52\t1.3\t0\t52\n",
      "9\t217\t0.3\t0\t216 1\n",
      "10\t273\t0.1\t1\t271 2\n",
      "11\t38\t0.0\t1\t38\n",
      "12\t34\t0.0\t1\t34\n",
      "13\t27\t0.0\t1\t27\n",
      "14\t31\t0.0\t1\t30 1\n",
      "15\t82\t0.0\t1\t82\n",
      "16\t87\t0.0\t1\t85 2\n",
      "17\t157\t0.0\t1\t153 4\n",
      "18\t203\t0.0\t1\t191 9 3\n",
      "19\t418\t0.0\t1\t300 117 1\n",
      "20\t12\t0.0\t2\t1 1 10\n",
      "21\t57\t0.0\t2\t0 57\n",
      "214\t1\t0.0\t2\t0 1\n",
      "243\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/358.U_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/358.U_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/358.U_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/358.U_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.00 s (25 us/read; 2.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,517\n",
      "  Read 1 with adapter:                      45 (0.1%)\n",
      "  Read 2 with adapter:                   1,921 (2.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        78,517 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,648,234 bp\n",
      "  Read 1:    17,527,646 bp\n",
      "  Read 2:    18,120,588 bp\n",
      "Total written (filtered):     35,620,448 bp (99.9%)\n",
      "  Read 1:    17,527,511 bp\n",
      "  Read 2:    18,092,937 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 45 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t1226.8\t0\t45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1921 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1226.8\t0\t6\n",
      "4\t2\t306.7\t0\t2\n",
      "5\t5\t76.7\t0\t5\n",
      "6\t8\t19.2\t0\t8\n",
      "7\t47\t4.8\t0\t47\n",
      "8\t80\t1.2\t0\t80\n",
      "9\t223\t0.3\t0\t223\n",
      "10\t329\t0.1\t1\t327 2\n",
      "11\t31\t0.0\t1\t31\n",
      "12\t41\t0.0\t1\t41\n",
      "13\t22\t0.0\t1\t22\n",
      "14\t69\t0.0\t1\t68 1\n",
      "15\t82\t0.0\t1\t81 1\n",
      "16\t101\t0.0\t1\t99 2\n",
      "17\t195\t0.0\t1\t191 4\n",
      "18\t203\t0.0\t1\t187 9 7\n",
      "19\t411\t0.0\t1\t265 144 2\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t58\t0.0\t2\t0 57 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/359.D_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/359.D_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/359.D_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/359.D_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.91 s (16 us/read; 3.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            238,196\n",
      "  Read 1 with adapter:                     134 (0.1%)\n",
      "  Read 2 with adapter:                   9,998 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       238,196 (100.0%)\n",
      "\n",
      "Total basepairs processed:   108,521,503 bp\n",
      "  Read 1:    53,324,564 bp\n",
      "  Read 2:    55,196,939 bp\n",
      "Total written (filtered):    108,396,863 bp (99.9%)\n",
      "  Read 1:    53,323,633 bp\n",
      "  Read 2:    55,073,230 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 134 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t130\t3721.8\t0\t130\n",
      "27\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "211\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9998 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t3721.8\t0\t45\n",
      "4\t22\t930.5\t0\t22\n",
      "5\t36\t232.6\t0\t36\n",
      "6\t106\t58.2\t0\t106\n",
      "7\t581\t14.5\t0\t581\n",
      "8\t628\t3.6\t0\t628\n",
      "9\t2096\t0.9\t0\t2093 3\n",
      "10\t2592\t0.2\t1\t2584 8\n",
      "11\t111\t0.1\t1\t108 3\n",
      "12\t128\t0.0\t1\t125 3\n",
      "13\t117\t0.0\t1\t117\n",
      "14\t205\t0.0\t1\t202 3\n",
      "15\t243\t0.0\t1\t240 3\n",
      "16\t335\t0.0\t1\t314 21\n",
      "17\t575\t0.0\t1\t561 14\n",
      "18\t609\t0.0\t1\t570 27 12\n",
      "19\t1369\t0.0\t1\t905 461 3\n",
      "20\t30\t0.0\t2\t0 1 29\n",
      "21\t157\t0.0\t2\t0 156 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t7\t0.0\t2\t7\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/359.U_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/359.U_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/359.U_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/359.U_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.27 s (22 us/read; 2.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,674\n",
      "  Read 1 with adapter:                      55 (0.1%)\n",
      "  Read 2 with adapter:                   1,134 (1.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       103,674 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,140,754 bp\n",
      "  Read 1:    23,194,180 bp\n",
      "  Read 2:    23,946,574 bp\n",
      "Total written (filtered):     47,121,234 bp (100.0%)\n",
      "  Read 1:    23,193,933 bp\n",
      "  Read 2:    23,927,301 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t1619.9\t0\t53\n",
      "24\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1134 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1619.9\t0\t3\n",
      "5\t1\t101.2\t0\t1\n",
      "6\t1\t25.3\t0\t1\n",
      "7\t11\t6.3\t0\t11\n",
      "8\t5\t1.6\t0\t5\n",
      "9\t43\t0.4\t0\t43\n",
      "10\t88\t0.1\t1\t88\n",
      "11\t12\t0.0\t1\t12\n",
      "12\t24\t0.0\t1\t24\n",
      "13\t19\t0.0\t1\t19\n",
      "14\t34\t0.0\t1\t33 1\n",
      "15\t43\t0.0\t1\t42 1\n",
      "16\t60\t0.0\t1\t58 2\n",
      "17\t133\t0.0\t1\t130 3\n",
      "18\t165\t0.0\t1\t158 6 1\n",
      "19\t410\t0.0\t1\t292 117 1\n",
      "20\t15\t0.0\t2\t0 0 15\n",
      "21\t64\t0.0\t2\t0 64\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "217\t2\t0.0\t2\t2\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/351.D_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/351.D_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/351.D_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/351.D_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/351.U_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/351.U_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/351.U_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/351.U_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/352.D_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/352.D_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/352.D_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/352.D_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/352.U_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/352.U_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/352.U_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/352.U_14_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/353.H_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/353.H_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/353.H_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/353.H_26_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/354.H_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/354.H_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/354.H_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/354.H_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/355.H_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/355.H_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/355.H_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/355.H_24_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/356.D_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/356.D_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/356.D_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/356.D_3_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/356.U_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/356.U_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/356.U_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/356.U_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/357.D_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/357.D_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/357.D_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/357.D_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/357.U_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/357.U_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/357.U_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/357.U_18_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/358.D_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/358.D_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/358.D_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/358.D_21_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/358.U_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/358.U_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/358.U_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/358.U_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/359.D_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/359.D_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/359.D_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/359.D_15_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/359.U_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/359.U_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/359.U_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/359.U_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/360.U_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/360.U_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/360.U_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/360.U_40_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/360.U_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y2nhnblr/360.U_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/360.U_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-jy9e58le/5a25a290-d9e9-46c7-b0c9-28a4afab70e0/data/360.U_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.15 s (23 us/read; 2.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            136,430\n",
      "  Read 1 with adapter:                      50 (0.0%)\n",
      "  Read 2 with adapter:                   1,598 (1.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       136,430 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,088,263 bp\n",
      "  Read 1:    30,551,619 bp\n",
      "  Read 2:    31,536,644 bp\n",
      "Total written (filtered):     62,061,773 bp (100.0%)\n",
      "  Read 1:    30,551,445 bp\n",
      "  Read 2:    31,510,328 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 50 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t2131.7\t0\t49\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1598 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t2131.7\t0\t4\n",
      "4\t4\t532.9\t0\t4\n",
      "5\t1\t133.2\t0\t1\n",
      "6\t4\t33.3\t0\t4\n",
      "7\t14\t8.3\t0\t14\n",
      "8\t19\t2.1\t0\t19\n",
      "9\t65\t0.5\t0\t64 1\n",
      "10\t164\t0.1\t1\t162 2\n",
      "11\t32\t0.0\t1\t32\n",
      "12\t39\t0.0\t1\t38 1\n",
      "13\t21\t0.0\t1\t21\n",
      "14\t44\t0.0\t1\t43 1\n",
      "15\t76\t0.0\t1\t74 2\n",
      "16\t105\t0.0\t1\t104 1\n",
      "17\t188\t0.0\t1\t184 4\n",
      "18\t226\t0.0\t1\t214 6 6\n",
      "19\t502\t0.0\t1\t360 141 1\n",
      "20\t12\t0.0\t2\t1 0 11\n",
      "21\t75\t0.0\t2\t0 75\n",
      "204\t1\t0.0\t2\t0 1\n",
      "217\t2\t0.0\t2\t2\n",
      "\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGmicro2018_49-96.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/360.D_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/360.D_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/360.D_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/360.D_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.01 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,533\n",
      "  Read 1 with adapter:                      31 (0.0%)\n",
      "  Read 2 with adapter:                     929 (1.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        90,533 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,198,361 bp\n",
      "  Read 1:    20,271,847 bp\n",
      "  Read 2:    20,926,514 bp\n",
      "Total written (filtered):     41,182,664 bp (100.0%)\n",
      "  Read 1:    20,271,632 bp\n",
      "  Read 2:    20,911,032 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 31 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t1414.6\t0\t30\n",
      "125\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 929 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t88.4\t0\t1\n",
      "6\t2\t22.1\t0\t2\n",
      "7\t4\t5.5\t0\t4\n",
      "8\t8\t1.4\t0\t8\n",
      "9\t47\t0.3\t0\t45 2\n",
      "10\t105\t0.1\t1\t103 2\n",
      "11\t17\t0.0\t1\t17\n",
      "12\t8\t0.0\t1\t8\n",
      "13\t18\t0.0\t1\t18\n",
      "14\t15\t0.0\t1\t15\n",
      "15\t43\t0.0\t1\t43\n",
      "16\t70\t0.0\t1\t70\n",
      "17\t91\t0.0\t1\t90 1\n",
      "18\t169\t0.0\t1\t166 2 1\n",
      "19\t264\t0.0\t1\t207 56 1\n",
      "20\t15\t0.0\t2\t0 0 15\n",
      "21\t50\t0.0\t2\t0 50\n",
      "217\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/361.H_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/361.H_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/361.H_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/361.H_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.34 s (18 us/read; 3.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            187,186\n",
      "  Read 1 with adapter:                     109 (0.1%)\n",
      "  Read 2 with adapter:                   2,895 (1.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       187,186 (100.0%)\n",
      "\n",
      "Total basepairs processed:    85,171,607 bp\n",
      "  Read 1:    41,896,817 bp\n",
      "  Read 2:    43,274,790 bp\n",
      "Total written (filtered):     85,127,695 bp (99.9%)\n",
      "  Read 1:    41,896,468 bp\n",
      "  Read 2:    43,231,227 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t108\t2924.8\t0\t108\n",
      "25\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2895 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t2924.8\t0\t10\n",
      "4\t2\t731.2\t0\t2\n",
      "5\t4\t182.8\t0\t4\n",
      "6\t8\t45.7\t0\t8\n",
      "7\t42\t11.4\t0\t42\n",
      "8\t44\t2.9\t0\t44\n",
      "9\t204\t0.7\t0\t204\n",
      "10\t596\t0.2\t1\t595 1\n",
      "11\t95\t0.0\t1\t94 1\n",
      "12\t78\t0.0\t1\t78\n",
      "13\t39\t0.0\t1\t39\n",
      "14\t43\t0.0\t1\t43\n",
      "15\t151\t0.0\t1\t149 2\n",
      "16\t142\t0.0\t1\t139 3\n",
      "17\t235\t0.0\t1\t227 8\n",
      "18\t376\t0.0\t1\t366 9 1\n",
      "19\t650\t0.0\t1\t484 166\n",
      "20\t23\t0.0\t2\t0 0 23\n",
      "21\t150\t0.0\t2\t0 150\n",
      "215\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/362.H_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/362.H_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/362.H_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/362.H_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.85 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,998\n",
      "  Read 1 with adapter:                      42 (0.1%)\n",
      "  Read 2 with adapter:                     868 (1.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,998 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,583,402 bp\n",
      "  Read 1:    17,016,187 bp\n",
      "  Read 2:    17,567,215 bp\n",
      "Total written (filtered):     34,569,686 bp (100.0%)\n",
      "  Read 1:    17,016,061 bp\n",
      "  Read 2:    17,553,625 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 42 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t1187.5\t0\t42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 868 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1187.5\t0\t1\n",
      "6\t3\t18.6\t0\t3\n",
      "7\t12\t4.6\t0\t12\n",
      "8\t13\t1.2\t0\t13\n",
      "9\t42\t0.3\t0\t42\n",
      "10\t130\t0.1\t1\t130\n",
      "11\t20\t0.0\t1\t19 1\n",
      "12\t18\t0.0\t1\t18\n",
      "13\t12\t0.0\t1\t12\n",
      "14\t15\t0.0\t1\t15\n",
      "15\t47\t0.0\t1\t47\n",
      "16\t57\t0.0\t1\t56 1\n",
      "17\t96\t0.0\t1\t93 3\n",
      "18\t115\t0.0\t1\t110 3 2\n",
      "19\t229\t0.0\t1\t159 70\n",
      "20\t11\t0.0\t2\t0 0 11\n",
      "21\t46\t0.0\t2\t0 46\n",
      "45\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/363.H_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/363.H_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/363.H_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/363.H_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.23 s (31 us/read; 1.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,411\n",
      "  Read 1 with adapter:                      20 (0.1%)\n",
      "  Read 2 with adapter:                     673 (1.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        39,411 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,836,067 bp\n",
      "  Read 1:     8,768,127 bp\n",
      "  Read 2:     9,067,940 bp\n",
      "Total written (filtered):     17,825,502 bp (99.9%)\n",
      "  Read 1:     8,767,854 bp\n",
      "  Read 2:     9,057,648 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t615.8\t0\t19\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 673 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t615.8\t0\t1\n",
      "4\t1\t153.9\t0\t1\n",
      "5\t2\t38.5\t0\t2\n",
      "6\t1\t9.6\t0\t1\n",
      "7\t10\t2.4\t0\t10\n",
      "8\t9\t0.6\t0\t9\n",
      "9\t35\t0.2\t0\t35\n",
      "10\t90\t0.0\t1\t90\n",
      "11\t32\t0.0\t1\t32\n",
      "12\t28\t0.0\t1\t28\n",
      "13\t7\t0.0\t1\t7\n",
      "14\t10\t0.0\t1\t10\n",
      "15\t46\t0.0\t1\t45 1\n",
      "16\t43\t0.0\t1\t43\n",
      "17\t65\t0.0\t1\t65\n",
      "18\t117\t0.0\t1\t113 4\n",
      "19\t143\t0.0\t1\t109 34\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t27\t0.0\t2\t0 27\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/364.D_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/364.D_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/364.D_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/364.D_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.54 s (21 us/read; 2.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            217,037\n",
      "  Read 1 with adapter:                     120 (0.1%)\n",
      "  Read 2 with adapter:                   7,126 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       217,037 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,877,215 bp\n",
      "  Read 1:    48,605,330 bp\n",
      "  Read 2:    50,271,885 bp\n",
      "Total written (filtered):     98,785,288 bp (99.9%)\n",
      "  Read 1:    48,604,287 bp\n",
      "  Read 2:    50,181,001 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 120 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t112\t3391.2\t0\t112\n",
      "9\t2\t0.8\t0\t2\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7126 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t3391.2\t0\t50\n",
      "4\t12\t847.8\t0\t12\n",
      "5\t18\t212.0\t0\t18\n",
      "6\t48\t53.0\t0\t48\n",
      "7\t264\t13.2\t0\t264\n",
      "8\t359\t3.3\t0\t359\n",
      "9\t1399\t0.8\t0\t1399\n",
      "10\t1811\t0.2\t1\t1804 7\n",
      "11\t98\t0.1\t1\t98\n",
      "12\t126\t0.0\t1\t125 1\n",
      "13\t130\t0.0\t1\t130\n",
      "14\t171\t0.0\t1\t166 5\n",
      "15\t213\t0.0\t1\t210 3\n",
      "16\t248\t0.0\t1\t240 8\n",
      "17\t461\t0.0\t1\t448 13\n",
      "18\t507\t0.0\t1\t484 12 11\n",
      "19\t1051\t0.0\t1\t698 351 2\n",
      "20\t21\t0.0\t2\t0 0 21\n",
      "21\t135\t0.0\t2\t0 134 1\n",
      "215\t1\t0.0\t2\t0 0 1\n",
      "216\t3\t0.0\t2\t3\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/364.U_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/364.U_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/364.U_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/364.U_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.40 s (21 us/read; 2.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            162,169\n",
      "  Read 1 with adapter:                     103 (0.1%)\n",
      "  Read 2 with adapter:                   4,862 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       162,169 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,862,911 bp\n",
      "  Read 1:    36,313,904 bp\n",
      "  Read 2:    37,549,007 bp\n",
      "Total written (filtered):     73,799,300 bp (99.9%)\n",
      "  Read 1:    36,312,926 bp\n",
      "  Read 2:    37,486,374 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t2533.9\t0\t85\n",
      "27\t14\t0.0\t1\t14\n",
      "38\t2\t0.0\t1\t2\n",
      "54\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4862 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t2533.9\t0\t19\n",
      "4\t4\t633.5\t0\t4\n",
      "5\t4\t158.4\t0\t4\n",
      "6\t33\t39.6\t0\t33\n",
      "7\t170\t9.9\t0\t170\n",
      "8\t260\t2.5\t0\t260\n",
      "9\t878\t0.6\t0\t878\n",
      "10\t1225\t0.2\t1\t1223 2\n",
      "11\t63\t0.0\t1\t63\n",
      "12\t79\t0.0\t1\t77 2\n",
      "13\t67\t0.0\t1\t65 2\n",
      "14\t120\t0.0\t1\t117 3\n",
      "15\t162\t0.0\t1\t159 3\n",
      "16\t196\t0.0\t1\t190 6\n",
      "17\t360\t0.0\t1\t344 16\n",
      "18\t366\t0.0\t1\t347 11 8\n",
      "19\t757\t0.0\t1\t460 295 2\n",
      "20\t12\t0.0\t2\t0 1 11\n",
      "21\t85\t0.0\t2\t0 84 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/365.D_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/365.D_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/365.D_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/365.D_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.27 s (23 us/read; 2.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            143,963\n",
      "  Read 1 with adapter:                     111 (0.1%)\n",
      "  Read 2 with adapter:                   5,566 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       143,963 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,598,323 bp\n",
      "  Read 1:    32,237,313 bp\n",
      "  Read 2:    33,361,010 bp\n",
      "Total written (filtered):     65,527,741 bp (99.9%)\n",
      "  Read 1:    32,236,260 bp\n",
      "  Read 2:    33,291,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 111 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t100\t2249.4\t0\t100\n",
      "23\t1\t0.0\t1\t0 1\n",
      "27\t6\t0.0\t1\t6\n",
      "48\t1\t0.0\t1\t1\n",
      "151\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5566 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t2249.4\t0\t42\n",
      "4\t3\t562.4\t0\t3\n",
      "5\t15\t140.6\t0\t15\n",
      "6\t47\t35.1\t0\t47\n",
      "7\t194\t8.8\t0\t194\n",
      "8\t296\t2.2\t0\t296\n",
      "9\t1165\t0.5\t0\t1165\n",
      "10\t1428\t0.1\t1\t1424 4\n",
      "11\t68\t0.0\t1\t68\n",
      "12\t97\t0.0\t1\t92 5\n",
      "13\t109\t0.0\t1\t108 1\n",
      "14\t163\t0.0\t1\t160 3\n",
      "15\t189\t0.0\t1\t186 3\n",
      "16\t220\t0.0\t1\t216 4\n",
      "17\t354\t0.0\t1\t345 9\n",
      "18\t342\t0.0\t1\t312 23 7\n",
      "19\t730\t0.0\t1\t444 285 1\n",
      "20\t15\t0.0\t2\t0 2 13\n",
      "21\t86\t0.0\t2\t0 86\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/365.U_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/365.U_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/365.U_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/365.U_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.37 s (21 us/read; 2.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            205,597\n",
      "  Read 1 with adapter:                      96 (0.0%)\n",
      "  Read 2 with adapter:                   7,895 (3.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       205,597 (100.0%)\n",
      "\n",
      "Total basepairs processed:    93,679,159 bp\n",
      "  Read 1:    46,035,201 bp\n",
      "  Read 2:    47,643,958 bp\n",
      "Total written (filtered):     93,577,409 bp (99.9%)\n",
      "  Read 1:    46,034,762 bp\n",
      "  Read 2:    47,542,647 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 96 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t95\t3212.5\t0\t95\n",
      "154\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7895 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t3212.5\t0\t43\n",
      "4\t9\t803.1\t0\t9\n",
      "5\t23\t200.8\t0\t23\n",
      "6\t60\t50.2\t0\t60\n",
      "7\t271\t12.5\t0\t271\n",
      "8\t386\t3.1\t0\t386\n",
      "9\t1474\t0.8\t0\t1472 2\n",
      "10\t2101\t0.2\t1\t2092 9\n",
      "11\t113\t0.0\t1\t112 1\n",
      "12\t166\t0.0\t1\t164 2\n",
      "13\t188\t0.0\t1\t186 2\n",
      "14\t215\t0.0\t1\t205 10\n",
      "15\t246\t0.0\t1\t241 5\n",
      "16\t283\t0.0\t1\t274 9\n",
      "17\t476\t0.0\t1\t457 19\n",
      "18\t606\t0.0\t1\t575 15 16\n",
      "19\t1049\t0.0\t1\t664 383 2\n",
      "20\t29\t0.0\t2\t0 0 29\n",
      "21\t143\t0.0\t2\t0 143\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "41\t1\t0.0\t2\t1\n",
      "126\t1\t0.0\t2\t1\n",
      "207\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t3\t0.0\t2\t3\n",
      "229\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/366.D_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/366.D_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/366.D_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/366.D_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.14 s (22 us/read; 2.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            143,852\n",
      "  Read 1 with adapter:                      89 (0.1%)\n",
      "  Read 2 with adapter:                   4,840 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       143,852 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,533,685 bp\n",
      "  Read 1:    32,211,811 bp\n",
      "  Read 2:    33,321,874 bp\n",
      "Total written (filtered):     65,471,009 bp (99.9%)\n",
      "  Read 1:    32,211,077 bp\n",
      "  Read 2:    33,259,932 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 89 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t81\t2247.7\t0\t81\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t3\t0.0\t1\t3\n",
      "120\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4840 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t2247.7\t0\t28\n",
      "4\t5\t561.9\t0\t5\n",
      "5\t13\t140.5\t0\t13\n",
      "6\t24\t35.1\t0\t24\n",
      "7\t174\t8.8\t0\t174\n",
      "8\t266\t2.2\t0\t266\n",
      "9\t900\t0.5\t0\t900\n",
      "10\t1209\t0.1\t1\t1200 9\n",
      "11\t57\t0.0\t1\t57\n",
      "12\t73\t0.0\t1\t71 2\n",
      "13\t88\t0.0\t1\t88\n",
      "14\t155\t0.0\t1\t152 3\n",
      "15\t150\t0.0\t1\t146 4\n",
      "16\t201\t0.0\t1\t197 4\n",
      "17\t346\t0.0\t1\t342 4\n",
      "18\t337\t0.0\t1\t313 17 7\n",
      "19\t710\t0.0\t1\t458 252\n",
      "20\t12\t0.0\t2\t0 0 12\n",
      "21\t89\t0.0\t2\t0 89\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/366.U_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/366.U_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/366.U_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/366.U_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.00 s (20 us/read; 3.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            204,857\n",
      "  Read 1 with adapter:                      91 (0.0%)\n",
      "  Read 2 with adapter:                   7,198 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       204,857 (100.0%)\n",
      "\n",
      "Total basepairs processed:    93,321,447 bp\n",
      "  Read 1:    45,864,961 bp\n",
      "  Read 2:    47,456,486 bp\n",
      "Total written (filtered):     93,226,565 bp (99.9%)\n",
      "  Read 1:    45,863,680 bp\n",
      "  Read 2:    47,362,885 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t3200.9\t0\t79\n",
      "4\t1\t800.2\t0\t1\n",
      "27\t5\t0.0\t1\t4 1\n",
      "101\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t1 1\n",
      "174\t1\t0.0\t1\t0 1\n",
      "187\t1\t0.0\t1\t1\n",
      "203\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7198 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t3200.9\t0\t53\n",
      "4\t9\t800.2\t0\t9\n",
      "5\t14\t200.1\t0\t14\n",
      "6\t42\t50.0\t0\t42\n",
      "7\t221\t12.5\t0\t221\n",
      "8\t315\t3.1\t0\t315\n",
      "9\t1261\t0.8\t0\t1260 1\n",
      "10\t1906\t0.2\t1\t1896 10\n",
      "11\t128\t0.0\t1\t124 4\n",
      "12\t147\t0.0\t1\t145 2\n",
      "13\t148\t0.0\t1\t147 1\n",
      "14\t191\t0.0\t1\t189 2\n",
      "15\t258\t0.0\t1\t253 5\n",
      "16\t254\t0.0\t1\t250 4\n",
      "17\t504\t0.0\t1\t486 18\n",
      "18\t530\t0.0\t1\t499 20 11\n",
      "19\t1036\t0.0\t1\t675 358 3\n",
      "20\t33\t0.0\t2\t0 2 31\n",
      "21\t138\t0.0\t2\t0 138\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t1\n",
      "125\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t4\t0.0\t2\t3 0 1\n",
      "217\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/367.D_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/367.D_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/367.D_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/367.D_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.18 s (17 us/read; 3.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            296,626\n",
      "  Read 1 with adapter:                     193 (0.1%)\n",
      "  Read 2 with adapter:                   8,254 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       296,626 (100.0%)\n",
      "\n",
      "Total basepairs processed:   135,100,777 bp\n",
      "  Read 1:    66,426,477 bp\n",
      "  Read 2:    68,674,300 bp\n",
      "Total written (filtered):    134,988,063 bp (99.9%)\n",
      "  Read 1:    66,424,839 bp\n",
      "  Read 2:    68,563,224 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 193 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t176\t4634.8\t0\t176\n",
      "25\t5\t0.0\t1\t5\n",
      "27\t6\t0.0\t1\t6\n",
      "38\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "106\t1\t0.0\t1\t1\n",
      "170\t1\t0.0\t1\t1\n",
      "187\t1\t0.0\t1\t1\n",
      "221\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8254 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t4634.8\t0\t47\n",
      "4\t7\t1158.7\t0\t7\n",
      "5\t17\t289.7\t0\t17\n",
      "6\t42\t72.4\t0\t42\n",
      "7\t228\t18.1\t0\t228\n",
      "8\t302\t4.5\t0\t302\n",
      "9\t1381\t1.1\t0\t1380 1\n",
      "10\t1931\t0.3\t1\t1918 13\n",
      "11\t130\t0.1\t1\t129 1\n",
      "12\t152\t0.0\t1\t152\n",
      "13\t167\t0.0\t1\t166 1\n",
      "14\t237\t0.0\t1\t230 7\n",
      "15\t288\t0.0\t1\t282 6\n",
      "16\t300\t0.0\t1\t294 6\n",
      "17\t582\t0.0\t1\t562 20\n",
      "18\t733\t0.0\t1\t684 38 11\n",
      "19\t1487\t0.0\t1\t990 494 3\n",
      "20\t45\t0.0\t2\t0 1 44\n",
      "21\t172\t0.0\t2\t0 168 4\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/367.U_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/367.U_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/367.U_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/367.U_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.35 s (20 us/read; 3.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            170,770\n",
      "  Read 1 with adapter:                     111 (0.1%)\n",
      "  Read 2 with adapter:                   4,470 (2.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       170,770 (100.0%)\n",
      "\n",
      "Total basepairs processed:    77,775,476 bp\n",
      "  Read 1:    38,242,444 bp\n",
      "  Read 2:    39,533,032 bp\n",
      "Total written (filtered):     77,714,560 bp (99.9%)\n",
      "  Read 1:    38,241,342 bp\n",
      "  Read 2:    39,473,218 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 111 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t103\t2668.3\t0\t103\n",
      "24\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "121\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "219\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4470 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t2668.3\t0\t20\n",
      "4\t3\t667.1\t0\t3\n",
      "5\t5\t166.8\t0\t5\n",
      "6\t18\t41.7\t0\t18\n",
      "7\t134\t10.4\t0\t134\n",
      "8\t169\t2.6\t0\t169\n",
      "9\t733\t0.7\t0\t733\n",
      "10\t1134\t0.2\t1\t1127 7\n",
      "11\t75\t0.0\t1\t75\n",
      "12\t83\t0.0\t1\t82 1\n",
      "13\t79\t0.0\t1\t79\n",
      "14\t84\t0.0\t1\t83 1\n",
      "15\t176\t0.0\t1\t170 6\n",
      "16\t153\t0.0\t1\t151 2\n",
      "17\t328\t0.0\t1\t324 4\n",
      "18\t399\t0.0\t1\t371 17 11\n",
      "19\t733\t0.0\t1\t479 254\n",
      "20\t25\t0.0\t2\t0 1 24\n",
      "21\t114\t0.0\t2\t0 114\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/368.D_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/368.D_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/368.D_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/368.D_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.49 s (18 us/read; 3.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            243,134\n",
      "  Read 1 with adapter:                     117 (0.0%)\n",
      "  Read 2 with adapter:                   8,602 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       243,134 (100.0%)\n",
      "\n",
      "Total basepairs processed:   110,769,894 bp\n",
      "  Read 1:    54,444,430 bp\n",
      "  Read 2:    56,325,464 bp\n",
      "Total written (filtered):    110,653,567 bp (99.9%)\n",
      "  Read 1:    54,443,805 bp\n",
      "  Read 2:    56,209,762 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 117 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t3799.0\t0\t109\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t5\t0.0\t1\t5\n",
      "38\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8602 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t3799.0\t0\t44\n",
      "4\t6\t949.7\t0\t6\n",
      "5\t13\t237.4\t0\t13\n",
      "6\t47\t59.4\t0\t47\n",
      "7\t256\t14.8\t0\t256\n",
      "8\t377\t3.7\t0\t377\n",
      "9\t1390\t0.9\t0\t1390\n",
      "10\t2156\t0.2\t1\t2134 22\n",
      "11\t134\t0.1\t1\t131 3\n",
      "12\t153\t0.0\t1\t152 1\n",
      "13\t151\t0.0\t1\t148 3\n",
      "14\t207\t0.0\t1\t195 12\n",
      "15\t295\t0.0\t1\t280 15\n",
      "16\t303\t0.0\t1\t295 8\n",
      "17\t443\t0.0\t1\t426 17\n",
      "18\t704\t0.0\t1\t649 38 17\n",
      "19\t1291\t0.0\t1\t838 451 2\n",
      "20\t457\t0.0\t2\t0 2 455\n",
      "21\t166\t0.0\t2\t0 163 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "189\t1\t0.0\t2\t1\n",
      "206\t1\t0.0\t2\t1\n",
      "216\t5\t0.0\t2\t5\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/368.U_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/368.U_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/368.U_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/368.U_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.57 s (20 us/read; 3.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            178,272\n",
      "  Read 1 with adapter:                     109 (0.1%)\n",
      "  Read 2 with adapter:                   6,413 (3.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       178,272 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,213,512 bp\n",
      "  Read 1:    39,922,038 bp\n",
      "  Read 2:    41,291,474 bp\n",
      "Total written (filtered):     81,130,357 bp (99.9%)\n",
      "  Read 1:    39,920,117 bp\n",
      "  Read 2:    41,210,240 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t2785.5\t0\t91\n",
      "16\t1\t0.0\t1\t0 1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t4\t0.0\t1\t4\n",
      "119\t1\t0.0\t1\t1\n",
      "120\t10\t0.0\t1\t10\n",
      "180\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6413 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t2785.5\t0\t40\n",
      "4\t11\t696.4\t0\t11\n",
      "5\t17\t174.1\t0\t17\n",
      "6\t49\t43.5\t0\t49\n",
      "7\t264\t10.9\t0\t264\n",
      "8\t342\t2.7\t0\t342\n",
      "9\t1303\t0.7\t0\t1300 3\n",
      "10\t1614\t0.2\t1\t1605 9\n",
      "11\t90\t0.0\t1\t90\n",
      "12\t98\t0.0\t1\t98\n",
      "13\t136\t0.0\t1\t135 1\n",
      "14\t164\t0.0\t1\t162 2\n",
      "15\t197\t0.0\t1\t193 4\n",
      "16\t221\t0.0\t1\t216 5\n",
      "17\t425\t0.0\t1\t414 11\n",
      "18\t405\t0.0\t1\t378 15 12\n",
      "19\t869\t0.0\t1\t579 289 1\n",
      "20\t27\t0.0\t2\t0 1 26\n",
      "21\t134\t0.0\t2\t0 132 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "216\t3\t0.0\t2\t2 1\n",
      "217\t1\t0.0\t2\t1\n",
      "230\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/369.H_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/369.H_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/369.H_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/369.H_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.01 s (20 us/read; 2.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            198,178\n",
      "  Read 1 with adapter:                     133 (0.1%)\n",
      "  Read 2 with adapter:                   5,428 (2.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       198,178 (100.0%)\n",
      "\n",
      "Total basepairs processed:    90,246,492 bp\n",
      "  Read 1:    44,369,534 bp\n",
      "  Read 2:    45,876,958 bp\n",
      "Total written (filtered):     90,173,281 bp (99.9%)\n",
      "  Read 1:    44,368,043 bp\n",
      "  Read 2:    45,805,238 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 133 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t124\t3096.5\t0\t124\n",
      "25\t1\t0.0\t1\t1\n",
      "38\t2\t0.0\t1\t2\n",
      "116\t1\t0.0\t1\t1\n",
      "119\t1\t0.0\t1\t1\n",
      "180\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t1\n",
      "195\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5428 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t3096.5\t0\t39\n",
      "4\t2\t774.1\t0\t2\n",
      "5\t8\t193.5\t0\t8\n",
      "6\t30\t48.4\t0\t30\n",
      "7\t177\t12.1\t0\t177\n",
      "8\t234\t3.0\t0\t234\n",
      "9\t961\t0.8\t0\t959 2\n",
      "10\t1365\t0.2\t1\t1357 8\n",
      "11\t78\t0.0\t1\t77 1\n",
      "12\t88\t0.0\t1\t88\n",
      "13\t93\t0.0\t1\t93\n",
      "14\t117\t0.0\t1\t112 5\n",
      "15\t164\t0.0\t1\t158 6\n",
      "16\t209\t0.0\t1\t200 9\n",
      "17\t368\t0.0\t1\t361 7\n",
      "18\t463\t0.0\t1\t440 16 7\n",
      "19\t876\t0.0\t1\t576 298 2\n",
      "20\t26\t0.0\t2\t0 0 26\n",
      "21\t124\t0.0\t2\t0 122 2\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t2\t0.0\t2\t1 1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/360.D_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/360.D_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/360.D_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/360.D_21_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/361.H_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/361.H_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/361.H_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/361.H_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/362.H_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/362.H_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/362.H_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/362.H_1_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/363.H_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/363.H_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/363.H_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/363.H_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/364.D_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/364.D_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/364.D_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/364.D_27_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/364.U_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/364.U_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/364.U_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/364.U_18_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/365.D_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/365.D_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/365.D_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/365.D_31_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/365.U_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/365.U_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/365.U_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/365.U_4_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/366.D_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/366.D_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/366.D_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/366.D_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/366.U_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/366.U_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/366.U_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/366.U_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/367.D_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/367.D_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/367.D_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/367.D_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/367.U_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/367.U_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/367.U_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/367.U_7_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/368.D_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/368.D_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/368.D_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/368.D_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/368.U_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/368.U_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/368.U_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/368.U_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/369.H_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/369.H_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/369.H_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/369.H_11_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/370.H_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/370.H_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/370.H_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/370.H_15_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/370.H_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/370.H_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/370.H_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/370.H_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.54 s (19 us/read; 3.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            233,630\n",
      "  Read 1 with adapter:                     131 (0.1%)\n",
      "  Read 2 with adapter:                   6,494 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       233,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:   106,399,347 bp\n",
      "  Read 1:    52,312,922 bp\n",
      "  Read 2:    54,086,425 bp\n",
      "Total written (filtered):    106,311,213 bp (99.9%)\n",
      "  Read 1:    52,311,903 bp\n",
      "  Read 2:    53,999,310 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 131 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t121\t3650.5\t0\t121\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t5\t0.0\t1\t5\n",
      "38\t2\t0.0\t1\t2\n",
      "202\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6494 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t3650.5\t0\t46\n",
      "4\t4\t912.6\t0\t4\n",
      "5\t11\t228.2\t0\t11\n",
      "6\t36\t57.0\t0\t36\n",
      "7\t198\t14.3\t0\t198\n",
      "8\t204\t3.6\t0\t204\n",
      "9\t983\t0.9\t0\t981 2\n",
      "10\t1577\t0.2\t1\t1570 7\n",
      "11\t132\t0.1\t1\t131 1\n",
      "12\t177\t0.0\t1\t176 1\n",
      "13\t158\t0.0\t1\t157 1\n",
      "14\t192\t0.0\t1\t182 10\n",
      "15\t264\t0.0\t1\t257 7\n",
      "16\t221\t0.0\t1\t214 7\n",
      "17\t484\t0.0\t1\t462 22\n",
      "18\t591\t0.0\t1\t553 22 16\n",
      "19\t1057\t0.0\t1\t700 356 1\n",
      "20\t24\t0.0\t2\t0 2 22\n",
      "21\t130\t0.0\t2\t0 126 4\n",
      "216\t3\t0.0\t2\t3\n",
      "217\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/371.H_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/371.H_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/371.H_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/371.H_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.70 s (20 us/read; 2.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            180,998\n",
      "  Read 1 with adapter:                     114 (0.1%)\n",
      "  Read 2 with adapter:                   4,847 (2.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       180,998 (100.0%)\n",
      "\n",
      "Total basepairs processed:    82,421,287 bp\n",
      "  Read 1:    40,523,405 bp\n",
      "  Read 2:    41,897,882 bp\n",
      "Total written (filtered):     82,354,808 bp (99.9%)\n",
      "  Read 1:    40,522,128 bp\n",
      "  Read 2:    41,832,680 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 114 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t99\t2828.1\t0\t99\n",
      "25\t4\t0.0\t1\t4\n",
      "27\t5\t0.0\t1\t5\n",
      "38\t2\t0.0\t1\t2\n",
      "45\t1\t0.0\t1\t1\n",
      "188\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4847 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t2828.1\t0\t35\n",
      "4\t10\t707.0\t0\t10\n",
      "5\t9\t176.8\t0\t9\n",
      "6\t28\t44.2\t0\t28\n",
      "7\t125\t11.0\t0\t125\n",
      "8\t195\t2.8\t0\t195\n",
      "9\t792\t0.7\t0\t792\n",
      "10\t1194\t0.2\t1\t1190 4\n",
      "11\t89\t0.0\t1\t88 1\n",
      "12\t92\t0.0\t1\t92\n",
      "13\t87\t0.0\t1\t86 1\n",
      "14\t105\t0.0\t1\t100 5\n",
      "15\t183\t0.0\t1\t181 2\n",
      "16\t194\t0.0\t1\t189 5\n",
      "17\t348\t0.0\t1\t338 10\n",
      "18\t407\t0.0\t1\t389 11 7\n",
      "19\t808\t0.0\t1\t527 280 1\n",
      "20\t21\t0.0\t2\t0 1 20\n",
      "21\t115\t0.0\t2\t0 115\n",
      "22\t3\t0.0\t2\t0 1 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "208\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t2 1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/372.D_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/372.D_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/372.D_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/372.D_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.51 s (20 us/read; 3.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            128,703\n",
      "  Read 1 with adapter:                      64 (0.0%)\n",
      "  Read 2 with adapter:                   6,321 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       128,703 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,516,553 bp\n",
      "  Read 1:    28,732,988 bp\n",
      "  Read 2:    29,783,565 bp\n",
      "Total written (filtered):     58,441,524 bp (99.9%)\n",
      "  Read 1:    28,731,918 bp\n",
      "  Read 2:    29,709,606 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 64 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t55\t2011.0\t0\t55\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "53\t1\t0.0\t1\t1\n",
      "101\t2\t0.0\t1\t2\n",
      "132\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t1\n",
      "223\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6321 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t2011.0\t0\t40\n",
      "4\t10\t502.7\t0\t10\n",
      "5\t15\t125.7\t0\t15\n",
      "6\t69\t31.4\t0\t69\n",
      "7\t356\t7.9\t0\t356\n",
      "8\t376\t2.0\t0\t376\n",
      "9\t1598\t0.5\t0\t1597 1\n",
      "10\t1872\t0.1\t1\t1860 12\n",
      "11\t74\t0.0\t1\t74\n",
      "12\t72\t0.0\t1\t71 1\n",
      "13\t96\t0.0\t1\t96\n",
      "14\t102\t0.0\t1\t99 3\n",
      "15\t159\t0.0\t1\t152 7\n",
      "16\t143\t0.0\t1\t139 4\n",
      "17\t256\t0.0\t1\t243 13\n",
      "18\t338\t0.0\t1\t313 14 11\n",
      "19\t633\t0.0\t1\t403 226 4\n",
      "20\t20\t0.0\t2\t0 1 19\n",
      "21\t83\t0.0\t2\t0 82 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "61\t3\t0.0\t2\t3\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "214\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/372.U_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/372.U_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/372.U_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/372.U_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.09 s (35 us/read; 1.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,743\n",
      "  Read 1 with adapter:                      22 (0.1%)\n",
      "  Read 2 with adapter:                   1,273 (4.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,743 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,783,771 bp\n",
      "  Read 1:     6,762,177 bp\n",
      "  Read 2:     7,021,594 bp\n",
      "Total written (filtered):     13,766,557 bp (99.9%)\n",
      "  Read 1:     6,761,492 bp\n",
      "  Read 2:     7,005,065 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t480.4\t0\t19\n",
      "196\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1273 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t480.4\t0\t4\n",
      "4\t2\t120.1\t0\t2\n",
      "5\t3\t30.0\t0\t3\n",
      "6\t12\t7.5\t0\t12\n",
      "7\t50\t1.9\t0\t50\n",
      "8\t75\t0.5\t0\t75\n",
      "9\t243\t0.1\t0\t243\n",
      "10\t306\t0.0\t1\t306\n",
      "11\t10\t0.0\t1\t10\n",
      "12\t19\t0.0\t1\t19\n",
      "13\t23\t0.0\t1\t22 1\n",
      "14\t33\t0.0\t1\t29 4\n",
      "15\t44\t0.0\t1\t43 1\n",
      "16\t49\t0.0\t1\t47 2\n",
      "17\t91\t0.0\t1\t87 4\n",
      "18\t91\t0.0\t1\t88 2 1\n",
      "19\t182\t0.0\t1\t114 68\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t27\t0.0\t2\t0 27\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/373.D_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/373.D_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/373.D_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/373.D_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.77 s (19 us/read; 3.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            145,049\n",
      "  Read 1 with adapter:                      99 (0.1%)\n",
      "  Read 2 with adapter:                   6,027 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       145,049 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,963,800 bp\n",
      "  Read 1:    32,403,685 bp\n",
      "  Read 2:    33,560,115 bp\n",
      "Total written (filtered):     65,888,574 bp (99.9%)\n",
      "  Read 1:    32,401,704 bp\n",
      "  Read 2:    33,486,870 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t2266.4\t0\t85\n",
      "53\t1\t0.0\t1\t1\n",
      "99\t1\t0.0\t1\t1\n",
      "100\t8\t0.0\t1\t8\n",
      "152\t1\t0.0\t1\t1\n",
      "190\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6027 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t2266.4\t0\t37\n",
      "4\t9\t566.6\t0\t9\n",
      "5\t18\t141.6\t0\t18\n",
      "6\t51\t35.4\t0\t51\n",
      "7\t283\t8.9\t0\t283\n",
      "8\t323\t2.2\t0\t323\n",
      "9\t1430\t0.6\t0\t1430\n",
      "10\t1801\t0.1\t1\t1795 6\n",
      "11\t56\t0.0\t1\t55 1\n",
      "12\t59\t0.0\t1\t59\n",
      "13\t66\t0.0\t1\t66\n",
      "14\t95\t0.0\t1\t92 3\n",
      "15\t142\t0.0\t1\t141 1\n",
      "16\t151\t0.0\t1\t149 2\n",
      "17\t290\t0.0\t1\t276 14\n",
      "18\t365\t0.0\t1\t339 17 9\n",
      "19\t708\t0.0\t1\t449 258 1\n",
      "20\t21\t0.0\t2\t0 0 21\n",
      "21\t112\t0.0\t2\t0 111 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "116\t1\t0.0\t2\t1\n",
      "202\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t3\n",
      "217\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/373.U_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/373.U_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/373.U_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/373.U_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.52 s (28 us/read; 2.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,708\n",
      "  Read 1 with adapter:                      33 (0.1%)\n",
      "  Read 2 with adapter:                   1,529 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,708 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,240,749 bp\n",
      "  Read 1:    11,908,650 bp\n",
      "  Read 2:    12,332,099 bp\n",
      "Total written (filtered):     24,219,574 bp (99.9%)\n",
      "  Read 1:    11,908,069 bp\n",
      "  Read 2:    12,311,505 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 33 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t839.2\t0\t28\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "210\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1529 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t839.2\t0\t6\n",
      "4\t1\t209.8\t0\t1\n",
      "5\t3\t52.4\t0\t3\n",
      "6\t11\t13.1\t0\t11\n",
      "7\t45\t3.3\t0\t45\n",
      "8\t63\t0.8\t0\t63\n",
      "9\t251\t0.2\t0\t251\n",
      "10\t322\t0.1\t1\t321 1\n",
      "11\t23\t0.0\t1\t23\n",
      "12\t30\t0.0\t1\t30\n",
      "13\t28\t0.0\t1\t28\n",
      "14\t39\t0.0\t1\t39\n",
      "15\t65\t0.0\t1\t65\n",
      "16\t58\t0.0\t1\t55 3\n",
      "17\t134\t0.0\t1\t131 3\n",
      "18\t139\t0.0\t1\t131 4 4\n",
      "19\t273\t0.0\t1\t182 91\n",
      "20\t7\t0.0\t2\t0 1 6\n",
      "21\t30\t0.0\t2\t0 29 1\n",
      "30\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/374.D_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/374.D_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/374.D_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/374.D_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.31 s (22 us/read; 2.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,980\n",
      "  Read 1 with adapter:                      72 (0.1%)\n",
      "  Read 2 with adapter:                   4,513 (4.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       104,980 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,766,108 bp\n",
      "  Read 1:    23,466,125 bp\n",
      "  Read 2:    24,299,983 bp\n",
      "Total written (filtered):     47,711,775 bp (99.9%)\n",
      "  Read 1:    23,465,403 bp\n",
      "  Read 2:    24,246,372 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 72 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t67\t1640.3\t0\t67\n",
      "27\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4513 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1640.3\t0\t29\n",
      "4\t7\t410.1\t0\t7\n",
      "5\t14\t102.5\t0\t14\n",
      "6\t50\t25.6\t0\t50\n",
      "7\t224\t6.4\t0\t224\n",
      "8\t340\t1.6\t0\t340\n",
      "9\t1075\t0.4\t0\t1073 2\n",
      "10\t1267\t0.1\t1\t1262 5\n",
      "11\t35\t0.0\t1\t34 1\n",
      "12\t38\t0.0\t1\t38\n",
      "13\t58\t0.0\t1\t58\n",
      "14\t81\t0.0\t1\t78 3\n",
      "15\t88\t0.0\t1\t87 1\n",
      "16\t139\t0.0\t1\t136 3\n",
      "17\t224\t0.0\t1\t220 4\n",
      "18\t267\t0.0\t1\t251 10 6\n",
      "19\t500\t0.0\t1\t310 190\n",
      "20\t12\t0.0\t2\t0 2 10\n",
      "21\t62\t0.0\t2\t0 62\n",
      "129\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/374.U_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/374.U_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/374.U_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/374.U_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.73 s (28 us/read; 2.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,255\n",
      "  Read 1 with adapter:                      37 (0.1%)\n",
      "  Read 2 with adapter:                   1,807 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        61,255 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,773,134 bp\n",
      "  Read 1:    13,648,775 bp\n",
      "  Read 2:    14,124,359 bp\n",
      "Total written (filtered):     27,748,448 bp (99.9%)\n",
      "  Read 1:    13,648,305 bp\n",
      "  Read 2:    14,100,143 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 37 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t957.1\t0\t32\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "79\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1807 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t957.1\t0\t10\n",
      "4\t2\t239.3\t0\t2\n",
      "5\t3\t59.8\t0\t3\n",
      "6\t10\t15.0\t0\t10\n",
      "7\t50\t3.7\t0\t50\n",
      "8\t81\t0.9\t0\t81\n",
      "9\t257\t0.2\t0\t256 1\n",
      "10\t424\t0.1\t1\t421 3\n",
      "11\t28\t0.0\t1\t28\n",
      "12\t44\t0.0\t1\t43 1\n",
      "13\t35\t0.0\t1\t34 1\n",
      "14\t61\t0.0\t1\t57 4\n",
      "15\t73\t0.0\t1\t71 2\n",
      "16\t76\t0.0\t1\t72 4\n",
      "17\t135\t0.0\t1\t129 6\n",
      "18\t153\t0.0\t1\t139 9 5\n",
      "19\t307\t0.0\t1\t198 108 1\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t54\t0.0\t2\t0 54\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/375.D_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/375.D_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/375.D_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/375.D_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.76 s (17 us/read; 3.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            160,215\n",
      "  Read 1 with adapter:                     106 (0.1%)\n",
      "  Read 2 with adapter:                   8,284 (5.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       160,215 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,033,115 bp\n",
      "  Read 1:    35,869,052 bp\n",
      "  Read 2:    37,164,063 bp\n",
      "Total written (filtered):     72,935,247 bp (99.9%)\n",
      "  Read 1:    35,865,595 bp\n",
      "  Read 2:    37,069,652 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 106 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t75\t2503.4\t0\t75\n",
      "27\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t0 1\n",
      "76\t1\t0.0\t1\t1\n",
      "100\t6\t0.0\t1\t6\n",
      "101\t8\t0.0\t1\t8\n",
      "102\t6\t0.0\t1\t5 1\n",
      "103\t6\t0.0\t1\t5 1\n",
      "200\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8284 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t81\t2503.4\t0\t81\n",
      "4\t15\t625.8\t0\t15\n",
      "5\t24\t156.5\t0\t24\n",
      "6\t66\t39.1\t0\t66\n",
      "7\t458\t9.8\t0\t458\n",
      "8\t586\t2.4\t0\t586\n",
      "9\t2148\t0.6\t0\t2147 1\n",
      "10\t2564\t0.2\t1\t2556 8\n",
      "11\t88\t0.0\t1\t88\n",
      "12\t61\t0.0\t1\t61\n",
      "13\t70\t0.0\t1\t69 1\n",
      "14\t112\t0.0\t1\t109 3\n",
      "15\t155\t0.0\t1\t152 3\n",
      "16\t189\t0.0\t1\t185 4\n",
      "17\t350\t0.0\t1\t340 10\n",
      "18\t415\t0.0\t1\t388 17 10\n",
      "19\t773\t0.0\t1\t475 296 2\n",
      "20\t29\t0.0\t2\t0 0 29\n",
      "21\t97\t0.0\t2\t0 97\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/375.U_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/375.U_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/375.U_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/375.U_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.37 s (34 us/read; 1.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,847\n",
      "  Read 1 with adapter:                      19 (0.0%)\n",
      "  Read 2 with adapter:                   1,222 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        40,847 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,475,965 bp\n",
      "  Read 1:     9,077,162 bp\n",
      "  Read 2:     9,398,803 bp\n",
      "Total written (filtered):     18,459,217 bp (99.9%)\n",
      "  Read 1:     9,076,917 bp\n",
      "  Read 2:     9,382,300 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t638.2\t0\t17\n",
      "27\t1\t0.0\t1\t1\n",
      "167\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1222 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t638.2\t0\t12\n",
      "5\t2\t39.9\t0\t2\n",
      "6\t9\t10.0\t0\t9\n",
      "7\t38\t2.5\t0\t38\n",
      "8\t47\t0.6\t0\t47\n",
      "9\t180\t0.2\t0\t180\n",
      "10\t259\t0.0\t1\t258 1\n",
      "11\t22\t0.0\t1\t22\n",
      "12\t11\t0.0\t1\t11\n",
      "13\t32\t0.0\t1\t32\n",
      "14\t36\t0.0\t1\t36\n",
      "15\t50\t0.0\t1\t50\n",
      "16\t63\t0.0\t1\t61 2\n",
      "17\t106\t0.0\t1\t104 2\n",
      "18\t106\t0.0\t1\t96 8 2\n",
      "19\t218\t0.0\t1\t145 73\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t27\t0.0\t2\t0 27\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/377.H_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/377.H_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/377.H_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/377.H_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.11 s (38 us/read; 1.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,677\n",
      "  Read 1 with adapter:                      18 (0.1%)\n",
      "  Read 2 with adapter:                     826 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,677 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,358,672 bp\n",
      "  Read 1:     6,561,941 bp\n",
      "  Read 2:     6,796,731 bp\n",
      "Total written (filtered):     13,346,579 bp (99.9%)\n",
      "  Read 1:     6,561,248 bp\n",
      "  Read 2:     6,785,331 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 18 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t463.7\t0\t15\n",
      "216\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 826 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t463.7\t0\t3\n",
      "4\t1\t115.9\t0\t1\n",
      "6\t12\t7.2\t0\t12\n",
      "7\t16\t1.8\t0\t16\n",
      "8\t37\t0.5\t0\t37\n",
      "9\t128\t0.1\t0\t128\n",
      "10\t198\t0.0\t1\t195 3\n",
      "11\t18\t0.0\t1\t18\n",
      "12\t13\t0.0\t1\t13\n",
      "13\t13\t0.0\t1\t13\n",
      "14\t27\t0.0\t1\t26 1\n",
      "15\t27\t0.0\t1\t25 2\n",
      "16\t32\t0.0\t1\t30 2\n",
      "17\t61\t0.0\t1\t61\n",
      "18\t69\t0.0\t1\t67 0 2\n",
      "19\t143\t0.0\t1\t101 42\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t21\t0.0\t2\t0 21\n",
      "115\t4\t0.0\t2\t3 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/378.H_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/378.H_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/378.H_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/378.H_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.98 s (43 us/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,632\n",
      "  Read 1 with adapter:                      19 (0.1%)\n",
      "  Read 2 with adapter:                     259 (1.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,632 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,250,337 bp\n",
      "  Read 1:     5,040,043 bp\n",
      "  Read 2:     5,210,294 bp\n",
      "Total written (filtered):     10,245,868 bp (100.0%)\n",
      "  Read 1:     5,039,986 bp\n",
      "  Read 2:     5,205,882 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t353.6\t0\t19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 259 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "7\t1\t1.4\t0\t1\n",
      "8\t2\t0.3\t0\t2\n",
      "9\t8\t0.1\t0\t8\n",
      "10\t26\t0.0\t1\t26\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t10\t0.0\t1\t10\n",
      "15\t17\t0.0\t1\t17\n",
      "16\t17\t0.0\t1\t17\n",
      "17\t40\t0.0\t1\t39 1\n",
      "18\t36\t0.0\t1\t32 2 2\n",
      "19\t70\t0.0\t1\t48 22\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t16\t0.0\t2\t0 16\n",
      "207\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/379.H_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/379.H_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/379.H_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/379.H_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.78 s (25 us/read; 2.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             71,352\n",
      "  Read 1 with adapter:                      34 (0.0%)\n",
      "  Read 2 with adapter:                   1,815 (2.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        71,352 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,357,912 bp\n",
      "  Read 1:    15,907,813 bp\n",
      "  Read 2:    16,450,099 bp\n",
      "Total written (filtered):     32,332,222 bp (99.9%)\n",
      "  Read 1:    15,907,341 bp\n",
      "  Read 2:    16,424,881 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 34 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t1114.9\t0\t30\n",
      "21\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "118\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1815 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1114.9\t0\t9\n",
      "4\t3\t278.7\t0\t3\n",
      "5\t5\t69.7\t0\t5\n",
      "6\t6\t17.4\t0\t6\n",
      "7\t44\t4.4\t0\t44\n",
      "8\t60\t1.1\t0\t60\n",
      "9\t255\t0.3\t0\t255\n",
      "10\t422\t0.1\t1\t421 1\n",
      "11\t22\t0.0\t1\t22\n",
      "12\t34\t0.0\t1\t33 1\n",
      "13\t43\t0.0\t1\t43\n",
      "14\t56\t0.0\t1\t55 1\n",
      "15\t72\t0.0\t1\t69 3\n",
      "16\t78\t0.0\t1\t76 2\n",
      "17\t146\t0.0\t1\t140 6\n",
      "18\t147\t0.0\t1\t137 6 4\n",
      "19\t341\t0.0\t1\t211 127 3\n",
      "20\t17\t0.0\t2\t0 0 17\n",
      "21\t53\t0.0\t2\t0 53\n",
      "217\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/380.D_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/380.D_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/380.D_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/380.D_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.17 s (17 us/read; 3.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            184,224\n",
      "  Read 1 with adapter:                     128 (0.1%)\n",
      "  Read 2 with adapter:                   8,720 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       184,224 (100.0%)\n",
      "\n",
      "Total basepairs processed:    83,919,603 bp\n",
      "  Read 1:    41,222,722 bp\n",
      "  Read 2:    42,696,881 bp\n",
      "Total written (filtered):     83,813,909 bp (99.9%)\n",
      "  Read 1:    41,218,971 bp\n",
      "  Read 2:    42,594,938 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 128 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t2878.5\t0\t91\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t6\t0.0\t1\t6\n",
      "50\t1\t0.0\t1\t1\n",
      "94\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "101\t2\t0.0\t1\t2\n",
      "102\t17\t0.0\t1\t17\n",
      "103\t2\t0.0\t1\t2\n",
      "104\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8720 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t2878.5\t0\t49\n",
      "4\t9\t719.6\t0\t9\n",
      "5\t23\t179.9\t0\t23\n",
      "6\t51\t45.0\t0\t51\n",
      "7\t348\t11.2\t0\t348\n",
      "8\t452\t2.8\t0\t452\n",
      "9\t1930\t0.7\t0\t1927 3\n",
      "10\t3303\t0.2\t1\t3285 18\n",
      "11\t124\t0.0\t1\t121 3\n",
      "12\t81\t0.0\t1\t81\n",
      "13\t78\t0.0\t1\t78\n",
      "14\t71\t0.0\t1\t69 2\n",
      "15\t166\t0.0\t1\t162 4\n",
      "16\t182\t0.0\t1\t179 3\n",
      "17\t330\t0.0\t1\t314 16\n",
      "18\t467\t0.0\t1\t447 16 4\n",
      "19\t900\t0.0\t1\t602 297 1\n",
      "20\t28\t0.0\t2\t0 0 28\n",
      "21\t122\t0.0\t2\t0 121 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "210\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/380.U_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/380.U_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/380.U_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/380.U_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.33 s (32 us/read; 1.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,044\n",
      "  Read 1 with adapter:                      37 (0.1%)\n",
      "  Read 2 with adapter:                   1,656 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,044 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,463,081 bp\n",
      "  Read 1:     9,062,872 bp\n",
      "  Read 2:     9,400,209 bp\n",
      "Total written (filtered):     18,440,028 bp (99.9%)\n",
      "  Read 1:     9,061,065 bp\n",
      "  Read 2:     9,378,963 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 37 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t641.3\t0\t23\n",
      "25\t4\t0.0\t1\t4\n",
      "27\t2\t0.0\t1\t2\n",
      "100\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1656 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t641.3\t0\t8\n",
      "4\t1\t160.3\t0\t1\n",
      "5\t5\t40.1\t0\t5\n",
      "6\t13\t10.0\t0\t13\n",
      "7\t55\t2.5\t0\t55\n",
      "8\t100\t0.6\t0\t100\n",
      "9\t322\t0.2\t0\t321 1\n",
      "10\t413\t0.0\t1\t409 4\n",
      "11\t24\t0.0\t1\t24\n",
      "12\t24\t0.0\t1\t24\n",
      "13\t32\t0.0\t1\t31 1\n",
      "14\t46\t0.0\t1\t44 2\n",
      "15\t52\t0.0\t1\t51 1\n",
      "16\t74\t0.0\t1\t74\n",
      "17\t116\t0.0\t1\t109 7\n",
      "18\t115\t0.0\t1\t107 6 2\n",
      "19\t224\t0.0\t1\t147 77\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t25\t0.0\t2\t0 25\n",
      "92\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/381.D_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/381.D_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/381.D_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/381.D_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.26 s (21 us/read; 2.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            152,051\n",
      "  Read 1 with adapter:                     107 (0.1%)\n",
      "  Read 2 with adapter:                   6,094 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       152,051 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,177,789 bp\n",
      "  Read 1:    33,989,644 bp\n",
      "  Read 2:    35,188,145 bp\n",
      "Total written (filtered):     69,100,484 bp (99.9%)\n",
      "  Read 1:    33,987,347 bp\n",
      "  Read 2:    35,113,137 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 107 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t90\t2375.8\t0\t90\n",
      "25\t6\t0.0\t1\t6\n",
      "27\t1\t0.0\t1\t1\n",
      "92\t2\t0.0\t1\t2\n",
      "151\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t3 1\n",
      "223\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6094 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t2375.8\t0\t33\n",
      "4\t5\t593.9\t0\t5\n",
      "5\t20\t148.5\t0\t20\n",
      "6\t53\t37.1\t0\t53\n",
      "7\t299\t9.3\t0\t299\n",
      "8\t329\t2.3\t0\t329\n",
      "9\t1346\t0.6\t0\t1342 4\n",
      "10\t1719\t0.1\t1\t1714 5\n",
      "11\t80\t0.0\t1\t80\n",
      "12\t80\t0.0\t1\t78 2\n",
      "13\t91\t0.0\t1\t91\n",
      "14\t139\t0.0\t1\t137 2\n",
      "15\t159\t0.0\t1\t157 2\n",
      "16\t165\t0.0\t1\t158 7\n",
      "17\t350\t0.0\t1\t334 16\n",
      "18\t370\t0.0\t1\t351 14 5\n",
      "19\t737\t0.0\t1\t475 260 2\n",
      "20\t17\t0.0\t2\t0 0 17\n",
      "21\t93\t0.0\t2\t0 93\n",
      "53\t1\t0.0\t2\t0 1\n",
      "103\t1\t0.0\t2\t0 1\n",
      "112\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "216\t4\t0.0\t2\t3 0 1\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/371.H_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/371.H_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/371.H_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/371.H_3_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/372.D_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/372.D_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/372.D_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/372.D_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/372.U_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/372.U_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/372.U_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/372.U_8_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/373.D_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/373.D_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/373.D_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/373.D_20_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/373.U_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/373.U_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/373.U_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/373.U_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/374.D_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/374.D_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/374.D_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/374.D_17_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/374.U_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/374.U_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/374.U_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/374.U_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/375.D_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/375.D_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/375.D_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/375.D_14_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/375.U_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/375.U_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/375.U_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/375.U_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/377.H_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/377.H_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/377.H_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/377.H_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/378.H_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/378.H_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/378.H_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/378.H_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/379.H_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/379.H_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/379.H_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/379.H_35_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/380.D_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/380.D_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/380.D_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/380.D_25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/380.U_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/380.U_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/380.U_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/380.U_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/381.D_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/381.D_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/381.D_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/381.D_2_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/381.U_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/381.U_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/381.U_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/381.U_42_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/381.U_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/381.U_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/381.U_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/381.U_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.80 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,530\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                     188 (1.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,530 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,108,666 bp\n",
      "  Read 1:     2,508,707 bp\n",
      "  Read 2:     2,599,959 bp\n",
      "Total written (filtered):      5,105,667 bp (99.9%)\n",
      "  Read 1:     2,508,483 bp\n",
      "  Read 2:     2,597,184 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t180.2\t0\t3\n",
      "215\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 188 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t180.2\t0\t1\n",
      "7\t5\t0.7\t0\t5\n",
      "8\t5\t0.2\t0\t5\n",
      "9\t16\t0.0\t0\t16\n",
      "10\t33\t0.0\t1\t33\n",
      "11\t6\t0.0\t1\t6\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t5\t0.0\t1\t5\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t11\t0.0\t1\t11\n",
      "16\t11\t0.0\t1\t11\n",
      "17\t12\t0.0\t1\t12\n",
      "18\t22\t0.0\t1\t21 1\n",
      "19\t48\t0.0\t1\t33 15\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t7\t0.0\t2\t0 7\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/382.D_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/382.D_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/382.D_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/382.D_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.82 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,392\n",
      "  Read 1 with adapter:                      58 (0.1%)\n",
      "  Read 2 with adapter:                   3,494 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        74,392 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,820,524 bp\n",
      "  Read 1:    16,604,172 bp\n",
      "  Read 2:    17,216,352 bp\n",
      "Total written (filtered):     33,778,102 bp (99.9%)\n",
      "  Read 1:    16,602,940 bp\n",
      "  Read 2:    17,175,162 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 58 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t1162.4\t0\t48\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "103\t1\t0.0\t1\t1\n",
      "118\t2\t0.0\t1\t2\n",
      "212\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3494 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t1162.4\t0\t24\n",
      "4\t7\t290.6\t0\t7\n",
      "5\t12\t72.6\t0\t12\n",
      "6\t28\t18.2\t0\t28\n",
      "7\t159\t4.5\t0\t159\n",
      "8\t228\t1.1\t0\t228\n",
      "9\t865\t0.3\t0\t864 1\n",
      "10\t1010\t0.1\t1\t1006 4\n",
      "11\t43\t0.0\t1\t43\n",
      "12\t34\t0.0\t1\t34\n",
      "13\t31\t0.0\t1\t31\n",
      "14\t62\t0.0\t1\t61 1\n",
      "15\t81\t0.0\t1\t77 4\n",
      "16\t106\t0.0\t1\t99 7\n",
      "17\t191\t0.0\t1\t190 1\n",
      "18\t204\t0.0\t1\t195 9\n",
      "19\t356\t0.0\t1\t227 127 2\n",
      "20\t15\t0.0\t2\t0 1 14\n",
      "21\t36\t0.0\t2\t0 36\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/382.U_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/382.U_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/382.U_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/382.U_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.40 s (29 us/read; 2.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,460\n",
      "  Read 1 with adapter:                      36 (0.1%)\n",
      "  Read 2 with adapter:                     830 (1.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,460 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,885,608 bp\n",
      "  Read 1:    10,745,362 bp\n",
      "  Read 2:    11,140,246 bp\n",
      "Total written (filtered):     21,871,836 bp (99.9%)\n",
      "  Read 1:    10,744,594 bp\n",
      "  Read 2:    11,127,242 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 36 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t757.2\t0\t31\n",
      "27\t2\t0.0\t1\t2\n",
      "192\t1\t0.0\t1\t0 1\n",
      "213\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 830 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t757.2\t0\t3\n",
      "5\t2\t47.3\t0\t2\n",
      "6\t8\t11.8\t0\t8\n",
      "7\t16\t3.0\t0\t16\n",
      "8\t14\t0.7\t0\t14\n",
      "9\t84\t0.2\t0\t84\n",
      "10\t142\t0.0\t1\t142\n",
      "11\t23\t0.0\t1\t23\n",
      "12\t22\t0.0\t1\t22\n",
      "13\t19\t0.0\t1\t19\n",
      "14\t23\t0.0\t1\t20 3\n",
      "15\t41\t0.0\t1\t41\n",
      "16\t49\t0.0\t1\t48 1\n",
      "17\t78\t0.0\t1\t77 1\n",
      "18\t91\t0.0\t1\t87 3 1\n",
      "19\t186\t0.0\t1\t133 53\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t21\t0.0\t2\t0 20 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "206\t3\t0.0\t2\t2 1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/383.D_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/383.D_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/383.D_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/383.D_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.26 s (22 us/read; 2.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,649\n",
      "  Read 1 with adapter:                      58 (0.1%)\n",
      "  Read 2 with adapter:                   3,222 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,649 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,812,228 bp\n",
      "  Read 1:    22,521,050 bp\n",
      "  Read 2:    23,291,178 bp\n",
      "Total written (filtered):     45,771,584 bp (99.9%)\n",
      "  Read 1:    22,520,742 bp\n",
      "  Read 2:    23,250,842 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 58 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t1572.6\t0\t53\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "32\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3222 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t1572.6\t0\t17\n",
      "4\t5\t393.2\t0\t5\n",
      "5\t5\t98.3\t0\t5\n",
      "6\t23\t24.6\t0\t23\n",
      "7\t96\t6.1\t0\t96\n",
      "8\t124\t1.5\t0\t124\n",
      "9\t603\t0.4\t0\t601 2\n",
      "10\t1065\t0.1\t1\t1060 5\n",
      "11\t59\t0.0\t1\t59\n",
      "12\t38\t0.0\t1\t38\n",
      "13\t33\t0.0\t1\t33\n",
      "14\t39\t0.0\t1\t38 1\n",
      "15\t81\t0.0\t1\t81\n",
      "16\t111\t0.0\t1\t109 2\n",
      "17\t190\t0.0\t1\t186 4\n",
      "18\t207\t0.0\t1\t193 12 2\n",
      "19\t442\t0.0\t1\t325 115 2\n",
      "20\t15\t0.0\t2\t0 0 15\n",
      "21\t68\t0.0\t2\t0 67 1\n",
      "208\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/383.U_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/383.U_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/383.U_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/383.U_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.87 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,647\n",
      "  Read 1 with adapter:                      14 (0.1%)\n",
      "  Read 2 with adapter:                     442 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        14,647 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,431,432 bp\n",
      "  Read 1:     3,152,107 bp\n",
      "  Read 2:     3,279,325 bp\n",
      "Total written (filtered):      6,423,700 bp (99.9%)\n",
      "  Read 1:     3,150,328 bp\n",
      "  Read 2:     3,273,372 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t228.9\t0\t4\n",
      "27\t2\t0.0\t1\t2\n",
      "204\t1\t0.0\t1\t0 1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t5\t0.0\t1\t4 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 442 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t228.9\t0\t2\n",
      "5\t2\t14.3\t0\t2\n",
      "6\t2\t3.6\t0\t2\n",
      "7\t11\t0.9\t0\t11\n",
      "8\t25\t0.2\t0\t25\n",
      "9\t59\t0.1\t0\t59\n",
      "10\t123\t0.0\t1\t121 2\n",
      "11\t10\t0.0\t1\t9 1\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t11\t0.0\t1\t11\n",
      "14\t12\t0.0\t1\t12\n",
      "15\t19\t0.0\t1\t18 1\n",
      "16\t13\t0.0\t1\t13\n",
      "17\t34\t0.0\t1\t32 2\n",
      "18\t39\t0.0\t1\t39\n",
      "19\t61\t0.0\t1\t39 22\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t9\t0.0\t2\t0 9\n",
      "213\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/384.D_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/384.D_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/384.D_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/384.D_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.14 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,590\n",
      "  Read 1 with adapter:                      57 (0.1%)\n",
      "  Read 2 with adapter:                   3,817 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        94,590 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,895,380 bp\n",
      "  Read 1:    21,067,991 bp\n",
      "  Read 2:    21,827,389 bp\n",
      "Total written (filtered):     42,846,739 bp (99.9%)\n",
      "  Read 1:    21,066,640 bp\n",
      "  Read 2:    21,780,099 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 57 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t51\t1478.0\t0\t51\n",
      "120\t1\t0.0\t1\t1\n",
      "215\t2\t0.0\t1\t2\n",
      "216\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3817 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t1478.0\t0\t27\n",
      "4\t5\t369.5\t0\t5\n",
      "5\t9\t92.4\t0\t9\n",
      "6\t25\t23.1\t0\t25\n",
      "7\t163\t5.8\t0\t163\n",
      "8\t190\t1.4\t0\t190\n",
      "9\t790\t0.4\t0\t790\n",
      "10\t1107\t0.1\t1\t1101 6\n",
      "11\t55\t0.0\t1\t55\n",
      "12\t39\t0.0\t1\t39\n",
      "13\t52\t0.0\t1\t50 2\n",
      "14\t73\t0.0\t1\t72 1\n",
      "15\t108\t0.0\t1\t106 2\n",
      "16\t114\t0.0\t1\t111 3\n",
      "17\t226\t0.0\t1\t218 8\n",
      "18\t248\t0.0\t1\t234 9 5\n",
      "19\t526\t0.0\t1\t338 188\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t52\t0.0\t2\t0 52\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "208\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/384.U_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/384.U_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/384.U_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/384.U_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.42 s (27 us/read; 2.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,960\n",
      "  Read 1 with adapter:                      28 (0.1%)\n",
      "  Read 2 with adapter:                   1,790 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,960 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,949,156 bp\n",
      "  Read 1:    11,763,211 bp\n",
      "  Read 2:    12,185,945 bp\n",
      "Total written (filtered):     23,925,694 bp (99.9%)\n",
      "  Read 1:    11,762,728 bp\n",
      "  Read 2:    12,162,966 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 28 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t827.5\t0\t25\n",
      "27\t1\t0.0\t1\t1\n",
      "174\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1790 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t827.5\t0\t13\n",
      "4\t2\t206.9\t0\t2\n",
      "5\t3\t51.7\t0\t3\n",
      "6\t15\t12.9\t0\t15\n",
      "7\t62\t3.2\t0\t62\n",
      "8\t69\t0.8\t0\t69\n",
      "9\t345\t0.2\t0\t345\n",
      "10\t497\t0.1\t1\t495 2\n",
      "11\t22\t0.0\t1\t22\n",
      "12\t28\t0.0\t1\t28\n",
      "13\t31\t0.0\t1\t31\n",
      "14\t40\t0.0\t1\t39 1\n",
      "15\t46\t0.0\t1\t46\n",
      "16\t56\t0.0\t1\t55 1\n",
      "17\t133\t0.0\t1\t131 2\n",
      "18\t130\t0.0\t1\t125 5\n",
      "19\t257\t0.0\t1\t162 95\n",
      "20\t8\t0.0\t2\t0 0 8\n",
      "21\t30\t0.0\t2\t0 30\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/385.H_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/385.H_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/385.H_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/385.H_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.16 s (37 us/read; 1.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,078\n",
      "  Read 1 with adapter:                      24 (0.1%)\n",
      "  Read 2 with adapter:                   1,234 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        31,078 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,974,716 bp\n",
      "  Read 1:     6,861,212 bp\n",
      "  Read 2:     7,113,504 bp\n",
      "Total written (filtered):     13,957,832 bp (99.9%)\n",
      "  Read 1:     6,860,265 bp\n",
      "  Read 2:     7,097,567 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 24 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t485.6\t0\t16\n",
      "27\t2\t0.0\t1\t2\n",
      "84\t1\t0.0\t1\t1\n",
      "101\t2\t0.0\t1\t1 1\n",
      "127\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1234 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t485.6\t0\t11\n",
      "4\t2\t121.4\t0\t2\n",
      "6\t4\t7.6\t0\t4\n",
      "7\t34\t1.9\t0\t34\n",
      "8\t72\t0.5\t0\t72\n",
      "9\t213\t0.1\t0\t213\n",
      "10\t312\t0.0\t1\t311 1\n",
      "11\t21\t0.0\t1\t21\n",
      "12\t26\t0.0\t1\t26\n",
      "13\t24\t0.0\t1\t24\n",
      "14\t31\t0.0\t1\t31\n",
      "15\t47\t0.0\t1\t47\n",
      "16\t60\t0.0\t1\t57 3\n",
      "17\t93\t0.0\t1\t88 5\n",
      "18\t81\t0.0\t1\t78 1 2\n",
      "19\t169\t0.0\t1\t115 54\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t29\t0.0\t2\t0 28 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/386.H_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/386.H_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/386.H_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/386.H_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.26 s (36 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,417\n",
      "  Read 1 with adapter:                      18 (0.1%)\n",
      "  Read 2 with adapter:                   1,419 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,417 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,960,159 bp\n",
      "  Read 1:     7,834,636 bp\n",
      "  Read 2:     8,125,523 bp\n",
      "Total written (filtered):     15,941,019 bp (99.9%)\n",
      "  Read 1:     7,833,741 bp\n",
      "  Read 2:     8,107,278 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 18 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t553.4\t0\t13\n",
      "27\t1\t0.0\t1\t1\n",
      "190\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1419 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t553.4\t0\t13\n",
      "4\t3\t138.3\t0\t3\n",
      "5\t2\t34.6\t0\t2\n",
      "6\t11\t8.6\t0\t11\n",
      "7\t46\t2.2\t0\t46\n",
      "8\t68\t0.5\t0\t68\n",
      "9\t245\t0.1\t0\t245\n",
      "10\t353\t0.0\t1\t351 2\n",
      "11\t18\t0.0\t1\t18\n",
      "12\t26\t0.0\t1\t26\n",
      "13\t20\t0.0\t1\t20\n",
      "14\t46\t0.0\t1\t44 2\n",
      "15\t54\t0.0\t1\t54\n",
      "16\t57\t0.0\t1\t55 2\n",
      "17\t118\t0.0\t1\t116 2\n",
      "18\t112\t0.0\t1\t109 1 2\n",
      "19\t202\t0.0\t1\t116 86\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t21\t0.0\t2\t0 21\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/387.H_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/387.H_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/387.H_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/387.H_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.71 s (28 us/read; 2.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,147\n",
      "  Read 1 with adapter:                      48 (0.1%)\n",
      "  Read 2 with adapter:                   2,227 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        60,147 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,190,209 bp\n",
      "  Read 1:    13,353,843 bp\n",
      "  Read 2:    13,836,366 bp\n",
      "Total written (filtered):     27,160,864 bp (99.9%)\n",
      "  Read 1:    13,353,161 bp\n",
      "  Read 2:    13,807,703 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 48 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t939.8\t0\t39\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t6\t0.0\t1\t6\n",
      "162\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2227 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t939.8\t0\t15\n",
      "4\t2\t234.9\t0\t2\n",
      "5\t5\t58.7\t0\t5\n",
      "6\t16\t14.7\t0\t16\n",
      "7\t85\t3.7\t0\t85\n",
      "8\t118\t0.9\t0\t118\n",
      "9\t403\t0.2\t0\t403\n",
      "10\t572\t0.1\t1\t572\n",
      "11\t32\t0.0\t1\t32\n",
      "12\t39\t0.0\t1\t39\n",
      "13\t34\t0.0\t1\t34\n",
      "14\t51\t0.0\t1\t47 4\n",
      "15\t86\t0.0\t1\t85 1\n",
      "16\t101\t0.0\t1\t96 5\n",
      "17\t121\t0.0\t1\t116 5\n",
      "18\t188\t0.0\t1\t175 7 6\n",
      "19\t318\t0.0\t1\t203 115\n",
      "20\t15\t0.0\t2\t0 1 14\n",
      "21\t24\t0.0\t2\t0 24\n",
      "217\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/388.D_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/388.D_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/388.D_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/388.D_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.07 s (20 us/read; 3.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            208,570\n",
      "  Read 1 with adapter:                      80 (0.0%)\n",
      "  Read 2 with adapter:                   6,341 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       208,570 (100.0%)\n",
      "\n",
      "Total basepairs processed:    94,797,263 bp\n",
      "  Read 1:    46,648,437 bp\n",
      "  Read 2:    48,148,826 bp\n",
      "Total written (filtered):     94,715,176 bp (99.9%)\n",
      "  Read 1:    46,646,794 bp\n",
      "  Read 2:    48,068,382 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 80 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t3258.9\t0\t73\n",
      "141\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6341 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t3258.9\t0\t54\n",
      "4\t9\t814.7\t0\t9\n",
      "5\t24\t203.7\t0\t24\n",
      "6\t56\t50.9\t0\t56\n",
      "7\t205\t12.7\t0\t205\n",
      "8\t318\t3.2\t0\t318\n",
      "9\t1188\t0.8\t0\t1185 3\n",
      "10\t1811\t0.2\t1\t1805 6\n",
      "11\t108\t0.0\t1\t107 1\n",
      "12\t95\t0.0\t1\t93 2\n",
      "13\t89\t0.0\t1\t88 1\n",
      "14\t116\t0.0\t1\t112 4\n",
      "15\t185\t0.0\t1\t184 1\n",
      "16\t202\t0.0\t1\t196 6\n",
      "17\t362\t0.0\t1\t357 5\n",
      "18\t438\t0.0\t1\t408 23 7\n",
      "19\t920\t0.0\t1\t622 298\n",
      "20\t15\t0.0\t2\t0 0 15\n",
      "21\t135\t0.0\t2\t0 134 1\n",
      "22\t5\t0.0\t2\t0 2 3\n",
      "115\t1\t0.0\t2\t1\n",
      "129\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "239\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/388.U_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/388.U_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/388.U_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/388.U_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.98 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            212,486\n",
      "  Read 1 with adapter:                      43 (0.0%)\n",
      "  Read 2 with adapter:                   6,207 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       212,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    96,618,786 bp\n",
      "  Read 1:    47,549,856 bp\n",
      "  Read 2:    49,068,930 bp\n",
      "Total written (filtered):     96,538,373 bp (99.9%)\n",
      "  Read 1:    47,548,636 bp\n",
      "  Read 2:    48,989,737 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 43 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t3320.1\t0\t35\n",
      "27\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "63\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6207 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t3320.1\t0\t47\n",
      "4\t7\t830.0\t0\t7\n",
      "5\t20\t207.5\t0\t20\n",
      "6\t54\t51.9\t0\t54\n",
      "7\t232\t13.0\t0\t232\n",
      "8\t286\t3.2\t0\t286\n",
      "9\t1186\t0.8\t0\t1184 2\n",
      "10\t1750\t0.2\t1\t1743 7\n",
      "11\t81\t0.1\t1\t80 1\n",
      "12\t107\t0.0\t1\t106 1\n",
      "13\t89\t0.0\t1\t87 2\n",
      "14\t107\t0.0\t1\t106 1\n",
      "15\t182\t0.0\t1\t181 1\n",
      "16\t189\t0.0\t1\t183 6\n",
      "17\t338\t0.0\t1\t319 19\n",
      "18\t441\t0.0\t1\t424 14 3\n",
      "19\t911\t0.0\t1\t612 299\n",
      "20\t21\t0.0\t2\t0 0 21\n",
      "21\t153\t0.0\t2\t0 153\n",
      "214\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t0 0 1\n",
      "226\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/389.D_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/389.D_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/389.D_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/389.D_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.07 s (28 us/read; 2.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,583\n",
      "  Read 1 with adapter:                      28 (0.1%)\n",
      "  Read 2 with adapter:                   1,681 (4.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        38,583 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,276,969 bp\n",
      "  Read 1:     8,464,600 bp\n",
      "  Read 2:     8,812,369 bp\n",
      "Total written (filtered):     17,254,930 bp (99.9%)\n",
      "  Read 1:     8,462,948 bp\n",
      "  Read 2:     8,791,982 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 28 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t602.9\t0\t18\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "217\t7\t0.0\t1\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1681 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t602.9\t0\t19\n",
      "4\t1\t150.7\t0\t1\n",
      "5\t4\t37.7\t0\t4\n",
      "6\t15\t9.4\t0\t15\n",
      "7\t85\t2.4\t0\t85\n",
      "8\t130\t0.6\t0\t130\n",
      "9\t335\t0.1\t0\t335\n",
      "10\t472\t0.0\t1\t469 3\n",
      "11\t18\t0.0\t1\t18\n",
      "12\t11\t0.0\t1\t10 1\n",
      "13\t15\t0.0\t1\t15\n",
      "14\t18\t0.0\t1\t17 1\n",
      "15\t53\t0.0\t1\t52 1\n",
      "16\t55\t0.0\t1\t54 1\n",
      "17\t93\t0.0\t1\t88 5\n",
      "18\t116\t0.0\t1\t108 7 1\n",
      "19\t211\t0.0\t1\t138 73\n",
      "20\t8\t0.0\t2\t1 0 7\n",
      "21\t21\t0.0\t2\t0 20 1\n",
      "115\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/389.U_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/389.U_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/389.U_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/389.U_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.69 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,672\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                     490 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,097,310 bp\n",
      "  Read 1:     2,496,495 bp\n",
      "  Read 2:     2,600,815 bp\n",
      "Total written (filtered):      5,091,021 bp (99.9%)\n",
      "  Read 1:     2,496,269 bp\n",
      "  Read 2:     2,594,752 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t182.4\t0\t3\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 490 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t182.4\t0\t2\n",
      "4\t3\t45.6\t0\t3\n",
      "5\t1\t11.4\t0\t1\n",
      "6\t1\t2.8\t0\t1\n",
      "7\t19\t0.7\t0\t19\n",
      "8\t23\t0.2\t0\t23\n",
      "9\t94\t0.0\t0\t94\n",
      "10\t143\t0.0\t1\t142 1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t10\t0.0\t1\t10\n",
      "13\t11\t0.0\t1\t11\n",
      "14\t15\t0.0\t1\t15\n",
      "15\t18\t0.0\t1\t16 2\n",
      "16\t15\t0.0\t1\t14 1\n",
      "17\t30\t0.0\t1\t30\n",
      "18\t38\t0.0\t1\t34 3 1\n",
      "19\t61\t0.0\t1\t38 22 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t3\t0.0\t2\t0 3\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/390.D_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/390.D_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/390.D_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/390.D_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.88 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,296\n",
      "  Read 1 with adapter:                      29 (0.0%)\n",
      "  Read 2 with adapter:                   4,231 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       130,296 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,181,100 bp\n",
      "  Read 1:    29,118,831 bp\n",
      "  Read 2:    30,062,269 bp\n",
      "Total written (filtered):     59,126,813 bp (99.9%)\n",
      "  Read 1:    29,118,179 bp\n",
      "  Read 2:    30,008,634 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t2035.9\t0\t26\n",
      "139\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4231 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t2035.9\t0\t33\n",
      "4\t3\t509.0\t0\t3\n",
      "5\t14\t127.2\t0\t14\n",
      "6\t37\t31.8\t0\t37\n",
      "7\t138\t8.0\t0\t138\n",
      "8\t217\t2.0\t0\t217\n",
      "9\t846\t0.5\t0\t845 1\n",
      "10\t1201\t0.1\t1\t1197 4\n",
      "11\t57\t0.0\t1\t56 1\n",
      "12\t75\t0.0\t1\t73 2\n",
      "13\t53\t0.0\t1\t52 1\n",
      "14\t74\t0.0\t1\t74\n",
      "15\t109\t0.0\t1\t109\n",
      "16\t140\t0.0\t1\t133 7\n",
      "17\t244\t0.0\t1\t238 6\n",
      "18\t280\t0.0\t1\t264 12 4\n",
      "19\t622\t0.0\t1\t407 214 1\n",
      "20\t6\t0.0\t2\t0 1 5\n",
      "21\t77\t0.0\t2\t0 77\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "215\t2\t0.0\t2\t2\n",
      "216\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/390.U_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/390.U_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/390.U_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/390.U_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.92 s (20 us/read; 3.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            146,249\n",
      "  Read 1 with adapter:                      43 (0.0%)\n",
      "  Read 2 with adapter:                   4,982 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       146,249 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,424,235 bp\n",
      "  Read 1:    32,682,376 bp\n",
      "  Read 2:    33,741,859 bp\n",
      "Total written (filtered):     66,360,571 bp (99.9%)\n",
      "  Read 1:    32,681,448 bp\n",
      "  Read 2:    33,679,123 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 43 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t2285.1\t0\t36\n",
      "27\t3\t0.0\t1\t3\n",
      "89\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4982 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t2285.1\t0\t34\n",
      "4\t8\t571.3\t0\t8\n",
      "5\t27\t142.8\t0\t27\n",
      "6\t55\t35.7\t0\t55\n",
      "7\t201\t8.9\t0\t201\n",
      "8\t259\t2.2\t0\t259\n",
      "9\t1040\t0.6\t0\t1039 1\n",
      "10\t1388\t0.1\t1\t1386 2\n",
      "11\t59\t0.0\t1\t59\n",
      "12\t63\t0.0\t1\t63\n",
      "13\t60\t0.0\t1\t60\n",
      "14\t101\t0.0\t1\t97 4\n",
      "15\t133\t0.0\t1\t128 5\n",
      "16\t137\t0.0\t1\t135 2\n",
      "17\t271\t0.0\t1\t264 7\n",
      "18\t318\t0.0\t1\t288 19 11\n",
      "19\t720\t0.0\t1\t449 271\n",
      "20\t14\t0.0\t2\t0 1 13\n",
      "21\t87\t0.0\t2\t0 87\n",
      "216\t5\t0.0\t2\t5\n",
      "217\t2\t0.0\t2\t1 1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/382.D_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/382.D_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/382.D_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/382.D_16_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/382.U_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/382.U_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/382.U_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/382.U_9_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/383.D_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/383.D_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/383.D_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/383.D_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/383.U_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/383.U_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/383.U_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/383.U_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/384.D_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/384.D_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/384.D_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/384.D_33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/384.U_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/384.U_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/384.U_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/384.U_24_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/385.H_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/385.H_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/385.H_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/385.H_32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/386.H_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/386.H_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/386.H_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/386.H_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/387.H_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/387.H_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/387.H_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/387.H_6_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/388.D_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/388.D_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/388.D_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/388.D_12_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/388.U_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/388.U_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/388.U_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/388.U_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/389.D_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/389.D_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/389.D_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/389.D_29_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/389.U_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/389.U_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/389.U_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/389.U_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/390.D_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/390.D_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/390.D_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/390.D_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/390.U_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/390.U_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/390.U_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/390.U_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/391.U_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/391.U_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/391.U_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/391.U_26_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/391.U_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kc87guhf/391.U_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/391.U_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-3624litc/a366bae3-3809-46d6-97f9-48111b432d0d/data/391.U_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.43 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,487\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                   3,918 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,487 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,880,816 bp\n",
      "  Read 1:    25,529,481 bp\n",
      "  Read 2:    26,351,335 bp\n",
      "Total written (filtered):     51,831,537 bp (99.9%)\n",
      "  Read 1:    25,529,026 bp\n",
      "  Read 2:    26,302,511 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1788.9\t0\t7\n",
      "217\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3918 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t1788.9\t0\t31\n",
      "4\t3\t447.2\t0\t3\n",
      "5\t21\t111.8\t0\t21\n",
      "6\t34\t28.0\t0\t34\n",
      "7\t148\t7.0\t0\t148\n",
      "8\t216\t1.7\t0\t216\n",
      "9\t841\t0.4\t0\t841\n",
      "10\t1158\t0.1\t1\t1155 3\n",
      "11\t68\t0.0\t1\t67 1\n",
      "12\t28\t0.0\t1\t28\n",
      "13\t34\t0.0\t1\t34\n",
      "14\t56\t0.0\t1\t53 3\n",
      "15\t92\t0.0\t1\t89 3\n",
      "16\t111\t0.0\t1\t108 3\n",
      "17\t208\t0.0\t1\t202 6\n",
      "18\t241\t0.0\t1\t222 15 4\n",
      "19\t542\t0.0\t1\t362 180\n",
      "20\t8\t0.0\t2\t0 0 8\n",
      "21\t72\t0.0\t2\t0 71 1\n",
      "107\t1\t0.0\t2\t1\n",
      "216\t5\t0.0\t2\t3 2\n",
      "\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGmicro2018_97-144.qza\n"
     ]
    }
   ],
   "source": [
    "\n",
    "%%bash\n",
    "\n",
    "for f1 in demux_Muller_SWGmicro2018_*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "\n",
    "qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences $f1 \\\n",
    "--p-cores 16 \\\n",
    "--p-front-f GTGCCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACVSGGGTATCTAAT \\\n",
    "--o-trimmed-sequences trim_$newfile \\\n",
    "--verbose\n",
    "\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: trim_demux_Muller_SWGmicro2018_145-192.qza.qzv\n",
      "Saved Visualization to: trim_demux_Muller_SWGmicro2018_1-48.qza.qzv\n",
      "Saved Visualization to: trim_demux_Muller_SWGmicro2018_193-240.qza.qzv\n",
      "Saved Visualization to: trim_demux_Muller_SWGmicro2018_241-288.qza.qzv\n",
      "Saved Visualization to: trim_demux_Muller_SWGmicro2018_289-336.qza.qzv\n",
      "Saved Visualization to: trim_demux_Muller_SWGmicro2018_337-38.qza.qzv\n",
      "Saved Visualization to: trim_demux_Muller_SWGmicro2018_49-96.qza.qzv\n",
      "Saved Visualization to: trim_demux_Muller_SWGmicro2018_97-144.qza.qzv\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for f1 in trim_demux_Muller_SWGmicro2018*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "qiime demux summarize  --i-data $f1  --o-visualization $newfile.qzv\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "for f1 in Folder_SWGmicrobiome2018/trim_demux_Muller_SWGmicro2018_*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime dada2 denoise-paired --i-demultiplexed-seqs $f1 --p-trunc-len-f 220 --p-trunc-len-r  220 --p-n-threads 10 --o-representative-sequences Folder_SWGmicrobiome2018/rep_$fn.qza --o-table Folder_SWGmicrobiome2018/table_$fn.qza --o-denoising-stats Folder_SWGmicrobiome2018/dada_$fn.qza \n",
    "\n",
    "done\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_145-192.qza\n",
      "Saved FeatureData[Sequence] to: Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_145-192.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_SWGmicrobiome2018/dada_trim_demux_Muller_SWGmicro2018_145-192.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_1-48.qza\n",
      "Saved FeatureData[Sequence] to: Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_1-48.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_SWGmicrobiome2018/dada_trim_demux_Muller_SWGmicro2018_1-48.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_193-240.qza\n",
      "Saved FeatureData[Sequence] to: Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_193-240.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_SWGmicrobiome2018/dada_trim_demux_Muller_SWGmicro2018_193-240.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_241-288.qza\n",
      "Saved FeatureData[Sequence] to: Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_241-288.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_SWGmicrobiome2018/dada_trim_demux_Muller_SWGmicro2018_241-288.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_289-336.qza\n",
      "Saved FeatureData[Sequence] to: Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_289-336.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_SWGmicrobiome2018/dada_trim_demux_Muller_SWGmicro2018_289-336.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_337-38.qza\n",
      "Saved FeatureData[Sequence] to: Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_337-38.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_SWGmicrobiome2018/dada_trim_demux_Muller_SWGmicro2018_337-38.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_49-96.qza\n",
      "Saved FeatureData[Sequence] to: Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_49-96.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_SWGmicrobiome2018/dada_trim_demux_Muller_SWGmicro2018_49-96.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_97-144.qza\n",
      "Saved FeatureData[Sequence] to: Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_97-144.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_SWGmicrobiome2018/dada_trim_demux_Muller_SWGmicro2018_97-144.qza\n"
     ]
    }
   ],
   "source": [
    "\n",
    "%%bash\n",
    "\n",
    "for f1 in Folder_SWGmicrobiome2018/trim_demux_Muller_SWGmicro2018_*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime dada2 denoise-single --i-demultiplexed-seqs $f1 --p-max-ee 3 --p-trunc-len 130 --p-n-threads 10 --o-representative-sequences Folder_SWGmicrobiome2018/rep_$fn.qza --o-table Folder_SWGmicrobiome2018/table_$fn.qza --o-denoising-stats Folder_SWGmicrobiome2018/dada_$fn.qza \n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_SWGmicrobiome2018/dada_trim_demux_Muller_SWGmicro2018_97-144.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_97-144.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file Folder_SWGmicrobiome2018/dada_trim_demux_Muller_SWGmicro2018_97-144.qza \\\n",
    "  --o-visualization Folder_SWGmicrobiome2018/dada_trim_demux_Muller_SWGmicro2018_97-144.qzv\n",
    "    \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_97-144.qza \\\n",
    " --o-visualization Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_97-144.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table merge-seqs \\\n",
    "--i-data Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_1-48.qza \\\n",
    "--i-data Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_145-192.qza \\\n",
    "--i-data Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_193-240.qza \\\n",
    "--i-data Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_241-288.qza \\\n",
    "--i-data Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_289-336.qza \\\n",
    "--i-data Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_337-38.qza \\\n",
    "--i-data Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_49-96.qza \\\n",
    "--i-data Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018_97-144.qza \\\n",
    "--o-merged-data Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_SWGmicrobiome2018/table_Muller_SWGmicro2018.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table merge \\\n",
    " --i-tables Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_1-48.qza \\\n",
    " --i-tables Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_145-192.qza \\\n",
    " --i-tables Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_193-240.qza \\\n",
    " --i-tables Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_241-288.qza \\\n",
    " --i-tables Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_289-336.qza \\\n",
    " --i-tables Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_337-38.qza \\\n",
    " --i-tables Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_49-96.qza \\\n",
    " --i-tables Folder_SWGmicrobiome2018/table_trim_demux_Muller_SWGmicro2018_97-144.qza \\\n",
    " --o-merged-table Folder_SWGmicrobiome2018/table_Muller_SWGmicro2018.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_SWGmicrobiome2018/table_Muller_SWGmicro2018.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file Folder_SWGmicrobiome2018/table_Muller_SWGmicro2018.qza \\\n",
    "   --o-visualization Folder_SWGmicrobiome2018/table_Muller_SWGmicro2018.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Taxonomy] to: Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier /space/home/rosales/databases/SILVA_DBs/silva-138-99-nb-classifier.qza \\\n",
    "  --i-reads Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018.qza \\\n",
    "  --p-n-jobs -7 \\\n",
    "  --o-classification Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: Folder_SWGmicrobiome2018/taxa-bar-Muller_SWGmicrobiome2018.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qza \\\n",
    "  --o-visualization Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qzv\n",
    "\n",
    "\n",
    "!qiime taxa barplot \\\n",
    "  --i-table  Folder_SWGmicrobiome2018/table-BacArc_Muller_SWGmicrobiome2018.qza \\\n",
    "  --i-taxonomy  Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qza \\\n",
    "  --m-metadata-file  metadata/metadata_Mulller_SWG.txt \\\n",
    "  --o-visualization  Folder_SWGmicrobiome2018/taxa-bar-Muller_SWGmicrobiome2018.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qza \\\n",
    "  --o-visualization Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qzv\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_SWGmicrobiome2018/table-BacArc_Muller_SWGmicrobiome2018.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_SWGmicrobiome2018/rep_seqs_BacArc_Muller_SWGmicrobiome2018.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime taxa filter-table \\\n",
    "  --i-table Folder_SWGmicrobiome2018/table_Muller_SWGmicro2018.qza \\\n",
    "  --i-taxonomy  Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qza \\\n",
    "  --p-exclude mitochondria,chloroplast \\\n",
    "--o-filtered-table Folder_SWGmicrobiome2018/table-BacArc_Muller_SWGmicrobiome2018.qza\n",
    "\n",
    "!qiime taxa filter-seqs \\\n",
    "  --i-sequences  Folder_SWGmicrobiome2018/rep_trim_demux_Muller_SWGmicro2018.qza \\\n",
    "  --i-taxonomy   Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qza \\\n",
    "  --p-exclude mitochondria,chloroplast \\\n",
    "  --o-filtered-sequences Folder_SWGmicrobiome2018/rep_seqs_BacArc_Muller_SWGmicrobiome2018.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: Folder_SWGmicrobiome2018/taxa-bar-Muller_SWGmicrobiome.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file  Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qza \\\n",
    "  --o-visualization Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qzv\n",
    "\n",
    "!qiime taxa barplot \\\n",
    "  --i-table Folder_SWGmicrobiome2018/table-BacArc_Muller_SWGmicrobiome2018.qza \\\n",
    "  --i-taxonomy Folder_SWGmicrobiome2018/taxonomy_Muller_SWGmicrobiome2018.qza \\\n",
    "  --m-metadata-file  metadata/metadata_Mulller_SWG.txt \\\n",
    "  --o-visualization Folder_SWGmicrobiome2018/taxa-bar-Muller_SWGmicrobiome.qzv"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
