{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "env: STUDY=Apprill_transmissionTissue2019\n"
     ]
    }
   ],
   "source": [
    "%env STUDY Apprill_transmissionTissue2019\n",
    "dir_path='/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_2019TransmissionExpTissue/'\n",
    "wdir= \"/space/home/rosales/projects/2020_SCTLD_meta-analysis/\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_2019TransmissionExpTissue\n",
      "/space/home/rosales/projects/2020_SCTLD_meta-analysis\n"
     ]
    }
   ],
   "source": [
    "%cd $dir_path\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_$STUDY.txt\n",
    "%cd $wdir"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as ps\n",
    "import numpy as np\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_Apprill_transmissionTissue2019.txt',\n",
    "                                    names=header_list)\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_S\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/projects/2020_SCTLD_meta-analysis\r\n"
     ]
    }
   ],
   "source": [
    "!pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: cannot create directory ‘folder_Apprill_transmissionTissue2019’: File exists\r\n"
     ]
    }
   ],
   "source": [
    "!mkdir folder_Apprill_transmissionTissue2019"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/anaconda2/envs/qiime2-2020.2/bin/qiime\r\n"
     ]
    }
   ],
   "source": [
    "!which qiime"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "!grep \"sample-id\\|LibS*\" /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue.csv \\\n",
    "> /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibS.csv \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibS.csv as PairedEndFastqManifestPhred33V2 to folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibS.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!grep \"sample-id\\|LibS\" /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue.csv \\\n",
    "> /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibS.csv \n",
    "\n",
    "!qiime tools import \\\n",
    "--output-path folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibS.qza \\\n",
    "--type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "--input-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibS.csv \\\n",
    "--input-format PairedEndFastqManifestPhred33V2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibT.csv as PairedEndFastqManifestPhred33V2 to folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibT.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!grep \"sample-id\\|LibT\" /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue.csv \\\n",
    "> /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibT.csv \n",
    "\n",
    "!qiime tools import \\\n",
    "--output-path folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibT.qza \\\n",
    "--type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "--input-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibT.csv \\\n",
    "--input-format PairedEndFastqManifestPhred33V2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibU.csv as PairedEndFastqManifestPhred33V2 to folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibU.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!grep \"sample-id\\|LibU\" /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue.csv \\\n",
    "> /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibU.csv \n",
    "\n",
    "!qiime tools import \\\n",
    "--output-path folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibU.qza \\\n",
    "--type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "--input-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibU.csv \\\n",
    "--input-format PairedEndFastqManifestPhred33V2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibV.csv as PairedEndFastqManifestPhred33V2 to folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibV.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!grep \"sample-id\\|LibV\" /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue.csv \\\n",
    "> /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibV.csv \n",
    "\n",
    "!qiime tools import \\\n",
    "--output-path folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibV.qza \\\n",
    "--type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "--input-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibV.csv \\\n",
    "--input-format PairedEndFastqManifestPhred33V2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibY.csv as PairedEndFastqManifestPhred33V2 to folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibY.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!grep \"sample-id\\|LibY\" /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue.csv \\\n",
    "> /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibY.csv \n",
    "\n",
    "!qiime tools import \\\n",
    "--output-path folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibY.qza \\\n",
    "--type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "--input-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_2019TransmissionExpTissue_LibY.csv \\\n",
    "--input-format PairedEndFastqManifestPhred33V2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibS.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibT.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibU.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibV.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibY.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibS.qza \\\n",
    "--o-visualization folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibS.qzv\n",
    "\n",
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibT.qza \\\n",
    "--o-visualization folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibT.qzv\n",
    "\n",
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibU.qza \\\n",
    "--o-visualization folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibU.qzv\n",
    "\n",
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibV.qza \\\n",
    "--o-visualization folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibV.qzv\n",
    "\n",
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibY.qza \\\n",
    "--o-visualization folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibY.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1226A-TF-D-MC-T_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1226A-TF-D-MC-T_39_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1226A-TF-D-MC-T_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1226A-TF-D-MC-T_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1226A-TF-D-MC-T_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1226A-TF-D-MC-T_39_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1226A-TF-D-MC-T_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1226A-TF-D-MC-T_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       108,086 reads  @     22.8 µs/read;   2.64 M reads/minute\n",
      "Finished in 2.49 s (23 us/read; 2.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,086\n",
      "  Read 1 with adapter:                 101,746 (94.1%)\n",
      "  Read 2 with adapter:                 101,268 (93.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,086 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,559,124 bp\n",
      "  Read 1:    16,280,371 bp\n",
      "  Read 2:    16,278,753 bp\n",
      "Total written (filtered):     28,601,604 bp (87.8%)\n",
      "  Read 1:    14,347,389 bp\n",
      "  Read 2:    14,254,215 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101746 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t394\t0.0\t1\t47 347\n",
      "19\t101089\t0.0\t1\t92779 8310\n",
      "20\t252\t0.0\t1\t48 204\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 101268 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1688.8\t0\t7\n",
      "4\t15\t422.2\t0\t15\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t4\t0.0\t1\t3 1\n",
      "19\t663\t0.0\t1\t94 569\n",
      "20\t100339\t0.0\t2\t97825 2514\n",
      "21\t235\t0.0\t2\t52 183\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1301-TF-H-DL-T_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1301-TF-H-DL-T_40_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1301-TF-H-DL-T_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1301-TF-H-DL-T_40_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1301-TF-H-DL-T_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1301-TF-H-DL-T_40_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1301-TF-H-DL-T_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1301-TF-H-DL-T_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        95,955 reads  @     22.8 µs/read;   2.64 M reads/minute\n",
      "Finished in 2.21 s (23 us/read; 2.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             95,955\n",
      "  Read 1 with adapter:                  90,908 (94.7%)\n",
      "  Read 2 with adapter:                  90,497 (94.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        95,955 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,925,230 bp\n",
      "  Read 1:    14,470,669 bp\n",
      "  Read 2:    14,454,561 bp\n",
      "Total written (filtered):     25,389,016 bp (87.8%)\n",
      "  Read 1:    12,743,630 bp\n",
      "  Read 2:    12,645,386 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90908 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t0 3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t359\t0.0\t1\t32 327\n",
      "19\t90314\t0.0\t1\t83940 6374\n",
      "20\t217\t0.0\t1\t42 175\n",
      "34\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 90497 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1499.3\t0\t6\n",
      "4\t11\t374.8\t0\t11\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t8\t0.0\t1\t4 4\n",
      "19\t630\t0.0\t1\t90 540\n",
      "20\t89644\t0.0\t2\t87500 2144\n",
      "21\t192\t0.0\t2\t47 145\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1305A-TF-H-OA-T_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1305A-TF-H-OA-T_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1305A-TF-H-OA-T_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1305A-TF-H-OA-T_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1305A-TF-H-OA-T_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1305A-TF-H-OA-T_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1305A-TF-H-OA-T_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1305A-TF-H-OA-T_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        80,453 reads  @     26.3 µs/read;   2.28 M reads/minute\n",
      "Finished in 2.14 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,453\n",
      "  Read 1 with adapter:                  75,718 (94.1%)\n",
      "  Read 2 with adapter:                  75,408 (93.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        80,453 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,262,053 bp\n",
      "  Read 1:    12,140,354 bp\n",
      "  Read 2:    12,121,699 bp\n",
      "Total written (filtered):     21,316,169 bp (87.9%)\n",
      "  Read 1:    10,701,873 bp\n",
      "  Read 2:    10,614,296 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 75718 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t271\t0.0\t1\t27 244\n",
      "19\t75269\t0.0\t1\t68796 6473\n",
      "20\t166\t0.0\t1\t40 126\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 75408 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1257.1\t0\t6\n",
      "4\t12\t314.3\t0\t12\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t7\t0.0\t1\t7\n",
      "19\t592\t0.0\t1\t77 515\n",
      "20\t74628\t0.0\t2\t72772 1856\n",
      "21\t160\t0.0\t2\t27 133\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1313-TF-D-DL-T_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1313-TF-D-DL-T_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1313-TF-D-DL-T_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1313-TF-D-DL-T_42_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1313-TF-D-DL-T_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1313-TF-D-DL-T_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1313-TF-D-DL-T_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1313-TF-D-DL-T_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       133,733 reads  @     22.5 µs/read;   2.66 M reads/minute\n",
      "Finished in 3.04 s (23 us/read; 2.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            133,733\n",
      "  Read 1 with adapter:                 126,249 (94.4%)\n",
      "  Read 2 with adapter:                 125,666 (94.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       133,733 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,212,124 bp\n",
      "  Read 1:    20,113,319 bp\n",
      "  Read 2:    20,098,805 bp\n",
      "Total written (filtered):     35,301,654 bp (87.8%)\n",
      "  Read 1:    17,714,988 bp\n",
      "  Read 2:    17,586,666 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 126249 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.5\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t6\t0.0\t1\t4 2\n",
      "13\t5\t0.0\t1\t5\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t568\t0.0\t1\t57 511\n",
      "19\t125344\t0.0\t1\t115760 9584\n",
      "20\t309\t0.0\t1\t55 254\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 125666 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t2089.6\t0\t11\n",
      "4\t12\t522.4\t0\t12\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t3\t0.0\t1\t3\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t11\t0.0\t1\t9 2\n",
      "19\t989\t0.0\t1\t139 850\n",
      "20\t124324\t0.0\t2\t121252 3072\n",
      "21\t300\t0.0\t2\t70 230\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1323-TF-H-DL-T_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1323-TF-H-DL-T_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1323-TF-H-DL-T_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1323-TF-H-DL-T_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1323-TF-H-DL-T_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1323-TF-H-DL-T_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1323-TF-H-DL-T_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1323-TF-H-DL-T_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       152,743 reads  @     21.6 µs/read;   2.78 M reads/minute\n",
      "Finished in 3.33 s (22 us/read; 2.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            152,743\n",
      "  Read 1 with adapter:                 146,694 (96.0%)\n",
      "  Read 2 with adapter:                 146,062 (95.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       152,743 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,956,600 bp\n",
      "  Read 1:    22,992,610 bp\n",
      "  Read 2:    22,963,990 bp\n",
      "Total written (filtered):     40,250,137 bp (87.6%)\n",
      "  Read 1:    20,205,917 bp\n",
      "  Read 2:    20,044,220 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 146694 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t10\t0.0\t1\t6 4\n",
      "18\t807\t0.0\t1\t102 705\n",
      "19\t145463\t0.0\t1\t131555 13908\n",
      "20\t403\t0.0\t1\t70 333\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 146062 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t2386.6\t0\t12\n",
      "4\t18\t596.7\t0\t18\n",
      "12\t3\t0.0\t1\t1 2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t18\t0.0\t1\t9 9\n",
      "19\t1236\t0.0\t1\t201 1035\n",
      "20\t144407\t0.0\t2\t140852 3555\n",
      "21\t352\t0.0\t2\t61 291\n",
      "22\t4\t0.0\t2\t1 3\n",
      "26\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1338I-T1-H-OA-T_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1338I-T1-H-OA-T_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1338I-T1-H-OA-T_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1338I-T1-H-OA-T_44_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1338I-T1-H-OA-T_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1338I-T1-H-OA-T_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1338I-T1-H-OA-T_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1338I-T1-H-OA-T_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       110,506 reads  @     20.7 µs/read;   2.90 M reads/minute\n",
      "Finished in 2.32 s (21 us/read; 2.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,506\n",
      "  Read 1 with adapter:                 104,384 (94.5%)\n",
      "  Read 2 with adapter:                 103,759 (93.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       110,506 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,145,811 bp\n",
      "  Read 1:    16,573,191 bp\n",
      "  Read 2:    16,572,620 bp\n",
      "Total written (filtered):     29,088,375 bp (87.8%)\n",
      "  Read 1:    14,590,133 bp\n",
      "  Read 2:    14,498,242 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 104384 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t13\t0.0\t1\t4 9\n",
      "18\t469\t0.0\t1\t48 421\n",
      "19\t103630\t0.0\t1\t96074 7556\n",
      "20\t261\t0.0\t1\t47 214\n",
      "33\t1\t0.0\t1\t1\n",
      "43\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 103759 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1726.7\t0\t12\n",
      "4\t6\t431.7\t0\t6\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t5\t0.0\t1\t2 3\n",
      "19\t721\t0.0\t1\t101 620\n",
      "20\t102766\t0.0\t2\t100323 2443\n",
      "21\t245\t0.0\t2\t50 195\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1339C-TF-H-SS-T_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1339C-TF-H-SS-T_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1339C-TF-H-SS-T_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1339C-TF-H-SS-T_45_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1339C-TF-H-SS-T_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1339C-TF-H-SS-T_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1339C-TF-H-SS-T_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1339C-TF-H-SS-T_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        94,733 reads  @     23.9 µs/read;   2.52 M reads/minute\n",
      "Finished in 2.28 s (24 us/read; 2.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,733\n",
      "  Read 1 with adapter:                  90,545 (95.6%)\n",
      "  Read 2 with adapter:                  90,237 (95.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        94,733 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,497,081 bp\n",
      "  Read 1:    14,253,426 bp\n",
      "  Read 2:    14,243,655 bp\n",
      "Total written (filtered):     24,973,216 bp (87.6%)\n",
      "  Read 1:    12,533,349 bp\n",
      "  Read 2:    12,439,867 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90545 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t453\t0.0\t1\t51 402\n",
      "19\t89860\t0.0\t1\t79215 10645\n",
      "20\t220\t0.0\t1\t39 181\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 90237 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1480.2\t0\t5\n",
      "4\t18\t370.1\t0\t18\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t10\t0.0\t1\t8 2\n",
      "19\t725\t0.0\t1\t105 620\n",
      "20\t89289\t0.0\t2\t87039 2250\n",
      "21\t186\t0.0\t2\t45 141\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1345B-TF-D-CN-T_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1345B-TF-D-CN-T_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1345B-TF-D-CN-T_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1345B-TF-D-CN-T_46_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1345B-TF-D-CN-T_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1345B-TF-D-CN-T_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1345B-TF-D-CN-T_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1345B-TF-D-CN-T_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        93,518 reads  @     23.6 µs/read;   2.54 M reads/minute\n",
      "Finished in 2.23 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,518\n",
      "  Read 1 with adapter:                  87,799 (93.9%)\n",
      "  Read 2 with adapter:                  87,491 (93.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        93,518 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,188,606 bp\n",
      "  Read 1:    14,092,536 bp\n",
      "  Read 2:    14,096,070 bp\n",
      "Total written (filtered):     24,771,733 bp (87.9%)\n",
      "  Read 1:    12,424,598 bp\n",
      "  Read 2:    12,347,135 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 87799 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t7\t0.0\t1\t5 2\n",
      "18\t343\t0.0\t1\t35 308\n",
      "19\t87277\t0.0\t1\t78680 8597\n",
      "20\t162\t0.0\t1\t33 129\n",
      "21\t2\t0.0\t1\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 87491 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1461.2\t0\t12\n",
      "4\t11\t365.3\t0\t11\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t9\t0.0\t1\t7 2\n",
      "19\t610\t0.0\t1\t92 518\n",
      "20\t86681\t0.0\t2\t84552 2129\n",
      "21\t162\t0.0\t2\t32 130\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1350F-TF-H-SS-T_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1350F-TF-H-SS-T_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1350F-TF-H-SS-T_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1350F-TF-H-SS-T_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1350F-TF-H-SS-T_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1350F-TF-H-SS-T_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1350F-TF-H-SS-T_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1350F-TF-H-SS-T_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        76,787 reads  @     22.3 µs/read;   2.69 M reads/minute\n",
      "Finished in 1.74 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             76,787\n",
      "  Read 1 with adapter:                  72,295 (94.2%)\n",
      "  Read 2 with adapter:                  72,047 (93.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        76,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,842,764 bp\n",
      "  Read 1:    11,427,556 bp\n",
      "  Read 2:    11,415,208 bp\n",
      "Total written (filtered):     20,029,446 bp (87.7%)\n",
      "  Read 1:    10,054,224 bp\n",
      "  Read 2:     9,975,222 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 72295 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.3\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t438\t0.0\t1\t38 400\n",
      "19\t71642\t0.0\t1\t64298 7344\n",
      "20\t206\t0.0\t1\t38 168\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 72047 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1199.8\t0\t9\n",
      "4\t17\t299.9\t0\t17\n",
      "10\t3\t0.1\t1\t3\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t7\t0.0\t1\t4 3\n",
      "19\t631\t0.0\t1\t82 549\n",
      "20\t71175\t0.0\t2\t69209 1966\n",
      "21\t196\t0.0\t2\t35 161\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1364B-TF-D-OA-T_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1364B-TF-D-OA-T_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1364B-TF-D-OA-T_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1364B-TF-D-OA-T_48_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1364B-TF-D-OA-T_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1364B-TF-D-OA-T_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1364B-TF-D-OA-T_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1364B-TF-D-OA-T_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       108,177 reads  @     22.3 µs/read;   2.69 M reads/minute\n",
      "Finished in 2.43 s (22 us/read; 2.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,177\n",
      "  Read 1 with adapter:                 103,794 (95.9%)\n",
      "  Read 2 with adapter:                 103,327 (95.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,177 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,641,533 bp\n",
      "  Read 1:    16,322,073 bp\n",
      "  Read 2:    16,319,460 bp\n",
      "Total written (filtered):     28,604,140 bp (87.6%)\n",
      "  Read 1:    14,350,263 bp\n",
      "  Read 2:    14,253,877 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103794 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t8\t0.0\t1\t7 1\n",
      "18\t456\t0.0\t1\t55 401\n",
      "19\t103103\t0.0\t1\t94115 8988\n",
      "20\t222\t0.0\t1\t38 184\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 103327 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1690.3\t0\t9\n",
      "4\t12\t422.6\t0\t12\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t12\t0.0\t1\t6 6\n",
      "19\t796\t0.0\t1\t119 677\n",
      "20\t102258\t0.0\t2\t99507 2751\n",
      "21\t235\t0.0\t2\t41 194\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1364E-T1-H-OA-T_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1364E-T1-H-OA-T_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1364E-T1-H-OA-T_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1364E-T1-H-OA-T_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1364E-T1-H-OA-T_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1364E-T1-H-OA-T_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1364E-T1-H-OA-T_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1364E-T1-H-OA-T_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        74,511 reads  @     21.5 µs/read;   2.79 M reads/minute\n",
      "Finished in 1.63 s (22 us/read; 2.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,511\n",
      "  Read 1 with adapter:                  69,994 (93.9%)\n",
      "  Read 2 with adapter:                  69,765 (93.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        74,511 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,478,823 bp\n",
      "  Read 1:    11,239,421 bp\n",
      "  Read 2:    11,239,402 bp\n",
      "Total written (filtered):     19,754,543 bp (87.9%)\n",
      "  Read 1:     9,909,694 bp\n",
      "  Read 2:     9,844,849 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 69994 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t268\t0.0\t1\t27 241\n",
      "19\t69549\t0.0\t1\t61967 7582\n",
      "20\t166\t0.0\t1\t39 127\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 69765 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1164.2\t0\t11\n",
      "4\t8\t291.1\t0\t8\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t2\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t6\t0.0\t1\t4 2\n",
      "19\t517\t0.0\t1\t82 435\n",
      "20\t69071\t0.0\t2\t67506 1565\n",
      "21\t145\t0.0\t2\t37 108\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1365-TF-D-DL-T_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1365-TF-D-DL-T_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1365-TF-D-DL-T_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1365-TF-D-DL-T_50_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1365-TF-D-DL-T_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1365-TF-D-DL-T_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1365-TF-D-DL-T_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1365-TF-D-DL-T_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       104,986 reads  @     20.5 µs/read;   2.92 M reads/minute\n",
      "Finished in 2.18 s (21 us/read; 2.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,986\n",
      "  Read 1 with adapter:                  98,929 (94.2%)\n",
      "  Read 2 with adapter:                  98,345 (93.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       104,986 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,666,355 bp\n",
      "  Read 1:    15,831,629 bp\n",
      "  Read 2:    15,834,726 bp\n",
      "Total written (filtered):     27,821,306 bp (87.9%)\n",
      "  Read 1:    13,952,276 bp\n",
      "  Read 2:    13,869,030 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 98929 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t534\t0.0\t1\t56 478\n",
      "19\t98094\t0.0\t1\t89777 8317\n",
      "20\t289\t0.0\t1\t45 244\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 98345 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1640.4\t0\t16\n",
      "4\t13\t410.1\t0\t13\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t5\t0.0\t1\t0 5\n",
      "19\t948\t0.0\t1\t121 827\n",
      "20\t97078\t0.0\t2\t94383 2695\n",
      "21\t277\t0.0\t2\t60 217\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1369-T1-H-DL-T_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1369-T1-H-DL-T_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1369-T1-H-DL-T_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1369-T1-H-DL-T_51_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1369-T1-H-DL-T_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1369-T1-H-DL-T_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1369-T1-H-DL-T_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1369-T1-H-DL-T_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       173,152 reads  @     22.1 µs/read;   2.72 M reads/minute\n",
      "Finished in 3.85 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            173,152\n",
      "  Read 1 with adapter:                 163,576 (94.5%)\n",
      "  Read 2 with adapter:                 162,725 (94.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       173,152 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,019,854 bp\n",
      "  Read 1:    26,030,813 bp\n",
      "  Read 2:    25,989,041 bp\n",
      "Total written (filtered):     45,659,245 bp (87.8%)\n",
      "  Read 1:    22,923,282 bp\n",
      "  Read 2:    22,735,963 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 163576 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.7\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t10\t0.0\t1\t4 6\n",
      "18\t785\t0.0\t1\t82 703\n",
      "19\t162297\t0.0\t1\t149784 12513\n",
      "20\t469\t0.0\t1\t83 386\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 162725 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t2705.5\t0\t12\n",
      "4\t13\t676.4\t0\t13\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t28\t0.0\t1\t18 10\n",
      "19\t1315\t0.0\t1\t212 1103\n",
      "20\t160954\t0.0\t2\t156826 4128\n",
      "21\t394\t0.0\t2\t77 317\n",
      "22\t2\t0.0\t2\t0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1370B-TF-H-CN-T_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1370B-TF-H-CN-T_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1370B-TF-H-CN-T_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1370B-TF-H-CN-T_52_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1370B-TF-H-CN-T_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1370B-TF-H-CN-T_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1370B-TF-H-CN-T_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1370B-TF-H-CN-T_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       107,170 reads  @     24.9 µs/read;   2.41 M reads/minute\n",
      "Finished in 2.69 s (25 us/read; 2.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,170\n",
      "  Read 1 with adapter:                 100,789 (94.0%)\n",
      "  Read 2 with adapter:                 100,298 (93.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       107,170 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,288,802 bp\n",
      "  Read 1:    16,161,905 bp\n",
      "  Read 2:    16,126,897 bp\n",
      "Total written (filtered):     28,369,042 bp (87.9%)\n",
      "  Read 1:    14,247,139 bp\n",
      "  Read 2:    14,121,903 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 100789 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t408\t0.0\t1\t29 379\n",
      "19\t100145\t0.0\t1\t90340 9805\n",
      "20\t223\t0.0\t1\t48 175\n",
      "25\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 100298 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t1674.5\t0\t14\n",
      "4\t13\t418.6\t0\t13\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t11\t0.0\t1\t6 5\n",
      "19\t704\t0.0\t1\t96 608\n",
      "20\t99328\t0.0\t2\t96853 2475\n",
      "21\t224\t0.0\t2\t39 185\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1379C-TF-D-MC-T_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1379C-TF-D-MC-T_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1379C-TF-D-MC-T_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1379C-TF-D-MC-T_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1379C-TF-D-MC-T_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1379C-TF-D-MC-T_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1379C-TF-D-MC-T_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1379C-TF-D-MC-T_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       103,562 reads  @     24.2 µs/read;   2.48 M reads/minute\n",
      "Finished in 2.53 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,562\n",
      "  Read 1 with adapter:                  97,921 (94.6%)\n",
      "  Read 2 with adapter:                  97,494 (94.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       103,562 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,209,471 bp\n",
      "  Read 1:    15,612,921 bp\n",
      "  Read 2:    15,596,550 bp\n",
      "Total written (filtered):     27,400,036 bp (87.8%)\n",
      "  Read 1:    13,752,591 bp\n",
      "  Read 2:    13,647,445 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 97921 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t426\t0.0\t1\t42 384\n",
      "19\t97162\t0.0\t1\t90361 6801\n",
      "20\t322\t0.0\t1\t70 252\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 97494 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1618.2\t0\t9\n",
      "4\t8\t404.5\t0\t8\n",
      "11\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t10\t0.0\t1\t5 5\n",
      "19\t694\t0.0\t1\t113 581\n",
      "20\t96519\t0.0\t2\t94188 2331\n",
      "21\t247\t0.0\t2\t46 201\n",
      "22\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1610E-TF-H-OA-T_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1610E-TF-H-OA-T_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1610E-TF-H-OA-T_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1610E-TF-H-OA-T_54_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1610E-TF-H-OA-T_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1610E-TF-H-OA-T_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1610E-TF-H-OA-T_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1610E-TF-H-OA-T_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       125,701 reads  @     18.5 µs/read;   3.24 M reads/minute\n",
      "Finished in 2.36 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,701\n",
      "  Read 1 with adapter:                 118,958 (94.6%)\n",
      "  Read 2 with adapter:                 118,269 (94.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       125,701 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,835,763 bp\n",
      "  Read 1:    18,948,705 bp\n",
      "  Read 2:    18,887,058 bp\n",
      "Total written (filtered):     33,211,512 bp (87.8%)\n",
      "  Read 1:    16,688,828 bp\n",
      "  Read 2:    16,522,684 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 118958 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t4\t0.0\t1\t2 2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t563\t0.0\t1\t50 513\n",
      "19\t118086\t0.0\t1\t109789 8297\n",
      "20\t283\t0.0\t1\t51 232\n",
      "21\t1\t0.0\t1\t0 1\n",
      "39\t1\t0.0\t1\t1\n",
      "60\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 118269 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1964.1\t0\t8\n",
      "4\t11\t491.0\t0\t11\n",
      "12\t3\t0.0\t1\t3\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t13\t0.0\t1\t2 11\n",
      "19\t937\t0.0\t1\t126 811\n",
      "20\t116999\t0.0\t2\t114132 2867\n",
      "21\t293\t0.0\t2\t64 229\n",
      "24\t1\t0.0\t2\t1\n",
      "28\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1613A-TF-D-SS-T_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1613A-TF-D-SS-T_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1613A-TF-D-SS-T_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1613A-TF-D-SS-T_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1613A-TF-D-SS-T_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1613A-TF-D-SS-T_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1613A-TF-D-SS-T_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1613A-TF-D-SS-T_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       124,324 reads  @     23.4 µs/read;   2.56 M reads/minute\n",
      "Finished in 2.93 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            124,324\n",
      "  Read 1 with adapter:                 116,859 (94.0%)\n",
      "  Read 2 with adapter:                 116,261 (93.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       124,324 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,419,344 bp\n",
      "  Read 1:    18,722,291 bp\n",
      "  Read 2:    18,697,053 bp\n",
      "Total written (filtered):     32,875,449 bp (87.9%)\n",
      "  Read 1:    16,502,348 bp\n",
      "  Read 2:    16,373,101 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 116859 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t0 2\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t576\t0.0\t1\t61 515\n",
      "19\t115970\t0.0\t1\t104869 11101\n",
      "20\t292\t0.0\t1\t60 232\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 116261 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1942.6\t0\t16\n",
      "4\t15\t485.6\t0\t15\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t6\t0.0\t1\t0 6\n",
      "19\t966\t0.0\t1\t152 814\n",
      "20\t114971\t0.0\t2\t111759 3212\n",
      "21\t277\t0.0\t2\t77 200\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1633B-TF-H-PA-T_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1633B-TF-H-PA-T_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1633B-TF-H-PA-T_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1633B-TF-H-PA-T_56_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1633B-TF-H-PA-T_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1633B-TF-H-PA-T_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1633B-TF-H-PA-T_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1633B-TF-H-PA-T_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        95,357 reads  @     23.6 µs/read;   2.54 M reads/minute\n",
      "Finished in 2.27 s (24 us/read; 2.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             95,357\n",
      "  Read 1 with adapter:                  89,820 (94.2%)\n",
      "  Read 2 with adapter:                  89,439 (93.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        95,357 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,756,409 bp\n",
      "  Read 1:    14,377,911 bp\n",
      "  Read 2:    14,378,498 bp\n",
      "Total written (filtered):     25,262,352 bp (87.8%)\n",
      "  Read 1:    12,671,633 bp\n",
      "  Read 2:    12,590,719 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 89820 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t7\t0.0\t1\t1 6\n",
      "18\t489\t0.0\t1\t45 444\n",
      "19\t89028\t0.0\t1\t80437 8591\n",
      "20\t281\t0.0\t1\t43 238\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 89439 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1490.0\t0\t3\n",
      "4\t17\t372.5\t0\t17\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t4\t0.0\t1\t4\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t822\t0.0\t1\t98 724\n",
      "20\t88342\t0.0\t2\t86043 2299\n",
      "21\t233\t0.0\t2\t45 188\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1633D-T1-H-PA-T_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1633D-T1-H-PA-T_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1633D-T1-H-PA-T_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1633D-T1-H-PA-T_57_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1633D-T1-H-PA-T_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1633D-T1-H-PA-T_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1633D-T1-H-PA-T_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1633D-T1-H-PA-T_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        84,105 reads  @     21.4 µs/read;   2.81 M reads/minute\n",
      "Finished in 1.82 s (22 us/read; 2.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,105\n",
      "  Read 1 with adapter:                  79,237 (94.2%)\n",
      "  Read 2 with adapter:                  79,668 (94.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        84,105 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,371,294 bp\n",
      "  Read 1:    12,685,629 bp\n",
      "  Read 2:    12,685,665 bp\n",
      "Total written (filtered):     22,273,662 bp (87.8%)\n",
      "  Read 1:    11,180,298 bp\n",
      "  Read 2:    11,093,364 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 79237 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t343\t0.0\t1\t27 316\n",
      "19\t78659\t0.0\t1\t61257 17402\n",
      "20\t223\t0.0\t1\t42 181\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 79668 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t1314.1\t0\t13\n",
      "4\t16\t328.5\t0\t16\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t10\t0.0\t1\t8 2\n",
      "19\t769\t0.0\t1\t89 680\n",
      "20\t78636\t0.0\t2\t76843 1793\n",
      "21\t214\t0.0\t2\t40 174\n",
      "22\t1\t0.0\t2\t1\n",
      "24\t1\t0.0\t2\t1\n",
      "37\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1654B-TF-H-SS-T_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1654B-TF-H-SS-T_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1654B-TF-H-SS-T_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1654B-TF-H-SS-T_58_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1654B-TF-H-SS-T_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1654B-TF-H-SS-T_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1654B-TF-H-SS-T_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1654B-TF-H-SS-T_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        89,664 reads  @     21.8 µs/read;   2.76 M reads/minute\n",
      "Finished in 1.98 s (22 us/read; 2.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             89,664\n",
      "  Read 1 with adapter:                  84,255 (94.0%)\n",
      "  Read 2 with adapter:                  83,753 (93.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        89,664 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,997,224 bp\n",
      "  Read 1:    13,502,752 bp\n",
      "  Read 2:    13,494,472 bp\n",
      "Total written (filtered):     23,722,627 bp (87.9%)\n",
      "  Read 1:    11,902,207 bp\n",
      "  Read 2:    11,820,420 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 84255 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t6\t0.0\t1\t3 3\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t0 4\n",
      "18\t458\t0.0\t1\t58 400\n",
      "19\t83562\t0.0\t1\t74216 9346\n",
      "20\t221\t0.0\t1\t54 167\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 83753 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1401.0\t0\t9\n",
      "4\t15\t350.2\t0\t15\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t767\t0.0\t1\t96 671\n",
      "20\t82738\t0.0\t2\t80515 2223\n",
      "21\t207\t0.0\t2\t41 166\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1654C-T1-H-SS-T_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1654C-T1-H-SS-T_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1654C-T1-H-SS-T_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1654C-T1-H-SS-T_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1654C-T1-H-SS-T_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1654C-T1-H-SS-T_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1654C-T1-H-SS-T_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1654C-T1-H-SS-T_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        83,152 reads  @     23.1 µs/read;   2.60 M reads/minute\n",
      "Finished in 1.95 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             83,152\n",
      "  Read 1 with adapter:                  79,088 (95.1%)\n",
      "  Read 2 with adapter:                  78,898 (94.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        83,152 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,041,945 bp\n",
      "  Read 1:    12,528,830 bp\n",
      "  Read 2:    12,513,115 bp\n",
      "Total written (filtered):     21,962,522 bp (87.7%)\n",
      "  Read 1:    11,026,432 bp\n",
      "  Read 2:    10,936,090 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 79088 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.3\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t4\t0.0\t1\t3 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t7\t0.0\t1\t2 5\n",
      "18\t399\t0.0\t1\t35 364\n",
      "19\t78471\t0.0\t1\t66907 11564\n",
      "20\t201\t0.0\t1\t28 173\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 78898 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1299.2\t0\t9\n",
      "4\t11\t324.8\t0\t11\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t3 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t5\t0.0\t1\t2 3\n",
      "19\t764\t0.0\t1\t109 655\n",
      "20\t77893\t0.0\t2\t76121 1772\n",
      "21\t207\t0.0\t2\t38 169\n",
      "23\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659B-T1-H-SS-T_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659B-T1-H-SS-T_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659B-T1-H-SS-T_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659B-T1-H-SS-T_60_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659B-T1-H-SS-T_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659B-T1-H-SS-T_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659B-T1-H-SS-T_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659B-T1-H-SS-T_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       142,423 reads  @     20.0 µs/read;   3.00 M reads/minute\n",
      "Finished in 2.87 s (20 us/read; 2.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,423\n",
      "  Read 1 with adapter:                 133,900 (94.0%)\n",
      "  Read 2 with adapter:                 133,232 (93.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       142,423 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,743,423 bp\n",
      "  Read 1:    21,435,252 bp\n",
      "  Read 2:    21,308,171 bp\n",
      "Total written (filtered):     37,536,647 bp (87.8%)\n",
      "  Read 1:    18,891,724 bp\n",
      "  Read 2:    18,644,923 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 133900 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t4\t0.5\t0\t4\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t5\t0.0\t1\t4 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t11\t0.0\t1\t3 8\n",
      "18\t794\t0.0\t1\t81 713\n",
      "19\t132736\t0.0\t1\t121198 11538\n",
      "20\t340\t0.0\t1\t64 276\n",
      "31\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 133232 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t2225.4\t0\t15\n",
      "4\t17\t556.3\t0\t17\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t6\t0.0\t1\t3 3\n",
      "19\t1121\t0.0\t1\t151 970\n",
      "20\t131738\t0.0\t2\t128396 3342\n",
      "21\t322\t0.0\t2\t70 252\n",
      "26\t2\t0.0\t2\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659DTF-H-SS-T_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659DTF-H-SS-T_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659DTF-H-SS-T_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659DTF-H-SS-T_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659DTF-H-SS-T_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659DTF-H-SS-T_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659DTF-H-SS-T_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659DTF-H-SS-T_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       100,016 reads  @     24.5 µs/read;   2.45 M reads/minute\n",
      "Finished in 2.48 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,016\n",
      "  Read 1 with adapter:                  93,690 (93.7%)\n",
      "  Read 2 with adapter:                  93,429 (93.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,016 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,953,654 bp\n",
      "  Read 1:    15,038,019 bp\n",
      "  Read 2:    14,915,635 bp\n",
      "Total written (filtered):     26,306,117 bp (87.8%)\n",
      "  Read 1:    13,258,277 bp\n",
      "  Read 2:    13,047,840 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 93690 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t8\t0.0\t1\t4 4\n",
      "18\t589\t0.0\t1\t34 555\n",
      "19\t92821\t0.0\t1\t82993 9828\n",
      "20\t267\t0.0\t1\t70 197\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 93429 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1562.8\t0\t2\n",
      "4\t6\t390.7\t0\t6\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t9\t0.0\t1\t5 4\n",
      "19\t821\t0.0\t1\t99 722\n",
      "20\t92351\t0.0\t2\t89955 2396\n",
      "21\t230\t0.0\t2\t55 175\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659E-TF-D-SS-T_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659E-TF-D-SS-T_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659E-TF-D-SS-T_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659E-TF-D-SS-T_62_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659E-TF-D-SS-T_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1659E-TF-D-SS-T_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659E-TF-D-SS-T_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1659E-TF-D-SS-T_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       122,414 reads  @     23.2 µs/read;   2.59 M reads/minute\n",
      "Finished in 2.87 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            122,414\n",
      "  Read 1 with adapter:                 115,284 (94.2%)\n",
      "  Read 2 with adapter:                 114,713 (93.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       122,414 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,854,772 bp\n",
      "  Read 1:    18,473,490 bp\n",
      "  Read 2:    18,381,282 bp\n",
      "Total written (filtered):     32,371,319 bp (87.8%)\n",
      "  Read 1:    16,283,430 bp\n",
      "  Read 2:    16,087,889 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 115284 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t517\t0.0\t1\t54 463\n",
      "19\t114486\t0.0\t1\t105346 9140\n",
      "20\t261\t0.0\t1\t58 203\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 114713 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1912.7\t0\t11\n",
      "4\t9\t478.2\t0\t9\n",
      "12\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t11\t0.0\t1\t7 4\n",
      "19\t741\t0.0\t1\t108 633\n",
      "20\t113708\t0.0\t2\t111021 2687\n",
      "21\t230\t0.0\t2\t56 174\n",
      "28\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1666C-TF-D-PA-T_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1666C-TF-D-PA-T_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1666C-TF-D-PA-T_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1666C-TF-D-PA-T_63_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1666C-TF-D-PA-T_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1666C-TF-D-PA-T_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1666C-TF-D-PA-T_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1666C-TF-D-PA-T_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        89,580 reads  @     23.8 µs/read;   2.52 M reads/minute\n",
      "Finished in 2.15 s (24 us/read; 2.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             89,580\n",
      "  Read 1 with adapter:                  84,075 (93.9%)\n",
      "  Read 2 with adapter:                  83,811 (93.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        89,580 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,025,269 bp\n",
      "  Read 1:    13,512,485 bp\n",
      "  Read 2:    13,512,784 bp\n",
      "Total written (filtered):     23,752,771 bp (87.9%)\n",
      "  Read 1:    11,915,340 bp\n",
      "  Read 2:    11,837,431 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 84075 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t5\t0.0\t1\t0 5\n",
      "18\t427\t0.0\t1\t51 376\n",
      "19\t83431\t0.0\t1\t74083 9348\n",
      "20\t202\t0.0\t1\t28 174\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 83811 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1399.7\t0\t8\n",
      "4\t11\t349.9\t0\t11\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t794\t0.0\t1\t109 685\n",
      "20\t82752\t0.0\t2\t80740 2012\n",
      "21\t233\t0.0\t2\t48 185\n",
      "22\t2\t0.0\t2\t0 2\n",
      "53\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1666E-T1-H-PA-T_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1666E-T1-H-PA-T_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1666E-T1-H-PA-T_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1666E-T1-H-PA-T_64_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1666E-T1-H-PA-T_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1666E-T1-H-PA-T_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1666E-T1-H-PA-T_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1666E-T1-H-PA-T_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       161,324 reads  @     20.2 µs/read;   2.97 M reads/minute\n",
      "Finished in 3.27 s (20 us/read; 2.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,324\n",
      "  Read 1 with adapter:                 151,395 (93.8%)\n",
      "  Read 2 with adapter:                 150,780 (93.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       161,324 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,679,775 bp\n",
      "  Read 1:    24,341,082 bp\n",
      "  Read 2:    24,338,693 bp\n",
      "Total written (filtered):     42,789,903 bp (87.9%)\n",
      "  Read 1:    21,465,075 bp\n",
      "  Read 2:    21,324,828 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 151395 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t8\t0.0\t1\t7 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t7\t0.0\t1\t5 2\n",
      "17\t10\t0.0\t1\t1 9\n",
      "18\t744\t0.0\t1\t69 675\n",
      "19\t150238\t0.0\t1\t134392 15846\n",
      "20\t383\t0.0\t1\t81 302\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 150780 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t2520.7\t0\t15\n",
      "4\t19\t630.2\t0\t19\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t7\t0.0\t1\t3 4\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t17\t0.0\t1\t6 11\n",
      "19\t1435\t0.0\t1\t163 1272\n",
      "20\t148913\t0.0\t2\t144668 4245\n",
      "21\t365\t0.0\t2\t69 296\n",
      "24\t1\t0.0\t2\t1\n",
      "30\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1692A-TF-D-CN-T_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1692A-TF-D-CN-T_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1692A-TF-D-CN-T_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1692A-TF-D-CN-T_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1692A-TF-D-CN-T_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1692A-TF-D-CN-T_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1692A-TF-D-CN-T_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1692A-TF-D-CN-T_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       109,600 reads  @     23.1 µs/read;   2.59 M reads/minute\n",
      "Finished in 2.56 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,600\n",
      "  Read 1 with adapter:                 102,833 (93.8%)\n",
      "  Read 2 with adapter:                 102,496 (93.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,600 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,079,625 bp\n",
      "  Read 1:    16,537,482 bp\n",
      "  Read 2:    16,542,143 bp\n",
      "Total written (filtered):     29,077,069 bp (87.9%)\n",
      "  Read 1:    14,583,918 bp\n",
      "  Read 2:    14,493,151 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 102833 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t1 1\n",
      "12\t2\t0.0\t1\t2\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t425\t0.0\t1\t44 381\n",
      "19\t102207\t0.0\t1\t92793 9414\n",
      "20\t196\t0.0\t1\t44 152\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 102496 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1712.5\t0\t16\n",
      "4\t5\t428.1\t0\t5\n",
      "11\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t740\t0.0\t1\t122 618\n",
      "20\t101511\t0.0\t2\t98873 2638\n",
      "21\t209\t0.0\t2\t47 162\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1831D-T1-H-OA-T_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1831D-T1-H-OA-T_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1831D-T1-H-OA-T_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1831D-T1-H-OA-T_66_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1831D-T1-H-OA-T_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1831D-T1-H-OA-T_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1831D-T1-H-OA-T_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1831D-T1-H-OA-T_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       144,986 reads  @     22.6 µs/read;   2.65 M reads/minute\n",
      "Finished in 3.32 s (23 us/read; 2.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            144,986\n",
      "  Read 1 with adapter:                 136,531 (94.2%)\n",
      "  Read 2 with adapter:                 135,915 (93.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       144,986 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,695,161 bp\n",
      "  Read 1:    21,852,440 bp\n",
      "  Read 2:    21,842,721 bp\n",
      "Total written (filtered):     38,384,476 bp (87.8%)\n",
      "  Read 1:    19,258,756 bp\n",
      "  Read 2:    19,125,720 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 136531 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.6\t0\t2\n",
      "10\t2\t0.1\t1\t1 1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t5\t0.0\t1\t4 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t614\t0.0\t1\t78 536\n",
      "19\t135562\t0.0\t1\t123042 12520\n",
      "20\t332\t0.0\t1\t63 269\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 135915 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t2265.4\t0\t13\n",
      "4\t13\t566.4\t0\t13\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t16\t0.0\t1\t9 7\n",
      "19\t1059\t0.0\t1\t149 910\n",
      "20\t134500\t0.0\t2\t131331 3169\n",
      "21\t300\t0.0\t2\t75 225\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1848C-T1-H-CN-T_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1848C-T1-H-CN-T_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1848C-T1-H-CN-T_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1848C-T1-H-CN-T_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1848C-T1-H-CN-T_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1848C-T1-H-CN-T_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1848C-T1-H-CN-T_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1848C-T1-H-CN-T_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        79,725 reads  @     23.2 µs/read;   2.59 M reads/minute\n",
      "Finished in 1.88 s (24 us/read; 2.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,725\n",
      "  Read 1 with adapter:                  74,895 (93.9%)\n",
      "  Read 2 with adapter:                  74,637 (93.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        79,725 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,045,924 bp\n",
      "  Read 1:    12,028,186 bp\n",
      "  Read 2:    12,017,738 bp\n",
      "Total written (filtered):     21,130,940 bp (87.9%)\n",
      "  Read 1:    10,605,344 bp\n",
      "  Read 2:    10,525,596 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 74895 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t306\t0.0\t1\t43 263\n",
      "19\t74418\t0.0\t1\t66624 7794\n",
      "20\t164\t0.0\t1\t38 126\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 74637 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1245.7\t0\t9\n",
      "4\t7\t311.4\t0\t7\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t6\t0.0\t1\t4 2\n",
      "19\t481\t0.0\t1\t66 415\n",
      "20\t73955\t0.0\t2\t72060 1895\n",
      "21\t173\t0.0\t2\t41 132\n",
      "27\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1858-TF-D-PA-T_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1858-TF-D-PA-T_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1858-TF-D-PA-T_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1858-TF-D-PA-T_68_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1858-TF-D-PA-T_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1858-TF-D-PA-T_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1858-TF-D-PA-T_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1858-TF-D-PA-T_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       125,644 reads  @     23.2 µs/read;   2.59 M reads/minute\n",
      "Finished in 2.94 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,644\n",
      "  Read 1 with adapter:                 116,810 (93.0%)\n",
      "  Read 2 with adapter:                 116,912 (93.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       125,644 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,914,026 bp\n",
      "  Read 1:    18,958,565 bp\n",
      "  Read 2:    18,955,461 bp\n",
      "Total written (filtered):     33,358,391 bp (88.0%)\n",
      "  Read 1:    16,739,598 bp\n",
      "  Read 2:    16,618,793 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 116810 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t6\t0.0\t1\t3 3\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t572\t0.0\t1\t60 512\n",
      "19\t115964\t0.0\t1\t94027 21937\n",
      "20\t251\t0.0\t1\t51 200\n",
      "22\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 116912 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t1963.2\t0\t23\n",
      "4\t21\t490.8\t0\t21\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t16\t0.0\t1\t11 5\n",
      "19\t1058\t0.0\t1\t132 926\n",
      "20\t115483\t0.0\t2\t113032 2451\n",
      "21\t300\t0.0\t2\t51 249\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1870A-TF-H-PS-T_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1870A-TF-H-PS-T_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1870A-TF-H-PS-T_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1870A-TF-H-PS-T_69_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1870A-TF-H-PS-T_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-1870A-TF-H-PS-T_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1870A-TF-H-PS-T_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-1870A-TF-H-PS-T_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       106,559 reads  @     25.0 µs/read;   2.40 M reads/minute\n",
      "Finished in 2.69 s (25 us/read; 2.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,559\n",
      "  Read 1 with adapter:                  97,896 (91.9%)\n",
      "  Read 2 with adapter:                  98,797 (92.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,559 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,096,762 bp\n",
      "  Read 1:    16,069,940 bp\n",
      "  Read 2:    16,026,822 bp\n",
      "Total written (filtered):     28,262,062 bp (88.1%)\n",
      "  Read 1:    14,210,151 bp\n",
      "  Read 2:    14,051,911 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 97896 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t0 2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t2 2\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t380\t0.0\t1\t45 335\n",
      "19\t97276\t0.0\t1\t68577 28699\n",
      "20\t222\t0.0\t1\t38 184\n",
      "25\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 98797 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1665.0\t0\t9\n",
      "4\t24\t416.2\t0\t24\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t5\t0.0\t1\t4 1\n",
      "19\t673\t0.0\t1\t114 559\n",
      "20\t97868\t0.0\t2\t95993 1875\n",
      "21\t212\t0.0\t2\t54 158\n",
      "22\t1\t0.0\t2\t0 1\n",
      "26\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-2186E-T1-H-PA-T_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-2186E-T1-H-PA-T_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-2186E-T1-H-PA-T_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-2186E-T1-H-PA-T_70_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-2186E-T1-H-PA-T_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-2186E-T1-H-PA-T_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-2186E-T1-H-PA-T_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-2186E-T1-H-PA-T_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        78,178 reads  @     22.8 µs/read;   2.63 M reads/minute\n",
      "Finished in 1.81 s (23 us/read; 2.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,178\n",
      "  Read 1 with adapter:                  73,391 (93.9%)\n",
      "  Read 2 with adapter:                  73,074 (93.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        78,178 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,514,209 bp\n",
      "  Read 1:    11,757,891 bp\n",
      "  Read 2:    11,756,318 bp\n",
      "Total written (filtered):     20,659,450 bp (87.9%)\n",
      "  Read 1:    10,363,721 bp\n",
      "  Read 2:    10,295,729 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 73391 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t387\t0.0\t1\t42 345\n",
      "19\t72830\t0.0\t1\t65349 7481\n",
      "20\t166\t0.0\t1\t31 135\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 73074 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1221.5\t0\t8\n",
      "4\t9\t305.4\t0\t9\n",
      "9\t1\t0.3\t0\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t7\t0.0\t1\t6 1\n",
      "18\t9\t0.0\t1\t6 3\n",
      "19\t736\t0.0\t1\t107 629\n",
      "20\t72123\t0.0\t2\t70362 1761\n",
      "21\t179\t0.0\t2\t37 142\n",
      "22\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-2199D-TF-D-SS-T_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-2199D-TF-D-SS-T_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-2199D-TF-D-SS-T_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-2199D-TF-D-SS-T_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-2199D-TF-D-SS-T_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-2199D-TF-D-SS-T_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-2199D-TF-D-SS-T_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-2199D-TF-D-SS-T_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       226,134 reads  @     18.4 µs/read;   3.26 M reads/minute\n",
      "Finished in 4.19 s (19 us/read; 3.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            226,134\n",
      "  Read 1 with adapter:                 211,631 (93.6%)\n",
      "  Read 2 with adapter:                 210,856 (93.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       226,134 (100.0%)\n",
      "\n",
      "Total basepairs processed:    68,056,764 bp\n",
      "  Read 1:    34,120,392 bp\n",
      "  Read 2:    33,936,372 bp\n",
      "Total written (filtered):     59,822,081 bp (87.9%)\n",
      "  Read 1:    30,100,333 bp\n",
      "  Read 2:    29,721,748 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 211631 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.9\t0\t3\n",
      "11\t3\t0.1\t1\t2 1\n",
      "12\t7\t0.0\t1\t6 1\n",
      "13\t4\t0.0\t1\t3 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t5\t0.0\t1\t0 5\n",
      "17\t9\t0.0\t1\t5 4\n",
      "18\t1304\t0.0\t1\t109 1195\n",
      "19\t209752\t0.0\t1\t185861 23891\n",
      "20\t540\t0.0\t1\t90 450\n",
      "21\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 210856 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t3533.3\t0\t24\n",
      "4\t31\t883.3\t0\t31\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t2\t0.1\t1\t2\n",
      "12\t4\t0.0\t1\t2 2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t0 3\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t14\t0.0\t1\t6 8\n",
      "19\t1955\t0.0\t1\t264 1691\n",
      "20\t208315\t0.0\t2\t203202 5113\n",
      "21\t500\t0.0\t2\t90 410\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-3FLAT-DD-OA-T_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-3FLAT-DD-OA-T_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-3FLAT-DD-OA-T_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-3FLAT-DD-OA-T_72_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-3FLAT-DD-OA-T_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-3FLAT-DD-OA-T_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-3FLAT-DD-OA-T_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-3FLAT-DD-OA-T_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       103,836 reads  @     24.7 µs/read;   2.43 M reads/minute\n",
      "Finished in 2.60 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,836\n",
      "  Read 1 with adapter:                  99,200 (95.5%)\n",
      "  Read 2 with adapter:                  98,688 (95.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       103,836 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,315,772 bp\n",
      "  Read 1:    15,671,254 bp\n",
      "  Read 2:    15,644,518 bp\n",
      "Total written (filtered):     27,458,118 bp (87.7%)\n",
      "  Read 1:    13,786,575 bp\n",
      "  Read 2:    13,671,543 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99200 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t369\t0.0\t1\t48 321\n",
      "19\t98522\t0.0\t1\t91842 6680\n",
      "20\t291\t0.0\t1\t49 242\n",
      "21\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 98688 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1622.4\t0\t9\n",
      "4\t10\t405.6\t0\t10\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t6\t0.0\t1\t3 3\n",
      "19\t678\t0.0\t1\t99 579\n",
      "20\t97769\t0.0\t2\t95455 2314\n",
      "21\t213\t0.0\t2\t44 169\n",
      "24\t2\t0.0\t2\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-6FLAT-DD-OA-T_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-6FLAT-DD-OA-T_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-6FLAT-DD-OA-T_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-6FLAT-DD-OA-T_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-6FLAT-DD-OA-T_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-6FLAT-DD-OA-T_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-6FLAT-DD-OA-T_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-6FLAT-DD-OA-T_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        73,280 reads  @     24.2 µs/read;   2.48 M reads/minute\n",
      "Finished in 1.80 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,280\n",
      "  Read 1 with adapter:                  69,492 (94.8%)\n",
      "  Read 2 with adapter:                  69,114 (94.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        73,280 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,925,986 bp\n",
      "  Read 1:    10,969,445 bp\n",
      "  Read 2:    10,956,541 bp\n",
      "Total written (filtered):     19,224,119 bp (87.7%)\n",
      "  Read 1:     9,649,261 bp\n",
      "  Read 2:     9,574,858 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 69492 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t3\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t4\n",
      "18\t296\t0.0\t1\t36 260\n",
      "19\t69019\t0.0\t1\t63635 5384\n",
      "20\t169\t0.0\t1\t32 137\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 69114 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1145.0\t0\t2\n",
      "4\t12\t286.2\t0\t12\n",
      "11\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t2\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t6\t0.0\t1\t2 4\n",
      "19\t503\t0.0\t1\t81 422\n",
      "20\t68413\t0.0\t2\t66572 1841\n",
      "21\t172\t0.0\t2\t27 145\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-CONTROL4_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-CONTROL4_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-CONTROL4_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-CONTROL4_74_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-CONTROL4_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-CONTROL4_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-CONTROL4_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-CONTROL4_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        10,636 reads  @     64.5 µs/read;   0.93 M reads/minute\n",
      "Finished in 0.72 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,636\n",
      "  Read 1 with adapter:                  10,095 (94.9%)\n",
      "  Read 2 with adapter:                  10,055 (94.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        10,636 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,169,604 bp\n",
      "  Read 1:     1,584,969 bp\n",
      "  Read 2:     1,584,635 bp\n",
      "Total written (filtered):      2,776,828 bp (87.6%)\n",
      "  Read 1:     1,393,215 bp\n",
      "  Read 2:     1,383,613 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 10095 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t68\t0.0\t1\t8 60\n",
      "19\t10007\t0.0\t1\t9028 979\n",
      "20\t19\t0.0\t1\t1 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10055 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t41.5\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "19\t79\t0.0\t1\t14 65\n",
      "20\t9946\t0.0\t2\t9668 278\n",
      "21\t27\t0.0\t2\t6 21\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-Mock1_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-Mock1_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-Mock1_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-Mock1_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-Mock1_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS-Mock1_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-Mock1_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS-Mock1_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        79,554 reads  @     25.2 µs/read;   2.38 M reads/minute\n",
      "Finished in 2.03 s (25 us/read; 2.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,554\n",
      "  Read 1 with adapter:                  74,665 (93.9%)\n",
      "  Read 2 with adapter:                  74,391 (93.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        79,554 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,994,296 bp\n",
      "  Read 1:    12,004,213 bp\n",
      "  Read 2:    11,990,083 bp\n",
      "Total written (filtered):     21,089,099 bp (87.9%)\n",
      "  Read 1:    10,585,897 bp\n",
      "  Read 2:    10,503,202 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 74665 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t7\t0.0\t1\t2 5\n",
      "18\t454\t0.0\t1\t44 410\n",
      "19\t74010\t0.0\t1\t66403 7607\n",
      "20\t187\t0.0\t1\t41 146\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 74391 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1243.0\t0\t8\n",
      "4\t14\t310.8\t0\t14\n",
      "13\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t7\t0.0\t1\t6 1\n",
      "19\t725\t0.0\t1\t104 621\n",
      "20\t73451\t0.0\t2\t71372 2079\n",
      "21\t181\t0.0\t2\t37 144\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS1-1659E-TF-D-SS-T_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS1-1659E-TF-D-SS-T_38_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS1-1659E-TF-D-SS-T_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS1-1659E-TF-D-SS-T_38_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS1-1659E-TF-D-SS-T_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-r6m4l00v/LibS1-1659E-TF-D-SS-T_38_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS1-1659E-TF-D-SS-T_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-3ikussqw/818dbce3-19c0-4faa-b717-0298225b7179/data/LibS1-1659E-TF-D-SS-T_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       101,843 reads  @     23.2 µs/read;   2.59 M reads/minute\n",
      "Finished in 2.39 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,843\n",
      "  Read 1 with adapter:                  95,859 (94.1%)\n",
      "  Read 2 with adapter:                  95,284 (93.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       101,843 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,675,284 bp\n",
      "  Read 1:    15,349,128 bp\n",
      "  Read 2:    15,326,156 bp\n",
      "Total written (filtered):     26,949,460 bp (87.9%)\n",
      "  Read 1:    13,528,139 bp\n",
      "  Read 2:    13,421,321 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 95859 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t0 3\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t501\t0.0\t1\t56 445\n",
      "19\t95091\t0.0\t1\t86700 8391\n",
      "20\t247\t0.0\t1\t51 196\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 95284 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1591.3\t0\t10\n",
      "4\t4\t397.8\t0\t4\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t11\t0.0\t1\t4 7\n",
      "19\t821\t0.0\t1\t93 728\n",
      "20\t94187\t0.0\t2\t91505 2682\n",
      "21\t243\t0.0\t2\t50 193\n",
      "33\t2\t0.0\t2\t2\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibS.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1301-T1-H-DL-T_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1301-T1-H-DL-T_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1301-T1-H-DL-T_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1301-T1-H-DL-T_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1301-T1-H-DL-T_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1301-T1-H-DL-T_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1301-T1-H-DL-T_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1301-T1-H-DL-T_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       132,379 reads  @     19.6 µs/read;   3.06 M reads/minute\n",
      "Finished in 2.62 s (20 us/read; 3.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            132,379\n",
      "  Read 1 with adapter:                 131,865 (99.6%)\n",
      "  Read 2 with adapter:                 131,864 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       132,379 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,769,649 bp\n",
      "  Read 1:    19,891,852 bp\n",
      "  Read 2:    19,877,797 bp\n",
      "Total written (filtered):     34,628,312 bp (87.1%)\n",
      "  Read 1:    17,386,857 bp\n",
      "  Read 2:    17,241,455 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 131865 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t0 2\n",
      "13\t2\t0.0\t1\t2\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t12\t0.0\t1\t5 7\n",
      "18\t722\t0.0\t1\t74 648\n",
      "19\t130755\t0.0\t1\t121869 8886\n",
      "20\t364\t0.0\t1\t57 307\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 131864 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t5\t0.0\t1\t1 4\n",
      "15\t4\t0.0\t1\t1 3\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t11\t0.0\t1\t5 6\n",
      "19\t1143\t0.0\t1\t170 973\n",
      "20\t130376\t0.0\t2\t127409 2967\n",
      "21\t317\t0.0\t2\t64 253\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1301-TF-D-DL-T_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1301-TF-D-DL-T_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1301-TF-D-DL-T_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1301-TF-D-DL-T_42_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1301-TF-D-DL-T_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1301-TF-D-DL-T_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1301-TF-D-DL-T_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1301-TF-D-DL-T_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        84,469 reads  @     20.6 µs/read;   2.91 M reads/minute\n",
      "Finished in 1.77 s (21 us/read; 2.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,469\n",
      "  Read 1 with adapter:                  84,139 (99.6%)\n",
      "  Read 2 with adapter:                  84,043 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        84,469 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,479,333 bp\n",
      "  Read 1:    12,743,536 bp\n",
      "  Read 2:    12,735,797 bp\n",
      "Total written (filtered):     22,200,746 bp (87.1%)\n",
      "  Read 1:    11,145,153 bp\n",
      "  Read 2:    11,055,593 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 84139 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.3\t0\t2\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t3\t0.0\t1\t3\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t446\t0.0\t1\t57 389\n",
      "19\t83442\t0.0\t1\t78483 4959\n",
      "20\t240\t0.0\t1\t50 190\n",
      "31\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 84043 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t11\t0.0\t1\t5 6\n",
      "19\t796\t0.0\t1\t109 687\n",
      "20\t83013\t0.0\t2\t81191 1822\n",
      "21\t213\t0.0\t2\t40 173\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1305C-T1-H-OA-T_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1305C-T1-H-OA-T_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1305C-T1-H-OA-T_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1305C-T1-H-OA-T_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1305C-T1-H-OA-T_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1305C-T1-H-OA-T_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1305C-T1-H-OA-T_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1305C-T1-H-OA-T_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       137,824 reads  @     17.4 µs/read;   3.45 M reads/minute\n",
      "Finished in 2.43 s (18 us/read; 3.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            137,824\n",
      "  Read 1 with adapter:                 137,329 (99.6%)\n",
      "  Read 2 with adapter:                 137,235 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       137,824 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,505,301 bp\n",
      "  Read 1:    20,767,636 bp\n",
      "  Read 2:    20,737,665 bp\n",
      "Total written (filtered):     36,152,689 bp (87.1%)\n",
      "  Read 1:    18,158,814 bp\n",
      "  Read 2:    17,993,875 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 137329 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.5\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t7\t0.0\t1\t5 2\n",
      "16\t4\t0.0\t1\t0 4\n",
      "17\t9\t0.0\t1\t7 2\n",
      "18\t735\t0.0\t1\t99 636\n",
      "19\t136148\t0.0\t1\t127486 8662\n",
      "20\t415\t0.0\t1\t67 348\n",
      "22\t1\t0.0\t1\t0 1\n",
      "43\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 137235 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t13\t0.0\t1\t8 5\n",
      "19\t1193\t0.0\t1\t164 1029\n",
      "20\t135651\t0.0\t2\t132655 2996\n",
      "21\t363\t0.0\t2\t84 279\n",
      "22\t1\t0.0\t2\t0 1\n",
      "44\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1312-T1-H-DL-T_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1312-T1-H-DL-T_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1312-T1-H-DL-T_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1312-T1-H-DL-T_44_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1312-T1-H-DL-T_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1312-T1-H-DL-T_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1312-T1-H-DL-T_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1312-T1-H-DL-T_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:05       269,512 reads  @     21.1 µs/read;   2.84 M reads/minute\n",
      "Finished in 5.72 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            269,512\n",
      "  Read 1 with adapter:                 268,384 (99.6%)\n",
      "  Read 2 with adapter:                 268,430 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       269,512 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,089,642 bp\n",
      "  Read 1:    40,596,122 bp\n",
      "  Read 2:    40,493,520 bp\n",
      "Total written (filtered):     70,624,417 bp (87.1%)\n",
      "  Read 1:    35,497,643 bp\n",
      "  Read 2:    35,126,774 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 268384 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t65.8\t0\t1\n",
      "9\t2\t1.0\t0\t2\n",
      "11\t1\t0.1\t1\t1\n",
      "12\t8\t0.0\t1\t6 2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t7\t0.0\t1\t2 5\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t7\t0.0\t1\t3 4\n",
      "17\t14\t0.0\t1\t5 9\n",
      "18\t1413\t0.0\t1\t121 1292\n",
      "19\t266144\t0.0\t1\t249012 17132\n",
      "20\t781\t0.0\t1\t111 670\n",
      "25\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 268430 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t1052.8\t0\t1\n",
      "10\t2\t0.3\t1\t2\n",
      "11\t2\t0.1\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t7\t0.0\t1\t3 4\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t11\t0.0\t1\t8 3\n",
      "18\t29\t0.0\t1\t12 17\n",
      "19\t2356\t0.0\t1\t363 1993\n",
      "20\t265305\t0.0\t2\t259517 5788\n",
      "21\t705\t0.0\t2\t142 563\n",
      "22\t3\t0.0\t2\t1 2\n",
      "45\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1313TF-H-DL-T_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1313TF-H-DL-T_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1313TF-H-DL-T_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1313TF-H-DL-T_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1313TF-H-DL-T_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1313TF-H-DL-T_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1313TF-H-DL-T_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1313TF-H-DL-T_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02       161,064 reads  @     17.6 µs/read;   3.40 M reads/minute\n",
      "Finished in 2.86 s (18 us/read; 3.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,064\n",
      "  Read 1 with adapter:                 160,452 (99.6%)\n",
      "  Read 2 with adapter:                 160,472 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       161,064 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,574,215 bp\n",
      "  Read 1:    24,307,078 bp\n",
      "  Read 2:    24,267,137 bp\n",
      "Total written (filtered):     42,317,759 bp (87.1%)\n",
      "  Read 1:    21,259,002 bp\n",
      "  Read 2:    21,058,757 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 160452 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t2\t0.2\t1\t1 1\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t4\t0.0\t1\t3 1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t11\t0.0\t1\t7 4\n",
      "18\t836\t0.0\t1\t104 732\n",
      "19\t159141\t0.0\t1\t149124 10017\n",
      "20\t443\t0.0\t1\t74 369\n",
      "21\t1\t0.0\t1\t0 1\n",
      "34\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 160472 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t3\t0.0\t1\t2 1\n",
      "17\t6\t0.0\t1\t5 1\n",
      "18\t18\t0.0\t1\t5 13\n",
      "19\t1346\t0.0\t1\t212 1134\n",
      "20\t158696\t0.0\t2\t154933 3763\n",
      "21\t398\t0.0\t2\t86 312\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1319G-T1-H-SS-T_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1319G-T1-H-SS-T_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1319G-T1-H-SS-T_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1319G-T1-H-SS-T_46_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1319G-T1-H-SS-T_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1319G-T1-H-SS-T_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1319G-T1-H-SS-T_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1319G-T1-H-SS-T_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       120,924 reads  @     21.5 µs/read;   2.79 M reads/minute\n",
      "Finished in 2.62 s (22 us/read; 2.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,924\n",
      "  Read 1 with adapter:                 120,402 (99.6%)\n",
      "  Read 2 with adapter:                 120,424 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       120,924 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,007,992 bp\n",
      "  Read 1:    18,010,645 bp\n",
      "  Read 2:    17,997,347 bp\n",
      "Total written (filtered):     31,312,971 bp (87.0%)\n",
      "  Read 1:    15,723,369 bp\n",
      "  Read 2:    15,589,602 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 120402 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t1 2\n",
      "13\t4\t0.0\t1\t3 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t11\t0.0\t1\t7 4\n",
      "18\t591\t0.0\t1\t60 531\n",
      "19\t119477\t0.0\t1\t111223 8254\n",
      "20\t308\t0.0\t1\t44 264\n",
      "22\t1\t0.0\t1\t0 1\n",
      "23\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 120424 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "18\t17\t0.0\t1\t12 5\n",
      "19\t1014\t0.0\t1\t145 869\n",
      "20\t119081\t0.0\t2\t116628 2453\n",
      "21\t307\t0.0\t2\t63 244\n",
      "22\t1\t0.0\t2\t0 1\n",
      "41\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1326E-TF-H-PS-T_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1326E-TF-H-PS-T_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1326E-TF-H-PS-T_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1326E-TF-H-PS-T_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1326E-TF-H-PS-T_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1326E-TF-H-PS-T_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1326E-TF-H-PS-T_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1326E-TF-H-PS-T_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        87,244 reads  @     23.7 µs/read;   2.53 M reads/minute\n",
      "Finished in 2.10 s (24 us/read; 2.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             87,244\n",
      "  Read 1 with adapter:                  86,845 (99.5%)\n",
      "  Read 2 with adapter:                  86,898 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        87,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,332,878 bp\n",
      "  Read 1:    13,172,246 bp\n",
      "  Read 2:    13,160,632 bp\n",
      "Total written (filtered):     22,945,524 bp (87.1%)\n",
      "  Read 1:    11,522,387 bp\n",
      "  Read 2:    11,423,137 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 86845 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t392\t0.0\t1\t40 352\n",
      "19\t86226\t0.0\t1\t80308 5918\n",
      "20\t219\t0.0\t1\t44 175\n",
      "26\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 86898 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t10\t0.0\t1\t3 7\n",
      "19\t619\t0.0\t1\t93 526\n",
      "20\t86053\t0.0\t2\t84058 1995\n",
      "21\t209\t0.0\t2\t51 158\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1327-T1-H-OA-T_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1327-T1-H-OA-T_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1327-T1-H-OA-T_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1327-T1-H-OA-T_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1327-T1-H-OA-T_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1327-T1-H-OA-T_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1327-T1-H-OA-T_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1327-T1-H-OA-T_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02       108,979 reads  @     23.8 µs/read;   2.52 M reads/minute\n",
      "Finished in 2.63 s (24 us/read; 2.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,979\n",
      "  Read 1 with adapter:                 108,654 (99.7%)\n",
      "  Read 2 with adapter:                 108,624 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,979 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,874,194 bp\n",
      "  Read 1:    16,454,009 bp\n",
      "  Read 2:    16,420,185 bp\n",
      "Total written (filtered):     28,638,017 bp (87.1%)\n",
      "  Read 1:    14,389,808 bp\n",
      "  Read 2:    14,248,209 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108654 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t423\t0.0\t1\t40 383\n",
      "19\t107964\t0.0\t1\t101613 6351\n",
      "20\t255\t0.0\t1\t51 204\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108624 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t4\t0.0\t1\t1 3\n",
      "19\t752\t0.0\t1\t149 603\n",
      "20\t107598\t0.0\t2\t105326 2272\n",
      "21\t266\t0.0\t2\t56 210\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1327B-TF-H-OA-T_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1327B-TF-H-OA-T_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1327B-TF-H-OA-T_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1327B-TF-H-OA-T_48_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1327B-TF-H-OA-T_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1327B-TF-H-OA-T_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1327B-TF-H-OA-T_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1327B-TF-H-OA-T_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       189,126 reads  @     20.9 µs/read;   2.87 M reads/minute\n",
      "Finished in 3.99 s (21 us/read; 2.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            189,126\n",
      "  Read 1 with adapter:                 188,475 (99.7%)\n",
      "  Read 2 with adapter:                 188,449 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       189,126 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,034,996 bp\n",
      "  Read 1:    28,523,680 bp\n",
      "  Read 2:    28,511,316 bp\n",
      "Total written (filtered):     49,686,426 bp (87.1%)\n",
      "  Read 1:    24,942,829 bp\n",
      "  Read 2:    24,743,597 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 188475 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t738.8\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t9\t0.0\t1\t4 5\n",
      "17\t12\t0.0\t1\t7 5\n",
      "18\t755\t0.0\t1\t104 651\n",
      "19\t187038\t0.0\t1\t176979 10059\n",
      "20\t649\t0.0\t1\t87 562\n",
      "22\t1\t0.0\t1\t0 1\n",
      "29\t1\t0.0\t1\t1\n",
      "31\t1\t0.0\t1\t1\n",
      "37\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 188449 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t738.8\t0\t2\n",
      "9\t2\t0.7\t0\t2\n",
      "11\t3\t0.0\t1\t2 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t3\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t20\t0.0\t1\t14 6\n",
      "19\t1542\t0.0\t1\t244 1298\n",
      "20\t186411\t0.0\t2\t182786 3625\n",
      "21\t458\t0.0\t2\t100 358\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1335-T1-H-DL-T_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1335-T1-H-DL-T_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1335-T1-H-DL-T_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1335-T1-H-DL-T_50_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1335-T1-H-DL-T_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1335-T1-H-DL-T_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1335-T1-H-DL-T_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1335-T1-H-DL-T_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        47,486 reads  @     27.3 µs/read;   2.20 M reads/minute\n",
      "Finished in 1.33 s (28 us/read; 2.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,486\n",
      "  Read 1 with adapter:                  47,219 (99.4%)\n",
      "  Read 2 with adapter:                  47,275 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        47,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,195,373 bp\n",
      "  Read 1:     7,106,904 bp\n",
      "  Read 2:     7,088,469 bp\n",
      "Total written (filtered):     12,353,104 bp (87.0%)\n",
      "  Read 1:     6,209,851 bp\n",
      "  Read 2:     6,143,253 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 47219 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t217\t0.0\t1\t26 191\n",
      "19\t46863\t0.0\t1\t43020 3843\n",
      "20\t133\t0.0\t1\t19 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 47275 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t2\n",
      "19\t384\t0.0\t1\t53 331\n",
      "20\t46769\t0.0\t2\t45840 929\n",
      "21\t118\t0.0\t2\t32 86\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1338A-TF-H-OA-T_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1338A-TF-H-OA-T_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1338A-TF-H-OA-T_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1338A-TF-H-OA-T_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1338A-TF-H-OA-T_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1338A-TF-H-OA-T_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1338A-TF-H-OA-T_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1338A-TF-H-OA-T_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       200,477 reads  @     20.3 µs/read;   2.95 M reads/minute\n",
      "Finished in 4.10 s (20 us/read; 2.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            200,477\n",
      "  Read 1 with adapter:                 199,501 (99.5%)\n",
      "  Read 2 with adapter:                 199,861 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       200,477 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,462,665 bp\n",
      "  Read 1:    30,246,802 bp\n",
      "  Read 2:    30,215,863 bp\n",
      "Total written (filtered):     52,676,255 bp (87.1%)\n",
      "  Read 1:    26,456,627 bp\n",
      "  Read 2:    26,219,628 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 199501 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t3\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t7\t0.0\t1\t4 3\n",
      "17\t12\t0.0\t1\t5 7\n",
      "18\t789\t0.0\t1\t92 697\n",
      "19\t198179\t0.0\t1\t183315 14864\n",
      "20\t505\t0.0\t1\t81 424\n",
      "29\t1\t0.0\t1\t1\n",
      "34\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 199861 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t3132.5\t0\t1\n",
      "4\t1\t783.1\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t6\t0.0\t1\t5 1\n",
      "18\t12\t0.0\t1\t8 4\n",
      "19\t1331\t0.0\t1\t240 1091\n",
      "20\t198070\t0.0\t2\t194102 3968\n",
      "21\t434\t0.0\t2\t104 330\n",
      "22\t1\t0.0\t2\t1\n",
      "29\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1339A-TF-D-SS-T_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1339A-TF-D-SS-T_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1339A-TF-D-SS-T_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1339A-TF-D-SS-T_52_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1339A-TF-D-SS-T_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1339A-TF-D-SS-T_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1339A-TF-D-SS-T_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1339A-TF-D-SS-T_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       134,535 reads  @     22.0 µs/read;   2.73 M reads/minute\n",
      "Finished in 2.99 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            134,535\n",
      "  Read 1 with adapter:                 133,958 (99.6%)\n",
      "  Read 2 with adapter:                 133,955 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       134,535 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,278,104 bp\n",
      "  Read 1:    20,146,100 bp\n",
      "  Read 2:    20,132,004 bp\n",
      "Total written (filtered):     35,054,891 bp (87.0%)\n",
      "  Read 1:    17,601,228 bp\n",
      "  Read 2:    17,453,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 133958 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t651\t0.0\t1\t78 573\n",
      "19\t132923\t0.0\t1\t124451 8472\n",
      "20\t374\t0.0\t1\t58 316\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 133955 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t13\t0.0\t1\t7 6\n",
      "19\t1000\t0.0\t1\t139 861\n",
      "20\t132629\t0.0\t2\t129850 2779\n",
      "21\t303\t0.0\t2\t67 236\n",
      "22\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1342B-T1-H-OA-T_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1342B-T1-H-OA-T_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1342B-T1-H-OA-T_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1342B-T1-H-OA-T_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1342B-T1-H-OA-T_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1342B-T1-H-OA-T_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1342B-T1-H-OA-T_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1342B-T1-H-OA-T_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       152,600 reads  @     20.1 µs/read;   2.98 M reads/minute\n",
      "Finished in 3.10 s (20 us/read; 2.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            152,600\n",
      "  Read 1 with adapter:                 151,964 (99.6%)\n",
      "  Read 2 with adapter:                 152,004 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       152,600 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,005,797 bp\n",
      "  Read 1:    23,010,181 bp\n",
      "  Read 2:    22,995,616 bp\n",
      "Total written (filtered):     40,079,657 bp (87.1%)\n",
      "  Read 1:    20,123,224 bp\n",
      "  Read 2:    19,956,433 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 151964 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.6\t0\t2\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t10\t0.0\t1\t6 4\n",
      "18\t695\t0.0\t1\t85 610\n",
      "19\t150834\t0.0\t1\t141023 9811\n",
      "20\t408\t0.0\t1\t64 344\n",
      "21\t1\t0.0\t1\t1\n",
      "52\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 152004 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2384.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t3\t0.0\t1\t3\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t11\t0.0\t1\t5 6\n",
      "19\t1158\t0.0\t1\t180 978\n",
      "20\t150455\t0.0\t2\t147306 3149\n",
      "21\t367\t0.0\t2\t89 278\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1342D-TF-H-OA-T_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1342D-TF-H-OA-T_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1342D-TF-H-OA-T_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1342D-TF-H-OA-T_54_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1342D-TF-H-OA-T_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1342D-TF-H-OA-T_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1342D-TF-H-OA-T_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1342D-TF-H-OA-T_54_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       180,737 reads  @     18.3 µs/read;   3.28 M reads/minute\n",
      "Finished in 3.34 s (18 us/read; 3.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            180,737\n",
      "  Read 1 with adapter:                 180,098 (99.6%)\n",
      "  Read 2 with adapter:                 180,086 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       180,737 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,482,802 bp\n",
      "  Read 1:    27,288,056 bp\n",
      "  Read 2:    27,194,746 bp\n",
      "Total written (filtered):     47,460,734 bp (87.1%)\n",
      "  Read 1:    23,866,620 bp\n",
      "  Read 2:    23,594,114 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 180098 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t4\t0.7\t0\t4\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t3\t0.0\t1\t3\n",
      "16\t7\t0.0\t1\t5 2\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t857\t0.0\t1\t70 787\n",
      "19\t178726\t0.0\t1\t168262 10464\n",
      "20\t491\t0.0\t1\t79 412\n",
      "21\t1\t0.0\t1\t1\n",
      "31\t1\t0.0\t1\t1\n",
      "54\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 180086 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t4\t0.0\t1\t1 3\n",
      "15\t8\t0.0\t1\t4 4\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t15\t0.0\t1\t7 8\n",
      "19\t1347\t0.0\t1\t211 1136\n",
      "20\t178285\t0.0\t2\t174379 3906\n",
      "21\t415\t0.0\t2\t98 317\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1345-T1-H-CN-T_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1345-T1-H-CN-T_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1345-T1-H-CN-T_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1345-T1-H-CN-T_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1345-T1-H-CN-T_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1345-T1-H-CN-T_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1345-T1-H-CN-T_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1345-T1-H-CN-T_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        64,329 reads  @     23.9 µs/read;   2.51 M reads/minute\n",
      "Finished in 1.57 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,329\n",
      "  Read 1 with adapter:                  64,019 (99.5%)\n",
      "  Read 2 with adapter:                  64,114 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,329 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,420,686 bp\n",
      "  Read 1:     9,712,792 bp\n",
      "  Read 2:     9,707,894 bp\n",
      "Total written (filtered):     16,922,585 bp (87.1%)\n",
      "  Read 1:     8,496,609 bp\n",
      "  Read 2:     8,425,976 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 64019 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.2\t0\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t281\t0.0\t1\t24 257\n",
      "19\t63561\t0.0\t1\t59061 4500\n",
      "20\t166\t0.0\t1\t24 142\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 64114 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t4\t0.0\t1\t1 3\n",
      "19\t467\t0.0\t1\t76 391\n",
      "20\t63507\t0.0\t2\t62127 1380\n",
      "21\t133\t0.0\t2\t31 102\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1350G-TF-D-SS-T_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1350G-TF-D-SS-T_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1350G-TF-D-SS-T_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1350G-TF-D-SS-T_56_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1350G-TF-D-SS-T_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1350G-TF-D-SS-T_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1350G-TF-D-SS-T_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1350G-TF-D-SS-T_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       103,522 reads  @     23.8 µs/read;   2.52 M reads/minute\n",
      "Finished in 2.50 s (24 us/read; 2.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,522\n",
      "  Read 1 with adapter:                 103,065 (99.6%)\n",
      "  Read 2 with adapter:                 103,003 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       103,522 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,223,300 bp\n",
      "  Read 1:    15,616,852 bp\n",
      "  Read 2:    15,606,448 bp\n",
      "Total written (filtered):     27,206,158 bp (87.1%)\n",
      "  Read 1:    13,658,985 bp\n",
      "  Read 2:    13,547,173 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103065 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t0 3\n",
      "17\t12\t0.0\t1\t8 4\n",
      "18\t619\t0.0\t1\t57 562\n",
      "19\t102096\t0.0\t1\t94990 7106\n",
      "20\t330\t0.0\t1\t60 270\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 103003 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t5\t0.0\t1\t4 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t14\t0.0\t1\t4 10\n",
      "19\t981\t0.0\t1\t138 843\n",
      "20\t101689\t0.0\t2\t99256 2433\n",
      "21\t304\t0.0\t2\t51 253\n",
      "23\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1356C-T1-H-MC-T_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1356C-T1-H-MC-T_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1356C-T1-H-MC-T_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1356C-T1-H-MC-T_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1356C-T1-H-MC-T_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1356C-T1-H-MC-T_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1356C-T1-H-MC-T_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1356C-T1-H-MC-T_57_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       139,265 reads  @     22.1 µs/read;   2.72 M reads/minute\n",
      "Finished in 3.11 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            139,265\n",
      "  Read 1 with adapter:                 138,590 (99.5%)\n",
      "  Read 2 with adapter:                 138,690 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       139,265 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,846,331 bp\n",
      "  Read 1:    20,952,780 bp\n",
      "  Read 2:    20,893,551 bp\n",
      "Total written (filtered):     36,440,501 bp (87.1%)\n",
      "  Read 1:    18,319,980 bp\n",
      "  Read 2:    18,120,521 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 138590 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t11\t0.0\t1\t7 4\n",
      "13\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t8\t0.0\t1\t3 5\n",
      "17\t12\t0.0\t1\t7 5\n",
      "18\t590\t0.0\t1\t61 529\n",
      "19\t137608\t0.0\t1\t126923 10685\n",
      "20\t354\t0.0\t1\t69 285\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 138690 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t12\t0.0\t1\t8 4\n",
      "19\t1023\t0.0\t1\t192 831\n",
      "20\t137329\t0.0\t2\t134533 2796\n",
      "21\t316\t0.0\t2\t55 261\n",
      "22\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1360A-T1-H-MC-T_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1360A-T1-H-MC-T_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1360A-T1-H-MC-T_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1360A-T1-H-MC-T_58_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1360A-T1-H-MC-T_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1360A-T1-H-MC-T_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1360A-T1-H-MC-T_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1360A-T1-H-MC-T_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       137,277 reads  @     21.2 µs/read;   2.83 M reads/minute\n",
      "Finished in 2.94 s (21 us/read; 2.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            137,277\n",
      "  Read 1 with adapter:                 136,782 (99.6%)\n",
      "  Read 2 with adapter:                 136,828 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       137,277 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,420,012 bp\n",
      "  Read 1:    20,727,231 bp\n",
      "  Read 2:    20,692,781 bp\n",
      "Total written (filtered):     36,085,565 bp (87.1%)\n",
      "  Read 1:    18,128,698 bp\n",
      "  Read 2:    17,956,867 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 136782 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.5\t0\t2 1\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t4\t0.0\t1\t3 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t560\t0.0\t1\t65 495\n",
      "19\t135875\t0.0\t1\t127257 8618\n",
      "20\t324\t0.0\t1\t68 256\n",
      "42\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 136828 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2145.0\t0\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t885\t0.0\t1\t126 759\n",
      "20\t135628\t0.0\t2\t132526 3102\n",
      "21\t298\t0.0\t2\t59 239\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1364A-TF-H-OA-T_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1364A-TF-H-OA-T_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1364A-TF-H-OA-T_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1364A-TF-H-OA-T_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1364A-TF-H-OA-T_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1364A-TF-H-OA-T_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1364A-TF-H-OA-T_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1364A-TF-H-OA-T_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        25,927 reads  @     32.1 µs/read;   1.87 M reads/minute\n",
      "Finished in 0.87 s (34 us/read; 1.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,927\n",
      "  Read 1 with adapter:                  25,804 (99.5%)\n",
      "  Read 2 with adapter:                  25,830 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,927 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,808,798 bp\n",
      "  Read 1:     3,911,676 bp\n",
      "  Read 2:     3,897,122 bp\n",
      "Total written (filtered):      6,802,190 bp (87.1%)\n",
      "  Read 1:     3,421,486 bp\n",
      "  Read 2:     3,380,704 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 25804 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t143\t0.0\t1\t14 129\n",
      "19\t25584\t0.0\t1\t23612 1972\n",
      "20\t73\t0.0\t1\t19 54\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 25830 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t207\t0.0\t1\t31 176\n",
      "20\t25557\t0.0\t2\t24993 564\n",
      "21\t59\t0.0\t2\t12 47\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1369-TF-H-DL-T_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1369-TF-H-DL-T_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1369-TF-H-DL-T_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1369-TF-H-DL-T_60_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1369-TF-H-DL-T_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1369-TF-H-DL-T_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1369-TF-H-DL-T_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1369-TF-H-DL-T_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        52,058 reads  @     27.1 µs/read;   2.22 M reads/minute\n",
      "Finished in 1.44 s (28 us/read; 2.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,058\n",
      "  Read 1 with adapter:                  51,829 (99.6%)\n",
      "  Read 2 with adapter:                  51,870 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,058 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,673,896 bp\n",
      "  Read 1:     7,841,731 bp\n",
      "  Read 2:     7,832,165 bp\n",
      "Total written (filtered):     13,652,253 bp (87.1%)\n",
      "  Read 1:     6,857,126 bp\n",
      "  Read 2:     6,795,127 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 51829 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t263\t0.0\t1\t24 239\n",
      "19\t51413\t0.0\t1\t47986 3427\n",
      "20\t147\t0.0\t1\t19 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 51870 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t2\t0.0\t1\t1 1\n",
      "19\t438\t0.0\t1\t64 374\n",
      "20\t51308\t0.0\t2\t50237 1071\n",
      "21\t114\t0.0\t2\t22 92\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1610C-T1-H-OA-T_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1610C-T1-H-OA-T_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1610C-T1-H-OA-T_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1610C-T1-H-OA-T_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1610C-T1-H-OA-T_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1610C-T1-H-OA-T_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1610C-T1-H-OA-T_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1610C-T1-H-OA-T_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       114,700 reads  @     21.1 µs/read;   2.85 M reads/minute\n",
      "Finished in 2.44 s (21 us/read; 2.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,700\n",
      "  Read 1 with adapter:                 114,192 (99.6%)\n",
      "  Read 2 with adapter:                 114,209 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,700 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,590,690 bp\n",
      "  Read 1:    17,302,957 bp\n",
      "  Read 2:    17,287,733 bp\n",
      "Total written (filtered):     30,137,935 bp (87.1%)\n",
      "  Read 1:    15,133,658 bp\n",
      "  Read 2:    15,004,277 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 114192 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t583\t0.0\t1\t60 523\n",
      "19\t113317\t0.0\t1\t105405 7912\n",
      "20\t277\t0.0\t1\t55 222\n",
      "22\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 114209 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1792.2\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t3\t0.0\t1\t0 3\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t14\t0.0\t1\t8 6\n",
      "19\t867\t0.0\t1\t109 758\n",
      "20\t113053\t0.0\t2\t110508 2545\n",
      "21\t259\t0.0\t2\t62 197\n",
      "24\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1613B-TF-H-SS-T_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1613B-TF-H-SS-T_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1613B-TF-H-SS-T_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1613B-TF-H-SS-T_62_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1613B-TF-H-SS-T_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1613B-TF-H-SS-T_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1613B-TF-H-SS-T_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1613B-TF-H-SS-T_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       165,387 reads  @     16.7 µs/read;   3.59 M reads/minute\n",
      "Finished in 2.80 s (17 us/read; 3.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            165,387\n",
      "  Read 1 with adapter:                 164,696 (99.6%)\n",
      "  Read 2 with adapter:                 164,824 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       165,387 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,800,853 bp\n",
      "  Read 1:    24,907,975 bp\n",
      "  Read 2:    24,892,878 bp\n",
      "Total written (filtered):     43,376,544 bp (87.1%)\n",
      "  Read 1:    21,779,147 bp\n",
      "  Read 2:    21,597,397 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 164696 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t4\t0.0\t1\t4\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t4\t0.0\t1\t0 4\n",
      "18\t771\t0.0\t1\t80 691\n",
      "19\t163487\t0.0\t1\t153251 10236\n",
      "20\t426\t0.0\t1\t80 346\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 164824 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t646.0\t0\t1\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t18\t0.0\t1\t10 8\n",
      "19\t1269\t0.0\t1\t189 1080\n",
      "20\t163155\t0.0\t2\t159930 3225\n",
      "21\t374\t0.0\t2\t85 289\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1633A-TF-D-PA-T_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1633A-TF-D-PA-T_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1633A-TF-D-PA-T_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1633A-TF-D-PA-T_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1633A-TF-D-PA-T_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1633A-TF-D-PA-T_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1633A-TF-D-PA-T_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1633A-TF-D-PA-T_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        55,605 reads  @     26.6 µs/read;   2.26 M reads/minute\n",
      "Finished in 1.51 s (27 us/read; 2.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,605\n",
      "  Read 1 with adapter:                  55,356 (99.6%)\n",
      "  Read 2 with adapter:                  55,340 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,605 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,767,303 bp\n",
      "  Read 1:     8,386,326 bp\n",
      "  Read 2:     8,380,977 bp\n",
      "Total written (filtered):     14,609,361 bp (87.1%)\n",
      "  Read 1:     7,334,781 bp\n",
      "  Read 2:     7,274,580 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55356 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t322\t0.0\t1\t44 278\n",
      "19\t54842\t0.0\t1\t51428 3414\n",
      "20\t175\t0.0\t1\t33 142\n",
      "31\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 55340 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t0 1\n",
      "18\t7\t0.0\t1\t2 5\n",
      "19\t536\t0.0\t1\t73 463\n",
      "20\t54643\t0.0\t2\t53457 1186\n",
      "21\t153\t0.0\t2\t25 128\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1662D-TF-H-MC-T_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1662D-TF-H-MC-T_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1662D-TF-H-MC-T_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1662D-TF-H-MC-T_64_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1662D-TF-H-MC-T_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1662D-TF-H-MC-T_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1662D-TF-H-MC-T_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1662D-TF-H-MC-T_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       178,586 reads  @     20.7 µs/read;   2.90 M reads/minute\n",
      "Finished in 3.73 s (21 us/read; 2.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            178,586\n",
      "  Read 1 with adapter:                 177,954 (99.6%)\n",
      "  Read 2 with adapter:                 177,908 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       178,586 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,662,538 bp\n",
      "  Read 1:    26,887,485 bp\n",
      "  Read 2:    26,775,053 bp\n",
      "Total written (filtered):     46,724,714 bp (87.1%)\n",
      "  Read 1:    23,506,852 bp\n",
      "  Read 2:    23,217,862 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 177954 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.7\t0\t2 1\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t13\t0.0\t1\t7 6\n",
      "18\t882\t0.0\t1\t103 779\n",
      "19\t176524\t0.0\t1\t165673 10851\n",
      "20\t515\t0.0\t1\t78 437\n",
      "21\t1\t0.0\t1\t1\n",
      "30\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 177908 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2790.4\t0\t1\n",
      "4\t1\t697.6\t0\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "18\t11\t0.0\t1\t4 7\n",
      "19\t1357\t0.0\t1\t192 1165\n",
      "20\t176104\t0.0\t2\t172023 4081\n",
      "21\t430\t0.0\t2\t89 341\n",
      "24\t1\t0.0\t2\t1\n",
      "39\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1666B-TF-H-PA-T_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1666B-TF-H-PA-T_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1666B-TF-H-PA-T_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1666B-TF-H-PA-T_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1666B-TF-H-PA-T_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1666B-TF-H-PA-T_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1666B-TF-H-PA-T_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1666B-TF-H-PA-T_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       142,821 reads  @     21.8 µs/read;   2.75 M reads/minute\n",
      "Finished in 3.14 s (22 us/read; 2.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,821\n",
      "  Read 1 with adapter:                 142,283 (99.6%)\n",
      "  Read 2 with adapter:                 142,197 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       142,821 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,107,170 bp\n",
      "  Read 1:    21,559,176 bp\n",
      "  Read 2:    21,547,994 bp\n",
      "Total written (filtered):     37,561,500 bp (87.1%)\n",
      "  Read 1:    18,856,292 bp\n",
      "  Read 2:    18,705,208 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 142283 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t10\t0.0\t1\t8 2\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t11\t0.0\t1\t8 3\n",
      "18\t800\t0.0\t1\t80 720\n",
      "19\t141030\t0.0\t1\t132083 8947\n",
      "20\t426\t0.0\t1\t88 338\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 142197 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t16\t0.0\t1\t7 9\n",
      "19\t1424\t0.0\t1\t193 1231\n",
      "20\t140373\t0.0\t2\t137201 3172\n",
      "21\t365\t0.0\t2\t79 286\n",
      "28\t1\t0.0\t2\t0 1\n",
      "39\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1666E-T1-H-PA-T_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1666E-T1-H-PA-T_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1666E-T1-H-PA-T_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1666E-T1-H-PA-T_66_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1666E-T1-H-PA-T_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1666E-T1-H-PA-T_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1666E-T1-H-PA-T_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1666E-T1-H-PA-T_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:06       316,791 reads  @     19.6 µs/read;   3.06 M reads/minute\n",
      "Finished in 6.24 s (20 us/read; 3.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            316,791\n",
      "  Read 1 with adapter:                 315,298 (99.5%)\n",
      "  Read 2 with adapter:                 315,427 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       316,791 (100.0%)\n",
      "\n",
      "Total basepairs processed:    95,609,612 bp\n",
      "  Read 1:    47,820,367 bp\n",
      "  Read 2:    47,789,245 bp\n",
      "Total written (filtered):     83,313,804 bp (87.1%)\n",
      "  Read 1:    41,830,825 bp\n",
      "  Read 2:    41,482,979 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 315298 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t1237.5\t0\t1\n",
      "9\t4\t1.2\t0\t4\n",
      "10\t3\t0.3\t1\t3\n",
      "11\t1\t0.1\t1\t1\n",
      "12\t10\t0.0\t1\t8 2\n",
      "13\t6\t0.0\t1\t3 3\n",
      "14\t4\t0.0\t1\t0 4\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t14\t0.0\t1\t6 8\n",
      "18\t1699\t0.0\t1\t160 1539\n",
      "19\t312708\t0.0\t1\t289971 22737\n",
      "20\t838\t0.0\t1\t143 695\n",
      "21\t2\t0.0\t1\t1 1\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 315427 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t1237.5\t0\t1\n",
      "11\t5\t0.1\t1\t5\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t7\t0.0\t1\t4 3\n",
      "18\t19\t0.0\t1\t11 8\n",
      "19\t2920\t0.0\t1\t405 2515\n",
      "20\t311648\t0.0\t2\t303419 8229\n",
      "21\t816\t0.0\t2\t191 625\n",
      "23\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1692C-T1-H-CN-T_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1692C-T1-H-CN-T_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1692C-T1-H-CN-T_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1692C-T1-H-CN-T_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1692C-T1-H-CN-T_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1692C-T1-H-CN-T_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1692C-T1-H-CN-T_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1692C-T1-H-CN-T_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       128,723 reads  @     19.2 µs/read;   3.12 M reads/minute\n",
      "Finished in 2.50 s (19 us/read; 3.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            128,723\n",
      "  Read 1 with adapter:                 128,173 (99.6%)\n",
      "  Read 2 with adapter:                 128,281 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       128,723 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,737,615 bp\n",
      "  Read 1:    19,390,003 bp\n",
      "  Read 2:    19,347,612 bp\n",
      "Total written (filtered):     33,737,711 bp (87.1%)\n",
      "  Read 1:    16,955,015 bp\n",
      "  Read 2:    16,782,696 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 128173 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t0 1\n",
      "12\t5\t0.0\t1\t3 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t601\t0.0\t1\t72 529\n",
      "19\t127242\t0.0\t1\t118514 8728\n",
      "20\t314\t0.0\t1\t56 258\n",
      "21\t2\t0.0\t1\t2\n",
      "41\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 128281 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t9\t0.0\t1\t6 3\n",
      "19\t926\t0.0\t1\t133 793\n",
      "20\t127063\t0.0\t2\t124119 2944\n",
      "21\t273\t0.0\t2\t68 205\n",
      "44\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1827C-TF-H-PA-T_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1827C-TF-H-PA-T_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1827C-TF-H-PA-T_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1827C-TF-H-PA-T_68_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1827C-TF-H-PA-T_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1827C-TF-H-PA-T_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1827C-TF-H-PA-T_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1827C-TF-H-PA-T_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01       101,103 reads  @     17.5 µs/read;   3.43 M reads/minute\n",
      "Finished in 1.79 s (18 us/read; 3.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,103\n",
      "  Read 1 with adapter:                 100,689 (99.6%)\n",
      "  Read 2 with adapter:                 100,693 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       101,103 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,471,575 bp\n",
      "  Read 1:    15,239,106 bp\n",
      "  Read 2:    15,232,469 bp\n",
      "Total written (filtered):     26,545,583 bp (87.1%)\n",
      "  Read 1:    13,326,308 bp\n",
      "  Read 2:    13,219,275 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 100689 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "16\t8\t0.0\t1\t3 5\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t529\t0.0\t1\t44 485\n",
      "19\t99825\t0.0\t1\t93143 6682\n",
      "20\t314\t0.0\t1\t62 252\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 100693 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t10\t0.0\t1\t6 4\n",
      "19\t885\t0.0\t1\t127 758\n",
      "20\t99506\t0.0\t2\t97162 2344\n",
      "21\t281\t0.0\t2\t60 221\n",
      "23\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1827E-TF-D-PA-T_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1827E-TF-D-PA-T_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1827E-TF-D-PA-T_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1827E-TF-D-PA-T_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1827E-TF-D-PA-T_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1827E-TF-D-PA-T_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1827E-TF-D-PA-T_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1827E-TF-D-PA-T_69_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        79,081 reads  @     23.5 µs/read;   2.55 M reads/minute\n",
      "Finished in 1.89 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,081\n",
      "  Read 1 with adapter:                  78,721 (99.5%)\n",
      "  Read 2 with adapter:                  78,764 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        79,081 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,849,330 bp\n",
      "  Read 1:    11,928,811 bp\n",
      "  Read 2:    11,920,519 bp\n",
      "Total written (filtered):     20,779,136 bp (87.1%)\n",
      "  Read 1:    10,433,364 bp\n",
      "  Read 2:    10,345,772 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 78721 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t5\t0.0\t1\t4 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t406\t0.0\t1\t46 360\n",
      "19\t78077\t0.0\t1\t72140 5937\n",
      "20\t224\t0.0\t1\t38 186\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 78764 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t4\t0.0\t1\t3 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t11\t0.0\t1\t7 4\n",
      "19\t668\t0.0\t1\t118 550\n",
      "20\t77888\t0.0\t2\t76561 1327\n",
      "21\t185\t0.0\t2\t35 150\n",
      "22\t1\t0.0\t2\t1\n",
      "23\t1\t0.0\t2\t1\n",
      "31\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1842A-TF-D-PS-T_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1842A-TF-D-PS-T_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1842A-TF-D-PS-T_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1842A-TF-D-PS-T_70_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1842A-TF-D-PS-T_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1842A-TF-D-PS-T_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1842A-TF-D-PS-T_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1842A-TF-D-PS-T_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       154,234 reads  @     17.8 µs/read;   3.38 M reads/minute\n",
      "Finished in 2.77 s (18 us/read; 3.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            154,234\n",
      "  Read 1 with adapter:                 153,627 (99.6%)\n",
      "  Read 2 with adapter:                 153,762 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       154,234 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,501,788 bp\n",
      "  Read 1:    23,277,160 bp\n",
      "  Read 2:    23,224,628 bp\n",
      "Total written (filtered):     40,508,733 bp (87.1%)\n",
      "  Read 1:    20,358,532 bp\n",
      "  Read 2:    20,150,201 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 153627 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t4\n",
      "17\t9\t0.0\t1\t4 5\n",
      "18\t611\t0.0\t1\t62 549\n",
      "19\t152619\t0.0\t1\t142597 10022\n",
      "20\t376\t0.0\t1\t79 297\n",
      "21\t2\t0.0\t1\t0 2\n",
      "26\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 153762 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t602.5\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t1073\t0.0\t1\t189 884\n",
      "20\t152343\t0.0\t2\t148612 3731\n",
      "21\t333\t0.0\t2\t78 255\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1842B-TF-H-PS-T_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1842B-TF-H-PS-T_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1842B-TF-H-PS-T_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1842B-TF-H-PS-T_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1842B-TF-H-PS-T_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1842B-TF-H-PS-T_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1842B-TF-H-PS-T_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1842B-TF-H-PS-T_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        44,155 reads  @     26.1 µs/read;   2.30 M reads/minute\n",
      "Finished in 1.18 s (27 us/read; 2.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,155\n",
      "  Read 1 with adapter:                  43,958 (99.6%)\n",
      "  Read 2 with adapter:                  43,995 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,155 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,301,999 bp\n",
      "  Read 1:     6,660,009 bp\n",
      "  Read 2:     6,641,990 bp\n",
      "Total written (filtered):     11,587,279 bp (87.1%)\n",
      "  Read 1:     5,824,934 bp\n",
      "  Read 2:     5,762,345 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 43958 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t225\t0.0\t1\t28 197\n",
      "19\t43612\t0.0\t1\t40403 3209\n",
      "20\t118\t0.0\t1\t20 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 43995 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t0 1\n",
      "18\t4\t0.0\t1\t1 3\n",
      "19\t345\t0.0\t1\t47 298\n",
      "20\t43540\t0.0\t2\t42555 985\n",
      "21\t105\t0.0\t2\t23 82\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1848B-TF-H-CN-T_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1848B-TF-H-CN-T_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1848B-TF-H-CN-T_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1848B-TF-H-CN-T_72_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1848B-TF-H-CN-T_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1848B-TF-H-CN-T_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1848B-TF-H-CN-T_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1848B-TF-H-CN-T_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        75,777 reads  @     24.2 µs/read;   2.48 M reads/minute\n",
      "Finished in 1.86 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,777\n",
      "  Read 1 with adapter:                  75,217 (99.3%)\n",
      "  Read 2 with adapter:                  75,503 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,777 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,869,053 bp\n",
      "  Read 1:    11,437,282 bp\n",
      "  Read 2:    11,431,771 bp\n",
      "Total written (filtered):     19,930,420 bp (87.2%)\n",
      "  Read 1:    10,008,278 bp\n",
      "  Read 2:     9,922,142 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 75217 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t302\t0.0\t1\t27 275\n",
      "19\t74734\t0.0\t1\t67213 7521\n",
      "20\t176\t0.0\t1\t45 131\n",
      "45\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 75503 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t7\t0.0\t1\t5 2\n",
      "19\t524\t0.0\t1\t92 432\n",
      "20\t74828\t0.0\t2\t73440 1388\n",
      "21\t137\t0.0\t2\t28 109\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1858A-TF-H-PA-T_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1858A-TF-H-PA-T_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1858A-TF-H-PA-T_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1858A-TF-H-PA-T_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1858A-TF-H-PA-T_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1858A-TF-H-PA-T_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1858A-TF-H-PA-T_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1858A-TF-H-PA-T_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        64,752 reads  @     23.4 µs/read;   2.57 M reads/minute\n",
      "Finished in 1.54 s (24 us/read; 2.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,752\n",
      "  Read 1 with adapter:                  64,507 (99.6%)\n",
      "  Read 2 with adapter:                  64,474 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,752 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,536,840 bp\n",
      "  Read 1:     9,771,168 bp\n",
      "  Read 2:     9,765,672 bp\n",
      "Total written (filtered):     17,022,327 bp (87.1%)\n",
      "  Read 1:     8,545,692 bp\n",
      "  Read 2:     8,476,635 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 64507 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t342\t0.0\t1\t38 304\n",
      "19\t63967\t0.0\t1\t60364 3603\n",
      "20\t190\t0.0\t1\t28 162\n",
      "45\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 64474 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t6\t0.0\t1\t4 2\n",
      "19\t572\t0.0\t1\t76 496\n",
      "20\t63726\t0.0\t2\t62343 1383\n",
      "21\t166\t0.0\t2\t34 132\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1858D-T1-H-PA-T_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1858D-T1-H-PA-T_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1858D-T1-H-PA-T_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1858D-T1-H-PA-T_74_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1858D-T1-H-PA-T_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1858D-T1-H-PA-T_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1858D-T1-H-PA-T_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1858D-T1-H-PA-T_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        60,790 reads  @     24.9 µs/read;   2.41 M reads/minute\n",
      "Finished in 1.54 s (25 us/read; 2.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,790\n",
      "  Read 1 with adapter:                  60,502 (99.5%)\n",
      "  Read 2 with adapter:                  60,518 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        60,790 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,331,901 bp\n",
      "  Read 1:     9,168,035 bp\n",
      "  Read 2:     9,163,866 bp\n",
      "Total written (filtered):     15,972,600 bp (87.1%)\n",
      "  Read 1:     8,018,658 bp\n",
      "  Read 2:     7,953,942 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 60502 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t288\t0.0\t1\t26 262\n",
      "19\t60034\t0.0\t1\t55981 4053\n",
      "20\t170\t0.0\t1\t28 142\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 60518 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "18\t6\t0.0\t1\t3 3\n",
      "19\t563\t0.0\t1\t70 493\n",
      "20\t59768\t0.0\t2\t58288 1480\n",
      "21\t176\t0.0\t2\t39 137\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1863I-T1-H-CN-T_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1863I-T1-H-CN-T_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1863I-T1-H-CN-T_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1863I-T1-H-CN-T_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1863I-T1-H-CN-T_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1863I-T1-H-CN-T_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1863I-T1-H-CN-T_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1863I-T1-H-CN-T_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        77,022 reads  @     24.7 µs/read;   2.43 M reads/minute\n",
      "Finished in 1.94 s (25 us/read; 2.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,022\n",
      "  Read 1 with adapter:                  76,694 (99.6%)\n",
      "  Read 2 with adapter:                  76,720 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        77,022 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,054,285 bp\n",
      "  Read 1:    11,533,775 bp\n",
      "  Read 2:    11,520,510 bp\n",
      "Total written (filtered):     20,063,379 bp (87.0%)\n",
      "  Read 1:    10,076,793 bp\n",
      "  Read 2:     9,986,586 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 76694 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t372\t0.0\t1\t52 320\n",
      "19\t76109\t0.0\t1\t71211 4898\n",
      "20\t203\t0.0\t1\t40 163\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 76720 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t12\t0.0\t1\t6 6\n",
      "19\t617\t0.0\t1\t111 506\n",
      "20\t75894\t0.0\t2\t74281 1613\n",
      "21\t188\t0.0\t2\t37 151\n",
      "23\t1\t0.0\t2\t1\n",
      "30\t1\t0.0\t2\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1870C-T1-H-PS-T_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1870C-T1-H-PS-T_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1870C-T1-H-PS-T_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1870C-T1-H-PS-T_76_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1870C-T1-H-PS-T_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-1870C-T1-H-PS-T_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1870C-T1-H-PS-T_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-1870C-T1-H-PS-T_76_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       114,213 reads  @     22.5 µs/read;   2.67 M reads/minute\n",
      "Finished in 2.60 s (23 us/read; 2.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,213\n",
      "  Read 1 with adapter:                 113,825 (99.7%)\n",
      "  Read 2 with adapter:                 113,819 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,406,151 bp\n",
      "  Read 1:    17,218,985 bp\n",
      "  Read 2:    17,187,166 bp\n",
      "Total written (filtered):     29,968,028 bp (87.1%)\n",
      "  Read 1:    15,056,607 bp\n",
      "  Read 2:    14,911,421 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113825 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t4\t0.0\t1\t3 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t464\t0.0\t1\t47 417\n",
      "19\t113070\t0.0\t1\t106666 6404\n",
      "20\t271\t0.0\t1\t66 205\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 113819 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t10\t0.0\t1\t5 5\n",
      "19\t832\t0.0\t1\t145 687\n",
      "20\t112719\t0.0\t2\t110496 2223\n",
      "21\t251\t0.0\t2\t68 183\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-2186A-TF-D-PA-T_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-2186A-TF-D-PA-T_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-2186A-TF-D-PA-T_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-2186A-TF-D-PA-T_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-2186A-TF-D-PA-T_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-2186A-TF-D-PA-T_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-2186A-TF-D-PA-T_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-2186A-TF-D-PA-T_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       161,151 reads  @     18.7 µs/read;   3.20 M reads/minute\n",
      "Finished in 3.05 s (19 us/read; 3.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,151\n",
      "  Read 1 with adapter:                 160,424 (99.5%)\n",
      "  Read 2 with adapter:                 160,440 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       161,151 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,644,562 bp\n",
      "  Read 1:    24,328,874 bp\n",
      "  Read 2:    24,315,688 bp\n",
      "Total written (filtered):     42,389,490 bp (87.1%)\n",
      "  Read 1:    21,281,354 bp\n",
      "  Read 2:    21,108,136 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 160424 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.6\t0\t2\n",
      "12\t4\t0.0\t1\t4\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t11\t0.0\t1\t4 7\n",
      "18\t880\t0.0\t1\t89 791\n",
      "19\t159088\t0.0\t1\t147285 11803\n",
      "20\t431\t0.0\t1\t92 339\n",
      "28\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 160440 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t629.5\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t19\t0.0\t1\t9 10\n",
      "19\t1558\t0.0\t1\t200 1358\n",
      "20\t158398\t0.0\t2\t154564 3834\n",
      "21\t451\t0.0\t2\t91 360\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-2199B-T1-H-SS-T_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-2199B-T1-H-SS-T_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-2199B-T1-H-SS-T_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-2199B-T1-H-SS-T_78_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-2199B-T1-H-SS-T_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-2199B-T1-H-SS-T_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-2199B-T1-H-SS-T_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-2199B-T1-H-SS-T_78_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       156,159 reads  @     19.2 µs/read;   3.13 M reads/minute\n",
      "Finished in 3.02 s (19 us/read; 3.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            156,159\n",
      "  Read 1 with adapter:                 155,273 (99.4%)\n",
      "  Read 2 with adapter:                 155,452 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       156,159 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,086,628 bp\n",
      "  Read 1:    23,576,690 bp\n",
      "  Read 2:    23,509,938 bp\n",
      "Total written (filtered):     41,029,072 bp (87.1%)\n",
      "  Read 1:    20,627,101 bp\n",
      "  Read 2:    20,401,971 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 155273 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.6\t0\t2\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t12\t0.0\t1\t4 8\n",
      "17\t13\t0.0\t1\t7 6\n",
      "18\t906\t0.0\t1\t78 828\n",
      "19\t153874\t0.0\t1\t141056 12818\n",
      "20\t455\t0.0\t1\t92 363\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 155452 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t610.0\t0\t1\n",
      "9\t1\t0.6\t0\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t18\t0.0\t1\t13 5\n",
      "19\t1364\t0.0\t1\t206 1158\n",
      "20\t153669\t0.0\t2\t149983 3686\n",
      "21\t388\t0.0\t2\t90 298\n",
      "32\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-Control-2_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-Control-2_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-Control-2_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-Control-2_79_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-Control-2_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-Control-2_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-Control-2_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-Control-2_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00           248 reads  @   1049.9 µs/read;   0.06 M reads/minute\n",
      "Finished in 0.28 s (1138 us/read; 0.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                248\n",
      "  Read 1 with adapter:                     243 (98.0%)\n",
      "  Read 2 with adapter:                     246 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):           248 (100.0%)\n",
      "\n",
      "Total basepairs processed:        74,625 bp\n",
      "  Read 1:        37,340 bp\n",
      "  Read 2:        37,285 bp\n",
      "Total written (filtered):         65,096 bp (87.2%)\n",
      "  Read 1:        32,726 bp\n",
      "  Read 2:        32,370 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 243 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 3\n",
      "19\t240\t0.0\t1\t208 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 246 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t5\t0.0\t1\t1 4\n",
      "20\t241\t0.0\t2\t236 5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-Mock1_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-Mock1_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-Mock1_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-Mock1_80_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-Mock1_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-Mock1_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-Mock1_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-Mock1_80_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       167,028 reads  @     19.5 µs/read;   3.07 M reads/minute\n",
      "Finished in 3.29 s (20 us/read; 3.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            167,028\n",
      "  Read 1 with adapter:                 166,191 (99.5%)\n",
      "  Read 2 with adapter:                 166,168 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       167,028 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,393,990 bp\n",
      "  Read 1:    25,220,558 bp\n",
      "  Read 2:    25,173,432 bp\n",
      "Total written (filtered):     43,915,048 bp (87.1%)\n",
      "  Read 1:    22,063,632 bp\n",
      "  Read 2:    21,851,416 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 166191 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t2.5\t0\t1\n",
      "9\t4\t0.6\t0\t4\n",
      "10\t2\t0.2\t1\t2\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t5\t0.0\t1\t3 2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t5\t0.0\t1\t1 4\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t1001\t0.0\t1\t94 907\n",
      "19\t164697\t0.0\t1\t152914 11783\n",
      "20\t462\t0.0\t1\t92 370\n",
      "23\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 166168 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t4\t0.0\t1\t3 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t0 5\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t20\t0.0\t1\t15 5\n",
      "19\t1640\t0.0\t1\t209 1431\n",
      "20\t164038\t0.0\t2\t159650 4388\n",
      "21\t451\t0.0\t2\t83 368\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-PCR-CTRL_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-PCR-CTRL_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-PCR-CTRL_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-PCR-CTRL_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-PCR-CTRL_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jqb5oa7z/LibT-PCR-CTRL_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-PCR-CTRL_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-iggp93fi/17374cfe-0265-4ebc-a366-4198826d1b6e/data/LibT-PCR-CTRL_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        51,656 reads  @     25.5 µs/read;   2.35 M reads/minute\n",
      "Finished in 1.35 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,656\n",
      "  Read 1 with adapter:                  51,475 (99.6%)\n",
      "  Read 2 with adapter:                  51,426 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,656 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,412,003 bp\n",
      "  Read 1:     7,707,985 bp\n",
      "  Read 2:     7,704,018 bp\n",
      "Total written (filtered):     13,406,031 bp (87.0%)\n",
      "  Read 1:     6,730,139 bp\n",
      "  Read 2:     6,675,892 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 51475 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t328\t0.0\t1\t27 301\n",
      "19\t50974\t0.0\t1\t48093 2881\n",
      "20\t169\t0.0\t1\t22 147\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 51426 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "18\t3\t0.0\t1\t2 1\n",
      "19\t497\t0.0\t1\t69 428\n",
      "20\t50788\t0.0\t2\t49554 1234\n",
      "21\t134\t0.0\t2\t25 109\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibT.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1226C-T1-H-MC-T_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1226C-T1-H-MC-T_39_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1226C-T1-H-MC-T_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1226C-T1-H-MC-T_39_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1226C-T1-H-MC-T_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1226C-T1-H-MC-T_39_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1226C-T1-H-MC-T_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1226C-T1-H-MC-T_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       124,868 reads  @     20.9 µs/read;   2.88 M reads/minute\n",
      "Finished in 2.66 s (21 us/read; 2.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            124,868\n",
      "  Read 1 with adapter:                 121,485 (97.3%)\n",
      "  Read 2 with adapter:                 122,811 (98.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       124,868 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,571,666 bp\n",
      "  Read 1:    17,815,606 bp\n",
      "  Read 2:    17,756,060 bp\n",
      "Total written (filtered):     30,810,528 bp (86.6%)\n",
      "  Read 1:    15,509,661 bp\n",
      "  Read 2:    15,300,867 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 121485 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t121.9\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t5\t0.0\t1\t3 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t9\t0.0\t1\t5 4\n",
      "18\t2835\t0.0\t1\t55 2780\n",
      "19\t117957\t0.0\t1\t109588 8369\n",
      "20\t668\t0.0\t1\t61 607\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 122811 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1951.1\t0\t11\n",
      "4\t5\t487.8\t0\t5\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t13\t0.0\t1\t7 6\n",
      "19\t983\t0.0\t1\t138 845\n",
      "20\t121495\t0.0\t2\t117663 3832\n",
      "21\t295\t0.0\t2\t75 220\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1295A-T1-H-PA-T_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1295A-T1-H-PA-T_40_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1295A-T1-H-PA-T_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1295A-T1-H-PA-T_40_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1295A-T1-H-PA-T_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1295A-T1-H-PA-T_40_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1295A-T1-H-PA-T_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1295A-T1-H-PA-T_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       108,903 reads  @     21.7 µs/read;   2.77 M reads/minute\n",
      "Finished in 2.41 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,903\n",
      "  Read 1 with adapter:                 106,044 (97.4%)\n",
      "  Read 2 with adapter:                 107,719 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,903 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,796,056 bp\n",
      "  Read 1:    16,404,620 bp\n",
      "  Read 2:    16,391,436 bp\n",
      "Total written (filtered):     28,628,358 bp (87.3%)\n",
      "  Read 1:    14,390,280 bp\n",
      "  Read 2:    14,238,078 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 106044 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t5\t0.0\t1\t1 4\n",
      "18\t1088\t0.0\t1\t44 1044\n",
      "19\t104271\t0.0\t1\t93392 10879\n",
      "20\t669\t0.0\t1\t47 622\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 107719 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t1701.6\t0\t14\n",
      "4\t2\t425.4\t0\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t974\t0.0\t1\t126 848\n",
      "20\t106425\t0.0\t2\t103817 2608\n",
      "21\t287\t0.0\t2\t67 220\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1304E-TF-D-PA-T_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1304E-TF-D-PA-T_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1304E-TF-D-PA-T_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1304E-TF-D-PA-T_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1304E-TF-D-PA-T_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1304E-TF-D-PA-T_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1304E-TF-D-PA-T_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1304E-TF-D-PA-T_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        54,614 reads  @     27.7 µs/read;   2.17 M reads/minute\n",
      "Finished in 1.56 s (29 us/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,614\n",
      "  Read 1 with adapter:                  52,230 (95.6%)\n",
      "  Read 2 with adapter:                  54,014 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,614 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,480,564 bp\n",
      "  Read 1:     8,243,080 bp\n",
      "  Read 2:     8,237,484 bp\n",
      "Total written (filtered):     14,408,719 bp (87.4%)\n",
      "  Read 1:     7,251,054 bp\n",
      "  Read 2:     7,157,665 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 52230 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t477\t0.0\t1\t29 448\n",
      "19\t51582\t0.0\t1\t44217 7365\n",
      "20\t164\t0.0\t1\t22 142\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 54014 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t853.3\t0\t3\n",
      "15\t1\t0.0\t1\t1\n",
      "18\t3\t0.0\t1\t1 2\n",
      "19\t545\t0.0\t1\t83 462\n",
      "20\t53316\t0.0\t2\t52034 1282\n",
      "21\t146\t0.0\t2\t22 124\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1305B-TF-D-OA-T_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1305B-TF-D-OA-T_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1305B-TF-D-OA-T_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1305B-TF-D-OA-T_42_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1305B-TF-D-OA-T_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1305B-TF-D-OA-T_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1305B-TF-D-OA-T_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1305B-TF-D-OA-T_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        63,790 reads  @     25.3 µs/read;   2.37 M reads/minute\n",
      "Finished in 1.66 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,790\n",
      "  Read 1 with adapter:                  61,956 (97.1%)\n",
      "  Read 2 with adapter:                  62,841 (98.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        63,790 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,240,648 bp\n",
      "  Read 1:     9,625,977 bp\n",
      "  Read 2:     9,614,671 bp\n",
      "Total written (filtered):     16,807,897 bp (87.4%)\n",
      "  Read 1:     8,449,530 bp\n",
      "  Read 2:     8,358,367 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 61956 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t249.2\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t1078\t0.0\t1\t34 1044\n",
      "19\t60411\t0.0\t1\t55437 4974\n",
      "20\t450\t0.0\t1\t42 408\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 62841 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t996.7\t0\t6\n",
      "4\t1\t249.2\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t7\t0.0\t1\t3 4\n",
      "19\t482\t0.0\t1\t77 405\n",
      "20\t62197\t0.0\t2\t60399 1798\n",
      "21\t141\t0.0\t2\t36 105\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1312-TF-H-DL-T_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1312-TF-H-DL-T_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1312-TF-H-DL-T_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1312-TF-H-DL-T_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1312-TF-H-DL-T_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1312-TF-H-DL-T_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1312-TF-H-DL-T_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1312-TF-H-DL-T_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        68,953 reads  @     24.4 µs/read;   2.46 M reads/minute\n",
      "Finished in 1.74 s (25 us/read; 2.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,953\n",
      "  Read 1 with adapter:                  67,202 (97.5%)\n",
      "  Read 2 with adapter:                  67,862 (98.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        68,953 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,575,065 bp\n",
      "  Read 1:    10,292,640 bp\n",
      "  Read 2:    10,282,425 bp\n",
      "Total written (filtered):     17,942,182 bp (87.2%)\n",
      "  Read 1:     9,016,273 bp\n",
      "  Read 2:     8,925,909 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 67202 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t3\t0.0\t1\t3\n",
      "15\t4\t0.0\t1\t2 2\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t1119\t0.0\t1\t31 1088\n",
      "19\t65351\t0.0\t1\t60653 4698\n",
      "20\t715\t0.0\t1\t44 671\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 67862 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1077.4\t0\t10\n",
      "4\t1\t269.3\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "18\t7\t0.0\t1\t5 2\n",
      "19\t680\t0.0\t1\t70 610\n",
      "20\t66982\t0.0\t2\t64742 2240\n",
      "21\t178\t0.0\t2\t31 147\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1319H-TF-D-SS-T_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1319H-TF-D-SS-T_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1319H-TF-D-SS-T_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1319H-TF-D-SS-T_44_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1319H-TF-D-SS-T_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1319H-TF-D-SS-T_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1319H-TF-D-SS-T_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1319H-TF-D-SS-T_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        42,338 reads  @     28.5 µs/read;   2.10 M reads/minute\n",
      "Finished in 1.26 s (30 us/read; 2.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,338\n",
      "  Read 1 with adapter:                  41,148 (97.2%)\n",
      "  Read 2 with adapter:                  41,688 (98.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,338 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,736,876 bp\n",
      "  Read 1:     6,374,216 bp\n",
      "  Read 2:     6,362,660 bp\n",
      "Total written (filtered):     11,122,416 bp (87.3%)\n",
      "  Read 1:     5,593,156 bp\n",
      "  Read 2:     5,529,260 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 41148 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t2\t0.0\t1\t1 1\n",
      "18\t863\t0.0\t1\t15 848\n",
      "19\t40172\t0.0\t1\t36764 3408\n",
      "20\t110\t0.0\t1\t23 87\n",
      "26\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 41688 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t661.5\t0\t5\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t4\t0.0\t1\t3 1\n",
      "19\t344\t0.0\t1\t47 297\n",
      "20\t41245\t0.0\t2\t40041 1204\n",
      "21\t88\t0.0\t2\t17 71\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1323-T1-H-DL-T_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1323-T1-H-DL-T_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1323-T1-H-DL-T_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1323-T1-H-DL-T_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1323-T1-H-DL-T_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1323-T1-H-DL-T_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1323-T1-H-DL-T_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1323-T1-H-DL-T_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        47,146 reads  @     27.2 µs/read;   2.21 M reads/minute\n",
      "Finished in 1.37 s (29 us/read; 2.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,146\n",
      "  Read 1 with adapter:                  45,913 (97.4%)\n",
      "  Read 2 with adapter:                  46,471 (98.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        47,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,112,506 bp\n",
      "  Read 1:     7,065,171 bp\n",
      "  Read 2:     7,047,335 bp\n",
      "Total written (filtered):     12,311,807 bp (87.2%)\n",
      "  Read 1:     6,193,530 bp\n",
      "  Read 2:     6,118,277 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 45913 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t880\t0.0\t1\t18 862\n",
      "19\t44829\t0.0\t1\t42081 2748\n",
      "20\t199\t0.0\t1\t16 183\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 46471 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t736.7\t0\t4\n",
      "4\t2\t184.2\t0\t2\n",
      "10\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t342\t0.0\t1\t42 300\n",
      "20\t46007\t0.0\t2\t44714 1293\n",
      "21\t109\t0.0\t2\t18 91\n",
      "22\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1342C-TF-D-OA-T_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1342C-TF-D-OA-T_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1342C-TF-D-OA-T_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1342C-TF-D-OA-T_46_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1342C-TF-D-OA-T_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1342C-TF-D-OA-T_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1342C-TF-D-OA-T_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1342C-TF-D-OA-T_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        93,374 reads  @     22.7 µs/read;   2.64 M reads/minute\n",
      "Finished in 2.17 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,374\n",
      "  Read 1 with adapter:                  90,873 (97.3%)\n",
      "  Read 2 with adapter:                  91,802 (98.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        93,374 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,136,198 bp\n",
      "  Read 1:    14,075,159 bp\n",
      "  Read 2:    14,061,039 bp\n",
      "Total written (filtered):     24,575,868 bp (87.3%)\n",
      "  Read 1:    12,350,077 bp\n",
      "  Read 2:    12,225,791 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90873 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t7\t0.0\t1\t5 2\n",
      "18\t1956\t0.0\t1\t37 1919\n",
      "19\t88403\t0.0\t1\t82207 6196\n",
      "20\t497\t0.0\t1\t47 450\n",
      "39\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 91802 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1459.0\t0\t8\n",
      "4\t5\t364.7\t0\t5\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t11\t0.0\t1\t7 4\n",
      "19\t735\t0.0\t1\t94 641\n",
      "20\t90796\t0.0\t2\t87961 2835\n",
      "21\t235\t0.0\t2\t49 186\n",
      "30\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1345A-TF-H-CN-T_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1345A-TF-H-CN-T_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1345A-TF-H-CN-T_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1345A-TF-H-CN-T_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1345A-TF-H-CN-T_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1345A-TF-H-CN-T_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1345A-TF-H-CN-T_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1345A-TF-H-CN-T_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       179,484 reads  @     20.4 µs/read;   2.94 M reads/minute\n",
      "Finished in 3.71 s (21 us/read; 2.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            179,484\n",
      "  Read 1 with adapter:                 174,766 (97.4%)\n",
      "  Read 2 with adapter:                 176,319 (98.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       179,484 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,177,118 bp\n",
      "  Read 1:    27,096,719 bp\n",
      "  Read 2:    27,080,399 bp\n",
      "Total written (filtered):     47,334,466 bp (87.4%)\n",
      "  Read 1:    23,779,143 bp\n",
      "  Read 2:    23,555,323 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 174766 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t701.1\t0\t1\n",
      "5\t1\t175.3\t0\t1\n",
      "9\t1\t0.7\t0\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t4\t0.0\t1\t1 3\n",
      "15\t2\t0.0\t1\t0 2\n",
      "17\t10\t0.0\t1\t8 2\n",
      "18\t3296\t0.0\t1\t76 3220\n",
      "19\t171017\t0.0\t1\t157927 13090\n",
      "20\t429\t0.0\t1\t70 359\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 176319 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t2804.4\t0\t20\n",
      "4\t3\t701.1\t0\t3\n",
      "10\t1\t0.2\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t6\t0.0\t1\t4 2\n",
      "19\t1243\t0.0\t1\t164 1079\n",
      "20\t174656\t0.0\t2\t169365 5291\n",
      "21\t382\t0.0\t2\t98 284\n",
      "22\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1347A-TF-D-PS-T_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1347A-TF-D-PS-T_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1347A-TF-D-PS-T_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1347A-TF-D-PS-T_48_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1347A-TF-D-PS-T_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1347A-TF-D-PS-T_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1347A-TF-D-PS-T_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1347A-TF-D-PS-T_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        74,934 reads  @     22.7 µs/read;   2.64 M reads/minute\n",
      "Finished in 1.75 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,934\n",
      "  Read 1 with adapter:                  72,725 (97.1%)\n",
      "  Read 2 with adapter:                  73,592 (98.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        74,934 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,127,693 bp\n",
      "  Read 1:    10,592,334 bp\n",
      "  Read 2:    10,535,359 bp\n",
      "Total written (filtered):     18,275,267 bp (86.5%)\n",
      "  Read 1:     9,211,236 bp\n",
      "  Read 2:     9,064,031 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 72725 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t4\t0.0\t1\t3 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t959\t0.0\t1\t47 912\n",
      "19\t71434\t0.0\t1\t65925 5509\n",
      "20\t318\t0.0\t1\t35 283\n",
      "23\t1\t0.0\t1\t1\n",
      "30\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 73592 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1170.8\t0\t5\n",
      "4\t1\t292.7\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t7\t0.0\t1\t1 6\n",
      "19\t539\t0.0\t1\t74 465\n",
      "20\t72880\t0.0\t2\t70716 2164\n",
      "21\t154\t0.0\t2\t30 124\n",
      "22\t1\t0.0\t2\t0 1\n",
      "29\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1350D-T1-H-SS-T_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1350D-T1-H-SS-T_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1350D-T1-H-SS-T_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1350D-T1-H-SS-T_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1350D-T1-H-SS-T_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1350D-T1-H-SS-T_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1350D-T1-H-SS-T_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1350D-T1-H-SS-T_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       139,450 reads  @     21.8 µs/read;   2.75 M reads/minute\n",
      "Finished in 3.09 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            139,450\n",
      "  Read 1 with adapter:                 136,000 (97.5%)\n",
      "  Read 2 with adapter:                 136,539 (97.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       139,450 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,853,109 bp\n",
      "  Read 1:    20,939,535 bp\n",
      "  Read 2:    20,913,574 bp\n",
      "Total written (filtered):     36,542,680 bp (87.3%)\n",
      "  Read 1:    18,358,408 bp\n",
      "  Read 2:    18,184,272 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 136000 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t3079\t0.0\t1\t52 3027\n",
      "19\t132596\t0.0\t1\t123464 9132\n",
      "20\t304\t0.0\t1\t56 248\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 136539 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t2178.9\t0\t26\n",
      "4\t4\t544.7\t0\t4\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t10\t0.0\t1\t5 5\n",
      "19\t1225\t0.0\t1\t148 1077\n",
      "20\t134939\t0.0\t2\t130277 4662\n",
      "21\t322\t0.0\t2\t62 260\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1370A-TF-D-CN-T_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1370A-TF-D-CN-T_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1370A-TF-D-CN-T_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1370A-TF-D-CN-T_50_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1370A-TF-D-CN-T_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1370A-TF-D-CN-T_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1370A-TF-D-CN-T_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1370A-TF-D-CN-T_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        55,059 reads  @     27.8 µs/read;   2.16 M reads/minute\n",
      "Finished in 1.59 s (29 us/read; 2.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,059\n",
      "  Read 1 with adapter:                  53,465 (97.1%)\n",
      "  Read 2 with adapter:                  54,422 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,059 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,608,588 bp\n",
      "  Read 1:     8,307,606 bp\n",
      "  Read 2:     8,300,982 bp\n",
      "Total written (filtered):     14,505,467 bp (87.3%)\n",
      "  Read 1:     7,292,527 bp\n",
      "  Read 2:     7,212,940 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 53465 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t847\t0.0\t1\t19 828\n",
      "19\t52499\t0.0\t1\t47695 4804\n",
      "20\t113\t0.0\t1\t19 94\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 54422 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t860.3\t0\t6\n",
      "9\t1\t0.2\t0\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "18\t5\t0.0\t1\t4 1\n",
      "19\t367\t0.0\t1\t48 319\n",
      "20\t53924\t0.0\t2\t52395 1529\n",
      "21\t116\t0.0\t2\t15 101\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1370E-T1-H-CN-T_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1370E-T1-H-CN-T_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1370E-T1-H-CN-T_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1370E-T1-H-CN-T_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1370E-T1-H-CN-T_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1370E-T1-H-CN-T_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1370E-T1-H-CN-T_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1370E-T1-H-CN-T_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        86,042 reads  @     23.7 µs/read;   2.54 M reads/minute\n",
      "Finished in 2.09 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,042\n",
      "  Read 1 with adapter:                  83,844 (97.4%)\n",
      "  Read 2 with adapter:                  84,617 (98.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        86,042 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,744,677 bp\n",
      "  Read 1:    12,885,594 bp\n",
      "  Read 2:    12,859,083 bp\n",
      "Total written (filtered):     22,461,771 bp (87.2%)\n",
      "  Read 1:    11,294,364 bp\n",
      "  Read 2:    11,167,407 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 83844 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.3\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t1947\t0.0\t1\t40 1907\n",
      "19\t81695\t0.0\t1\t76555 5140\n",
      "20\t191\t0.0\t1\t33 158\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 84617 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1344.4\t0\t11\n",
      "4\t2\t336.1\t0\t2\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t9\t0.0\t1\t5 4\n",
      "19\t559\t0.0\t1\t82 477\n",
      "20\t83845\t0.0\t2\t81171 2674\n",
      "21\t183\t0.0\t2\t35 148\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1654A-TF-D-SS-T_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1654A-TF-D-SS-T_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1654A-TF-D-SS-T_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1654A-TF-D-SS-T_52_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1654A-TF-D-SS-T_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1654A-TF-D-SS-T_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1654A-TF-D-SS-T_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1654A-TF-D-SS-T_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        35,486 reads  @     32.6 µs/read;   1.84 M reads/minute\n",
      "Finished in 1.21 s (34 us/read; 1.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,486\n",
      "  Read 1 with adapter:                  34,689 (97.8%)\n",
      "  Read 2 with adapter:                  35,064 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,607,321 bp\n",
      "  Read 1:     5,310,326 bp\n",
      "  Read 2:     5,296,995 bp\n",
      "Total written (filtered):      9,247,895 bp (87.2%)\n",
      "  Read 1:     4,651,909 bp\n",
      "  Read 2:     4,595,986 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 34689 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t752\t0.0\t1\t15 737\n",
      "19\t33828\t0.0\t1\t31889 1939\n",
      "20\t103\t0.0\t1\t13 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 35064 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t554.5\t0\t5\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t4\t0.0\t1\t1 3\n",
      "19\t259\t0.0\t1\t35 224\n",
      "20\t34710\t0.0\t2\t33819 891\n",
      "21\t85\t0.0\t2\t16 69\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1661A-TF-D-CN-T_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1661A-TF-D-CN-T_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1661A-TF-D-CN-T_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1661A-TF-D-CN-T_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1661A-TF-D-CN-T_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1661A-TF-D-CN-T_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1661A-TF-D-CN-T_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1661A-TF-D-CN-T_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       126,884 reads  @     22.0 µs/read;   2.73 M reads/minute\n",
      "Finished in 2.85 s (22 us/read; 2.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            126,884\n",
      "  Read 1 with adapter:                 121,435 (95.7%)\n",
      "  Read 2 with adapter:                 125,686 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       126,884 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,240,570 bp\n",
      "  Read 1:    19,127,849 bp\n",
      "  Read 2:    19,112,721 bp\n",
      "Total written (filtered):     33,421,322 bp (87.4%)\n",
      "  Read 1:    16,821,473 bp\n",
      "  Read 2:    16,599,849 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 121435 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.5\t0\t3\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t11\t0.0\t1\t5 6\n",
      "18\t1092\t0.0\t1\t53 1039\n",
      "19\t120038\t0.0\t1\t102441 17597\n",
      "20\t279\t0.0\t1\t52 227\n",
      "26\t1\t0.0\t1\t1\n",
      "43\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 125686 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1982.6\t0\t10\n",
      "4\t4\t495.6\t0\t4\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t5\t0.0\t1\t2 3\n",
      "19\t866\t0.0\t1\t147 719\n",
      "20\t124492\t0.0\t2\t121357 3135\n",
      "21\t301\t0.0\t2\t81 220\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1661C-T1-H-CN-T_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1661C-T1-H-CN-T_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1661C-T1-H-CN-T_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1661C-T1-H-CN-T_54_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1661C-T1-H-CN-T_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1661C-T1-H-CN-T_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1661C-T1-H-CN-T_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1661C-T1-H-CN-T_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        48,732 reads  @     26.2 µs/read;   2.29 M reads/minute\n",
      "Finished in 1.34 s (28 us/read; 2.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,732\n",
      "  Read 1 with adapter:                  47,437 (97.3%)\n",
      "  Read 2 with adapter:                  48,044 (98.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,732 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,707,797 bp\n",
      "  Read 1:     7,357,127 bp\n",
      "  Read 2:     7,350,670 bp\n",
      "Total written (filtered):     12,846,797 bp (87.3%)\n",
      "  Read 1:     6,456,729 bp\n",
      "  Read 2:     6,390,068 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 47437 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t986\t0.0\t1\t21 965\n",
      "19\t46355\t0.0\t1\t42729 3626\n",
      "20\t93\t0.0\t1\t17 76\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 48044 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t761.4\t0\t3\n",
      "18\t6\t0.0\t1\t5 1\n",
      "19\t333\t0.0\t1\t44 289\n",
      "20\t47593\t0.0\t2\t46119 1474\n",
      "21\t106\t0.0\t2\t32 74\n",
      "22\t1\t0.0\t2\t1\n",
      "25\t2\t0.0\t2\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1666E-T1-H-PA-T_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1666E-T1-H-PA-T_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1666E-T1-H-PA-T_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1666E-T1-H-PA-T_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1666E-T1-H-PA-T_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1666E-T1-H-PA-T_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1666E-T1-H-PA-T_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1666E-T1-H-PA-T_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       112,925 reads  @     22.1 µs/read;   2.72 M reads/minute\n",
      "Finished in 2.54 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,925\n",
      "  Read 1 with adapter:                 110,016 (97.4%)\n",
      "  Read 2 with adapter:                 111,020 (98.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       112,925 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,074,721 bp\n",
      "  Read 1:    17,045,478 bp\n",
      "  Read 2:    17,029,243 bp\n",
      "Total written (filtered):     29,766,832 bp (87.4%)\n",
      "  Read 1:    14,956,980 bp\n",
      "  Read 2:    14,809,852 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110016 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t110.3\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t3\t0.0\t1\t3\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t7\t0.0\t1\t4 3\n",
      "17\t10\t0.0\t1\t5 5\n",
      "18\t2104\t0.0\t1\t51 2053\n",
      "19\t107482\t0.0\t1\t98540 8942\n",
      "20\t405\t0.0\t1\t43 362\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 111020 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1764.5\t0\t11\n",
      "4\t2\t441.1\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t11\t0.0\t1\t4 7\n",
      "19\t1028\t0.0\t1\t126 902\n",
      "20\t109667\t0.0\t2\t106080 3587\n",
      "21\t293\t0.0\t2\t62 231\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1672D-TF-D-PS-T_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1672D-TF-D-PS-T_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1672D-TF-D-PS-T_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1672D-TF-D-PS-T_56_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1672D-TF-D-PS-T_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1672D-TF-D-PS-T_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1672D-TF-D-PS-T_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1672D-TF-D-PS-T_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        65,512 reads  @     25.2 µs/read;   2.38 M reads/minute\n",
      "Finished in 1.70 s (26 us/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,512\n",
      "  Read 1 with adapter:                  63,755 (97.3%)\n",
      "  Read 2 with adapter:                  64,553 (98.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        65,512 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,765,875 bp\n",
      "  Read 1:     9,891,534 bp\n",
      "  Read 2:     9,874,341 bp\n",
      "Total written (filtered):     17,265,135 bp (87.3%)\n",
      "  Read 1:     8,681,384 bp\n",
      "  Read 2:     8,583,751 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 63755 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t64.0\t0\t1\n",
      "9\t1\t0.2\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t1295\t0.0\t1\t31 1264\n",
      "19\t62283\t0.0\t1\t57618 4665\n",
      "20\t164\t0.0\t1\t34 130\n",
      "31\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 64553 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1023.6\t0\t7\n",
      "10\t2\t0.1\t1\t2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t433\t0.0\t1\t48 385\n",
      "20\t63969\t0.0\t2\t62030 1939\n",
      "21\t134\t0.0\t2\t38 96\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1681A-TF-D-PA-T_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1681A-TF-D-PA-T_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1681A-TF-D-PA-T_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1681A-TF-D-PA-T_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1681A-TF-D-PA-T_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1681A-TF-D-PA-T_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1681A-TF-D-PA-T_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1681A-TF-D-PA-T_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        75,076 reads  @     26.1 µs/read;   2.30 M reads/minute\n",
      "Finished in 2.01 s (27 us/read; 2.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,076\n",
      "  Read 1 with adapter:                  73,185 (97.5%)\n",
      "  Read 2 with adapter:                  73,686 (98.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,076 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,657,756 bp\n",
      "  Read 1:    11,334,087 bp\n",
      "  Read 2:    11,323,669 bp\n",
      "Total written (filtered):     19,795,444 bp (87.4%)\n",
      "  Read 1:     9,944,820 bp\n",
      "  Read 2:     9,850,624 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 73185 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t1476\t0.0\t1\t30 1446\n",
      "19\t71406\t0.0\t1\t66205 5201\n",
      "20\t288\t0.0\t1\t37 251\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 73686 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1173.1\t0\t11\n",
      "4\t2\t293.3\t0\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t4\t0.0\t1\t2 2\n",
      "19\t632\t0.0\t1\t76 556\n",
      "20\t72836\t0.0\t2\t70447 2389\n",
      "21\t197\t0.0\t2\t41 156\n",
      "22\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1681B-TF-H-PA-T_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1681B-TF-H-PA-T_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1681B-TF-H-PA-T_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1681B-TF-H-PA-T_58_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1681B-TF-H-PA-T_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1681B-TF-H-PA-T_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1681B-TF-H-PA-T_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1681B-TF-H-PA-T_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        57,088 reads  @     25.5 µs/read;   2.35 M reads/minute\n",
      "Finished in 1.51 s (26 us/read; 2.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,088\n",
      "  Read 1 with adapter:                  55,440 (97.1%)\n",
      "  Read 2 with adapter:                  56,194 (98.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        57,088 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,221,869 bp\n",
      "  Read 1:     8,615,127 bp\n",
      "  Read 2:     8,606,742 bp\n",
      "Total written (filtered):     15,045,822 bp (87.4%)\n",
      "  Read 1:     7,562,419 bp\n",
      "  Read 2:     7,483,403 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55440 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.2\t0\t3\n",
      "13\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t813\t0.0\t1\t31 782\n",
      "19\t54433\t0.0\t1\t48957 5476\n",
      "20\t180\t0.0\t1\t19 161\n",
      "22\t1\t0.0\t1\t1\n",
      "34\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 56194 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t892.0\t0\t6\n",
      "11\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t6\t0.0\t1\t4 2\n",
      "19\t550\t0.0\t1\t73 477\n",
      "20\t55490\t0.0\t2\t53811 1679\n",
      "21\t139\t0.0\t2\t27 112\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1692B-TF-H-CN-T_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1692B-TF-H-CN-T_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1692B-TF-H-CN-T_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1692B-TF-H-CN-T_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1692B-TF-H-CN-T_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1692B-TF-H-CN-T_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1692B-TF-H-CN-T_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1692B-TF-H-CN-T_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       137,501 reads  @     22.6 µs/read;   2.66 M reads/minute\n",
      "Finished in 3.15 s (23 us/read; 2.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            137,501\n",
      "  Read 1 with adapter:                 133,840 (97.3%)\n",
      "  Read 2 with adapter:                 134,887 (98.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       137,501 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,900,663 bp\n",
      "  Read 1:    20,464,245 bp\n",
      "  Read 2:    20,436,418 bp\n",
      "Total written (filtered):     35,663,799 bp (87.2%)\n",
      "  Read 1:    17,924,125 bp\n",
      "  Read 2:    17,739,674 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 133840 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t537.1\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t8\t0.0\t1\t6 2\n",
      "18\t3061\t0.0\t1\t70 2991\n",
      "19\t130457\t0.0\t1\t121775 8682\n",
      "20\t299\t0.0\t1\t57 242\n",
      "34\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 134887 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t2148.5\t0\t16\n",
      "4\t1\t537.1\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t7\t0.0\t1\t2 5\n",
      "19\t944\t0.0\t1\t99 845\n",
      "20\t133625\t0.0\t2\t129084 4541\n",
      "21\t286\t0.0\t2\t56 230\n",
      "23\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1695-T1-H-DL-T_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1695-T1-H-DL-T_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1695-T1-H-DL-T_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1695-T1-H-DL-T_60_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1695-T1-H-DL-T_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1695-T1-H-DL-T_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1695-T1-H-DL-T_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1695-T1-H-DL-T_60_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       106,075 reads  @     22.0 µs/read;   2.72 M reads/minute\n",
      "Finished in 2.39 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,075\n",
      "  Read 1 with adapter:                 103,518 (97.6%)\n",
      "  Read 2 with adapter:                 104,395 (98.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,075 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,649,587 bp\n",
      "  Read 1:    15,355,561 bp\n",
      "  Read 2:    15,294,026 bp\n",
      "Total written (filtered):     26,597,488 bp (86.8%)\n",
      "  Read 1:    13,390,572 bp\n",
      "  Read 2:    13,206,916 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103518 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t6\t0.0\t1\t5 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t6\t0.0\t1\t3 3\n",
      "17\t10\t0.0\t1\t4 6\n",
      "18\t2250\t0.0\t1\t55 2195\n",
      "19\t100822\t0.0\t1\t94908 5914\n",
      "20\t419\t0.0\t1\t48 371\n",
      "23\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t1\t1\n",
      "57\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 104395 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1657.4\t0\t12\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t746\t0.0\t1\t117 629\n",
      "20\t103387\t0.0\t2\t100281 3106\n",
      "21\t232\t0.0\t2\t47 185\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1695-TF-H-DL-T_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1695-TF-H-DL-T_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1695-TF-H-DL-T_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1695-TF-H-DL-T_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1695-TF-H-DL-T_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1695-TF-H-DL-T_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1695-TF-H-DL-T_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1695-TF-H-DL-T_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        51,780 reads  @     28.0 µs/read;   2.14 M reads/minute\n",
      "Finished in 1.52 s (29 us/read; 2.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,780\n",
      "  Read 1 with adapter:                  50,169 (96.9%)\n",
      "  Read 2 with adapter:                  50,970 (98.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,780 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,562,362 bp\n",
      "  Read 1:     7,792,813 bp\n",
      "  Read 2:     7,769,549 bp\n",
      "Total written (filtered):     13,591,213 bp (87.3%)\n",
      "  Read 1:     6,840,670 bp\n",
      "  Read 2:     6,750,543 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 50169 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t50.6\t0\t1\n",
      "8\t1\t0.8\t0\t1\n",
      "9\t2\t0.2\t0\t2\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t1197\t0.0\t1\t13 1184\n",
      "19\t48761\t0.0\t1\t45600 3161\n",
      "20\t201\t0.0\t1\t17 184\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 50970 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t809.1\t0\t5\n",
      "4\t2\t202.3\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t7\t0.0\t1\t4 3\n",
      "19\t369\t0.0\t1\t62 307\n",
      "20\t50453\t0.0\t2\t49062 1391\n",
      "21\t130\t0.0\t2\t20 110\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826C-TF-D-SS-T_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826C-TF-D-SS-T_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826C-TF-D-SS-T_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826C-TF-D-SS-T_62_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826C-TF-D-SS-T_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826C-TF-D-SS-T_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826C-TF-D-SS-T_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826C-TF-D-SS-T_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        64,505 reads  @     23.7 µs/read;   2.53 M reads/minute\n",
      "Finished in 1.57 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,505\n",
      "  Read 1 with adapter:                  62,763 (97.3%)\n",
      "  Read 2 with adapter:                  63,233 (98.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,505 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,412,768 bp\n",
      "  Read 1:     9,714,738 bp\n",
      "  Read 2:     9,698,030 bp\n",
      "Total written (filtered):     16,957,391 bp (87.4%)\n",
      "  Read 1:     8,523,509 bp\n",
      "  Read 2:     8,433,882 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 62763 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t1439\t0.0\t1\t26 1413\n",
      "19\t61129\t0.0\t1\t56684 4445\n",
      "20\t191\t0.0\t1\t25 166\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 63233 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1007.9\t0\t4\n",
      "4\t1\t252.0\t0\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "14\t5\t0.0\t1\t0 5\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t8\t0.0\t1\t4 4\n",
      "19\t508\t0.0\t1\t76 432\n",
      "20\t62535\t0.0\t2\t60493 2042\n",
      "21\t164\t0.0\t2\t29 135\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826D-T1-H-SS-T_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826D-T1-H-SS-T_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826D-T1-H-SS-T_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826D-T1-H-SS-T_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826D-T1-H-SS-T_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826D-T1-H-SS-T_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826D-T1-H-SS-T_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826D-T1-H-SS-T_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02       114,565 reads  @     21.0 µs/read;   2.86 M reads/minute\n",
      "Finished in 2.45 s (21 us/read; 2.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,565\n",
      "  Read 1 with adapter:                 111,417 (97.3%)\n",
      "  Read 2 with adapter:                 112,564 (98.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,565 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,576,873 bp\n",
      "  Read 1:    17,295,220 bp\n",
      "  Read 2:    17,281,653 bp\n",
      "Total written (filtered):     30,211,683 bp (87.4%)\n",
      "  Read 1:    15,180,385 bp\n",
      "  Read 2:    15,031,298 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 111417 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t3\t0.0\t1\t2 1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t7\t0.0\t1\t5 2\n",
      "18\t2289\t0.0\t1\t53 2236\n",
      "19\t108856\t0.0\t1\t100291 8565\n",
      "20\t255\t0.0\t1\t40 215\n",
      "34\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 112564 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t1790.1\t0\t13\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "18\t6\t0.0\t1\t1 5\n",
      "19\t982\t0.0\t1\t133 849\n",
      "20\t111262\t0.0\t2\t107815 3447\n",
      "21\t296\t0.0\t2\t64 232\n",
      "32\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826E-TF-H-SS-T_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826E-TF-H-SS-T_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826E-TF-H-SS-T_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826E-TF-H-SS-T_64_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826E-TF-H-SS-T_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1826E-TF-H-SS-T_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826E-TF-H-SS-T_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1826E-TF-H-SS-T_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        65,315 reads  @     25.0 µs/read;   2.40 M reads/minute\n",
      "Finished in 1.68 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,315\n",
      "  Read 1 with adapter:                  63,467 (97.2%)\n",
      "  Read 2 with adapter:                  64,451 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        65,315 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,690,971 bp\n",
      "  Read 1:     9,849,812 bp\n",
      "  Read 2:     9,841,159 bp\n",
      "Total written (filtered):     17,197,656 bp (87.3%)\n",
      "  Read 1:     8,644,895 bp\n",
      "  Read 2:     8,552,761 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 63467 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t9\t0.0\t1\t6 3\n",
      "18\t1068\t0.0\t1\t34 1034\n",
      "19\t62203\t0.0\t1\t55937 6266\n",
      "20\t176\t0.0\t1\t31 145\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 64451 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1020.5\t0\t9\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "18\t2\t0.0\t1\t1 1\n",
      "19\t586\t0.0\t1\t75 511\n",
      "20\t63695\t0.0\t2\t61957 1738\n",
      "21\t155\t0.0\t2\t32 123\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1831A-TF-D-OA-T_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1831A-TF-D-OA-T_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1831A-TF-D-OA-T_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1831A-TF-D-OA-T_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1831A-TF-D-OA-T_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1831A-TF-D-OA-T_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1831A-TF-D-OA-T_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1831A-TF-D-OA-T_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        52,610 reads  @     25.3 µs/read;   2.37 M reads/minute\n",
      "Finished in 1.38 s (26 us/read; 2.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,610\n",
      "  Read 1 with adapter:                  51,035 (97.0%)\n",
      "  Read 2 with adapter:                  51,911 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,610 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,844,782 bp\n",
      "  Read 1:     7,929,894 bp\n",
      "  Read 2:     7,914,888 bp\n",
      "Total written (filtered):     13,838,094 bp (87.3%)\n",
      "  Read 1:     6,961,052 bp\n",
      "  Read 2:     6,877,042 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 51035 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t205.5\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t912\t0.0\t1\t22 890\n",
      "19\t49982\t0.0\t1\t45707 4275\n",
      "20\t133\t0.0\t1\t23 110\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 51911 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t822.0\t0\t5\n",
      "4\t1\t205.5\t0\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "18\t9\t0.0\t1\t5 4\n",
      "19\t355\t0.0\t1\t38 317\n",
      "20\t51436\t0.0\t2\t49974 1462\n",
      "21\t103\t0.0\t2\t15 88\n",
      "22\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1833B-TF-D-CN-T_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1833B-TF-D-CN-T_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1833B-TF-D-CN-T_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1833B-TF-D-CN-T_66_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1833B-TF-D-CN-T_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1833B-TF-D-CN-T_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1833B-TF-D-CN-T_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1833B-TF-D-CN-T_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       113,722 reads  @     22.3 µs/read;   2.69 M reads/minute\n",
      "Finished in 2.59 s (23 us/read; 2.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,722\n",
      "  Read 1 with adapter:                 110,124 (96.8%)\n",
      "  Read 2 with adapter:                 111,681 (98.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,722 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,991,528 bp\n",
      "  Read 1:    17,016,805 bp\n",
      "  Read 2:    16,974,723 bp\n",
      "Total written (filtered):     29,668,385 bp (87.3%)\n",
      "  Read 1:    14,926,431 bp\n",
      "  Read 2:    14,741,954 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110124 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t444.2\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t2267\t0.0\t1\t50 2217\n",
      "19\t107501\t0.0\t1\t99703 7798\n",
      "20\t340\t0.0\t1\t62 278\n",
      "29\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 111681 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1776.9\t0\t10\n",
      "4\t4\t444.2\t0\t4\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t9\t0.0\t1\t7 2\n",
      "19\t769\t0.0\t1\t116 653\n",
      "20\t110651\t0.0\t2\t107189 3462\n",
      "21\t225\t0.0\t2\t45 180\n",
      "22\t1\t0.0\t2\t0 1\n",
      "28\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1848A-TF-D-CN-T_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1848A-TF-D-CN-T_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1848A-TF-D-CN-T_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1848A-TF-D-CN-T_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1848A-TF-D-CN-T_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1848A-TF-D-CN-T_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1848A-TF-D-CN-T_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1848A-TF-D-CN-T_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        64,393 reads  @     22.4 µs/read;   2.67 M reads/minute\n",
      "Finished in 1.50 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,393\n",
      "  Read 1 with adapter:                  62,712 (97.4%)\n",
      "  Read 2 with adapter:                  63,463 (98.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,393 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,598,109 bp\n",
      "  Read 1:     9,300,864 bp\n",
      "  Read 2:     9,297,245 bp\n",
      "Total written (filtered):     16,139,064 bp (86.8%)\n",
      "  Read 1:     8,110,427 bp\n",
      "  Read 2:     8,028,637 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 62712 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t6\t0.0\t1\t5 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t1194\t0.0\t1\t34 1160\n",
      "19\t61322\t0.0\t1\t56026 5296\n",
      "20\t177\t0.0\t1\t19 158\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 63463 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1006.1\t0\t15\n",
      "5\t1\t62.9\t0\t1\n",
      "10\t2\t0.1\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t9\t0.0\t1\t9\n",
      "19\t470\t0.0\t1\t85 385\n",
      "20\t62829\t0.0\t2\t61173 1656\n",
      "21\t135\t0.0\t2\t33 102\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1863-TF-D-CN-T_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1863-TF-D-CN-T_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1863-TF-D-CN-T_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1863-TF-D-CN-T_68_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1863-TF-D-CN-T_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1863-TF-D-CN-T_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1863-TF-D-CN-T_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1863-TF-D-CN-T_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        90,028 reads  @     20.5 µs/read;   2.92 M reads/minute\n",
      "Finished in 1.90 s (21 us/read; 2.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,028\n",
      "  Read 1 with adapter:                  87,874 (97.6%)\n",
      "  Read 2 with adapter:                  88,565 (98.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        90,028 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,100,178 bp\n",
      "  Read 1:    13,556,228 bp\n",
      "  Read 2:    13,543,950 bp\n",
      "Total written (filtered):     23,661,576 bp (87.3%)\n",
      "  Read 1:    11,888,300 bp\n",
      "  Read 2:    11,773,276 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 87874 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t1843\t0.0\t1\t37 1806\n",
      "19\t85829\t0.0\t1\t80469 5360\n",
      "20\t193\t0.0\t1\t38 155\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 88565 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1406.7\t0\t10\n",
      "4\t1\t351.7\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t6\t0.0\t1\t0 6\n",
      "19\t585\t0.0\t1\t79 506\n",
      "20\t87766\t0.0\t2\t85093 2673\n",
      "21\t190\t0.0\t2\t41 149\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1864C-T1-H-PS-T_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1864C-T1-H-PS-T_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1864C-T1-H-PS-T_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1864C-T1-H-PS-T_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1864C-T1-H-PS-T_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1864C-T1-H-PS-T_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1864C-T1-H-PS-T_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1864C-T1-H-PS-T_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        41,434 reads  @     28.7 µs/read;   2.09 M reads/minute\n",
      "Finished in 1.25 s (30 us/read; 1.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,434\n",
      "  Read 1 with adapter:                  40,508 (97.8%)\n",
      "  Read 2 with adapter:                  40,867 (98.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,482,988 bp\n",
      "  Read 1:     6,249,105 bp\n",
      "  Read 2:     6,233,883 bp\n",
      "Total written (filtered):     10,897,008 bp (87.3%)\n",
      "  Read 1:     5,480,186 bp\n",
      "  Read 2:     5,416,822 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 40508 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t860\t0.0\t1\t12 848\n",
      "19\t39496\t0.0\t1\t37021 2475\n",
      "20\t146\t0.0\t1\t26 120\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 40867 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t647.4\t0\t4\n",
      "4\t1\t161.9\t0\t1\n",
      "10\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "18\t1\t0.0\t1\t0 1\n",
      "19\t248\t0.0\t1\t30 218\n",
      "20\t40523\t0.0\t2\t39324 1199\n",
      "21\t86\t0.0\t2\t13 73\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1870D-TF-D-PS-T_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1870D-TF-D-PS-T_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1870D-TF-D-PS-T_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1870D-TF-D-PS-T_70_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1870D-TF-D-PS-T_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1870D-TF-D-PS-T_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1870D-TF-D-PS-T_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1870D-TF-D-PS-T_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       101,820 reads  @     24.7 µs/read;   2.43 M reads/minute\n",
      "Finished in 2.57 s (25 us/read; 2.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,820\n",
      "  Read 1 with adapter:                  99,312 (97.5%)\n",
      "  Read 2 with adapter:                 100,658 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       101,820 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,733,337 bp\n",
      "  Read 1:    15,373,585 bp\n",
      "  Read 2:    15,359,752 bp\n",
      "Total written (filtered):     26,835,713 bp (87.3%)\n",
      "  Read 1:    13,488,370 bp\n",
      "  Read 2:    13,347,343 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99312 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t397.7\t0\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t1892\t0.0\t1\t45 1847\n",
      "19\t97219\t0.0\t1\t89352 7867\n",
      "20\t189\t0.0\t1\t39 150\n",
      "36\t1\t0.0\t1\t1\n",
      "42\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 100658 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1590.9\t0\t9\n",
      "4\t8\t397.7\t0\t8\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t8\t0.0\t1\t6 2\n",
      "19\t630\t0.0\t1\t76 554\n",
      "20\t99802\t0.0\t2\t97209 2593\n",
      "21\t197\t0.0\t2\t37 160\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1FLAT-DD-PS-T_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1FLAT-DD-PS-T_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1FLAT-DD-PS-T_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1FLAT-DD-PS-T_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1FLAT-DD-PS-T_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-1FLAT-DD-PS-T_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1FLAT-DD-PS-T_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-1FLAT-DD-PS-T_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        75,097 reads  @     25.1 µs/read;   2.39 M reads/minute\n",
      "Finished in 1.93 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,097\n",
      "  Read 1 with adapter:                  73,607 (98.0%)\n",
      "  Read 2 with adapter:                  73,969 (98.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,097 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,657,065 bp\n",
      "  Read 1:    11,336,343 bp\n",
      "  Read 2:    11,320,722 bp\n",
      "Total written (filtered):     19,781,091 bp (87.3%)\n",
      "  Read 1:     9,939,159 bp\n",
      "  Read 2:     9,841,932 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 73607 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t5 1\n",
      "18\t1510\t0.0\t1\t38 1472\n",
      "19\t71905\t0.0\t1\t66310 5595\n",
      "20\t184\t0.0\t1\t31 153\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 73969 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t1173.4\t0\t13\n",
      "4\t2\t293.3\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t5\t0.0\t1\t2 3\n",
      "19\t493\t0.0\t1\t66 427\n",
      "20\t73298\t0.0\t2\t71070 2228\n",
      "21\t155\t0.0\t2\t24 131\n",
      "44\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-2186-TF-H-PA-T_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-2186-TF-H-PA-T_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-2186-TF-H-PA-T_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-2186-TF-H-PA-T_72_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-2186-TF-H-PA-T_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-2186-TF-H-PA-T_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-2186-TF-H-PA-T_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-2186-TF-H-PA-T_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        35,784 reads  @     30.7 µs/read;   1.95 M reads/minute\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.14 s (32 us/read; 1.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,784\n",
      "  Read 1 with adapter:                  34,900 (97.5%)\n",
      "  Read 2 with adapter:                  35,142 (98.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,784 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,784,106 bp\n",
      "  Read 1:     5,396,068 bp\n",
      "  Read 2:     5,388,038 bp\n",
      "Total written (filtered):      9,419,094 bp (87.3%)\n",
      "  Read 1:     4,733,629 bp\n",
      "  Read 2:     4,685,465 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 34900 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.1\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "18\t770\t0.0\t1\t22 748\n",
      "19\t33991\t0.0\t1\t31780 2211\n",
      "20\t136\t0.0\t1\t19 117\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 35142 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t559.1\t0\t1\n",
      "4\t1\t139.8\t0\t1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t4\t0.0\t1\t3 1\n",
      "19\t299\t0.0\t1\t35 264\n",
      "20\t34752\t0.0\t2\t33649 1103\n",
      "21\t82\t0.0\t2\t20 62\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-1_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-1_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-1_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-1_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-1_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-1_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-1_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-1_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        54,883 reads  @     24.4 µs/read;   2.46 M reads/minute\n",
      "Finished in 1.39 s (25 us/read; 2.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,883\n",
      "  Read 1 with adapter:                  53,937 (98.3%)\n",
      "  Read 2 with adapter:                  54,223 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,883 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,132,048 bp\n",
      "  Read 1:     8,070,777 bp\n",
      "  Read 2:     8,061,271 bp\n",
      "Total written (filtered):     14,024,131 bp (86.9%)\n",
      "  Read 1:     7,046,806 bp\n",
      "  Read 2:     6,977,325 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 53937 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t964\t0.0\t1\t22 942\n",
      "19\t52824\t0.0\t1\t49585 3239\n",
      "20\t146\t0.0\t1\t23 123\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 54223 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t857.5\t0\t8\n",
      "4\t1\t214.4\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t7\t0.0\t1\t3 4\n",
      "19\t440\t0.0\t1\t65 375\n",
      "20\t53653\t0.0\t2\t52069 1584\n",
      "21\t110\t0.0\t2\t22 88\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-7_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-7_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-7_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-7_74_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-7_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-7_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-7_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-7_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        13,461 reads  @     54.2 µs/read;   1.11 M reads/minute\n",
      "Finished in 0.78 s (58 us/read; 1.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,461\n",
      "  Read 1 with adapter:                  12,973 (96.4%)\n",
      "  Read 2 with adapter:                  13,208 (98.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        13,461 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,003,875 bp\n",
      "  Read 1:     2,002,912 bp\n",
      "  Read 2:     2,000,963 bp\n",
      "Total written (filtered):      3,493,697 bp (87.3%)\n",
      "  Read 1:     1,756,809 bp\n",
      "  Read 2:     1,736,888 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12973 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.0\t1\t2\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t397\t0.0\t1\t7 390\n",
      "19\t12541\t0.0\t1\t11665 876\n",
      "20\t31\t0.0\t1\t6 25\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13208 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t112\t0.0\t1\t11 101\n",
      "20\t13056\t0.0\t2\t12599 457\n",
      "21\t37\t0.0\t2\t8 29\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-A_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-A_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-A_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-A_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-A_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-A_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-A_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-A_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00         7,969 reads  @     76.7 µs/read;   0.78 M reads/minute\n",
      "Finished in 0.66 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              7,969\n",
      "  Read 1 with adapter:                   7,714 (96.8%)\n",
      "  Read 2 with adapter:                   7,837 (98.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         7,969 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,337,735 bp\n",
      "  Read 1:     1,169,450 bp\n",
      "  Read 2:     1,168,285 bp\n",
      "Total written (filtered):      2,034,731 bp (87.0%)\n",
      "  Read 1:     1,023,097 bp\n",
      "  Read 2:     1,011,634 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7714 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t0 2\n",
      "18\t213\t0.0\t1\t3 210\n",
      "19\t7483\t0.0\t1\t6877 606\n",
      "20\t16\t0.0\t1\t4 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7837 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t124.5\t0\t2\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t74\t0.0\t1\t7 67\n",
      "20\t7739\t0.0\t2\t7503 236\n",
      "21\t21\t0.0\t2\t4 17\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-B_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-B_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-B_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-B_76_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-B_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-Control-B_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-B_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-Control-B_76_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00         7,889 reads  @     67.6 µs/read;   0.89 M reads/minute\n",
      "Finished in 0.58 s (74 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              7,889\n",
      "  Read 1 with adapter:                   7,739 (98.1%)\n",
      "  Read 2 with adapter:                   7,618 (96.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         7,889 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,033,917 bp\n",
      "  Read 1:       528,889 bp\n",
      "  Read 2:       505,028 bp\n",
      "Total written (filtered):        734,709 bp (71.1%)\n",
      "  Read 1:       381,941 bp\n",
      "  Read 2:       352,768 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7739 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t110\t0.0\t1\t3 107\n",
      "19\t7608\t0.0\t1\t7343 265\n",
      "20\t20\t0.0\t1\t1 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7618 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t123.3\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "19\t83\t0.0\t1\t13 70\n",
      "20\t7516\t0.0\t2\t7292 224\n",
      "21\t15\t0.0\t2\t3 12\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-mock2_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-mock2_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-mock2_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-mock2_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-mock2_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-y17umvnj/LibU2-mock2_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-mock2_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-qm507jsp/28044d30-520b-47e6-8f59-865b13e9046f/data/LibU2-mock2_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        57,696 reads  @     27.3 µs/read;   2.20 M reads/minute\n",
      "Finished in 1.62 s (28 us/read; 2.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,696\n",
      "  Read 1 with adapter:                  55,709 (96.6%)\n",
      "  Read 2 with adapter:                  56,639 (98.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        57,696 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,403,627 bp\n",
      "  Read 1:     8,711,822 bp\n",
      "  Read 2:     8,691,805 bp\n",
      "Total written (filtered):     15,214,527 bp (87.4%)\n",
      "  Read 1:     7,654,871 bp\n",
      "  Read 2:     7,559,656 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55709 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t56.3\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t6\t0.0\t1\t0 6\n",
      "18\t1621\t0.0\t1\t26 1595\n",
      "19\t53926\t0.0\t1\t50304 3622\n",
      "20\t144\t0.0\t1\t26 118\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 56639 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t901.5\t0\t10\n",
      "9\t1\t0.2\t0\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t8\t0.0\t1\t4 4\n",
      "19\t565\t0.0\t1\t68 497\n",
      "20\t55913\t0.0\t2\t53810 2103\n",
      "21\t140\t0.0\t2\t31 109\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibU.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1295D-TF-D-PA-T_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1295D-TF-D-PA-T_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1295D-TF-D-PA-T_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1295D-TF-D-PA-T_42_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1295D-TF-D-PA-T_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1295D-TF-D-PA-T_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1295D-TF-D-PA-T_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1295D-TF-D-PA-T_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       170,045 reads  @     20.3 µs/read;   2.96 M reads/minute\n",
      "Finished in 3.48 s (20 us/read; 2.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            170,045\n",
      "  Read 1 with adapter:                 168,030 (98.8%)\n",
      "  Read 2 with adapter:                 167,818 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       170,045 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,289,678 bp\n",
      "  Read 1:    25,653,863 bp\n",
      "  Read 2:    25,635,815 bp\n",
      "Total written (filtered):     44,742,789 bp (87.2%)\n",
      "  Read 1:    22,461,815 bp\n",
      "  Read 2:    22,280,974 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 168030 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t2.6\t0\t1\n",
      "9\t1\t0.6\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t7\t0.0\t1\t4 3\n",
      "17\t15\t0.0\t1\t9 6\n",
      "18\t818\t0.0\t1\t88 730\n",
      "19\t166751\t0.0\t1\t146930 19821\n",
      "20\t422\t0.0\t1\t74 348\n",
      "24\t1\t0.0\t1\t1\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 167818 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t2657.0\t0\t7\n",
      "4\t7\t664.2\t0\t7\n",
      "11\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t5\t0.0\t1\t1 4\n",
      "18\t24\t0.0\t1\t12 12\n",
      "19\t1566\t0.0\t1\t223 1343\n",
      "20\t165789\t0.0\t2\t161990 3799\n",
      "21\t406\t0.0\t2\t90 316\n",
      "22\t1\t0.0\t2\t1\n",
      "23\t1\t0.0\t2\t0 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1295E-TF-H-PA-T_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1295E-TF-H-PA-T_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1295E-TF-H-PA-T_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1295E-TF-H-PA-T_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1295E-TF-H-PA-T_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1295E-TF-H-PA-T_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1295E-TF-H-PA-T_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1295E-TF-H-PA-T_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        91,675 reads  @     20.1 µs/read;   2.98 M reads/minute\n",
      "Finished in 1.87 s (20 us/read; 2.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,675\n",
      "  Read 1 with adapter:                  90,529 (98.7%)\n",
      "  Read 2 with adapter:                  90,478 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        91,675 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,655,373 bp\n",
      "  Read 1:    13,831,048 bp\n",
      "  Read 2:    13,824,325 bp\n",
      "Total written (filtered):     24,126,803 bp (87.2%)\n",
      "  Read 1:    12,111,255 bp\n",
      "  Read 2:    12,015,548 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90529 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t358.1\t0\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t433\t0.0\t1\t42 391\n",
      "19\t89849\t0.0\t1\t77069 12780\n",
      "20\t231\t0.0\t1\t38 193\n",
      "32\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 90478 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1432.4\t0\t4\n",
      "4\t4\t358.1\t0\t4\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t13\t0.0\t1\t10 3\n",
      "19\t812\t0.0\t1\t122 690\n",
      "20\t89417\t0.0\t2\t87384 2033\n",
      "21\t218\t0.0\t2\t47 171\n",
      "24\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1304A-T1-H-PA-T_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1304A-T1-H-PA-T_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1304A-T1-H-PA-T_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1304A-T1-H-PA-T_44_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1304A-T1-H-PA-T_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1304A-T1-H-PA-T_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1304A-T1-H-PA-T_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1304A-T1-H-PA-T_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       112,960 reads  @     20.8 µs/read;   2.88 M reads/minute\n",
      "Finished in 2.38 s (21 us/read; 2.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,960\n",
      "  Read 1 with adapter:                 111,766 (98.9%)\n",
      "  Read 2 with adapter:                 111,596 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       112,960 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,044,285 bp\n",
      "  Read 1:    17,024,951 bp\n",
      "  Read 2:    17,019,334 bp\n",
      "Total written (filtered):     29,690,031 bp (87.2%)\n",
      "  Read 1:    14,901,730 bp\n",
      "  Read 2:    14,788,301 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 111766 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t441.2\t0\t1\n",
      "9\t5\t0.4\t0\t4 1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t519\t0.0\t1\t58 461\n",
      "19\t110877\t0.0\t1\t98565 12312\n",
      "20\t344\t0.0\t1\t55 289\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 111596 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1765.0\t0\t5\n",
      "4\t1\t441.2\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t18\t0.0\t1\t11 7\n",
      "19\t1001\t0.0\t1\t167 834\n",
      "20\t110281\t0.0\t2\t107348 2933\n",
      "21\t280\t0.0\t2\t52 228\n",
      "22\t2\t0.0\t2\t0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1304G-TF-H-PA-T_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1304G-TF-H-PA-T_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1304G-TF-H-PA-T_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1304G-TF-H-PA-T_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1304G-TF-H-PA-T_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1304G-TF-H-PA-T_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1304G-TF-H-PA-T_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1304G-TF-H-PA-T_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       135,749 reads  @     18.6 µs/read;   3.22 M reads/minute\n",
      "Finished in 2.55 s (19 us/read; 3.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            135,749\n",
      "  Read 1 with adapter:                 133,969 (98.7%)\n",
      "  Read 2 with adapter:                 133,881 (98.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       135,749 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,972,039 bp\n",
      "  Read 1:    20,489,693 bp\n",
      "  Read 2:    20,482,346 bp\n",
      "Total written (filtered):     35,750,755 bp (87.3%)\n",
      "  Read 1:    17,944,807 bp\n",
      "  Read 2:    17,805,948 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 133969 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t530.3\t0\t1\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t12\t0.0\t1\t5 7\n",
      "18\t678\t0.0\t1\t70 608\n",
      "19\t132967\t0.0\t1\t115335 17632\n",
      "20\t294\t0.0\t1\t44 250\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 133881 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t2121.1\t0\t10\n",
      "4\t5\t530.3\t0\t5\n",
      "11\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t13\t0.0\t1\t8 5\n",
      "19\t1217\t0.0\t1\t186 1031\n",
      "20\t132312\t0.0\t2\t128759 3553\n",
      "21\t316\t0.0\t2\t60 256\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1313-T1-H-DL-T_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1313-T1-H-DL-T_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1313-T1-H-DL-T_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1313-T1-H-DL-T_46_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1313-T1-H-DL-T_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1313-T1-H-DL-T_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1313-T1-H-DL-T_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1313-T1-H-DL-T_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       134,325 reads  @     21.7 µs/read;   2.76 M reads/minute\n",
      "Finished in 2.95 s (22 us/read; 2.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            134,325\n",
      "  Read 1 with adapter:                 132,734 (98.8%)\n",
      "  Read 2 with adapter:                 132,573 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       134,325 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,452,432 bp\n",
      "  Read 1:    20,235,354 bp\n",
      "  Read 2:    20,217,078 bp\n",
      "Total written (filtered):     35,280,460 bp (87.2%)\n",
      "  Read 1:    17,713,790 bp\n",
      "  Read 2:    17,566,670 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 132734 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t4\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t11\t0.0\t1\t6 5\n",
      "18\t630\t0.0\t1\t78 552\n",
      "19\t131727\t0.0\t1\t116576 15151\n",
      "20\t351\t0.0\t1\t70 281\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 132573 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t2098.8\t0\t8\n",
      "4\t1\t524.7\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t18\t0.0\t1\t7 11\n",
      "19\t1094\t0.0\t1\t163 931\n",
      "20\t131129\t0.0\t2\t127819 3310\n",
      "21\t309\t0.0\t2\t67 242\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1319A-TF-H-SS-T_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1319A-TF-H-SS-T_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1319A-TF-H-SS-T_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1319A-TF-H-SS-T_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1319A-TF-H-SS-T_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1319A-TF-H-SS-T_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1319A-TF-H-SS-T_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1319A-TF-H-SS-T_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       161,535 reads  @     21.2 µs/read;   2.83 M reads/minute\n",
      "Finished in 3.45 s (21 us/read; 2.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,535\n",
      "  Read 1 with adapter:                 158,700 (98.2%)\n",
      "  Read 2 with adapter:                 159,163 (98.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       161,535 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,261,902 bp\n",
      "  Read 1:    23,669,593 bp\n",
      "  Read 2:    23,592,309 bp\n",
      "Total written (filtered):     41,065,357 bp (86.9%)\n",
      "  Read 1:    20,654,821 bp\n",
      "  Read 2:    20,410,536 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 158700 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t631.0\t0\t2\n",
      "9\t3\t0.6\t0\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t830\t0.0\t1\t92 738\n",
      "19\t157474\t0.0\t1\t128381 29093\n",
      "20\t371\t0.0\t1\t80 291\n",
      "21\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "32\t2\t0.0\t1\t2\n",
      "33\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 159163 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t2524.0\t0\t10\n",
      "4\t5\t631.0\t0\t5\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t2 2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t14\t0.0\t1\t5 9\n",
      "19\t1596\t0.0\t1\t205 1391\n",
      "20\t157080\t0.0\t2\t152601 4479\n",
      "21\t446\t0.0\t2\t90 356\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320A-TF-H-PS-T_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320A-TF-H-PS-T_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320A-TF-H-PS-T_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320A-TF-H-PS-T_48_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320A-TF-H-PS-T_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320A-TF-H-PS-T_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320A-TF-H-PS-T_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320A-TF-H-PS-T_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        94,958 reads  @     21.1 µs/read;   2.84 M reads/minute\n",
      "Finished in 2.03 s (21 us/read; 2.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,958\n",
      "  Read 1 with adapter:                  91,403 (96.3%)\n",
      "  Read 2 with adapter:                  93,316 (98.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        94,958 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,564,631 bp\n",
      "  Read 1:    14,286,191 bp\n",
      "  Read 2:    14,278,440 bp\n",
      "Total written (filtered):     24,962,559 bp (87.4%)\n",
      "  Read 1:    12,549,738 bp\n",
      "  Read 2:    12,412,821 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91403 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t370.9\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t317\t0.0\t1\t39 278\n",
      "19\t90876\t0.0\t1\t56531 34345\n",
      "20\t194\t0.0\t1\t33 161\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 93316 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1483.7\t0\t4\n",
      "4\t3\t370.9\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t821\t0.0\t1\t125 696\n",
      "20\t92217\t0.0\t2\t89899 2318\n",
      "21\t256\t0.0\t2\t58 198\n",
      "22\t1\t0.0\t2\t1\n",
      "35\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320B-TF-D-PS-T_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320B-TF-D-PS-T_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320B-TF-D-PS-T_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320B-TF-D-PS-T_49_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320B-TF-D-PS-T_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320B-TF-D-PS-T_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320B-TF-D-PS-T_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320B-TF-D-PS-T_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        92,947 reads  @     23.1 µs/read;   2.60 M reads/minute\n",
      "Finished in 2.18 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,947\n",
      "  Read 1 with adapter:                  91,767 (98.7%)\n",
      "  Read 2 with adapter:                  91,775 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        92,947 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,030,292 bp\n",
      "  Read 1:    14,018,193 bp\n",
      "  Read 2:    14,012,099 bp\n",
      "Total written (filtered):     24,452,365 bp (87.2%)\n",
      "  Read 1:    12,274,913 bp\n",
      "  Read 2:    12,177,452 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91767 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t363.1\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t8\t0.0\t1\t2 6\n",
      "18\t431\t0.0\t1\t51 380\n",
      "19\t91102\t0.0\t1\t78437 12665\n",
      "20\t217\t0.0\t1\t46 171\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 91775 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1452.3\t0\t4\n",
      "4\t7\t363.1\t0\t7\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t5\t0.0\t1\t2 3\n",
      "19\t829\t0.0\t1\t134 695\n",
      "20\t90709\t0.0\t2\t88255 2454\n",
      "21\t210\t0.0\t2\t45 165\n",
      "22\t2\t0.0\t2\t0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320E-T1-H-PS-T_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320E-T1-H-PS-T_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320E-T1-H-PS-T_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320E-T1-H-PS-T_50_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320E-T1-H-PS-T_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1320E-T1-H-PS-T_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320E-T1-H-PS-T_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1320E-T1-H-PS-T_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       120,518 reads  @     20.3 µs/read;   2.95 M reads/minute\n",
      "Finished in 2.48 s (21 us/read; 2.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,518\n",
      "  Read 1 with adapter:                 118,641 (98.4%)\n",
      "  Read 2 with adapter:                 118,963 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       120,518 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,251,745 bp\n",
      "  Read 1:    18,132,477 bp\n",
      "  Read 2:    18,119,268 bp\n",
      "Total written (filtered):     31,619,471 bp (87.2%)\n",
      "  Read 1:    15,878,617 bp\n",
      "  Read 2:    15,740,854 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 118641 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t470.8\t0\t2\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t1 3\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t0 4\n",
      "17\t12\t0.0\t1\t5 7\n",
      "18\t455\t0.0\t1\t59 396\n",
      "19\t117888\t0.0\t1\t98608 19280\n",
      "20\t268\t0.0\t1\t53 215\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 118963 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1883.1\t0\t5\n",
      "4\t1\t470.8\t0\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t12\t0.0\t1\t2 10\n",
      "19\t942\t0.0\t1\t150 792\n",
      "20\t117717\t0.0\t2\t114835 2882\n",
      "21\t277\t0.0\t2\t54 223\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1326C-T1-H-PS-T_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1326C-T1-H-PS-T_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1326C-T1-H-PS-T_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1326C-T1-H-PS-T_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1326C-T1-H-PS-T_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1326C-T1-H-PS-T_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1326C-T1-H-PS-T_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1326C-T1-H-PS-T_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       124,079 reads  @     18.2 µs/read;   3.29 M reads/minute\n",
      "Finished in 2.29 s (18 us/read; 3.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            124,079\n",
      "  Read 1 with adapter:                 122,726 (98.9%)\n",
      "  Read 2 with adapter:                 122,655 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       124,079 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,263,125 bp\n",
      "  Read 1:    18,631,645 bp\n",
      "  Read 2:    18,631,480 bp\n",
      "Total written (filtered):     32,479,258 bp (87.2%)\n",
      "  Read 1:    16,300,082 bp\n",
      "  Read 2:    16,179,176 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 122726 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t5\t0.0\t1\t5\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t8\t0.0\t1\t3 5\n",
      "18\t479\t0.0\t1\t53 426\n",
      "19\t121926\t0.0\t1\t108043 13883\n",
      "20\t306\t0.0\t1\t82 224\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 122655 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1938.7\t0\t6\n",
      "4\t3\t484.7\t0\t3\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t9\t0.0\t1\t3 6\n",
      "19\t889\t0.0\t1\t135 754\n",
      "20\t121437\t0.0\t2\t118365 3072\n",
      "21\t300\t0.0\t2\t70 230\n",
      "40\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1326D-TF-D-PS-T_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1326D-TF-D-PS-T_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1326D-TF-D-PS-T_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1326D-TF-D-PS-T_52_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1326D-TF-D-PS-T_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1326D-TF-D-PS-T_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1326D-TF-D-PS-T_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1326D-TF-D-PS-T_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       113,230 reads  @     21.3 µs/read;   2.82 M reads/minute\n",
      "Finished in 2.43 s (22 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,230\n",
      "  Read 1 with adapter:                 111,815 (98.8%)\n",
      "  Read 2 with adapter:                 111,810 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,230 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,124,018 bp\n",
      "  Read 1:    17,064,337 bp\n",
      "  Read 2:    17,059,681 bp\n",
      "Total written (filtered):     29,764,382 bp (87.2%)\n",
      "  Read 1:    14,940,126 bp\n",
      "  Read 2:    14,824,256 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 111815 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t0 3\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t13\t0.0\t1\t5 8\n",
      "18\t409\t0.0\t1\t65 344\n",
      "19\t111126\t0.0\t1\t96497 14629\n",
      "20\t252\t0.0\t1\t41 211\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 111810 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1769.2\t0\t7\n",
      "4\t4\t442.3\t0\t4\n",
      "11\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t11\t0.0\t1\t6 5\n",
      "19\t827\t0.0\t1\t134 693\n",
      "20\t110676\t0.0\t2\t108132 2544\n",
      "21\t280\t0.0\t2\t63 217\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1335-TF-H-DL-T_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1335-TF-H-DL-T_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1335-TF-H-DL-T_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1335-TF-H-DL-T_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1335-TF-H-DL-T_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1335-TF-H-DL-T_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1335-TF-H-DL-T_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1335-TF-H-DL-T_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01       101,063 reads  @     19.0 µs/read;   3.15 M reads/minute\n",
      "Finished in 1.96 s (19 us/read; 3.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,063\n",
      "  Read 1 with adapter:                  99,962 (98.9%)\n",
      "  Read 2 with adapter:                  99,949 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       101,063 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,429,592 bp\n",
      "  Read 1:    15,224,784 bp\n",
      "  Read 2:    15,204,808 bp\n",
      "Total written (filtered):     26,532,392 bp (87.2%)\n",
      "  Read 1:    13,325,836 bp\n",
      "  Read 2:    13,206,556 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99962 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t533\t0.0\t1\t54 479\n",
      "19\t99152\t0.0\t1\t86765 12387\n",
      "20\t264\t0.0\t1\t49 215\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 99949 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1579.1\t0\t2\n",
      "4\t2\t394.8\t0\t2\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t15\t0.0\t1\t4 11\n",
      "19\t843\t0.0\t1\t127 716\n",
      "20\t98821\t0.0\t2\t96115 2706\n",
      "21\t257\t0.0\t2\t53 204\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1339B-T1-H-SS-T_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1339B-T1-H-SS-T_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1339B-T1-H-SS-T_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1339B-T1-H-SS-T_54_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1339B-T1-H-SS-T_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1339B-T1-H-SS-T_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1339B-T1-H-SS-T_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1339B-T1-H-SS-T_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       107,621 reads  @     19.6 µs/read;   3.06 M reads/minute\n",
      "Finished in 2.14 s (20 us/read; 3.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,621\n",
      "  Read 1 with adapter:                 106,333 (98.8%)\n",
      "  Read 2 with adapter:                 106,227 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       107,621 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,027,313 bp\n",
      "  Read 1:    16,017,705 bp\n",
      "  Read 2:    16,009,608 bp\n",
      "Total written (filtered):     27,883,579 bp (87.1%)\n",
      "  Read 1:    13,997,749 bp\n",
      "  Read 2:    13,885,830 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 106333 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t6\t0.0\t1\t6\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t1 4\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t546\t0.0\t1\t67 479\n",
      "19\t105511\t0.0\t1\t92549 12962\n",
      "20\t257\t0.0\t1\t58 199\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 106227 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1681.6\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t16\t0.0\t1\t8 8\n",
      "19\t957\t0.0\t1\t142 815\n",
      "20\t104927\t0.0\t2\t102265 2662\n",
      "21\t317\t0.0\t2\t71 246\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1347B-TF-H-PS-T_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1347B-TF-H-PS-T_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1347B-TF-H-PS-T_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1347B-TF-H-PS-T_55_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1347B-TF-H-PS-T_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1347B-TF-H-PS-T_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1347B-TF-H-PS-T_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1347B-TF-H-PS-T_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       114,320 reads  @     21.4 µs/read;   2.81 M reads/minute\n",
      "Finished in 2.47 s (22 us/read; 2.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,320\n",
      "  Read 1 with adapter:                 112,925 (98.8%)\n",
      "  Read 2 with adapter:                 112,943 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,320 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,475,565 bp\n",
      "  Read 1:    16,752,888 bp\n",
      "  Read 2:    16,722,677 bp\n",
      "Total written (filtered):     29,072,403 bp (86.8%)\n",
      "  Read 1:    14,607,731 bp\n",
      "  Read 2:    14,464,672 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 112925 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t446.6\t0\t2\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t6\t0.0\t1\t5 1\n",
      "13\t4\t0.0\t1\t3 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t572\t0.0\t1\t79 493\n",
      "19\t112043\t0.0\t1\t98161 13882\n",
      "20\t289\t0.0\t1\t57 232\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 112943 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1786.2\t0\t7\n",
      "4\t4\t446.6\t0\t4\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t890\t0.0\t1\t127 763\n",
      "20\t111764\t0.0\t2\t108854 2910\n",
      "21\t267\t0.0\t2\t62 205\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1347C-T1-H-PS-T_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1347C-T1-H-PS-T_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1347C-T1-H-PS-T_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1347C-T1-H-PS-T_56_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1347C-T1-H-PS-T_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1347C-T1-H-PS-T_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1347C-T1-H-PS-T_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1347C-T1-H-PS-T_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       112,143 reads  @     22.3 µs/read;   2.69 M reads/minute\n",
      "Finished in 2.53 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,143\n",
      "  Read 1 with adapter:                 110,786 (98.8%)\n",
      "  Read 2 with adapter:                 110,702 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       112,143 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,089,081 bp\n",
      "  Read 1:    16,551,993 bp\n",
      "  Read 2:    16,537,088 bp\n",
      "Total written (filtered):     28,771,570 bp (87.0%)\n",
      "  Read 1:    14,447,426 bp\n",
      "  Read 2:    14,324,144 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110786 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t11\t0.0\t1\t5 6\n",
      "18\t643\t0.0\t1\t54 589\n",
      "19\t109817\t0.0\t1\t96108 13709\n",
      "20\t301\t0.0\t1\t47 254\n",
      "21\t1\t0.0\t1\t1\n",
      "36\t1\t0.0\t1\t1\n",
      "54\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110702 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1752.2\t0\t9\n",
      "4\t2\t438.1\t0\t2\n",
      "12\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t7\t0.0\t1\t4 3\n",
      "19\t1112\t0.0\t1\t160 952\n",
      "20\t109288\t0.0\t2\t106191 3097\n",
      "21\t271\t0.0\t2\t48 223\n",
      "22\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1360C-TF-H-MC-T_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1360C-TF-H-MC-T_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1360C-TF-H-MC-T_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1360C-TF-H-MC-T_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1360C-TF-H-MC-T_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1360C-TF-H-MC-T_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1360C-TF-H-MC-T_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1360C-TF-H-MC-T_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       157,446 reads  @     21.6 µs/read;   2.78 M reads/minute\n",
      "Finished in 3.43 s (22 us/read; 2.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            157,446\n",
      "  Read 1 with adapter:                 155,776 (98.9%)\n",
      "  Read 2 with adapter:                 155,638 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       157,446 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,369,728 bp\n",
      "  Read 1:    23,700,567 bp\n",
      "  Read 2:    23,669,161 bp\n",
      "Total written (filtered):     41,298,582 bp (87.2%)\n",
      "  Read 1:    20,741,151 bp\n",
      "  Read 2:    20,557,431 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 155776 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t615.0\t0\t2\n",
      "9\t1\t0.6\t0\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t9\t0.0\t1\t3 6\n",
      "18\t590\t0.0\t1\t70 520\n",
      "19\t154812\t0.0\t1\t136369 18443\n",
      "20\t350\t0.0\t1\t64 286\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 155638 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t2460.1\t0\t14\n",
      "4\t4\t615.0\t0\t4\n",
      "11\t3\t0.0\t1\t2 1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t13\t0.0\t1\t3 10\n",
      "19\t982\t0.0\t1\t165 817\n",
      "20\t154277\t0.0\t2\t150755 3522\n",
      "21\t338\t0.0\t2\t68 270\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1363B-T1-H-MC-T_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1363B-T1-H-MC-T_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1363B-T1-H-MC-T_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1363B-T1-H-MC-T_58_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1363B-T1-H-MC-T_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1363B-T1-H-MC-T_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1363B-T1-H-MC-T_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1363B-T1-H-MC-T_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        93,652 reads  @     22.6 µs/read;   2.65 M reads/minute\n",
      "Finished in 2.14 s (23 us/read; 2.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,652\n",
      "  Read 1 with adapter:                  92,684 (99.0%)\n",
      "  Read 2 with adapter:                  92,626 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        93,652 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,031,878 bp\n",
      "  Read 1:    14,027,217 bp\n",
      "  Read 2:    14,004,661 bp\n",
      "Total written (filtered):     24,419,068 bp (87.1%)\n",
      "  Read 1:    12,266,393 bp\n",
      "  Read 2:    12,152,675 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 92684 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t5\t0.0\t1\t5\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t331\t0.0\t1\t45 286\n",
      "19\t92127\t0.0\t1\t81107 11020\n",
      "20\t214\t0.0\t1\t56 158\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 92626 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1463.3\t0\t3\n",
      "4\t3\t365.8\t0\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t605\t0.0\t1\t75 530\n",
      "20\t91814\t0.0\t2\t89768 2046\n",
      "21\t193\t0.0\t2\t38 155\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1363B-TF-D-MC-T_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1363B-TF-D-MC-T_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1363B-TF-D-MC-T_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1363B-TF-D-MC-T_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1363B-TF-D-MC-T_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1363B-TF-D-MC-T_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1363B-TF-D-MC-T_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1363B-TF-D-MC-T_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       114,928 reads  @     21.9 µs/read;   2.74 M reads/minute\n",
      "Finished in 2.54 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,928\n",
      "  Read 1 with adapter:                 113,653 (98.9%)\n",
      "  Read 2 with adapter:                 113,674 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,928 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,607,200 bp\n",
      "  Read 1:    17,328,432 bp\n",
      "  Read 2:    17,278,768 bp\n",
      "Total written (filtered):     30,175,215 bp (87.2%)\n",
      "  Read 1:    15,169,250 bp\n",
      "  Read 2:    15,005,965 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113653 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t448.9\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t7\t0.0\t1\t4 3\n",
      "18\t434\t0.0\t1\t69 365\n",
      "19\t112894\t0.0\t1\t98302 14592\n",
      "20\t304\t0.0\t1\t58 246\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 113674 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1795.8\t0\t5\n",
      "4\t1\t448.9\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t7\t0.0\t1\t3 4\n",
      "19\t798\t0.0\t1\t122 676\n",
      "20\t112601\t0.0\t2\t110057 2544\n",
      "21\t256\t0.0\t2\t68 188\n",
      "22\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1365-T1-H-DL-T_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1365-T1-H-DL-T_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1365-T1-H-DL-T_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1365-T1-H-DL-T_60_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1365-T1-H-DL-T_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1365-T1-H-DL-T_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1365-T1-H-DL-T_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1365-T1-H-DL-T_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       105,237 reads  @     24.2 µs/read;   2.48 M reads/minute\n",
      "Finished in 2.58 s (25 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            105,237\n",
      "  Read 1 with adapter:                 103,869 (98.7%)\n",
      "  Read 2 with adapter:                 103,927 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       105,237 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,566,309 bp\n",
      "  Read 1:    15,792,710 bp\n",
      "  Read 2:    15,773,599 bp\n",
      "Total written (filtered):     27,515,413 bp (87.2%)\n",
      "  Read 1:    13,819,493 bp\n",
      "  Read 2:    13,695,920 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103869 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t10\t0.0\t1\t5 5\n",
      "18\t523\t0.0\t1\t63 460\n",
      "19\t103065\t0.0\t1\t89438 13627\n",
      "20\t262\t0.0\t1\t40 222\n",
      "51\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 103927 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1644.3\t0\t11\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t12\t0.0\t1\t6 6\n",
      "19\t876\t0.0\t1\t127 749\n",
      "20\t102758\t0.0\t2\t100516 2242\n",
      "21\t263\t0.0\t2\t72 191\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368A-TF-H-MC-T_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368A-TF-H-MC-T_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368A-TF-H-MC-T_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368A-TF-H-MC-T_61_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368A-TF-H-MC-T_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368A-TF-H-MC-T_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368A-TF-H-MC-T_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368A-TF-H-MC-T_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       129,569 reads  @     21.0 µs/read;   2.85 M reads/minute\n",
      "Finished in 2.76 s (21 us/read; 2.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,569\n",
      "  Read 1 with adapter:                 128,043 (98.8%)\n",
      "  Read 2 with adapter:                 127,966 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       129,569 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,682,540 bp\n",
      "  Read 1:    19,388,163 bp\n",
      "  Read 2:    19,294,377 bp\n",
      "Total written (filtered):     33,691,596 bp (87.1%)\n",
      "  Read 1:    16,955,669 bp\n",
      "  Read 2:    16,735,927 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 128043 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t506.1\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t6\t0.0\t1\t4 2\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t13\t0.0\t1\t4 9\n",
      "18\t557\t0.0\t1\t71 486\n",
      "19\t127134\t0.0\t1\t112509 14625\n",
      "20\t322\t0.0\t1\t68 254\n",
      "23\t1\t0.0\t1\t1\n",
      "42\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 127966 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t2024.5\t0\t8\n",
      "4\t2\t506.1\t0\t2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t10\t0.0\t1\t4 6\n",
      "19\t943\t0.0\t1\t148 795\n",
      "20\t126722\t0.0\t2\t123524 3198\n",
      "21\t277\t0.0\t2\t59 218\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368B-T1-H-MC-T_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368B-T1-H-MC-T_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368B-T1-H-MC-T_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368B-T1-H-MC-T_62_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368B-T1-H-MC-T_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368B-T1-H-MC-T_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368B-T1-H-MC-T_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368B-T1-H-MC-T_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       126,222 reads  @     22.3 µs/read;   2.69 M reads/minute\n",
      "Finished in 2.84 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            126,222\n",
      "  Read 1 with adapter:                 125,093 (99.1%)\n",
      "  Read 2 with adapter:                 124,984 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       126,222 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,983,941 bp\n",
      "  Read 1:    19,009,175 bp\n",
      "  Read 2:    18,974,766 bp\n",
      "Total written (filtered):     33,108,392 bp (87.2%)\n",
      "  Read 1:    16,632,655 bp\n",
      "  Read 2:    16,475,737 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 125093 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t3\t0.0\t1\t0 3\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t413\t0.0\t1\t68 345\n",
      "19\t124389\t0.0\t1\t111241 13148\n",
      "20\t269\t0.0\t1\t64 205\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 124984 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1972.2\t0\t6\n",
      "4\t2\t493.1\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t706\t0.0\t1\t108 598\n",
      "20\t124044\t0.0\t2\t121443 2601\n",
      "21\t216\t0.0\t2\t55 161\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368E-TF-D-MC-T_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368E-TF-D-MC-T_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368E-TF-D-MC-T_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368E-TF-D-MC-T_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368E-TF-D-MC-T_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1368E-TF-D-MC-T_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368E-TF-D-MC-T_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1368E-TF-D-MC-T_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02       115,974 reads  @     18.4 µs/read;   3.26 M reads/minute\n",
      "Finished in 2.17 s (19 us/read; 3.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,974\n",
      "  Read 1 with adapter:                 114,923 (99.1%)\n",
      "  Read 2 with adapter:                 114,867 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,974 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,003,327 bp\n",
      "  Read 1:    17,510,551 bp\n",
      "  Read 2:    17,492,776 bp\n",
      "Total written (filtered):     30,523,260 bp (87.2%)\n",
      "  Read 1:    15,327,230 bp\n",
      "  Read 2:    15,196,030 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 114923 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t3\t0.0\t1\t0 3\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t406\t0.0\t1\t61 345\n",
      "19\t114283\t0.0\t1\t102306 11977\n",
      "20\t225\t0.0\t1\t49 176\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 114867 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1812.1\t0\t5\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t731\t0.0\t1\t118 613\n",
      "20\t113891\t0.0\t2\t111401 2490\n",
      "21\t230\t0.0\t2\t51 179\n",
      "22\t1\t0.0\t2\t1\n",
      "26\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1379B-T1-H-MC-T_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1379B-T1-H-MC-T_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1379B-T1-H-MC-T_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1379B-T1-H-MC-T_64_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1379B-T1-H-MC-T_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1379B-T1-H-MC-T_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1379B-T1-H-MC-T_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1379B-T1-H-MC-T_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02       126,069 reads  @     21.5 µs/read;   2.79 M reads/minute\n",
      "Finished in 2.74 s (22 us/read; 2.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            126,069\n",
      "  Read 1 with adapter:                 124,836 (99.0%)\n",
      "  Read 2 with adapter:                 124,645 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       126,069 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,747,660 bp\n",
      "  Read 1:    18,379,308 bp\n",
      "  Read 2:    18,368,352 bp\n",
      "Total written (filtered):     31,883,906 bp (86.8%)\n",
      "  Read 1:    16,007,629 bp\n",
      "  Read 2:    15,876,277 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 124836 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t510\t0.0\t1\t61 449\n",
      "19\t123938\t0.0\t1\t110599 13339\n",
      "20\t359\t0.0\t1\t58 301\n",
      "21\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t1\t0 1\n",
      "23\t1\t0.0\t1\t1\n",
      "30\t1\t0.0\t1\t1\n",
      "34\t1\t0.0\t1\t1\n",
      "49\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 124645 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1969.8\t0\t7\n",
      "4\t1\t492.5\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t7\t0.0\t1\t4 3\n",
      "19\t988\t0.0\t1\t136 852\n",
      "20\t123291\t0.0\t2\t119420 3871\n",
      "21\t345\t0.0\t2\t77 268\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1613A-T1-H-SS-T_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1613A-T1-H-SS-T_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1613A-T1-H-SS-T_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1613A-T1-H-SS-T_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1613A-T1-H-SS-T_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1613A-T1-H-SS-T_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1613A-T1-H-SS-T_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1613A-T1-H-SS-T_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       111,628 reads  @     21.0 µs/read;   2.85 M reads/minute\n",
      "Finished in 2.37 s (21 us/read; 2.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,628\n",
      "  Read 1 with adapter:                 109,897 (98.4%)\n",
      "  Read 2 with adapter:                 109,920 (98.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,628 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,477,871 bp\n",
      "  Read 1:    16,741,505 bp\n",
      "  Read 2:    16,736,366 bp\n",
      "Total written (filtered):     29,192,916 bp (87.2%)\n",
      "  Read 1:    14,653,854 bp\n",
      "  Read 2:    14,539,062 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109897 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t436.0\t0\t1\n",
      "8\t1\t1.7\t0\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t565\t0.0\t1\t36 529\n",
      "19\t109068\t0.0\t1\t94010 15058\n",
      "20\t245\t0.0\t1\t45 200\n",
      "35\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 109920 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1744.2\t0\t10\n",
      "4\t4\t436.0\t0\t4\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t10\t0.0\t1\t2 8\n",
      "19\t1056\t0.0\t1\t132 924\n",
      "20\t108519\t0.0\t2\t105396 3123\n",
      "21\t304\t0.0\t2\t69 235\n",
      "23\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1655C-T1-H-CN-T_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1655C-T1-H-CN-T_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1655C-T1-H-CN-T_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1655C-T1-H-CN-T_66_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1655C-T1-H-CN-T_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1655C-T1-H-CN-T_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1655C-T1-H-CN-T_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1655C-T1-H-CN-T_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       106,779 reads  @     22.6 µs/read;   2.66 M reads/minute\n",
      "Finished in 2.44 s (23 us/read; 2.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,779\n",
      "  Read 1 with adapter:                 105,188 (98.5%)\n",
      "  Read 2 with adapter:                 105,370 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,779 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,230,340 bp\n",
      "  Read 1:    16,121,962 bp\n",
      "  Read 2:    16,108,378 bp\n",
      "Total written (filtered):     28,125,284 bp (87.3%)\n",
      "  Read 1:    14,123,633 bp\n",
      "  Read 2:    14,001,651 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 105188 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "12\t4\t0.0\t1\t2 2\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t1 4\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t380\t0.0\t1\t47 333\n",
      "19\t104585\t0.0\t1\t87943 16642\n",
      "20\t209\t0.0\t1\t47 162\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 105370 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1668.4\t0\t4\n",
      "4\t5\t417.1\t0\t5\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t14\t0.0\t1\t8 6\n",
      "19\t768\t0.0\t1\t118 650\n",
      "20\t104308\t0.0\t2\t101629 2679\n",
      "21\t265\t0.0\t2\t46 219\n",
      "29\t2\t0.0\t2\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1661B-TF-H-CN-T_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1661B-TF-H-CN-T_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1661B-TF-H-CN-T_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1661B-TF-H-CN-T_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1661B-TF-H-CN-T_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1661B-TF-H-CN-T_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1661B-TF-H-CN-T_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1661B-TF-H-CN-T_67_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       109,260 reads  @     23.0 µs/read;   2.60 M reads/minute\n",
      "Finished in 2.54 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,260\n",
      "  Read 1 with adapter:                 108,043 (98.9%)\n",
      "  Read 2 with adapter:                 107,911 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,260 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,797,389 bp\n",
      "  Read 1:    16,409,988 bp\n",
      "  Read 2:    16,387,401 bp\n",
      "Total written (filtered):     28,587,274 bp (87.2%)\n",
      "  Read 1:    14,357,363 bp\n",
      "  Read 2:    14,229,911 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108043 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t426.8\t0\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t402\t0.0\t1\t57 345\n",
      "19\t107381\t0.0\t1\t94330 13051\n",
      "20\t245\t0.0\t1\t46 199\n",
      "21\t3\t0.0\t1\t2 1\n",
      "33\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 107911 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1707.2\t0\t10\n",
      "4\t3\t426.8\t0\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t12\t0.0\t1\t8 4\n",
      "19\t718\t0.0\t1\t128 590\n",
      "20\t106917\t0.0\t2\t104287 2630\n",
      "21\t248\t0.0\t2\t49 199\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1662H-TF-D-MC-T_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1662H-TF-D-MC-T_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1662H-TF-D-MC-T_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1662H-TF-D-MC-T_68_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1662H-TF-D-MC-T_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1662H-TF-D-MC-T_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1662H-TF-D-MC-T_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1662H-TF-D-MC-T_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       132,824 reads  @     18.8 µs/read;   3.19 M reads/minute\n",
      "Finished in 2.53 s (19 us/read; 3.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            132,824\n",
      "  Read 1 with adapter:                 131,115 (98.7%)\n",
      "  Read 2 with adapter:                 131,381 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       132,824 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,803,087 bp\n",
      "  Read 1:    19,920,269 bp\n",
      "  Read 2:    19,882,818 bp\n",
      "Total written (filtered):     34,685,416 bp (87.1%)\n",
      "  Read 1:    17,429,398 bp\n",
      "  Read 2:    17,256,018 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 131115 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t6\t0.0\t1\t5 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t3\t0.0\t1\t2 1\n",
      "17\t11\t0.0\t1\t4 7\n",
      "18\t520\t0.0\t1\t72 448\n",
      "19\t130272\t0.0\t1\t108476 21796\n",
      "20\t295\t0.0\t1\t49 246\n",
      "29\t1\t0.0\t1\t1\n",
      "36\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 131381 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t2075.4\t0\t7\n",
      "4\t3\t518.8\t0\t3\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t11\t0.0\t1\t6 5\n",
      "19\t894\t0.0\t1\t145 749\n",
      "20\t130158\t0.0\t2\t127536 2622\n",
      "21\t298\t0.0\t2\t75 223\n",
      "22\t2\t0.0\t2\t0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1666E-T1-H-PA-T_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1666E-T1-H-PA-T_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1666E-T1-H-PA-T_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1666E-T1-H-PA-T_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1666E-T1-H-PA-T_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1666E-T1-H-PA-T_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1666E-T1-H-PA-T_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1666E-T1-H-PA-T_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       148,649 reads  @     19.6 µs/read;   3.06 M reads/minute\n",
      "Finished in 2.94 s (20 us/read; 3.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            148,649\n",
      "  Read 1 with adapter:                 146,716 (98.7%)\n",
      "  Read 2 with adapter:                 146,792 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       148,649 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,866,803 bp\n",
      "  Read 1:    22,439,185 bp\n",
      "  Read 2:    22,427,618 bp\n",
      "Total written (filtered):     39,145,031 bp (87.2%)\n",
      "  Read 1:    19,652,021 bp\n",
      "  Read 2:    19,493,010 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 146716 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.6\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "14\t4\t0.0\t1\t2 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t677\t0.0\t1\t99 578\n",
      "19\t145680\t0.0\t1\t126274 19406\n",
      "20\t338\t0.0\t1\t67 271\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 146792 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t2322.6\t0\t7\n",
      "4\t2\t580.7\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t21\t0.0\t1\t9 12\n",
      "19\t1381\t0.0\t1\t169 1212\n",
      "20\t144962\t0.0\t2\t140970 3992\n",
      "21\t406\t0.0\t2\t70 336\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1669I-T1-H-MC-T_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1669I-T1-H-MC-T_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1669I-T1-H-MC-T_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1669I-T1-H-MC-T_70_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1669I-T1-H-MC-T_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1669I-T1-H-MC-T_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1669I-T1-H-MC-T_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1669I-T1-H-MC-T_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02       130,407 reads  @     22.0 µs/read;   2.73 M reads/minute\n",
      "Finished in 2.90 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,407\n",
      "  Read 1 with adapter:                 128,828 (98.8%)\n",
      "  Read 2 with adapter:                 129,009 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       130,407 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,252,692 bp\n",
      "  Read 1:    19,642,633 bp\n",
      "  Read 2:    19,610,059 bp\n",
      "Total written (filtered):     34,225,670 bp (87.2%)\n",
      "  Read 1:    17,195,106 bp\n",
      "  Read 2:    17,030,564 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 128828 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t509.4\t0\t1\n",
      "9\t2\t0.5\t0\t2\n",
      "12\t6\t0.0\t1\t5 1\n",
      "13\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t426\t0.0\t1\t59 367\n",
      "19\t128073\t0.0\t1\t109552 18521\n",
      "20\t314\t0.0\t1\t69 245\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 129009 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t2037.6\t0\t5\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t4\t0.0\t1\t1 3\n",
      "19\t825\t0.0\t1\t155 670\n",
      "20\t127917\t0.0\t2\t125048 2869\n",
      "21\t246\t0.0\t2\t62 184\n",
      "22\t2\t0.0\t2\t1 1\n",
      "24\t1\t0.0\t2\t1\n",
      "36\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1672B-T1-H-PS-T_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1672B-T1-H-PS-T_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1672B-T1-H-PS-T_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1672B-T1-H-PS-T_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1672B-T1-H-PS-T_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1672B-T1-H-PS-T_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1672B-T1-H-PS-T_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1672B-T1-H-PS-T_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       109,988 reads  @     24.0 µs/read;   2.50 M reads/minute\n",
      "Finished in 2.68 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,988\n",
      "  Read 1 with adapter:                 108,826 (98.9%)\n",
      "  Read 2 with adapter:                 108,547 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,988 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,186,116 bp\n",
      "  Read 1:    15,103,360 bp\n",
      "  Read 2:    15,082,756 bp\n",
      "Total written (filtered):     25,948,527 bp (86.0%)\n",
      "  Read 1:    13,036,016 bp\n",
      "  Read 2:    12,912,511 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108826 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t1 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t4\t0.0\t1\t1 3\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t532\t0.0\t1\t49 483\n",
      "19\t107992\t0.0\t1\t96198 11794\n",
      "20\t279\t0.0\t1\t44 235\n",
      "31\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108547 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1718.6\t0\t3\n",
      "4\t3\t429.6\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t12\t0.0\t1\t6 6\n",
      "19\t799\t0.0\t1\t131 668\n",
      "20\t107459\t0.0\t2\t104745 2714\n",
      "21\t263\t0.0\t2\t55 208\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1827D-T1-H-PA-T_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1827D-T1-H-PA-T_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1827D-T1-H-PA-T_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1827D-T1-H-PA-T_72_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1827D-T1-H-PA-T_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1827D-T1-H-PA-T_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1827D-T1-H-PA-T_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1827D-T1-H-PA-T_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       121,694 reads  @     23.4 µs/read;   2.57 M reads/minute\n",
      "Finished in 2.87 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            121,694\n",
      "  Read 1 with adapter:                 120,242 (98.8%)\n",
      "  Read 2 with adapter:                 120,273 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       121,694 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,726,153 bp\n",
      "  Read 1:    18,371,984 bp\n",
      "  Read 2:    18,354,169 bp\n",
      "Total written (filtered):     32,037,539 bp (87.2%)\n",
      "  Read 1:    16,087,758 bp\n",
      "  Read 2:    15,949,781 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 120242 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t475.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t578\t0.0\t1\t56 522\n",
      "19\t119345\t0.0\t1\t104361 14984\n",
      "20\t301\t0.0\t1\t46 255\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 120273 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1901.5\t0\t7\n",
      "4\t2\t475.4\t0\t2\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t2\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t15\t0.0\t1\t10 5\n",
      "19\t1146\t0.0\t1\t167 979\n",
      "20\t118793\t0.0\t2\t115547 3246\n",
      "21\t302\t0.0\t2\t66 236\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1831B-TF-H-OA-T_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1831B-TF-H-OA-T_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1831B-TF-H-OA-T_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1831B-TF-H-OA-T_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1831B-TF-H-OA-T_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1831B-TF-H-OA-T_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1831B-TF-H-OA-T_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1831B-TF-H-OA-T_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02       139,113 reads  @     21.5 µs/read;   2.79 M reads/minute\n",
      "Finished in 3.02 s (22 us/read; 2.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            139,113\n",
      "  Read 1 with adapter:                 137,465 (98.8%)\n",
      "  Read 2 with adapter:                 137,374 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       139,113 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,965,916 bp\n",
      "  Read 1:    20,497,134 bp\n",
      "  Read 2:    20,468,782 bp\n",
      "Total written (filtered):     35,607,953 bp (86.9%)\n",
      "  Read 1:    17,885,691 bp\n",
      "  Read 2:    17,722,262 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 137465 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t8\t0.0\t1\t3 5\n",
      "18\t603\t0.0\t1\t88 515\n",
      "19\t136530\t0.0\t1\t119529 17001\n",
      "20\t308\t0.0\t1\t61 247\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 137374 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t2173.6\t0\t3\n",
      "4\t6\t543.4\t0\t6\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t2\t0.0\t1\t2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t10\t0.0\t1\t3 7\n",
      "19\t1082\t0.0\t1\t149 933\n",
      "20\t135926\t0.0\t2\t132880 3046\n",
      "21\t337\t0.0\t2\t77 260\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1833A-TF-H-CN-T_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1833A-TF-H-CN-T_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1833A-TF-H-CN-T_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1833A-TF-H-CN-T_74_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1833A-TF-H-CN-T_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1833A-TF-H-CN-T_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1833A-TF-H-CN-T_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1833A-TF-H-CN-T_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        93,265 reads  @     23.4 µs/read;   2.57 M reads/minute\n",
      "Finished in 2.21 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,265\n",
      "  Read 1 with adapter:                  91,826 (98.5%)\n",
      "  Read 2 with adapter:                  92,100 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        93,265 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,140,139 bp\n",
      "  Read 1:    14,077,585 bp\n",
      "  Read 2:    14,062,554 bp\n",
      "Total written (filtered):     24,554,145 bp (87.3%)\n",
      "  Read 1:    12,333,075 bp\n",
      "  Read 2:    12,221,070 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91826 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t364.3\t0\t1\n",
      "9\t2\t0.4\t0\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t337\t0.0\t1\t48 289\n",
      "19\t91276\t0.0\t1\t75728 15548\n",
      "20\t203\t0.0\t1\t40 163\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 92100 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1457.3\t0\t4\n",
      "4\t2\t364.3\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t602\t0.0\t1\t79 523\n",
      "20\t91273\t0.0\t2\t89024 2249\n",
      "21\t209\t0.0\t2\t33 176\n",
      "39\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1835B-TF-H-OA-T_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1835B-TF-H-OA-T_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1835B-TF-H-OA-T_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1835B-TF-H-OA-T_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1835B-TF-H-OA-T_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1835B-TF-H-OA-T_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1835B-TF-H-OA-T_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1835B-TF-H-OA-T_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       115,481 reads  @     21.9 µs/read;   2.74 M reads/minute\n",
      "Finished in 2.56 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,481\n",
      "  Read 1 with adapter:                 114,131 (98.8%)\n",
      "  Read 2 with adapter:                 114,061 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,481 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,786,745 bp\n",
      "  Read 1:    17,402,647 bp\n",
      "  Read 2:    17,384,098 bp\n",
      "Total written (filtered):     30,338,130 bp (87.2%)\n",
      "  Read 1:    15,234,448 bp\n",
      "  Read 2:    15,103,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 114131 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t6\t0.0\t1\t5 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t2 2\n",
      "16\t5\t0.0\t1\t1 4\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t471\t0.0\t1\t47 424\n",
      "19\t113344\t0.0\t1\t99068 14276\n",
      "20\t291\t0.0\t1\t51 240\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 114061 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1804.4\t0\t8\n",
      "4\t2\t451.1\t0\t2\n",
      "11\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t6\t0.0\t1\t3 3\n",
      "19\t848\t0.0\t1\t140 708\n",
      "20\t112922\t0.0\t2\t110367 2555\n",
      "21\t262\t0.0\t2\t55 207\n",
      "22\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1835D-T1-H-OA-T_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1835D-T1-H-OA-T_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1835D-T1-H-OA-T_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1835D-T1-H-OA-T_76_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1835D-T1-H-OA-T_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1835D-T1-H-OA-T_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1835D-T1-H-OA-T_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1835D-T1-H-OA-T_76_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       128,240 reads  @     22.8 µs/read;   2.63 M reads/minute\n",
      "Finished in 2.95 s (23 us/read; 2.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            128,240\n",
      "  Read 1 with adapter:                 126,870 (98.9%)\n",
      "  Read 2 with adapter:                 126,688 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       128,240 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,456,508 bp\n",
      "  Read 1:    19,236,435 bp\n",
      "  Read 2:    19,220,073 bp\n",
      "Total written (filtered):     33,513,412 bp (87.1%)\n",
      "  Read 1:    16,826,274 bp\n",
      "  Read 2:    16,687,138 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 126870 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t8\t0.0\t1\t8\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t4\t0.0\t1\t4\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t612\t0.0\t1\t63 549\n",
      "19\t125865\t0.0\t1\t111492 14373\n",
      "20\t366\t0.0\t1\t75 291\n",
      "37\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 126688 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t2003.8\t0\t6\n",
      "4\t3\t500.9\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t15\t0.0\t1\t7 8\n",
      "19\t950\t0.0\t1\t151 799\n",
      "20\t125376\t0.0\t2\t122230 3146\n",
      "21\t329\t0.0\t2\t71 258\n",
      "41\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1842D-T1-H-PS-T_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1842D-T1-H-PS-T_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1842D-T1-H-PS-T_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1842D-T1-H-PS-T_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1842D-T1-H-PS-T_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1842D-T1-H-PS-T_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1842D-T1-H-PS-T_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1842D-T1-H-PS-T_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       150,417 reads  @     22.0 µs/read;   2.72 M reads/minute\n",
      "Finished in 3.35 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            150,417\n",
      "  Read 1 with adapter:                 148,518 (98.7%)\n",
      "  Read 2 with adapter:                 148,478 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       150,417 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,118,173 bp\n",
      "  Read 1:    22,574,653 bp\n",
      "  Read 2:    22,543,520 bp\n",
      "Total written (filtered):     39,328,275 bp (87.2%)\n",
      "  Read 1:    19,753,215 bp\n",
      "  Read 2:    19,575,060 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 148518 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t587.6\t0\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t9\t0.0\t1\t8 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t5\t0.0\t1\t4 1\n",
      "17\t10\t0.0\t1\t4 6\n",
      "18\t596\t0.0\t1\t92 504\n",
      "19\t147567\t0.0\t1\t127434 20133\n",
      "20\t323\t0.0\t1\t71 252\n",
      "37\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 148478 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t2350.3\t0\t9\n",
      "4\t3\t587.6\t0\t3\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t6\t0.0\t1\t2 4\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t12\t0.0\t1\t5 7\n",
      "19\t1160\t0.0\t1\t182 978\n",
      "20\t146942\t0.0\t2\t143247 3695\n",
      "21\t340\t0.0\t2\t72 268\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1863A-TF-H-CN-T_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1863A-TF-H-CN-T_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1863A-TF-H-CN-T_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1863A-TF-H-CN-T_78_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1863A-TF-H-CN-T_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1863A-TF-H-CN-T_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1863A-TF-H-CN-T_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1863A-TF-H-CN-T_78_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        98,530 reads  @     21.1 µs/read;   2.84 M reads/minute\n",
      "Finished in 2.11 s (21 us/read; 2.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             98,530\n",
      "  Read 1 with adapter:                  96,771 (98.2%)\n",
      "  Read 2 with adapter:                  97,167 (98.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        98,530 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,749,290 bp\n",
      "  Read 1:    14,876,436 bp\n",
      "  Read 2:    14,872,854 bp\n",
      "Total written (filtered):     25,968,098 bp (87.3%)\n",
      "  Read 1:    13,037,974 bp\n",
      "  Read 2:    12,930,124 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 96771 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t354\t0.0\t1\t40 314\n",
      "19\t96212\t0.0\t1\t77661 18551\n",
      "20\t195\t0.0\t1\t40 155\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 97167 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1539.5\t0\t7\n",
      "4\t1\t384.9\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t4\t0.0\t1\t3 1\n",
      "19\t677\t0.0\t1\t104 573\n",
      "20\t96244\t0.0\t2\t93932 2312\n",
      "21\t228\t0.0\t2\t42 186\n",
      "34\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1864A-TF-H-PS-T_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1864A-TF-H-PS-T_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1864A-TF-H-PS-T_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1864A-TF-H-PS-T_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1864A-TF-H-PS-T_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-1864A-TF-H-PS-T_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1864A-TF-H-PS-T_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-1864A-TF-H-PS-T_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02       106,349 reads  @     20.1 µs/read;   2.98 M reads/minute\n",
      "Finished in 2.16 s (20 us/read; 2.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,349\n",
      "  Read 1 with adapter:                 105,236 (99.0%)\n",
      "  Read 2 with adapter:                 105,111 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,349 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,085,931 bp\n",
      "  Read 1:    16,046,275 bp\n",
      "  Read 2:    16,039,656 bp\n",
      "Total written (filtered):     27,985,132 bp (87.2%)\n",
      "  Read 1:    14,047,012 bp\n",
      "  Read 2:    13,938,120 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 105236 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.4\t0\t3\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t2\t0.0\t1\t2\n",
      "16\t5\t0.0\t1\t0 5\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t393\t0.0\t1\t55 338\n",
      "19\t104568\t0.0\t1\t92877 11691\n",
      "20\t258\t0.0\t1\t54 204\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 105111 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1661.7\t0\t6\n",
      "4\t3\t415.4\t0\t3\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t11\t0.0\t1\t6 5\n",
      "19\t718\t0.0\t1\t118 600\n",
      "20\t104140\t0.0\t2\t101722 2418\n",
      "21\t225\t0.0\t2\t50 175\n",
      "22\t1\t0.0\t2\t1\n",
      "30\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-2199E-TF-H-SS-T_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-2199E-TF-H-SS-T_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-2199E-TF-H-SS-T_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-2199E-TF-H-SS-T_80_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-2199E-TF-H-SS-T_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-2199E-TF-H-SS-T_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-2199E-TF-H-SS-T_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-2199E-TF-H-SS-T_80_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       104,906 reads  @     23.1 µs/read;   2.59 M reads/minute\n",
      "Finished in 2.45 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,906\n",
      "  Read 1 with adapter:                 102,577 (97.8%)\n",
      "  Read 2 with adapter:                 103,301 (98.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       104,906 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,229,046 bp\n",
      "  Read 1:    15,665,827 bp\n",
      "  Read 2:    15,563,219 bp\n",
      "Total written (filtered):     27,215,163 bp (87.1%)\n",
      "  Read 1:    13,717,174 bp\n",
      "  Read 2:    13,497,989 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 102577 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t459\t0.0\t1\t60 399\n",
      "19\t101877\t0.0\t1\t76428 25449\n",
      "20\t225\t0.0\t1\t51 174\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 103301 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1639.2\t0\t5\n",
      "4\t3\t409.8\t0\t3\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t5\t0.0\t1\t2 3\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t852\t0.0\t1\t127 725\n",
      "20\t102148\t0.0\t2\t99683 2465\n",
      "21\t273\t0.0\t2\t52 221\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Control3_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Control3_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Control3_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Control3_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Control3_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Control3_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Control3_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Control3_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        75,641 reads  @     23.5 µs/read;   2.55 M reads/minute\n",
      "Finished in 1.81 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,641\n",
      "  Read 1 with adapter:                  74,816 (98.9%)\n",
      "  Read 2 with adapter:                  74,721 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,641 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,747,827 bp\n",
      "  Read 1:    11,378,742 bp\n",
      "  Read 2:    11,369,085 bp\n",
      "Total written (filtered):     19,832,705 bp (87.2%)\n",
      "  Read 1:     9,957,459 bp\n",
      "  Read 2:     9,875,246 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 74816 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t379\t0.0\t1\t28 351\n",
      "19\t74240\t0.0\t1\t64957 9283\n",
      "20\t190\t0.0\t1\t33 157\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 74721 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1181.9\t0\t2\n",
      "4\t1\t295.5\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t3\t0.0\t1\t0 3\n",
      "19\t681\t0.0\t1\t89 592\n",
      "20\t73853\t0.0\t2\t71939 1914\n",
      "21\t174\t0.0\t2\t32 142\n",
      "30\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Control5_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Control5_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Control5_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Control5_82_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Control5_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Control5_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Control5_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Control5_82_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        81,730 reads  @     22.0 µs/read;   2.72 M reads/minute\n",
      "Finished in 1.83 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,730\n",
      "  Read 1 with adapter:                  80,050 (97.9%)\n",
      "  Read 2 with adapter:                  80,559 (98.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        81,730 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,433,932 bp\n",
      "  Read 1:    12,221,275 bp\n",
      "  Read 2:    12,212,657 bp\n",
      "Total written (filtered):     21,302,709 bp (87.2%)\n",
      "  Read 1:    10,700,620 bp\n",
      "  Read 2:    10,602,089 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 80050 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t319.3\t0\t1\n",
      "9\t2\t0.3\t0\t2\n",
      "12\t4\t0.0\t1\t4\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t392\t0.0\t1\t37 355\n",
      "19\t79455\t0.0\t1\t60566 18889\n",
      "20\t185\t0.0\t1\t27 158\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 80559 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1277.0\t0\t4\n",
      "4\t2\t319.3\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t7\t0.0\t1\t2 5\n",
      "19\t668\t0.0\t1\t88 580\n",
      "20\t79668\t0.0\t2\t77681 1987\n",
      "21\t203\t0.0\t2\t49 154\n",
      "34\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Mock1_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Mock1_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Mock1_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Mock1_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Mock1_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-u5e0evf5/LibV-Mock1_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Mock1_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-cjbv3_po/c4f3e90d-cea4-4c92-aaf3-89260d3df2f0/data/LibV-Mock1_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        74,079 reads  @     25.4 µs/read;   2.36 M reads/minute\n",
      "Finished in 1.91 s (26 us/read; 2.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,079\n",
      "  Read 1 with adapter:                  73,116 (98.7%)\n",
      "  Read 2 with adapter:                  73,151 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        74,079 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,351,222 bp\n",
      "  Read 1:    11,185,006 bp\n",
      "  Read 2:    11,166,216 bp\n",
      "Total written (filtered):     19,499,971 bp (87.2%)\n",
      "  Read 1:     9,796,113 bp\n",
      "  Read 2:     9,703,858 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 73116 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t6\t0.0\t1\t4 2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t421\t0.0\t1\t37 384\n",
      "19\t72503\t0.0\t1\t62825 9678\n",
      "20\t177\t0.0\t1\t27 150\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 73151 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1157.5\t0\t3\n",
      "4\t2\t289.4\t0\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t4\t0.0\t1\t2 2\n",
      "19\t703\t0.0\t1\t100 603\n",
      "20\t72243\t0.0\t2\t70222 2021\n",
      "21\t187\t0.0\t2\t43 144\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibV.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1327A-TF-D-OA-T_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1327A-TF-D-OA-T_14_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1327A-TF-D-OA-T_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1327A-TF-D-OA-T_14_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1327A-TF-D-OA-T_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1327A-TF-D-OA-T_14_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1327A-TF-D-OA-T_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1327A-TF-D-OA-T_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       231,827 reads  @     20.2 µs/read;   2.97 M reads/minute\n",
      "Finished in 4.71 s (20 us/read; 2.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            231,827\n",
      "  Read 1 with adapter:                 230,706 (99.5%)\n",
      "  Read 2 with adapter:                 230,807 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       231,827 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,619,994 bp\n",
      "  Read 1:    34,906,688 bp\n",
      "  Read 2:    34,713,306 bp\n",
      "Total written (filtered):     60,622,264 bp (87.1%)\n",
      "  Read 1:    30,523,839 bp\n",
      "  Read 2:    30,098,425 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 230706 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.9\t0\t2\n",
      "10\t1\t0.2\t1\t1\n",
      "12\t10\t0.0\t1\t10\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t8\t0.0\t1\t6 2\n",
      "17\t11\t0.0\t1\t8 3\n",
      "18\t895\t0.0\t1\t113 782\n",
      "19\t229260\t0.0\t1\t211365 17895\n",
      "20\t512\t0.0\t1\t103 409\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 230807 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t8\t905.6\t0\t8\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t1\t0.1\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t5\t0.0\t1\t3 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "18\t23\t0.0\t1\t16 7\n",
      "19\t1507\t0.0\t1\t236 1271\n",
      "20\t228773\t0.0\t2\t219800 8973\n",
      "21\t486\t0.0\t2\t102 384\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1338C-TF-D-OA-T_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1338C-TF-D-OA-T_15_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1338C-TF-D-OA-T_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1338C-TF-D-OA-T_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1338C-TF-D-OA-T_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1338C-TF-D-OA-T_15_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1338C-TF-D-OA-T_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1338C-TF-D-OA-T_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        86,293 reads  @     24.5 µs/read;   2.45 M reads/minute\n",
      "Finished in 2.14 s (25 us/read; 2.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,293\n",
      "  Read 1 with adapter:                  85,761 (99.4%)\n",
      "  Read 2 with adapter:                  85,829 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        86,293 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,885,994 bp\n",
      "  Read 1:    12,971,773 bp\n",
      "  Read 2:    12,914,221 bp\n",
      "Total written (filtered):     22,540,721 bp (87.1%)\n",
      "  Read 1:    11,342,568 bp\n",
      "  Read 2:    11,198,153 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 85761 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t337.1\t0\t1\n",
      "9\t1\t0.3\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t382\t0.0\t1\t43 339\n",
      "19\t85129\t0.0\t1\t77489 7640\n",
      "20\t233\t0.0\t1\t53 180\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 85829 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1348.3\t0\t1\n",
      "4\t3\t337.1\t0\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t588\t0.0\t1\t98 490\n",
      "20\t85011\t0.0\t2\t81279 3732\n",
      "21\t206\t0.0\t2\t51 155\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1356E-TF-D-MC-T_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1356E-TF-D-MC-T_16_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1356E-TF-D-MC-T_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1356E-TF-D-MC-T_16_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1356E-TF-D-MC-T_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1356E-TF-D-MC-T_16_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1356E-TF-D-MC-T_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1356E-TF-D-MC-T_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       223,793 reads  @     21.1 µs/read;   2.85 M reads/minute\n",
      "Finished in 4.75 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            223,793\n",
      "  Read 1 with adapter:                 222,362 (99.4%)\n",
      "  Read 2 with adapter:                 222,753 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       223,793 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,946,647 bp\n",
      "  Read 1:    33,543,133 bp\n",
      "  Read 2:    33,403,514 bp\n",
      "Total written (filtered):     58,268,954 bp (87.0%)\n",
      "  Read 1:    29,318,967 bp\n",
      "  Read 2:    28,949,987 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 222362 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t2\t0.1\t1\t0 2\n",
      "12\t10\t0.0\t1\t7 3\n",
      "13\t4\t0.0\t1\t3 1\n",
      "14\t6\t0.0\t1\t2 4\n",
      "15\t4\t0.0\t1\t2 2\n",
      "16\t8\t0.0\t1\t6 2\n",
      "17\t17\t0.0\t1\t7 10\n",
      "18\t1057\t0.0\t1\t128 929\n",
      "19\t220685\t0.0\t1\t202021 18664\n",
      "20\t568\t0.0\t1\t105 463\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 222753 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t3496.8\t0\t1\n",
      "4\t20\t874.2\t0\t20\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t8\t0.0\t1\t8\n",
      "18\t21\t0.0\t1\t11 10\n",
      "19\t1618\t0.0\t1\t275 1343\n",
      "20\t220553\t0.0\t2\t211918 8635\n",
      "21\t525\t0.0\t2\t119 406\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1360D-TF-D-MC-T_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1360D-TF-D-MC-T_17_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1360D-TF-D-MC-T_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1360D-TF-D-MC-T_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1360D-TF-D-MC-T_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1360D-TF-D-MC-T_17_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1360D-TF-D-MC-T_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1360D-TF-D-MC-T_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       152,026 reads  @     20.8 µs/read;   2.89 M reads/minute\n",
      "Finished in 3.19 s (21 us/read; 2.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            152,026\n",
      "  Read 1 with adapter:                 151,278 (99.5%)\n",
      "  Read 2 with adapter:                 151,427 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       152,026 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,665,020 bp\n",
      "  Read 1:    22,881,766 bp\n",
      "  Read 2:    22,783,254 bp\n",
      "Total written (filtered):     39,763,373 bp (87.1%)\n",
      "  Read 1:    20,007,794 bp\n",
      "  Read 2:    19,755,579 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 151278 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t3\t0.0\t1\t3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t619\t0.0\t1\t74 545\n",
      "19\t150267\t0.0\t1\t140233 10034\n",
      "20\t376\t0.0\t1\t63 313\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 151427 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t6\t593.9\t0\t6\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t0 4\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t12\t0.0\t1\t9 3\n",
      "19\t1028\t0.0\t1\t171 857\n",
      "20\t150041\t0.0\t2\t144418 5623\n",
      "21\t329\t0.0\t2\t75 254\n",
      "22\t2\t0.0\t2\t0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1610D-TF-D-OA-T_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1610D-TF-D-OA-T_18_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1610D-TF-D-OA-T_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1610D-TF-D-OA-T_18_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1610D-TF-D-OA-T_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1610D-TF-D-OA-T_18_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1610D-TF-D-OA-T_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1610D-TF-D-OA-T_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:06       358,190 reads  @     18.1 µs/read;   3.32 M reads/minute\n",
      "Finished in 6.50 s (18 us/read; 3.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            358,190\n",
      "  Read 1 with adapter:                 355,750 (99.3%)\n",
      "  Read 2 with adapter:                 356,171 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       358,190 (100.0%)\n",
      "\n",
      "Total basepairs processed:   107,726,868 bp\n",
      "  Read 1:    53,980,406 bp\n",
      "  Read 2:    53,746,462 bp\n",
      "Total written (filtered):     93,847,777 bp (87.1%)\n",
      "  Read 1:    47,222,259 bp\n",
      "  Read 2:    46,625,518 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 355750 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t87.4\t0\t1\n",
      "9\t5\t1.4\t0\t5\n",
      "10\t2\t0.3\t1\t2\n",
      "11\t2\t0.1\t1\t2\n",
      "12\t13\t0.0\t1\t11 2\n",
      "13\t5\t0.0\t1\t4 1\n",
      "14\t5\t0.0\t1\t1 4\n",
      "15\t5\t0.0\t1\t5\n",
      "16\t8\t0.0\t1\t3 5\n",
      "17\t22\t0.0\t1\t8 14\n",
      "18\t1716\t0.0\t1\t208 1508\n",
      "19\t353033\t0.0\t1\t325703 27330\n",
      "20\t932\t0.0\t1\t178 754\n",
      "31\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 356171 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t14\t1399.2\t0\t14\n",
      "10\t1\t0.3\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t4\t0.0\t1\t2 2\n",
      "16\t10\t0.0\t1\t4 6\n",
      "17\t13\t0.0\t1\t6 7\n",
      "18\t36\t0.0\t1\t16 20\n",
      "19\t2960\t0.0\t1\t434 2526\n",
      "20\t352206\t0.0\t2\t339109 13097\n",
      "21\t918\t0.0\t2\t219 699\n",
      "22\t1\t0.0\t2\t1\n",
      "24\t1\t0.0\t2\t0 1\n",
      "25\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1666E-T1-H-PA-T_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1666E-T1-H-PA-T_19_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1666E-T1-H-PA-T_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1666E-T1-H-PA-T_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1666E-T1-H-PA-T_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1666E-T1-H-PA-T_19_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1666E-T1-H-PA-T_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1666E-T1-H-PA-T_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[      8<----] 00:00:13       778,510 reads  @     17.7 µs/read;   3.39 M reads/minute\n",
      "Finished in 13.81 s (18 us/read; 3.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            778,510\n",
      "  Read 1 with adapter:                 773,703 (99.4%)\n",
      "  Read 2 with adapter:                 774,405 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       778,510 (100.0%)\n",
      "\n",
      "Total basepairs processed:   234,503,894 bp\n",
      "  Read 1:   117,507,014 bp\n",
      "  Read 2:   116,996,880 bp\n",
      "Total written (filtered):    204,323,637 bp (87.1%)\n",
      "  Read 1:   102,808,962 bp\n",
      "  Read 2:   101,514,675 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 773703 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t11.9\t0\t1\n",
      "9\t4\t3.0\t0\t4\n",
      "10\t3\t0.7\t1\t3\n",
      "11\t4\t0.2\t1\t2 2\n",
      "12\t16\t0.0\t1\t15 1\n",
      "13\t12\t0.0\t1\t9 3\n",
      "14\t4\t0.0\t1\t1 3\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t15\t0.0\t1\t10 5\n",
      "17\t37\t0.0\t1\t22 15\n",
      "18\t3980\t0.0\t1\t391 3589\n",
      "19\t767566\t0.0\t1\t696594 70972\n",
      "20\t2056\t0.0\t1\t353 1703\n",
      "21\t1\t0.0\t1\t0 1\n",
      "31\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "47\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 774405 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t27\t3041.1\t0\t27\n",
      "8\t1\t11.9\t0\t1\n",
      "9\t1\t3.0\t0\t1\n",
      "10\t2\t0.7\t1\t2\n",
      "11\t4\t0.2\t1\t4\n",
      "12\t5\t0.0\t1\t3 2\n",
      "13\t9\t0.0\t1\t5 4\n",
      "14\t11\t0.0\t1\t3 8\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t16\t0.0\t1\t12 4\n",
      "18\t68\t0.0\t1\t25 43\n",
      "19\t6919\t0.0\t1\t926 5993\n",
      "20\t765429\t0.0\t2\t730654 34775\n",
      "21\t1904\t0.0\t2\t418 1486\n",
      "22\t1\t0.0\t2\t1\n",
      "33\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1835A-TF-D-OA-T_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1835A-TF-D-OA-T_20_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1835A-TF-D-OA-T_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1835A-TF-D-OA-T_20_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1835A-TF-D-OA-T_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1835A-TF-D-OA-T_20_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1835A-TF-D-OA-T_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1835A-TF-D-OA-T_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       142,530 reads  @     21.6 µs/read;   2.78 M reads/minute\n",
      "Finished in 3.11 s (22 us/read; 2.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,530\n",
      "  Read 1 with adapter:                 141,852 (99.5%)\n",
      "  Read 2 with adapter:                 141,889 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       142,530 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,486,457 bp\n",
      "  Read 1:    21,289,790 bp\n",
      "  Read 2:    21,196,667 bp\n",
      "Total written (filtered):     36,954,646 bp (87.0%)\n",
      "  Read 1:    18,594,934 bp\n",
      "  Read 2:    18,359,712 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 141852 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t3\t0.0\t1\t2 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t9\t0.0\t1\t6 3\n",
      "18\t608\t0.0\t1\t62 546\n",
      "19\t140891\t0.0\t1\t130286 10605\n",
      "20\t333\t0.0\t1\t72 261\n",
      "21\t1\t0.0\t1\t0 1\n",
      "37\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 141889 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t7\t556.8\t0\t7\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t12\t0.0\t1\t5 7\n",
      "19\t961\t0.0\t1\t163 798\n",
      "20\t140584\t0.0\t2\t135423 5161\n",
      "21\t317\t0.0\t2\t70 247\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1864B-TF-D-PS-T_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1864B-TF-D-PS-T_21_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1864B-TF-D-PS-T_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1864B-TF-D-PS-T_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1864B-TF-D-PS-T_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1864B-TF-D-PS-T_21_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1864B-TF-D-PS-T_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1864B-TF-D-PS-T_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04       212,740 reads  @     21.9 µs/read;   2.74 M reads/minute\n",
      "Finished in 4.69 s (22 us/read; 2.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            212,740\n",
      "  Read 1 with adapter:                 211,188 (99.3%)\n",
      "  Read 2 with adapter:                 211,840 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       212,740 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,630,134 bp\n",
      "  Read 1:    31,884,766 bp\n",
      "  Read 2:    31,745,368 bp\n",
      "Total written (filtered):     55,382,246 bp (87.0%)\n",
      "  Read 1:    27,872,672 bp\n",
      "  Read 2:    27,509,574 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 211188 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t2\t0.1\t1\t1 1\n",
      "12\t7\t0.0\t1\t6 1\n",
      "13\t3\t0.0\t1\t0 3\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t8\t0.0\t1\t4 4\n",
      "18\t831\t0.0\t1\t100 731\n",
      "19\t209866\t0.0\t1\t191360 18506\n",
      "20\t460\t0.0\t1\t86 374\n",
      "31\t1\t0.0\t1\t1\n",
      "39\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 211840 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t3324.1\t0\t1\n",
      "4\t8\t831.0\t0\t8\n",
      "10\t1\t0.2\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t20\t0.0\t1\t9 11\n",
      "19\t1290\t0.0\t1\t215 1075\n",
      "20\t210005\t0.0\t2\t201869 8136\n",
      "21\t509\t0.0\t2\t112 397\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1Flat-DD-OA-T_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1Flat-DD-OA-T_22_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1Flat-DD-OA-T_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1Flat-DD-OA-T_22_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1Flat-DD-OA-T_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-1Flat-DD-OA-T_22_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1Flat-DD-OA-T_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-1Flat-DD-OA-T_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       288,667 reads  @     17.8 µs/read;   3.37 M reads/minute\n",
      "Finished in 5.18 s (18 us/read; 3.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            288,667\n",
      "  Read 1 with adapter:                 287,240 (99.5%)\n",
      "  Read 2 with adapter:                 287,343 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       288,667 (100.0%)\n",
      "\n",
      "Total basepairs processed:    86,971,801 bp\n",
      "  Read 1:    43,566,847 bp\n",
      "  Read 2:    43,404,954 bp\n",
      "Total written (filtered):     75,769,856 bp (87.1%)\n",
      "  Read 1:    38,110,036 bp\n",
      "  Read 2:    37,659,820 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 287240 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t1.1\t0\t3\n",
      "10\t1\t0.3\t1\t0 1\n",
      "11\t3\t0.1\t1\t2 1\n",
      "12\t10\t0.0\t1\t10\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t18\t0.0\t1\t8 10\n",
      "18\t1246\t0.0\t1\t152 1094\n",
      "19\t285228\t0.0\t1\t265138 20090\n",
      "20\t716\t0.0\t1\t143 573\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 287343 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t4510.4\t0\t1\n",
      "4\t15\t1127.6\t0\t15\n",
      "9\t1\t1.1\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t5\t0.0\t1\t1 4\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t6\t0.0\t1\t1 5\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t18\t0.0\t1\t6 12\n",
      "19\t1973\t0.0\t1\t336 1637\n",
      "20\t284665\t0.0\t2\t274621 10044\n",
      "21\t645\t0.0\t2\t149 496\n",
      "23\t1\t0.0\t2\t1\n",
      "26\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-2Flat-DD-OA-T_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-2Flat-DD-OA-T_23_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-2Flat-DD-OA-T_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-2Flat-DD-OA-T_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-2Flat-DD-OA-T_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-2Flat-DD-OA-T_23_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-2Flat-DD-OA-T_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-2Flat-DD-OA-T_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[     8<-----] 00:00:12       608,563 reads  @     20.1 µs/read;   2.99 M reads/minute\n",
      "Finished in 12.26 s (20 us/read; 2.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            608,563\n",
      "  Read 1 with adapter:                 605,462 (99.5%)\n",
      "  Read 2 with adapter:                 605,871 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       608,563 (100.0%)\n",
      "\n",
      "Total basepairs processed:   183,329,138 bp\n",
      "  Read 1:    91,884,420 bp\n",
      "  Read 2:    91,444,718 bp\n",
      "Total written (filtered):    159,712,898 bp (87.1%)\n",
      "  Read 1:    80,382,159 bp\n",
      "  Read 2:    79,330,739 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 605462 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t4\t2.3\t0\t4\n",
      "10\t4\t0.6\t1\t4\n",
      "11\t3\t0.1\t1\t2 1\n",
      "12\t20\t0.0\t1\t17 3\n",
      "13\t7\t0.0\t1\t6 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t10\t0.0\t1\t4 6\n",
      "16\t16\t0.0\t1\t5 11\n",
      "17\t43\t0.0\t1\t18 25\n",
      "18\t2634\t0.0\t1\t304 2330\n",
      "19\t601176\t0.0\t1\t552024 49152\n",
      "20\t1539\t0.0\t1\t276 1263\n",
      "21\t2\t0.0\t1\t1 1\n",
      "32\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 605871 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t9508.8\t0\t3\n",
      "4\t29\t2377.2\t0\t29\n",
      "11\t4\t0.1\t1\t4\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t7\t0.0\t1\t2 5\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t11\t0.0\t1\t6 5\n",
      "18\t44\t0.0\t1\t28 16\n",
      "19\t3989\t0.0\t1\t642 3347\n",
      "20\t600469\t0.0\t2\t576858 23611\n",
      "21\t1304\t0.0\t2\t296 1008\n",
      "22\t1\t0.0\t2\t1\n",
      "24\t2\t0.0\t2\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-4Flat-DD-PS-T_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-4Flat-DD-PS-T_24_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-4Flat-DD-PS-T_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-4Flat-DD-PS-T_24_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-4Flat-DD-PS-T_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-4Flat-DD-PS-T_24_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-4Flat-DD-PS-T_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-4Flat-DD-PS-T_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[   8=-------] 00:00:08       479,159 reads  @     17.3 µs/read;   3.47 M reads/minute\n",
      "Finished in 8.33 s (17 us/read; 3.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            479,159\n",
      "  Read 1 with adapter:                 476,538 (99.5%)\n",
      "  Read 2 with adapter:                 476,981 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       479,159 (100.0%)\n",
      "\n",
      "Total basepairs processed:   142,628,514 bp\n",
      "  Read 1:    71,460,076 bp\n",
      "  Read 2:    71,168,438 bp\n",
      "Total written (filtered):    124,038,386 bp (87.0%)\n",
      "  Read 1:    62,406,898 bp\n",
      "  Read 2:    61,631,488 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 476538 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t1.8\t0\t1\n",
      "11\t2\t0.1\t1\t1 1\n",
      "12\t8\t0.0\t1\t7 1\n",
      "13\t5\t0.0\t1\t2 3\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t23\t0.0\t1\t15 8\n",
      "18\t2010\t0.0\t1\t220 1790\n",
      "19\t473356\t0.0\t1\t432279 41077\n",
      "20\t1121\t0.0\t1\t215 906\n",
      "26\t1\t0.0\t1\t1\n",
      "32\t1\t0.0\t1\t1\n",
      "37\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 476981 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t7486.9\t0\t1\n",
      "4\t17\t1871.7\t0\t17\n",
      "9\t1\t1.8\t0\t1\n",
      "11\t1\t0.1\t1\t1\n",
      "13\t6\t0.0\t1\t4 2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t6\t0.0\t1\t4 2\n",
      "17\t8\t0.0\t1\t7 1\n",
      "18\t40\t0.0\t1\t21 19\n",
      "19\t3216\t0.0\t1\t495 2721\n",
      "20\t472645\t0.0\t2\t452210 20435\n",
      "21\t1034\t0.0\t2\t221 813\n",
      "22\t1\t0.0\t2\t0 1\n",
      "30\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-7Flat-DD-PS-T_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-7Flat-DD-PS-T_25_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-7Flat-DD-PS-T_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-7Flat-DD-PS-T_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-7Flat-DD-PS-T_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-7Flat-DD-PS-T_25_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-7Flat-DD-PS-T_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-7Flat-DD-PS-T_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        96,562 reads  @     23.5 µs/read;   2.55 M reads/minute\n",
      "Finished in 2.31 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,562\n",
      "  Read 1 with adapter:                  96,045 (99.5%)\n",
      "  Read 2 with adapter:                  96,116 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        96,562 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,939,630 bp\n",
      "  Read 1:    14,498,846 bp\n",
      "  Read 2:    14,440,784 bp\n",
      "Total written (filtered):     25,193,281 bp (87.1%)\n",
      "  Read 1:    12,674,256 bp\n",
      "  Read 2:    12,519,025 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 96045 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t447\t0.0\t1\t62 385\n",
      "19\t95348\t0.0\t1\t88746 6602\n",
      "20\t239\t0.0\t1\t49 190\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 96116 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t6\t377.2\t0\t6\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t10\t0.0\t1\t7 3\n",
      "19\t656\t0.0\t1\t106 550\n",
      "20\t95189\t0.0\t2\t91688 3501\n",
      "21\t248\t0.0\t2\t58 190\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-Mock1_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-Mock1_26_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-Mock1_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-Mock1_26_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-Mock1_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-Mock1_26_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-Mock1_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-Mock1_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[    8<------] 00:00:09       499,633 reads  @     18.7 µs/read;   3.22 M reads/minute\n",
      "Finished in 9.36 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            499,633\n",
      "  Read 1 with adapter:                 496,562 (99.4%)\n",
      "  Read 2 with adapter:                 496,641 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       499,633 (100.0%)\n",
      "\n",
      "Total basepairs processed:   150,512,867 bp\n",
      "  Read 1:    75,442,246 bp\n",
      "  Read 2:    75,070,621 bp\n",
      "Total written (filtered):    131,151,441 bp (87.1%)\n",
      "  Read 1:    66,009,534 bp\n",
      "  Read 2:    65,141,907 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 496562 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t1951.7\t0\t2\n",
      "9\t2\t1.9\t0\t2\n",
      "10\t1\t0.5\t1\t1\n",
      "11\t4\t0.1\t1\t2 2\n",
      "12\t19\t0.0\t1\t18 1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t7\t0.0\t1\t3 4\n",
      "15\t5\t0.0\t1\t2 3\n",
      "16\t13\t0.0\t1\t5 8\n",
      "17\t30\t0.0\t1\t18 12\n",
      "18\t2921\t0.0\t1\t305 2616\n",
      "19\t492277\t0.0\t1\t446124 46153\n",
      "20\t1272\t0.0\t1\t251 1021\n",
      "22\t2\t0.0\t1\t1 1\n",
      "28\t1\t0.0\t1\t1\n",
      "36\t1\t0.0\t1\t1\n",
      "42\t1\t0.0\t1\t1\n",
      "43\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 496641 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t7806.8\t0\t1\n",
      "4\t17\t1951.7\t0\t17\n",
      "8\t1\t7.6\t0\t1\n",
      "9\t2\t1.9\t0\t2\n",
      "11\t3\t0.1\t1\t3\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t6\t0.0\t1\t4 2\n",
      "14\t5\t0.0\t1\t1 4\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t6\t0.0\t1\t2 4\n",
      "17\t13\t0.0\t1\t10 3\n",
      "18\t54\t0.0\t1\t27 27\n",
      "19\t4730\t0.0\t1\t629 4101\n",
      "20\t490560\t0.0\t2\t467826 22734\n",
      "21\t1234\t0.0\t2\t269 965\n",
      "22\t2\t0.0\t2\t2\n",
      "24\t1\t0.0\t2\t1\n",
      "26\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-PCR-Control_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-PCR-Control_27_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-PCR-Control_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-PCR-Control_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-PCR-Control_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-w2izk2vv/LibY-PCR-Control_27_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-PCR-Control_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-93xd7pex/349874d0-248a-4763-80c4-97f427a59fd5/data/LibY-PCR-Control_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8<----------] 00:00:00           926 reads  @    333.9 µs/read;   0.18 M reads/minute\n",
      "Finished in 0.36 s (388 us/read; 0.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                926\n",
      "  Read 1 with adapter:                     918 (99.1%)\n",
      "  Read 2 with adapter:                     918 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):           926 (100.0%)\n",
      "\n",
      "Total basepairs processed:       278,301 bp\n",
      "  Read 1:       139,497 bp\n",
      "  Read 2:       138,804 bp\n",
      "Total written (filtered):        242,506 bp (87.1%)\n",
      "  Read 1:       122,058 bp\n",
      "  Read 2:       120,448 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 918 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 3\n",
      "19\t915\t0.0\t1\t680 235\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 918 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t4\t0.0\t1\t0 4\n",
      "20\t914\t0.0\t2\t806 108\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibY.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "\n",
    "#trim PCR primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibS.qza \\\n",
    "--p-cores 2 \\\n",
    "--p-front-f GTGYCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACNVGGGTWTCTAAT \\\n",
    "--o-trimmed-sequences folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibS.qza \\\n",
    "--verbose\n",
    "\n",
    "\n",
    "#trim PCR primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibT.qza \\\n",
    "--p-cores 2 \\\n",
    "--p-front-f GTGYCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACNVGGGTWTCTAAT \\\n",
    "--o-trimmed-sequences folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibT.qza \\\n",
    "--verbose\n",
    "\n",
    "\n",
    "#trim PCR primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibU.qza \\\n",
    "--p-cores 2 \\\n",
    "--p-front-f GTGYCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACNVGGGTWTCTAAT \\\n",
    "--o-trimmed-sequences folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibU.qza \\\n",
    "--verbose\n",
    "\n",
    "\n",
    "#trim PCR primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibV.qza \\\n",
    "--p-cores 2 \\\n",
    "--p-front-f GTGYCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACNVGGGTWTCTAAT \\\n",
    "--o-trimmed-sequences folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibV.qza \\\n",
    "--verbose\n",
    "\n",
    "\n",
    "#trim PCR primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences folder_Apprill_transmissionTissue2019/Import_Apprill_2019TransmissionExpTissue_LibY.qza \\\n",
    "--p-cores 2 \\\n",
    "--p-front-f GTGYCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACNVGGGTWTCTAAT \\\n",
    "--o-trimmed-sequences folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibY.qza \\\n",
    "--verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibS..qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibT.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibU.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibV.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibY.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibS.qza \\\n",
    "--o-visualization folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibS.\n",
    "\n",
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibT.qza \\\n",
    "--o-visualization folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibT.qzv\n",
    "\n",
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibU.qza \\\n",
    "--o-visualization folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibU.qzv\n",
    "\n",
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibV.qza \\\n",
    "--o-visualization folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibV.qzv\n",
    "\n",
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibY.qza \\\n",
    "--o-visualization folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_LibY.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibS.qza\n",
      "Saved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibS.qza\n",
      "Saved SampleData[DADA2Stats] to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibS.qza\n",
      "Saved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibT.qza\n",
      "Saved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibT.qza\n",
      "Saved SampleData[DADA2Stats] to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibT.qza\n",
      "Saved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibU.qza\n",
      "Saved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibU.qza\n",
      "Saved SampleData[DADA2Stats] to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibU.qza\n",
      "Saved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibV.qza\n",
      "Saved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibV.qza\n",
      "Saved SampleData[DADA2Stats] to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibV.qza\n",
      "Saved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibY.qza\n",
      "Saved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibY.qza\n",
      "Saved SampleData[DADA2Stats] to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibY.qza\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "\n",
    "for f1 in folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_Lib*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime dada2 denoise-single --p-max-ee 3 --i-demultiplexed-seqs $f1 --p-trunc-len 0 --p-n-threads 10 --o-representative-sequences folder_Apprill_transmissionTissue2019/rep_$fn.qza --o-table folder_Apprill_transmissionTissue2019/table_$fn.qza --o-denoising-stats folder_Apprill_transmissionTissue2019/dada_$fn.qza \n",
    "#&> DADA2_denoising.log\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibS.qza\n",
      "Saved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibS.qza\n",
      "Saved SampleData[DADA2Stats] to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibS.qza\n",
      "Saved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibT.qza\n",
      "Saved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibT.qza\n",
      "Saved SampleData[DADA2Stats] to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibT.qza\n",
      "Saved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibU.qza\n",
      "Saved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibU.qza\n",
      "Saved SampleData[DADA2Stats] to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibU.qza\n",
      "Saved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibV.qza\n",
      "Saved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibV.qza\n",
      "Saved SampleData[DADA2Stats] to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibV.qza\n",
      "Saved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibY.qza\n",
      "Saved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibY.qza\n",
      "Saved SampleData[DADA2Stats] to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibY.qza\n"
     ]
    }
   ],
   "source": [
    "#%%bash\n",
    "\n",
    "#for f1 in folder_Apprill_transmissionTissue2019/trim_Apprill_2019TransmissionExpTissu_Lib*.qza\n",
    "#do\n",
    "#newfile=\"$(basename $f1)\"\n",
    "#fn=${newfile%.qza};\n",
    "\n",
    "#qiime dada2 denoise-paired --i-demultiplexed-seqs $f1 --p-trunc-len-f 130 --p-trunc-len-r  130 --p-n-threads 10 --o-representative-sequences folder_Apprill_transmissionTissue2019/rep_$fn.qza --o-table folder_Apprill_transmissionTissue2019/table_$fn.qza --o-denoising-stats folder_Apprill_transmissionTissue2019/dada_$fn.qza \n",
    "\n",
    "#done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table merge-seqs \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibS.qza \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibT.qza \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibU.qza \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibV.qza \\\n",
    "--i-data folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu_LibY.qza \\\n",
    "--o-merged-data folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu.qza\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "--m-input-file folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu.qza \\\n",
    "--o-visualization folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table_Apprill_2019TransmissionExpTissu.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table merge \\\n",
    " --i-tables folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibS.qza \\\n",
    " --i-tables folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibT.qza \\\n",
    " --i-tables folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibU.qza \\\n",
    " --i-tables folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibV.qza \\\n",
    " --i-tables folder_Apprill_transmissionTissue2019/table_trim_Apprill_2019TransmissionExpTissu_LibY.qza \\\n",
    " --o-merged-table folder_Apprill_transmissionTissue2019/table_Apprill_2019TransmissionExpTissu.qza\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/table_Apprill_2019TransmissionExpTissu.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file folder_Apprill_transmissionTissue2019/table_Apprill_2019TransmissionExpTissu.qza \\\n",
    "  --o-visualization folder_Apprill_transmissionTissue2019/table_Apprill_2019TransmissionExpTissu.qzv\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibS.qzv\n",
      "Saved Visualization to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibT.qzv\n",
      "Saved Visualization to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibU.qzv\n",
      "Saved Visualization to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibV.qzv\n",
      "Saved Visualization to: folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_LibY.qzv\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for f1 in folder_Apprill_transmissionTissue2019/dada_trim_Apprill_2019TransmissionExpTissu_Lib*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime metadata tabulate \\\n",
    "  --m-input-file $f1 \\\n",
    "  --o-visualization folder_Apprill_transmissionTissue2019/$fn\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Taxonomy] to: folder_Apprill_transmissionTissue2019/taxonomy_Apprill_2019TransmissionExpTissu.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier /space/home/rosales/databases/SILVA_DBs/silva-138-99-nb-classifier.qza \\\n",
    "  --i-reads folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu.qza \\\n",
    "  --p-n-jobs -7 \\\n",
    "  --o-classification folder_Apprill_transmissionTissue2019/taxonomy_Apprill_2019TransmissionExpTissu.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: folder_Apprill_transmissionTissue2019/rep-BacArc_Apprill_2019TransmissionExpTissu.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime taxa filter-seqs \\\n",
    "--i-sequences folder_Apprill_transmissionTissue2019/rep_trim_Apprill_2019TransmissionExpTissu.qza \\\n",
    "--i-taxonomy folder_Apprill_transmissionTissue2019/taxonomy_Apprill_2019TransmissionExpTissu.qza \\\n",
    "--p-exclude mitochondria,chloroplast \\\n",
    "--p-include d__Bacteria,d__Archaea \\\n",
    "--o-filtered-sequences folder_Apprill_transmissionTissue2019/rep-BacArc_Apprill_2019TransmissionExpTissu.qza\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: folder_Apprill_transmissionTissue2019/table-BacArc_Apprill_transmissionTissue2019.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime taxa filter-table \\\n",
    "--i-table folder_Apprill_transmissionTissue2019/table_Apprill_2019TransmissionExpTissu.qza \\\n",
    "--i-taxonomy folder_Apprill_transmissionTissue2019/taxonomy_Apprill_2019TransmissionExpTissu.qza \\\n",
    "--p-exclude mitochondria,chloroplast \\\n",
    "--p-include d__Bacteria,d__Archaea \\\n",
    "--o-filtered-table folder_Apprill_transmissionTissue2019/table-BacArc_Apprill_transmissionTissue2019.qza\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/table-BacArc_Apprill_transmissionTissue2019.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file folder_Apprill_transmissionTissue2019/table-BacArc_Apprill_transmissionTissue2019.qza \\\n",
    "  --o-visualization folder_Apprill_transmissionTissue2019/table-BacArc_Apprill_transmissionTissue2019.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/taxonomy_Apprill_2019TransmissionExpTissu.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file folder_Apprill_transmissionTissue2019/taxonomy_Apprill_2019TransmissionExpTissu.qza\\\n",
    "  --o-visualization folder_Apprill_transmissionTissue2019/taxonomy_Apprill_2019TransmissionExpTissu.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: folder_Apprill_transmissionTissue2019/taxa-bar-Apprill_2019TransmissionExpTissue.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime taxa barplot \\\n",
    "  --i-table folder_Apprill_transmissionTissue2019/table-BacArc_Apprill_transmissionTissue2019.qza \\\n",
    "  --i-taxonomy  folder_Apprill_transmissionTissue2019/taxonomy_Apprill_2019TransmissionExpTissu.qza \\\n",
    "  --m-metadata-file  metadata/metadata_Apprill_2019TransmissionExpTissue.txt \\\n",
    "  --o-visualization  folder_Apprill_transmissionTissue2019/taxa-bar-Apprill_2019TransmissionExpTissue.qzv"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
