{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Voss_ENS_LBTS1/201009_demultiplexed\n",
      "/space/home/rosales/projects/2020_SCTLD_meta-analysis\n"
     ]
    }
   ],
   "source": [
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Voss_ENS_LBTS1/201009_demultiplexed/\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_Voss_ENS_LBTS1.txt\n",
    "%cd /space/home/rosales/projects/2020_SCTLD_meta-analysis/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as ps\n",
    "import numpy as np\n",
    "\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_Voss_ENS_LBTS1.txt',\n",
    "                                    names=header_list)\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_S\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "dir_path=\"/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Voss_ENS_LBTS1/201009_demultiplexed/\"\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Voss_ENS_LBTS1.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir Folder_Voss_ENS_LBTS1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/miniconda3/envs/qiime2-2020.11/bin/qiime\r\n"
     ]
    }
   ],
   "source": [
    "!which qiime"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Voss_ENS_LBTS1.csv as PairedEndFastqManifestPhred33V2 to Folder_Voss_ENS_LBTS1/Import_Voss_ENS_LBTS1.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime tools import \\\n",
    "--output-path Folder_Voss_ENS_LBTS1/Import_Voss_ENS_LBTS1.qza \\\n",
    "--type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "--input-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Voss_ENS_LBTS1.csv \\\n",
    "--input-format PairedEndFastqManifestPhred33V2\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/miniconda3/envs/qiime2-2020.11/lib/python3.6/site-packages/seaborn/distributions.py:2551: FutureWarning: `distplot` is a deprecated function and will be removed in a future version. Please adapt your code to use either `displot` (a figure-level function with similar flexibility) or `histplot` (an axes-level function for histograms).\n",
      "  warnings.warn(msg, FutureWarning)\n",
      "/space/home/rosales/miniconda3/envs/qiime2-2020.11/lib/python3.6/site-packages/seaborn/distributions.py:2551: FutureWarning: `distplot` is a deprecated function and will be removed in a future version. Please adapt your code to use either `displot` (a figure-level function with similar flexibility) or `histplot` (an axes-level function for histograms).\n",
      "  warnings.warn(msg, FutureWarning)\n",
      "\u001b[32mSaved Visualization to: Folder_Voss_ENS_LBTS1/Import_Voss_ENS_LBTS1.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data  Folder_Voss_ENS_LBTS1/Import_Voss_ENS_LBTS1.qza \\\n",
    "--o-visualization  Folder_Voss_ENS_LBTS1/Import_Voss_ENS_LBTS1.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/1_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/1_226_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/1_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/1_226_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/1_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/1_226_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/1_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/1_226_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        19,624 reads  @     42.2 µs/read;   1.42 M reads/minute\n",
      "Finished in 0.86 s (44 us/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,624\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        19,624 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,926,448 bp\n",
      "  Read 1:     2,963,224 bp\n",
      "  Read 2:     2,963,224 bp\n",
      "Total written (filtered):      5,926,414 bp (100.0%)\n",
      "  Read 1:     2,963,200 bp\n",
      "  Read 2:     2,963,214 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t306.6\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t306.6\t0\t2\n",
      "4\t1\t76.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/10_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/10_220_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/10_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/10_220_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/10_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/10_220_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/10_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/10_220_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        42,712 reads  @     27.0 µs/read;   2.22 M reads/minute\n",
      "Finished in 1.19 s (28 us/read; 2.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,712\n",
      "  Read 1 with adapter:                      19 (0.0%)\n",
      "  Read 2 with adapter:                      16 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,899,024 bp\n",
      "  Read 1:     6,449,512 bp\n",
      "  Read 2:     6,449,512 bp\n",
      "Total written (filtered):     12,898,916 bp (100.0%)\n",
      "  Read 1:     6,449,454 bp\n",
      "  Read 2:     6,449,462 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t667.4\t0\t18\n",
      "4\t1\t166.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t667.4\t0\t14\n",
      "4\t2\t166.8\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/11_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/11_221_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/11_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/11_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/11_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/11_221_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/11_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/11_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        43,933 reads  @     28.2 µs/read;   2.13 M reads/minute\n",
      "Finished in 1.27 s (29 us/read; 2.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,933\n",
      "  Read 1 with adapter:                      22 (0.1%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        43,933 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,267,766 bp\n",
      "  Read 1:     6,633,883 bp\n",
      "  Read 2:     6,633,883 bp\n",
      "Total written (filtered):     13,267,660 bp (100.0%)\n",
      "  Read 1:     6,633,811 bp\n",
      "  Read 2:     6,633,849 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t686.5\t0\t19\n",
      "4\t2\t171.6\t0\t2\n",
      "7\t1\t2.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t686.5\t0\t10\n",
      "4\t1\t171.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/12_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/12_222_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/12_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/12_222_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/12_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/12_222_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/12_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/12_222_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        34,223 reads  @     30.5 µs/read;   1.97 M reads/minute\n",
      "Finished in 1.08 s (32 us/read; 1.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,223\n",
      "  Read 1 with adapter:                      16 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,223 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,335,346 bp\n",
      "  Read 1:     5,167,673 bp\n",
      "  Read 2:     5,167,673 bp\n",
      "Total written (filtered):     10,335,256 bp (100.0%)\n",
      "  Read 1:     5,167,623 bp\n",
      "  Read 2:     5,167,633 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t534.7\t0\t14\n",
      "4\t2\t133.7\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t534.7\t0\t9\n",
      "4\t2\t133.7\t0\t2\n",
      "5\t1\t33.4\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/13_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/13_223_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/13_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/13_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/13_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/13_223_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/13_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/13_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,281 reads  @     28.7 µs/read;   2.09 M reads/minute\n",
      "Finished in 1.45 s (29 us/read; 2.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,281\n",
      "  Read 1 with adapter:                      20 (0.0%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,281 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,882,862 bp\n",
      "  Read 1:     7,441,431 bp\n",
      "  Read 2:     7,441,431 bp\n",
      "Total written (filtered):     14,882,763 bp (100.0%)\n",
      "  Read 1:     7,441,369 bp\n",
      "  Read 2:     7,441,394 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t770.0\t0\t19\n",
      "5\t1\t48.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t770.0\t0\t8\n",
      "4\t2\t192.5\t0\t2\n",
      "5\t1\t48.1\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/14_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/14_224_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/14_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/14_224_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/14_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/14_224_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/14_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/14_224_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        45,003 reads  @     27.0 µs/read;   2.22 M reads/minute\n",
      "Finished in 1.25 s (28 us/read; 2.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,003\n",
      "  Read 1 with adapter:                      17 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,003 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,590,906 bp\n",
      "  Read 1:     6,795,453 bp\n",
      "  Read 2:     6,795,453 bp\n",
      "Total written (filtered):     13,590,816 bp (100.0%)\n",
      "  Read 1:     6,795,402 bp\n",
      "  Read 2:     6,795,414 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 17 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t703.2\t0\t17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t703.2\t0\t9\n",
      "4\t3\t175.8\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/15_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/15_225_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/15_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/15_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/15_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/15_225_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/15_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/15_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        39,623 reads  @     29.1 µs/read;   2.06 M reads/minute\n",
      "Finished in 1.19 s (30 us/read; 2.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,623\n",
      "  Read 1 with adapter:                      23 (0.1%)\n",
      "  Read 2 with adapter:                      17 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        39,623 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,966,146 bp\n",
      "  Read 1:     5,983,073 bp\n",
      "  Read 2:     5,983,073 bp\n",
      "Total written (filtered):     11,966,019 bp (100.0%)\n",
      "  Read 1:     5,983,002 bp\n",
      "  Read 2:     5,983,017 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t619.1\t0\t21\n",
      "4\t2\t154.8\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 17 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t619.1\t0\t14\n",
      "4\t1\t154.8\t0\t1\n",
      "5\t2\t38.7\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/2_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/2_235_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/2_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/2_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/2_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/2_235_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/2_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/2_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        50,161 reads  @     24.8 µs/read;   2.42 M reads/minute\n",
      "Finished in 1.28 s (26 us/read; 2.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,161\n",
      "  Read 1 with adapter:                      23 (0.0%)\n",
      "  Read 2 with adapter:                      16 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,161 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,148,622 bp\n",
      "  Read 1:     7,574,311 bp\n",
      "  Read 2:     7,574,311 bp\n",
      "Total written (filtered):     15,148,504 bp (100.0%)\n",
      "  Read 1:     7,574,241 bp\n",
      "  Read 2:     7,574,263 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t783.8\t0\t22\n",
      "4\t1\t195.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t783.8\t0\t16\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/285_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/285_227_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/285_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/285_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/285_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/285_227_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/285_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/285_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        54,864 reads  @     25.5 µs/read;   2.36 M reads/minute\n",
      "Finished in 1.43 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,864\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,864 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,568,928 bp\n",
      "  Read 1:     8,284,464 bp\n",
      "  Read 2:     8,284,464 bp\n",
      "Total written (filtered):     16,568,869 bp (100.0%)\n",
      "  Read 1:     8,284,439 bp\n",
      "  Read 2:     8,284,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t857.2\t0\t7\n",
      "4\t1\t214.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t857.2\t0\t10\n",
      "4\t1\t214.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/291_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/291_228_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/291_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/291_228_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/291_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/291_228_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/291_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/291_228_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        54,921 reads  @     24.4 µs/read;   2.46 M reads/minute\n",
      "Finished in 1.37 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,921\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                      19 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,921 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,586,142 bp\n",
      "  Read 1:     8,293,071 bp\n",
      "  Read 2:     8,293,071 bp\n",
      "Total written (filtered):     16,586,059 bp (100.0%)\n",
      "  Read 1:     8,293,053 bp\n",
      "  Read 2:     8,293,006 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t858.1\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t858.1\t0\t14\n",
      "4\t3\t214.5\t0\t3\n",
      "5\t1\t53.6\t0\t1\n",
      "6\t1\t13.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/292_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/292_229_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/292_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/292_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/292_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/292_229_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/292_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/292_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        72,060 reads  @     23.8 µs/read;   2.53 M reads/minute\n",
      "Finished in 1.74 s (24 us/read; 2.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,060\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                      16 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        72,060 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,762,120 bp\n",
      "  Read 1:    10,881,060 bp\n",
      "  Read 2:    10,881,060 bp\n",
      "Total written (filtered):     21,762,046 bp (100.0%)\n",
      "  Read 1:    10,881,036 bp\n",
      "  Read 2:    10,881,010 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1125.9\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t1125.9\t0\t14\n",
      "4\t2\t281.5\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/293_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/293_230_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/293_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/293_230_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/293_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/293_230_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/293_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/293_230_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        42,493 reads  @     25.6 µs/read;   2.34 M reads/minute\n",
      "Finished in 1.12 s (26 us/read; 2.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,493\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                       7 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,493 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,832,886 bp\n",
      "  Read 1:     6,416,443 bp\n",
      "  Read 2:     6,416,443 bp\n",
      "Total written (filtered):     12,832,856 bp (100.0%)\n",
      "  Read 1:     6,416,434 bp\n",
      "  Read 2:     6,416,422 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t664.0\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t664.0\t0\t7\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/296_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/296_231_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/296_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/296_231_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/296_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/296_231_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/296_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/296_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        70,686 reads  @     22.9 µs/read;   2.63 M reads/minute\n",
      "Finished in 1.64 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,686\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                      24 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        70,686 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,347,172 bp\n",
      "  Read 1:    10,673,586 bp\n",
      "  Read 2:    10,673,586 bp\n",
      "Total written (filtered):     21,347,085 bp (100.0%)\n",
      "  Read 1:    10,673,574 bp\n",
      "  Read 2:    10,673,511 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1104.5\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 24 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t1104.5\t0\t22\n",
      "4\t1\t276.1\t0\t1\n",
      "5\t1\t69.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/297_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/297_232_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/297_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/297_232_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/297_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/297_232_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/297_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/297_232_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        53,507 reads  @     24.3 µs/read;   2.47 M reads/minute\n",
      "Finished in 1.33 s (25 us/read; 2.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,507\n",
      "  Read 1 with adapter:                      10 (0.0%)\n",
      "  Read 2 with adapter:                      15 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,507 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,159,114 bp\n",
      "  Read 1:     8,079,557 bp\n",
      "  Read 2:     8,079,557 bp\n",
      "Total written (filtered):     16,159,031 bp (100.0%)\n",
      "  Read 1:     8,079,524 bp\n",
      "  Read 2:     8,079,507 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t836.0\t0\t9\n",
      "6\t1\t13.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t836.0\t0\t11\n",
      "4\t3\t209.0\t0\t3\n",
      "5\t1\t52.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/298_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/298_233_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/298_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/298_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/298_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/298_233_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/298_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/298_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        62,513 reads  @     23.9 µs/read;   2.51 M reads/minute\n",
      "Finished in 1.52 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,513\n",
      "  Read 1 with adapter:                      11 (0.0%)\n",
      "  Read 2 with adapter:                      23 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        62,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,878,926 bp\n",
      "  Read 1:     9,439,463 bp\n",
      "  Read 2:     9,439,463 bp\n",
      "Total written (filtered):     18,878,815 bp (100.0%)\n",
      "  Read 1:     9,439,430 bp\n",
      "  Read 2:     9,439,385 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t976.8\t0\t11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 23 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t976.8\t0\t18\n",
      "4\t3\t244.2\t0\t3\n",
      "5\t1\t61.0\t0\t1\n",
      "7\t1\t3.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/299_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/299_234_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/299_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/299_234_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/299_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/299_234_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/299_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/299_234_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        56,299 reads  @     25.0 µs/read;   2.40 M reads/minute\n",
      "Finished in 1.44 s (26 us/read; 2.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,299\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,299 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,002,298 bp\n",
      "  Read 1:     8,501,149 bp\n",
      "  Read 2:     8,501,149 bp\n",
      "Total written (filtered):     17,002,219 bp (100.0%)\n",
      "  Read 1:     8,501,134 bp\n",
      "  Read 2:     8,501,085 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t879.7\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t879.7\t0\t6\n",
      "5\t1\t55.0\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "30\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/3_111_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/3_331_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/3_111_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/3_331_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/3_111_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/3_331_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/3_111_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/3_331_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        37,845 reads  @     32.2 µs/read;   1.86 M reads/minute\n",
      "Finished in 1.26 s (33 us/read; 1.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,845\n",
      "  Read 1 with adapter:                      22 (0.1%)\n",
      "  Read 2 with adapter:                       7 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        37,845 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,429,190 bp\n",
      "  Read 1:     5,714,595 bp\n",
      "  Read 2:     5,714,595 bp\n",
      "Total written (filtered):     11,429,077 bp (100.0%)\n",
      "  Read 1:     5,714,529 bp\n",
      "  Read 2:     5,714,548 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t591.3\t0\t22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t591.3\t0\t4\n",
      "4\t2\t147.8\t0\t2\n",
      "27\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/300_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/300_236_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/300_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/300_236_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/300_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/300_236_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/300_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/300_236_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        56,267 reads  @     25.3 µs/read;   2.37 M reads/minute\n",
      "Finished in 1.46 s (26 us/read; 2.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,267\n",
      "  Read 1 with adapter:                      12 (0.0%)\n",
      "  Read 2 with adapter:                      15 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,267 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,992,634 bp\n",
      "  Read 1:     8,496,317 bp\n",
      "  Read 2:     8,496,317 bp\n",
      "Total written (filtered):     16,992,548 bp (100.0%)\n",
      "  Read 1:     8,496,281 bp\n",
      "  Read 2:     8,496,267 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t879.2\t0\t12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t879.2\t0\t12\n",
      "4\t1\t219.8\t0\t1\n",
      "5\t2\t54.9\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/301_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/301_237_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/301_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/301_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/301_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/301_237_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/301_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/301_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        52,261 reads  @     27.1 µs/read;   2.22 M reads/minute\n",
      "Finished in 1.44 s (28 us/read; 2.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,261\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                      19 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,261 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,782,822 bp\n",
      "  Read 1:     7,891,411 bp\n",
      "  Read 2:     7,891,411 bp\n",
      "Total written (filtered):     15,782,738 bp (100.0%)\n",
      "  Read 1:     7,891,395 bp\n",
      "  Read 2:     7,891,343 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t816.6\t0\t4\n",
      "4\t1\t204.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t816.6\t0\t13\n",
      "4\t4\t204.1\t0\t4\n",
      "5\t1\t51.0\t0\t1\n",
      "8\t1\t0.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/302_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/302_238_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/302_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/302_238_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/302_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/302_238_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/302_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/302_238_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        52,010 reads  @     25.1 µs/read;   2.39 M reads/minute\n",
      "Finished in 1.34 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,010\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,010 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,707,020 bp\n",
      "  Read 1:     7,853,510 bp\n",
      "  Read 2:     7,853,510 bp\n",
      "Total written (filtered):     15,706,960 bp (100.0%)\n",
      "  Read 1:     7,853,489 bp\n",
      "  Read 2:     7,853,471 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t812.7\t0\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t812.7\t0\t10\n",
      "4\t1\t203.2\t0\t1\n",
      "5\t1\t50.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/303_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/303_239_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/303_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/303_239_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/303_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/303_239_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/303_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/303_239_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        52,891 reads  @     27.4 µs/read;   2.19 M reads/minute\n",
      "Finished in 1.48 s (28 us/read; 2.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,891\n",
      "  Read 1 with adapter:                      23 (0.0%)\n",
      "  Read 2 with adapter:                      19 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,891 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,973,082 bp\n",
      "  Read 1:     7,986,541 bp\n",
      "  Read 2:     7,986,541 bp\n",
      "Total written (filtered):     15,972,951 bp (100.0%)\n",
      "  Read 1:     7,986,472 bp\n",
      "  Read 2:     7,986,479 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t826.4\t0\t23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t826.4\t0\t16\n",
      "4\t1\t206.6\t0\t1\n",
      "5\t2\t51.7\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/304_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/304_240_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/304_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/304_240_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/304_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/304_240_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/304_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/304_240_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        41,360 reads  @     25.3 µs/read;   2.38 M reads/minute\n",
      "Finished in 1.07 s (26 us/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,360\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,360 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,490,720 bp\n",
      "  Read 1:     6,245,360 bp\n",
      "  Read 2:     6,245,360 bp\n",
      "Total written (filtered):     12,490,673 bp (100.0%)\n",
      "  Read 1:     6,245,340 bp\n",
      "  Read 2:     6,245,333 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t646.2\t0\t4\n",
      "4\t2\t161.6\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t646.2\t0\t9\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/305_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/305_241_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/305_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/305_241_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/305_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/305_241_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/305_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/305_241_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        56,234 reads  @     24.3 µs/read;   2.47 M reads/minute\n",
      "Finished in 1.39 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,234\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,234 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,982,668 bp\n",
      "  Read 1:     8,491,334 bp\n",
      "  Read 2:     8,491,334 bp\n",
      "Total written (filtered):     16,982,607 bp (100.0%)\n",
      "  Read 1:     8,491,310 bp\n",
      "  Read 2:     8,491,297 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t878.7\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t878.7\t0\t11\n",
      "4\t1\t219.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/306_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/306_242_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/306_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/306_242_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/306_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/306_242_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/306_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/306_242_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        60,860 reads  @     26.0 µs/read;   2.31 M reads/minute\n",
      "Finished in 1.62 s (27 us/read; 2.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,860\n",
      "  Read 1 with adapter:                      13 (0.0%)\n",
      "  Read 2 with adapter:                      15 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        60,860 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,379,720 bp\n",
      "  Read 1:     9,189,860 bp\n",
      "  Read 2:     9,189,860 bp\n",
      "Total written (filtered):     18,379,630 bp (100.0%)\n",
      "  Read 1:     9,189,820 bp\n",
      "  Read 2:     9,189,810 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t950.9\t0\t12\n",
      "4\t1\t237.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t950.9\t0\t11\n",
      "4\t3\t237.7\t0\t3\n",
      "5\t1\t59.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/307_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/307_243_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/307_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/307_243_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/307_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/307_243_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/307_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/307_243_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        53,231 reads  @     27.1 µs/read;   2.21 M reads/minute\n",
      "Finished in 1.48 s (28 us/read; 2.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,231\n",
      "  Read 1 with adapter:                      18 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,231 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,075,762 bp\n",
      "  Read 1:     8,037,881 bp\n",
      "  Read 2:     8,037,881 bp\n",
      "Total written (filtered):     16,075,676 bp (100.0%)\n",
      "  Read 1:     8,037,827 bp\n",
      "  Read 2:     8,037,849 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 18 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t831.7\t0\t18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t831.7\t0\t8\n",
      "4\t2\t207.9\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/308_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/308_244_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/308_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/308_244_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/308_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/308_244_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/308_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/308_244_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        67,594 reads  @     22.2 µs/read;   2.70 M reads/minute\n",
      "Finished in 1.52 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,594\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                      17 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        67,594 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,413,388 bp\n",
      "  Read 1:    10,206,694 bp\n",
      "  Read 2:    10,206,694 bp\n",
      "Total written (filtered):     20,413,308 bp (100.0%)\n",
      "  Read 1:    10,206,669 bp\n",
      "  Read 2:    10,206,639 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1056.2\t0\t7\n",
      "4\t1\t264.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 17 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1056.2\t0\t15\n",
      "4\t1\t264.0\t0\t1\n",
      "6\t1\t16.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/309_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/309_245_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/309_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/309_245_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/309_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/309_245_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/309_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/309_245_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        56,009 reads  @     25.9 µs/read;   2.31 M reads/minute\n",
      "Finished in 1.48 s (26 us/read; 2.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,009\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                      15 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,009 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,914,718 bp\n",
      "  Read 1:     8,457,359 bp\n",
      "  Read 2:     8,457,359 bp\n",
      "Total written (filtered):     16,914,659 bp (100.0%)\n",
      "  Read 1:     8,457,347 bp\n",
      "  Read 2:     8,457,312 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t875.1\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t875.1\t0\t13\n",
      "4\t2\t218.8\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/310_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/310_246_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/310_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/310_246_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/310_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/310_246_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/310_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/310_246_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        52,275 reads  @     25.1 µs/read;   2.39 M reads/minute\n",
      "Finished in 1.34 s (26 us/read; 2.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,275\n",
      "  Read 1 with adapter:                      14 (0.0%)\n",
      "  Read 2 with adapter:                      17 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,275 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,787,050 bp\n",
      "  Read 1:     7,893,525 bp\n",
      "  Read 2:     7,893,525 bp\n",
      "Total written (filtered):     15,786,940 bp (100.0%)\n",
      "  Read 1:     7,893,482 bp\n",
      "  Read 2:     7,893,458 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t816.8\t0\t13\n",
      "4\t1\t204.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 17 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t816.8\t0\t12\n",
      "4\t3\t204.2\t0\t3\n",
      "6\t1\t12.8\t0\t1\n",
      "13\t1\t0.0\t1\t0 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/311_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/311_247_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/311_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/311_247_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/311_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/311_247_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/311_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/311_247_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        54,675 reads  @     26.1 µs/read;   2.30 M reads/minute\n",
      "Finished in 1.46 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,675\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                      18 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,675 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,511,850 bp\n",
      "  Read 1:     8,255,925 bp\n",
      "  Read 2:     8,255,925 bp\n",
      "Total written (filtered):     16,511,770 bp (100.0%)\n",
      "  Read 1:     8,255,899 bp\n",
      "  Read 2:     8,255,871 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t854.3\t0\t7\n",
      "5\t1\t53.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 18 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t854.3\t0\t18\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/312_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/312_248_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/312_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/312_248_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/312_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/312_248_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/312_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/312_248_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        55,507 reads  @     25.3 µs/read;   2.37 M reads/minute\n",
      "Finished in 1.43 s (26 us/read; 2.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,507\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,507 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,763,114 bp\n",
      "  Read 1:     8,381,557 bp\n",
      "  Read 2:     8,381,557 bp\n",
      "Total written (filtered):     16,763,066 bp (100.0%)\n",
      "  Read 1:     8,381,542 bp\n",
      "  Read 2:     8,381,524 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t867.3\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t867.3\t0\t11\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/313_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/313_249_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/313_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/313_249_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/313_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/313_249_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/313_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/313_249_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        57,329 reads  @     25.8 µs/read;   2.32 M reads/minute\n",
      "Finished in 1.51 s (26 us/read; 2.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,329\n",
      "  Read 1 with adapter:                      13 (0.0%)\n",
      "  Read 2 with adapter:                      22 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        57,329 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,313,358 bp\n",
      "  Read 1:     8,656,679 bp\n",
      "  Read 2:     8,656,679 bp\n",
      "Total written (filtered):     17,313,248 bp (100.0%)\n",
      "  Read 1:     8,656,638 bp\n",
      "  Read 2:     8,656,610 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t895.8\t0\t11\n",
      "4\t2\t223.9\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 22 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t895.8\t0\t20\n",
      "4\t1\t223.9\t0\t1\n",
      "5\t1\t56.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/314_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/314_250_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/314_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/314_250_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/314_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/314_250_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/314_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/314_250_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        41,542 reads  @     25.1 µs/read;   2.39 M reads/minute\n",
      "Finished in 1.07 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,542\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,542 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,545,684 bp\n",
      "  Read 1:     6,272,842 bp\n",
      "  Read 2:     6,272,842 bp\n",
      "Total written (filtered):     12,545,639 bp (100.0%)\n",
      "  Read 1:     6,272,833 bp\n",
      "  Read 2:     6,272,806 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t649.1\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t649.1\t0\t9\n",
      "4\t1\t162.3\t0\t1\n",
      "5\t1\t40.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/315_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/315_251_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/315_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/315_251_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/315_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/315_251_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/315_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/315_251_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        51,382 reads  @     27.6 µs/read;   2.17 M reads/minute\n",
      "Finished in 1.45 s (28 us/read; 2.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,382\n",
      "  Read 1 with adapter:                      11 (0.0%)\n",
      "  Read 2 with adapter:                      18 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,382 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,517,364 bp\n",
      "  Read 1:     7,758,682 bp\n",
      "  Read 2:     7,758,682 bp\n",
      "Total written (filtered):     15,517,273 bp (100.0%)\n",
      "  Read 1:     7,758,648 bp\n",
      "  Read 2:     7,758,625 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t802.8\t0\t10\n",
      "4\t1\t200.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 18 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t802.8\t0\t15\n",
      "4\t3\t200.7\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/316_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/316_252_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/316_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/316_252_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/316_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/316_252_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/316_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/316_252_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        48,472 reads  @     27.8 µs/read;   2.16 M reads/minute\n",
      "Finished in 1.37 s (28 us/read; 2.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,472\n",
      "  Read 1 with adapter:                      19 (0.0%)\n",
      "  Read 2 with adapter:                      17 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,472 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,638,544 bp\n",
      "  Read 1:     7,319,272 bp\n",
      "  Read 2:     7,319,272 bp\n",
      "Total written (filtered):     14,638,424 bp (100.0%)\n",
      "  Read 1:     7,319,214 bp\n",
      "  Read 2:     7,319,210 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t757.4\t0\t18\n",
      "4\t1\t189.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 17 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t757.4\t0\t10\n",
      "4\t5\t189.3\t0\t5\n",
      "6\t2\t11.8\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/317_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/317_253_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/317_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/317_253_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/317_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/317_253_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/317_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/317_253_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        37,207 reads  @     31.5 µs/read;   1.90 M reads/minute\n",
      "Finished in 1.21 s (32 us/read; 1.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,207\n",
      "  Read 1 with adapter:                      17 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        37,207 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,236,514 bp\n",
      "  Read 1:     5,618,257 bp\n",
      "  Read 2:     5,618,257 bp\n",
      "Total written (filtered):     11,236,444 bp (100.0%)\n",
      "  Read 1:     5,618,205 bp\n",
      "  Read 2:     5,618,239 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 17 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t581.4\t0\t16\n",
      "4\t1\t145.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t581.4\t0\t6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/318_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/318_254_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/318_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/318_254_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/318_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/318_254_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/318_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/318_254_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        63,741 reads  @     24.3 µs/read;   2.47 M reads/minute\n",
      "Finished in 1.58 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,741\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                      14 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        63,741 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,249,782 bp\n",
      "  Read 1:     9,624,891 bp\n",
      "  Read 2:     9,624,891 bp\n",
      "Total written (filtered):     19,249,717 bp (100.0%)\n",
      "  Read 1:     9,624,868 bp\n",
      "  Read 2:     9,624,849 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t996.0\t0\t5\n",
      "4\t2\t249.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t996.0\t0\t14\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/319_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/319_255_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/319_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/319_255_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/319_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/319_255_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/319_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/319_255_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        62,606 reads  @     23.0 µs/read;   2.61 M reads/minute\n",
      "Finished in 1.48 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,606\n",
      "  Read 1 with adapter:                      14 (0.0%)\n",
      "  Read 2 with adapter:                      26 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        62,606 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,907,012 bp\n",
      "  Read 1:     9,453,506 bp\n",
      "  Read 2:     9,453,506 bp\n",
      "Total written (filtered):     18,906,888 bp (100.0%)\n",
      "  Read 1:     9,453,462 bp\n",
      "  Read 2:     9,453,426 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t978.2\t0\t13\n",
      "5\t1\t61.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 26 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t978.2\t0\t24\n",
      "4\t2\t244.6\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/320_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/320_256_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/320_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/320_256_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/320_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/320_256_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/320_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/320_256_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        51,518 reads  @     26.8 µs/read;   2.24 M reads/minute\n",
      "Finished in 1.41 s (27 us/read; 2.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,518\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,518 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,558,436 bp\n",
      "  Read 1:     7,779,218 bp\n",
      "  Read 2:     7,779,218 bp\n",
      "Total written (filtered):     15,558,372 bp (100.0%)\n",
      "  Read 1:     7,779,194 bp\n",
      "  Read 2:     7,779,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t805.0\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t805.0\t0\t10\n",
      "4\t1\t201.2\t0\t1\n",
      "6\t1\t12.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/321_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/321_257_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/321_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/321_257_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/321_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/321_257_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/321_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/321_257_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        47,498 reads  @     25.1 µs/read;   2.39 M reads/minute\n",
      "Finished in 1.22 s (26 us/read; 2.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,498\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        47,498 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,344,396 bp\n",
      "  Read 1:     7,172,198 bp\n",
      "  Read 2:     7,172,198 bp\n",
      "Total written (filtered):     14,344,360 bp (100.0%)\n",
      "  Read 1:     7,172,180 bp\n",
      "  Read 2:     7,172,180 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t742.2\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t742.2\t0\t6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/322_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/322_258_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/322_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/322_258_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/322_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/322_258_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/322_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/322_258_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        65,715 reads  @     21.9 µs/read;   2.74 M reads/minute\n",
      "Finished in 1.46 s (22 us/read; 2.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,715\n",
      "  Read 1 with adapter:                      16 (0.0%)\n",
      "  Read 2 with adapter:                      21 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        65,715 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,845,930 bp\n",
      "  Read 1:     9,922,965 bp\n",
      "  Read 2:     9,922,965 bp\n",
      "Total written (filtered):     19,845,797 bp (100.0%)\n",
      "  Read 1:     9,922,901 bp\n",
      "  Read 2:     9,922,896 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t1026.8\t0\t14\n",
      "11\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 21 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t1026.8\t0\t17\n",
      "4\t3\t256.7\t0\t3\n",
      "6\t1\t16.0\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/323_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/323_259_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/323_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/323_259_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/323_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/323_259_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/323_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/323_259_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        44,332 reads  @     26.8 µs/read;   2.24 M reads/minute\n",
      "Finished in 1.22 s (27 us/read; 2.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,332\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,332 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,388,264 bp\n",
      "  Read 1:     6,694,132 bp\n",
      "  Read 2:     6,694,132 bp\n",
      "Total written (filtered):     13,388,214 bp (100.0%)\n",
      "  Read 1:     6,694,107 bp\n",
      "  Read 2:     6,694,107 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t692.7\t0\t7\n",
      "4\t1\t173.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t692.7\t0\t7\n",
      "4\t1\t173.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/325_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/325_260_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/325_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/325_260_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/325_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/325_260_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/325_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/325_260_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,379 reads  @     24.9 µs/read;   2.41 M reads/minute\n",
      "Finished in 1.26 s (26 us/read; 2.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,379\n",
      "  Read 1 with adapter:                      14 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,379 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,912,458 bp\n",
      "  Read 1:     7,456,229 bp\n",
      "  Read 2:     7,456,229 bp\n",
      "Total written (filtered):     14,912,404 bp (100.0%)\n",
      "  Read 1:     7,456,187 bp\n",
      "  Read 2:     7,456,217 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t771.5\t0\t14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t771.5\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/326_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/326_261_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/326_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/326_261_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/326_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/326_261_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/326_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/326_261_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,403 reads  @     25.1 µs/read;   2.39 M reads/minute\n",
      "Finished in 1.27 s (26 us/read; 2.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,403\n",
      "  Read 1 with adapter:                      10 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,403 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,919,706 bp\n",
      "  Read 1:     7,459,853 bp\n",
      "  Read 2:     7,459,853 bp\n",
      "Total written (filtered):     14,919,640 bp (100.0%)\n",
      "  Read 1:     7,459,823 bp\n",
      "  Read 2:     7,459,817 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t771.9\t0\t10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t771.9\t0\t12\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/327_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/327_262_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/327_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/327_262_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/327_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/327_262_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/327_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/327_262_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,010 reads  @     23.6 µs/read;   2.54 M reads/minute\n",
      "Finished in 1.19 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,010\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,010 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,801,020 bp\n",
      "  Read 1:     7,400,510 bp\n",
      "  Read 2:     7,400,510 bp\n",
      "Total written (filtered):     14,800,969 bp (100.0%)\n",
      "  Read 1:     7,400,495 bp\n",
      "  Read 2:     7,400,474 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t765.8\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t765.8\t0\t12\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/328_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/328_263_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/328_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/328_263_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/328_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/328_263_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/328_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/328_263_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        47,084 reads  @     24.9 µs/read;   2.41 M reads/minute\n",
      "Finished in 1.19 s (25 us/read; 2.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,084\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                      13 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        47,084 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,219,368 bp\n",
      "  Read 1:     7,109,684 bp\n",
      "  Read 2:     7,109,684 bp\n",
      "Total written (filtered):     14,219,311 bp (100.0%)\n",
      "  Read 1:     7,109,672 bp\n",
      "  Read 2:     7,109,639 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t735.7\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t735.7\t0\t11\n",
      "4\t1\t183.9\t0\t1\n",
      "8\t1\t0.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/329_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/329_264_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/329_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/329_264_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/329_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/329_264_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/329_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/329_264_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,181 reads  @     25.4 µs/read;   2.36 M reads/minute\n",
      "Finished in 1.28 s (26 us/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,181\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                      17 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,181 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,852,662 bp\n",
      "  Read 1:     7,426,331 bp\n",
      "  Read 2:     7,426,331 bp\n",
      "Total written (filtered):     14,852,586 bp (100.0%)\n",
      "  Read 1:     7,426,309 bp\n",
      "  Read 2:     7,426,277 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t768.5\t0\t6\n",
      "4\t1\t192.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 17 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t768.5\t0\t16\n",
      "6\t1\t12.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/331_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/331_265_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/331_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/331_265_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/331_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/331_265_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/331_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/331_265_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        41,587 reads  @     26.1 µs/read;   2.30 M reads/minute\n",
      "Finished in 1.12 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,587\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                      13 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,587 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,559,274 bp\n",
      "  Read 1:     6,279,637 bp\n",
      "  Read 2:     6,279,637 bp\n",
      "Total written (filtered):     12,559,207 bp (100.0%)\n",
      "  Read 1:     6,279,610 bp\n",
      "  Read 2:     6,279,597 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t649.8\t0\t9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t649.8\t0\t12\n",
      "4\t1\t162.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/332_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/332_266_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/332_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/332_266_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/332_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/332_266_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/332_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/332_266_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        45,766 reads  @     26.4 µs/read;   2.27 M reads/minute\n",
      "Finished in 1.24 s (27 us/read; 2.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,766\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                      14 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,766 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,821,332 bp\n",
      "  Read 1:     6,910,666 bp\n",
      "  Read 2:     6,910,666 bp\n",
      "Total written (filtered):     13,821,268 bp (100.0%)\n",
      "  Read 1:     6,910,644 bp\n",
      "  Read 2:     6,910,624 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t715.1\t0\t6\n",
      "4\t1\t178.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t715.1\t0\t14\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/333_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/333_267_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/333_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/333_267_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/333_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/333_267_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/333_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/333_267_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        40,677 reads  @     27.2 µs/read;   2.20 M reads/minute\n",
      "Finished in 1.14 s (28 us/read; 2.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,677\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        40,677 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,284,454 bp\n",
      "  Read 1:     6,142,227 bp\n",
      "  Read 2:     6,142,227 bp\n",
      "Total written (filtered):     12,284,409 bp (100.0%)\n",
      "  Read 1:     6,142,209 bp\n",
      "  Read 2:     6,142,200 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t635.6\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t635.6\t0\t9\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/334_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/334_268_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/334_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/334_268_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/334_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/334_268_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/334_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/334_268_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,698 reads  @     25.5 µs/read;   2.35 M reads/minute\n",
      "Finished in 1.30 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,698\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,698 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,008,796 bp\n",
      "  Read 1:     7,504,398 bp\n",
      "  Read 2:     7,504,398 bp\n",
      "Total written (filtered):     15,008,766 bp (100.0%)\n",
      "  Read 1:     7,504,392 bp\n",
      "  Read 2:     7,504,374 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t776.5\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t776.5\t0\t8\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/335_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/335_269_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/335_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/335_269_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/335_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/335_269_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/335_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/335_269_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        64,460 reads  @     22.0 µs/read;   2.73 M reads/minute\n",
      "Finished in 1.44 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,460\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                      19 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,460 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,466,920 bp\n",
      "  Read 1:     9,733,460 bp\n",
      "  Read 2:     9,733,460 bp\n",
      "Total written (filtered):     19,466,834 bp (100.0%)\n",
      "  Read 1:     9,733,435 bp\n",
      "  Read 2:     9,733,399 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1007.2\t0\t5\n",
      "5\t2\t62.9\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1007.2\t0\t15\n",
      "4\t4\t251.8\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/336_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/336_270_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/336_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/336_270_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/336_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/336_270_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/336_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/336_270_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        51,471 reads  @     26.7 µs/read;   2.25 M reads/minute\n",
      "Finished in 1.41 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,471\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                      20 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,471 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,544,242 bp\n",
      "  Read 1:     7,772,121 bp\n",
      "  Read 2:     7,772,121 bp\n",
      "Total written (filtered):     15,544,151 bp (100.0%)\n",
      "  Read 1:     7,772,097 bp\n",
      "  Read 2:     7,772,054 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t804.2\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 20 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t804.2\t0\t14\n",
      "4\t5\t201.1\t0\t5\n",
      "5\t1\t50.3\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/337_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/337_271_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/337_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/337_271_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/337_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/337_271_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/337_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/337_271_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        35,558 reads  @     32.2 µs/read;   1.86 M reads/minute\n",
      "Finished in 1.18 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,558\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,558 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,738,516 bp\n",
      "  Read 1:     5,369,258 bp\n",
      "  Read 2:     5,369,258 bp\n",
      "Total written (filtered):     10,738,474 bp (100.0%)\n",
      "  Read 1:     5,369,234 bp\n",
      "  Read 2:     5,369,240 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t555.6\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t555.6\t0\t6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/338_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/338_272_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/338_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/338_272_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/338_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/338_272_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/338_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/338_272_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        43,123 reads  @     30.8 µs/read;   1.95 M reads/minute\n",
      "Finished in 1.36 s (32 us/read; 1.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,123\n",
      "  Read 1 with adapter:                      14 (0.0%)\n",
      "  Read 2 with adapter:                      14 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        43,123 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,023,146 bp\n",
      "  Read 1:     6,511,573 bp\n",
      "  Read 2:     6,511,573 bp\n",
      "Total written (filtered):     13,023,032 bp (100.0%)\n",
      "  Read 1:     6,511,531 bp\n",
      "  Read 2:     6,511,501 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t673.8\t0\t14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t673.8\t0\t11\n",
      "4\t2\t168.4\t0\t2\n",
      "31\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/339_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/339_273_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/339_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/339_273_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/339_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/339_273_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/339_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/339_273_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        42,699 reads  @     28.9 µs/read;   2.08 M reads/minute\n",
      "Finished in 1.27 s (30 us/read; 2.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,699\n",
      "  Read 1 with adapter:                      11 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,699 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,895,098 bp\n",
      "  Read 1:     6,447,549 bp\n",
      "  Read 2:     6,447,549 bp\n",
      "Total written (filtered):     12,895,030 bp (100.0%)\n",
      "  Read 1:     6,447,516 bp\n",
      "  Read 2:     6,447,514 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t667.2\t0\t11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t667.2\t0\t7\n",
      "4\t1\t166.8\t0\t1\n",
      "5\t2\t41.7\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/340_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/340_274_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/340_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/340_274_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/340_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/340_274_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/340_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/340_274_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        43,880 reads  @     27.9 µs/read;   2.15 M reads/minute\n",
      "Finished in 1.26 s (29 us/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,880\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        43,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,251,760 bp\n",
      "  Read 1:     6,625,880 bp\n",
      "  Read 2:     6,625,880 bp\n",
      "Total written (filtered):     13,251,714 bp (100.0%)\n",
      "  Read 1:     6,625,859 bp\n",
      "  Read 2:     6,625,855 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t685.6\t0\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t685.6\t0\t7\n",
      "4\t1\t171.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/341_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/341_275_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/341_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/341_275_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/341_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/341_275_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/341_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/341_275_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        47,548 reads  @     25.0 µs/read;   2.40 M reads/minute\n",
      "Finished in 1.22 s (26 us/read; 2.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,548\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        47,548 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,359,496 bp\n",
      "  Read 1:     7,179,748 bp\n",
      "  Read 2:     7,179,748 bp\n",
      "Total written (filtered):     14,359,460 bp (100.0%)\n",
      "  Read 1:     7,179,739 bp\n",
      "  Read 2:     7,179,721 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t742.9\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t742.9\t0\t9\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/342_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/342_276_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/342_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/342_276_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/342_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/342_276_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/342_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/342_276_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        52,662 reads  @     24.4 µs/read;   2.46 M reads/minute\n",
      "Finished in 1.32 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,662\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,662 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,903,924 bp\n",
      "  Read 1:     7,951,962 bp\n",
      "  Read 2:     7,951,962 bp\n",
      "Total written (filtered):     15,903,865 bp (100.0%)\n",
      "  Read 1:     7,951,935 bp\n",
      "  Read 2:     7,951,930 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t822.8\t0\t9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t822.8\t0\t8\n",
      "4\t2\t205.7\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/343_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/343_277_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/343_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/343_277_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/343_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/343_277_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/343_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/343_277_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        55,762 reads  @     26.3 µs/read;   2.28 M reads/minute\n",
      "Finished in 1.50 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,762\n",
      "  Read 1 with adapter:                      12 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,762 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,840,124 bp\n",
      "  Read 1:     8,420,062 bp\n",
      "  Read 2:     8,420,062 bp\n",
      "Total written (filtered):     16,840,059 bp (100.0%)\n",
      "  Read 1:     8,420,026 bp\n",
      "  Read 2:     8,420,033 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t871.3\t0\t12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t871.3\t0\t7\n",
      "4\t2\t217.8\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/344_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/344_278_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/344_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/344_278_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/344_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/344_278_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/344_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/344_278_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        33,056 reads  @     37.3 µs/read;   1.61 M reads/minute\n",
      "Finished in 1.27 s (38 us/read; 1.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,056\n",
      "  Read 1 with adapter:                      10 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,056 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,982,912 bp\n",
      "  Read 1:     4,991,456 bp\n",
      "  Read 2:     4,991,456 bp\n",
      "Total written (filtered):      9,982,858 bp (100.0%)\n",
      "  Read 1:     4,991,426 bp\n",
      "  Read 2:     4,991,432 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t516.5\t0\t10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t516.5\t0\t8\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/345_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/345_279_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/345_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/345_279_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/345_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/345_279_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/345_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/345_279_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        51,372 reads  @     26.3 µs/read;   2.28 M reads/minute\n",
      "Finished in 1.39 s (27 us/read; 2.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,372\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,372 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,514,344 bp\n",
      "  Read 1:     7,757,172 bp\n",
      "  Read 2:     7,757,172 bp\n",
      "Total written (filtered):     15,514,295 bp (100.0%)\n",
      "  Read 1:     7,757,148 bp\n",
      "  Read 2:     7,757,147 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t802.7\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t802.7\t0\t7\n",
      "4\t1\t200.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/346_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/346_280_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/346_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/346_280_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/346_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/346_280_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/346_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/346_280_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        55,383 reads  @     25.2 µs/read;   2.38 M reads/minute\n",
      "Finished in 1.42 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,383\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                      19 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,383 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,725,666 bp\n",
      "  Read 1:     8,362,833 bp\n",
      "  Read 2:     8,362,833 bp\n",
      "Total written (filtered):     16,725,588 bp (100.0%)\n",
      "  Read 1:     8,362,821 bp\n",
      "  Read 2:     8,362,767 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t865.4\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t865.4\t0\t14\n",
      "4\t3\t216.3\t0\t3\n",
      "5\t1\t54.1\t0\t1\n",
      "7\t1\t3.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/347_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/347_281_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/347_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/347_281_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/347_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/347_281_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/347_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/347_281_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        44,454 reads  @     25.3 µs/read;   2.37 M reads/minute\n",
      "Finished in 1.15 s (26 us/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,454\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,454 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,425,108 bp\n",
      "  Read 1:     6,712,554 bp\n",
      "  Read 2:     6,712,554 bp\n",
      "Total written (filtered):     13,425,078 bp (100.0%)\n",
      "  Read 1:     6,712,536 bp\n",
      "  Read 2:     6,712,542 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t694.6\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t694.6\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/348_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/348_282_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/348_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/348_282_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/348_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/348_282_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/348_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/348_282_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        41,836 reads  @     28.5 µs/read;   2.10 M reads/minute\n",
      "Finished in 1.23 s (29 us/read; 2.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,836\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,836 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,634,472 bp\n",
      "  Read 1:     6,317,236 bp\n",
      "  Read 2:     6,317,236 bp\n",
      "Total written (filtered):     12,634,445 bp (100.0%)\n",
      "  Read 1:     6,317,221 bp\n",
      "  Read 2:     6,317,224 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t653.7\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t653.7\t0\t2\n",
      "6\t1\t10.2\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/349_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/349_283_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/349_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/349_283_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/349_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/349_283_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/349_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/349_283_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        50,324 reads  @     25.5 µs/read;   2.35 M reads/minute\n",
      "Finished in 1.31 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,324\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                      14 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,324 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,197,848 bp\n",
      "  Read 1:     7,598,924 bp\n",
      "  Read 2:     7,598,924 bp\n",
      "Total written (filtered):     15,197,788 bp (100.0%)\n",
      "  Read 1:     7,598,909 bp\n",
      "  Read 2:     7,598,879 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t786.3\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t786.3\t0\t11\n",
      "4\t3\t196.6\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/350_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/350_284_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/350_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/350_284_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/350_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/350_284_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/350_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/350_284_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        47,466 reads  @     25.2 µs/read;   2.38 M reads/minute\n",
      "Finished in 1.22 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,466\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        47,466 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,334,732 bp\n",
      "  Read 1:     7,167,366 bp\n",
      "  Read 2:     7,167,366 bp\n",
      "Total written (filtered):     14,334,669 bp (100.0%)\n",
      "  Read 1:     7,167,339 bp\n",
      "  Read 2:     7,167,330 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t741.7\t0\t9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t741.7\t0\t3\n",
      "4\t1\t185.4\t0\t1\n",
      "5\t2\t46.4\t0\t2\n",
      "6\t1\t11.6\t0\t1\n",
      "7\t1\t2.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/351_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/351_285_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/351_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/351_285_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/351_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/351_285_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/351_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/351_285_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        56,061 reads  @     24.6 µs/read;   2.44 M reads/minute\n",
      "Finished in 1.41 s (25 us/read; 2.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,061\n",
      "  Read 1 with adapter:                      13 (0.0%)\n",
      "  Read 2 with adapter:                      13 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,061 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,930,422 bp\n",
      "  Read 1:     8,465,211 bp\n",
      "  Read 2:     8,465,211 bp\n",
      "Total written (filtered):     16,930,336 bp (100.0%)\n",
      "  Read 1:     8,465,170 bp\n",
      "  Read 2:     8,465,166 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t876.0\t0\t12\n",
      "5\t1\t54.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t876.0\t0\t9\n",
      "4\t3\t219.0\t0\t3\n",
      "6\t1\t13.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/352_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/352_286_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/352_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/352_286_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/352_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/352_286_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/352_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/352_286_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        50,000 reads  @     24.4 µs/read;   2.46 M reads/minute\n",
      "Finished in 1.25 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,000\n",
      "  Read 1 with adapter:                      17 (0.0%)\n",
      "  Read 2 with adapter:                      15 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,000 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,100,000 bp\n",
      "  Read 1:     7,550,000 bp\n",
      "  Read 2:     7,550,000 bp\n",
      "Total written (filtered):     15,099,901 bp (100.0%)\n",
      "  Read 1:     7,549,948 bp\n",
      "  Read 2:     7,549,953 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 17 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t781.2\t0\t16\n",
      "4\t1\t195.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t781.2\t0\t13\n",
      "4\t2\t195.3\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/353_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/353_287_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/353_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/353_287_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/353_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/353_287_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/353_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/353_287_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        48,741 reads  @     26.5 µs/read;   2.27 M reads/minute\n",
      "Finished in 1.33 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,741\n",
      "  Read 1 with adapter:                      14 (0.0%)\n",
      "  Read 2 with adapter:                      13 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,741 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,719,782 bp\n",
      "  Read 1:     7,359,891 bp\n",
      "  Read 2:     7,359,891 bp\n",
      "Total written (filtered):     14,719,694 bp (100.0%)\n",
      "  Read 1:     7,359,848 bp\n",
      "  Read 2:     7,359,846 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t761.6\t0\t13\n",
      "4\t1\t190.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t761.6\t0\t11\n",
      "5\t1\t47.6\t0\t1\n",
      "7\t1\t3.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/354_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/354_288_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/354_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/354_288_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/354_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/354_288_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/354_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/354_288_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        48,993 reads  @     24.6 µs/read;   2.44 M reads/minute\n",
      "Finished in 1.24 s (25 us/read; 2.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,993\n",
      "  Read 1 with adapter:                      27 (0.1%)\n",
      "  Read 2 with adapter:                      16 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,993 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,795,886 bp\n",
      "  Read 1:     7,397,943 bp\n",
      "  Read 2:     7,397,943 bp\n",
      "Total written (filtered):     14,795,753 bp (100.0%)\n",
      "  Read 1:     7,397,860 bp\n",
      "  Read 2:     7,397,893 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 27 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t765.5\t0\t25\n",
      "4\t2\t191.4\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t765.5\t0\t14\n",
      "4\t2\t191.4\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/355_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/355_289_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/355_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/355_289_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/355_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/355_289_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/355_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/355_289_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        53,900 reads  @     28.3 µs/read;   2.12 M reads/minute\n",
      "Finished in 1.56 s (29 us/read; 2.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,900\n",
      "  Read 1 with adapter:                      23 (0.0%)\n",
      "  Read 2 with adapter:                      22 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,900 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,277,800 bp\n",
      "  Read 1:     8,138,900 bp\n",
      "  Read 2:     8,138,900 bp\n",
      "Total written (filtered):     16,277,655 bp (100.0%)\n",
      "  Read 1:     8,138,828 bp\n",
      "  Read 2:     8,138,827 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t842.2\t0\t21\n",
      "4\t1\t210.5\t0\t1\n",
      "5\t1\t52.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 22 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t842.2\t0\t17\n",
      "4\t4\t210.5\t0\t4\n",
      "6\t1\t13.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/356_70_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/356_290_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/356_70_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/356_290_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/356_70_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/356_290_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/356_70_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/356_290_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        44,324 reads  @     26.9 µs/read;   2.23 M reads/minute\n",
      "Finished in 1.22 s (28 us/read; 2.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,324\n",
      "  Read 1 with adapter:                      11 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,324 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,385,848 bp\n",
      "  Read 1:     6,692,924 bp\n",
      "  Read 2:     6,692,924 bp\n",
      "Total written (filtered):     13,385,795 bp (100.0%)\n",
      "  Read 1:     6,692,891 bp\n",
      "  Read 2:     6,692,904 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t692.6\t0\t11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t692.6\t0\t5\n",
      "5\t1\t43.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/357_71_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/357_291_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/357_71_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/357_291_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/357_71_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/357_291_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/357_71_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/357_291_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,522 reads  @     26.7 µs/read;   2.24 M reads/minute\n",
      "Finished in 1.36 s (27 us/read; 2.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,522\n",
      "  Read 1 with adapter:                      19 (0.0%)\n",
      "  Read 2 with adapter:                      13 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,522 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,955,644 bp\n",
      "  Read 1:     7,477,822 bp\n",
      "  Read 2:     7,477,822 bp\n",
      "Total written (filtered):     14,955,540 bp (100.0%)\n",
      "  Read 1:     7,477,762 bp\n",
      "  Read 2:     7,477,778 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t773.8\t0\t17\n",
      "4\t1\t193.4\t0\t1\n",
      "5\t1\t48.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t773.8\t0\t9\n",
      "4\t3\t193.4\t0\t3\n",
      "5\t1\t48.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/358_72_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/358_292_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/358_72_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/358_292_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/358_72_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/358_292_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/358_72_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/358_292_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        57,644 reads  @     28.0 µs/read;   2.15 M reads/minute\n",
      "Finished in 1.65 s (29 us/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,644\n",
      "  Read 1 with adapter:                      20 (0.0%)\n",
      "  Read 2 with adapter:                      19 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        57,644 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,408,488 bp\n",
      "  Read 1:     8,704,244 bp\n",
      "  Read 2:     8,704,244 bp\n",
      "Total written (filtered):     17,408,357 bp (100.0%)\n",
      "  Read 1:     8,704,182 bp\n",
      "  Read 2:     8,704,175 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t900.7\t0\t18\n",
      "4\t2\t225.2\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t900.7\t0\t14\n",
      "5\t3\t56.3\t0\t3\n",
      "6\t2\t14.1\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/359_73_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/359_293_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/359_73_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/359_293_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/359_73_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/359_293_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/359_73_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/359_293_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        45,836 reads  @     27.9 µs/read;   2.15 M reads/minute\n",
      "Finished in 1.32 s (29 us/read; 2.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,836\n",
      "  Read 1 with adapter:                      14 (0.0%)\n",
      "  Read 2 with adapter:                       7 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,836 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,842,472 bp\n",
      "  Read 1:     6,921,236 bp\n",
      "  Read 2:     6,921,236 bp\n",
      "Total written (filtered):     13,842,406 bp (100.0%)\n",
      "  Read 1:     6,921,194 bp\n",
      "  Read 2:     6,921,212 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t716.2\t0\t14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t716.2\t0\t5\n",
      "4\t1\t179.0\t0\t1\n",
      "5\t1\t44.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/360_74_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/360_294_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/360_74_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/360_294_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/360_74_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/360_294_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/360_74_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/360_294_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        45,485 reads  @     27.5 µs/read;   2.18 M reads/minute\n",
      "Finished in 1.28 s (28 us/read; 2.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,485\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                      17 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,485 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,736,470 bp\n",
      "  Read 1:     6,868,235 bp\n",
      "  Read 2:     6,868,235 bp\n",
      "Total written (filtered):     13,736,370 bp (100.0%)\n",
      "  Read 1:     6,868,188 bp\n",
      "  Read 2:     6,868,182 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t710.7\t0\t8\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 17 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t710.7\t0\t15\n",
      "4\t2\t177.7\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/361_75_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/361_295_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/361_75_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/361_295_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/361_75_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/361_295_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/361_75_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/361_295_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        52,846 reads  @     28.7 µs/read;   2.09 M reads/minute\n",
      "Finished in 1.56 s (29 us/read; 2.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,846\n",
      "  Read 1 with adapter:                      15 (0.0%)\n",
      "  Read 2 with adapter:                      14 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,846 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,959,492 bp\n",
      "  Read 1:     7,979,746 bp\n",
      "  Read 2:     7,979,746 bp\n",
      "Total written (filtered):     15,959,398 bp (100.0%)\n",
      "  Read 1:     7,979,699 bp\n",
      "  Read 2:     7,979,699 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t825.7\t0\t14\n",
      "5\t1\t51.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t825.7\t0\t12\n",
      "4\t1\t206.4\t0\t1\n",
      "7\t1\t3.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/362_76_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/362_296_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/362_76_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/362_296_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/362_76_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/362_296_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/362_76_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/362_296_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        38,796 reads  @     31.2 µs/read;   1.92 M reads/minute\n",
      "Finished in 1.25 s (32 us/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,796\n",
      "  Read 1 with adapter:                      16 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        38,796 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,716,392 bp\n",
      "  Read 1:     5,858,196 bp\n",
      "  Read 2:     5,858,196 bp\n",
      "Total written (filtered):     11,716,311 bp (100.0%)\n",
      "  Read 1:     5,858,143 bp\n",
      "  Read 2:     5,858,168 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t606.2\t0\t13\n",
      "4\t2\t151.5\t0\t2\n",
      "6\t1\t9.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t606.2\t0\t8\n",
      "4\t1\t151.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/363_77_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/363_297_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/363_77_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/363_297_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/363_77_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/363_297_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/363_77_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/363_297_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        55,268 reads  @     29.9 µs/read;   2.01 M reads/minute\n",
      "Finished in 1.69 s (31 us/read; 1.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,268\n",
      "  Read 1 with adapter:                      20 (0.0%)\n",
      "  Read 2 with adapter:                      14 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,268 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,690,936 bp\n",
      "  Read 1:     8,345,468 bp\n",
      "  Read 2:     8,345,468 bp\n",
      "Total written (filtered):     16,690,827 bp (100.0%)\n",
      "  Read 1:     8,345,405 bp\n",
      "  Read 2:     8,345,422 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t863.6\t0\t17\n",
      "4\t3\t215.9\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t863.6\t0\t11\n",
      "4\t2\t215.9\t0\t2\n",
      "5\t1\t54.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/364_78_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/364_298_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/364_78_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/364_298_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/364_78_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/364_298_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/364_78_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/364_298_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        42,041 reads  @     27.7 µs/read;   2.16 M reads/minute\n",
      "Finished in 1.20 s (28 us/read; 2.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,041\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                      13 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,041 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,696,382 bp\n",
      "  Read 1:     6,348,191 bp\n",
      "  Read 2:     6,348,191 bp\n",
      "Total written (filtered):     12,696,311 bp (100.0%)\n",
      "  Read 1:     6,348,182 bp\n",
      "  Read 2:     6,348,129 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t656.9\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t656.9\t0\t10\n",
      "6\t2\t10.3\t0\t2\n",
      "20\t1\t0.0\t2\t0 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/365_79_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/365_299_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/365_79_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/365_299_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/365_79_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/365_299_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/365_79_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/365_299_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        38,647 reads  @     31.3 µs/read;   1.92 M reads/minute\n",
      "Finished in 1.24 s (32 us/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,647\n",
      "  Read 1 with adapter:                      10 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        38,647 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,671,394 bp\n",
      "  Read 1:     5,835,697 bp\n",
      "  Read 2:     5,835,697 bp\n",
      "Total written (filtered):     11,671,326 bp (100.0%)\n",
      "  Read 1:     5,835,666 bp\n",
      "  Read 2:     5,835,660 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t603.9\t0\t9\n",
      "4\t1\t151.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t603.9\t0\t11\n",
      "4\t1\t151.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/366_80_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/366_300_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/366_80_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/366_300_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/366_80_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/366_300_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/366_80_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/366_300_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        56,395 reads  @     27.2 µs/read;   2.21 M reads/minute\n",
      "Finished in 1.56 s (28 us/read; 2.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,395\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                      20 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,395 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,031,290 bp\n",
      "  Read 1:     8,515,645 bp\n",
      "  Read 2:     8,515,645 bp\n",
      "Total written (filtered):     17,031,202 bp (100.0%)\n",
      "  Read 1:     8,515,626 bp\n",
      "  Read 2:     8,515,576 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t881.2\t0\t5\n",
      "4\t1\t220.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 20 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t881.2\t0\t16\n",
      "4\t1\t220.3\t0\t1\n",
      "5\t1\t55.1\t0\t1\n",
      "6\t2\t13.8\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/367_81_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/367_301_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/367_81_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/367_301_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/367_81_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/367_301_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/367_81_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/367_301_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        58,216 reads  @     27.3 µs/read;   2.20 M reads/minute\n",
      "Finished in 1.62 s (28 us/read; 2.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,216\n",
      "  Read 1 with adapter:                      14 (0.0%)\n",
      "  Read 2 with adapter:                      19 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        58,216 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,581,232 bp\n",
      "  Read 1:     8,790,616 bp\n",
      "  Read 2:     8,790,616 bp\n",
      "Total written (filtered):     17,581,124 bp (100.0%)\n",
      "  Read 1:     8,790,574 bp\n",
      "  Read 2:     8,790,550 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t909.6\t0\t14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t909.6\t0\t14\n",
      "4\t3\t227.4\t0\t3\n",
      "6\t2\t14.2\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/368_82_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/368_302_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/368_82_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/368_302_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/368_82_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/368_302_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/368_82_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/368_302_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,430 reads  @     36.1 µs/read;   1.66 M reads/minute\n",
      "Finished in 1.21 s (37 us/read; 1.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,430\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,430 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,793,860 bp\n",
      "  Read 1:     4,896,930 bp\n",
      "  Read 2:     4,896,930 bp\n",
      "Total written (filtered):      9,793,813 bp (100.0%)\n",
      "  Read 1:     4,896,924 bp\n",
      "  Read 2:     4,896,889 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t506.7\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t506.7\t0\t3\n",
      "32\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/369_83_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/369_303_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/369_83_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/369_303_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/369_83_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/369_303_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/369_83_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/369_303_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        40,590 reads  @     30.0 µs/read;   2.00 M reads/minute\n",
      "Finished in 1.25 s (31 us/read; 1.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,590\n",
      "  Read 1 with adapter:                      15 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        40,590 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,258,180 bp\n",
      "  Read 1:     6,129,090 bp\n",
      "  Read 2:     6,129,090 bp\n",
      "Total written (filtered):     12,258,113 bp (100.0%)\n",
      "  Read 1:     6,129,044 bp\n",
      "  Read 2:     6,129,069 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t634.2\t0\t14\n",
      "4\t1\t158.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t634.2\t0\t4\n",
      "4\t1\t158.6\t0\t1\n",
      "5\t1\t39.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/370_84_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/370_304_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/370_84_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/370_304_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/370_84_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/370_304_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/370_84_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/370_304_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        54,980 reads  @     29.9 µs/read;   2.01 M reads/minute\n",
      "Finished in 1.68 s (31 us/read; 1.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,980\n",
      "  Read 1 with adapter:                      10 (0.0%)\n",
      "  Read 2 with adapter:                      15 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,980 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,603,960 bp\n",
      "  Read 1:     8,301,980 bp\n",
      "  Read 2:     8,301,980 bp\n",
      "Total written (filtered):     16,603,877 bp (100.0%)\n",
      "  Read 1:     8,301,947 bp\n",
      "  Read 2:     8,301,930 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t859.1\t0\t7\n",
      "4\t3\t214.8\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t859.1\t0\t12\n",
      "4\t2\t214.8\t0\t2\n",
      "6\t1\t13.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/371_85_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/371_305_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/371_85_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/371_305_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/371_85_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/371_305_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/371_85_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/371_305_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        40,262 reads  @     29.7 µs/read;   2.02 M reads/minute\n",
      "Finished in 1.22 s (30 us/read; 1.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,262\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        40,262 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,159,124 bp\n",
      "  Read 1:     6,079,562 bp\n",
      "  Read 2:     6,079,562 bp\n",
      "Total written (filtered):     12,159,077 bp (100.0%)\n",
      "  Read 1:     6,079,543 bp\n",
      "  Read 2:     6,079,534 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t629.1\t0\t5\n",
      "4\t1\t157.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t629.1\t0\t8\n",
      "4\t1\t157.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/372_86_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/372_306_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/372_86_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/372_306_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/372_86_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/372_306_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/372_86_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/372_306_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        41,397 reads  @     26.1 µs/read;   2.30 M reads/minute\n",
      "Finished in 1.11 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,397\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,397 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,501,894 bp\n",
      "  Read 1:     6,250,947 bp\n",
      "  Read 2:     6,250,947 bp\n",
      "Total written (filtered):     12,501,847 bp (100.0%)\n",
      "  Read 1:     6,250,928 bp\n",
      "  Read 2:     6,250,919 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t646.8\t0\t5\n",
      "4\t1\t161.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t646.8\t0\t8\n",
      "4\t1\t161.7\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/373_87_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/373_307_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/373_87_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/373_307_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/373_87_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/373_307_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/373_87_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/373_307_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        40,726 reads  @     27.2 µs/read;   2.20 M reads/minute\n",
      "Finished in 1.14 s (28 us/read; 2.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,726\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        40,726 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,299,252 bp\n",
      "  Read 1:     6,149,626 bp\n",
      "  Read 2:     6,149,626 bp\n",
      "Total written (filtered):     12,299,227 bp (100.0%)\n",
      "  Read 1:     6,149,626 bp\n",
      "  Read 2:     6,149,601 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t636.3\t0\t7\n",
      "4\t1\t159.1\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/374_88_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/374_308_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/374_88_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/374_308_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/374_88_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/374_308_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/374_88_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/374_308_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        40,225 reads  @     27.3 µs/read;   2.19 M reads/minute\n",
      "Finished in 1.13 s (28 us/read; 2.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,225\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        40,225 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,147,950 bp\n",
      "  Read 1:     6,073,975 bp\n",
      "  Read 2:     6,073,975 bp\n",
      "Total written (filtered):     12,147,923 bp (100.0%)\n",
      "  Read 1:     6,073,966 bp\n",
      "  Read 2:     6,073,957 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t628.5\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t628.5\t0\t6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/375_89_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/375_309_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/375_89_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/375_309_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/375_89_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/375_309_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/375_89_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/375_309_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,544 reads  @     25.0 µs/read;   2.40 M reads/minute\n",
      "Finished in 1.27 s (26 us/read; 2.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,544\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,544 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,962,288 bp\n",
      "  Read 1:     7,481,144 bp\n",
      "  Read 2:     7,481,144 bp\n",
      "Total written (filtered):     14,962,239 bp (100.0%)\n",
      "  Read 1:     7,481,120 bp\n",
      "  Read 2:     7,481,119 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t774.1\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t774.1\t0\t7\n",
      "4\t1\t193.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/376_90_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/376_310_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/376_90_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/376_310_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/376_90_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/376_310_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/376_90_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/376_310_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,039 reads  @     24.4 µs/read;   2.46 M reads/minute\n",
      "Finished in 1.22 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,039\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,039 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,809,778 bp\n",
      "  Read 1:     7,404,889 bp\n",
      "  Read 2:     7,404,889 bp\n",
      "Total written (filtered):     14,809,733 bp (100.0%)\n",
      "  Read 1:     7,404,883 bp\n",
      "  Read 2:     7,404,850 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t766.2\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t766.2\t0\t6\n",
      "6\t1\t12.0\t0\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/377_91_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/377_311_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/377_91_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/377_311_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/377_91_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/377_311_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/377_91_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/377_311_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        50,842 reads  @     26.6 µs/read;   2.26 M reads/minute\n",
      "Finished in 1.38 s (27 us/read; 2.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,842\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                      18 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,842 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,354,284 bp\n",
      "  Read 1:     7,677,142 bp\n",
      "  Read 2:     7,677,142 bp\n",
      "Total written (filtered):     15,354,181 bp (100.0%)\n",
      "  Read 1:     7,677,133 bp\n",
      "  Read 2:     7,677,048 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t794.4\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 18 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t794.4\t0\t13\n",
      "5\t2\t49.7\t0\t2\n",
      "15\t3\t0.0\t1\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/378_92_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/378_312_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/378_92_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/378_312_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/378_92_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/378_312_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/378_92_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/378_312_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        62,533 reads  @     26.1 µs/read;   2.30 M reads/minute\n",
      "Finished in 1.67 s (27 us/read; 2.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,533\n",
      "  Read 1 with adapter:                      14 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        62,533 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,884,966 bp\n",
      "  Read 1:     9,442,483 bp\n",
      "  Read 2:     9,442,483 bp\n",
      "Total written (filtered):     18,884,879 bp (100.0%)\n",
      "  Read 1:     9,442,420 bp\n",
      "  Read 2:     9,442,459 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t977.1\t0\t12\n",
      "4\t1\t244.3\t0\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t977.1\t0\t8\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/379_93_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/379_313_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/379_93_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/379_313_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/379_93_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/379_313_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/379_93_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/379_313_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        58,027 reads  @     25.7 µs/read;   2.34 M reads/minute\n",
      "Finished in 1.52 s (26 us/read; 2.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,027\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                      21 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        58,027 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,524,154 bp\n",
      "  Read 1:     8,762,077 bp\n",
      "  Read 2:     8,762,077 bp\n",
      "Total written (filtered):     17,524,068 bp (100.0%)\n",
      "  Read 1:     8,762,058 bp\n",
      "  Read 2:     8,762,010 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t906.7\t0\t5\n",
      "4\t1\t226.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 21 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t906.7\t0\t18\n",
      "4\t2\t226.7\t0\t2\n",
      "5\t1\t56.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/380_94_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/380_314_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/380_94_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/380_314_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/380_94_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/380_314_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/380_94_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/380_314_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        58,509 reads  @     24.2 µs/read;   2.48 M reads/minute\n",
      "Finished in 1.44 s (25 us/read; 2.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,509\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        58,509 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,669,718 bp\n",
      "  Read 1:     8,834,859 bp\n",
      "  Read 2:     8,834,859 bp\n",
      "Total written (filtered):     17,669,673 bp (100.0%)\n",
      "  Read 1:     8,834,850 bp\n",
      "  Read 2:     8,834,823 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t914.2\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t914.2\t0\t12\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/381_95_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/381_315_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/381_95_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/381_315_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/381_95_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/381_315_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/381_95_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/381_315_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        53,634 reads  @     27.1 µs/read;   2.22 M reads/minute\n",
      "Finished in 1.48 s (28 us/read; 2.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,634\n",
      "  Read 1 with adapter:                      16 (0.0%)\n",
      "  Read 2 with adapter:                      16 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,634 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,197,468 bp\n",
      "  Read 1:     8,098,734 bp\n",
      "  Read 2:     8,098,734 bp\n",
      "Total written (filtered):     16,197,369 bp (100.0%)\n",
      "  Read 1:     8,098,683 bp\n",
      "  Read 2:     8,098,686 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t838.0\t0\t13\n",
      "4\t3\t209.5\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t838.0\t0\t16\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/382_96_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/382_316_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/382_96_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/382_316_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/382_96_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/382_316_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/382_96_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/382_316_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        60,258 reads  @     25.6 µs/read;   2.35 M reads/minute\n",
      "Finished in 1.57 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,258\n",
      "  Read 1 with adapter:                      13 (0.0%)\n",
      "  Read 2 with adapter:                      18 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        60,258 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,197,916 bp\n",
      "  Read 1:     9,098,958 bp\n",
      "  Read 2:     9,098,958 bp\n",
      "Total written (filtered):     18,197,819 bp (100.0%)\n",
      "  Read 1:     9,098,918 bp\n",
      "  Read 2:     9,098,901 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t941.5\t0\t12\n",
      "4\t1\t235.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 18 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t941.5\t0\t15\n",
      "4\t3\t235.4\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/383_97_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/383_317_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/383_97_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/383_317_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/383_97_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/383_317_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/383_97_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/383_317_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        61,770 reads  @     23.7 µs/read;   2.53 M reads/minute\n",
      "Finished in 1.50 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,770\n",
      "  Read 1 with adapter:                      11 (0.0%)\n",
      "  Read 2 with adapter:                      18 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        61,770 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,654,540 bp\n",
      "  Read 1:     9,327,270 bp\n",
      "  Read 2:     9,327,270 bp\n",
      "Total written (filtered):     18,654,445 bp (100.0%)\n",
      "  Read 1:     9,327,232 bp\n",
      "  Read 2:     9,327,213 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t965.2\t0\t9\n",
      "4\t1\t241.3\t0\t1\n",
      "7\t1\t3.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 18 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t965.2\t0\t16\n",
      "4\t1\t241.3\t0\t1\n",
      "5\t1\t60.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/384_98_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/384_318_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/384_98_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/384_318_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/384_98_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/384_318_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/384_98_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/384_318_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        55,227 reads  @     27.4 µs/read;   2.19 M reads/minute\n",
      "Finished in 1.54 s (28 us/read; 2.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,227\n",
      "  Read 1 with adapter:                      13 (0.0%)\n",
      "  Read 2 with adapter:                      23 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,227 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,678,554 bp\n",
      "  Read 1:     8,339,277 bp\n",
      "  Read 2:     8,339,277 bp\n",
      "Total written (filtered):     16,678,439 bp (100.0%)\n",
      "  Read 1:     8,339,236 bp\n",
      "  Read 2:     8,339,203 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t862.9\t0\t11\n",
      "4\t2\t215.7\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 23 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t862.9\t0\t22\n",
      "8\t1\t0.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/385_99_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/385_319_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/385_99_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/385_319_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/385_99_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/385_319_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/385_99_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/385_319_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        65,838 reads  @     24.2 µs/read;   2.48 M reads/minute\n",
      "Finished in 1.63 s (25 us/read; 2.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,838\n",
      "  Read 1 with adapter:                      12 (0.0%)\n",
      "  Read 2 with adapter:                      19 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        65,838 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,883,076 bp\n",
      "  Read 1:     9,941,538 bp\n",
      "  Read 2:     9,941,538 bp\n",
      "Total written (filtered):     19,882,978 bp (100.0%)\n",
      "  Read 1:     9,941,499 bp\n",
      "  Read 2:     9,941,479 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1028.7\t0\t10\n",
      "4\t1\t257.2\t0\t1\n",
      "5\t1\t64.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t1028.7\t0\t17\n",
      "4\t2\t257.2\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/386_100_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/386_320_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/386_100_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/386_320_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/386_100_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/386_320_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/386_100_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/386_320_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        50,573 reads  @     26.1 µs/read;   2.30 M reads/minute\n",
      "Finished in 1.35 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,573\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                      14 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,573 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,273,046 bp\n",
      "  Read 1:     7,636,523 bp\n",
      "  Read 2:     7,636,523 bp\n",
      "Total written (filtered):     15,272,977 bp (100.0%)\n",
      "  Read 1:     7,636,496 bp\n",
      "  Read 2:     7,636,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t790.2\t0\t9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t790.2\t0\t14\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/387_101_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/387_321_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/387_101_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/387_321_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/387_101_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/387_321_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/387_101_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/387_321_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,527 reads  @     23.9 µs/read;   2.51 M reads/minute\n",
      "Finished in 1.21 s (24 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,527\n",
      "  Read 1 with adapter:                      14 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,527 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,957,154 bp\n",
      "  Read 1:     7,478,577 bp\n",
      "  Read 2:     7,478,577 bp\n",
      "Total written (filtered):     14,957,061 bp (100.0%)\n",
      "  Read 1:     7,478,514 bp\n",
      "  Read 2:     7,478,547 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t773.9\t0\t11\n",
      "10\t3\t0.0\t1\t0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t773.9\t0\t10\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/391_102_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/391_322_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/391_102_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/391_322_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/391_102_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/391_322_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/391_102_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/391_322_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        61,267 reads  @     24.4 µs/read;   2.46 M reads/minute\n",
      "Finished in 1.52 s (25 us/read; 2.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,267\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                      15 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        61,267 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,502,634 bp\n",
      "  Read 1:     9,251,317 bp\n",
      "  Read 2:     9,251,317 bp\n",
      "Total written (filtered):     18,502,548 bp (100.0%)\n",
      "  Read 1:     9,251,306 bp\n",
      "  Read 2:     9,251,242 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t957.3\t0\t2\n",
      "5\t1\t59.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t957.3\t0\t14\n",
      "33\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/392_103_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/392_323_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/392_103_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/392_323_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/392_103_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/392_323_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/392_103_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/392_323_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        57,755 reads  @     25.0 µs/read;   2.40 M reads/minute\n",
      "Finished in 1.47 s (26 us/read; 2.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,755\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        57,755 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,442,010 bp\n",
      "  Read 1:     8,721,005 bp\n",
      "  Read 2:     8,721,005 bp\n",
      "Total written (filtered):     17,441,965 bp (100.0%)\n",
      "  Read 1:     8,720,993 bp\n",
      "  Read 2:     8,720,972 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t902.4\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t902.4\t0\t8\n",
      "4\t1\t225.6\t0\t1\n",
      "5\t1\t56.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/393_104_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/393_324_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/393_104_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/393_324_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/393_104_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/393_324_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/393_104_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/393_324_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        35,818 reads  @     27.7 µs/read;   2.16 M reads/minute\n",
      "Finished in 1.02 s (29 us/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,818\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,818 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,817,036 bp\n",
      "  Read 1:     5,408,518 bp\n",
      "  Read 2:     5,408,518 bp\n",
      "Total written (filtered):     10,816,994 bp (100.0%)\n",
      "  Read 1:     5,408,509 bp\n",
      "  Read 2:     5,408,485 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t559.7\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t559.7\t0\t7\n",
      "4\t3\t139.9\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/394_105_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/394_325_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/394_105_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/394_325_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/394_105_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/394_325_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/394_105_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/394_325_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        31,304 reads  @     31.7 µs/read;   1.89 M reads/minute\n",
      "Finished in 1.02 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,304\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                      13 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        31,304 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,453,808 bp\n",
      "  Read 1:     4,726,904 bp\n",
      "  Read 2:     4,726,904 bp\n",
      "Total written (filtered):      9,453,762 bp (100.0%)\n",
      "  Read 1:     4,726,898 bp\n",
      "  Read 2:     4,726,864 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t489.1\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t489.1\t0\t12\n",
      "4\t1\t122.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/395_106_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/395_326_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/395_106_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/395_326_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/395_106_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/395_326_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/395_106_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/395_326_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        34,302 reads  @     32.9 µs/read;   1.82 M reads/minute\n",
      "Finished in 1.16 s (34 us/read; 1.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,302\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,302 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,359,204 bp\n",
      "  Read 1:     5,179,602 bp\n",
      "  Read 2:     5,179,602 bp\n",
      "Total written (filtered):     10,359,168 bp (100.0%)\n",
      "  Read 1:     5,179,596 bp\n",
      "  Read 2:     5,179,572 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t536.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t536.0\t0\t10\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/396_107_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/396_327_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/396_107_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/396_327_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/396_107_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/396_327_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/396_107_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/396_327_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        33,842 reads  @     31.1 µs/read;   1.93 M reads/minute\n",
      "Finished in 1.08 s (32 us/read; 1.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,842\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,842 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,220,284 bp\n",
      "  Read 1:     5,110,142 bp\n",
      "  Read 2:     5,110,142 bp\n",
      "Total written (filtered):     10,220,238 bp (100.0%)\n",
      "  Read 1:     5,110,130 bp\n",
      "  Read 2:     5,110,108 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t528.8\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t528.8\t0\t7\n",
      "13\t1\t0.0\t1\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/397_108_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/397_328_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/397_108_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/397_328_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/397_108_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/397_328_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/397_108_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/397_328_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        15,082 reads  @     49.7 µs/read;   1.21 M reads/minute\n",
      "Finished in 0.78 s (52 us/read; 1.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,082\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        15,082 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,554,764 bp\n",
      "  Read 1:     2,277,382 bp\n",
      "  Read 2:     2,277,382 bp\n",
      "Total written (filtered):      4,554,752 bp (100.0%)\n",
      "  Read 1:     2,277,376 bp\n",
      "  Read 2:     2,277,376 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t235.7\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t235.7\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/398_109_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/398_329_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/398_109_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/398_329_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/398_109_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/398_329_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/398_109_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/398_329_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        26,534 reads  @     32.1 µs/read;   1.87 M reads/minute\n",
      "Finished in 0.88 s (33 us/read; 1.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,534\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        26,534 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,013,268 bp\n",
      "  Read 1:     4,006,634 bp\n",
      "  Read 2:     4,006,634 bp\n",
      "Total written (filtered):      8,013,248 bp (100.0%)\n",
      "  Read 1:     4,006,634 bp\n",
      "  Read 2:     4,006,614 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t414.6\t0\t4\n",
      "4\t2\t103.6\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/399_110_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/399_330_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/399_110_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/399_330_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/399_110_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/399_330_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/399_110_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/399_330_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        29,119 reads  @     30.9 µs/read;   1.94 M reads/minute\n",
      "Finished in 0.93 s (32 us/read; 1.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,119\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,119 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,793,938 bp\n",
      "  Read 1:     4,396,969 bp\n",
      "  Read 2:     4,396,969 bp\n",
      "Total written (filtered):      8,793,893 bp (100.0%)\n",
      "  Read 1:     4,396,951 bp\n",
      "  Read 2:     4,396,942 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t455.0\t0\t3\n",
      "4\t1\t113.7\t0\t1\n",
      "5\t1\t28.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t455.0\t0\t9\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/4_212_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/4_432_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/4_212_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/4_432_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/4_212_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/4_432_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/4_212_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/4_432_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        62,377 reads  @     26.3 µs/read;   2.28 M reads/minute\n",
      "Finished in 1.68 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,377\n",
      "  Read 1 with adapter:                      20 (0.0%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        62,377 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,837,854 bp\n",
      "  Read 1:     9,418,927 bp\n",
      "  Read 2:     9,418,927 bp\n",
      "Total written (filtered):     18,837,755 bp (100.0%)\n",
      "  Read 1:     9,418,865 bp\n",
      "  Read 2:     9,418,890 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t974.6\t0\t18\n",
      "4\t2\t243.7\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t974.6\t0\t8\n",
      "4\t2\t243.7\t0\t2\n",
      "5\t1\t60.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/400_112_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/400_332_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/400_112_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/400_332_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/400_112_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/400_332_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/400_112_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/400_332_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        32,189 reads  @     31.3 µs/read;   1.92 M reads/minute\n",
      "Finished in 1.03 s (32 us/read; 1.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,189\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,721,078 bp\n",
      "  Read 1:     4,860,539 bp\n",
      "  Read 2:     4,860,539 bp\n",
      "Total written (filtered):      9,721,026 bp (100.0%)\n",
      "  Read 1:     4,860,527 bp\n",
      "  Read 2:     4,860,499 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t503.0\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t503.0\t0\t8\n",
      "4\t1\t125.7\t0\t1\n",
      "5\t1\t31.4\t0\t1\n",
      "7\t1\t2.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/401_113_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/401_333_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/401_113_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/401_333_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/401_113_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/401_333_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/401_113_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/401_333_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        25,273 reads  @     33.8 µs/read;   1.78 M reads/minute\n",
      "Finished in 0.88 s (35 us/read; 1.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,273\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,273 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,632,446 bp\n",
      "  Read 1:     3,816,223 bp\n",
      "  Read 2:     3,816,223 bp\n",
      "Total written (filtered):      7,632,416 bp (100.0%)\n",
      "  Read 1:     3,816,208 bp\n",
      "  Read 2:     3,816,208 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t394.9\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t394.9\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/402_114_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/402_334_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/402_114_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/402_334_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/402_114_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/402_334_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/402_114_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/402_334_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        29,320 reads  @     32.4 µs/read;   1.85 M reads/minute\n",
      "Finished in 0.98 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,320\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,320 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,854,640 bp\n",
      "  Read 1:     4,427,320 bp\n",
      "  Read 2:     4,427,320 bp\n",
      "Total written (filtered):      8,854,604 bp (100.0%)\n",
      "  Read 1:     4,427,302 bp\n",
      "  Read 2:     4,427,302 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t458.1\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t458.1\t0\t6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/403_115_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/403_335_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/403_115_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/403_335_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/403_115_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/403_335_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/403_115_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/403_335_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        23,308 reads  @     31.7 µs/read;   1.89 M reads/minute\n",
      "Finished in 0.77 s (33 us/read; 1.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,308\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,308 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,039,016 bp\n",
      "  Read 1:     3,519,508 bp\n",
      "  Read 2:     3,519,508 bp\n",
      "Total written (filtered):      7,039,010 bp (100.0%)\n",
      "  Read 1:     3,519,505 bp\n",
      "  Read 2:     3,519,505 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t364.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t364.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/404_116_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/404_336_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/404_116_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/404_336_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/404_116_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/404_336_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/404_116_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/404_336_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        27,837 reads  @     31.8 µs/read;   1.88 M reads/minute\n",
      "Finished in 0.92 s (33 us/read; 1.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,837\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,837 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,406,774 bp\n",
      "  Read 1:     4,203,387 bp\n",
      "  Read 2:     4,203,387 bp\n",
      "Total written (filtered):      8,406,728 bp (100.0%)\n",
      "  Read 1:     4,203,372 bp\n",
      "  Read 2:     4,203,356 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t435.0\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t435.0\t0\t9\n",
      "4\t1\t108.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/405_117_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/405_337_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/405_117_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/405_337_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/405_117_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/405_337_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/405_117_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/405_337_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        23,142 reads  @     34.4 µs/read;   1.75 M reads/minute\n",
      "Finished in 0.82 s (35 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,142\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,142 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,988,884 bp\n",
      "  Read 1:     3,494,442 bp\n",
      "  Read 2:     3,494,442 bp\n",
      "Total written (filtered):      6,988,863 bp (100.0%)\n",
      "  Read 1:     3,494,436 bp\n",
      "  Read 2:     3,494,427 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t361.6\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t361.6\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/406_118_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/406_338_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/406_118_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/406_338_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/406_118_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/406_338_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/406_118_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/406_338_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,927 reads  @     32.6 µs/read;   1.84 M reads/minute\n",
      "Finished in 1.10 s (34 us/read; 1.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,927\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,927 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,943,954 bp\n",
      "  Read 1:     4,971,977 bp\n",
      "  Read 2:     4,971,977 bp\n",
      "Total written (filtered):      9,943,891 bp (100.0%)\n",
      "  Read 1:     4,971,956 bp\n",
      "  Read 2:     4,971,935 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t514.5\t0\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t514.5\t0\t6\n",
      "4\t1\t128.6\t0\t1\n",
      "5\t4\t32.2\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/407_119_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/407_339_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/407_119_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/407_339_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/407_119_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/407_339_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/407_119_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/407_339_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        26,351 reads  @     32.1 µs/read;   1.87 M reads/minute\n",
      "Finished in 0.87 s (33 us/read; 1.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,351\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        26,351 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,958,002 bp\n",
      "  Read 1:     3,979,001 bp\n",
      "  Read 2:     3,979,001 bp\n",
      "Total written (filtered):      7,957,936 bp (100.0%)\n",
      "  Read 1:     3,978,988 bp\n",
      "  Read 2:     3,978,948 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t411.7\t0\t2\n",
      "7\t1\t1.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t411.7\t0\t9\n",
      "26\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/408_120_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/408_340_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/408_120_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/408_340_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/408_120_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/408_340_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/408_120_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/408_340_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        34,139 reads  @     29.8 µs/read;   2.01 M reads/minute\n",
      "Finished in 1.05 s (31 us/read; 1.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,139\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,139 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,309,978 bp\n",
      "  Read 1:     5,154,989 bp\n",
      "  Read 2:     5,154,989 bp\n",
      "Total written (filtered):     10,309,950 bp (100.0%)\n",
      "  Read 1:     5,154,979 bp\n",
      "  Read 2:     5,154,971 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t533.4\t0\t2\n",
      "4\t1\t133.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t533.4\t0\t6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/409_121_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/409_341_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/409_121_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/409_341_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/409_121_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/409_341_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/409_121_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/409_341_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        17,729 reads  @     46.1 µs/read;   1.30 M reads/minute\n",
      "Finished in 0.85 s (48 us/read; 1.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,729\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        17,729 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,354,158 bp\n",
      "  Read 1:     2,677,079 bp\n",
      "  Read 2:     2,677,079 bp\n",
      "Total written (filtered):      5,354,134 bp (100.0%)\n",
      "  Read 1:     2,677,064 bp\n",
      "  Read 2:     2,677,070 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t277.0\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t277.0\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/410_122_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/410_342_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/410_122_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/410_342_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/410_122_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/410_342_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/410_122_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/410_342_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        35,562 reads  @     32.3 µs/read;   1.86 M reads/minute\n",
      "Finished in 1.18 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,562\n",
      "  Read 1 with adapter:                      19 (0.1%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,562 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,739,724 bp\n",
      "  Read 1:     5,369,862 bp\n",
      "  Read 2:     5,369,862 bp\n",
      "Total written (filtered):     10,739,613 bp (100.0%)\n",
      "  Read 1:     5,369,784 bp\n",
      "  Read 2:     5,369,829 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t555.7\t0\t16\n",
      "10\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t555.7\t0\t7\n",
      "6\t2\t8.7\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/411_123_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/411_343_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/411_123_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/411_343_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/411_123_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/411_343_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/411_123_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/411_343_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        34,580 reads  @     27.8 µs/read;   2.16 M reads/minute\n",
      "Finished in 0.99 s (29 us/read; 2.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,580\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,580 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,443,160 bp\n",
      "  Read 1:     5,221,580 bp\n",
      "  Read 2:     5,221,580 bp\n",
      "Total written (filtered):     10,443,137 bp (100.0%)\n",
      "  Read 1:     5,221,575 bp\n",
      "  Read 2:     5,221,562 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t33.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t540.3\t0\t6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/412_124_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/412_344_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/412_124_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/412_344_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/412_124_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/412_344_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/412_124_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/412_344_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,216 reads  @     34.0 µs/read;   1.76 M reads/minute\n",
      "Finished in 1.12 s (35 us/read; 1.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,216\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,216 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,729,232 bp\n",
      "  Read 1:     4,864,616 bp\n",
      "  Read 2:     4,864,616 bp\n",
      "Total written (filtered):      9,729,202 bp (100.0%)\n",
      "  Read 1:     4,864,601 bp\n",
      "  Read 2:     4,864,601 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t503.4\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t503.4\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/413_125_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/413_345_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/413_125_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/413_345_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/413_125_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/413_345_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/413_125_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/413_345_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        31,329 reads  @     31.7 µs/read;   1.89 M reads/minute\n",
      "Finished in 1.02 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,329\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                       7 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        31,329 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,461,358 bp\n",
      "  Read 1:     4,730,679 bp\n",
      "  Read 2:     4,730,679 bp\n",
      "Total written (filtered):      9,461,320 bp (100.0%)\n",
      "  Read 1:     4,730,663 bp\n",
      "  Read 2:     4,730,657 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t489.5\t0\t3\n",
      "7\t1\t1.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t489.5\t0\t6\n",
      "4\t1\t122.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/414_126_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/414_346_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/414_126_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/414_346_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/414_126_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/414_346_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/414_126_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/414_346_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        22,224 reads  @     33.9 µs/read;   1.77 M reads/minute\n",
      "Finished in 0.78 s (35 us/read; 1.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,224\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,224 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,711,648 bp\n",
      "  Read 1:     3,355,824 bp\n",
      "  Read 2:     3,355,824 bp\n",
      "Total written (filtered):      6,711,633 bp (100.0%)\n",
      "  Read 1:     3,355,818 bp\n",
      "  Read 2:     3,355,815 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t347.2\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t347.2\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/415_127_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/415_347_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/415_127_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/415_347_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/415_127_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/415_347_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/415_127_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/415_347_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        30,299 reads  @     29.2 µs/read;   2.05 M reads/minute\n",
      "Finished in 0.91 s (30 us/read; 1.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,299\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,299 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,150,298 bp\n",
      "  Read 1:     4,575,149 bp\n",
      "  Read 2:     4,575,149 bp\n",
      "Total written (filtered):      9,150,276 bp (100.0%)\n",
      "  Read 1:     4,575,145 bp\n",
      "  Read 2:     4,575,131 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t118.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t473.4\t0\t6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/416_128_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/416_348_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/416_128_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/416_348_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/416_128_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/416_348_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/416_128_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/416_348_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        28,410 reads  @     27.5 µs/read;   2.18 M reads/minute\n",
      "Finished in 0.81 s (28 us/read; 2.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,410\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                       7 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,410 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,579,820 bp\n",
      "  Read 1:     4,289,910 bp\n",
      "  Read 2:     4,289,910 bp\n",
      "Total written (filtered):      8,579,790 bp (100.0%)\n",
      "  Read 1:     4,289,901 bp\n",
      "  Read 2:     4,289,889 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t443.9\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t443.9\t0\t7\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/417_129_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/417_349_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/417_129_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/417_349_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/417_129_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/417_349_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/417_129_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/417_349_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        27,090 reads  @     30.1 µs/read;   2.00 M reads/minute\n",
      "Finished in 0.84 s (31 us/read; 1.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,090\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,090 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,181,180 bp\n",
      "  Read 1:     4,090,590 bp\n",
      "  Read 2:     4,090,590 bp\n",
      "Total written (filtered):      8,181,144 bp (100.0%)\n",
      "  Read 1:     4,090,576 bp\n",
      "  Read 2:     4,090,568 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t423.3\t0\t3\n",
      "5\t1\t26.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t423.3\t0\t4\n",
      "4\t1\t105.8\t0\t1\n",
      "6\t1\t6.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/418_130_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/418_350_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/418_130_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/418_350_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/418_130_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/418_350_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/418_130_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/418_350_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        14,490 reads  @     47.1 µs/read;   1.27 M reads/minute\n",
      "Finished in 0.71 s (49 us/read; 1.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,490\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        14,490 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,375,980 bp\n",
      "  Read 1:     2,187,990 bp\n",
      "  Read 2:     2,187,990 bp\n",
      "Total written (filtered):      4,375,974 bp (100.0%)\n",
      "  Read 1:     2,187,987 bp\n",
      "  Read 2:     2,187,987 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t226.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t226.4\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/419_131_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/419_351_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/419_131_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/419_351_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/419_131_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/419_351_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/419_131_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/419_351_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        27,838 reads  @     29.6 µs/read;   2.03 M reads/minute\n",
      "Finished in 0.86 s (31 us/read; 1.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,838\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,838 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,407,076 bp\n",
      "  Read 1:     4,203,538 bp\n",
      "  Read 2:     4,203,538 bp\n",
      "Total written (filtered):      8,407,064 bp (100.0%)\n",
      "  Read 1:     4,203,535 bp\n",
      "  Read 2:     4,203,529 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t435.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t435.0\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/420_132_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/420_352_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/420_132_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/420_352_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/420_132_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/420_352_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/420_132_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/420_352_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        30,343 reads  @     31.8 µs/read;   1.89 M reads/minute\n",
      "Finished in 0.99 s (33 us/read; 1.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,343\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,343 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,163,586 bp\n",
      "  Read 1:     4,581,793 bp\n",
      "  Read 2:     4,581,793 bp\n",
      "Total written (filtered):      9,163,542 bp (100.0%)\n",
      "  Read 1:     4,581,781 bp\n",
      "  Read 2:     4,581,761 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t474.1\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t474.1\t0\t8\n",
      "4\t2\t118.5\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/421_133_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/421_353_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/421_133_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/421_353_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/421_133_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/421_353_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/421_133_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/421_353_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        24,825 reads  @     31.1 µs/read;   1.93 M reads/minute\n",
      "Finished in 0.79 s (32 us/read; 1.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,825\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        24,825 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,497,150 bp\n",
      "  Read 1:     3,748,575 bp\n",
      "  Read 2:     3,748,575 bp\n",
      "Total written (filtered):      7,497,127 bp (100.0%)\n",
      "  Read 1:     3,748,563 bp\n",
      "  Read 2:     3,748,564 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t387.9\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t387.9\t0\t2\n",
      "5\t1\t24.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/422_134_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/422_354_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/422_134_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/422_354_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/422_134_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/422_354_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/422_134_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/422_354_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,362 reads  @     34.6 µs/read;   1.74 M reads/minute\n",
      "Finished in 1.15 s (36 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,362\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,362 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,773,324 bp\n",
      "  Read 1:     4,886,662 bp\n",
      "  Read 2:     4,886,662 bp\n",
      "Total written (filtered):      9,773,288 bp (100.0%)\n",
      "  Read 1:     4,886,638 bp\n",
      "  Read 2:     4,886,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t505.7\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t505.7\t0\t4\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/423_135_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/423_355_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/423_135_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/423_355_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/423_135_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/423_355_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/423_135_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/423_355_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        29,892 reads  @     33.6 µs/read;   1.79 M reads/minute\n",
      "Finished in 1.04 s (35 us/read; 1.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,892\n",
      "  Read 1 with adapter:                      12 (0.0%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,892 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,027,384 bp\n",
      "  Read 1:     4,513,692 bp\n",
      "  Read 2:     4,513,692 bp\n",
      "Total written (filtered):      9,027,333 bp (100.0%)\n",
      "  Read 1:     4,513,656 bp\n",
      "  Read 2:     4,513,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t467.1\t0\t12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t467.1\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/424_136_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/424_356_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/424_136_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/424_356_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/424_136_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/424_356_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/424_136_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/424_356_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        26,175 reads  @     33.6 µs/read;   1.78 M reads/minute\n",
      "Finished in 0.91 s (35 us/read; 1.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,175\n",
      "  Read 1 with adapter:                      11 (0.0%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        26,175 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,904,850 bp\n",
      "  Read 1:     3,952,425 bp\n",
      "  Read 2:     3,952,425 bp\n",
      "Total written (filtered):      7,904,809 bp (100.0%)\n",
      "  Read 1:     3,952,392 bp\n",
      "  Read 2:     3,952,417 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t409.0\t0\t11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t409.0\t0\t1\n",
      "5\t1\t25.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/425_137_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/425_357_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/425_137_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/425_357_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/425_137_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/425_357_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/425_137_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/425_357_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        27,073 reads  @     35.0 µs/read;   1.71 M reads/minute\n",
      "Finished in 0.98 s (36 us/read; 1.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,073\n",
      "  Read 1 with adapter:                      18 (0.1%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,073 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,176,046 bp\n",
      "  Read 1:     4,088,023 bp\n",
      "  Read 2:     4,088,023 bp\n",
      "Total written (filtered):      8,175,963 bp (100.0%)\n",
      "  Read 1:     4,087,969 bp\n",
      "  Read 2:     4,087,994 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 18 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t423.0\t0\t18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t423.0\t0\t7\n",
      "4\t2\t105.8\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/426_138_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/426_358_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/426_138_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/426_358_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/426_138_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/426_358_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/426_138_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/426_358_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        26,205 reads  @     31.7 µs/read;   1.90 M reads/minute\n",
      "Finished in 0.86 s (33 us/read; 1.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,205\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        26,205 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,913,910 bp\n",
      "  Read 1:     3,956,955 bp\n",
      "  Read 2:     3,956,955 bp\n",
      "Total written (filtered):      7,913,885 bp (100.0%)\n",
      "  Read 1:     3,956,936 bp\n",
      "  Read 2:     3,956,949 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t409.5\t0\t5\n",
      "4\t1\t102.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t409.5\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/427_139_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/427_359_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/427_139_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/427_359_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/427_139_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/427_359_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/427_139_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/427_359_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        34,200 reads  @     33.6 µs/read;   1.79 M reads/minute\n",
      "Finished in 1.17 s (34 us/read; 1.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,200\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,200 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,328,400 bp\n",
      "  Read 1:     5,164,200 bp\n",
      "  Read 2:     5,164,200 bp\n",
      "Total written (filtered):     10,328,375 bp (100.0%)\n",
      "  Read 1:     5,164,190 bp\n",
      "  Read 2:     5,164,185 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t534.4\t0\t2\n",
      "4\t1\t133.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t534.4\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/428_140_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/428_360_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/428_140_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/428_360_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/428_140_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/428_360_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/428_140_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/428_360_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        23,868 reads  @     35.7 µs/read;   1.68 M reads/minute\n",
      "Finished in 0.88 s (37 us/read; 1.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,868\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,868 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,208,136 bp\n",
      "  Read 1:     3,604,068 bp\n",
      "  Read 2:     3,604,068 bp\n",
      "Total written (filtered):      7,208,105 bp (100.0%)\n",
      "  Read 1:     3,604,062 bp\n",
      "  Read 2:     3,604,043 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t372.9\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t372.9\t0\t7\n",
      "4\t1\t93.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/429_141_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/429_361_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/429_141_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/429_361_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/429_141_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/429_361_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/429_141_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/429_361_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        33,405 reads  @     34.4 µs/read;   1.74 M reads/minute\n",
      "Finished in 1.19 s (36 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,405\n",
      "  Read 1 with adapter:                      10 (0.0%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,405 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,088,310 bp\n",
      "  Read 1:     5,044,155 bp\n",
      "  Read 2:     5,044,155 bp\n",
      "Total written (filtered):     10,088,246 bp (100.0%)\n",
      "  Read 1:     5,044,125 bp\n",
      "  Read 2:     5,044,121 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t522.0\t0\t10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t522.0\t0\t10\n",
      "4\t1\t130.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/430_142_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/430_362_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/430_142_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/430_362_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/430_142_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/430_362_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/430_142_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/430_362_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,169 reads  @     33.5 µs/read;   1.79 M reads/minute\n",
      "Finished in 1.11 s (34 us/read; 1.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,169\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,169 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,715,038 bp\n",
      "  Read 1:     4,857,519 bp\n",
      "  Read 2:     4,857,519 bp\n",
      "Total written (filtered):      9,714,976 bp (100.0%)\n",
      "  Read 1:     4,857,490 bp\n",
      "  Read 2:     4,857,486 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t502.6\t0\t8\n",
      "5\t1\t31.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t502.6\t0\t8\n",
      "4\t1\t125.7\t0\t1\n",
      "5\t1\t31.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/431_143_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/431_363_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/431_143_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/431_363_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/431_143_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/431_363_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/431_143_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/431_363_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        29,077 reads  @     30.1 µs/read;   1.99 M reads/minute\n",
      "Finished in 0.90 s (31 us/read; 1.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,077\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,077 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,781,254 bp\n",
      "  Read 1:     4,390,627 bp\n",
      "  Read 2:     4,390,627 bp\n",
      "Total written (filtered):      8,781,226 bp (100.0%)\n",
      "  Read 1:     4,390,602 bp\n",
      "  Read 2:     4,390,624 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t454.3\t0\t2\n",
      "7\t1\t1.8\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t454.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/432_144_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/432_364_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/432_144_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/432_364_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/432_144_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/432_364_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/432_144_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/432_364_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        38,471 reads  @     24.2 µs/read;   2.48 M reads/minute\n",
      "Finished in 0.96 s (25 us/read; 2.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,471\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        38,471 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,618,242 bp\n",
      "  Read 1:     5,809,121 bp\n",
      "  Read 2:     5,809,121 bp\n",
      "Total written (filtered):     11,618,208 bp (100.0%)\n",
      "  Read 1:     5,809,111 bp\n",
      "  Read 2:     5,809,097 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t601.1\t0\t2\n",
      "4\t1\t150.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t601.1\t0\t8\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/433_145_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/433_365_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/433_145_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/433_365_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/433_145_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/433_365_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/433_145_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/433_365_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        34,586 reads  @     29.3 µs/read;   2.05 M reads/minute\n",
      "Finished in 1.04 s (30 us/read; 1.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,586\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,586 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,444,972 bp\n",
      "  Read 1:     5,222,486 bp\n",
      "  Read 2:     5,222,486 bp\n",
      "Total written (filtered):     10,444,951 bp (100.0%)\n",
      "  Read 1:     5,222,486 bp\n",
      "  Read 2:     5,222,465 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t540.4\t0\t5\n",
      "6\t1\t8.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/434_146_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/434_366_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/434_146_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/434_366_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/434_146_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/434_366_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/434_146_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/434_366_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        35,175 reads  @     30.0 µs/read;   2.00 M reads/minute\n",
      "Finished in 1.09 s (31 us/read; 1.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,175\n",
      "  Read 1 with adapter:                      20 (0.1%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,175 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,622,850 bp\n",
      "  Read 1:     5,311,425 bp\n",
      "  Read 2:     5,311,425 bp\n",
      "Total written (filtered):     10,622,749 bp (100.0%)\n",
      "  Read 1:     5,311,363 bp\n",
      "  Read 2:     5,311,386 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t549.6\t0\t18\n",
      "4\t2\t137.4\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t549.6\t0\t7\n",
      "4\t3\t137.4\t0\t3\n",
      "6\t1\t8.6\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/435_147_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/435_367_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/435_147_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/435_367_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/435_147_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/435_367_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/435_147_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/435_367_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        24,646 reads  @     30.1 µs/read;   2.00 M reads/minute\n",
      "Finished in 0.77 s (31 us/read; 1.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,646\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        24,646 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,443,092 bp\n",
      "  Read 1:     3,721,546 bp\n",
      "  Read 2:     3,721,546 bp\n",
      "Total written (filtered):      7,443,059 bp (100.0%)\n",
      "  Read 1:     3,721,525 bp\n",
      "  Read 2:     3,721,534 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t385.1\t0\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t385.1\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/436_148_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/436_368_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/436_148_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/436_368_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/436_148_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/436_368_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/436_148_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/436_368_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        17,123 reads  @     40.7 µs/read;   1.47 M reads/minute\n",
      "Finished in 0.73 s (43 us/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,123\n",
      "  Read 1 with adapter:                      12 (0.1%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        17,123 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,171,146 bp\n",
      "  Read 1:     2,585,573 bp\n",
      "  Read 2:     2,585,573 bp\n",
      "Total written (filtered):      5,171,097 bp (100.0%)\n",
      "  Read 1:     2,585,537 bp\n",
      "  Read 2:     2,585,560 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t267.5\t0\t12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t267.5\t0\t3\n",
      "4\t1\t66.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/437_149_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/437_369_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/437_149_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/437_369_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/437_149_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/437_369_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/437_149_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/437_369_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        34,946 reads  @     30.2 µs/read;   1.99 M reads/minute\n",
      "Finished in 1.08 s (31 us/read; 1.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,946\n",
      "  Read 1 with adapter:                      10 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,946 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,553,692 bp\n",
      "  Read 1:     5,276,846 bp\n",
      "  Read 2:     5,276,846 bp\n",
      "Total written (filtered):     10,553,616 bp (100.0%)\n",
      "  Read 1:     5,276,806 bp\n",
      "  Read 2:     5,276,810 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t546.0\t0\t9\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t546.0\t0\t6\n",
      "4\t2\t136.5\t0\t2\n",
      "5\t2\t34.1\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/438_150_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/438_370_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/438_150_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/438_370_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/438_150_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/438_370_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/438_150_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/438_370_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        28,340 reads  @     28.0 µs/read;   2.14 M reads/minute\n",
      "Finished in 0.82 s (29 us/read; 2.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,340\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,340 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,558,680 bp\n",
      "  Read 1:     4,279,340 bp\n",
      "  Read 2:     4,279,340 bp\n",
      "Total written (filtered):      8,558,654 bp (100.0%)\n",
      "  Read 1:     4,279,331 bp\n",
      "  Read 2:     4,279,323 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t442.8\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t442.8\t0\t4\n",
      "5\t1\t27.7\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/439_151_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/439_371_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/439_151_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/439_371_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/439_151_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/439_371_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/439_151_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/439_371_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        23,915 reads  @     29.9 µs/read;   2.01 M reads/minute\n",
      "Finished in 0.75 s (31 us/read; 1.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,915\n",
      "  Read 1 with adapter:                      11 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,915 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,222,330 bp\n",
      "  Read 1:     3,611,165 bp\n",
      "  Read 2:     3,611,165 bp\n",
      "Total written (filtered):      7,222,280 bp (100.0%)\n",
      "  Read 1:     3,611,132 bp\n",
      "  Read 2:     3,611,148 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t373.7\t0\t11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t373.7\t0\t2\n",
      "5\t1\t23.4\t0\t1\n",
      "6\t1\t5.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/440_152_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/440_372_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/440_152_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/440_372_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/440_152_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/440_372_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/440_152_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/440_372_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        28,113 reads  @     28.8 µs/read;   2.09 M reads/minute\n",
      "Finished in 0.83 s (30 us/read; 2.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,113\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,113 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,490,126 bp\n",
      "  Read 1:     4,245,063 bp\n",
      "  Read 2:     4,245,063 bp\n",
      "Total written (filtered):      8,490,114 bp (100.0%)\n",
      "  Read 1:     4,245,057 bp\n",
      "  Read 2:     4,245,057 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t439.3\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t439.3\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/441_153_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/441_373_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/441_153_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/441_373_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/441_153_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/441_373_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/441_153_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/441_373_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        34,141 reads  @     29.9 µs/read;   2.00 M reads/minute\n",
      "Finished in 1.05 s (31 us/read; 1.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,141\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,141 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,310,582 bp\n",
      "  Read 1:     5,155,291 bp\n",
      "  Read 2:     5,155,291 bp\n",
      "Total written (filtered):     10,310,554 bp (100.0%)\n",
      "  Read 1:     5,155,288 bp\n",
      "  Read 2:     5,155,266 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t533.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t533.5\t0\t7\n",
      "4\t1\t133.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/442_154_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/442_374_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/442_154_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/442_374_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/442_154_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/442_374_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/442_154_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/442_374_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        31,105 reads  @     29.9 µs/read;   2.01 M reads/minute\n",
      "Finished in 0.96 s (31 us/read; 1.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,105\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        31,105 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,393,710 bp\n",
      "  Read 1:     4,696,855 bp\n",
      "  Read 2:     4,696,855 bp\n",
      "Total written (filtered):      9,393,673 bp (100.0%)\n",
      "  Read 1:     4,696,849 bp\n",
      "  Read 2:     4,696,824 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t486.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t486.0\t0\t5\n",
      "4\t1\t121.5\t0\t1\n",
      "6\t2\t7.6\t0\t2\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/443_155_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/443_375_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/443_155_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/443_375_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/443_155_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/443_375_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/443_155_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/443_375_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        34,505 reads  @     32.1 µs/read;   1.87 M reads/minute\n",
      "Finished in 1.14 s (33 us/read; 1.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,505\n",
      "  Read 1 with adapter:                      12 (0.0%)\n",
      "  Read 2 with adapter:                      13 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,505 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,420,510 bp\n",
      "  Read 1:     5,210,255 bp\n",
      "  Read 2:     5,210,255 bp\n",
      "Total written (filtered):     10,420,433 bp (100.0%)\n",
      "  Read 1:     5,210,219 bp\n",
      "  Read 2:     5,210,214 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t539.1\t0\t12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t539.1\t0\t11\n",
      "4\t2\t134.8\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/444_156_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/444_376_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/444_156_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/444_376_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/444_156_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/444_376_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/444_156_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/444_376_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,390 reads  @     32.7 µs/read;   1.84 M reads/minute\n",
      "Finished in 1.08 s (33 us/read; 1.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,390\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,390 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,781,780 bp\n",
      "  Read 1:     4,890,890 bp\n",
      "  Read 2:     4,890,890 bp\n",
      "Total written (filtered):      9,781,739 bp (100.0%)\n",
      "  Read 1:     4,890,881 bp\n",
      "  Read 2:     4,890,858 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t126.5\t0\t1\n",
      "5\t1\t31.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t506.1\t0\t9\n",
      "5\t1\t31.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/445_157_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/445_377_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/445_157_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/445_377_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/445_157_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/445_377_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/445_157_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/445_377_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,489 reads  @     33.3 µs/read;   1.80 M reads/minute\n",
      "Finished in 1.11 s (34 us/read; 1.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,489\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       7 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,489 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,811,678 bp\n",
      "  Read 1:     4,905,839 bp\n",
      "  Read 2:     4,905,839 bp\n",
      "Total written (filtered):      9,811,656 bp (100.0%)\n",
      "  Read 1:     4,905,839 bp\n",
      "  Read 2:     4,905,817 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t507.6\t0\t6\n",
      "4\t1\t126.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/446_158_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/446_378_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/446_158_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/446_378_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/446_158_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/446_378_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/446_158_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/446_378_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        31,723 reads  @     32.7 µs/read;   1.83 M reads/minute\n",
      "Finished in 1.07 s (34 us/read; 1.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,723\n",
      "  Read 1 with adapter:                      24 (0.1%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        31,723 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,580,346 bp\n",
      "  Read 1:     4,790,173 bp\n",
      "  Read 2:     4,790,173 bp\n",
      "Total written (filtered):      9,580,257 bp (100.0%)\n",
      "  Read 1:     4,790,100 bp\n",
      "  Read 2:     4,790,157 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 24 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t495.7\t0\t23\n",
      "4\t1\t123.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t495.7\t0\t4\n",
      "4\t1\t123.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/447_159_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/447_379_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/447_159_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/447_379_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/447_159_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/447_379_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/447_159_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/447_379_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        23,209 reads  @     33.9 µs/read;   1.77 M reads/minute\n",
      "Finished in 0.81 s (35 us/read; 1.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,209\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,209 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,009,118 bp\n",
      "  Read 1:     3,504,559 bp\n",
      "  Read 2:     3,504,559 bp\n",
      "Total written (filtered):      7,009,103 bp (100.0%)\n",
      "  Read 1:     3,504,556 bp\n",
      "  Read 2:     3,504,547 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t362.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t362.6\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/448_160_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/448_380_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/448_160_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/448_380_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/448_160_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/448_380_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/448_160_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/448_380_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        28,311 reads  @     31.1 µs/read;   1.93 M reads/minute\n",
      "Finished in 0.90 s (32 us/read; 1.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,311\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,311 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,549,922 bp\n",
      "  Read 1:     4,274,961 bp\n",
      "  Read 2:     4,274,961 bp\n",
      "Total written (filtered):      8,549,898 bp (100.0%)\n",
      "  Read 1:     4,274,955 bp\n",
      "  Read 2:     4,274,943 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t442.4\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t442.4\t0\t6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/449_161_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/449_381_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/449_161_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/449_381_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/449_161_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/449_381_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/449_161_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/449_381_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        32,237 reads  @     30.5 µs/read;   1.97 M reads/minute\n",
      "Finished in 1.01 s (31 us/read; 1.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,237\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,237 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,735,574 bp\n",
      "  Read 1:     4,867,787 bp\n",
      "  Read 2:     4,867,787 bp\n",
      "Total written (filtered):      9,735,559 bp (100.0%)\n",
      "  Read 1:     4,867,784 bp\n",
      "  Read 2:     4,867,775 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t503.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t503.7\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/450_162_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/450_382_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/450_162_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/450_382_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/450_162_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/450_382_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/450_162_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/450_382_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        33,509 reads  @     30.7 µs/read;   1.96 M reads/minute\n",
      "Finished in 1.06 s (32 us/read; 1.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,509\n",
      "  Read 1 with adapter:                      12 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,509 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,119,718 bp\n",
      "  Read 1:     5,059,859 bp\n",
      "  Read 2:     5,059,859 bp\n",
      "Total written (filtered):     10,119,658 bp (100.0%)\n",
      "  Read 1:     5,059,822 bp\n",
      "  Read 2:     5,059,836 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t523.6\t0\t11\n",
      "4\t1\t130.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t523.6\t0\t3\n",
      "4\t1\t130.9\t0\t1\n",
      "5\t2\t32.7\t0\t2\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/451_163_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/451_383_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/451_163_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/451_383_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/451_163_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/451_383_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/451_163_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/451_383_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,245 reads  @     33.6 µs/read;   1.79 M reads/minute\n",
      "Finished in 1.11 s (35 us/read; 1.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,245\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,245 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,737,990 bp\n",
      "  Read 1:     4,868,995 bp\n",
      "  Read 2:     4,868,995 bp\n",
      "Total written (filtered):      9,737,954 bp (100.0%)\n",
      "  Read 1:     4,868,989 bp\n",
      "  Read 2:     4,868,965 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t503.8\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t503.8\t0\t10\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/452_164_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/452_384_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/452_164_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/452_384_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/452_164_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/452_384_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/452_164_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/452_384_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        29,954 reads  @     31.5 µs/read;   1.90 M reads/minute\n",
      "Finished in 0.97 s (33 us/read; 1.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,954\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,954 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,046,108 bp\n",
      "  Read 1:     4,523,054 bp\n",
      "  Read 2:     4,523,054 bp\n",
      "Total written (filtered):      9,046,075 bp (100.0%)\n",
      "  Read 1:     4,523,036 bp\n",
      "  Read 2:     4,523,039 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t468.0\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t468.0\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/453_165_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/453_385_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/453_165_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/453_385_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/453_165_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/453_385_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/453_165_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/453_385_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        37,391 reads  @     29.3 µs/read;   2.05 M reads/minute\n",
      "Finished in 1.13 s (30 us/read; 1.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,391\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                      18 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        37,391 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,292,082 bp\n",
      "  Read 1:     5,646,041 bp\n",
      "  Read 2:     5,646,041 bp\n",
      "Total written (filtered):     11,292,001 bp (100.0%)\n",
      "  Read 1:     5,646,017 bp\n",
      "  Read 2:     5,645,984 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t584.2\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 18 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t584.2\t0\t15\n",
      "4\t3\t146.1\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/454_166_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/454_386_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/454_166_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/454_386_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/454_166_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/454_386_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/454_166_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/454_386_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        27,839 reads  @     29.8 µs/read;   2.01 M reads/minute\n",
      "Finished in 0.86 s (31 us/read; 1.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,839\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,839 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,407,378 bp\n",
      "  Read 1:     4,203,689 bp\n",
      "  Read 2:     4,203,689 bp\n",
      "Total written (filtered):      8,407,320 bp (100.0%)\n",
      "  Read 1:     4,203,686 bp\n",
      "  Read 2:     4,203,634 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t435.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t435.0\t0\t8\n",
      "31\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/455_167_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/455_387_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/455_167_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/455_387_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/455_167_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/455_387_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/455_167_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/455_387_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        22,233 reads  @     39.9 µs/read;   1.50 M reads/minute\n",
      "Finished in 0.92 s (41 us/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,233\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,233 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,714,366 bp\n",
      "  Read 1:     3,357,183 bp\n",
      "  Read 2:     3,357,183 bp\n",
      "Total written (filtered):      6,714,342 bp (100.0%)\n",
      "  Read 1:     3,357,165 bp\n",
      "  Read 2:     3,357,177 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t347.4\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t347.4\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/456_168_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/456_388_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/456_168_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/456_388_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/456_168_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/456_388_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/456_168_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/456_388_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        30,569 reads  @     31.6 µs/read;   1.90 M reads/minute\n",
      "Finished in 1.00 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,569\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,569 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,231,838 bp\n",
      "  Read 1:     4,615,919 bp\n",
      "  Read 2:     4,615,919 bp\n",
      "Total written (filtered):      9,231,811 bp (100.0%)\n",
      "  Read 1:     4,615,907 bp\n",
      "  Read 2:     4,615,904 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t477.6\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t477.6\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/457_169_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/457_389_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/457_169_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/457_389_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/457_169_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/457_389_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/457_169_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/457_389_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,656 reads  @     34.0 µs/read;   1.76 M reads/minute\n",
      "Finished in 1.14 s (35 us/read; 1.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,656\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,656 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,862,112 bp\n",
      "  Read 1:     4,931,056 bp\n",
      "  Read 2:     4,931,056 bp\n",
      "Total written (filtered):      9,862,070 bp (100.0%)\n",
      "  Read 1:     4,931,041 bp\n",
      "  Read 2:     4,931,029 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t510.2\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t510.2\t0\t6\n",
      "4\t1\t127.6\t0\t1\n",
      "5\t1\t31.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/458_170_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/458_390_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/458_170_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/458_390_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/458_170_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/458_390_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/458_170_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/458_390_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        27,626 reads  @     31.4 µs/read;   1.91 M reads/minute\n",
      "Finished in 0.90 s (32 us/read; 1.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,626\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,626 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,343,052 bp\n",
      "  Read 1:     4,171,526 bp\n",
      "  Read 2:     4,171,526 bp\n",
      "Total written (filtered):      8,343,013 bp (100.0%)\n",
      "  Read 1:     4,171,514 bp\n",
      "  Read 2:     4,171,499 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t431.7\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t431.7\t0\t9\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/459_171_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/459_391_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/459_171_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/459_391_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/459_171_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/459_391_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/459_171_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/459_391_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        13,749 reads  @     52.1 µs/read;   1.15 M reads/minute\n",
      "Finished in 0.75 s (55 us/read; 1.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,749\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        13,749 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,152,198 bp\n",
      "  Read 1:     2,076,099 bp\n",
      "  Read 2:     2,076,099 bp\n",
      "Total written (filtered):      4,152,189 bp (100.0%)\n",
      "  Read 1:     2,076,093 bp\n",
      "  Read 2:     2,076,096 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t214.8\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t214.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/460_172_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/460_392_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/460_172_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/460_392_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/460_172_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/460_392_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/460_172_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/460_392_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        25,722 reads  @     31.2 µs/read;   1.92 M reads/minute\n",
      "Finished in 0.82 s (32 us/read; 1.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,722\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,722 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,768,044 bp\n",
      "  Read 1:     3,884,022 bp\n",
      "  Read 2:     3,884,022 bp\n",
      "Total written (filtered):      7,768,004 bp (100.0%)\n",
      "  Read 1:     3,884,001 bp\n",
      "  Read 2:     3,884,003 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t401.9\t0\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t401.9\t0\t5\n",
      "4\t1\t100.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/461_173_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/461_393_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/461_173_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/461_393_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/461_173_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/461_393_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/461_173_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/461_393_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        28,152 reads  @     32.1 µs/read;   1.87 M reads/minute\n",
      "Finished in 0.93 s (33 us/read; 1.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,152\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,152 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,501,904 bp\n",
      "  Read 1:     4,250,952 bp\n",
      "  Read 2:     4,250,952 bp\n",
      "Total written (filtered):      8,501,873 bp (100.0%)\n",
      "  Read 1:     4,250,949 bp\n",
      "  Read 2:     4,250,924 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t439.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t439.9\t0\t5\n",
      "4\t2\t110.0\t0\t2\n",
      "5\t1\t27.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/462_174_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/462_394_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/462_174_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/462_394_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/462_174_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/462_394_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/462_174_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/462_394_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        25,117 reads  @     33.0 µs/read;   1.82 M reads/minute\n",
      "Finished in 0.85 s (34 us/read; 1.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,117\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,117 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,585,334 bp\n",
      "  Read 1:     3,792,667 bp\n",
      "  Read 2:     3,792,667 bp\n",
      "Total written (filtered):      7,585,291 bp (100.0%)\n",
      "  Read 1:     3,792,648 bp\n",
      "  Read 2:     3,792,643 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t392.5\t0\t5\n",
      "4\t1\t98.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t392.5\t0\t8\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/463_175_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/463_395_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/463_175_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/463_395_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/463_175_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/463_395_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/463_175_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/463_395_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,129 reads  @     33.2 µs/read;   1.81 M reads/minute\n",
      "Finished in 1.10 s (34 us/read; 1.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,129\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,129 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,702,958 bp\n",
      "  Read 1:     4,851,479 bp\n",
      "  Read 2:     4,851,479 bp\n",
      "Total written (filtered):      9,702,923 bp (100.0%)\n",
      "  Read 1:     4,851,461 bp\n",
      "  Read 2:     4,851,462 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t502.0\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t502.0\t0\t3\n",
      "4\t2\t125.5\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/464_176_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/464_396_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/464_176_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/464_396_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/464_176_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/464_396_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/464_176_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/464_396_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        29,013 reads  @     32.6 µs/read;   1.84 M reads/minute\n",
      "Finished in 0.98 s (34 us/read; 1.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,013\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                       7 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,013 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,761,926 bp\n",
      "  Read 1:     4,380,963 bp\n",
      "  Read 2:     4,380,963 bp\n",
      "Total written (filtered):      8,761,895 bp (100.0%)\n",
      "  Read 1:     4,380,954 bp\n",
      "  Read 2:     4,380,941 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t453.3\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t453.3\t0\t6\n",
      "4\t1\t113.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/465_177_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/465_397_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/465_177_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/465_397_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/465_177_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/465_397_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/465_177_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/465_397_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        28,707 reads  @     29.2 µs/read;   2.05 M reads/minute\n",
      "Finished in 0.87 s (30 us/read; 1.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,707\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,707 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,669,514 bp\n",
      "  Read 1:     4,334,757 bp\n",
      "  Read 2:     4,334,757 bp\n",
      "Total written (filtered):      8,669,486 bp (100.0%)\n",
      "  Read 1:     4,334,754 bp\n",
      "  Read 2:     4,334,732 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t448.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t448.5\t0\t7\n",
      "4\t1\t112.1\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/466_178_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/466_398_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/466_178_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/466_398_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/466_178_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/466_398_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/466_178_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/466_398_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        14,243 reads  @     52.7 µs/read;   1.14 M reads/minute\n",
      "Finished in 0.78 s (55 us/read; 1.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,243\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        14,243 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,301,386 bp\n",
      "  Read 1:     2,150,693 bp\n",
      "  Read 2:     2,150,693 bp\n",
      "Total written (filtered):      4,301,358 bp (100.0%)\n",
      "  Read 1:     2,150,677 bp\n",
      "  Read 2:     2,150,681 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t222.5\t0\t4\n",
      "4\t1\t55.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t222.5\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/467_179_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/467_399_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/467_179_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/467_399_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/467_179_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/467_399_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/467_179_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/467_399_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        23,234 reads  @     33.4 µs/read;   1.79 M reads/minute\n",
      "Finished in 0.80 s (34 us/read; 1.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,234\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,234 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,016,668 bp\n",
      "  Read 1:     3,508,334 bp\n",
      "  Read 2:     3,508,334 bp\n",
      "Total written (filtered):      7,016,640 bp (100.0%)\n",
      "  Read 1:     3,508,318 bp\n",
      "  Read 2:     3,508,322 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t363.0\t0\t4\n",
      "4\t1\t90.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t363.0\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/468_180_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/468_400_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/468_180_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/468_400_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/468_180_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/468_400_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/468_180_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/468_400_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        29,323 reads  @     32.8 µs/read;   1.83 M reads/minute\n",
      "Finished in 0.99 s (34 us/read; 1.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,323\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,323 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,855,546 bp\n",
      "  Read 1:     4,427,773 bp\n",
      "  Read 2:     4,427,773 bp\n",
      "Total written (filtered):      8,855,500 bp (100.0%)\n",
      "  Read 1:     4,427,757 bp\n",
      "  Read 2:     4,427,743 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t458.2\t0\t4\n",
      "4\t1\t114.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t458.2\t0\t8\n",
      "6\t1\t7.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/469_181_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/469_401_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/469_181_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/469_401_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/469_181_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/469_401_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/469_181_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/469_401_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        29,526 reads  @     31.7 µs/read;   1.90 M reads/minute\n",
      "Finished in 0.96 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,526\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,526 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,916,852 bp\n",
      "  Read 1:     4,458,426 bp\n",
      "  Read 2:     4,458,426 bp\n",
      "Total written (filtered):      8,916,821 bp (100.0%)\n",
      "  Read 1:     4,458,423 bp\n",
      "  Read 2:     4,458,398 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t461.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t461.3\t0\t8\n",
      "4\t1\t115.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/470_182_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/470_402_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/470_182_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/470_402_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/470_182_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/470_402_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/470_182_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/470_402_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        27,436 reads  @     31.3 µs/read;   1.92 M reads/minute\n",
      "Finished in 0.89 s (32 us/read; 1.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,436\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,436 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,285,672 bp\n",
      "  Read 1:     4,142,836 bp\n",
      "  Read 2:     4,142,836 bp\n",
      "Total written (filtered):      8,285,660 bp (100.0%)\n",
      "  Read 1:     4,142,830 bp\n",
      "  Read 2:     4,142,830 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t428.7\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t428.7\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/471_183_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/471_403_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/471_183_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/471_403_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/471_183_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/471_403_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/471_183_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/471_403_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        28,764 reads  @     31.3 µs/read;   1.91 M reads/minute\n",
      "Finished in 0.93 s (32 us/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,764\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       7 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,764 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,686,728 bp\n",
      "  Read 1:     4,343,364 bp\n",
      "  Read 2:     4,343,364 bp\n",
      "Total written (filtered):      8,686,702 bp (100.0%)\n",
      "  Read 1:     4,343,364 bp\n",
      "  Read 2:     4,343,338 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t449.4\t0\t5\n",
      "4\t1\t112.4\t0\t1\n",
      "7\t1\t1.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/472_184_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/472_404_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/472_184_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/472_404_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/472_184_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/472_404_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/472_184_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/472_404_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        27,606 reads  @     32.9 µs/read;   1.82 M reads/minute\n",
      "Finished in 0.94 s (34 us/read; 1.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,606\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,606 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,337,012 bp\n",
      "  Read 1:     4,168,506 bp\n",
      "  Read 2:     4,168,506 bp\n",
      "Total written (filtered):      8,336,991 bp (100.0%)\n",
      "  Read 1:     4,168,491 bp\n",
      "  Read 2:     4,168,500 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t431.3\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t431.3\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/473_185_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/473_405_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/473_185_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/473_405_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/473_185_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/473_405_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/473_185_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/473_405_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        24,284 reads  @     32.6 µs/read;   1.84 M reads/minute\n",
      "Finished in 0.82 s (34 us/read; 1.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,284\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        24,284 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,333,768 bp\n",
      "  Read 1:     3,666,884 bp\n",
      "  Read 2:     3,666,884 bp\n",
      "Total written (filtered):      7,333,724 bp (100.0%)\n",
      "  Read 1:     3,666,863 bp\n",
      "  Read 2:     3,666,861 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t379.4\t0\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t379.4\t0\t5\n",
      "8\t1\t0.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/474_186_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/474_406_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/474_186_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/474_406_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/474_186_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/474_406_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/474_186_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/474_406_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        25,176 reads  @     35.1 µs/read;   1.71 M reads/minute\n",
      "Finished in 0.91 s (36 us/read; 1.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,176\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,176 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,603,152 bp\n",
      "  Read 1:     3,801,576 bp\n",
      "  Read 2:     3,801,576 bp\n",
      "Total written (filtered):      7,603,125 bp (100.0%)\n",
      "  Read 1:     3,801,558 bp\n",
      "  Read 2:     3,801,567 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t393.4\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t393.4\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/475_187_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/475_407_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/475_187_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/475_407_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/475_187_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/475_407_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/475_187_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/475_407_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        30,434 reads  @     32.1 µs/read;   1.87 M reads/minute\n",
      "Finished in 1.01 s (33 us/read; 1.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,434\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,191,068 bp\n",
      "  Read 1:     4,595,534 bp\n",
      "  Read 2:     4,595,534 bp\n",
      "Total written (filtered):      9,191,034 bp (100.0%)\n",
      "  Read 1:     4,595,524 bp\n",
      "  Read 2:     4,595,510 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t475.5\t0\t2\n",
      "4\t1\t118.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t475.5\t0\t8\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/476_188_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/476_408_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/476_188_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/476_408_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/476_188_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/476_408_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/476_188_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/476_408_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        10,493 reads  @     65.1 µs/read;   0.92 M reads/minute\n",
      "Finished in 0.71 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,493\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        10,493 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,168,886 bp\n",
      "  Read 1:     1,584,443 bp\n",
      "  Read 2:     1,584,443 bp\n",
      "Total written (filtered):      3,168,877 bp (100.0%)\n",
      "  Read 1:     1,584,443 bp\n",
      "  Read 2:     1,584,434 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t164.0\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/477_189_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/477_409_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/477_189_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/477_409_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/477_189_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/477_409_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/477_189_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/477_409_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        41,636 reads  @     26.0 µs/read;   2.31 M reads/minute\n",
      "Finished in 1.11 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,636\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                      13 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,636 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,574,072 bp\n",
      "  Read 1:     6,287,036 bp\n",
      "  Read 2:     6,287,036 bp\n",
      "Total written (filtered):     12,574,027 bp (100.0%)\n",
      "  Read 1:     6,287,030 bp\n",
      "  Read 2:     6,286,997 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 2 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t650.6\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t650.6\t0\t13\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/478_190_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/478_410_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/478_190_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/478_410_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/478_190_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/478_410_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/478_190_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/478_410_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        25,760 reads  @     29.2 µs/read;   2.06 M reads/minute\n",
      "Finished in 0.78 s (30 us/read; 1.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,760\n",
      "  Read 1 with adapter:                       4 (0.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,760 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,779,520 bp\n",
      "  Read 1:     3,889,760 bp\n",
      "  Read 2:     3,889,760 bp\n",
      "Total written (filtered):      7,779,499 bp (100.0%)\n",
      "  Read 1:     3,889,748 bp\n",
      "  Read 2:     3,889,751 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t402.5\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t402.5\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/479_191_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/479_411_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/479_191_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/479_411_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/479_191_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/479_411_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/479_191_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/479_411_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        21,957 reads  @     37.2 µs/read;   1.61 M reads/minute\n",
      "Finished in 0.84 s (38 us/read; 1.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,957\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        21,957 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,631,014 bp\n",
      "  Read 1:     3,315,507 bp\n",
      "  Read 2:     3,315,507 bp\n",
      "Total written (filtered):      6,630,983 bp (100.0%)\n",
      "  Read 1:     3,315,489 bp\n",
      "  Read 2:     3,315,494 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t343.1\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t343.1\t0\t3\n",
      "4\t1\t85.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/480_192_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/480_412_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/480_192_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/480_412_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/480_192_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/480_412_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/480_192_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/480_412_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        32,822 reads  @     30.5 µs/read;   1.97 M reads/minute\n",
      "Finished in 1.04 s (32 us/read; 1.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,822\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,822 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,912,244 bp\n",
      "  Read 1:     4,956,122 bp\n",
      "  Read 2:     4,956,122 bp\n",
      "Total written (filtered):      9,912,172 bp (100.0%)\n",
      "  Read 1:     4,956,113 bp\n",
      "  Read 2:     4,956,059 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t512.8\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t512.8\t0\t8\n",
      "4\t2\t128.2\t0\t2\n",
      "31\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/481_193_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/481_413_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/481_193_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/481_413_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/481_193_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/481_413_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/481_193_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/481_413_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        25,573 reads  @     33.7 µs/read;   1.78 M reads/minute\n",
      "Finished in 0.89 s (35 us/read; 1.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,573\n",
      "  Read 1 with adapter:                       3 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,573 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,723,046 bp\n",
      "  Read 1:     3,861,523 bp\n",
      "  Read 2:     3,861,523 bp\n",
      "Total written (filtered):      7,723,005 bp (100.0%)\n",
      "  Read 1:     3,861,514 bp\n",
      "  Read 2:     3,861,491 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t399.6\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t399.6\t0\t8\n",
      "4\t2\t99.9\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/482_194_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/482_414_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/482_194_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/482_414_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/482_194_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/482_414_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/482_194_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/482_414_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        39,089 reads  @     31.3 µs/read;   1.92 M reads/minute\n",
      "Finished in 1.26 s (32 us/read; 1.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,089\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        39,089 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,804,878 bp\n",
      "  Read 1:     5,902,439 bp\n",
      "  Read 2:     5,902,439 bp\n",
      "Total written (filtered):     11,804,816 bp (100.0%)\n",
      "  Read 1:     5,902,417 bp\n",
      "  Read 2:     5,902,399 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t610.8\t0\t6\n",
      "4\t1\t152.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t610.8\t0\t10\n",
      "4\t1\t152.7\t0\t1\n",
      "6\t1\t9.5\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/483_195_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/483_415_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/483_195_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/483_415_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/483_195_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/483_415_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/483_195_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/483_415_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        34,326 reads  @     31.7 µs/read;   1.90 M reads/minute\n",
      "Finished in 1.12 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,326\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,366,452 bp\n",
      "  Read 1:     5,183,226 bp\n",
      "  Read 2:     5,183,226 bp\n",
      "Total written (filtered):     10,366,399 bp (100.0%)\n",
      "  Read 1:     5,183,199 bp\n",
      "  Read 2:     5,183,200 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t536.3\t0\t9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t536.3\t0\t6\n",
      "4\t2\t134.1\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/484_196_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/484_416_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/484_196_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/484_416_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/484_196_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/484_416_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/484_196_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/484_416_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        33,999 reads  @     31.3 µs/read;   1.92 M reads/minute\n",
      "Finished in 1.09 s (32 us/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,999\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,999 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,267,698 bp\n",
      "  Read 1:     5,133,849 bp\n",
      "  Read 2:     5,133,849 bp\n",
      "Total written (filtered):     10,267,654 bp (100.0%)\n",
      "  Read 1:     5,133,825 bp\n",
      "  Read 2:     5,133,829 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t531.2\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t531.2\t0\t5\n",
      "5\t1\t33.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/485_197_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/485_417_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/485_197_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/485_417_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/485_197_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/485_417_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/485_197_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/485_417_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        31,099 reads  @     31.7 µs/read;   1.89 M reads/minute\n",
      "Finished in 1.02 s (33 us/read; 1.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,099\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                      10 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        31,099 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,391,898 bp\n",
      "  Read 1:     4,695,949 bp\n",
      "  Read 2:     4,695,949 bp\n",
      "Total written (filtered):      9,391,845 bp (100.0%)\n",
      "  Read 1:     4,695,931 bp\n",
      "  Read 2:     4,695,914 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t485.9\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t485.9\t0\t7\n",
      "4\t2\t121.5\t0\t2\n",
      "6\t1\t7.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/486_198_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/486_418_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/486_198_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/486_418_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/486_198_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/486_418_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/486_198_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/486_418_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        27,884 reads  @     36.4 µs/read;   1.65 M reads/minute\n",
      "Finished in 1.05 s (38 us/read; 1.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,884\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,884 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,420,968 bp\n",
      "  Read 1:     4,210,484 bp\n",
      "  Read 2:     4,210,484 bp\n",
      "Total written (filtered):      8,420,918 bp (100.0%)\n",
      "  Read 1:     4,210,460 bp\n",
      "  Read 2:     4,210,458 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t435.7\t0\t5\n",
      "4\t1\t108.9\t0\t1\n",
      "5\t1\t27.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t435.7\t0\t6\n",
      "4\t2\t108.9\t0\t2\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/487_199_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/487_419_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/487_199_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/487_419_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/487_199_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/487_419_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/487_199_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/487_419_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        28,768 reads  @     32.5 µs/read;   1.85 M reads/minute\n",
      "Finished in 0.96 s (34 us/read; 1.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,768\n",
      "  Read 1 with adapter:                       7 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,768 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,687,936 bp\n",
      "  Read 1:     4,343,968 bp\n",
      "  Read 2:     4,343,968 bp\n",
      "Total written (filtered):      8,687,899 bp (100.0%)\n",
      "  Read 1:     4,343,943 bp\n",
      "  Read 2:     4,343,956 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 7 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t449.5\t0\t6\n",
      "7\t1\t1.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t449.5\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/488_200_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/488_420_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/488_200_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/488_420_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/488_200_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/488_420_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/488_200_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/488_420_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        21,729 reads  @     34.5 µs/read;   1.74 M reads/minute\n",
      "Finished in 0.78 s (36 us/read; 1.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,729\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        21,729 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,562,158 bp\n",
      "  Read 1:     3,281,079 bp\n",
      "  Read 2:     3,281,079 bp\n",
      "Total written (filtered):      6,562,140 bp (100.0%)\n",
      "  Read 1:     3,281,076 bp\n",
      "  Read 2:     3,281,064 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t339.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t339.5\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/489_201_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/489_421_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/489_201_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/489_421_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/489_201_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/489_421_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/489_201_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/489_421_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        47,080 reads  @     23.1 µs/read;   2.59 M reads/minute\n",
      "Finished in 1.12 s (24 us/read; 2.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,080\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                      22 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        47,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,218,160 bp\n",
      "  Read 1:     7,109,080 bp\n",
      "  Read 2:     7,109,080 bp\n",
      "Total written (filtered):     14,218,073 bp (100.0%)\n",
      "  Read 1:     7,109,062 bp\n",
      "  Read 2:     7,109,011 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t735.6\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 22 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t735.6\t0\t19\n",
      "4\t3\t183.9\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/490_202_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/490_422_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/490_202_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/490_422_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/490_202_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/490_422_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/490_202_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/490_422_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        18,292 reads  @     40.7 µs/read;   1.47 M reads/minute\n",
      "Finished in 0.78 s (43 us/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,292\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        18,292 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,524,184 bp\n",
      "  Read 1:     2,762,092 bp\n",
      "  Read 2:     2,762,092 bp\n",
      "Total written (filtered):      5,524,181 bp (100.0%)\n",
      "  Read 1:     2,762,092 bp\n",
      "  Read 2:     2,762,089 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t285.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/491_203_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/491_423_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/491_203_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/491_423_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/491_203_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/491_423_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/491_203_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/491_423_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        48,963 reads  @     24.7 µs/read;   2.43 M reads/minute\n",
      "Finished in 1.24 s (25 us/read; 2.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,963\n",
      "  Read 1 with adapter:                      16 (0.0%)\n",
      "  Read 2 with adapter:                      17 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,963 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,786,826 bp\n",
      "  Read 1:     7,393,413 bp\n",
      "  Read 2:     7,393,413 bp\n",
      "Total written (filtered):     14,786,705 bp (100.0%)\n",
      "  Read 1:     7,393,365 bp\n",
      "  Read 2:     7,393,340 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t765.0\t0\t16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 17 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t765.0\t0\t9\n",
      "4\t4\t191.3\t0\t4\n",
      "5\t2\t47.8\t0\t2\n",
      "10\t2\t0.0\t1\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/492_204_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/492_424_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/492_204_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/492_424_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/492_204_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/492_424_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/492_204_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/492_424_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        54,890 reads  @     28.1 µs/read;   2.14 M reads/minute\n",
      "Finished in 1.58 s (29 us/read; 2.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,890\n",
      "  Read 1 with adapter:                      20 (0.0%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,576,780 bp\n",
      "  Read 1:     8,288,390 bp\n",
      "  Read 2:     8,288,390 bp\n",
      "Total written (filtered):     16,576,463 bp (100.0%)\n",
      "  Read 1:     8,288,098 bp\n",
      "  Read 2:     8,288,365 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t857.7\t0\t16\n",
      "4\t2\t214.4\t0\t2\n",
      "118\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t857.7\t0\t7\n",
      "4\t1\t214.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/493_205_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/493_425_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/493_205_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/493_425_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/493_205_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/493_425_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/493_205_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/493_425_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        55,222 reads  @     27.0 µs/read;   2.22 M reads/minute\n",
      "Finished in 1.52 s (28 us/read; 2.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,222\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                      11 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,222 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,677,044 bp\n",
      "  Read 1:     8,338,522 bp\n",
      "  Read 2:     8,338,522 bp\n",
      "Total written (filtered):     16,676,981 bp (100.0%)\n",
      "  Read 1:     8,338,495 bp\n",
      "  Read 2:     8,338,486 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t862.8\t0\t9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t862.8\t0\t9\n",
      "4\t1\t215.7\t0\t1\n",
      "5\t1\t53.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/494_206_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/494_426_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/494_206_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/494_426_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/494_206_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/494_426_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/494_206_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/494_426_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        55,418 reads  @     26.2 µs/read;   2.29 M reads/minute\n",
      "Finished in 1.48 s (27 us/read; 2.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,418\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                      22 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,418 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,736,236 bp\n",
      "  Read 1:     8,368,118 bp\n",
      "  Read 2:     8,368,118 bp\n",
      "Total written (filtered):     16,736,137 bp (100.0%)\n",
      "  Read 1:     8,368,091 bp\n",
      "  Read 2:     8,368,046 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 8 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t865.9\t0\t6\n",
      "4\t1\t216.5\t0\t1\n",
      "5\t1\t54.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 22 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t865.9\t0\t19\n",
      "4\t1\t216.5\t0\t1\n",
      "5\t1\t54.1\t0\t1\n",
      "6\t1\t13.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/495_207_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/495_427_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/495_207_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/495_427_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/495_207_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/495_427_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/495_207_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/495_427_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        42,794 reads  @     27.7 µs/read;   2.17 M reads/minute\n",
      "Finished in 1.22 s (28 us/read; 2.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,794\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,923,788 bp\n",
      "  Read 1:     6,461,894 bp\n",
      "  Read 2:     6,461,894 bp\n",
      "Total written (filtered):     12,923,758 bp (100.0%)\n",
      "  Read 1:     6,461,879 bp\n",
      "  Read 2:     6,461,879 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t668.7\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t668.7\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/496_208_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/496_428_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/496_208_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/496_428_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/496_208_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/496_428_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/496_208_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/496_428_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        59,408 reads  @     26.3 µs/read;   2.28 M reads/minute\n",
      "Finished in 1.60 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,408\n",
      "  Read 1 with adapter:                      11 (0.0%)\n",
      "  Read 2 with adapter:                      14 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        59,408 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,941,216 bp\n",
      "  Read 1:     8,970,608 bp\n",
      "  Read 2:     8,970,608 bp\n",
      "Total written (filtered):     17,941,140 bp (100.0%)\n",
      "  Read 1:     8,970,575 bp\n",
      "  Read 2:     8,970,565 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t928.2\t0\t11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t928.2\t0\t13\n",
      "4\t1\t232.1\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/497_209_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/497_429_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/497_209_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/497_429_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/497_209_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/497_429_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/497_209_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/497_429_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        57,824 reads  @     24.2 µs/read;   2.48 M reads/minute\n",
      "Finished in 1.44 s (25 us/read; 2.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,824\n",
      "  Read 1 with adapter:                      13 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        57,824 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,462,848 bp\n",
      "  Read 1:     8,731,424 bp\n",
      "  Read 2:     8,731,424 bp\n",
      "Total written (filtered):     17,462,767 bp (100.0%)\n",
      "  Read 1:     8,731,383 bp\n",
      "  Read 2:     8,731,384 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t903.5\t0\t11\n",
      "4\t2\t225.9\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t903.5\t0\t9\n",
      "4\t2\t225.9\t0\t2\n",
      "5\t1\t56.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/498_210_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/498_430_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/498_210_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/498_430_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/498_210_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/498_430_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/498_210_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/498_430_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,610 reads  @     26.3 µs/read;   2.28 M reads/minute\n",
      "Finished in 1.33 s (27 us/read; 2.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,610\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,610 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,982,220 bp\n",
      "  Read 1:     7,491,110 bp\n",
      "  Read 2:     7,491,110 bp\n",
      "Total written (filtered):     14,982,161 bp (100.0%)\n",
      "  Read 1:     7,491,091 bp\n",
      "  Read 2:     7,491,070 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t775.2\t0\t5\n",
      "4\t1\t193.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t775.2\t0\t10\n",
      "4\t1\t193.8\t0\t1\n",
      "6\t1\t12.1\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/499_211_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/499_431_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/499_211_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/499_431_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/499_211_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/499_431_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/499_211_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/499_431_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        56,944 reads  @     24.1 µs/read;   2.49 M reads/minute\n",
      "Finished in 1.41 s (25 us/read; 2.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,944\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                      14 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,944 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,197,088 bp\n",
      "  Read 1:     8,598,544 bp\n",
      "  Read 2:     8,598,544 bp\n",
      "Total written (filtered):     17,197,027 bp (100.0%)\n",
      "  Read 1:     8,598,526 bp\n",
      "  Read 2:     8,598,501 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t889.8\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t889.8\t0\t13\n",
      "4\t1\t222.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/5_214_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/5_434_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/5_214_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/5_434_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/5_214_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/5_434_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/5_214_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/5_434_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        45,536 reads  @     27.5 µs/read;   2.18 M reads/minute\n",
      "Finished in 1.29 s (28 us/read; 2.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,536\n",
      "  Read 1 with adapter:                      15 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,536 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,751,872 bp\n",
      "  Read 1:     6,875,936 bp\n",
      "  Read 2:     6,875,936 bp\n",
      "Total written (filtered):     13,751,769 bp (100.0%)\n",
      "  Read 1:     6,875,891 bp\n",
      "  Read 2:     6,875,878 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t711.5\t0\t15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t711.5\t0\t11\n",
      "25\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/500_213_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/500_433_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/500_213_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/500_433_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/500_213_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/500_433_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/500_213_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/500_433_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,646 reads  @     27.7 µs/read;   2.17 M reads/minute\n",
      "Finished in 1.41 s (28 us/read; 2.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,646\n",
      "  Read 1 with adapter:                      11 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,646 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,993,092 bp\n",
      "  Read 1:     7,496,546 bp\n",
      "  Read 2:     7,496,546 bp\n",
      "Total written (filtered):     14,993,023 bp (100.0%)\n",
      "  Read 1:     7,496,513 bp\n",
      "  Read 2:     7,496,510 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t775.7\t0\t11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t775.7\t0\t12\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/6_215_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/6_435_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/6_215_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/6_435_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/6_215_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/6_435_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/6_215_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/6_435_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        49,917 reads  @     27.5 µs/read;   2.18 M reads/minute\n",
      "Finished in 1.41 s (28 us/read; 2.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,917\n",
      "  Read 1 with adapter:                      21 (0.0%)\n",
      "  Read 2 with adapter:                      12 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,917 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,074,934 bp\n",
      "  Read 1:     7,537,467 bp\n",
      "  Read 2:     7,537,467 bp\n",
      "Total written (filtered):     15,074,833 bp (100.0%)\n",
      "  Read 1:     7,537,403 bp\n",
      "  Read 2:     7,537,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t780.0\t0\t20\n",
      "4\t1\t195.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t780.0\t0\t11\n",
      "4\t1\t195.0\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/7_216_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/7_436_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/7_216_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/7_436_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/7_216_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/7_436_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/7_216_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/7_436_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        47,090 reads  @     26.8 µs/read;   2.24 M reads/minute\n",
      "Finished in 1.29 s (27 us/read; 2.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,090\n",
      "  Read 1 with adapter:                      24 (0.1%)\n",
      "  Read 2 with adapter:                      14 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        47,090 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,221,180 bp\n",
      "  Read 1:     7,110,590 bp\n",
      "  Read 2:     7,110,590 bp\n",
      "Total written (filtered):     14,221,061 bp (100.0%)\n",
      "  Read 1:     7,110,516 bp\n",
      "  Read 2:     7,110,545 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 24 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t735.8\t0\t23\n",
      "5\t1\t46.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t735.8\t0\t12\n",
      "4\t1\t183.9\t0\t1\n",
      "5\t1\t46.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/8_217_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/8_437_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/8_217_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/8_437_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/8_217_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/8_437_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/8_217_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/8_437_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        48,771 reads  @     28.1 µs/read;   2.14 M reads/minute\n",
      "Finished in 1.40 s (29 us/read; 2.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,771\n",
      "  Read 1 with adapter:                      25 (0.1%)\n",
      "  Read 2 with adapter:                      16 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,771 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,728,842 bp\n",
      "  Read 1:     7,364,421 bp\n",
      "  Read 2:     7,364,421 bp\n",
      "Total written (filtered):     14,728,692 bp (100.0%)\n",
      "  Read 1:     7,364,344 bp\n",
      "  Read 2:     7,364,348 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 25 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t762.0\t0\t23\n",
      "4\t2\t190.5\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t762.0\t0\t9\n",
      "4\t4\t190.5\t0\t4\n",
      "6\t1\t11.9\t0\t1\n",
      "8\t1\t0.7\t0\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/9_218_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/9_438_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/9_218_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/9_438_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/9_218_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/9_438_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/9_218_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/9_438_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        52,467 reads  @     26.9 µs/read;   2.23 M reads/minute\n",
      "Finished in 1.44 s (28 us/read; 2.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,467\n",
      "  Read 1 with adapter:                      33 (0.1%)\n",
      "  Read 2 with adapter:                      13 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,467 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,845,034 bp\n",
      "  Read 1:     7,922,517 bp\n",
      "  Read 2:     7,922,517 bp\n",
      "Total written (filtered):     15,844,887 bp (100.0%)\n",
      "  Read 1:     7,922,413 bp\n",
      "  Read 2:     7,922,474 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 33 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t819.8\t0\t30\n",
      "4\t2\t204.9\t0\t2\n",
      "6\t1\t12.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t819.8\t0\t11\n",
      "4\t1\t204.9\t0\t1\n",
      "6\t1\t12.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/MC_219_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/MC_439_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/MC_219_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/MC_439_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.1 with Python 3.6.12\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/MC_219_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-kz27gwdh/MC_439_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/MC_219_L001_R1_001.fastq.gz /tmp/qiime2-archive-m7yz_9_q/a5b124cf-fda6-4e98-8c90-79cabc7ad4cc/data/MC_439_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        46,700 reads  @     26.6 µs/read;   2.25 M reads/minute\n",
      "Finished in 1.27 s (27 us/read; 2.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,700\n",
      "  Read 1 with adapter:                      16 (0.0%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        46,700 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,103,400 bp\n",
      "  Read 1:     7,051,700 bp\n",
      "  Read 2:     7,051,700 bp\n",
      "Total written (filtered):     14,103,338 bp (100.0%)\n",
      "  Read 1:     7,051,652 bp\n",
      "  Read 2:     7,051,686 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t729.7\t0\t16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t729.7\t0\t3\n",
      "5\t1\t45.6\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: Folder_Voss_ENS_LBTS1/trim_Voss_ENS_LBTS1.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#trim PCR primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences Folder_Voss_ENS_LBTS1/Import_Voss_ENS_LBTS1.qza \\\n",
    "--p-cores 2 \\\n",
    "--p-front-f GTGYCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACNVGGGTWTCTAAT \\\n",
    "--o-trimmed-sequences Folder_Voss_ENS_LBTS1/trim_Voss_ENS_LBTS1.qza \\\n",
    "--verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/miniconda3/envs/qiime2-2020.11/lib/python3.6/site-packages/seaborn/distributions.py:2551: FutureWarning: `distplot` is a deprecated function and will be removed in a future version. Please adapt your code to use either `displot` (a figure-level function with similar flexibility) or `histplot` (an axes-level function for histograms).\n",
      "  warnings.warn(msg, FutureWarning)\n",
      "/space/home/rosales/miniconda3/envs/qiime2-2020.11/lib/python3.6/site-packages/seaborn/distributions.py:2551: FutureWarning: `distplot` is a deprecated function and will be removed in a future version. Please adapt your code to use either `displot` (a figure-level function with similar flexibility) or `histplot` (an axes-level function for histograms).\n",
      "  warnings.warn(msg, FutureWarning)\n",
      "\u001b[32mSaved Visualization to: Folder_Voss_ENS_LBTS1/trim_Voss_ENS_LBTS1.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data  Folder_Voss_ENS_LBTS1/trim_Voss_ENS_LBTS1.qza \\\n",
    "--o-visualization  Folder_Voss_ENS_LBTS1/trim_Voss_ENS_LBTS1.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_Voss_ENS_LBTS1/table-Voss_ENS_LBTS1.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_Voss_ENS_LBTS1/rep_seqs_Voss_ENS_LBTS1.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: Folder_Voss_ENS_LBTS1/dada_stats_Voss_ENS_LBTS1.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime dada2 denoise-single \\\n",
    "--i-demultiplexed-seqs Folder_Voss_ENS_LBTS1/trim_Voss_ENS_LBTS1.qza \\\n",
    "--p-trunc-len 130 \\\n",
    "--p-n-threads 10  \\\n",
    "--p-max-ee 3 \\\n",
    "--o-representative-sequences Folder_Voss_ENS_LBTS1/rep_seqs_Voss_ENS_LBTS1.qza \\\n",
    "--o-table Folder_Voss_ENS_LBTS1/table-Voss_ENS_LBTS1.qza \\\n",
    "--o-denoising-stats Folder_Voss_ENS_LBTS1/dada_stats_Voss_ENS_LBTS1"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
