{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/projects/2020_SCTLD_meta-analysis\n"
     ]
    }
   ],
   "source": [
    "%cd /space/home/rosales/projects/2020_SCTLD_meta-analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/Sam_139_207_emp-paired-end-sequences/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_Sam_139_207.qza\u001b[0m\n",
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/Sam_1_69_emp-paired-end-sequences/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGtransmissionstudies_Sam_1_69.qza\u001b[0m\n",
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/Sam_208_276_emp-paired-end-sequences/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGtransmissionstudies_Sam_208_276.qza\u001b[0m\n",
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/Sam_277_236_emp-paired-end-sequences/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGtransmissionstudies_Sam_277_236.qza\u001b[0m\n",
      "\u001b[32mImported /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/Sam_70_138_emp-paired-end-sequences/ as EMPPairedEndDirFmt to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGtransmissionstudies_Sam_70_138.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/Sam_139_207_emp-paired-end-sequences/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGmicrobiome2018_Sam_139_207.qza\n",
    "\n",
    "\n",
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/Sam_1_69_emp-paired-end-sequences/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGtransmissionstudies_Sam_1_69.qza\n",
    "\n",
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/Sam_208_276_emp-paired-end-sequences/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGtransmissionstudies_Sam_208_276.qza\n",
    "\n",
    "\n",
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/Sam_277_236_emp-paired-end-sequences/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGtransmissionstudies_Sam_277_236.qza\n",
    "\n",
    "!qiime tools import \\\n",
    "--type EMPPairedEndSequences \\\n",
    "--input-path /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/Sam_70_138_emp-paired-end-sequences/ \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Import_Muller_SWGtransmissionstudies_Sam_70_138.qza\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGtransmissionstudies_13_207.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGtransmissionstudies_13_207.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGtransmissionstudies_Sam_139_207.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/sample-metadata-Sam13-207.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGtransmissionstudies_13_207.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGtransmissionstudies_13_207"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGtransmissionstudies_1_69.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGtransmissionstudies_1_69.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGtransmissionstudies_Sam_1_69.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/sample-metadata-Sam1-69.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGtransmissionstudies_1_69.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGtransmissionstudies_1_69"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGtransmissionstudies_208-276.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGtransmissionstudies_208-276.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGtransmissionstudies_Sam_208_276.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/sample-metadata-Sam208-276.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGtransmissionstudies_208-276.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGtransmissionstudies_208-276"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGtransmissionstudies_277_236.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGtransmissionstudies_277_236.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGtransmissionstudies_Sam_277_236.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/sample-metadata-Sam277-326.tsv \\\n",
    "--o-per-sample-sequences demux_Muller_SWGtransmissionstudies_277_236.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGtransmissionstudies_277_236"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: demux_Muller_SWGtransmissionstudies_70_138.qza\u001b[0m\n",
      "\u001b[32mSaved ErrorCorrectionDetails to: demux_err_Muller_SWGtransmissionstudies_70_138.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux emp-paired \\\n",
    "--i-seqs Import_Muller_SWGtransmissionstudies_Sam_70_138.qza \\\n",
    "--m-barcodes-file /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Muller_SWGtransmissionstudies_fastqfiles/sample-metadata-Sam70-138.tsv\\\n",
    "--o-per-sample-sequences demux_Muller_SWGtransmissionstudies_70_138.qza \\\n",
    "--m-barcodes-column BarcodeSequence \\\n",
    "--p-no-golay-error-correction \\\n",
    "--o-error-correction-details demux_err_Muller_SWGtransmissionstudies_70_138"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D1.7.25.19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D1.7.25.19_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D1.7.25.19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D1.7.25.19_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.08 s (17 us/read; 3.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            241,681\n",
      "  Read 1 with adapter:                      78 (0.0%)\n",
      "  Read 2 with adapter:                  12,694 (5.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       241,681 (100.0%)\n",
      "\n",
      "Total basepairs processed:   109,741,027 bp\n",
      "  Read 1:    53,943,127 bp\n",
      "  Read 2:    55,797,900 bp\n",
      "Total written (filtered):    109,570,728 bp (99.8%)\n",
      "  Read 1:    53,942,337 bp\n",
      "  Read 2:    55,628,391 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 78 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t71\t3776.3\t0\t71\n",
      "20\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "54\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12694 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t216\t3776.3\t0\t216\n",
      "4\t23\t944.1\t0\t23\n",
      "5\t78\t236.0\t0\t78\n",
      "6\t106\t59.0\t0\t106\n",
      "7\t248\t14.8\t0\t248\n",
      "8\t323\t3.7\t0\t323\n",
      "9\t1251\t0.9\t0\t1245 6\n",
      "10\t4159\t0.2\t1\t4105 54\n",
      "11\t216\t0.1\t1\t214 2\n",
      "12\t137\t0.0\t1\t133 4\n",
      "13\t102\t0.0\t1\t94 8\n",
      "14\t260\t0.0\t1\t242 18\n",
      "15\t278\t0.0\t1\t265 13\n",
      "16\t767\t0.0\t1\t739 28\n",
      "17\t1171\t0.0\t1\t1138 33\n",
      "18\t667\t0.0\t1\t591 57 19\n",
      "19\t2360\t0.0\t1\t1440 911 9\n",
      "20\t3\t0.0\t2\t1 1 1\n",
      "21\t310\t0.0\t2\t0 304 6\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "53\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t0 1\n",
      "140\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "213\t1\t0.0\t2\t0 1\n",
      "214\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D2.5percent.8.16.19_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D2.5percent.8.16.19_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D2.5percent.8.16.19_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D2.5percent.8.16.19_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.68 s (16 us/read; 3.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            166,338\n",
      "  Read 1 with adapter:                     120 (0.1%)\n",
      "  Read 2 with adapter:                   4,963 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       166,338 (100.0%)\n",
      "\n",
      "Total basepairs processed:    75,794,197 bp\n",
      "  Read 1:    37,247,826 bp\n",
      "  Read 2:    38,546,371 bp\n",
      "Total written (filtered):     75,717,639 bp (99.9%)\n",
      "  Read 1:    37,247,391 bp\n",
      "  Read 2:    38,470,248 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 120 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t118\t2599.0\t0\t118\n",
      "29\t1\t0.0\t1\t1\n",
      "52\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4963 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t2599.0\t0\t18\n",
      "4\t11\t649.8\t0\t11\n",
      "5\t7\t162.4\t0\t7\n",
      "6\t22\t40.6\t0\t22\n",
      "7\t100\t10.2\t0\t100\n",
      "8\t140\t2.5\t0\t140\n",
      "9\t396\t0.6\t0\t395 1\n",
      "10\t620\t0.2\t1\t616 4\n",
      "11\t59\t0.0\t1\t57 2\n",
      "12\t133\t0.0\t1\t129 4\n",
      "13\t157\t0.0\t1\t152 5\n",
      "14\t227\t0.0\t1\t208 19\n",
      "15\t144\t0.0\t1\t135 9\n",
      "16\t380\t0.0\t1\t371 9\n",
      "17\t573\t0.0\t1\t555 18\n",
      "18\t353\t0.0\t1\t304 28 21\n",
      "19\t1370\t0.0\t1\t821 547 2\n",
      "20\t1\t0.0\t2\t0 1\n",
      "21\t224\t0.0\t2\t0 221 3\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "100\t2\t0.0\t2\t2\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "184\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D2.7.25.19_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D2.7.25.19_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D2.7.25.19_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D2.7.25.19_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.40 s (19 us/read; 3.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            127,314\n",
      "  Read 1 with adapter:                     163 (0.1%)\n",
      "  Read 2 with adapter:                   4,209 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       127,314 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,940,127 bp\n",
      "  Read 1:    28,466,103 bp\n",
      "  Read 2:    29,474,024 bp\n",
      "Total written (filtered):     57,879,103 bp (99.9%)\n",
      "  Read 1:    28,465,202 bp\n",
      "  Read 2:    29,413,901 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 163 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t156\t1989.3\t0\t156\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t1\n",
      "86\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4209 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t1989.3\t0\t23\n",
      "4\t9\t497.3\t0\t9\n",
      "5\t11\t124.3\t0\t11\n",
      "6\t17\t31.1\t0\t17\n",
      "7\t53\t7.8\t0\t53\n",
      "8\t100\t1.9\t0\t100\n",
      "9\t357\t0.5\t0\t356 1\n",
      "10\t1143\t0.1\t1\t1123 20\n",
      "11\t80\t0.0\t1\t75 5\n",
      "12\t51\t0.0\t1\t48 3\n",
      "13\t53\t0.0\t1\t53\n",
      "14\t86\t0.0\t1\t80 6\n",
      "15\t109\t0.0\t1\t106 3\n",
      "16\t272\t0.0\t1\t265 7\n",
      "17\t406\t0.0\t1\t398 8\n",
      "18\t299\t0.0\t1\t280 17 2\n",
      "19\t978\t0.0\t1\t608 368 2\n",
      "20\t1\t0.0\t2\t1\n",
      "21\t150\t0.0\t2\t0 148 2\n",
      "22\t5\t0.0\t2\t0 0 5\n",
      "56\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D3.5percent.8.7.19_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D3.5percent.8.7.19_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D3.5percent.8.7.19_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D3.5percent.8.7.19_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.64 s (14 us/read; 4.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            393,197\n",
      "  Read 1 with adapter:                     677 (0.2%)\n",
      "  Read 2 with adapter:                  39,930 (10.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       393,197 (100.0%)\n",
      "\n",
      "Total basepairs processed:   179,737,798 bp\n",
      "  Read 1:    88,021,109 bp\n",
      "  Read 2:    91,716,689 bp\n",
      "Total written (filtered):    179,257,665 bp (99.7%)\n",
      "  Read 1:    88,017,945 bp\n",
      "  Read 2:    91,239,720 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 677 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t653\t6143.7\t0\t653\n",
      "27\t6\t0.0\t1\t6\n",
      "29\t14\t0.0\t1\t14\n",
      "128\t1\t0.0\t1\t0 1\n",
      "132\t1\t0.0\t1\t1\n",
      "156\t1\t0.0\t1\t1\n",
      "221\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 39930 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t304\t6143.7\t0\t304\n",
      "4\t185\t1535.9\t0\t185\n",
      "5\t298\t384.0\t0\t298\n",
      "6\t722\t96.0\t0\t722\n",
      "7\t2190\t24.0\t0\t2190\n",
      "8\t3315\t6.0\t0\t3315\n",
      "9\t9353\t1.5\t0\t9341 12\n",
      "10\t8988\t0.4\t1\t8871 117\n",
      "11\t435\t0.1\t1\t424 11\n",
      "12\t573\t0.0\t1\t553 20\n",
      "13\t890\t0.0\t1\t862 28\n",
      "14\t1243\t0.0\t1\t1177 66\n",
      "15\t779\t0.0\t1\t728 51\n",
      "16\t2081\t0.0\t1\t2006 75\n",
      "17\t2280\t0.0\t1\t2171 109\n",
      "18\t1370\t0.0\t1\t1175 119 76\n",
      "19\t4276\t0.0\t1\t2348 1906 22\n",
      "20\t10\t0.0\t2\t1 6 3\n",
      "21\t526\t0.0\t2\t0 516 10\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "35\t2\t0.0\t2\t2\n",
      "40\t1\t0.0\t2\t1\n",
      "46\t1\t0.0\t2\t0 1\n",
      "48\t2\t0.0\t2\t2\n",
      "50\t2\t0.0\t2\t2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t2\t0.0\t2\t1 0 1\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t2\n",
      "63\t2\t0.0\t2\t1 1\n",
      "64\t2\t0.0\t2\t1 1\n",
      "65\t1\t0.0\t2\t0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t0 1\n",
      "71\t1\t0.0\t2\t1\n",
      "74\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t0 1\n",
      "78\t4\t0.0\t2\t3 1\n",
      "79\t2\t0.0\t2\t2\n",
      "80\t1\t0.0\t2\t1\n",
      "82\t1\t0.0\t2\t1\n",
      "83\t1\t0.0\t2\t0 1\n",
      "84\t2\t0.0\t2\t2\n",
      "86\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "90\t2\t0.0\t2\t1 0 1\n",
      "94\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "101\t1\t0.0\t2\t1\n",
      "110\t1\t0.0\t2\t1\n",
      "111\t1\t0.0\t2\t1\n",
      "114\t1\t0.0\t2\t1\n",
      "125\t1\t0.0\t2\t1\n",
      "127\t1\t0.0\t2\t1\n",
      "144\t1\t0.0\t2\t0 1\n",
      "150\t1\t0.0\t2\t1\n",
      "152\t1\t0.0\t2\t1\n",
      "157\t1\t0.0\t2\t1\n",
      "162\t1\t0.0\t2\t1\n",
      "187\t1\t0.0\t2\t1\n",
      "188\t1\t0.0\t2\t1\n",
      "197\t1\t0.0\t2\t1\n",
      "203\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "218\t3\t0.0\t2\t2 1\n",
      "232\t1\t0.0\t2\t1\n",
      "235\t1\t0.0\t2\t1\n",
      "236\t2\t0.0\t2\t1 0 1\n",
      "237\t3\t0.0\t2\t2 0 1\n",
      "238\t3\t0.0\t2\t3\n",
      "239\t1\t0.0\t2\t1\n",
      "240\t3\t0.0\t2\t1 1 1\n",
      "241\t4\t0.0\t2\t2 1 1\n",
      "242\t1\t0.0\t2\t1\n",
      "243\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D3.7.25.19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D3.7.25.19_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D3.7.25.19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D3.7.25.19_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.95 s (22 us/read; 2.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             87,413\n",
      "  Read 1 with adapter:                      59 (0.1%)\n",
      "  Read 2 with adapter:                   1,991 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        87,413 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,800,180 bp\n",
      "  Read 1:    19,564,544 bp\n",
      "  Read 2:    20,235,636 bp\n",
      "Total written (filtered):     39,768,697 bp (99.9%)\n",
      "  Read 1:    19,564,199 bp\n",
      "  Read 2:    20,204,498 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 59 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t57\t1365.8\t0\t57\n",
      "27\t1\t0.0\t1\t1\n",
      "147\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1991 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t1365.8\t0\t13\n",
      "4\t5\t341.5\t0\t5\n",
      "5\t7\t85.4\t0\t7\n",
      "6\t6\t21.3\t0\t6\n",
      "7\t30\t5.3\t0\t30\n",
      "8\t50\t1.3\t0\t50\n",
      "9\t113\t0.3\t0\t111 2\n",
      "10\t297\t0.1\t1\t296 1\n",
      "11\t32\t0.0\t1\t32\n",
      "12\t41\t0.0\t1\t41\n",
      "13\t20\t0.0\t1\t19 1\n",
      "14\t60\t0.0\t1\t56 4\n",
      "15\t50\t0.0\t1\t48 2\n",
      "16\t158\t0.0\t1\t155 3\n",
      "17\t229\t0.0\t1\t221 8\n",
      "18\t179\t0.0\t1\t164 12 3\n",
      "19\t586\t0.0\t1\t377 208 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t108\t0.0\t2\t0 107 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t0 1\n",
      "167\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D4.5percent.8.16.19_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D4.5percent.8.16.19_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D4.5percent.8.16.19_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D4.5percent.8.16.19_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.79 s (16 us/read; 3.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            178,492\n",
      "  Read 1 with adapter:                     227 (0.1%)\n",
      "  Read 2 with adapter:                   7,766 (4.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       178,492 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,322,656 bp\n",
      "  Read 1:    39,940,187 bp\n",
      "  Read 2:    41,382,469 bp\n",
      "Total written (filtered):     81,218,428 bp (99.9%)\n",
      "  Read 1:    39,938,936 bp\n",
      "  Read 2:    41,279,492 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 227 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t220\t2788.9\t0\t220\n",
      "23\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "116\t1\t0.0\t1\t1\n",
      "158\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7766 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t2788.9\t0\t45\n",
      "4\t12\t697.2\t0\t12\n",
      "5\t12\t174.3\t0\t12\n",
      "6\t24\t43.6\t0\t24\n",
      "7\t142\t10.9\t0\t142\n",
      "8\t183\t2.7\t0\t183\n",
      "9\t834\t0.7\t0\t825 9\n",
      "10\t2931\t0.2\t1\t2900 31\n",
      "11\t139\t0.0\t1\t136 3\n",
      "12\t98\t0.0\t1\t98\n",
      "13\t56\t0.0\t1\t53 3\n",
      "14\t121\t0.0\t1\t114 7\n",
      "15\t129\t0.0\t1\t122 7\n",
      "16\t391\t0.0\t1\t374 17\n",
      "17\t587\t0.0\t1\t568 19\n",
      "18\t412\t0.0\t1\t392 13 7\n",
      "19\t1415\t0.0\t1\t875 531 9\n",
      "20\t2\t0.0\t2\t0 2\n",
      "21\t213\t0.0\t2\t0 212 1\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "36\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 0 1\n",
      "228\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D4.7.25.19_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D4.7.25.19_62_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D4.7.25.19_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D4.7.25.19_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.42 s (19 us/read; 3.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,666\n",
      "  Read 1 with adapter:                     115 (0.1%)\n",
      "  Read 2 with adapter:                   3,802 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       129,666 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,057,142 bp\n",
      "  Read 1:    29,022,654 bp\n",
      "  Read 2:    30,034,488 bp\n",
      "Total written (filtered):     58,999,455 bp (99.9%)\n",
      "  Read 1:    29,021,739 bp\n",
      "  Read 2:    29,977,716 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 115 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t108\t2026.0\t0\t108\n",
      "25\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t1\t0.0\t1\t1\n",
      "50\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3802 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t2026.0\t0\t22\n",
      "4\t8\t506.5\t0\t8\n",
      "5\t9\t126.6\t0\t9\n",
      "6\t6\t31.7\t0\t6\n",
      "7\t50\t7.9\t0\t50\n",
      "8\t75\t2.0\t0\t75\n",
      "9\t246\t0.5\t0\t245 1\n",
      "10\t884\t0.1\t1\t873 11\n",
      "11\t70\t0.0\t1\t69 1\n",
      "12\t59\t0.0\t1\t59\n",
      "13\t46\t0.0\t1\t45 1\n",
      "14\t78\t0.0\t1\t72 6\n",
      "15\t91\t0.0\t1\t86 5\n",
      "16\t266\t0.0\t1\t262 4\n",
      "17\t396\t0.0\t1\t386 10\n",
      "18\t314\t0.0\t1\t287 21 6\n",
      "19\t985\t0.0\t1\t638 342 5\n",
      "20\t3\t0.0\t2\t0 2 1\n",
      "21\t177\t0.0\t2\t0 175 2\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "70\t3\t0.0\t2\t3\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t2 1\n",
      "100\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D1.7.25.19_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D1.7.25.19_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D1.7.25.19_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D1.7.25.19_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.43 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,652\n",
      "  Read 1 with adapter:                     165 (0.1%)\n",
      "  Read 2 with adapter:                   4,521 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       129,652 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,978,269 bp\n",
      "  Read 1:    28,974,709 bp\n",
      "  Read 2:    30,003,560 bp\n",
      "Total written (filtered):     58,912,242 bp (99.9%)\n",
      "  Read 1:    28,973,635 bp\n",
      "  Read 2:    29,938,607 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 165 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t154\t2025.8\t0\t154\n",
      "12\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "166\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4521 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t2025.8\t0\t26\n",
      "4\t7\t506.5\t0\t7\n",
      "5\t11\t126.6\t0\t11\n",
      "6\t20\t31.7\t0\t20\n",
      "7\t70\t7.9\t0\t70\n",
      "8\t94\t2.0\t0\t94\n",
      "9\t375\t0.5\t0\t373 2\n",
      "10\t1311\t0.1\t1\t1289 22\n",
      "11\t90\t0.0\t1\t90\n",
      "12\t63\t0.0\t1\t62 1\n",
      "13\t50\t0.0\t1\t46 4\n",
      "14\t78\t0.0\t1\t72 6\n",
      "15\t102\t0.0\t1\t97 5\n",
      "16\t286\t0.0\t1\t283 3\n",
      "17\t489\t0.0\t1\t476 13\n",
      "18\t293\t0.0\t1\t268 21 4\n",
      "19\t975\t0.0\t1\t603 369 3\n",
      "21\t159\t0.0\t2\t0 158 1\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "99\t2\t0.0\t2\t0 2\n",
      "134\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t1\n",
      "181\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D2.5percent.8.14.19_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D2.5percent.8.14.19_52_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D2.5percent.8.14.19_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D2.5percent.8.14.19_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.60 s (15 us/read; 3.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            236,635\n",
      "  Read 1 with adapter:                     301 (0.1%)\n",
      "  Read 2 with adapter:                  13,205 (5.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       236,635 (100.0%)\n",
      "\n",
      "Total basepairs processed:   107,960,270 bp\n",
      "  Read 1:    52,991,040 bp\n",
      "  Read 2:    54,969,230 bp\n",
      "Total written (filtered):    107,786,771 bp (99.8%)\n",
      "  Read 1:    52,989,615 bp\n",
      "  Read 2:    54,797,156 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 301 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t289\t3697.4\t0\t289\n",
      "9\t1\t0.9\t0\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t2\t0.0\t1\t2\n",
      "55\t1\t0.0\t1\t1\n",
      "104\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13205 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t120\t3697.4\t0\t120\n",
      "4\t33\t924.4\t0\t33\n",
      "5\t41\t231.1\t0\t41\n",
      "6\t74\t57.8\t0\t74\n",
      "7\t347\t14.4\t0\t347\n",
      "8\t437\t3.6\t0\t437\n",
      "9\t1609\t0.9\t0\t1604 5\n",
      "10\t4668\t0.2\t1\t4604 64\n",
      "11\t172\t0.1\t1\t167 5\n",
      "12\t111\t0.0\t1\t108 3\n",
      "13\t148\t0.0\t1\t140 8\n",
      "14\t267\t0.0\t1\t253 14\n",
      "15\t244\t0.0\t1\t236 8\n",
      "16\t792\t0.0\t1\t763 29\n",
      "17\t1018\t0.0\t1\t976 42\n",
      "18\t672\t0.0\t1\t615 41 16\n",
      "19\t2117\t0.0\t1\t1290 814 13\n",
      "20\t4\t0.0\t2\t2 2\n",
      "21\t302\t0.0\t2\t0 297 5\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t3\n",
      "100\t1\t0.0\t2\t1\n",
      "116\t1\t0.0\t2\t0 1\n",
      "138\t1\t0.0\t2\t0 1\n",
      "156\t1\t0.0\t2\t1\n",
      "162\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "202\t1\t0.0\t2\t1\n",
      "217\t3\t0.0\t2\t2 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D2.7.25.19_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D2.7.25.19_55_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D2.7.25.19_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D2.7.25.19_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.97 s (17 us/read; 3.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            178,310\n",
      "  Read 1 with adapter:                     198 (0.1%)\n",
      "  Read 2 with adapter:                   7,119 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       178,310 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,257,526 bp\n",
      "  Read 1:    39,910,709 bp\n",
      "  Read 2:    41,346,817 bp\n",
      "Total written (filtered):     81,157,826 bp (99.9%)\n",
      "  Read 1:    39,909,806 bp\n",
      "  Read 2:    41,248,020 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 198 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t185\t2786.1\t0\t185\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t1\t0.0\t1\t0 1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7119 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t2786.1\t0\t46\n",
      "4\t10\t696.5\t0\t10\n",
      "5\t24\t174.1\t0\t24\n",
      "6\t28\t43.5\t0\t28\n",
      "7\t174\t10.9\t0\t174\n",
      "8\t181\t2.7\t0\t181\n",
      "9\t675\t0.7\t0\t669 6\n",
      "10\t2089\t0.2\t1\t2073 16\n",
      "11\t116\t0.0\t1\t114 2\n",
      "12\t101\t0.0\t1\t100 1\n",
      "13\t90\t0.0\t1\t86 4\n",
      "14\t168\t0.0\t1\t156 12\n",
      "15\t174\t0.0\t1\t169 5\n",
      "16\t467\t0.0\t1\t443 24\n",
      "17\t663\t0.0\t1\t628 35\n",
      "18\t462\t0.0\t1\t424 26 12\n",
      "19\t1385\t0.0\t1\t864 519 2\n",
      "20\t4\t0.0\t2\t0 2 2\n",
      "21\t243\t0.0\t2\t0 242 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "50\t2\t0.0\t2\t2\n",
      "58\t2\t0.0\t2\t1 0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t0 1\n",
      "79\t5\t0.0\t2\t5\n",
      "172\t1\t0.0\t2\t1\n",
      "221\t1\t0.0\t2\t0 0 1\n",
      "238\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D3.5percent.9.4.19_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D3.5percent.9.4.19_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D3.5percent.9.4.19_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D3.5percent.9.4.19_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.92 s (16 us/read; 3.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            245,997\n",
      "  Read 1 with adapter:                     403 (0.2%)\n",
      "  Read 2 with adapter:                  11,973 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       245,997 (100.0%)\n",
      "\n",
      "Total basepairs processed:   112,184,935 bp\n",
      "  Read 1:    55,084,731 bp\n",
      "  Read 2:    57,100,204 bp\n",
      "Total written (filtered):    112,024,645 bp (99.9%)\n",
      "  Read 1:    55,082,997 bp\n",
      "  Read 2:    56,941,648 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 403 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t395\t3843.7\t0\t395\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "55\t2\t0.0\t1\t2\n",
      "113\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11973 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t104\t3843.7\t0\t104\n",
      "4\t33\t960.9\t0\t33\n",
      "5\t36\t240.2\t0\t36\n",
      "6\t85\t60.1\t0\t85\n",
      "7\t263\t15.0\t0\t263\n",
      "8\t422\t3.8\t0\t422\n",
      "9\t1406\t0.9\t0\t1398 8\n",
      "10\t3909\t0.2\t1\t3860 49\n",
      "11\t178\t0.1\t1\t173 5\n",
      "12\t121\t0.0\t1\t120 1\n",
      "13\t155\t0.0\t1\t146 9\n",
      "14\t265\t0.0\t1\t252 13\n",
      "15\t257\t0.0\t1\t245 12\n",
      "16\t664\t0.0\t1\t644 20\n",
      "17\t1002\t0.0\t1\t971 31\n",
      "18\t607\t0.0\t1\t560 38 9\n",
      "19\t2133\t0.0\t1\t1340 785 8\n",
      "20\t5\t0.0\t2\t0 4 1\n",
      "21\t301\t0.0\t2\t0 299 2\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "30\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t0 1\n",
      "114\t1\t0.0\t2\t1\n",
      "142\t1\t0.0\t2\t0 1\n",
      "182\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t3\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D3.7.25.19_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D3.7.25.19_53_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D3.7.25.19_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D3.7.25.19_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.75 s (20 us/read; 3.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            137,991\n",
      "  Read 1 with adapter:                     161 (0.1%)\n",
      "  Read 2 with adapter:                   4,808 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       137,991 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,780,373 bp\n",
      "  Read 1:    30,841,711 bp\n",
      "  Read 2:    31,938,662 bp\n",
      "Total written (filtered):     62,708,483 bp (99.9%)\n",
      "  Read 1:    30,840,376 bp\n",
      "  Read 2:    31,868,107 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 161 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t147\t2156.1\t0\t147\n",
      "23\t2\t0.0\t1\t2\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t2\t0.0\t1\t2\n",
      "62\t1\t0.0\t1\t1\n",
      "141\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4808 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t2156.1\t0\t31\n",
      "4\t8\t539.0\t0\t8\n",
      "5\t11\t134.8\t0\t11\n",
      "6\t18\t33.7\t0\t18\n",
      "7\t86\t8.4\t0\t86\n",
      "8\t109\t2.1\t0\t109\n",
      "9\t389\t0.5\t0\t387 2\n",
      "10\t1176\t0.1\t1\t1162 14\n",
      "11\t69\t0.0\t1\t65 4\n",
      "12\t50\t0.0\t1\t50\n",
      "13\t62\t0.0\t1\t59 3\n",
      "14\t92\t0.0\t1\t84 8\n",
      "15\t126\t0.0\t1\t120 6\n",
      "16\t371\t0.0\t1\t356 15\n",
      "17\t518\t0.0\t1\t496 22\n",
      "18\t332\t0.0\t1\t301 22 9\n",
      "19\t1174\t0.0\t1\t725 446 3\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t163\t0.0\t2\t0 161 2\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "30\t1\t0.0\t2\t1\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "53\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t2\n",
      "77\t2\t0.0\t2\t1 1\n",
      "79\t3\t0.0\t2\t3\n",
      "99\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.5percent.8.16.19_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.5percent.8.16.19_51_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.5percent.8.16.19_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.5percent.8.16.19_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.60 s (19 us/read; 3.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            135,384\n",
      "  Read 1 with adapter:                     184 (0.1%)\n",
      "  Read 2 with adapter:                   4,146 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       135,384 (100.0%)\n",
      "\n",
      "Total basepairs processed:    61,629,044 bp\n",
      "  Read 1:    30,285,863 bp\n",
      "  Read 2:    31,343,181 bp\n",
      "Total written (filtered):     61,568,095 bp (99.9%)\n",
      "  Read 1:    30,285,165 bp\n",
      "  Read 2:    31,282,930 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 184 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t178\t2115.4\t0\t178\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4146 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t2115.4\t0\t19\n",
      "4\t8\t528.8\t0\t8\n",
      "5\t10\t132.2\t0\t10\n",
      "6\t15\t33.1\t0\t15\n",
      "7\t62\t8.3\t0\t62\n",
      "8\t99\t2.1\t0\t99\n",
      "9\t331\t0.5\t0\t330 1\n",
      "10\t1026\t0.1\t1\t1011 15\n",
      "11\t82\t0.0\t1\t82\n",
      "12\t70\t0.0\t1\t70\n",
      "13\t44\t0.0\t1\t44\n",
      "14\t95\t0.0\t1\t88 7\n",
      "15\t104\t0.0\t1\t98 6\n",
      "16\t289\t0.0\t1\t282 7\n",
      "17\t436\t0.0\t1\t424 12\n",
      "18\t298\t0.0\t1\t279 15 4\n",
      "19\t993\t0.0\t1\t615 374 4\n",
      "20\t2\t0.0\t2\t1 0 1\n",
      "21\t154\t0.0\t2\t0 151 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "213\t1\t0.0\t2\t0 0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D1.7.25.19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D1.7.25.19_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D1.7.25.19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D1.7.25.19_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D2.5percent.8.16.19_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D2.5percent.8.16.19_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D2.5percent.8.16.19_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D2.5percent.8.16.19_9_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D2.7.25.19_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D2.7.25.19_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D2.7.25.19_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D2.7.25.19_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D3.5percent.8.7.19_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D3.5percent.8.7.19_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D3.5percent.8.7.19_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D3.5percent.8.7.19_8_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D3.7.25.19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D3.7.25.19_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D3.7.25.19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D3.7.25.19_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D4.5percent.8.16.19_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D4.5percent.8.16.19_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D4.5percent.8.16.19_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D4.5percent.8.16.19_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D4.7.25.19_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCA.D4.7.25.19_62_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D4.7.25.19_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCA.D4.7.25.19_62_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D1.7.25.19_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D1.7.25.19_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D1.7.25.19_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D1.7.25.19_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D2.5percent.8.14.19_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D2.5percent.8.14.19_52_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D2.5percent.8.14.19_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D2.5percent.8.14.19_52_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D2.7.25.19_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D2.7.25.19_55_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D2.7.25.19_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D2.7.25.19_55_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D3.5percent.9.4.19_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D3.5percent.9.4.19_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D3.5percent.9.4.19_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D3.5percent.9.4.19_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D3.7.25.19_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D3.7.25.19_53_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D3.7.25.19_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D3.7.25.19_53_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.5percent.8.16.19_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.5percent.8.16.19_51_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.5percent.8.16.19_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.5percent.8.16.19_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.5percent.9.12.19_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.5percent.9.12.19_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.5percent.9.12.19_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.5percent.9.12.19_21_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.5percent.9.12.19_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.5percent.9.12.19_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.5percent.9.12.19_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.5percent.9.12.19_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.75 s (17 us/read; 3.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            274,246\n",
      "  Read 1 with adapter:                     396 (0.1%)\n",
      "  Read 2 with adapter:                  24,828 (9.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       274,246 (100.0%)\n",
      "\n",
      "Total basepairs processed:   125,311,850 bp\n",
      "  Read 1:    61,392,892 bp\n",
      "  Read 2:    63,918,958 bp\n",
      "Total written (filtered):    124,996,239 bp (99.7%)\n",
      "  Read 1:    61,390,418 bp\n",
      "  Read 2:    63,605,821 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 396 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t371\t4285.1\t0\t371\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t17\t0.0\t1\t17\n",
      "37\t1\t0.0\t1\t1\n",
      "39\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "134\t2\t0.0\t1\t2\n",
      "197\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 24828 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t198\t4285.1\t0\t198\n",
      "4\t88\t1071.3\t0\t88\n",
      "5\t133\t267.8\t0\t133\n",
      "6\t360\t67.0\t0\t360\n",
      "7\t1044\t16.7\t0\t1044\n",
      "8\t1767\t4.2\t0\t1767\n",
      "9\t5331\t1.0\t0\t5318 13\n",
      "10\t5450\t0.3\t1\t5392 58\n",
      "11\t337\t0.1\t1\t329 8\n",
      "12\t475\t0.0\t1\t464 11\n",
      "13\t613\t0.0\t1\t584 29\n",
      "14\t954\t0.0\t1\t898 56\n",
      "15\t560\t0.0\t1\t522 38\n",
      "16\t1436\t0.0\t1\t1375 61\n",
      "17\t1581\t0.0\t1\t1489 92\n",
      "18\t951\t0.0\t1\t828 81 42\n",
      "19\t2979\t0.0\t1\t1712 1255 12\n",
      "20\t6\t0.0\t2\t0 3 3\n",
      "21\t406\t0.0\t2\t0 400 6\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "25\t1\t0.0\t2\t1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t3\t0.0\t2\t3\n",
      "65\t5\t0.0\t2\t2 0 3\n",
      "66\t1\t0.0\t2\t0 1\n",
      "67\t1\t0.0\t2\t0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t3\t0.0\t2\t3\n",
      "70\t10\t0.0\t2\t8 0 2\n",
      "71\t9\t0.0\t2\t1 1 7\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "75\t2\t0.0\t2\t2\n",
      "76\t1\t0.0\t2\t1\n",
      "77\t3\t0.0\t2\t1 1 1\n",
      "79\t30\t0.0\t2\t26 1 3\n",
      "80\t1\t0.0\t2\t1\n",
      "81\t1\t0.0\t2\t1\n",
      "82\t1\t0.0\t2\t0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 1\n",
      "87\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t1\n",
      "96\t2\t0.0\t2\t2\n",
      "99\t2\t0.0\t2\t1 0 1\n",
      "100\t3\t0.0\t2\t2 1\n",
      "101\t1\t0.0\t2\t1\n",
      "104\t2\t0.0\t2\t1 1\n",
      "106\t1\t0.0\t2\t1\n",
      "134\t1\t0.0\t2\t1\n",
      "141\t1\t0.0\t2\t1\n",
      "145\t1\t0.0\t2\t0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "172\t3\t0.0\t2\t1 0 2\n",
      "179\t2\t0.0\t2\t1 1\n",
      "181\t1\t0.0\t2\t1\n",
      "182\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t0 1\n",
      "186\t7\t0.0\t2\t5 2\n",
      "194\t2\t0.0\t2\t2\n",
      "205\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "225\t1\t0.0\t2\t0 1\n",
      "226\t1\t0.0\t2\t0 1\n",
      "236\t1\t0.0\t2\t0 1\n",
      "240\t1\t0.0\t2\t1\n",
      "244\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.7.25.19_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.7.25.19_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.7.25.19_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.7.25.19_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.48 s (19 us/read; 3.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            180,731\n",
      "  Read 1 with adapter:                     258 (0.1%)\n",
      "  Read 2 with adapter:                   6,621 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       180,731 (100.0%)\n",
      "\n",
      "Total basepairs processed:    82,347,544 bp\n",
      "  Read 1:    40,452,688 bp\n",
      "  Read 2:    41,894,856 bp\n",
      "Total written (filtered):     82,251,902 bp (99.9%)\n",
      "  Read 1:    40,450,580 bp\n",
      "  Read 2:    41,801,322 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 258 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t236\t2823.9\t0\t236\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t4\t0.0\t1\t4\n",
      "88\t2\t0.0\t1\t2\n",
      "133\t1\t0.0\t1\t1\n",
      "145\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t1\n",
      "196\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6621 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t2823.9\t0\t36\n",
      "4\t10\t706.0\t0\t10\n",
      "5\t16\t176.5\t0\t16\n",
      "6\t28\t44.1\t0\t28\n",
      "7\t121\t11.0\t0\t121\n",
      "8\t140\t2.8\t0\t140\n",
      "9\t552\t0.7\t0\t550 2\n",
      "10\t1899\t0.2\t1\t1866 33\n",
      "11\t101\t0.0\t1\t100 1\n",
      "12\t108\t0.0\t1\t105 3\n",
      "13\t106\t0.0\t1\t104 2\n",
      "14\t176\t0.0\t1\t165 11\n",
      "15\t162\t0.0\t1\t159 3\n",
      "16\t409\t0.0\t1\t391 18\n",
      "17\t649\t0.0\t1\t623 26\n",
      "18\t480\t0.0\t1\t430 42 8\n",
      "19\t1387\t0.0\t1\t851 526 10\n",
      "20\t4\t0.0\t2\t0 3 1\n",
      "21\t223\t0.0\t2\t0 222 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "52\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 1\n",
      "88\t1\t0.0\t2\t1\n",
      "101\t1\t0.0\t2\t1\n",
      "183\t1\t0.0\t2\t0 1\n",
      "214\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D1.7.25.19_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D1.7.25.19_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D1.7.25.19_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D1.7.25.19_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.14 s (19 us/read; 3.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,769\n",
      "  Read 1 with adapter:                     115 (0.1%)\n",
      "  Read 2 with adapter:                   3,104 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       112,769 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,328,764 bp\n",
      "  Read 1:    25,224,186 bp\n",
      "  Read 2:    26,104,578 bp\n",
      "Total written (filtered):     51,280,580 bp (99.9%)\n",
      "  Read 1:    25,223,597 bp\n",
      "  Read 2:    26,056,983 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 115 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t1762.0\t0\t109\n",
      "23\t1\t0.0\t1\t0 1\n",
      "26\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t1\t0.0\t1\t1\n",
      "130\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3104 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t1762.0\t0\t18\n",
      "4\t3\t440.5\t0\t3\n",
      "5\t4\t110.1\t0\t4\n",
      "6\t10\t27.5\t0\t10\n",
      "7\t33\t6.9\t0\t33\n",
      "8\t63\t1.7\t0\t63\n",
      "9\t243\t0.4\t0\t242 1\n",
      "10\t665\t0.1\t1\t657 8\n",
      "11\t54\t0.0\t1\t53 1\n",
      "12\t41\t0.0\t1\t40 1\n",
      "13\t41\t0.0\t1\t41\n",
      "14\t65\t0.0\t1\t63 2\n",
      "15\t66\t0.0\t1\t61 5\n",
      "16\t209\t0.0\t1\t207 2\n",
      "17\t333\t0.0\t1\t319 14\n",
      "18\t206\t0.0\t1\t190 11 5\n",
      "19\t875\t0.0\t1\t590 281 4\n",
      "20\t5\t0.0\t2\t3 2\n",
      "21\t147\t0.0\t2\t0 145 2\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "51\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t6\t0.0\t2\t6\n",
      "87\t1\t0.0\t2\t0 1\n",
      "100\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "218\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D2.5percent.8.7.19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D2.5percent.8.7.19_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D2.5percent.8.7.19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D2.5percent.8.7.19_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.72 s (15 us/read; 3.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            246,315\n",
      "  Read 1 with adapter:                     261 (0.1%)\n",
      "  Read 2 with adapter:                  19,905 (8.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       246,315 (100.0%)\n",
      "\n",
      "Total basepairs processed:   112,510,395 bp\n",
      "  Read 1:    55,159,532 bp\n",
      "  Read 2:    57,350,863 bp\n",
      "Total written (filtered):    112,265,160 bp (99.8%)\n",
      "  Read 1:    55,158,650 bp\n",
      "  Read 2:    57,106,510 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 261 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t257\t3848.7\t0\t257\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 19905 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t200\t3848.7\t0\t200\n",
      "4\t46\t962.2\t0\t46\n",
      "5\t65\t240.5\t0\t65\n",
      "6\t132\t60.1\t0\t132\n",
      "7\t645\t15.0\t0\t645\n",
      "8\t745\t3.8\t0\t745\n",
      "9\t2869\t0.9\t0\t2863 6\n",
      "10\t7935\t0.2\t1\t7831 104\n",
      "11\t252\t0.1\t1\t247 5\n",
      "12\t175\t0.0\t1\t172 3\n",
      "13\t207\t0.0\t1\t198 9\n",
      "14\t427\t0.0\t1\t399 28\n",
      "15\t384\t0.0\t1\t368 16\n",
      "16\t1008\t0.0\t1\t984 24\n",
      "17\t1285\t0.0\t1\t1234 51\n",
      "18\t747\t0.0\t1\t688 41 18\n",
      "19\t2406\t0.0\t1\t1391 1003 12\n",
      "20\t6\t0.0\t2\t0 5 1\n",
      "21\t328\t0.0\t2\t0 323 5\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t1\n",
      "55\t2\t0.0\t2\t2\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t3\t0.0\t2\t1 0 2\n",
      "74\t2\t0.0\t2\t1 1\n",
      "78\t2\t0.0\t2\t2\n",
      "79\t3\t0.0\t2\t3\n",
      "99\t1\t0.0\t2\t1\n",
      "100\t3\t0.0\t2\t2 0 1\n",
      "117\t1\t0.0\t2\t1\n",
      "179\t1\t0.0\t2\t1\n",
      "186\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t0 1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "247\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D2.7.25.19_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D2.7.25.19_54_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D2.7.25.19_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D2.7.25.19_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.54 s (22 us/read; 2.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,421\n",
      "  Read 1 with adapter:                      98 (0.1%)\n",
      "  Read 2 with adapter:                   3,594 (5.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        70,421 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,974,152 bp\n",
      "  Read 1:    15,691,103 bp\n",
      "  Read 2:    16,283,049 bp\n",
      "Total written (filtered):     31,925,728 bp (99.8%)\n",
      "  Read 1:    15,689,788 bp\n",
      "  Read 2:    16,235,940 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 98 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t89\t1100.3\t0\t89\n",
      "29\t2\t0.0\t1\t2\n",
      "51\t1\t0.0\t1\t1\n",
      "100\t2\t0.0\t1\t2\n",
      "119\t1\t0.0\t1\t1\n",
      "186\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3594 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t1100.3\t0\t25\n",
      "4\t10\t275.1\t0\t10\n",
      "5\t9\t68.8\t0\t9\n",
      "6\t17\t17.2\t0\t17\n",
      "7\t78\t4.3\t0\t78\n",
      "8\t106\t1.1\t0\t106\n",
      "9\t439\t0.3\t0\t436 3\n",
      "10\t1331\t0.1\t1\t1316 15\n",
      "11\t52\t0.0\t1\t51 1\n",
      "12\t31\t0.0\t1\t30 1\n",
      "13\t27\t0.0\t1\t25 2\n",
      "14\t56\t0.0\t1\t48 8\n",
      "15\t77\t0.0\t1\t76 1\n",
      "16\t177\t0.0\t1\t173 4\n",
      "17\t269\t0.0\t1\t262 7\n",
      "18\t188\t0.0\t1\t171 14 3\n",
      "19\t602\t0.0\t1\t354 245 3\n",
      "20\t1\t0.0\t2\t0 1\n",
      "21\t89\t0.0\t2\t0 88 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "70\t2\t0.0\t2\t2\n",
      "79\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D3.5percent.8.7.19_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D3.5percent.8.7.19_56_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D3.5percent.8.7.19_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D3.5percent.8.7.19_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.13 s (15 us/read; 4.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            213,105\n",
      "  Read 1 with adapter:                     282 (0.1%)\n",
      "  Read 2 with adapter:                  10,815 (5.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       213,105 (100.0%)\n",
      "\n",
      "Total basepairs processed:    97,196,337 bp\n",
      "  Read 1:    47,717,714 bp\n",
      "  Read 2:    49,478,623 bp\n",
      "Total written (filtered):     97,054,007 bp (99.9%)\n",
      "  Read 1:    47,716,585 bp\n",
      "  Read 2:    49,337,422 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 282 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t270\t3329.8\t0\t270\n",
      "21\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t6\t0.0\t1\t6\n",
      "29\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10815 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t3329.8\t0\t72\n",
      "4\t15\t832.4\t0\t15\n",
      "5\t26\t208.1\t0\t26\n",
      "6\t63\t52.0\t0\t63\n",
      "7\t219\t13.0\t0\t219\n",
      "8\t274\t3.3\t0\t274\n",
      "9\t1230\t0.8\t0\t1227 3\n",
      "10\t4151\t0.2\t1\t4096 55\n",
      "11\t172\t0.1\t1\t168 4\n",
      "12\t99\t0.0\t1\t98 1\n",
      "13\t100\t0.0\t1\t95 5\n",
      "14\t178\t0.0\t1\t169 9\n",
      "15\t203\t0.0\t1\t195 8\n",
      "16\t581\t0.0\t1\t565 16\n",
      "17\t817\t0.0\t1\t799 18\n",
      "18\t518\t0.0\t1\t478 34 6\n",
      "19\t1797\t0.0\t1\t1115 674 8\n",
      "20\t3\t0.0\t2\t0 3\n",
      "21\t278\t0.0\t2\t0 274 4\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "55\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t2 1 1\n",
      "100\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t1\n",
      "141\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D3.7.25.19_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D3.7.25.19_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D3.7.25.19_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D3.7.25.19_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.00 s (19 us/read; 3.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,348\n",
      "  Read 1 with adapter:                     132 (0.1%)\n",
      "  Read 2 with adapter:                   1,578 (1.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,348 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,412,535 bp\n",
      "  Read 1:    23,809,294 bp\n",
      "  Read 2:    24,603,241 bp\n",
      "Total written (filtered):     48,384,561 bp (99.9%)\n",
      "  Read 1:    23,808,503 bp\n",
      "  Read 2:    24,576,058 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 132 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t121\t1661.7\t0\t121\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t0 1\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t6\t0.0\t1\t6\n",
      "85\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1578 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1661.7\t0\t6\n",
      "5\t2\t103.9\t0\t2\n",
      "6\t7\t26.0\t0\t7\n",
      "7\t16\t6.5\t0\t16\n",
      "8\t17\t1.6\t0\t17\n",
      "9\t41\t0.4\t0\t41\n",
      "10\t99\t0.1\t1\t95 4\n",
      "11\t19\t0.0\t1\t19\n",
      "12\t27\t0.0\t1\t27\n",
      "13\t21\t0.0\t1\t21\n",
      "14\t40\t0.0\t1\t36 4\n",
      "15\t30\t0.0\t1\t28 2\n",
      "16\t115\t0.0\t1\t111 4\n",
      "17\t228\t0.0\t1\t224 4\n",
      "18\t152\t0.0\t1\t140 10 2\n",
      "19\t640\t0.0\t1\t415 223 2\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t112\t0.0\t2\t0 110 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "217\t2\t0.0\t2\t2\n",
      "230\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D4.5percent.8.8.19_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D4.5percent.8.8.19_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D4.5percent.8.8.19_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D4.5percent.8.8.19_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.68 s (15 us/read; 4.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            448,286\n",
      "  Read 1 with adapter:                     574 (0.1%)\n",
      "  Read 2 with adapter:                  37,155 (8.3%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       448,285 (100.0%)\n",
      "\n",
      "Total basepairs processed:   204,732,825 bp\n",
      "  Read 1:   100,349,804 bp\n",
      "  Read 2:   104,383,021 bp\n",
      "Total written (filtered):    204,243,728 bp (99.8%)\n",
      "  Read 1:   100,345,698 bp\n",
      "  Read 2:   103,898,030 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 574 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t542\t7004.5\t0\t542\n",
      "5\t1\t437.8\t0\t1\n",
      "21\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t0 1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t3\t0.0\t1\t3\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t9\t0.0\t1\t9\n",
      "68\t1\t0.0\t1\t0 1\n",
      "87\t1\t0.0\t1\t0 1\n",
      "92\t5\t0.0\t1\t5\n",
      "100\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t0 1\n",
      "171\t1\t0.0\t1\t0 1\n",
      "178\t1\t0.0\t1\t1\n",
      "185\t1\t0.0\t1\t0 1\n",
      "195\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 37155 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t253\t7004.5\t0\t253\n",
      "4\t95\t1751.1\t0\t95\n",
      "5\t142\t437.8\t0\t142\n",
      "6\t368\t109.4\t0\t368\n",
      "7\t1091\t27.4\t0\t1091\n",
      "8\t1791\t6.8\t0\t1791\n",
      "9\t6136\t1.7\t0\t6124 12\n",
      "10\t8732\t0.4\t1\t8633 99\n",
      "11\t644\t0.1\t1\t628 16\n",
      "12\t1008\t0.0\t1\t977 31\n",
      "13\t1349\t0.0\t1\t1283 66\n",
      "14\t1821\t0.0\t1\t1740 81\n",
      "15\t1019\t0.0\t1\t951 68\n",
      "16\t2664\t0.0\t1\t2535 129\n",
      "17\t2691\t0.0\t1\t2590 101\n",
      "18\t1592\t0.0\t1\t1355 131 106\n",
      "19\t4974\t0.0\t1\t2742 2206 26\n",
      "20\t10\t0.0\t2\t2 6 2\n",
      "21\t618\t0.0\t2\t0 614 4\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "28\t1\t0.0\t2\t1\n",
      "30\t1\t0.0\t2\t1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "50\t1\t0.0\t2\t1\n",
      "55\t3\t0.0\t2\t3\n",
      "57\t2\t0.0\t2\t0 1 1\n",
      "58\t2\t0.0\t2\t1 1\n",
      "59\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t0 1\n",
      "63\t4\t0.0\t2\t3 0 1\n",
      "64\t3\t0.0\t2\t3\n",
      "65\t2\t0.0\t2\t2\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t4\t0.0\t2\t3 1\n",
      "71\t8\t0.0\t2\t2 0 6\n",
      "73\t1\t0.0\t2\t1\n",
      "75\t2\t0.0\t2\t0 2\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 1 1\n",
      "79\t34\t0.0\t2\t28 3 3\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t1\n",
      "86\t2\t0.0\t2\t0 2\n",
      "87\t1\t0.0\t2\t1\n",
      "89\t4\t0.0\t2\t4\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t1\n",
      "100\t2\t0.0\t2\t2\n",
      "113\t1\t0.0\t2\t1\n",
      "114\t2\t0.0\t2\t2\n",
      "115\t1\t0.0\t2\t1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "153\t2\t0.0\t2\t0 1 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "169\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t1\n",
      "175\t1\t0.0\t2\t0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t1\n",
      "183\t1\t0.0\t2\t1\n",
      "186\t2\t0.0\t2\t1 1\n",
      "190\t1\t0.0\t2\t0 1\n",
      "195\t1\t0.0\t2\t1\n",
      "198\t1\t0.0\t2\t1\n",
      "215\t2\t0.0\t2\t1 1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t3\t0.0\t2\t3\n",
      "222\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "237\t2\t0.0\t2\t2\n",
      "241\t1\t0.0\t2\t1\n",
      "242\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t0 0 1\n",
      "251\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D4.7.25.19_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D4.7.25.19_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D4.7.25.19_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D4.7.25.19_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.92 s (33 us/read; 1.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,904\n",
      "  Read 1 with adapter:                      38 (0.1%)\n",
      "  Read 2 with adapter:                     969 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,904 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,476,372 bp\n",
      "  Read 1:     6,119,064 bp\n",
      "  Read 2:     6,357,308 bp\n",
      "Total written (filtered):     12,460,072 bp (99.9%)\n",
      "  Read 1:     6,116,722 bp\n",
      "  Read 2:     6,343,350 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 38 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t436.0\t0\t24\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t2 1\n",
      "217\t6\t0.0\t1\t5 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 969 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t436.0\t0\t5\n",
      "5\t1\t27.2\t0\t1\n",
      "6\t4\t6.8\t0\t4\n",
      "7\t17\t1.7\t0\t17\n",
      "8\t26\t0.4\t0\t26\n",
      "9\t91\t0.1\t0\t90 1\n",
      "10\t234\t0.0\t1\t234\n",
      "11\t30\t0.0\t1\t29 1\n",
      "12\t25\t0.0\t1\t25\n",
      "13\t16\t0.0\t1\t16\n",
      "14\t18\t0.0\t1\t17 1\n",
      "15\t25\t0.0\t1\t24 1\n",
      "16\t83\t0.0\t1\t80 3\n",
      "17\t102\t0.0\t1\t100 2\n",
      "18\t62\t0.0\t1\t59 3\n",
      "19\t181\t0.0\t1\t119 61 1\n",
      "21\t45\t0.0\t2\t0 45\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D1.7.25.19_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D1.7.25.19_65_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D1.7.25.19_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D1.7.25.19_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.60 s (28 us/read; 2.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,045\n",
      "  Read 1 with adapter:                      60 (0.1%)\n",
      "  Read 2 with adapter:                   2,417 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        58,045 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,211,847 bp\n",
      "  Read 1:    12,865,617 bp\n",
      "  Read 2:    13,346,230 bp\n",
      "Total written (filtered):     26,175,545 bp (99.9%)\n",
      "  Read 1:    12,864,234 bp\n",
      "  Read 2:    13,311,311 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 60 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t907.0\t0\t50\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t0 2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2417 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t907.0\t0\t23\n",
      "4\t6\t226.7\t0\t6\n",
      "5\t7\t56.7\t0\t7\n",
      "6\t7\t14.2\t0\t7\n",
      "7\t44\t3.5\t0\t44\n",
      "8\t53\t0.9\t0\t53\n",
      "9\t230\t0.2\t0\t229 1\n",
      "10\t622\t0.1\t1\t613 9\n",
      "11\t24\t0.0\t1\t24\n",
      "12\t44\t0.0\t1\t44\n",
      "13\t38\t0.0\t1\t36 2\n",
      "14\t68\t0.0\t1\t64 4\n",
      "15\t71\t0.0\t1\t67 4\n",
      "16\t170\t0.0\t1\t165 5\n",
      "17\t274\t0.0\t1\t263 11\n",
      "18\t158\t0.0\t1\t138 14 6\n",
      "19\t492\t0.0\t1\t283 204 5\n",
      "20\t1\t0.0\t2\t0 1\n",
      "21\t71\t0.0\t2\t0 71\n",
      "22\t5\t0.0\t2\t0 0 5\n",
      "70\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t1 0 1\n",
      "148\t1\t0.0\t2\t1\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "185\t1\t0.0\t2\t0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "224\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D2.7.25.19_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D2.7.25.19_69_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D2.7.25.19_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D2.7.25.19_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.79 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,227\n",
      "  Read 1 with adapter:                       9 (0.1%)\n",
      "  Read 2 with adapter:                     669 (6.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        10,227 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,358,913 bp\n",
      "  Read 1:     2,124,285 bp\n",
      "  Read 2:     2,234,628 bp\n",
      "Total written (filtered):      4,349,405 bp (99.8%)\n",
      "  Read 1:     2,123,379 bp\n",
      "  Read 2:     2,226,026 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t159.8\t0\t4\n",
      "26\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 669 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t159.8\t0\t4\n",
      "5\t4\t10.0\t0\t4\n",
      "6\t3\t2.5\t0\t3\n",
      "7\t10\t0.6\t0\t10\n",
      "8\t19\t0.2\t0\t19\n",
      "9\t92\t0.0\t0\t92\n",
      "10\t233\t0.0\t1\t231 2\n",
      "11\t9\t0.0\t1\t8 1\n",
      "12\t13\t0.0\t1\t13\n",
      "13\t11\t0.0\t1\t11\n",
      "14\t18\t0.0\t1\t16 2\n",
      "15\t20\t0.0\t1\t20\n",
      "16\t45\t0.0\t1\t43 2\n",
      "17\t58\t0.0\t1\t57 1\n",
      "18\t26\t0.0\t1\t23 1 2\n",
      "19\t89\t0.0\t1\t50 39\n",
      "21\t14\t0.0\t2\t0 14\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D3.7.25.19_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D3.7.25.19_64_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D3.7.25.19_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D3.7.25.19_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.97 s (43 us/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,636\n",
      "  Read 1 with adapter:                      29 (0.1%)\n",
      "  Read 2 with adapter:                     860 (3.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,636 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,067,934 bp\n",
      "  Read 1:     4,936,110 bp\n",
      "  Read 2:     5,131,824 bp\n",
      "Total written (filtered):     10,054,411 bp (99.9%)\n",
      "  Read 1:     4,935,167 bp\n",
      "  Read 2:     5,119,244 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t353.7\t0\t25\n",
      "217\t4\t0.0\t1\t3 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 860 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t353.7\t0\t4\n",
      "5\t4\t22.1\t0\t4\n",
      "6\t4\t5.5\t0\t4\n",
      "7\t11\t1.4\t0\t11\n",
      "8\t25\t0.3\t0\t25\n",
      "9\t82\t0.1\t0\t82\n",
      "10\t193\t0.0\t1\t190 3\n",
      "11\t8\t0.0\t1\t8\n",
      "12\t11\t0.0\t1\t11\n",
      "13\t18\t0.0\t1\t17 1\n",
      "14\t28\t0.0\t1\t27 1\n",
      "15\t22\t0.0\t1\t22\n",
      "16\t70\t0.0\t1\t70\n",
      "17\t89\t0.0\t1\t87 2\n",
      "18\t68\t0.0\t1\t61 5 2\n",
      "19\t198\t0.0\t1\t114 83 1\n",
      "21\t23\t0.0\t2\t0 22 1\n",
      "83\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D1.8.12.19_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D1.8.12.19_63_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D1.8.12.19_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D1.8.12.19_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.99 s (42 us/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,721\n",
      "  Read 1 with adapter:                      30 (0.1%)\n",
      "  Read 2 with adapter:                   1,398 (5.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,721 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,505,889 bp\n",
      "  Read 1:     5,142,126 bp\n",
      "  Read 2:     5,363,763 bp\n",
      "Total written (filtered):     10,486,116 bp (99.8%)\n",
      "  Read 1:     5,140,918 bp\n",
      "  Read 2:     5,345,198 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 30 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t370.6\t0\t23\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t2 1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1398 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t370.6\t0\t9\n",
      "4\t6\t92.7\t0\t6\n",
      "5\t9\t23.2\t0\t9\n",
      "6\t11\t5.8\t0\t11\n",
      "7\t23\t1.4\t0\t23\n",
      "8\t41\t0.4\t0\t41\n",
      "9\t138\t0.1\t0\t138\n",
      "10\t443\t0.0\t1\t436 7\n",
      "11\t23\t0.0\t1\t23\n",
      "12\t14\t0.0\t1\t14\n",
      "13\t26\t0.0\t1\t26\n",
      "14\t46\t0.0\t1\t44 2\n",
      "15\t39\t0.0\t1\t39\n",
      "16\t92\t0.0\t1\t89 3\n",
      "17\t152\t0.0\t1\t145 7\n",
      "18\t49\t0.0\t1\t43 4 2\n",
      "19\t251\t0.0\t1\t156 95\n",
      "21\t25\t0.0\t2\t0 24 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D2.8.12.19_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D2.8.12.19_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D2.8.12.19_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D2.8.12.19_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.32 s (31 us/read; 1.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,820\n",
      "  Read 1 with adapter:                      35 (0.1%)\n",
      "  Read 2 with adapter:                   2,268 (5.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,820 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,252,674 bp\n",
      "  Read 1:     9,440,545 bp\n",
      "  Read 2:     9,812,129 bp\n",
      "Total written (filtered):     19,220,307 bp (99.8%)\n",
      "  Read 1:     9,439,428 bp\n",
      "  Read 2:     9,780,879 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 35 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t669.1\t0\t27\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2268 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t669.1\t0\t17\n",
      "4\t1\t167.3\t0\t1\n",
      "5\t8\t41.8\t0\t8\n",
      "6\t11\t10.5\t0\t11\n",
      "7\t48\t2.6\t0\t48\n",
      "8\t54\t0.7\t0\t54\n",
      "9\t196\t0.2\t0\t195 1\n",
      "10\t702\t0.0\t1\t693 9\n",
      "11\t34\t0.0\t1\t34\n",
      "12\t29\t0.0\t1\t28 1\n",
      "13\t30\t0.0\t1\t25 5\n",
      "14\t56\t0.0\t1\t54 2\n",
      "15\t62\t0.0\t1\t58 4\n",
      "16\t179\t0.0\t1\t174 5\n",
      "17\t254\t0.0\t1\t247 7\n",
      "18\t117\t0.0\t1\t108 7 2\n",
      "19\t421\t0.0\t1\t263 158\n",
      "21\t45\t0.0\t2\t0 45\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.7.25.19_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCB.D4.7.25.19_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.7.25.19_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCB.D4.7.25.19_12_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D1.7.25.19_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D1.7.25.19_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D1.7.25.19_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D1.7.25.19_16_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D2.5percent.8.7.19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D2.5percent.8.7.19_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D2.5percent.8.7.19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D2.5percent.8.7.19_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D2.7.25.19_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D2.7.25.19_54_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D2.7.25.19_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D2.7.25.19_54_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D3.5percent.8.7.19_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D3.5percent.8.7.19_56_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D3.5percent.8.7.19_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D3.5percent.8.7.19_56_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D3.7.25.19_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D3.7.25.19_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D3.7.25.19_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D3.7.25.19_25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D4.5percent.8.8.19_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D4.5percent.8.8.19_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D4.5percent.8.8.19_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D4.5percent.8.8.19_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D4.7.25.19_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCC.D4.7.25.19_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D4.7.25.19_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCC.D4.7.25.19_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D1.7.25.19_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D1.7.25.19_65_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D1.7.25.19_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D1.7.25.19_65_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D2.7.25.19_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D2.7.25.19_69_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D2.7.25.19_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D2.7.25.19_69_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D3.7.25.19_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.DCD.D3.7.25.19_64_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D3.7.25.19_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.DCD.D3.7.25.19_64_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D1.8.12.19_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D1.8.12.19_63_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D1.8.12.19_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D1.8.12.19_63_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D2.8.12.19_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D2.8.12.19_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D2.8.12.19_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D2.8.12.19_27_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D3.8.12.19_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D3.8.12.19_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D3.8.12.19_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D3.8.12.19_39_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D3.8.12.19_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D3.8.12.19_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D3.8.12.19_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D3.8.12.19_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.78 s (22 us/read; 2.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,249\n",
      "  Read 1 with adapter:                      84 (0.1%)\n",
      "  Read 2 with adapter:                   4,368 (5.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        80,249 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,334,736 bp\n",
      "  Read 1:    17,826,667 bp\n",
      "  Read 2:    18,508,069 bp\n",
      "Total written (filtered):     36,272,795 bp (99.8%)\n",
      "  Read 1:    17,825,245 bp\n",
      "  Read 2:    18,447,550 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 84 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t75\t1253.9\t0\t75\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "216\t2\t0.0\t1\t1 1\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4368 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t1253.9\t0\t47\n",
      "4\t4\t313.5\t0\t4\n",
      "5\t8\t78.4\t0\t8\n",
      "6\t19\t19.6\t0\t19\n",
      "7\t67\t4.9\t0\t67\n",
      "8\t92\t1.2\t0\t92\n",
      "9\t341\t0.3\t0\t339 2\n",
      "10\t1352\t0.1\t1\t1336 16\n",
      "11\t67\t0.0\t1\t66 1\n",
      "12\t62\t0.0\t1\t62\n",
      "13\t62\t0.0\t1\t58 4\n",
      "14\t163\t0.0\t1\t151 12\n",
      "15\t110\t0.0\t1\t100 10\n",
      "16\t320\t0.0\t1\t309 11\n",
      "17\t455\t0.0\t1\t443 12\n",
      "18\t259\t0.0\t1\t228 22 9\n",
      "19\t842\t0.0\t1\t530 305 7\n",
      "20\t1\t0.0\t2\t1\n",
      "21\t84\t0.0\t2\t0 84\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "246\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D4.8.12.19_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D4.8.12.19_49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D4.8.12.19_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D4.8.12.19_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.37 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,144\n",
      "  Read 1 with adapter:                      60 (0.1%)\n",
      "  Read 2 with adapter:                   2,729 (5.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,144 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,571,996 bp\n",
      "  Read 1:    11,070,244 bp\n",
      "  Read 2:    11,501,752 bp\n",
      "Total written (filtered):     22,532,541 bp (99.8%)\n",
      "  Read 1:    11,069,035 bp\n",
      "  Read 2:    11,463,506 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 60 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t783.5\t0\t53\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "133\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2729 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t783.5\t0\t22\n",
      "4\t2\t195.9\t0\t2\n",
      "5\t10\t49.0\t0\t10\n",
      "6\t7\t12.2\t0\t7\n",
      "7\t51\t3.1\t0\t51\n",
      "8\t51\t0.8\t0\t51\n",
      "9\t234\t0.2\t0\t234\n",
      "10\t797\t0.0\t1\t789 8\n",
      "11\t40\t0.0\t1\t39 1\n",
      "12\t38\t0.0\t1\t36 2\n",
      "13\t46\t0.0\t1\t44 2\n",
      "14\t81\t0.0\t1\t75 6\n",
      "15\t86\t0.0\t1\t83 3\n",
      "16\t215\t0.0\t1\t209 6\n",
      "17\t271\t0.0\t1\t259 12\n",
      "18\t178\t0.0\t1\t155 17 6\n",
      "19\t526\t0.0\t1\t315 210 1\n",
      "20\t2\t0.0\t2\t0 2\n",
      "21\t64\t0.0\t2\t0 63 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t3\n",
      "115\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.18.A.control.6.25.19_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.18.A.control.6.25.19_61_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.18.A.control.6.25.19_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.18.A.control.6.25.19_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.53 s (27 us/read; 2.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,775\n",
      "  Read 1 with adapter:                      45 (0.1%)\n",
      "  Read 2 with adapter:                   2,501 (4.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,775 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,164,098 bp\n",
      "  Read 1:    12,345,718 bp\n",
      "  Read 2:    12,818,380 bp\n",
      "Total written (filtered):     25,127,260 bp (99.9%)\n",
      "  Read 1:    12,344,891 bp\n",
      "  Read 2:    12,782,369 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 45 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t871.5\t0\t40\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "223\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2501 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t871.5\t0\t23\n",
      "4\t2\t217.9\t0\t2\n",
      "5\t10\t54.5\t0\t10\n",
      "6\t16\t13.6\t0\t16\n",
      "7\t39\t3.4\t0\t39\n",
      "8\t43\t0.9\t0\t43\n",
      "9\t202\t0.2\t0\t200 2\n",
      "10\t661\t0.1\t1\t654 7\n",
      "11\t37\t0.0\t1\t36 1\n",
      "12\t32\t0.0\t1\t29 3\n",
      "13\t30\t0.0\t1\t28 2\n",
      "14\t69\t0.0\t1\t62 7\n",
      "15\t75\t0.0\t1\t74 1\n",
      "16\t194\t0.0\t1\t185 9\n",
      "17\t254\t0.0\t1\t244 10\n",
      "18\t175\t0.0\t1\t162 9 4\n",
      "19\t561\t0.0\t1\t354 204 3\n",
      "20\t3\t0.0\t2\t3\n",
      "21\t67\t0.0\t2\t0 66 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t3\n",
      "176\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.18.D.control.6.25.19_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.18.D.control.6.25.19_60_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.18.D.control.6.25.19_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.18.D.control.6.25.19_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.89 s (53 us/read; 1.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,850\n",
      "  Read 1 with adapter:                      17 (0.1%)\n",
      "  Read 2 with adapter:                     957 (5.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        16,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,362,167 bp\n",
      "  Read 1:     3,606,825 bp\n",
      "  Read 2:     3,755,342 bp\n",
      "Total written (filtered):      7,347,780 bp (99.8%)\n",
      "  Read 1:     3,605,243 bp\n",
      "  Read 2:     3,742,537 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 17 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t263.3\t0\t8\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t1\n",
      "215\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 957 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t263.3\t0\t8\n",
      "4\t1\t65.8\t0\t1\n",
      "5\t3\t16.5\t0\t3\n",
      "6\t7\t4.1\t0\t7\n",
      "7\t22\t1.0\t0\t22\n",
      "8\t23\t0.3\t0\t23\n",
      "9\t107\t0.1\t0\t107\n",
      "10\t290\t0.0\t1\t287 3\n",
      "11\t8\t0.0\t1\t8\n",
      "12\t11\t0.0\t1\t11\n",
      "13\t21\t0.0\t1\t20 1\n",
      "14\t25\t0.0\t1\t24 1\n",
      "15\t29\t0.0\t1\t28 1\n",
      "16\t71\t0.0\t1\t71\n",
      "17\t84\t0.0\t1\t83 1\n",
      "18\t61\t0.0\t1\t56 1 4\n",
      "19\t161\t0.0\t1\t92 69\n",
      "20\t5\t0.0\t2\t4 0 1\n",
      "21\t20\t0.0\t2\t0 20\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.7.B.control.6.25.19_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.7.B.control.6.25.19_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.7.B.control.6.25.19_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.7.B.control.6.25.19_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.82 s (17 us/read; 3.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            166,050\n",
      "  Read 1 with adapter:                     132 (0.1%)\n",
      "  Read 2 with adapter:                   7,798 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       166,050 (100.0%)\n",
      "\n",
      "Total basepairs processed:    75,560,635 bp\n",
      "  Read 1:    37,099,747 bp\n",
      "  Read 2:    38,460,888 bp\n",
      "Total written (filtered):     75,454,291 bp (99.9%)\n",
      "  Read 1:    37,098,207 bp\n",
      "  Read 2:    38,356,084 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 132 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t115\t2594.5\t0\t115\n",
      "21\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t3\t0.0\t1\t3\n",
      "29\t5\t0.0\t1\t5\n",
      "203\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7798 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t2594.5\t0\t85\n",
      "4\t15\t648.6\t0\t15\n",
      "5\t25\t162.2\t0\t25\n",
      "6\t21\t40.5\t0\t21\n",
      "7\t149\t10.1\t0\t149\n",
      "8\t182\t2.5\t0\t182\n",
      "9\t665\t0.6\t0\t659 6\n",
      "10\t2860\t0.2\t1\t2841 19\n",
      "11\t122\t0.0\t1\t122\n",
      "12\t67\t0.0\t1\t66 1\n",
      "13\t51\t0.0\t1\t48 3\n",
      "14\t149\t0.0\t1\t141 8\n",
      "15\t165\t0.0\t1\t163 2\n",
      "16\t477\t0.0\t1\t466 11\n",
      "17\t695\t0.0\t1\t675 20\n",
      "18\t412\t0.0\t1\t379 24 9\n",
      "19\t1434\t0.0\t1\t870 561 3\n",
      "20\t5\t0.0\t2\t4 1\n",
      "21\t202\t0.0\t2\t0 200 2\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "30\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t2\n",
      "214\t1\t0.0\t2\t1\n",
      "218\t2\t0.0\t2\t2\n",
      "238\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.7.F.control.6.25.19_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.7.F.control.6.25.19_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.7.F.control.6.25.19_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.7.F.control.6.25.19_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.99 s (18 us/read; 3.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            169,165\n",
      "  Read 1 with adapter:                     225 (0.1%)\n",
      "  Read 2 with adapter:                   6,142 (3.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       169,165 (100.0%)\n",
      "\n",
      "Total basepairs processed:    77,061,545 bp\n",
      "  Read 1:    37,855,135 bp\n",
      "  Read 2:    39,206,410 bp\n",
      "Total written (filtered):     76,971,546 bp (99.9%)\n",
      "  Read 1:    37,853,901 bp\n",
      "  Read 2:    39,117,645 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 225 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t215\t2643.2\t0\t215\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t3\t0.0\t1\t3\n",
      "78\t1\t0.0\t1\t1\n",
      "135\t1\t0.0\t1\t1\n",
      "184\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6142 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t2643.2\t0\t28\n",
      "4\t7\t660.8\t0\t7\n",
      "5\t16\t165.2\t0\t16\n",
      "6\t23\t41.3\t0\t23\n",
      "7\t101\t10.3\t0\t101\n",
      "8\t126\t2.6\t0\t126\n",
      "9\t531\t0.6\t0\t528 3\n",
      "10\t1783\t0.2\t1\t1768 15\n",
      "11\t94\t0.0\t1\t93 1\n",
      "12\t89\t0.0\t1\t87 2\n",
      "13\t59\t0.0\t1\t57 2\n",
      "14\t130\t0.0\t1\t120 10\n",
      "15\t138\t0.0\t1\t127 11\n",
      "16\t380\t0.0\t1\t374 6\n",
      "17\t599\t0.0\t1\t573 26\n",
      "18\t426\t0.0\t1\t399 18 9\n",
      "19\t1342\t0.0\t1\t829 506 7\n",
      "20\t8\t0.0\t2\t6 2\n",
      "21\t229\t0.0\t2\t0 224 5\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "64\t1\t0.0\t2\t1\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "74\t2\t0.0\t2\t0 1 1\n",
      "79\t2\t0.0\t2\t2\n",
      "101\t1\t0.0\t2\t1\n",
      "102\t1\t0.0\t2\t1\n",
      "140\t1\t0.0\t2\t0 1\n",
      "158\t1\t0.0\t2\t1\n",
      "179\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t0 1 1\n",
      "227\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCA.7.23.19_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCA.7.23.19_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCA.7.23.19_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCA.7.23.19_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.78 s (19 us/read; 3.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            253,664\n",
      "  Read 1 with adapter:                     149 (0.1%)\n",
      "  Read 2 with adapter:                  11,426 (4.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       253,664 (100.0%)\n",
      "\n",
      "Total basepairs processed:   115,315,122 bp\n",
      "  Read 1:    56,690,261 bp\n",
      "  Read 2:    58,624,861 bp\n",
      "Total written (filtered):    115,151,597 bp (99.9%)\n",
      "  Read 1:    56,688,282 bp\n",
      "  Read 2:    58,463,315 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 149 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t125\t3963.5\t0\t125\n",
      "27\t10\t0.0\t1\t10\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t7\t0.0\t1\t7\n",
      "87\t1\t0.0\t1\t1\n",
      "149\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11426 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t3963.5\t0\t85\n",
      "4\t28\t990.9\t0\t28\n",
      "5\t54\t247.7\t0\t54\n",
      "6\t108\t61.9\t0\t108\n",
      "7\t280\t15.5\t0\t280\n",
      "8\t392\t3.9\t0\t392\n",
      "9\t1461\t1.0\t0\t1458 3\n",
      "10\t2036\t0.2\t1\t2002 34\n",
      "11\t150\t0.1\t1\t147 3\n",
      "12\t222\t0.0\t1\t214 8\n",
      "13\t246\t0.0\t1\t232 14\n",
      "14\t399\t0.0\t1\t374 25\n",
      "15\t307\t0.0\t1\t292 15\n",
      "16\t842\t0.0\t1\t810 32\n",
      "17\t1231\t0.0\t1\t1188 43\n",
      "18\t693\t0.0\t1\t627 47 19\n",
      "19\t2519\t0.0\t1\t1539 971 9\n",
      "20\t7\t0.0\t2\t3 3 1\n",
      "21\t335\t0.0\t2\t1 330 4\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "34\t1\t0.0\t2\t1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "58\t2\t0.0\t2\t0 1 1\n",
      "70\t2\t0.0\t2\t2\n",
      "79\t2\t0.0\t2\t2\n",
      "95\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "214\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCA.7.25.19_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCA.7.25.19_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCA.7.25.19_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCA.7.25.19_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.99 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             87,729\n",
      "  Read 1 with adapter:                      88 (0.1%)\n",
      "  Read 2 with adapter:                   6,125 (7.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        87,729 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,254,382 bp\n",
      "  Read 1:    19,239,650 bp\n",
      "  Read 2:    20,014,732 bp\n",
      "Total written (filtered):     39,170,787 bp (99.8%)\n",
      "  Read 1:    19,237,411 bp\n",
      "  Read 2:    19,933,376 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 88 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t1370.8\t0\t72\n",
      "24\t1\t0.0\t1\t0 1\n",
      "25\t2\t0.0\t1\t1 1\n",
      "27\t2\t0.0\t1\t1 1\n",
      "29\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t1\n",
      "198\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t3\t0.0\t1\t2 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6125 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t41\t1370.8\t0\t41\n",
      "4\t9\t342.7\t0\t9\n",
      "5\t24\t85.7\t0\t24\n",
      "6\t63\t21.4\t0\t63\n",
      "7\t194\t5.4\t0\t194\n",
      "8\t308\t1.3\t0\t308\n",
      "9\t939\t0.3\t0\t932 7\n",
      "10\t1313\t0.1\t1\t1304 9\n",
      "11\t82\t0.0\t1\t79 3\n",
      "12\t126\t0.0\t1\t124 2\n",
      "13\t151\t0.0\t1\t145 6\n",
      "14\t272\t0.0\t1\t260 12\n",
      "15\t170\t0.0\t1\t157 13\n",
      "16\t487\t0.0\t1\t462 25\n",
      "17\t538\t0.0\t1\t519 19\n",
      "18\t291\t0.0\t1\t252 21 18\n",
      "19\t962\t0.0\t1\t552 409 1\n",
      "20\t7\t0.0\t2\t3 3 1\n",
      "21\t133\t0.0\t2\t0 130 3\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "54\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "82\t1\t0.0\t2\t0 1\n",
      "95\t1\t0.0\t2\t1\n",
      "124\t1\t0.0\t2\t1\n",
      "147\t1\t0.0\t2\t0 1\n",
      "240\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCB.7.23.19_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCB.7.23.19_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCB.7.23.19_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCB.7.23.19_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.42 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,843\n",
      "  Read 1 with adapter:                      64 (0.1%)\n",
      "  Read 2 with adapter:                   2,260 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,843 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,656,545 bp\n",
      "  Read 1:    12,596,744 bp\n",
      "  Read 2:    13,059,801 bp\n",
      "Total written (filtered):     25,623,418 bp (99.9%)\n",
      "  Read 1:    12,595,223 bp\n",
      "  Read 2:    13,028,195 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 64 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t888.2\t0\t56\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "215\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t2 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2260 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t888.2\t0\t21\n",
      "5\t9\t55.5\t0\t9\n",
      "6\t11\t13.9\t0\t11\n",
      "7\t25\t3.5\t0\t25\n",
      "8\t50\t0.9\t0\t50\n",
      "9\t196\t0.2\t0\t193 3\n",
      "10\t703\t0.1\t1\t694 9\n",
      "11\t37\t0.0\t1\t36 1\n",
      "12\t35\t0.0\t1\t35\n",
      "13\t23\t0.0\t1\t22 1\n",
      "14\t47\t0.0\t1\t47\n",
      "15\t50\t0.0\t1\t48 2\n",
      "16\t161\t0.0\t1\t157 4\n",
      "17\t231\t0.0\t1\t223 8\n",
      "18\t125\t0.0\t1\t117 7 1\n",
      "19\t457\t0.0\t1\t288 169\n",
      "20\t1\t0.0\t2\t0 1\n",
      "21\t74\t0.0\t2\t0 74\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCB.7.25.19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCB.7.25.19_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCB.7.25.19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCB.7.25.19_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.53 s (17 us/read; 3.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            381,495\n",
      "  Read 1 with adapter:                     355 (0.1%)\n",
      "  Read 2 with adapter:                  18,703 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       381,495 (100.0%)\n",
      "\n",
      "Total basepairs processed:   173,825,965 bp\n",
      "  Read 1:    85,347,239 bp\n",
      "  Read 2:    88,478,726 bp\n",
      "Total written (filtered):    173,561,900 bp (99.8%)\n",
      "  Read 1:    85,344,828 bp\n",
      "  Read 2:    88,217,072 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 355 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t335\t5960.9\t0\t335\n",
      "6\t1\t93.1\t0\t1\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t6\t0.0\t1\t5 1\n",
      "130\t1\t0.0\t1\t1\n",
      "150\t1\t0.0\t1\t0 1\n",
      "170\t2\t0.0\t1\t2\n",
      "202\t1\t0.0\t1\t1\n",
      "221\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18703 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t113\t5960.9\t0\t113\n",
      "4\t80\t1490.2\t0\t80\n",
      "5\t86\t372.6\t0\t86\n",
      "6\t181\t93.1\t0\t181\n",
      "7\t473\t23.3\t0\t473\n",
      "8\t840\t5.8\t0\t840\n",
      "9\t2976\t1.5\t0\t2969 7\n",
      "10\t3268\t0.4\t1\t3235 33\n",
      "11\t198\t0.1\t1\t192 6\n",
      "12\t289\t0.0\t1\t284 5\n",
      "13\t373\t0.0\t1\t355 18\n",
      "14\t581\t0.0\t1\t541 40\n",
      "15\t450\t0.0\t1\t426 24\n",
      "16\t1257\t0.0\t1\t1202 55\n",
      "17\t1823\t0.0\t1\t1757 66\n",
      "18\t1128\t0.0\t1\t998 87 43\n",
      "19\t3977\t0.0\t1\t2349 1607 21\n",
      "20\t2\t0.0\t2\t1 1\n",
      "21\t517\t0.0\t2\t0 509 8\n",
      "22\t31\t0.0\t2\t0 1 30\n",
      "30\t1\t0.0\t2\t1\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "50\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 1\n",
      "63\t3\t0.0\t2\t3\n",
      "64\t2\t0.0\t2\t2\n",
      "65\t2\t0.0\t2\t2\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t0 1 1\n",
      "70\t3\t0.0\t2\t1 0 2\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "74\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t2\n",
      "79\t8\t0.0\t2\t7 1\n",
      "80\t1\t0.0\t2\t1\n",
      "97\t3\t0.0\t2\t2 1\n",
      "100\t1\t0.0\t2\t1\n",
      "118\t1\t0.0\t2\t1\n",
      "133\t1\t0.0\t2\t1\n",
      "179\t1\t0.0\t2\t1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "189\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t6\t0.0\t2\t5 0 1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "236\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.7.23.19_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.7.23.19_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.7.23.19_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.7.23.19_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.85 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,013\n",
      "  Read 1 with adapter:                      77 (0.1%)\n",
      "  Read 2 with adapter:                   3,896 (5.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,013 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,703,488 bp\n",
      "  Read 1:    16,523,513 bp\n",
      "  Read 2:    17,179,975 bp\n",
      "Total written (filtered):     33,643,031 bp (99.8%)\n",
      "  Read 1:    16,521,188 bp\n",
      "  Read 2:    17,121,843 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 77 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t1172.1\t0\t61\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t4\t0.0\t1\t4\n",
      "207\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "218\t4\t0.0\t1\t4\n",
      "219\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3896 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1172.1\t0\t16\n",
      "4\t11\t293.0\t0\t11\n",
      "5\t10\t73.3\t0\t10\n",
      "6\t30\t18.3\t0\t30\n",
      "7\t71\t4.6\t0\t71\n",
      "8\t116\t1.1\t0\t116\n",
      "9\t407\t0.3\t0\t407\n",
      "10\t614\t0.1\t1\t603 11\n",
      "11\t68\t0.0\t1\t64 4\n",
      "12\t109\t0.0\t1\t108 1\n",
      "13\t130\t0.0\t1\t125 5\n",
      "14\t218\t0.0\t1\t204 14\n",
      "15\t132\t0.0\t1\t120 12\n",
      "16\t362\t0.0\t1\t346 16\n",
      "17\t418\t0.0\t1\t403 15\n",
      "18\t237\t0.0\t1\t213 11 13\n",
      "19\t813\t0.0\t1\t470 341 2\n",
      "20\t4\t0.0\t2\t0 1 3\n",
      "21\t97\t0.0\t2\t0 94 3\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t1\n",
      "55\t2\t0.0\t2\t2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "70\t5\t0.0\t2\t2 0 3\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t3\n",
      "93\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t1\n",
      "146\t1\t0.0\t2\t0 1\n",
      "148\t1\t0.0\t2\t1\n",
      "155\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "176\t1\t0.0\t2\t1\n",
      "186\t2\t0.0\t2\t1 0 1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "236\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D1.7.25.19_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D1.7.25.19_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D1.7.25.19_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D1.7.25.19_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.42 s (22 us/read; 2.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,951\n",
      "  Read 1 with adapter:                      26 (0.0%)\n",
      "  Read 2 with adapter:                   5,075 (4.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,951 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,753,673 bp\n",
      "  Read 1:    24,459,373 bp\n",
      "  Read 2:    25,294,300 bp\n",
      "Total written (filtered):     49,682,461 bp (99.9%)\n",
      "  Read 1:    24,458,011 bp\n",
      "  Read 2:    25,224,450 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 26 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t1718.0\t0\t14\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t2\t0.0\t1\t2\n",
      "91\t1\t0.0\t1\t1\n",
      "197\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t0 1\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5075 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t1718.0\t0\t39\n",
      "4\t15\t429.5\t0\t15\n",
      "5\t19\t107.4\t0\t19\n",
      "6\t54\t26.8\t0\t54\n",
      "7\t152\t6.7\t0\t152\n",
      "8\t253\t1.7\t0\t253\n",
      "9\t920\t0.4\t0\t919 1\n",
      "10\t830\t0.1\t1\t821 9\n",
      "11\t35\t0.0\t1\t34 1\n",
      "12\t49\t0.0\t1\t47 2\n",
      "13\t62\t0.0\t1\t59 3\n",
      "14\t135\t0.0\t1\t122 13\n",
      "15\t110\t0.0\t1\t104 6\n",
      "16\t293\t0.0\t1\t283 10\n",
      "17\t509\t0.0\t1\t489 20\n",
      "18\t335\t0.0\t1\t302 27 6\n",
      "19\t1110\t0.0\t1\t666 438 6\n",
      "20\t2\t0.0\t2\t0 2\n",
      "21\t137\t0.0\t2\t0 136 1\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "66\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t2\n",
      "158\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D2.7.25.19_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D2.7.25.19_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D2.7.25.19_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D2.7.25.19_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.75 s (17 us/read; 3.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            405,906\n",
      "  Read 1 with adapter:                     310 (0.1%)\n",
      "  Read 2 with adapter:                  24,845 (6.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       405,906 (100.0%)\n",
      "\n",
      "Total basepairs processed:   184,827,985 bp\n",
      "  Read 1:    90,746,823 bp\n",
      "  Read 2:    94,081,162 bp\n",
      "Total written (filtered):    184,504,833 bp (99.8%)\n",
      "  Read 1:    90,743,720 bp\n",
      "  Read 2:    93,761,113 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 310 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t277\t6342.3\t0\t277\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t8\t0.0\t1\t7 1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t11\t0.0\t1\t10 1\n",
      "37\t2\t0.0\t1\t2\n",
      "38\t2\t0.0\t1\t2\n",
      "81\t1\t0.0\t1\t1\n",
      "163\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 24845 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t201\t6342.3\t0\t201\n",
      "4\t97\t1585.6\t0\t97\n",
      "5\t199\t396.4\t0\t199\n",
      "6\t398\t99.1\t0\t398\n",
      "7\t933\t24.8\t0\t933\n",
      "8\t1632\t6.2\t0\t1632\n",
      "9\t5120\t1.5\t0\t5119 1\n",
      "10\t4629\t0.4\t1\t4575 54\n",
      "11\t240\t0.1\t1\t232 8\n",
      "12\t290\t0.0\t1\t284 6\n",
      "13\t375\t0.0\t1\t363 12\n",
      "14\t647\t0.0\t1\t601 46\n",
      "15\t521\t0.0\t1\t493 28\n",
      "16\t1326\t0.0\t1\t1279 47\n",
      "17\t2098\t0.0\t1\t2023 75\n",
      "18\t1273\t0.0\t1\t1139 93 41\n",
      "19\t4224\t0.0\t1\t2468 1738 18\n",
      "20\t11\t0.0\t2\t0 9 2\n",
      "21\t571\t0.0\t2\t0 562 9\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "30\t2\t0.0\t2\t2\n",
      "35\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t0 1\n",
      "59\t1\t0.0\t2\t0 1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t0 1\n",
      "66\t1\t0.0\t2\t0 1\n",
      "70\t2\t0.0\t2\t2\n",
      "79\t1\t0.0\t2\t1\n",
      "102\t2\t0.0\t2\t2\n",
      "118\t1\t0.0\t2\t1\n",
      "133\t1\t0.0\t2\t1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "240\t3\t0.0\t2\t0 2 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D3.7.25.19_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D3.7.25.19_68_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D3.7.25.19_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D3.7.25.19_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.00 s (43 us/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,999\n",
      "  Read 1 with adapter:                      21 (0.1%)\n",
      "  Read 2 with adapter:                   1,872 (8.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,999 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,083,630 bp\n",
      "  Read 1:     4,921,104 bp\n",
      "  Read 2:     5,162,526 bp\n",
      "Total written (filtered):     10,057,853 bp (99.7%)\n",
      "  Read 1:     4,920,538 bp\n",
      "  Read 2:     5,137,315 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t359.4\t0\t16\n",
      "26\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1872 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t359.4\t0\t8\n",
      "4\t2\t89.8\t0\t2\n",
      "5\t11\t22.5\t0\t11\n",
      "6\t15\t5.6\t0\t15\n",
      "7\t46\t1.4\t0\t46\n",
      "8\t87\t0.4\t0\t87\n",
      "9\t327\t0.1\t0\t327\n",
      "10\t356\t0.0\t1\t354 2\n",
      "11\t27\t0.0\t1\t27\n",
      "12\t40\t0.0\t1\t39 1\n",
      "13\t70\t0.0\t1\t68 2\n",
      "14\t89\t0.0\t1\t86 3\n",
      "15\t52\t0.0\t1\t49 3\n",
      "16\t161\t0.0\t1\t152 9\n",
      "17\t150\t0.0\t1\t148 2\n",
      "18\t99\t0.0\t1\t81 10 8\n",
      "19\t293\t0.0\t1\t163 130\n",
      "21\t32\t0.0\t2\t0 31 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t1\n",
      "99\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t1\n",
      "130\t1\t0.0\t2\t1\n",
      "184\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D4.8.12.19_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.MCC.D4.8.12.19_49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D4.8.12.19_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.MCC.D4.8.12.19_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.18.A.control.6.25.19_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.18.A.control.6.25.19_61_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.18.A.control.6.25.19_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.18.A.control.6.25.19_61_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.18.D.control.6.25.19_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.18.D.control.6.25.19_60_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.18.D.control.6.25.19_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.18.D.control.6.25.19_60_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.7.B.control.6.25.19_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.7.B.control.6.25.19_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.7.B.control.6.25.19_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.7.B.control.6.25.19_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.7.F.control.6.25.19_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/M.cav.Tank.7.F.control.6.25.19_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.7.F.control.6.25.19_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/M.cav.Tank.7.F.control.6.25.19_35_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCA.7.23.19_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCA.7.23.19_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCA.7.23.19_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCA.7.23.19_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCA.7.25.19_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCA.7.25.19_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCA.7.25.19_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCA.7.25.19_11_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCB.7.23.19_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCB.7.23.19_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCB.7.23.19_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCB.7.23.19_18_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCB.7.25.19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCB.7.25.19_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCB.7.25.19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCB.7.25.19_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.7.23.19_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.7.23.19_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.7.23.19_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.7.23.19_2_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D1.7.25.19_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D1.7.25.19_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D1.7.25.19_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D1.7.25.19_17_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D2.7.25.19_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D2.7.25.19_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D2.7.25.19_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D2.7.25.19_14_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D3.7.25.19_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D3.7.25.19_68_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D3.7.25.19_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D3.7.25.19_68_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D4.7.25.19_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D4.7.25.19_66_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D4.7.25.19_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D4.7.25.19_66_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D4.7.25.19_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCC.D4.7.25.19_66_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D4.7.25.19_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCC.D4.7.25.19_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.59 s (23 us/read; 2.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,331\n",
      "  Read 1 with adapter:                      88 (0.1%)\n",
      "  Read 2 with adapter:                   4,092 (5.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        69,331 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,005,809 bp\n",
      "  Read 1:    15,183,028 bp\n",
      "  Read 2:    15,822,781 bp\n",
      "Total written (filtered):     30,945,526 bp (99.8%)\n",
      "  Read 1:    15,181,093 bp\n",
      "  Read 2:    15,764,433 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 88 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t1083.3\t0\t77\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "144\t1\t0.0\t1\t1\n",
      "182\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t4\t0.0\t1\t3 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4092 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1083.3\t0\t15\n",
      "4\t7\t270.8\t0\t7\n",
      "5\t13\t67.7\t0\t13\n",
      "6\t35\t16.9\t0\t35\n",
      "7\t94\t4.2\t0\t94\n",
      "8\t169\t1.1\t0\t169\n",
      "9\t580\t0.3\t0\t580\n",
      "10\t744\t0.1\t1\t733 11\n",
      "11\t55\t0.0\t1\t55\n",
      "12\t104\t0.0\t1\t99 5\n",
      "13\t145\t0.0\t1\t143 2\n",
      "14\t179\t0.0\t1\t174 5\n",
      "15\t106\t0.0\t1\t98 8\n",
      "16\t318\t0.0\t1\t307 11\n",
      "17\t381\t0.0\t1\t372 9\n",
      "18\t229\t0.0\t1\t205 17 7\n",
      "19\t808\t0.0\t1\t454 349 5\n",
      "21\t86\t0.0\t2\t0 82 4\n",
      "22\t5\t0.0\t2\t0 0 5\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t1\n",
      "115\t4\t0.0\t2\t4\n",
      "121\t1\t0.0\t2\t1\n",
      "152\t2\t0.0\t2\t1 1\n",
      "164\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "244\t1\t0.0\t2\t1\n",
      "246\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCD.7.23.19_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCD.7.23.19_67_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCD.7.23.19_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCD.7.23.19_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.44 s (32 us/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,538\n",
      "  Read 1 with adapter:                      63 (0.1%)\n",
      "  Read 2 with adapter:                   3,371 (7.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,538 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,649,148 bp\n",
      "  Read 1:     9,606,903 bp\n",
      "  Read 2:    10,042,245 bp\n",
      "Total written (filtered):     19,598,540 bp (99.7%)\n",
      "  Read 1:     9,603,657 bp\n",
      "  Read 2:     9,994,883 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 63 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t695.9\t0\t45\n",
      "12\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "218\t9\t0.0\t1\t9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3371 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t695.9\t0\t14\n",
      "4\t7\t174.0\t0\t7\n",
      "5\t11\t43.5\t0\t11\n",
      "6\t23\t10.9\t0\t23\n",
      "7\t85\t2.7\t0\t85\n",
      "8\t151\t0.7\t0\t151\n",
      "9\t481\t0.2\t0\t480 1\n",
      "10\t713\t0.0\t1\t707 6\n",
      "11\t61\t0.0\t1\t61\n",
      "12\t80\t0.0\t1\t79 1\n",
      "13\t107\t0.0\t1\t104 3\n",
      "14\t157\t0.0\t1\t145 12\n",
      "15\t112\t0.0\t1\t101 11\n",
      "16\t265\t0.0\t1\t254 11\n",
      "17\t319\t0.0\t1\t307 12\n",
      "18\t162\t0.0\t1\t131 19 12\n",
      "19\t521\t0.0\t1\t295 225 1\n",
      "20\t3\t0.0\t2\t1 1 1\n",
      "21\t74\t0.0\t2\t0 74\n",
      "30\t2\t0.0\t2\t2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t2\n",
      "76\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t4\t0.0\t2\t3 1\n",
      "123\t1\t0.0\t2\t1\n",
      "160\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "200\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCD.7.25.19_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCD.7.25.19_58_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCD.7.25.19_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCD.7.25.19_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.71 s (25 us/read; 2.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,013\n",
      "  Read 1 with adapter:                      53 (0.1%)\n",
      "  Read 2 with adapter:                   4,209 (6.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        68,013 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,584,134 bp\n",
      "  Read 1:    14,996,406 bp\n",
      "  Read 2:    15,587,728 bp\n",
      "Total written (filtered):     30,523,420 bp (99.8%)\n",
      "  Read 1:    14,995,131 bp\n",
      "  Read 2:    15,528,289 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 53 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t1062.7\t0\t46\n",
      "26\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4209 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t1062.7\t0\t36\n",
      "4\t9\t265.7\t0\t9\n",
      "5\t20\t66.4\t0\t20\n",
      "6\t36\t16.6\t0\t36\n",
      "7\t96\t4.2\t0\t96\n",
      "8\t167\t1.0\t0\t167\n",
      "9\t532\t0.3\t0\t532\n",
      "10\t740\t0.1\t1\t732 8\n",
      "11\t55\t0.0\t1\t54 1\n",
      "12\t78\t0.0\t1\t75 3\n",
      "13\t124\t0.0\t1\t122 2\n",
      "14\t177\t0.0\t1\t169 8\n",
      "15\t123\t0.0\t1\t118 5\n",
      "16\t339\t0.0\t1\t328 11\n",
      "17\t445\t0.0\t1\t426 19\n",
      "18\t265\t0.0\t1\t224 24 17\n",
      "19\t851\t0.0\t1\t481 369 1\n",
      "20\t5\t0.0\t2\t1 2 2\n",
      "21\t96\t0.0\t2\t0 95 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "52\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t3\n",
      "100\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "223\t1\t0.0\t2\t1\n",
      "246\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCE.7.23.19_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCE.7.23.19_57_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCE.7.23.19_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCE.7.23.19_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.03 s (24 us/read; 2.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,503\n",
      "  Read 1 with adapter:                     101 (0.1%)\n",
      "  Read 2 with adapter:                   5,906 (6.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        85,503 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,753,691 bp\n",
      "  Read 1:    18,997,799 bp\n",
      "  Read 2:    19,755,892 bp\n",
      "Total written (filtered):     38,665,863 bp (99.8%)\n",
      "  Read 1:    18,995,711 bp\n",
      "  Read 2:    19,670,152 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t1336.0\t0\t88\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t0 1\n",
      "28\t1\t0.0\t1\t0 1\n",
      "50\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "203\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "218\t3\t0.0\t1\t2 1\n",
      "221\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5906 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t1336.0\t0\t32\n",
      "4\t8\t334.0\t0\t8\n",
      "5\t20\t83.5\t0\t20\n",
      "6\t48\t20.9\t0\t48\n",
      "7\t125\t5.2\t0\t125\n",
      "8\t224\t1.3\t0\t224\n",
      "9\t716\t0.3\t0\t715 1\n",
      "10\t1196\t0.1\t1\t1181 15\n",
      "11\t97\t0.0\t1\t95 2\n",
      "12\t172\t0.0\t1\t165 7\n",
      "13\t249\t0.0\t1\t241 8\n",
      "14\t305\t0.0\t1\t291 14\n",
      "15\t213\t0.0\t1\t200 13\n",
      "16\t493\t0.0\t1\t484 9\n",
      "17\t568\t0.0\t1\t549 19\n",
      "18\t324\t0.0\t1\t286 25 13\n",
      "19\t935\t0.0\t1\t553 379 3\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t128\t0.0\t2\t0 127 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "30\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t2\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "76\t3\t0.0\t2\t3\n",
      "79\t4\t0.0\t2\t2 0 2\n",
      "80\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "111\t1\t0.0\t2\t1\n",
      "120\t1\t0.0\t2\t1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t1\n",
      "150\t3\t0.0\t2\t3\n",
      "151\t2\t0.0\t2\t2\n",
      "152\t2\t0.0\t2\t2\n",
      "153\t2\t0.0\t2\t2\n",
      "154\t1\t0.0\t2\t1\n",
      "155\t2\t0.0\t2\t2\n",
      "158\t1\t0.0\t2\t0 1\n",
      "162\t1\t0.0\t2\t1\n",
      "164\t1\t0.0\t2\t0 1\n",
      "165\t3\t0.0\t2\t3\n",
      "166\t4\t0.0\t2\t3 1\n",
      "167\t1\t0.0\t2\t1\n",
      "169\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "237\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCE.7.25.19_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCE.7.25.19_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCE.7.25.19_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCE.7.25.19_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.37 s (18 us/read; 3.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,571\n",
      "  Read 1 with adapter:                      86 (0.1%)\n",
      "  Read 2 with adapter:                   8,007 (6.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       130,571 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,168,001 bp\n",
      "  Read 1:    29,010,842 bp\n",
      "  Read 2:    30,157,159 bp\n",
      "Total written (filtered):     59,053,161 bp (99.8%)\n",
      "  Read 1:    29,008,541 bp\n",
      "  Read 2:    30,044,620 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 86 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t71\t2040.2\t0\t71\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "195\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t5\t0.0\t1\t4 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8007 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t2040.2\t0\t34\n",
      "4\t12\t510.0\t0\t12\n",
      "5\t21\t127.5\t0\t21\n",
      "6\t65\t31.9\t0\t65\n",
      "7\t145\t8.0\t0\t145\n",
      "8\t309\t2.0\t0\t309\n",
      "9\t1127\t0.5\t0\t1127\n",
      "10\t1369\t0.1\t1\t1356 13\n",
      "11\t140\t0.0\t1\t136 4\n",
      "12\t203\t0.0\t1\t196 7\n",
      "13\t291\t0.0\t1\t272 19\n",
      "14\t435\t0.0\t1\t412 23\n",
      "15\t239\t0.0\t1\t230 9\n",
      "16\t694\t0.0\t1\t670 24\n",
      "17\t789\t0.0\t1\t763 26\n",
      "18\t471\t0.0\t1\t415 34 22\n",
      "19\t1452\t0.0\t1\t827 620 5\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t176\t0.0\t2\t0 174 2\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t1 2\n",
      "85\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t0 1\n",
      "113\t1\t0.0\t2\t1\n",
      "121\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t0 1\n",
      "167\t1\t0.0\t2\t0 1\n",
      "173\t1\t0.0\t2\t1\n",
      "176\t1\t0.0\t2\t1\n",
      "211\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCF.7.23.19_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCF.7.23.19_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCF.7.23.19_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCF.7.23.19_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.92 s (16 us/read; 3.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            433,329\n",
      "  Read 1 with adapter:                     203 (0.0%)\n",
      "  Read 2 with adapter:                  25,197 (5.8%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       433,328 (100.0%)\n",
      "\n",
      "Total basepairs processed:   197,223,863 bp\n",
      "  Read 1:    96,889,990 bp\n",
      "  Read 2:   100,333,873 bp\n",
      "Total written (filtered):    196,877,050 bp (99.8%)\n",
      "  Read 1:    96,887,782 bp\n",
      "  Read 2:    99,989,268 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 203 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t187\t6770.8\t0\t187\n",
      "24\t2\t0.0\t1\t2\n",
      "26\t2\t0.0\t1\t2\n",
      "29\t5\t0.0\t1\t4 1\n",
      "38\t1\t0.0\t1\t1\n",
      "143\t1\t0.0\t1\t1\n",
      "160\t1\t0.0\t1\t1\n",
      "186\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t2 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 25197 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t202\t6770.8\t0\t202\n",
      "4\t66\t1692.7\t0\t66\n",
      "5\t139\t423.2\t0\t139\n",
      "6\t284\t105.8\t0\t284\n",
      "7\t596\t26.4\t0\t596\n",
      "8\t1097\t6.6\t0\t1097\n",
      "9\t3733\t1.7\t0\t3726 7\n",
      "10\t5085\t0.4\t1\t5010 75\n",
      "11\t322\t0.1\t1\t312 10\n",
      "12\t419\t0.0\t1\t414 5\n",
      "13\t550\t0.0\t1\t525 25\n",
      "14\t850\t0.0\t1\t798 52\n",
      "15\t666\t0.0\t1\t628 38\n",
      "16\t1880\t0.0\t1\t1804 76\n",
      "17\t2411\t0.0\t1\t2321 90\n",
      "18\t1432\t0.0\t1\t1246 121 65\n",
      "19\t4746\t0.0\t1\t2780 1944 22\n",
      "20\t10\t0.0\t2\t0 10\n",
      "21\t621\t0.0\t2\t0 607 14\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "30\t1\t0.0\t2\t1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "70\t2\t0.0\t2\t0 1 1\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "78\t3\t0.0\t2\t3\n",
      "79\t5\t0.0\t2\t4 0 1\n",
      "88\t2\t0.0\t2\t2\n",
      "89\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "93\t2\t0.0\t2\t1 0 1\n",
      "100\t1\t0.0\t2\t1\n",
      "102\t1\t0.0\t2\t1\n",
      "107\t1\t0.0\t2\t0 1\n",
      "158\t1\t0.0\t2\t1\n",
      "159\t1\t0.0\t2\t1\n",
      "162\t1\t0.0\t2\t0 1\n",
      "165\t2\t0.0\t2\t1 0 1\n",
      "167\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t2\t0.0\t2\t1 1\n",
      "220\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "230\t1\t0.0\t2\t0 0 1\n",
      "239\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCF.7.25.19_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCF.7.25.19_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCF.7.25.19_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCF.7.25.19_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.17 s (16 us/read; 3.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            324,297\n",
      "  Read 1 with adapter:                     458 (0.1%)\n",
      "  Read 2 with adapter:                  20,555 (6.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       324,297 (100.0%)\n",
      "\n",
      "Total basepairs processed:   147,486,373 bp\n",
      "  Read 1:    72,365,217 bp\n",
      "  Read 2:    75,121,156 bp\n",
      "Total written (filtered):    147,203,475 bp (99.8%)\n",
      "  Read 1:    72,360,586 bp\n",
      "  Read 2:    74,842,889 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 458 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t424\t5067.1\t0\t424\n",
      "13\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t4\t0.0\t1\t4\n",
      "29\t7\t0.0\t1\t6 1\n",
      "30\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "76\t1\t0.0\t1\t1\n",
      "81\t1\t0.0\t1\t1\n",
      "109\t1\t0.0\t1\t1\n",
      "153\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "218\t6\t0.0\t1\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 20555 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t129\t5067.1\t0\t129\n",
      "4\t31\t1266.8\t0\t31\n",
      "5\t63\t316.7\t0\t63\n",
      "6\t196\t79.2\t0\t196\n",
      "7\t487\t19.8\t0\t487\n",
      "8\t906\t4.9\t0\t906\n",
      "9\t2979\t1.2\t0\t2967 12\n",
      "10\t4197\t0.3\t1\t4139 58\n",
      "11\t359\t0.1\t1\t349 10\n",
      "12\t493\t0.0\t1\t477 16\n",
      "13\t650\t0.0\t1\t627 23\n",
      "14\t902\t0.0\t1\t853 49\n",
      "15\t580\t0.0\t1\t546 34\n",
      "16\t1639\t0.0\t1\t1578 61\n",
      "17\t1817\t0.0\t1\t1736 81\n",
      "18\t1115\t0.0\t1\t985 80 50\n",
      "19\t3526\t0.0\t1\t1992 1521 13\n",
      "20\t5\t0.0\t2\t0 4 1\n",
      "21\t424\t0.0\t2\t0 413 11\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "23\t1\t0.0\t2\t1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t1\n",
      "57\t2\t0.0\t2\t1 0 1\n",
      "58\t2\t0.0\t2\t1 0 1\n",
      "59\t2\t0.0\t2\t0 1 1\n",
      "64\t1\t0.0\t2\t1\n",
      "67\t2\t0.0\t2\t2\n",
      "70\t1\t0.0\t2\t1\n",
      "72\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "82\t1\t0.0\t2\t1\n",
      "85\t1\t0.0\t2\t1\n",
      "88\t2\t0.0\t2\t1 1\n",
      "106\t1\t0.0\t2\t1\n",
      "115\t2\t0.0\t2\t1 0 1\n",
      "157\t1\t0.0\t2\t1\n",
      "176\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "224\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCG.7.23.19_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCG.7.23.19_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCG.7.23.19_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCG.7.23.19_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 9.50 s (16 us/read; 3.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            584,847\n",
      "  Read 1 with adapter:                     199 (0.0%)\n",
      "  Read 2 with adapter:                  34,260 (5.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       584,847 (100.0%)\n",
      "\n",
      "Total basepairs processed:   266,219,033 bp\n",
      "  Read 1:   130,824,469 bp\n",
      "  Read 2:   135,394,564 bp\n",
      "Total written (filtered):    265,791,786 bp (99.8%)\n",
      "  Read 1:   130,822,022 bp\n",
      "  Read 2:   134,969,764 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 199 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t172\t9138.2\t0\t172\n",
      "4\t1\t2284.6\t0\t1\n",
      "11\t1\t0.1\t1\t1\n",
      "23\t1\t0.0\t1\t0 1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t10\t0.0\t1\t10\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t5\t0.0\t1\t5\n",
      "130\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "218\t5\t0.0\t1\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 34260 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t344\t9138.2\t0\t344\n",
      "4\t174\t2284.6\t0\t174\n",
      "5\t319\t571.1\t0\t319\n",
      "6\t609\t142.8\t0\t609\n",
      "7\t1472\t35.7\t0\t1472\n",
      "8\t2785\t8.9\t0\t2785\n",
      "9\t8389\t2.2\t0\t8362 27\n",
      "10\t6657\t0.6\t1\t6562 95\n",
      "11\t216\t0.1\t1\t205 11\n",
      "12\t169\t0.0\t1\t162 7\n",
      "13\t187\t0.0\t1\t172 15\n",
      "14\t465\t0.0\t1\t427 38\n",
      "15\t438\t0.0\t1\t424 14\n",
      "16\t1314\t0.0\t1\t1267 47\n",
      "17\t2553\t0.0\t1\t2473 80\n",
      "18\t1626\t0.0\t1\t1457 129 40\n",
      "19\t5641\t0.0\t1\t3403 2213 25\n",
      "20\t8\t0.0\t2\t0 2 6\n",
      "21\t816\t0.0\t2\t0 805 11\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "30\t2\t0.0\t2\t2\n",
      "35\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t4\t0.0\t2\t2 2\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t1 1 1\n",
      "96\t1\t0.0\t2\t0 1\n",
      "100\t2\t0.0\t2\t2\n",
      "150\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t4\t0.0\t2\t3 1\n",
      "218\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCG.7.25.19_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCG.7.25.19_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCG.7.25.19_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCG.7.25.19_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.30 s (17 us/read; 3.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            438,135\n",
      "  Read 1 with adapter:                     225 (0.1%)\n",
      "  Read 2 with adapter:                  23,728 (5.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       438,135 (100.0%)\n",
      "\n",
      "Total basepairs processed:   199,405,360 bp\n",
      "  Read 1:    97,960,216 bp\n",
      "  Read 2:   101,445,144 bp\n",
      "Total written (filtered):    199,077,484 bp (99.8%)\n",
      "  Read 1:    97,958,399 bp\n",
      "  Read 2:   101,119,085 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 225 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t208\t6845.9\t0\t208\n",
      "15\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t3\t0.0\t1\t3\n",
      "50\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t1\n",
      "81\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t1\n",
      "156\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 23728 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t216\t6845.9\t0\t216\n",
      "4\t78\t1711.5\t0\t78\n",
      "5\t120\t427.9\t0\t120\n",
      "6\t264\t107.0\t0\t264\n",
      "7\t559\t26.7\t0\t559\n",
      "8\t1050\t6.7\t0\t1050\n",
      "9\t3579\t1.7\t0\t3571 8\n",
      "10\t4679\t0.4\t1\t4618 61\n",
      "11\t285\t0.1\t1\t279 6\n",
      "12\t333\t0.0\t1\t316 17\n",
      "13\t498\t0.0\t1\t479 19\n",
      "14\t814\t0.0\t1\t767 47\n",
      "15\t606\t0.0\t1\t576 30\n",
      "16\t1657\t0.0\t1\t1581 76\n",
      "17\t2309\t0.0\t1\t2211 98\n",
      "18\t1376\t0.0\t1\t1200 112 64\n",
      "19\t4603\t0.0\t1\t2705 1870 28\n",
      "20\t10\t0.0\t2\t0 3 7\n",
      "21\t603\t0.0\t2\t0 595 8\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "30\t2\t0.0\t2\t2\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "47\t2\t0.0\t2\t2\n",
      "51\t2\t0.0\t2\t1 0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "59\t2\t0.0\t2\t2\n",
      "62\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "68\t2\t0.0\t2\t0 0 2\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t7\t0.0\t2\t6 1\n",
      "85\t1\t0.0\t2\t0 1\n",
      "100\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t1\n",
      "129\t1\t0.0\t2\t1\n",
      "132\t1\t0.0\t2\t1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t1\n",
      "169\t1\t0.0\t2\t0 1\n",
      "171\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t1\n",
      "195\t1\t0.0\t2\t1\n",
      "212\t1\t0.0\t2\t1\n",
      "213\t2\t0.0\t2\t1 0 1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "217\t2\t0.0\t2\t2\n",
      "218\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "247\t1\t0.0\t2\t0 1\n",
      "248\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.1.D.5percent.9.8.19_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.1.D.5percent.9.8.19_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.1.D.5percent.9.8.19_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.1.D.5percent.9.8.19_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.62 s (14 us/read; 4.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            253,343\n",
      "  Read 1 with adapter:                     258 (0.1%)\n",
      "  Read 2 with adapter:                  16,181 (6.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       253,343 (100.0%)\n",
      "\n",
      "Total basepairs processed:   115,591,680 bp\n",
      "  Read 1:    56,719,037 bp\n",
      "  Read 2:    58,872,643 bp\n",
      "Total written (filtered):    115,392,640 bp (99.8%)\n",
      "  Read 1:    56,718,093 bp\n",
      "  Read 2:    58,674,547 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 258 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t251\t3958.5\t0\t251\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 16181 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t151\t3958.5\t0\t151\n",
      "4\t68\t989.6\t0\t68\n",
      "5\t107\t247.4\t0\t107\n",
      "6\t254\t61.9\t0\t254\n",
      "7\t815\t15.5\t0\t815\n",
      "8\t1431\t3.9\t0\t1431\n",
      "9\t4182\t1.0\t0\t4174 8\n",
      "10\t3168\t0.2\t1\t3132 36\n",
      "11\t75\t0.1\t1\t71 4\n",
      "12\t86\t0.0\t1\t85 1\n",
      "13\t134\t0.0\t1\t129 5\n",
      "14\t207\t0.0\t1\t190 17\n",
      "15\t212\t0.0\t1\t199 13\n",
      "16\t562\t0.0\t1\t540 22\n",
      "17\t1146\t0.0\t1\t1108 38\n",
      "18\t713\t0.0\t1\t643 51 19\n",
      "19\t2499\t0.0\t1\t1481 1008 10\n",
      "20\t8\t0.0\t2\t3 4 1\n",
      "21\t317\t0.0\t2\t0 312 5\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "30\t2\t0.0\t2\t2\n",
      "34\t1\t0.0\t2\t1\n",
      "35\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "70\t3\t0.0\t2\t3\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "78\t2\t0.0\t2\t2\n",
      "79\t1\t0.0\t2\t1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t1\n",
      "185\t2\t0.0\t2\t2\n",
      "211\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t0 1\n",
      "224\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.10.D.5percent.9.10.19_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.10.D.5percent.9.10.19_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.10.D.5percent.9.10.19_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.10.D.5percent.9.10.19_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.93 s (15 us/read; 3.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            261,559\n",
      "  Read 1 with adapter:                  11,040 (4.2%)\n",
      "  Read 2 with adapter:                  16,563 (6.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       261,559 (100.0%)\n",
      "\n",
      "Total basepairs processed:   119,196,974 bp\n",
      "  Read 1:    58,465,957 bp\n",
      "  Read 2:    60,731,017 bp\n",
      "Total written (filtered):    118,650,182 bp (99.5%)\n",
      "  Read 1:    58,134,278 bp\n",
      "  Read 2:    60,515,904 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11040 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t257\t4086.9\t0\t257\n",
      "9\t2\t1.0\t0\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "23\t2\t0.0\t1\t2\n",
      "25\t5\t0.0\t1\t5\n",
      "26\t7\t0.0\t1\t5 2\n",
      "27\t43\t0.0\t1\t32 11\n",
      "28\t298\t0.0\t1\t162 136\n",
      "29\t10117\t0.0\t1\t9907 210\n",
      "30\t23\t0.0\t1\t18 5\n",
      "31\t2\t0.0\t1\t1 1\n",
      "32\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "42\t3\t0.0\t1\t3\n",
      "46\t1\t0.0\t1\t1\n",
      "56\t3\t0.0\t1\t3\n",
      "57\t16\t0.0\t1\t12 4\n",
      "58\t65\t0.0\t1\t56 9\n",
      "59\t2\t0.0\t1\t1 1\n",
      "61\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t1\n",
      "84\t1\t0.0\t1\t1\n",
      "87\t4\t0.0\t1\t3 1\n",
      "88\t8\t0.0\t1\t4 4\n",
      "89\t56\t0.0\t1\t52 4\n",
      "91\t6\t0.0\t1\t3 3\n",
      "92\t50\t0.0\t1\t47 3\n",
      "94\t1\t0.0\t1\t1\n",
      "145\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "154\t4\t0.0\t1\t2 2\n",
      "155\t49\t0.0\t1\t46 3\n",
      "167\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t3\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 16563 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t114\t4086.9\t0\t114\n",
      "4\t45\t1021.7\t0\t45\n",
      "5\t74\t255.4\t0\t74\n",
      "6\t183\t63.9\t0\t183\n",
      "7\t558\t16.0\t0\t558\n",
      "8\t950\t4.0\t0\t950\n",
      "9\t3125\t1.0\t0\t3116 9\n",
      "10\t3523\t0.2\t1\t3483 40\n",
      "11\t185\t0.1\t1\t179 6\n",
      "12\t290\t0.0\t1\t288 2\n",
      "13\t387\t0.0\t1\t374 13\n",
      "14\t552\t0.0\t1\t529 23\n",
      "15\t414\t0.0\t1\t387 27\n",
      "16\t966\t0.0\t1\t931 35\n",
      "17\t1348\t0.0\t1\t1306 42\n",
      "18\t774\t0.0\t1\t691 54 29\n",
      "19\t2689\t0.0\t1\t1545 1133 11\n",
      "20\t7\t0.0\t2\t2 5\n",
      "21\t340\t0.0\t2\t0 337 3\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "62\t3\t0.0\t2\t3\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t1 1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t1 0 1\n",
      "71\t2\t0.0\t2\t2\n",
      "73\t1\t0.0\t2\t1\n",
      "79\t8\t0.0\t2\t4 4\n",
      "92\t1\t0.0\t2\t1\n",
      "104\t1\t0.0\t2\t1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "169\t1\t0.0\t2\t1\n",
      "171\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t1\n",
      "183\t1\t0.0\t2\t1\n",
      "231\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.10.E.5percent.9.9.19_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.10.E.5percent.9.9.19_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.10.E.5percent.9.9.19_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.10.E.5percent.9.9.19_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.65 s (14 us/read; 4.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            483,713\n",
      "  Read 1 with adapter:                     531 (0.1%)\n",
      "  Read 2 with adapter:                  36,306 (7.5%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       483,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:   220,804,914 bp\n",
      "  Read 1:   108,265,603 bp\n",
      "  Read 2:   112,539,311 bp\n",
      "Total written (filtered):    220,340,627 bp (99.8%)\n",
      "  Read 1:   108,262,237 bp\n",
      "  Read 2:   112,078,390 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 531 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t508\t7558.0\t0\t508\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t8\t0.0\t1\t8\n",
      "71\t1\t0.0\t1\t1\n",
      "78\t1\t0.0\t1\t0 1\n",
      "89\t1\t0.0\t1\t1\n",
      "101\t2\t0.0\t1\t2\n",
      "106\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t1\n",
      "172\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 36306 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t242\t7558.0\t0\t242\n",
      "4\t96\t1889.5\t0\t96\n",
      "5\t159\t472.4\t0\t159\n",
      "6\t396\t118.1\t0\t396\n",
      "7\t1199\t29.5\t0\t1199\n",
      "8\t2154\t7.4\t0\t2154\n",
      "9\t6890\t1.8\t0\t6871 19\n",
      "10\t8985\t0.5\t1\t8877 108\n",
      "11\t456\t0.1\t1\t445 11\n",
      "12\t580\t0.0\t1\t562 18\n",
      "13\t821\t0.0\t1\t786 35\n",
      "14\t1232\t0.0\t1\t1157 75\n",
      "15\t956\t0.0\t1\t899 57\n",
      "16\t2215\t0.0\t1\t2112 103\n",
      "17\t2535\t0.0\t1\t2434 101\n",
      "18\t1607\t0.0\t1\t1406 117 84\n",
      "19\t5035\t0.0\t1\t2858 2146 31\n",
      "20\t9\t0.0\t2\t0 8 1\n",
      "21\t621\t0.0\t2\t0 605 16\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "30\t1\t0.0\t2\t1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t2\n",
      "56\t2\t0.0\t2\t2\n",
      "60\t1\t0.0\t2\t1\n",
      "65\t3\t0.0\t2\t3\n",
      "66\t4\t0.0\t2\t3 0 1\n",
      "67\t2\t0.0\t2\t1 1\n",
      "68\t3\t0.0\t2\t3\n",
      "69\t4\t0.0\t2\t3 1\n",
      "70\t7\t0.0\t2\t5 1 1\n",
      "71\t4\t0.0\t2\t0 0 4\n",
      "73\t1\t0.0\t2\t1\n",
      "74\t2\t0.0\t2\t2\n",
      "75\t1\t0.0\t2\t1\n",
      "76\t2\t0.0\t2\t2\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t2\n",
      "79\t12\t0.0\t2\t11 1\n",
      "80\t1\t0.0\t2\t1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t1\n",
      "84\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "99\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 1\n",
      "167\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "208\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t2\t0.0\t2\t2\n",
      "224\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t0 1\n",
      "242\t1\t0.0\t2\t0 1\n",
      "248\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.11.D.5percent.8.10.19_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.11.D.5percent.8.10.19_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.11.D.5percent.8.10.19_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.11.D.5percent.8.10.19_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.30 s (15 us/read; 4.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            365,178\n",
      "  Read 1 with adapter:                     457 (0.1%)\n",
      "  Read 2 with adapter:                  36,565 (10.0%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       365,177 (100.0%)\n",
      "\n",
      "Total basepairs processed:   166,929,810 bp\n",
      "  Read 1:    81,752,660 bp\n",
      "  Read 2:    85,177,150 bp\n",
      "Total written (filtered):    166,487,080 bp (99.7%)\n",
      "  Read 1:    81,749,486 bp\n",
      "  Read 2:    84,737,594 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 457 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t435\t5705.9\t0\t435\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t7\t0.0\t1\t7\n",
      "89\t2\t0.0\t1\t2\n",
      "140\t1\t0.0\t1\t1\n",
      "170\t1\t0.0\t1\t1\n",
      "175\t1\t0.0\t1\t0 1\n",
      "192\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 36565 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t341\t5705.9\t0\t341\n",
      "4\t116\t1426.5\t0\t116\n",
      "5\t230\t356.6\t0\t230\n",
      "6\t493\t89.2\t0\t493\n",
      "7\t1685\t22.3\t0\t1685\n",
      "8\t2708\t5.6\t0\t2708\n",
      "9\t8315\t1.4\t0\t8297 18\n",
      "10\t9394\t0.3\t1\t9284 110\n",
      "11\t422\t0.1\t1\t412 10\n",
      "12\t505\t0.0\t1\t484 21\n",
      "13\t767\t0.0\t1\t736 31\n",
      "14\t1110\t0.0\t1\t1063 47\n",
      "15\t711\t0.0\t1\t666 45\n",
      "16\t1883\t0.0\t1\t1818 65\n",
      "17\t2178\t0.0\t1\t2059 119\n",
      "18\t1246\t0.0\t1\t1086 93 67\n",
      "19\t3853\t0.0\t1\t2143 1692 18\n",
      "20\t6\t0.0\t2\t0 6\n",
      "21\t482\t0.0\t2\t0 465 17\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "30\t1\t0.0\t2\t1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t1\n",
      "50\t2\t0.0\t2\t1 1\n",
      "51\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "55\t2\t0.0\t2\t1 0 1\n",
      "56\t1\t0.0\t2\t1\n",
      "58\t2\t0.0\t2\t1 0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t3\t0.0\t2\t3\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t2\t0.0\t2\t2\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t4\t0.0\t2\t2 2\n",
      "71\t5\t0.0\t2\t0 0 5\n",
      "72\t2\t0.0\t2\t2\n",
      "74\t2\t0.0\t2\t2\n",
      "75\t3\t0.0\t2\t2 0 1\n",
      "76\t2\t0.0\t2\t2\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t16\t0.0\t2\t10 5 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t1\n",
      "88\t2\t0.0\t2\t2\n",
      "91\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t1\n",
      "147\t1\t0.0\t2\t1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "170\t1\t0.0\t2\t0 1\n",
      "171\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t1\n",
      "191\t1\t0.0\t2\t1\n",
      "192\t1\t0.0\t2\t1\n",
      "194\t1\t0.0\t2\t1\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t0 1\n",
      "232\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "238\t3\t0.0\t2\t2 0 1\n",
      "239\t2\t0.0\t2\t2\n",
      "240\t1\t0.0\t2\t1\n",
      "241\t1\t0.0\t2\t1\n",
      "242\t4\t0.0\t2\t3 1\n",
      "247\t1\t0.0\t2\t1\n",
      "251\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.12.D.5percent.9.11.19_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.12.D.5percent.9.11.19_50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.12.D.5percent.9.11.19_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.12.D.5percent.9.11.19_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.32 s (15 us/read; 4.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            363,600\n",
      "  Read 1 with adapter:                     366 (0.1%)\n",
      "  Read 2 with adapter:                  16,446 (4.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       363,600 (100.0%)\n",
      "\n",
      "Total basepairs processed:   165,784,103 bp\n",
      "  Read 1:    81,410,561 bp\n",
      "  Read 2:    84,373,542 bp\n",
      "Total written (filtered):    165,557,852 bp (99.9%)\n",
      "  Read 1:    81,408,670 bp\n",
      "  Read 2:    84,149,182 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 366 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t352\t5681.2\t0\t352\n",
      "18\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t5\t0.0\t1\t5\n",
      "61\t1\t0.0\t1\t0 1\n",
      "141\t1\t0.0\t1\t1\n",
      "151\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 16446 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t103\t5681.2\t0\t103\n",
      "4\t63\t1420.3\t0\t63\n",
      "5\t79\t355.1\t0\t79\n",
      "6\t168\t88.8\t0\t168\n",
      "7\t480\t22.2\t0\t480\n",
      "8\t779\t5.5\t0\t779\n",
      "9\t2632\t1.4\t0\t2628 4\n",
      "10\t3500\t0.3\t1\t3462 38\n",
      "11\t157\t0.1\t1\t154 3\n",
      "12\t209\t0.0\t1\t202 7\n",
      "13\t221\t0.0\t1\t215 6\n",
      "14\t435\t0.0\t1\t400 35\n",
      "15\t331\t0.0\t1\t312 19\n",
      "16\t982\t0.0\t1\t940 42\n",
      "17\t1508\t0.0\t1\t1459 49\n",
      "18\t987\t0.0\t1\t880 79 28\n",
      "19\t3239\t0.0\t1\t1997 1227 15\n",
      "20\t6\t0.0\t2\t1 4 1\n",
      "21\t501\t0.0\t2\t0 493 8\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "27\t1\t0.0\t2\t0 1\n",
      "30\t4\t0.0\t2\t4\n",
      "54\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "70\t4\t0.0\t2\t4\n",
      "71\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t0 1\n",
      "78\t3\t0.0\t2\t2 1\n",
      "79\t5\t0.0\t2\t4 1\n",
      "100\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t1\n",
      "160\t1\t0.0\t2\t1\n",
      "186\t3\t0.0\t2\t2 0 1\n",
      "211\t1\t0.0\t2\t1\n",
      "213\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "218\t2\t0.0\t2\t2\n",
      "222\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCD.7.23.19_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCD.7.23.19_67_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCD.7.23.19_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCD.7.23.19_67_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCD.7.25.19_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCD.7.25.19_58_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCD.7.25.19_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCD.7.25.19_58_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCE.7.23.19_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCE.7.23.19_57_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCE.7.23.19_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCE.7.23.19_57_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCE.7.25.19_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCE.7.25.19_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCE.7.25.19_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCE.7.25.19_26_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCF.7.23.19_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCF.7.23.19_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCF.7.23.19_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCF.7.23.19_6_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCF.7.25.19_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCF.7.25.19_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCF.7.25.19_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCF.7.25.19_33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCG.7.23.19_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCG.7.23.19_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCG.7.23.19_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCG.7.23.19_29_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCG.7.25.19_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.HCG.7.25.19_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCG.7.25.19_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.HCG.7.25.19_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.1.D.5percent.9.8.19_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.1.D.5percent.9.8.19_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.1.D.5percent.9.8.19_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.1.D.5percent.9.8.19_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.10.D.5percent.9.10.19_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.10.D.5percent.9.10.19_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.10.D.5percent.9.10.19_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.10.D.5percent.9.10.19_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.10.E.5percent.9.9.19_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.10.E.5percent.9.9.19_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.10.E.5percent.9.9.19_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.10.E.5percent.9.9.19_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.11.D.5percent.8.10.19_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.11.D.5percent.8.10.19_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.11.D.5percent.8.10.19_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.11.D.5percent.8.10.19_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.12.D.5percent.9.11.19_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.12.D.5percent.9.11.19_50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.12.D.5percent.9.11.19_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.12.D.5percent.9.11.19_50_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.14.B.5percent.8.15.19_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.14.B.5percent.8.15.19_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.14.B.5percent.8.15.19_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.14.B.5percent.8.15.19_4_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.14.B.5percent.8.15.19_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.14.B.5percent.8.15.19_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.14.B.5percent.8.15.19_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.14.B.5percent.8.15.19_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.77 s (15 us/read; 3.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            245,796\n",
      "  Read 1 with adapter:                     243 (0.1%)\n",
      "  Read 2 with adapter:                  25,592 (10.4%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       245,795 (100.0%)\n",
      "\n",
      "Total basepairs processed:   112,376,888 bp\n",
      "  Read 1:    55,027,603 bp\n",
      "  Read 2:    57,349,285 bp\n",
      "Total written (filtered):    112,068,786 bp (99.7%)\n",
      "  Read 1:    55,025,786 bp\n",
      "  Read 2:    57,043,000 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 243 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t227\t3840.6\t0\t227\n",
      "9\t1\t0.9\t0\t1\n",
      "19\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t5\t0.0\t1\t5\n",
      "29\t4\t0.0\t1\t4\n",
      "92\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "195\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 25592 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t192\t3840.6\t0\t192\n",
      "4\t90\t960.1\t0\t90\n",
      "5\t172\t240.0\t0\t172\n",
      "6\t354\t60.0\t0\t354\n",
      "7\t1479\t15.0\t0\t1479\n",
      "8\t1992\t3.8\t0\t1992\n",
      "9\t6073\t0.9\t0\t6058 15\n",
      "10\t6293\t0.2\t1\t6229 64\n",
      "11\t273\t0.1\t1\t266 7\n",
      "12\t314\t0.0\t1\t302 12\n",
      "13\t536\t0.0\t1\t516 20\n",
      "14\t787\t0.0\t1\t762 25\n",
      "15\t507\t0.0\t1\t475 32\n",
      "16\t1272\t0.0\t1\t1227 45\n",
      "17\t1398\t0.0\t1\t1335 63\n",
      "18\t828\t0.0\t1\t708 66 54\n",
      "19\t2622\t0.0\t1\t1412 1197 13\n",
      "20\t7\t0.0\t2\t0 6 1\n",
      "21\t307\t0.0\t2\t0 303 4\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t1\n",
      "54\t3\t0.0\t2\t2 1\n",
      "55\t1\t0.0\t2\t1\n",
      "57\t3\t0.0\t2\t3\n",
      "60\t3\t0.0\t2\t0 2 1\n",
      "61\t1\t0.0\t2\t1\n",
      "64\t3\t0.0\t2\t3\n",
      "65\t2\t0.0\t2\t2\n",
      "66\t2\t0.0\t2\t1 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t1 0 1\n",
      "71\t2\t0.0\t2\t1 0 1\n",
      "76\t2\t0.0\t2\t2\n",
      "77\t2\t0.0\t2\t2\n",
      "79\t12\t0.0\t2\t10 1 1\n",
      "85\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t1\n",
      "124\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "187\t1\t0.0\t2\t0 0 1\n",
      "192\t1\t0.0\t2\t1\n",
      "193\t1\t0.0\t2\t1\n",
      "207\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t0 1\n",
      "238\t4\t0.0\t2\t1 2 1\n",
      "239\t1\t0.0\t2\t1\n",
      "240\t1\t0.0\t2\t1\n",
      "241\t3\t0.0\t2\t1 1 1\n",
      "242\t4\t0.0\t2\t2 2\n",
      "243\t2\t0.0\t2\t1 1\n",
      "246\t1\t0.0\t2\t0 1\n",
      "247\t2\t0.0\t2\t0 1 1\n",
      "249\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.17.D.5percent.7.29.19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.17.D.5percent.7.29.19_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.17.D.5percent.7.29.19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.17.D.5percent.7.29.19_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.03 s (15 us/read; 4.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            274,902\n",
      "  Read 1 with adapter:                     418 (0.2%)\n",
      "  Read 2 with adapter:                  29,053 (10.6%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       274,901 (100.0%)\n",
      "\n",
      "Total basepairs processed:   125,663,748 bp\n",
      "  Read 1:    61,527,941 bp\n",
      "  Read 2:    64,135,807 bp\n",
      "Total written (filtered):    125,318,203 bp (99.7%)\n",
      "  Read 1:    61,525,196 bp\n",
      "  Read 2:    63,793,007 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 418 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t406\t4295.3\t0\t406\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "64\t1\t0.0\t1\t1\n",
      "156\t2\t0.0\t1\t2\n",
      "158\t1\t0.0\t1\t1\n",
      "194\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 29053 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t239\t4295.3\t0\t239\n",
      "4\t82\t1073.8\t0\t82\n",
      "5\t157\t268.5\t0\t157\n",
      "6\t387\t67.1\t0\t387\n",
      "7\t1300\t16.8\t0\t1300\n",
      "8\t2130\t4.2\t0\t2130\n",
      "9\t6587\t1.0\t0\t6577 10\n",
      "10\t8285\t0.3\t1\t8190 95\n",
      "11\t300\t0.1\t1\t295 5\n",
      "12\t340\t0.0\t1\t327 13\n",
      "13\t568\t0.0\t1\t540 28\n",
      "14\t854\t0.0\t1\t815 39\n",
      "15\t563\t0.0\t1\t528 35\n",
      "16\t1367\t0.0\t1\t1317 50\n",
      "17\t1593\t0.0\t1\t1524 69\n",
      "18\t939\t0.0\t1\t802 72 65\n",
      "19\t2912\t0.0\t1\t1584 1314 14\n",
      "20\t8\t0.0\t2\t1 5 2\n",
      "21\t355\t0.0\t2\t0 354 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "55\t2\t0.0\t2\t1 1\n",
      "59\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t5\t0.0\t2\t4 1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t3\t0.0\t2\t2 0 1\n",
      "68\t2\t0.0\t2\t1 0 1\n",
      "69\t2\t0.0\t2\t1 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t2\n",
      "74\t2\t0.0\t2\t1 0 1\n",
      "76\t1\t0.0\t2\t0 1\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t2\n",
      "79\t5\t0.0\t2\t5\n",
      "80\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t0 1\n",
      "88\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t1\n",
      "125\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t1\n",
      "162\t1\t0.0\t2\t0 1\n",
      "172\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "230\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t1\n",
      "242\t3\t0.0\t2\t1 1 1\n",
      "245\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.17.G.5percent.9.4.19_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.17.G.5percent.9.4.19_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.17.G.5percent.9.4.19_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.17.G.5percent.9.4.19_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 8.27 s (15 us/read; 4.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            565,645\n",
      "  Read 1 with adapter:                     359 (0.1%)\n",
      "  Read 2 with adapter:                  36,895 (6.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       565,645 (100.0%)\n",
      "\n",
      "Total basepairs processed:   257,941,698 bp\n",
      "  Read 1:   126,638,734 bp\n",
      "  Read 2:   131,302,964 bp\n",
      "Total written (filtered):    257,456,161 bp (99.8%)\n",
      "  Read 1:   126,636,528 bp\n",
      "  Read 2:   130,819,633 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 359 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t346\t8838.2\t0\t346\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t4\t0.0\t1\t4\n",
      "53\t1\t0.0\t1\t1\n",
      "158\t1\t0.0\t1\t1\n",
      "169\t1\t0.0\t1\t1\n",
      "177\t1\t0.0\t1\t1\n",
      "202\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 36895 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t308\t8838.2\t0\t308\n",
      "4\t121\t2209.6\t0\t121\n",
      "5\t202\t552.4\t0\t202\n",
      "6\t422\t138.1\t0\t422\n",
      "7\t1098\t34.5\t0\t1098\n",
      "8\t1988\t8.6\t0\t1988\n",
      "9\t6475\t2.2\t0\t6454 21\n",
      "10\t8099\t0.5\t1\t8002 97\n",
      "11\t430\t0.1\t1\t413 17\n",
      "12\t554\t0.0\t1\t544 10\n",
      "13\t735\t0.0\t1\t706 29\n",
      "14\t1112\t0.0\t1\t1041 71\n",
      "15\t890\t0.0\t1\t848 42\n",
      "16\t2401\t0.0\t1\t2300 101\n",
      "17\t3124\t0.0\t1\t3009 115\n",
      "18\t1850\t0.0\t1\t1641 138 71\n",
      "19\t6237\t0.0\t1\t3569 2636 32\n",
      "20\t17\t0.0\t2\t1 14 2\n",
      "21\t750\t0.0\t2\t0 738 12\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "40\t1\t0.0\t2\t1\n",
      "44\t2\t0.0\t2\t2\n",
      "46\t1\t0.0\t2\t1\n",
      "49\t2\t0.0\t2\t2\n",
      "55\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t1 1\n",
      "68\t2\t0.0\t2\t2\n",
      "70\t4\t0.0\t2\t4\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t5\t0.0\t2\t3 1 1\n",
      "124\t1\t0.0\t2\t1\n",
      "132\t1\t0.0\t2\t1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t1\n",
      "158\t1\t0.0\t2\t1\n",
      "161\t1\t0.0\t2\t1\n",
      "169\t2\t0.0\t2\t0 0 2\n",
      "182\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "215\t2\t0.0\t2\t1 1\n",
      "217\t4\t0.0\t2\t1 3\n",
      "218\t3\t0.0\t2\t1 1 1\n",
      "228\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.19.B.5percent.8.1.19_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.19.B.5percent.8.1.19_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.19.B.5percent.8.1.19_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.19.B.5percent.8.1.19_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.15 s (13 us/read; 4.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            466,078\n",
      "  Read 1 with adapter:                     492 (0.1%)\n",
      "  Read 2 with adapter:                  43,249 (9.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       466,078 (100.0%)\n",
      "\n",
      "Total basepairs processed:   212,993,776 bp\n",
      "  Read 1:   104,356,281 bp\n",
      "  Read 2:   108,637,495 bp\n",
      "Total written (filtered):    212,449,273 bp (99.7%)\n",
      "  Read 1:   104,353,857 bp\n",
      "  Read 2:   108,095,416 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 492 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t471\t7282.5\t0\t471\n",
      "10\t1\t0.4\t1\t1\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t7\t0.0\t1\t7\n",
      "38\t1\t0.0\t1\t1\n",
      "168\t1\t0.0\t1\t0 1\n",
      "176\t1\t0.0\t1\t1\n",
      "191\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 43249 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t307\t7282.5\t0\t307\n",
      "4\t130\t1820.6\t0\t130\n",
      "5\t240\t455.2\t0\t240\n",
      "6\t612\t113.8\t0\t612\n",
      "7\t1851\t28.4\t0\t1851\n",
      "8\t2842\t7.1\t0\t2842\n",
      "9\t9129\t1.8\t0\t9113 16\n",
      "10\t10596\t0.4\t1\t10494 102\n",
      "11\t569\t0.1\t1\t557 12\n",
      "12\t678\t0.0\t1\t663 15\n",
      "13\t1059\t0.0\t1\t1022 37\n",
      "14\t1636\t0.0\t1\t1552 84\n",
      "15\t998\t0.0\t1\t921 77\n",
      "16\t2485\t0.0\t1\t2386 99\n",
      "17\t2746\t0.0\t1\t2622 124\n",
      "18\t1579\t0.0\t1\t1353 136 90\n",
      "19\t4968\t0.0\t1\t2758 2174 36\n",
      "20\t11\t0.0\t2\t1 8 2\n",
      "21\t632\t0.0\t2\t0 626 6\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t0 1\n",
      "56\t1\t0.0\t2\t1\n",
      "58\t3\t0.0\t2\t1 0 2\n",
      "59\t2\t0.0\t2\t1 0 1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t7\t0.0\t2\t6 1\n",
      "64\t1\t0.0\t2\t1\n",
      "66\t4\t0.0\t2\t2 1 1\n",
      "67\t6\t0.0\t2\t5 0 1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t4\t0.0\t2\t4\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "72\t3\t0.0\t2\t3\n",
      "74\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t11\t0.0\t2\t10 0 1\n",
      "85\t1\t0.0\t2\t1\n",
      "89\t2\t0.0\t2\t2\n",
      "94\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "97\t2\t0.0\t2\t2\n",
      "103\t2\t0.0\t2\t1 1\n",
      "104\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t1\n",
      "118\t4\t0.0\t2\t2 2\n",
      "120\t3\t0.0\t2\t2 0 1\n",
      "121\t1\t0.0\t2\t1\n",
      "129\t1\t0.0\t2\t0 1\n",
      "140\t1\t0.0\t2\t0 1\n",
      "141\t1\t0.0\t2\t1\n",
      "164\t1\t0.0\t2\t1\n",
      "167\t1\t0.0\t2\t1\n",
      "173\t2\t0.0\t2\t2\n",
      "185\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "192\t2\t0.0\t2\t2\n",
      "196\t1\t0.0\t2\t0 0 1\n",
      "197\t1\t0.0\t2\t1\n",
      "212\t1\t0.0\t2\t1\n",
      "213\t1\t0.0\t2\t1\n",
      "218\t2\t0.0\t2\t2\n",
      "225\t2\t0.0\t2\t1 0 1\n",
      "227\t1\t0.0\t2\t1\n",
      "232\t1\t0.0\t2\t0 1\n",
      "236\t1\t0.0\t2\t0 1\n",
      "237\t5\t0.0\t2\t3 2\n",
      "238\t4\t0.0\t2\t3 1\n",
      "239\t4\t0.0\t2\t3 1\n",
      "240\t10\t0.0\t2\t4 5 1\n",
      "241\t6\t0.0\t2\t5 0 1\n",
      "242\t8\t0.0\t2\t8\n",
      "243\t4\t0.0\t2\t2 2\n",
      "244\t1\t0.0\t2\t0 1\n",
      "245\t3\t0.0\t2\t1 2\n",
      "246\t2\t0.0\t2\t1 1\n",
      "248\t1\t0.0\t2\t0 1\n",
      "249\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.2.F.5percent.7.29.19_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.2.F.5percent.7.29.19_59_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.2.F.5percent.7.29.19_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.2.F.5percent.7.29.19_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.75 s (18 us/read; 3.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            152,506\n",
      "  Read 1 with adapter:                     225 (0.1%)\n",
      "  Read 2 with adapter:                  12,395 (8.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       152,506 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,370,913 bp\n",
      "  Read 1:    33,993,229 bp\n",
      "  Read 2:    35,377,684 bp\n",
      "Total written (filtered):     69,207,771 bp (99.8%)\n",
      "  Read 1:    33,991,809 bp\n",
      "  Read 2:    35,215,962 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 225 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t217\t2382.9\t0\t217\n",
      "19\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "207\t1\t0.0\t1\t1\n",
      "211\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12395 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t2382.9\t0\t72\n",
      "4\t37\t595.7\t0\t37\n",
      "5\t65\t148.9\t0\t65\n",
      "6\t119\t37.2\t0\t119\n",
      "7\t425\t9.3\t0\t425\n",
      "8\t675\t2.3\t0\t675\n",
      "9\t2295\t0.6\t0\t2291 4\n",
      "10\t3138\t0.1\t1\t3098 40\n",
      "11\t158\t0.0\t1\t155 3\n",
      "12\t202\t0.0\t1\t188 14\n",
      "13\t297\t0.0\t1\t287 10\n",
      "14\t485\t0.0\t1\t456 29\n",
      "15\t292\t0.0\t1\t268 24\n",
      "16\t766\t0.0\t1\t738 28\n",
      "17\t860\t0.0\t1\t823 37\n",
      "18\t555\t0.0\t1\t478 52 25\n",
      "19\t1670\t0.0\t1\t918 744 8\n",
      "20\t3\t0.0\t2\t0 3\n",
      "21\t215\t0.0\t2\t0 212 3\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "27\t1\t0.0\t2\t1\n",
      "30\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "57\t2\t0.0\t2\t1 1\n",
      "59\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "70\t4\t0.0\t2\t3 0 1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "79\t3\t0.0\t2\t1 2\n",
      "81\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t0 1\n",
      "90\t1\t0.0\t2\t1\n",
      "93\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t0 1\n",
      "107\t1\t0.0\t2\t1\n",
      "111\t1\t0.0\t2\t1\n",
      "132\t1\t0.0\t2\t1\n",
      "136\t1\t0.0\t2\t1\n",
      "148\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t0 1\n",
      "165\t1\t0.0\t2\t0 1\n",
      "166\t1\t0.0\t2\t0 1\n",
      "167\t1\t0.0\t2\t1\n",
      "169\t1\t0.0\t2\t1\n",
      "177\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "188\t1\t0.0\t2\t1\n",
      "204\t1\t0.0\t2\t1\n",
      "206\t1\t0.0\t2\t1\n",
      "212\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "224\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "240\t2\t0.0\t2\t2\n",
      "241\t4\t0.0\t2\t3 1\n",
      "242\t3\t0.0\t2\t2 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.20.B.5percent.9.5.19_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.20.B.5percent.9.5.19_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.20.B.5percent.9.5.19_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.20.B.5percent.9.5.19_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.90 s (15 us/read; 4.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            269,226\n",
      "  Read 1 with adapter:                     401 (0.1%)\n",
      "  Read 2 with adapter:                  17,575 (6.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       269,226 (100.0%)\n",
      "\n",
      "Total basepairs processed:   122,864,054 bp\n",
      "  Read 1:    60,272,743 bp\n",
      "  Read 2:    62,591,311 bp\n",
      "Total written (filtered):    122,626,853 bp (99.8%)\n",
      "  Read 1:    60,270,764 bp\n",
      "  Read 2:    62,356,089 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 401 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t387\t4206.7\t0\t387\n",
      "9\t1\t1.0\t0\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t4\t0.0\t1\t4\n",
      "51\t1\t0.0\t1\t0 1\n",
      "89\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 17575 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t107\t4206.7\t0\t107\n",
      "4\t54\t1051.7\t0\t54\n",
      "5\t68\t262.9\t0\t68\n",
      "6\t174\t65.7\t0\t174\n",
      "7\t489\t16.4\t0\t489\n",
      "8\t804\t4.1\t0\t804\n",
      "9\t2787\t1.0\t0\t2777 10\n",
      "10\t3737\t0.3\t1\t3688 49\n",
      "11\t233\t0.1\t1\t223 10\n",
      "12\t357\t0.0\t1\t342 15\n",
      "13\t554\t0.0\t1\t533 21\n",
      "14\t726\t0.0\t1\t696 30\n",
      "15\t483\t0.0\t1\t451 32\n",
      "16\t1327\t0.0\t1\t1271 56\n",
      "17\t1557\t0.0\t1\t1491 66\n",
      "18\t897\t0.0\t1\t800 69 28\n",
      "19\t2804\t0.0\t1\t1649 1141 14\n",
      "20\t5\t0.0\t2\t1 3 1\n",
      "21\t352\t0.0\t2\t0 343 9\n",
      "22\t20\t0.0\t2\t0 1 19\n",
      "30\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "61\t3\t0.0\t2\t3\n",
      "62\t2\t0.0\t2\t1 0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "70\t4\t0.0\t2\t4\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t2 0 1\n",
      "88\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t1\n",
      "113\t1\t0.0\t2\t1\n",
      "123\t1\t0.0\t2\t1\n",
      "148\t1\t0.0\t2\t1\n",
      "160\t1\t0.0\t2\t0 1\n",
      "161\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t1\n",
      "179\t1\t0.0\t2\t1\n",
      "180\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t2\t0.0\t2\t2\n",
      "226\t2\t0.0\t2\t1 1\n",
      "232\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.20.E.5percent.9.10.19_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.20.E.5percent.9.10.19_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.20.E.5percent.9.10.19_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.20.E.5percent.9.10.19_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.50 s (12 us/read; 4.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            453,095\n",
      "  Read 1 with adapter:                     412 (0.1%)\n",
      "  Read 2 with adapter:                  14,079 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       453,095 (100.0%)\n",
      "\n",
      "Total basepairs processed:   206,458,650 bp\n",
      "  Read 1:   101,447,329 bp\n",
      "  Read 2:   105,011,321 bp\n",
      "Total written (filtered):    206,243,953 bp (99.9%)\n",
      "  Read 1:   101,444,922 bp\n",
      "  Read 2:   104,799,031 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 412 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t376\t7079.6\t0\t376\n",
      "13\t2\t0.0\t1\t2\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t22\t0.0\t1\t22\n",
      "89\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14079 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t7079.6\t0\t73\n",
      "4\t30\t1769.9\t0\t30\n",
      "5\t29\t442.5\t0\t29\n",
      "6\t77\t110.6\t0\t77\n",
      "7\t201\t27.7\t0\t201\n",
      "8\t366\t6.9\t0\t366\n",
      "9\t1417\t1.7\t0\t1415 2\n",
      "10\t2188\t0.4\t1\t2158 30\n",
      "11\t158\t0.1\t1\t153 5\n",
      "12\t246\t0.0\t1\t239 7\n",
      "13\t302\t0.0\t1\t289 13\n",
      "14\t412\t0.0\t1\t388 24\n",
      "15\t355\t0.0\t1\t337 18\n",
      "16\t956\t0.0\t1\t921 35\n",
      "17\t1640\t0.0\t1\t1590 50\n",
      "18\t1072\t0.0\t1\t977 70 25\n",
      "19\t3922\t0.0\t1\t2445 1452 25\n",
      "20\t7\t0.0\t2\t1 3 3\n",
      "21\t582\t0.0\t2\t0 570 12\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "30\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t2\n",
      "70\t2\t0.0\t2\t2\n",
      "79\t7\t0.0\t2\t4 3\n",
      "88\t1\t0.0\t2\t1\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t1\n",
      "179\t1\t0.0\t2\t1\n",
      "194\t1\t0.0\t2\t1\n",
      "217\t3\t0.0\t2\t1 1 1\n",
      "218\t1\t0.0\t2\t1\n",
      "228\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.3.E.5percent.9.11.19_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.3.E.5percent.9.11.19_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.3.E.5percent.9.11.19_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.3.E.5percent.9.11.19_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.87 s (12 us/read; 4.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            476,842\n",
      "  Read 1 with adapter:                     402 (0.1%)\n",
      "  Read 2 with adapter:                  23,044 (4.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       476,842 (100.0%)\n",
      "\n",
      "Total basepairs processed:   217,437,314 bp\n",
      "  Read 1:   106,756,761 bp\n",
      "  Read 2:   110,680,553 bp\n",
      "Total written (filtered):    217,116,611 bp (99.9%)\n",
      "  Read 1:   106,754,908 bp\n",
      "  Read 2:   110,361,703 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 402 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t389\t7450.7\t0\t389\n",
      "6\t1\t116.4\t0\t1\n",
      "23\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t5\t0.0\t1\t5\n",
      "66\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 23044 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t145\t7450.7\t0\t145\n",
      "4\t64\t1862.7\t0\t64\n",
      "5\t77\t465.7\t0\t77\n",
      "6\t206\t116.4\t0\t206\n",
      "7\t501\t29.1\t0\t501\n",
      "8\t913\t7.3\t0\t913\n",
      "9\t3412\t1.8\t0\t3403 9\n",
      "10\t4187\t0.5\t1\t4141 46\n",
      "11\t354\t0.1\t1\t350 4\n",
      "12\t538\t0.0\t1\t529 9\n",
      "13\t706\t0.0\t1\t679 27\n",
      "14\t941\t0.0\t1\t887 54\n",
      "15\t563\t0.0\t1\t525 38\n",
      "16\t1593\t0.0\t1\t1523 70\n",
      "17\t2171\t0.0\t1\t2083 88\n",
      "18\t1373\t0.0\t1\t1224 97 52\n",
      "19\t4611\t0.0\t1\t2746 1842 23\n",
      "20\t7\t0.0\t2\t1 4 2\n",
      "21\t612\t0.0\t2\t0 601 11\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t2\t0.0\t2\t2\n",
      "66\t2\t0.0\t2\t0 1 1\n",
      "67\t1\t0.0\t2\t1\n",
      "70\t3\t0.0\t2\t2 0 1\n",
      "71\t4\t0.0\t2\t1 0 3\n",
      "74\t1\t0.0\t2\t1\n",
      "79\t5\t0.0\t2\t4 1\n",
      "96\t1\t0.0\t2\t1\n",
      "126\t1\t0.0\t2\t0 1\n",
      "143\t2\t0.0\t2\t2\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "202\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.4.B.5percent.8.5.19_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.4.B.5percent.8.5.19_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.4.B.5percent.8.5.19_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.4.B.5percent.8.5.19_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.56 s (16 us/read; 3.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            222,438\n",
      "  Read 1 with adapter:                     249 (0.1%)\n",
      "  Read 2 with adapter:                  20,506 (9.2%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       222,437 (100.0%)\n",
      "\n",
      "Total basepairs processed:   101,652,444 bp\n",
      "  Read 1:    49,803,564 bp\n",
      "  Read 2:    51,848,880 bp\n",
      "Total written (filtered):    101,398,945 bp (99.8%)\n",
      "  Read 1:    49,802,267 bp\n",
      "  Read 2:    51,596,678 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 249 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t239\t3475.6\t0\t239\n",
      "5\t1\t217.2\t0\t1\n",
      "9\t1\t0.8\t0\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "29\t5\t0.0\t1\t5\n",
      "71\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 20506 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t106\t3475.6\t0\t106\n",
      "4\t65\t868.9\t0\t65\n",
      "5\t116\t217.2\t0\t116\n",
      "6\t283\t54.3\t0\t283\n",
      "7\t848\t13.6\t0\t848\n",
      "8\t1409\t3.4\t0\t1409\n",
      "9\t4560\t0.8\t0\t4549 11\n",
      "10\t4883\t0.2\t1\t4828 55\n",
      "11\t267\t0.1\t1\t259 8\n",
      "12\t338\t0.0\t1\t328 10\n",
      "13\t518\t0.0\t1\t499 19\n",
      "14\t726\t0.0\t1\t686 40\n",
      "15\t436\t0.0\t1\t409 27\n",
      "16\t1105\t0.0\t1\t1063 42\n",
      "17\t1341\t0.0\t1\t1267 74\n",
      "18\t750\t0.0\t1\t646 63 41\n",
      "19\t2372\t0.0\t1\t1308 1054 10\n",
      "20\t7\t0.0\t2\t0 6 1\n",
      "21\t290\t0.0\t2\t0 288 2\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "40\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t1\n",
      "54\t2\t0.0\t2\t2\n",
      "55\t2\t0.0\t2\t2\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 1\n",
      "59\t2\t0.0\t2\t1 1\n",
      "60\t3\t0.0\t2\t2 0 1\n",
      "61\t2\t0.0\t2\t2\n",
      "62\t2\t0.0\t2\t1 1\n",
      "63\t3\t0.0\t2\t2 0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t4\t0.0\t2\t3 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t5\t0.0\t2\t3 0 2\n",
      "69\t2\t0.0\t2\t1 0 1\n",
      "70\t3\t0.0\t2\t3\n",
      "71\t4\t0.0\t2\t2 0 2\n",
      "77\t2\t0.0\t2\t2\n",
      "79\t10\t0.0\t2\t7 1 2\n",
      "81\t1\t0.0\t2\t1\n",
      "83\t1\t0.0\t2\t0 1\n",
      "86\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "113\t1\t0.0\t2\t0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t1\n",
      "194\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t0 0 1\n",
      "218\t3\t0.0\t2\t3\n",
      "231\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "251\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.4.G.5percent.8.5.19_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.4.G.5percent.8.5.19_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.4.G.5percent.8.5.19_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.4.G.5percent.8.5.19_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.93 s (15 us/read; 4.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            331,050\n",
      "  Read 1 with adapter:                     463 (0.1%)\n",
      "  Read 2 with adapter:                  30,630 (9.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       331,050 (100.0%)\n",
      "\n",
      "Total basepairs processed:   151,272,773 bp\n",
      "  Read 1:    74,121,083 bp\n",
      "  Read 2:    77,151,690 bp\n",
      "Total written (filtered):    150,896,176 bp (99.8%)\n",
      "  Read 1:    74,118,343 bp\n",
      "  Read 2:    76,777,833 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 463 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t445\t5172.7\t0\t445\n",
      "14\t1\t0.0\t1\t1\n",
      "18\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "29\t6\t0.0\t1\t6\n",
      "67\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "118\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "136\t1\t0.0\t1\t1\n",
      "141\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 30630 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t226\t5172.7\t0\t226\n",
      "4\t112\t1293.2\t0\t112\n",
      "5\t148\t323.3\t0\t148\n",
      "6\t391\t80.8\t0\t391\n",
      "7\t1151\t20.2\t0\t1151\n",
      "8\t2041\t5.1\t0\t2041\n",
      "9\t6514\t1.3\t0\t6500 14\n",
      "10\t7812\t0.3\t1\t7735 77\n",
      "11\t355\t0.1\t1\t347 8\n",
      "12\t490\t0.0\t1\t475 15\n",
      "13\t760\t0.0\t1\t734 26\n",
      "14\t1074\t0.0\t1\t1021 53\n",
      "15\t645\t0.0\t1\t594 51\n",
      "16\t1671\t0.0\t1\t1590 81\n",
      "17\t2000\t0.0\t1\t1906 94\n",
      "18\t1098\t0.0\t1\t946 92 60\n",
      "19\t3582\t0.0\t1\t2002 1564 16\n",
      "20\t9\t0.0\t2\t1 6 2\n",
      "21\t477\t0.0\t2\t0 468 9\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "47\t2\t0.0\t2\t2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t1 1\n",
      "55\t2\t0.0\t2\t1 0 1\n",
      "57\t2\t0.0\t2\t0 1 1\n",
      "58\t4\t0.0\t2\t3 1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "65\t2\t0.0\t2\t2\n",
      "66\t4\t0.0\t2\t4\n",
      "67\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t2\n",
      "79\t3\t0.0\t2\t2 1\n",
      "82\t1\t0.0\t2\t1\n",
      "85\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t1\n",
      "97\t1\t0.0\t2\t1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t0 1\n",
      "198\t1\t0.0\t2\t1\n",
      "204\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "218\t1\t0.0\t2\t1\n",
      "233\t1\t0.0\t2\t1\n",
      "240\t1\t0.0\t2\t1\n",
      "241\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.5.B.5percent.9.2.19_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.5.B.5percent.9.2.19_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.5.B.5percent.9.2.19_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.5.B.5percent.9.2.19_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.55 s (18 us/read; 3.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,898\n",
      "  Read 1 with adapter:                     186 (0.1%)\n",
      "  Read 2 with adapter:                   9,753 (6.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       142,898 (100.0%)\n",
      "\n",
      "Total basepairs processed:    64,976,625 bp\n",
      "  Read 1:    31,862,544 bp\n",
      "  Read 2:    33,114,081 bp\n",
      "Total written (filtered):     64,845,313 bp (99.8%)\n",
      "  Read 1:    31,860,660 bp\n",
      "  Read 2:    32,984,653 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 186 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t178\t2232.8\t0\t178\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t1 1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9753 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t2232.8\t0\t80\n",
      "4\t29\t558.2\t0\t29\n",
      "5\t39\t139.5\t0\t39\n",
      "6\t101\t34.9\t0\t101\n",
      "7\t292\t8.7\t0\t292\n",
      "8\t509\t2.2\t0\t509\n",
      "9\t1775\t0.5\t0\t1773 2\n",
      "10\t2198\t0.1\t1\t2166 32\n",
      "11\t126\t0.0\t1\t125 1\n",
      "12\t182\t0.0\t1\t172 10\n",
      "13\t206\t0.0\t1\t193 13\n",
      "14\t345\t0.0\t1\t328 17\n",
      "15\t203\t0.0\t1\t193 10\n",
      "16\t615\t0.0\t1\t595 20\n",
      "17\t766\t0.0\t1\t730 36\n",
      "18\t519\t0.0\t1\t452 43 24\n",
      "19\t1526\t0.0\t1\t882 638 6\n",
      "20\t1\t0.0\t2\t0 1\n",
      "21\t188\t0.0\t2\t0 183 5\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "30\t1\t0.0\t2\t1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "70\t4\t0.0\t2\t2 1 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t5\t0.0\t2\t5\n",
      "81\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "99\t1\t0.0\t2\t1\n",
      "110\t1\t0.0\t2\t0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "157\t2\t0.0\t2\t2\n",
      "160\t1\t0.0\t2\t1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t1\n",
      "175\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t3\t0.0\t2\t1 1 1\n",
      "226\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.5.D.5percent.9.11.19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.5.D.5percent.9.11.19_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.5.D.5percent.9.11.19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.5.D.5percent.9.11.19_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.02 s (12 us/read; 4.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            325,156\n",
      "  Read 1 with adapter:                     360 (0.1%)\n",
      "  Read 2 with adapter:                  26,523 (8.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       325,156 (100.0%)\n",
      "\n",
      "Total basepairs processed:   148,148,431 bp\n",
      "  Read 1:    72,607,317 bp\n",
      "  Read 2:    75,541,114 bp\n",
      "Total written (filtered):    147,830,355 bp (99.8%)\n",
      "  Read 1:    72,604,991 bp\n",
      "  Read 2:    75,225,364 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 360 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t339\t5080.6\t0\t339\n",
      "25\t4\t0.0\t1\t4\n",
      "26\t7\t0.0\t1\t6 1\n",
      "27\t4\t0.0\t1\t4\n",
      "29\t1\t0.0\t1\t1\n",
      "107\t1\t0.0\t1\t0 1\n",
      "132\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 26523 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t194\t5080.6\t0\t194\n",
      "4\t115\t1270.1\t0\t115\n",
      "5\t182\t317.5\t0\t182\n",
      "6\t417\t79.4\t0\t417\n",
      "7\t1213\t19.8\t0\t1213\n",
      "8\t2027\t5.0\t0\t2027\n",
      "9\t6396\t1.2\t0\t6386 10\n",
      "10\t6442\t0.3\t1\t6373 69\n",
      "11\t261\t0.1\t1\t253 8\n",
      "12\t260\t0.0\t1\t255 5\n",
      "13\t398\t0.0\t1\t382 16\n",
      "14\t631\t0.0\t1\t593 38\n",
      "15\t477\t0.0\t1\t450 27\n",
      "16\t1249\t0.0\t1\t1192 57\n",
      "17\t1600\t0.0\t1\t1523 77\n",
      "18\t1021\t0.0\t1\t906 74 41\n",
      "19\t3165\t0.0\t1\t1816 1337 12\n",
      "20\t10\t0.0\t2\t1 7 2\n",
      "21\t426\t0.0\t2\t0 418 8\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "30\t2\t0.0\t2\t2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "63\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t4\n",
      "93\t2\t0.0\t2\t2\n",
      "159\t1\t0.0\t2\t1\n",
      "204\t1\t0.0\t2\t1\n",
      "215\t2\t0.0\t2\t2\n",
      "237\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.8.D.5percent.9.6.19_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.8.D.5percent.9.6.19_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.8.D.5percent.9.6.19_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.8.D.5percent.9.6.19_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.17 s (15 us/read; 3.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            403,612\n",
      "  Read 1 with adapter:                     321 (0.1%)\n",
      "  Read 2 with adapter:                  28,917 (7.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       403,612 (100.0%)\n",
      "\n",
      "Total basepairs processed:   184,177,821 bp\n",
      "  Read 1:    90,359,522 bp\n",
      "  Read 2:    93,818,299 bp\n",
      "Total written (filtered):    183,803,165 bp (99.8%)\n",
      "  Read 1:    90,357,560 bp\n",
      "  Read 2:    93,445,605 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 321 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t308\t6306.4\t0\t308\n",
      "26\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t4\t0.0\t1\t4\n",
      "30\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "182\t1\t0.0\t1\t1\n",
      "194\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 28917 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t233\t6306.4\t0\t233\n",
      "4\t101\t1576.6\t0\t101\n",
      "5\t144\t394.2\t0\t144\n",
      "6\t326\t98.5\t0\t326\n",
      "7\t1054\t24.6\t0\t1054\n",
      "8\t1746\t6.2\t0\t1746\n",
      "9\t5625\t1.5\t0\t5611 14\n",
      "10\t5900\t0.4\t1\t5835 65\n",
      "11\t362\t0.1\t1\t349 13\n",
      "12\t461\t0.0\t1\t455 6\n",
      "13\t669\t0.0\t1\t643 26\n",
      "14\t995\t0.0\t1\t943 52\n",
      "15\t646\t0.0\t1\t611 35\n",
      "16\t1797\t0.0\t1\t1719 78\n",
      "17\t2343\t0.0\t1\t2257 86\n",
      "18\t1353\t0.0\t1\t1196 100 57\n",
      "19\t4515\t0.0\t1\t2647 1838 30\n",
      "20\t9\t0.0\t2\t0 8 1\n",
      "21\t546\t0.0\t2\t0 535 11\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "30\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t1 1\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t4\t0.0\t2\t4\n",
      "57\t2\t0.0\t2\t0 2\n",
      "61\t3\t0.0\t2\t3\n",
      "63\t2\t0.0\t2\t2\n",
      "64\t2\t0.0\t2\t1 1\n",
      "65\t2\t0.0\t2\t2\n",
      "66\t6\t0.0\t2\t4 1 1\n",
      "67\t2\t0.0\t2\t1 0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t4\t0.0\t2\t3 1\n",
      "71\t3\t0.0\t2\t1 0 2\n",
      "73\t1\t0.0\t2\t1\n",
      "74\t1\t0.0\t2\t1\n",
      "78\t3\t0.0\t2\t1 2\n",
      "79\t9\t0.0\t2\t7 2\n",
      "86\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t1\n",
      "125\t1\t0.0\t2\t0 1\n",
      "158\t2\t0.0\t2\t2\n",
      "162\t1\t0.0\t2\t0 1\n",
      "163\t1\t0.0\t2\t1\n",
      "175\t1\t0.0\t2\t0 1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "226\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.8.G.5percent.8.21.19_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.8.G.5percent.8.21.19_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.8.G.5percent.8.21.19_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.8.G.5percent.8.21.19_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.29 s (14 us/read; 4.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            521,350\n",
      "  Read 1 with adapter:                     468 (0.1%)\n",
      "  Read 2 with adapter:                  45,798 (8.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       521,350 (100.0%)\n",
      "\n",
      "Total basepairs processed:   238,165,440 bp\n",
      "  Read 1:   116,730,192 bp\n",
      "  Read 2:   121,435,248 bp\n",
      "Total written (filtered):    237,598,307 bp (99.8%)\n",
      "  Read 1:   116,727,673 bp\n",
      "  Read 2:   120,870,634 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 468 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t446\t8146.1\t0\t446\n",
      "21\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "29\t10\t0.0\t1\t9 1\n",
      "50\t1\t0.0\t1\t1\n",
      "86\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t0 1\n",
      "89\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "170\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 45798 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t335\t8146.1\t0\t335\n",
      "4\t123\t2036.5\t0\t123\n",
      "5\t227\t509.1\t0\t227\n",
      "6\t568\t127.3\t0\t568\n",
      "7\t1768\t31.8\t0\t1768\n",
      "8\t2919\t8.0\t0\t2919\n",
      "9\t9521\t2.0\t0\t9496 25\n",
      "10\t11414\t0.5\t1\t11270 144\n",
      "11\t552\t0.1\t1\t541 11\n",
      "12\t714\t0.0\t1\t696 18\n",
      "13\t1118\t0.0\t1\t1075 43\n",
      "14\t1609\t0.0\t1\t1528 81\n",
      "15\t1077\t0.0\t1\t1008 69\n",
      "16\t2700\t0.0\t1\t2586 114\n",
      "17\t2963\t0.0\t1\t2829 134\n",
      "18\t1752\t0.0\t1\t1524 143 85\n",
      "19\t5627\t0.0\t1\t3124 2468 35\n",
      "20\t12\t0.0\t2\t3 5 4\n",
      "21\t692\t0.0\t2\t0 680 12\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "30\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t1\n",
      "49\t2\t0.0\t2\t2\n",
      "50\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t5\t0.0\t2\t5\n",
      "64\t2\t0.0\t2\t1 0 1\n",
      "67\t2\t0.0\t2\t2\n",
      "69\t4\t0.0\t2\t4\n",
      "70\t5\t0.0\t2\t3 0 2\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "73\t2\t0.0\t2\t2\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t13\t0.0\t2\t12 1\n",
      "80\t2\t0.0\t2\t2\n",
      "84\t1\t0.0\t2\t1\n",
      "86\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "96\t2\t0.0\t2\t1 1\n",
      "103\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t1\n",
      "141\t1\t0.0\t2\t1\n",
      "160\t1\t0.0\t2\t1\n",
      "161\t1\t0.0\t2\t0 1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "178\t1\t0.0\t2\t1\n",
      "192\t1\t0.0\t2\t1\n",
      "207\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "229\t1\t0.0\t2\t1\n",
      "235\t1\t0.0\t2\t0 1\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "238\t2\t0.0\t2\t2\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.17.D.5percent.7.29.19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.17.D.5percent.7.29.19_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.17.D.5percent.7.29.19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.17.D.5percent.7.29.19_1_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.17.G.5percent.9.4.19_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.17.G.5percent.9.4.19_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.17.G.5percent.9.4.19_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.17.G.5percent.9.4.19_7_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.19.B.5percent.8.1.19_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.19.B.5percent.8.1.19_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.19.B.5percent.8.1.19_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.19.B.5percent.8.1.19_20_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.2.F.5percent.7.29.19_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.2.F.5percent.7.29.19_59_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.2.F.5percent.7.29.19_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.2.F.5percent.7.29.19_59_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.20.B.5percent.9.5.19_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.20.B.5percent.9.5.19_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.20.B.5percent.9.5.19_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.20.B.5percent.9.5.19_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.20.E.5percent.9.10.19_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.20.E.5percent.9.10.19_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.20.E.5percent.9.10.19_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.20.E.5percent.9.10.19_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.3.E.5percent.9.11.19_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.3.E.5percent.9.11.19_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.3.E.5percent.9.11.19_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.3.E.5percent.9.11.19_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.4.B.5percent.8.5.19_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.4.B.5percent.8.5.19_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.4.B.5percent.8.5.19_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.4.B.5percent.8.5.19_3_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.4.G.5percent.8.5.19_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.4.G.5percent.8.5.19_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.4.G.5percent.8.5.19_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.4.G.5percent.8.5.19_24_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.5.B.5percent.9.2.19_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.5.B.5percent.9.2.19_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.5.B.5percent.9.2.19_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.5.B.5percent.9.2.19_32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.5.D.5percent.9.11.19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.5.D.5percent.9.11.19_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.5.D.5percent.9.11.19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.5.D.5percent.9.11.19_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.8.D.5percent.9.6.19_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.8.D.5percent.9.6.19_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.8.D.5percent.9.6.19_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.8.D.5percent.9.6.19_15_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.8.G.5percent.8.21.19_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-b3ovgkp9/O.fav.Tank.8.G.5percent.8.21.19_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.8.G.5percent.8.21.19_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-959gts87/031527b2-909b-4f13-86e8-bac6d2b8187b/data/O.fav.Tank.8.G.5percent.8.21.19_31_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGtransmissionstudies_13_207.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.4.26.19_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.4.26.19_58_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.4.26.19_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.4.26.19_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.84 s (13 us/read; 4.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            297,072\n",
      "  Read 1 with adapter:                     321 (0.1%)\n",
      "  Read 2 with adapter:                   6,697 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       297,072 (100.0%)\n",
      "\n",
      "Total basepairs processed:   135,356,135 bp\n",
      "  Read 1:    66,533,382 bp\n",
      "  Read 2:    68,822,753 bp\n",
      "Total written (filtered):    135,247,642 bp (99.9%)\n",
      "  Read 1:    66,531,965 bp\n",
      "  Read 2:    68,715,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 321 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t305\t4641.8\t0\t305\n",
      "26\t5\t0.0\t1\t4 1\n",
      "27\t5\t0.0\t1\t5\n",
      "29\t5\t0.0\t1\t5\n",
      "92\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6697 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t4641.8\t0\t31\n",
      "4\t8\t1160.4\t0\t8\n",
      "5\t20\t290.1\t0\t20\n",
      "6\t49\t72.5\t0\t49\n",
      "7\t99\t18.1\t0\t99\n",
      "8\t128\t4.5\t0\t128\n",
      "9\t491\t1.1\t0\t491\n",
      "10\t761\t0.3\t1\t756 5\n",
      "11\t71\t0.1\t1\t69 2\n",
      "12\t79\t0.0\t1\t78 1\n",
      "13\t67\t0.0\t1\t61 6\n",
      "14\t171\t0.0\t1\t151 20\n",
      "15\t127\t0.0\t1\t122 5\n",
      "16\t484\t0.0\t1\t463 21\n",
      "17\t911\t0.0\t1\t885 26\n",
      "18\t555\t0.0\t1\t507 40 8\n",
      "19\t2230\t0.0\t1\t1418 803 9\n",
      "20\t5\t0.0\t2\t2 3\n",
      "21\t380\t0.0\t2\t0 368 12\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "61\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t2\n",
      "79\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t0 0 1\n",
      "217\t5\t0.0\t2\t4 0 1\n",
      "228\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.F.5.4.19_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.F.5.4.19_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.F.5.4.19_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.F.5.4.19_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.45 s (15 us/read; 4.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            234,199\n",
      "  Read 1 with adapter:                     209 (0.1%)\n",
      "  Read 2 with adapter:                   5,073 (2.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       234,199 (100.0%)\n",
      "\n",
      "Total basepairs processed:   106,686,058 bp\n",
      "  Read 1:    52,442,314 bp\n",
      "  Read 2:    54,243,744 bp\n",
      "Total written (filtered):    106,604,890 bp (99.9%)\n",
      "  Read 1:    52,441,242 bp\n",
      "  Read 2:    54,163,648 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 209 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t202\t3659.4\t0\t202\n",
      "19\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t1 1\n",
      "29\t2\t0.0\t1\t2\n",
      "140\t1\t0.0\t1\t1\n",
      "197\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5073 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t3659.4\t0\t42\n",
      "4\t5\t914.8\t0\t5\n",
      "5\t27\t228.7\t0\t27\n",
      "6\t88\t57.2\t0\t88\n",
      "7\t192\t14.3\t0\t192\n",
      "8\t119\t3.6\t0\t119\n",
      "9\t397\t0.9\t0\t397\n",
      "10\t366\t0.2\t1\t363 3\n",
      "11\t33\t0.1\t1\t32 1\n",
      "12\t61\t0.0\t1\t59 2\n",
      "13\t43\t0.0\t1\t42 1\n",
      "14\t106\t0.0\t1\t98 8\n",
      "15\t113\t0.0\t1\t106 7\n",
      "16\t332\t0.0\t1\t322 10\n",
      "17\t669\t0.0\t1\t647 22\n",
      "18\t405\t0.0\t1\t382 19 4\n",
      "19\t1762\t0.0\t1\t1183 574 5\n",
      "20\t5\t0.0\t2\t2 1 2\n",
      "21\t292\t0.0\t2\t0 284 8\n",
      "22\t6\t0.0\t2\t0 1 5\n",
      "48\t1\t0.0\t2\t0 1\n",
      "52\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "84\t1\t0.0\t2\t1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.N.5.4.19_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.N.5.4.19_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.N.5.4.19_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.N.5.4.19_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.72 s (13 us/read; 4.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            532,444\n",
      "  Read 1 with adapter:                     574 (0.1%)\n",
      "  Read 2 with adapter:                  25,729 (4.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       532,444 (100.0%)\n",
      "\n",
      "Total basepairs processed:   242,878,880 bp\n",
      "  Read 1:   119,240,731 bp\n",
      "  Read 2:   123,638,149 bp\n",
      "Total written (filtered):    242,528,111 bp (99.9%)\n",
      "  Read 1:   119,238,443 bp\n",
      "  Read 2:   123,289,668 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 574 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t559\t8319.4\t0\t559\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t5\t0.0\t1\t5\n",
      "43\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 25729 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t173\t8319.4\t0\t173\n",
      "4\t64\t2079.9\t0\t64\n",
      "5\t59\t520.0\t0\t59\n",
      "6\t157\t130.0\t0\t157\n",
      "7\t547\t32.5\t0\t547\n",
      "8\t723\t8.1\t0\t723\n",
      "9\t2847\t2.0\t0\t2837 10\n",
      "10\t8137\t0.5\t1\t8029 108\n",
      "11\t299\t0.1\t1\t294 5\n",
      "12\t210\t0.0\t1\t208 2\n",
      "13\t230\t0.0\t1\t223 7\n",
      "14\t540\t0.0\t1\t494 46\n",
      "15\t583\t0.0\t1\t563 20\n",
      "16\t1658\t0.0\t1\t1612 46\n",
      "17\t2420\t0.0\t1\t2332 88\n",
      "18\t1394\t0.0\t1\t1263 98 33\n",
      "19\t4879\t0.0\t1\t2879 1976 24\n",
      "20\t20\t0.0\t2\t9 8 3\n",
      "21\t737\t0.0\t2\t0 724 13\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "43\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t2\n",
      "96\t1\t0.0\t2\t0 1\n",
      "98\t1\t0.0\t2\t1\n",
      "119\t1\t0.0\t2\t1\n",
      "139\t1\t0.0\t2\t0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t2\t0.0\t2\t2\n",
      "244\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D2.4.26.19_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D2.4.26.19_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D2.4.26.19_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D2.4.26.19_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.97 s (18 us/read; 3.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            168,529\n",
      "  Read 1 with adapter:                     108 (0.1%)\n",
      "  Read 2 with adapter:                   3,561 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       168,529 (100.0%)\n",
      "\n",
      "Total basepairs processed:    76,711,633 bp\n",
      "  Read 1:    37,709,411 bp\n",
      "  Read 2:    39,002,222 bp\n",
      "Total written (filtered):     76,653,004 bp (99.9%)\n",
      "  Read 1:    37,708,479 bp\n",
      "  Read 2:    38,944,525 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t101\t2633.3\t0\t101\n",
      "23\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "89\t2\t0.0\t1\t2\n",
      "155\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3561 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t2633.3\t0\t31\n",
      "4\t4\t658.3\t0\t4\n",
      "5\t16\t164.6\t0\t16\n",
      "6\t59\t41.1\t0\t59\n",
      "7\t78\t10.3\t0\t78\n",
      "8\t48\t2.6\t0\t48\n",
      "9\t213\t0.6\t0\t213\n",
      "10\t242\t0.2\t1\t240 2\n",
      "11\t29\t0.0\t1\t29\n",
      "12\t64\t0.0\t1\t62 2\n",
      "13\t46\t0.0\t1\t44 2\n",
      "14\t89\t0.0\t1\t84 5\n",
      "15\t88\t0.0\t1\t85 3\n",
      "16\t263\t0.0\t1\t258 5\n",
      "17\t466\t0.0\t1\t449 17\n",
      "18\t290\t0.0\t1\t266 16 8\n",
      "19\t1323\t0.0\t1\t809 506 8\n",
      "20\t1\t0.0\t2\t1\n",
      "21\t204\t0.0\t2\t0 204\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "73\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "217\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D2.5.4.19_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D2.5.4.19_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D2.5.4.19_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D2.5.4.19_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.44 s (14 us/read; 4.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            454,513\n",
      "  Read 1 with adapter:                     515 (0.1%)\n",
      "  Read 2 with adapter:                  19,888 (4.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       454,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:   207,113,625 bp\n",
      "  Read 1:   101,701,510 bp\n",
      "  Read 2:   105,412,115 bp\n",
      "Total written (filtered):    206,838,304 bp (99.9%)\n",
      "  Read 1:   101,697,883 bp\n",
      "  Read 2:   105,140,421 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 515 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t483\t7101.8\t0\t483\n",
      "8\t1\t6.9\t0\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t10\t0.0\t1\t10\n",
      "27\t4\t0.0\t1\t4\n",
      "29\t4\t0.0\t1\t4\n",
      "55\t1\t0.0\t1\t1\n",
      "89\t2\t0.0\t1\t2\n",
      "155\t1\t0.0\t1\t1\n",
      "180\t1\t0.0\t1\t1\n",
      "184\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 19888 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t162\t7101.8\t0\t162\n",
      "4\t54\t1775.4\t0\t54\n",
      "5\t61\t443.9\t0\t61\n",
      "6\t103\t111.0\t0\t103\n",
      "7\t362\t27.7\t0\t362\n",
      "8\t507\t6.9\t0\t507\n",
      "9\t2005\t1.7\t0\t1994 11\n",
      "10\t6371\t0.4\t1\t6305 66\n",
      "11\t268\t0.1\t1\t265 3\n",
      "12\t192\t0.0\t1\t189 3\n",
      "13\t169\t0.0\t1\t162 7\n",
      "14\t400\t0.0\t1\t372 28\n",
      "15\t444\t0.0\t1\t424 20\n",
      "16\t1273\t0.0\t1\t1237 36\n",
      "17\t1769\t0.0\t1\t1723 46\n",
      "18\t1169\t0.0\t1\t1075 64 30\n",
      "19\t3932\t0.0\t1\t2360 1553 19\n",
      "20\t13\t0.0\t2\t6 7\n",
      "21\t588\t0.0\t2\t0 582 6\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "54\t2\t0.0\t2\t1 1\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t2\t0.0\t2\t2\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "86\t1\t0.0\t2\t1\n",
      "143\t1\t0.0\t2\t0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t4\t0.0\t2\t3 1\n",
      "225\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.4.26.19_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.4.26.19_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.4.26.19_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.4.26.19_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.70 s (17 us/read; 3.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            219,536\n",
      "  Read 1 with adapter:                     348 (0.2%)\n",
      "  Read 2 with adapter:                   5,814 (2.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       219,536 (100.0%)\n",
      "\n",
      "Total basepairs processed:    99,804,718 bp\n",
      "  Read 1:    49,042,207 bp\n",
      "  Read 2:    50,762,511 bp\n",
      "Total written (filtered):     99,713,681 bp (99.9%)\n",
      "  Read 1:    49,039,957 bp\n",
      "  Read 2:    50,673,724 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 348 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t337\t3430.2\t0\t337\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t2\t0.0\t1\t2\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5814 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t3430.2\t0\t31\n",
      "4\t12\t857.6\t0\t12\n",
      "5\t15\t214.4\t0\t15\n",
      "6\t26\t53.6\t0\t26\n",
      "7\t74\t13.4\t0\t74\n",
      "8\t93\t3.3\t0\t93\n",
      "9\t372\t0.8\t0\t370 2\n",
      "10\t1140\t0.2\t1\t1127 13\n",
      "11\t85\t0.1\t1\t82 3\n",
      "12\t76\t0.0\t1\t73 3\n",
      "13\t85\t0.0\t1\t81 4\n",
      "14\t124\t0.0\t1\t116 8\n",
      "15\t141\t0.0\t1\t136 5\n",
      "16\t421\t0.0\t1\t403 18\n",
      "17\t713\t0.0\t1\t697 16\n",
      "18\t461\t0.0\t1\t415 31 15\n",
      "19\t1651\t0.0\t1\t1021 621 9\n",
      "20\t4\t0.0\t2\t4\n",
      "21\t279\t0.0\t2\t0 278 1\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "139\t1\t0.0\t2\t0 1\n",
      "217\t4\t0.0\t2\t3 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.5.4.19_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.5.4.19_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.5.4.19_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.5.4.19_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.67 s (14 us/read; 4.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            419,665\n",
      "  Read 1 with adapter:                     382 (0.1%)\n",
      "  Read 2 with adapter:                  17,162 (4.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       419,665 (100.0%)\n",
      "\n",
      "Total basepairs processed:   190,945,973 bp\n",
      "  Read 1:    93,762,738 bp\n",
      "  Read 2:    97,183,235 bp\n",
      "Total written (filtered):    190,703,708 bp (99.9%)\n",
      "  Read 1:    93,758,509 bp\n",
      "  Read 2:    96,945,199 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 382 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t360\t6557.3\t0\t360\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t5\t0.0\t1\t5\n",
      "100\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t7\t0.0\t1\t4 3\n",
      "217\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 17162 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t102\t6557.3\t0\t102\n",
      "4\t30\t1639.3\t0\t30\n",
      "5\t47\t409.8\t0\t47\n",
      "6\t74\t102.5\t0\t74\n",
      "7\t293\t25.6\t0\t293\n",
      "8\t434\t6.4\t0\t434\n",
      "9\t1695\t1.6\t0\t1690 5\n",
      "10\t5187\t0.4\t1\t5135 52\n",
      "11\t239\t0.1\t1\t232 7\n",
      "12\t167\t0.0\t1\t162 5\n",
      "13\t148\t0.0\t1\t145 3\n",
      "14\t384\t0.0\t1\t351 33\n",
      "15\t409\t0.0\t1\t391 18\n",
      "16\t1133\t0.0\t1\t1099 34\n",
      "17\t1728\t0.0\t1\t1662 66\n",
      "18\t1013\t0.0\t1\t922 65 26\n",
      "19\t3473\t0.0\t1\t2122 1333 18\n",
      "20\t12\t0.0\t2\t5 4 3\n",
      "21\t554\t0.0\t2\t0 549 5\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "40\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t7\t0.0\t2\t5 0 2\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "77\t2\t0.0\t2\t2\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t3\n",
      "186\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.95percent.5.31.19_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.95percent.5.31.19_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.95percent.5.31.19_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.95percent.5.31.19_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.71 s (14 us/read; 4.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            468,990\n",
      "  Read 1 with adapter:                     738 (0.2%)\n",
      "  Read 2 with adapter:                  42,485 (9.1%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       468,989 (100.0%)\n",
      "\n",
      "Total basepairs processed:   214,472,293 bp\n",
      "  Read 1:   105,024,443 bp\n",
      "  Read 2:   109,447,850 bp\n",
      "Total written (filtered):    213,956,617 bp (99.8%)\n",
      "  Read 1:   105,020,569 bp\n",
      "  Read 2:   108,936,048 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 738 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t714\t7328.0\t0\t714\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t9\t0.0\t1\t9\n",
      "51\t1\t0.0\t1\t1\n",
      "89\t3\t0.0\t1\t3\n",
      "92\t1\t0.0\t1\t1\n",
      "147\t1\t0.0\t1\t1\n",
      "163\t1\t0.0\t1\t1\n",
      "178\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 42485 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t447\t7328.0\t0\t447\n",
      "4\t240\t1832.0\t0\t240\n",
      "5\t427\t458.0\t0\t427\n",
      "6\t1092\t114.5\t0\t1092\n",
      "7\t2905\t28.6\t0\t2905\n",
      "8\t3940\t7.2\t0\t3940\n",
      "9\t9682\t1.8\t0\t9665 17\n",
      "10\t8302\t0.4\t1\t8197 105\n",
      "11\t330\t0.1\t1\t319 11\n",
      "12\t487\t0.0\t1\t479 8\n",
      "13\t724\t0.0\t1\t688 36\n",
      "14\t1163\t0.0\t1\t1087 76\n",
      "15\t808\t0.0\t1\t761 47\n",
      "16\t2104\t0.0\t1\t2020 84\n",
      "17\t2495\t0.0\t1\t2377 118\n",
      "18\t1490\t0.0\t1\t1289 115 86\n",
      "19\t5062\t0.0\t1\t2705 2328 29\n",
      "20\t26\t0.0\t2\t16 6 4\n",
      "21\t596\t0.0\t2\t0 583 13\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 1\n",
      "46\t1\t0.0\t2\t1\n",
      "48\t3\t0.0\t2\t0 2 1\n",
      "50\t1\t0.0\t2\t1\n",
      "54\t3\t0.0\t2\t3\n",
      "55\t3\t0.0\t2\t3\n",
      "56\t3\t0.0\t2\t3\n",
      "57\t5\t0.0\t2\t5\n",
      "58\t3\t0.0\t2\t2 0 1\n",
      "62\t3\t0.0\t2\t3\n",
      "63\t9\t0.0\t2\t9\n",
      "64\t5\t0.0\t2\t5\n",
      "65\t2\t0.0\t2\t2\n",
      "66\t2\t0.0\t2\t1 1\n",
      "67\t5\t0.0\t2\t4 1\n",
      "69\t1\t0.0\t2\t0 1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "72\t2\t0.0\t2\t1 1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t3\t0.0\t2\t3\n",
      "82\t1\t0.0\t2\t0 1\n",
      "83\t1\t0.0\t2\t1\n",
      "84\t1\t0.0\t2\t0 1\n",
      "85\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t0 1\n",
      "92\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t1\n",
      "125\t1\t0.0\t2\t1\n",
      "126\t1\t0.0\t2\t1\n",
      "133\t1\t0.0\t2\t1\n",
      "141\t1\t0.0\t2\t1\n",
      "170\t1\t0.0\t2\t1\n",
      "173\t2\t0.0\t2\t2\n",
      "174\t1\t0.0\t2\t1\n",
      "198\t1\t0.0\t2\t1\n",
      "210\t1\t0.0\t2\t1\n",
      "211\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t5\t0.0\t2\t4 0 1\n",
      "218\t2\t0.0\t2\t1 1\n",
      "229\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 0 1\n",
      "238\t7\t0.0\t2\t4 1 2\n",
      "239\t7\t0.0\t2\t3 2 2\n",
      "240\t5\t0.0\t2\t3 1 1\n",
      "241\t5\t0.0\t2\t4 0 1\n",
      "242\t6\t0.0\t2\t2 2 2\n",
      "243\t4\t0.0\t2\t2 2\n",
      "244\t1\t0.0\t2\t0 1\n",
      "245\t2\t0.0\t2\t1 1\n",
      "246\t2\t0.0\t2\t1 1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D4.4.26.19_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D4.4.26.19_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D4.4.26.19_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D4.4.26.19_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.26 s (19 us/read; 3.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            219,029\n",
      "  Read 1 with adapter:                     203 (0.1%)\n",
      "  Read 2 with adapter:                   7,201 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       219,029 (100.0%)\n",
      "\n",
      "Total basepairs processed:    99,678,490 bp\n",
      "  Read 1:    48,966,648 bp\n",
      "  Read 2:    50,711,842 bp\n",
      "Total written (filtered):     99,564,751 bp (99.9%)\n",
      "  Read 1:    48,964,549 bp\n",
      "  Read 2:    50,600,202 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 203 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t189\t3422.3\t0\t189\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "87\t2\t0.0\t1\t2\n",
      "88\t1\t0.0\t1\t1\n",
      "89\t2\t0.0\t1\t2\n",
      "155\t1\t0.0\t1\t1\n",
      "179\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7201 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t3422.3\t0\t40\n",
      "4\t22\t855.6\t0\t22\n",
      "5\t15\t213.9\t0\t15\n",
      "6\t32\t53.5\t0\t32\n",
      "7\t92\t13.4\t0\t92\n",
      "8\t171\t3.3\t0\t171\n",
      "9\t532\t0.8\t0\t531 1\n",
      "10\t899\t0.2\t1\t885 14\n",
      "11\t87\t0.1\t1\t84 3\n",
      "12\t199\t0.0\t1\t198 1\n",
      "13\t233\t0.0\t1\t221 12\n",
      "14\t300\t0.0\t1\t278 22\n",
      "15\t189\t0.0\t1\t178 11\n",
      "16\t601\t0.0\t1\t578 23\n",
      "17\t839\t0.0\t1\t809 30\n",
      "18\t514\t0.0\t1\t456 36 22\n",
      "19\t2081\t0.0\t1\t1234 840 7\n",
      "20\t4\t0.0\t2\t2 0 2\n",
      "21\t318\t0.0\t2\t0 316 2\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t0 1\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t9\t0.0\t2\t8 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t2\n",
      "100\t1\t0.0\t2\t0 1\n",
      "114\t1\t0.0\t2\t1\n",
      "151\t1\t0.0\t2\t1\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D4.5.4.19_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D4.5.4.19_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D4.5.4.19_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D4.5.4.19_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.05 s (13 us/read; 4.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            396,841\n",
      "  Read 1 with adapter:                     384 (0.1%)\n",
      "  Read 2 with adapter:                  15,560 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       396,841 (100.0%)\n",
      "\n",
      "Total basepairs processed:   180,661,517 bp\n",
      "  Read 1:    88,726,495 bp\n",
      "  Read 2:    91,935,022 bp\n",
      "Total written (filtered):    180,445,139 bp (99.9%)\n",
      "  Read 1:    88,723,584 bp\n",
      "  Read 2:    91,721,555 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 384 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t369\t6200.6\t0\t369\n",
      "22\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t0 1\n",
      "53\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t3 1\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15560 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t6200.6\t0\t109\n",
      "4\t41\t1550.2\t0\t41\n",
      "5\t41\t387.5\t0\t41\n",
      "6\t125\t96.9\t0\t125\n",
      "7\t328\t24.2\t0\t328\n",
      "8\t503\t6.1\t0\t503\n",
      "9\t1712\t1.5\t0\t1709 3\n",
      "10\t4575\t0.4\t1\t4536 39\n",
      "11\t209\t0.1\t1\t203 6\n",
      "12\t162\t0.0\t1\t159 3\n",
      "13\t164\t0.0\t1\t156 8\n",
      "14\t317\t0.0\t1\t296 21\n",
      "15\t325\t0.0\t1\t310 15\n",
      "16\t1021\t0.0\t1\t992 29\n",
      "17\t1385\t0.0\t1\t1352 33\n",
      "18\t910\t0.0\t1\t842 53 15\n",
      "19\t3148\t0.0\t1\t1913 1223 12\n",
      "20\t5\t0.0\t2\t3 2\n",
      "21\t443\t0.0\t2\t0 436 7\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "217\t4\t0.0\t2\t4\n",
      "236\t1\t0.0\t2\t1\n",
      "242\t3\t0.0\t2\t2 1\n",
      "247\t2\t0.0\t2\t0 2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.4.26.19_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.4.26.19_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.4.26.19_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.4.26.19_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 8.39 s (13 us/read; 4.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            653,142\n",
      "  Read 1 with adapter:                     804 (0.1%)\n",
      "  Read 2 with adapter:                  30,778 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       653,142 (100.0%)\n",
      "\n",
      "Total basepairs processed:   297,965,016 bp\n",
      "  Read 1:   146,269,513 bp\n",
      "  Read 2:   151,695,503 bp\n",
      "Total written (filtered):    297,557,959 bp (99.9%)\n",
      "  Read 1:   146,265,427 bp\n",
      "  Read 2:   151,292,532 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 804 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t775\t10205.3\t0\t775\n",
      "22\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t3\t0.0\t1\t3\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t3\t0.0\t1\t2 1\n",
      "29\t4\t0.0\t1\t4\n",
      "30\t1\t0.0\t1\t0 1\n",
      "92\t2\t0.0\t1\t2\n",
      "137\t1\t0.0\t1\t1\n",
      "156\t1\t0.0\t1\t1\n",
      "157\t1\t0.0\t1\t1\n",
      "176\t1\t0.0\t1\t0 1\n",
      "185\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 30778 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t248\t10205.3\t0\t248\n",
      "4\t107\t2551.3\t0\t107\n",
      "5\t116\t637.8\t0\t116\n",
      "6\t306\t159.5\t0\t306\n",
      "7\t924\t39.9\t0\t924\n",
      "8\t1527\t10.0\t0\t1527\n",
      "9\t5149\t2.5\t0\t5131 18\n",
      "10\t8385\t0.6\t1\t8299 86\n",
      "11\t285\t0.2\t1\t275 10\n",
      "12\t264\t0.0\t1\t258 6\n",
      "13\t344\t0.0\t1\t334 10\n",
      "14\t551\t0.0\t1\t518 33\n",
      "15\t531\t0.0\t1\t512 19\n",
      "16\t1663\t0.0\t1\t1618 45\n",
      "17\t2490\t0.0\t1\t2411 79\n",
      "18\t1609\t0.0\t1\t1422 136 51\n",
      "19\t5380\t0.0\t1\t3104 2253 23\n",
      "20\t21\t0.0\t2\t9 7 5\n",
      "21\t793\t0.0\t2\t0 789 4\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "23\t1\t0.0\t2\t1\n",
      "25\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t2\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t1 0 1\n",
      "70\t4\t0.0\t2\t3 1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "79\t7\t0.0\t2\t6 1\n",
      "84\t1\t0.0\t2\t1\n",
      "86\t1\t0.0\t2\t1\n",
      "89\t2\t0.0\t2\t2\n",
      "100\t3\t0.0\t2\t3\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "119\t1\t0.0\t2\t0 1\n",
      "128\t1\t0.0\t2\t0 1\n",
      "143\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "181\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t3\n",
      "217\t7\t0.0\t2\t6 1\n",
      "223\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.F.5.4.19_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.F.5.4.19_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.F.5.4.19_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.F.5.4.19_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.41 s (14 us/read; 4.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            460,533\n",
      "  Read 1 with adapter:                     532 (0.1%)\n",
      "  Read 2 with adapter:                  18,411 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       460,533 (100.0%)\n",
      "\n",
      "Total basepairs processed:   209,961,878 bp\n",
      "  Read 1:   103,116,984 bp\n",
      "  Read 2:   106,844,894 bp\n",
      "Total written (filtered):    209,699,209 bp (99.9%)\n",
      "  Read 1:   103,114,582 bp\n",
      "  Read 2:   106,584,627 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 532 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t508\t7195.8\t0\t508\n",
      "20\t1\t0.0\t1\t1\n",
      "24\t5\t0.0\t1\t5\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t6\t0.0\t1\t6\n",
      "89\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18411 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t154\t7195.8\t0\t154\n",
      "4\t43\t1799.0\t0\t43\n",
      "5\t57\t449.7\t0\t57\n",
      "6\t109\t112.4\t0\t109\n",
      "7\t346\t28.1\t0\t346\n",
      "8\t479\t7.0\t0\t479\n",
      "9\t1856\t1.8\t0\t1851 5\n",
      "10\t5112\t0.4\t1\t5048 64\n",
      "11\t220\t0.1\t1\t211 9\n",
      "12\t178\t0.0\t1\t175 3\n",
      "13\t196\t0.0\t1\t189 7\n",
      "14\t393\t0.0\t1\t354 39\n",
      "15\t397\t0.0\t1\t379 18\n",
      "16\t1219\t0.0\t1\t1178 41\n",
      "17\t1823\t0.0\t1\t1774 49\n",
      "18\t1125\t0.0\t1\t1036 69 20\n",
      "19\t4023\t0.0\t1\t2453 1549 21\n",
      "20\t16\t0.0\t2\t10 4 2\n",
      "21\t606\t0.0\t2\t0 597 9\n",
      "22\t16\t0.0\t2\t0 1 15\n",
      "27\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t3\t0.0\t2\t1 1 1\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "74\t2\t0.0\t2\t2\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t5\t0.0\t2\t3 1 1\n",
      "87\t1\t0.0\t2\t0 1\n",
      "88\t2\t0.0\t2\t1 1\n",
      "105\t1\t0.0\t2\t1\n",
      "161\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "189\t1\t0.0\t2\t0 1\n",
      "210\t1\t0.0\t2\t1\n",
      "215\t2\t0.0\t2\t2\n",
      "216\t2\t0.0\t2\t0 2\n",
      "217\t5\t0.0\t2\t5\n",
      "236\t2\t0.0\t2\t1 1\n",
      "244\t1\t0.0\t2\t0 0 1\n",
      "248\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.N.5.4.19_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.N.5.4.19_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.N.5.4.19_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.N.5.4.19_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.56 s (16 us/read; 3.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            417,468\n",
      "  Read 1 with adapter:                     442 (0.1%)\n",
      "  Read 2 with adapter:                  36,796 (8.8%)\n",
      "Pairs that were too short:                   2 (0.0%)\n",
      "Pairs written (passing filters):       417,466 (100.0%)\n",
      "\n",
      "Total basepairs processed:   190,871,957 bp\n",
      "  Read 1:    93,475,035 bp\n",
      "  Read 2:    97,396,922 bp\n",
      "Total written (filtered):    190,407,976 bp (99.8%)\n",
      "  Read 1:    93,471,892 bp\n",
      "  Read 2:    96,936,084 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 442 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t421\t6522.9\t0\t421\n",
      "8\t1\t6.4\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t7\t0.0\t1\t7\n",
      "32\t1\t0.0\t1\t1\n",
      "51\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "101\t4\t0.0\t1\t4\n",
      "102\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "195\t1\t0.0\t1\t1\n",
      "202\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 36796 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t344\t6522.9\t0\t344\n",
      "4\t182\t1630.7\t0\t182\n",
      "5\t319\t407.7\t0\t319\n",
      "6\t759\t101.9\t0\t759\n",
      "7\t2028\t25.5\t0\t2028\n",
      "8\t3029\t6.4\t0\t3029\n",
      "9\t7972\t1.6\t0\t7956 16\n",
      "10\t7592\t0.4\t1\t7519 73\n",
      "11\t379\t0.1\t1\t370 9\n",
      "12\t518\t0.0\t1\t500 18\n",
      "13\t788\t0.0\t1\t753 35\n",
      "14\t1188\t0.0\t1\t1132 56\n",
      "15\t771\t0.0\t1\t720 51\n",
      "16\t2047\t0.0\t1\t1976 71\n",
      "17\t2247\t0.0\t1\t2169 78\n",
      "18\t1380\t0.0\t1\t1172 100 108\n",
      "19\t4491\t0.0\t1\t2433 2038 20\n",
      "20\t22\t0.0\t2\t15 5 2\n",
      "21\t550\t0.0\t2\t0 541 9\n",
      "22\t29\t0.0\t2\t0 2 27\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t1\n",
      "54\t3\t0.0\t2\t3\n",
      "55\t2\t0.0\t2\t2\n",
      "56\t1\t0.0\t2\t0 1\n",
      "57\t3\t0.0\t2\t3\n",
      "58\t4\t0.0\t2\t2 1 1\n",
      "59\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t6\t0.0\t2\t5 1\n",
      "64\t4\t0.0\t2\t4\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t2\t0.0\t2\t1 0 1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t4\t0.0\t2\t3 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t7\t0.0\t2\t6 1\n",
      "82\t1\t0.0\t2\t1\n",
      "84\t1\t0.0\t2\t1\n",
      "87\t2\t0.0\t2\t1 1\n",
      "88\t5\t0.0\t2\t4 0 1\n",
      "89\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "93\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "95\t2\t0.0\t2\t2\n",
      "96\t1\t0.0\t2\t1\n",
      "99\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t1\n",
      "125\t1\t0.0\t2\t1\n",
      "128\t1\t0.0\t2\t1\n",
      "131\t1\t0.0\t2\t1\n",
      "140\t1\t0.0\t2\t1\n",
      "142\t1\t0.0\t2\t1\n",
      "146\t1\t0.0\t2\t1\n",
      "147\t1\t0.0\t2\t1\n",
      "148\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t1\n",
      "160\t1\t0.0\t2\t1\n",
      "161\t2\t0.0\t2\t2\n",
      "170\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t0 1\n",
      "180\t1\t0.0\t2\t1\n",
      "183\t1\t0.0\t2\t1\n",
      "184\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "199\t1\t0.0\t2\t1\n",
      "200\t1\t0.0\t2\t1\n",
      "207\t2\t0.0\t2\t0 1 1\n",
      "215\t3\t0.0\t2\t2 1\n",
      "216\t3\t0.0\t2\t2 1\n",
      "226\t1\t0.0\t2\t1\n",
      "233\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t1\n",
      "236\t2\t0.0\t2\t2\n",
      "237\t6\t0.0\t2\t6\n",
      "238\t4\t0.0\t2\t3 1\n",
      "239\t3\t0.0\t2\t2 0 1\n",
      "240\t6\t0.0\t2\t6\n",
      "241\t11\t0.0\t2\t7 2 2\n",
      "242\t6\t0.0\t2\t5 1\n",
      "243\t3\t0.0\t2\t3\n",
      "244\t3\t0.0\t2\t3\n",
      "245\t1\t0.0\t2\t1\n",
      "247\t1\t0.0\t2\t1\n",
      "248\t2\t0.0\t2\t1 1\n",
      "249\t2\t0.0\t2\t2\n",
      "250\t3\t0.0\t2\t2 0 1\n",
      "251\t1\t0.0\t2\t0 0 1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.4.26.19_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.4.26.19_58_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.4.26.19_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.4.26.19_58_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.F.5.4.19_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.F.5.4.19_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.F.5.4.19_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.F.5.4.19_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.N.5.4.19_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D1.N.5.4.19_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.N.5.4.19_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D1.N.5.4.19_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D2.4.26.19_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D2.4.26.19_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D2.4.26.19_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D2.4.26.19_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D2.5.4.19_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D2.5.4.19_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D2.5.4.19_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D2.5.4.19_2_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.4.26.19_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.4.26.19_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.4.26.19_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.4.26.19_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.5.4.19_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.5.4.19_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.5.4.19_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.5.4.19_27_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.95percent.5.31.19_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D3.95percent.5.31.19_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.95percent.5.31.19_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D3.95percent.5.31.19_17_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D4.4.26.19_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D4.4.26.19_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D4.4.26.19_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D4.4.26.19_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D4.5.4.19_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCA.D4.5.4.19_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D4.5.4.19_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCA.D4.5.4.19_16_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.4.26.19_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.4.26.19_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.4.26.19_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.4.26.19_35_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.F.5.4.19_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.F.5.4.19_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.F.5.4.19_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.F.5.4.19_9_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.N.5.4.19_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D1.N.5.4.19_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.N.5.4.19_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D1.N.5.4.19_20_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.4.26.19_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.4.26.19_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.4.26.19_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.4.26.19_48_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.4.26.19_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.4.26.19_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.4.26.19_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.4.26.19_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.84 s (16 us/read; 3.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            240,507\n",
      "  Read 1 with adapter:                     248 (0.1%)\n",
      "  Read 2 with adapter:                   6,302 (2.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       240,507 (100.0%)\n",
      "\n",
      "Total basepairs processed:   109,487,972 bp\n",
      "  Read 1:    53,803,387 bp\n",
      "  Read 2:    55,684,585 bp\n",
      "Total written (filtered):    109,390,591 bp (99.9%)\n",
      "  Read 1:    53,801,988 bp\n",
      "  Read 2:    55,588,603 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 248 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t236\t3757.9\t0\t236\n",
      "24\t3\t0.0\t1\t3\n",
      "25\t2\t0.0\t1\t1 1\n",
      "26\t3\t0.0\t1\t2 1\n",
      "29\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6302 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t3757.9\t0\t27\n",
      "4\t17\t939.5\t0\t17\n",
      "5\t25\t234.9\t0\t25\n",
      "6\t31\t58.7\t0\t31\n",
      "7\t112\t14.7\t0\t112\n",
      "8\t159\t3.7\t0\t159\n",
      "9\t482\t0.9\t0\t480 2\n",
      "10\t1210\t0.2\t1\t1193 17\n",
      "11\t90\t0.1\t1\t89 1\n",
      "12\t85\t0.0\t1\t85\n",
      "13\t83\t0.0\t1\t80 3\n",
      "14\t134\t0.0\t1\t128 6\n",
      "15\t154\t0.0\t1\t149 5\n",
      "16\t456\t0.0\t1\t445 11\n",
      "17\t714\t0.0\t1\t686 28\n",
      "18\t472\t0.0\t1\t444 21 7\n",
      "19\t1753\t0.0\t1\t1131 614 8\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t274\t0.0\t2\t0 272 2\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "48\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t4\t0.0\t2\t4\n",
      "217\t2\t0.0\t2\t2\n",
      "230\t1\t0.0\t2\t0 0 1\n",
      "236\t1\t0.0\t2\t0 1\n",
      "237\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.5.4.19_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.5.4.19_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.5.4.19_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.5.4.19_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.40 s (17 us/read; 3.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            252,616\n",
      "  Read 1 with adapter:                     164 (0.1%)\n",
      "  Read 2 with adapter:                   5,608 (2.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       252,616 (100.0%)\n",
      "\n",
      "Total basepairs processed:   114,986,632 bp\n",
      "  Read 1:    56,521,145 bp\n",
      "  Read 2:    58,465,487 bp\n",
      "Total written (filtered):    114,896,481 bp (99.9%)\n",
      "  Read 1:    56,520,203 bp\n",
      "  Read 2:    58,376,278 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 164 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t157\t3947.1\t0\t157\n",
      "15\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t1\t0.0\t1\t1\n",
      "131\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5608 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t3947.1\t0\t22\n",
      "4\t15\t986.8\t0\t15\n",
      "5\t20\t246.7\t0\t20\n",
      "6\t20\t61.7\t0\t20\n",
      "7\t74\t15.4\t0\t74\n",
      "8\t83\t3.9\t0\t83\n",
      "9\t276\t1.0\t0\t275 1\n",
      "10\t862\t0.2\t1\t855 7\n",
      "11\t82\t0.1\t1\t81 1\n",
      "12\t117\t0.0\t1\t117\n",
      "13\t56\t0.0\t1\t54 2\n",
      "14\t111\t0.0\t1\t104 7\n",
      "15\t136\t0.0\t1\t133 3\n",
      "16\t415\t0.0\t1\t401 14\n",
      "17\t653\t0.0\t1\t631 22\n",
      "18\t489\t0.0\t1\t451 26 12\n",
      "19\t1835\t0.0\t1\t1118 709 8\n",
      "20\t6\t0.0\t2\t2 2 2\n",
      "21\t320\t0.0\t2\t0 313 7\n",
      "22\t5\t0.0\t2\t0 1 4\n",
      "70\t1\t0.0\t2\t0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "100\t3\t0.0\t2\t1 2\n",
      "205\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t0 1\n",
      "210\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "230\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.5percent.5.31.19_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.5percent.5.31.19_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.5percent.5.31.19_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.5percent.5.31.19_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.09 s (14 us/read; 4.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            508,682\n",
      "  Read 1 with adapter:                     547 (0.1%)\n",
      "  Read 2 with adapter:                  27,197 (5.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       508,682 (100.0%)\n",
      "\n",
      "Total basepairs processed:   232,061,089 bp\n",
      "  Read 1:   113,916,431 bp\n",
      "  Read 2:   118,144,658 bp\n",
      "Total written (filtered):    231,686,508 bp (99.8%)\n",
      "  Read 1:   113,914,072 bp\n",
      "  Read 2:   117,772,436 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 547 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t535\t7948.2\t0\t535\n",
      "20\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t2\t0.0\t1\t2\n",
      "29\t3\t0.0\t1\t3\n",
      "100\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "221\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 27197 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t233\t7948.2\t0\t233\n",
      "4\t70\t1987.0\t0\t70\n",
      "5\t104\t496.8\t0\t104\n",
      "6\t215\t124.2\t0\t215\n",
      "7\t624\t31.0\t0\t624\n",
      "8\t929\t7.8\t0\t929\n",
      "9\t3291\t1.9\t0\t3282 9\n",
      "10\t7875\t0.5\t1\t7791 84\n",
      "11\t277\t0.1\t1\t267 10\n",
      "12\t256\t0.0\t1\t251 5\n",
      "13\t274\t0.0\t1\t266 8\n",
      "14\t568\t0.0\t1\t529 39\n",
      "15\t515\t0.0\t1\t490 25\n",
      "16\t1523\t0.0\t1\t1471 52\n",
      "17\t2368\t0.0\t1\t2302 66\n",
      "18\t1375\t0.0\t1\t1246 103 26\n",
      "19\t4670\t0.0\t1\t2815 1827 28\n",
      "20\t1280\t0.0\t2\t1257 15 8\n",
      "21\t640\t0.0\t2\t0 630 10\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "40\t38\t0.0\t2\t37 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 1\n",
      "60\t2\t0.0\t2\t1 0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t3\t0.0\t2\t2 0 1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t3\t0.0\t2\t3\n",
      "73\t1\t0.0\t2\t1\n",
      "79\t9\t0.0\t2\t6 2 1\n",
      "80\t2\t0.0\t2\t0 2\n",
      "84\t1\t0.0\t2\t1\n",
      "85\t1\t0.0\t2\t1\n",
      "100\t3\t0.0\t2\t2 1\n",
      "150\t1\t0.0\t2\t1\n",
      "161\t1\t0.0\t2\t1\n",
      "206\t1\t0.0\t2\t0 1\n",
      "209\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t5\t0.0\t2\t4 1\n",
      "235\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.4.26.19_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.4.26.19_60_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.4.26.19_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.4.26.19_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.94 s (16 us/read; 3.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            305,444\n",
      "  Read 1 with adapter:                     214 (0.1%)\n",
      "  Read 2 with adapter:                   7,219 (2.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       305,444 (100.0%)\n",
      "\n",
      "Total basepairs processed:   139,059,486 bp\n",
      "  Read 1:    68,348,056 bp\n",
      "  Read 2:    70,711,430 bp\n",
      "Total written (filtered):    138,944,785 bp (99.9%)\n",
      "  Read 1:    68,346,921 bp\n",
      "  Read 2:    70,597,864 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 214 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t201\t4772.6\t0\t201\n",
      "25\t6\t0.0\t1\t4 2\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7219 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t4772.6\t0\t33\n",
      "4\t11\t1193.1\t0\t11\n",
      "5\t11\t298.3\t0\t11\n",
      "6\t36\t74.6\t0\t36\n",
      "7\t96\t18.6\t0\t96\n",
      "8\t116\t4.7\t0\t116\n",
      "9\t401\t1.2\t0\t401\n",
      "10\t1307\t0.3\t1\t1298 9\n",
      "11\t108\t0.1\t1\t106 2\n",
      "12\t81\t0.0\t1\t81\n",
      "13\t71\t0.0\t1\t70 1\n",
      "14\t141\t0.0\t1\t130 11\n",
      "15\t180\t0.0\t1\t172 8\n",
      "16\t522\t0.0\t1\t504 18\n",
      "17\t881\t0.0\t1\t861 20\n",
      "18\t579\t0.0\t1\t533 32 14\n",
      "19\t2223\t0.0\t1\t1398 811 14\n",
      "20\t5\t0.0\t2\t2 3\n",
      "21\t376\t0.0\t2\t0 373 3\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "56\t3\t0.0\t2\t3\n",
      "65\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t0 1\n",
      "70\t3\t0.0\t2\t1 2\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t0 1\n",
      "79\t9\t0.0\t2\t9\n",
      "88\t1\t0.0\t2\t1\n",
      "124\t1\t0.0\t2\t1\n",
      "146\t1\t0.0\t2\t1\n",
      "166\t2\t0.0\t2\t2\n",
      "209\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.5percent.5.31.19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.5percent.5.31.19_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.5percent.5.31.19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.5percent.5.31.19_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.23 s (15 us/read; 4.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            483,008\n",
      "  Read 1 with adapter:                     465 (0.1%)\n",
      "  Read 2 with adapter:                  26,721 (5.5%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       483,007 (100.0%)\n",
      "\n",
      "Total basepairs processed:   220,393,193 bp\n",
      "  Read 1:   108,157,356 bp\n",
      "  Read 2:   112,235,837 bp\n",
      "Total written (filtered):    220,037,910 bp (99.8%)\n",
      "  Read 1:   108,155,260 bp\n",
      "  Read 2:   111,882,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 465 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t453\t7547.0\t0\t453\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t5\t0.0\t1\t5\n",
      "29\t3\t0.0\t1\t3\n",
      "89\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 26721 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t260\t7547.0\t0\t260\n",
      "4\t69\t1886.8\t0\t69\n",
      "5\t90\t471.7\t0\t90\n",
      "6\t222\t117.9\t0\t222\n",
      "7\t777\t29.5\t0\t777\n",
      "8\t1015\t7.4\t0\t1015\n",
      "9\t3621\t1.8\t0\t3611 10\n",
      "10\t8071\t0.5\t1\t7978 93\n",
      "11\t313\t0.1\t1\t307 6\n",
      "12\t288\t0.0\t1\t277 11\n",
      "13\t332\t0.0\t1\t321 11\n",
      "14\t654\t0.0\t1\t619 35\n",
      "15\t596\t0.0\t1\t572 24\n",
      "16\t1630\t0.0\t1\t1561 69\n",
      "17\t2174\t0.0\t1\t2094 80\n",
      "18\t1349\t0.0\t1\t1242 80 27\n",
      "19\t4527\t0.0\t1\t2674 1834 19\n",
      "20\t18\t0.0\t2\t9 5 4\n",
      "21\t623\t0.0\t2\t0 609 14\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "67\t2\t0.0\t2\t1 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t3\t0.0\t2\t2 0 1\n",
      "71\t7\t0.0\t2\t0 0 7\n",
      "72\t2\t0.0\t2\t1 1\n",
      "78\t5\t0.0\t2\t5\n",
      "79\t19\t0.0\t2\t18 1\n",
      "91\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t0 1\n",
      "111\t1\t0.0\t2\t0 1\n",
      "166\t1\t0.0\t2\t0 1\n",
      "186\t2\t0.0\t2\t1 1\n",
      "215\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t0 0 1\n",
      "241\t3\t0.0\t2\t3\n",
      "242\t2\t0.0\t2\t1 1\n",
      "251\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.F.5.4.19_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.F.5.4.19_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.F.5.4.19_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.F.5.4.19_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.32 s (14 us/read; 4.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            453,780\n",
      "  Read 1 with adapter:                     548 (0.1%)\n",
      "  Read 2 with adapter:                  18,931 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       453,780 (100.0%)\n",
      "\n",
      "Total basepairs processed:   206,854,282 bp\n",
      "  Read 1:   101,585,729 bp\n",
      "  Read 2:   105,268,553 bp\n",
      "Total written (filtered):    206,595,263 bp (99.9%)\n",
      "  Read 1:   101,582,655 bp\n",
      "  Read 2:   105,012,608 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 548 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t532\t7090.3\t0\t532\n",
      "24\t2\t0.0\t1\t2\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t4\t0.0\t1\t4\n",
      "89\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "131\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "175\t1\t0.0\t1\t0 1\n",
      "197\t1\t0.0\t1\t1\n",
      "198\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18931 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t124\t7090.3\t0\t124\n",
      "4\t49\t1772.6\t0\t49\n",
      "5\t69\t443.1\t0\t69\n",
      "6\t100\t110.8\t0\t100\n",
      "7\t373\t27.7\t0\t373\n",
      "8\t583\t6.9\t0\t583\n",
      "9\t2115\t1.7\t0\t2101 14\n",
      "10\t6045\t0.4\t1\t5990 55\n",
      "11\t250\t0.1\t1\t249 1\n",
      "12\t199\t0.0\t1\t189 10\n",
      "13\t128\t0.0\t1\t124 4\n",
      "14\t352\t0.0\t1\t323 29\n",
      "15\t373\t0.0\t1\t358 15\n",
      "16\t1102\t0.0\t1\t1062 40\n",
      "17\t1712\t0.0\t1\t1661 51\n",
      "18\t989\t0.0\t1\t906 63 20\n",
      "19\t3774\t0.0\t1\t2409 1344 21\n",
      "20\t12\t0.0\t2\t5 2 5\n",
      "21\t544\t0.0\t2\t0 539 5\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "199\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t4\t0.0\t2\t4\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.N.5.4.19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.N.5.4.19_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.N.5.4.19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.N.5.4.19_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.89 s (16 us/read; 3.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            359,375\n",
      "  Read 1 with adapter:                     414 (0.1%)\n",
      "  Read 2 with adapter:                  30,299 (8.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       359,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:   164,274,466 bp\n",
      "  Read 1:    80,468,597 bp\n",
      "  Read 2:    83,805,869 bp\n",
      "Total written (filtered):    163,904,117 bp (99.8%)\n",
      "  Read 1:    80,466,685 bp\n",
      "  Read 2:    83,437,432 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 414 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t396\t5615.2\t0\t396\n",
      "21\t1\t0.0\t1\t1\n",
      "27\t4\t0.0\t1\t4\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t8\t0.0\t1\t8\n",
      "56\t1\t0.0\t1\t1\n",
      "62\t2\t0.0\t1\t2\n",
      "155\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 30299 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t312\t5615.2\t0\t312\n",
      "4\t181\t1403.8\t0\t181\n",
      "5\t332\t351.0\t0\t332\n",
      "6\t799\t87.7\t0\t799\n",
      "7\t1940\t21.9\t0\t1940\n",
      "8\t2795\t5.5\t0\t2795\n",
      "9\t6697\t1.4\t0\t6685 12\n",
      "10\t5623\t0.3\t1\t5539 84\n",
      "11\t282\t0.1\t1\t272 10\n",
      "12\t380\t0.0\t1\t368 12\n",
      "13\t603\t0.0\t1\t582 21\n",
      "14\t844\t0.0\t1\t810 34\n",
      "15\t601\t0.0\t1\t562 39\n",
      "16\t1517\t0.0\t1\t1445 72\n",
      "17\t1895\t0.0\t1\t1792 103\n",
      "18\t1198\t0.0\t1\t1028 94 76\n",
      "19\t3677\t0.0\t1\t2027 1635 15\n",
      "20\t15\t0.0\t2\t7 6 2\n",
      "21\t472\t0.0\t2\t0 463 9\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "32\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "49\t3\t0.0\t2\t2 1\n",
      "52\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "56\t2\t0.0\t2\t1 0 1\n",
      "57\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "60\t2\t0.0\t2\t2\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t4\t0.0\t2\t4\n",
      "64\t4\t0.0\t2\t4\n",
      "65\t3\t0.0\t2\t3\n",
      "66\t3\t0.0\t2\t2 1\n",
      "67\t3\t0.0\t2\t2 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t3\t0.0\t2\t2 0 1\n",
      "70\t8\t0.0\t2\t7 1\n",
      "71\t6\t0.0\t2\t2 0 4\n",
      "73\t1\t0.0\t2\t1\n",
      "74\t2\t0.0\t2\t1 1\n",
      "79\t15\t0.0\t2\t14 1\n",
      "84\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t0 1\n",
      "91\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t1\n",
      "125\t1\t0.0\t2\t1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t1 1\n",
      "145\t1\t0.0\t2\t1\n",
      "146\t1\t0.0\t2\t1\n",
      "147\t2\t0.0\t2\t2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "166\t3\t0.0\t2\t2 1\n",
      "184\t1\t0.0\t2\t1\n",
      "185\t2\t0.0\t2\t1 0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "194\t1\t0.0\t2\t1\n",
      "197\t1\t0.0\t2\t0 1\n",
      "203\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t0 1\n",
      "210\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "235\t1\t0.0\t2\t1\n",
      "237\t3\t0.0\t2\t3\n",
      "238\t2\t0.0\t2\t1 1\n",
      "239\t2\t0.0\t2\t1 0 1\n",
      "240\t1\t0.0\t2\t1\n",
      "241\t1\t0.0\t2\t0 0 1\n",
      "242\t1\t0.0\t2\t1\n",
      "243\t1\t0.0\t2\t1\n",
      "245\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D4.4.26.19_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D4.4.26.19_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D4.4.26.19_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D4.4.26.19_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.20 s (16 us/read; 3.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            376,359\n",
      "  Read 1 with adapter:                     355 (0.1%)\n",
      "  Read 2 with adapter:                  10,162 (2.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       376,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:   171,213,357 bp\n",
      "  Read 1:    84,139,790 bp\n",
      "  Read 2:    87,073,567 bp\n",
      "Total written (filtered):    171,056,388 bp (99.9%)\n",
      "  Read 1:    84,136,394 bp\n",
      "  Read 2:    86,919,994 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 355 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t326\t5880.6\t0\t326\n",
      "9\t1\t1.4\t0\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t3\t0.0\t1\t2 1\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t9\t0.0\t1\t9\n",
      "89\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t1\n",
      "175\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10162 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t5880.6\t0\t53\n",
      "4\t19\t1470.2\t0\t19\n",
      "5\t25\t367.5\t0\t25\n",
      "6\t54\t91.9\t0\t54\n",
      "7\t126\t23.0\t0\t126\n",
      "8\t193\t5.7\t0\t193\n",
      "9\t650\t1.4\t0\t647 3\n",
      "10\t2302\t0.4\t1\t2274 28\n",
      "11\t182\t0.1\t1\t178 4\n",
      "12\t171\t0.0\t1\t169 2\n",
      "13\t100\t0.0\t1\t96 4\n",
      "14\t231\t0.0\t1\t218 13\n",
      "15\t227\t0.0\t1\t219 8\n",
      "16\t740\t0.0\t1\t715 25\n",
      "17\t1135\t0.0\t1\t1105 30\n",
      "18\t733\t0.0\t1\t679 41 13\n",
      "19\t2717\t0.0\t1\t1662 1047 8\n",
      "20\t7\t0.0\t2\t1 3 3\n",
      "21\t454\t0.0\t2\t0 450 4\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "63\t2\t0.0\t2\t2\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 1\n",
      "73\t1\t0.0\t2\t1\n",
      "79\t10\t0.0\t2\t9 0 1\n",
      "92\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t6\t0.0\t2\t5 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D4.5.4.19_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D4.5.4.19_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D4.5.4.19_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D4.5.4.19_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.40 s (14 us/read; 4.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            457,941\n",
      "  Read 1 with adapter:                     538 (0.1%)\n",
      "  Read 2 with adapter:                  22,600 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       457,941 (100.0%)\n",
      "\n",
      "Total basepairs processed:   208,882,212 bp\n",
      "  Read 1:   102,547,941 bp\n",
      "  Read 2:   106,334,271 bp\n",
      "Total written (filtered):    208,582,169 bp (99.9%)\n",
      "  Read 1:   102,545,286 bp\n",
      "  Read 2:   106,036,883 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 538 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t518\t7155.3\t0\t518\n",
      "18\t1\t0.0\t1\t0 1\n",
      "21\t2\t0.0\t1\t2\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "29\t9\t0.0\t1\t8 1\n",
      "92\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "150\t1\t0.0\t1\t1\n",
      "179\t1\t0.0\t1\t0 1\n",
      "186\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 22600 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t150\t7155.3\t0\t150\n",
      "4\t51\t1788.8\t0\t51\n",
      "5\t71\t447.2\t0\t71\n",
      "6\t155\t111.8\t0\t155\n",
      "7\t508\t28.0\t0\t508\n",
      "8\t768\t7.0\t0\t768\n",
      "9\t2902\t1.7\t0\t2890 12\n",
      "10\t7522\t0.4\t1\t7440 82\n",
      "11\t290\t0.1\t1\t287 3\n",
      "12\t221\t0.0\t1\t213 8\n",
      "13\t195\t0.0\t1\t188 7\n",
      "14\t458\t0.0\t1\t433 25\n",
      "15\t457\t0.0\t1\t436 21\n",
      "16\t1315\t0.0\t1\t1287 28\n",
      "17\t1768\t0.0\t1\t1700 68\n",
      "18\t1189\t0.0\t1\t1089 78 22\n",
      "19\t3900\t0.0\t1\t2374 1512 14\n",
      "20\t14\t0.0\t2\t7 5 2\n",
      "21\t622\t0.0\t2\t0 611 11\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "40\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "67\t3\t0.0\t2\t3\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t1 1\n",
      "96\t1\t0.0\t2\t1\n",
      "105\t2\t0.0\t2\t2\n",
      "210\t2\t0.0\t2\t1 1\n",
      "213\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t1 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.4.26.19_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.4.26.19_69_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.4.26.19_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.4.26.19_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 0.31 s (1191 us/read; 0.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                258\n",
      "  Read 1 with adapter:                       4 (1.6%)\n",
      "  Read 2 with adapter:                       4 (1.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):           258 (100.0%)\n",
      "\n",
      "Total basepairs processed:       117,636 bp\n",
      "  Read 1:        57,753 bp\n",
      "  Read 2:        59,883 bp\n",
      "Total written (filtered):        117,569 bp (99.9%)\n",
      "  Read 1:        57,741 bp\n",
      "  Read 2:        59,828 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t4.0\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "19\t1\t0.0\t1\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.F.5.4.19_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.F.5.4.19_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.F.5.4.19_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.F.5.4.19_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.42 s (12 us/read; 5.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            545,081\n",
      "  Read 1 with adapter:                     867 (0.2%)\n",
      "  Read 2 with adapter:                  32,371 (5.9%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       545,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:   248,775,185 bp\n",
      "  Read 1:   122,076,432 bp\n",
      "  Read 2:   126,698,753 bp\n",
      "Total written (filtered):    248,352,924 bp (99.8%)\n",
      "  Read 1:   122,072,576 bp\n",
      "  Read 2:   126,280,348 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 867 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t848\t8516.9\t0\t848\n",
      "10\t3\t0.5\t1\t3\n",
      "18\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t0 1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t4\t0.0\t1\t4\n",
      "92\t1\t0.0\t1\t1\n",
      "100\t2\t0.0\t1\t2\n",
      "113\t1\t0.0\t1\t1\n",
      "178\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 32371 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t297\t8516.9\t0\t297\n",
      "4\t82\t2129.2\t0\t82\n",
      "5\t99\t532.3\t0\t99\n",
      "6\t216\t133.1\t0\t216\n",
      "7\t880\t33.3\t0\t880\n",
      "8\t1239\t8.3\t0\t1239\n",
      "9\t4450\t2.1\t0\t4426 24\n",
      "10\t10953\t0.5\t1\t10835 118\n",
      "11\t343\t0.1\t1\t339 4\n",
      "12\t233\t0.0\t1\t227 6\n",
      "13\t276\t0.0\t1\t271 5\n",
      "14\t629\t0.0\t1\t594 35\n",
      "15\t642\t0.0\t1\t610 32\n",
      "16\t1926\t0.0\t1\t1865 61\n",
      "17\t2580\t0.0\t1\t2502 78\n",
      "18\t1607\t0.0\t1\t1454 114 39\n",
      "19\t5137\t0.0\t1\t2921 2192 24\n",
      "20\t22\t0.0\t2\t14 5 3\n",
      "21\t701\t0.0\t2\t0 690 11\n",
      "22\t35\t0.0\t2\t0 1 34\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 1\n",
      "52\t1\t0.0\t2\t0 1\n",
      "54\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "118\t1\t0.0\t2\t0 1\n",
      "119\t1\t0.0\t2\t1\n",
      "162\t1\t0.0\t2\t1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "192\t1\t0.0\t2\t0 1\n",
      "197\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t0 0 1\n",
      "238\t1\t0.0\t2\t1\n",
      "241\t2\t0.0\t2\t1 1\n",
      "242\t3\t0.0\t2\t3\n",
      "243\t2\t0.0\t2\t1 1\n",
      "246\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.N.5.4.19_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.N.5.4.19_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.N.5.4.19_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.N.5.4.19_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.42 s (14 us/read; 4.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            393,869\n",
      "  Read 1 with adapter:                     292 (0.1%)\n",
      "  Read 2 with adapter:                  13,834 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       393,869 (100.0%)\n",
      "\n",
      "Total basepairs processed:   179,520,613 bp\n",
      "  Read 1:    88,189,055 bp\n",
      "  Read 2:    91,331,558 bp\n",
      "Total written (filtered):    179,322,886 bp (99.9%)\n",
      "  Read 1:    88,187,346 bp\n",
      "  Read 2:    91,135,540 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 292 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t281\t6154.2\t0\t281\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "55\t3\t0.0\t1\t3\n",
      "130\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13834 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t104\t6154.2\t0\t104\n",
      "4\t33\t1538.6\t0\t33\n",
      "5\t41\t384.6\t0\t41\n",
      "6\t98\t96.2\t0\t98\n",
      "7\t302\t24.0\t0\t302\n",
      "8\t410\t6.0\t0\t410\n",
      "9\t1407\t1.5\t0\t1405 2\n",
      "10\t3502\t0.4\t1\t3463 39\n",
      "11\t181\t0.1\t1\t177 4\n",
      "12\t148\t0.0\t1\t146 2\n",
      "13\t153\t0.0\t1\t147 6\n",
      "14\t303\t0.0\t1\t270 33\n",
      "15\t260\t0.0\t1\t252 8\n",
      "16\t962\t0.0\t1\t934 28\n",
      "17\t1428\t0.0\t1\t1385 43\n",
      "18\t869\t0.0\t1\t811 43 15\n",
      "19\t3095\t0.0\t1\t1984 1101 10\n",
      "20\t12\t0.0\t2\t9 3\n",
      "21\t487\t0.0\t2\t0 481 6\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "70\t3\t0.0\t2\t2 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t5\t0.0\t2\t5\n",
      "87\t1\t0.0\t2\t1\n",
      "100\t2\t0.0\t2\t2\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t1\n",
      "177\t2\t0.0\t2\t0 2\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D2.4.26.19_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D2.4.26.19_66_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D2.4.26.19_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D2.4.26.19_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.81 s (22 us/read; 2.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,038\n",
      "  Read 1 with adapter:                      93 (0.1%)\n",
      "  Read 2 with adapter:                   3,243 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        82,038 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,631,632 bp\n",
      "  Read 1:    17,960,008 bp\n",
      "  Read 2:    18,671,624 bp\n",
      "Total written (filtered):     36,580,755 bp (99.9%)\n",
      "  Read 1:    17,957,342 bp\n",
      "  Read 2:    18,623,413 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 93 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t1281.8\t0\t80\n",
      "25\t2\t0.0\t1\t2\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t3\t0.0\t1\t2 1\n",
      "216\t2\t0.0\t1\t1 1\n",
      "217\t5\t0.0\t1\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3243 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1281.8\t0\t16\n",
      "4\t6\t320.5\t0\t6\n",
      "5\t8\t80.1\t0\t8\n",
      "6\t13\t20.0\t0\t13\n",
      "7\t54\t5.0\t0\t54\n",
      "8\t78\t1.3\t0\t78\n",
      "9\t297\t0.3\t0\t296 1\n",
      "10\t767\t0.1\t1\t758 9\n",
      "11\t24\t0.0\t1\t22 2\n",
      "12\t31\t0.0\t1\t31\n",
      "13\t52\t0.0\t1\t51 1\n",
      "14\t97\t0.0\t1\t93 4\n",
      "15\t96\t0.0\t1\t90 6\n",
      "16\t251\t0.0\t1\t240 11\n",
      "17\t379\t0.0\t1\t369 10\n",
      "18\t217\t0.0\t1\t194 20 3\n",
      "19\t726\t0.0\t1\t457 266 3\n",
      "20\t2\t0.0\t2\t0 2\n",
      "21\t108\t0.0\t2\t0 106 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "70\t3\t0.0\t2\t3\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t7\t0.0\t2\t7\n",
      "161\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "245\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D2.5.4.19_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D2.5.4.19_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D2.5.4.19_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D2.5.4.19_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.47 s (14 us/read; 4.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            450,628\n",
      "  Read 1 with adapter:                     369 (0.1%)\n",
      "  Read 2 with adapter:                  14,244 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       450,628 (100.0%)\n",
      "\n",
      "Total basepairs processed:   205,245,244 bp\n",
      "  Read 1:   100,870,066 bp\n",
      "  Read 2:   104,375,178 bp\n",
      "Total written (filtered):    205,022,119 bp (99.9%)\n",
      "  Read 1:   100,868,099 bp\n",
      "  Read 2:   104,154,020 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 369 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t354\t7041.1\t0\t354\n",
      "24\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t7\t0.0\t1\t7\n",
      "74\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14244 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t124\t7041.1\t0\t124\n",
      "4\t33\t1760.3\t0\t33\n",
      "5\t32\t440.1\t0\t32\n",
      "6\t45\t110.0\t0\t45\n",
      "7\t175\t27.5\t0\t175\n",
      "8\t263\t6.9\t0\t263\n",
      "9\t1045\t1.7\t0\t1043 2\n",
      "10\t3534\t0.4\t1\t3500 34\n",
      "11\t217\t0.1\t1\t210 7\n",
      "12\t154\t0.0\t1\t151 3\n",
      "13\t148\t0.0\t1\t142 6\n",
      "14\t312\t0.0\t1\t291 21\n",
      "15\t304\t0.0\t1\t292 12\n",
      "16\t1026\t0.0\t1\t996 30\n",
      "17\t1554\t0.0\t1\t1502 52\n",
      "18\t993\t0.0\t1\t935 44 14\n",
      "19\t3605\t0.0\t1\t2279 1309 17\n",
      "20\t8\t0.0\t2\t4 3 1\n",
      "21\t540\t0.0\t2\t0 536 4\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "55\t3\t0.0\t2\t2 1\n",
      "59\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t0 1\n",
      "68\t2\t0.0\t2\t1 0 1\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t7\t0.0\t2\t0 0 7\n",
      "74\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t1 1\n",
      "79\t11\t0.0\t2\t11\n",
      "93\t1\t0.0\t2\t0 1\n",
      "94\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "151\t1\t0.0\t2\t1\n",
      "152\t1\t0.0\t2\t0 1\n",
      "153\t4\t0.0\t2\t4\n",
      "154\t1\t0.0\t2\t1\n",
      "155\t1\t0.0\t2\t1\n",
      "156\t4\t0.0\t2\t4\n",
      "157\t4\t0.0\t2\t3 1\n",
      "158\t6\t0.0\t2\t6\n",
      "159\t5\t0.0\t2\t4 1\n",
      "160\t4\t0.0\t2\t3 1\n",
      "161\t3\t0.0\t2\t2 1\n",
      "162\t3\t0.0\t2\t3\n",
      "163\t5\t0.0\t2\t3 2\n",
      "164\t11\t0.0\t2\t3 5 3\n",
      "165\t21\t0.0\t2\t15 5 1\n",
      "166\t4\t0.0\t2\t2 1 1\n",
      "199\t1\t0.0\t2\t1\n",
      "210\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "224\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.5.4.19_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.5.4.19_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.5.4.19_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.5.4.19_33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.5percent.5.31.19_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D2.5percent.5.31.19_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.5percent.5.31.19_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D2.5percent.5.31.19_24_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.4.26.19_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.4.26.19_60_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.4.26.19_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.4.26.19_60_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.5percent.5.31.19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.5percent.5.31.19_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.5percent.5.31.19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.5percent.5.31.19_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.F.5.4.19_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.F.5.4.19_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.F.5.4.19_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.F.5.4.19_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.N.5.4.19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D3.N.5.4.19_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.N.5.4.19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D3.N.5.4.19_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D4.4.26.19_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D4.4.26.19_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D4.4.26.19_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D4.4.26.19_31_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D4.5.4.19_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCB.D4.5.4.19_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D4.5.4.19_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCB.D4.5.4.19_29_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.4.26.19_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.4.26.19_69_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.4.26.19_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.4.26.19_69_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.F.5.4.19_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.F.5.4.19_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.F.5.4.19_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.F.5.4.19_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.N.5.4.19_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D1.N.5.4.19_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.N.5.4.19_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D1.N.5.4.19_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D2.4.26.19_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D2.4.26.19_66_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D2.4.26.19_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D2.4.26.19_66_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D2.5.4.19_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D2.5.4.19_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D2.5.4.19_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D2.5.4.19_32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D3.4.26.19_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D3.4.26.19_50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D3.4.26.19_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D3.4.26.19_50_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D3.4.26.19_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D3.4.26.19_50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D3.4.26.19_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D3.4.26.19_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.82 s (17 us/read; 3.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            334,056\n",
      "  Read 1 with adapter:                     345 (0.1%)\n",
      "  Read 2 with adapter:                  10,521 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       334,056 (100.0%)\n",
      "\n",
      "Total basepairs processed:   151,983,317 bp\n",
      "  Read 1:    74,661,389 bp\n",
      "  Read 2:    77,321,928 bp\n",
      "Total written (filtered):    151,825,069 bp (99.9%)\n",
      "  Read 1:    74,658,150 bp\n",
      "  Read 2:    77,166,919 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 345 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t330\t5219.6\t0\t330\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "179\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t0 1\n",
      "211\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t4\t0.0\t1\t3 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10521 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t78\t5219.6\t0\t78\n",
      "4\t11\t1304.9\t0\t11\n",
      "5\t17\t326.2\t0\t17\n",
      "6\t37\t81.6\t0\t37\n",
      "7\t167\t20.4\t0\t167\n",
      "8\t227\t5.1\t0\t227\n",
      "9\t809\t1.3\t0\t803 6\n",
      "10\t2592\t0.3\t1\t2564 28\n",
      "11\t135\t0.1\t1\t132 3\n",
      "12\t122\t0.0\t1\t121 1\n",
      "13\t124\t0.0\t1\t120 4\n",
      "14\t233\t0.0\t1\t219 14\n",
      "15\t228\t0.0\t1\t222 6\n",
      "16\t771\t0.0\t1\t736 35\n",
      "17\t1088\t0.0\t1\t1048 40\n",
      "18\t716\t0.0\t1\t655 47 14\n",
      "19\t2722\t0.0\t1\t1707 1009 6\n",
      "20\t10\t0.0\t2\t6 2 2\n",
      "21\t390\t0.0\t2\t0 389 1\n",
      "22\t19\t0.0\t2\t0 2 17\n",
      "55\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "68\t2\t0.0\t2\t1 1\n",
      "76\t1\t0.0\t2\t0 1\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t0 2\n",
      "79\t7\t0.0\t2\t7\n",
      "85\t1\t0.0\t2\t0 1\n",
      "100\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "190\t1\t0.0\t2\t1\n",
      "193\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D3.5.4.19_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D3.5.4.19_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D3.5.4.19_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D3.5.4.19_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.75 s (20 us/read; 3.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            137,815\n",
      "  Read 1 with adapter:                     136 (0.1%)\n",
      "  Read 2 with adapter:                   4,622 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       137,815 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,306,809 bp\n",
      "  Read 1:    30,593,498 bp\n",
      "  Read 2:    31,713,311 bp\n",
      "Total written (filtered):     62,235,560 bp (99.9%)\n",
      "  Read 1:    30,590,679 bp\n",
      "  Read 2:    31,644,881 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 136 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t122\t2153.4\t0\t122\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "216\t7\t0.0\t1\t6 1\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4622 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t2153.4\t0\t33\n",
      "4\t8\t538.3\t0\t8\n",
      "5\t11\t134.6\t0\t11\n",
      "6\t28\t33.6\t0\t28\n",
      "7\t81\t8.4\t0\t81\n",
      "8\t110\t2.1\t0\t110\n",
      "9\t378\t0.5\t0\t378\n",
      "10\t976\t0.1\t1\t960 16\n",
      "11\t62\t0.0\t1\t60 2\n",
      "12\t65\t0.0\t1\t64 1\n",
      "13\t55\t0.0\t1\t54 1\n",
      "14\t128\t0.0\t1\t119 9\n",
      "15\t116\t0.0\t1\t113 3\n",
      "16\t329\t0.0\t1\t320 9\n",
      "17\t551\t0.0\t1\t537 14\n",
      "18\t316\t0.0\t1\t285 24 7\n",
      "19\t1150\t0.0\t1\t717 428 5\n",
      "20\t6\t0.0\t2\t4 1 1\n",
      "21\t202\t0.0\t2\t0 198 4\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t2 1 1\n",
      "151\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.4.26.19_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.4.26.19_49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.4.26.19_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.4.26.19_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.00 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            160,270\n",
      "  Read 1 with adapter:                     220 (0.1%)\n",
      "  Read 2 with adapter:                   5,583 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       160,270 (100.0%)\n",
      "\n",
      "Total basepairs processed:    72,554,917 bp\n",
      "  Read 1:    35,623,465 bp\n",
      "  Read 2:    36,931,452 bp\n",
      "Total written (filtered):     72,472,657 bp (99.9%)\n",
      "  Read 1:    35,620,811 bp\n",
      "  Read 2:    36,851,846 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 220 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t207\t2504.2\t0\t207\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "205\t1\t0.0\t1\t1\n",
      "208\t1\t0.0\t1\t0 1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t2 1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5583 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t2504.2\t0\t30\n",
      "4\t12\t626.1\t0\t12\n",
      "5\t18\t156.5\t0\t18\n",
      "6\t42\t39.1\t0\t42\n",
      "7\t124\t9.8\t0\t124\n",
      "8\t124\t2.4\t0\t124\n",
      "9\t522\t0.6\t0\t521 1\n",
      "10\t1421\t0.2\t1\t1400 21\n",
      "11\t81\t0.0\t1\t80 1\n",
      "12\t57\t0.0\t1\t56 1\n",
      "13\t60\t0.0\t1\t59 1\n",
      "14\t124\t0.0\t1\t117 7\n",
      "15\t128\t0.0\t1\t125 3\n",
      "16\t395\t0.0\t1\t381 14\n",
      "17\t550\t0.0\t1\t533 17\n",
      "18\t369\t0.0\t1\t335 22 12\n",
      "19\t1297\t0.0\t1\t800 493 4\n",
      "20\t3\t0.0\t2\t1 2\n",
      "21\t214\t0.0\t2\t0 209 5\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "102\t1\t0.0\t2\t0 1\n",
      "195\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.F.5.4.19_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.F.5.4.19_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.F.5.4.19_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.F.5.4.19_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.13 s (19 us/read; 3.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            276,264\n",
      "  Read 1 with adapter:                     294 (0.1%)\n",
      "  Read 2 with adapter:                   8,355 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       276,264 (100.0%)\n",
      "\n",
      "Total basepairs processed:   125,769,932 bp\n",
      "  Read 1:    61,791,190 bp\n",
      "  Read 2:    63,978,742 bp\n",
      "Total written (filtered):    125,643,546 bp (99.9%)\n",
      "  Read 1:    61,789,015 bp\n",
      "  Read 2:    63,854,531 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 294 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t281\t4316.6\t0\t281\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "92\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8355 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t65\t4316.6\t0\t65\n",
      "4\t25\t1079.2\t0\t25\n",
      "5\t30\t269.8\t0\t30\n",
      "6\t49\t67.4\t0\t49\n",
      "7\t134\t16.9\t0\t134\n",
      "8\t194\t4.2\t0\t194\n",
      "9\t651\t1.1\t0\t647 4\n",
      "10\t1831\t0.3\t1\t1811 20\n",
      "11\t124\t0.1\t1\t123 1\n",
      "12\t114\t0.0\t1\t111 3\n",
      "13\t119\t0.0\t1\t117 2\n",
      "14\t228\t0.0\t1\t209 19\n",
      "15\t183\t0.0\t1\t174 9\n",
      "16\t599\t0.0\t1\t589 10\n",
      "17\t886\t0.0\t1\t862 24\n",
      "18\t601\t0.0\t1\t547 41 13\n",
      "19\t2148\t0.0\t1\t1305 834 9\n",
      "20\t7\t0.0\t2\t3 1 3\n",
      "21\t326\t0.0\t2\t0 324 2\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t1\t0.0\t2\t0 1\n",
      "70\t3\t0.0\t2\t3\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "78\t2\t0.0\t2\t0 1 1\n",
      "79\t12\t0.0\t2\t8 3 1\n",
      "88\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t1\n",
      "100\t2\t0.0\t2\t1 0 1\n",
      "128\t1\t0.0\t2\t1\n",
      "144\t1\t0.0\t2\t1\n",
      "151\t1\t0.0\t2\t0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.N.5.4.19_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.N.5.4.19_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.N.5.4.19_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.N.5.4.19_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.85 s (13 us/read; 4.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            527,969\n",
      "  Read 1 with adapter:                     824 (0.2%)\n",
      "  Read 2 with adapter:                  38,399 (7.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       527,969 (100.0%)\n",
      "\n",
      "Total basepairs processed:   241,173,860 bp\n",
      "  Read 1:   118,232,066 bp\n",
      "  Read 2:   122,941,794 bp\n",
      "Total written (filtered):    240,667,728 bp (99.8%)\n",
      "  Read 1:   118,227,561 bp\n",
      "  Read 2:   122,440,167 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 824 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t784\t8249.5\t0\t784\n",
      "17\t1\t0.0\t1\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t4\t0.0\t1\t4\n",
      "37\t1\t0.0\t1\t1\n",
      "46\t1\t0.0\t1\t1\n",
      "51\t1\t0.0\t1\t1\n",
      "54\t3\t0.0\t1\t1 2\n",
      "55\t22\t0.0\t1\t22\n",
      "89\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "130\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 38399 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t323\t8249.5\t0\t323\n",
      "4\t173\t2062.4\t0\t173\n",
      "5\t230\t515.6\t0\t230\n",
      "6\t585\t128.9\t0\t585\n",
      "7\t1549\t32.2\t0\t1549\n",
      "8\t2376\t8.1\t0\t2376\n",
      "9\t6879\t2.0\t0\t6864 15\n",
      "10\t7565\t0.5\t1\t7493 72\n",
      "11\t460\t0.1\t1\t452 8\n",
      "12\t641\t0.0\t1\t625 16\n",
      "13\t959\t0.0\t1\t930 29\n",
      "14\t1575\t0.0\t1\t1488 87\n",
      "15\t1046\t0.0\t1\t978 68\n",
      "16\t2522\t0.0\t1\t2417 105\n",
      "17\t2873\t0.0\t1\t2757 116\n",
      "18\t1791\t0.0\t1\t1508 153 130\n",
      "19\t5931\t0.0\t1\t3127 2774 30\n",
      "20\t20\t0.0\t2\t8 11 1\n",
      "21\t767\t0.0\t2\t0 756 11\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "48\t1\t0.0\t2\t1\n",
      "49\t2\t0.0\t2\t2\n",
      "52\t1\t0.0\t2\t1\n",
      "54\t3\t0.0\t2\t2 1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t3\t0.0\t2\t2 0 1\n",
      "59\t3\t0.0\t2\t2 0 1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t8\t0.0\t2\t8\n",
      "64\t6\t0.0\t2\t6\n",
      "65\t2\t0.0\t2\t1 0 1\n",
      "66\t5\t0.0\t2\t5\n",
      "67\t6\t0.0\t2\t6\n",
      "68\t7\t0.0\t2\t7\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t2\t0.0\t2\t2\n",
      "73\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t0 1\n",
      "77\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "97\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "107\t2\t0.0\t2\t2\n",
      "111\t1\t0.0\t2\t1\n",
      "112\t1\t0.0\t2\t1\n",
      "118\t1\t0.0\t2\t1\n",
      "126\t1\t0.0\t2\t1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t1\n",
      "160\t1\t0.0\t2\t0 1\n",
      "171\t1\t0.0\t2\t1\n",
      "172\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "184\t1\t0.0\t2\t1\n",
      "197\t1\t0.0\t2\t0 1\n",
      "217\t4\t0.0\t2\t3 0 1\n",
      "225\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t0 1\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t2\t0.0\t2\t1 1\n",
      "238\t2\t0.0\t2\t2\n",
      "239\t4\t0.0\t2\t3 1\n",
      "240\t3\t0.0\t2\t2 0 1\n",
      "241\t7\t0.0\t2\t5 0 2\n",
      "242\t3\t0.0\t2\t2 1\n",
      "243\t5\t0.0\t2\t4 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D1.4.26.19_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D1.4.26.19_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D1.4.26.19_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D1.4.26.19_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.22 s (13 us/read; 4.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            392,199\n",
      "  Read 1 with adapter:                     348 (0.1%)\n",
      "  Read 2 with adapter:                  18,732 (4.8%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       392,198 (100.0%)\n",
      "\n",
      "Total basepairs processed:   178,675,401 bp\n",
      "  Read 1:    87,719,913 bp\n",
      "  Read 2:    90,955,488 bp\n",
      "Total written (filtered):    178,423,122 bp (99.9%)\n",
      "  Read 1:    87,717,275 bp\n",
      "  Read 2:    90,705,847 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 348 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t329\t6128.1\t0\t329\n",
      "26\t6\t0.0\t1\t6\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t5\t0.0\t1\t5\n",
      "101\t2\t0.0\t1\t2\n",
      "144\t1\t0.0\t1\t1\n",
      "156\t1\t0.0\t1\t1\n",
      "162\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18732 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t122\t6128.1\t0\t122\n",
      "4\t55\t1532.0\t0\t55\n",
      "5\t51\t383.0\t0\t51\n",
      "6\t126\t95.8\t0\t126\n",
      "7\t404\t23.9\t0\t404\n",
      "8\t576\t6.0\t0\t576\n",
      "9\t2310\t1.5\t0\t2297 13\n",
      "10\t6216\t0.4\t1\t6146 70\n",
      "11\t272\t0.1\t1\t269 3\n",
      "12\t168\t0.0\t1\t167 1\n",
      "13\t150\t0.0\t1\t143 7\n",
      "14\t322\t0.0\t1\t301 21\n",
      "15\t376\t0.0\t1\t363 13\n",
      "16\t1077\t0.0\t1\t1043 34\n",
      "17\t1489\t0.0\t1\t1434 55\n",
      "18\t971\t0.0\t1\t900 55 16\n",
      "19\t3292\t0.0\t1\t1970 1308 14\n",
      "20\t198\t0.0\t2\t185 10 3\n",
      "21\t502\t0.0\t2\t0 498 4\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t3\t0.0\t2\t1 1 1\n",
      "71\t4\t0.0\t2\t0 0 4\n",
      "76\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t2\n",
      "79\t8\t0.0\t2\t6 2\n",
      "80\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "151\t1\t0.0\t2\t0 1\n",
      "185\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "223\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D2.4.26.19_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D2.4.26.19_65_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D2.4.26.19_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D2.4.26.19_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.88 s (22 us/read; 2.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             83,978\n",
      "  Read 1 with adapter:                     100 (0.1%)\n",
      "  Read 2 with adapter:                   3,273 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        83,978 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,521,380 bp\n",
      "  Read 1:    18,405,280 bp\n",
      "  Read 2:    19,116,100 bp\n",
      "Total written (filtered):     37,470,881 bp (99.9%)\n",
      "  Read 1:    18,401,334 bp\n",
      "  Read 2:    19,069,547 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 100 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t1312.2\t0\t79\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "155\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t11\t0.0\t1\t8 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3273 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t1312.2\t0\t21\n",
      "4\t6\t328.0\t0\t6\n",
      "5\t10\t82.0\t0\t10\n",
      "6\t19\t20.5\t0\t19\n",
      "7\t54\t5.1\t0\t54\n",
      "8\t77\t1.3\t0\t77\n",
      "9\t320\t0.3\t0\t319 1\n",
      "10\t882\t0.1\t1\t876 6\n",
      "11\t41\t0.0\t1\t39 2\n",
      "12\t38\t0.0\t1\t37 1\n",
      "13\t31\t0.0\t1\t30 1\n",
      "14\t67\t0.0\t1\t62 5\n",
      "15\t65\t0.0\t1\t62 3\n",
      "16\t264\t0.0\t1\t258 6\n",
      "17\t340\t0.0\t1\t330 10\n",
      "18\t229\t0.0\t1\t209 15 5\n",
      "19\t712\t0.0\t1\t420 289 3\n",
      "21\t83\t0.0\t2\t0 83\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t3\n",
      "100\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D2.5.4.19_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D2.5.4.19_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D2.5.4.19_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D2.5.4.19_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.39 s (14 us/read; 4.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            375,083\n",
      "  Read 1 with adapter:                     437 (0.1%)\n",
      "  Read 2 with adapter:                  18,046 (4.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       375,083 (100.0%)\n",
      "\n",
      "Total basepairs processed:   170,811,121 bp\n",
      "  Read 1:    83,846,341 bp\n",
      "  Read 2:    86,964,780 bp\n",
      "Total written (filtered):    170,562,746 bp (99.9%)\n",
      "  Read 1:    83,843,000 bp\n",
      "  Read 2:    86,719,746 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 437 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t410\t5860.7\t0\t410\n",
      "14\t1\t0.0\t1\t1\n",
      "24\t4\t0.0\t1\t4\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t11\t0.0\t1\t10 1\n",
      "48\t1\t0.0\t1\t1\n",
      "87\t1\t0.0\t1\t1\n",
      "194\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t0 1\n",
      "217\t5\t0.0\t1\t4 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18046 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t157\t5860.7\t0\t157\n",
      "4\t47\t1465.2\t0\t47\n",
      "5\t49\t366.3\t0\t49\n",
      "6\t126\t91.6\t0\t126\n",
      "7\t366\t22.9\t0\t366\n",
      "8\t534\t5.7\t0\t534\n",
      "9\t2078\t1.4\t0\t2064 14\n",
      "10\t5604\t0.4\t1\t5558 46\n",
      "11\t208\t0.1\t1\t206 2\n",
      "12\t166\t0.0\t1\t162 4\n",
      "13\t210\t0.0\t1\t203 7\n",
      "14\t423\t0.0\t1\t390 33\n",
      "15\t383\t0.0\t1\t365 18\n",
      "16\t1167\t0.0\t1\t1131 36\n",
      "17\t1614\t0.0\t1\t1554 60\n",
      "18\t952\t0.0\t1\t871 60 21\n",
      "19\t3408\t0.0\t1\t2050 1349 9\n",
      "20\t17\t0.0\t2\t10 5 2\n",
      "21\t487\t0.0\t2\t0 477 10\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t1\n",
      "53\t2\t0.0\t2\t1 0 1\n",
      "54\t2\t0.0\t2\t2\n",
      "61\t1\t0.0\t2\t1\n",
      "70\t4\t0.0\t2\t3 0 1\n",
      "79\t7\t0.0\t2\t6 1\n",
      "100\t1\t0.0\t2\t1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "184\t1\t0.0\t2\t0 1\n",
      "205\t1\t0.0\t2\t1\n",
      "208\t2\t0.0\t2\t2\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t4\t0.0\t2\t4\n",
      "225\t1\t0.0\t2\t1\n",
      "228\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D3.4.26.19_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D3.4.26.19_62_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D3.4.26.19_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D3.4.26.19_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.04 s (19 us/read; 3.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            164,183\n",
      "  Read 1 with adapter:                     186 (0.1%)\n",
      "  Read 2 with adapter:                   6,406 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       164,183 (100.0%)\n",
      "\n",
      "Total basepairs processed:    74,163,782 bp\n",
      "  Read 1:    36,397,958 bp\n",
      "  Read 2:    37,765,824 bp\n",
      "Total written (filtered):     74,069,677 bp (99.9%)\n",
      "  Read 1:    36,394,299 bp\n",
      "  Read 2:    37,675,378 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 186 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t168\t2565.4\t0\t168\n",
      "16\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "62\t1\t0.0\t1\t1\n",
      "211\t1\t0.0\t1\t1\n",
      "215\t2\t0.0\t1\t1 1\n",
      "216\t3\t0.0\t1\t2 1\n",
      "217\t8\t0.0\t1\t7 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6406 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t2565.4\t0\t44\n",
      "4\t24\t641.3\t0\t24\n",
      "5\t14\t160.3\t0\t14\n",
      "6\t35\t40.1\t0\t35\n",
      "7\t131\t10.0\t0\t131\n",
      "8\t166\t2.5\t0\t166\n",
      "9\t620\t0.6\t0\t616 4\n",
      "10\t1717\t0.2\t1\t1695 22\n",
      "11\t93\t0.0\t1\t92 1\n",
      "12\t68\t0.0\t1\t67 1\n",
      "13\t77\t0.0\t1\t73 4\n",
      "14\t154\t0.0\t1\t142 12\n",
      "15\t149\t0.0\t1\t147 2\n",
      "16\t415\t0.0\t1\t399 16\n",
      "17\t583\t0.0\t1\t573 10\n",
      "18\t444\t0.0\t1\t414 25 5\n",
      "19\t1415\t0.0\t1\t842 569 4\n",
      "20\t7\t0.0\t2\t6 0 1\n",
      "21\t223\t0.0\t2\t0 219 4\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "62\t1\t0.0\t2\t0 1\n",
      "67\t2\t0.0\t2\t2\n",
      "70\t1\t0.0\t2\t0 1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D3.5.4.19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D3.5.4.19_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D3.5.4.19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D3.5.4.19_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 9.65 s (15 us/read; 4.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            652,835\n",
      "  Read 1 with adapter:                     941 (0.1%)\n",
      "  Read 2 with adapter:                  58,473 (9.0%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       652,834 (100.0%)\n",
      "\n",
      "Total basepairs processed:   298,517,498 bp\n",
      "  Read 1:   146,177,513 bp\n",
      "  Read 2:   152,339,985 bp\n",
      "Total written (filtered):    297,775,029 bp (99.8%)\n",
      "  Read 1:   146,172,640 bp\n",
      "  Read 2:   151,602,389 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 941 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t918\t10200.5\t0\t918\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t8\t0.0\t1\t8\n",
      "57\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "151\t1\t0.0\t1\t1\n",
      "153\t1\t0.0\t1\t1\n",
      "154\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t0 1\n",
      "178\t1\t0.0\t1\t1\n",
      "197\t1\t0.0\t1\t0 1\n",
      "209\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 58473 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t549\t10200.5\t0\t549\n",
      "4\t283\t2550.1\t0\t283\n",
      "5\t473\t637.5\t0\t473\n",
      "6\t1255\t159.4\t0\t1255\n",
      "7\t3363\t39.8\t0\t3363\n",
      "8\t4911\t10.0\t0\t4911\n",
      "9\t12649\t2.5\t0\t12625 24\n",
      "10\t12246\t0.6\t1\t12108 138\n",
      "11\t568\t0.2\t1\t549 19\n",
      "12\t710\t0.0\t1\t693 17\n",
      "13\t1117\t0.0\t1\t1059 58\n",
      "14\t1806\t0.0\t1\t1710 96\n",
      "15\t1262\t0.0\t1\t1178 84\n",
      "16\t3001\t0.0\t1\t2850 151\n",
      "17\t3474\t0.0\t1\t3297 177\n",
      "18\t2194\t0.0\t1\t1883 150 161\n",
      "19\t7164\t0.0\t1\t3777 3346 41\n",
      "20\t33\t0.0\t2\t18 12 3\n",
      "21\t904\t0.0\t2\t0 897 7\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "30\t1\t0.0\t2\t1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "46\t2\t0.0\t2\t2\n",
      "49\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "53\t2\t0.0\t2\t1 0 1\n",
      "54\t8\t0.0\t2\t6 2\n",
      "55\t5\t0.0\t2\t4 1\n",
      "56\t11\t0.0\t2\t8 3\n",
      "57\t7\t0.0\t2\t5 1 1\n",
      "58\t8\t0.0\t2\t4 1 3\n",
      "59\t4\t0.0\t2\t2 1 1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t6\t0.0\t2\t2 2 2\n",
      "62\t7\t0.0\t2\t7\n",
      "63\t24\t0.0\t2\t21 1 2\n",
      "64\t28\t0.0\t2\t24 4\n",
      "65\t39\t0.0\t2\t31 7 1\n",
      "66\t35\t0.0\t2\t29 2 4\n",
      "67\t14\t0.0\t2\t10 3 1\n",
      "68\t4\t0.0\t2\t4\n",
      "69\t7\t0.0\t2\t5 1 1\n",
      "70\t15\t0.0\t2\t10 5\n",
      "71\t14\t0.0\t2\t3 1 10\n",
      "72\t3\t0.0\t2\t3\n",
      "73\t5\t0.0\t2\t4 1\n",
      "74\t1\t0.0\t2\t0 1\n",
      "75\t3\t0.0\t2\t2 1\n",
      "76\t2\t0.0\t2\t2\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t6\t0.0\t2\t3 1 2\n",
      "79\t30\t0.0\t2\t21 5 4\n",
      "80\t2\t0.0\t2\t2\n",
      "82\t1\t0.0\t2\t1\n",
      "83\t1\t0.0\t2\t1\n",
      "84\t6\t0.0\t2\t6\n",
      "85\t2\t0.0\t2\t2\n",
      "86\t6\t0.0\t2\t4 2\n",
      "87\t11\t0.0\t2\t10 0 1\n",
      "88\t4\t0.0\t2\t3 1\n",
      "89\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t2\t0.0\t2\t2\n",
      "92\t2\t0.0\t2\t1 0 1\n",
      "94\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "97\t1\t0.0\t2\t1\n",
      "99\t2\t0.0\t2\t2\n",
      "100\t6\t0.0\t2\t4 1 1\n",
      "101\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "106\t1\t0.0\t2\t1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t1\n",
      "114\t1\t0.0\t2\t1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 1\n",
      "119\t1\t0.0\t2\t0 1\n",
      "125\t2\t0.0\t2\t1 1\n",
      "127\t1\t0.0\t2\t0 1\n",
      "128\t1\t0.0\t2\t1\n",
      "137\t1\t0.0\t2\t1\n",
      "138\t1\t0.0\t2\t1\n",
      "139\t2\t0.0\t2\t1 1\n",
      "140\t1\t0.0\t2\t1\n",
      "141\t1\t0.0\t2\t0 1\n",
      "142\t1\t0.0\t2\t1\n",
      "146\t1\t0.0\t2\t1\n",
      "147\t1\t0.0\t2\t1\n",
      "150\t5\t0.0\t2\t4 0 1\n",
      "151\t1\t0.0\t2\t0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t1\n",
      "169\t1\t0.0\t2\t1\n",
      "171\t3\t0.0\t2\t3\n",
      "172\t1\t0.0\t2\t1\n",
      "173\t3\t0.0\t2\t1 0 2\n",
      "174\t1\t0.0\t2\t0 1\n",
      "179\t1\t0.0\t2\t0 1\n",
      "181\t3\t0.0\t2\t3\n",
      "186\t2\t0.0\t2\t1 1\n",
      "193\t1\t0.0\t2\t1\n",
      "194\t1\t0.0\t2\t0 1\n",
      "199\t2\t0.0\t2\t2\n",
      "203\t1\t0.0\t2\t1\n",
      "205\t1\t0.0\t2\t1\n",
      "206\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t1\n",
      "211\t1\t0.0\t2\t1\n",
      "214\t2\t0.0\t2\t2\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t4\t0.0\t2\t4\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t0 1\n",
      "225\t1\t0.0\t2\t0 1\n",
      "226\t2\t0.0\t2\t1 1\n",
      "227\t2\t0.0\t2\t2\n",
      "237\t3\t0.0\t2\t3\n",
      "238\t8\t0.0\t2\t6 2\n",
      "239\t7\t0.0\t2\t4 2 1\n",
      "240\t7\t0.0\t2\t5 1 1\n",
      "241\t7\t0.0\t2\t5 2\n",
      "242\t6\t0.0\t2\t3 2 1\n",
      "243\t4\t0.0\t2\t3 1\n",
      "244\t2\t0.0\t2\t1 1\n",
      "245\t2\t0.0\t2\t1 1\n",
      "246\t2\t0.0\t2\t2\n",
      "247\t1\t0.0\t2\t1\n",
      "249\t3\t0.0\t2\t1 1 1\n",
      "250\t2\t0.0\t2\t2\n",
      "251\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.4.26.19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.4.26.19_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.4.26.19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.4.26.19_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.45 s (15 us/read; 4.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            430,994\n",
      "  Read 1 with adapter:                     410 (0.1%)\n",
      "  Read 2 with adapter:                  20,400 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       430,994 (100.0%)\n",
      "\n",
      "Total basepairs processed:   196,289,282 bp\n",
      "  Read 1:    96,362,802 bp\n",
      "  Read 2:    99,926,480 bp\n",
      "Total written (filtered):    196,010,591 bp (99.9%)\n",
      "  Read 1:    96,359,483 bp\n",
      "  Read 2:    99,651,108 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 410 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t381\t6734.3\t0\t381\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t5\t0.0\t1\t4 1\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t4\t0.0\t1\t4\n",
      "86\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t0 1\n",
      "180\t2\t0.0\t1\t1 1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 20400 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t163\t6734.3\t0\t163\n",
      "4\t51\t1683.6\t0\t51\n",
      "5\t54\t420.9\t0\t54\n",
      "6\t111\t105.2\t0\t111\n",
      "7\t416\t26.3\t0\t416\n",
      "8\t570\t6.6\t0\t570\n",
      "9\t2373\t1.6\t0\t2367 6\n",
      "10\t6399\t0.4\t1\t6333 66\n",
      "11\t239\t0.1\t1\t237 2\n",
      "12\t212\t0.0\t1\t209 3\n",
      "13\t199\t0.0\t1\t194 5\n",
      "14\t445\t0.0\t1\t414 31\n",
      "15\t454\t0.0\t1\t433 21\n",
      "16\t1382\t0.0\t1\t1351 31\n",
      "17\t1779\t0.0\t1\t1718 61\n",
      "18\t1119\t0.0\t1\t1021 78 20\n",
      "19\t3813\t0.0\t1\t2322 1476 15\n",
      "20\t12\t0.0\t2\t7 3 2\n",
      "21\t564\t0.0\t2\t0 553 11\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t2 1 1\n",
      "84\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t0 0 1\n",
      "237\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.5.4.19_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.5.4.19_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.5.4.19_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.5.4.19_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.79 s (14 us/read; 4.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            405,453\n",
      "  Read 1 with adapter:                     405 (0.1%)\n",
      "  Read 2 with adapter:                  36,314 (9.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       405,453 (100.0%)\n",
      "\n",
      "Total basepairs processed:   185,382,000 bp\n",
      "  Read 1:    90,784,197 bp\n",
      "  Read 2:    94,597,803 bp\n",
      "Total written (filtered):    184,938,368 bp (99.8%)\n",
      "  Read 1:    90,781,876 bp\n",
      "  Read 2:    94,156,492 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 405 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t386\t6335.2\t0\t386\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t5\t0.0\t1\t5\n",
      "81\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t0 1\n",
      "172\t1\t0.0\t1\t0 1\n",
      "192\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 36314 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t419\t6335.2\t0\t419\n",
      "4\t189\t1583.8\t0\t189\n",
      "5\t388\t396.0\t0\t388\n",
      "6\t980\t99.0\t0\t980\n",
      "7\t2497\t24.7\t0\t2497\n",
      "8\t3446\t6.2\t0\t3446\n",
      "9\t8198\t1.5\t0\t8184 14\n",
      "10\t6791\t0.4\t1\t6709 82\n",
      "11\t305\t0.1\t1\t292 13\n",
      "12\t424\t0.0\t1\t410 14\n",
      "13\t680\t0.0\t1\t657 23\n",
      "14\t1054\t0.0\t1\t992 62\n",
      "15\t714\t0.0\t1\t674 40\n",
      "16\t1783\t0.0\t1\t1699 84\n",
      "17\t2161\t0.0\t1\t2067 94\n",
      "18\t1276\t0.0\t1\t1109 89 78\n",
      "19\t4282\t0.0\t1\t2345 1912 25\n",
      "20\t20\t0.0\t2\t11 7 2\n",
      "21\t528\t0.0\t2\t0 520 8\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "49\t1\t0.0\t2\t0 1\n",
      "51\t1\t0.0\t2\t1\n",
      "53\t3\t0.0\t2\t3\n",
      "55\t6\t0.0\t2\t3 2 1\n",
      "56\t3\t0.0\t2\t2 0 1\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "62\t3\t0.0\t2\t2 1\n",
      "63\t4\t0.0\t2\t3 1\n",
      "65\t3\t0.0\t2\t2 0 1\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t3\t0.0\t2\t1 2\n",
      "69\t2\t0.0\t2\t2\n",
      "70\t7\t0.0\t2\t4 1 2\n",
      "71\t5\t0.0\t2\t1 0 4\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "74\t3\t0.0\t2\t1 0 2\n",
      "76\t4\t0.0\t2\t4\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t14\t0.0\t2\t12 1 1\n",
      "85\t1\t0.0\t2\t1\n",
      "86\t2\t0.0\t2\t2\n",
      "88\t1\t0.0\t2\t1\n",
      "91\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t1\n",
      "100\t3\t0.0\t2\t3\n",
      "104\t2\t0.0\t2\t2\n",
      "105\t1\t0.0\t2\t0 1\n",
      "106\t1\t0.0\t2\t1\n",
      "107\t2\t0.0\t2\t2\n",
      "108\t1\t0.0\t2\t1\n",
      "112\t1\t0.0\t2\t1\n",
      "132\t1\t0.0\t2\t1\n",
      "137\t1\t0.0\t2\t1\n",
      "138\t4\t0.0\t2\t4\n",
      "139\t1\t0.0\t2\t1\n",
      "142\t1\t0.0\t2\t1\n",
      "147\t2\t0.0\t2\t2\n",
      "151\t1\t0.0\t2\t1\n",
      "152\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t1\n",
      "157\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "169\t2\t0.0\t2\t2\n",
      "175\t1\t0.0\t2\t1\n",
      "178\t1\t0.0\t2\t1\n",
      "182\t1\t0.0\t2\t0 1\n",
      "183\t3\t0.0\t2\t3\n",
      "187\t1\t0.0\t2\t0 1\n",
      "195\t1\t0.0\t2\t1\n",
      "197\t1\t0.0\t2\t0 1\n",
      "199\t1\t0.0\t2\t1\n",
      "204\t1\t0.0\t2\t1\n",
      "217\t4\t0.0\t2\t1 3\n",
      "228\t1\t0.0\t2\t1\n",
      "230\t1\t0.0\t2\t1\n",
      "231\t1\t0.0\t2\t1\n",
      "232\t2\t0.0\t2\t2\n",
      "237\t2\t0.0\t2\t2\n",
      "238\t9\t0.0\t2\t6 1 2\n",
      "239\t2\t0.0\t2\t1 0 1\n",
      "240\t1\t0.0\t2\t1\n",
      "241\t5\t0.0\t2\t3 1 1\n",
      "242\t3\t0.0\t2\t2 1\n",
      "243\t2\t0.0\t2\t0 2\n",
      "244\t3\t0.0\t2\t2 1\n",
      "246\t2\t0.0\t2\t2\n",
      "249\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.5percent.5.31.19_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.5percent.5.31.19_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.5percent.5.31.19_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.5percent.5.31.19_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.59 s (17 us/read; 3.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            434,233\n",
      "  Read 1 with adapter:                     404 (0.1%)\n",
      "  Read 2 with adapter:                  32,123 (7.4%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       434,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:   198,324,343 bp\n",
      "  Read 1:    97,240,224 bp\n",
      "  Read 2:   101,084,119 bp\n",
      "Total written (filtered):    197,913,658 bp (99.8%)\n",
      "  Read 1:    97,237,823 bp\n",
      "  Read 2:   100,675,835 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 404 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t389\t6784.9\t0\t389\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t4\t0.0\t1\t4\n",
      "51\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "102\t1\t0.0\t1\t1\n",
      "125\t1\t0.0\t1\t1\n",
      "195\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 32123 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t229\t6784.9\t0\t229\n",
      "4\t79\t1696.2\t0\t79\n",
      "5\t116\t424.1\t0\t116\n",
      "6\t231\t106.0\t0\t231\n",
      "7\t870\t26.5\t0\t870\n",
      "8\t1284\t6.6\t0\t1284\n",
      "9\t4732\t1.7\t0\t4715 17\n",
      "10\t10726\t0.4\t1\t10597 129\n",
      "11\t385\t0.1\t1\t376 9\n",
      "12\t325\t0.0\t1\t319 6\n",
      "13\t488\t0.0\t1\t469 19\n",
      "14\t972\t0.0\t1\t907 65\n",
      "15\t772\t0.0\t1\t736 36\n",
      "16\t2025\t0.0\t1\t1949 76\n",
      "17\t2424\t0.0\t1\t2317 107\n",
      "18\t1413\t0.0\t1\t1267 100 46\n",
      "19\t4371\t0.0\t1\t2397 1956 18\n",
      "20\t13\t0.0\t2\t7 4 2\n",
      "21\t588\t0.0\t2\t0 579 9\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t2\t0.0\t2\t1 1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t2\t0.0\t2\t2\n",
      "67\t5\t0.0\t2\t5\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t2\n",
      "70\t4\t0.0\t2\t3 0 1\n",
      "71\t2\t0.0\t2\t0 1 1\n",
      "74\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t0 1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t10\t0.0\t2\t9 0 1\n",
      "99\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t1\n",
      "121\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "205\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "250\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCA.4.23.19_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCA.4.23.19_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCA.4.23.19_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCA.4.23.19_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 8.37 s (15 us/read; 3.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            554,802\n",
      "  Read 1 with adapter:                     471 (0.1%)\n",
      "  Read 2 with adapter:                  18,541 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       554,802 (100.0%)\n",
      "\n",
      "Total basepairs processed:   252,640,322 bp\n",
      "  Read 1:   124,141,018 bp\n",
      "  Read 2:   128,499,304 bp\n",
      "Total written (filtered):    252,358,135 bp (99.9%)\n",
      "  Read 1:   124,138,322 bp\n",
      "  Read 2:   128,219,813 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 471 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t441\t8668.8\t0\t441\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t8\t0.0\t1\t7 1\n",
      "27\t9\t0.0\t1\t8 1\n",
      "29\t6\t0.0\t1\t5 1\n",
      "35\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18541 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t154\t8668.8\t0\t154\n",
      "4\t52\t2167.2\t0\t52\n",
      "5\t46\t541.8\t0\t46\n",
      "6\t110\t135.4\t0\t110\n",
      "7\t269\t33.9\t0\t269\n",
      "8\t419\t8.5\t0\t419\n",
      "9\t1441\t2.1\t0\t1437 4\n",
      "10\t2403\t0.5\t1\t2379 24\n",
      "11\t308\t0.1\t1\t299 9\n",
      "12\t1004\t0.0\t1\t988 16\n",
      "13\t411\t0.0\t1\t398 13\n",
      "14\t655\t0.0\t1\t616 39\n",
      "15\t544\t0.0\t1\t519 25\n",
      "16\t1505\t0.0\t1\t1456 49\n",
      "17\t2223\t0.0\t1\t2154 69\n",
      "18\t1332\t0.0\t1\t1191 90 51\n",
      "19\t4928\t0.0\t1\t2998 1904 26\n",
      "20\t10\t0.0\t2\t6 3 1\n",
      "21\t660\t0.0\t2\t0 649 11\n",
      "22\t42\t0.0\t2\t0 1 41\n",
      "52\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "71\t3\t0.0\t2\t2 0 1\n",
      "72\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t6\t0.0\t2\t4 1 1\n",
      "91\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "152\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t1\n",
      "240\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCA.4.26.19_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCA.4.26.19_64_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCA.4.26.19_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCA.4.26.19_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.36 s (26 us/read; 2.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,939\n",
      "  Read 1 with adapter:                      63 (0.1%)\n",
      "  Read 2 with adapter:                   1,016 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,939 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,463,665 bp\n",
      "  Read 1:    11,527,835 bp\n",
      "  Read 2:    11,935,830 bp\n",
      "Total written (filtered):     23,445,799 bp (99.9%)\n",
      "  Read 1:    11,526,321 bp\n",
      "  Read 2:    11,919,478 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 63 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t55\t811.5\t0\t55\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t3 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1016 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t811.5\t0\t6\n",
      "4\t2\t202.9\t0\t2\n",
      "5\t5\t50.7\t0\t5\n",
      "6\t1\t12.7\t0\t1\n",
      "7\t10\t3.2\t0\t10\n",
      "8\t17\t0.8\t0\t17\n",
      "9\t35\t0.2\t0\t35\n",
      "10\t121\t0.0\t1\t121\n",
      "11\t15\t0.0\t1\t15\n",
      "12\t13\t0.0\t1\t13\n",
      "13\t15\t0.0\t1\t15\n",
      "14\t19\t0.0\t1\t17 2\n",
      "15\t33\t0.0\t1\t30 3\n",
      "16\t88\t0.0\t1\t85 3\n",
      "17\t119\t0.0\t1\t114 5\n",
      "18\t115\t0.0\t1\t107 4 4\n",
      "19\t329\t0.0\t1\t219 110\n",
      "21\t71\t0.0\t2\t0 68 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D3.5.4.19_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D3.5.4.19_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D3.5.4.19_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D3.5.4.19_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.4.26.19_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.4.26.19_49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.4.26.19_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.4.26.19_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.F.5.4.19_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.F.5.4.19_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.F.5.4.19_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.F.5.4.19_18_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.N.5.4.19_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCC.D4.N.5.4.19_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.N.5.4.19_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCC.D4.N.5.4.19_8_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D1.4.26.19_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D1.4.26.19_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D1.4.26.19_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D1.4.26.19_25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D2.4.26.19_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D2.4.26.19_65_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D2.4.26.19_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D2.4.26.19_65_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D2.5.4.19_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D2.5.4.19_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D2.5.4.19_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D2.5.4.19_12_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D3.4.26.19_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D3.4.26.19_62_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D3.4.26.19_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D3.4.26.19_62_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D3.5.4.19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D3.5.4.19_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D3.5.4.19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D3.5.4.19_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.4.26.19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.4.26.19_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.4.26.19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.4.26.19_1_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.5.4.19_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.5.4.19_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.5.4.19_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.5.4.19_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.5percent.5.31.19_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.DCD.D4.5percent.5.31.19_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.5percent.5.31.19_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.DCD.D4.5percent.5.31.19_11_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCA.4.23.19_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCA.4.23.19_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCA.4.23.19_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCA.4.23.19_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCA.4.26.19_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCA.4.26.19_64_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCA.4.26.19_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCA.4.26.19_64_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCB.4.23.19_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCB.4.23.19_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCB.4.23.19_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCB.4.23.19_6_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCB.4.23.19_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCB.4.23.19_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCB.4.23.19_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCB.4.23.19_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.94 s (17 us/read; 3.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,990\n",
      "  Read 1 with adapter:                     125 (0.1%)\n",
      "  Read 2 with adapter:                   2,259 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,990 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,915,207 bp\n",
      "  Read 1:    25,521,263 bp\n",
      "  Read 2:    26,393,944 bp\n",
      "Total written (filtered):     51,878,243 bp (99.9%)\n",
      "  Read 1:    25,520,207 bp\n",
      "  Read 2:    26,358,036 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 125 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t117\t1781.1\t0\t117\n",
      "15\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "30\t1\t0.0\t1\t1\n",
      "179\t1\t0.0\t1\t1\n",
      "181\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2259 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1781.1\t0\t9\n",
      "4\t3\t445.3\t0\t3\n",
      "5\t2\t111.3\t0\t2\n",
      "6\t5\t27.8\t0\t5\n",
      "7\t8\t7.0\t0\t8\n",
      "8\t15\t1.7\t0\t15\n",
      "9\t80\t0.4\t0\t79 1\n",
      "10\t336\t0.1\t1\t333 3\n",
      "11\t52\t0.0\t1\t51 1\n",
      "12\t59\t0.0\t1\t58 1\n",
      "13\t33\t0.0\t1\t31 2\n",
      "14\t44\t0.0\t1\t37 7\n",
      "15\t75\t0.0\t1\t73 2\n",
      "16\t182\t0.0\t1\t176 6\n",
      "17\t279\t0.0\t1\t271 8\n",
      "18\t218\t0.0\t1\t203 12 3\n",
      "19\t754\t0.0\t1\t461 288 5\n",
      "20\t6\t0.0\t2\t5 1\n",
      "21\t96\t0.0\t2\t0 94 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "38\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCB.4.26.19_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCB.4.26.19_55_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCB.4.26.19_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCB.4.26.19_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.63 s (18 us/read; 3.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            146,075\n",
      "  Read 1 with adapter:                     135 (0.1%)\n",
      "  Read 2 with adapter:                   3,235 (2.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       146,075 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,325,957 bp\n",
      "  Read 1:    32,592,867 bp\n",
      "  Read 2:    33,733,090 bp\n",
      "Total written (filtered):     66,272,197 bp (99.9%)\n",
      "  Read 1:    32,591,984 bp\n",
      "  Read 2:    33,680,213 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 135 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t131\t2282.4\t0\t131\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3235 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t2282.4\t0\t20\n",
      "4\t7\t570.6\t0\t7\n",
      "5\t8\t142.7\t0\t8\n",
      "6\t11\t35.7\t0\t11\n",
      "7\t22\t8.9\t0\t22\n",
      "8\t32\t2.2\t0\t32\n",
      "9\t128\t0.6\t0\t127 1\n",
      "10\t482\t0.1\t1\t476 6\n",
      "11\t45\t0.0\t1\t45\n",
      "12\t42\t0.0\t1\t41 1\n",
      "13\t34\t0.0\t1\t33 1\n",
      "14\t67\t0.0\t1\t61 6\n",
      "15\t88\t0.0\t1\t84 4\n",
      "16\t269\t0.0\t1\t262 7\n",
      "17\t428\t0.0\t1\t423 5\n",
      "18\t270\t0.0\t1\t256 10 4\n",
      "19\t1089\t0.0\t1\t697 388 4\n",
      "20\t3\t0.0\t2\t1 1 1\n",
      "21\t176\t0.0\t2\t0 175 1\n",
      "22\t5\t0.0\t2\t0 0 5\n",
      "79\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t4\t0.0\t2\t4\n",
      "227\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.4.23.19_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.4.23.19_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.4.23.19_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.4.23.19_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.39 s (13 us/read; 4.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            549,237\n",
      "  Read 1 with adapter:                     709 (0.1%)\n",
      "  Read 2 with adapter:                  18,179 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       549,237 (100.0%)\n",
      "\n",
      "Total basepairs processed:   250,200,855 bp\n",
      "  Read 1:   122,920,316 bp\n",
      "  Read 2:   127,280,539 bp\n",
      "Total written (filtered):    249,918,287 bp (99.9%)\n",
      "  Read 1:   122,916,341 bp\n",
      "  Read 2:   127,001,946 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 709 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t685\t8581.8\t0\t685\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t9\t0.0\t1\t9\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "35\t1\t0.0\t1\t1\n",
      "190\t1\t0.0\t1\t1\n",
      "201\t2\t0.0\t1\t1 1\n",
      "210\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18179 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t99\t8581.8\t0\t99\n",
      "4\t28\t2145.5\t0\t28\n",
      "5\t46\t536.4\t0\t46\n",
      "6\t128\t134.1\t0\t128\n",
      "7\t239\t33.5\t0\t239\n",
      "8\t469\t8.4\t0\t469\n",
      "9\t1513\t2.1\t0\t1513\n",
      "10\t2160\t0.5\t1\t2143 17\n",
      "11\t293\t0.1\t1\t284 9\n",
      "12\t549\t0.0\t1\t540 9\n",
      "13\t602\t0.0\t1\t582 20\n",
      "14\t756\t0.0\t1\t705 51\n",
      "15\t481\t0.0\t1\t453 28\n",
      "16\t1363\t0.0\t1\t1308 55\n",
      "17\t2288\t0.0\t1\t2201 87\n",
      "18\t1339\t0.0\t1\t1182 97 60\n",
      "19\t5017\t0.0\t1\t2939 2056 22\n",
      "20\t18\t0.0\t2\t14 4\n",
      "21\t725\t0.0\t2\t0 711 14\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "41\t1\t0.0\t2\t1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t1\n",
      "79\t6\t0.0\t2\t4 0 2\n",
      "88\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "122\t1\t0.0\t2\t0 1\n",
      "128\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t0 1\n",
      "187\t1\t0.0\t2\t0 0 1\n",
      "194\t2\t0.0\t2\t1 1\n",
      "213\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t4\t0.0\t2\t3 1\n",
      "226\t1\t0.0\t2\t0 1\n",
      "234\t1\t0.0\t2\t0 1\n",
      "238\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D1.4.26.19_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D1.4.26.19_53_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D1.4.26.19_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D1.4.26.19_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.09 s (35 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,845\n",
      "  Read 1 with adapter:                      28 (0.1%)\n",
      "  Read 2 with adapter:                     582 (1.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,845 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,819,120 bp\n",
      "  Read 1:     6,784,004 bp\n",
      "  Read 2:     7,035,116 bp\n",
      "Total written (filtered):     13,808,993 bp (99.9%)\n",
      "  Read 1:     6,783,441 bp\n",
      "  Read 2:     7,025,552 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 28 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t482.0\t0\t24\n",
      "29\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 582 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t482.0\t0\t2\n",
      "4\t2\t120.5\t0\t2\n",
      "6\t6\t7.5\t0\t6\n",
      "7\t7\t1.9\t0\t7\n",
      "8\t9\t0.5\t0\t9\n",
      "9\t22\t0.1\t0\t22\n",
      "10\t63\t0.0\t1\t61 2\n",
      "11\t14\t0.0\t1\t14\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t20\t0.0\t1\t20\n",
      "15\t18\t0.0\t1\t16 2\n",
      "16\t47\t0.0\t1\t46 1\n",
      "17\t75\t0.0\t1\t74 1\n",
      "18\t63\t0.0\t1\t57 5 1\n",
      "19\t187\t0.0\t1\t122 65\n",
      "21\t33\t0.0\t2\t0 33\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "70\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D1.5.4.19_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D1.5.4.19_63_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D1.5.4.19_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D1.5.4.19_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.58 s (22 us/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            119,080\n",
      "  Read 1 with adapter:                      91 (0.1%)\n",
      "  Read 2 with adapter:                   5,159 (4.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       119,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,882,625 bp\n",
      "  Read 1:    25,927,688 bp\n",
      "  Read 2:    26,954,937 bp\n",
      "Total written (filtered):     52,803,436 bp (99.9%)\n",
      "  Read 1:    25,921,645 bp\n",
      "  Read 2:    26,881,791 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t1860.6\t0\t59\n",
      "6\t1\t29.1\t0\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "173\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t0 1\n",
      "208\t1\t0.0\t1\t0 1\n",
      "213\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t2\t0.0\t1\t1 1\n",
      "216\t9\t0.0\t1\t7 2\n",
      "217\t10\t0.0\t1\t9 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5159 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t1860.6\t0\t40\n",
      "4\t13\t465.2\t0\t13\n",
      "5\t8\t116.3\t0\t8\n",
      "6\t34\t29.1\t0\t34\n",
      "7\t93\t7.3\t0\t93\n",
      "8\t113\t1.8\t0\t113\n",
      "9\t479\t0.5\t0\t477 2\n",
      "10\t1469\t0.1\t1\t1452 17\n",
      "11\t65\t0.0\t1\t64 1\n",
      "12\t49\t0.0\t1\t49\n",
      "13\t61\t0.0\t1\t61\n",
      "14\t100\t0.0\t1\t94 6\n",
      "15\t158\t0.0\t1\t153 5\n",
      "16\t383\t0.0\t1\t369 14\n",
      "17\t546\t0.0\t1\t523 23\n",
      "18\t329\t0.0\t1\t294 30 5\n",
      "19\t1066\t0.0\t1\t648 416 2\n",
      "20\t4\t0.0\t2\t2 2\n",
      "21\t124\t0.0\t2\t0 122 2\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "28\t1\t0.0\t2\t1\n",
      "67\t2\t0.0\t2\t2\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t11\t0.0\t2\t9 2\n",
      "153\t1\t0.0\t2\t0 1\n",
      "171\t1\t0.0\t2\t1\n",
      "186\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D2.4.26.19_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D2.4.26.19_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D2.4.26.19_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D2.4.26.19_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.56 s (26 us/read; 2.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,587\n",
      "  Read 1 with adapter:                      42 (0.1%)\n",
      "  Read 2 with adapter:                   1,161 (1.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        59,587 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,944,561 bp\n",
      "  Read 1:    13,239,458 bp\n",
      "  Read 2:    13,705,103 bp\n",
      "Total written (filtered):     26,925,096 bp (99.9%)\n",
      "  Read 1:    13,238,624 bp\n",
      "  Read 2:    13,686,472 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 42 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t931.0\t0\t36\n",
      "26\t3\t0.0\t1\t3\n",
      "214\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1161 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t931.0\t0\t6\n",
      "4\t3\t232.8\t0\t3\n",
      "5\t3\t58.2\t0\t3\n",
      "6\t8\t14.5\t0\t8\n",
      "7\t18\t3.6\t0\t18\n",
      "8\t17\t0.9\t0\t17\n",
      "9\t40\t0.2\t0\t40\n",
      "10\t101\t0.1\t1\t98 3\n",
      "11\t20\t0.0\t1\t20\n",
      "12\t31\t0.0\t1\t31\n",
      "13\t18\t0.0\t1\t18\n",
      "14\t30\t0.0\t1\t28 2\n",
      "15\t47\t0.0\t1\t45 2\n",
      "16\t98\t0.0\t1\t97 1\n",
      "17\t185\t0.0\t1\t174 11\n",
      "18\t113\t0.0\t1\t109 3 1\n",
      "19\t369\t0.0\t1\t224 141 4\n",
      "21\t48\t0.0\t2\t0 47 1\n",
      "22\t5\t0.0\t2\t0 1 4\n",
      "79\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D2.5.4.19_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D2.5.4.19_61_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D2.5.4.19_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D2.5.4.19_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.38 s (30 us/read; 1.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,555\n",
      "  Read 1 with adapter:                      40 (0.1%)\n",
      "  Read 2 with adapter:                   2,147 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,555 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,904,888 bp\n",
      "  Read 1:     9,741,154 bp\n",
      "  Read 2:    10,163,734 bp\n",
      "Total written (filtered):     19,873,042 bp (99.8%)\n",
      "  Read 1:     9,739,443 bp\n",
      "  Read 2:    10,133,599 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 40 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t711.8\t0\t29\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t5\t0.0\t1\t3 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2147 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t711.8\t0\t16\n",
      "5\t4\t44.5\t0\t4\n",
      "6\t13\t11.1\t0\t13\n",
      "7\t25\t2.8\t0\t25\n",
      "8\t54\t0.7\t0\t54\n",
      "9\t199\t0.2\t0\t199\n",
      "10\t616\t0.0\t1\t609 7\n",
      "11\t33\t0.0\t1\t33\n",
      "12\t17\t0.0\t1\t16 1\n",
      "13\t13\t0.0\t1\t13\n",
      "14\t65\t0.0\t1\t63 2\n",
      "15\t65\t0.0\t1\t65\n",
      "16\t135\t0.0\t1\t132 3\n",
      "17\t249\t0.0\t1\t236 13\n",
      "18\t139\t0.0\t1\t127 9 3\n",
      "19\t435\t0.0\t1\t253 181 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t61\t0.0\t2\t0 61\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D3.4.26.19_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D3.4.26.19_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D3.4.26.19_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D3.4.26.19_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.77 s (19 us/read; 3.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            148,679\n",
      "  Read 1 with adapter:                     178 (0.1%)\n",
      "  Read 2 with adapter:                   3,304 (2.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       148,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,606,546 bp\n",
      "  Read 1:    33,230,128 bp\n",
      "  Read 2:    34,376,418 bp\n",
      "Total written (filtered):     67,552,256 bp (99.9%)\n",
      "  Read 1:    33,228,226 bp\n",
      "  Read 2:    34,324,030 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 178 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t167\t2323.1\t0\t167\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "100\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3304 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t2323.1\t0\t14\n",
      "4\t10\t580.8\t0\t10\n",
      "5\t11\t145.2\t0\t11\n",
      "6\t10\t36.3\t0\t10\n",
      "7\t41\t9.1\t0\t41\n",
      "8\t48\t2.3\t0\t48\n",
      "9\t172\t0.6\t0\t172\n",
      "10\t394\t0.1\t1\t391 3\n",
      "11\t69\t0.0\t1\t66 3\n",
      "12\t84\t0.0\t1\t83 1\n",
      "13\t53\t0.0\t1\t50 3\n",
      "14\t80\t0.0\t1\t73 7\n",
      "15\t109\t0.0\t1\t107 2\n",
      "16\t290\t0.0\t1\t282 8\n",
      "17\t424\t0.0\t1\t414 10\n",
      "18\t306\t0.0\t1\t276 22 8\n",
      "19\t1023\t0.0\t1\t617 402 4\n",
      "20\t1\t0.0\t2\t1\n",
      "21\t161\t0.0\t2\t0 160 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D3.5.4.19_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D3.5.4.19_68_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D3.5.4.19_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D3.5.4.19_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.17 s (40 us/read; 1.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,512\n",
      "  Read 1 with adapter:                      21 (0.1%)\n",
      "  Read 2 with adapter:                   1,445 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,512 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,647,799 bp\n",
      "  Read 1:     6,173,926 bp\n",
      "  Read 2:     6,473,873 bp\n",
      "Total written (filtered):     12,626,345 bp (99.8%)\n",
      "  Read 1:     6,172,560 bp\n",
      "  Read 2:     6,453,785 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t461.1\t0\t14\n",
      "29\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1445 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t461.1\t0\t8\n",
      "4\t2\t115.3\t0\t2\n",
      "5\t3\t28.8\t0\t3\n",
      "6\t8\t7.2\t0\t8\n",
      "7\t28\t1.8\t0\t28\n",
      "8\t44\t0.5\t0\t44\n",
      "9\t167\t0.1\t0\t167\n",
      "10\t429\t0.0\t1\t422 7\n",
      "11\t20\t0.0\t1\t18 2\n",
      "12\t20\t0.0\t1\t20\n",
      "13\t11\t0.0\t1\t11\n",
      "14\t33\t0.0\t1\t30 3\n",
      "15\t29\t0.0\t1\t28 1\n",
      "16\t113\t0.0\t1\t112 1\n",
      "17\t147\t0.0\t1\t143 4\n",
      "18\t82\t0.0\t1\t72 9 1\n",
      "19\t251\t0.0\t1\t159 92\n",
      "20\t1\t0.0\t2\t0 1\n",
      "21\t42\t0.0\t2\t0 42\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "99\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "192\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D4.4.26.19_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D4.4.26.19_56_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D4.4.26.19_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D4.4.26.19_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.23 s (29 us/read; 2.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,137\n",
      "  Read 1 with adapter:                      34 (0.1%)\n",
      "  Read 2 with adapter:                   1,003 (2.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,137 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,709,427 bp\n",
      "  Read 1:     9,184,989 bp\n",
      "  Read 2:     9,524,438 bp\n",
      "Total written (filtered):     18,691,839 bp (99.9%)\n",
      "  Read 1:     9,183,177 bp\n",
      "  Read 2:     9,508,662 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 34 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t658.4\t0\t26\n",
      "215\t1\t0.0\t1\t1\n",
      "217\t7\t0.0\t1\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1003 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t658.4\t0\t3\n",
      "4\t1\t164.6\t0\t1\n",
      "5\t3\t41.1\t0\t3\n",
      "6\t5\t10.3\t0\t5\n",
      "7\t9\t2.6\t0\t9\n",
      "8\t10\t0.6\t0\t10\n",
      "9\t55\t0.2\t0\t54 1\n",
      "10\t147\t0.0\t1\t147\n",
      "11\t8\t0.0\t1\t8\n",
      "12\t15\t0.0\t1\t15\n",
      "13\t24\t0.0\t1\t24\n",
      "14\t16\t0.0\t1\t16\n",
      "15\t34\t0.0\t1\t34\n",
      "16\t74\t0.0\t1\t73 1\n",
      "17\t140\t0.0\t1\t135 5\n",
      "18\t104\t0.0\t1\t98 2 4\n",
      "19\t298\t0.0\t1\t180 118\n",
      "21\t56\t0.0\t2\t0 56\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D4.5.4.19_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D4.5.4.19_67_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D4.5.4.19_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D4.5.4.19_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.13 s (30 us/read; 1.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,158\n",
      "  Read 1 with adapter:                      57 (0.2%)\n",
      "  Read 2 with adapter:                   2,123 (5.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        37,158 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,949,702 bp\n",
      "  Read 1:     7,793,093 bp\n",
      "  Read 2:     8,156,609 bp\n",
      "Total written (filtered):     15,915,170 bp (99.8%)\n",
      "  Read 1:     7,787,508 bp\n",
      "  Read 2:     8,127,662 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 57 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t580.6\t0\t27\n",
      "13\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "205\t2\t0.0\t1\t1 1\n",
      "206\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t7\t0.0\t1\t6 1\n",
      "217\t13\t0.0\t1\t13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2123 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t580.6\t0\t17\n",
      "4\t3\t145.1\t0\t3\n",
      "5\t7\t36.3\t0\t7\n",
      "6\t13\t9.1\t0\t13\n",
      "7\t37\t2.3\t0\t37\n",
      "8\t50\t0.6\t0\t50\n",
      "9\t219\t0.1\t0\t218 1\n",
      "10\t675\t0.0\t1\t666 9\n",
      "11\t31\t0.0\t1\t31\n",
      "12\t20\t0.0\t1\t20\n",
      "13\t23\t0.0\t1\t21 2\n",
      "14\t49\t0.0\t1\t47 2\n",
      "15\t48\t0.0\t1\t46 2\n",
      "16\t140\t0.0\t1\t136 4\n",
      "17\t187\t0.0\t1\t181 6\n",
      "18\t137\t0.0\t1\t124 11 2\n",
      "19\t405\t0.0\t1\t226 176 3\n",
      "20\t1\t0.0\t2\t1\n",
      "21\t57\t0.0\t2\t0 56 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCD.4.23.19_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCD.4.23.19_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCD.4.23.19_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCD.4.23.19_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.26 s (17 us/read; 3.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            374,178\n",
      "  Read 1 with adapter:                     233 (0.1%)\n",
      "  Read 2 with adapter:                  16,535 (4.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       374,178 (100.0%)\n",
      "\n",
      "Total basepairs processed:   170,425,044 bp\n",
      "  Read 1:    83,729,696 bp\n",
      "  Read 2:    86,695,348 bp\n",
      "Total written (filtered):    170,177,009 bp (99.9%)\n",
      "  Read 1:    83,727,773 bp\n",
      "  Read 2:    86,449,236 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 233 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t216\t5846.5\t0\t216\n",
      "18\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t4\t0.0\t1\t4\n",
      "29\t4\t0.0\t1\t4\n",
      "30\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "118\t1\t0.0\t1\t0 1\n",
      "173\t1\t0.0\t1\t1\n",
      "177\t1\t0.0\t1\t0 1\n",
      "193\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 16535 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t104\t5846.5\t0\t104\n",
      "4\t38\t1461.6\t0\t38\n",
      "5\t48\t365.4\t0\t48\n",
      "6\t96\t91.4\t0\t96\n",
      "7\t261\t22.8\t0\t261\n",
      "8\t438\t5.7\t0\t438\n",
      "9\t1525\t1.4\t0\t1523 2\n",
      "10\t2410\t0.4\t1\t2378 32\n",
      "11\t234\t0.1\t1\t231 3\n",
      "12\t500\t0.0\t1\t490 10\n",
      "13\t639\t0.0\t1\t617 22\n",
      "14\t932\t0.0\t1\t877 55\n",
      "15\t564\t0.0\t1\t540 24\n",
      "16\t1382\t0.0\t1\t1318 64\n",
      "17\t1902\t0.0\t1\t1822 80\n",
      "18\t1095\t0.0\t1\t970 80 45\n",
      "19\t3735\t0.0\t1\t2190 1532 13\n",
      "20\t24\t0.0\t2\t17 6 1\n",
      "21\t519\t0.0\t2\t0 511 8\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "50\t2\t0.0\t2\t1 1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t4\t0.0\t2\t1 0 3\n",
      "74\t3\t0.0\t2\t3\n",
      "75\t1\t0.0\t2\t0 1\n",
      "78\t2\t0.0\t2\t0 1 1\n",
      "79\t10\t0.0\t2\t8 1 1\n",
      "88\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "93\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 1\n",
      "166\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t1\n",
      "186\t3\t0.0\t2\t2 1\n",
      "216\t5\t0.0\t2\t2 2 1\n",
      "217\t1\t0.0\t2\t1\n",
      "223\t1\t0.0\t2\t1\n",
      "237\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCD.4.26.19_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCD.4.26.19_59_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCD.4.26.19_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCD.4.26.19_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.01 s (19 us/read; 3.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            158,342\n",
      "  Read 1 with adapter:                     157 (0.1%)\n",
      "  Read 2 with adapter:                   7,205 (4.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       158,342 (100.0%)\n",
      "\n",
      "Total basepairs processed:    71,741,307 bp\n",
      "  Read 1:    35,215,778 bp\n",
      "  Read 2:    36,525,529 bp\n",
      "Total written (filtered):     71,631,798 bp (99.8%)\n",
      "  Read 1:    35,213,062 bp\n",
      "  Read 2:    36,418,736 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 157 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t138\t2474.1\t0\t138\n",
      "26\t2\t0.0\t1\t1 1\n",
      "27\t4\t0.0\t1\t4\n",
      "29\t3\t0.0\t1\t3\n",
      "107\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t6\t0.0\t1\t5 1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7205 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t2474.1\t0\t37\n",
      "4\t12\t618.5\t0\t12\n",
      "5\t17\t154.6\t0\t17\n",
      "6\t56\t38.7\t0\t56\n",
      "7\t116\t9.7\t0\t116\n",
      "8\t205\t2.4\t0\t205\n",
      "9\t754\t0.6\t0\t751 3\n",
      "10\t985\t0.2\t1\t971 14\n",
      "11\t139\t0.0\t1\t138 1\n",
      "12\t242\t0.0\t1\t239 3\n",
      "13\t295\t0.0\t1\t284 11\n",
      "14\t357\t0.0\t1\t330 27\n",
      "15\t243\t0.0\t1\t221 22\n",
      "16\t618\t0.0\t1\t592 26\n",
      "17\t838\t0.0\t1\t799 39\n",
      "18\t459\t0.0\t1\t393 41 25\n",
      "19\t1596\t0.0\t1\t926 659 11\n",
      "20\t9\t0.0\t2\t4 4 1\n",
      "21\t197\t0.0\t2\t0 192 5\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "23\t2\t0.0\t2\t1 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "71\t4\t0.0\t2\t0 0 4\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t5\t0.0\t2\t4 0 1\n",
      "98\t1\t0.0\t2\t1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t1\n",
      "157\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t0 1\n",
      "206\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCE.4.23.19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCE.4.23.19_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCE.4.23.19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCE.4.23.19_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.67 s (17 us/read; 3.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            154,545\n",
      "  Read 1 with adapter:                     160 (0.1%)\n",
      "  Read 2 with adapter:                   3,316 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       154,545 (100.0%)\n",
      "\n",
      "Total basepairs processed:    70,222,158 bp\n",
      "  Read 1:    34,518,366 bp\n",
      "  Read 2:    35,703,792 bp\n",
      "Total written (filtered):     70,167,674 bp (99.9%)\n",
      "  Read 1:    34,516,821 bp\n",
      "  Read 2:    35,650,853 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 160 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t147\t2414.8\t0\t147\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t5\t0.0\t1\t5\n",
      "29\t3\t0.0\t1\t3\n",
      "207\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3316 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t2414.8\t0\t18\n",
      "4\t3\t603.7\t0\t3\n",
      "5\t6\t150.9\t0\t6\n",
      "6\t5\t37.7\t0\t5\n",
      "7\t30\t9.4\t0\t30\n",
      "8\t38\t2.4\t0\t38\n",
      "9\t148\t0.6\t0\t148\n",
      "10\t554\t0.1\t1\t547 7\n",
      "11\t65\t0.0\t1\t62 3\n",
      "12\t53\t0.0\t1\t51 2\n",
      "13\t34\t0.0\t1\t32 2\n",
      "14\t64\t0.0\t1\t60 4\n",
      "15\t80\t0.0\t1\t76 4\n",
      "16\t273\t0.0\t1\t272 1\n",
      "17\t404\t0.0\t1\t395 9\n",
      "18\t289\t0.0\t1\t265 16 8\n",
      "19\t1065\t0.0\t1\t688 371 6\n",
      "20\t6\t0.0\t2\t3 1 2\n",
      "21\t169\t0.0\t2\t0 168 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "52\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "80\t1\t0.0\t2\t0 1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "214\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCE.4.26.19_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCE.4.26.19_57_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCE.4.26.19_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCE.4.26.19_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.23 s (19 us/read; 3.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,783\n",
      "  Read 1 with adapter:                     164 (0.1%)\n",
      "  Read 2 with adapter:                   3,474 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,783 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,369,957 bp\n",
      "  Read 1:    25,717,989 bp\n",
      "  Read 2:    26,651,968 bp\n",
      "Total written (filtered):     52,315,514 bp (99.9%)\n",
      "  Read 1:    25,716,186 bp\n",
      "  Read 2:    26,599,328 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 164 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t148\t1809.1\t0\t148\n",
      "20\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t3 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3474 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1809.1\t0\t16\n",
      "4\t3\t452.3\t0\t3\n",
      "5\t7\t113.1\t0\t7\n",
      "6\t16\t28.3\t0\t16\n",
      "7\t38\t7.1\t0\t38\n",
      "8\t68\t1.8\t0\t68\n",
      "9\t190\t0.4\t0\t189 1\n",
      "10\t786\t0.1\t1\t781 5\n",
      "11\t57\t0.0\t1\t55 2\n",
      "12\t57\t0.0\t1\t57\n",
      "13\t38\t0.0\t1\t35 3\n",
      "14\t61\t0.0\t1\t53 8\n",
      "15\t84\t0.0\t1\t84\n",
      "16\t269\t0.0\t1\t258 11\n",
      "17\t379\t0.0\t1\t368 11\n",
      "18\t271\t0.0\t1\t256 11 4\n",
      "19\t975\t0.0\t1\t623 348 4\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t150\t0.0\t2\t0 146 4\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "56\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "217\t2\t0.0\t2\t2\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCB.4.26.19_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCB.4.26.19_55_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCB.4.26.19_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCB.4.26.19_55_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.4.23.19_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.4.23.19_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.4.23.19_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.4.23.19_3_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D1.4.26.19_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D1.4.26.19_53_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D1.4.26.19_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D1.4.26.19_53_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D1.5.4.19_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D1.5.4.19_63_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D1.5.4.19_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D1.5.4.19_63_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D2.4.26.19_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D2.4.26.19_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D2.4.26.19_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D2.4.26.19_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D2.5.4.19_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D2.5.4.19_61_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D2.5.4.19_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D2.5.4.19_61_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D3.4.26.19_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D3.4.26.19_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D3.4.26.19_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D3.4.26.19_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D3.5.4.19_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D3.5.4.19_68_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D3.5.4.19_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D3.5.4.19_68_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D4.4.26.19_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D4.4.26.19_56_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D4.4.26.19_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D4.4.26.19_56_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D4.5.4.19_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCC.D4.5.4.19_67_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D4.5.4.19_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCC.D4.5.4.19_67_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCD.4.23.19_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCD.4.23.19_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCD.4.23.19_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCD.4.23.19_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCD.4.26.19_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCD.4.26.19_59_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCD.4.26.19_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCD.4.26.19_59_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCE.4.23.19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCE.4.23.19_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCE.4.23.19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCE.4.23.19_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCE.4.26.19_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCE.4.26.19_57_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCE.4.26.19_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCE.4.26.19_57_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCF.4.23.19_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCF.4.23.19_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCF.4.23.19_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCF.4.23.19_4_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCF.4.23.19_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCF.4.23.19_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCF.4.23.19_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCF.4.23.19_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.82 s (15 us/read; 3.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            517,854\n",
      "  Read 1 with adapter:                     466 (0.1%)\n",
      "  Read 2 with adapter:                  13,035 (2.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       517,854 (100.0%)\n",
      "\n",
      "Total basepairs processed:   235,923,826 bp\n",
      "  Read 1:   115,960,557 bp\n",
      "  Read 2:   119,963,269 bp\n",
      "Total written (filtered):    235,709,955 bp (99.9%)\n",
      "  Read 1:   115,957,302 bp\n",
      "  Read 2:   119,752,653 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 466 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t434\t8091.5\t0\t434\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t17\t0.0\t1\t15 2\n",
      "92\t1\t0.0\t1\t1\n",
      "103\t1\t0.0\t1\t1\n",
      "155\t6\t0.0\t1\t6\n",
      "179\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13035 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t8091.5\t0\t49\n",
      "4\t21\t2022.9\t0\t21\n",
      "5\t24\t505.7\t0\t24\n",
      "6\t87\t126.4\t0\t87\n",
      "7\t147\t31.6\t0\t147\n",
      "8\t196\t7.9\t0\t196\n",
      "9\t686\t2.0\t0\t685 1\n",
      "10\t1216\t0.5\t1\t1206 10\n",
      "11\t165\t0.1\t1\t158 7\n",
      "12\t352\t0.0\t1\t344 8\n",
      "13\t311\t0.0\t1\t296 15\n",
      "14\t525\t0.0\t1\t469 56\n",
      "15\t375\t0.0\t1\t357 18\n",
      "16\t1069\t0.0\t1\t1032 37\n",
      "17\t1763\t0.0\t1\t1713 50\n",
      "18\t1039\t0.0\t1\t933 66 40\n",
      "19\t4258\t0.0\t1\t2521 1726 11\n",
      "20\t15\t0.0\t2\t6 7 2\n",
      "21\t678\t0.0\t2\t0 673 5\n",
      "22\t33\t0.0\t2\t0 1 32\n",
      "61\t2\t0.0\t2\t2\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t2\t0.0\t2\t2\n",
      "79\t3\t0.0\t2\t3\n",
      "86\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 1\n",
      "158\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "195\t1\t0.0\t2\t1\n",
      "208\t1\t0.0\t2\t1\n",
      "215\t2\t0.0\t2\t2\n",
      "217\t5\t0.0\t2\t4 0 1\n",
      "233\t1\t0.0\t2\t1\n",
      "248\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCF.4.26.19_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCF.4.26.19_51_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCF.4.26.19_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCF.4.26.19_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.75 s (21 us/read; 2.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            133,993\n",
      "  Read 1 with adapter:                     121 (0.1%)\n",
      "  Read 2 with adapter:                   5,003 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       133,993 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,625,768 bp\n",
      "  Read 1:    29,758,333 bp\n",
      "  Read 2:    30,867,435 bp\n",
      "Total written (filtered):     60,545,591 bp (99.9%)\n",
      "  Read 1:    29,755,180 bp\n",
      "  Read 2:    30,790,411 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 121 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t103\t2093.6\t0\t103\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "155\t4\t0.0\t1\t3 1\n",
      "174\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t1 3\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5003 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t2093.6\t0\t27\n",
      "4\t11\t523.4\t0\t11\n",
      "5\t9\t130.9\t0\t9\n",
      "6\t24\t32.7\t0\t24\n",
      "7\t82\t8.2\t0\t82\n",
      "8\t101\t2.0\t0\t101\n",
      "9\t393\t0.5\t0\t392 1\n",
      "10\t594\t0.1\t1\t590 4\n",
      "11\t66\t0.0\t1\t66\n",
      "12\t171\t0.0\t1\t171\n",
      "13\t146\t0.0\t1\t139 7\n",
      "14\t268\t0.0\t1\t244 24\n",
      "15\t158\t0.0\t1\t150 8\n",
      "16\t434\t0.0\t1\t410 24\n",
      "17\t619\t0.0\t1\t600 19\n",
      "18\t366\t0.0\t1\t317 29 20\n",
      "19\t1338\t0.0\t1\t780 555 3\n",
      "20\t7\t0.0\t2\t2 5\n",
      "21\t166\t0.0\t2\t0 163 3\n",
      "22\t5\t0.0\t2\t0 0 5\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "70\t3\t0.0\t2\t3\n",
      "78\t2\t0.0\t2\t2\n",
      "79\t4\t0.0\t2\t3 1\n",
      "99\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "193\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "235\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCG.4.23.19_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCG.4.23.19_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCG.4.23.19_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCG.4.23.19_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.34 s (15 us/read; 4.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            426,035\n",
      "  Read 1 with adapter:                     191 (0.0%)\n",
      "  Read 2 with adapter:                  10,105 (2.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       426,035 (100.0%)\n",
      "\n",
      "Total basepairs processed:   194,023,459 bp\n",
      "  Read 1:    95,358,155 bp\n",
      "  Read 2:    98,665,304 bp\n",
      "Total written (filtered):    193,865,317 bp (99.9%)\n",
      "  Read 1:    95,356,676 bp\n",
      "  Read 2:    98,508,641 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 191 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t179\t6656.8\t0\t179\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t2 1\n",
      "47\t1\t0.0\t1\t1\n",
      "57\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10105 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t57\t6656.8\t0\t57\n",
      "4\t9\t1664.2\t0\t9\n",
      "5\t34\t416.0\t0\t34\n",
      "6\t43\t104.0\t0\t43\n",
      "7\t97\t26.0\t0\t97\n",
      "8\t123\t6.5\t0\t123\n",
      "9\t534\t1.6\t0\t533 1\n",
      "10\t1988\t0.4\t1\t1961 27\n",
      "11\t175\t0.1\t1\t173 2\n",
      "12\t141\t0.0\t1\t139 2\n",
      "13\t104\t0.0\t1\t98 6\n",
      "14\t195\t0.0\t1\t176 19\n",
      "15\t257\t0.0\t1\t248 9\n",
      "16\t736\t0.0\t1\t712 24\n",
      "17\t1191\t0.0\t1\t1157 34\n",
      "18\t806\t0.0\t1\t748 49 9\n",
      "19\t3068\t0.0\t1\t1978 1079 11\n",
      "20\t15\t0.0\t2\t7 6 2\n",
      "21\t506\t0.0\t2\t0 502 4\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t2\n",
      "143\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t0 2\n",
      "217\t3\t0.0\t2\t3\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCG.4.26.19_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCG.4.26.19_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCG.4.26.19_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCG.4.26.19_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.89 s (17 us/read; 3.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,434\n",
      "  Read 1 with adapter:                     129 (0.1%)\n",
      "  Read 2 with adapter:                   2,842 (2.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,511,887 bp\n",
      "  Read 1:    24,814,982 bp\n",
      "  Read 2:    25,696,905 bp\n",
      "Total written (filtered):     50,466,524 bp (99.9%)\n",
      "  Read 1:    24,812,565 bp\n",
      "  Read 2:    25,653,959 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 129 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t115\t1741.2\t0\t115\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t1\n",
      "211\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "216\t5\t0.0\t1\t4 1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2842 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t1741.2\t0\t13\n",
      "4\t6\t435.3\t0\t6\n",
      "5\t1\t108.8\t0\t1\n",
      "6\t12\t27.2\t0\t12\n",
      "7\t37\t6.8\t0\t37\n",
      "8\t34\t1.7\t0\t34\n",
      "9\t168\t0.4\t0\t168\n",
      "10\t620\t0.1\t1\t617 3\n",
      "11\t56\t0.0\t1\t55 1\n",
      "12\t48\t0.0\t1\t48\n",
      "13\t35\t0.0\t1\t34 1\n",
      "14\t69\t0.0\t1\t64 5\n",
      "15\t57\t0.0\t1\t53 4\n",
      "16\t223\t0.0\t1\t216 7\n",
      "17\t313\t0.0\t1\t304 9\n",
      "18\t231\t0.0\t1\t210 16 5\n",
      "19\t785\t0.0\t1\t485 299 1\n",
      "20\t2\t0.0\t2\t1 1\n",
      "21\t125\t0.0\t2\t0 122 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.1.D.5percent.5.29.19_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.1.D.5percent.5.29.19_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.1.D.5percent.5.29.19_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.1.D.5percent.5.29.19_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.34 s (15 us/read; 4.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            298,377\n",
      "  Read 1 with adapter:                     357 (0.1%)\n",
      "  Read 2 with adapter:                   8,023 (2.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       298,377 (100.0%)\n",
      "\n",
      "Total basepairs processed:   135,959,376 bp\n",
      "  Read 1:    66,813,527 bp\n",
      "  Read 2:    69,145,849 bp\n",
      "Total written (filtered):    135,832,883 bp (99.9%)\n",
      "  Read 1:    66,811,849 bp\n",
      "  Read 2:    69,021,034 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 357 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t343\t4662.1\t0\t343\n",
      "19\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t7\t0.0\t1\t7\n",
      "100\t1\t0.0\t1\t0 1\n",
      "220\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8023 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t51\t4662.1\t0\t51\n",
      "4\t17\t1165.5\t0\t17\n",
      "5\t27\t291.4\t0\t27\n",
      "6\t56\t72.8\t0\t56\n",
      "7\t132\t18.2\t0\t132\n",
      "8\t168\t4.6\t0\t168\n",
      "9\t624\t1.1\t0\t621 3\n",
      "10\t1019\t0.3\t1\t1007 12\n",
      "11\t69\t0.1\t1\t68 1\n",
      "12\t142\t0.0\t1\t141 1\n",
      "13\t134\t0.0\t1\t132 2\n",
      "14\t242\t0.0\t1\t222 20\n",
      "15\t204\t0.0\t1\t190 14\n",
      "16\t600\t0.0\t1\t582 18\n",
      "17\t1081\t0.0\t1\t1047 34\n",
      "18\t591\t0.0\t1\t534 38 19\n",
      "19\t2440\t0.0\t1\t1486 947 7\n",
      "20\t12\t0.0\t2\t7 3 2\n",
      "21\t384\t0.0\t2\t0 378 6\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t3\n",
      "87\t1\t0.0\t2\t0 1\n",
      "100\t1\t0.0\t2\t0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t1\n",
      "140\t1\t0.0\t2\t1\n",
      "161\t1\t0.0\t2\t0 1\n",
      "208\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.1.E.5percent.5.21.19_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.1.E.5percent.5.21.19_54_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.1.E.5percent.5.21.19_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.1.E.5percent.5.21.19_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.49 s (16 us/read; 3.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            218,940\n",
      "  Read 1 with adapter:                     193 (0.1%)\n",
      "  Read 2 with adapter:                   9,028 (4.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       218,940 (100.0%)\n",
      "\n",
      "Total basepairs processed:    99,726,988 bp\n",
      "  Read 1:    48,971,018 bp\n",
      "  Read 2:    50,755,970 bp\n",
      "Total written (filtered):     99,604,019 bp (99.9%)\n",
      "  Read 1:    48,969,721 bp\n",
      "  Read 2:    50,634,298 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 193 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t185\t3420.9\t0\t185\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t1 1\n",
      "29\t1\t0.0\t1\t1\n",
      "101\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9028 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t55\t3420.9\t0\t55\n",
      "4\t20\t855.2\t0\t20\n",
      "5\t24\t213.8\t0\t24\n",
      "6\t67\t53.5\t0\t67\n",
      "7\t214\t13.4\t0\t214\n",
      "8\t274\t3.3\t0\t274\n",
      "9\t962\t0.8\t0\t960 2\n",
      "10\t2935\t0.2\t1\t2900 35\n",
      "11\t136\t0.1\t1\t132 4\n",
      "12\t113\t0.0\t1\t112 1\n",
      "13\t78\t0.0\t1\t73 5\n",
      "14\t129\t0.0\t1\t117 12\n",
      "15\t169\t0.0\t1\t163 6\n",
      "16\t512\t0.0\t1\t502 10\n",
      "17\t748\t0.0\t1\t732 16\n",
      "18\t519\t0.0\t1\t472 37 10\n",
      "19\t1745\t0.0\t1\t1094 644 7\n",
      "20\t7\t0.0\t2\t3 2 2\n",
      "21\t304\t0.0\t2\t0 302 2\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "60\t1\t0.0\t2\t0 1\n",
      "79\t3\t0.0\t2\t3\n",
      "154\t1\t0.0\t2\t1\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.10.G.5percent.4.29.19_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.10.G.5percent.4.29.19_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.10.G.5percent.4.29.19_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.10.G.5percent.4.29.19_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.99 s (13 us/read; 4.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            464,851\n",
      "  Read 1 with adapter:                     433 (0.1%)\n",
      "  Read 2 with adapter:                  21,639 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       464,851 (100.0%)\n",
      "\n",
      "Total basepairs processed:   211,998,248 bp\n",
      "  Read 1:   104,077,925 bp\n",
      "  Read 2:   107,920,323 bp\n",
      "Total written (filtered):    211,680,457 bp (99.9%)\n",
      "  Read 1:   104,075,053 bp\n",
      "  Read 2:   107,605,404 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 433 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t414\t7263.3\t0\t414\n",
      "10\t1\t0.4\t1\t1\n",
      "26\t1\t0.0\t1\t0 1\n",
      "29\t5\t0.0\t1\t5\n",
      "36\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "89\t4\t0.0\t1\t4\n",
      "92\t1\t0.0\t1\t1\n",
      "155\t2\t0.0\t1\t2\n",
      "166\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 21639 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t120\t7263.3\t0\t120\n",
      "4\t39\t1815.8\t0\t39\n",
      "5\t61\t454.0\t0\t61\n",
      "6\t167\t113.5\t0\t167\n",
      "7\t395\t28.4\t0\t395\n",
      "8\t677\t7.1\t0\t677\n",
      "9\t2216\t1.8\t0\t2212 4\n",
      "10\t3339\t0.4\t1\t3299 40\n",
      "11\t289\t0.1\t1\t277 12\n",
      "12\t575\t0.0\t1\t564 11\n",
      "13\t755\t0.0\t1\t728 27\n",
      "14\t1075\t0.0\t1\t1018 57\n",
      "15\t696\t0.0\t1\t659 37\n",
      "16\t1914\t0.0\t1\t1839 75\n",
      "17\t2340\t0.0\t1\t2248 92\n",
      "18\t1420\t0.0\t1\t1272 94 54\n",
      "19\t4856\t0.0\t1\t2887 1946 23\n",
      "20\t13\t0.0\t2\t4 4 5\n",
      "21\t627\t0.0\t2\t0 621 6\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t2\t0.0\t2\t1 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 1\n",
      "77\t2\t0.0\t2\t1 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t5\t0.0\t2\t5\n",
      "80\t1\t0.0\t2\t0 1\n",
      "86\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "91\t2\t0.0\t2\t0 1 1\n",
      "101\t1\t0.0\t2\t0 1\n",
      "124\t1\t0.0\t2\t1\n",
      "158\t1\t0.0\t2\t1\n",
      "181\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "223\t1\t0.0\t2\t1\n",
      "228\t1\t0.0\t2\t1\n",
      "235\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.11.G.5percent.4.29.19_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.11.G.5percent.4.29.19_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.11.G.5percent.4.29.19_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.11.G.5percent.4.29.19_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.76 s (14 us/read; 4.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            416,062\n",
      "  Read 1 with adapter:                  12,922 (3.1%)\n",
      "  Read 2 with adapter:                  16,466 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       416,062 (100.0%)\n",
      "\n",
      "Total basepairs processed:   189,237,507 bp\n",
      "  Read 1:    92,906,830 bp\n",
      "  Read 2:    96,330,677 bp\n",
      "Total written (filtered):    188,632,021 bp (99.7%)\n",
      "  Read 1:    92,527,501 bp\n",
      "  Read 2:    96,104,520 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 12922 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t416\t6501.0\t0\t416\n",
      "6\t1\t101.6\t0\t1\n",
      "9\t1\t1.6\t0\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "19\t1\t0.0\t1\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "23\t7\t0.0\t1\t7\n",
      "25\t4\t0.0\t1\t4\n",
      "26\t18\t0.0\t1\t14 4\n",
      "27\t128\t0.0\t1\t110 18\n",
      "28\t322\t0.0\t1\t190 132\n",
      "29\t11680\t0.0\t1\t11437 243\n",
      "30\t52\t0.0\t1\t30 22\n",
      "31\t12\t0.0\t1\t11 1\n",
      "32\t3\t0.0\t1\t3\n",
      "33\t2\t0.0\t1\t2\n",
      "48\t2\t0.0\t1\t2\n",
      "49\t1\t0.0\t1\t1\n",
      "52\t2\t0.0\t1\t2\n",
      "54\t2\t0.0\t1\t2\n",
      "55\t1\t0.0\t1\t1\n",
      "56\t4\t0.0\t1\t3 1\n",
      "57\t35\t0.0\t1\t31 4\n",
      "58\t65\t0.0\t1\t57 8\n",
      "59\t2\t0.0\t1\t2\n",
      "64\t1\t0.0\t1\t1\n",
      "85\t2\t0.0\t1\t2\n",
      "86\t1\t0.0\t1\t0 1\n",
      "87\t2\t0.0\t1\t2\n",
      "88\t4\t0.0\t1\t1 3\n",
      "89\t45\t0.0\t1\t44 1\n",
      "90\t2\t0.0\t1\t2\n",
      "91\t3\t0.0\t1\t2 1\n",
      "92\t52\t0.0\t1\t52\n",
      "93\t3\t0.0\t1\t1 2\n",
      "100\t3\t0.0\t1\t3\n",
      "101\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "154\t1\t0.0\t1\t0 1\n",
      "155\t29\t0.0\t1\t26 3\n",
      "215\t2\t0.0\t1\t1 1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t6\t0.0\t1\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 16466 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t124\t6501.0\t0\t124\n",
      "4\t48\t1625.2\t0\t48\n",
      "5\t72\t406.3\t0\t72\n",
      "6\t186\t101.6\t0\t186\n",
      "7\t478\t25.4\t0\t478\n",
      "8\t820\t6.3\t0\t820\n",
      "9\t2640\t1.6\t0\t2635 5\n",
      "10\t3236\t0.4\t1\t3192 44\n",
      "11\t131\t0.1\t1\t131\n",
      "12\t180\t0.0\t1\t176 4\n",
      "13\t182\t0.0\t1\t172 10\n",
      "14\t349\t0.0\t1\t329 20\n",
      "15\t309\t0.0\t1\t300 9\n",
      "16\t961\t0.0\t1\t936 25\n",
      "17\t1517\t0.0\t1\t1479 38\n",
      "18\t981\t0.0\t1\t898 55 28\n",
      "19\t3641\t0.0\t1\t2198 1426 17\n",
      "20\t13\t0.0\t2\t10 1 2\n",
      "21\t547\t0.0\t2\t0 539 8\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t0 1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t2 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 1\n",
      "109\t1\t0.0\t2\t0 1\n",
      "114\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t0 1\n",
      "163\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "181\t1\t0.0\t2\t1\n",
      "212\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.4.E.5percent.5.3.19_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.4.E.5percent.5.3.19_52_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.4.E.5percent.5.3.19_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.4.E.5percent.5.3.19_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.97 s (19 us/read; 3.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,878\n",
      "  Read 1 with adapter:                      98 (0.1%)\n",
      "  Read 2 with adapter:                   5,486 (5.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,878 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,455,185 bp\n",
      "  Read 1:    22,297,835 bp\n",
      "  Read 2:    23,157,350 bp\n",
      "Total written (filtered):     45,378,901 bp (99.8%)\n",
      "  Read 1:    22,295,070 bp\n",
      "  Read 2:    23,083,831 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 98 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t81\t1576.2\t0\t81\n",
      "25\t3\t0.0\t1\t3\n",
      "29\t3\t0.0\t1\t3\n",
      "204\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t1\n",
      "216\t6\t0.0\t1\t5 1\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5486 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t1576.2\t0\t42\n",
      "4\t15\t394.1\t0\t15\n",
      "5\t13\t98.5\t0\t13\n",
      "6\t30\t24.6\t0\t30\n",
      "7\t108\t6.2\t0\t108\n",
      "8\t180\t1.5\t0\t180\n",
      "9\t665\t0.4\t0\t661 4\n",
      "10\t1708\t0.1\t1\t1679 29\n",
      "11\t77\t0.0\t1\t74 3\n",
      "12\t51\t0.0\t1\t48 3\n",
      "13\t56\t0.0\t1\t54 2\n",
      "14\t134\t0.0\t1\t122 12\n",
      "15\t128\t0.0\t1\t123 5\n",
      "16\t340\t0.0\t1\t330 10\n",
      "17\t461\t0.0\t1\t452 9\n",
      "18\t344\t0.0\t1\t312 25 7\n",
      "19\t991\t0.0\t1\t605 383 3\n",
      "20\t4\t0.0\t2\t0 3 1\n",
      "21\t128\t0.0\t2\t0 128\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "58\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "212\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.4.G.5percent.4.27.19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.4.G.5percent.4.27.19_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.4.G.5percent.4.27.19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.4.G.5percent.4.27.19_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 11.61 s (14 us/read; 4.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            831,586\n",
      "  Read 1 with adapter:                     792 (0.1%)\n",
      "  Read 2 with adapter:                  33,001 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       831,586 (100.0%)\n",
      "\n",
      "Total basepairs processed:   378,459,780 bp\n",
      "  Read 1:   185,961,175 bp\n",
      "  Read 2:   192,498,605 bp\n",
      "Total written (filtered):    378,008,454 bp (99.9%)\n",
      "  Read 1:   185,956,768 bp\n",
      "  Read 2:   192,051,686 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 792 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t778\t12993.5\t0\t778\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "29\t2\t0.0\t1\t2\n",
      "207\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t6\t0.0\t1\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 33001 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t502\t12993.5\t0\t502\n",
      "4\t89\t3248.4\t0\t89\n",
      "5\t106\t812.1\t0\t106\n",
      "6\t202\t203.0\t0\t202\n",
      "7\t621\t50.8\t0\t621\n",
      "8\t866\t12.7\t0\t866\n",
      "9\t3388\t3.2\t0\t3365 23\n",
      "10\t10756\t0.8\t1\t10629 127\n",
      "11\t541\t0.2\t1\t528 13\n",
      "12\t753\t0.0\t1\t742 11\n",
      "13\t152\t0.0\t1\t143 9\n",
      "14\t428\t0.0\t1\t402 26\n",
      "15\t652\t0.0\t1\t630 22\n",
      "16\t1628\t0.0\t1\t1576 52\n",
      "17\t2824\t0.0\t1\t2716 108\n",
      "18\t1723\t0.0\t1\t1595 110 18\n",
      "19\t6664\t0.0\t1\t4248 2394 22\n",
      "20\t9\t0.0\t2\t3 4 2\n",
      "21\t997\t0.0\t2\t0 986 11\n",
      "22\t54\t0.0\t2\t0 0 54\n",
      "25\t3\t0.0\t2\t0 3\n",
      "47\t1\t0.0\t2\t0 1\n",
      "56\t1\t0.0\t2\t0 1\n",
      "62\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 1\n",
      "73\t1\t0.0\t2\t0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t4\n",
      "129\t1\t0.0\t2\t0 1\n",
      "137\t1\t0.0\t2\t1\n",
      "151\t1\t0.0\t2\t1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "176\t1\t0.0\t2\t1\n",
      "198\t2\t0.0\t2\t2\n",
      "214\t1\t0.0\t2\t1\n",
      "215\t3\t0.0\t2\t2 1\n",
      "216\t5\t0.0\t2\t5\n",
      "217\t6\t0.0\t2\t6\n",
      "224\t1\t0.0\t2\t0 1\n",
      "227\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t3\t0.0\t2\t2 1\n",
      "238\t1\t0.0\t2\t1\n",
      "245\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.6.D.5percent.4.27.19_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.6.D.5percent.4.27.19_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.6.D.5percent.4.27.19_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.6.D.5percent.4.27.19_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.20 s (15 us/read; 3.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            277,580\n",
      "  Read 1 with adapter:                     325 (0.1%)\n",
      "  Read 2 with adapter:                  12,365 (4.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       277,580 (100.0%)\n",
      "\n",
      "Total basepairs processed:   126,315,743 bp\n",
      "  Read 1:    62,013,988 bp\n",
      "  Read 2:    64,301,755 bp\n",
      "Total written (filtered):    126,145,731 bp (99.9%)\n",
      "  Read 1:    62,010,977 bp\n",
      "  Read 2:    64,134,754 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 325 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t297\t4337.2\t0\t297\n",
      "9\t1\t1.1\t0\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t5\t0.0\t1\t5\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t5\t0.0\t1\t5\n",
      "29\t4\t0.0\t1\t4\n",
      "100\t4\t0.0\t1\t4\n",
      "101\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12365 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t90\t4337.2\t0\t90\n",
      "4\t23\t1084.3\t0\t23\n",
      "5\t34\t271.1\t0\t34\n",
      "6\t65\t67.8\t0\t65\n",
      "7\t273\t16.9\t0\t273\n",
      "8\t361\t4.2\t0\t361\n",
      "9\t1472\t1.1\t0\t1461 11\n",
      "10\t3879\t0.3\t1\t3832 47\n",
      "11\t188\t0.1\t1\t186 2\n",
      "12\t134\t0.0\t1\t131 3\n",
      "13\t126\t0.0\t1\t119 7\n",
      "14\t245\t0.0\t1\t229 16\n",
      "15\t261\t0.0\t1\t250 11\n",
      "16\t730\t0.0\t1\t703 27\n",
      "17\t1054\t0.0\t1\t1021 33\n",
      "18\t686\t0.0\t1\t618 50 18\n",
      "19\t2369\t0.0\t1\t1448 909 12\n",
      "20\t8\t0.0\t2\t6 1 1\n",
      "21\t341\t0.0\t2\t0 334 7\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t7\t0.0\t2\t4 2 1\n",
      "159\t1\t0.0\t2\t1\n",
      "215\t3\t0.0\t2\t3\n",
      "217\t1\t0.0\t2\t0 0 1\n",
      "227\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.8.A.5percent.4.30.19_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.8.A.5percent.4.30.19_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.8.A.5percent.4.30.19_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.8.A.5percent.4.30.19_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.39 s (26 us/read; 2.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,852\n",
      "  Read 1 with adapter:                     358 (0.7%)\n",
      "  Read 2 with adapter:                   2,617 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,852 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,867,853 bp\n",
      "  Read 1:    11,690,710 bp\n",
      "  Read 2:    12,177,143 bp\n",
      "Total written (filtered):     23,799,232 bp (99.7%)\n",
      "  Read 1:    11,657,485 bp\n",
      "  Read 2:    12,141,747 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 358 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t841.4\t0\t42\n",
      "4\t1\t210.4\t0\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "72\t1\t0.0\t1\t1\n",
      "99\t7\t0.0\t1\t2 5\n",
      "100\t270\t0.0\t1\t264 6\n",
      "101\t19\t0.0\t1\t18 1\n",
      "131\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t0 1\n",
      "207\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t2 2\n",
      "217\t8\t0.0\t1\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2617 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t841.4\t0\t24\n",
      "4\t5\t210.4\t0\t5\n",
      "5\t5\t52.6\t0\t5\n",
      "6\t11\t13.1\t0\t11\n",
      "7\t44\t3.3\t0\t44\n",
      "8\t87\t0.8\t0\t87\n",
      "9\t300\t0.2\t0\t299 1\n",
      "10\t848\t0.1\t1\t843 5\n",
      "11\t33\t0.0\t1\t33\n",
      "12\t20\t0.0\t1\t19 1\n",
      "13\t17\t0.0\t1\t17\n",
      "14\t50\t0.0\t1\t47 3\n",
      "15\t58\t0.0\t1\t57 1\n",
      "16\t193\t0.0\t1\t188 5\n",
      "17\t234\t0.0\t1\t230 4\n",
      "18\t146\t0.0\t1\t131 12 3\n",
      "19\t460\t0.0\t1\t291 168 1\n",
      "20\t3\t0.0\t2\t0 1 2\n",
      "21\t71\t0.0\t2\t0 70 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "79\t2\t0.0\t2\t2\n",
      "186\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCF.4.26.19_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCF.4.26.19_51_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCF.4.26.19_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCF.4.26.19_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCG.4.23.19_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCG.4.23.19_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCG.4.23.19_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCG.4.23.19_26_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCG.4.26.19_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.HCG.4.26.19_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCG.4.26.19_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.HCG.4.26.19_14_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.1.D.5percent.5.29.19_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.1.D.5percent.5.29.19_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.1.D.5percent.5.29.19_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.1.D.5percent.5.29.19_7_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.1.E.5percent.5.21.19_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.1.E.5percent.5.21.19_54_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.1.E.5percent.5.21.19_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.1.E.5percent.5.21.19_54_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.10.G.5percent.4.29.19_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.10.G.5percent.4.29.19_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.10.G.5percent.4.29.19_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.10.G.5percent.4.29.19_15_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.11.G.5percent.4.29.19_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.11.G.5percent.4.29.19_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.11.G.5percent.4.29.19_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.11.G.5percent.4.29.19_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.4.E.5percent.5.3.19_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.4.E.5percent.5.3.19_52_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.4.E.5percent.5.3.19_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.4.E.5percent.5.3.19_52_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.4.G.5percent.4.27.19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.4.G.5percent.4.27.19_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.4.G.5percent.4.27.19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.4.G.5percent.4.27.19_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.6.D.5percent.4.27.19_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.6.D.5percent.4.27.19_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.6.D.5percent.4.27.19_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.6.D.5percent.4.27.19_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.8.A.5percent.4.30.19_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xgoslv5e/M.cav.Tank.8.A.5percent.4.30.19_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.8.A.5percent.4.30.19_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-9tuznbly/014c014b-c591-494e-867c-3e24f95dbc65/data/M.cav.Tank.8.A.5percent.4.30.19_21_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGtransmissionstudies_1_69.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.B.NT.5percent.9.15.19_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.B.NT.5percent.9.15.19_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.B.NT.5percent.9.15.19_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.B.NT.5percent.9.15.19_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.24 s (13 us/read; 4.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            315,238\n",
      "  Read 1 with adapter:                     350 (0.1%)\n",
      "  Read 2 with adapter:                  31,997 (10.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       315,238 (100.0%)\n",
      "\n",
      "Total basepairs processed:   144,169,448 bp\n",
      "  Read 1:    70,588,137 bp\n",
      "  Read 2:    73,581,311 bp\n",
      "Total written (filtered):    143,802,773 bp (99.7%)\n",
      "  Read 1:    70,586,455 bp\n",
      "  Read 2:    73,216,318 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 350 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t337\t4925.6\t0\t337\n",
      "13\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t5\t0.0\t1\t4 1\n",
      "27\t1\t0.0\t1\t1\n",
      "34\t1\t0.0\t1\t0 1\n",
      "181\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 31997 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t301\t4925.6\t0\t301\n",
      "4\t135\t1231.4\t0\t135\n",
      "5\t263\t307.8\t0\t263\n",
      "6\t623\t77.0\t0\t623\n",
      "7\t2076\t19.2\t0\t2076\n",
      "8\t3005\t4.8\t0\t3005\n",
      "9\t8062\t1.2\t0\t8043 19\n",
      "10\t7495\t0.3\t1\t7410 85\n",
      "11\t216\t0.1\t1\t213 3\n",
      "12\t263\t0.0\t1\t258 5\n",
      "13\t493\t0.0\t1\t479 14\n",
      "14\t683\t0.0\t1\t648 35\n",
      "15\t507\t0.0\t1\t479 28\n",
      "16\t1372\t0.0\t1\t1318 54\n",
      "17\t1771\t0.0\t1\t1691 80\n",
      "18\t1115\t0.0\t1\t979 87 49\n",
      "19\t3143\t0.0\t1\t1727 1403 13\n",
      "20\t15\t0.0\t2\t2 8 5\n",
      "21\t415\t0.0\t2\t0 411 4\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "30\t1\t0.0\t2\t1\n",
      "33\t1\t0.0\t2\t1\n",
      "47\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "79\t4\t0.0\t2\t3 0 1\n",
      "80\t1\t0.0\t2\t0 1\n",
      "89\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t1\n",
      "177\t1\t0.0\t2\t0 1\n",
      "206\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.B.NT.75percent.9.16.19._7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.B.NT.75percent.9.16.19._7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.B.NT.75percent.9.16.19._7_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.B.NT.75percent.9.16.19._7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.00 s (16 us/read; 3.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            370,900\n",
      "  Read 1 with adapter:                     199 (0.1%)\n",
      "  Read 2 with adapter:                  25,135 (6.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       370,900 (100.0%)\n",
      "\n",
      "Total basepairs processed:   169,172,867 bp\n",
      "  Read 1:    83,031,920 bp\n",
      "  Read 2:    86,140,947 bp\n",
      "Total written (filtered):    168,840,176 bp (99.8%)\n",
      "  Read 1:    83,031,094 bp\n",
      "  Read 2:    85,809,082 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 199 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t193\t5795.3\t0\t193\n",
      "6\t1\t90.6\t0\t1\n",
      "21\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t1\t0.0\t1\t1\n",
      "137\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 25135 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t202\t5795.3\t0\t202\n",
      "4\t70\t1448.8\t0\t70\n",
      "5\t119\t362.2\t0\t119\n",
      "6\t289\t90.6\t0\t289\n",
      "7\t741\t22.6\t0\t741\n",
      "8\t1313\t5.7\t0\t1313\n",
      "9\t4091\t1.4\t0\t4076 15\n",
      "10\t5451\t0.4\t1\t5394 57\n",
      "11\t355\t0.1\t1\t341 14\n",
      "12\t517\t0.0\t1\t506 11\n",
      "13\t606\t0.0\t1\t591 15\n",
      "14\t986\t0.0\t1\t931 55\n",
      "15\t624\t0.0\t1\t586 38\n",
      "16\t1692\t0.0\t1\t1626 66\n",
      "17\t2237\t0.0\t1\t2151 86\n",
      "18\t1246\t0.0\t1\t1098 89 59\n",
      "19\t4027\t0.0\t1\t2333 1680 14\n",
      "20\t4\t0.0\t2\t0 3 1\n",
      "21\t493\t0.0\t2\t0 485 8\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "46\t1\t0.0\t2\t1\n",
      "47\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "58\t2\t0.0\t2\t2\n",
      "60\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "71\t2\t0.0\t2\t1 0 1\n",
      "78\t2\t0.0\t2\t1 1\n",
      "79\t4\t0.0\t2\t4\n",
      "85\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "93\t1\t0.0\t2\t1\n",
      "120\t2\t0.0\t2\t1 1\n",
      "129\t1\t0.0\t2\t0 1\n",
      "193\t1\t0.0\t2\t1\n",
      "197\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "229\t1\t0.0\t2\t1\n",
      "236\t2\t0.0\t2\t2\n",
      "239\t1\t0.0\t2\t0 0 1\n",
      "241\t3\t0.0\t2\t3\n",
      "245\t2\t0.0\t2\t2\n",
      "246\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.25percent.9.12.19_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.25percent.9.12.19_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.25percent.9.12.19_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.25percent.9.12.19_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.48 s (18 us/read; 3.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            306,248\n",
      "  Read 1 with adapter:                     257 (0.1%)\n",
      "  Read 2 with adapter:                  21,142 (6.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       306,248 (100.0%)\n",
      "\n",
      "Total basepairs processed:   139,552,796 bp\n",
      "  Read 1:    68,487,161 bp\n",
      "  Read 2:    71,065,635 bp\n",
      "Total written (filtered):    139,277,551 bp (99.8%)\n",
      "  Read 1:    68,485,376 bp\n",
      "  Read 2:    70,792,175 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 257 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t244\t4785.1\t0\t244\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t3\t0.0\t1\t3\n",
      "38\t1\t0.0\t1\t0 1\n",
      "155\t1\t0.0\t1\t0 1\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 21142 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t164\t4785.1\t0\t164\n",
      "4\t66\t1196.3\t0\t66\n",
      "5\t132\t299.1\t0\t132\n",
      "6\t291\t74.8\t0\t291\n",
      "7\t726\t18.7\t0\t726\n",
      "8\t1313\t4.7\t0\t1313\n",
      "9\t3761\t1.2\t0\t3751 10\n",
      "10\t4725\t0.3\t1\t4669 56\n",
      "11\t222\t0.1\t1\t219 3\n",
      "12\t265\t0.0\t1\t258 7\n",
      "13\t373\t0.0\t1\t352 21\n",
      "14\t643\t0.0\t1\t611 32\n",
      "15\t501\t0.0\t1\t469 32\n",
      "16\t1364\t0.0\t1\t1321 43\n",
      "17\t1773\t0.0\t1\t1699 74\n",
      "18\t1023\t0.0\t1\t917 70 36\n",
      "19\t3319\t0.0\t1\t1917 1383 19\n",
      "20\t15\t0.0\t2\t2 8 5\n",
      "21\t414\t0.0\t2\t0 409 5\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "61\t2\t0.0\t2\t1 0 1\n",
      "62\t2\t0.0\t2\t2\n",
      "64\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t3\t0.0\t2\t1 1 1\n",
      "71\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 1\n",
      "116\t1\t0.0\t2\t0 1\n",
      "119\t1\t0.0\t2\t1\n",
      "124\t1\t0.0\t2\t1\n",
      "139\t1\t0.0\t2\t0 1\n",
      "150\t1\t0.0\t2\t1\n",
      "152\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t0 1\n",
      "210\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "233\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.50percent.9.16.19_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.50percent.9.16.19_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.50percent.9.16.19_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.50percent.9.16.19_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.30 s (13 us/read; 4.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            325,720\n",
      "  Read 1 with adapter:                     359 (0.1%)\n",
      "  Read 2 with adapter:                  31,469 (9.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       325,720 (100.0%)\n",
      "\n",
      "Total basepairs processed:   148,919,343 bp\n",
      "  Read 1:    72,934,616 bp\n",
      "  Read 2:    75,984,727 bp\n",
      "Total written (filtered):    148,539,502 bp (99.7%)\n",
      "  Read 1:    72,929,911 bp\n",
      "  Read 2:    75,609,591 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 359 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t316\t5089.4\t0\t316\n",
      "26\t1\t0.0\t1\t1\n",
      "29\t4\t0.0\t1\t4\n",
      "30\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "42\t1\t0.0\t1\t1\n",
      "50\t2\t0.0\t1\t2\n",
      "51\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "101\t28\t0.0\t1\t27 1\n",
      "113\t1\t0.0\t1\t1\n",
      "133\t1\t0.0\t1\t0 1\n",
      "180\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 31469 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t214\t5089.4\t0\t214\n",
      "4\t122\t1272.3\t0\t122\n",
      "5\t195\t318.1\t0\t195\n",
      "6\t428\t79.5\t0\t428\n",
      "7\t1577\t19.9\t0\t1577\n",
      "8\t2399\t5.0\t0\t2399\n",
      "9\t7005\t1.2\t0\t6983 22\n",
      "10\t8199\t0.3\t1\t8082 117\n",
      "11\t329\t0.1\t1\t318 11\n",
      "12\t419\t0.0\t1\t409 10\n",
      "13\t617\t0.0\t1\t590 27\n",
      "14\t893\t0.0\t1\t847 46\n",
      "15\t629\t0.0\t1\t591 38\n",
      "16\t1513\t0.0\t1\t1461 52\n",
      "17\t1822\t0.0\t1\t1757 65\n",
      "18\t1115\t0.0\t1\t978 76 61\n",
      "19\t3514\t0.0\t1\t1914 1586 14\n",
      "20\t8\t0.0\t2\t2 6\n",
      "21\t407\t0.0\t2\t0 397 10\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "63\t2\t0.0\t2\t2\n",
      "65\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t2\n",
      "70\t2\t0.0\t2\t0 1 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "79\t5\t0.0\t2\t4 1\n",
      "82\t1\t0.0\t2\t1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "88\t2\t0.0\t2\t1 1\n",
      "91\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t1\n",
      "102\t1\t0.0\t2\t0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t1\n",
      "114\t1\t0.0\t2\t1\n",
      "116\t1\t0.0\t2\t1\n",
      "137\t1\t0.0\t2\t1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t1\n",
      "165\t2\t0.0\t2\t2\n",
      "216\t1\t0.0\t2\t1\n",
      "220\t1\t0.0\t2\t1\n",
      "222\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t0 1\n",
      "240\t2\t0.0\t2\t0 1 1\n",
      "241\t1\t0.0\t2\t0 1\n",
      "242\t1\t0.0\t2\t1\n",
      "248\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.75percent.9.18.19._33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.75percent.9.18.19._33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.75percent.9.18.19._33_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.75percent.9.18.19._33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.37 s (14 us/read; 4.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            319,298\n",
      "  Read 1 with adapter:                     394 (0.1%)\n",
      "  Read 2 with adapter:                  31,939 (10.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       319,298 (100.0%)\n",
      "\n",
      "Total basepairs processed:   145,998,576 bp\n",
      "  Read 1:    71,499,632 bp\n",
      "  Read 2:    74,498,944 bp\n",
      "Total written (filtered):    145,631,243 bp (99.7%)\n",
      "  Read 1:    71,498,092 bp\n",
      "  Read 2:    74,133,151 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 394 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t383\t4989.0\t0\t383\n",
      "9\t1\t1.2\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "26\t4\t0.0\t1\t3 1\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t1\t0.0\t1\t1\n",
      "156\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 31939 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t262\t4989.0\t0\t262\n",
      "4\t115\t1247.3\t0\t115\n",
      "5\t196\t311.8\t0\t196\n",
      "6\t551\t78.0\t0\t551\n",
      "7\t1850\t19.5\t0\t1850\n",
      "8\t2887\t4.9\t0\t2887\n",
      "9\t8046\t1.2\t0\t8025 21\n",
      "10\t8164\t0.3\t1\t8071 93\n",
      "11\t223\t0.1\t1\t220 3\n",
      "12\t295\t0.0\t1\t286 9\n",
      "13\t415\t0.0\t1\t399 16\n",
      "14\t724\t0.0\t1\t690 34\n",
      "15\t488\t0.0\t1\t463 25\n",
      "16\t1343\t0.0\t1\t1294 49\n",
      "17\t1682\t0.0\t1\t1621 61\n",
      "18\t1058\t0.0\t1\t931 72 55\n",
      "19\t3190\t0.0\t1\t1778 1393 19\n",
      "20\t11\t0.0\t2\t3 7 1\n",
      "21\t400\t0.0\t2\t0 389 11\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "30\t1\t0.0\t2\t1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "112\t1\t0.0\t2\t1\n",
      "136\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "156\t1\t0.0\t2\t1\n",
      "170\t1\t0.0\t2\t1\n",
      "208\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "242\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.25percent.9.13.19_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.25percent.9.13.19_62_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.25percent.9.13.19_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.25percent.9.13.19_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.87 s (20 us/read; 3.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            145,202\n",
      "  Read 1 with adapter:                      95 (0.1%)\n",
      "  Read 2 with adapter:                  10,463 (7.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       145,202 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,936,717 bp\n",
      "  Read 1:    32,330,963 bp\n",
      "  Read 2:    33,605,754 bp\n",
      "Total written (filtered):     65,798,867 bp (99.8%)\n",
      "  Read 1:    32,329,972 bp\n",
      "  Read 2:    33,468,895 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 95 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t2268.8\t0\t88\n",
      "19\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10463 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t2268.8\t0\t73\n",
      "4\t28\t567.2\t0\t28\n",
      "5\t43\t141.8\t0\t43\n",
      "6\t130\t35.4\t0\t130\n",
      "7\t370\t8.9\t0\t370\n",
      "8\t602\t2.2\t0\t602\n",
      "9\t1833\t0.6\t0\t1826 7\n",
      "10\t2164\t0.1\t1\t2143 21\n",
      "11\t111\t0.0\t1\t108 3\n",
      "12\t158\t0.0\t1\t153 5\n",
      "13\t219\t0.0\t1\t217 2\n",
      "14\t388\t0.0\t1\t373 15\n",
      "15\t257\t0.0\t1\t250 7\n",
      "16\t695\t0.0\t1\t658 37\n",
      "17\t947\t0.0\t1\t899 48\n",
      "18\t528\t0.0\t1\t467 28 33\n",
      "19\t1681\t0.0\t1\t933 739 9\n",
      "20\t7\t0.0\t2\t1 3 3\n",
      "21\t207\t0.0\t2\t0 205 2\n",
      "22\t8\t0.0\t2\t0 1 7\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t1 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t0 1\n",
      "127\t1\t0.0\t2\t1\n",
      "144\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t1\n",
      "205\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.5percent.9.12.19_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.5percent.9.12.19_51_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.5percent.9.12.19_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.5percent.9.12.19_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.03 s (14 us/read; 4.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            363,587\n",
      "  Read 1 with adapter:                     290 (0.1%)\n",
      "  Read 2 with adapter:                  27,108 (7.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       363,587 (100.0%)\n",
      "\n",
      "Total basepairs processed:   165,964,839 bp\n",
      "  Read 1:    81,382,905 bp\n",
      "  Read 2:    84,581,934 bp\n",
      "Total written (filtered):    165,622,016 bp (99.8%)\n",
      "  Read 1:    81,381,155 bp\n",
      "  Read 2:    84,240,861 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 290 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t274\t5681.0\t0\t274\n",
      "25\t5\t0.0\t1\t5\n",
      "26\t3\t0.0\t1\t2 1\n",
      "27\t1\t0.0\t1\t0 1\n",
      "29\t3\t0.0\t1\t3\n",
      "75\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 27108 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t201\t5681.0\t0\t201\n",
      "4\t86\t1420.3\t0\t86\n",
      "5\t121\t355.1\t0\t121\n",
      "6\t326\t88.8\t0\t326\n",
      "7\t1147\t22.2\t0\t1147\n",
      "8\t1801\t5.5\t0\t1801\n",
      "9\t5402\t1.4\t0\t5391 11\n",
      "10\t6312\t0.3\t1\t6241 71\n",
      "11\t228\t0.1\t1\t222 6\n",
      "12\t349\t0.0\t1\t343 6\n",
      "13\t493\t0.0\t1\t479 14\n",
      "14\t751\t0.0\t1\t713 38\n",
      "15\t612\t0.0\t1\t571 41\n",
      "16\t1565\t0.0\t1\t1513 52\n",
      "17\t2018\t0.0\t1\t1939 79\n",
      "18\t1221\t0.0\t1\t1071 90 60\n",
      "19\t3902\t0.0\t1\t2184 1701 17\n",
      "20\t21\t0.0\t2\t9 7 5\n",
      "21\t488\t0.0\t2\t0 481 7\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "30\t2\t0.0\t2\t2\n",
      "52\t1\t0.0\t2\t1\n",
      "55\t3\t0.0\t2\t0 2 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t2\n",
      "69\t2\t0.0\t2\t1 0 1\n",
      "71\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t3 1\n",
      "84\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t1\n",
      "91\t1\t0.0\t2\t0 1\n",
      "123\t2\t0.0\t2\t1 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t1\n",
      "146\t1\t0.0\t2\t1\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "185\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "218\t3\t0.0\t2\t3\n",
      "225\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.75percent.9.16.19._49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.75percent.9.16.19._49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.75percent.9.16.19._49_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.75percent.9.16.19._49_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.99 s (14 us/read; 4.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            278,126\n",
      "  Read 1 with adapter:                     361 (0.1%)\n",
      "  Read 2 with adapter:                  20,348 (7.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       278,126 (100.0%)\n",
      "\n",
      "Total basepairs processed:   126,910,108 bp\n",
      "  Read 1:    62,225,717 bp\n",
      "  Read 2:    64,684,391 bp\n",
      "Total written (filtered):    126,649,718 bp (99.8%)\n",
      "  Read 1:    62,223,745 bp\n",
      "  Read 2:    64,425,973 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 361 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t351\t4345.7\t0\t351\n",
      "25\t2\t0.0\t1\t2\n",
      "29\t2\t0.0\t1\t2\n",
      "51\t2\t0.0\t1\t2\n",
      "128\t1\t0.0\t1\t0 1\n",
      "164\t1\t0.0\t1\t0 1\n",
      "199\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 20348 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t141\t4345.7\t0\t141\n",
      "4\t48\t1086.4\t0\t48\n",
      "5\t118\t271.6\t0\t118\n",
      "6\t188\t67.9\t0\t188\n",
      "7\t697\t17.0\t0\t697\n",
      "8\t1159\t4.2\t0\t1159\n",
      "9\t3528\t1.1\t0\t3521 7\n",
      "10\t5268\t0.3\t1\t5207 61\n",
      "11\t253\t0.1\t1\t244 9\n",
      "12\t306\t0.0\t1\t296 10\n",
      "13\t486\t0.0\t1\t474 12\n",
      "14\t714\t0.0\t1\t682 32\n",
      "15\t464\t0.0\t1\t429 35\n",
      "16\t1249\t0.0\t1\t1192 57\n",
      "17\t1466\t0.0\t1\t1403 63\n",
      "18\t881\t0.0\t1\t763 65 53\n",
      "19\t2961\t0.0\t1\t1648 1292 21\n",
      "20\t4\t0.0\t2\t0 4\n",
      "21\t365\t0.0\t2\t0 361 4\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "30\t1\t0.0\t2\t1\n",
      "47\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t0 1\n",
      "52\t2\t0.0\t2\t2\n",
      "58\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t3\n",
      "86\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "91\t1\t0.0\t2\t1\n",
      "92\t2\t0.0\t2\t2\n",
      "104\t1\t0.0\t2\t1\n",
      "124\t1\t0.0\t2\t1\n",
      "138\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "223\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.25percent.9.15.19_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.25percent.9.15.19_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.25percent.9.15.19_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.25percent.9.15.19_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.95 s (17 us/read; 3.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            177,189\n",
      "  Read 1 with adapter:                     194 (0.1%)\n",
      "  Read 2 with adapter:                  13,115 (7.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       177,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:    80,909,360 bp\n",
      "  Read 1:    39,673,649 bp\n",
      "  Read 2:    41,235,711 bp\n",
      "Total written (filtered):     80,754,503 bp (99.8%)\n",
      "  Read 1:    39,672,178 bp\n",
      "  Read 2:    41,082,325 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 194 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t172\t2768.6\t0\t172\n",
      "6\t1\t43.3\t0\t1\n",
      "9\t1\t0.7\t0\t1\n",
      "25\t3\t0.0\t1\t2 1\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t5\t0.0\t1\t5\n",
      "29\t1\t0.0\t1\t1\n",
      "101\t3\t0.0\t1\t3\n",
      "113\t1\t0.0\t1\t1\n",
      "129\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13115 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t126\t2768.6\t0\t126\n",
      "4\t57\t692.1\t0\t57\n",
      "5\t110\t173.0\t0\t110\n",
      "6\t280\t43.3\t0\t280\n",
      "7\t877\t10.8\t0\t877\n",
      "8\t1340\t2.7\t0\t1340\n",
      "9\t3403\t0.7\t0\t3399 4\n",
      "10\t2675\t0.2\t1\t2640 35\n",
      "11\t65\t0.0\t1\t65\n",
      "12\t54\t0.0\t1\t53 1\n",
      "13\t75\t0.0\t1\t72 3\n",
      "14\t180\t0.0\t1\t171 9\n",
      "15\t179\t0.0\t1\t164 15\n",
      "16\t457\t0.0\t1\t437 20\n",
      "17\t754\t0.0\t1\t727 27\n",
      "18\t513\t0.0\t1\t463 34 16\n",
      "19\t1703\t0.0\t1\t1001 690 12\n",
      "20\t10\t0.0\t2\t3 3 4\n",
      "21\t226\t0.0\t2\t0 224 2\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "30\t4\t0.0\t2\t4\n",
      "34\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t0 1\n",
      "66\t1\t0.0\t2\t0 1\n",
      "68\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t2\n",
      "101\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "138\t1\t0.0\t2\t1\n",
      "143\t1\t0.0\t2\t0 1\n",
      "151\t1\t0.0\t2\t1\n",
      "172\t1\t0.0\t2\t1\n",
      "196\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.5percent.9.14.19_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.5percent.9.14.19_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.5percent.9.14.19_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.5percent.9.14.19_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.21 s (15 us/read; 4.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            351,200\n",
      "  Read 1 with adapter:                     253 (0.1%)\n",
      "  Read 2 with adapter:                  32,329 (9.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       351,200 (100.0%)\n",
      "\n",
      "Total basepairs processed:   160,502,138 bp\n",
      "  Read 1:    78,641,868 bp\n",
      "  Read 2:    81,860,270 bp\n",
      "Total written (filtered):    160,107,130 bp (99.8%)\n",
      "  Read 1:    78,640,781 bp\n",
      "  Read 2:    81,466,349 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 253 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t245\t5487.5\t0\t245\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t2\t0.0\t1\t2\n",
      "89\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 32329 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t219\t5487.5\t0\t219\n",
      "4\t91\t1371.9\t0\t91\n",
      "5\t233\t343.0\t0\t233\n",
      "6\t416\t85.7\t0\t416\n",
      "7\t1485\t21.4\t0\t1485\n",
      "8\t2289\t5.4\t0\t2289\n",
      "9\t6715\t1.3\t0\t6706 9\n",
      "10\t7924\t0.3\t1\t7835 89\n",
      "11\t393\t0.1\t1\t385 8\n",
      "12\t487\t0.0\t1\t474 13\n",
      "13\t707\t0.0\t1\t668 39\n",
      "14\t1019\t0.0\t1\t975 44\n",
      "15\t681\t0.0\t1\t634 47\n",
      "16\t1859\t0.0\t1\t1770 89\n",
      "17\t2072\t0.0\t1\t1981 91\n",
      "18\t1313\t0.0\t1\t1121 116 76\n",
      "19\t3902\t0.0\t1\t2061 1821 20\n",
      "20\t14\t0.0\t2\t0 9 5\n",
      "21\t449\t0.0\t2\t0 441 8\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "30\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t2\t0.0\t2\t2\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "71\t2\t0.0\t2\t1 0 1\n",
      "72\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t4\n",
      "83\t1\t0.0\t2\t1\n",
      "88\t2\t0.0\t2\t2\n",
      "89\t2\t0.0\t2\t2\n",
      "97\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t1\n",
      "104\t1\t0.0\t2\t1\n",
      "109\t2\t0.0\t2\t1 1\n",
      "134\t1\t0.0\t2\t1\n",
      "144\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t0 1\n",
      "176\t1\t0.0\t2\t0 1\n",
      "178\t1\t0.0\t2\t0 1\n",
      "181\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.75percent.9.18.19._16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.75percent.9.18.19._16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.75percent.9.18.19._16_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.75percent.9.18.19._16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.49 s (14 us/read; 4.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            386,762\n",
      "  Read 1 with adapter:                     491 (0.1%)\n",
      "  Read 2 with adapter:                  32,049 (8.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       386,762 (100.0%)\n",
      "\n",
      "Total basepairs processed:   176,644,280 bp\n",
      "  Read 1:    86,598,162 bp\n",
      "  Read 2:    90,046,118 bp\n",
      "Total written (filtered):    176,253,186 bp (99.8%)\n",
      "  Read 1:    86,596,185 bp\n",
      "  Read 2:    89,657,001 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 491 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t482\t6043.2\t0\t482\n",
      "4\t1\t1510.8\t0\t1\n",
      "6\t1\t94.4\t0\t1\n",
      "7\t1\t23.6\t0\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "141\t1\t0.0\t1\t1\n",
      "166\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 32049 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t256\t6043.2\t0\t256\n",
      "4\t96\t1510.8\t0\t96\n",
      "5\t207\t377.7\t0\t207\n",
      "6\t453\t94.4\t0\t453\n",
      "7\t1432\t23.6\t0\t1432\n",
      "8\t2242\t5.9\t0\t2242\n",
      "9\t6880\t1.5\t0\t6856 24\n",
      "10\t7950\t0.4\t1\t7854 96\n",
      "11\t309\t0.1\t1\t303 6\n",
      "12\t392\t0.0\t1\t374 18\n",
      "13\t497\t0.0\t1\t477 20\n",
      "14\t869\t0.0\t1\t816 53\n",
      "15\t653\t0.0\t1\t616 37\n",
      "16\t1643\t0.0\t1\t1586 57\n",
      "17\t2264\t0.0\t1\t2183 81\n",
      "18\t1276\t0.0\t1\t1104 120 52\n",
      "19\t4096\t0.0\t1\t2344 1720 32\n",
      "20\t9\t0.0\t2\t0 7 2\n",
      "21\t480\t0.0\t2\t0 477 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "46\t1\t0.0\t2\t1\n",
      "47\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t2\t0.0\t2\t1 1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t2\t0.0\t2\t1 0 1\n",
      "77\t1\t0.0\t2\t1\n",
      "93\t1\t0.0\t2\t1\n",
      "120\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.B.75percent.9.5.19._60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.B.75percent.9.5.19._60_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.B.75percent.9.5.19._60_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.B.75percent.9.5.19._60_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.41 s (16 us/read; 3.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            278,344\n",
      "  Read 1 with adapter:                     246 (0.1%)\n",
      "  Read 2 with adapter:                  24,508 (8.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       278,344 (100.0%)\n",
      "\n",
      "Total basepairs processed:   127,165,786 bp\n",
      "  Read 1:    62,308,428 bp\n",
      "  Read 2:    64,857,358 bp\n",
      "Total written (filtered):    126,867,807 bp (99.8%)\n",
      "  Read 1:    62,307,250 bp\n",
      "  Read 2:    64,560,557 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 246 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t240\t4349.1\t0\t240\n",
      "14\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "35\t1\t0.0\t1\t1\n",
      "52\t1\t0.0\t1\t1\n",
      "156\t1\t0.0\t1\t1\n",
      "176\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 24508 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t184\t4349.1\t0\t184\n",
      "4\t97\t1087.3\t0\t97\n",
      "5\t147\t271.8\t0\t147\n",
      "6\t320\t68.0\t0\t320\n",
      "7\t1097\t17.0\t0\t1097\n",
      "8\t1793\t4.2\t0\t1793\n",
      "9\t5134\t1.1\t0\t5121 13\n",
      "10\t6235\t0.3\t1\t6143 92\n",
      "11\t265\t0.1\t1\t257 8\n",
      "12\t313\t0.0\t1\t303 10\n",
      "13\t488\t0.0\t1\t468 20\n",
      "14\t742\t0.0\t1\t694 48\n",
      "15\t502\t0.0\t1\t468 34\n",
      "16\t1301\t0.0\t1\t1248 53\n",
      "17\t1473\t0.0\t1\t1415 58\n",
      "18\t968\t0.0\t1\t837 82 49\n",
      "19\t3015\t0.0\t1\t1685 1319 11\n",
      "20\t10\t0.0\t2\t0 8 2\n",
      "21\t367\t0.0\t2\t0 364 3\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t3\t0.0\t2\t2 1\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "62\t2\t0.0\t2\t2\n",
      "63\t1\t0.0\t2\t0 1\n",
      "67\t2\t0.0\t2\t1 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "81\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "97\t1\t0.0\t2\t1\n",
      "101\t2\t0.0\t2\t2\n",
      "109\t1\t0.0\t2\t1\n",
      "132\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "173\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t0 1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.B.NT.5percent.9.15.19_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.B.NT.5percent.9.15.19_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.B.NT.5percent.9.15.19_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.B.NT.5percent.9.15.19_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.B.NT.75percent.9.16.19._7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.B.NT.75percent.9.16.19._7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.B.NT.75percent.9.16.19._7_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.B.NT.75percent.9.16.19._7_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.25percent.9.12.19_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.25percent.9.12.19_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.25percent.9.12.19_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.25percent.9.12.19_15_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.50percent.9.16.19_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.50percent.9.16.19_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.50percent.9.16.19_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.50percent.9.16.19_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.75percent.9.18.19._33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.D.75percent.9.18.19._33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.75percent.9.18.19._33_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.D.75percent.9.18.19._33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.25percent.9.13.19_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.25percent.9.13.19_62_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.25percent.9.13.19_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.25percent.9.13.19_62_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.5percent.9.12.19_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.5percent.9.12.19_51_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.5percent.9.12.19_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.5percent.9.12.19_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.75percent.9.16.19._49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.F.75percent.9.16.19._49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.75percent.9.16.19._49_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.F.75percent.9.16.19._49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.25percent.9.15.19_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.25percent.9.15.19_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.25percent.9.15.19_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.25percent.9.15.19_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.5percent.9.14.19_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.5percent.9.14.19_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.5percent.9.14.19_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.5percent.9.14.19_31_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.75percent.9.18.19._16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.1.G.75percent.9.18.19._16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.75percent.9.18.19._16_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.1.G.75percent.9.18.19._16_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.B.75percent.9.5.19._60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.B.75percent.9.5.19._60_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.B.75percent.9.5.19._60_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.B.75percent.9.5.19._60_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.25percent.9.13.19_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.25percent.9.13.19_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.25percent.9.13.19_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.25percent.9.13.19_12_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.25percent.9.13.19_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.25percent.9.13.19_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.25percent.9.13.19_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.25percent.9.13.19_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.55 s (13 us/read; 4.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            346,336\n",
      "  Read 1 with adapter:                     301 (0.1%)\n",
      "  Read 2 with adapter:                  33,683 (9.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       346,336 (100.0%)\n",
      "\n",
      "Total basepairs processed:   158,346,170 bp\n",
      "  Read 1:    77,542,821 bp\n",
      "  Read 2:    80,803,349 bp\n",
      "Total written (filtered):    157,965,567 bp (99.8%)\n",
      "  Read 1:    77,541,403 bp\n",
      "  Read 2:    80,424,164 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 301 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t290\t5411.5\t0\t290\n",
      "24\t1\t0.0\t1\t0 1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "66\t1\t0.0\t1\t1\n",
      "101\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 33683 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t317\t5411.5\t0\t317\n",
      "4\t134\t1352.9\t0\t134\n",
      "5\t296\t338.2\t0\t296\n",
      "6\t734\t84.6\t0\t734\n",
      "7\t2444\t21.1\t0\t2444\n",
      "8\t3379\t5.3\t0\t3379\n",
      "9\t9135\t1.3\t0\t9120 15\n",
      "10\t7318\t0.3\t1\t7243 75\n",
      "11\t204\t0.1\t1\t200 4\n",
      "12\t284\t0.0\t1\t280 4\n",
      "13\t378\t0.0\t1\t364 14\n",
      "14\t579\t0.0\t1\t556 23\n",
      "15\t435\t0.0\t1\t408 27\n",
      "16\t1260\t0.0\t1\t1210 50\n",
      "17\t1739\t0.0\t1\t1671 68\n",
      "18\t1122\t0.0\t1\t988 91 43\n",
      "19\t3448\t0.0\t1\t1934 1496 18\n",
      "20\t4\t0.0\t2\t1 2 1\n",
      "21\t433\t0.0\t2\t0 420 13\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "30\t1\t0.0\t2\t1\n",
      "35\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t1\n",
      "73\t1\t0.0\t2\t1\n",
      "79\t5\t0.0\t2\t1 2 2\n",
      "84\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t0 1\n",
      "143\t1\t0.0\t2\t0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 1\n",
      "176\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.50percent.9.15.19_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.50percent.9.15.19_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.50percent.9.15.19_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.50percent.9.15.19_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.24 s (14 us/read; 4.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            295,871\n",
      "  Read 1 with adapter:                     304 (0.1%)\n",
      "  Read 2 with adapter:                  27,609 (9.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       295,871 (100.0%)\n",
      "\n",
      "Total basepairs processed:   135,232,759 bp\n",
      "  Read 1:    66,242,639 bp\n",
      "  Read 2:    68,990,120 bp\n",
      "Total written (filtered):    134,910,782 bp (99.8%)\n",
      "  Read 1:    66,241,149 bp\n",
      "  Read 2:    68,669,633 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 304 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t291\t4623.0\t0\t291\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t3\t0.0\t1\t3\n",
      "128\t1\t0.0\t1\t0 1\n",
      "192\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 27609 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t231\t4623.0\t0\t231\n",
      "4\t115\t1155.7\t0\t115\n",
      "5\t197\t288.9\t0\t197\n",
      "6\t489\t72.2\t0\t489\n",
      "7\t1636\t18.1\t0\t1636\n",
      "8\t2554\t4.5\t0\t2554\n",
      "9\t6778\t1.1\t0\t6769 9\n",
      "10\t6499\t0.3\t1\t6419 80\n",
      "11\t205\t0.1\t1\t199 6\n",
      "12\t251\t0.0\t1\t246 5\n",
      "13\t356\t0.0\t1\t341 15\n",
      "14\t600\t0.0\t1\t582 18\n",
      "15\t434\t0.0\t1\t417 17\n",
      "16\t1226\t0.0\t1\t1180 46\n",
      "17\t1589\t0.0\t1\t1520 69\n",
      "18\t997\t0.0\t1\t877 73 47\n",
      "19\t3039\t0.0\t1\t1703 1324 12\n",
      "20\t9\t0.0\t2\t3 4 2\n",
      "21\t363\t0.0\t2\t0 354 9\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "30\t2\t0.0\t2\t2\n",
      "47\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t2\t0.0\t2\t1 1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "71\t3\t0.0\t2\t1 1 1\n",
      "75\t2\t0.0\t2\t2\n",
      "84\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "205\t1\t0.0\t2\t0 1\n",
      "209\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.75percent.9.17.19._57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.75percent.9.17.19._57_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.75percent.9.17.19._57_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.75percent.9.17.19._57_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.42 s (15 us/read; 4.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            304,504\n",
      "  Read 1 with adapter:                     213 (0.1%)\n",
      "  Read 2 with adapter:                  26,022 (8.5%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       304,503 (100.0%)\n",
      "\n",
      "Total basepairs processed:   139,136,202 bp\n",
      "  Read 1:    68,181,209 bp\n",
      "  Read 2:    70,954,993 bp\n",
      "Total written (filtered):    138,814,739 bp (99.8%)\n",
      "  Read 1:    68,179,822 bp\n",
      "  Read 2:    70,634,917 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 213 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t201\t4757.9\t0\t201\n",
      "6\t1\t74.3\t0\t1\n",
      "9\t1\t1.2\t0\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "84\t1\t0.0\t1\t1\n",
      "107\t1\t0.0\t1\t1\n",
      "172\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 26022 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t208\t4757.9\t0\t208\n",
      "4\t58\t1189.5\t0\t58\n",
      "5\t161\t297.4\t0\t161\n",
      "6\t325\t74.3\t0\t325\n",
      "7\t1126\t18.6\t0\t1126\n",
      "8\t1804\t4.6\t0\t1804\n",
      "9\t5330\t1.2\t0\t5312 18\n",
      "10\t6530\t0.3\t1\t6462 68\n",
      "11\t284\t0.1\t1\t275 9\n",
      "12\t335\t0.0\t1\t329 6\n",
      "13\t522\t0.0\t1\t500 22\n",
      "14\t819\t0.0\t1\t790 29\n",
      "15\t564\t0.0\t1\t521 43\n",
      "16\t1449\t0.0\t1\t1384 65\n",
      "17\t1713\t0.0\t1\t1658 55\n",
      "18\t1107\t0.0\t1\t950 104 53\n",
      "19\t3240\t0.0\t1\t1794 1436 10\n",
      "20\t8\t0.0\t2\t1 5 2\n",
      "21\t378\t0.0\t2\t0 373 5\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t1\n",
      "50\t2\t0.0\t2\t1 1\n",
      "51\t1\t0.0\t2\t1\n",
      "56\t2\t0.0\t2\t2\n",
      "60\t1\t0.0\t2\t1\n",
      "63\t2\t0.0\t2\t1 0 1\n",
      "64\t1\t0.0\t2\t0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t1 1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "73\t2\t0.0\t2\t2\n",
      "74\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t1 0 1\n",
      "79\t4\t0.0\t2\t4\n",
      "89\t2\t0.0\t2\t2\n",
      "98\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t1\n",
      "155\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t0 1\n",
      "194\t1\t0.0\t2\t1\n",
      "213\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "241\t2\t0.0\t2\t0 0 2\n",
      "242\t1\t0.0\t2\t1\n",
      "243\t2\t0.0\t2\t2\n",
      "251\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.E.25percent.9.13.19_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.E.25percent.9.13.19_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.E.25percent.9.13.19_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.E.25percent.9.13.19_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.63 s (16 us/read; 3.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            165,428\n",
      "  Read 1 with adapter:                     137 (0.1%)\n",
      "  Read 2 with adapter:                  12,419 (7.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       165,428 (100.0%)\n",
      "\n",
      "Total basepairs processed:    75,551,556 bp\n",
      "  Read 1:    37,036,640 bp\n",
      "  Read 2:    38,514,916 bp\n",
      "Total written (filtered):     75,406,532 bp (99.8%)\n",
      "  Read 1:    37,035,915 bp\n",
      "  Read 2:    38,370,617 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 137 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t131\t2584.8\t0\t131\n",
      "4\t1\t646.2\t0\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12419 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t123\t2584.8\t0\t123\n",
      "4\t47\t646.2\t0\t47\n",
      "5\t85\t161.6\t0\t85\n",
      "6\t252\t40.4\t0\t252\n",
      "7\t824\t10.1\t0\t824\n",
      "8\t1343\t2.5\t0\t1343\n",
      "9\t3354\t0.6\t0\t3344 10\n",
      "10\t2426\t0.2\t1\t2395 31\n",
      "11\t45\t0.0\t1\t44 1\n",
      "12\t62\t0.0\t1\t60 2\n",
      "13\t83\t0.0\t1\t77 6\n",
      "14\t152\t0.0\t1\t136 16\n",
      "15\t131\t0.0\t1\t125 6\n",
      "16\t419\t0.0\t1\t406 13\n",
      "17\t764\t0.0\t1\t733 31\n",
      "18\t482\t0.0\t1\t434 32 16\n",
      "19\t1600\t0.0\t1\t933 662 5\n",
      "20\t9\t0.0\t2\t1 8\n",
      "21\t195\t0.0\t2\t0 193 2\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "30\t1\t0.0\t2\t1\n",
      "39\t1\t0.0\t2\t1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "48\t2\t0.0\t2\t2\n",
      "51\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "74\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "147\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t0 1\n",
      "169\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.E.75percent.9.17.19._50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.E.75percent.9.17.19._50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.E.75percent.9.17.19._50_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.E.75percent.9.17.19._50_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.61 s (14 us/read; 4.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            340,745\n",
      "  Read 1 with adapter:                     504 (0.1%)\n",
      "  Read 2 with adapter:                  33,081 (9.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       340,745 (100.0%)\n",
      "\n",
      "Total basepairs processed:   155,781,952 bp\n",
      "  Read 1:    76,300,244 bp\n",
      "  Read 2:    79,481,708 bp\n",
      "Total written (filtered):    155,390,688 bp (99.7%)\n",
      "  Read 1:    76,297,966 bp\n",
      "  Read 2:    79,092,722 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 504 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t495\t5324.1\t0\t495\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t2\t0.0\t1\t2\n",
      "113\t1\t0.0\t1\t1\n",
      "145\t1\t0.0\t1\t1\n",
      "177\t1\t0.0\t1\t0 1\n",
      "219\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 33081 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t251\t5324.1\t0\t251\n",
      "4\t101\t1331.0\t0\t101\n",
      "5\t257\t332.8\t0\t257\n",
      "6\t487\t83.2\t0\t487\n",
      "7\t1658\t20.8\t0\t1658\n",
      "8\t2529\t5.2\t0\t2529\n",
      "9\t7238\t1.3\t0\t7230 8\n",
      "10\t9024\t0.3\t1\t8917 107\n",
      "11\t323\t0.1\t1\t314 9\n",
      "12\t379\t0.0\t1\t371 8\n",
      "13\t652\t0.0\t1\t622 30\n",
      "14\t837\t0.0\t1\t802 35\n",
      "15\t586\t0.0\t1\t551 35\n",
      "16\t1617\t0.0\t1\t1558 59\n",
      "17\t1887\t0.0\t1\t1806 81\n",
      "18\t1119\t0.0\t1\t957 95 67\n",
      "19\t3691\t0.0\t1\t1998 1671 22\n",
      "20\t8\t0.0\t2\t0 7 1\n",
      "21\t409\t0.0\t2\t0 402 7\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "55\t1\t0.0\t2\t0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "82\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "172\t1\t0.0\t2\t1\n",
      "207\t2\t0.0\t2\t2\n",
      "217\t2\t0.0\t2\t2\n",
      "218\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.25percent.8.27.19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.25percent.8.27.19_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.25percent.8.27.19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.25percent.8.27.19_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.25 s (15 us/read; 4.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            353,029\n",
      "  Read 1 with adapter:                     232 (0.1%)\n",
      "  Read 2 with adapter:                  21,748 (6.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       353,029 (100.0%)\n",
      "\n",
      "Total basepairs processed:   161,047,380 bp\n",
      "  Read 1:    79,040,487 bp\n",
      "  Read 2:    82,006,893 bp\n",
      "Total written (filtered):    160,767,399 bp (99.8%)\n",
      "  Read 1:    79,038,929 bp\n",
      "  Read 2:    81,728,470 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 232 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t207\t5516.1\t0\t207\n",
      "22\t1\t0.0\t1\t1\n",
      "25\t4\t0.0\t1\t3 1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t13\t0.0\t1\t13\n",
      "29\t2\t0.0\t1\t2\n",
      "87\t1\t0.0\t1\t1\n",
      "116\t1\t0.0\t1\t0 1\n",
      "151\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 21748 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t222\t5516.1\t0\t222\n",
      "4\t52\t1379.0\t0\t52\n",
      "5\t100\t344.8\t0\t100\n",
      "6\t164\t86.2\t0\t164\n",
      "7\t566\t21.5\t0\t566\n",
      "8\t806\t5.4\t0\t806\n",
      "9\t2787\t1.3\t0\t2782 5\n",
      "10\t7667\t0.3\t1\t7585 82\n",
      "11\t261\t0.1\t1\t255 6\n",
      "12\t205\t0.0\t1\t205\n",
      "13\t215\t0.0\t1\t199 16\n",
      "14\t401\t0.0\t1\t375 26\n",
      "15\t440\t0.0\t1\t419 21\n",
      "16\t1219\t0.0\t1\t1184 35\n",
      "17\t1737\t0.0\t1\t1684 53\n",
      "18\t1102\t0.0\t1\t1019 65 18\n",
      "19\t3275\t0.0\t1\t1942 1316 17\n",
      "20\t9\t0.0\t2\t3 3 3\n",
      "21\t480\t0.0\t2\t0 467 13\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "48\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "69\t2\t0.0\t2\t0 1 1\n",
      "75\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t0 2\n",
      "79\t2\t0.0\t2\t2\n",
      "100\t1\t0.0\t2\t0 1\n",
      "105\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "230\t1\t0.0\t2\t1\n",
      "242\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.50percent.9.4.19_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.50percent.9.4.19_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.50percent.9.4.19_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.50percent.9.4.19_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.36 s (16 us/read; 3.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            331,266\n",
      "  Read 1 with adapter:                     367 (0.1%)\n",
      "  Read 2 with adapter:                  32,519 (9.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       331,266 (100.0%)\n",
      "\n",
      "Total basepairs processed:   151,429,936 bp\n",
      "  Read 1:    74,167,129 bp\n",
      "  Read 2:    77,262,807 bp\n",
      "Total written (filtered):    151,045,568 bp (99.7%)\n",
      "  Read 1:    74,165,119 bp\n",
      "  Read 2:    76,880,449 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 367 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t357\t5176.0\t0\t357\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t0 1\n",
      "131\t1\t0.0\t1\t1\n",
      "151\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 32519 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t254\t5176.0\t0\t254\n",
      "4\t129\t1294.0\t0\t129\n",
      "5\t250\t323.5\t0\t250\n",
      "6\t498\t80.9\t0\t498\n",
      "7\t1790\t20.2\t0\t1790\n",
      "8\t2784\t5.1\t0\t2784\n",
      "9\t7611\t1.3\t0\t7599 12\n",
      "10\t8108\t0.3\t1\t8009 99\n",
      "11\t295\t0.1\t1\t289 6\n",
      "12\t344\t0.0\t1\t335 9\n",
      "13\t533\t0.0\t1\t512 21\n",
      "14\t839\t0.0\t1\t800 39\n",
      "15\t582\t0.0\t1\t540 42\n",
      "16\t1465\t0.0\t1\t1394 71\n",
      "17\t1833\t0.0\t1\t1746 87\n",
      "18\t1195\t0.0\t1\t1050 80 65\n",
      "19\t3501\t0.0\t1\t1905 1574 22\n",
      "20\t9\t0.0\t2\t1 8\n",
      "21\t429\t0.0\t2\t0 420 9\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "30\t1\t0.0\t2\t1\n",
      "50\t2\t0.0\t2\t2\n",
      "53\t1\t0.0\t2\t0 1\n",
      "54\t2\t0.0\t2\t1 1\n",
      "55\t2\t0.0\t2\t2\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t2\t0.0\t2\t0 1 1\n",
      "64\t2\t0.0\t2\t1 0 1\n",
      "66\t2\t0.0\t2\t1 1\n",
      "67\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t1 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t6\t0.0\t2\t6\n",
      "83\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "93\t2\t0.0\t2\t2\n",
      "98\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t1\n",
      "122\t1\t0.0\t2\t1\n",
      "139\t1\t0.0\t2\t1\n",
      "193\t1\t0.0\t2\t0 0 1\n",
      "205\t1\t0.0\t2\t0 1\n",
      "214\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "225\t1\t0.0\t2\t1\n",
      "240\t1\t0.0\t2\t1\n",
      "241\t1\t0.0\t2\t0 0 1\n",
      "242\t1\t0.0\t2\t0 1\n",
      "248\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.5percent.8.26.19_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.5percent.8.26.19_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.5percent.8.26.19_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.5percent.8.26.19_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.04 s (18 us/read; 3.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            280,599\n",
      "  Read 1 with adapter:                     181 (0.1%)\n",
      "  Read 2 with adapter:                  19,334 (6.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       280,599 (100.0%)\n",
      "\n",
      "Total basepairs processed:   127,791,687 bp\n",
      "  Read 1:    62,708,343 bp\n",
      "  Read 2:    65,083,344 bp\n",
      "Total written (filtered):    127,540,856 bp (99.8%)\n",
      "  Read 1:    62,706,373 bp\n",
      "  Read 2:    64,834,483 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 181 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t168\t4384.4\t0\t168\n",
      "23\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "141\t1\t0.0\t1\t1\n",
      "208\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 19334 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t173\t4384.4\t0\t173\n",
      "4\t86\t1096.1\t0\t86\n",
      "5\t117\t274.0\t0\t117\n",
      "6\t257\t68.5\t0\t257\n",
      "7\t667\t17.1\t0\t667\n",
      "8\t1087\t4.3\t0\t1087\n",
      "9\t3661\t1.1\t0\t3653 8\n",
      "10\t4189\t0.3\t1\t4135 54\n",
      "11\t206\t0.1\t1\t202 4\n",
      "12\t263\t0.0\t1\t259 4\n",
      "13\t328\t0.0\t1\t310 18\n",
      "14\t575\t0.0\t1\t549 26\n",
      "15\t458\t0.0\t1\t439 19\n",
      "16\t1223\t0.0\t1\t1167 56\n",
      "17\t1679\t0.0\t1\t1624 55\n",
      "18\t907\t0.0\t1\t801 62 44\n",
      "19\t3044\t0.0\t1\t1680 1343 21\n",
      "20\t6\t0.0\t2\t0 5 1\n",
      "21\t366\t0.0\t2\t0 361 5\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t2\n",
      "73\t1\t0.0\t2\t1\n",
      "79\t5\t0.0\t2\t5\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "170\t1\t0.0\t2\t1\n",
      "186\t2\t0.0\t2\t1 1\n",
      "192\t1\t0.0\t2\t0 1\n",
      "193\t1\t0.0\t2\t1\n",
      "233\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.75percent.9.6.19._21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.75percent.9.6.19._21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.75percent.9.6.19._21_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.75percent.9.6.19._21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.74 s (14 us/read; 4.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            330,014\n",
      "  Read 1 with adapter:                     310 (0.1%)\n",
      "  Read 2 with adapter:                  31,236 (9.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       330,014 (100.0%)\n",
      "\n",
      "Total basepairs processed:   150,819,322 bp\n",
      "  Read 1:    73,881,632 bp\n",
      "  Read 2:    76,937,690 bp\n",
      "Total written (filtered):    150,449,942 bp (99.8%)\n",
      "  Read 1:    73,879,908 bp\n",
      "  Read 2:    76,570,034 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 310 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t303\t5156.5\t0\t303\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "153\t1\t0.0\t1\t1\n",
      "183\t1\t0.0\t1\t0 1\n",
      "210\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 31236 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t253\t5156.5\t0\t253\n",
      "4\t110\t1289.1\t0\t110\n",
      "5\t199\t322.3\t0\t199\n",
      "6\t480\t80.6\t0\t480\n",
      "7\t1742\t20.1\t0\t1742\n",
      "8\t2561\t5.0\t0\t2561\n",
      "9\t7207\t1.3\t0\t7195 12\n",
      "10\t7958\t0.3\t1\t7879 79\n",
      "11\t285\t0.1\t1\t280 5\n",
      "12\t338\t0.0\t1\t331 7\n",
      "13\t517\t0.0\t1\t494 23\n",
      "14\t795\t0.0\t1\t754 41\n",
      "15\t548\t0.0\t1\t515 33\n",
      "16\t1472\t0.0\t1\t1425 47\n",
      "17\t1755\t0.0\t1\t1675 80\n",
      "18\t1064\t0.0\t1\t927 74 63\n",
      "19\t3440\t0.0\t1\t1943 1480 17\n",
      "20\t9\t0.0\t2\t0 6 3\n",
      "21\t445\t0.0\t2\t0 427 18\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "30\t1\t0.0\t2\t1\n",
      "46\t1\t0.0\t2\t1\n",
      "56\t3\t0.0\t2\t3\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t2 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "92\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t1\n",
      "105\t2\t0.0\t2\t1 1\n",
      "114\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "128\t1\t0.0\t2\t1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t0 1\n",
      "185\t1\t0.0\t2\t0 1\n",
      "186\t1\t0.0\t2\t0 1\n",
      "218\t2\t0.0\t2\t2\n",
      "238\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.D.25percent.9.10.19_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.D.25percent.9.10.19_65_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.D.25percent.9.10.19_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.D.25percent.9.10.19_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.75 s (16 us/read; 3.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            231,502\n",
      "  Read 1 with adapter:                     328 (0.1%)\n",
      "  Read 2 with adapter:                  18,088 (7.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       231,502 (100.0%)\n",
      "\n",
      "Total basepairs processed:   105,694,332 bp\n",
      "  Read 1:    51,806,257 bp\n",
      "  Read 2:    53,888,075 bp\n",
      "Total written (filtered):    105,473,881 bp (99.8%)\n",
      "  Read 1:    51,804,134 bp\n",
      "  Read 2:    53,669,747 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 328 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t315\t3617.2\t0\t315\n",
      "19\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t4\t0.0\t1\t4\n",
      "101\t2\t0.0\t1\t2\n",
      "144\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18088 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t159\t3617.2\t0\t159\n",
      "4\t56\t904.3\t0\t56\n",
      "5\t105\t226.1\t0\t105\n",
      "6\t258\t56.5\t0\t258\n",
      "7\t944\t14.1\t0\t944\n",
      "8\t1457\t3.5\t0\t1457\n",
      "9\t4271\t0.9\t0\t4265 6\n",
      "10\t3977\t0.2\t1\t3930 47\n",
      "11\t139\t0.1\t1\t134 5\n",
      "12\t190\t0.0\t1\t185 5\n",
      "13\t280\t0.0\t1\t266 14\n",
      "14\t418\t0.0\t1\t386 32\n",
      "15\t301\t0.0\t1\t282 19\n",
      "16\t811\t0.0\t1\t777 34\n",
      "17\t1239\t0.0\t1\t1192 47\n",
      "18\t726\t0.0\t1\t649 50 27\n",
      "19\t2403\t0.0\t1\t1380 1016 7\n",
      "20\t8\t0.0\t2\t0 8\n",
      "21\t310\t0.0\t2\t0 305 5\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "28\t1\t0.0\t2\t1\n",
      "30\t1\t0.0\t2\t1\n",
      "34\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "53\t2\t0.0\t2\t2\n",
      "55\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t1 1\n",
      "117\t1\t0.0\t2\t1\n",
      "148\t1\t0.0\t2\t0 1\n",
      "173\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.D.50percent.9.17.19_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.D.50percent.9.17.19_54_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.D.50percent.9.17.19_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.D.50percent.9.17.19_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.41 s (16 us/read; 3.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            281,206\n",
      "  Read 1 with adapter:                     341 (0.1%)\n",
      "  Read 2 with adapter:                  27,017 (9.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       281,206 (100.0%)\n",
      "\n",
      "Total basepairs processed:   128,569,426 bp\n",
      "  Read 1:    62,965,572 bp\n",
      "  Read 2:    65,603,854 bp\n",
      "Total written (filtered):    128,250,692 bp (99.8%)\n",
      "  Read 1:    62,964,091 bp\n",
      "  Read 2:    65,286,601 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 341 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t333\t4393.8\t0\t333\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "29\t2\t0.0\t1\t2\n",
      "62\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "208\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 27017 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t244\t4393.8\t0\t244\n",
      "4\t99\t1098.5\t0\t99\n",
      "5\t190\t274.6\t0\t190\n",
      "6\t435\t68.7\t0\t435\n",
      "7\t1513\t17.2\t0\t1513\n",
      "8\t2224\t4.3\t0\t2224\n",
      "9\t6490\t1.1\t0\t6471 19\n",
      "10\t6624\t0.3\t1\t6533 91\n",
      "11\t240\t0.1\t1\t233 7\n",
      "12\t292\t0.0\t1\t281 11\n",
      "13\t414\t0.0\t1\t398 16\n",
      "14\t647\t0.0\t1\t620 27\n",
      "15\t461\t0.0\t1\t426 35\n",
      "16\t1218\t0.0\t1\t1166 52\n",
      "17\t1538\t0.0\t1\t1483 55\n",
      "18\t954\t0.0\t1\t818 83 53\n",
      "19\t2976\t0.0\t1\t1649 1310 17\n",
      "20\t8\t0.0\t2\t2 5 1\n",
      "21\t391\t0.0\t2\t0 384 7\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t2\t0.0\t2\t2\n",
      "65\t2\t0.0\t2\t2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 1\n",
      "72\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t6\t0.0\t2\t4 1 1\n",
      "96\t1\t0.0\t2\t1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t1\n",
      "151\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "192\t1\t0.0\t2\t0 1\n",
      "200\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "232\t1\t0.0\t2\t0 1\n",
      "234\t2\t0.0\t2\t2\n",
      "243\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.F.NT.5percent.9.16.19_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.F.NT.5percent.9.16.19_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.F.NT.5percent.9.16.19_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.F.NT.5percent.9.16.19_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.44 s (14 us/read; 4.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            324,367\n",
      "  Read 1 with adapter:                     319 (0.1%)\n",
      "  Read 2 with adapter:                  18,374 (5.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       324,367 (100.0%)\n",
      "\n",
      "Total basepairs processed:   147,970,192 bp\n",
      "  Read 1:    72,615,064 bp\n",
      "  Read 2:    75,355,128 bp\n",
      "Total written (filtered):    147,720,317 bp (99.8%)\n",
      "  Read 1:    72,612,780 bp\n",
      "  Read 2:    75,107,537 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 319 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t300\t5068.2\t0\t300\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t6\t0.0\t1\t6\n",
      "29\t3\t0.0\t1\t3\n",
      "153\t3\t0.0\t1\t3\n",
      "158\t1\t0.0\t1\t1\n",
      "208\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18374 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t105\t5068.2\t0\t105\n",
      "4\t62\t1267.1\t0\t62\n",
      "5\t81\t316.8\t0\t81\n",
      "6\t200\t79.2\t0\t200\n",
      "7\t534\t19.8\t0\t534\n",
      "8\t935\t4.9\t0\t935\n",
      "9\t2952\t1.2\t0\t2943 9\n",
      "10\t3611\t0.3\t1\t3574 37\n",
      "11\t220\t0.1\t1\t215 5\n",
      "12\t352\t0.0\t1\t345 7\n",
      "13\t451\t0.0\t1\t431 20\n",
      "14\t665\t0.0\t1\t640 25\n",
      "15\t441\t0.0\t1\t420 21\n",
      "16\t1143\t0.0\t1\t1093 50\n",
      "17\t1711\t0.0\t1\t1657 54\n",
      "18\t1016\t0.0\t1\t889 89 38\n",
      "19\t3357\t0.0\t1\t1973 1373 11\n",
      "20\t15\t0.0\t2\t7 7 1\n",
      "21\t463\t0.0\t2\t0 454 9\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "30\t2\t0.0\t2\t2\n",
      "34\t1\t0.0\t2\t1\n",
      "35\t1\t0.0\t2\t1\n",
      "44\t2\t0.0\t2\t2\n",
      "49\t2\t0.0\t2\t2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t1\t0.0\t2\t0 1\n",
      "70\t2\t0.0\t2\t1 1\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "78\t3\t0.0\t2\t3\n",
      "87\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "202\t1\t0.0\t2\t1\n",
      "218\t5\t0.0\t2\t4 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.G.NT.5percent.9.18.19_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.G.NT.5percent.9.18.19_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.G.NT.5percent.9.18.19_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.G.NT.5percent.9.18.19_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.78 s (15 us/read; 4.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            389,606\n",
      "  Read 1 with adapter:                     309 (0.1%)\n",
      "  Read 2 with adapter:                  32,659 (8.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       389,606 (100.0%)\n",
      "\n",
      "Total basepairs processed:   177,965,630 bp\n",
      "  Read 1:    87,220,905 bp\n",
      "  Read 2:    90,744,725 bp\n",
      "Total written (filtered):    177,554,350 bp (99.8%)\n",
      "  Read 1:    87,219,452 bp\n",
      "  Read 2:    90,334,898 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 309 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t298\t6087.6\t0\t298\n",
      "15\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "162\t1\t0.0\t1\t1\n",
      "172\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 32659 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t251\t6087.6\t0\t251\n",
      "4\t114\t1521.9\t0\t114\n",
      "5\t159\t380.5\t0\t159\n",
      "6\t430\t95.1\t0\t430\n",
      "7\t1287\t23.8\t0\t1287\n",
      "8\t1981\t5.9\t0\t1981\n",
      "9\t6085\t1.5\t0\t6067 18\n",
      "10\t8302\t0.4\t1\t8213 89\n",
      "11\t372\t0.1\t1\t362 10\n",
      "12\t523\t0.0\t1\t502 21\n",
      "13\t738\t0.0\t1\t710 28\n",
      "14\t1084\t0.0\t1\t1038 46\n",
      "15\t790\t0.0\t1\t734 56\n",
      "16\t1883\t0.0\t1\t1801 82\n",
      "17\t2305\t0.0\t1\t2214 91\n",
      "18\t1392\t0.0\t1\t1201 110 81\n",
      "19\t4337\t0.0\t1\t2334 1977 26\n",
      "20\t12\t0.0\t2\t5 6 1\n",
      "21\t520\t0.0\t2\t0 512 8\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "30\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t0 1\n",
      "54\t2\t0.0\t2\t1 1\n",
      "55\t3\t0.0\t2\t3\n",
      "56\t2\t0.0\t2\t1 1\n",
      "57\t3\t0.0\t2\t2 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t2\n",
      "65\t1\t0.0\t2\t0 1\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t3\t0.0\t2\t2 0 1\n",
      "70\t4\t0.0\t2\t2 1 1\n",
      "71\t4\t0.0\t2\t0 0 4\n",
      "79\t5\t0.0\t2\t3 0 2\n",
      "81\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "97\t1\t0.0\t2\t1\n",
      "99\t1\t0.0\t2\t1\n",
      "122\t2\t0.0\t2\t2\n",
      "129\t1\t0.0\t2\t1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t1\n",
      "151\t2\t0.0\t2\t1 0 1\n",
      "174\t2\t0.0\t2\t1 0 1\n",
      "180\t2\t0.0\t2\t2\n",
      "212\t1\t0.0\t2\t0 1\n",
      "214\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "219\t1\t0.0\t2\t1\n",
      "222\t1\t0.0\t2\t1\n",
      "235\t2\t0.0\t2\t1 1\n",
      "237\t2\t0.0\t2\t2\n",
      "247\t1\t0.0\t2\t0 1\n",
      "248\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.50percent.9.15.19_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.50percent.9.15.19_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.50percent.9.15.19_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.50percent.9.15.19_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.75percent.9.17.19._57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.D.75percent.9.17.19._57_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.75percent.9.17.19._57_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.D.75percent.9.17.19._57_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.E.25percent.9.13.19_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.E.25percent.9.13.19_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.E.25percent.9.13.19_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.E.25percent.9.13.19_35_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.E.75percent.9.17.19._50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.E.75percent.9.17.19._50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.E.75percent.9.17.19._50_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.E.75percent.9.17.19._50_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.25percent.8.27.19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.25percent.8.27.19_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.25percent.8.27.19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.25percent.8.27.19_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.50percent.9.4.19_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.50percent.9.4.19_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.50percent.9.4.19_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.50percent.9.4.19_20_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.5percent.8.26.19_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.5percent.8.26.19_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.5percent.8.26.19_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.5percent.8.26.19_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.75percent.9.6.19._21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.10.G.75percent.9.6.19._21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.75percent.9.6.19._21_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.10.G.75percent.9.6.19._21_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.D.25percent.9.10.19_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.D.25percent.9.10.19_65_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.D.25percent.9.10.19_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.D.25percent.9.10.19_65_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.D.50percent.9.17.19_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.D.50percent.9.17.19_54_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.D.50percent.9.17.19_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.D.50percent.9.17.19_54_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.F.NT.5percent.9.16.19_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.F.NT.5percent.9.16.19_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.F.NT.5percent.9.16.19_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.F.NT.5percent.9.16.19_14_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.G.NT.5percent.9.18.19_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.G.NT.5percent.9.18.19_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.G.NT.5percent.9.18.19_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.G.NT.5percent.9.18.19_8_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.G.NT.5percent.9.19.19_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.G.NT.5percent.9.19.19_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.G.NT.5percent.9.19.19_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.G.NT.5percent.9.19.19_11_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.G.NT.5percent.9.19.19_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.11.G.NT.5percent.9.19.19_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.G.NT.5percent.9.19.19_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.11.G.NT.5percent.9.19.19_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.89 s (16 us/read; 3.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            432,924\n",
      "  Read 1 with adapter:                     281 (0.1%)\n",
      "  Read 2 with adapter:                  31,763 (7.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       432,924 (100.0%)\n",
      "\n",
      "Total basepairs processed:   197,418,465 bp\n",
      "  Read 1:    96,853,820 bp\n",
      "  Read 2:   100,564,645 bp\n",
      "Total written (filtered):    197,018,764 bp (99.8%)\n",
      "  Read 1:    96,851,997 bp\n",
      "  Read 2:   100,166,767 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 281 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t272\t6764.4\t0\t272\n",
      "26\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t1\n",
      "39\t2\t0.0\t1\t2\n",
      "207\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 31763 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t291\t6764.4\t0\t291\n",
      "4\t112\t1691.1\t0\t112\n",
      "5\t225\t422.8\t0\t225\n",
      "6\t419\t105.7\t0\t419\n",
      "7\t1163\t26.4\t0\t1163\n",
      "8\t1814\t6.6\t0\t1814\n",
      "9\t5982\t1.7\t0\t5962 20\n",
      "10\t8031\t0.4\t1\t7945 86\n",
      "11\t329\t0.1\t1\t320 9\n",
      "12\t410\t0.0\t1\t404 6\n",
      "13\t551\t0.0\t1\t522 29\n",
      "14\t948\t0.0\t1\t891 57\n",
      "15\t675\t0.0\t1\t646 29\n",
      "16\t1864\t0.0\t1\t1787 77\n",
      "17\t2323\t0.0\t1\t2218 105\n",
      "18\t1397\t0.0\t1\t1219 113 65\n",
      "19\t4603\t0.0\t1\t2545 2024 34\n",
      "20\t11\t0.0\t2\t1 7 3\n",
      "21\t565\t0.0\t2\t0 564 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "35\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t1\n",
      "63\t2\t0.0\t2\t2\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "70\t3\t0.0\t2\t2 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t3\n",
      "93\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t1\n",
      "165\t1\t0.0\t2\t0 1\n",
      "193\t1\t0.0\t2\t1\n",
      "202\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.D.NT.5percent.9.16.19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.D.NT.5percent.9.16.19_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.D.NT.5percent.9.16.19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.D.NT.5percent.9.16.19_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.92 s (16 us/read; 3.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            314,667\n",
      "  Read 1 with adapter:                     375 (0.1%)\n",
      "  Read 2 with adapter:                  23,710 (7.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       314,667 (100.0%)\n",
      "\n",
      "Total basepairs processed:   143,653,880 bp\n",
      "  Read 1:    70,433,963 bp\n",
      "  Read 2:    73,219,917 bp\n",
      "Total written (filtered):    143,353,801 bp (99.8%)\n",
      "  Read 1:    70,431,835 bp\n",
      "  Read 2:    72,921,966 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 375 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t361\t4916.7\t0\t361\n",
      "15\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "29\t3\t0.0\t1\t3\n",
      "55\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "134\t1\t0.0\t1\t1\n",
      "166\t2\t0.0\t1\t2\n",
      "206\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 23710 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t178\t4916.7\t0\t178\n",
      "4\t76\t1229.2\t0\t76\n",
      "5\t113\t307.3\t0\t113\n",
      "6\t281\t76.8\t0\t281\n",
      "7\t917\t19.2\t0\t917\n",
      "8\t1455\t4.8\t0\t1455\n",
      "9\t4636\t1.2\t0\t4618 18\n",
      "10\t5613\t0.3\t1\t5555 58\n",
      "11\t256\t0.1\t1\t250 6\n",
      "12\t350\t0.0\t1\t342 8\n",
      "13\t490\t0.0\t1\t472 18\n",
      "14\t754\t0.0\t1\t712 42\n",
      "15\t551\t0.0\t1\t522 29\n",
      "16\t1336\t0.0\t1\t1279 57\n",
      "17\t1782\t0.0\t1\t1713 69\n",
      "18\t1083\t0.0\t1\t946 87 50\n",
      "19\t3353\t0.0\t1\t1920 1416 17\n",
      "20\t14\t0.0\t2\t3 9 2\n",
      "21\t432\t0.0\t2\t0 425 7\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "30\t1\t0.0\t2\t1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t1\n",
      "59\t2\t0.0\t2\t1 0 1\n",
      "64\t2\t0.0\t2\t2\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t3\t0.0\t2\t3\n",
      "67\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "79\t4\t0.0\t2\t3 1\n",
      "80\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t0 0 1\n",
      "238\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.E.50percent.9.7.19_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.E.50percent.9.7.19_69_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.E.50percent.9.7.19_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.E.50percent.9.7.19_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.42 s (15 us/read; 3.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            224,510\n",
      "  Read 1 with adapter:                     268 (0.1%)\n",
      "  Read 2 with adapter:                  15,682 (7.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       224,510 (100.0%)\n",
      "\n",
      "Total basepairs processed:   102,298,000 bp\n",
      "  Read 1:    50,161,226 bp\n",
      "  Read 2:    52,136,774 bp\n",
      "Total written (filtered):    102,092,209 bp (99.8%)\n",
      "  Read 1:    50,159,320 bp\n",
      "  Read 2:    51,932,889 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 268 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t256\t3508.0\t0\t256\n",
      "23\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "92\t7\t0.0\t1\t6 1\n",
      "207\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15682 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t113\t3508.0\t0\t113\n",
      "4\t38\t877.0\t0\t38\n",
      "5\t70\t219.2\t0\t70\n",
      "6\t146\t54.8\t0\t146\n",
      "7\t535\t13.7\t0\t535\n",
      "8\t847\t3.4\t0\t847\n",
      "9\t2730\t0.9\t0\t2724 6\n",
      "10\t3599\t0.2\t1\t3562 37\n",
      "11\t184\t0.1\t1\t180 4\n",
      "12\t272\t0.0\t1\t267 5\n",
      "13\t363\t0.0\t1\t347 16\n",
      "14\t557\t0.0\t1\t533 24\n",
      "15\t365\t0.0\t1\t347 18\n",
      "16\t1035\t0.0\t1\t1000 35\n",
      "17\t1285\t0.0\t1\t1235 50\n",
      "18\t788\t0.0\t1\t709 49 30\n",
      "19\t2393\t0.0\t1\t1352 1027 14\n",
      "20\t3\t0.0\t2\t0 3\n",
      "21\t320\t0.0\t2\t0 316 4\n",
      "22\t17\t0.0\t2\t0 1 16\n",
      "30\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "65\t2\t0.0\t2\t2\n",
      "67\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t2\n",
      "88\t2\t0.0\t2\t2\n",
      "102\t1\t0.0\t2\t1\n",
      "119\t1\t0.0\t2\t0 1\n",
      "129\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t1\n",
      "171\t1\t0.0\t2\t1\n",
      "175\t1\t0.0\t2\t0 1\n",
      "218\t1\t0.0\t2\t0 0 1\n",
      "239\t1\t0.0\t2\t1\n",
      "241\t1\t0.0\t2\t0 1\n",
      "242\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.E.75percent.9.8.19._25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.E.75percent.9.8.19._25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.E.75percent.9.8.19._25_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.E.75percent.9.8.19._25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.27 s (17 us/read; 3.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            250,033\n",
      "  Read 1 with adapter:                     266 (0.1%)\n",
      "  Read 2 with adapter:                  14,951 (6.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       250,033 (100.0%)\n",
      "\n",
      "Total basepairs processed:   114,003,329 bp\n",
      "  Read 1:    55,935,165 bp\n",
      "  Read 2:    58,068,164 bp\n",
      "Total written (filtered):    113,796,922 bp (99.8%)\n",
      "  Read 1:    55,933,904 bp\n",
      "  Read 2:    57,863,018 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 266 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t260\t3906.8\t0\t260\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t0 1\n",
      "38\t1\t0.0\t1\t1\n",
      "84\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14951 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t107\t3906.8\t0\t107\n",
      "4\t32\t976.7\t0\t32\n",
      "5\t63\t244.2\t0\t63\n",
      "6\t120\t61.0\t0\t120\n",
      "7\t342\t15.3\t0\t342\n",
      "8\t645\t3.8\t0\t645\n",
      "9\t1891\t1.0\t0\t1884 7\n",
      "10\t3122\t0.2\t1\t3080 42\n",
      "11\t193\t0.1\t1\t183 10\n",
      "12\t350\t0.0\t1\t347 3\n",
      "13\t401\t0.0\t1\t380 21\n",
      "14\t653\t0.0\t1\t621 32\n",
      "15\t458\t0.0\t1\t435 23\n",
      "16\t1147\t0.0\t1\t1102 45\n",
      "17\t1468\t0.0\t1\t1408 60\n",
      "18\t822\t0.0\t1\t717 59 46\n",
      "19\t2737\t0.0\t1\t1590 1133 14\n",
      "20\t4\t0.0\t2\t0 3 1\n",
      "21\t351\t0.0\t2\t0 345 6\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t2 0 2\n",
      "88\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t0 1\n",
      "213\t2\t0.0\t2\t2\n",
      "238\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.G.25percent.9.17.19_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.G.25percent.9.17.19_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.G.25percent.9.17.19_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.G.25percent.9.17.19_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.35 s (15 us/read; 4.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            290,948\n",
      "  Read 1 with adapter:                     465 (0.2%)\n",
      "  Read 2 with adapter:                  22,368 (7.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       290,948 (100.0%)\n",
      "\n",
      "Total basepairs processed:   132,857,439 bp\n",
      "  Read 1:    65,128,546 bp\n",
      "  Read 2:    67,728,893 bp\n",
      "Total written (filtered):    132,586,447 bp (99.8%)\n",
      "  Read 1:    65,126,643 bp\n",
      "  Read 2:    67,459,804 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 465 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t459\t4546.1\t0\t459\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "207\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 22368 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t203\t4546.1\t0\t203\n",
      "4\t94\t1136.5\t0\t94\n",
      "5\t135\t284.1\t0\t135\n",
      "6\t379\t71.0\t0\t379\n",
      "7\t1069\t17.8\t0\t1069\n",
      "8\t1802\t4.4\t0\t1802\n",
      "9\t5217\t1.1\t0\t5210 7\n",
      "10\t5012\t0.3\t1\t4943 69\n",
      "11\t188\t0.1\t1\t183 5\n",
      "12\t219\t0.0\t1\t216 3\n",
      "13\t301\t0.0\t1\t285 16\n",
      "14\t531\t0.0\t1\t500 31\n",
      "15\t384\t0.0\t1\t351 33\n",
      "16\t1058\t0.0\t1\t1022 36\n",
      "17\t1498\t0.0\t1\t1442 56\n",
      "18\t947\t0.0\t1\t841 78 28\n",
      "19\t2883\t0.0\t1\t1682 1185 16\n",
      "20\t9\t0.0\t2\t2 6 1\n",
      "21\t413\t0.0\t2\t0 405 8\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "46\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "77\t2\t0.0\t2\t2\n",
      "92\t1\t0.0\t2\t1\n",
      "127\t1\t0.0\t2\t1\n",
      "178\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.18.G.75percent.8.12.19._58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.18.G.75percent.8.12.19._58_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.18.G.75percent.8.12.19._58_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.18.G.75percent.8.12.19._58_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.47 s (15 us/read; 3.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            292,102\n",
      "  Read 1 with adapter:                     308 (0.1%)\n",
      "  Read 2 with adapter:                  33,118 (11.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       292,102 (100.0%)\n",
      "\n",
      "Total basepairs processed:   133,652,202 bp\n",
      "  Read 1:    65,405,127 bp\n",
      "  Read 2:    68,247,075 bp\n",
      "Total written (filtered):    133,275,864 bp (99.7%)\n",
      "  Read 1:    65,403,583 bp\n",
      "  Read 2:    67,872,281 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 308 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t301\t4564.1\t0\t301\n",
      "25\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t1 1\n",
      "137\t1\t0.0\t1\t1\n",
      "140\t2\t0.0\t1\t2\n",
      "141\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 33118 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t290\t4564.1\t0\t290\n",
      "4\t136\t1141.0\t0\t136\n",
      "5\t269\t285.3\t0\t269\n",
      "6\t674\t71.3\t0\t674\n",
      "7\t2197\t17.8\t0\t2197\n",
      "8\t3094\t4.5\t0\t3094\n",
      "9\t8470\t1.1\t0\t8453 17\n",
      "10\t8146\t0.3\t1\t8044 102\n",
      "11\t291\t0.1\t1\t280 11\n",
      "12\t319\t0.0\t1\t310 9\n",
      "13\t522\t0.0\t1\t497 25\n",
      "14\t754\t0.0\t1\t713 41\n",
      "15\t587\t0.0\t1\t538 49\n",
      "16\t1342\t0.0\t1\t1278 64\n",
      "17\t1619\t0.0\t1\t1561 58\n",
      "18\t981\t0.0\t1\t859 66 56\n",
      "19\t3006\t0.0\t1\t1616 1370 20\n",
      "20\t10\t0.0\t2\t3 6 1\n",
      "21\t355\t0.0\t2\t0 351 4\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "46\t1\t0.0\t2\t1\n",
      "53\t3\t0.0\t2\t2 1\n",
      "54\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "70\t3\t0.0\t2\t3\n",
      "71\t1\t0.0\t2\t1\n",
      "72\t1\t0.0\t2\t1\n",
      "74\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "84\t1\t0.0\t2\t1\n",
      "86\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "107\t1\t0.0\t2\t1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t1\n",
      "146\t1\t0.0\t2\t0 1\n",
      "150\t1\t0.0\t2\t0 1\n",
      "152\t1\t0.0\t2\t0 1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "157\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t1\n",
      "229\t2\t0.0\t2\t1 1\n",
      "234\t2\t0.0\t2\t1 1\n",
      "235\t2\t0.0\t2\t2\n",
      "238\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t1\n",
      "242\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.A.25percent.8.21.19_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.A.25percent.8.21.19_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.A.25percent.8.21.19_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.A.25percent.8.21.19_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.08 s (15 us/read; 3.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            269,845\n",
      "  Read 1 with adapter:                     320 (0.1%)\n",
      "  Read 2 with adapter:                  21,984 (8.1%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       269,844 (100.0%)\n",
      "\n",
      "Total basepairs processed:   123,287,396 bp\n",
      "  Read 1:    60,421,702 bp\n",
      "  Read 2:    62,865,694 bp\n",
      "Total written (filtered):    123,032,112 bp (99.8%)\n",
      "  Read 1:    60,419,938 bp\n",
      "  Read 2:    62,612,174 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 320 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t312\t4216.3\t0\t312\n",
      "19\t1\t0.0\t1\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t1 1\n",
      "128\t1\t0.0\t1\t0 1\n",
      "140\t1\t0.0\t1\t0 1\n",
      "219\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 21984 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t217\t4216.3\t0\t217\n",
      "4\t104\t1054.1\t0\t104\n",
      "5\t160\t263.5\t0\t160\n",
      "6\t471\t65.9\t0\t471\n",
      "7\t1485\t16.5\t0\t1485\n",
      "8\t2229\t4.1\t0\t2229\n",
      "9\t5817\t1.0\t0\t5807 10\n",
      "10\t4821\t0.3\t1\t4756 65\n",
      "11\t112\t0.1\t1\t105 7\n",
      "12\t103\t0.0\t1\t100 3\n",
      "13\t151\t0.0\t1\t144 7\n",
      "14\t276\t0.0\t1\t261 15\n",
      "15\t271\t0.0\t1\t261 10\n",
      "16\t699\t0.0\t1\t676 23\n",
      "17\t1253\t0.0\t1\t1205 48\n",
      "18\t832\t0.0\t1\t766 51 15\n",
      "19\t2563\t0.0\t1\t1484 1064 15\n",
      "20\t6\t0.0\t2\t0 4 2\n",
      "21\t362\t0.0\t2\t0 354 8\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "29\t1\t0.0\t2\t0 1\n",
      "30\t3\t0.0\t2\t3\n",
      "34\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t1\n",
      "63\t2\t0.0\t2\t2\n",
      "69\t1\t0.0\t2\t0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t4\n",
      "81\t1\t0.0\t2\t1\n",
      "111\t1\t0.0\t2\t1\n",
      "136\t1\t0.0\t2\t1\n",
      "138\t1\t0.0\t2\t1\n",
      "160\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "189\t1\t0.0\t2\t1\n",
      "192\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "228\t1\t0.0\t2\t0 1\n",
      "239\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.A.5percent.8.20.19_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.A.5percent.8.20.19_68_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.A.5percent.8.20.19_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.A.5percent.8.20.19_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.20 s (14 us/read; 4.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            359,152\n",
      "  Read 1 with adapter:                     502 (0.1%)\n",
      "  Read 2 with adapter:                  30,300 (8.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       359,152 (100.0%)\n",
      "\n",
      "Total basepairs processed:   164,018,675 bp\n",
      "  Read 1:    80,388,856 bp\n",
      "  Read 2:    83,629,819 bp\n",
      "Total written (filtered):    163,647,197 bp (99.8%)\n",
      "  Read 1:    80,387,034 bp\n",
      "  Read 2:    83,260,163 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 502 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t492\t5611.8\t0\t492\n",
      "20\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "84\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 30300 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t213\t5611.8\t0\t213\n",
      "4\t105\t1402.9\t0\t105\n",
      "5\t186\t350.7\t0\t186\n",
      "6\t427\t87.7\t0\t427\n",
      "7\t1414\t21.9\t0\t1414\n",
      "8\t2168\t5.5\t0\t2168\n",
      "9\t6536\t1.4\t0\t6528 8\n",
      "10\t7533\t0.3\t1\t7453 80\n",
      "11\t289\t0.1\t1\t284 5\n",
      "12\t323\t0.0\t1\t311 12\n",
      "13\t510\t0.0\t1\t487 23\n",
      "14\t805\t0.0\t1\t756 49\n",
      "15\t621\t0.0\t1\t586 35\n",
      "16\t1636\t0.0\t1\t1553 83\n",
      "17\t1989\t0.0\t1\t1907 82\n",
      "18\t1249\t0.0\t1\t1085 110 54\n",
      "19\t3761\t0.0\t1\t2088 1646 27\n",
      "20\t11\t0.0\t2\t1 6 4\n",
      "21\t468\t0.0\t2\t0 460 8\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "54\t2\t0.0\t2\t2\n",
      "55\t1\t0.0\t2\t0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t0 1\n",
      "79\t3\t0.0\t2\t3\n",
      "112\t1\t0.0\t2\t1\n",
      "128\t1\t0.0\t2\t1\n",
      "141\t1\t0.0\t2\t0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t1\n",
      "165\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "232\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "237\t3\t0.0\t2\t3\n",
      "238\t1\t0.0\t2\t1\n",
      "239\t2\t0.0\t2\t2\n",
      "240\t1\t0.0\t2\t0 0 1\n",
      "241\t2\t0.0\t2\t1 1\n",
      "247\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.B.25percent.8.3.19_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.B.25percent.8.3.19_56_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.B.25percent.8.3.19_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.B.25percent.8.3.19_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.08 s (16 us/read; 3.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            198,460\n",
      "  Read 1 with adapter:                     186 (0.1%)\n",
      "  Read 2 with adapter:                  12,174 (6.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       198,460 (100.0%)\n",
      "\n",
      "Total basepairs processed:    90,575,522 bp\n",
      "  Read 1:    44,431,401 bp\n",
      "  Read 2:    46,144,121 bp\n",
      "Total written (filtered):     90,429,086 bp (99.8%)\n",
      "  Read 1:    44,430,747 bp\n",
      "  Read 2:    45,998,339 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 186 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t182\t3100.9\t0\t182\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12174 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t101\t3100.9\t0\t101\n",
      "4\t49\t775.2\t0\t49\n",
      "5\t78\t193.8\t0\t78\n",
      "6\t219\t48.5\t0\t219\n",
      "7\t727\t12.1\t0\t727\n",
      "8\t1121\t3.0\t0\t1121\n",
      "9\t2997\t0.8\t0\t2991 6\n",
      "10\t2544\t0.2\t1\t2525 19\n",
      "11\t61\t0.0\t1\t57 4\n",
      "12\t87\t0.0\t1\t84 3\n",
      "13\t86\t0.0\t1\t82 4\n",
      "14\t192\t0.0\t1\t177 15\n",
      "15\t139\t0.0\t1\t133 6\n",
      "16\t458\t0.0\t1\t439 19\n",
      "17\t771\t0.0\t1\t739 32\n",
      "18\t517\t0.0\t1\t485 25 7\n",
      "19\t1719\t0.0\t1\t1043 670 6\n",
      "20\t9\t0.0\t2\t5 2 2\n",
      "21\t266\t0.0\t2\t0 264 2\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "23\t1\t0.0\t2\t1\n",
      "29\t1\t0.0\t2\t1\n",
      "30\t8\t0.0\t2\t7 1\n",
      "33\t1\t0.0\t2\t1\n",
      "35\t1\t0.0\t2\t1\n",
      "46\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t1\n",
      "82\t1\t0.0\t2\t1\n",
      "84\t1\t0.0\t2\t1\n",
      "86\t2\t0.0\t2\t2\n",
      "230\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.B.50percent.8.5.19_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.B.50percent.8.5.19_63_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.B.50percent.8.5.19_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.B.50percent.8.5.19_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.17 s (14 us/read; 4.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            293,299\n",
      "  Read 1 with adapter:                     255 (0.1%)\n",
      "  Read 2 with adapter:                  25,238 (8.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       293,299 (100.0%)\n",
      "\n",
      "Total basepairs processed:   134,015,990 bp\n",
      "  Read 1:    65,671,336 bp\n",
      "  Read 2:    68,344,654 bp\n",
      "Total written (filtered):    133,708,358 bp (99.8%)\n",
      "  Read 1:    65,669,921 bp\n",
      "  Read 2:    68,038,437 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 255 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t246\t4582.8\t0\t246\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t4\t0.0\t1\t4\n",
      "113\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "144\t1\t0.0\t1\t1\n",
      "151\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 25238 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t173\t4582.8\t0\t173\n",
      "4\t86\t1145.7\t0\t86\n",
      "5\t157\t286.4\t0\t157\n",
      "6\t376\t71.6\t0\t376\n",
      "7\t1207\t17.9\t0\t1207\n",
      "8\t1872\t4.5\t0\t1872\n",
      "9\t5332\t1.1\t0\t5321 11\n",
      "10\t6240\t0.3\t1\t6158 82\n",
      "11\t243\t0.1\t1\t238 5\n",
      "12\t336\t0.0\t1\t323 13\n",
      "13\t454\t0.0\t1\t429 25\n",
      "14\t738\t0.0\t1\t699 39\n",
      "15\t522\t0.0\t1\t478 44\n",
      "16\t1266\t0.0\t1\t1217 49\n",
      "17\t1656\t0.0\t1\t1581 75\n",
      "18\t992\t0.0\t1\t870 80 42\n",
      "19\t3146\t0.0\t1\t1694 1437 15\n",
      "20\t8\t0.0\t2\t2 3 3\n",
      "21\t380\t0.0\t2\t0 376 4\n",
      "22\t22\t0.0\t2\t0 1 21\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t0 1\n",
      "55\t1\t0.0\t2\t0 1\n",
      "56\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "67\t3\t0.0\t2\t3\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "70\t4\t0.0\t2\t2 1 1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "73\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "92\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "133\t1\t0.0\t2\t1\n",
      "235\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 1\n",
      "240\t1\t0.0\t2\t0 0 1\n",
      "241\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.50percent.8.5.19_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.50percent.8.5.19_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.50percent.8.5.19_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.50percent.8.5.19_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.68 s (13 us/read; 4.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            272,988\n",
      "  Read 1 with adapter:                     246 (0.1%)\n",
      "  Read 2 with adapter:                  24,870 (9.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       272,988 (100.0%)\n",
      "\n",
      "Total basepairs processed:   124,751,339 bp\n",
      "  Read 1:    61,107,155 bp\n",
      "  Read 2:    63,644,184 bp\n",
      "Total written (filtered):    124,464,821 bp (99.8%)\n",
      "  Read 1:    61,105,697 bp\n",
      "  Read 2:    63,359,124 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 246 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t238\t4265.4\t0\t238\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "113\t2\t0.0\t1\t2\n",
      "195\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 24870 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t234\t4265.4\t0\t234\n",
      "4\t119\t1066.4\t0\t119\n",
      "5\t190\t266.6\t0\t190\n",
      "6\t518\t66.6\t0\t518\n",
      "7\t1659\t16.7\t0\t1659\n",
      "8\t2423\t4.2\t0\t2423\n",
      "9\t6540\t1.0\t0\t6528 12\n",
      "10\t5439\t0.3\t1\t5374 65\n",
      "11\t141\t0.1\t1\t134 7\n",
      "12\t218\t0.0\t1\t216 2\n",
      "13\t244\t0.0\t1\t235 9\n",
      "14\t422\t0.0\t1\t401 21\n",
      "15\t334\t0.0\t1\t323 11\n",
      "16\t964\t0.0\t1\t923 41\n",
      "17\t1443\t0.0\t1\t1394 49\n",
      "18\t881\t0.0\t1\t781 65 35\n",
      "19\t2700\t0.0\t1\t1505 1183 12\n",
      "20\t11\t0.0\t2\t1 6 4\n",
      "21\t351\t0.0\t2\t0 346 5\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "30\t1\t0.0\t2\t1\n",
      "47\t1\t0.0\t2\t1\n",
      "54\t2\t0.0\t2\t2\n",
      "55\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "74\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "93\t1\t0.0\t2\t0 1\n",
      "98\t1\t0.0\t2\t1\n",
      "186\t2\t0.0\t2\t2\n",
      "215\t3\t0.0\t2\t2 1\n",
      "237\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.5percent.8.4.19_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.5percent.8.4.19_52_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.5percent.8.4.19_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.5percent.8.4.19_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.10 s (14 us/read; 4.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            288,884\n",
      "  Read 1 with adapter:                     282 (0.1%)\n",
      "  Read 2 with adapter:                  19,696 (6.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       288,884 (100.0%)\n",
      "\n",
      "Total basepairs processed:   131,860,658 bp\n",
      "  Read 1:    64,660,465 bp\n",
      "  Read 2:    67,200,193 bp\n",
      "Total written (filtered):    131,617,253 bp (99.8%)\n",
      "  Read 1:    64,659,077 bp\n",
      "  Read 2:    66,958,176 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 282 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t275\t4513.8\t0\t275\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t0 1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 19696 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t158\t4513.8\t0\t158\n",
      "4\t89\t1128.5\t0\t89\n",
      "5\t98\t282.1\t0\t98\n",
      "6\t299\t70.5\t0\t299\n",
      "7\t870\t17.6\t0\t870\n",
      "8\t1384\t4.4\t0\t1384\n",
      "9\t4373\t1.1\t0\t4364 9\n",
      "10\t4672\t0.3\t1\t4607 65\n",
      "11\t124\t0.1\t1\t119 5\n",
      "12\t191\t0.0\t1\t187 4\n",
      "13\t236\t0.0\t1\t222 14\n",
      "14\t460\t0.0\t1\t438 22\n",
      "15\t314\t0.0\t1\t298 16\n",
      "16\t940\t0.0\t1\t895 45\n",
      "17\t1424\t0.0\t1\t1368 56\n",
      "18\t801\t0.0\t1\t716 62 23\n",
      "19\t2837\t0.0\t1\t1654 1166 17\n",
      "20\t5\t0.0\t2\t0 4 1\n",
      "21\t381\t0.0\t2\t0 377 4\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "30\t1\t0.0\t2\t1\n",
      "31\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "68\t2\t0.0\t2\t2\n",
      "80\t1\t0.0\t2\t1\n",
      "81\t1\t0.0\t2\t1\n",
      "92\t1\t0.0\t2\t0 1\n",
      "119\t1\t0.0\t2\t0 1\n",
      "165\t1\t0.0\t2\t0 1\n",
      "174\t2\t0.0\t2\t0 2\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.75percent.8.7.19._24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.75percent.8.7.19._24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.75percent.8.7.19._24_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.75percent.8.7.19._24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.35 s (16 us/read; 3.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            273,978\n",
      "  Read 1 with adapter:                     366 (0.1%)\n",
      "  Read 2 with adapter:                  25,731 (9.4%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       273,977 (100.0%)\n",
      "\n",
      "Total basepairs processed:   125,147,427 bp\n",
      "  Read 1:    61,299,074 bp\n",
      "  Read 2:    63,848,353 bp\n",
      "Total written (filtered):    124,839,901 bp (99.8%)\n",
      "  Read 1:    61,297,324 bp\n",
      "  Read 2:    63,542,577 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 366 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t357\t4280.9\t0\t357\n",
      "13\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t2\t0.0\t1\t1 1\n",
      "62\t1\t0.0\t1\t1\n",
      "221\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 25731 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t181\t4280.9\t0\t181\n",
      "4\t84\t1070.2\t0\t84\n",
      "5\t161\t267.6\t0\t161\n",
      "6\t347\t66.9\t0\t347\n",
      "7\t1346\t16.7\t0\t1346\n",
      "8\t1969\t4.2\t0\t1969\n",
      "9\t5893\t1.0\t0\t5882 11\n",
      "10\t6698\t0.3\t1\t6615 83\n",
      "11\t242\t0.1\t1\t234 8\n",
      "12\t304\t0.0\t1\t302 2\n",
      "13\t480\t0.0\t1\t461 19\n",
      "14\t730\t0.0\t1\t692 38\n",
      "15\t429\t0.0\t1\t410 19\n",
      "16\t1222\t0.0\t1\t1173 49\n",
      "17\t1483\t0.0\t1\t1410 73\n",
      "18\t893\t0.0\t1\t777 71 45\n",
      "19\t2837\t0.0\t1\t1566 1252 19\n",
      "20\t6\t0.0\t2\t0 5 1\n",
      "21\t373\t0.0\t2\t0 370 3\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "30\t1\t0.0\t2\t1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "48\t3\t0.0\t2\t2 0 1\n",
      "52\t2\t0.0\t2\t1 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t2\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 1\n",
      "86\t1\t0.0\t2\t1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t1\n",
      "106\t1\t0.0\t2\t1\n",
      "112\t1\t0.0\t2\t1\n",
      "114\t1\t0.0\t2\t1\n",
      "122\t2\t0.0\t2\t2\n",
      "162\t1\t0.0\t2\t1\n",
      "165\t1\t0.0\t2\t1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t1 0 1\n",
      "226\t1\t0.0\t2\t1\n",
      "235\t1\t0.0\t2\t1\n",
      "237\t2\t0.0\t2\t2\n",
      "241\t1\t0.0\t2\t1\n",
      "242\t1\t0.0\t2\t0 1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.D.NT.5percent.9.16.19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.D.NT.5percent.9.16.19_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.D.NT.5percent.9.16.19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.D.NT.5percent.9.16.19_1_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.E.50percent.9.7.19_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.E.50percent.9.7.19_69_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.E.50percent.9.7.19_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.E.50percent.9.7.19_69_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.E.75percent.9.8.19._25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.E.75percent.9.8.19._25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.E.75percent.9.8.19._25_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.E.75percent.9.8.19._25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.G.25percent.9.17.19_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.17.G.25percent.9.17.19_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.G.25percent.9.17.19_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.17.G.25percent.9.17.19_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.18.G.75percent.8.12.19._58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.18.G.75percent.8.12.19._58_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.18.G.75percent.8.12.19._58_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.18.G.75percent.8.12.19._58_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.A.25percent.8.21.19_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.A.25percent.8.21.19_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.A.25percent.8.21.19_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.A.25percent.8.21.19_29_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.A.5percent.8.20.19_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.A.5percent.8.20.19_68_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.A.5percent.8.20.19_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.A.5percent.8.20.19_68_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.B.25percent.8.3.19_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.B.25percent.8.3.19_56_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.B.25percent.8.3.19_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.B.25percent.8.3.19_56_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.B.50percent.8.5.19_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.B.50percent.8.5.19_63_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.B.50percent.8.5.19_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.B.50percent.8.5.19_63_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.50percent.8.5.19_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.50percent.8.5.19_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.50percent.8.5.19_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.50percent.8.5.19_2_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.5percent.8.4.19_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.5percent.8.4.19_52_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.5percent.8.4.19_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.5percent.8.4.19_52_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.75percent.8.7.19._24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.19.E.NT.75percent.8.7.19._24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.75percent.8.7.19._24_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.19.E.NT.75percent.8.7.19._24_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.50percent.7.30.19_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.50percent.7.30.19_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.50percent.7.30.19_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.50percent.7.30.19_6_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.50percent.7.30.19_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.50percent.7.30.19_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.50percent.7.30.19_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.50percent.7.30.19_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.71 s (13 us/read; 4.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            287,600\n",
      "  Read 1 with adapter:                     292 (0.1%)\n",
      "  Read 2 with adapter:                  25,337 (8.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       287,600 (100.0%)\n",
      "\n",
      "Total basepairs processed:   131,441,757 bp\n",
      "  Read 1:    64,394,095 bp\n",
      "  Read 2:    67,047,662 bp\n",
      "Total written (filtered):    131,147,471 bp (99.8%)\n",
      "  Read 1:    64,392,633 bp\n",
      "  Read 2:    66,754,838 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 292 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t282\t4493.8\t0\t282\n",
      "19\t1\t0.0\t1\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "29\t2\t0.0\t1\t2\n",
      "88\t1\t0.0\t1\t1\n",
      "93\t1\t0.0\t1\t1\n",
      "124\t1\t0.0\t1\t1\n",
      "164\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 25337 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t221\t4493.8\t0\t221\n",
      "4\t91\t1123.4\t0\t91\n",
      "5\t212\t280.9\t0\t212\n",
      "6\t467\t70.2\t0\t467\n",
      "7\t1603\t17.6\t0\t1603\n",
      "8\t2400\t4.4\t0\t2400\n",
      "9\t6531\t1.1\t0\t6520 11\n",
      "10\t5685\t0.3\t1\t5607 78\n",
      "11\t142\t0.1\t1\t134 8\n",
      "12\t178\t0.0\t1\t170 8\n",
      "13\t223\t0.0\t1\t216 7\n",
      "14\t400\t0.0\t1\t382 18\n",
      "15\t368\t0.0\t1\t350 18\n",
      "16\t1016\t0.0\t1\t976 40\n",
      "17\t1463\t0.0\t1\t1409 54\n",
      "18\t944\t0.0\t1\t841 75 28\n",
      "19\t2984\t0.0\t1\t1708 1252 24\n",
      "20\t9\t0.0\t2\t3 2 4\n",
      "21\t364\t0.0\t2\t0 360 4\n",
      "22\t26\t0.0\t2\t0 1 25\n",
      "49\t1\t0.0\t2\t1\n",
      "51\t2\t0.0\t2\t0 1 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t1\n",
      "138\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.5percent.7.29.19_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.5percent.7.29.19_61_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.5percent.7.29.19_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.5percent.7.29.19_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.50 s (14 us/read; 4.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            311,071\n",
      "  Read 1 with adapter:                     244 (0.1%)\n",
      "  Read 2 with adapter:                  18,531 (6.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       311,071 (100.0%)\n",
      "\n",
      "Total basepairs processed:   141,902,842 bp\n",
      "  Read 1:    69,618,377 bp\n",
      "  Read 2:    72,284,465 bp\n",
      "Total written (filtered):    141,662,443 bp (99.8%)\n",
      "  Read 1:    69,616,522 bp\n",
      "  Read 2:    72,045,921 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 244 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t228\t4860.5\t0\t228\n",
      "10\t1\t0.3\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t2 1\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t1\n",
      "84\t1\t0.0\t1\t1\n",
      "140\t2\t0.0\t1\t2\n",
      "141\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18531 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t103\t4860.5\t0\t103\n",
      "4\t64\t1215.1\t0\t64\n",
      "5\t87\t303.8\t0\t87\n",
      "6\t227\t75.9\t0\t227\n",
      "7\t688\t19.0\t0\t688\n",
      "8\t1149\t4.7\t0\t1149\n",
      "9\t3667\t1.2\t0\t3658 9\n",
      "10\t4022\t0.3\t1\t3983 39\n",
      "11\t148\t0.1\t1\t146 2\n",
      "12\t225\t0.0\t1\t225\n",
      "13\t292\t0.0\t1\t281 11\n",
      "14\t459\t0.0\t1\t422 37\n",
      "15\t352\t0.0\t1\t336 16\n",
      "16\t959\t0.0\t1\t924 35\n",
      "17\t1560\t0.0\t1\t1499 61\n",
      "18\t1015\t0.0\t1\t892 84 39\n",
      "19\t3061\t0.0\t1\t1789 1256 16\n",
      "20\t9\t0.0\t2\t1 8\n",
      "21\t413\t0.0\t2\t0 404 9\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "52\t1\t0.0\t2\t0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "63\t2\t0.0\t2\t2\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t2\t0.0\t2\t1 0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t3 1\n",
      "99\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.75percent.7.31.19._32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.75percent.7.31.19._32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.75percent.7.31.19._32_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.75percent.7.31.19._32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.83 s (14 us/read; 4.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            283,110\n",
      "  Read 1 with adapter:                     330 (0.1%)\n",
      "  Read 2 with adapter:                  25,640 (9.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       283,110 (100.0%)\n",
      "\n",
      "Total basepairs processed:   129,382,004 bp\n",
      "  Read 1:    63,384,879 bp\n",
      "  Read 2:    65,997,125 bp\n",
      "Total written (filtered):    129,073,964 bp (99.8%)\n",
      "  Read 1:    63,383,122 bp\n",
      "  Read 2:    65,690,842 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 330 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t319\t4423.6\t0\t319\n",
      "22\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "78\t1\t0.0\t1\t0 1\n",
      "82\t1\t0.0\t1\t1\n",
      "88\t2\t0.0\t1\t1 1\n",
      "92\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t0 1\n",
      "152\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 25640 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t184\t4423.6\t0\t184\n",
      "4\t82\t1105.9\t0\t82\n",
      "5\t170\t276.5\t0\t170\n",
      "6\t355\t69.1\t0\t355\n",
      "7\t1230\t17.3\t0\t1230\n",
      "8\t1984\t4.3\t0\t1984\n",
      "9\t5484\t1.1\t0\t5473 11\n",
      "10\t6678\t0.3\t1\t6611 67\n",
      "11\t234\t0.1\t1\t223 11\n",
      "12\t321\t0.0\t1\t313 8\n",
      "13\t426\t0.0\t1\t413 13\n",
      "14\t731\t0.0\t1\t693 38\n",
      "15\t524\t0.0\t1\t493 31\n",
      "16\t1234\t0.0\t1\t1189 45\n",
      "17\t1573\t0.0\t1\t1514 59\n",
      "18\t1023\t0.0\t1\t900 70 53\n",
      "19\t3007\t0.0\t1\t1649 1340 18\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t363\t0.0\t2\t0 359 4\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "46\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "68\t2\t0.0\t2\t2\n",
      "69\t1\t0.0\t2\t1\n",
      "71\t3\t0.0\t2\t2 0 1\n",
      "79\t3\t0.0\t2\t2 1\n",
      "83\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "173\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.F.25percent.8.1.19_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.F.25percent.8.1.19_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.F.25percent.8.1.19_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.F.25percent.8.1.19_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.72 s (14 us/read; 4.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            264,825\n",
      "  Read 1 with adapter:                     291 (0.1%)\n",
      "  Read 2 with adapter:                  26,355 (10.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       264,825 (100.0%)\n",
      "\n",
      "Total basepairs processed:   121,082,299 bp\n",
      "  Read 1:    59,290,555 bp\n",
      "  Read 2:    61,791,744 bp\n",
      "Total written (filtered):    120,783,857 bp (99.8%)\n",
      "  Read 1:    59,289,588 bp\n",
      "  Read 2:    61,494,269 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 291 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t287\t4137.9\t0\t287\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 26355 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t246\t4137.9\t0\t246\n",
      "4\t132\t1034.5\t0\t132\n",
      "5\t228\t258.6\t0\t228\n",
      "6\t614\t64.7\t0\t614\n",
      "7\t2015\t16.2\t0\t2015\n",
      "8\t2761\t4.0\t0\t2761\n",
      "9\t7118\t1.0\t0\t7108 10\n",
      "10\t5454\t0.3\t1\t5393 61\n",
      "11\t158\t0.1\t1\t155 3\n",
      "12\t209\t0.0\t1\t208 1\n",
      "13\t339\t0.0\t1\t324 15\n",
      "14\t510\t0.0\t1\t486 24\n",
      "15\t349\t0.0\t1\t328 21\n",
      "16\t979\t0.0\t1\t947 32\n",
      "17\t1369\t0.0\t1\t1323 46\n",
      "18\t884\t0.0\t1\t811 48 25\n",
      "19\t2623\t0.0\t1\t1466 1146 11\n",
      "20\t5\t0.0\t2\t0 4 1\n",
      "21\t320\t0.0\t2\t0 315 5\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "30\t1\t0.0\t2\t1\n",
      "46\t1\t0.0\t2\t0 1\n",
      "49\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t0 1\n",
      "58\t1\t0.0\t2\t0 1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t2\t0.0\t2\t2\n",
      "64\t1\t0.0\t2\t0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t1\n",
      "74\t2\t0.0\t2\t2\n",
      "76\t1\t0.0\t2\t0 1\n",
      "79\t2\t0.0\t2\t2\n",
      "85\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t1\n",
      "171\t1\t0.0\t2\t1\n",
      "197\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "222\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "239\t2\t0.0\t2\t1 1\n",
      "241\t1\t0.0\t2\t1\n",
      "242\t2\t0.0\t2\t2\n",
      "244\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.F.75percent.8.3.19._3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.F.75percent.8.3.19._3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.F.75percent.8.3.19._3_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.F.75percent.8.3.19._3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.30 s (15 us/read; 3.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            281,937\n",
      "  Read 1 with adapter:                     203 (0.1%)\n",
      "  Read 2 with adapter:                  23,637 (8.4%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       281,936 (100.0%)\n",
      "\n",
      "Total basepairs processed:   128,798,340 bp\n",
      "  Read 1:    63,123,349 bp\n",
      "  Read 2:    65,674,991 bp\n",
      "Total written (filtered):    128,504,224 bp (99.8%)\n",
      "  Read 1:    63,121,828 bp\n",
      "  Read 2:    65,382,396 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 203 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t193\t4405.3\t0\t193\n",
      "7\t1\t17.2\t0\t1\n",
      "9\t1\t1.1\t0\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "87\t1\t0.0\t1\t1\n",
      "153\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 23637 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t183\t4405.3\t0\t183\n",
      "4\t63\t1101.3\t0\t63\n",
      "5\t125\t275.3\t0\t125\n",
      "6\t255\t68.8\t0\t255\n",
      "7\t1096\t17.2\t0\t1096\n",
      "8\t1609\t4.3\t0\t1609\n",
      "9\t4752\t1.1\t0\t4737 15\n",
      "10\t5480\t0.3\t1\t5422 58\n",
      "11\t296\t0.1\t1\t291 5\n",
      "12\t380\t0.0\t1\t374 6\n",
      "13\t525\t0.0\t1\t509 16\n",
      "14\t790\t0.0\t1\t755 35\n",
      "15\t564\t0.0\t1\t527 37\n",
      "16\t1313\t0.0\t1\t1255 58\n",
      "17\t1670\t0.0\t1\t1596 74\n",
      "18\t989\t0.0\t1\t848 85 56\n",
      "19\t3136\t0.0\t1\t1721 1393 22\n",
      "20\t6\t0.0\t2\t0 5 1\n",
      "21\t365\t0.0\t2\t0 359 6\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "57\t2\t0.0\t2\t1 0 1\n",
      "60\t2\t0.0\t2\t1 0 1\n",
      "64\t1\t0.0\t2\t0 1\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t3\n",
      "88\t2\t0.0\t2\t1 0 1\n",
      "93\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t1\n",
      "100\t2\t0.0\t2\t2\n",
      "115\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t1\n",
      "222\t1\t0.0\t2\t0 0 1\n",
      "238\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.20.B.25percent.9.9.19_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.20.B.25percent.9.9.19_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.20.B.25percent.9.9.19_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.20.B.25percent.9.9.19_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.85 s (13 us/read; 4.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            296,597\n",
      "  Read 1 with adapter:                     347 (0.1%)\n",
      "  Read 2 with adapter:                  19,385 (6.5%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       296,596 (100.0%)\n",
      "\n",
      "Total basepairs processed:   135,288,986 bp\n",
      "  Read 1:    66,349,504 bp\n",
      "  Read 2:    68,939,482 bp\n",
      "Total written (filtered):    135,041,531 bp (99.8%)\n",
      "  Read 1:    66,347,290 bp\n",
      "  Read 2:    68,694,241 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 347 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t335\t4634.3\t0\t335\n",
      "22\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "29\t3\t0.0\t1\t3\n",
      "132\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 19385 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t143\t4634.3\t0\t143\n",
      "4\t51\t1158.6\t0\t51\n",
      "5\t81\t289.6\t0\t81\n",
      "6\t238\t72.4\t0\t238\n",
      "7\t741\t18.1\t0\t741\n",
      "8\t1247\t4.5\t0\t1247\n",
      "9\t3855\t1.1\t0\t3848 7\n",
      "10\t4601\t0.3\t1\t4549 52\n",
      "11\t178\t0.1\t1\t171 7\n",
      "12\t201\t0.0\t1\t195 6\n",
      "13\t307\t0.0\t1\t296 11\n",
      "14\t497\t0.0\t1\t473 24\n",
      "15\t398\t0.0\t1\t375 23\n",
      "16\t1095\t0.0\t1\t1059 36\n",
      "17\t1541\t0.0\t1\t1474 67\n",
      "18\t883\t0.0\t1\t763 92 28\n",
      "19\t2897\t0.0\t1\t1645 1234 18\n",
      "20\t17\t0.0\t2\t3 13 1\n",
      "21\t376\t0.0\t2\t0 371 5\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "34\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t5\t0.0\t2\t5\n",
      "133\t1\t0.0\t2\t1\n",
      "139\t1\t0.0\t2\t1\n",
      "140\t1\t0.0\t2\t1\n",
      "161\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t1\n",
      "241\t1\t0.0\t2\t1\n",
      "251\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.20.G.NT.50percent.9.20.19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.20.G.NT.50percent.9.20.19_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.20.G.NT.50percent.9.20.19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.20.G.NT.50percent.9.20.19_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.59 s (14 us/read; 4.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            331,841\n",
      "  Read 1 with adapter:                     309 (0.1%)\n",
      "  Read 2 with adapter:                  35,406 (10.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       331,841 (100.0%)\n",
      "\n",
      "Total basepairs processed:   151,771,843 bp\n",
      "  Read 1:    74,298,838 bp\n",
      "  Read 2:    77,473,005 bp\n",
      "Total written (filtered):    151,365,171 bp (99.7%)\n",
      "  Read 1:    74,297,347 bp\n",
      "  Read 2:    77,067,824 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 309 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t297\t5185.0\t0\t297\n",
      "6\t1\t81.0\t0\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t2\t0.0\t1\t2\n",
      "87\t1\t0.0\t1\t1\n",
      "99\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 35406 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t322\t5185.0\t0\t322\n",
      "4\t120\t1296.3\t0\t120\n",
      "5\t326\t324.1\t0\t326\n",
      "6\t554\t81.0\t0\t554\n",
      "7\t2123\t20.3\t0\t2123\n",
      "8\t3102\t5.1\t0\t3102\n",
      "9\t8634\t1.3\t0\t8620 14\n",
      "10\t9106\t0.3\t1\t9006 100\n",
      "11\t318\t0.1\t1\t305 13\n",
      "12\t370\t0.0\t1\t352 18\n",
      "13\t575\t0.0\t1\t558 17\n",
      "14\t795\t0.0\t1\t752 43\n",
      "15\t579\t0.0\t1\t544 35\n",
      "16\t1473\t0.0\t1\t1420 53\n",
      "17\t1863\t0.0\t1\t1777 86\n",
      "18\t1164\t0.0\t1\t1004 82 78\n",
      "19\t3500\t0.0\t1\t1867 1613 20\n",
      "20\t8\t0.0\t2\t2 4 2\n",
      "21\t429\t0.0\t2\t0 417 12\n",
      "22\t14\t0.0\t2\t0 1 13\n",
      "48\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t2\n",
      "55\t2\t0.0\t2\t1 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "65\t2\t0.0\t2\t2\n",
      "66\t2\t0.0\t2\t2\n",
      "70\t2\t0.0\t2\t1 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t2 2\n",
      "81\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 1\n",
      "147\t1\t0.0\t2\t1\n",
      "154\t2\t0.0\t2\t1 1\n",
      "161\t2\t0.0\t2\t1 0 1\n",
      "239\t1\t0.0\t2\t0 0 1\n",
      "242\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.B.25percent.8.11.19_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.B.25percent.8.11.19_59_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.B.25percent.8.11.19_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.B.25percent.8.11.19_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.26 s (16 us/read; 3.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            262,004\n",
      "  Read 1 with adapter:                     297 (0.1%)\n",
      "  Read 2 with adapter:                  25,928 (9.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       262,004 (100.0%)\n",
      "\n",
      "Total basepairs processed:   119,742,834 bp\n",
      "  Read 1:    58,612,067 bp\n",
      "  Read 2:    61,130,767 bp\n",
      "Total written (filtered):    119,439,279 bp (99.7%)\n",
      "  Read 1:    58,610,597 bp\n",
      "  Read 2:    60,828,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 297 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t288\t4093.8\t0\t288\n",
      "21\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "55\t1\t0.0\t1\t1\n",
      "173\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 25928 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t274\t4093.8\t0\t274\n",
      "4\t141\t1023.5\t0\t141\n",
      "5\t217\t255.9\t0\t217\n",
      "6\t542\t64.0\t0\t542\n",
      "7\t1806\t16.0\t0\t1806\n",
      "8\t2362\t4.0\t0\t2362\n",
      "9\t6267\t1.0\t0\t6256 11\n",
      "10\t5635\t0.2\t1\t5564 71\n",
      "11\t188\t0.1\t1\t186 2\n",
      "12\t297\t0.0\t1\t288 9\n",
      "13\t382\t0.0\t1\t371 11\n",
      "14\t568\t0.0\t1\t550 18\n",
      "15\t466\t0.0\t1\t433 33\n",
      "16\t1209\t0.0\t1\t1151 58\n",
      "17\t1480\t0.0\t1\t1409 71\n",
      "18\t893\t0.0\t1\t772 67 54\n",
      "19\t2796\t0.0\t1\t1582 1201 13\n",
      "20\t11\t0.0\t2\t1 7 3\n",
      "21\t336\t0.0\t2\t0 333 3\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "30\t1\t0.0\t2\t1\n",
      "46\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t1 0 1\n",
      "71\t3\t0.0\t2\t0 1 2\n",
      "72\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t2 1\n",
      "80\t2\t0.0\t2\t2\n",
      "82\t1\t0.0\t2\t1\n",
      "83\t1\t0.0\t2\t1\n",
      "86\t1\t0.0\t2\t1\n",
      "93\t1\t0.0\t2\t0 1\n",
      "95\t1\t0.0\t2\t1\n",
      "97\t1\t0.0\t2\t1\n",
      "103\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "142\t2\t0.0\t2\t2\n",
      "154\t1\t0.0\t2\t1\n",
      "172\t1\t0.0\t2\t1\n",
      "218\t2\t0.0\t2\t2\n",
      "222\t1\t0.0\t2\t0 1\n",
      "237\t1\t0.0\t2\t1\n",
      "239\t2\t0.0\t2\t2\n",
      "240\t2\t0.0\t2\t2\n",
      "241\t1\t0.0\t2\t1\n",
      "242\t1\t0.0\t2\t0 1\n",
      "247\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.B.75percent.8.13.19._64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.B.75percent.8.13.19._64_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.B.75percent.8.13.19._64_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.B.75percent.8.13.19._64_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.47 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            184,839\n",
      "  Read 1 with adapter:                     222 (0.1%)\n",
      "  Read 2 with adapter:                  15,290 (8.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       184,839 (100.0%)\n",
      "\n",
      "Total basepairs processed:    84,264,327 bp\n",
      "  Read 1:    41,282,634 bp\n",
      "  Read 2:    42,981,693 bp\n",
      "Total written (filtered):     84,070,323 bp (99.8%)\n",
      "  Read 1:    41,280,262 bp\n",
      "  Read 2:    42,790,061 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 222 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t189\t2888.1\t0\t189\n",
      "19\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t23\t0.0\t1\t22 1\n",
      "37\t1\t0.0\t1\t1\n",
      "107\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15290 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t119\t2888.1\t0\t119\n",
      "4\t42\t722.0\t0\t42\n",
      "5\t97\t180.5\t0\t97\n",
      "6\t194\t45.1\t0\t194\n",
      "7\t681\t11.3\t0\t681\n",
      "8\t962\t2.8\t0\t962\n",
      "9\t3006\t0.7\t0\t2999 7\n",
      "10\t3804\t0.2\t1\t3765 39\n",
      "11\t163\t0.0\t1\t157 6\n",
      "12\t207\t0.0\t1\t202 5\n",
      "13\t316\t0.0\t1\t301 15\n",
      "14\t472\t0.0\t1\t455 17\n",
      "15\t318\t0.0\t1\t299 19\n",
      "16\t870\t0.0\t1\t824 46\n",
      "17\t1026\t0.0\t1\t975 51\n",
      "18\t674\t0.0\t1\t571 61 42\n",
      "19\t2027\t0.0\t1\t1093 921 13\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t256\t0.0\t2\t0 254 2\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "30\t1\t0.0\t2\t1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t0 1\n",
      "66\t3\t0.0\t2\t3\n",
      "67\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t0 1\n",
      "71\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t3\n",
      "80\t2\t0.0\t2\t2\n",
      "82\t1\t0.0\t2\t1\n",
      "85\t1\t0.0\t2\t1\n",
      "92\t2\t0.0\t2\t2\n",
      "93\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "99\t2\t0.0\t2\t1 1\n",
      "100\t1\t0.0\t2\t0 1\n",
      "111\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 1\n",
      "119\t1\t0.0\t2\t1\n",
      "122\t1\t0.0\t2\t0 1\n",
      "130\t1\t0.0\t2\t1\n",
      "131\t1\t0.0\t2\t1\n",
      "146\t1\t0.0\t2\t1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t0 1\n",
      "190\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t1\n",
      "248\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.25percent.8.5.19_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.25percent.8.5.19_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.25percent.8.5.19_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.25percent.8.5.19_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.49 s (14 us/read; 4.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            256,889\n",
      "  Read 1 with adapter:                     196 (0.1%)\n",
      "  Read 2 with adapter:                  25,340 (9.9%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       256,888 (100.0%)\n",
      "\n",
      "Total basepairs processed:   117,465,582 bp\n",
      "  Read 1:    57,521,085 bp\n",
      "  Read 2:    59,944,497 bp\n",
      "Total written (filtered):    117,172,248 bp (99.8%)\n",
      "  Read 1:    57,519,941 bp\n",
      "  Read 2:    59,652,307 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 196 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t190\t4013.9\t0\t190\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t2\t0.0\t1\t2\n",
      "100\t1\t0.0\t1\t1\n",
      "141\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 25340 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t202\t4013.9\t0\t202\n",
      "4\t109\t1003.5\t0\t109\n",
      "5\t191\t250.9\t0\t191\n",
      "6\t541\t62.7\t0\t541\n",
      "7\t1772\t15.7\t0\t1772\n",
      "8\t2393\t3.9\t0\t2393\n",
      "9\t6565\t1.0\t0\t6551 14\n",
      "10\t5600\t0.2\t1\t5546 54\n",
      "11\t170\t0.1\t1\t165 5\n",
      "12\t213\t0.0\t1\t208 5\n",
      "13\t299\t0.0\t1\t293 6\n",
      "14\t508\t0.0\t1\t482 26\n",
      "15\t391\t0.0\t1\t363 28\n",
      "16\t1023\t0.0\t1\t987 36\n",
      "17\t1410\t0.0\t1\t1360 50\n",
      "18\t901\t0.0\t1\t799 75 27\n",
      "19\t2653\t0.0\t1\t1502 1137 14\n",
      "20\t4\t0.0\t2\t2 2\n",
      "21\t343\t0.0\t2\t0 335 8\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "54\t4\t0.0\t2\t4\n",
      "55\t3\t0.0\t2\t3\n",
      "57\t1\t0.0\t2\t1\n",
      "60\t2\t0.0\t2\t0 1 1\n",
      "61\t2\t0.0\t2\t1 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t0 1\n",
      "79\t2\t0.0\t2\t2\n",
      "84\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t0 1\n",
      "101\t2\t0.0\t2\t0 2\n",
      "104\t1\t0.0\t2\t1\n",
      "127\t1\t0.0\t2\t1\n",
      "140\t1\t0.0\t2\t1\n",
      "145\t1\t0.0\t2\t1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "238\t4\t0.0\t2\t4\n",
      "241\t1\t0.0\t2\t0 0 1\n",
      "245\t1\t0.0\t2\t0 0 1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.50percent.8.11.19_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.50percent.8.11.19_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.50percent.8.11.19_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.50percent.8.11.19_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.44 s (14 us/read; 4.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            308,661\n",
      "  Read 1 with adapter:                     421 (0.1%)\n",
      "  Read 2 with adapter:                  23,226 (7.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       308,661 (100.0%)\n",
      "\n",
      "Total basepairs processed:   140,973,718 bp\n",
      "  Read 1:    69,116,520 bp\n",
      "  Read 2:    71,857,198 bp\n",
      "Total written (filtered):    140,680,945 bp (99.8%)\n",
      "  Read 1:    69,113,894 bp\n",
      "  Read 2:    71,567,051 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 421 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t400\t4822.8\t0\t400\n",
      "20\t1\t0.0\t1\t1\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "26\t5\t0.0\t1\t5\n",
      "29\t5\t0.0\t1\t5\n",
      "30\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "150\t2\t0.0\t1\t2\n",
      "151\t2\t0.0\t1\t1 1\n",
      "200\t1\t0.0\t1\t0 1\n",
      "201\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 23226 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t139\t4822.8\t0\t139\n",
      "4\t90\t1205.7\t0\t90\n",
      "5\t126\t301.4\t0\t126\n",
      "6\t309\t75.4\t0\t309\n",
      "7\t955\t18.8\t0\t955\n",
      "8\t1553\t4.7\t0\t1553\n",
      "9\t4729\t1.2\t0\t4719 10\n",
      "10\t5320\t0.3\t1\t5258 62\n",
      "11\t236\t0.1\t1\t231 5\n",
      "12\t370\t0.0\t1\t363 7\n",
      "13\t473\t0.0\t1\t460 13\n",
      "14\t753\t0.0\t1\t708 45\n",
      "15\t472\t0.0\t1\t448 24\n",
      "16\t1257\t0.0\t1\t1206 51\n",
      "17\t1736\t0.0\t1\t1664 72\n",
      "18\t1054\t0.0\t1\t931 77 46\n",
      "19\t3185\t0.0\t1\t1849 1320 16\n",
      "20\t5\t0.0\t2\t1 4\n",
      "21\t414\t0.0\t2\t0 410 4\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "46\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "53\t2\t0.0\t2\t1 1\n",
      "57\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "63\t3\t0.0\t2\t2 1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "72\t1\t0.0\t2\t1\n",
      "73\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t2 1\n",
      "84\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "125\t1\t0.0\t2\t0 1\n",
      "141\t1\t0.0\t2\t1\n",
      "164\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.75percent.8.12.19._66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.75percent.8.12.19._66_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.75percent.8.12.19._66_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.75percent.8.12.19._66_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.21 s (14 us/read; 4.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            295,706\n",
      "  Read 1 with adapter:                     211 (0.1%)\n",
      "  Read 2 with adapter:                  21,218 (7.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       295,706 (100.0%)\n",
      "\n",
      "Total basepairs processed:   134,978,772 bp\n",
      "  Read 1:    66,191,654 bp\n",
      "  Read 2:    68,787,118 bp\n",
      "Total written (filtered):    134,704,023 bp (99.8%)\n",
      "  Read 1:    66,189,637 bp\n",
      "  Read 2:    68,514,386 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 211 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t200\t4620.4\t0\t200\n",
      "19\t2\t0.0\t1\t2\n",
      "21\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "206\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t3\t0.0\t1\t2 1\n",
      "223\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 21218 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t130\t4620.4\t0\t130\n",
      "4\t48\t1155.1\t0\t48\n",
      "5\t99\t288.8\t0\t99\n",
      "6\t252\t72.2\t0\t252\n",
      "7\t765\t18.0\t0\t765\n",
      "8\t1244\t4.5\t0\t1244\n",
      "9\t3809\t1.1\t0\t3802 7\n",
      "10\t5043\t0.3\t1\t4999 44\n",
      "11\t224\t0.1\t1\t218 6\n",
      "12\t332\t0.0\t1\t328 4\n",
      "13\t438\t0.0\t1\t417 21\n",
      "14\t724\t0.0\t1\t674 50\n",
      "15\t508\t0.0\t1\t472 36\n",
      "16\t1278\t0.0\t1\t1235 43\n",
      "17\t1681\t0.0\t1\t1626 55\n",
      "18\t982\t0.0\t1\t875 71 36\n",
      "19\t3201\t0.0\t1\t1826 1353 22\n",
      "20\t8\t0.0\t2\t0 6 2\n",
      "21\t399\t0.0\t2\t0 395 4\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t1 1\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t2\t0.0\t2\t1 0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t2\n",
      "79\t3\t0.0\t2\t1 2\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 1\n",
      "103\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "128\t2\t0.0\t2\t1 1\n",
      "186\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t1\n",
      "241\t1\t0.0\t2\t1\n",
      "242\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.G.25percent.8.10.19_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.G.25percent.8.10.19_55_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.G.25percent.8.10.19_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.G.25percent.8.10.19_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.71 s (16 us/read; 3.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            231,925\n",
      "  Read 1 with adapter:                     233 (0.1%)\n",
      "  Read 2 with adapter:                  18,461 (8.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       231,925 (100.0%)\n",
      "\n",
      "Total basepairs processed:   105,916,019 bp\n",
      "  Read 1:    51,914,789 bp\n",
      "  Read 2:    54,001,230 bp\n",
      "Total written (filtered):    105,698,714 bp (99.8%)\n",
      "  Read 1:    51,913,082 bp\n",
      "  Read 2:    53,785,632 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 233 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t224\t3623.8\t0\t224\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "101\t3\t0.0\t1\t3\n",
      "123\t1\t0.0\t1\t1\n",
      "166\t1\t0.0\t1\t0 1\n",
      "176\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18461 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t189\t3623.8\t0\t189\n",
      "4\t110\t906.0\t0\t110\n",
      "5\t148\t226.5\t0\t148\n",
      "6\t382\t56.6\t0\t382\n",
      "7\t1168\t14.2\t0\t1168\n",
      "8\t1811\t3.5\t0\t1811\n",
      "9\t4614\t0.9\t0\t4601 13\n",
      "10\t4043\t0.2\t1\t3998 45\n",
      "11\t88\t0.1\t1\t85 3\n",
      "12\t104\t0.0\t1\t103 1\n",
      "13\t135\t0.0\t1\t125 10\n",
      "14\t275\t0.0\t1\t259 16\n",
      "15\t237\t0.0\t1\t224 13\n",
      "16\t671\t0.0\t1\t649 22\n",
      "17\t1120\t0.0\t1\t1082 38\n",
      "18\t710\t0.0\t1\t628 58 24\n",
      "19\t2299\t0.0\t1\t1375 910 14\n",
      "20\t9\t0.0\t2\t2 3 4\n",
      "21\t312\t0.0\t2\t0 307 5\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "30\t3\t0.0\t2\t3\n",
      "58\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 1\n",
      "175\t1\t0.0\t2\t0 1\n",
      "178\t1\t0.0\t2\t0 1\n",
      "216\t3\t0.0\t2\t2 0 1\n",
      "217\t2\t0.0\t2\t2\n",
      "226\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.5percent.7.29.19_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.5percent.7.29.19_61_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.5percent.7.29.19_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.5percent.7.29.19_61_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.75percent.7.31.19._32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.D.75percent.7.31.19._32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.75percent.7.31.19._32_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.D.75percent.7.31.19._32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.F.25percent.8.1.19_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.F.25percent.8.1.19_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.F.25percent.8.1.19_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.F.25percent.8.1.19_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.F.75percent.8.3.19._3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.2.F.75percent.8.3.19._3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.F.75percent.8.3.19._3_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.2.F.75percent.8.3.19._3_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.20.B.25percent.9.9.19_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.20.B.25percent.9.9.19_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.20.B.25percent.9.9.19_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.20.B.25percent.9.9.19_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.20.G.NT.50percent.9.20.19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.20.G.NT.50percent.9.20.19_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.20.G.NT.50percent.9.20.19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.20.G.NT.50percent.9.20.19_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.B.25percent.8.11.19_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.B.25percent.8.11.19_59_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.B.25percent.8.11.19_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.B.25percent.8.11.19_59_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.B.75percent.8.13.19._64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.B.75percent.8.13.19._64_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.B.75percent.8.13.19._64_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.B.75percent.8.13.19._64_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.25percent.8.5.19_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.25percent.8.5.19_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.25percent.8.5.19_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.25percent.8.5.19_26_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.50percent.8.11.19_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.50percent.8.11.19_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.50percent.8.11.19_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.50percent.8.11.19_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.75percent.8.12.19._66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.F.75percent.8.12.19._66_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.75percent.8.12.19._66_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.F.75percent.8.12.19._66_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.G.25percent.8.10.19_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.G.25percent.8.10.19_55_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.G.25percent.8.10.19_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.G.25percent.8.10.19_55_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.G.75percent.8.12.19._27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.G.75percent.8.12.19._27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.G.75percent.8.12.19._27_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.G.75percent.8.12.19._27_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.G.75percent.8.12.19._27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.4.G.75percent.8.12.19._27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.G.75percent.8.12.19._27_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.4.G.75percent.8.12.19._27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.19 s (14 us/read; 4.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            308,230\n",
      "  Read 1 with adapter:                     288 (0.1%)\n",
      "  Read 2 with adapter:                  25,864 (8.4%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       308,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:   140,838,289 bp\n",
      "  Read 1:    69,017,587 bp\n",
      "  Read 2:    71,820,702 bp\n",
      "Total written (filtered):    140,522,839 bp (99.8%)\n",
      "  Read 1:    69,016,199 bp\n",
      "  Read 2:    71,506,640 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 288 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t286\t4816.1\t0\t286\n",
      "153\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 25864 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t181\t4816.1\t0\t181\n",
      "4\t68\t1204.0\t0\t68\n",
      "5\t148\t301.0\t0\t148\n",
      "6\t372\t75.3\t0\t372\n",
      "7\t1226\t18.8\t0\t1226\n",
      "8\t1937\t4.7\t0\t1937\n",
      "9\t5720\t1.2\t0\t5709 11\n",
      "10\t6269\t0.3\t1\t6189 80\n",
      "11\t205\t0.1\t1\t199 6\n",
      "12\t295\t0.0\t1\t287 8\n",
      "13\t426\t0.0\t1\t407 19\n",
      "14\t698\t0.0\t1\t660 38\n",
      "15\t526\t0.0\t1\t490 36\n",
      "16\t1374\t0.0\t1\t1310 64\n",
      "17\t1687\t0.0\t1\t1625 62\n",
      "18\t1004\t0.0\t1\t897 66 41\n",
      "19\t3268\t0.0\t1\t1820 1442 6\n",
      "20\t6\t0.0\t2\t0 4 2\n",
      "21\t400\t0.0\t2\t0 393 7\n",
      "22\t23\t0.0\t2\t0 1 22\n",
      "48\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "58\t3\t0.0\t2\t2 0 1\n",
      "62\t2\t0.0\t2\t2\n",
      "65\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t2\t0.0\t2\t0 1 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t6\t0.0\t2\t3 3\n",
      "93\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t0 0 1\n",
      "251\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.25percent.9.3.19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.25percent.9.3.19_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.25percent.9.3.19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.25percent.9.3.19_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.49 s (17 us/read; 3.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            145,823\n",
      "  Read 1 with adapter:                     138 (0.1%)\n",
      "  Read 2 with adapter:                  10,317 (7.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       145,823 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,338,707 bp\n",
      "  Read 1:    32,518,884 bp\n",
      "  Read 2:    33,819,823 bp\n",
      "Total written (filtered):     66,209,473 bp (99.8%)\n",
      "  Read 1:    32,517,985 bp\n",
      "  Read 2:    33,691,488 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 138 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t128\t2278.5\t0\t128\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t0 1\n",
      "89\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10317 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t84\t2278.5\t0\t84\n",
      "4\t44\t569.6\t0\t44\n",
      "5\t44\t142.4\t0\t44\n",
      "6\t144\t35.6\t0\t144\n",
      "7\t439\t8.9\t0\t439\n",
      "8\t708\t2.2\t0\t708\n",
      "9\t2236\t0.6\t0\t2232 4\n",
      "10\t2383\t0.1\t1\t2342 41\n",
      "11\t79\t0.0\t1\t75 4\n",
      "12\t125\t0.0\t1\t123 2\n",
      "13\t138\t0.0\t1\t135 3\n",
      "14\t248\t0.0\t1\t232 16\n",
      "15\t215\t0.0\t1\t207 8\n",
      "16\t515\t0.0\t1\t498 17\n",
      "17\t700\t0.0\t1\t669 31\n",
      "18\t476\t0.0\t1\t436 28 12\n",
      "19\t1517\t0.0\t1\t911 600 6\n",
      "20\t8\t0.0\t2\t2 5 1\n",
      "21\t187\t0.0\t2\t0 185 2\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "55\t1\t0.0\t2\t0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "79\t2\t0.0\t2\t2\n",
      "91\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "116\t1\t0.0\t2\t1\n",
      "122\t1\t0.0\t2\t1\n",
      "128\t1\t0.0\t2\t0 1\n",
      "183\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.50percent.9.5.19_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.50percent.9.5.19_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.50percent.9.5.19_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.50percent.9.5.19_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.76 s (13 us/read; 4.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            291,799\n",
      "  Read 1 with adapter:                     242 (0.1%)\n",
      "  Read 2 with adapter:                  20,581 (7.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       291,799 (100.0%)\n",
      "\n",
      "Total basepairs processed:   133,128,847 bp\n",
      "  Read 1:    65,277,110 bp\n",
      "  Read 2:    67,851,737 bp\n",
      "Total written (filtered):    132,866,903 bp (99.8%)\n",
      "  Read 1:    65,276,054 bp\n",
      "  Read 2:    67,590,849 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 242 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t239\t4559.4\t0\t239\n",
      "29\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 20581 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t152\t4559.4\t0\t152\n",
      "4\t42\t1139.8\t0\t42\n",
      "5\t98\t285.0\t0\t98\n",
      "6\t192\t71.2\t0\t192\n",
      "7\t702\t17.8\t0\t702\n",
      "8\t1140\t4.5\t0\t1140\n",
      "9\t3573\t1.1\t0\t3558 15\n",
      "10\t5758\t0.3\t1\t5681 77\n",
      "11\t207\t0.1\t1\t198 9\n",
      "12\t200\t0.0\t1\t196 4\n",
      "13\t271\t0.0\t1\t263 8\n",
      "14\t600\t0.0\t1\t576 24\n",
      "15\t442\t0.0\t1\t415 27\n",
      "16\t1223\t0.0\t1\t1174 49\n",
      "17\t1580\t0.0\t1\t1518 62\n",
      "18\t929\t0.0\t1\t820 70 39\n",
      "19\t3044\t0.0\t1\t1727 1304 13\n",
      "20\t5\t0.0\t2\t0 4 1\n",
      "21\t374\t0.0\t2\t0 364 10\n",
      "22\t29\t0.0\t2\t0 1 28\n",
      "30\t3\t0.0\t2\t3\n",
      "47\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "74\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "151\t1\t0.0\t2\t0 1\n",
      "186\t1\t0.0\t2\t0 1\n",
      "196\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t0 0 1\n",
      "226\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.75percent.9.9.19._46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.75percent.9.9.19._46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.75percent.9.9.19._46_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.75percent.9.9.19._46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.30 s (14 us/read; 4.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            165,530\n",
      "  Read 1 with adapter:                     177 (0.1%)\n",
      "  Read 2 with adapter:                  11,952 (7.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       165,530 (100.0%)\n",
      "\n",
      "Total basepairs processed:    75,212,307 bp\n",
      "  Read 1:    36,863,304 bp\n",
      "  Read 2:    38,349,003 bp\n",
      "Total written (filtered):     75,058,361 bp (99.8%)\n",
      "  Read 1:    36,861,042 bp\n",
      "  Read 2:    38,197,319 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 177 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t168\t2586.4\t0\t168\n",
      "27\t1\t0.0\t1\t1\n",
      "208\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t3\t0.0\t1\t1 2\n",
      "219\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11952 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t87\t2586.4\t0\t87\n",
      "4\t20\t646.6\t0\t20\n",
      "5\t56\t161.7\t0\t56\n",
      "6\t118\t40.4\t0\t118\n",
      "7\t410\t10.1\t0\t410\n",
      "8\t663\t2.5\t0\t663\n",
      "9\t2161\t0.6\t0\t2154 7\n",
      "10\t3098\t0.2\t1\t3063 35\n",
      "11\t127\t0.0\t1\t122 5\n",
      "12\t184\t0.0\t1\t175 9\n",
      "13\t265\t0.0\t1\t256 9\n",
      "14\t361\t0.0\t1\t345 16\n",
      "15\t260\t0.0\t1\t246 14\n",
      "16\t704\t0.0\t1\t683 21\n",
      "17\t913\t0.0\t1\t876 37\n",
      "18\t560\t0.0\t1\t498 44 18\n",
      "19\t1719\t0.0\t1\t966 745 8\n",
      "20\t2\t0.0\t2\t0 2\n",
      "21\t223\t0.0\t2\t0 218 5\n",
      "22\t10\t0.0\t2\t0 1 9\n",
      "30\t1\t0.0\t2\t1\n",
      "46\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "127\t1\t0.0\t2\t1\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "193\t1\t0.0\t2\t1\n",
      "205\t1\t0.0\t2\t0 1\n",
      "218\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.E.final.9.25.19_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.E.final.9.25.19_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.E.final.9.25.19_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.E.final.9.25.19_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.00 s (19 us/read; 3.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            211,440\n",
      "  Read 1 with adapter:                     264 (0.1%)\n",
      "  Read 2 with adapter:                   8,001 (3.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       211,440 (100.0%)\n",
      "\n",
      "Total basepairs processed:    96,287,357 bp\n",
      "  Read 1:    47,286,491 bp\n",
      "  Read 2:    49,000,866 bp\n",
      "Total written (filtered):     96,165,476 bp (99.9%)\n",
      "  Read 1:    47,284,977 bp\n",
      "  Read 2:    48,880,499 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 264 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t257\t3303.8\t0\t257\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t0 1\n",
      "92\t1\t0.0\t1\t1\n",
      "133\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8001 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t3303.8\t0\t39\n",
      "4\t19\t825.9\t0\t19\n",
      "5\t16\t206.5\t0\t16\n",
      "6\t43\t51.6\t0\t43\n",
      "7\t148\t12.9\t0\t148\n",
      "8\t220\t3.2\t0\t220\n",
      "9\t700\t0.8\t0\t697 3\n",
      "10\t1111\t0.2\t1\t1091 20\n",
      "11\t109\t0.1\t1\t104 5\n",
      "12\t173\t0.0\t1\t167 6\n",
      "13\t230\t0.0\t1\t225 5\n",
      "14\t340\t0.0\t1\t326 14\n",
      "15\t257\t0.0\t1\t239 18\n",
      "16\t665\t0.0\t1\t631 34\n",
      "17\t977\t0.0\t1\t949 28\n",
      "18\t586\t0.0\t1\t511 49 26\n",
      "19\t2042\t0.0\t1\t1214 823 5\n",
      "20\t4\t0.0\t2\t2 2\n",
      "21\t285\t0.0\t2\t0 284 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t1\n",
      "142\t1\t0.0\t2\t0 1\n",
      "148\t1\t0.0\t2\t0 1\n",
      "161\t1\t0.0\t2\t0 1\n",
      "174\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.6.B.control.8.1.19_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.6.B.control.8.1.19_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.6.B.control.8.1.19_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.6.B.control.8.1.19_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.99 s (17 us/read; 3.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            172,279\n",
      "  Read 1 with adapter:                     166 (0.1%)\n",
      "  Read 2 with adapter:                  11,955 (6.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       172,279 (100.0%)\n",
      "\n",
      "Total basepairs processed:    78,404,156 bp\n",
      "  Read 1:    38,443,525 bp\n",
      "  Read 2:    39,960,631 bp\n",
      "Total written (filtered):     78,243,599 bp (99.8%)\n",
      "  Read 1:    38,441,246 bp\n",
      "  Read 2:    39,802,353 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 166 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t152\t2691.9\t0\t152\n",
      "9\t1\t0.7\t0\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "170\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t4\n",
      "218\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11955 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t69\t2691.9\t0\t69\n",
      "4\t23\t673.0\t0\t23\n",
      "5\t45\t168.2\t0\t45\n",
      "6\t114\t42.1\t0\t114\n",
      "7\t372\t10.5\t0\t372\n",
      "8\t582\t2.6\t0\t582\n",
      "9\t1850\t0.7\t0\t1847 3\n",
      "10\t2666\t0.2\t1\t2620 46\n",
      "11\t164\t0.0\t1\t159 5\n",
      "12\t257\t0.0\t1\t247 10\n",
      "13\t360\t0.0\t1\t341 19\n",
      "14\t461\t0.0\t1\t435 26\n",
      "15\t314\t0.0\t1\t298 16\n",
      "16\t791\t0.0\t1\t762 29\n",
      "17\t1013\t0.0\t1\t976 37\n",
      "18\t646\t0.0\t1\t558 58 30\n",
      "19\t1915\t0.0\t1\t1057 849 9\n",
      "20\t43\t0.0\t2\t35 5 3\n",
      "21\t241\t0.0\t2\t0 235 6\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "55\t2\t0.0\t2\t2\n",
      "58\t1\t0.0\t2\t0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "74\t1\t0.0\t2\t0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t7\t0.0\t2\t7\n",
      "83\t1\t0.0\t2\t1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.7.B.final.9.25.19_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.7.B.final.9.25.19_53_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.7.B.final.9.25.19_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.7.B.final.9.25.19_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.54 s (17 us/read; 3.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            323,966\n",
      "  Read 1 with adapter:                     131 (0.0%)\n",
      "  Read 2 with adapter:                  17,483 (5.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       323,966 (100.0%)\n",
      "\n",
      "Total basepairs processed:   147,394,000 bp\n",
      "  Read 1:    72,421,089 bp\n",
      "  Read 2:    74,972,911 bp\n",
      "Total written (filtered):    147,164,837 bp (99.8%)\n",
      "  Read 1:    72,420,179 bp\n",
      "  Read 2:    74,744,658 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 131 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t125\t5062.0\t0\t125\n",
      "26\t1\t0.0\t1\t0 1\n",
      "27\t2\t0.0\t1\t2\n",
      "31\t1\t0.0\t1\t1\n",
      "208\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 17483 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t247\t5062.0\t0\t247\n",
      "4\t24\t1265.5\t0\t24\n",
      "5\t87\t316.4\t0\t87\n",
      "6\t153\t79.1\t0\t153\n",
      "7\t360\t19.8\t0\t360\n",
      "8\t519\t4.9\t0\t519\n",
      "9\t1760\t1.2\t0\t1745 15\n",
      "10\t6108\t0.3\t1\t6022 86\n",
      "11\t282\t0.1\t1\t278 4\n",
      "12\t194\t0.0\t1\t193 1\n",
      "13\t156\t0.0\t1\t142 14\n",
      "14\t394\t0.0\t1\t358 36\n",
      "15\t392\t0.0\t1\t373 19\n",
      "16\t967\t0.0\t1\t936 31\n",
      "17\t1488\t0.0\t1\t1438 50\n",
      "18\t971\t0.0\t1\t872 79 20\n",
      "19\t2932\t0.0\t1\t1835 1078 19\n",
      "20\t7\t0.0\t2\t1 5 1\n",
      "21\t417\t0.0\t2\t0 411 6\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.A.final.9.25.19_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.A.final.9.25.19_67_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.A.final.9.25.19_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.A.final.9.25.19_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.12 s (19 us/read; 3.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            162,006\n",
      "  Read 1 with adapter:                      92 (0.1%)\n",
      "  Read 2 with adapter:                   9,809 (6.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       162,006 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,482,640 bp\n",
      "  Read 1:    36,067,151 bp\n",
      "  Read 2:    37,415,489 bp\n",
      "Total written (filtered):     73,343,719 bp (99.8%)\n",
      "  Read 1:    36,064,653 bp\n",
      "  Read 2:    37,279,066 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 92 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t2531.3\t0\t80\n",
      "24\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "198\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "218\t4\t0.0\t1\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9809 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t74\t2531.3\t0\t74\n",
      "4\t21\t632.8\t0\t21\n",
      "5\t34\t158.2\t0\t34\n",
      "6\t82\t39.6\t0\t82\n",
      "7\t211\t9.9\t0\t211\n",
      "8\t403\t2.5\t0\t403\n",
      "9\t1171\t0.6\t0\t1164 7\n",
      "10\t2016\t0.2\t1\t1994 22\n",
      "11\t136\t0.0\t1\t134 2\n",
      "12\t191\t0.0\t1\t185 6\n",
      "13\t231\t0.0\t1\t221 10\n",
      "14\t387\t0.0\t1\t370 17\n",
      "15\t297\t0.0\t1\t274 23\n",
      "16\t754\t0.0\t1\t727 27\n",
      "17\t1018\t0.0\t1\t977 41\n",
      "18\t631\t0.0\t1\t561 48 22\n",
      "19\t1905\t0.0\t1\t1094 803 8\n",
      "20\t2\t0.0\t2\t0 2\n",
      "21\t225\t0.0\t2\t0 224 1\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "63\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t5\t0.0\t2\t4 1\n",
      "99\t1\t0.0\t2\t1\n",
      "126\t2\t0.0\t2\t2\n",
      "218\t1\t0.0\t2\t1\n",
      "224\t1\t0.0\t2\t0 1\n",
      "228\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.B.NT.5percent.9.16.19_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.B.NT.5percent.9.16.19_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.B.NT.5percent.9.16.19_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.B.NT.5percent.9.16.19_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.89 s (12 us/read; 4.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            392,091\n",
      "  Read 1 with adapter:                     429 (0.1%)\n",
      "  Read 2 with adapter:                  28,749 (7.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       392,091 (100.0%)\n",
      "\n",
      "Total basepairs processed:   179,069,832 bp\n",
      "  Read 1:    87,800,162 bp\n",
      "  Read 2:    91,269,670 bp\n",
      "Total written (filtered):    178,715,185 bp (99.8%)\n",
      "  Read 1:    87,798,070 bp\n",
      "  Read 2:    90,917,115 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 429 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t410\t6126.4\t0\t410\n",
      "22\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t2 1\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "61\t5\t0.0\t1\t4 1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 28749 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t176\t6126.4\t0\t176\n",
      "4\t97\t1531.6\t0\t97\n",
      "5\t164\t382.9\t0\t164\n",
      "6\t384\t95.7\t0\t384\n",
      "7\t1345\t23.9\t0\t1345\n",
      "8\t2125\t6.0\t0\t2125\n",
      "9\t6083\t1.5\t0\t6066 17\n",
      "10\t6997\t0.4\t1\t6920 77\n",
      "11\t234\t0.1\t1\t228 6\n",
      "12\t304\t0.0\t1\t298 6\n",
      "13\t374\t0.0\t1\t358 16\n",
      "14\t665\t0.0\t1\t621 44\n",
      "15\t529\t0.0\t1\t507 22\n",
      "16\t1483\t0.0\t1\t1435 48\n",
      "17\t2057\t0.0\t1\t1976 81\n",
      "18\t1272\t0.0\t1\t1120 100 52\n",
      "19\t3942\t0.0\t1\t2234 1680 28\n",
      "20\t16\t0.0\t2\t0 13 3\n",
      "21\t444\t0.0\t2\t0 435 9\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "30\t3\t0.0\t2\t3\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t3\t0.0\t2\t3\n",
      "66\t2\t0.0\t2\t2\n",
      "75\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "82\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "140\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.B.NT.5percent.9.19.19_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.B.NT.5percent.9.19.19_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.B.NT.5percent.9.19.19_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.B.NT.5percent.9.19.19_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.29 s (15 us/read; 3.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            480,538\n",
      "  Read 1 with adapter:                     555 (0.1%)\n",
      "  Read 2 with adapter:                  24,238 (5.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       480,538 (100.0%)\n",
      "\n",
      "Total basepairs processed:   219,053,079 bp\n",
      "  Read 1:   107,536,900 bp\n",
      "  Read 2:   111,516,179 bp\n",
      "Total written (filtered):    218,701,482 bp (99.8%)\n",
      "  Read 1:   107,532,762 bp\n",
      "  Read 2:   111,168,720 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 555 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t510\t7508.4\t0\t510\n",
      "20\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t22\t0.0\t1\t21 1\n",
      "29\t3\t0.0\t1\t3\n",
      "52\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t1\n",
      "61\t1\t0.0\t1\t1\n",
      "65\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "93\t1\t0.0\t1\t1\n",
      "130\t1\t0.0\t1\t1\n",
      "143\t1\t0.0\t1\t1\n",
      "146\t2\t0.0\t1\t2\n",
      "213\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 24238 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t143\t7508.4\t0\t143\n",
      "4\t45\t1877.1\t0\t45\n",
      "5\t85\t469.3\t0\t85\n",
      "6\t222\t117.3\t0\t222\n",
      "7\t665\t29.3\t0\t665\n",
      "8\t793\t7.3\t0\t793\n",
      "9\t2560\t1.8\t0\t2555 5\n",
      "10\t3847\t0.5\t1\t3803 44\n",
      "11\t364\t0.1\t1\t355 9\n",
      "12\t559\t0.0\t1\t545 14\n",
      "13\t805\t0.0\t1\t784 21\n",
      "14\t1158\t0.0\t1\t1095 63\n",
      "15\t757\t0.0\t1\t707 50\n",
      "16\t2033\t0.0\t1\t1945 88\n",
      "17\t2692\t0.0\t1\t2575 117\n",
      "18\t1562\t0.0\t1\t1379 128 55\n",
      "19\t5229\t0.0\t1\t3003 2201 25\n",
      "20\t19\t0.0\t2\t5 11 3\n",
      "21\t639\t0.0\t2\t0 625 14\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "68\t2\t0.0\t2\t2\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "79\t6\t0.0\t2\t3 2 1\n",
      "100\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 1\n",
      "167\t1\t0.0\t2\t1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "192\t1\t0.0\t2\t1\n",
      "204\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "229\t1\t0.0\t2\t1\n",
      "231\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.25percent.8.25.19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.25percent.8.25.19_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.25percent.8.25.19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.25percent.8.25.19_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.01 s (15 us/read; 4.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            403,102\n",
      "  Read 1 with adapter:                     608 (0.2%)\n",
      "  Read 2 with adapter:                  35,115 (8.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       403,102 (100.0%)\n",
      "\n",
      "Total basepairs processed:   184,179,178 bp\n",
      "  Read 1:    90,268,489 bp\n",
      "  Read 2:    93,910,689 bp\n",
      "Total written (filtered):    183,757,039 bp (99.8%)\n",
      "  Read 1:    90,266,146 bp\n",
      "  Read 2:    93,490,893 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 608 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t594\t6298.5\t0\t594\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t6\t0.0\t1\t6\n",
      "29\t1\t0.0\t1\t1\n",
      "62\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "130\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 35115 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t291\t6298.5\t0\t291\n",
      "4\t141\t1574.6\t0\t141\n",
      "5\t271\t393.7\t0\t271\n",
      "6\t581\t98.4\t0\t581\n",
      "7\t1868\t24.6\t0\t1868\n",
      "8\t2741\t6.2\t0\t2741\n",
      "9\t8097\t1.5\t0\t8085 12\n",
      "10\t8165\t0.4\t1\t8069 96\n",
      "11\t276\t0.1\t1\t266 10\n",
      "12\t378\t0.0\t1\t371 7\n",
      "13\t536\t0.0\t1\t518 18\n",
      "14\t834\t0.0\t1\t791 43\n",
      "15\t706\t0.0\t1\t660 46\n",
      "16\t1783\t0.0\t1\t1726 57\n",
      "17\t2264\t0.0\t1\t2163 101\n",
      "18\t1371\t0.0\t1\t1194 116 61\n",
      "19\t4231\t0.0\t1\t2368 1835 28\n",
      "20\t12\t0.0\t2\t3 8 1\n",
      "21\t513\t0.0\t2\t0 503 10\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "46\t1\t0.0\t2\t1\n",
      "47\t1\t0.0\t2\t0 1\n",
      "48\t3\t0.0\t2\t3\n",
      "49\t1\t0.0\t2\t0 1\n",
      "53\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "143\t1\t0.0\t2\t1\n",
      "172\t1\t0.0\t2\t1\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "199\t1\t0.0\t2\t0 0 1\n",
      "202\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "237\t2\t0.0\t2\t1 0 1\n",
      "238\t1\t0.0\t2\t1\n",
      "240\t3\t0.0\t2\t3\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.50percent.8.26.19_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.50percent.8.26.19_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.50percent.8.26.19_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.50percent.8.26.19_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.87 s (14 us/read; 4.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            358,140\n",
      "  Read 1 with adapter:                     374 (0.1%)\n",
      "  Read 2 with adapter:                  35,324 (9.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       358,140 (100.0%)\n",
      "\n",
      "Total basepairs processed:   163,751,542 bp\n",
      "  Read 1:    80,201,108 bp\n",
      "  Read 2:    83,550,434 bp\n",
      "Total written (filtered):    163,334,901 bp (99.7%)\n",
      "  Read 1:    80,199,540 bp\n",
      "  Read 2:    83,135,361 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 374 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t364\t5595.9\t0\t364\n",
      "20\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t2\t0.0\t1\t1 1\n",
      "29\t1\t0.0\t1\t0 1\n",
      "129\t1\t0.0\t1\t1\n",
      "141\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 35324 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t328\t5595.9\t0\t328\n",
      "4\t138\t1399.0\t0\t138\n",
      "5\t269\t349.7\t0\t269\n",
      "6\t631\t87.4\t0\t631\n",
      "7\t2215\t21.9\t0\t2215\n",
      "8\t3041\t5.5\t0\t3041\n",
      "9\t8525\t1.4\t0\t8507 18\n",
      "10\t8230\t0.3\t1\t8148 82\n",
      "11\t277\t0.1\t1\t269 8\n",
      "12\t372\t0.0\t1\t356 16\n",
      "13\t492\t0.0\t1\t480 12\n",
      "14\t816\t0.0\t1\t770 46\n",
      "15\t612\t0.0\t1\t576 36\n",
      "16\t1602\t0.0\t1\t1533 69\n",
      "17\t2066\t0.0\t1\t1995 71\n",
      "18\t1270\t0.0\t1\t1130 91 49\n",
      "19\t3906\t0.0\t1\t2086 1792 28\n",
      "20\t5\t0.0\t2\t1 4\n",
      "21\t468\t0.0\t2\t0 459 9\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "54\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "70\t2\t0.0\t2\t1 0 1\n",
      "72\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t2 1\n",
      "87\t2\t0.0\t2\t2\n",
      "92\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t1\n",
      "107\t2\t0.0\t2\t2\n",
      "108\t1\t0.0\t2\t1\n",
      "118\t1\t0.0\t2\t1\n",
      "150\t2\t0.0\t2\t1 0 1\n",
      "177\t1\t0.0\t2\t1\n",
      "198\t1\t0.0\t2\t1\n",
      "208\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "233\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "239\t5\t0.0\t2\t4 1\n",
      "240\t1\t0.0\t2\t1\n",
      "241\t1\t0.0\t2\t0 1\n",
      "242\t1\t0.0\t2\t1\n",
      "243\t2\t0.0\t2\t1 0 1\n",
      "245\t1\t0.0\t2\t1\n",
      "246\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.75percent.9.4.19._17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.75percent.9.4.19._17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.75percent.9.4.19._17_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.75percent.9.4.19._17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.73 s (14 us/read; 4.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            403,579\n",
      "  Read 1 with adapter:                     602 (0.1%)\n",
      "  Read 2 with adapter:                  40,465 (10.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       403,579 (100.0%)\n",
      "\n",
      "Total basepairs processed:   184,524,793 bp\n",
      "  Read 1:    90,364,301 bp\n",
      "  Read 2:    94,160,492 bp\n",
      "Total written (filtered):    184,049,712 bp (99.7%)\n",
      "  Read 1:    90,361,053 bp\n",
      "  Read 2:    93,688,659 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 602 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t582\t6305.9\t0\t582\n",
      "13\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "48\t1\t0.0\t1\t1\n",
      "54\t1\t0.0\t1\t1\n",
      "55\t9\t0.0\t1\t9\n",
      "62\t1\t0.0\t1\t1\n",
      "109\t2\t0.0\t1\t2\n",
      "137\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t0 1\n",
      "211\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 40465 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t327\t6305.9\t0\t327\n",
      "4\t131\t1576.5\t0\t131\n",
      "5\t273\t394.1\t0\t273\n",
      "6\t624\t98.5\t0\t624\n",
      "7\t2050\t24.6\t0\t2050\n",
      "8\t3201\t6.2\t0\t3201\n",
      "9\t9448\t1.5\t0\t9426 22\n",
      "10\t10793\t0.4\t1\t10670 123\n",
      "11\t347\t0.1\t1\t337 10\n",
      "12\t448\t0.0\t1\t435 13\n",
      "13\t733\t0.0\t1\t703 30\n",
      "14\t1016\t0.0\t1\t969 47\n",
      "15\t703\t0.0\t1\t654 49\n",
      "16\t1905\t0.0\t1\t1836 69\n",
      "17\t2280\t0.0\t1\t2181 99\n",
      "18\t1306\t0.0\t1\t1118 114 74\n",
      "19\t4297\t0.0\t1\t2368 1906 23\n",
      "20\t13\t0.0\t2\t1 8 4\n",
      "21\t511\t0.0\t2\t0 507 4\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t2\t0.0\t2\t2\n",
      "59\t2\t0.0\t2\t2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "63\t5\t0.0\t2\t5\n",
      "64\t3\t0.0\t2\t3\n",
      "65\t2\t0.0\t2\t2\n",
      "66\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "73\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "135\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t1\n",
      "158\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "240\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.D.control.7.29.19_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.D.control.7.29.19_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.D.control.7.29.19_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.D.control.7.29.19_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.02 s (21 us/read; 2.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             95,460\n",
      "  Read 1 with adapter:                     101 (0.1%)\n",
      "  Read 2 with adapter:                   7,645 (8.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        95,460 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,105,570 bp\n",
      "  Read 1:    21,108,761 bp\n",
      "  Read 2:    21,996,809 bp\n",
      "Total written (filtered):     43,008,419 bp (99.8%)\n",
      "  Read 1:    21,107,460 bp\n",
      "  Read 2:    21,900,959 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t92\t1491.6\t0\t92\n",
      "21\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t0 1\n",
      "29\t1\t0.0\t1\t1\n",
      "61\t1\t0.0\t1\t0 1\n",
      "212\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t0 1\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7645 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t55\t1491.6\t0\t55\n",
      "4\t15\t372.9\t0\t15\n",
      "5\t39\t93.2\t0\t39\n",
      "6\t68\t23.3\t0\t68\n",
      "7\t307\t5.8\t0\t307\n",
      "8\t503\t1.5\t0\t503\n",
      "9\t1457\t0.4\t0\t1452 5\n",
      "10\t1898\t0.1\t1\t1875 23\n",
      "11\t82\t0.0\t1\t81 1\n",
      "12\t103\t0.0\t1\t98 5\n",
      "13\t176\t0.0\t1\t172 4\n",
      "14\t269\t0.0\t1\t256 13\n",
      "15\t182\t0.0\t1\t165 17\n",
      "16\t450\t0.0\t1\t436 14\n",
      "17\t514\t0.0\t1\t494 20\n",
      "18\t346\t0.0\t1\t309 20 17\n",
      "19\t1039\t0.0\t1\t573 459 7\n",
      "20\t3\t0.0\t2\t2 1\n",
      "21\t128\t0.0\t2\t0 126 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t1 1\n",
      "97\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "185\t1\t0.0\t2\t1\n",
      "203\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.25percent.9.3.19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.25percent.9.3.19_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.25percent.9.3.19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.25percent.9.3.19_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.50percent.9.5.19_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.50percent.9.5.19_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.50percent.9.5.19_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.50percent.9.5.19_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.75percent.9.9.19._46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.B.75percent.9.9.19._46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.75percent.9.9.19._46_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.B.75percent.9.9.19._46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.E.final.9.25.19_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.5.E.final.9.25.19_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.E.final.9.25.19_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.5.E.final.9.25.19_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.6.B.control.8.1.19_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.6.B.control.8.1.19_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.6.B.control.8.1.19_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.6.B.control.8.1.19_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.7.B.final.9.25.19_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.7.B.final.9.25.19_53_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.7.B.final.9.25.19_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.7.B.final.9.25.19_53_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.A.final.9.25.19_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.A.final.9.25.19_67_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.A.final.9.25.19_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.A.final.9.25.19_67_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.B.NT.5percent.9.16.19_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.B.NT.5percent.9.16.19_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.B.NT.5percent.9.16.19_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.B.NT.5percent.9.16.19_9_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.B.NT.5percent.9.19.19_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.B.NT.5percent.9.19.19_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.B.NT.5percent.9.19.19_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.B.NT.5percent.9.19.19_18_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.25percent.8.25.19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.25percent.8.25.19_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.25percent.8.25.19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.25percent.8.25.19_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.50percent.8.26.19_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.50percent.8.26.19_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.50percent.8.26.19_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.50percent.8.26.19_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.75percent.9.4.19._17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.8.G.75percent.9.4.19._17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.75percent.9.4.19._17_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.8.G.75percent.9.4.19._17_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.D.control.7.29.19_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.D.control.7.29.19_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.D.control.7.29.19_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.D.control.7.29.19_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.D.control.7.30.19_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.D.control.7.30.19_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.D.control.7.30.19_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.D.control.7.30.19_38_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.D.control.7.30.19_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.D.control.7.30.19_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.D.control.7.30.19_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.D.control.7.30.19_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.62 s (16 us/read; 3.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            349,624\n",
      "  Read 1 with adapter:                     231 (0.1%)\n",
      "  Read 2 with adapter:                  23,965 (6.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       349,624 (100.0%)\n",
      "\n",
      "Total basepairs processed:   159,488,192 bp\n",
      "  Read 1:    78,257,510 bp\n",
      "  Read 2:    81,230,682 bp\n",
      "Total written (filtered):    159,179,739 bp (99.8%)\n",
      "  Read 1:    78,256,019 bp\n",
      "  Read 2:    80,923,720 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 231 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t219\t5462.9\t0\t219\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t3\t0.0\t1\t3\n",
      "29\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "101\t3\t0.0\t1\t2 1\n",
      "177\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 23965 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t230\t5462.9\t0\t230\n",
      "4\t76\t1365.7\t0\t76\n",
      "5\t141\t341.4\t0\t141\n",
      "6\t302\t85.4\t0\t302\n",
      "7\t871\t21.3\t0\t871\n",
      "8\t1435\t5.3\t0\t1435\n",
      "9\t4411\t1.3\t0\t4392 19\n",
      "10\t5370\t0.3\t1\t5306 64\n",
      "11\t219\t0.1\t1\t212 7\n",
      "12\t296\t0.0\t1\t291 5\n",
      "13\t380\t0.0\t1\t365 15\n",
      "14\t711\t0.0\t1\t668 43\n",
      "15\t612\t0.0\t1\t587 25\n",
      "16\t1528\t0.0\t1\t1459 69\n",
      "17\t1944\t0.0\t1\t1870 74\n",
      "18\t1172\t0.0\t1\t1043 81 48\n",
      "19\t3713\t0.0\t1\t2089 1603 21\n",
      "20\t11\t0.0\t2\t0 8 3\n",
      "21\t486\t0.0\t2\t0 481 5\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "30\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t2\t0.0\t2\t0 1 1\n",
      "67\t2\t0.0\t2\t1 1\n",
      "69\t2\t0.0\t2\t2\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "79\t6\t0.0\t2\t4 2\n",
      "84\t2\t0.0\t2\t0 1 1\n",
      "98\t1\t0.0\t2\t1\n",
      "102\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.F.control.7.29.19_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.F.control.7.29.19_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.F.control.7.29.19_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.F.control.7.29.19_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.88 s (18 us/read; 3.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            218,741\n",
      "  Read 1 with adapter:                   4,167 (1.9%)\n",
      "  Read 2 with adapter:                   9,841 (4.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       218,741 (100.0%)\n",
      "\n",
      "Total basepairs processed:    99,570,711 bp\n",
      "  Read 1:    48,886,465 bp\n",
      "  Read 2:    50,684,246 bp\n",
      "Total written (filtered):     99,313,939 bp (99.7%)\n",
      "  Read 1:    48,765,604 bp\n",
      "  Read 2:    50,548,335 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 4167 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t207\t3417.8\t0\t207\n",
      "15\t1\t0.0\t1\t1\n",
      "19\t1\t0.0\t1\t0 1\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t1 1\n",
      "26\t6\t0.0\t1\t4 2\n",
      "27\t23\t0.0\t1\t19 4\n",
      "28\t112\t0.0\t1\t57 55\n",
      "29\t3720\t0.0\t1\t3646 74\n",
      "30\t9\t0.0\t1\t5 4\n",
      "31\t3\t0.0\t1\t3\n",
      "33\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "57\t8\t0.0\t1\t6 2\n",
      "58\t18\t0.0\t1\t15 3\n",
      "87\t2\t0.0\t1\t2\n",
      "88\t3\t0.0\t1\t0 3\n",
      "89\t14\t0.0\t1\t14\n",
      "92\t17\t0.0\t1\t16 1\n",
      "154\t2\t0.0\t1\t2\n",
      "155\t8\t0.0\t1\t8\n",
      "161\t1\t0.0\t1\t1\n",
      "166\t1\t0.0\t1\t1\n",
      "206\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t0 1\n",
      "218\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9841 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t76\t3417.8\t0\t76\n",
      "4\t25\t854.5\t0\t25\n",
      "5\t28\t213.6\t0\t28\n",
      "6\t108\t53.4\t0\t108\n",
      "7\t253\t13.4\t0\t253\n",
      "8\t500\t3.3\t0\t500\n",
      "9\t1542\t0.8\t0\t1537 5\n",
      "10\t1905\t0.2\t1\t1888 17\n",
      "11\t83\t0.1\t1\t75 8\n",
      "12\t110\t0.0\t1\t107 3\n",
      "13\t157\t0.0\t1\t150 7\n",
      "14\t254\t0.0\t1\t242 12\n",
      "15\t208\t0.0\t1\t196 12\n",
      "16\t621\t0.0\t1\t592 29\n",
      "17\t982\t0.0\t1\t956 26\n",
      "18\t573\t0.0\t1\t514 41 18\n",
      "19\t2114\t0.0\t1\t1277 830 7\n",
      "20\t4\t0.0\t2\t0 3 1\n",
      "21\t275\t0.0\t2\t0 273 2\n",
      "22\t8\t0.0\t2\t0 1 7\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "144\t2\t0.0\t2\t2\n",
      "186\t1\t0.0\t2\t1\n",
      "206\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "241\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/Ofav.Tank.18.D.control.7.31.19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/Ofav.Tank.18.D.control.7.31.19_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/Ofav.Tank.18.D.control.7.31.19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/Ofav.Tank.18.D.control.7.31.19_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.27 s (20 us/read; 3.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,994\n",
      "  Read 1 with adapter:                     120 (0.1%)\n",
      "  Read 2 with adapter:                   9,463 (8.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,994 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,661,049 bp\n",
      "  Read 1:    25,800,272 bp\n",
      "  Read 2:    26,860,777 bp\n",
      "Total written (filtered):     52,541,059 bp (99.8%)\n",
      "  Read 1:    25,797,753 bp\n",
      "  Read 2:    26,743,306 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 120 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t107\t1812.4\t0\t107\n",
      "26\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t1\n",
      "91\t1\t0.0\t1\t0 1\n",
      "162\t1\t0.0\t1\t1\n",
      "168\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "208\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t3\t0.0\t1\t2 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9463 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t1812.4\t0\t59\n",
      "4\t32\t453.1\t0\t32\n",
      "5\t49\t113.3\t0\t49\n",
      "6\t144\t28.3\t0\t144\n",
      "7\t504\t7.1\t0\t504\n",
      "8\t645\t1.8\t0\t645\n",
      "9\t1885\t0.4\t0\t1877 8\n",
      "10\t2164\t0.1\t1\t2135 29\n",
      "11\t90\t0.0\t1\t85 5\n",
      "12\t130\t0.0\t1\t125 5\n",
      "13\t173\t0.0\t1\t168 5\n",
      "14\t307\t0.0\t1\t293 14\n",
      "15\t215\t0.0\t1\t198 17\n",
      "16\t516\t0.0\t1\t494 22\n",
      "17\t664\t0.0\t1\t631 33\n",
      "18\t397\t0.0\t1\t346 34 17\n",
      "19\t1292\t0.0\t1\t738 551 3\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t171\t0.0\t2\t0 166 5\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t2\t0.0\t2\t1 1\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t2\t0.0\t2\t1 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t0 1\n",
      "68\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t1\n",
      "81\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "172\t1\t0.0\t2\t1\n",
      "240\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/Ofav.Tank.6.F.control.8.1.19_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/Ofav.Tank.6.F.control.8.1.19_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/Ofav.Tank.6.F.control.8.1.19_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/Ofav.Tank.6.F.control.8.1.19_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.96 s (20 us/read; 3.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            150,724\n",
      "  Read 1 with adapter:                     163 (0.1%)\n",
      "  Read 2 with adapter:                   9,431 (6.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       150,724 (100.0%)\n",
      "\n",
      "Total basepairs processed:    68,694,177 bp\n",
      "  Read 1:    33,692,768 bp\n",
      "  Read 2:    35,001,409 bp\n",
      "Total written (filtered):     68,571,812 bp (99.8%)\n",
      "  Read 1:    33,691,819 bp\n",
      "  Read 2:    34,879,993 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 163 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t156\t2355.1\t0\t156\n",
      "21\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t2 1\n",
      "27\t1\t0.0\t1\t1\n",
      "139\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9431 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t2355.1\t0\t79\n",
      "4\t40\t588.8\t0\t40\n",
      "5\t41\t147.2\t0\t41\n",
      "6\t117\t36.8\t0\t117\n",
      "7\t365\t9.2\t0\t365\n",
      "8\t548\t2.3\t0\t548\n",
      "9\t1711\t0.6\t0\t1702 9\n",
      "10\t2244\t0.1\t1\t2213 31\n",
      "11\t100\t0.0\t1\t98 2\n",
      "12\t111\t0.0\t1\t108 3\n",
      "13\t157\t0.0\t1\t149 8\n",
      "14\t238\t0.0\t1\t225 13\n",
      "15\t196\t0.0\t1\t185 11\n",
      "16\t552\t0.0\t1\t523 29\n",
      "17\t745\t0.0\t1\t721 24\n",
      "18\t430\t0.0\t1\t378 42 10\n",
      "19\t1516\t0.0\t1\t881 628 7\n",
      "20\t2\t0.0\t2\t1 0 1\n",
      "21\t216\t0.0\t2\t0 211 5\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 1\n",
      "79\t4\t0.0\t2\t3 1\n",
      "97\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t1\n",
      "154\t2\t0.0\t2\t1 1\n",
      "183\t1\t0.0\t2\t0 1\n",
      "238\t1\t0.0\t2\t0 1\n",
      "244\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.F.control.7.29.19_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/O.fav.Tank.9.F.control.7.29.19_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.F.control.7.29.19_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/O.fav.Tank.9.F.control.7.29.19_4_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/Ofav.Tank.18.D.control.7.31.19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/Ofav.Tank.18.D.control.7.31.19_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/Ofav.Tank.18.D.control.7.31.19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/Ofav.Tank.18.D.control.7.31.19_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/Ofav.Tank.6.F.control.8.1.19_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-xixy9uh1/Ofav.Tank.6.F.control.8.1.19_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/Ofav.Tank.6.F.control.8.1.19_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-lkj281t3/c17bda13-f047-4497-a76f-1ab6e0ce4cf3/data/Ofav.Tank.6.F.control.8.1.19_43_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGtransmissionstudies_208-276.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O..fav.Tank.6.F.control.9.13.19_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O..fav.Tank.6.F.control.9.13.19_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O..fav.Tank.6.F.control.9.13.19_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O..fav.Tank.6.F.control.9.13.19_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.06 s (17 us/read; 3.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            183,403\n",
      "  Read 1 with adapter:                     135 (0.1%)\n",
      "  Read 2 with adapter:                  10,699 (5.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       183,403 (100.0%)\n",
      "\n",
      "Total basepairs processed:    83,511,110 bp\n",
      "  Read 1:    40,982,794 bp\n",
      "  Read 2:    42,528,316 bp\n",
      "Total written (filtered):     83,360,880 bp (99.8%)\n",
      "  Read 1:    40,982,150 bp\n",
      "  Read 2:    42,378,730 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 135 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t132\t2865.7\t0\t132\n",
      "25\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t1\n",
      "135\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10699 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t63\t2865.7\t0\t63\n",
      "4\t13\t716.4\t0\t13\n",
      "5\t38\t179.1\t0\t38\n",
      "6\t43\t44.8\t0\t43\n",
      "7\t176\t11.2\t0\t176\n",
      "8\t314\t2.8\t0\t314\n",
      "9\t1105\t0.7\t0\t1102 3\n",
      "10\t2676\t0.2\t1\t2642 34\n",
      "11\t157\t0.0\t1\t150 7\n",
      "12\t189\t0.0\t1\t185 4\n",
      "13\t289\t0.0\t1\t282 7\n",
      "14\t464\t0.0\t1\t430 34\n",
      "15\t385\t0.0\t1\t363 22\n",
      "16\t861\t0.0\t1\t825 36\n",
      "17\t1009\t0.0\t1\t971 38\n",
      "18\t625\t0.0\t1\t544 52 29\n",
      "19\t1980\t0.0\t1\t1122 848 10\n",
      "20\t3\t0.0\t2\t0 2 1\n",
      "21\t272\t0.0\t2\t0 268 4\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "50\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t0 1\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t2\t0.0\t2\t1 0 1\n",
      "73\t1\t0.0\t2\t1\n",
      "80\t1\t0.0\t2\t1\n",
      "82\t1\t0.0\t2\t1\n",
      "83\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t1\n",
      "91\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "107\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t1\n",
      "126\t1\t0.0\t2\t1\n",
      "127\t1\t0.0\t2\t1\n",
      "141\t1\t0.0\t2\t0 1\n",
      "143\t1\t0.0\t2\t1\n",
      "212\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.8.20.19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.8.20.19_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.8.20.19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.8.20.19_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 6.07 s (17 us/read; 3.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            356,613\n",
      "  Read 1 with adapter:                     199 (0.1%)\n",
      "  Read 2 with adapter:                  17,682 (5.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       356,613 (100.0%)\n",
      "\n",
      "Total basepairs processed:   162,358,605 bp\n",
      "  Read 1:    79,805,331 bp\n",
      "  Read 2:    82,553,274 bp\n",
      "Total written (filtered):    162,113,716 bp (99.8%)\n",
      "  Read 1:    79,804,466 bp\n",
      "  Read 2:    82,309,250 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 199 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t187\t5572.1\t0\t187\n",
      "9\t1\t1.4\t0\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t5\t0.0\t1\t5\n",
      "32\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 17682 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t117\t5572.1\t0\t117\n",
      "4\t44\t1393.0\t0\t44\n",
      "5\t94\t348.3\t0\t94\n",
      "6\t177\t87.1\t0\t177\n",
      "7\t445\t21.8\t0\t445\n",
      "8\t714\t5.4\t0\t714\n",
      "9\t2590\t1.4\t0\t2586 4\n",
      "10\t3235\t0.3\t1\t3183 52\n",
      "11\t232\t0.1\t1\t227 5\n",
      "12\t328\t0.0\t1\t325 3\n",
      "13\t406\t0.0\t1\t396 10\n",
      "14\t594\t0.0\t1\t553 41\n",
      "15\t431\t0.0\t1\t412 19\n",
      "16\t1221\t0.0\t1\t1166 55\n",
      "17\t1815\t0.0\t1\t1766 49\n",
      "18\t1085\t0.0\t1\t960 91 34\n",
      "19\t3607\t0.0\t1\t2106 1486 15\n",
      "20\t12\t0.0\t2\t1 7 4\n",
      "21\t501\t0.0\t2\t0 497 4\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "44\t1\t0.0\t2\t1\n",
      "49\t2\t0.0\t2\t2\n",
      "57\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "133\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t1\n",
      "210\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.8.21.19_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.8.21.19_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.8.21.19_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.8.21.19_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.61 s (19 us/read; 3.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            185,700\n",
      "  Read 1 with adapter:                     108 (0.1%)\n",
      "  Read 2 with adapter:                  12,367 (6.7%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       185,699 (100.0%)\n",
      "\n",
      "Total basepairs processed:    84,434,403 bp\n",
      "  Read 1:    41,444,680 bp\n",
      "  Read 2:    42,989,723 bp\n",
      "Total written (filtered):     84,266,109 bp (99.8%)\n",
      "  Read 1:    41,442,582 bp\n",
      "  Read 2:    42,823,527 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t96\t2901.6\t0\t96\n",
      "19\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "176\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t0 1\n",
      "212\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12367 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t2901.6\t0\t73\n",
      "4\t22\t725.4\t0\t22\n",
      "5\t41\t181.3\t0\t41\n",
      "6\t128\t45.3\t0\t128\n",
      "7\t342\t11.3\t0\t342\n",
      "8\t604\t2.8\t0\t604\n",
      "9\t1917\t0.7\t0\t1912 5\n",
      "10\t2418\t0.2\t1\t2391 27\n",
      "11\t199\t0.0\t1\t193 6\n",
      "12\t283\t0.0\t1\t275 8\n",
      "13\t358\t0.0\t1\t345 13\n",
      "14\t516\t0.0\t1\t487 29\n",
      "15\t367\t0.0\t1\t338 29\n",
      "16\t923\t0.0\t1\t879 44\n",
      "17\t1142\t0.0\t1\t1096 46\n",
      "18\t677\t0.0\t1\t572 55 50\n",
      "19\t2060\t0.0\t1\t1123 928 9\n",
      "20\t11\t0.0\t2\t0 7 4\n",
      "21\t248\t0.0\t2\t0 244 4\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "30\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t3\t0.0\t2\t2 1\n",
      "65\t1\t0.0\t2\t0 1\n",
      "69\t1\t0.0\t2\t0 1\n",
      "70\t2\t0.0\t2\t1 1\n",
      "72\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "80\t2\t0.0\t2\t2\n",
      "83\t1\t0.0\t2\t1\n",
      "97\t1\t0.0\t2\t1\n",
      "114\t1\t0.0\t2\t1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "228\t1\t0.0\t2\t1\n",
      "251\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.9.25.19_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.9.25.19_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.9.25.19_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.9.25.19_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.97 s (18 us/read; 3.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            336,737\n",
      "  Read 1 with adapter:                     388 (0.1%)\n",
      "  Read 2 with adapter:                  11,587 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       336,737 (100.0%)\n",
      "\n",
      "Total basepairs processed:   153,100,496 bp\n",
      "  Read 1:    75,263,895 bp\n",
      "  Read 2:    77,836,601 bp\n",
      "Total written (filtered):    152,920,189 bp (99.9%)\n",
      "  Read 1:    75,261,325 bp\n",
      "  Read 2:    77,658,864 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 388 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t364\t5261.5\t0\t364\n",
      "27\t3\t0.0\t1\t3\n",
      "36\t1\t0.0\t1\t1\n",
      "55\t16\t0.0\t1\t16\n",
      "63\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t0 1\n",
      "113\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11587 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t52\t5261.5\t0\t52\n",
      "4\t24\t1315.4\t0\t24\n",
      "5\t44\t328.8\t0\t44\n",
      "6\t65\t82.2\t0\t65\n",
      "7\t111\t20.6\t0\t111\n",
      "8\t214\t5.1\t0\t214\n",
      "9\t728\t1.3\t0\t724 4\n",
      "10\t1396\t0.3\t1\t1385 11\n",
      "11\t158\t0.1\t1\t152 6\n",
      "12\t339\t0.0\t1\t331 8\n",
      "13\t449\t0.0\t1\t430 19\n",
      "14\t597\t0.0\t1\t550 47\n",
      "15\t392\t0.0\t1\t372 20\n",
      "16\t1115\t0.0\t1\t1063 52\n",
      "17\t1561\t0.0\t1\t1510 51\n",
      "18\t911\t0.0\t1\t808 65 38\n",
      "19\t2984\t0.0\t1\t1752 1222 10\n",
      "20\t7\t0.0\t2\t1 5 1\n",
      "21\t414\t0.0\t2\t0 412 2\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "59\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t1\n",
      "176\t1\t0.0\t2\t1\n",
      "190\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.8.7.19_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.8.7.19_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.8.7.19_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.8.7.19_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.41 s (17 us/read; 3.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            324,845\n",
      "  Read 1 with adapter:                      91 (0.0%)\n",
      "  Read 2 with adapter:                  22,455 (6.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       324,845 (100.0%)\n",
      "\n",
      "Total basepairs processed:   147,669,101 bp\n",
      "  Read 1:    72,512,040 bp\n",
      "  Read 2:    75,157,061 bp\n",
      "Total written (filtered):    147,378,021 bp (99.8%)\n",
      "  Read 1:    72,509,727 bp\n",
      "  Read 2:    74,868,294 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 91 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t5075.7\t0\t80\n",
      "25\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t0 2\n",
      "217\t2\t0.0\t1\t1 1\n",
      "218\t5\t0.0\t1\t4 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 22455 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t149\t5075.7\t0\t149\n",
      "4\t67\t1268.9\t0\t67\n",
      "5\t127\t317.2\t0\t127\n",
      "6\t276\t79.3\t0\t276\n",
      "7\t771\t19.8\t0\t771\n",
      "8\t1307\t5.0\t0\t1307\n",
      "9\t4281\t1.2\t0\t4275 6\n",
      "10\t5196\t0.3\t1\t5134 62\n",
      "11\t282\t0.1\t1\t272 10\n",
      "12\t289\t0.0\t1\t284 5\n",
      "13\t389\t0.0\t1\t382 7\n",
      "14\t627\t0.0\t1\t591 36\n",
      "15\t529\t0.0\t1\t507 22\n",
      "16\t1374\t0.0\t1\t1328 46\n",
      "17\t1779\t0.0\t1\t1698 81\n",
      "18\t1056\t0.0\t1\t929 90 37\n",
      "19\t3431\t0.0\t1\t1922 1488 21\n",
      "20\t7\t0.0\t2\t1 4 2\n",
      "21\t458\t0.0\t2\t0 450 8\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "52\t2\t0.0\t2\t2\n",
      "58\t2\t0.0\t2\t1 0 1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t1 1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t1\t0.0\t2\t0 1\n",
      "97\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t1\n",
      "151\t2\t0.0\t2\t2\n",
      "155\t1\t0.0\t2\t0 1\n",
      "164\t1\t0.0\t2\t0 1\n",
      "165\t1\t0.0\t2\t0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "199\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t3\t0.0\t2\t2 0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t2\t0.0\t2\t1 1\n",
      "222\t1\t0.0\t2\t0 1\n",
      "229\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.11.19_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.11.19_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.11.19_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.11.19_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.18 s (17 us/read; 3.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            187,712\n",
      "  Read 1 with adapter:                     157 (0.1%)\n",
      "  Read 2 with adapter:                  10,009 (5.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       187,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    85,530,144 bp\n",
      "  Read 1:    41,983,324 bp\n",
      "  Read 2:    43,546,820 bp\n",
      "Total written (filtered):     85,391,985 bp (99.8%)\n",
      "  Read 1:    41,982,694 bp\n",
      "  Read 2:    43,409,291 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 157 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t150\t2933.0\t0\t150\n",
      "23\t1\t0.0\t1\t1\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10009 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t2933.0\t0\t42\n",
      "4\t34\t733.2\t0\t34\n",
      "5\t21\t183.3\t0\t21\n",
      "6\t71\t45.8\t0\t71\n",
      "7\t230\t11.5\t0\t230\n",
      "8\t382\t2.9\t0\t382\n",
      "9\t1390\t0.7\t0\t1390\n",
      "10\t2154\t0.2\t1\t2123 31\n",
      "11\t160\t0.0\t1\t158 2\n",
      "12\t219\t0.0\t1\t213 6\n",
      "13\t294\t0.0\t1\t280 14\n",
      "14\t385\t0.0\t1\t357 28\n",
      "15\t264\t0.0\t1\t241 23\n",
      "16\t727\t0.0\t1\t701 26\n",
      "17\t951\t0.0\t1\t908 43\n",
      "18\t568\t0.0\t1\t496 46 26\n",
      "19\t1852\t0.0\t1\t1125 721 6\n",
      "20\t5\t0.0\t2\t0 4 1\n",
      "21\t226\t0.0\t2\t0 224 2\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t0 1\n",
      "131\t1\t0.0\t2\t1\n",
      "149\t1\t0.0\t2\t1\n",
      "175\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "218\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.17.19_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.17.19_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.17.19_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.17.19_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.24 s (16 us/read; 3.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            260,972\n",
      "  Read 1 with adapter:                     345 (0.1%)\n",
      "  Read 2 with adapter:                  20,141 (7.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       260,972 (100.0%)\n",
      "\n",
      "Total basepairs processed:   119,152,659 bp\n",
      "  Read 1:    58,426,372 bp\n",
      "  Read 2:    60,726,287 bp\n",
      "Total written (filtered):    118,892,185 bp (99.8%)\n",
      "  Read 1:    58,424,904 bp\n",
      "  Read 2:    60,467,281 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 345 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t340\t4077.7\t0\t340\n",
      "16\t1\t0.0\t1\t1\n",
      "24\t2\t0.0\t1\t2\n",
      "190\t1\t0.0\t1\t1\n",
      "194\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 20141 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t4077.7\t0\t91\n",
      "4\t41\t1019.4\t0\t41\n",
      "5\t75\t254.9\t0\t75\n",
      "6\t169\t63.7\t0\t169\n",
      "7\t627\t15.9\t0\t627\n",
      "8\t1004\t4.0\t0\t1004\n",
      "9\t3551\t1.0\t0\t3541 10\n",
      "10\t4879\t0.2\t1\t4824 55\n",
      "11\t294\t0.1\t1\t291 3\n",
      "12\t474\t0.0\t1\t458 16\n",
      "13\t665\t0.0\t1\t639 26\n",
      "14\t826\t0.0\t1\t784 42\n",
      "15\t513\t0.0\t1\t482 31\n",
      "16\t1350\t0.0\t1\t1297 53\n",
      "17\t1528\t0.0\t1\t1460 68\n",
      "18\t882\t0.0\t1\t779 61 42\n",
      "19\t2777\t0.0\t1\t1537 1224 16\n",
      "20\t3\t0.0\t2\t0 2 1\n",
      "21\t347\t0.0\t2\t0 343 4\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "52\t1\t0.0\t2\t1\n",
      "54\t2\t0.0\t2\t1 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t3\t0.0\t2\t2 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "119\t1\t0.0\t2\t1\n",
      "165\t1\t0.0\t2\t0 1\n",
      "170\t1\t0.0\t2\t0 1\n",
      "214\t2\t0.0\t2\t2\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "218\t3\t0.0\t2\t3\n",
      "237\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.25.19_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.25.19_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.25.19_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.25.19_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.54 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,753\n",
      "  Read 1 with adapter:                      57 (0.1%)\n",
      "  Read 2 with adapter:                   4,703 (8.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        57,753 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,885,269 bp\n",
      "  Read 1:    12,670,709 bp\n",
      "  Read 2:    13,214,560 bp\n",
      "Total written (filtered):     25,818,646 bp (99.7%)\n",
      "  Read 1:    12,668,609 bp\n",
      "  Read 2:    13,150,037 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 57 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t902.4\t0\t48\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t4\t0.0\t1\t3 1\n",
      "218\t4\t0.0\t1\t2 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4703 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t902.4\t0\t19\n",
      "4\t8\t225.6\t0\t8\n",
      "5\t12\t56.4\t0\t12\n",
      "6\t18\t14.1\t0\t18\n",
      "7\t115\t3.5\t0\t115\n",
      "8\t195\t0.9\t0\t195\n",
      "9\t687\t0.2\t0\t685 2\n",
      "10\t972\t0.1\t1\t964 8\n",
      "11\t88\t0.0\t1\t83 5\n",
      "12\t151\t0.0\t1\t146 5\n",
      "13\t199\t0.0\t1\t191 8\n",
      "14\t263\t0.0\t1\t253 10\n",
      "15\t155\t0.0\t1\t143 12\n",
      "16\t365\t0.0\t1\t345 20\n",
      "17\t397\t0.0\t1\t378 19\n",
      "18\t232\t0.0\t1\t202 17 13\n",
      "19\t735\t0.0\t1\t398 334 3\n",
      "21\t74\t0.0\t2\t0 73 1\n",
      "22\t3\t0.0\t2\t0 1 2\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "129\t1\t0.0\t2\t1\n",
      "157\t1\t0.0\t2\t1\n",
      "165\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t2\t0.0\t2\t2\n",
      "219\t1\t0.0\t2\t0 1\n",
      "226\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.7.19_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.7.19_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.7.19_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.7.19_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.95 s (18 us/read; 3.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            276,882\n",
      "  Read 1 with adapter:                     302 (0.1%)\n",
      "  Read 2 with adapter:                  22,685 (8.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       276,882 (100.0%)\n",
      "\n",
      "Total basepairs processed:   126,419,236 bp\n",
      "  Read 1:    61,966,930 bp\n",
      "  Read 2:    64,452,306 bp\n",
      "Total written (filtered):    126,126,437 bp (99.8%)\n",
      "  Read 1:    61,965,349 bp\n",
      "  Read 2:    64,161,088 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 302 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t292\t4326.3\t0\t292\n",
      "25\t3\t0.0\t1\t1 2\n",
      "26\t3\t0.0\t1\t3\n",
      "79\t1\t0.0\t1\t1\n",
      "116\t1\t0.0\t1\t1\n",
      "153\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 22685 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t155\t4326.3\t0\t155\n",
      "4\t79\t1081.6\t0\t79\n",
      "5\t99\t270.4\t0\t99\n",
      "6\t270\t67.6\t0\t270\n",
      "7\t860\t16.9\t0\t860\n",
      "8\t1350\t4.2\t0\t1350\n",
      "9\t4308\t1.1\t0\t4295 13\n",
      "10\t4980\t0.3\t1\t4918 62\n",
      "11\t349\t0.1\t1\t339 10\n",
      "12\t506\t0.0\t1\t491 15\n",
      "13\t706\t0.0\t1\t681 25\n",
      "14\t904\t0.0\t1\t844 60\n",
      "15\t623\t0.0\t1\t577 46\n",
      "16\t1370\t0.0\t1\t1321 49\n",
      "17\t1673\t0.0\t1\t1604 69\n",
      "18\t977\t0.0\t1\t815 92 70\n",
      "19\t2962\t0.0\t1\t1618 1331 13\n",
      "20\t5\t0.0\t2\t0 4 1\n",
      "21\t383\t0.0\t2\t0 374 9\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "30\t3\t0.0\t2\t3\n",
      "48\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t1\n",
      "52\t3\t0.0\t2\t2 0 1\n",
      "53\t1\t0.0\t2\t1\n",
      "55\t2\t0.0\t2\t2\n",
      "56\t1\t0.0\t2\t1\n",
      "57\t2\t0.0\t2\t1 0 1\n",
      "58\t3\t0.0\t2\t2 0 1\n",
      "59\t2\t0.0\t2\t2\n",
      "60\t2\t0.0\t2\t1 0 1\n",
      "61\t6\t0.0\t2\t6\n",
      "62\t4\t0.0\t2\t3 1\n",
      "63\t4\t0.0\t2\t4\n",
      "64\t7\t0.0\t2\t6 0 1\n",
      "65\t3\t0.0\t2\t2 0 1\n",
      "66\t2\t0.0\t2\t1 1\n",
      "67\t3\t0.0\t2\t2 0 1\n",
      "68\t5\t0.0\t2\t5\n",
      "69\t6\t0.0\t2\t6\n",
      "73\t2\t0.0\t2\t2\n",
      "74\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "77\t2\t0.0\t2\t1 1\n",
      "78\t5\t0.0\t2\t4 1\n",
      "79\t5\t0.0\t2\t3 1 1\n",
      "80\t1\t0.0\t2\t1\n",
      "87\t2\t0.0\t2\t2\n",
      "90\t2\t0.0\t2\t1 1\n",
      "92\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t1\n",
      "111\t1\t0.0\t2\t1\n",
      "116\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t1\n",
      "127\t1\t0.0\t2\t0 1\n",
      "140\t1\t0.0\t2\t0 1\n",
      "143\t1\t0.0\t2\t1\n",
      "150\t3\t0.0\t2\t3\n",
      "153\t2\t0.0\t2\t0 1 1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t1\n",
      "171\t1\t0.0\t2\t0 1\n",
      "207\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.9.19_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.9.19_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.9.19_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.9.19_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.30 s (17 us/read; 3.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            253,531\n",
      "  Read 1 with adapter:                     168 (0.1%)\n",
      "  Read 2 with adapter:                   9,321 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       253,531 (100.0%)\n",
      "\n",
      "Total basepairs processed:   115,470,984 bp\n",
      "  Read 1:    56,746,932 bp\n",
      "  Read 2:    58,724,052 bp\n",
      "Total written (filtered):    115,330,882 bp (99.9%)\n",
      "  Read 1:    56,745,835 bp\n",
      "  Read 2:    58,585,047 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 168 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t158\t3961.4\t0\t158\n",
      "26\t5\t0.0\t1\t5\n",
      "87\t1\t0.0\t1\t1\n",
      "92\t3\t0.0\t1\t3\n",
      "130\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9321 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t3961.4\t0\t42\n",
      "4\t28\t990.4\t0\t28\n",
      "5\t18\t247.6\t0\t18\n",
      "6\t55\t61.9\t0\t55\n",
      "7\t141\t15.5\t0\t141\n",
      "8\t245\t3.9\t0\t245\n",
      "9\t965\t1.0\t0\t963 2\n",
      "10\t1372\t0.2\t1\t1352 20\n",
      "11\t131\t0.1\t1\t126 5\n",
      "12\t249\t0.0\t1\t243 6\n",
      "13\t266\t0.0\t1\t257 9\n",
      "14\t357\t0.0\t1\t333 24\n",
      "15\t248\t0.0\t1\t237 11\n",
      "16\t702\t0.0\t1\t682 20\n",
      "17\t1137\t0.0\t1\t1107 30\n",
      "18\t656\t0.0\t1\t575 51 30\n",
      "19\t2325\t0.0\t1\t1323 990 12\n",
      "20\t8\t0.0\t2\t1 5 2\n",
      "21\t346\t0.0\t2\t0 340 6\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "53\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "65\t2\t0.0\t2\t1 1\n",
      "67\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t0 1\n",
      "109\t1\t0.0\t2\t0 1\n",
      "115\t1\t0.0\t2\t0 1\n",
      "191\t1\t0.0\t2\t1\n",
      "218\t3\t0.0\t2\t2 1\n",
      "235\t1\t0.0\t2\t1\n",
      "244\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.10.19_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.10.19_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.10.19_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.10.19_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.30 s (19 us/read; 3.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            169,835\n",
      "  Read 1 with adapter:                     134 (0.1%)\n",
      "  Read 2 with adapter:                  13,207 (7.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       169,835 (100.0%)\n",
      "\n",
      "Total basepairs processed:    77,124,817 bp\n",
      "  Read 1:    37,838,651 bp\n",
      "  Read 2:    39,286,166 bp\n",
      "Total written (filtered):     76,954,226 bp (99.8%)\n",
      "  Read 1:    37,837,123 bp\n",
      "  Read 2:    39,117,103 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 134 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t125\t2653.7\t0\t125\n",
      "24\t1\t0.0\t1\t0 1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "206\t3\t0.0\t1\t3\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13207 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t75\t2653.7\t0\t75\n",
      "4\t31\t663.4\t0\t31\n",
      "5\t73\t165.9\t0\t73\n",
      "6\t164\t41.5\t0\t164\n",
      "7\t467\t10.4\t0\t467\n",
      "8\t787\t2.6\t0\t787\n",
      "9\t2372\t0.6\t0\t2364 8\n",
      "10\t3075\t0.2\t1\t3033 42\n",
      "11\t153\t0.0\t1\t148 5\n",
      "12\t177\t0.0\t1\t170 7\n",
      "13\t235\t0.0\t1\t230 5\n",
      "14\t383\t0.0\t1\t360 23\n",
      "15\t311\t0.0\t1\t295 16\n",
      "16\t891\t0.0\t1\t863 28\n",
      "17\t1114\t0.0\t1\t1073 41\n",
      "18\t627\t0.0\t1\t535 62 30\n",
      "19\t2025\t0.0\t1\t1115 900 10\n",
      "20\t3\t0.0\t2\t0 3\n",
      "21\t215\t0.0\t2\t0 213 2\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "30\t1\t0.0\t2\t1\n",
      "45\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t3\t0.0\t2\t3\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "116\t1\t0.0\t2\t1\n",
      "189\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.12.19_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.12.19_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.12.19_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.12.19_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.56 s (17 us/read; 3.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            324,183\n",
      "  Read 1 with adapter:                     190 (0.1%)\n",
      "  Read 2 with adapter:                  20,119 (6.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       324,183 (100.0%)\n",
      "\n",
      "Total basepairs processed:   147,507,240 bp\n",
      "  Read 1:    72,459,301 bp\n",
      "  Read 2:    75,047,939 bp\n",
      "Total written (filtered):    147,238,354 bp (99.8%)\n",
      "  Read 1:    72,457,079 bp\n",
      "  Read 2:    74,781,275 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 190 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t176\t5065.4\t0\t176\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t3\t0.0\t1\t3\n",
      "37\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "211\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 20119 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t156\t5065.4\t0\t156\n",
      "4\t47\t1266.3\t0\t47\n",
      "5\t91\t316.6\t0\t91\n",
      "6\t202\t79.1\t0\t202\n",
      "7\t527\t19.8\t0\t527\n",
      "8\t1011\t4.9\t0\t1011\n",
      "9\t3344\t1.2\t0\t3335 9\n",
      "10\t4395\t0.3\t1\t4343 52\n",
      "11\t263\t0.1\t1\t257 6\n",
      "12\t304\t0.0\t1\t291 13\n",
      "13\t421\t0.0\t1\t403 18\n",
      "14\t699\t0.0\t1\t656 43\n",
      "15\t595\t0.0\t1\t560 35\n",
      "16\t1360\t0.0\t1\t1310 50\n",
      "17\t1664\t0.0\t1\t1592 72\n",
      "18\t1064\t0.0\t1\t932 80 52\n",
      "19\t3455\t0.0\t1\t1943 1490 22\n",
      "20\t9\t0.0\t2\t1 5 3\n",
      "21\t471\t0.0\t2\t0 463 8\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "30\t1\t0.0\t2\t1\n",
      "58\t2\t0.0\t2\t2\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t2\t0.0\t2\t1 1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "93\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t0 1\n",
      "172\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "217\t3\t0.0\t2\t2 1\n",
      "218\t1\t0.0\t2\t1\n",
      "242\t1\t0.0\t2\t1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O..fav.Tank.6.F.control.9.13.19_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O..fav.Tank.6.F.control.9.13.19_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O..fav.Tank.6.F.control.9.13.19_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O..fav.Tank.6.F.control.9.13.19_27_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.8.20.19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.8.20.19_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.8.20.19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.8.20.19_13_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.8.21.19_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.8.21.19_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.8.21.19_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.8.21.19_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.9.25.19_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.A.control.9.25.19_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.9.25.19_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.A.control.9.25.19_5_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.8.7.19_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.8.7.19_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.8.7.19_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.8.7.19_15_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.11.19_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.11.19_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.11.19_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.11.19_18_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.17.19_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.17.19_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.17.19_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.17.19_16_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.25.19_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.25.19_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.25.19_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.25.19_12_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.7.19_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.7.19_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.7.19_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.7.19_31_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.9.19_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.E.control.9.9.19_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.9.19_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.E.control.9.9.19_6_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.10.19_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.10.19_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.10.19_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.10.19_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.12.19_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.12.19_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.12.19_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.12.19_17_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.25.19_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.25.19_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.25.19_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.25.19_9_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.25.19_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.25.19_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.25.19_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.25.19_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.23 s (18 us/read; 3.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            177,158\n",
      "  Read 1 with adapter:                     108 (0.1%)\n",
      "  Read 2 with adapter:                   8,419 (4.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       177,158 (100.0%)\n",
      "\n",
      "Total basepairs processed:    80,639,426 bp\n",
      "  Read 1:    39,632,636 bp\n",
      "  Read 2:    41,006,790 bp\n",
      "Total written (filtered):     80,531,176 bp (99.9%)\n",
      "  Read 1:    39,631,738 bp\n",
      "  Read 2:    40,899,438 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t103\t2768.1\t0\t103\n",
      "18\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8419 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t2768.1\t0\t59\n",
      "4\t35\t692.0\t0\t35\n",
      "5\t53\t173.0\t0\t53\n",
      "6\t100\t43.3\t0\t100\n",
      "7\t371\t10.8\t0\t371\n",
      "8\t624\t2.7\t0\t624\n",
      "9\t2050\t0.7\t0\t2049 1\n",
      "10\t1590\t0.2\t1\t1567 23\n",
      "11\t31\t0.0\t1\t30 1\n",
      "12\t25\t0.0\t1\t24 1\n",
      "13\t33\t0.0\t1\t32 1\n",
      "14\t110\t0.0\t1\t101 9\n",
      "15\t127\t0.0\t1\t124 3\n",
      "16\t370\t0.0\t1\t354 16\n",
      "17\t622\t0.0\t1\t607 15\n",
      "18\t455\t0.0\t1\t417 25 13\n",
      "19\t1504\t0.0\t1\t931 566 7\n",
      "20\t5\t0.0\t2\t4 1\n",
      "21\t226\t0.0\t2\t0 224 2\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "23\t1\t0.0\t2\t1\n",
      "30\t3\t0.0\t2\t3\n",
      "46\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 1\n",
      "102\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.27.19_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.27.19_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.27.19_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.27.19_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.26 s (18 us/read; 3.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            233,793\n",
      "  Read 1 with adapter:                     297 (0.1%)\n",
      "  Read 2 with adapter:                  20,471 (8.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       233,793 (100.0%)\n",
      "\n",
      "Total basepairs processed:   106,750,756 bp\n",
      "  Read 1:    52,327,402 bp\n",
      "  Read 2:    54,423,354 bp\n",
      "Total written (filtered):    106,496,850 bp (99.8%)\n",
      "  Read 1:    52,326,003 bp\n",
      "  Read 2:    54,170,847 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 297 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t287\t3653.0\t0\t287\n",
      "14\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "64\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 20471 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t143\t3653.0\t0\t143\n",
      "4\t79\t913.3\t0\t79\n",
      "5\t108\t228.3\t0\t108\n",
      "6\t252\t57.1\t0\t252\n",
      "7\t941\t14.3\t0\t941\n",
      "8\t1502\t3.6\t0\t1502\n",
      "9\t4387\t0.9\t0\t4379 8\n",
      "10\t4459\t0.2\t1\t4385 74\n",
      "11\t282\t0.1\t1\t276 6\n",
      "12\t352\t0.0\t1\t346 6\n",
      "13\t484\t0.0\t1\t465 19\n",
      "14\t704\t0.0\t1\t665 39\n",
      "15\t462\t0.0\t1\t417 45\n",
      "16\t1193\t0.0\t1\t1136 57\n",
      "17\t1392\t0.0\t1\t1318 74\n",
      "18\t824\t0.0\t1\t713 63 48\n",
      "19\t2534\t0.0\t1\t1373 1143 18\n",
      "20\t6\t0.0\t2\t1 3 2\n",
      "21\t305\t0.0\t2\t0 297 8\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "52\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t3\t0.0\t2\t3\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t2\t0.0\t2\t1 1\n",
      "59\t2\t0.0\t2\t1 0 1\n",
      "60\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t6\t0.0\t2\t6\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t5\t0.0\t2\t2 2 1\n",
      "72\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "83\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t1\n",
      "108\t1\t0.0\t2\t1\n",
      "128\t1\t0.0\t2\t1\n",
      "145\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "157\t1\t0.0\t2\t1\n",
      "205\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "220\t1\t0.0\t2\t1\n",
      "230\t1\t0.0\t2\t0 1\n",
      "245\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.5.19_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.5.19_49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.5.19_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.5.19_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.64 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,795\n",
      "  Read 1 with adapter:                      46 (0.0%)\n",
      "  Read 2 with adapter:                   4,105 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       117,795 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,469,414 bp\n",
      "  Read 1:    26,300,628 bp\n",
      "  Read 2:    27,168,786 bp\n",
      "Total written (filtered):     53,410,705 bp (99.9%)\n",
      "  Read 1:    26,299,407 bp\n",
      "  Read 2:    27,111,298 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 46 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t1840.5\t0\t39\n",
      "13\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "200\t1\t0.0\t1\t0 1\n",
      "214\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t0 1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4105 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t1840.5\t0\t26\n",
      "4\t12\t460.1\t0\t12\n",
      "5\t18\t115.0\t0\t18\n",
      "6\t42\t28.8\t0\t42\n",
      "7\t105\t7.2\t0\t105\n",
      "8\t213\t1.8\t0\t213\n",
      "9\t713\t0.4\t0\t712 1\n",
      "10\t701\t0.1\t1\t693 8\n",
      "11\t31\t0.0\t1\t29 2\n",
      "12\t19\t0.0\t1\t18 1\n",
      "13\t19\t0.0\t1\t18 1\n",
      "14\t56\t0.0\t1\t50 6\n",
      "15\t77\t0.0\t1\t73 4\n",
      "16\t236\t0.0\t1\t225 11\n",
      "17\t424\t0.0\t1\t412 12\n",
      "18\t267\t0.0\t1\t252 12 3\n",
      "19\t975\t0.0\t1\t589 383 3\n",
      "20\t4\t0.0\t2\t0 3 1\n",
      "21\t150\t0.0\t2\t0 150\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "53\t1\t0.0\t2\t1\n",
      "147\t1\t0.0\t2\t1\n",
      "152\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.A.control.9.4.19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.A.control.9.4.19_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.A.control.9.4.19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.A.control.9.4.19_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.97 s (16 us/read; 3.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            242,178\n",
      "  Read 1 with adapter:                     225 (0.1%)\n",
      "  Read 2 with adapter:                  19,424 (8.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       242,178 (100.0%)\n",
      "\n",
      "Total basepairs processed:   110,349,032 bp\n",
      "  Read 1:    54,109,750 bp\n",
      "  Read 2:    56,239,282 bp\n",
      "Total written (filtered):    110,097,950 bp (99.8%)\n",
      "  Read 1:    54,107,385 bp\n",
      "  Read 2:    55,990,565 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 225 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t211\t3784.0\t0\t211\n",
      "25\t1\t0.0\t1\t0 1\n",
      "26\t5\t0.0\t1\t5\n",
      "138\t1\t0.0\t1\t1\n",
      "161\t1\t0.0\t1\t1\n",
      "207\t2\t0.0\t1\t2\n",
      "214\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 19424 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t119\t3784.0\t0\t119\n",
      "4\t68\t946.0\t0\t68\n",
      "5\t92\t236.5\t0\t92\n",
      "6\t220\t59.1\t0\t220\n",
      "7\t836\t14.8\t0\t836\n",
      "8\t1292\t3.7\t0\t1292\n",
      "9\t3998\t0.9\t0\t3986 12\n",
      "10\t4226\t0.2\t1\t4182 44\n",
      "11\t213\t0.1\t1\t208 5\n",
      "12\t299\t0.0\t1\t292 7\n",
      "13\t424\t0.0\t1\t403 21\n",
      "14\t653\t0.0\t1\t615 38\n",
      "15\t442\t0.0\t1\t417 25\n",
      "16\t1153\t0.0\t1\t1119 34\n",
      "17\t1384\t0.0\t1\t1322 62\n",
      "18\t867\t0.0\t1\t748 72 47\n",
      "19\t2695\t0.0\t1\t1504 1180 11\n",
      "20\t6\t0.0\t2\t0 6\n",
      "21\t324\t0.0\t2\t0 321 3\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "46\t1\t0.0\t2\t1\n",
      "49\t3\t0.0\t2\t3\n",
      "52\t2\t0.0\t2\t2\n",
      "53\t5\t0.0\t2\t4 1\n",
      "54\t9\t0.0\t2\t8 0 1\n",
      "55\t7\t0.0\t2\t7\n",
      "56\t3\t0.0\t2\t1 2\n",
      "57\t7\t0.0\t2\t2 3 2\n",
      "58\t9\t0.0\t2\t6 2 1\n",
      "59\t4\t0.0\t2\t3 0 1\n",
      "60\t3\t0.0\t2\t3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 1\n",
      "63\t2\t0.0\t2\t1 0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t1 1\n",
      "67\t3\t0.0\t2\t3\n",
      "69\t1\t0.0\t2\t1\n",
      "72\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t0 1\n",
      "107\t2\t0.0\t2\t2\n",
      "109\t1\t0.0\t2\t1\n",
      "115\t2\t0.0\t2\t1 0 1\n",
      "147\t1\t0.0\t2\t1\n",
      "171\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "225\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "240\t1\t0.0\t2\t0 1\n",
      "241\t1\t0.0\t2\t1\n",
      "242\t4\t0.0\t2\t4\n",
      "243\t2\t0.0\t2\t2\n",
      "244\t3\t0.0\t2\t2 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.8.10.19_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.8.10.19_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.8.10.19_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.8.10.19_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.18 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,258\n",
      "  Read 1 with adapter:                      72 (0.1%)\n",
      "  Read 2 with adapter:                   7,616 (7.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        96,258 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,488,253 bp\n",
      "  Read 1:    21,312,413 bp\n",
      "  Read 2:    22,175,840 bp\n",
      "Total written (filtered):     43,384,005 bp (99.8%)\n",
      "  Read 1:    21,311,265 bp\n",
      "  Read 2:    22,072,740 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 72 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t1504.0\t0\t58\n",
      "20\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t0 1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t6\t0.0\t1\t6\n",
      "141\t1\t0.0\t1\t1\n",
      "151\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7616 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t41\t1504.0\t0\t41\n",
      "4\t15\t376.0\t0\t15\n",
      "5\t18\t94.0\t0\t18\n",
      "6\t43\t23.5\t0\t43\n",
      "7\t212\t5.9\t0\t212\n",
      "8\t347\t1.5\t0\t347\n",
      "9\t1097\t0.4\t0\t1095 2\n",
      "10\t1680\t0.1\t1\t1659 21\n",
      "11\t114\t0.0\t1\t108 6\n",
      "12\t193\t0.0\t1\t184 9\n",
      "13\t284\t0.0\t1\t273 11\n",
      "14\t387\t0.0\t1\t359 28\n",
      "15\t226\t0.0\t1\t210 16\n",
      "16\t570\t0.0\t1\t544 26\n",
      "17\t658\t0.0\t1\t619 39\n",
      "18\t370\t0.0\t1\t312 43 15\n",
      "19\t1154\t0.0\t1\t604 546 4\n",
      "20\t2\t0.0\t2\t0 2\n",
      "21\t164\t0.0\t2\t0 158 6\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "30\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "54\t2\t0.0\t2\t2\n",
      "55\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "72\t1\t0.0\t2\t1\n",
      "73\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t1 1 1\n",
      "120\t1\t0.0\t2\t1\n",
      "124\t1\t0.0\t2\t1\n",
      "151\t1\t0.0\t2\t1\n",
      "172\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t3\n",
      "217\t1\t0.0\t2\t0 1\n",
      "221\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.10.19_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.10.19_50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.10.19_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.10.19_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.34 s (32 us/read; 1.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,908\n",
      "  Read 1 with adapter:                      29 (0.1%)\n",
      "  Read 2 with adapter:                   2,037 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,908 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,836,940 bp\n",
      "  Read 1:     9,240,762 bp\n",
      "  Read 2:     9,596,178 bp\n",
      "Total written (filtered):     18,806,646 bp (99.8%)\n",
      "  Read 1:     9,240,030 bp\n",
      "  Read 2:     9,566,616 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t654.8\t0\t26\n",
      "218\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2037 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t654.8\t0\t7\n",
      "4\t1\t163.7\t0\t1\n",
      "5\t4\t40.9\t0\t4\n",
      "6\t6\t10.2\t0\t6\n",
      "7\t30\t2.6\t0\t30\n",
      "8\t47\t0.6\t0\t47\n",
      "9\t178\t0.2\t0\t178\n",
      "10\t319\t0.0\t1\t313 6\n",
      "11\t51\t0.0\t1\t51\n",
      "12\t71\t0.0\t1\t65 6\n",
      "13\t99\t0.0\t1\t95 4\n",
      "14\t114\t0.0\t1\t110 4\n",
      "15\t88\t0.0\t1\t78 10\n",
      "16\t204\t0.0\t1\t194 10\n",
      "17\t242\t0.0\t1\t225 17\n",
      "18\t125\t0.0\t1\t103 12 10\n",
      "19\t404\t0.0\t1\t251 149 4\n",
      "21\t46\t0.0\t2\t0 46\n",
      "217\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.12.19_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.12.19_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.12.19_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.12.19_4_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.62 s (20 us/read; 3.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,819\n",
      "  Read 1 with adapter:                      96 (0.1%)\n",
      "  Read 2 with adapter:                   8,845 (6.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       130,819 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,293,761 bp\n",
      "  Read 1:    29,078,158 bp\n",
      "  Read 2:    30,215,603 bp\n",
      "Total written (filtered):     59,172,774 bp (99.8%)\n",
      "  Read 1:    29,076,447 bp\n",
      "  Read 2:    30,096,327 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 96 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t87\t2044.0\t0\t87\n",
      "6\t1\t31.9\t0\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t1\n",
      "186\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8845 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t2044.0\t0\t53\n",
      "4\t24\t511.0\t0\t24\n",
      "5\t31\t127.8\t0\t31\n",
      "6\t79\t31.9\t0\t79\n",
      "7\t229\t8.0\t0\t229\n",
      "8\t410\t2.0\t0\t410\n",
      "9\t1301\t0.5\t0\t1300 1\n",
      "10\t1779\t0.1\t1\t1763 16\n",
      "11\t132\t0.0\t1\t126 6\n",
      "12\t179\t0.0\t1\t172 7\n",
      "13\t260\t0.0\t1\t255 5\n",
      "14\t372\t0.0\t1\t346 26\n",
      "15\t289\t0.0\t1\t272 17\n",
      "16\t695\t0.0\t1\t668 27\n",
      "17\t852\t0.0\t1\t816 36\n",
      "18\t492\t0.0\t1\t422 45 25\n",
      "19\t1463\t0.0\t1\t799 657 7\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t176\t0.0\t2\t0 171 5\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "30\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t2\n",
      "70\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "110\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "124\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.15.19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.15.19_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.15.19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.15.19_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.91 s (18 us/read; 3.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            157,522\n",
      "  Read 1 with adapter:                     145 (0.1%)\n",
      "  Read 2 with adapter:                  12,564 (8.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       157,522 (100.0%)\n",
      "\n",
      "Total basepairs processed:    71,724,690 bp\n",
      "  Read 1:    35,157,085 bp\n",
      "  Read 2:    36,567,605 bp\n",
      "Total written (filtered):     71,561,729 bp (99.8%)\n",
      "  Read 1:    35,156,017 bp\n",
      "  Read 2:    36,405,712 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 145 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t137\t2461.3\t0\t137\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "101\t5\t0.0\t1\t5\n",
      "102\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12564 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t62\t2461.3\t0\t62\n",
      "4\t23\t615.3\t0\t23\n",
      "5\t35\t153.8\t0\t35\n",
      "6\t99\t38.5\t0\t99\n",
      "7\t344\t9.6\t0\t344\n",
      "8\t597\t2.4\t0\t597\n",
      "9\t2163\t0.6\t0\t2154 9\n",
      "10\t3167\t0.2\t1\t3139 28\n",
      "11\t231\t0.0\t1\t223 8\n",
      "12\t301\t0.0\t1\t294 7\n",
      "13\t404\t0.0\t1\t389 15\n",
      "14\t533\t0.0\t1\t511 22\n",
      "15\t359\t0.0\t1\t334 25\n",
      "16\t867\t0.0\t1\t819 48\n",
      "17\t932\t0.0\t1\t874 58\n",
      "18\t579\t0.0\t1\t507 38 34\n",
      "19\t1630\t0.0\t1\t921 705 4\n",
      "20\t3\t0.0\t2\t0 1 2\n",
      "21\t187\t0.0\t2\t0 185 2\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "46\t1\t0.0\t2\t1\n",
      "57\t2\t0.0\t2\t1 1\n",
      "58\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t2\t0.0\t2\t2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t1 0 1\n",
      "67\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t1 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t1\n",
      "72\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t1\n",
      "86\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "97\t1\t0.0\t2\t1\n",
      "105\t1\t0.0\t2\t1\n",
      "107\t1\t0.0\t2\t1\n",
      "132\t1\t0.0\t2\t1\n",
      "137\t2\t0.0\t2\t1 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 2\n",
      "214\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.16.19_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.16.19_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.16.19_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.16.19_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.32 s (14 us/read; 4.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            303,205\n",
      "  Read 1 with adapter:                     263 (0.1%)\n",
      "  Read 2 with adapter:                  22,086 (7.3%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       303,204 (100.0%)\n",
      "\n",
      "Total basepairs processed:   138,371,628 bp\n",
      "  Read 1:    67,864,116 bp\n",
      "  Read 2:    70,507,512 bp\n",
      "Total written (filtered):    138,074,489 bp (99.8%)\n",
      "  Read 1:    67,862,551 bp\n",
      "  Read 2:    70,211,938 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 263 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t252\t4737.6\t0\t252\n",
      "20\t2\t0.0\t1\t2\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "58\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "92\t2\t0.0\t1\t2\n",
      "111\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 22086 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t135\t4737.6\t0\t135\n",
      "4\t42\t1184.4\t0\t42\n",
      "5\t63\t296.1\t0\t63\n",
      "6\t149\t74.0\t0\t149\n",
      "7\t572\t18.5\t0\t572\n",
      "8\t847\t4.6\t0\t847\n",
      "9\t3208\t1.2\t0\t3203 5\n",
      "10\t5007\t0.3\t1\t4953 54\n",
      "11\t428\t0.1\t1\t417 11\n",
      "12\t636\t0.0\t1\t614 22\n",
      "13\t861\t0.0\t1\t833 28\n",
      "14\t1155\t0.0\t1\t1106 49\n",
      "15\t679\t0.0\t1\t634 45\n",
      "16\t1712\t0.0\t1\t1634 78\n",
      "17\t1874\t0.0\t1\t1792 82\n",
      "18\t1052\t0.0\t1\t909 85 58\n",
      "19\t3175\t0.0\t1\t1703 1451 21\n",
      "20\t10\t0.0\t2\t1 8 1\n",
      "21\t397\t0.0\t2\t0 389 8\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "47\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "58\t2\t0.0\t2\t1 0 1\n",
      "62\t1\t0.0\t2\t1\n",
      "64\t3\t0.0\t2\t3\n",
      "65\t3\t0.0\t2\t3\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t3\t0.0\t2\t3\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t3\t0.0\t2\t3\n",
      "71\t2\t0.0\t2\t1 0 1\n",
      "72\t2\t0.0\t2\t1 0 1\n",
      "73\t2\t0.0\t2\t2\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t1 1\n",
      "79\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t1\n",
      "164\t1\t0.0\t2\t1\n",
      "174\t2\t0.0\t2\t2\n",
      "197\t1\t0.0\t2\t1\n",
      "204\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "240\t1\t0.0\t2\t1\n",
      "242\t4\t0.0\t2\t4\n",
      "243\t1\t0.0\t2\t1\n",
      "247\t1\t0.0\t2\t0 1\n",
      "249\t2\t0.0\t2\t1 0 1\n",
      "250\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.18.19_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.18.19_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.18.19_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.18.19_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.81 s (20 us/read; 2.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            139,236\n",
      "  Read 1 with adapter:                     133 (0.1%)\n",
      "  Read 2 with adapter:                   6,936 (5.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       139,236 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,293,857 bp\n",
      "  Read 1:    31,082,210 bp\n",
      "  Read 2:    32,211,647 bp\n",
      "Total written (filtered):     63,191,810 bp (99.8%)\n",
      "  Read 1:    31,081,095 bp\n",
      "  Read 2:    32,110,715 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 133 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t126\t2175.6\t0\t126\n",
      "6\t1\t34.0\t0\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t1\n",
      "173\t1\t0.0\t1\t0 1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6936 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t2175.6\t0\t27\n",
      "4\t7\t543.9\t0\t7\n",
      "5\t14\t136.0\t0\t14\n",
      "6\t30\t34.0\t0\t30\n",
      "7\t133\t8.5\t0\t133\n",
      "8\t229\t2.1\t0\t229\n",
      "9\t707\t0.5\t0\t705 2\n",
      "10\t1165\t0.1\t1\t1153 12\n",
      "11\t115\t0.0\t1\t112 3\n",
      "12\t200\t0.0\t1\t193 7\n",
      "13\t258\t0.0\t1\t250 8\n",
      "14\t339\t0.0\t1\t324 15\n",
      "15\t219\t0.0\t1\t208 11\n",
      "16\t577\t0.0\t1\t556 21\n",
      "17\t805\t0.0\t1\t773 32\n",
      "18\t454\t0.0\t1\t409 31 14\n",
      "19\t1434\t0.0\t1\t839 590 5\n",
      "20\t5\t0.0\t2\t0 4 1\n",
      "21\t193\t0.0\t2\t0 189 4\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "51\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "57\t2\t0.0\t2\t2\n",
      "58\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "86\t2\t0.0\t2\t1 0 1\n",
      "87\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t1\n",
      "213\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "243\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.8.21.19_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.8.21.19_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.8.21.19_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.8.21.19_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.97 s (18 us/read; 3.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            162,221\n",
      "  Read 1 with adapter:                      87 (0.1%)\n",
      "  Read 2 with adapter:                  11,616 (7.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       162,221 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,574,175 bp\n",
      "  Read 1:    36,112,165 bp\n",
      "  Read 2:    37,462,010 bp\n",
      "Total written (filtered):     73,424,470 bp (99.8%)\n",
      "  Read 1:    36,110,378 bp\n",
      "  Read 2:    37,314,092 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 87 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t70\t2534.7\t0\t70\n",
      "10\t1\t0.2\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t4\t0.0\t1\t4\n",
      "116\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t3 1\n",
      "219\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11616 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t2534.7\t0\t79\n",
      "4\t54\t633.7\t0\t54\n",
      "5\t82\t158.4\t0\t82\n",
      "6\t182\t39.6\t0\t182\n",
      "7\t455\t9.9\t0\t455\n",
      "8\t796\t2.5\t0\t796\n",
      "9\t2460\t0.6\t0\t2449 11\n",
      "10\t2175\t0.2\t1\t2161 14\n",
      "11\t111\t0.0\t1\t106 5\n",
      "12\t124\t0.0\t1\t121 3\n",
      "13\t187\t0.0\t1\t183 4\n",
      "14\t290\t0.0\t1\t278 12\n",
      "15\t253\t0.0\t1\t243 10\n",
      "16\t699\t0.0\t1\t672 27\n",
      "17\t958\t0.0\t1\t929 29\n",
      "18\t607\t0.0\t1\t532 49 26\n",
      "19\t1858\t0.0\t1\t1029 816 13\n",
      "20\t2\t0.0\t2\t0 2\n",
      "21\t220\t0.0\t2\t0 218 2\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "30\t2\t0.0\t2\t2\n",
      "44\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t2\n",
      "86\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.8.26.19_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.8.26.19_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.8.26.19_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.8.26.19_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.43 s (17 us/read; 3.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            440,183\n",
      "  Read 1 with adapter:                     159 (0.0%)\n",
      "  Read 2 with adapter:                  29,907 (6.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       440,183 (100.0%)\n",
      "\n",
      "Total basepairs processed:   200,533,120 bp\n",
      "  Read 1:    98,462,634 bp\n",
      "  Read 2:   102,070,486 bp\n",
      "Total written (filtered):    200,153,185 bp (99.8%)\n",
      "  Read 1:    98,461,085 bp\n",
      "  Read 2:   101,692,100 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 159 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t140\t6877.9\t0\t140\n",
      "6\t1\t107.5\t0\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t7\t0.0\t1\t7\n",
      "27\t2\t0.0\t1\t2\n",
      "34\t1\t0.0\t1\t1\n",
      "35\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 29907 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t219\t6877.9\t0\t219\n",
      "4\t112\t1719.5\t0\t112\n",
      "5\t186\t429.9\t0\t186\n",
      "6\t433\t107.5\t0\t433\n",
      "7\t1199\t26.9\t0\t1199\n",
      "8\t2065\t6.7\t0\t2065\n",
      "9\t6415\t1.7\t0\t6402 13\n",
      "10\t6030\t0.4\t1\t5959 71\n",
      "11\t270\t0.1\t1\t263 7\n",
      "12\t403\t0.0\t1\t399 4\n",
      "13\t490\t0.0\t1\t468 22\n",
      "14\t740\t0.0\t1\t693 47\n",
      "15\t624\t0.0\t1\t593 31\n",
      "16\t1627\t0.0\t1\t1571 56\n",
      "17\t2386\t0.0\t1\t2292 94\n",
      "18\t1470\t0.0\t1\t1287 108 75\n",
      "19\t4518\t0.0\t1\t2542 1957 19\n",
      "20\t15\t0.0\t2\t1 7 7\n",
      "21\t640\t0.0\t2\t0 628 12\n",
      "22\t32\t0.0\t2\t0 1 31\n",
      "30\t1\t0.0\t2\t1\n",
      "44\t1\t0.0\t2\t1\n",
      "47\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "54\t2\t0.0\t2\t2\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "63\t4\t0.0\t2\t4\n",
      "64\t2\t0.0\t2\t2\n",
      "65\t2\t0.0\t2\t2\n",
      "69\t2\t0.0\t2\t1 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t1\n",
      "74\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "131\t1\t0.0\t2\t1\n",
      "210\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "223\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "237\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.9.4.19_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.9.4.19_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.9.4.19_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.9.4.19_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 8.06 s (16 us/read; 3.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            515,946\n",
      "  Read 1 with adapter:                     339 (0.1%)\n",
      "  Read 2 with adapter:                  32,783 (6.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       515,946 (100.0%)\n",
      "\n",
      "Total basepairs processed:   235,044,444 bp\n",
      "  Read 1:   115,450,941 bp\n",
      "  Read 2:   119,593,503 bp\n",
      "Total written (filtered):    234,621,612 bp (99.8%)\n",
      "  Read 1:   115,447,928 bp\n",
      "  Read 2:   119,173,684 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 339 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t313\t8061.7\t0\t313\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "18\t1\t0.0\t1\t0 1\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t8\t0.0\t1\t8\n",
      "27\t2\t0.0\t1\t2\n",
      "38\t2\t0.0\t1\t2\n",
      "92\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t2\n",
      "222\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 32783 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t255\t8061.7\t0\t255\n",
      "4\t134\t2015.4\t0\t134\n",
      "5\t204\t503.9\t0\t204\n",
      "6\t458\t126.0\t0\t458\n",
      "7\t1104\t31.5\t0\t1104\n",
      "8\t2069\t7.9\t0\t2069\n",
      "9\t6752\t2.0\t0\t6741 11\n",
      "10\t6811\t0.5\t1\t6717 94\n",
      "11\t290\t0.1\t1\t287 3\n",
      "12\t335\t0.0\t1\t331 4\n",
      "13\t470\t0.0\t1\t454 16\n",
      "14\t782\t0.0\t1\t725 57\n",
      "15\t730\t0.0\t1\t698 32\n",
      "16\t1855\t0.0\t1\t1796 59\n",
      "17\t2736\t0.0\t1\t2636 100\n",
      "18\t1618\t0.0\t1\t1431 129 58\n",
      "19\t5398\t0.0\t1\t3069 2302 27\n",
      "20\t13\t0.0\t2\t3 5 5\n",
      "21\t707\t0.0\t2\t0 697 10\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "30\t2\t0.0\t2\t2\n",
      "46\t1\t0.0\t2\t0 1\n",
      "49\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "54\t2\t0.0\t2\t1 1\n",
      "55\t1\t0.0\t2\t0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t2\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t2\t0.0\t2\t2\n",
      "226\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "241\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.11.19_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.11.19_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.11.19_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.11.19_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.39 s (17 us/read; 3.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            196,529\n",
      "  Read 1 with adapter:                     159 (0.1%)\n",
      "  Read 2 with adapter:                  12,516 (6.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       196,529 (100.0%)\n",
      "\n",
      "Total basepairs processed:    89,551,034 bp\n",
      "  Read 1:    43,943,658 bp\n",
      "  Read 2:    45,607,376 bp\n",
      "Total written (filtered):     89,381,706 bp (99.8%)\n",
      "  Read 1:    43,942,406 bp\n",
      "  Read 2:    45,439,300 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 159 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t149\t3070.8\t0\t149\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t5\t0.0\t1\t5\n",
      "198\t1\t0.0\t1\t0 1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12516 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t68\t3070.8\t0\t68\n",
      "4\t34\t767.7\t0\t34\n",
      "5\t53\t191.9\t0\t53\n",
      "6\t122\t48.0\t0\t122\n",
      "7\t370\t12.0\t0\t370\n",
      "8\t560\t3.0\t0\t560\n",
      "9\t2037\t0.7\t0\t2026 11\n",
      "10\t2597\t0.2\t1\t2565 32\n",
      "11\t172\t0.0\t1\t167 5\n",
      "12\t257\t0.0\t1\t252 5\n",
      "13\t348\t0.0\t1\t333 15\n",
      "14\t500\t0.0\t1\t460 40\n",
      "15\t358\t0.0\t1\t340 18\n",
      "16\t873\t0.0\t1\t823 50\n",
      "17\t1071\t0.0\t1\t1029 42\n",
      "18\t687\t0.0\t1\t588 68 31\n",
      "19\t2072\t0.0\t1\t1142 916 14\n",
      "20\t7\t0.0\t2\t1 4 2\n",
      "21\t281\t0.0\t2\t0 275 6\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "23\t1\t0.0\t2\t0 1\n",
      "46\t1\t0.0\t2\t0 1\n",
      "54\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t0 1\n",
      "67\t3\t0.0\t2\t3\n",
      "71\t1\t0.0\t2\t1\n",
      "72\t2\t0.0\t2\t2\n",
      "79\t1\t0.0\t2\t1\n",
      "80\t1\t0.0\t2\t1\n",
      "90\t3\t0.0\t2\t2 1\n",
      "93\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "97\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t1\n",
      "102\t1\t0.0\t2\t1\n",
      "156\t1\t0.0\t2\t1\n",
      "164\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "218\t1\t0.0\t2\t0 0 1\n",
      "226\t1\t0.0\t2\t1\n",
      "230\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.27.19_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.27.19_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.27.19_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.27.19_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.5.19_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.16.G.control.8.5.19_49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.5.19_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.16.G.control.8.5.19_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.A.control.9.4.19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.A.control.9.4.19_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.A.control.9.4.19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.A.control.9.4.19_30_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.8.10.19_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.8.10.19_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.8.10.19_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.8.10.19_11_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.10.19_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.10.19_50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.10.19_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.10.19_50_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.12.19_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.12.19_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.12.19_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.12.19_4_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.15.19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.15.19_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.15.19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.15.19_28_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.16.19_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.16.19_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.16.19_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.16.19_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.18.19_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.D.control.9.18.19_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.18.19_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.D.control.9.18.19_33_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.8.21.19_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.8.21.19_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.8.21.19_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.8.21.19_32_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.8.26.19_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.8.26.19_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.8.26.19_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.8.26.19_26_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.9.4.19_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.18.G.control.9.4.19_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.9.4.19_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.18.G.control.9.4.19_21_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.11.19_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.11.19_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.11.19_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.11.19_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.13.19_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.13.19_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.13.19_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.13.19_14_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.13.19_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.13.19_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.13.19_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.13.19_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.48 s (17 us/read; 3.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            210,580\n",
      "  Read 1 with adapter:                     142 (0.1%)\n",
      "  Read 2 with adapter:                  14,232 (6.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       210,580 (100.0%)\n",
      "\n",
      "Total basepairs processed:    96,022,382 bp\n",
      "  Read 1:    47,104,295 bp\n",
      "  Read 2:    48,918,087 bp\n",
      "Total written (filtered):     95,834,194 bp (99.8%)\n",
      "  Read 1:    47,102,759 bp\n",
      "  Read 2:    48,731,435 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 142 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t128\t3290.3\t0\t128\n",
      "5\t1\t205.6\t0\t1\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t1\n",
      "84\t2\t0.0\t1\t2\n",
      "120\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14232 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t3290.3\t0\t77\n",
      "4\t50\t822.6\t0\t50\n",
      "5\t57\t205.6\t0\t57\n",
      "6\t141\t51.4\t0\t141\n",
      "7\t511\t12.9\t0\t511\n",
      "8\t737\t3.2\t0\t737\n",
      "9\t2591\t0.8\t0\t2584 7\n",
      "10\t2882\t0.2\t1\t2847 35\n",
      "11\t187\t0.1\t1\t182 5\n",
      "12\t302\t0.0\t1\t294 8\n",
      "13\t416\t0.0\t1\t406 10\n",
      "14\t545\t0.0\t1\t513 32\n",
      "15\t372\t0.0\t1\t349 23\n",
      "16\t901\t0.0\t1\t874 27\n",
      "17\t1151\t0.0\t1\t1107 44\n",
      "18\t712\t0.0\t1\t623 56 33\n",
      "19\t2244\t0.0\t1\t1240 988 16\n",
      "20\t4\t0.0\t2\t3 1\n",
      "21\t301\t0.0\t2\t0 296 5\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "30\t4\t0.0\t2\t4\n",
      "43\t1\t0.0\t2\t0 1\n",
      "46\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "62\t2\t0.0\t2\t1 0 1\n",
      "63\t3\t0.0\t2\t2 1\n",
      "64\t2\t0.0\t2\t2\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t3\t0.0\t2\t3\n",
      "68\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "72\t1\t0.0\t2\t1\n",
      "73\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "92\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "220\t1\t0.0\t2\t1\n",
      "223\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "242\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.3.19_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.3.19_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.3.19_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.3.19_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.46 s (20 us/read; 3.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,239\n",
      "  Read 1 with adapter:                      59 (0.1%)\n",
      "  Read 2 with adapter:                   1,457 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        73,239 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,333,607 bp\n",
      "  Read 1:    16,394,428 bp\n",
      "  Read 2:    16,939,179 bp\n",
      "Total written (filtered):     33,310,862 bp (99.9%)\n",
      "  Read 1:    16,393,987 bp\n",
      "  Read 2:    16,916,875 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 59 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t55\t1144.4\t0\t55\n",
      "20\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "202\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1457 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1144.4\t0\t8\n",
      "5\t6\t71.5\t0\t6\n",
      "6\t5\t17.9\t0\t5\n",
      "7\t24\t4.5\t0\t24\n",
      "8\t40\t1.1\t0\t40\n",
      "9\t161\t0.3\t0\t160 1\n",
      "10\t225\t0.1\t1\t223 2\n",
      "11\t7\t0.0\t1\t7\n",
      "12\t8\t0.0\t1\t8\n",
      "13\t12\t0.0\t1\t12\n",
      "14\t14\t0.0\t1\t14\n",
      "15\t31\t0.0\t1\t31\n",
      "16\t71\t0.0\t1\t68 3\n",
      "17\t178\t0.0\t1\t171 7\n",
      "18\t138\t0.0\t1\t122 13 3\n",
      "19\t449\t0.0\t1\t300 149\n",
      "20\t8\t0.0\t2\t7 0 1\n",
      "21\t69\t0.0\t2\t0 68 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "213\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.5.19_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.5.19_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.5.19_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.5.19_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.76 s (17 us/read; 3.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            227,406\n",
      "  Read 1 with adapter:                     167 (0.1%)\n",
      "  Read 2 with adapter:                  16,885 (7.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       227,406 (100.0%)\n",
      "\n",
      "Total basepairs processed:   103,606,957 bp\n",
      "  Read 1:    50,813,717 bp\n",
      "  Read 2:    52,793,240 bp\n",
      "Total written (filtered):    103,381,660 bp (99.8%)\n",
      "  Read 1:    50,812,456 bp\n",
      "  Read 2:    52,569,204 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 167 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t153\t3553.2\t0\t153\n",
      "18\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t5\t0.0\t1\t5\n",
      "116\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t0 1\n",
      "213\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 16885 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t99\t3553.2\t0\t99\n",
      "4\t23\t888.3\t0\t23\n",
      "5\t72\t222.1\t0\t72\n",
      "6\t123\t55.5\t0\t123\n",
      "7\t434\t13.9\t0\t434\n",
      "8\t747\t3.5\t0\t747\n",
      "9\t2594\t0.9\t0\t2583 11\n",
      "10\t4252\t0.2\t1\t4208 44\n",
      "11\t272\t0.1\t1\t265 7\n",
      "12\t333\t0.0\t1\t318 15\n",
      "13\t459\t0.0\t1\t444 15\n",
      "14\t685\t0.0\t1\t652 33\n",
      "15\t498\t0.0\t1\t462 36\n",
      "16\t1233\t0.0\t1\t1190 43\n",
      "17\t1330\t0.0\t1\t1270 60\n",
      "18\t789\t0.0\t1\t678 52 59\n",
      "19\t2557\t0.0\t1\t1333 1210 14\n",
      "20\t12\t0.0\t2\t6 4 2\n",
      "21\t310\t0.0\t2\t0 305 5\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "30\t1\t0.0\t2\t1\n",
      "38\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "60\t2\t0.0\t2\t2\n",
      "61\t3\t0.0\t2\t1 0 2\n",
      "64\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t0 1\n",
      "71\t1\t0.0\t2\t0 1\n",
      "72\t2\t0.0\t2\t1 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t1\n",
      "82\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "91\t1\t0.0\t2\t1\n",
      "92\t2\t0.0\t2\t1 1\n",
      "93\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "99\t2\t0.0\t2\t2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t1\n",
      "112\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "119\t1\t0.0\t2\t1\n",
      "130\t1\t0.0\t2\t0 1\n",
      "133\t1\t0.0\t2\t1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "183\t1\t0.0\t2\t1\n",
      "193\t1\t0.0\t2\t1\n",
      "206\t1\t0.0\t2\t1\n",
      "218\t2\t0.0\t2\t2\n",
      "223\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "245\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.2.19_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.2.19_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.2.19_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.2.19_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.87 s (14 us/read; 4.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            336,317\n",
      "  Read 1 with adapter:                     283 (0.1%)\n",
      "  Read 2 with adapter:                  20,884 (6.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       336,317 (100.0%)\n",
      "\n",
      "Total basepairs processed:   153,367,367 bp\n",
      "  Read 1:    75,270,016 bp\n",
      "  Read 2:    78,097,351 bp\n",
      "Total written (filtered):    153,091,939 bp (99.8%)\n",
      "  Read 1:    75,268,792 bp\n",
      "  Read 2:    77,823,147 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 283 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t269\t5255.0\t0\t269\n",
      "13\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "25\t5\t0.0\t1\t5\n",
      "26\t6\t0.0\t1\t6\n",
      "101\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 20884 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t126\t5255.0\t0\t126\n",
      "4\t75\t1313.7\t0\t75\n",
      "5\t87\t328.4\t0\t87\n",
      "6\t188\t82.1\t0\t188\n",
      "7\t666\t20.5\t0\t666\n",
      "8\t1133\t5.1\t0\t1133\n",
      "9\t3899\t1.3\t0\t3896 3\n",
      "10\t4297\t0.3\t1\t4264 33\n",
      "11\t273\t0.1\t1\t265 8\n",
      "12\t353\t0.0\t1\t343 10\n",
      "13\t458\t0.0\t1\t442 16\n",
      "14\t663\t0.0\t1\t624 39\n",
      "15\t486\t0.0\t1\t459 27\n",
      "16\t1227\t0.0\t1\t1194 33\n",
      "17\t1875\t0.0\t1\t1805 70\n",
      "18\t1166\t0.0\t1\t1038 78 50\n",
      "19\t3370\t0.0\t1\t1891 1463 16\n",
      "20\t13\t0.0\t2\t1 7 5\n",
      "21\t464\t0.0\t2\t0 463 1\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "30\t2\t0.0\t2\t2\n",
      "45\t1\t0.0\t2\t1\n",
      "47\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t1\n",
      "51\t2\t0.0\t2\t2\n",
      "53\t1\t0.0\t2\t1\n",
      "56\t2\t0.0\t2\t2\n",
      "59\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t2\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t1 1\n",
      "152\t1\t0.0\t2\t1\n",
      "190\t1\t0.0\t2\t1\n",
      "193\t1\t0.0\t2\t1\n",
      "208\t2\t0.0\t2\t1 0 1\n",
      "214\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "218\t3\t0.0\t2\t2 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.3.19_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.3.19_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.3.19_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.3.19_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.78 s (16 us/read; 3.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            487,589\n",
      "  Read 1 with adapter:                     433 (0.1%)\n",
      "  Read 2 with adapter:                  30,973 (6.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       487,589 (100.0%)\n",
      "\n",
      "Total basepairs processed:   222,209,946 bp\n",
      "  Read 1:   109,072,167 bp\n",
      "  Read 2:   113,137,779 bp\n",
      "Total written (filtered):    221,781,203 bp (99.8%)\n",
      "  Read 1:   109,070,121 bp\n",
      "  Read 2:   112,711,082 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 433 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t420\t7618.6\t0\t420\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t3\t0.0\t1\t3\n",
      "111\t1\t0.0\t1\t1\n",
      "207\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 30973 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t186\t7618.6\t0\t186\n",
      "4\t65\t1904.6\t0\t65\n",
      "5\t118\t476.2\t0\t118\n",
      "6\t282\t119.0\t0\t282\n",
      "7\t838\t29.8\t0\t838\n",
      "8\t1432\t7.4\t0\t1432\n",
      "9\t4868\t1.9\t0\t4849 19\n",
      "10\t6428\t0.5\t1\t6353 75\n",
      "11\t418\t0.1\t1\t411 7\n",
      "12\t532\t0.0\t1\t521 11\n",
      "13\t813\t0.0\t1\t775 38\n",
      "14\t1234\t0.0\t1\t1170 64\n",
      "15\t896\t0.0\t1\t837 59\n",
      "16\t2154\t0.0\t1\t2059 95\n",
      "17\t2863\t0.0\t1\t2740 123\n",
      "18\t1668\t0.0\t1\t1462 116 90\n",
      "19\t5301\t0.0\t1\t2860 2414 27\n",
      "20\t14\t0.0\t2\t1 9 4\n",
      "21\t674\t0.0\t2\t0 667 7\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "33\t1\t0.0\t2\t0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t0 1\n",
      "57\t1\t0.0\t2\t0 1\n",
      "58\t2\t0.0\t2\t0 1 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t4\t0.0\t2\t2 0 2\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t1\n",
      "69\t3\t0.0\t2\t0 2 1\n",
      "71\t2\t0.0\t2\t2\n",
      "72\t1\t0.0\t2\t1\n",
      "73\t2\t0.0\t2\t1 1\n",
      "75\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "81\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t0 1\n",
      "115\t99\t0.0\t2\t77 19 3\n",
      "124\t2\t0.0\t2\t2\n",
      "128\t2\t0.0\t2\t2\n",
      "129\t1\t0.0\t2\t1\n",
      "131\t3\t0.0\t2\t2 1\n",
      "132\t3\t0.0\t2\t3\n",
      "134\t1\t0.0\t2\t1\n",
      "139\t1\t0.0\t2\t0 1\n",
      "158\t1\t0.0\t2\t1\n",
      "170\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t0 1\n",
      "182\t1\t0.0\t2\t1\n",
      "207\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.5.19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.5.19_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.5.19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.5.19_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.70 s (19 us/read; 3.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            196,616\n",
      "  Read 1 with adapter:                     119 (0.1%)\n",
      "  Read 2 with adapter:                  12,606 (6.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       196,616 (100.0%)\n",
      "\n",
      "Total basepairs processed:    89,699,685 bp\n",
      "  Read 1:    44,010,258 bp\n",
      "  Read 2:    45,689,427 bp\n",
      "Total written (filtered):     89,533,424 bp (99.8%)\n",
      "  Read 1:    44,009,073 bp\n",
      "  Read 2:    45,524,351 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 119 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t103\t3072.1\t0\t103\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t1 1\n",
      "26\t6\t0.0\t1\t5 1\n",
      "27\t3\t0.0\t1\t3\n",
      "52\t1\t0.0\t1\t1\n",
      "79\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12606 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t87\t3072.1\t0\t87\n",
      "4\t60\t768.0\t0\t60\n",
      "5\t41\t192.0\t0\t41\n",
      "6\t125\t48.0\t0\t125\n",
      "7\t491\t12.0\t0\t491\n",
      "8\t678\t3.0\t0\t678\n",
      "9\t2328\t0.8\t0\t2320 8\n",
      "10\t2445\t0.2\t1\t2420 25\n",
      "11\t149\t0.0\t1\t144 5\n",
      "12\t252\t0.0\t1\t249 3\n",
      "13\t322\t0.0\t1\t311 11\n",
      "14\t504\t0.0\t1\t478 26\n",
      "15\t319\t0.0\t1\t311 8\n",
      "16\t774\t0.0\t1\t730 44\n",
      "17\t1097\t0.0\t1\t1045 52\n",
      "18\t664\t0.0\t1\t579 58 27\n",
      "19\t1949\t0.0\t1\t1086 856 7\n",
      "20\t5\t0.0\t2\t0 4 1\n",
      "21\t276\t0.0\t2\t0 270 6\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "30\t2\t0.0\t2\t2\n",
      "46\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t0 1\n",
      "58\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t0 1\n",
      "63\t1\t0.0\t2\t0 1\n",
      "64\t2\t0.0\t2\t2\n",
      "66\t1\t0.0\t2\t1\n",
      "72\t1\t0.0\t2\t1\n",
      "84\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "90\t2\t0.0\t2\t2\n",
      "105\t1\t0.0\t2\t1\n",
      "151\t1\t0.0\t2\t1\n",
      "167\t1\t0.0\t2\t1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "235\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.8.5.19_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.8.5.19_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.8.5.19_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.8.5.19_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.73 s (19 us/read; 3.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            145,508\n",
      "  Read 1 with adapter:                     187 (0.1%)\n",
      "  Read 2 with adapter:                   8,367 (5.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       145,508 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,085,194 bp\n",
      "  Read 1:    32,429,155 bp\n",
      "  Read 2:    33,656,039 bp\n",
      "Total written (filtered):     65,963,548 bp (99.8%)\n",
      "  Read 1:    32,425,314 bp\n",
      "  Read 2:    33,538,234 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 187 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t134\t2273.6\t0\t134\n",
      "17\t1\t0.0\t1\t1\n",
      "26\t5\t0.0\t1\t3 2\n",
      "27\t35\t0.0\t1\t35\n",
      "87\t1\t0.0\t1\t1\n",
      "151\t1\t0.0\t1\t1\n",
      "190\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t0 1\n",
      "207\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t2 1\n",
      "218\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8367 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t2273.6\t0\t44\n",
      "4\t9\t568.4\t0\t9\n",
      "5\t24\t142.1\t0\t24\n",
      "6\t54\t35.5\t0\t54\n",
      "7\t161\t8.9\t0\t161\n",
      "8\t266\t2.2\t0\t266\n",
      "9\t966\t0.6\t0\t961 5\n",
      "10\t1602\t0.1\t1\t1578 24\n",
      "11\t123\t0.0\t1\t119 4\n",
      "12\t232\t0.0\t1\t226 6\n",
      "13\t315\t0.0\t1\t298 17\n",
      "14\t472\t0.0\t1\t450 22\n",
      "15\t299\t0.0\t1\t286 13\n",
      "16\t680\t0.0\t1\t649 31\n",
      "17\t836\t0.0\t1\t790 46\n",
      "18\t520\t0.0\t1\t454 42 24\n",
      "19\t1515\t0.0\t1\t850 657 8\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t215\t0.0\t2\t0 210 5\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "47\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "67\t2\t0.0\t2\t1 1\n",
      "69\t1\t0.0\t2\t1\n",
      "73\t1\t0.0\t2\t1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 1\n",
      "141\t1\t0.0\t2\t1\n",
      "203\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.10.19_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.10.19_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.10.19_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.10.19_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.31 s (14 us/read; 4.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            232,354\n",
      "  Read 1 with adapter:                     272 (0.1%)\n",
      "  Read 2 with adapter:                  10,778 (4.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       232,354 (100.0%)\n",
      "\n",
      "Total basepairs processed:   105,891,998 bp\n",
      "  Read 1:    52,001,907 bp\n",
      "  Read 2:    53,890,091 bp\n",
      "Total written (filtered):    105,737,634 bp (99.9%)\n",
      "  Read 1:    51,999,264 bp\n",
      "  Read 2:    53,738,370 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 272 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t245\t3630.5\t0\t245\n",
      "22\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t4\t0.0\t1\t4\n",
      "28\t1\t0.0\t1\t1\n",
      "68\t1\t0.0\t1\t1\n",
      "90\t1\t0.0\t1\t0 1\n",
      "91\t6\t0.0\t1\t6\n",
      "92\t4\t0.0\t1\t4\n",
      "132\t1\t0.0\t1\t1\n",
      "195\t1\t0.0\t1\t1\n",
      "198\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10778 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t52\t3630.5\t0\t52\n",
      "4\t29\t907.6\t0\t29\n",
      "5\t29\t226.9\t0\t29\n",
      "6\t64\t56.7\t0\t64\n",
      "7\t197\t14.2\t0\t197\n",
      "8\t358\t3.5\t0\t358\n",
      "9\t1329\t0.9\t0\t1326 3\n",
      "10\t1916\t0.2\t1\t1895 21\n",
      "11\t210\t0.1\t1\t206 4\n",
      "12\t333\t0.0\t1\t320 13\n",
      "13\t396\t0.0\t1\t383 13\n",
      "14\t494\t0.0\t1\t466 28\n",
      "15\t301\t0.0\t1\t280 21\n",
      "16\t818\t0.0\t1\t790 28\n",
      "17\t1097\t0.0\t1\t1050 47\n",
      "18\t648\t0.0\t1\t571 46 31\n",
      "19\t2171\t0.0\t1\t1262 899 10\n",
      "20\t6\t0.0\t2\t2 1 3\n",
      "21\t299\t0.0\t2\t0 294 5\n",
      "22\t23\t0.0\t2\t0 1 22\n",
      "30\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "128\t1\t0.0\t2\t1\n",
      "142\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.13.19_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.13.19_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.13.19_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.13.19_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.11 s (14 us/read; 4.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            224,484\n",
      "  Read 1 with adapter:                     198 (0.1%)\n",
      "  Read 2 with adapter:                  14,396 (6.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       224,484 (100.0%)\n",
      "\n",
      "Total basepairs processed:   102,385,489 bp\n",
      "  Read 1:    50,232,670 bp\n",
      "  Read 2:    52,152,819 bp\n",
      "Total written (filtered):    102,201,250 bp (99.8%)\n",
      "  Read 1:    50,231,962 bp\n",
      "  Read 2:    51,969,288 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 198 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t193\t3507.6\t0\t193\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14396 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t104\t3507.6\t0\t104\n",
      "4\t33\t876.9\t0\t33\n",
      "5\t70\t219.2\t0\t70\n",
      "6\t108\t54.8\t0\t108\n",
      "7\t447\t13.7\t0\t447\n",
      "8\t807\t3.4\t0\t807\n",
      "9\t2553\t0.9\t0\t2542 11\n",
      "10\t3946\t0.2\t1\t3895 51\n",
      "11\t136\t0.1\t1\t134 2\n",
      "12\t161\t0.0\t1\t160 1\n",
      "13\t196\t0.0\t1\t185 11\n",
      "14\t375\t0.0\t1\t358 17\n",
      "15\t322\t0.0\t1\t304 18\n",
      "16\t856\t0.0\t1\t837 19\n",
      "17\t1143\t0.0\t1\t1111 32\n",
      "18\t701\t0.0\t1\t639 47 15\n",
      "19\t2108\t0.0\t1\t1209 882 17\n",
      "20\t9\t0.0\t2\t1 4 4\n",
      "21\t296\t0.0\t2\t0 292 4\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "30\t2\t0.0\t2\t2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t2\n",
      "48\t1\t0.0\t2\t1\n",
      "51\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "91\t1\t0.0\t2\t1\n",
      "120\t2\t0.0\t2\t2\n",
      "180\t1\t0.0\t2\t1\n",
      "202\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "237\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.8.19_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.8.19_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.8.19_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.8.19_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.78 s (18 us/read; 3.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            153,819\n",
      "  Read 1 with adapter:                     171 (0.1%)\n",
      "  Read 2 with adapter:                   8,020 (5.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       153,819 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,997,836 bp\n",
      "  Read 1:    34,362,584 bp\n",
      "  Read 2:    35,635,252 bp\n",
      "Total written (filtered):     69,883,687 bp (99.8%)\n",
      "  Read 1:    34,360,960 bp\n",
      "  Read 2:    35,522,727 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 171 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t156\t2403.4\t0\t156\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t6\t0.0\t1\t5 1\n",
      "27\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "157\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8020 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t2403.4\t0\t44\n",
      "4\t28\t600.9\t0\t28\n",
      "5\t27\t150.2\t0\t27\n",
      "6\t52\t37.6\t0\t52\n",
      "7\t166\t9.4\t0\t166\n",
      "8\t251\t2.3\t0\t251\n",
      "9\t1043\t0.6\t0\t1043\n",
      "10\t1407\t0.1\t1\t1384 23\n",
      "11\t144\t0.0\t1\t144\n",
      "12\t218\t0.0\t1\t212 6\n",
      "13\t281\t0.0\t1\t269 12\n",
      "14\t379\t0.0\t1\t363 16\n",
      "15\t261\t0.0\t1\t247 14\n",
      "16\t611\t0.0\t1\t588 23\n",
      "17\t827\t0.0\t1\t795 32\n",
      "18\t516\t0.0\t1\t431 47 38\n",
      "19\t1555\t0.0\t1\t896 652 7\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t176\t0.0\t2\t0 176\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "46\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 1\n",
      "76\t1\t0.0\t2\t1\n",
      "81\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t1\n",
      "155\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.F.control.8.11.19_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.F.control.8.11.19_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.F.control.8.11.19_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.F.control.8.11.19_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.36 s (19 us/read; 3.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            126,759\n",
      "  Read 1 with adapter:                      90 (0.1%)\n",
      "  Read 2 with adapter:                   7,189 (5.7%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       126,758 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,657,425 bp\n",
      "  Read 1:    28,296,285 bp\n",
      "  Read 2:    29,361,140 bp\n",
      "Total written (filtered):     57,564,104 bp (99.8%)\n",
      "  Read 1:    28,294,740 bp\n",
      "  Read 2:    29,269,364 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t1980.6\t0\t80\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "79\t1\t0.0\t1\t1\n",
      "142\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "207\t2\t0.0\t1\t2\n",
      "218\t1\t0.0\t1\t1\n",
      "223\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7189 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t1980.6\t0\t56\n",
      "4\t34\t495.2\t0\t34\n",
      "5\t24\t123.8\t0\t24\n",
      "6\t104\t30.9\t0\t104\n",
      "7\t305\t7.7\t0\t305\n",
      "8\t506\t1.9\t0\t506\n",
      "9\t1640\t0.5\t0\t1634 6\n",
      "10\t1403\t0.1\t1\t1383 20\n",
      "11\t43\t0.0\t1\t38 5\n",
      "12\t47\t0.0\t1\t47\n",
      "13\t48\t0.0\t1\t45 3\n",
      "14\t131\t0.0\t1\t125 6\n",
      "15\t129\t0.0\t1\t123 6\n",
      "16\t352\t0.0\t1\t342 10\n",
      "17\t593\t0.0\t1\t576 17\n",
      "18\t345\t0.0\t1\t312 19 14\n",
      "19\t1225\t0.0\t1\t719 501 5\n",
      "20\t3\t0.0\t2\t1 1 1\n",
      "21\t178\t0.0\t2\t0 175 3\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "30\t4\t0.0\t2\t4\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.F.control.8.5.19_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.F.control.8.5.19_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.F.control.8.5.19_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.F.control.8.5.19_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.59 s (16 us/read; 3.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            292,920\n",
      "  Read 1 with adapter:                     323 (0.1%)\n",
      "  Read 2 with adapter:                  22,670 (7.7%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       292,919 (100.0%)\n",
      "\n",
      "Total basepairs processed:   133,631,502 bp\n",
      "  Read 1:    65,531,631 bp\n",
      "  Read 2:    68,099,871 bp\n",
      "Total written (filtered):    133,336,598 bp (99.8%)\n",
      "  Read 1:    65,528,885 bp\n",
      "  Read 2:    67,807,713 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 323 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t299\t4576.9\t0\t299\n",
      "21\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t9\t0.0\t1\t9\n",
      "27\t2\t0.0\t1\t2\n",
      "35\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t1\n",
      "73\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t1\n",
      "215\t3\t0.0\t1\t2 1\n",
      "216\t1\t0.0\t1\t1\n",
      "223\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 22670 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t133\t4576.9\t0\t133\n",
      "4\t71\t1144.2\t0\t71\n",
      "5\t98\t286.1\t0\t98\n",
      "6\t198\t71.5\t0\t198\n",
      "7\t792\t17.9\t0\t792\n",
      "8\t1293\t4.5\t0\t1293\n",
      "9\t4206\t1.1\t0\t4192 14\n",
      "10\t5312\t0.3\t1\t5255 57\n",
      "11\t307\t0.1\t1\t299 8\n",
      "12\t400\t0.0\t1\t387 13\n",
      "13\t602\t0.0\t1\t576 26\n",
      "14\t855\t0.0\t1\t805 50\n",
      "15\t596\t0.0\t1\t568 28\n",
      "16\t1462\t0.0\t1\t1398 64\n",
      "17\t1724\t0.0\t1\t1629 95\n",
      "18\t1003\t0.0\t1\t863 79 61\n",
      "19\t3074\t0.0\t1\t1717 1338 19\n",
      "20\t8\t0.0\t2\t2 4 2\n",
      "21\t445\t0.0\t2\t0 438 7\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "30\t1\t0.0\t2\t1\n",
      "46\t2\t0.0\t2\t2\n",
      "49\t2\t0.0\t2\t1 1\n",
      "53\t1\t0.0\t2\t1\n",
      "54\t2\t0.0\t2\t2\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "58\t5\t0.0\t2\t4 0 1\n",
      "59\t8\t0.0\t2\t0 1 7\n",
      "62\t1\t0.0\t2\t1\n",
      "65\t5\t0.0\t2\t5\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t2\t0.0\t2\t2\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "72\t1\t0.0\t2\t1\n",
      "73\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "76\t2\t0.0\t2\t2\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "87\t3\t0.0\t2\t3\n",
      "88\t3\t0.0\t2\t1 1 1\n",
      "94\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "106\t1\t0.0\t2\t1\n",
      "113\t1\t0.0\t2\t1\n",
      "116\t1\t0.0\t2\t1\n",
      "133\t1\t0.0\t2\t1\n",
      "145\t1\t0.0\t2\t0 1\n",
      "147\t1\t0.0\t2\t1\n",
      "158\t1\t0.0\t2\t0 1\n",
      "167\t1\t0.0\t2\t0 1\n",
      "168\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t0 1\n",
      "174\t3\t0.0\t2\t2 0 1\n",
      "186\t1\t0.0\t2\t1\n",
      "229\t1\t0.0\t2\t0 1\n",
      "234\t1\t0.0\t2\t0 1\n",
      "237\t1\t0.0\t2\t1\n",
      "240\t1\t0.0\t2\t1\n",
      "243\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.A.control.9.19.19_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.A.control.9.19.19_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.A.control.9.19.19_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.A.control.9.19.19_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.03 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             89,903\n",
      "  Read 1 with adapter:                      52 (0.1%)\n",
      "  Read 2 with adapter:                   4,623 (5.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        89,903 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,652,419 bp\n",
      "  Read 1:    19,948,992 bp\n",
      "  Read 2:    20,703,427 bp\n",
      "Total written (filtered):     40,584,968 bp (99.8%)\n",
      "  Read 1:    19,947,921 bp\n",
      "  Read 2:    20,637,047 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 52 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t1404.7\t0\t44\n",
      "15\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "112\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4623 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t1404.7\t0\t23\n",
      "4\t12\t351.2\t0\t12\n",
      "5\t13\t87.8\t0\t13\n",
      "6\t29\t21.9\t0\t29\n",
      "7\t77\t5.5\t0\t77\n",
      "8\t158\t1.4\t0\t158\n",
      "9\t534\t0.3\t0\t533 1\n",
      "10\t763\t0.1\t1\t755 8\n",
      "11\t55\t0.0\t1\t55\n",
      "12\t99\t0.0\t1\t99\n",
      "13\t148\t0.0\t1\t143 5\n",
      "14\t224\t0.0\t1\t216 8\n",
      "15\t184\t0.0\t1\t174 10\n",
      "16\t422\t0.0\t1\t406 16\n",
      "17\t470\t0.0\t1\t451 19\n",
      "18\t278\t0.0\t1\t253 13 12\n",
      "19\t993\t0.0\t1\t563 426 4\n",
      "20\t2\t0.0\t2\t0 2\n",
      "21\t124\t0.0\t2\t0 123 1\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "64\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "119\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.11.19_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.11.19_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.11.19_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.11.19_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 1.92 s (16 us/read; 3.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            123,476\n",
      "  Read 1 with adapter:                     130 (0.1%)\n",
      "  Read 2 with adapter:                  10,060 (8.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       123,476 (100.0%)\n",
      "\n",
      "Total basepairs processed:    56,065,592 bp\n",
      "  Read 1:    27,471,976 bp\n",
      "  Read 2:    28,593,616 bp\n",
      "Total written (filtered):     55,937,975 bp (99.8%)\n",
      "  Read 1:    27,467,327 bp\n",
      "  Read 2:    28,470,648 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 130 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t100\t1929.3\t0\t100\n",
      "21\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t2\t0.0\t1\t2\n",
      "191\t1\t0.0\t1\t1\n",
      "192\t3\t0.0\t1\t2 1\n",
      "193\t12\t0.0\t1\t10 2\n",
      "194\t1\t0.0\t1\t0 1\n",
      "198\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "219\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10060 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t1929.3\t0\t47\n",
      "4\t32\t482.3\t0\t32\n",
      "5\t41\t120.6\t0\t41\n",
      "6\t114\t30.1\t0\t114\n",
      "7\t455\t7.5\t0\t455\n",
      "8\t699\t1.9\t0\t699\n",
      "9\t2181\t0.5\t0\t2173 8\n",
      "10\t2519\t0.1\t1\t2484 35\n",
      "11\t116\t0.0\t1\t113 3\n",
      "12\t124\t0.0\t1\t118 6\n",
      "13\t162\t0.0\t1\t157 5\n",
      "14\t273\t0.0\t1\t258 15\n",
      "15\t235\t0.0\t1\t219 16\n",
      "16\t517\t0.0\t1\t498 19\n",
      "17\t686\t0.0\t1\t658 28\n",
      "18\t453\t0.0\t1\t400 32 21\n",
      "19\t1232\t0.0\t1\t711 515 6\n",
      "20\t2\t0.0\t2\t1 1\n",
      "21\t159\t0.0\t2\t0 156 3\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 1\n",
      "224\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.3.19_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.3.19_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.3.19_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.3.19_23_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.5.19_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.8.5.19_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.5.19_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.8.5.19_36_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.2.19_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.2.19_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.2.19_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.2.19_10_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.3.19_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.3.19_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.3.19_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.3.19_3_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.5.19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.B.control.9.5.19_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.5.19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.B.control.9.5.19_19_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.8.5.19_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.8.5.19_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.8.5.19_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.8.5.19_29_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.10.19_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.10.19_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.10.19_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.10.19_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.13.19_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.13.19_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.13.19_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.13.19_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.8.19_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.E.control.9.8.19_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.8.19_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.E.control.9.8.19_20_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.F.control.8.11.19_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.F.control.8.11.19_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.F.control.8.11.19_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.F.control.8.11.19_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.F.control.8.5.19_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.6.F.control.8.5.19_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.F.control.8.5.19_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.6.F.control.8.5.19_34_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.A.control.9.19.19_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.A.control.9.19.19_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.A.control.9.19.19_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.A.control.9.19.19_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.11.19_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.11.19_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.11.19_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.11.19_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.13.19_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.13.19_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.13.19_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.13.19_2_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.13.19_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.13.19_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.13.19_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.13.19_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.72 s (15 us/read; 3.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            179,918\n",
      "  Read 1 with adapter:                     161 (0.1%)\n",
      "  Read 2 with adapter:                  11,883 (6.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       179,918 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,976,068 bp\n",
      "  Read 1:    40,210,522 bp\n",
      "  Read 2:    41,765,546 bp\n",
      "Total written (filtered):     81,823,285 bp (99.8%)\n",
      "  Read 1:    40,209,741 bp\n",
      "  Read 2:    41,613,544 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 161 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t156\t2811.2\t0\t156\n",
      "27\t3\t0.0\t1\t3\n",
      "116\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11883 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t2811.2\t0\t79\n",
      "4\t28\t702.8\t0\t28\n",
      "5\t44\t175.7\t0\t44\n",
      "6\t97\t43.9\t0\t97\n",
      "7\t326\t11.0\t0\t326\n",
      "8\t543\t2.7\t0\t543\n",
      "9\t2007\t0.7\t0\t2003 4\n",
      "10\t3368\t0.2\t1\t3340 28\n",
      "11\t137\t0.0\t1\t131 6\n",
      "12\t201\t0.0\t1\t195 6\n",
      "13\t253\t0.0\t1\t244 9\n",
      "14\t376\t0.0\t1\t345 31\n",
      "15\t269\t0.0\t1\t254 15\n",
      "16\t735\t0.0\t1\t696 39\n",
      "17\t827\t0.0\t1\t783 44\n",
      "18\t558\t0.0\t1\t498 40 20\n",
      "19\t1742\t0.0\t1\t935 794 13\n",
      "20\t6\t0.0\t2\t0 4 2\n",
      "21\t261\t0.0\t2\t0 257 4\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "46\t2\t0.0\t2\t2\n",
      "49\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "186\t2\t0.0\t2\t0 1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.17.19_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.17.19_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.17.19_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.17.19_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.31 s (20 us/read; 2.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,245\n",
      "  Read 1 with adapter:                      87 (0.1%)\n",
      "  Read 2 with adapter:                   5,141 (4.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,245 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,405,119 bp\n",
      "  Read 1:    25,244,087 bp\n",
      "  Read 2:    26,161,032 bp\n",
      "Total written (filtered):     51,330,888 bp (99.9%)\n",
      "  Read 1:    25,243,696 bp\n",
      "  Read 2:    26,087,192 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 87 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t1769.5\t0\t85\n",
      "27\t1\t0.0\t1\t1\n",
      "109\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5141 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t1769.5\t0\t27\n",
      "4\t8\t442.4\t0\t8\n",
      "5\t9\t110.6\t0\t9\n",
      "6\t39\t27.6\t0\t39\n",
      "7\t90\t6.9\t0\t90\n",
      "8\t156\t1.7\t0\t156\n",
      "9\t534\t0.4\t0\t532 2\n",
      "10\t869\t0.1\t1\t860 9\n",
      "11\t65\t0.0\t1\t63 2\n",
      "12\t130\t0.0\t1\t129 1\n",
      "13\t182\t0.0\t1\t168 14\n",
      "14\t254\t0.0\t1\t233 21\n",
      "15\t177\t0.0\t1\t169 8\n",
      "16\t400\t0.0\t1\t375 25\n",
      "17\t542\t0.0\t1\t525 17\n",
      "18\t361\t0.0\t1\t312 31 18\n",
      "19\t1140\t0.0\t1\t654 483 3\n",
      "20\t2\t0.0\t2\t1 1\n",
      "21\t150\t0.0\t2\t0 149 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "52\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.6.19_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.6.19_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.6.19_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.6.19_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 3.60 s (13 us/read; 4.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            280,488\n",
      "  Read 1 with adapter:                     262 (0.1%)\n",
      "  Read 2 with adapter:                  13,300 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       280,488 (100.0%)\n",
      "\n",
      "Total basepairs processed:   127,844,640 bp\n",
      "  Read 1:    62,775,227 bp\n",
      "  Read 2:    65,069,413 bp\n",
      "Total written (filtered):    127,672,347 bp (99.9%)\n",
      "  Read 1:    62,773,734 bp\n",
      "  Read 2:    64,898,613 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 262 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t251\t4382.6\t0\t251\n",
      "25\t4\t0.0\t1\t4\n",
      "26\t4\t0.0\t1\t4\n",
      "111\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13300 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t4382.6\t0\t79\n",
      "4\t55\t1095.7\t0\t55\n",
      "5\t34\t273.9\t0\t34\n",
      "6\t147\t68.5\t0\t147\n",
      "7\t501\t17.1\t0\t501\n",
      "8\t884\t4.3\t0\t884\n",
      "9\t2911\t1.1\t0\t2909 2\n",
      "10\t2921\t0.3\t1\t2884 37\n",
      "11\t71\t0.1\t1\t69 2\n",
      "12\t58\t0.0\t1\t56 2\n",
      "13\t83\t0.0\t1\t82 1\n",
      "14\t177\t0.0\t1\t161 16\n",
      "15\t157\t0.0\t1\t141 16\n",
      "16\t601\t0.0\t1\t585 16\n",
      "17\t1058\t0.0\t1\t1019 39\n",
      "18\t748\t0.0\t1\t685 41 22\n",
      "19\t2440\t0.0\t1\t1453 972 15\n",
      "20\t6\t0.0\t2\t1 3 2\n",
      "21\t347\t0.0\t2\t0 342 5\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "107\t1\t0.0\t2\t1\n",
      "204\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.8.19_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.8.19_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.8.19_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.8.19_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.85 s (13 us/read; 4.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            216,681\n",
      "  Read 1 with adapter:                     304 (0.1%)\n",
      "  Read 2 with adapter:                  15,862 (7.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       216,681 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,708,167 bp\n",
      "  Read 1:    48,401,252 bp\n",
      "  Read 2:    50,306,915 bp\n",
      "Total written (filtered):     98,512,376 bp (99.8%)\n",
      "  Read 1:    48,399,725 bp\n",
      "  Read 2:    50,112,651 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 304 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t285\t3385.6\t0\t285\n",
      "19\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t4\t0.0\t1\t3 1\n",
      "26\t7\t0.0\t1\t6 1\n",
      "27\t4\t0.0\t1\t4\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15862 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t102\t3385.6\t0\t102\n",
      "4\t49\t846.4\t0\t49\n",
      "5\t71\t211.6\t0\t71\n",
      "6\t176\t52.9\t0\t176\n",
      "7\t591\t13.2\t0\t591\n",
      "8\t1015\t3.3\t0\t1015\n",
      "9\t3438\t0.8\t0\t3435 3\n",
      "10\t4021\t0.2\t1\t3973 48\n",
      "11\t177\t0.1\t1\t174 3\n",
      "12\t232\t0.0\t1\t220 12\n",
      "13\t321\t0.0\t1\t307 14\n",
      "14\t434\t0.0\t1\t415 19\n",
      "15\t342\t0.0\t1\t320 22\n",
      "16\t842\t0.0\t1\t813 29\n",
      "17\t1081\t0.0\t1\t1035 46\n",
      "18\t703\t0.0\t1\t637 45 21\n",
      "19\t1968\t0.0\t1\t1128 827 13\n",
      "20\t3\t0.0\t2\t2 1\n",
      "21\t277\t0.0\t2\t0 275 2\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "30\t1\t0.0\t2\t1\n",
      "46\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "128\t1\t0.0\t2\t1\n",
      "202\t1\t0.0\t2\t1\n",
      "208\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.8.12.19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.8.12.19_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.8.12.19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.8.12.19_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 7.05 s (18 us/read; 3.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            402,732\n",
      "  Read 1 with adapter:                     162 (0.0%)\n",
      "  Read 2 with adapter:                  15,814 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       402,732 (100.0%)\n",
      "\n",
      "Total basepairs processed:   183,188,891 bp\n",
      "  Read 1:    90,104,904 bp\n",
      "  Read 2:    93,083,987 bp\n",
      "Total written (filtered):    182,962,084 bp (99.9%)\n",
      "  Read 1:    90,104,048 bp\n",
      "  Read 2:    92,858,036 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 162 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t154\t6292.7\t0\t154\n",
      "18\t1\t0.0\t1\t0 1\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t2\t0.0\t1\t2\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15814 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t128\t6292.7\t0\t128\n",
      "4\t57\t1573.2\t0\t57\n",
      "5\t54\t393.3\t0\t54\n",
      "6\t152\t98.3\t0\t152\n",
      "7\t314\t24.6\t0\t314\n",
      "8\t561\t6.1\t0\t561\n",
      "9\t2025\t1.5\t0\t2018 7\n",
      "10\t2712\t0.4\t1\t2680 32\n",
      "11\t203\t0.1\t1\t199 4\n",
      "12\t265\t0.0\t1\t264 1\n",
      "13\t295\t0.0\t1\t282 13\n",
      "14\t510\t0.0\t1\t474 36\n",
      "15\t422\t0.0\t1\t405 17\n",
      "16\t1079\t0.0\t1\t1049 30\n",
      "17\t1729\t0.0\t1\t1660 69\n",
      "18\t1043\t0.0\t1\t920 82 41\n",
      "19\t3676\t0.0\t1\t2144 1512 20\n",
      "20\t6\t0.0\t2\t0 3 3\n",
      "21\t540\t0.0\t2\t0 531 9\n",
      "22\t28\t0.0\t2\t0 1 27\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t2\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "72\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t1\n",
      "204\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "218\t4\t0.0\t2\t3 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.8.3.19_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.8.3.19_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.8.3.19_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.8.3.19_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 4.65 s (14 us/read; 4.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            336,362\n",
      "  Read 1 with adapter:                     325 (0.1%)\n",
      "  Read 2 with adapter:                  26,028 (7.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       336,362 (100.0%)\n",
      "\n",
      "Total basepairs processed:   153,494,630 bp\n",
      "  Read 1:    75,263,743 bp\n",
      "  Read 2:    78,230,887 bp\n",
      "Total written (filtered):    153,174,244 bp (99.8%)\n",
      "  Read 1:    75,261,665 bp\n",
      "  Read 2:    77,912,579 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 325 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t302\t5255.7\t0\t302\n",
      "25\t4\t0.0\t1\t4\n",
      "26\t10\t0.0\t1\t9 1\n",
      "27\t4\t0.0\t1\t4\n",
      "52\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "153\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 26028 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t151\t5255.7\t0\t151\n",
      "4\t103\t1313.9\t0\t103\n",
      "5\t129\t328.5\t0\t129\n",
      "6\t353\t82.1\t0\t353\n",
      "7\t1245\t20.5\t0\t1245\n",
      "8\t1986\t5.1\t0\t1986\n",
      "9\t6020\t1.3\t0\t6008 12\n",
      "10\t5639\t0.3\t1\t5582 57\n",
      "11\t295\t0.1\t1\t289 6\n",
      "12\t358\t0.0\t1\t348 10\n",
      "13\t491\t0.0\t1\t482 9\n",
      "14\t713\t0.0\t1\t687 26\n",
      "15\t525\t0.0\t1\t495 30\n",
      "16\t1333\t0.0\t1\t1274 59\n",
      "17\t1735\t0.0\t1\t1686 49\n",
      "18\t1080\t0.0\t1\t951 80 49\n",
      "19\t3361\t0.0\t1\t1831 1512 18\n",
      "20\t11\t0.0\t2\t2 6 3\n",
      "21\t436\t0.0\t2\t0 429 7\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t1\n",
      "46\t2\t0.0\t2\t2\n",
      "49\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t0 1\n",
      "55\t2\t0.0\t2\t2\n",
      "56\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t0 1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t3\t0.0\t2\t3\n",
      "70\t1\t0.0\t2\t1\n",
      "72\t2\t0.0\t2\t2\n",
      "74\t3\t0.0\t2\t0 3\n",
      "79\t1\t0.0\t2\t1\n",
      "88\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "91\t1\t0.0\t2\t1\n",
      "92\t1\t0.0\t2\t1\n",
      "102\t1\t0.0\t2\t1\n",
      "106\t1\t0.0\t2\t1\n",
      "110\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t1\n",
      "144\t1\t0.0\t2\t1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t0 1\n",
      "246\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.14.19_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.14.19_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.14.19_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.14.19_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.52 s (17 us/read; 3.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            326,898\n",
      "  Read 1 with adapter:                     219 (0.1%)\n",
      "  Read 2 with adapter:                  11,355 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       326,898 (100.0%)\n",
      "\n",
      "Total basepairs processed:   148,681,363 bp\n",
      "  Read 1:    73,113,453 bp\n",
      "  Read 2:    75,567,910 bp\n",
      "Total written (filtered):    148,506,939 bp (99.9%)\n",
      "  Read 1:    73,112,522 bp\n",
      "  Read 2:    75,394,417 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 219 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t211\t5107.8\t0\t211\n",
      "9\t1\t1.2\t0\t1\n",
      "10\t1\t0.3\t1\t1\n",
      "18\t1\t0.0\t1\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "68\t1\t0.0\t1\t1\n",
      "112\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11355 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t5107.8\t0\t49\n",
      "4\t30\t1276.9\t0\t30\n",
      "5\t30\t319.2\t0\t30\n",
      "6\t51\t79.8\t0\t51\n",
      "7\t128\t20.0\t0\t128\n",
      "8\t220\t5.0\t0\t220\n",
      "9\t840\t1.2\t0\t835 5\n",
      "10\t1526\t0.3\t1\t1507 19\n",
      "11\t170\t0.1\t1\t168 2\n",
      "12\t262\t0.0\t1\t255 7\n",
      "13\t342\t0.0\t1\t321 21\n",
      "14\t485\t0.0\t1\t450 35\n",
      "15\t362\t0.0\t1\t340 22\n",
      "16\t1047\t0.0\t1\t999 48\n",
      "17\t1465\t0.0\t1\t1421 44\n",
      "18\t865\t0.0\t1\t771 60 34\n",
      "19\t2989\t0.0\t1\t1757 1214 18\n",
      "20\t11\t0.0\t2\t2 5 4\n",
      "21\t450\t0.0\t2\t0 442 8\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "25\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "71\t1\t0.0\t2\t1\n",
      "152\t1\t0.0\t2\t1\n",
      "156\t1\t0.0\t2\t1\n",
      "204\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "226\t1\t0.0\t2\t0 0 1\n",
      "243\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.16.19_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.16.19_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.16.19_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.16.19_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.56 s (15 us/read; 3.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            364,743\n",
      "  Read 1 with adapter:                     264 (0.1%)\n",
      "  Read 2 with adapter:                  23,314 (6.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       364,743 (100.0%)\n",
      "\n",
      "Total basepairs processed:   166,116,814 bp\n",
      "  Read 1:    81,541,740 bp\n",
      "  Read 2:    84,575,074 bp\n",
      "Total written (filtered):    165,805,472 bp (99.8%)\n",
      "  Read 1:    81,540,245 bp\n",
      "  Read 2:    84,265,227 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 264 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t252\t5699.1\t0\t252\n",
      "25\t3\t0.0\t1\t1 2\n",
      "26\t6\t0.0\t1\t6\n",
      "110\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "202\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 23314 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t132\t5699.1\t0\t132\n",
      "4\t57\t1424.8\t0\t57\n",
      "5\t102\t356.2\t0\t102\n",
      "6\t210\t89.0\t0\t210\n",
      "7\t673\t22.3\t0\t673\n",
      "8\t1084\t5.6\t0\t1084\n",
      "9\t3718\t1.4\t0\t3706 12\n",
      "10\t5232\t0.3\t1\t5162 70\n",
      "11\t270\t0.1\t1\t264 6\n",
      "12\t358\t0.0\t1\t350 8\n",
      "13\t466\t0.0\t1\t448 18\n",
      "14\t819\t0.0\t1\t765 54\n",
      "15\t625\t0.0\t1\t587 38\n",
      "16\t1635\t0.0\t1\t1561 74\n",
      "17\t2152\t0.0\t1\t2056 96\n",
      "18\t1252\t0.0\t1\t1093 106 53\n",
      "19\t3977\t0.0\t1\t2254 1701 22\n",
      "20\t13\t0.0\t2\t0 8 5\n",
      "21\t488\t0.0\t2\t0 481 7\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "49\t2\t0.0\t2\t1 0 1\n",
      "52\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "59\t2\t0.0\t2\t1 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "72\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "81\t1\t0.0\t2\t1\n",
      "83\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.20.19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.20.19_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.20.19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.20.19_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 5.70 s (16 us/read; 3.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            357,898\n",
      "  Read 1 with adapter:                     453 (0.1%)\n",
      "  Read 2 with adapter:                  18,275 (5.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       357,898 (100.0%)\n",
      "\n",
      "Total basepairs processed:   162,990,314 bp\n",
      "  Read 1:    80,051,589 bp\n",
      "  Read 2:    82,938,725 bp\n",
      "Total written (filtered):    162,728,165 bp (99.8%)\n",
      "  Read 1:    80,048,780 bp\n",
      "  Read 2:    82,679,385 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 453 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t442\t5592.2\t0\t442\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "47\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18275 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t119\t5592.2\t0\t119\n",
      "4\t31\t1398.0\t0\t31\n",
      "5\t33\t349.5\t0\t33\n",
      "6\t145\t87.4\t0\t145\n",
      "7\t315\t21.8\t0\t315\n",
      "8\t559\t5.5\t0\t559\n",
      "9\t2059\t1.4\t0\t2053 6\n",
      "10\t3187\t0.3\t1\t3162 25\n",
      "11\t303\t0.1\t1\t298 5\n",
      "12\t548\t0.0\t1\t533 15\n",
      "13\t669\t0.0\t1\t638 31\n",
      "14\t914\t0.0\t1\t855 59\n",
      "15\t588\t0.0\t1\t557 31\n",
      "16\t1437\t0.0\t1\t1391 46\n",
      "17\t1936\t0.0\t1\t1859 77\n",
      "18\t1160\t0.0\t1\t1018 83 59\n",
      "19\t3733\t0.0\t1\t2152 1564 17\n",
      "20\t9\t0.0\t2\t2 5 2\n",
      "21\t482\t0.0\t2\t0 472 10\n",
      "22\t31\t0.0\t2\t0 1 30\n",
      "25\t1\t0.0\t2\t1\n",
      "39\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "65\t2\t0.0\t2\t2\n",
      "68\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t0 1\n",
      "85\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "116\t1\t0.0\t2\t1\n",
      "117\t1\t0.0\t2\t1\n",
      "145\t1\t0.0\t2\t1\n",
      "160\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.25.19_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.25.19_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.25.19_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.25.19_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "Finished in 2.74 s (17 us/read; 3.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            160,267\n",
      "  Read 1 with adapter:                     128 (0.1%)\n",
      "  Read 2 with adapter:                   3,766 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       160,267 (100.0%)\n",
      "\n",
      "Total basepairs processed:    72,934,078 bp\n",
      "  Read 1:    35,845,618 bp\n",
      "  Read 2:    37,088,460 bp\n",
      "Total written (filtered):     72,872,244 bp (99.9%)\n",
      "  Read 1:    35,844,698 bp\n",
      "  Read 2:    37,027,546 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 128 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t125\t2504.2\t0\t125\n",
      "123\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3766 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t2504.2\t0\t23\n",
      "4\t4\t626.0\t0\t4\n",
      "5\t4\t156.5\t0\t4\n",
      "6\t13\t39.1\t0\t13\n",
      "7\t39\t9.8\t0\t39\n",
      "8\t49\t2.4\t0\t49\n",
      "9\t203\t0.6\t0\t200 3\n",
      "10\t556\t0.2\t1\t548 8\n",
      "11\t24\t0.0\t1\t24\n",
      "12\t55\t0.0\t1\t55\n",
      "13\t41\t0.0\t1\t41\n",
      "14\t80\t0.0\t1\t77 3\n",
      "15\t104\t0.0\t1\t101 3\n",
      "16\t304\t0.0\t1\t290 14\n",
      "17\t516\t0.0\t1\t499 17\n",
      "18\t319\t0.0\t1\t286 29 4\n",
      "19\t1211\t0.0\t1\t767 435 9\n",
      "20\t4\t0.0\t2\t1 3\n",
      "21\t198\t0.0\t2\t0 193 5\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "30\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "220\t1\t0.0\t2\t0 1\n",
      "230\t2\t0.0\t2\t2\n",
      "234\t2\t0.0\t2\t2\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.17.19_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.17.19_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.17.19_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.17.19_7_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.6.19_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.6.19_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.6.19_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.6.19_24_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.8.19_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.D.control.9.8.19_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.8.19_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.D.control.9.8.19_25_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.8.12.19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.8.12.19_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.8.12.19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.8.12.19_1_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.8.3.19_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.8.3.19_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.8.3.19_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.8.3.19_8_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.14.19_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.14.19_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.14.19_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.14.19_22_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.16.19_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.16.19_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.16.19_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.16.19_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.20.19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.20.19_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.20.19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.20.19_37_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.25.19_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-slo7x_cf/O.fav.Tank.9.F.control.9.25.19_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.25.19_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-pkx5zeyu/e9046409-855c-40aa-af28-aa0834329b05/data/O.fav.Tank.9.F.control.9.25.19_35_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGtransmissionstudies_277_236.qza\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for f1 in demux_Muller_SWGtransmissionstudies_*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "\n",
    "qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences $f1 \\\n",
    "--p-cores 16 \\\n",
    "--p-front-f GTGCCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACVSGGGTATCTAAT \\\n",
    "--o-trimmed-sequences trim_$newfile \\\n",
    "--verbose\n",
    "\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M..cav.Tank.1.E.90percent.6.4.19_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M..cav.Tank.1.E.90percent.6.4.19_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M..cav.Tank.1.E.90percent.6.4.19_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M..cav.Tank.1.E.90percent.6.4.19_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M..cav.Tank.1.E.90percent.6.4.19_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M..cav.Tank.1.E.90percent.6.4.19_5_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M..cav.Tank.1.E.90percent.6.4.19_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M..cav.Tank.1.E.90percent.6.4.19_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       363,705 reads  @     14.1 µs/read;   4.27 M reads/minute\n",
      "Finished in 5.14 s (14 us/read; 4.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            363,705\n",
      "  Read 1 with adapter:                     397 (0.1%)\n",
      "  Read 2 with adapter:                  16,828 (4.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       363,705 (100.0%)\n",
      "\n",
      "Total basepairs processed:   165,859,959 bp\n",
      "  Read 1:    81,444,672 bp\n",
      "  Read 2:    84,415,287 bp\n",
      "Total written (filtered):    165,623,127 bp (99.9%)\n",
      "  Read 1:    81,439,177 bp\n",
      "  Read 2:    84,183,950 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 397 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t349\t5682.9\t0\t349\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t3\t0.0\t1\t3\n",
      "100\t3\t0.0\t1\t0 3\n",
      "101\t37\t0.0\t1\t36 1\n",
      "102\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 16828 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t164\t5682.9\t0\t164\n",
      "4\t24\t1420.7\t0\t24\n",
      "5\t38\t355.2\t0\t38\n",
      "6\t84\t88.8\t0\t84\n",
      "7\t326\t22.2\t0\t326\n",
      "8\t433\t5.5\t0\t433\n",
      "9\t1708\t1.4\t0\t1696 12\n",
      "10\t5240\t0.3\t1\t5186 54\n",
      "11\t214\t0.1\t1\t212 2\n",
      "12\t174\t0.0\t1\t172 2\n",
      "13\t162\t0.0\t1\t154 8\n",
      "14\t349\t0.0\t1\t334 15\n",
      "15\t368\t0.0\t1\t355 13\n",
      "16\t1098\t0.0\t1\t1069 29\n",
      "17\t1590\t0.0\t1\t1535 55\n",
      "18\t961\t0.0\t1\t881 55 25\n",
      "19\t3232\t0.0\t1\t2004 1216 12\n",
      "20\t179\t0.0\t2\t175 4\n",
      "21\t444\t0.0\t2\t0 433 11\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t6\t0.0\t2\t5 1\n",
      "100\t1\t0.0\t2\t1\n",
      "215\t2\t0.0\t2\t2\n",
      "217\t3\t0.0\t2\t3\n",
      "224\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t0 1\n",
      "227\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.25percent.5.30.19_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.25percent.5.30.19_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.25percent.5.30.19_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.25percent.5.30.19_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.25percent.5.30.19_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.25percent.5.30.19_47_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.25percent.5.30.19_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.25percent.5.30.19_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       336,741 reads  @     14.0 µs/read;   4.28 M reads/minute\n",
      "Finished in 4.75 s (14 us/read; 4.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            336,741\n",
      "  Read 1 with adapter:                     294 (0.1%)\n",
      "  Read 2 with adapter:                  15,642 (4.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       336,741 (100.0%)\n",
      "\n",
      "Total basepairs processed:   153,575,242 bp\n",
      "  Read 1:    75,406,463 bp\n",
      "  Read 2:    78,168,779 bp\n",
      "Total written (filtered):    153,349,818 bp (99.9%)\n",
      "  Read 1:    75,404,914 bp\n",
      "  Read 2:    77,944,904 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 294 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t279\t5261.6\t0\t279\n",
      "22\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t7\t0.0\t1\t7\n",
      "27\t3\t0.0\t1\t3\n",
      "162\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15642 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t84\t5261.6\t0\t84\n",
      "4\t36\t1315.4\t0\t36\n",
      "5\t55\t328.8\t0\t55\n",
      "6\t127\t82.2\t0\t127\n",
      "7\t385\t20.6\t0\t385\n",
      "8\t540\t5.1\t0\t540\n",
      "9\t1850\t1.3\t0\t1845 5\n",
      "10\t2531\t0.3\t1\t2508 23\n",
      "11\t201\t0.1\t1\t199 2\n",
      "12\t479\t0.0\t1\t468 11\n",
      "13\t526\t0.0\t1\t509 17\n",
      "14\t729\t0.0\t1\t687 42\n",
      "15\t460\t0.0\t1\t423 37\n",
      "16\t1213\t0.0\t1\t1171 42\n",
      "17\t1646\t0.0\t1\t1596 50\n",
      "18\t940\t0.0\t1\t821 76 43\n",
      "19\t3319\t0.0\t1\t1918 1382 19\n",
      "20\t5\t0.0\t2\t1 2 2\n",
      "21\t451\t0.0\t2\t0 441 10\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "46\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t5\t0.0\t2\t4 1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t1 0 1\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t4\t0.0\t2\t1 0 3\n",
      "79\t3\t0.0\t2\t2 1\n",
      "92\t1\t0.0\t2\t1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t2\n",
      "144\t1\t0.0\t2\t1\n",
      "151\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t1\n",
      "172\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "229\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "243\t1\t0.0\t2\t0 0 1\n",
      "244\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.50percent.5.31.19_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.50percent.5.31.19_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.50percent.5.31.19_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.50percent.5.31.19_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.50percent.5.31.19_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.50percent.5.31.19_41_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.50percent.5.31.19_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.50percent.5.31.19_41_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       309,123 reads  @     16.1 µs/read;   3.72 M reads/minute\n",
      "Finished in 5.02 s (16 us/read; 3.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            309,123\n",
      "  Read 1 with adapter:                     347 (0.1%)\n",
      "  Read 2 with adapter:                  11,439 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       309,123 (100.0%)\n",
      "\n",
      "Total basepairs processed:   140,908,327 bp\n",
      "  Read 1:    69,216,198 bp\n",
      "  Read 2:    71,692,129 bp\n",
      "Total written (filtered):    140,742,281 bp (99.9%)\n",
      "  Read 1:    69,214,679 bp\n",
      "  Read 2:    71,527,602 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 347 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t337\t4830.0\t0\t337\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t4\t0.0\t1\t4\n",
      "53\t2\t0.0\t1\t2\n",
      "217\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 11439 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t66\t4830.0\t0\t66\n",
      "4\t18\t1207.5\t0\t18\n",
      "5\t31\t301.9\t0\t31\n",
      "6\t46\t75.5\t0\t46\n",
      "7\t189\t18.9\t0\t189\n",
      "8\t255\t4.7\t0\t255\n",
      "9\t1024\t1.2\t0\t1021 3\n",
      "10\t3101\t0.3\t1\t3067 34\n",
      "11\t163\t0.1\t1\t159 4\n",
      "12\t97\t0.0\t1\t97\n",
      "13\t157\t0.0\t1\t148 9\n",
      "14\t249\t0.0\t1\t232 17\n",
      "15\t273\t0.0\t1\t265 8\n",
      "16\t783\t0.0\t1\t762 21\n",
      "17\t1173\t0.0\t1\t1124 49\n",
      "18\t753\t0.0\t1\t705 38 10\n",
      "19\t2603\t0.0\t1\t1587 1005 11\n",
      "20\t3\t0.0\t2\t1 2\n",
      "21\t417\t0.0\t2\t0 408 9\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t3\t0.0\t2\t1 0 2\n",
      "72\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t1 0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "92\t1\t0.0\t2\t1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t2\n",
      "109\t1\t0.0\t2\t1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "185\t1\t0.0\t2\t0 1\n",
      "186\t3\t0.0\t2\t1 1 1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.5percent.5.29.19_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.5percent.5.29.19_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.5percent.5.29.19_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.5percent.5.29.19_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.5percent.5.29.19_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.5percent.5.29.19_39_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.5percent.5.29.19_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.5percent.5.29.19_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       296,550 reads  @     15.6 µs/read;   3.86 M reads/minute\n",
      "Finished in 4.64 s (16 us/read; 3.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            296,550\n",
      "  Read 1 with adapter:                     258 (0.1%)\n",
      "  Read 2 with adapter:                  10,668 (3.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       296,550 (100.0%)\n",
      "\n",
      "Total basepairs processed:   135,158,236 bp\n",
      "  Read 1:    66,399,586 bp\n",
      "  Read 2:    68,758,650 bp\n",
      "Total written (filtered):    135,007,344 bp (99.9%)\n",
      "  Read 1:    66,398,045 bp\n",
      "  Read 2:    68,609,299 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 258 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t250\t4633.6\t0\t250\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "100\t3\t0.0\t1\t3\n",
      "197\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10668 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t78\t4633.6\t0\t78\n",
      "4\t18\t1158.4\t0\t18\n",
      "5\t23\t289.6\t0\t23\n",
      "6\t57\t72.4\t0\t57\n",
      "7\t182\t18.1\t0\t182\n",
      "8\t233\t4.5\t0\t233\n",
      "9\t888\t1.1\t0\t883 5\n",
      "10\t3343\t0.3\t1\t3305 38\n",
      "11\t177\t0.1\t1\t176 1\n",
      "12\t101\t0.0\t1\t100 1\n",
      "13\t77\t0.0\t1\t76 1\n",
      "14\t157\t0.0\t1\t148 9\n",
      "15\t248\t0.0\t1\t236 12\n",
      "16\t654\t0.0\t1\t629 25\n",
      "17\t1030\t0.0\t1\t1001 29\n",
      "18\t651\t0.0\t1\t595 46 10\n",
      "19\t2361\t0.0\t1\t1496 858 7\n",
      "20\t7\t0.0\t2\t0 6 1\n",
      "21\t361\t0.0\t2\t0 357 4\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "41\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 1\n",
      "81\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t0 1\n",
      "142\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t0 1\n",
      "229\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.75percent.6.4.19_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.75percent.6.4.19_63_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.75percent.6.4.19_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.75percent.6.4.19_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.75percent.6.4.19_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.B.75percent.6.4.19_63_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.75percent.6.4.19_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.B.75percent.6.4.19_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       395,981 reads  @     11.9 µs/read;   5.03 M reads/minute\n",
      "Finished in 4.74 s (12 us/read; 5.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            395,981\n",
      "  Read 1 with adapter:                     402 (0.1%)\n",
      "  Read 2 with adapter:                  21,865 (5.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       395,981 (100.0%)\n",
      "\n",
      "Total basepairs processed:   180,633,959 bp\n",
      "  Read 1:    88,664,560 bp\n",
      "  Read 2:    91,969,399 bp\n",
      "Total written (filtered):    180,345,011 bp (99.8%)\n",
      "  Read 1:    88,662,373 bp\n",
      "  Read 2:    91,682,638 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 402 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t377\t6187.2\t0\t377\n",
      "22\t1\t0.0\t1\t1\n",
      "25\t4\t0.0\t1\t4\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t11\t0.0\t1\t11\n",
      "29\t1\t0.0\t1\t1\n",
      "153\t1\t0.0\t1\t0 1\n",
      "159\t1\t0.0\t1\t0 1\n",
      "166\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 21865 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t178\t6187.2\t0\t178\n",
      "4\t41\t1546.8\t0\t41\n",
      "5\t79\t386.7\t0\t79\n",
      "6\t127\t96.7\t0\t127\n",
      "7\t486\t24.2\t0\t486\n",
      "8\t627\t6.0\t0\t627\n",
      "9\t2730\t1.5\t0\t2716 14\n",
      "10\t7585\t0.4\t1\t7503 82\n",
      "11\t255\t0.1\t1\t254 1\n",
      "12\t179\t0.0\t1\t176 3\n",
      "13\t203\t0.0\t1\t189 14\n",
      "14\t431\t0.0\t1\t415 16\n",
      "15\t464\t0.0\t1\t450 14\n",
      "16\t1277\t0.0\t1\t1243 34\n",
      "17\t1788\t0.0\t1\t1728 60\n",
      "18\t1144\t0.0\t1\t1053 76 15\n",
      "19\t3706\t0.0\t1\t2243 1448 15\n",
      "20\t9\t0.0\t2\t3 5 1\n",
      "21\t519\t0.0\t2\t0 512 7\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "37\t1\t0.0\t2\t1\n",
      "46\t1\t0.0\t2\t0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t1 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t1\n",
      "129\t1\t0.0\t2\t1\n",
      "210\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "246\t1\t0.0\t2\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.25percent.5.31.19_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.25percent.5.31.19_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.25percent.5.31.19_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.25percent.5.31.19_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.25percent.5.31.19_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.25percent.5.31.19_33_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.25percent.5.31.19_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.25percent.5.31.19_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       295,541 reads  @     13.9 µs/read;   4.30 M reads/minute\n",
      "Finished in 4.15 s (14 us/read; 4.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            295,541\n",
      "  Read 1 with adapter:                     286 (0.1%)\n",
      "  Read 2 with adapter:                  13,443 (4.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       295,541 (100.0%)\n",
      "\n",
      "Total basepairs processed:   134,722,257 bp\n",
      "  Read 1:    66,153,143 bp\n",
      "  Read 2:    68,569,114 bp\n",
      "Total written (filtered):    134,538,052 bp (99.9%)\n",
      "  Read 1:    66,150,989 bp\n",
      "  Read 2:    68,387,063 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 286 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t266\t4617.8\t0\t266\n",
      "15\t1\t0.0\t1\t1\n",
      "21\t1\t0.0\t1\t1\n",
      "25\t7\t0.0\t1\t7\n",
      "27\t4\t0.0\t1\t4\n",
      "45\t2\t0.0\t1\t2\n",
      "156\t1\t0.0\t1\t1\n",
      "191\t1\t0.0\t1\t1\n",
      "192\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13443 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t128\t4617.8\t0\t128\n",
      "4\t27\t1154.5\t0\t27\n",
      "5\t44\t288.6\t0\t44\n",
      "6\t70\t72.2\t0\t70\n",
      "7\t296\t18.0\t0\t296\n",
      "8\t385\t4.5\t0\t385\n",
      "9\t1463\t1.1\t0\t1454 9\n",
      "10\t4256\t0.3\t1\t4207 49\n",
      "11\t159\t0.1\t1\t158 1\n",
      "12\t147\t0.0\t1\t144 3\n",
      "13\t121\t0.0\t1\t116 5\n",
      "14\t266\t0.0\t1\t253 13\n",
      "15\t309\t0.0\t1\t300 9\n",
      "16\t846\t0.0\t1\t820 26\n",
      "17\t1205\t0.0\t1\t1178 27\n",
      "18\t755\t0.0\t1\t697 42 16\n",
      "19\t2535\t0.0\t1\t1593 936 6\n",
      "20\t8\t0.0\t2\t3 2 3\n",
      "21\t385\t0.0\t2\t0 374 11\n",
      "22\t17\t0.0\t2\t0 1 16\n",
      "50\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 1\n",
      "69\t3\t0.0\t2\t2 1\n",
      "70\t1\t0.0\t2\t0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t1 1\n",
      "79\t3\t0.0\t2\t2 1\n",
      "87\t1\t0.0\t2\t0 1\n",
      "100\t1\t0.0\t2\t1\n",
      "185\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.50percent.6.3.19_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.50percent.6.3.19_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.50percent.6.3.19_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.50percent.6.3.19_40_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.50percent.6.3.19_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.50percent.6.3.19_40_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.50percent.6.3.19_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.50percent.6.3.19_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       315,270 reads  @     12.8 µs/read;   4.69 M reads/minute\n",
      "Finished in 4.05 s (13 us/read; 4.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            315,270\n",
      "  Read 1 with adapter:                     310 (0.1%)\n",
      "  Read 2 with adapter:                  17,192 (5.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       315,270 (100.0%)\n",
      "\n",
      "Total basepairs processed:   143,840,024 bp\n",
      "  Read 1:    70,605,690 bp\n",
      "  Read 2:    73,234,334 bp\n",
      "Total written (filtered):    143,608,541 bp (99.8%)\n",
      "  Read 1:    70,604,461 bp\n",
      "  Read 2:    73,004,080 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 310 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t303\t4926.1\t0\t303\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "166\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 17192 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t136\t4926.1\t0\t136\n",
      "4\t36\t1231.5\t0\t36\n",
      "5\t49\t307.9\t0\t49\n",
      "6\t113\t77.0\t0\t113\n",
      "7\t374\t19.2\t0\t374\n",
      "8\t569\t4.8\t0\t569\n",
      "9\t2050\t1.2\t0\t2044 6\n",
      "10\t5696\t0.3\t1\t5649 47\n",
      "11\t196\t0.1\t1\t194 2\n",
      "12\t123\t0.0\t1\t119 4\n",
      "13\t165\t0.0\t1\t163 2\n",
      "14\t351\t0.0\t1\t331 20\n",
      "15\t378\t0.0\t1\t360 18\n",
      "16\t999\t0.0\t1\t967 32\n",
      "17\t1504\t0.0\t1\t1450 54\n",
      "18\t916\t0.0\t1\t843 56 17\n",
      "19\t3076\t0.0\t1\t1884 1178 14\n",
      "20\t8\t0.0\t2\t1 6 1\n",
      "21\t405\t0.0\t2\t0 399 6\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t1\n",
      "79\t7\t0.0\t2\t6 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t1\n",
      "151\t1\t0.0\t2\t1\n",
      "168\t1\t0.0\t2\t1\n",
      "212\t1\t0.0\t2\t1\n",
      "217\t3\t0.0\t2\t3\n",
      "223\t1\t0.0\t2\t1\n",
      "235\t1\t0.0\t2\t1\n",
      "240\t2\t0.0\t2\t2\n",
      "243\t2\t0.0\t2\t1 1\n",
      "244\t1\t0.0\t2\t0 0 1\n",
      "246\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.75percent.6.6.19_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.75percent.6.6.19_54_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.75percent.6.6.19_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.75percent.6.6.19_54_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.75percent.6.6.19_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.D.75percent.6.6.19_54_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.75percent.6.6.19_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.D.75percent.6.6.19_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       418,962 reads  @     12.7 µs/read;   4.72 M reads/minute\n",
      "Finished in 5.35 s (13 us/read; 4.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            418,962\n",
      "  Read 1 with adapter:                     506 (0.1%)\n",
      "  Read 2 with adapter:                  29,013 (6.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       418,962 (100.0%)\n",
      "\n",
      "Total basepairs processed:   191,293,616 bp\n",
      "  Read 1:    93,829,947 bp\n",
      "  Read 2:    97,463,669 bp\n",
      "Total written (filtered):    190,917,888 bp (99.8%)\n",
      "  Read 1:    93,827,712 bp\n",
      "  Read 2:    97,090,176 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 506 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t489\t6546.3\t0\t489\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t5\t0.0\t1\t4 1\n",
      "83\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 29013 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t374\t6546.3\t0\t374\n",
      "4\t59\t1636.6\t0\t59\n",
      "5\t95\t409.1\t0\t95\n",
      "6\t196\t102.3\t0\t196\n",
      "7\t739\t25.6\t0\t739\n",
      "8\t903\t6.4\t0\t903\n",
      "9\t3559\t1.6\t0\t3534 25\n",
      "10\t10243\t0.4\t1\t10150 93\n",
      "11\t336\t0.1\t1\t330 6\n",
      "12\t236\t0.0\t1\t231 5\n",
      "13\t276\t0.0\t1\t265 11\n",
      "14\t777\t0.0\t1\t737 40\n",
      "15\t705\t0.0\t1\t674 31\n",
      "16\t1918\t0.0\t1\t1861 57\n",
      "17\t2334\t0.0\t1\t2243 91\n",
      "18\t1351\t0.0\t1\t1222 103 26\n",
      "19\t4288\t0.0\t1\t2381 1891 16\n",
      "20\t14\t0.0\t2\t1 11 2\n",
      "21\t562\t0.0\t2\t0 557 5\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "30\t1\t0.0\t2\t1\n",
      "48\t1\t0.0\t2\t0 1\n",
      "52\t2\t0.0\t2\t2\n",
      "58\t2\t0.0\t2\t1 1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t0 1\n",
      "101\t1\t0.0\t2\t1\n",
      "111\t1\t0.0\t2\t1\n",
      "144\t1\t0.0\t2\t1\n",
      "161\t2\t0.0\t2\t0 1 1\n",
      "186\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "241\t1\t0.0\t2\t0 1\n",
      "246\t1\t0.0\t2\t1\n",
      "248\t2\t0.0\t2\t0 1 1\n",
      "249\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.25percent.5.28.19_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.25percent.5.28.19_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.25percent.5.28.19_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.25percent.5.28.19_27_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.25percent.5.28.19_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.25percent.5.28.19_27_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.25percent.5.28.19_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.25percent.5.28.19_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       316,132 reads  @     13.5 µs/read;   4.43 M reads/minute\n",
      "Finished in 4.32 s (14 us/read; 4.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            316,132\n",
      "  Read 1 with adapter:                     237 (0.1%)\n",
      "  Read 2 with adapter:                  15,662 (5.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       316,132 (100.0%)\n",
      "\n",
      "Total basepairs processed:   144,103,874 bp\n",
      "  Read 1:    70,750,472 bp\n",
      "  Read 2:    73,353,402 bp\n",
      "Total written (filtered):    143,900,266 bp (99.9%)\n",
      "  Read 1:    70,749,515 bp\n",
      "  Read 2:    73,150,751 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 237 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t226\t4939.6\t0\t226\n",
      "18\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t2 1\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15662 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t133\t4939.6\t0\t133\n",
      "4\t35\t1234.9\t0\t35\n",
      "5\t43\t308.7\t0\t43\n",
      "6\t76\t77.2\t0\t76\n",
      "7\t320\t19.3\t0\t320\n",
      "8\t466\t4.8\t0\t466\n",
      "9\t1935\t1.2\t0\t1920 15\n",
      "10\t5891\t0.3\t1\t5836 55\n",
      "11\t231\t0.1\t1\t227 4\n",
      "12\t133\t0.0\t1\t132 1\n",
      "13\t95\t0.0\t1\t93 2\n",
      "14\t245\t0.0\t1\t233 12\n",
      "15\t249\t0.0\t1\t243 6\n",
      "16\t784\t0.0\t1\t751 33\n",
      "17\t1212\t0.0\t1\t1165 47\n",
      "18\t826\t0.0\t1\t766 47 13\n",
      "19\t2564\t0.0\t1\t1639 911 14\n",
      "20\t8\t0.0\t2\t2 4 2\n",
      "21\t387\t0.0\t2\t0 374 13\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "33\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t5\t0.0\t2\t2 2 1\n",
      "149\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.50percent.6.4.19_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.50percent.6.4.19_56_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.50percent.6.4.19_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.50percent.6.4.19_56_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.50percent.6.4.19_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.50percent.6.4.19_56_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.50percent.6.4.19_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.50percent.6.4.19_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       219,083 reads  @     14.9 µs/read;   4.02 M reads/minute\n",
      "Finished in 3.29 s (15 us/read; 3.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            219,083\n",
      "  Read 1 with adapter:                     242 (0.1%)\n",
      "  Read 2 with adapter:                  12,683 (5.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       219,083 (100.0%)\n",
      "\n",
      "Total basepairs processed:    99,762,949 bp\n",
      "  Read 1:    48,951,192 bp\n",
      "  Read 2:    50,811,757 bp\n",
      "Total written (filtered):     99,598,307 bp (99.8%)\n",
      "  Read 1:    48,949,521 bp\n",
      "  Read 2:    50,648,786 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 242 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t232\t3423.2\t0\t232\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t2\t0.0\t1\t2\n",
      "193\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12683 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t117\t3423.2\t0\t117\n",
      "4\t39\t855.8\t0\t39\n",
      "5\t47\t213.9\t0\t47\n",
      "6\t72\t53.5\t0\t72\n",
      "7\t293\t13.4\t0\t293\n",
      "8\t432\t3.3\t0\t432\n",
      "9\t1660\t0.8\t0\t1651 9\n",
      "10\t4546\t0.2\t1\t4494 52\n",
      "11\t127\t0.1\t1\t125 2\n",
      "12\t104\t0.0\t1\t102 2\n",
      "13\t112\t0.0\t1\t109 3\n",
      "14\t293\t0.0\t1\t277 16\n",
      "15\t255\t0.0\t1\t247 8\n",
      "16\t722\t0.0\t1\t695 27\n",
      "17\t970\t0.0\t1\t948 22\n",
      "18\t647\t0.0\t1\t585 50 12\n",
      "19\t1972\t0.0\t1\t1151 809 12\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t257\t0.0\t2\t0 255 2\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "46\t1\t0.0\t2\t0 1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "246\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.75percent.6.4.19_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.75percent.6.4.19_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.75percent.6.4.19_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.75percent.6.4.19_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.75percent.6.4.19_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.1.E.75percent.6.4.19_3_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.75percent.6.4.19_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.1.E.75percent.6.4.19_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       342,415 reads  @     13.1 µs/read;   4.60 M reads/minute\n",
      "Finished in 4.49 s (13 us/read; 4.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            342,415\n",
      "  Read 1 with adapter:                     364 (0.1%)\n",
      "  Read 2 with adapter:                  18,052 (5.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       342,415 (100.0%)\n",
      "\n",
      "Total basepairs processed:   156,092,257 bp\n",
      "  Read 1:    76,621,611 bp\n",
      "  Read 2:    79,470,646 bp\n",
      "Total written (filtered):    155,853,739 bp (99.8%)\n",
      "  Read 1:    76,618,742 bp\n",
      "  Read 2:    79,234,997 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 364 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t341\t5350.2\t0\t341\n",
      "25\t4\t0.0\t1\t4\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t6\t0.0\t1\t6\n",
      "84\t1\t0.0\t1\t1\n",
      "101\t2\t0.0\t1\t2\n",
      "162\t1\t0.0\t1\t1\n",
      "166\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t2 1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 18052 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t166\t5350.2\t0\t166\n",
      "4\t31\t1337.6\t0\t31\n",
      "5\t41\t334.4\t0\t41\n",
      "6\t79\t83.6\t0\t79\n",
      "7\t373\t20.9\t0\t373\n",
      "8\t518\t5.2\t0\t518\n",
      "9\t2178\t1.3\t0\t2169 9\n",
      "10\t6679\t0.3\t1\t6590 89\n",
      "11\t278\t0.1\t1\t276 2\n",
      "12\t154\t0.0\t1\t148 6\n",
      "13\t134\t0.0\t1\t131 3\n",
      "14\t297\t0.0\t1\t283 14\n",
      "15\t324\t0.0\t1\t314 10\n",
      "16\t937\t0.0\t1\t898 39\n",
      "17\t1448\t0.0\t1\t1388 60\n",
      "18\t947\t0.0\t1\t867 64 16\n",
      "19\t2995\t0.0\t1\t1816 1164 15\n",
      "20\t9\t0.0\t2\t4 3 2\n",
      "21\t428\t0.0\t2\t0 417 11\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "216\t4\t0.0\t2\t3 1\n",
      "217\t1\t0.0\t2\t0 0 1\n",
      "241\t1\t0.0\t2\t1\n",
      "247\t1\t0.0\t2\t0 1\n",
      "248\t1\t0.0\t2\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.10.B.final.6.25.19_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.10.B.final.6.25.19_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.10.B.final.6.25.19_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.10.B.final.6.25.19_10_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.10.B.final.6.25.19_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.10.B.final.6.25.19_10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.10.B.final.6.25.19_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.10.B.final.6.25.19_10_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       253,117 reads  @     16.7 µs/read;   3.60 M reads/minute\n",
      "Finished in 4.25 s (17 us/read; 3.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            253,117\n",
      "  Read 1 with adapter:                     238 (0.1%)\n",
      "  Read 2 with adapter:                   9,794 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       253,117 (100.0%)\n",
      "\n",
      "Total basepairs processed:   115,164,377 bp\n",
      "  Read 1:    56,569,712 bp\n",
      "  Read 2:    58,594,665 bp\n",
      "Total written (filtered):    115,023,884 bp (99.9%)\n",
      "  Read 1:    56,568,295 bp\n",
      "  Read 2:    58,455,589 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 238 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t232\t3955.0\t0\t232\n",
      "26\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "205\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9794 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t108\t3955.0\t0\t108\n",
      "4\t13\t988.7\t0\t13\n",
      "5\t20\t247.2\t0\t20\n",
      "6\t47\t61.8\t0\t47\n",
      "7\t135\t15.4\t0\t135\n",
      "8\t217\t3.9\t0\t217\n",
      "9\t823\t1.0\t0\t818 5\n",
      "10\t2837\t0.2\t1\t2799 38\n",
      "11\t125\t0.1\t1\t121 4\n",
      "12\t99\t0.0\t1\t98 1\n",
      "13\t88\t0.0\t1\t84 4\n",
      "14\t203\t0.0\t1\t185 18\n",
      "15\t217\t0.0\t1\t208 9\n",
      "16\t726\t0.0\t1\t711 15\n",
      "17\t1053\t0.0\t1\t1024 29\n",
      "18\t634\t0.0\t1\t582 42 10\n",
      "19\t2097\t0.0\t1\t1324 762 11\n",
      "20\t6\t0.0\t2\t2 3 1\n",
      "21\t316\t0.0\t2\t0 312 4\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t3\n",
      "80\t2\t0.0\t2\t2\n",
      "117\t1\t0.0\t2\t0 1\n",
      "173\t1\t0.0\t2\t1\n",
      "186\t4\t0.0\t2\t3 1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.F.final.6.25.19_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.F.final.6.25.19_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.F.final.6.25.19_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.F.final.6.25.19_44_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.F.final.6.25.19_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.F.final.6.25.19_44_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.F.final.6.25.19_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.F.final.6.25.19_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       189,597 reads  @     16.9 µs/read;   3.56 M reads/minute\n",
      "Finished in 3.22 s (17 us/read; 3.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            189,597\n",
      "  Read 1 with adapter:                     203 (0.1%)\n",
      "  Read 2 with adapter:                   6,019 (3.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       189,597 (100.0%)\n",
      "\n",
      "Total basepairs processed:    86,145,652 bp\n",
      "  Read 1:    42,323,213 bp\n",
      "  Read 2:    43,822,439 bp\n",
      "Total written (filtered):     86,056,758 bp (99.9%)\n",
      "  Read 1:    42,321,611 bp\n",
      "  Read 2:    43,735,147 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 203 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t193\t2962.5\t0\t193\n",
      "24\t2\t0.0\t1\t2\n",
      "27\t4\t0.0\t1\t4\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6019 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t41\t2962.5\t0\t41\n",
      "4\t12\t740.6\t0\t12\n",
      "5\t15\t185.2\t0\t15\n",
      "6\t22\t46.3\t0\t22\n",
      "7\t98\t11.6\t0\t98\n",
      "8\t121\t2.9\t0\t121\n",
      "9\t488\t0.7\t0\t484 4\n",
      "10\t1608\t0.2\t1\t1591 17\n",
      "11\t90\t0.0\t1\t89 1\n",
      "12\t60\t0.0\t1\t59 1\n",
      "13\t55\t0.0\t1\t52 3\n",
      "14\t112\t0.0\t1\t105 7\n",
      "15\t136\t0.0\t1\t133 3\n",
      "16\t391\t0.0\t1\t380 11\n",
      "17\t608\t0.0\t1\t589 19\n",
      "18\t407\t0.0\t1\t378 24 5\n",
      "19\t1511\t0.0\t1\t946 558 7\n",
      "20\t2\t0.0\t2\t1 1\n",
      "21\t228\t0.0\t2\t0 226 2\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t2\t0.0\t2\t2\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.25percent.6.14.19_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.25percent.6.14.19_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.25percent.6.14.19_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.25percent.6.14.19_26_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.25percent.6.14.19_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.25percent.6.14.19_26_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.25percent.6.14.19_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.25percent.6.14.19_26_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       160,805 reads  @     22.0 µs/read;   2.72 M reads/minute\n",
      "Finished in 3.58 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            160,805\n",
      "  Read 1 with adapter:                     248 (0.2%)\n",
      "  Read 2 with adapter:                   6,545 (4.1%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       160,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,011,846 bp\n",
      "  Read 1:    35,851,697 bp\n",
      "  Read 2:    37,160,149 bp\n",
      "Total written (filtered):     72,914,212 bp (99.9%)\n",
      "  Read 1:    35,848,353 bp\n",
      "  Read 2:    37,065,859 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 248 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t232\t2512.6\t0\t232\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t1\n",
      "133\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t2 1\n",
      "217\t6\t0.0\t1\t5 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6545 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t2512.6\t0\t48\n",
      "4\t11\t628.1\t0\t11\n",
      "5\t14\t157.0\t0\t14\n",
      "6\t27\t39.3\t0\t27\n",
      "7\t103\t9.8\t0\t103\n",
      "8\t154\t2.5\t0\t154\n",
      "9\t590\t0.6\t0\t588 2\n",
      "10\t1700\t0.2\t1\t1671 29\n",
      "11\t101\t0.0\t1\t99 2\n",
      "12\t87\t0.0\t1\t84 3\n",
      "13\t71\t0.0\t1\t66 5\n",
      "14\t163\t0.0\t1\t154 9\n",
      "15\t140\t0.0\t1\t133 7\n",
      "16\t495\t0.0\t1\t475 20\n",
      "17\t692\t0.0\t1\t670 22\n",
      "18\t466\t0.0\t1\t426 32 8\n",
      "19\t1466\t0.0\t1\t931 527 8\n",
      "20\t4\t0.0\t2\t2 2\n",
      "21\t188\t0.0\t2\t0 185 3\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "51\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "124\t1\t0.0\t2\t0 1\n",
      "146\t2\t0.0\t2\t2\n",
      "153\t2\t0.0\t2\t0 1 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "251\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.50percent.6.17.19_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.50percent.6.17.19_60_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.50percent.6.17.19_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.50percent.6.17.19_60_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.50percent.6.17.19_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.50percent.6.17.19_60_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.50percent.6.17.19_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.50percent.6.17.19_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       284,570 reads  @     18.1 µs/read;   3.32 M reads/minute\n",
      "Finished in 5.17 s (18 us/read; 3.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            284,570\n",
      "  Read 1 with adapter:                     320 (0.1%)\n",
      "  Read 2 with adapter:                  13,165 (4.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       284,570 (100.0%)\n",
      "\n",
      "Total basepairs processed:   129,724,290 bp\n",
      "  Read 1:    63,698,172 bp\n",
      "  Read 2:    66,026,118 bp\n",
      "Total written (filtered):    129,541,458 bp (99.9%)\n",
      "  Read 1:    63,695,608 bp\n",
      "  Read 2:    65,845,850 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 320 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t303\t4446.4\t0\t303\n",
      "21\t1\t0.0\t1\t0 1\n",
      "25\t4\t0.0\t1\t4\n",
      "26\t5\t0.0\t1\t5\n",
      "151\t1\t0.0\t1\t1\n",
      "191\t1\t0.0\t1\t0 1\n",
      "203\t1\t0.0\t1\t0 1\n",
      "211\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13165 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t113\t4446.4\t0\t113\n",
      "4\t27\t1111.6\t0\t27\n",
      "5\t44\t277.9\t0\t44\n",
      "6\t68\t69.5\t0\t68\n",
      "7\t237\t17.4\t0\t237\n",
      "8\t355\t4.3\t0\t355\n",
      "9\t1420\t1.1\t0\t1411 9\n",
      "10\t4069\t0.3\t1\t4032 37\n",
      "11\t177\t0.1\t1\t174 3\n",
      "12\t150\t0.0\t1\t148 2\n",
      "13\t127\t0.0\t1\t121 6\n",
      "14\t296\t0.0\t1\t282 14\n",
      "15\t302\t0.0\t1\t286 16\n",
      "16\t875\t0.0\t1\t836 39\n",
      "17\t1243\t0.0\t1\t1206 37\n",
      "18\t750\t0.0\t1\t676 45 29\n",
      "19\t2540\t0.0\t1\t1521 1007 12\n",
      "20\t7\t0.0\t2\t2 2 3\n",
      "21\t328\t0.0\t2\t0 317 11\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "49\t1\t0.0\t2\t0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "56\t2\t0.0\t2\t2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 1\n",
      "78\t2\t0.0\t2\t2\n",
      "79\t5\t0.0\t2\t3 0 2\n",
      "140\t1\t0.0\t2\t1\n",
      "161\t1\t0.0\t2\t1\n",
      "166\t1\t0.0\t2\t1\n",
      "167\t1\t0.0\t2\t1\n",
      "183\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "223\t1\t0.0\t2\t1\n",
      "239\t1\t0.0\t2\t1\n",
      "246\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.5percent.6.13.19_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.5percent.6.13.19_64_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.5percent.6.13.19_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.5percent.6.13.19_64_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.5percent.6.13.19_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.5percent.6.13.19_64_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.5percent.6.13.19_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.5percent.6.13.19_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        78,236 reads  @     21.5 µs/read;   2.79 M reads/minute\n",
      "Finished in 1.71 s (22 us/read; 2.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,236\n",
      "  Read 1 with adapter:                      59 (0.1%)\n",
      "  Read 2 with adapter:                   4,752 (6.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        78,236 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,190,163 bp\n",
      "  Read 1:    17,246,924 bp\n",
      "  Read 2:    17,943,239 bp\n",
      "Total written (filtered):     35,123,342 bp (99.8%)\n",
      "  Read 1:    17,243,304 bp\n",
      "  Read 2:    17,880,038 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 59 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t1222.4\t0\t42\n",
      "26\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t4\n",
      "217\t12\t0.0\t1\t12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4752 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t1222.4\t0\t40\n",
      "4\t11\t305.6\t0\t11\n",
      "5\t11\t76.4\t0\t11\n",
      "6\t24\t19.1\t0\t24\n",
      "7\t111\t4.8\t0\t111\n",
      "8\t122\t1.2\t0\t122\n",
      "9\t568\t0.3\t0\t565 3\n",
      "10\t1557\t0.1\t1\t1539 18\n",
      "11\t58\t0.0\t1\t57 1\n",
      "12\t43\t0.0\t1\t42 1\n",
      "13\t41\t0.0\t1\t40 1\n",
      "14\t128\t0.0\t1\t120 8\n",
      "15\t103\t0.0\t1\t98 5\n",
      "16\t332\t0.0\t1\t326 6\n",
      "17\t434\t0.0\t1\t412 22\n",
      "18\t267\t0.0\t1\t244 17 6\n",
      "19\t803\t0.0\t1\t452 346 5\n",
      "20\t1\t0.0\t2\t1\n",
      "21\t87\t0.0\t2\t0 85 2\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "48\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "93\t2\t0.0\t2\t1 0 1\n",
      "237\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.75percent.6.17.19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.75percent.6.17.19_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.75percent.6.17.19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.75percent.6.17.19_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.75percent.6.17.19_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.11.G.75percent.6.17.19_19_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.75percent.6.17.19_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.11.G.75percent.6.17.19_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       298,523 reads  @     18.2 µs/read;   3.30 M reads/minute\n",
      "Finished in 5.47 s (18 us/read; 3.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            298,523\n",
      "  Read 1 with adapter:                     350 (0.1%)\n",
      "  Read 2 with adapter:                  20,136 (6.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       298,523 (100.0%)\n",
      "\n",
      "Total basepairs processed:   136,246,880 bp\n",
      "  Read 1:    66,834,719 bp\n",
      "  Read 2:    69,412,161 bp\n",
      "Total written (filtered):    135,984,869 bp (99.8%)\n",
      "  Read 1:    66,833,197 bp\n",
      "  Read 2:    69,151,672 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 350 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t345\t4664.4\t0\t345\n",
      "25\t2\t0.0\t1\t2\n",
      "89\t1\t0.0\t1\t1\n",
      "131\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 20136 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t153\t4664.4\t0\t153\n",
      "4\t40\t1166.1\t0\t40\n",
      "5\t65\t291.5\t0\t65\n",
      "6\t126\t72.9\t0\t126\n",
      "7\t468\t18.2\t0\t468\n",
      "8\t654\t4.6\t0\t654\n",
      "9\t2706\t1.1\t0\t2694 12\n",
      "10\t7034\t0.3\t1\t6952 82\n",
      "11\t238\t0.1\t1\t234 4\n",
      "12\t199\t0.0\t1\t196 3\n",
      "13\t256\t0.0\t1\t244 12\n",
      "14\t533\t0.0\t1\t513 20\n",
      "15\t415\t0.0\t1\t389 26\n",
      "16\t1208\t0.0\t1\t1156 52\n",
      "17\t1635\t0.0\t1\t1574 61\n",
      "18\t977\t0.0\t1\t886 68 23\n",
      "19\t3006\t0.0\t1\t1772 1222 12\n",
      "20\t4\t0.0\t2\t0 2 2\n",
      "21\t371\t0.0\t2\t0 364 7\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t1\t0.0\t2\t0 1\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t1 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t1 1\n",
      "79\t8\t0.0\t2\t7 1\n",
      "92\t1\t0.0\t2\t0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "210\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "227\t1\t0.0\t2\t1\n",
      "241\t2\t0.0\t2\t2\n",
      "244\t1\t0.0\t2\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.A.control.6.6.19_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.A.control.6.6.19_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.A.control.6.6.19_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.A.control.6.6.19_38_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.A.control.6.6.19_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.A.control.6.6.19_38_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.A.control.6.6.19_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.A.control.6.6.19_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       220,003 reads  @     14.1 µs/read;   4.25 M reads/minute\n",
      "Finished in 3.14 s (14 us/read; 4.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            220,003\n",
      "  Read 1 with adapter:                     225 (0.1%)\n",
      "  Read 2 with adapter:                   7,966 (3.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       220,003 (100.0%)\n",
      "\n",
      "Total basepairs processed:   100,177,886 bp\n",
      "  Read 1:    49,207,712 bp\n",
      "  Read 2:    50,970,174 bp\n",
      "Total written (filtered):    100,067,092 bp (99.9%)\n",
      "  Read 1:    49,206,799 bp\n",
      "  Read 2:    50,860,293 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 225 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t223\t3437.5\t0\t223\n",
      "27\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7966 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t3437.5\t0\t56\n",
      "4\t19\t859.4\t0\t19\n",
      "5\t20\t214.8\t0\t20\n",
      "6\t40\t53.7\t0\t40\n",
      "7\t113\t13.4\t0\t113\n",
      "8\t182\t3.4\t0\t182\n",
      "9\t801\t0.8\t0\t797 4\n",
      "10\t2546\t0.2\t1\t2521 25\n",
      "11\t139\t0.1\t1\t135 4\n",
      "12\t81\t0.0\t1\t79 2\n",
      "13\t70\t0.0\t1\t69 1\n",
      "14\t116\t0.0\t1\t106 10\n",
      "15\t156\t0.0\t1\t149 7\n",
      "16\t431\t0.0\t1\t420 11\n",
      "17\t728\t0.0\t1\t698 30\n",
      "18\t477\t0.0\t1\t444 28 5\n",
      "19\t1697\t0.0\t1\t1049 635 13\n",
      "20\t6\t0.0\t2\t0 3 3\n",
      "21\t269\t0.0\t2\t0 263 6\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "53\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t1 1 1\n",
      "90\t1\t0.0\t2\t0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.F.control.5.29.19_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.F.control.5.29.19_67_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.F.control.5.29.19_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.F.control.5.29.19_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.F.control.5.29.19_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.F.control.5.29.19_67_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.F.control.5.29.19_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.F.control.5.29.19_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[8=----------] 00:00:03       208,808 reads  @     14.5 µs/read;   4.15 M reads/minute\n",
      "Finished in 3.05 s (15 us/read; 4.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            208,808\n",
      "  Read 1 with adapter:                     150 (0.1%)\n",
      "  Read 2 with adapter:                   7,557 (3.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       208,808 (100.0%)\n",
      "\n",
      "Total basepairs processed:    94,964,335 bp\n",
      "  Read 1:    46,644,415 bp\n",
      "  Read 2:    48,319,920 bp\n",
      "Total written (filtered):     94,855,869 bp (99.9%)\n",
      "  Read 1:    46,642,210 bp\n",
      "  Read 2:    48,213,659 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 150 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t131\t3262.6\t0\t131\n",
      "21\t2\t0.0\t1\t2\n",
      "24\t3\t0.0\t1\t3\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t4\t0.0\t1\t4\n",
      "213\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t5\t0.0\t1\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7557 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t3262.6\t0\t39\n",
      "4\t14\t815.7\t0\t14\n",
      "5\t12\t203.9\t0\t12\n",
      "6\t16\t51.0\t0\t16\n",
      "7\t107\t12.7\t0\t107\n",
      "8\t152\t3.2\t0\t152\n",
      "9\t654\t0.8\t0\t649 5\n",
      "10\t2278\t0.2\t1\t2249 29\n",
      "11\t119\t0.0\t1\t119\n",
      "12\t71\t0.0\t1\t68 3\n",
      "13\t62\t0.0\t1\t61 1\n",
      "14\t131\t0.0\t1\t120 11\n",
      "15\t192\t0.0\t1\t187 5\n",
      "16\t496\t0.0\t1\t476 20\n",
      "17\t789\t0.0\t1\t765 24\n",
      "18\t463\t0.0\t1\t431 31 1\n",
      "19\t1704\t0.0\t1\t1076 626 2\n",
      "20\t4\t0.0\t2\t0 2 2\n",
      "21\t239\t0.0\t2\t0 235 4\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.F.control.6.6.19_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.F.control.6.6.19_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.F.control.6.6.19_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.F.control.6.6.19_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.F.control.6.6.19_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.F.control.6.6.19_17_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.F.control.6.6.19_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.F.control.6.6.19_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       191,682 reads  @     18.4 µs/read;   3.26 M reads/minute\n",
      "Finished in 3.56 s (19 us/read; 3.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            191,682\n",
      "  Read 1 with adapter:                     145 (0.1%)\n",
      "  Read 2 with adapter:                   6,605 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       191,682 (100.0%)\n",
      "\n",
      "Total basepairs processed:    87,081,112 bp\n",
      "  Read 1:    42,774,436 bp\n",
      "  Read 2:    44,306,676 bp\n",
      "Total written (filtered):     86,982,998 bp (99.9%)\n",
      "  Read 1:    42,772,087 bp\n",
      "  Read 2:    44,210,911 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 145 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t133\t2995.0\t0\t133\n",
      "27\t3\t0.0\t1\t3\n",
      "133\t1\t0.0\t1\t0 1\n",
      "217\t8\t0.0\t1\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6605 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t2995.0\t0\t50\n",
      "4\t14\t748.8\t0\t14\n",
      "5\t15\t187.2\t0\t15\n",
      "6\t23\t46.8\t0\t23\n",
      "7\t110\t11.7\t0\t110\n",
      "8\t121\t2.9\t0\t121\n",
      "9\t520\t0.7\t0\t516 4\n",
      "10\t1676\t0.2\t1\t1654 22\n",
      "11\t92\t0.0\t1\t91 1\n",
      "12\t79\t0.0\t1\t78 1\n",
      "13\t59\t0.0\t1\t58 1\n",
      "14\t155\t0.0\t1\t144 11\n",
      "15\t153\t0.0\t1\t148 5\n",
      "16\t487\t0.0\t1\t472 15\n",
      "17\t747\t0.0\t1\t728 19\n",
      "18\t464\t0.0\t1\t421 35 8\n",
      "19\t1581\t0.0\t1\t1000 577 4\n",
      "21\t240\t0.0\t2\t0 235 5\n",
      "22\t10\t0.0\t2\t0 1 9\n",
      "70\t3\t0.0\t2\t1 1 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t2\n",
      "100\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.13.19_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.13.19_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.13.19_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.13.19_12_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.13.19_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.13.19_12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.13.19_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.13.19_12_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       160,762 reads  @     16.9 µs/read;   3.55 M reads/minute\n",
      "Finished in 2.74 s (17 us/read; 3.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            160,762\n",
      "  Read 1 with adapter:                     105 (0.1%)\n",
      "  Read 2 with adapter:                   3,608 (2.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       160,762 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,167,218 bp\n",
      "  Read 1:    35,965,589 bp\n",
      "  Read 2:    37,201,629 bp\n",
      "Total written (filtered):     73,109,152 bp (99.9%)\n",
      "  Read 1:    35,964,829 bp\n",
      "  Read 2:    37,144,323 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 105 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t102\t2511.9\t0\t102\n",
      "25\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3608 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t2511.9\t0\t13\n",
      "4\t4\t628.0\t0\t4\n",
      "5\t6\t157.0\t0\t6\n",
      "6\t11\t39.2\t0\t11\n",
      "7\t30\t9.8\t0\t30\n",
      "8\t45\t2.5\t0\t45\n",
      "9\t192\t0.6\t0\t192\n",
      "10\t591\t0.2\t1\t583 8\n",
      "11\t51\t0.0\t1\t50 1\n",
      "12\t63\t0.0\t1\t63\n",
      "13\t50\t0.0\t1\t49 1\n",
      "14\t70\t0.0\t1\t63 7\n",
      "15\t100\t0.0\t1\t98 2\n",
      "16\t288\t0.0\t1\t282 6\n",
      "17\t448\t0.0\t1\t436 12\n",
      "18\t339\t0.0\t1\t319 14 6\n",
      "19\t1119\t0.0\t1\t711 404 4\n",
      "20\t2\t0.0\t2\t1 1\n",
      "21\t173\t0.0\t2\t0 172 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t1\n",
      "70\t2\t0.0\t2\t2\n",
      "79\t2\t0.0\t2\t2\n",
      "100\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "246\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.14.19_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.14.19_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.14.19_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.14.19_48_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.14.19_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.14.19_48_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.14.19_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.14.19_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       242,196 reads  @     17.0 µs/read;   3.52 M reads/minute\n",
      "Finished in 4.15 s (17 us/read; 3.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            242,196\n",
      "  Read 1 with adapter:                     408 (0.2%)\n",
      "  Read 2 with adapter:                   7,230 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       242,196 (100.0%)\n",
      "\n",
      "Total basepairs processed:   110,356,438 bp\n",
      "  Read 1:    54,228,655 bp\n",
      "  Read 2:    56,127,783 bp\n",
      "Total written (filtered):    110,246,809 bp (99.9%)\n",
      "  Read 1:    54,226,553 bp\n",
      "  Read 2:    56,020,256 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 408 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t395\t3784.3\t0\t395\n",
      "9\t2\t0.9\t0\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "55\t2\t0.0\t1\t2\n",
      "74\t1\t0.0\t1\t1\n",
      "117\t1\t0.0\t1\t1\n",
      "118\t1\t0.0\t1\t1\n",
      "174\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7230 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t3784.3\t0\t37\n",
      "4\t12\t946.1\t0\t12\n",
      "5\t11\t236.5\t0\t11\n",
      "6\t21\t59.1\t0\t21\n",
      "7\t93\t14.8\t0\t93\n",
      "8\t150\t3.7\t0\t150\n",
      "9\t574\t0.9\t0\t573 1\n",
      "10\t1641\t0.2\t1\t1622 19\n",
      "11\t114\t0.1\t1\t113 1\n",
      "12\t89\t0.0\t1\t85 4\n",
      "13\t97\t0.0\t1\t95 2\n",
      "14\t144\t0.0\t1\t133 11\n",
      "15\t208\t0.0\t1\t204 4\n",
      "16\t483\t0.0\t1\t470 13\n",
      "17\t811\t0.0\t1\t787 24\n",
      "18\t523\t0.0\t1\t495 22 6\n",
      "19\t1888\t0.0\t1\t1220 667 1\n",
      "20\t5\t0.0\t2\t2 3\n",
      "21\t307\t0.0\t2\t0 302 5\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "52\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "68\t2\t0.0\t2\t2\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.17.19_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.17.19_51_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.17.19_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.17.19_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.17.19_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.13.G.control.5.17.19_51_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.17.19_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.13.G.control.5.17.19_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       127,411 reads  @     18.7 µs/read;   3.21 M reads/minute\n",
      "Finished in 2.41 s (19 us/read; 3.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            127,411\n",
      "  Read 1 with adapter:                     158 (0.1%)\n",
      "  Read 2 with adapter:                   5,360 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       127,411 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,784,505 bp\n",
      "  Read 1:    28,362,808 bp\n",
      "  Read 2:    29,421,697 bp\n",
      "Total written (filtered):     57,704,991 bp (99.9%)\n",
      "  Read 1:    28,360,176 bp\n",
      "  Read 2:    29,344,815 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 158 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t146\t1990.8\t0\t146\n",
      "5\t1\t124.4\t0\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t2 1\n",
      "217\t6\t0.0\t1\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5360 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t1990.8\t0\t48\n",
      "4\t9\t497.7\t0\t9\n",
      "5\t11\t124.4\t0\t11\n",
      "6\t21\t31.1\t0\t21\n",
      "7\t90\t7.8\t0\t90\n",
      "8\t115\t1.9\t0\t115\n",
      "9\t480\t0.5\t0\t474 6\n",
      "10\t1514\t0.1\t1\t1492 22\n",
      "11\t66\t0.0\t1\t65 1\n",
      "12\t57\t0.0\t1\t56 1\n",
      "13\t74\t0.0\t1\t69 5\n",
      "14\t112\t0.0\t1\t104 8\n",
      "15\t151\t0.0\t1\t145 6\n",
      "16\t369\t0.0\t1\t355 14\n",
      "17\t536\t0.0\t1\t518 18\n",
      "18\t323\t0.0\t1\t294 24 5\n",
      "19\t1167\t0.0\t1\t740 418 9\n",
      "20\t17\t0.0\t2\t15 2\n",
      "21\t176\t0.0\t2\t0 173 3\n",
      "22\t8\t0.0\t2\t0 1 7\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t3\t0.0\t2\t0 2 1\n",
      "183\t1\t0.0\t2\t0 1\n",
      "238\t2\t0.0\t2\t2\n",
      "245\t1\t0.0\t2\t1\n",
      "248\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.A.final.6.25.19_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.A.final.6.25.19_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.A.final.6.25.19_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.A.final.6.25.19_11_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.A.final.6.25.19_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.A.final.6.25.19_11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.A.final.6.25.19_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.A.final.6.25.19_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01       110,618 reads  @     18.0 µs/read;   3.33 M reads/minute\n",
      "Finished in 2.03 s (18 us/read; 3.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,618\n",
      "  Read 1 with adapter:                     111 (0.1%)\n",
      "  Read 2 with adapter:                   4,377 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       110,618 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,019,522 bp\n",
      "  Read 1:    24,557,392 bp\n",
      "  Read 2:    25,462,130 bp\n",
      "Total written (filtered):     49,955,591 bp (99.9%)\n",
      "  Read 1:    24,556,049 bp\n",
      "  Read 2:    25,399,542 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 111 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t106\t1728.4\t0\t106\n",
      "160\t1\t0.0\t1\t1\n",
      "216\t3\t0.0\t1\t2 1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4377 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t1728.4\t0\t34\n",
      "4\t11\t432.1\t0\t11\n",
      "5\t10\t108.0\t0\t10\n",
      "6\t17\t27.0\t0\t17\n",
      "7\t60\t6.8\t0\t60\n",
      "8\t102\t1.7\t0\t102\n",
      "9\t388\t0.4\t0\t385 3\n",
      "10\t1140\t0.1\t1\t1126 14\n",
      "11\t60\t0.0\t1\t60\n",
      "12\t41\t0.0\t1\t41\n",
      "13\t45\t0.0\t1\t45\n",
      "14\t94\t0.0\t1\t83 11\n",
      "15\t116\t0.0\t1\t111 5\n",
      "16\t306\t0.0\t1\t297 9\n",
      "17\t481\t0.0\t1\t464 17\n",
      "18\t310\t0.0\t1\t279 20 11\n",
      "19\t992\t0.0\t1\t620 369 3\n",
      "20\t1\t0.0\t2\t1\n",
      "21\t160\t0.0\t2\t0 157 3\n",
      "22\t5\t0.0\t2\t0 0 5\n",
      "47\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "212\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.25percent.5.13.19_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.25percent.5.13.19_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.25percent.5.13.19_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.25percent.5.13.19_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.25percent.5.13.19_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.25percent.5.13.19_21_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.25percent.5.13.19_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.25percent.5.13.19_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       257,651 reads  @     14.6 µs/read;   4.11 M reads/minute\n",
      "Finished in 3.79 s (15 us/read; 4.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            257,651\n",
      "  Read 1 with adapter:                     253 (0.1%)\n",
      "  Read 2 with adapter:                   5,029 (2.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       257,651 (100.0%)\n",
      "\n",
      "Total basepairs processed:   117,363,879 bp\n",
      "  Read 1:    57,701,407 bp\n",
      "  Read 2:    59,662,472 bp\n",
      "Total written (filtered):    117,280,323 bp (99.9%)\n",
      "  Read 1:    57,700,467 bp\n",
      "  Read 2:    59,579,856 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 253 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t251\t4025.8\t0\t251\n",
      "27\t1\t0.0\t1\t1\n",
      "160\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5029 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t4025.8\t0\t17\n",
      "4\t6\t1006.4\t0\t6\n",
      "5\t17\t251.6\t0\t17\n",
      "6\t29\t62.9\t0\t29\n",
      "7\t62\t15.7\t0\t62\n",
      "8\t91\t3.9\t0\t91\n",
      "9\t238\t1.0\t0\t237 1\n",
      "10\t517\t0.2\t1\t512 5\n",
      "11\t37\t0.1\t1\t36 1\n",
      "12\t49\t0.0\t1\t47 2\n",
      "13\t63\t0.0\t1\t59 4\n",
      "14\t99\t0.0\t1\t91 8\n",
      "15\t114\t0.0\t1\t110 4\n",
      "16\t378\t0.0\t1\t361 17\n",
      "17\t687\t0.0\t1\t664 23\n",
      "18\t461\t0.0\t1\t421 32 8\n",
      "19\t1835\t0.0\t1\t1187 646 2\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t310\t0.0\t2\t0 306 4\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "115\t1\t0.0\t2\t1\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.50percent.5.14.19_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.50percent.5.14.19_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.50percent.5.14.19_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.50percent.5.14.19_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.50percent.5.14.19_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.50percent.5.14.19_29_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.50percent.5.14.19_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.50percent.5.14.19_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       341,194 reads  @     14.0 µs/read;   4.27 M reads/minute\n",
      "Finished in 4.82 s (14 us/read; 4.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            341,194\n",
      "  Read 1 with adapter:                     470 (0.1%)\n",
      "  Read 2 with adapter:                  15,481 (4.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       341,194 (100.0%)\n",
      "\n",
      "Total basepairs processed:   155,611,997 bp\n",
      "  Read 1:    76,410,496 bp\n",
      "  Read 2:    79,201,501 bp\n",
      "Total written (filtered):    155,398,819 bp (99.9%)\n",
      "  Read 1:    76,408,194 bp\n",
      "  Read 2:    78,990,625 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 470 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t454\t5331.2\t0\t454\n",
      "21\t1\t0.0\t1\t1\n",
      "24\t3\t0.0\t1\t3\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t2\t0.0\t1\t2\n",
      "174\t1\t0.0\t1\t1\n",
      "219\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15481 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t135\t5331.2\t0\t135\n",
      "4\t32\t1332.8\t0\t32\n",
      "5\t38\t333.2\t0\t38\n",
      "6\t75\t83.3\t0\t75\n",
      "7\t279\t20.8\t0\t279\n",
      "8\t408\t5.2\t0\t408\n",
      "9\t1600\t1.3\t0\t1595 5\n",
      "10\t4978\t0.3\t1\t4924 54\n",
      "11\t208\t0.1\t1\t207 1\n",
      "12\t141\t0.0\t1\t139 2\n",
      "13\t121\t0.0\t1\t117 4\n",
      "14\t298\t0.0\t1\t283 15\n",
      "15\t344\t0.0\t1\t331 13\n",
      "16\t953\t0.0\t1\t925 28\n",
      "17\t1376\t0.0\t1\t1331 45\n",
      "18\t933\t0.0\t1\t854 61 18\n",
      "19\t3104\t0.0\t1\t1909 1181 14\n",
      "20\t2\t0.0\t2\t1 1\n",
      "21\t423\t0.0\t2\t0 420 3\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "55\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "76\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t2 1 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "240\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.5percent.5.12.19_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.5percent.5.12.19_53_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.5percent.5.12.19_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.5percent.5.12.19_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.5percent.5.12.19_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.5percent.5.12.19_53_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.5percent.5.12.19_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.5percent.5.12.19_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       413,362 reads  @     13.1 µs/read;   4.58 M reads/minute\n",
      "Finished in 5.44 s (13 us/read; 4.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            413,362\n",
      "  Read 1 with adapter:                     403 (0.1%)\n",
      "  Read 2 with adapter:                  14,306 (3.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       413,362 (100.0%)\n",
      "\n",
      "Total basepairs processed:   188,432,994 bp\n",
      "  Read 1:    92,572,329 bp\n",
      "  Read 2:    95,860,665 bp\n",
      "Total written (filtered):    188,223,209 bp (99.9%)\n",
      "  Read 1:    92,570,696 bp\n",
      "  Read 2:    95,652,513 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 403 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t389\t6458.8\t0\t389\n",
      "9\t1\t1.6\t0\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t5\t0.0\t1\t5\n",
      "29\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "119\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14306 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t98\t6458.8\t0\t98\n",
      "4\t50\t1614.7\t0\t50\n",
      "5\t64\t403.7\t0\t64\n",
      "6\t154\t100.9\t0\t154\n",
      "7\t321\t25.2\t0\t321\n",
      "8\t525\t6.3\t0\t525\n",
      "9\t1945\t1.6\t0\t1942 3\n",
      "10\t2438\t0.4\t1\t2420 18\n",
      "11\t114\t0.1\t1\t112 2\n",
      "12\t170\t0.0\t1\t163 7\n",
      "13\t191\t0.0\t1\t184 7\n",
      "14\t347\t0.0\t1\t324 23\n",
      "15\t296\t0.0\t1\t277 19\n",
      "16\t934\t0.0\t1\t905 29\n",
      "17\t1577\t0.0\t1\t1512 65\n",
      "18\t977\t0.0\t1\t885 61 31\n",
      "19\t3468\t0.0\t1\t2165 1292 11\n",
      "20\t17\t0.0\t2\t8 5 4\n",
      "21\t538\t0.0\t2\t0 534 4\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "30\t1\t0.0\t2\t1\n",
      "38\t1\t0.0\t2\t1\n",
      "40\t1\t0.0\t2\t0 1\n",
      "49\t2\t0.0\t2\t1 1\n",
      "50\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t0 1\n",
      "53\t1\t0.0\t2\t0 1\n",
      "54\t1\t0.0\t2\t0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t2\t0.0\t2\t2\n",
      "63\t2\t0.0\t2\t2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t2\t0.0\t2\t2\n",
      "68\t2\t0.0\t2\t2\n",
      "69\t2\t0.0\t2\t1 0 1\n",
      "70\t2\t0.0\t2\t1 1\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "74\t1\t0.0\t2\t0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t10\t0.0\t2\t7 3\n",
      "81\t1\t0.0\t2\t1\n",
      "86\t1\t0.0\t2\t1\n",
      "94\t1\t0.0\t2\t1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t1\n",
      "145\t2\t0.0\t2\t2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t1\n",
      "173\t1\t0.0\t2\t0 1\n",
      "179\t1\t0.0\t2\t0 1\n",
      "196\t1\t0.0\t2\t0 1\n",
      "204\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "237\t1\t0.0\t2\t0 1\n",
      "242\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.75percent.5.17.19_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.75percent.5.17.19_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.75percent.5.17.19_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.75percent.5.17.19_24_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.75percent.5.17.19_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.15.G.75percent.5.17.19_24_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.75percent.5.17.19_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.15.G.75percent.5.17.19_24_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       376,997 reads  @     12.7 µs/read;   4.72 M reads/minute\n",
      "Finished in 4.82 s (13 us/read; 4.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            376,997\n",
      "  Read 1 with adapter:                     305 (0.1%)\n",
      "  Read 2 with adapter:                  24,256 (6.4%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       376,996 (100.0%)\n",
      "\n",
      "Total basepairs processed:   172,112,681 bp\n",
      "  Read 1:    84,425,964 bp\n",
      "  Read 2:    87,686,717 bp\n",
      "Total written (filtered):    171,795,141 bp (99.8%)\n",
      "  Read 1:    84,424,418 bp\n",
      "  Read 2:    87,370,723 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 305 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t294\t5890.6\t0\t294\n",
      "19\t1\t0.0\t1\t1\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t4\t0.0\t1\t3 1\n",
      "27\t1\t0.0\t1\t1\n",
      "200\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 24256 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t184\t5890.6\t0\t184\n",
      "4\t124\t1472.6\t0\t124\n",
      "5\t161\t368.2\t0\t161\n",
      "6\t402\t92.0\t0\t402\n",
      "7\t1087\t23.0\t0\t1087\n",
      "8\t1641\t5.8\t0\t1641\n",
      "9\t4762\t1.4\t0\t4752 10\n",
      "10\t4401\t0.4\t1\t4355 46\n",
      "11\t224\t0.1\t1\t221 3\n",
      "12\t328\t0.0\t1\t318 10\n",
      "13\t478\t0.0\t1\t460 18\n",
      "14\t827\t0.0\t1\t784 43\n",
      "15\t594\t0.0\t1\t567 27\n",
      "16\t1428\t0.0\t1\t1370 58\n",
      "17\t1998\t0.0\t1\t1937 61\n",
      "18\t1192\t0.0\t1\t1047 86 59\n",
      "19\t3788\t0.0\t1\t2197 1577 14\n",
      "20\t7\t0.0\t2\t3 2 2\n",
      "21\t517\t0.0\t2\t0 506 11\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "29\t1\t0.0\t2\t1\n",
      "30\t4\t0.0\t2\t4\n",
      "34\t1\t0.0\t2\t1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t1\n",
      "47\t1\t0.0\t2\t1\n",
      "51\t3\t0.0\t2\t3\n",
      "53\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t2\t0.0\t2\t2\n",
      "58\t2\t0.0\t2\t1 1\n",
      "59\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t7\t0.0\t2\t7\n",
      "64\t2\t0.0\t2\t1 1\n",
      "65\t5\t0.0\t2\t3 1 1\n",
      "66\t2\t0.0\t2\t2\n",
      "67\t3\t0.0\t2\t1 0 2\n",
      "68\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t2\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t4\t0.0\t2\t2 0 2\n",
      "73\t3\t0.0\t2\t3\n",
      "76\t1\t0.0\t2\t0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "79\t5\t0.0\t2\t4 1\n",
      "81\t1\t0.0\t2\t1\n",
      "82\t1\t0.0\t2\t1\n",
      "83\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t1 1\n",
      "129\t1\t0.0\t2\t1\n",
      "172\t1\t0.0\t2\t1\n",
      "194\t1\t0.0\t2\t1\n",
      "228\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t2\t0.0\t2\t0 2\n",
      "238\t1\t0.0\t2\t0 0 1\n",
      "240\t1\t0.0\t2\t1\n",
      "241\t5\t0.0\t2\t2 3\n",
      "242\t3\t0.0\t2\t1 2\n",
      "243\t1\t0.0\t2\t1\n",
      "245\t2\t0.0\t2\t2\n",
      "249\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.A.control.6.5.19_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.A.control.6.5.19_58_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.A.control.6.5.19_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.A.control.6.5.19_58_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.A.control.6.5.19_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.A.control.6.5.19_58_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.A.control.6.5.19_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.A.control.6.5.19_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03       179,056 reads  @     17.0 µs/read;   3.52 M reads/minute\n",
      "Finished in 3.08 s (17 us/read; 3.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            179,056\n",
      "  Read 1 with adapter:                     136 (0.1%)\n",
      "  Read 2 with adapter:                   7,776 (4.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       179,056 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,337,274 bp\n",
      "  Read 1:    39,935,670 bp\n",
      "  Read 2:    41,401,604 bp\n",
      "Total written (filtered):     81,227,049 bp (99.9%)\n",
      "  Read 1:    39,933,909 bp\n",
      "  Read 2:    41,293,140 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 136 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t125\t2797.8\t0\t125\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "99\t1\t0.0\t1\t0 1\n",
      "101\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t2 2\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7776 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t2797.8\t0\t77\n",
      "4\t11\t699.4\t0\t11\n",
      "5\t15\t174.9\t0\t15\n",
      "6\t32\t43.7\t0\t32\n",
      "7\t134\t10.9\t0\t134\n",
      "8\t156\t2.7\t0\t156\n",
      "9\t695\t0.7\t0\t691 4\n",
      "10\t2286\t0.2\t1\t2263 23\n",
      "11\t109\t0.0\t1\t107 2\n",
      "12\t90\t0.0\t1\t90\n",
      "13\t60\t0.0\t1\t59 1\n",
      "14\t184\t0.0\t1\t176 8\n",
      "15\t188\t0.0\t1\t178 10\n",
      "16\t551\t0.0\t1\t536 15\n",
      "17\t841\t0.0\t1\t812 29\n",
      "18\t471\t0.0\t1\t438 23 10\n",
      "19\t1633\t0.0\t1\t1017 610 6\n",
      "20\t2\t0.0\t2\t1 1\n",
      "21\t229\t0.0\t2\t0 226 3\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.D.control.6.3.19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.D.control.6.3.19_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.D.control.6.3.19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.D.control.6.3.19_28_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.D.control.6.3.19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.D.control.6.3.19_28_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.D.control.6.3.19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.D.control.6.3.19_28_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       134,869 reads  @     17.7 µs/read;   3.39 M reads/minute\n",
      "Finished in 2.41 s (18 us/read; 3.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            134,869\n",
      "  Read 1 with adapter:                     129 (0.1%)\n",
      "  Read 2 with adapter:                   3,266 (2.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       134,869 (100.0%)\n",
      "\n",
      "Total basepairs processed:    61,381,064 bp\n",
      "  Read 1:    30,169,405 bp\n",
      "  Read 2:    31,211,659 bp\n",
      "Total written (filtered):     61,329,523 bp (99.9%)\n",
      "  Read 1:    30,168,732 bp\n",
      "  Read 2:    31,160,791 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 129 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t125\t2107.3\t0\t125\n",
      "27\t3\t0.0\t1\t3\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3266 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t2107.3\t0\t14\n",
      "4\t8\t526.8\t0\t8\n",
      "5\t6\t131.7\t0\t6\n",
      "6\t7\t32.9\t0\t7\n",
      "7\t33\t8.2\t0\t33\n",
      "8\t45\t2.1\t0\t45\n",
      "9\t159\t0.5\t0\t159\n",
      "10\t594\t0.1\t1\t586 8\n",
      "11\t52\t0.0\t1\t50 2\n",
      "12\t39\t0.0\t1\t38 1\n",
      "13\t23\t0.0\t1\t23\n",
      "14\t76\t0.0\t1\t72 4\n",
      "15\t98\t0.0\t1\t95 3\n",
      "16\t252\t0.0\t1\t246 6\n",
      "17\t356\t0.0\t1\t341 15\n",
      "18\t309\t0.0\t1\t289 14 6\n",
      "19\t1020\t0.0\t1\t668 348 4\n",
      "20\t5\t0.0\t2\t0 4 1\n",
      "21\t167\t0.0\t2\t0 164 3\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.D.control.6.5.19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.D.control.6.5.19_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.D.control.6.5.19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.D.control.6.5.19_30_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.D.control.6.5.19_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.D.control.6.5.19_30_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.D.control.6.5.19_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.D.control.6.5.19_30_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       144,061 reads  @     16.9 µs/read;   3.56 M reads/minute\n",
      "Finished in 2.45 s (17 us/read; 3.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            144,061\n",
      "  Read 1 with adapter:                     223 (0.2%)\n",
      "  Read 2 with adapter:                   3,292 (2.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       144,061 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,553,563 bp\n",
      "  Read 1:    32,222,185 bp\n",
      "  Read 2:    33,331,378 bp\n",
      "Total written (filtered):     65,499,825 bp (99.9%)\n",
      "  Read 1:    32,220,711 bp\n",
      "  Read 2:    33,279,114 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 223 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t218\t2251.0\t0\t218\n",
      "27\t1\t0.0\t1\t1\n",
      "167\t1\t0.0\t1\t1\n",
      "194\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3292 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t2251.0\t0\t11\n",
      "4\t9\t562.7\t0\t9\n",
      "5\t3\t140.7\t0\t3\n",
      "6\t4\t35.2\t0\t4\n",
      "7\t29\t8.8\t0\t29\n",
      "8\t40\t2.2\t0\t40\n",
      "9\t152\t0.5\t0\t150 2\n",
      "10\t565\t0.1\t1\t555 10\n",
      "11\t57\t0.0\t1\t55 2\n",
      "12\t55\t0.0\t1\t54 1\n",
      "13\t49\t0.0\t1\t48 1\n",
      "14\t80\t0.0\t1\t75 5\n",
      "15\t90\t0.0\t1\t86 4\n",
      "16\t240\t0.0\t1\t235 5\n",
      "17\t375\t0.0\t1\t368 7\n",
      "18\t272\t0.0\t1\t253 16 3\n",
      "19\t1084\t0.0\t1\t691 388 5\n",
      "20\t1\t0.0\t2\t0 1\n",
      "21\t168\t0.0\t2\t0 165 3\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 1\n",
      "95\t1\t0.0\t2\t1\n",
      "102\t1\t0.0\t2\t0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "237\t1\t0.0\t2\t1\n",
      "244\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.10.19_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.10.19_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.10.19_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.10.19_35_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.10.19_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.10.19_35_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.10.19_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.10.19_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       256,535 reads  @     14.0 µs/read;   4.27 M reads/minute\n",
      "Finished in 3.63 s (14 us/read; 4.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            256,535\n",
      "  Read 1 with adapter:                     253 (0.1%)\n",
      "  Read 2 with adapter:                  10,857 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       256,535 (100.0%)\n",
      "\n",
      "Total basepairs processed:   116,832,326 bp\n",
      "  Read 1:    57,373,207 bp\n",
      "  Read 2:    59,459,119 bp\n",
      "Total written (filtered):    116,683,025 bp (99.9%)\n",
      "  Read 1:    57,370,274 bp\n",
      "  Read 2:    59,312,751 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 253 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t228\t4008.4\t0\t228\n",
      "12\t2\t0.0\t1\t2\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "99\t2\t0.0\t1\t2\n",
      "100\t15\t0.0\t1\t15\n",
      "210\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10857 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t4008.4\t0\t72\n",
      "4\t26\t1002.1\t0\t26\n",
      "5\t28\t250.5\t0\t28\n",
      "6\t44\t62.6\t0\t44\n",
      "7\t190\t15.7\t0\t190\n",
      "8\t281\t3.9\t0\t281\n",
      "9\t1190\t1.0\t0\t1184 6\n",
      "10\t3857\t0.2\t1\t3818 39\n",
      "11\t184\t0.1\t1\t178 6\n",
      "12\t111\t0.0\t1\t107 4\n",
      "13\t69\t0.0\t1\t68 1\n",
      "14\t167\t0.0\t1\t161 6\n",
      "15\t212\t0.0\t1\t206 6\n",
      "16\t593\t0.0\t1\t576 17\n",
      "17\t927\t0.0\t1\t902 25\n",
      "18\t571\t0.0\t1\t529 37 5\n",
      "19\t1986\t0.0\t1\t1234 744 8\n",
      "20\t4\t0.0\t2\t1 3\n",
      "21\t313\t0.0\t2\t0 313\n",
      "22\t11\t0.0\t2\t0 1 10\n",
      "70\t1\t0.0\t2\t1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 1\n",
      "107\t1\t0.0\t2\t1\n",
      "115\t11\t0.0\t2\t2 1 8\n",
      "186\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t3\n",
      "217\t2\t0.0\t2\t2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.25.19_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.25.19_68_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.25.19_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.25.19_68_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.25.19_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.25.19_68_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.25.19_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.25.19_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        62,281 reads  @     24.0 µs/read;   2.50 M reads/minute\n",
      "Finished in 1.52 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,281\n",
      "  Read 1 with adapter:                      90 (0.1%)\n",
      "  Read 2 with adapter:                   3,081 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        62,281 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,448,478 bp\n",
      "  Read 1:    13,430,257 bp\n",
      "  Read 2:    14,018,221 bp\n",
      "Total written (filtered):     27,396,062 bp (99.8%)\n",
      "  Read 1:    13,425,107 bp\n",
      "  Read 2:    13,970,955 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t973.1\t0\t59\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t2\t0.0\t1\t2\n",
      "204\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t8\t0.0\t1\t7 1\n",
      "217\t11\t0.0\t1\t8 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3081 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t973.1\t0\t9\n",
      "5\t6\t60.8\t0\t6\n",
      "6\t7\t15.2\t0\t7\n",
      "7\t30\t3.8\t0\t30\n",
      "8\t46\t1.0\t0\t46\n",
      "9\t148\t0.2\t0\t147 1\n",
      "10\t609\t0.1\t1\t599 10\n",
      "11\t22\t0.0\t1\t22\n",
      "12\t20\t0.0\t1\t20\n",
      "13\t44\t0.0\t1\t42 2\n",
      "14\t113\t0.0\t1\t104 9\n",
      "15\t106\t0.0\t1\t100 6\n",
      "16\t311\t0.0\t1\t305 6\n",
      "17\t462\t0.0\t1\t445 17\n",
      "18\t271\t0.0\t1\t245 17 9\n",
      "19\t768\t0.0\t1\t428 338 2\n",
      "20\t3\t0.0\t2\t2 1\n",
      "21\t97\t0.0\t2\t0 96 1\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "79\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.5.19_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.5.19_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.5.19_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.5.19_16_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.5.19_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.18.E.control.6.5.19_16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.5.19_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.18.E.control.6.5.19_16_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        92,277 reads  @     18.9 µs/read;   3.17 M reads/minute\n",
      "Finished in 1.77 s (19 us/read; 3.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,277\n",
      "  Read 1 with adapter:                      85 (0.1%)\n",
      "  Read 2 with adapter:                   1,928 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        92,277 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,905,244 bp\n",
      "  Read 1:    20,593,769 bp\n",
      "  Read 2:    21,311,475 bp\n",
      "Total written (filtered):     41,873,890 bp (99.9%)\n",
      "  Read 1:    20,593,299 bp\n",
      "  Read 2:    21,280,591 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 85 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t84\t1441.8\t0\t84\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 1928 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1441.8\t0\t6\n",
      "4\t2\t360.5\t0\t2\n",
      "5\t4\t90.1\t0\t4\n",
      "6\t2\t22.5\t0\t2\n",
      "7\t9\t5.6\t0\t9\n",
      "8\t8\t1.4\t0\t8\n",
      "9\t88\t0.4\t0\t88\n",
      "10\t271\t0.1\t1\t268 3\n",
      "11\t46\t0.0\t1\t46\n",
      "12\t49\t0.0\t1\t49\n",
      "13\t32\t0.0\t1\t32\n",
      "14\t47\t0.0\t1\t45 2\n",
      "15\t51\t0.0\t1\t47 4\n",
      "16\t150\t0.0\t1\t147 3\n",
      "17\t264\t0.0\t1\t259 5\n",
      "18\t184\t0.0\t1\t174 10\n",
      "19\t591\t0.0\t1\t399 192\n",
      "20\t1\t0.0\t2\t0 1\n",
      "21\t115\t0.0\t2\t0 114 1\n",
      "22\t6\t0.0\t2\t0 1 5\n",
      "93\t1\t0.0\t2\t1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.5.30.19_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.5.30.19_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.5.30.19_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.5.30.19_23_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.5.30.19_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.5.30.19_23_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.5.30.19_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.5.30.19_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       275,022 reads  @     15.0 µs/read;   4.00 M reads/minute\n",
      "Finished in 4.15 s (15 us/read; 3.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            275,022\n",
      "  Read 1 with adapter:                     287 (0.1%)\n",
      "  Read 2 with adapter:                   8,314 (3.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       275,022 (100.0%)\n",
      "\n",
      "Total basepairs processed:   125,275,074 bp\n",
      "  Read 1:    61,560,331 bp\n",
      "  Read 2:    63,714,743 bp\n",
      "Total written (filtered):    125,152,470 bp (99.9%)\n",
      "  Read 1:    61,559,066 bp\n",
      "  Read 2:    63,593,404 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 287 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t277\t4297.2\t0\t277\n",
      "9\t1\t1.0\t0\t1\n",
      "25\t4\t0.0\t1\t4\n",
      "27\t4\t0.0\t1\t4\n",
      "217\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 8314 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t57\t4297.2\t0\t57\n",
      "4\t18\t1074.3\t0\t18\n",
      "5\t17\t268.6\t0\t17\n",
      "6\t33\t67.1\t0\t33\n",
      "7\t80\t16.8\t0\t80\n",
      "8\t157\t4.2\t0\t157\n",
      "9\t552\t1.0\t0\t551 1\n",
      "10\t2128\t0.3\t1\t2117 11\n",
      "11\t110\t0.1\t1\t110\n",
      "12\t107\t0.0\t1\t107\n",
      "13\t88\t0.0\t1\t84 4\n",
      "14\t176\t0.0\t1\t161 15\n",
      "15\t194\t0.0\t1\t188 6\n",
      "16\t623\t0.0\t1\t607 16\n",
      "17\t925\t0.0\t1\t902 23\n",
      "18\t591\t0.0\t1\t549 38 4\n",
      "19\t2108\t0.0\t1\t1305 798 5\n",
      "20\t4\t0.0\t2\t0 3 1\n",
      "21\t338\t0.0\t2\t0 335 3\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.5.31.19_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.5.31.19_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.5.31.19_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.5.31.19_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.5.31.19_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.5.31.19_45_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.5.31.19_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.5.31.19_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       259,838 reads  @     14.9 µs/read;   4.03 M reads/minute\n",
      "Finished in 3.90 s (15 us/read; 4.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            259,838\n",
      "  Read 1 with adapter:                     170 (0.1%)\n",
      "  Read 2 with adapter:                   9,528 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       259,838 (100.0%)\n",
      "\n",
      "Total basepairs processed:   118,307,735 bp\n",
      "  Read 1:    58,115,694 bp\n",
      "  Read 2:    60,192,041 bp\n",
      "Total written (filtered):    118,170,199 bp (99.9%)\n",
      "  Read 1:    58,114,433 bp\n",
      "  Read 2:    60,055,766 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 170 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t165\t4060.0\t0\t165\n",
      "26\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9528 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t60\t4060.0\t0\t60\n",
      "4\t19\t1015.0\t0\t19\n",
      "5\t19\t253.7\t0\t19\n",
      "6\t69\t63.4\t0\t69\n",
      "7\t164\t15.9\t0\t164\n",
      "8\t234\t4.0\t0\t234\n",
      "9\t885\t1.0\t0\t880 5\n",
      "10\t2496\t0.2\t1\t2467 29\n",
      "11\t139\t0.1\t1\t138 1\n",
      "12\t82\t0.0\t1\t81 1\n",
      "13\t98\t0.0\t1\t93 5\n",
      "14\t227\t0.0\t1\t216 11\n",
      "15\t204\t0.0\t1\t201 3\n",
      "16\t647\t0.0\t1\t618 29\n",
      "17\t1010\t0.0\t1\t982 28\n",
      "18\t656\t0.0\t1\t612 35 9\n",
      "19\t2173\t0.0\t1\t1333 829 11\n",
      "20\t5\t0.0\t2\t0 5\n",
      "21\t320\t0.0\t2\t0 318 2\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t1 2\n",
      "93\t1\t0.0\t2\t1\n",
      "217\t5\t0.0\t2\t5\n",
      "222\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.6.4.19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.6.4.19_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.6.4.19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.6.4.19_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.6.4.19_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.B.control.6.4.19_1_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.6.4.19_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.B.control.6.4.19_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       294,695 reads  @     14.5 µs/read;   4.13 M reads/minute\n",
      "Finished in 4.31 s (15 us/read; 4.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            294,695\n",
      "  Read 1 with adapter:                     231 (0.1%)\n",
      "  Read 2 with adapter:                   7,908 (2.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       294,695 (100.0%)\n",
      "\n",
      "Total basepairs processed:   134,278,108 bp\n",
      "  Read 1:    65,994,849 bp\n",
      "  Read 2:    68,283,259 bp\n",
      "Total written (filtered):    134,159,809 bp (99.9%)\n",
      "  Read 1:    65,993,943 bp\n",
      "  Read 2:    68,165,866 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 231 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t228\t4604.6\t0\t228\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "171\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7908 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t4604.6\t0\t36\n",
      "4\t16\t1151.2\t0\t16\n",
      "5\t18\t287.8\t0\t18\n",
      "6\t34\t71.9\t0\t34\n",
      "7\t92\t18.0\t0\t92\n",
      "8\t144\t4.5\t0\t144\n",
      "9\t564\t1.1\t0\t562 2\n",
      "10\t1957\t0.3\t1\t1933 24\n",
      "11\t149\t0.1\t1\t147 2\n",
      "12\t80\t0.0\t1\t79 1\n",
      "13\t65\t0.0\t1\t64 1\n",
      "14\t136\t0.0\t1\t125 11\n",
      "15\t162\t0.0\t1\t160 2\n",
      "16\t530\t0.0\t1\t517 13\n",
      "17\t875\t0.0\t1\t851 24\n",
      "18\t554\t0.0\t1\t515 30 9\n",
      "19\t2098\t0.0\t1\t1346 746 6\n",
      "20\t6\t0.0\t2\t3 3\n",
      "21\t373\t0.0\t2\t0 370 3\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t0 1\n",
      "71\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t3\n",
      "89\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t0 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.D.control.5.31.19_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.D.control.5.31.19_59_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.D.control.5.31.19_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.D.control.5.31.19_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.D.control.5.31.19_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.D.control.5.31.19_59_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.D.control.5.31.19_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.D.control.5.31.19_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        72,347 reads  @     23.0 µs/read;   2.61 M reads/minute\n",
      "Finished in 1.70 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,347\n",
      "  Read 1 with adapter:                      56 (0.1%)\n",
      "  Read 2 with adapter:                   2,700 (3.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        72,347 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,619,510 bp\n",
      "  Read 1:    16,007,933 bp\n",
      "  Read 2:    16,611,577 bp\n",
      "Total written (filtered):     32,578,566 bp (99.9%)\n",
      "  Read 1:    16,006,767 bp\n",
      "  Read 2:    16,571,799 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 56 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t1130.4\t0\t50\n",
      "61\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2700 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1130.4\t0\t16\n",
      "4\t1\t282.6\t0\t1\n",
      "5\t7\t70.7\t0\t7\n",
      "6\t18\t17.7\t0\t18\n",
      "7\t52\t4.4\t0\t52\n",
      "8\t48\t1.1\t0\t48\n",
      "9\t196\t0.3\t0\t195 1\n",
      "10\t606\t0.1\t1\t599 7\n",
      "11\t39\t0.0\t1\t36 3\n",
      "12\t34\t0.0\t1\t33 1\n",
      "13\t37\t0.0\t1\t37\n",
      "14\t80\t0.0\t1\t78 2\n",
      "15\t76\t0.0\t1\t75 1\n",
      "16\t218\t0.0\t1\t210 8\n",
      "17\t324\t0.0\t1\t311 13\n",
      "18\t189\t0.0\t1\t170 17 2\n",
      "19\t653\t0.0\t1\t400 252 1\n",
      "20\t1\t0.0\t2\t1\n",
      "21\t94\t0.0\t2\t0 91 3\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t0 1\n",
      "237\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.D.control.6.6.19_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.D.control.6.6.19_65_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.D.control.6.6.19_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.D.control.6.6.19_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.D.control.6.6.19_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.D.control.6.6.19_65_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.D.control.6.6.19_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.D.control.6.6.19_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       150,876 reads  @     18.4 µs/read;   3.26 M reads/minute\n",
      "Finished in 2.80 s (19 us/read; 3.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            150,876\n",
      "  Read 1 with adapter:                     207 (0.1%)\n",
      "  Read 2 with adapter:                   4,628 (3.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       150,876 (100.0%)\n",
      "\n",
      "Total basepairs processed:    68,729,507 bp\n",
      "  Read 1:    33,769,521 bp\n",
      "  Read 2:    34,959,986 bp\n",
      "Total written (filtered):     68,659,976 bp (99.9%)\n",
      "  Read 1:    33,767,282 bp\n",
      "  Read 2:    34,892,694 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 207 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t163\t2357.4\t0\t163\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t9\t0.0\t1\t9\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t26\t0.0\t1\t26\n",
      "88\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4628 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t2357.4\t0\t29\n",
      "4\t10\t589.4\t0\t10\n",
      "5\t16\t147.3\t0\t16\n",
      "6\t25\t36.8\t0\t25\n",
      "7\t73\t9.2\t0\t73\n",
      "8\t116\t2.3\t0\t116\n",
      "9\t412\t0.6\t0\t411 1\n",
      "10\t1143\t0.1\t1\t1135 8\n",
      "11\t82\t0.0\t1\t81 1\n",
      "12\t62\t0.0\t1\t62\n",
      "13\t48\t0.0\t1\t47 1\n",
      "14\t108\t0.0\t1\t98 10\n",
      "15\t112\t0.0\t1\t108 4\n",
      "16\t298\t0.0\t1\t290 8\n",
      "17\t469\t0.0\t1\t457 12\n",
      "18\t323\t0.0\t1\t302 15 6\n",
      "19\t1106\t0.0\t1\t690 411 5\n",
      "20\t5\t0.0\t2\t2 2 1\n",
      "21\t176\t0.0\t2\t0 174 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 1\n",
      "56\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t2\n",
      "79\t2\t0.0\t2\t1 1\n",
      "217\t2\t0.0\t2\t2\n",
      "237\t2\t0.0\t2\t1 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.4.28.19_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.4.28.19_62_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.4.28.19_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.4.28.19_62_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.4.28.19_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.4.28.19_62_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.4.28.19_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.4.28.19_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       152,706 reads  @     16.7 µs/read;   3.59 M reads/minute\n",
      "Finished in 2.58 s (17 us/read; 3.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            152,706\n",
      "  Read 1 with adapter:                     148 (0.1%)\n",
      "  Read 2 with adapter:                   7,081 (4.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       152,706 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,289,300 bp\n",
      "  Read 1:    34,010,803 bp\n",
      "  Read 2:    35,278,497 bp\n",
      "Total written (filtered):     69,188,406 bp (99.9%)\n",
      "  Read 1:    34,007,302 bp\n",
      "  Read 2:    35,181,104 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 148 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t128\t2386.0\t0\t128\n",
      "19\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t0 1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "216\t4\t0.0\t1\t2 2\n",
      "217\t8\t0.0\t1\t7 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7081 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t76\t2386.0\t0\t76\n",
      "4\t12\t596.5\t0\t12\n",
      "5\t19\t149.1\t0\t19\n",
      "6\t35\t37.3\t0\t35\n",
      "7\t107\t9.3\t0\t107\n",
      "8\t161\t2.3\t0\t161\n",
      "9\t721\t0.6\t0\t719 2\n",
      "10\t2176\t0.1\t1\t2161 15\n",
      "11\t93\t0.0\t1\t90 3\n",
      "12\t52\t0.0\t1\t51 1\n",
      "13\t60\t0.0\t1\t58 2\n",
      "14\t166\t0.0\t1\t153 13\n",
      "15\t176\t0.0\t1\t175 1\n",
      "16\t483\t0.0\t1\t466 17\n",
      "17\t689\t0.0\t1\t670 19\n",
      "18\t425\t0.0\t1\t377 36 12\n",
      "19\t1421\t0.0\t1\t882 536 3\n",
      "20\t2\t0.0\t2\t1 1\n",
      "21\t194\t0.0\t2\t0 192 2\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "75\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t2\n",
      "109\t1\t0.0\t2\t1\n",
      "142\t1\t0.0\t2\t0 1\n",
      "239\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.5.28.19_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.5.28.19_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.5.28.19_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.5.28.19_25_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.5.28.19_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.5.28.19_25_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.5.28.19_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.5.28.19_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:07       486,725 reads  @     15.5 µs/read;   3.88 M reads/minute\n",
      "Finished in 7.55 s (16 us/read; 3.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            486,725\n",
      "  Read 1 with adapter:                     688 (0.1%)\n",
      "  Read 2 with adapter:                  41,992 (8.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       486,725 (100.0%)\n",
      "\n",
      "Total basepairs processed:   222,463,006 bp\n",
      "  Read 1:   108,973,460 bp\n",
      "  Read 2:   113,489,546 bp\n",
      "Total written (filtered):    221,930,441 bp (99.8%)\n",
      "  Read 1:   108,970,012 bp\n",
      "  Read 2:   112,960,429 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 688 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t674\t7605.1\t0\t674\n",
      "14\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t1\n",
      "106\t1\t0.0\t1\t1\n",
      "121\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "151\t3\t0.0\t1\t3\n",
      "156\t1\t0.0\t1\t1\n",
      "161\t1\t0.0\t1\t1\n",
      "166\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 41992 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t340\t7605.1\t0\t340\n",
      "4\t181\t1901.3\t0\t181\n",
      "5\t228\t475.3\t0\t228\n",
      "6\t604\t118.8\t0\t604\n",
      "7\t1769\t29.7\t0\t1769\n",
      "8\t2899\t7.4\t0\t2899\n",
      "9\t8134\t1.9\t0\t8115 19\n",
      "10\t9807\t0.5\t1\t9677 130\n",
      "11\t579\t0.1\t1\t564 15\n",
      "12\t690\t0.0\t1\t667 23\n",
      "13\t1070\t0.0\t1\t1026 44\n",
      "14\t1526\t0.0\t1\t1472 54\n",
      "15\t1050\t0.0\t1\t971 79\n",
      "16\t2481\t0.0\t1\t2361 120\n",
      "17\t2718\t0.0\t1\t2605 113\n",
      "18\t1717\t0.0\t1\t1434 162 121\n",
      "19\t5275\t0.0\t1\t2844 2402 29\n",
      "20\t12\t0.0\t2\t2 8 2\n",
      "21\t661\t0.0\t2\t0 653 8\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t1\n",
      "39\t1\t0.0\t2\t1\n",
      "41\t1\t0.0\t2\t0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "46\t1\t0.0\t2\t1\n",
      "47\t2\t0.0\t2\t1 1\n",
      "50\t2\t0.0\t2\t0 2\n",
      "53\t1\t0.0\t2\t1\n",
      "54\t2\t0.0\t2\t1 1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t5\t0.0\t2\t5\n",
      "57\t2\t0.0\t2\t2\n",
      "58\t5\t0.0\t2\t4 0 1\n",
      "59\t3\t0.0\t2\t1 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t3\t0.0\t2\t2 0 1\n",
      "62\t4\t0.0\t2\t4\n",
      "63\t10\t0.0\t2\t10\n",
      "64\t7\t0.0\t2\t6 1\n",
      "65\t10\t0.0\t2\t10\n",
      "66\t4\t0.0\t2\t3 0 1\n",
      "67\t8\t0.0\t2\t7 0 1\n",
      "68\t5\t0.0\t2\t3 2\n",
      "69\t6\t0.0\t2\t2 1 3\n",
      "70\t15\t0.0\t2\t9 1 5\n",
      "71\t14\t0.0\t2\t3 1 10\n",
      "72\t1\t0.0\t2\t1\n",
      "73\t2\t0.0\t2\t2\n",
      "74\t3\t0.0\t2\t2 0 1\n",
      "75\t1\t0.0\t2\t1\n",
      "76\t2\t0.0\t2\t2\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t33\t0.0\t2\t25 5 3\n",
      "80\t1\t0.0\t2\t1\n",
      "84\t1\t0.0\t2\t1\n",
      "85\t1\t0.0\t2\t0 1\n",
      "86\t1\t0.0\t2\t1\n",
      "87\t1\t0.0\t2\t1\n",
      "88\t3\t0.0\t2\t2 0 1\n",
      "89\t2\t0.0\t2\t2\n",
      "90\t1\t0.0\t2\t1\n",
      "92\t4\t0.0\t2\t3 1\n",
      "97\t1\t0.0\t2\t1\n",
      "99\t1\t0.0\t2\t0 1\n",
      "100\t1\t0.0\t2\t1\n",
      "101\t2\t0.0\t2\t1 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t1\n",
      "112\t1\t0.0\t2\t1\n",
      "115\t2\t0.0\t2\t1 0 1\n",
      "119\t1\t0.0\t2\t1\n",
      "124\t2\t0.0\t2\t1 1\n",
      "127\t1\t0.0\t2\t1\n",
      "143\t1\t0.0\t2\t1\n",
      "146\t2\t0.0\t2\t1 0 1\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "156\t1\t0.0\t2\t1\n",
      "157\t1\t0.0\t2\t1\n",
      "158\t1\t0.0\t2\t0 1\n",
      "161\t1\t0.0\t2\t0 1\n",
      "162\t1\t0.0\t2\t0 1\n",
      "169\t1\t0.0\t2\t1\n",
      "172\t2\t0.0\t2\t2\n",
      "175\t1\t0.0\t2\t0 1\n",
      "184\t2\t0.0\t2\t2\n",
      "186\t2\t0.0\t2\t1 0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t3\t0.0\t2\t2 1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "234\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.5.6.19_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.5.6.19_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.5.6.19_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.5.6.19_32_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.5.6.19_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.5.6.19_32_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.5.6.19_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.5.6.19_32_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       170,153 reads  @     18.1 µs/read;   3.32 M reads/minute\n",
      "Finished in 3.11 s (18 us/read; 3.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            170,153\n",
      "  Read 1 with adapter:                     211 (0.1%)\n",
      "  Read 2 with adapter:                   5,625 (3.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       170,153 (100.0%)\n",
      "\n",
      "Total basepairs processed:    77,319,002 bp\n",
      "  Read 1:    37,981,451 bp\n",
      "  Read 2:    39,337,551 bp\n",
      "Total written (filtered):     77,235,207 bp (99.9%)\n",
      "  Read 1:    37,979,309 bp\n",
      "  Read 2:    39,255,898 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 211 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t201\t2658.6\t0\t201\n",
      "20\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t1 1\n",
      "218\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5625 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t2658.6\t0\t28\n",
      "4\t14\t664.7\t0\t14\n",
      "5\t16\t166.2\t0\t16\n",
      "6\t23\t41.5\t0\t23\n",
      "7\t83\t10.4\t0\t83\n",
      "8\t89\t2.6\t0\t89\n",
      "9\t414\t0.6\t0\t413 1\n",
      "10\t1412\t0.2\t1\t1391 21\n",
      "11\t87\t0.0\t1\t85 2\n",
      "12\t65\t0.0\t1\t62 3\n",
      "13\t58\t0.0\t1\t56 2\n",
      "14\t127\t0.0\t1\t122 5\n",
      "15\t152\t0.0\t1\t148 4\n",
      "16\t405\t0.0\t1\t395 10\n",
      "17\t652\t0.0\t1\t627 25\n",
      "18\t403\t0.0\t1\t374 24 5\n",
      "19\t1377\t0.0\t1\t894 480 3\n",
      "20\t1\t0.0\t2\t1\n",
      "21\t212\t0.0\t2\t0 210 2\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.6.4.19_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.6.4.19_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.6.4.19_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.6.4.19_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.6.4.19_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.2.E.control.6.4.19_15_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.6.4.19_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.2.E.control.6.4.19_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       198,177 reads  @     16.0 µs/read;   3.74 M reads/minute\n",
      "Finished in 3.20 s (16 us/read; 3.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            198,177\n",
      "  Read 1 with adapter:                     222 (0.1%)\n",
      "  Read 2 with adapter:                   5,340 (2.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       198,177 (100.0%)\n",
      "\n",
      "Total basepairs processed:    90,260,869 bp\n",
      "  Read 1:    44,354,118 bp\n",
      "  Read 2:    45,906,751 bp\n",
      "Total written (filtered):     90,179,922 bp (99.9%)\n",
      "  Read 1:    44,352,934 bp\n",
      "  Read 2:    45,826,988 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 222 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t216\t3096.5\t0\t216\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5340 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t3096.5\t0\t18\n",
      "4\t10\t774.1\t0\t10\n",
      "5\t13\t193.5\t0\t13\n",
      "6\t13\t48.4\t0\t13\n",
      "7\t65\t12.1\t0\t65\n",
      "8\t79\t3.0\t0\t79\n",
      "9\t340\t0.8\t0\t336 4\n",
      "10\t1299\t0.2\t1\t1287 12\n",
      "11\t108\t0.0\t1\t105 3\n",
      "12\t70\t0.0\t1\t69 1\n",
      "13\t57\t0.0\t1\t56 1\n",
      "14\t102\t0.0\t1\t96 6\n",
      "15\t132\t0.0\t1\t129 3\n",
      "16\t350\t0.0\t1\t341 9\n",
      "17\t571\t0.0\t1\t559 12\n",
      "18\t414\t0.0\t1\t382 29 3\n",
      "19\t1439\t0.0\t1\t909 517 13\n",
      "21\t249\t0.0\t2\t0 243 6\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "55\t1\t0.0\t2\t1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t2 0 1\n",
      "216\t2\t0.0\t2\t2\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.D.25percent.6.5.19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.D.25percent.6.5.19_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.D.25percent.6.5.19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.D.25percent.6.5.19_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.D.25percent.6.5.19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.D.25percent.6.5.19_13_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.D.25percent.6.5.19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.D.25percent.6.5.19_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       253,952 reads  @     14.1 µs/read;   4.25 M reads/minute\n",
      "Finished in 3.61 s (14 us/read; 4.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            253,952\n",
      "  Read 1 with adapter:                     247 (0.1%)\n",
      "  Read 2 with adapter:                   7,120 (2.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       253,952 (100.0%)\n",
      "\n",
      "Total basepairs processed:   115,687,231 bp\n",
      "  Read 1:    56,849,233 bp\n",
      "  Read 2:    58,837,998 bp\n",
      "Total written (filtered):    115,577,122 bp (99.9%)\n",
      "  Read 1:    56,847,471 bp\n",
      "  Read 2:    58,729,651 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 247 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t234\t3968.0\t0\t234\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t2\t0.0\t1\t1 1\n",
      "176\t1\t0.0\t1\t1\n",
      "217\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 7120 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t3968.0\t0\t30\n",
      "4\t13\t992.0\t0\t13\n",
      "5\t28\t248.0\t0\t28\n",
      "6\t47\t62.0\t0\t47\n",
      "7\t154\t15.5\t0\t154\n",
      "8\t179\t3.9\t0\t179\n",
      "9\t584\t1.0\t0\t583 1\n",
      "10\t1045\t0.2\t1\t1033 12\n",
      "11\t61\t0.1\t1\t60 1\n",
      "12\t122\t0.0\t1\t118 4\n",
      "13\t102\t0.0\t1\t93 9\n",
      "14\t218\t0.0\t1\t203 15\n",
      "15\t157\t0.0\t1\t146 11\n",
      "16\t531\t0.0\t1\t520 11\n",
      "17\t881\t0.0\t1\t850 31\n",
      "18\t541\t0.0\t1\t501 29 11\n",
      "19\t2075\t0.0\t1\t1265 800 10\n",
      "20\t4\t0.0\t2\t1 1 2\n",
      "21\t329\t0.0\t2\t0 326 3\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t2 1\n",
      "217\t2\t0.0\t2\t2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.D.5percent.6.3.19_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.D.5percent.6.3.19_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.D.5percent.6.3.19_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.D.5percent.6.3.19_6_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.D.5percent.6.3.19_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.D.5percent.6.3.19_6_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.D.5percent.6.3.19_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.D.5percent.6.3.19_6_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       538,323 reads  @     12.3 µs/read;   4.86 M reads/minute\n",
      "Finished in 6.67 s (12 us/read; 4.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            538,323\n",
      "  Read 1 with adapter:                     677 (0.1%)\n",
      "  Read 2 with adapter:                  22,532 (4.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       538,323 (100.0%)\n",
      "\n",
      "Total basepairs processed:   245,435,666 bp\n",
      "  Read 1:   120,529,838 bp\n",
      "  Read 2:   124,905,828 bp\n",
      "Total written (filtered):    245,110,377 bp (99.9%)\n",
      "  Read 1:   120,525,879 bp\n",
      "  Read 2:   124,584,498 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 677 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t654\t8411.3\t0\t654\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t5\t0.0\t1\t5\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "49\t1\t0.0\t1\t1\n",
      "51\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t1\n",
      "132\t1\t0.0\t1\t1\n",
      "144\t1\t0.0\t1\t1\n",
      "151\t1\t0.0\t1\t0 1\n",
      "166\t3\t0.0\t1\t3\n",
      "186\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "210\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 22532 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t161\t8411.3\t0\t161\n",
      "4\t78\t2102.8\t0\t78\n",
      "5\t66\t525.7\t0\t66\n",
      "6\t213\t131.4\t0\t213\n",
      "7\t518\t32.9\t0\t518\n",
      "8\t753\t8.2\t0\t753\n",
      "9\t2750\t2.1\t0\t2743 7\n",
      "10\t3809\t0.5\t1\t3758 51\n",
      "11\t287\t0.1\t1\t282 5\n",
      "12\t468\t0.0\t1\t456 12\n",
      "13\t596\t0.0\t1\t578 18\n",
      "14\t851\t0.0\t1\t815 36\n",
      "15\t595\t0.0\t1\t559 36\n",
      "16\t1734\t0.0\t1\t1672 62\n",
      "17\t2451\t0.0\t1\t2364 87\n",
      "18\t1427\t0.0\t1\t1276 94 57\n",
      "19\t5027\t0.0\t1\t2999 2007 21\n",
      "20\t17\t0.0\t2\t7 9 1\n",
      "21\t684\t0.0\t2\t0 671 13\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "39\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "56\t2\t0.0\t2\t2\n",
      "63\t2\t0.0\t2\t1 1\n",
      "70\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "79\t4\t0.0\t2\t4\n",
      "87\t1\t0.0\t2\t1\n",
      "95\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t1\n",
      "132\t1\t0.0\t2\t1\n",
      "153\t1\t0.0\t2\t1\n",
      "160\t1\t0.0\t2\t1\n",
      "201\t1\t0.0\t2\t1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "217\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "237\t1\t0.0\t2\t1\n",
      "248\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.25percent.6.5.19_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.25percent.6.5.19_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.25percent.6.5.19_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.25percent.6.5.19_36_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.25percent.6.5.19_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.25percent.6.5.19_36_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.25percent.6.5.19_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.25percent.6.5.19_36_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       331,604 reads  @     13.1 µs/read;   4.58 M reads/minute\n",
      "Finished in 4.37 s (13 us/read; 4.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            331,604\n",
      "  Read 1 with adapter:                     363 (0.1%)\n",
      "  Read 2 with adapter:                  10,977 (3.3%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       331,603 (100.0%)\n",
      "\n",
      "Total basepairs processed:   151,141,089 bp\n",
      "  Read 1:    74,258,191 bp\n",
      "  Read 2:    76,882,898 bp\n",
      "Total written (filtered):    150,974,295 bp (99.9%)\n",
      "  Read 1:    74,256,696 bp\n",
      "  Read 2:    76,717,599 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 363 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t360\t5181.3\t0\t360\n",
      "24\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10977 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t5181.3\t0\t56\n",
      "4\t37\t1295.3\t0\t37\n",
      "5\t32\t323.8\t0\t32\n",
      "6\t78\t81.0\t0\t78\n",
      "7\t260\t20.2\t0\t260\n",
      "8\t306\t5.1\t0\t306\n",
      "9\t831\t1.3\t0\t831\n",
      "10\t1520\t0.3\t1\t1500 20\n",
      "11\t127\t0.1\t1\t123 4\n",
      "12\t243\t0.0\t1\t238 5\n",
      "13\t353\t0.0\t1\t342 11\n",
      "14\t524\t0.0\t1\t501 23\n",
      "15\t310\t0.0\t1\t293 17\n",
      "16\t869\t0.0\t1\t838 31\n",
      "17\t1296\t0.0\t1\t1254 42\n",
      "18\t784\t0.0\t1\t699 59 26\n",
      "19\t2892\t0.0\t1\t1768 1116 8\n",
      "20\t6\t0.0\t2\t1 5\n",
      "21\t411\t0.0\t2\t0 406 5\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "55\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t0 1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "79\t3\t0.0\t2\t3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "186\t2\t0.0\t2\t2\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.5percent.5.30.19_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.5percent.5.30.19_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.5percent.5.30.19_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.5percent.5.30.19_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.5percent.5.30.19_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.5percent.5.30.19_7_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.5percent.5.30.19_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.5percent.5.30.19_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       488,981 reads  @     13.4 µs/read;   4.48 M reads/minute\n",
      "Finished in 6.59 s (13 us/read; 4.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            488,981\n",
      "  Read 1 with adapter:                     583 (0.1%)\n",
      "  Read 2 with adapter:                  42,331 (8.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       488,981 (100.0%)\n",
      "\n",
      "Total basepairs processed:   223,488,306 bp\n",
      "  Read 1:   109,487,518 bp\n",
      "  Read 2:   114,000,788 bp\n",
      "Total written (filtered):    222,983,447 bp (99.8%)\n",
      "  Read 1:   109,484,979 bp\n",
      "  Read 2:   113,498,468 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 583 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t565\t7640.3\t0\t565\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "18\t1\t0.0\t1\t1\n",
      "25\t3\t0.0\t1\t3\n",
      "26\t7\t0.0\t1\t7\n",
      "27\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "109\t1\t0.0\t1\t0 1\n",
      "125\t1\t0.0\t1\t1\n",
      "191\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 42331 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t491\t7640.3\t0\t491\n",
      "4\t296\t1910.1\t0\t296\n",
      "5\t553\t477.5\t0\t553\n",
      "6\t1322\t119.4\t0\t1322\n",
      "7\t3380\t29.8\t0\t3380\n",
      "8\t4351\t7.5\t0\t4351\n",
      "9\t9599\t1.9\t0\t9581 18\n",
      "10\t7184\t0.5\t1\t7114 70\n",
      "11\t295\t0.1\t1\t284 11\n",
      "12\t428\t0.0\t1\t420 8\n",
      "13\t645\t0.0\t1\t630 15\n",
      "14\t1045\t0.0\t1\t998 47\n",
      "15\t798\t0.0\t1\t751 47\n",
      "16\t1908\t0.0\t1\t1843 65\n",
      "17\t2582\t0.0\t1\t2497 85\n",
      "18\t1529\t0.0\t1\t1311 124 94\n",
      "19\t5052\t0.0\t1\t2814 2219 19\n",
      "20\t17\t0.0\t2\t5 8 4\n",
      "21\t684\t0.0\t2\t1 676 7\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "30\t1\t0.0\t2\t1\n",
      "43\t4\t0.0\t2\t0 0 4\n",
      "46\t1\t0.0\t2\t1\n",
      "47\t2\t0.0\t2\t2\n",
      "48\t1\t0.0\t2\t1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t2\n",
      "53\t2\t0.0\t2\t1 1\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t2\t0.0\t2\t2\n",
      "57\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "60\t2\t0.0\t2\t1 1\n",
      "62\t3\t0.0\t2\t3\n",
      "63\t3\t0.0\t2\t3\n",
      "64\t2\t0.0\t2\t1 1\n",
      "65\t2\t0.0\t2\t2\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t4\t0.0\t2\t2 1 1\n",
      "69\t3\t0.0\t2\t3\n",
      "70\t3\t0.0\t2\t1 0 2\n",
      "71\t6\t0.0\t2\t1 0 5\n",
      "73\t4\t0.0\t2\t4\n",
      "74\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t3\t0.0\t2\t2 0 1\n",
      "79\t18\t0.0\t2\t12 4 2\n",
      "80\t1\t0.0\t2\t1\n",
      "81\t1\t0.0\t2\t1\n",
      "83\t1\t0.0\t2\t0 1\n",
      "84\t3\t0.0\t2\t3\n",
      "85\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t0 1\n",
      "108\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t1\n",
      "112\t2\t0.0\t2\t2\n",
      "113\t1\t0.0\t2\t1\n",
      "114\t1\t0.0\t2\t0 1\n",
      "118\t1\t0.0\t2\t0 1\n",
      "128\t1\t0.0\t2\t1\n",
      "131\t1\t0.0\t2\t1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "204\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "228\t1\t0.0\t2\t0 1\n",
      "233\t1\t0.0\t2\t1\n",
      "235\t1\t0.0\t2\t0 1\n",
      "237\t4\t0.0\t2\t3 0 1\n",
      "238\t3\t0.0\t2\t3\n",
      "239\t4\t0.0\t2\t1 3\n",
      "241\t5\t0.0\t2\t5\n",
      "242\t2\t0.0\t2\t1 0 1\n",
      "243\t3\t0.0\t2\t3\n",
      "244\t1\t0.0\t2\t0 1\n",
      "247\t1\t0.0\t2\t0 0 1\n",
      "248\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.25percent.6.4.19_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.25percent.6.4.19_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.25percent.6.4.19_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.25percent.6.4.19_34_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.25percent.6.4.19_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.25percent.6.4.19_34_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.25percent.6.4.19_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.25percent.6.4.19_34_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       521,738 reads  @     12.4 µs/read;   4.85 M reads/minute\n",
      "Finished in 6.47 s (12 us/read; 4.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            521,738\n",
      "  Read 1 with adapter:                     451 (0.1%)\n",
      "  Read 2 with adapter:                  23,108 (4.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       521,738 (100.0%)\n",
      "\n",
      "Total basepairs processed:   237,932,009 bp\n",
      "  Read 1:   116,842,861 bp\n",
      "  Read 2:   121,089,148 bp\n",
      "Total written (filtered):    237,588,980 bp (99.9%)\n",
      "  Read 1:   116,841,158 bp\n",
      "  Read 2:   120,747,822 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 451 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t440\t8152.2\t0\t440\n",
      "18\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "122\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 23108 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t143\t8152.2\t0\t143\n",
      "4\t81\t2038.0\t0\t81\n",
      "5\t72\t509.5\t0\t72\n",
      "6\t211\t127.4\t0\t211\n",
      "7\t523\t31.8\t0\t523\n",
      "8\t727\t8.0\t0\t727\n",
      "9\t2389\t2.0\t0\t2383 6\n",
      "10\t3607\t0.5\t1\t3572 35\n",
      "11\t292\t0.1\t1\t288 4\n",
      "12\t494\t0.0\t1\t487 7\n",
      "13\t641\t0.0\t1\t614 27\n",
      "14\t1013\t0.0\t1\t954 59\n",
      "15\t731\t0.0\t1\t685 46\n",
      "16\t1857\t0.0\t1\t1789 68\n",
      "17\t2622\t0.0\t1\t2537 85\n",
      "18\t1531\t0.0\t1\t1344 120 67\n",
      "19\t5371\t0.0\t1\t3123 2222 26\n",
      "20\t14\t0.0\t2\t4 9 1\n",
      "21\t684\t0.0\t2\t0 673 11\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "30\t5\t0.0\t2\t5\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "50\t1\t0.0\t2\t0 1\n",
      "51\t1\t0.0\t2\t1\n",
      "54\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "63\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "69\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "74\t3\t0.0\t2\t3\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t12\t0.0\t2\t9 1 2\n",
      "132\t1\t0.0\t2\t1\n",
      "142\t1\t0.0\t2\t0 1\n",
      "144\t1\t0.0\t2\t1\n",
      "159\t1\t0.0\t2\t1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t1\n",
      "210\t1\t0.0\t2\t1\n",
      "213\t1\t0.0\t2\t0 0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "239\t2\t0.0\t2\t1 0 1\n",
      "240\t2\t0.0\t2\t0 1 1\n",
      "241\t4\t0.0\t2\t4\n",
      "242\t3\t0.0\t2\t3\n",
      "243\t1\t0.0\t2\t1\n",
      "244\t3\t0.0\t2\t3\n",
      "245\t1\t0.0\t2\t1\n",
      "246\t1\t0.0\t2\t1\n",
      "248\t1\t0.0\t2\t0 1\n",
      "249\t1\t0.0\t2\t1\n",
      "250\t2\t0.0\t2\t1 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.50percent.6.6.19_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.50percent.6.6.19_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.50percent.6.6.19_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.50percent.6.6.19_4_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.50percent.6.6.19_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.50percent.6.6.19_4_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.50percent.6.6.19_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.50percent.6.6.19_4_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       372,617 reads  @     12.0 µs/read;   5.01 M reads/minute\n",
      "Finished in 4.48 s (12 us/read; 4.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            372,617\n",
      "  Read 1 with adapter:                     324 (0.1%)\n",
      "  Read 2 with adapter:                  24,293 (6.5%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       372,616 (100.0%)\n",
      "\n",
      "Total basepairs processed:   170,080,752 bp\n",
      "  Read 1:    83,444,194 bp\n",
      "  Read 2:    86,636,558 bp\n",
      "Total written (filtered):    169,769,625 bp (99.8%)\n",
      "  Read 1:    83,442,418 bp\n",
      "  Read 2:    86,327,207 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 324 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t313\t5822.1\t0\t313\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t2\t0.0\t1\t2\n",
      "87\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 24293 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t195\t5822.1\t0\t195\n",
      "4\t63\t1455.5\t0\t63\n",
      "5\t93\t363.9\t0\t93\n",
      "6\t159\t91.0\t0\t159\n",
      "7\t759\t22.7\t0\t759\n",
      "8\t865\t5.7\t0\t865\n",
      "9\t3317\t1.4\t0\t3303 14\n",
      "10\t8820\t0.4\t1\t8739 81\n",
      "11\t246\t0.1\t1\t243 3\n",
      "12\t203\t0.0\t1\t199 4\n",
      "13\t215\t0.0\t1\t206 9\n",
      "14\t498\t0.0\t1\t479 19\n",
      "15\t478\t0.0\t1\t453 25\n",
      "16\t1287\t0.0\t1\t1249 38\n",
      "17\t1814\t0.0\t1\t1761 53\n",
      "18\t1156\t0.0\t1\t1043 81 32\n",
      "19\t3600\t0.0\t1\t2097 1486 17\n",
      "20\t8\t0.0\t2\t1 6 1\n",
      "21\t481\t0.0\t2\t0 479 2\n",
      "22\t11\t0.0\t2\t0 1 10\n",
      "40\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t2\n",
      "84\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "104\t1\t0.0\t2\t0 1\n",
      "114\t1\t0.0\t2\t1\n",
      "150\t2\t0.0\t2\t1 0 1\n",
      "212\t2\t0.0\t2\t1 1\n",
      "215\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "233\t1\t0.0\t2\t1\n",
      "238\t1\t0.0\t2\t1\n",
      "241\t2\t0.0\t2\t1 1\n",
      "246\t1\t0.0\t2\t0 0 1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.75percent.6.10.19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.75percent.6.10.19_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.75percent.6.10.19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.75percent.6.10.19_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.75percent.6.10.19_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.E.NT.75percent.6.10.19_37_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.75percent.6.10.19_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.E.NT.75percent.6.10.19_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:05       338,009 reads  @     15.5 µs/read;   3.86 M reads/minute\n",
      "Finished in 5.28 s (16 us/read; 3.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            338,009\n",
      "  Read 1 with adapter:                     408 (0.1%)\n",
      "  Read 2 with adapter:                  20,865 (6.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       338,009 (100.0%)\n",
      "\n",
      "Total basepairs processed:   154,252,646 bp\n",
      "  Read 1:    75,684,722 bp\n",
      "  Read 2:    78,567,924 bp\n",
      "Total written (filtered):    153,976,950 bp (99.8%)\n",
      "  Read 1:    75,682,789 bp\n",
      "  Read 2:    78,294,161 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 408 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t398\t5281.4\t0\t398\n",
      "9\t1\t1.3\t0\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t1\t0.0\t1\t0 1\n",
      "166\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 20865 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t185\t5281.4\t0\t185\n",
      "4\t37\t1320.3\t0\t37\n",
      "5\t67\t330.1\t0\t67\n",
      "6\t107\t82.5\t0\t107\n",
      "7\t462\t20.6\t0\t462\n",
      "8\t640\t5.2\t0\t640\n",
      "9\t2731\t1.3\t0\t2714 17\n",
      "10\t6984\t0.3\t1\t6898 86\n",
      "11\t252\t0.1\t1\t247 5\n",
      "12\t190\t0.0\t1\t188 2\n",
      "13\t254\t0.0\t1\t244 10\n",
      "14\t494\t0.0\t1\t470 24\n",
      "15\t498\t0.0\t1\t477 21\n",
      "16\t1315\t0.0\t1\t1263 52\n",
      "17\t1766\t0.0\t1\t1708 58\n",
      "18\t1062\t0.0\t1\t970 66 26\n",
      "19\t3312\t0.0\t1\t1978 1318 16\n",
      "20\t6\t0.0\t2\t2 2 2\n",
      "21\t432\t0.0\t2\t0 423 9\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t1\n",
      "57\t2\t0.0\t2\t1 0 1\n",
      "58\t2\t0.0\t2\t2\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "67\t2\t0.0\t2\t1 0 1\n",
      "70\t3\t0.0\t2\t2 1\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "72\t1\t0.0\t2\t1\n",
      "73\t1\t0.0\t2\t1\n",
      "79\t9\t0.0\t2\t8 1\n",
      "82\t1\t0.0\t2\t0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 1\n",
      "216\t2\t0.0\t2\t2\n",
      "230\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.25percent.5.29.19_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.25percent.5.29.19_66_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.25percent.5.29.19_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.25percent.5.29.19_66_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.25percent.5.29.19_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.25percent.5.29.19_66_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.25percent.5.29.19_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.25percent.5.29.19_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       374,019 reads  @     11.7 µs/read;   5.14 M reads/minute\n",
      "Finished in 4.38 s (12 us/read; 5.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            374,019\n",
      "  Read 1 with adapter:                     398 (0.1%)\n",
      "  Read 2 with adapter:                  19,015 (5.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       374,019 (100.0%)\n",
      "\n",
      "Total basepairs processed:   170,602,732 bp\n",
      "  Read 1:    83,755,374 bp\n",
      "  Read 2:    86,847,358 bp\n",
      "Total written (filtered):    170,350,182 bp (99.9%)\n",
      "  Read 1:    83,753,681 bp\n",
      "  Read 2:    86,596,501 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 398 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t389\t5844.0\t0\t389\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t2\t0.0\t1\t1 1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t1\t0.0\t1\t1\n",
      "169\t1\t0.0\t1\t1\n",
      "180\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 19015 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t179\t5844.0\t0\t179\n",
      "4\t37\t1461.0\t0\t37\n",
      "5\t58\t365.3\t0\t58\n",
      "6\t135\t91.3\t0\t135\n",
      "7\t477\t22.8\t0\t477\n",
      "8\t633\t5.7\t0\t633\n",
      "9\t2380\t1.4\t0\t2369 11\n",
      "10\t6174\t0.4\t1\t6126 48\n",
      "11\t223\t0.1\t1\t221 2\n",
      "12\t163\t0.0\t1\t160 3\n",
      "13\t165\t0.0\t1\t157 8\n",
      "14\t375\t0.0\t1\t360 15\n",
      "15\t385\t0.0\t1\t374 11\n",
      "16\t1122\t0.0\t1\t1091 31\n",
      "17\t1620\t0.0\t1\t1571 49\n",
      "18\t979\t0.0\t1\t905 62 12\n",
      "19\t3384\t0.0\t1\t2056 1316 12\n",
      "20\t7\t0.0\t2\t0 4 3\n",
      "21\t485\t0.0\t2\t0 476 9\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "30\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "84\t1\t0.0\t2\t0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "215\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t2\t0.0\t2\t2\n",
      "241\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.5percent.5.27.19_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.5percent.5.27.19_61_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.5percent.5.27.19_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.5percent.5.27.19_61_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.5percent.5.27.19_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.5percent.5.27.19_61_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.5percent.5.27.19_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.5percent.5.27.19_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       232,369 reads  @     12.5 µs/read;   4.79 M reads/minute\n",
      "Finished in 2.93 s (13 us/read; 4.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            232,369\n",
      "  Read 1 with adapter:                     156 (0.1%)\n",
      "  Read 2 with adapter:                   4,355 (1.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       232,369 (100.0%)\n",
      "\n",
      "Total basepairs processed:   105,835,361 bp\n",
      "  Read 1:    52,035,277 bp\n",
      "  Read 2:    53,800,084 bp\n",
      "Total written (filtered):    105,762,296 bp (99.9%)\n",
      "  Read 1:    52,034,444 bp\n",
      "  Read 2:    53,727,852 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 156 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t150\t3630.8\t0\t150\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t3\t0.0\t1\t2 1\n",
      "132\t1\t0.0\t1\t1\n",
      "149\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4355 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t3630.8\t0\t27\n",
      "4\t9\t907.7\t0\t9\n",
      "5\t10\t226.9\t0\t10\n",
      "6\t23\t56.7\t0\t23\n",
      "7\t53\t14.2\t0\t53\n",
      "8\t84\t3.5\t0\t84\n",
      "9\t181\t0.9\t0\t181\n",
      "10\t392\t0.2\t1\t389 3\n",
      "11\t34\t0.1\t1\t34\n",
      "12\t40\t0.0\t1\t40\n",
      "13\t35\t0.0\t1\t35\n",
      "14\t96\t0.0\t1\t92 4\n",
      "15\t96\t0.0\t1\t94 2\n",
      "16\t307\t0.0\t1\t298 9\n",
      "17\t582\t0.0\t1\t566 16\n",
      "18\t392\t0.0\t1\t360 25 7\n",
      "19\t1667\t0.0\t1\t1099 565 3\n",
      "20\t11\t0.0\t2\t2 5 4\n",
      "21\t294\t0.0\t2\t0 291 3\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "27\t1\t0.0\t2\t1\n",
      "40\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t3\n",
      "217\t2\t0.0\t2\t2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.75percent.6.6.19_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.75percent.6.6.19_49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.75percent.6.6.19_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.75percent.6.6.19_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.75percent.6.6.19_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.20.F.75percent.6.6.19_49_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.75percent.6.6.19_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.20.F.75percent.6.6.19_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       242,216 reads  @     13.9 µs/read;   4.30 M reads/minute\n",
      "Finished in 3.40 s (14 us/read; 4.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            242,216\n",
      "  Read 1 with adapter:                     383 (0.2%)\n",
      "  Read 2 with adapter:                  15,371 (6.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       242,216 (100.0%)\n",
      "\n",
      "Total basepairs processed:   110,284,957 bp\n",
      "  Read 1:    54,102,074 bp\n",
      "  Read 2:    56,182,883 bp\n",
      "Total written (filtered):    110,084,540 bp (99.8%)\n",
      "  Read 1:    54,098,787 bp\n",
      "  Read 2:    55,985,753 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 383 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t364\t3784.6\t0\t364\n",
      "13\t1\t0.0\t1\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t4\t0.0\t1\t4\n",
      "216\t4\t0.0\t1\t4\n",
      "217\t4\t0.0\t1\t4\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 15371 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t128\t3784.6\t0\t128\n",
      "4\t25\t946.2\t0\t25\n",
      "5\t54\t236.5\t0\t54\n",
      "6\t95\t59.1\t0\t95\n",
      "7\t411\t14.8\t0\t411\n",
      "8\t477\t3.7\t0\t477\n",
      "9\t2045\t0.9\t0\t2038 7\n",
      "10\t5533\t0.2\t1\t5469 64\n",
      "11\t173\t0.1\t1\t170 3\n",
      "12\t133\t0.0\t1\t130 3\n",
      "13\t155\t0.0\t1\t151 4\n",
      "14\t351\t0.0\t1\t331 20\n",
      "15\t304\t0.0\t1\t293 11\n",
      "16\t855\t0.0\t1\t834 21\n",
      "17\t1239\t0.0\t1\t1200 39\n",
      "18\t736\t0.0\t1\t677 44 15\n",
      "19\t2330\t0.0\t1\t1418 905 7\n",
      "20\t7\t0.0\t2\t2 4 1\n",
      "21\t296\t0.0\t2\t0 288 8\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "79\t1\t0.0\t2\t1\n",
      "139\t1\t0.0\t2\t0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 0 1\n",
      "241\t2\t0.0\t2\t2\n",
      "246\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.3.D.final.6.25.19_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.3.D.final.6.25.19_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.3.D.final.6.25.19_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.3.D.final.6.25.19_8_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.3.D.final.6.25.19_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.3.D.final.6.25.19_8_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.3.D.final.6.25.19_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.3.D.final.6.25.19_8_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       240,410 reads  @     15.0 µs/read;   3.99 M reads/minute\n",
      "Finished in 3.65 s (15 us/read; 3.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            240,410\n",
      "  Read 1 with adapter:                     228 (0.1%)\n",
      "  Read 2 with adapter:                   5,436 (2.3%)\n",
      "Pairs that were too short:                   2 (0.0%)\n",
      "Pairs written (passing filters):       240,408 (100.0%)\n",
      "\n",
      "Total basepairs processed:   109,193,410 bp\n",
      "  Read 1:    53,669,255 bp\n",
      "  Read 2:    55,524,155 bp\n",
      "Total written (filtered):    109,102,547 bp (99.9%)\n",
      "  Read 1:    53,666,959 bp\n",
      "  Read 2:    55,435,588 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 228 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t219\t3756.4\t0\t219\n",
      "4\t1\t939.1\t0\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "116\t1\t0.0\t1\t1\n",
      "177\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "211\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5436 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t3756.4\t0\t22\n",
      "4\t7\t939.1\t0\t7\n",
      "5\t13\t234.8\t0\t13\n",
      "6\t20\t58.7\t0\t20\n",
      "7\t43\t14.7\t0\t43\n",
      "8\t76\t3.7\t0\t76\n",
      "9\t247\t0.9\t0\t245 2\n",
      "10\t783\t0.2\t1\t775 8\n",
      "11\t57\t0.1\t1\t53 4\n",
      "12\t52\t0.0\t1\t51 1\n",
      "13\t66\t0.0\t1\t64 2\n",
      "14\t136\t0.0\t1\t122 14\n",
      "15\t142\t0.0\t1\t135 7\n",
      "16\t418\t0.0\t1\t409 9\n",
      "17\t745\t0.0\t1\t726 19\n",
      "18\t508\t0.0\t1\t469 34 5\n",
      "19\t1764\t0.0\t1\t1119 637 8\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t302\t0.0\t2\t0 298 4\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "63\t2\t0.0\t2\t1 0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "70\t2\t0.0\t2\t1 0 1\n",
      "72\t1\t0.0\t2\t0 1\n",
      "75\t2\t0.0\t2\t1 1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t6\t0.0\t2\t6\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t1\n",
      "227\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t0 0 1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.E.25percent.5.6.19_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.E.25percent.5.6.19_57_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.E.25percent.5.6.19_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.E.25percent.5.6.19_57_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.E.25percent.5.6.19_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.E.25percent.5.6.19_57_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.E.25percent.5.6.19_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.E.25percent.5.6.19_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       462,977 reads  @     11.9 µs/read;   5.03 M reads/minute\n",
      "Finished in 5.55 s (12 us/read; 5.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            462,977\n",
      "  Read 1 with adapter:                     649 (0.1%)\n",
      "  Read 2 with adapter:                  19,570 (4.2%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       462,976 (100.0%)\n",
      "\n",
      "Total basepairs processed:   211,113,672 bp\n",
      "  Read 1:   103,681,751 bp\n",
      "  Read 2:   107,431,921 bp\n",
      "Total written (filtered):    210,814,598 bp (99.9%)\n",
      "  Read 1:   103,679,161 bp\n",
      "  Read 2:   107,135,437 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 649 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t640\t7234.0\t0\t640\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t2\t0.0\t1\t2\n",
      "32\t2\t0.0\t1\t2\n",
      "38\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 19570 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t84\t7234.0\t0\t84\n",
      "4\t43\t1808.5\t0\t43\n",
      "5\t54\t452.1\t0\t54\n",
      "6\t148\t113.0\t0\t148\n",
      "7\t335\t28.3\t0\t335\n",
      "8\t480\t7.1\t0\t480\n",
      "9\t1599\t1.8\t0\t1596 3\n",
      "10\t2745\t0.4\t1\t2718 27\n",
      "11\t279\t0.1\t1\t277 2\n",
      "12\t544\t0.0\t1\t521 23\n",
      "13\t669\t0.0\t1\t640 29\n",
      "14\t996\t0.0\t1\t937 59\n",
      "15\t602\t0.0\t1\t560 42\n",
      "16\t1748\t0.0\t1\t1676 72\n",
      "17\t2336\t0.0\t1\t2249 87\n",
      "18\t1383\t0.0\t1\t1208 114 61\n",
      "19\t4762\t0.0\t1\t2859 1877 26\n",
      "20\t15\t0.0\t2\t5 8 2\n",
      "21\t652\t0.0\t2\t0 643 9\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "29\t1\t0.0\t2\t0 1\n",
      "30\t1\t0.0\t2\t1\n",
      "42\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t1\n",
      "52\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t1\n",
      "57\t1\t0.0\t2\t1\n",
      "58\t1\t0.0\t2\t1\n",
      "59\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "70\t6\t0.0\t2\t4 0 2\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "73\t2\t0.0\t2\t1 0 1\n",
      "77\t2\t0.0\t2\t0 1 1\n",
      "78\t2\t0.0\t2\t1 1\n",
      "79\t3\t0.0\t2\t1 2\n",
      "84\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t1\n",
      "109\t1\t0.0\t2\t1\n",
      "112\t1\t0.0\t2\t1\n",
      "118\t1\t0.0\t2\t1\n",
      "144\t1\t0.0\t2\t1\n",
      "152\t1\t0.0\t2\t0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "183\t1\t0.0\t2\t1\n",
      "193\t1\t0.0\t2\t1\n",
      "198\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t3\t0.0\t2\t2 0 1\n",
      "229\t1\t0.0\t2\t1\n",
      "238\t2\t0.0\t2\t2\n",
      "239\t3\t0.0\t2\t3\n",
      "244\t1\t0.0\t2\t0 1\n",
      "245\t2\t0.0\t2\t2\n",
      "246\t3\t0.0\t2\t1 2\n",
      "249\t1\t0.0\t2\t1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.E.final.6.25.19_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.E.final.6.25.19_69_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.E.final.6.25.19_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.E.final.6.25.19_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.E.final.6.25.19_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.E.final.6.25.19_69_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.E.final.6.25.19_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.E.final.6.25.19_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        45,539 reads  @     27.7 µs/read;   2.16 M reads/minute\n",
      "Finished in 1.29 s (28 us/read; 2.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,539\n",
      "  Read 1 with adapter:                      67 (0.1%)\n",
      "  Read 2 with adapter:                   2,245 (4.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,539 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,309,115 bp\n",
      "  Read 1:     9,956,470 bp\n",
      "  Read 2:    10,352,645 bp\n",
      "Total written (filtered):     20,274,965 bp (99.8%)\n",
      "  Read 1:     9,954,230 bp\n",
      "  Read 2:    10,320,735 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 67 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t57\t711.5\t0\t57\n",
      "128\t1\t0.0\t1\t0 1\n",
      "207\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t0 2\n",
      "217\t6\t0.0\t1\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 2245 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t711.5\t0\t25\n",
      "4\t7\t177.9\t0\t7\n",
      "5\t3\t44.5\t0\t3\n",
      "6\t7\t11.1\t0\t7\n",
      "7\t39\t2.8\t0\t39\n",
      "8\t54\t0.7\t0\t54\n",
      "9\t174\t0.2\t0\t174\n",
      "10\t621\t0.0\t1\t614 7\n",
      "11\t26\t0.0\t1\t26\n",
      "12\t28\t0.0\t1\t28\n",
      "13\t22\t0.0\t1\t20 2\n",
      "14\t60\t0.0\t1\t59 1\n",
      "15\t66\t0.0\t1\t63 3\n",
      "16\t169\t0.0\t1\t161 8\n",
      "17\t223\t0.0\t1\t215 8\n",
      "18\t137\t0.0\t1\t125 7 5\n",
      "19\t514\t0.0\t1\t308 201 5\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t63\t0.0\t2\t0 63\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.G.25percent.5.6.19_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.G.25percent.5.6.19_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.G.25percent.5.6.19_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.G.25percent.5.6.19_2_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.G.25percent.5.6.19_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.G.25percent.5.6.19_2_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.G.25percent.5.6.19_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.G.25percent.5.6.19_2_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       264,220 reads  @     14.5 µs/read;   4.15 M reads/minute\n",
      "Finished in 3.85 s (15 us/read; 4.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            264,220\n",
      "  Read 1 with adapter:                     316 (0.1%)\n",
      "  Read 2 with adapter:                  14,721 (5.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       264,220 (100.0%)\n",
      "\n",
      "Total basepairs processed:   120,562,616 bp\n",
      "  Read 1:    59,161,831 bp\n",
      "  Read 2:    61,400,785 bp\n",
      "Total written (filtered):    120,357,026 bp (99.8%)\n",
      "  Read 1:    59,160,305 bp\n",
      "  Read 2:    61,196,721 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 316 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t311\t4128.4\t0\t311\n",
      "16\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "153\t1\t0.0\t1\t1\n",
      "194\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14721 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t97\t4128.4\t0\t97\n",
      "4\t47\t1032.1\t0\t47\n",
      "5\t66\t258.0\t0\t66\n",
      "6\t129\t64.5\t0\t129\n",
      "7\t414\t16.1\t0\t414\n",
      "8\t695\t4.0\t0\t695\n",
      "9\t2058\t1.0\t0\t2055 3\n",
      "10\t2583\t0.3\t1\t2558 25\n",
      "11\t197\t0.1\t1\t194 3\n",
      "12\t369\t0.0\t1\t363 6\n",
      "13\t482\t0.0\t1\t468 14\n",
      "14\t736\t0.0\t1\t694 42\n",
      "15\t404\t0.0\t1\t374 30\n",
      "16\t1083\t0.0\t1\t1029 54\n",
      "17\t1432\t0.0\t1\t1384 48\n",
      "18\t822\t0.0\t1\t730 52 40\n",
      "19\t2687\t0.0\t1\t1517 1162 8\n",
      "20\t6\t0.0\t2\t0 6\n",
      "21\t338\t0.0\t2\t0 336 2\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "30\t1\t0.0\t2\t1\n",
      "33\t1\t0.0\t2\t1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t1\n",
      "50\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t2\t0 1\n",
      "59\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "63\t2\t0.0\t2\t2\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t1\n",
      "67\t2\t0.0\t2\t0 1 1\n",
      "68\t3\t0.0\t2\t1 0 2\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "70\t4\t0.0\t2\t2 0 2\n",
      "71\t5\t0.0\t2\t1 0 4\n",
      "74\t3\t0.0\t2\t3\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t8\t0.0\t2\t5 1 2\n",
      "84\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t0 1\n",
      "90\t1\t0.0\t2\t0 1\n",
      "100\t2\t0.0\t2\t2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t1\n",
      "141\t1\t0.0\t2\t1\n",
      "144\t1\t0.0\t2\t1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "186\t2\t0.0\t2\t2\n",
      "187\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t2\n",
      "233\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.G.50percent.5.15.19_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.G.50percent.5.15.19_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.G.50percent.5.15.19_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.G.50percent.5.15.19_18_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.G.50percent.5.15.19_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.4.G.50percent.5.15.19_18_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.G.50percent.5.15.19_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.4.G.50percent.5.15.19_18_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       284,203 reads  @     15.1 µs/read;   3.97 M reads/minute\n",
      "Finished in 4.32 s (15 us/read; 3.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            284,203\n",
      "  Read 1 with adapter:                     335 (0.1%)\n",
      "  Read 2 with adapter:                  13,424 (4.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       284,203 (100.0%)\n",
      "\n",
      "Total basepairs processed:   129,603,105 bp\n",
      "  Read 1:    63,636,601 bp\n",
      "  Read 2:    65,966,504 bp\n",
      "Total written (filtered):    129,419,896 bp (99.9%)\n",
      "  Read 1:    63,634,574 bp\n",
      "  Read 2:    65,785,322 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 335 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t324\t4440.7\t0\t324\n",
      "11\t1\t0.1\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "27\t3\t0.0\t1\t3\n",
      "147\t1\t0.0\t1\t1\n",
      "160\t1\t0.0\t1\t1\n",
      "171\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 13424 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t93\t4440.7\t0\t93\n",
      "4\t35\t1110.2\t0\t35\n",
      "5\t49\t277.5\t0\t49\n",
      "6\t85\t69.4\t0\t85\n",
      "7\t307\t17.3\t0\t307\n",
      "8\t417\t4.3\t0\t417\n",
      "9\t1467\t1.1\t0\t1457 10\n",
      "10\t4143\t0.3\t1\t4102 41\n",
      "11\t165\t0.1\t1\t160 5\n",
      "12\t137\t0.0\t1\t135 2\n",
      "13\t131\t0.0\t1\t127 4\n",
      "14\t325\t0.0\t1\t302 23\n",
      "15\t306\t0.0\t1\t299 7\n",
      "16\t823\t0.0\t1\t792 31\n",
      "17\t1247\t0.0\t1\t1205 42\n",
      "18\t735\t0.0\t1\t676 46 13\n",
      "19\t2555\t0.0\t1\t1577 964 14\n",
      "20\t6\t0.0\t2\t0 4 2\n",
      "21\t370\t0.0\t2\t0 363 7\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t1\n",
      "70\t1\t0.0\t2\t1\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "75\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t6\t0.0\t2\t6\n",
      "217\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.A.25percent.6.5.19_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.A.25percent.6.5.19_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.A.25percent.6.5.19_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.A.25percent.6.5.19_46_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.A.25percent.6.5.19_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.A.25percent.6.5.19_46_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.A.25percent.6.5.19_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.A.25percent.6.5.19_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       278,765 reads  @     14.0 µs/read;   4.29 M reads/minute\n",
      "Finished in 3.93 s (14 us/read; 4.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            278,765\n",
      "  Read 1 with adapter:                     177 (0.1%)\n",
      "  Read 2 with adapter:                  10,899 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       278,765 (100.0%)\n",
      "\n",
      "Total basepairs processed:   126,985,450 bp\n",
      "  Read 1:    62,371,767 bp\n",
      "  Read 2:    64,613,683 bp\n",
      "Total written (filtered):    126,832,313 bp (99.9%)\n",
      "  Read 1:    62,370,019 bp\n",
      "  Read 2:    64,462,294 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 177 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t157\t4355.7\t0\t157\n",
      "12\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t8\t0.0\t1\t8\n",
      "29\t1\t0.0\t1\t1\n",
      "217\t4\t0.0\t1\t3 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 10899 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t97\t4355.7\t0\t97\n",
      "4\t18\t1088.9\t0\t18\n",
      "5\t34\t272.2\t0\t34\n",
      "6\t53\t68.1\t0\t53\n",
      "7\t175\t17.0\t0\t175\n",
      "8\t293\t4.3\t0\t293\n",
      "9\t1028\t1.1\t0\t1025 3\n",
      "10\t3294\t0.3\t1\t3249 45\n",
      "11\t182\t0.1\t1\t179 3\n",
      "12\t118\t0.0\t1\t117 1\n",
      "13\t87\t0.0\t1\t83 4\n",
      "14\t203\t0.0\t1\t195 8\n",
      "15\t244\t0.0\t1\t238 6\n",
      "16\t658\t0.0\t1\t640 18\n",
      "17\t1028\t0.0\t1\t996 32\n",
      "18\t660\t0.0\t1\t608 39 13\n",
      "19\t2338\t0.0\t1\t1473 857 8\n",
      "20\t7\t0.0\t2\t2 5\n",
      "21\t353\t0.0\t2\t0 345 8\n",
      "22\t14\t0.0\t2\t0 1 13\n",
      "57\t1\t0.0\t2\t1\n",
      "59\t2\t0.0\t2\t0 1 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t1 1 1\n",
      "114\t1\t0.0\t2\t1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t1\n",
      "217\t2\t0.0\t2\t1 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.A.75percent.6.6.19_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.A.75percent.6.6.19_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.A.75percent.6.6.19_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.A.75percent.6.6.19_22_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.A.75percent.6.6.19_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.A.75percent.6.6.19_22_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.A.75percent.6.6.19_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.A.75percent.6.6.19_22_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       298,097 reads  @     15.1 µs/read;   3.97 M reads/minute\n",
      "Finished in 4.54 s (15 us/read; 3.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            298,097\n",
      "  Read 1 with adapter:                     325 (0.1%)\n",
      "  Read 2 with adapter:                  12,275 (4.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       298,097 (100.0%)\n",
      "\n",
      "Total basepairs processed:   135,862,684 bp\n",
      "  Read 1:    66,728,283 bp\n",
      "  Read 2:    69,134,401 bp\n",
      "Total written (filtered):    135,685,954 bp (99.9%)\n",
      "  Read 1:    66,726,420 bp\n",
      "  Read 2:    68,959,534 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 325 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t312\t4657.8\t0\t312\n",
      "5\t1\t291.1\t0\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t2\t0.0\t1\t2\n",
      "35\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "153\t1\t0.0\t1\t1\n",
      "194\t1\t0.0\t1\t0 1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 12275 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t142\t4657.8\t0\t142\n",
      "4\t23\t1164.4\t0\t23\n",
      "5\t31\t291.1\t0\t31\n",
      "6\t69\t72.8\t0\t69\n",
      "7\t199\t18.2\t0\t199\n",
      "8\t290\t4.5\t0\t290\n",
      "9\t1040\t1.1\t0\t1032 8\n",
      "10\t3418\t0.3\t1\t3373 45\n",
      "11\t143\t0.1\t1\t142 1\n",
      "12\t147\t0.0\t1\t146 1\n",
      "13\t129\t0.0\t1\t122 7\n",
      "14\t296\t0.0\t1\t276 20\n",
      "15\t263\t0.0\t1\t249 14\n",
      "16\t821\t0.0\t1\t794 27\n",
      "17\t1297\t0.0\t1\t1254 43\n",
      "18\t761\t0.0\t1\t702 45 14\n",
      "19\t2726\t0.0\t1\t1710 1006 10\n",
      "20\t5\t0.0\t2\t0 4 1\n",
      "21\t427\t0.0\t2\t0 418 9\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t1\n",
      "62\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t1\n",
      "68\t1\t0.0\t2\t1\n",
      "70\t3\t0.0\t2\t1 2\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 1\n",
      "77\t1\t0.0\t2\t1\n",
      "79\t6\t0.0\t2\t4 1 1\n",
      "100\t1\t0.0\t2\t1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t1\n",
      "186\t2\t0.0\t2\t2\n",
      "216\t1\t0.0\t2\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.25percent.6.3.19_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.25percent.6.3.19_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.25percent.6.3.19_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.25percent.6.3.19_14_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.25percent.6.3.19_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.25percent.6.3.19_14_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.25percent.6.3.19_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.25percent.6.3.19_14_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       285,350 reads  @     14.3 µs/read;   4.20 M reads/minute\n",
      "Finished in 4.11 s (14 us/read; 4.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            285,350\n",
      "  Read 1 with adapter:                     347 (0.1%)\n",
      "  Read 2 with adapter:                   9,643 (3.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       285,350 (100.0%)\n",
      "\n",
      "Total basepairs processed:   130,050,535 bp\n",
      "  Read 1:    63,898,476 bp\n",
      "  Read 2:    66,152,059 bp\n",
      "Total written (filtered):    129,902,392 bp (99.9%)\n",
      "  Read 1:    63,896,712 bp\n",
      "  Read 2:    66,005,680 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 347 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t339\t4458.6\t0\t339\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "129\t1\t0.0\t1\t1\n",
      "144\t1\t0.0\t1\t1\n",
      "177\t1\t0.0\t1\t1\n",
      "197\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 9643 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t4458.6\t0\t43\n",
      "4\t13\t1114.6\t0\t13\n",
      "5\t29\t278.7\t0\t29\n",
      "6\t69\t69.7\t0\t69\n",
      "7\t139\t17.4\t0\t139\n",
      "8\t222\t4.4\t0\t222\n",
      "9\t721\t1.1\t0\t719 2\n",
      "10\t1198\t0.3\t1\t1187 11\n",
      "11\t168\t0.1\t1\t165 3\n",
      "12\t275\t0.0\t1\t269 6\n",
      "13\t370\t0.0\t1\t353 17\n",
      "14\t542\t0.0\t1\t520 22\n",
      "15\t298\t0.0\t1\t284 14\n",
      "16\t832\t0.0\t1\t800 32\n",
      "17\t1175\t0.0\t1\t1139 36\n",
      "18\t662\t0.0\t1\t583 55 24\n",
      "19\t2510\t0.0\t1\t1560 945 5\n",
      "20\t5\t0.0\t2\t2 2 1\n",
      "21\t351\t0.0\t2\t0 344 7\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t1\n",
      "123\t1\t0.0\t2\t1\n",
      "186\t1\t0.0\t2\t1\n",
      "191\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t1\n",
      "214\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.5percent.5.30.19_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.5percent.5.30.19_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.5percent.5.30.19_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.5percent.5.30.19_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.5percent.5.30.19_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.5percent.5.30.19_43_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.5percent.5.30.19_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.5percent.5.30.19_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       334,960 reads  @     13.3 µs/read;   4.50 M reads/minute\n",
      "Finished in 4.49 s (13 us/read; 4.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            334,960\n",
      "  Read 1 with adapter:                     286 (0.1%)\n",
      "  Read 2 with adapter:                  29,865 (8.9%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       334,959 (100.0%)\n",
      "\n",
      "Total basepairs processed:   153,128,204 bp\n",
      "  Read 1:    75,008,409 bp\n",
      "  Read 2:    78,119,795 bp\n",
      "Total written (filtered):    152,768,152 bp (99.8%)\n",
      "  Read 1:    75,006,439 bp\n",
      "  Read 2:    77,761,713 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 286 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t271\t5233.8\t0\t271\n",
      "15\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t0 1\n",
      "26\t7\t0.0\t1\t7\n",
      "27\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "166\t1\t0.0\t1\t1\n",
      "185\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 29865 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t317\t5233.8\t0\t317\n",
      "4\t184\t1308.4\t0\t184\n",
      "5\t316\t327.1\t0\t316\n",
      "6\t769\t81.8\t0\t769\n",
      "7\t2035\t20.4\t0\t2035\n",
      "8\t2791\t5.1\t0\t2791\n",
      "9\t6860\t1.3\t0\t6852 8\n",
      "10\t5750\t0.3\t1\t5693 57\n",
      "11\t259\t0.1\t1\t257 2\n",
      "12\t352\t0.0\t1\t343 9\n",
      "13\t530\t0.0\t1\t510 20\n",
      "14\t813\t0.0\t1\t775 38\n",
      "15\t587\t0.0\t1\t558 29\n",
      "16\t1382\t0.0\t1\t1331 51\n",
      "17\t1806\t0.0\t1\t1725 81\n",
      "18\t1089\t0.0\t1\t940 80 69\n",
      "19\t3439\t0.0\t1\t1877 1543 19\n",
      "20\t16\t0.0\t2\t10 5 1\n",
      "21\t461\t0.0\t2\t0 452 9\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "29\t1\t0.0\t2\t1\n",
      "33\t1\t0.0\t2\t1\n",
      "38\t1\t0.0\t2\t1\n",
      "39\t1\t0.0\t2\t0 1\n",
      "40\t1\t0.0\t2\t1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t3\n",
      "59\t2\t0.0\t2\t1 0 1\n",
      "61\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t1\n",
      "70\t4\t0.0\t2\t3 1\n",
      "71\t5\t0.0\t2\t0 1 4\n",
      "72\t1\t0.0\t2\t1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t2\t0.0\t2\t2\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t2\t0.0\t2\t1 0 1\n",
      "79\t2\t0.0\t2\t2\n",
      "83\t1\t0.0\t2\t1\n",
      "89\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "100\t1\t0.0\t2\t1\n",
      "110\t1\t0.0\t2\t1\n",
      "148\t1\t0.0\t2\t1\n",
      "149\t1\t0.0\t2\t0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "183\t1\t0.0\t2\t1\n",
      "187\t1\t0.0\t2\t1\n",
      "190\t1\t0.0\t2\t1\n",
      "193\t1\t0.0\t2\t1\n",
      "209\t1\t0.0\t2\t1\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t0 1\n",
      "221\t1\t0.0\t2\t1\n",
      "225\t1\t0.0\t2\t1\n",
      "228\t1\t0.0\t2\t1\n",
      "232\t1\t0.0\t2\t1\n",
      "234\t1\t0.0\t2\t1\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t3\t0.0\t2\t2 1\n",
      "238\t4\t0.0\t2\t2 1 1\n",
      "239\t1\t0.0\t2\t0 1\n",
      "240\t4\t0.0\t2\t3 0 1\n",
      "241\t2\t0.0\t2\t1 1\n",
      "242\t2\t0.0\t2\t1 1\n",
      "243\t3\t0.0\t2\t2 1\n",
      "245\t2\t0.0\t2\t1 1\n",
      "248\t1\t0.0\t2\t1\n",
      "250\t2\t0.0\t2\t0 1 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.75percent.6.5.19_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.75percent.6.5.19_52_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.75percent.6.5.19_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.75percent.6.5.19_52_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.75percent.6.5.19_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.D.75percent.6.5.19_52_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.75percent.6.5.19_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.D.75percent.6.5.19_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:07       585,172 reads  @     12.7 µs/read;   4.73 M reads/minute\n",
      "Finished in 7.44 s (13 us/read; 4.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            585,172\n",
      "  Read 1 with adapter:                     572 (0.1%)\n",
      "  Read 2 with adapter:                  23,478 (4.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       585,172 (100.0%)\n",
      "\n",
      "Total basepairs processed:   266,795,613 bp\n",
      "  Read 1:   131,037,255 bp\n",
      "  Read 2:   135,758,358 bp\n",
      "Total written (filtered):    266,439,766 bp (99.9%)\n",
      "  Read 1:   131,034,330 bp\n",
      "  Read 2:   135,405,436 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 572 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t558\t9143.3\t0\t558\n",
      "27\t5\t0.0\t1\t5\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "87\t1\t0.0\t1\t1\n",
      "124\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "166\t2\t0.0\t1\t2\n",
      "184\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 23478 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t120\t9143.3\t0\t120\n",
      "4\t38\t2285.8\t0\t38\n",
      "5\t43\t571.5\t0\t43\n",
      "6\t147\t142.9\t0\t147\n",
      "7\t342\t35.7\t0\t342\n",
      "8\t509\t8.9\t0\t509\n",
      "9\t1741\t2.2\t0\t1737 4\n",
      "10\t3472\t0.6\t1\t3441 31\n",
      "11\t424\t0.1\t1\t417 7\n",
      "12\t707\t0.0\t1\t687 20\n",
      "13\t888\t0.0\t1\t860 28\n",
      "14\t1153\t0.0\t1\t1092 61\n",
      "15\t731\t0.0\t1\t679 52\n",
      "16\t2099\t0.0\t1\t2035 64\n",
      "17\t2783\t0.0\t1\t2670 113\n",
      "18\t1623\t0.0\t1\t1464 94 65\n",
      "19\t5776\t0.0\t1\t3469 2283 24\n",
      "20\t18\t0.0\t2\t5 7 6\n",
      "21\t777\t0.0\t2\t0 763 14\n",
      "22\t25\t0.0\t2\t0 1 24\n",
      "30\t1\t0.0\t2\t1\n",
      "43\t4\t0.0\t2\t0 0 4\n",
      "46\t1\t0.0\t2\t1\n",
      "47\t1\t0.0\t2\t0 1\n",
      "52\t1\t0.0\t2\t1\n",
      "54\t2\t0.0\t2\t2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t1\n",
      "63\t2\t0.0\t2\t1 1\n",
      "64\t1\t0.0\t2\t1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t1\n",
      "69\t2\t0.0\t2\t2\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t4\t0.0\t2\t0 0 4\n",
      "72\t1\t0.0\t2\t1\n",
      "75\t1\t0.0\t2\t1\n",
      "76\t1\t0.0\t2\t1\n",
      "77\t1\t0.0\t2\t1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t12\t0.0\t2\t10 0 2\n",
      "80\t1\t0.0\t2\t1\n",
      "90\t1\t0.0\t2\t1\n",
      "93\t1\t0.0\t2\t1\n",
      "96\t1\t0.0\t2\t1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t1 0 1\n",
      "114\t1\t0.0\t2\t0 1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t1\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 1\n",
      "186\t2\t0.0\t2\t2\n",
      "216\t2\t0.0\t2\t2\n",
      "217\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.E.25percent.4.28.19_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.E.25percent.4.28.19_50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.E.25percent.4.28.19_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.E.25percent.4.28.19_50_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.E.25percent.4.28.19_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.E.25percent.4.28.19_50_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.E.25percent.4.28.19_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.E.25percent.4.28.19_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       317,743 reads  @     13.1 µs/read;   4.57 M reads/minute\n",
      "Finished in 4.20 s (13 us/read; 4.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            317,743\n",
      "  Read 1 with adapter:                     338 (0.1%)\n",
      "  Read 2 with adapter:                  16,167 (5.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       317,743 (100.0%)\n",
      "\n",
      "Total basepairs processed:   144,932,490 bp\n",
      "  Read 1:    71,151,573 bp\n",
      "  Read 2:    73,780,917 bp\n",
      "Total written (filtered):    144,721,595 bp (99.9%)\n",
      "  Read 1:    71,149,993 bp\n",
      "  Read 2:    73,571,602 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 338 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t316\t4964.7\t0\t316\n",
      "6\t1\t77.6\t0\t1\n",
      "25\t5\t0.0\t1\t5\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t10\t0.0\t1\t10\n",
      "101\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 16167 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t116\t4964.7\t0\t116\n",
      "4\t27\t1241.2\t0\t27\n",
      "5\t55\t310.3\t0\t55\n",
      "6\t107\t77.6\t0\t107\n",
      "7\t361\t19.4\t0\t361\n",
      "8\t543\t4.8\t0\t543\n",
      "9\t2079\t1.2\t0\t2069 10\n",
      "10\t5978\t0.3\t1\t5906 72\n",
      "11\t217\t0.1\t1\t214 3\n",
      "12\t111\t0.0\t1\t107 4\n",
      "13\t99\t0.0\t1\t97 2\n",
      "14\t235\t0.0\t1\t221 14\n",
      "15\t275\t0.0\t1\t267 8\n",
      "16\t792\t0.0\t1\t767 25\n",
      "17\t1243\t0.0\t1\t1205 38\n",
      "18\t784\t0.0\t1\t725 39 20\n",
      "19\t2707\t0.0\t1\t1642 1047 18\n",
      "20\t6\t0.0\t2\t1 4 1\n",
      "21\t405\t0.0\t2\t0 395 10\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "40\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t1\n",
      "79\t2\t0.0\t2\t1 1\n",
      "96\t1\t0.0\t2\t1\n",
      "192\t1\t0.0\t2\t1\n",
      "200\t1\t0.0\t2\t1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "217\t3\t0.0\t2\t2 1\n",
      "237\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.E.75percent.6.5.19_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.E.75percent.6.5.19_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.E.75percent.6.5.19_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.E.75percent.6.5.19_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.E.75percent.6.5.19_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.6.E.75percent.6.5.19_9_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.E.75percent.6.5.19_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.6.E.75percent.6.5.19_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       256,473 reads  @     13.9 µs/read;   4.33 M reads/minute\n",
      "Finished in 3.58 s (14 us/read; 4.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            256,473\n",
      "  Read 1 with adapter:                     278 (0.1%)\n",
      "  Read 2 with adapter:                   5,347 (2.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       256,473 (100.0%)\n",
      "\n",
      "Total basepairs processed:   116,821,554 bp\n",
      "  Read 1:    57,435,287 bp\n",
      "  Read 2:    59,386,267 bp\n",
      "Total written (filtered):    116,731,947 bp (99.9%)\n",
      "  Read 1:    57,434,384 bp\n",
      "  Read 2:    59,297,563 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 278 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t275\t4007.4\t0\t275\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 5347 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t4007.4\t0\t22\n",
      "4\t4\t1001.8\t0\t4\n",
      "5\t18\t250.5\t0\t18\n",
      "6\t19\t62.6\t0\t19\n",
      "7\t52\t15.7\t0\t52\n",
      "8\t86\t3.9\t0\t86\n",
      "9\t235\t1.0\t0\t233 2\n",
      "10\t437\t0.2\t1\t430 7\n",
      "11\t40\t0.1\t1\t39 1\n",
      "12\t76\t0.0\t1\t76\n",
      "13\t56\t0.0\t1\t54 2\n",
      "14\t149\t0.0\t1\t138 11\n",
      "15\t130\t0.0\t1\t126 4\n",
      "16\t434\t0.0\t1\t421 13\n",
      "17\t788\t0.0\t1\t763 25\n",
      "18\t508\t0.0\t1\t470 27 11\n",
      "19\t1955\t0.0\t1\t1229 722 4\n",
      "20\t2\t0.0\t2\t2\n",
      "21\t313\t0.0\t2\t0 310 3\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "55\t1\t0.0\t2\t1\n",
      "64\t1\t0.0\t2\t0 1\n",
      "78\t1\t0.0\t2\t1\n",
      "79\t1\t0.0\t2\t0 1\n",
      "216\t2\t0.0\t2\t1 1\n",
      "236\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.5.15.19_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.5.15.19_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.5.15.19_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.5.15.19_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.5.15.19_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.5.15.19_31_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.5.15.19_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.5.15.19_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        89,564 reads  @     22.7 µs/read;   2.64 M reads/minute\n",
      "Finished in 2.07 s (23 us/read; 2.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             89,564\n",
      "  Read 1 with adapter:                     102 (0.1%)\n",
      "  Read 2 with adapter:                   3,499 (3.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        89,564 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,493,984 bp\n",
      "  Read 1:    19,876,149 bp\n",
      "  Read 2:    20,617,835 bp\n",
      "Total written (filtered):     40,441,900 bp (99.9%)\n",
      "  Read 1:    19,874,989 bp\n",
      "  Read 2:    20,566,911 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 102 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t98\t1399.4\t0\t98\n",
      "216\t2\t0.0\t1\t1 1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 3499 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t1399.4\t0\t24\n",
      "4\t2\t349.9\t0\t2\n",
      "5\t12\t87.5\t0\t12\n",
      "6\t16\t21.9\t0\t16\n",
      "7\t53\t5.5\t0\t53\n",
      "8\t89\t1.4\t0\t89\n",
      "9\t254\t0.3\t0\t254\n",
      "10\t881\t0.1\t1\t872 9\n",
      "11\t48\t0.0\t1\t47 1\n",
      "12\t40\t0.0\t1\t37 3\n",
      "13\t39\t0.0\t1\t39\n",
      "14\t93\t0.0\t1\t88 5\n",
      "15\t103\t0.0\t1\t100 3\n",
      "16\t277\t0.0\t1\t268 9\n",
      "17\t382\t0.0\t1\t374 8\n",
      "18\t255\t0.0\t1\t239 13 3\n",
      "19\t811\t0.0\t1\t506 302 3\n",
      "20\t3\t0.0\t2\t0 3\n",
      "21\t106\t0.0\t2\t0 103 3\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t1\n",
      "60\t1\t0.0\t2\t1\n",
      "79\t3\t0.0\t2\t1 0 2\n",
      "227\t1\t0.0\t2\t1\n",
      "249\t1\t0.0\t2\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.5.6.19_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.5.6.19_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.5.6.19_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.5.6.19_42_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.5.6.19_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.5.6.19_42_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.5.6.19_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.5.6.19_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       231,648 reads  @     15.8 µs/read;   3.80 M reads/minute\n",
      "Finished in 3.68 s (16 us/read; 3.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            231,648\n",
      "  Read 1 with adapter:                     204 (0.1%)\n",
      "  Read 2 with adapter:                   6,758 (2.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       231,648 (100.0%)\n",
      "\n",
      "Total basepairs processed:   105,399,917 bp\n",
      "  Read 1:    51,793,166 bp\n",
      "  Read 2:    53,606,751 bp\n",
      "Total written (filtered):    105,297,464 bp (99.9%)\n",
      "  Read 1:    51,791,419 bp\n",
      "  Read 2:    53,506,045 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 204 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t196\t3619.5\t0\t196\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "35\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 6758 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t3619.5\t0\t49\n",
      "4\t16\t904.9\t0\t16\n",
      "5\t14\t226.2\t0\t14\n",
      "6\t33\t56.6\t0\t33\n",
      "7\t86\t14.1\t0\t86\n",
      "8\t109\t3.5\t0\t109\n",
      "9\t454\t0.9\t0\t450 4\n",
      "10\t1560\t0.2\t1\t1543 17\n",
      "11\t123\t0.1\t1\t123\n",
      "12\t139\t0.0\t1\t137 2\n",
      "13\t75\t0.0\t1\t75\n",
      "14\t117\t0.0\t1\t108 9\n",
      "15\t171\t0.0\t1\t168 3\n",
      "16\t500\t0.0\t1\t485 15\n",
      "17\t755\t0.0\t1\t738 17\n",
      "18\t504\t0.0\t1\t476 23 5\n",
      "19\t1772\t0.0\t1\t1178 588 6\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t260\t0.0\t2\t0 260\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "49\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "205\t1\t0.0\t2\t0 1\n",
      "212\t1\t0.0\t2\t1\n",
      "217\t3\t0.0\t2\t2 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.6.14.19_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.6.14.19_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.6.14.19_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.6.14.19_20_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.6.14.19_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.6.14.19_20_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.6.14.19_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.6.14.19_20_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       331,014 reads  @     13.6 µs/read;   4.42 M reads/minute\n",
      "Finished in 4.52 s (14 us/read; 4.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            331,014\n",
      "  Read 1 with adapter:                     262 (0.1%)\n",
      "  Read 2 with adapter:                  14,387 (4.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       331,014 (100.0%)\n",
      "\n",
      "Total basepairs processed:   150,932,109 bp\n",
      "  Read 1:    74,116,015 bp\n",
      "  Read 2:    76,816,094 bp\n",
      "Total written (filtered):    150,722,473 bp (99.9%)\n",
      "  Read 1:    74,114,477 bp\n",
      "  Read 2:    76,607,996 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 262 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t254\t5172.1\t0\t254\n",
      "5\t1\t323.3\t0\t1\n",
      "27\t3\t0.0\t1\t3\n",
      "83\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "222\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 14387 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t5172.1\t0\t79\n",
      "4\t36\t1293.0\t0\t36\n",
      "5\t35\t323.3\t0\t35\n",
      "6\t117\t80.8\t0\t117\n",
      "7\t232\t20.2\t0\t232\n",
      "8\t473\t5.1\t0\t473\n",
      "9\t1559\t1.3\t0\t1557 2\n",
      "10\t2323\t0.3\t1\t2298 25\n",
      "11\t212\t0.1\t1\t205 7\n",
      "12\t424\t0.0\t1\t414 10\n",
      "13\t439\t0.0\t1\t430 9\n",
      "14\t636\t0.0\t1\t604 32\n",
      "15\t400\t0.0\t1\t379 21\n",
      "16\t1139\t0.0\t1\t1082 57\n",
      "17\t1589\t0.0\t1\t1531 58\n",
      "18\t972\t0.0\t1\t863 78 31\n",
      "19\t3242\t0.0\t1\t1946 1279 17\n",
      "20\t8\t0.0\t2\t1 7\n",
      "21\t433\t0.0\t2\t0 430 3\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t1\n",
      "53\t1\t0.0\t2\t0 1\n",
      "54\t1\t0.0\t2\t1\n",
      "55\t1\t0.0\t2\t1\n",
      "61\t1\t0.0\t2\t0 1\n",
      "63\t1\t0.0\t2\t1\n",
      "68\t2\t0.0\t2\t2\n",
      "70\t2\t0.0\t2\t2\n",
      "71\t5\t0.0\t2\t1 0 4\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t1\n",
      "97\t1\t0.0\t2\t0 1\n",
      "100\t1\t0.0\t2\t1\n",
      "154\t1\t0.0\t2\t1\n",
      "235\t1\t0.0\t2\t0 1\n",
      "236\t1\t0.0\t2\t1\n",
      "237\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.6.17.19_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.6.17.19_55_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.6.17.19_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.6.17.19_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.6.17.19_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-97twbha3/M.cav.Tank.7.G.control.6.17.19_55_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACVSGGGTATCTAAT /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.6.17.19_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-2yfvdh80/03c2352d-9227-424f-9397-2bddb7f14f6c/data/M.cav.Tank.7.G.control.6.17.19_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       166,561 reads  @     18.7 µs/read;   3.21 M reads/minute\n",
      "Finished in 3.14 s (19 us/read; 3.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            166,561\n",
      "  Read 1 with adapter:                     145 (0.1%)\n",
      "  Read 2 with adapter:                   4,426 (2.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       166,561 (100.0%)\n",
      "\n",
      "Total basepairs processed:    75,683,348 bp\n",
      "  Read 1:    37,186,809 bp\n",
      "  Read 2:    38,496,539 bp\n",
      "Total written (filtered):     75,612,983 bp (99.9%)\n",
      "  Read 1:    37,185,339 bp\n",
      "  Read 2:    38,427,644 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 145 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t130\t2602.5\t0\t130\n",
      "20\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t5\t0.0\t1\t5\n",
      "28\t1\t0.0\t1\t1\n",
      "66\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACVSGGGTATCTAAT; Type: regular 5'; Length: 20; Trimmed: 4426 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t2602.5\t0\t17\n",
      "4\t2\t650.6\t0\t2\n",
      "5\t11\t162.7\t0\t11\n",
      "6\t21\t40.7\t0\t21\n",
      "7\t52\t10.2\t0\t52\n",
      "8\t70\t2.5\t0\t70\n",
      "9\t264\t0.6\t0\t262 2\n",
      "10\t859\t0.2\t1\t847 12\n",
      "11\t40\t0.0\t1\t40\n",
      "12\t49\t0.0\t1\t48 1\n",
      "13\t36\t0.0\t1\t34 2\n",
      "14\t103\t0.0\t1\t94 9\n",
      "15\t108\t0.0\t1\t106 2\n",
      "16\t336\t0.0\t1\t322 14\n",
      "17\t575\t0.0\t1\t560 15\n",
      "18\t364\t0.0\t1\t340 20 4\n",
      "19\t1278\t0.0\t1\t825 448 5\n",
      "20\t2\t0.0\t2\t0 2\n",
      "21\t213\t0.0\t2\t0 208 5\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "61\t1\t0.0\t2\t1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "70\t3\t0.0\t2\t3\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "79\t7\t0.0\t2\t5 2\n",
      "161\t1\t0.0\t2\t0 1\n",
      "247\t1\t0.0\t2\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: trim_demux_Muller_SWGtransmissionstudies_70_138.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "\n",
    "!qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences Folder_Muller_SWGtransmissionstudies/demux_Muller_SWGtransmissionstudies_70_138.qza \\\n",
    "--p-cores 16 \\\n",
    "--p-front-f GTGCCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACVSGGGTATCTAAT \\\n",
    "--o-trimmed-sequences trim_demux_Muller_SWGtransmissionstudies_70_138.qza \\\n",
    "--verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: trim_demux_Muller_SWGtransmissionstudies_70_138.qzv\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for f1 in trim_demux_Muller_SWGtransmissionstudies*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime demux summarize  --i-data $f1  --o-visualization $fn.qzv\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_13_207.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_13_207.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Muller_SWGtransmissionstudies/dada_trim_demux_Muller_SWGtransmissionstudies_13_207.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_1_69.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_1_69.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Muller_SWGtransmissionstudies/dada_trim_demux_Muller_SWGtransmissionstudies_1_69.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_208-276.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_208-276.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Muller_SWGtransmissionstudies/dada_trim_demux_Muller_SWGtransmissionstudies_208-276.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_277_236.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_277_236.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Muller_SWGtransmissionstudies/dada_trim_demux_Muller_SWGtransmissionstudies_277_236.qza\n"
     ]
    }
   ],
   "source": [
    "#%%bash\n",
    "\n",
    "#for f1 in Folder_Muller_SWGtransmissionstudies/trim_demux_Muller_SWGtransmissionstudies_*.qza\n",
    "#do\n",
    "#newfile=\"$(basename $f1)\"\n",
    "#fn=${newfile%.qza};\n",
    "\n",
    "#qiime dada2 denoise-paired --i-demultiplexed-seqs $f1 --p-trunc-len-f 220 --p-trunc-len-r  220 --p-n-threads 10 --o-representative-sequences Folder_Muller_SWGtransmissionstudies/rep_$fn.qza --o-table Folder_Muller_SWGtransmissionstudies/table_$fn.qza --o-denoising-stats Folder_Muller_SWGtransmissionstudies/dada_$fn.qza \n",
    "\n",
    "#done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_13_207.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_13_207.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Muller_SWGtransmissionstudies/dada_trim_demux_Muller_SWGtransmissionstudies_13_207.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_1_69.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_1_69.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Muller_SWGtransmissionstudies/dada_trim_demux_Muller_SWGtransmissionstudies_1_69.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_208-276.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_208-276.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Muller_SWGtransmissionstudies/dada_trim_demux_Muller_SWGtransmissionstudies_208-276.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_277_236.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_277_236.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Muller_SWGtransmissionstudies/dada_trim_demux_Muller_SWGtransmissionstudies_277_236.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_70_138.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_70_138.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Muller_SWGtransmissionstudies/dada_trim_demux_Muller_SWGtransmissionstudies_70_138.qza\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for f1 in Folder_Muller_SWGtransmissionstudies/trim_demux_Muller_SWGtransmissionstudies_*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime dada2 denoise-single --i-demultiplexed-seqs $f1 --p-max-ee 3 --p-trunc-len 130 --p-n-threads 10 --o-representative-sequences Folder_Muller_SWGtransmissionstudies/rep_$fn.qza --o-table Folder_Muller_SWGtransmissionstudies/table_$fn.qza --o-denoising-stats Folder_Muller_SWGtransmissionstudies/dada_$fn.qza \n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table merge-seqs \\\n",
    "--i-data Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_13_207.qza \\\n",
    "--i-data Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_1_69.qza \\\n",
    "--i-data Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_208-276.qza \\\n",
    "--i-data Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_277_236.qza \\\n",
    "--i-data Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies_70_138.qza \\\n",
    "--o-merged-data Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_Muller_SWGtransmissionstudies/table_Muller_SWGtransmissionstudies.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table merge \\\n",
    " --i-tables Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_13_207.qza \\\n",
    " --i-tables Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_1_69.qza \\\n",
    " --i-tables Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_208-276.qza \\\n",
    " --i-tables Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_277_236.qza \\\n",
    " --i-tables Folder_Muller_SWGtransmissionstudies/table_trim_demux_Muller_SWGtransmissionstudies_70_138.qza \\\n",
    " --o-merged-table Folder_Muller_SWGtransmissionstudies/table_Muller_SWGtransmissionstudies.qza\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Taxonomy] to: Folder_Muller_SWGtransmissionstudies/taxonomy_Muller_SWGtransmissionstudies.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier /space/home/rosales/databases/SILVA_DBs/silva-138-99-nb-classifier.qza \\\n",
    "  --i-reads Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies.qza \\\n",
    "  --p-n-jobs -7 \\\n",
    "  --o-classification Folder_Muller_SWGtransmissionstudies/taxonomy_Muller_SWGtransmissionstudies.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_Muller_SWGtransmissionstudies/taxonomy_Muller_SWGtransmissionstudies.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: Folder_Muller_SWGtransmissionstudies/taxa-bar-SWGTransmission.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file Folder_Muller_SWGtransmissionstudies/taxonomy_Muller_SWGtransmissionstudies.qza \\\n",
    "  --o-visualization Folder_Muller_SWGtransmissionstudies/taxonomy_Muller_SWGtransmissionstudies.qzv\n",
    "\n",
    "\n",
    "!qiime taxa barplot \\\n",
    "  --i-table Folder_Muller_SWGtransmissionstudies/table-BacArc_Muller_SWGtransmissionstudies.qza \\\n",
    "  --i-taxonomy Folder_Muller_SWGtransmissionstudies/taxonomy_Muller_SWGtransmissionstudies.qza \\\n",
    "  --m-metadata-file  metadata/metadata_Muller_SWGTransmission.txt \\\n",
    "  --o-visualization Folder_Muller_SWGtransmissionstudies/taxa-bar-SWGTransmission.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_Muller_SWGtransmissionstudies/table-BacArc_Muller_SWGtransmissionstudies.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_Muller_SWGtransmissionstudies/rep_seqs_BacArc_Muller_SWGtransmissionstudies.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime taxa filter-table \\\n",
    "  --i-table  Folder_Muller_SWGtransmissionstudies/table_Muller_SWGtransmissionstudies.qza \\\n",
    "  --i-taxonomy  Folder_Muller_SWGtransmissionstudies/taxonomy_Muller_SWGtransmissionstudies.qza \\\n",
    "  --p-exclude mitochondria,chloroplast \\\n",
    " --p-include d__Bacteria,d__Archaea \\\n",
    "--o-filtered-table Folder_Muller_SWGtransmissionstudies/table-BacArc_Muller_SWGtransmissionstudies.qza\n",
    "\n",
    "!qiime taxa filter-seqs \\\n",
    "  --i-sequences  Folder_Muller_SWGtransmissionstudies/rep_trim_demux_Muller_SWGtransmissionstudies.qza \\\n",
    "  --i-taxonomy   Folder_Muller_SWGtransmissionstudies/taxonomy_Muller_SWGtransmissionstudies.qza \\\n",
    "  --p-exclude mitochondria,chloroplast \\\n",
    " --p-include d__Bacteria,d__Archaea \\\n",
    "  --o-filtered-sequences Folder_Muller_SWGtransmissionstudies/rep_seqs_BacArc_Muller_SWGtransmissionstudies.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Usage: \u001b[34mqiime vsearch\u001b[0m [OPTIONS] COMMAND [ARGS]...\r\n",
      "\r\n",
      "  Description: This plugin wraps the vsearch application, and provides\r\n",
      "  methods for clustering and dereplicating features and sequences.\r\n",
      "\r\n",
      "  Plugin website: https://github.com/qiime2/q2-vsearch\r\n",
      "\r\n",
      "  Getting user support: Please post to the QIIME 2 forum for help with this\r\n",
      "  plugin: https://forum.qiime2.org\r\n",
      "\r\n",
      "\u001b[1mOptions\u001b[0m:\r\n",
      "  \u001b[34m--version\u001b[0m    Show the version and exit.\r\n",
      "  \u001b[34m--citations\u001b[0m  Show citations and exit.\r\n",
      "  \u001b[34m--help\u001b[0m       Show this message and exit.\r\n",
      "\r\n",
      "\u001b[1mCommands\u001b[0m:\r\n",
      "  \u001b[34mcluster-features-closed-reference\u001b[0m\r\n",
      "                                  Closed-reference clustering of features.\r\n",
      "  \u001b[34mcluster-features-de-novo\u001b[0m        De novo clustering of features.\r\n",
      "  \u001b[34mcluster-features-open-reference\u001b[0m\r\n",
      "                                  Open-reference clustering of features.\r\n",
      "  \u001b[34mdereplicate-sequences\u001b[0m           Dereplicate sequences.\r\n",
      "  \u001b[34mfastq-stats\u001b[0m                     Fastq stats with vsearch.\r\n",
      "  \u001b[34mjoin-pairs\u001b[0m                      Join paired-end reads.\r\n",
      "  \u001b[34muchime-denovo\u001b[0m                   De novo chimera filtering with vsearch.\r\n",
      "  \u001b[34muchime-ref\u001b[0m                      Reference-based chimera filtering with\r\n",
      "                                  vsearch.\r\n"
     ]
    }
   ],
   "source": [
    "!qiime vsearch --help"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
