{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/rawdata/2018-05-09_resitance_coral_micro/demux\n",
      "/space/home/rosales/projects/2020_SCTLD_meta-analysis\n"
     ]
    }
   ],
   "source": [
    "%cd /space/home/rosales/rawdata/2018-05-09_resitance_coral_micro/demux\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_Rosales_AcropraDis.txt\n",
    "%cd /space/home/rosales/projects/2020_SCTLD_meta-analysis/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as ps\n",
    "import numpy as np\n",
    "\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_Rosales_AcropraDis.txt',\n",
    "                                    names=header_list)\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_S\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "dir_path=\"/space/home/rosales/rawdata/2018-05-09_resitance_coral_micro/demux/\"\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Rosales_AcropraDis.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir Folder_Rosales_AcropraDis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/anaconda2/envs/qiime2-2020.2/bin/qiime\r\n"
     ]
    }
   ],
   "source": [
    "!which qiime"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Rosales_AcropraDis.csv as PairedEndFastqManifestPhred33V2 to Folder_Rosales_AcropraDis/Import_Rosales_AcropraDis.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime tools import \\\n",
    "--output-path Folder_Rosales_AcropraDis/Import_Rosales_AcropraDis.qza \\\n",
    "--type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "--input-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Rosales_AcropraDis.csv \\\n",
    "--input-format PairedEndFastqManifestPhred33V2\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_Rosales_AcropraDis/Import_Rosales_AcropraDis.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data Folder_Rosales_AcropraDis/Import_Rosales_AcropraDis.qza \\\n",
    "--o-visualization Folder_Rosales_AcropraDis/Import_Rosales_AcropraDis.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14iexposed071717L_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14iexposed071717L_96_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14iexposed071717L_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14iexposed071717L_96_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14iexposed071717L_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14iexposed071717L_96_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14iexposed071717L_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14iexposed071717L_96_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        50,752 reads  @     40.1 µs/read;   1.50 M reads/minute\n",
      "Finished in 2.10 s (41 us/read; 1.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,752\n",
      "  Read 1 with adapter:                      36 (0.1%)\n",
      "  Read 2 with adapter:                      19 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,752 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,654,208 bp\n",
      "  Read 1:    15,327,104 bp\n",
      "  Read 2:    15,327,104 bp\n",
      "Total written (filtered):     30,653,389 bp (100.0%)\n",
      "  Read 1:    15,326,995 bp\n",
      "  Read 2:    15,326,394 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 36 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t793.0\t0\t35\n",
      "4\t1\t198.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t793.0\t0\t13\n",
      "4\t2\t198.2\t0\t2\n",
      "6\t1\t12.4\t0\t1\n",
      "219\t3\t0.0\t2\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14jexposed071717L_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14jexposed071717L_97_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14jexposed071717L_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14jexposed071717L_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14jexposed071717L_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14jexposed071717L_97_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14jexposed071717L_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14jexposed071717L_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       161,665 reads  @     28.7 µs/read;   2.09 M reads/minute\n",
      "Finished in 4.67 s (29 us/read; 2.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,665\n",
      "  Read 1 with adapter:                      96 (0.1%)\n",
      "  Read 2 with adapter:                     138 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       161,665 (100.0%)\n",
      "\n",
      "Total basepairs processed:    97,645,660 bp\n",
      "  Read 1:    48,822,830 bp\n",
      "  Read 2:    48,822,830 bp\n",
      "Total written (filtered):     97,633,256 bp (100.0%)\n",
      "  Read 1:    48,822,528 bp\n",
      "  Read 2:    48,810,728 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 96 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t86\t2526.0\t0\t86\n",
      "4\t7\t631.5\t0\t7\n",
      "5\t2\t157.9\t0\t2\n",
      "6\t1\t39.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 138 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t67\t2526.0\t0\t67\n",
      "4\t11\t631.5\t0\t11\n",
      "5\t5\t157.9\t0\t5\n",
      "6\t1\t39.5\t0\t1\n",
      "219\t54\t0.0\t2\t34 20\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14kexposed071717L_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14kexposed071717L_98_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14kexposed071717L_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14kexposed071717L_98_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14kexposed071717L_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C14kexposed071717L_98_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14kexposed071717L_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C14kexposed071717L_98_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       212,300 reads  @     29.5 µs/read;   2.03 M reads/minute\n",
      "Finished in 6.30 s (30 us/read; 2.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            212,300\n",
      "  Read 1 with adapter:                     100 (0.0%)\n",
      "  Read 2 with adapter:                     171 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       212,300 (100.0%)\n",
      "\n",
      "Total basepairs processed:   128,229,200 bp\n",
      "  Read 1:    64,114,600 bp\n",
      "  Read 2:    64,114,600 bp\n",
      "Total written (filtered):    128,210,863 bp (100.0%)\n",
      "  Read 1:    64,114,284 bp\n",
      "  Read 2:    64,096,579 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 100 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t3317.2\t0\t88\n",
      "4\t8\t829.3\t0\t8\n",
      "5\t4\t207.3\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 171 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t3317.2\t0\t77\n",
      "4\t10\t829.3\t0\t10\n",
      "5\t2\t207.3\t0\t2\n",
      "6\t1\t51.8\t0\t1\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t3\t0.0\t2\t1 2\n",
      "219\t77\t0.0\t2\t59 18\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17abaseline070217H_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17abaseline070217H_99_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17abaseline070217H_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17abaseline070217H_99_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17abaseline070217H_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17abaseline070217H_99_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17abaseline070217H_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17abaseline070217H_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       117,036 reads  @     29.8 µs/read;   2.01 M reads/minute\n",
      "Finished in 3.52 s (30 us/read; 1.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,036\n",
      "  Read 1 with adapter:                      75 (0.1%)\n",
      "  Read 2 with adapter:                      56 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       117,036 (100.0%)\n",
      "\n",
      "Total basepairs processed:    70,689,744 bp\n",
      "  Read 1:    35,344,872 bp\n",
      "  Read 2:    35,344,872 bp\n",
      "Total written (filtered):     70,686,100 bp (100.0%)\n",
      "  Read 1:    35,344,645 bp\n",
      "  Read 2:    35,341,455 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 75 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t1828.7\t0\t73\n",
      "4\t2\t457.2\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 56 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t1828.7\t0\t33\n",
      "4\t7\t457.2\t0\t7\n",
      "6\t1\t28.6\t0\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t14\t0.0\t2\t10 4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17aexposed072817H_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17aexposed072817H_100_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17aexposed072817H_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17aexposed072817H_100_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17aexposed072817H_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17aexposed072817H_100_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17aexposed072817H_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17aexposed072817H_100_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:05       188,949 reads  @     30.0 µs/read;   2.00 M reads/minute\n",
      "Finished in 5.70 s (30 us/read; 1.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            188,949\n",
      "  Read 1 with adapter:                     115 (0.1%)\n",
      "  Read 2 with adapter:                     155 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       188,949 (100.0%)\n",
      "\n",
      "Total basepairs processed:   114,125,196 bp\n",
      "  Read 1:    57,062,598 bp\n",
      "  Read 2:    57,062,598 bp\n",
      "Total written (filtered):    114,110,722 bp (100.0%)\n",
      "  Read 1:    57,062,246 bp\n",
      "  Read 2:    57,048,476 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 115 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t2952.3\t0\t109\n",
      "4\t5\t738.1\t0\t5\n",
      "5\t1\t184.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 155 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t2952.3\t0\t73\n",
      "4\t14\t738.1\t0\t14\n",
      "5\t4\t184.5\t0\t4\n",
      "30\t1\t0.0\t2\t0 1\n",
      "219\t63\t0.0\t2\t41 22\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17bexposed072817H_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17bexposed072817H_101_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17bexposed072817H_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17bexposed072817H_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17bexposed072817H_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17bexposed072817H_101_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17bexposed072817H_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17bexposed072817H_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       195,665 reads  @     30.8 µs/read;   1.95 M reads/minute\n",
      "Finished in 6.06 s (31 us/read; 1.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            195,665\n",
      "  Read 1 with adapter:                     103 (0.1%)\n",
      "  Read 2 with adapter:                     117 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       195,665 (100.0%)\n",
      "\n",
      "Total basepairs processed:   118,181,660 bp\n",
      "  Read 1:    59,090,830 bp\n",
      "  Read 2:    59,090,830 bp\n",
      "Total written (filtered):    118,176,424 bp (100.0%)\n",
      "  Read 1:    59,090,510 bp\n",
      "  Read 2:    59,085,914 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t99\t3057.3\t0\t99\n",
      "5\t2\t191.1\t0\t2\n",
      "6\t1\t47.8\t0\t1\n",
      "7\t1\t11.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 117 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t74\t3057.3\t0\t74\n",
      "4\t17\t764.3\t0\t17\n",
      "5\t2\t191.1\t0\t2\n",
      "6\t3\t47.8\t0\t3\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t20\t0.0\t2\t12 8\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17cexposed072817H_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17cexposed072817H_102_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17cexposed072817H_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17cexposed072817H_102_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17cexposed072817H_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17cexposed072817H_102_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17cexposed072817H_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17cexposed072817H_102_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8=-------] 00:00:07       275,254 reads  @     28.4 µs/read;   2.11 M reads/minute\n",
      "Finished in 7.85 s (29 us/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            275,254\n",
      "  Read 1 with adapter:                     133 (0.0%)\n",
      "  Read 2 with adapter:                     179 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       275,254 (100.0%)\n",
      "\n",
      "Total basepairs processed:   166,253,416 bp\n",
      "  Read 1:    83,126,708 bp\n",
      "  Read 2:    83,126,708 bp\n",
      "Total written (filtered):    166,239,046 bp (100.0%)\n",
      "  Read 1:    83,126,293 bp\n",
      "  Read 2:    83,112,753 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 133 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t120\t4300.8\t0\t120\n",
      "4\t10\t1075.2\t0\t10\n",
      "5\t3\t268.8\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 179 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t95\t4300.8\t0\t95\n",
      "4\t16\t1075.2\t0\t16\n",
      "5\t5\t268.8\t0\t5\n",
      "6\t1\t67.2\t0\t1\n",
      "218\t3\t0.0\t2\t2 1\n",
      "219\t59\t0.0\t2\t46 13\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17dbaseline070217H_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17dbaseline070217H_103_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17dbaseline070217H_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17dbaseline070217H_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17dbaseline070217H_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17dbaseline070217H_103_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17dbaseline070217H_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17dbaseline070217H_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       216,802 reads  @     28.0 µs/read;   2.14 M reads/minute\n",
      "Finished in 6.11 s (28 us/read; 2.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            216,802\n",
      "  Read 1 with adapter:                      73 (0.0%)\n",
      "  Read 2 with adapter:                      99 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       216,802 (100.0%)\n",
      "\n",
      "Total basepairs processed:   130,948,408 bp\n",
      "  Read 1:    65,474,204 bp\n",
      "  Read 2:    65,474,204 bp\n",
      "Total written (filtered):    130,947,850 bp (100.0%)\n",
      "  Read 1:    65,473,970 bp\n",
      "  Read 2:    65,473,880 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 73 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t64\t3387.5\t0\t64\n",
      "4\t7\t846.9\t0\t7\n",
      "5\t1\t211.7\t0\t1\n",
      "9\t1\t0.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 99 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t83\t3387.5\t0\t83\n",
      "4\t10\t846.9\t0\t10\n",
      "5\t3\t211.7\t0\t3\n",
      "6\t1\t52.9\t0\t1\n",
      "7\t2\t13.2\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17eexposed072817H_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17eexposed072817H_104_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17eexposed072817H_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17eexposed072817H_104_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17eexposed072817H_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C17eexposed072817H_104_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17eexposed072817H_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C17eexposed072817H_104_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:07       259,911 reads  @     29.0 µs/read;   2.07 M reads/minute\n",
      "Finished in 7.58 s (29 us/read; 2.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            259,911\n",
      "  Read 1 with adapter:                     176 (0.1%)\n",
      "  Read 2 with adapter:                     201 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       259,911 (100.0%)\n",
      "\n",
      "Total basepairs processed:   156,986,244 bp\n",
      "  Read 1:    78,493,122 bp\n",
      "  Read 2:    78,493,122 bp\n",
      "Total written (filtered):    156,972,453 bp (100.0%)\n",
      "  Read 1:    78,492,518 bp\n",
      "  Read 2:    78,479,935 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 176 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t157\t4061.1\t0\t157\n",
      "4\t13\t1015.3\t0\t13\n",
      "5\t4\t253.8\t0\t4\n",
      "6\t1\t63.5\t0\t1\n",
      "55\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 201 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t114\t4061.1\t0\t114\n",
      "4\t19\t1015.3\t0\t19\n",
      "5\t6\t253.8\t0\t6\n",
      "6\t3\t63.5\t0\t3\n",
      "20\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t57\t0.0\t2\t46 11\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18abaseline070217M_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18abaseline070217M_105_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18abaseline070217M_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18abaseline070217M_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18abaseline070217M_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18abaseline070217M_105_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18abaseline070217M_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18abaseline070217M_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8=-------] 00:00:07       279,539 reads  @     27.8 µs/read;   2.16 M reads/minute\n",
      "Finished in 7.80 s (28 us/read; 2.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            279,539\n",
      "  Read 1 with adapter:                     123 (0.0%)\n",
      "  Read 2 with adapter:                     169 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       279,539 (100.0%)\n",
      "\n",
      "Total basepairs processed:   168,841,556 bp\n",
      "  Read 1:    84,420,778 bp\n",
      "  Read 2:    84,420,778 bp\n",
      "Total written (filtered):    168,830,273 bp (100.0%)\n",
      "  Read 1:    84,420,391 bp\n",
      "  Read 2:    84,409,882 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 123 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t110\t4367.8\t0\t110\n",
      "4\t9\t1091.9\t0\t9\n",
      "5\t3\t273.0\t0\t3\n",
      "6\t1\t68.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 169 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t106\t4367.8\t0\t106\n",
      "4\t11\t1091.9\t0\t11\n",
      "5\t3\t273.0\t0\t3\n",
      "7\t1\t17.1\t0\t1\n",
      "219\t48\t0.0\t2\t31 17\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18cbaseline070217M_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18cbaseline070217M_106_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18cbaseline070217M_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18cbaseline070217M_106_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18cbaseline070217M_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18cbaseline070217M_106_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18cbaseline070217M_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18cbaseline070217M_106_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:07       255,933 reads  @     29.6 µs/read;   2.03 M reads/minute\n",
      "Finished in 7.62 s (30 us/read; 2.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            255,933\n",
      "  Read 1 with adapter:                     124 (0.0%)\n",
      "  Read 2 with adapter:                     198 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       255,933 (100.0%)\n",
      "\n",
      "Total basepairs processed:   154,583,532 bp\n",
      "  Read 1:    77,291,766 bp\n",
      "  Read 2:    77,291,766 bp\n",
      "Total written (filtered):    154,567,803 bp (100.0%)\n",
      "  Read 1:    77,291,382 bp\n",
      "  Read 2:    77,276,421 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 124 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t116\t3999.0\t0\t116\n",
      "4\t5\t999.7\t0\t5\n",
      "5\t2\t249.9\t0\t2\n",
      "6\t1\t62.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 198 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t100\t3999.0\t0\t100\n",
      "4\t25\t999.7\t0\t25\n",
      "5\t1\t249.9\t0\t1\n",
      "6\t3\t62.5\t0\t3\n",
      "30\t1\t0.0\t2\t1\n",
      "219\t68\t0.0\t2\t49 19\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18cexposed072817M_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18cexposed072817M_107_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18cexposed072817M_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18cexposed072817M_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18cexposed072817M_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18cexposed072817M_107_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18cexposed072817M_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18cexposed072817M_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8=-------] 00:00:08       297,850 reads  @     28.4 µs/read;   2.11 M reads/minute\n",
      "Finished in 8.51 s (29 us/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            297,850\n",
      "  Read 1 with adapter:                     126 (0.0%)\n",
      "  Read 2 with adapter:                     181 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       297,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:   179,901,400 bp\n",
      "  Read 1:    89,950,700 bp\n",
      "  Read 2:    89,950,700 bp\n",
      "Total written (filtered):    179,883,819 bp (100.0%)\n",
      "  Read 1:    89,950,314 bp\n",
      "  Read 2:    89,933,505 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 126 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t121\t4653.9\t0\t121\n",
      "4\t3\t1163.5\t0\t3\n",
      "5\t1\t290.9\t0\t1\n",
      "6\t1\t72.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 181 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t90\t4653.9\t0\t90\n",
      "4\t7\t1163.5\t0\t7\n",
      "5\t5\t290.9\t0\t5\n",
      "6\t2\t72.7\t0\t2\n",
      "217\t1\t0.0\t2\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t75\t0.0\t2\t53 22\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18dexposed072817M_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18dexposed072817M_108_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18dexposed072817M_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18dexposed072817M_108_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18dexposed072817M_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18dexposed072817M_108_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18dexposed072817M_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18dexposed072817M_108_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       167,510 reads  @     28.0 µs/read;   2.14 M reads/minute\n",
      "Finished in 4.72 s (28 us/read; 2.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            167,510\n",
      "  Read 1 with adapter:                      95 (0.1%)\n",
      "  Read 2 with adapter:                     133 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       167,510 (100.0%)\n",
      "\n",
      "Total basepairs processed:   101,176,040 bp\n",
      "  Read 1:    50,588,020 bp\n",
      "  Read 2:    50,588,020 bp\n",
      "Total written (filtered):    101,164,306 bp (100.0%)\n",
      "  Read 1:    50,587,722 bp\n",
      "  Read 2:    50,576,584 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 95 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t86\t2617.3\t0\t86\n",
      "4\t5\t654.3\t0\t5\n",
      "5\t4\t163.6\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 133 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t65\t2617.3\t0\t65\n",
      "4\t14\t654.3\t0\t14\n",
      "5\t1\t163.6\t0\t1\n",
      "6\t1\t40.9\t0\t1\n",
      "7\t1\t10.2\t0\t1\n",
      "218\t2\t0.0\t2\t1 1\n",
      "219\t49\t0.0\t2\t39 10\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18eexposed072817M_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18eexposed072817M_109_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18eexposed072817M_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18eexposed072817M_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18eexposed072817M_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C18eexposed072817M_109_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18eexposed072817M_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C18eexposed072817M_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[   8=-------] 00:00:08       259,163 reads  @     31.4 µs/read;   1.91 M reads/minute\n",
      "Finished in 8.17 s (32 us/read; 1.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            259,163\n",
      "  Read 1 with adapter:                     151 (0.1%)\n",
      "  Read 2 with adapter:                     120 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       259,163 (100.0%)\n",
      "\n",
      "Total basepairs processed:   156,534,452 bp\n",
      "  Read 1:    78,267,226 bp\n",
      "  Read 2:    78,267,226 bp\n",
      "Total written (filtered):    156,531,662 bp (100.0%)\n",
      "  Read 1:    78,266,764 bp\n",
      "  Read 2:    78,264,898 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 151 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t144\t4049.4\t0\t144\n",
      "4\t5\t1012.4\t0\t5\n",
      "5\t2\t253.1\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 120 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t4049.4\t0\t91\n",
      "4\t17\t1012.4\t0\t17\n",
      "5\t2\t253.1\t0\t2\n",
      "6\t1\t63.3\t0\t1\n",
      "219\t9\t0.0\t2\t5 4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20abaseline070217M_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20abaseline070217M_110_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20abaseline070217M_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20abaseline070217M_110_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20abaseline070217M_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20abaseline070217M_110_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20abaseline070217M_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20abaseline070217M_110_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8=-------] 00:00:07       272,155 reads  @     28.9 µs/read;   2.07 M reads/minute\n",
      "Finished in 7.91 s (29 us/read; 2.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            272,155\n",
      "  Read 1 with adapter:                     139 (0.1%)\n",
      "  Read 2 with adapter:                     162 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       272,155 (100.0%)\n",
      "\n",
      "Total basepairs processed:   164,381,620 bp\n",
      "  Read 1:    82,190,810 bp\n",
      "  Read 2:    82,190,810 bp\n",
      "Total written (filtered):    164,369,190 bp (100.0%)\n",
      "  Read 1:    82,190,380 bp\n",
      "  Read 2:    82,178,810 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 139 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t131\t4252.4\t0\t131\n",
      "4\t5\t1063.1\t0\t5\n",
      "5\t1\t265.8\t0\t1\n",
      "6\t2\t66.4\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 162 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t4252.4\t0\t85\n",
      "4\t14\t1063.1\t0\t14\n",
      "5\t6\t265.8\t0\t6\n",
      "6\t1\t66.4\t0\t1\n",
      "7\t1\t16.6\t0\t1\n",
      "8\t1\t4.2\t0\t1\n",
      "32\t1\t0.0\t2\t0 1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t52\t0.0\t2\t38 14\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20bbaseline070217M_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20bbaseline070217M_111_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20bbaseline070217M_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20bbaseline070217M_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20bbaseline070217M_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20bbaseline070217M_111_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20bbaseline070217M_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20bbaseline070217M_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       209,989 reads  @     30.5 µs/read;   1.97 M reads/minute\n",
      "Finished in 6.45 s (31 us/read; 1.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            209,989\n",
      "  Read 1 with adapter:                      35 (0.0%)\n",
      "  Read 2 with adapter:                      62 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       209,989 (100.0%)\n",
      "\n",
      "Total basepairs processed:   126,833,356 bp\n",
      "  Read 1:    63,416,678 bp\n",
      "  Read 2:    63,416,678 bp\n",
      "Total written (filtered):    126,833,044 bp (100.0%)\n",
      "  Read 1:    63,416,558 bp\n",
      "  Read 2:    63,416,486 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 35 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t3281.1\t0\t29\n",
      "4\t4\t820.3\t0\t4\n",
      "7\t1\t12.8\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 62 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t3281.1\t0\t56\n",
      "4\t6\t820.3\t0\t6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20bexposed072817M_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20bexposed072817M_112_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20bexposed072817M_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20bexposed072817M_112_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20bexposed072817M_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20bexposed072817M_112_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20bexposed072817M_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20bexposed072817M_112_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8=-------] 00:00:08       268,900 reads  @     32.2 µs/read;   1.86 M reads/minute\n",
      "Finished in 8.69 s (32 us/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            268,900\n",
      "  Read 1 with adapter:                     109 (0.0%)\n",
      "  Read 2 with adapter:                     127 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       268,900 (100.0%)\n",
      "\n",
      "Total basepairs processed:   162,415,600 bp\n",
      "  Read 1:    81,207,800 bp\n",
      "  Read 2:    81,207,800 bp\n",
      "Total written (filtered):    162,414,394 bp (100.0%)\n",
      "  Read 1:    81,207,450 bp\n",
      "  Read 2:    81,206,944 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t99\t4201.6\t0\t99\n",
      "4\t7\t1050.4\t0\t7\n",
      "5\t1\t262.6\t0\t1\n",
      "10\t2\t0.3\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 127 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t100\t4201.6\t0\t100\n",
      "4\t16\t1050.4\t0\t16\n",
      "5\t7\t262.6\t0\t7\n",
      "6\t1\t65.6\t0\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "219\t2\t0.0\t2\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20cexposed072817M_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20cexposed072817M_113_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20cexposed072817M_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20cexposed072817M_113_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20cexposed072817M_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20cexposed072817M_113_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20cexposed072817M_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20cexposed072817M_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       213,699 reads  @     28.8 µs/read;   2.08 M reads/minute\n",
      "Finished in 6.18 s (29 us/read; 2.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            213,699\n",
      "  Read 1 with adapter:                     110 (0.1%)\n",
      "  Read 2 with adapter:                     142 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       213,699 (100.0%)\n",
      "\n",
      "Total basepairs processed:   129,074,196 bp\n",
      "  Read 1:    64,537,098 bp\n",
      "  Read 2:    64,537,098 bp\n",
      "Total written (filtered):    129,064,978 bp (100.0%)\n",
      "  Read 1:    64,536,754 bp\n",
      "  Read 2:    64,528,224 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t100\t3339.0\t0\t100\n",
      "4\t7\t834.8\t0\t7\n",
      "5\t2\t208.7\t0\t2\n",
      "6\t1\t52.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 142 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t84\t3339.0\t0\t84\n",
      "4\t15\t834.8\t0\t15\n",
      "5\t2\t208.7\t0\t2\n",
      "6\t2\t52.2\t0\t2\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t38\t0.0\t2\t31 7\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20dbaseline070217M_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20dbaseline070217M_114_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20dbaseline070217M_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20dbaseline070217M_114_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20dbaseline070217M_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20dbaseline070217M_114_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20dbaseline070217M_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20dbaseline070217M_114_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       174,371 reads  @     27.3 µs/read;   2.20 M reads/minute\n",
      "Finished in 4.80 s (28 us/read; 2.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            174,371\n",
      "  Read 1 with adapter:                     111 (0.1%)\n",
      "  Read 2 with adapter:                     121 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       174,371 (100.0%)\n",
      "\n",
      "Total basepairs processed:   105,320,084 bp\n",
      "  Read 1:    52,660,042 bp\n",
      "  Read 2:    52,660,042 bp\n",
      "Total written (filtered):    105,313,523 bp (100.0%)\n",
      "  Read 1:    52,659,703 bp\n",
      "  Read 2:    52,653,820 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 111 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t107\t2724.5\t0\t107\n",
      "4\t3\t681.1\t0\t3\n",
      "6\t1\t42.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 121 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t2724.5\t0\t77\n",
      "4\t10\t681.1\t0\t10\n",
      "5\t3\t170.3\t0\t3\n",
      "6\t3\t42.6\t0\t3\n",
      "7\t1\t10.6\t0\t1\n",
      "218\t2\t0.0\t2\t0 2\n",
      "219\t25\t0.0\t2\t19 6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20dexposed072817M_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20dexposed072817M_115_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20dexposed072817M_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20dexposed072817M_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20dexposed072817M_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20dexposed072817M_115_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20dexposed072817M_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20dexposed072817M_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:05       166,582 reads  @     32.4 µs/read;   1.85 M reads/minute\n",
      "Finished in 5.44 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            166,582\n",
      "  Read 1 with adapter:                      65 (0.0%)\n",
      "  Read 2 with adapter:                      53 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       166,582 (100.0%)\n",
      "\n",
      "Total basepairs processed:   100,615,528 bp\n",
      "  Read 1:    50,307,764 bp\n",
      "  Read 2:    50,307,764 bp\n",
      "Total written (filtered):    100,614,727 bp (100.0%)\n",
      "  Read 1:    50,307,562 bp\n",
      "  Read 2:    50,307,165 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 65 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t2602.8\t0\t58\n",
      "4\t7\t650.7\t0\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 53 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t2602.8\t0\t46\n",
      "4\t3\t650.7\t0\t3\n",
      "5\t1\t162.7\t0\t1\n",
      "6\t1\t40.7\t0\t1\n",
      "219\t2\t0.0\t2\t0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20eexposed072817M_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20eexposed072817M_116_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20eexposed072817M_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20eexposed072817M_116_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20eexposed072817M_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C20eexposed072817M_116_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20eexposed072817M_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C20eexposed072817M_116_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:06       223,086 reads  @     28.5 µs/read;   2.10 M reads/minute\n",
      "Finished in 6.40 s (29 us/read; 2.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            223,086\n",
      "  Read 1 with adapter:                     124 (0.1%)\n",
      "  Read 2 with adapter:                     210 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       223,086 (100.0%)\n",
      "\n",
      "Total basepairs processed:   134,743,944 bp\n",
      "  Read 1:    67,371,972 bp\n",
      "  Read 2:    67,371,972 bp\n",
      "Total written (filtered):    134,727,989 bp (100.0%)\n",
      "  Read 1:    67,371,589 bp\n",
      "  Read 2:    67,356,400 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 124 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t113\t3485.7\t0\t113\n",
      "4\t11\t871.4\t0\t11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 210 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t114\t3485.7\t0\t114\n",
      "4\t19\t871.4\t0\t19\n",
      "5\t6\t217.9\t0\t6\n",
      "6\t1\t54.5\t0\t1\n",
      "8\t1\t3.4\t0\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t68\t0.0\t2\t50 18\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21abaseline070217H_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21abaseline070217H_117_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21abaseline070217H_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21abaseline070217H_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21abaseline070217H_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21abaseline070217H_117_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21abaseline070217H_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21abaseline070217H_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       131,310 reads  @     29.4 µs/read;   2.04 M reads/minute\n",
      "Finished in 3.90 s (30 us/read; 2.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            131,310\n",
      "  Read 1 with adapter:                      76 (0.1%)\n",
      "  Read 2 with adapter:                     102 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       131,310 (100.0%)\n",
      "\n",
      "Total basepairs processed:    79,311,240 bp\n",
      "  Read 1:    39,655,620 bp\n",
      "  Read 2:    39,655,620 bp\n",
      "Total written (filtered):     79,300,519 bp (100.0%)\n",
      "  Read 1:    39,655,370 bp\n",
      "  Read 2:    39,645,149 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 76 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t2051.7\t0\t72\n",
      "4\t3\t512.9\t0\t3\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 102 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t2051.7\t0\t45\n",
      "4\t6\t512.9\t0\t6\n",
      "5\t4\t128.2\t0\t4\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t46\t0.0\t2\t26 20\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21cbaseline070217H_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21cbaseline070217H_118_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21cbaseline070217H_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21cbaseline070217H_118_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21cbaseline070217H_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21cbaseline070217H_118_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21cbaseline070217H_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21cbaseline070217H_118_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[       8<---] 00:00:15       571,913 reads  @     27.1 µs/read;   2.22 M reads/minute\n",
      "Finished in 15.53 s (27 us/read; 2.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            571,913\n",
      "  Read 1 with adapter:                     270 (0.0%)\n",
      "  Read 2 with adapter:                     362 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       571,913 (100.0%)\n",
      "\n",
      "Total basepairs processed:   345,435,452 bp\n",
      "  Read 1:   172,717,726 bp\n",
      "  Read 2:   172,717,726 bp\n",
      "Total written (filtered):    345,407,113 bp (100.0%)\n",
      "  Read 1:   172,716,895 bp\n",
      "  Read 2:   172,690,218 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 270 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t256\t8936.1\t0\t256\n",
      "4\t10\t2234.0\t0\t10\n",
      "5\t2\t558.5\t0\t2\n",
      "6\t1\t139.6\t0\t1\n",
      "7\t1\t34.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 362 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t196\t8936.1\t0\t196\n",
      "4\t31\t2234.0\t0\t31\n",
      "5\t6\t558.5\t0\t6\n",
      "6\t6\t139.6\t0\t6\n",
      "13\t1\t0.0\t1\t0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t121\t0.0\t2\t83 38\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21cexposed072817H_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21cexposed072817H_119_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21cexposed072817H_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21cexposed072817H_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21cexposed072817H_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21cexposed072817H_119_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21cexposed072817H_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21cexposed072817H_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       163,849 reads  @     31.8 µs/read;   1.88 M reads/minute\n",
      "Finished in 5.25 s (32 us/read; 1.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            163,849\n",
      "  Read 1 with adapter:                      90 (0.1%)\n",
      "  Read 2 with adapter:                     155 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       163,849 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,964,796 bp\n",
      "  Read 1:    49,482,398 bp\n",
      "  Read 2:    49,482,398 bp\n",
      "Total written (filtered):     98,952,988 bp (100.0%)\n",
      "  Read 1:    49,482,112 bp\n",
      "  Read 2:    49,470,876 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t82\t2560.1\t0\t82\n",
      "4\t3\t640.0\t0\t3\n",
      "5\t3\t160.0\t0\t3\n",
      "6\t1\t40.0\t0\t1\n",
      "7\t1\t10.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 155 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t75\t2560.1\t0\t75\n",
      "4\t19\t640.0\t0\t19\n",
      "5\t8\t160.0\t0\t8\n",
      "6\t1\t40.0\t0\t1\n",
      "8\t1\t2.5\t0\t1\n",
      "218\t2\t0.0\t2\t1 1\n",
      "219\t49\t0.0\t2\t37 12\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21dbaseline070217H_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21dbaseline070217H_120_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21dbaseline070217H_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21dbaseline070217H_120_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21dbaseline070217H_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C21dbaseline070217H_120_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21dbaseline070217H_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C21dbaseline070217H_120_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       187,917 reads  @     27.5 µs/read;   2.19 M reads/minute\n",
      "Finished in 5.29 s (28 us/read; 2.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            187,917\n",
      "  Read 1 with adapter:                      98 (0.1%)\n",
      "  Read 2 with adapter:                     190 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       187,917 (100.0%)\n",
      "\n",
      "Total basepairs processed:   113,501,868 bp\n",
      "  Read 1:    56,750,934 bp\n",
      "  Read 2:    56,750,934 bp\n",
      "Total written (filtered):    113,478,734 bp (100.0%)\n",
      "  Read 1:    56,750,634 bp\n",
      "  Read 2:    56,728,100 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 98 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t93\t2936.2\t0\t93\n",
      "4\t4\t734.1\t0\t4\n",
      "5\t1\t183.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 190 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t70\t2936.2\t0\t70\n",
      "4\t15\t734.1\t0\t15\n",
      "5\t1\t183.5\t0\t1\n",
      "6\t1\t45.9\t0\t1\n",
      "218\t4\t0.0\t2\t2 2\n",
      "219\t99\t0.0\t2\t77 22\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22abaseline070217_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22abaseline070217_121_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22abaseline070217_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22abaseline070217_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22abaseline070217_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22abaseline070217_121_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22abaseline070217_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22abaseline070217_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       135,082 reads  @     31.6 µs/read;   1.90 M reads/minute\n",
      "Finished in 4.31 s (32 us/read; 1.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            135,082\n",
      "  Read 1 with adapter:                     113 (0.1%)\n",
      "  Read 2 with adapter:                     156 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       135,082 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,589,528 bp\n",
      "  Read 1:    40,794,764 bp\n",
      "  Read 2:    40,794,764 bp\n",
      "Total written (filtered):     81,577,222 bp (100.0%)\n",
      "  Read 1:    40,794,411 bp\n",
      "  Read 2:    40,782,811 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t102\t2110.7\t0\t102\n",
      "4\t9\t527.7\t0\t9\n",
      "5\t1\t131.9\t0\t1\n",
      "6\t1\t33.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 156 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t83\t2110.7\t0\t83\n",
      "4\t15\t527.7\t0\t15\n",
      "5\t4\t131.9\t0\t4\n",
      "19\t1\t0.0\t1\t1\n",
      "218\t2\t0.0\t2\t0 2\n",
      "219\t51\t0.0\t2\t34 17\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22cexposed072817H_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22cexposed072817H_122_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22cexposed072817H_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22cexposed072817H_122_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22cexposed072817H_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22cexposed072817H_122_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22cexposed072817H_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22cexposed072817H_122_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       142,064 reads  @     29.1 µs/read;   2.06 M reads/minute\n",
      "Finished in 4.17 s (29 us/read; 2.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,064\n",
      "  Read 1 with adapter:                      67 (0.0%)\n",
      "  Read 2 with adapter:                      93 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       142,064 (100.0%)\n",
      "\n",
      "Total basepairs processed:    85,806,656 bp\n",
      "  Read 1:    42,903,328 bp\n",
      "  Read 2:    42,903,328 bp\n",
      "Total written (filtered):     85,797,724 bp (100.0%)\n",
      "  Read 1:    42,903,117 bp\n",
      "  Read 2:    42,894,607 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 67 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t64\t2219.8\t0\t64\n",
      "4\t1\t554.9\t0\t1\n",
      "5\t1\t138.7\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 93 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t41\t2219.8\t0\t41\n",
      "4\t8\t554.9\t0\t8\n",
      "5\t5\t138.7\t0\t5\n",
      "219\t39\t0.0\t2\t25 14\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22ebaseline070217H_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22ebaseline070217H_123_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22ebaseline070217H_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22ebaseline070217H_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22ebaseline070217H_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22ebaseline070217H_123_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22ebaseline070217H_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22ebaseline070217H_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:05       178,975 reads  @     31.9 µs/read;   1.88 M reads/minute\n",
      "Finished in 5.75 s (32 us/read; 1.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            178,975\n",
      "  Read 1 with adapter:                      93 (0.1%)\n",
      "  Read 2 with adapter:                     128 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       178,975 (100.0%)\n",
      "\n",
      "Total basepairs processed:   108,100,900 bp\n",
      "  Read 1:    54,050,450 bp\n",
      "  Read 2:    54,050,450 bp\n",
      "Total written (filtered):    108,087,656 bp (100.0%)\n",
      "  Read 1:    54,050,132 bp\n",
      "  Read 2:    54,037,524 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 93 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t2796.5\t0\t85\n",
      "4\t6\t699.1\t0\t6\n",
      "10\t1\t0.2\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 128 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t2796.5\t0\t59\n",
      "4\t8\t699.1\t0\t8\n",
      "5\t2\t174.8\t0\t2\n",
      "6\t1\t43.7\t0\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t57\t0.0\t2\t40 17\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22eexposed072817H_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22eexposed072817H_124_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22eexposed072817H_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22eexposed072817H_124_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22eexposed072817H_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C22eexposed072817H_124_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22eexposed072817H_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C22eexposed072817H_124_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       130,595 reads  @     32.0 µs/read;   1.87 M reads/minute\n",
      "Finished in 4.22 s (32 us/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,595\n",
      "  Read 1 with adapter:                      88 (0.1%)\n",
      "  Read 2 with adapter:                      75 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       130,595 (100.0%)\n",
      "\n",
      "Total basepairs processed:    78,879,380 bp\n",
      "  Read 1:    39,439,690 bp\n",
      "  Read 2:    39,439,690 bp\n",
      "Total written (filtered):     78,875,408 bp (100.0%)\n",
      "  Read 1:    39,439,411 bp\n",
      "  Read 2:    39,435,997 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 88 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t2040.5\t0\t79\n",
      "4\t5\t510.1\t0\t5\n",
      "5\t2\t127.5\t0\t2\n",
      "6\t2\t31.9\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 75 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t2040.5\t0\t48\n",
      "4\t10\t510.1\t0\t10\n",
      "5\t1\t127.5\t0\t1\n",
      "219\t16\t0.0\t2\t12 4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24aexposed072817L_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24aexposed072817L_125_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24aexposed072817L_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24aexposed072817L_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24aexposed072817L_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24aexposed072817L_125_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24aexposed072817L_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24aexposed072817L_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        96,147 reads  @     35.1 µs/read;   1.71 M reads/minute\n",
      "Finished in 3.41 s (36 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,147\n",
      "  Read 1 with adapter:                      24 (0.0%)\n",
      "  Read 2 with adapter:                      33 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        96,147 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,072,788 bp\n",
      "  Read 1:    29,036,394 bp\n",
      "  Read 2:    29,036,394 bp\n",
      "Total written (filtered):     58,072,610 bp (100.0%)\n",
      "  Read 1:    29,036,319 bp\n",
      "  Read 2:    29,036,291 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 24 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t1502.3\t0\t22\n",
      "4\t1\t375.6\t0\t1\n",
      "5\t1\t93.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 33 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1502.3\t0\t29\n",
      "4\t4\t375.6\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24bexposed072817L_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24bexposed072817L_126_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24bexposed072817L_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24bexposed072817L_126_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24bexposed072817L_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24bexposed072817L_126_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24bexposed072817L_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24bexposed072817L_126_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:05       203,501 reads  @     27.3 µs/read;   2.20 M reads/minute\n",
      "Finished in 5.59 s (27 us/read; 2.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            203,501\n",
      "  Read 1 with adapter:                      95 (0.0%)\n",
      "  Read 2 with adapter:                     142 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       203,501 (100.0%)\n",
      "\n",
      "Total basepairs processed:   122,914,604 bp\n",
      "  Read 1:    61,457,302 bp\n",
      "  Read 2:    61,457,302 bp\n",
      "Total written (filtered):    122,906,701 bp (100.0%)\n",
      "  Read 1:    61,457,000 bp\n",
      "  Read 2:    61,449,701 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 95 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t89\t3179.7\t0\t89\n",
      "4\t3\t794.9\t0\t3\n",
      "5\t1\t198.7\t0\t1\n",
      "9\t2\t0.8\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 142 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t95\t3179.7\t0\t95\n",
      "4\t7\t794.9\t0\t7\n",
      "5\t4\t198.7\t0\t4\n",
      "6\t2\t49.7\t0\t2\n",
      "30\t1\t0.0\t2\t0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t32\t0.0\t2\t21 11\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24cbaseline070217L_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24cbaseline070217L_127_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24cbaseline070217L_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24cbaseline070217L_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24cbaseline070217L_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24cbaseline070217L_127_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24cbaseline070217L_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24cbaseline070217L_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:05       178,668 reads  @     32.8 µs/read;   1.83 M reads/minute\n",
      "Finished in 5.89 s (33 us/read; 1.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            178,668\n",
      "  Read 1 with adapter:                     128 (0.1%)\n",
      "  Read 2 with adapter:                     183 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       178,668 (100.0%)\n",
      "\n",
      "Total basepairs processed:   107,915,472 bp\n",
      "  Read 1:    53,957,736 bp\n",
      "  Read 2:    53,957,736 bp\n",
      "Total written (filtered):    107,900,847 bp (100.0%)\n",
      "  Read 1:    53,957,302 bp\n",
      "  Read 2:    53,943,545 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 128 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t117\t2791.7\t0\t117\n",
      "4\t5\t697.9\t0\t5\n",
      "5\t2\t174.5\t0\t2\n",
      "6\t1\t43.6\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 183 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t95\t2791.7\t0\t95\n",
      "4\t17\t697.9\t0\t17\n",
      "5\t6\t174.5\t0\t6\n",
      "6\t2\t43.6\t0\t2\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t62\t0.0\t2\t52 10\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24dbaseline070217L_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24dbaseline070217L_128_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24dbaseline070217L_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24dbaseline070217L_128_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24dbaseline070217L_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24dbaseline070217L_128_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24dbaseline070217L_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24dbaseline070217L_128_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8=-------] 00:00:07       276,058 reads  @     28.4 µs/read;   2.11 M reads/minute\n",
      "Finished in 7.87 s (29 us/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            276,058\n",
      "  Read 1 with adapter:                     124 (0.0%)\n",
      "  Read 2 with adapter:                     201 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       276,058 (100.0%)\n",
      "\n",
      "Total basepairs processed:   166,739,032 bp\n",
      "  Read 1:    83,369,516 bp\n",
      "  Read 2:    83,369,516 bp\n",
      "Total written (filtered):    166,718,156 bp (100.0%)\n",
      "  Read 1:    83,369,128 bp\n",
      "  Read 2:    83,349,028 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 124 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t117\t4313.4\t0\t117\n",
      "4\t3\t1078.4\t0\t3\n",
      "5\t2\t269.6\t0\t2\n",
      "6\t1\t67.4\t0\t1\n",
      "9\t1\t1.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 201 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t95\t4313.4\t0\t95\n",
      "4\t11\t1078.4\t0\t11\n",
      "5\t3\t269.6\t0\t3\n",
      "218\t4\t0.0\t2\t2 2\n",
      "219\t88\t0.0\t2\t58 30\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24dexposed072817L_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24dexposed072817L_129_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24dexposed072817L_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24dexposed072817L_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24dexposed072817L_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24dexposed072817L_129_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24dexposed072817L_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24dexposed072817L_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       139,585 reads  @     29.6 µs/read;   2.03 M reads/minute\n",
      "Finished in 4.17 s (30 us/read; 2.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            139,585\n",
      "  Read 1 with adapter:                      46 (0.0%)\n",
      "  Read 2 with adapter:                      70 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       139,585 (100.0%)\n",
      "\n",
      "Total basepairs processed:    84,309,340 bp\n",
      "  Read 1:    42,154,670 bp\n",
      "  Read 2:    42,154,670 bp\n",
      "Total written (filtered):     84,299,487 bp (100.0%)\n",
      "  Read 1:    42,154,532 bp\n",
      "  Read 2:    42,144,955 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 46 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t2181.0\t0\t46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 70 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t2181.0\t0\t25\n",
      "4\t1\t545.3\t0\t1\n",
      "219\t44\t0.0\t2\t34 10\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24eexposed072817L_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24eexposed072817L_130_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24eexposed072817L_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24eexposed072817L_130_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24eexposed072817L_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C24eexposed072817L_130_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24eexposed072817L_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C24eexposed072817L_130_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:07       263,739 reads  @     28.2 µs/read;   2.12 M reads/minute\n",
      "Finished in 7.49 s (28 us/read; 2.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            263,739\n",
      "  Read 1 with adapter:                     113 (0.0%)\n",
      "  Read 2 with adapter:                     173 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       263,739 (100.0%)\n",
      "\n",
      "Total basepairs processed:   159,298,356 bp\n",
      "  Read 1:    79,649,178 bp\n",
      "  Read 2:    79,649,178 bp\n",
      "Total written (filtered):    159,283,864 bp (100.0%)\n",
      "  Read 1:    79,648,833 bp\n",
      "  Read 2:    79,635,031 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t4120.9\t0\t109\n",
      "4\t3\t1030.2\t0\t3\n",
      "6\t1\t64.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 173 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t95\t4120.9\t0\t95\n",
      "4\t9\t1030.2\t0\t9\n",
      "5\t6\t257.6\t0\t6\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t62\t0.0\t2\t47 15\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28bexposed071717M_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28bexposed071717M_131_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28bexposed071717M_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28bexposed071717M_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28bexposed071717M_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28bexposed071717M_131_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28bexposed071717M_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28bexposed071717M_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:06       233,587 reads  @     29.7 µs/read;   2.02 M reads/minute\n",
      "Finished in 6.97 s (30 us/read; 2.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            233,587\n",
      "  Read 1 with adapter:                      99 (0.0%)\n",
      "  Read 2 with adapter:                     127 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       233,587 (100.0%)\n",
      "\n",
      "Total basepairs processed:   141,086,548 bp\n",
      "  Read 1:    70,543,274 bp\n",
      "  Read 2:    70,543,274 bp\n",
      "Total written (filtered):    141,076,685 bp (100.0%)\n",
      "  Read 1:    70,542,945 bp\n",
      "  Read 2:    70,533,740 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t3649.8\t0\t91\n",
      "4\t3\t912.4\t0\t3\n",
      "5\t1\t228.1\t0\t1\n",
      "6\t1\t57.0\t0\t1\n",
      "10\t2\t0.2\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 127 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t65\t3649.8\t0\t65\n",
      "4\t11\t912.4\t0\t11\n",
      "5\t6\t228.1\t0\t6\n",
      "6\t1\t57.0\t0\t1\n",
      "30\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t0 1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t41\t0.0\t2\t31 10\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28cbaseline070717M_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28cbaseline070717M_132_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28cbaseline070717M_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28cbaseline070717M_132_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28cbaseline070717M_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28cbaseline070717M_132_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28cbaseline070717M_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28cbaseline070717M_132_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[    8=------] 00:00:10       382,428 reads  @     28.2 µs/read;   2.13 M reads/minute\n",
      "Finished in 10.83 s (28 us/read; 2.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            382,428\n",
      "  Read 1 with adapter:                     261 (0.1%)\n",
      "  Read 2 with adapter:                     334 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       382,428 (100.0%)\n",
      "\n",
      "Total basepairs processed:   230,986,512 bp\n",
      "  Read 1:   115,493,256 bp\n",
      "  Read 2:   115,493,256 bp\n",
      "Total written (filtered):    230,963,642 bp (100.0%)\n",
      "  Read 1:   115,492,416 bp\n",
      "  Read 2:   115,471,226 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 261 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t228\t5975.4\t0\t228\n",
      "4\t19\t1493.9\t0\t19\n",
      "5\t6\t373.5\t0\t6\n",
      "6\t6\t93.4\t0\t6\n",
      "7\t2\t23.3\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 334 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t184\t5975.4\t0\t184\n",
      "4\t40\t1493.9\t0\t40\n",
      "5\t6\t373.5\t0\t6\n",
      "6\t5\t93.4\t0\t5\n",
      "8\t2\t5.8\t0\t2\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t96\t0.0\t2\t69 27\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28cexposed071717M_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28cexposed071717M_133_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28cexposed071717M_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28cexposed071717M_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28cexposed071717M_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28cexposed071717M_133_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28cexposed071717M_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28cexposed071717M_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[   8=-------] 00:00:08       273,722 reads  @     29.5 µs/read;   2.03 M reads/minute\n",
      "Finished in 8.11 s (30 us/read; 2.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            273,722\n",
      "  Read 1 with adapter:                     130 (0.0%)\n",
      "  Read 2 with adapter:                     194 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       273,722 (100.0%)\n",
      "\n",
      "Total basepairs processed:   165,328,088 bp\n",
      "  Read 1:    82,664,044 bp\n",
      "  Read 2:    82,664,044 bp\n",
      "Total written (filtered):    165,315,835 bp (100.0%)\n",
      "  Read 1:    82,663,634 bp\n",
      "  Read 2:    82,652,201 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 130 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t118\t4276.9\t0\t118\n",
      "4\t10\t1069.2\t0\t10\n",
      "6\t1\t66.8\t0\t1\n",
      "10\t1\t0.3\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 194 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t120\t4276.9\t0\t120\n",
      "4\t16\t1069.2\t0\t16\n",
      "5\t4\t267.3\t0\t4\n",
      "6\t2\t66.8\t0\t2\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t51\t0.0\t2\t37 14\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28fbaseline070717M_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28fbaseline070717M_134_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28fbaseline070717M_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28fbaseline070717M_134_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28fbaseline070717M_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28fbaseline070717M_134_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28fbaseline070717M_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28fbaseline070717M_134_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       142,049 reads  @     31.2 µs/read;   1.92 M reads/minute\n",
      "Finished in 4.48 s (32 us/read; 1.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,049\n",
      "  Read 1 with adapter:                      56 (0.0%)\n",
      "  Read 2 with adapter:                      95 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       142,049 (100.0%)\n",
      "\n",
      "Total basepairs processed:    85,797,596 bp\n",
      "  Read 1:    42,898,798 bp\n",
      "  Read 2:    42,898,798 bp\n",
      "Total written (filtered):     85,790,637 bp (100.0%)\n",
      "  Read 1:    42,898,620 bp\n",
      "  Read 2:    42,892,017 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 56 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t52\t2219.5\t0\t52\n",
      "4\t3\t554.9\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 95 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t51\t2219.5\t0\t51\n",
      "4\t11\t554.9\t0\t11\n",
      "5\t3\t138.7\t0\t3\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t29\t0.0\t2\t25 4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28hexposed071717M_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28hexposed071717M_135_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28hexposed071717M_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28hexposed071717M_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28hexposed071717M_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28hexposed071717M_135_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28hexposed071717M_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28hexposed071717M_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       162,131 reads  @     31.4 µs/read;   1.91 M reads/minute\n",
      "Finished in 5.13 s (32 us/read; 1.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            162,131\n",
      "  Read 1 with adapter:                      52 (0.0%)\n",
      "  Read 2 with adapter:                      67 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       162,131 (100.0%)\n",
      "\n",
      "Total basepairs processed:    97,927,124 bp\n",
      "  Read 1:    48,963,562 bp\n",
      "  Read 2:    48,963,562 bp\n",
      "Total written (filtered):     97,920,498 bp (100.0%)\n",
      "  Read 1:    48,963,406 bp\n",
      "  Read 2:    48,957,092 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 52 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t52\t2533.3\t0\t52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 67 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t2533.3\t0\t34\n",
      "4\t3\t633.3\t0\t3\n",
      "5\t1\t158.3\t0\t1\n",
      "219\t29\t0.0\t2\t23 6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28ibaseline070717M_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28ibaseline070717M_136_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28ibaseline070717M_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28ibaseline070717M_136_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28ibaseline070717M_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28ibaseline070717M_136_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28ibaseline070717M_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28ibaseline070717M_136_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       211,000 reads  @     29.4 µs/read;   2.04 M reads/minute\n",
      "Finished in 6.24 s (30 us/read; 2.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            211,000\n",
      "  Read 1 with adapter:                      80 (0.0%)\n",
      "  Read 2 with adapter:                     142 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       211,000 (100.0%)\n",
      "\n",
      "Total basepairs processed:   127,444,000 bp\n",
      "  Read 1:    63,722,000 bp\n",
      "  Read 2:    63,722,000 bp\n",
      "Total written (filtered):    127,436,169 bp (100.0%)\n",
      "  Read 1:    63,721,749 bp\n",
      "  Read 2:    63,714,420 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 80 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t75\t3296.9\t0\t75\n",
      "4\t1\t824.2\t0\t1\n",
      "5\t3\t206.1\t0\t3\n",
      "7\t1\t12.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 142 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t3296.9\t0\t91\n",
      "4\t13\t824.2\t0\t13\n",
      "5\t3\t206.1\t0\t3\n",
      "6\t1\t51.5\t0\t1\n",
      "8\t1\t3.2\t0\t1\n",
      "218\t1\t0.0\t2\t0 1\n",
      "219\t32\t0.0\t2\t25 7\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28iexposed071717M_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28iexposed071717M_137_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28iexposed071717M_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28iexposed071717M_137_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28iexposed071717M_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28iexposed071717M_137_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28iexposed071717M_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28iexposed071717M_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       111,667 reads  @     31.9 µs/read;   1.88 M reads/minute\n",
      "Finished in 3.60 s (32 us/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,667\n",
      "  Read 1 with adapter:                      62 (0.1%)\n",
      "  Read 2 with adapter:                      79 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,667 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,446,868 bp\n",
      "  Read 1:    33,723,434 bp\n",
      "  Read 2:    33,723,434 bp\n",
      "Total written (filtered):     67,441,665 bp (100.0%)\n",
      "  Read 1:    33,723,231 bp\n",
      "  Read 2:    33,718,434 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 62 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t1744.8\t0\t54\n",
      "4\t4\t436.2\t0\t4\n",
      "5\t3\t109.0\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 79 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t1744.8\t0\t48\n",
      "4\t7\t436.2\t0\t7\n",
      "5\t2\t109.0\t0\t2\n",
      "219\t22\t0.0\t2\t19 3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28jexposed071717M_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28jexposed071717M_138_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28jexposed071717M_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28jexposed071717M_138_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28jexposed071717M_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C28jexposed071717M_138_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28jexposed071717M_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C28jexposed071717M_138_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       126,668 reads  @     31.9 µs/read;   1.88 M reads/minute\n",
      "Finished in 4.07 s (32 us/read; 1.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            126,668\n",
      "  Read 1 with adapter:                      82 (0.1%)\n",
      "  Read 2 with adapter:                      71 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       126,668 (100.0%)\n",
      "\n",
      "Total basepairs processed:    76,507,472 bp\n",
      "  Read 1:    38,253,736 bp\n",
      "  Read 2:    38,253,736 bp\n",
      "Total written (filtered):     76,501,560 bp (100.0%)\n",
      "  Read 1:    38,253,452 bp\n",
      "  Read 2:    38,248,108 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 82 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t66\t1979.2\t0\t66\n",
      "4\t10\t494.8\t0\t10\n",
      "5\t3\t123.7\t0\t3\n",
      "9\t2\t0.5\t0\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 71 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t1979.2\t0\t37\n",
      "4\t6\t494.8\t0\t6\n",
      "5\t2\t123.7\t0\t2\n",
      "8\t1\t1.9\t0\t1\n",
      "219\t25\t0.0\t2\t17 8\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29bbaseline070717M_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29bbaseline070717M_139_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29bbaseline070717M_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29bbaseline070717M_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29bbaseline070717M_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29bbaseline070717M_139_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29bbaseline070717M_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29bbaseline070717M_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       101,642 reads  @     32.2 µs/read;   1.86 M reads/minute\n",
      "Finished in 3.31 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,642\n",
      "  Read 1 with adapter:                      35 (0.0%)\n",
      "  Read 2 with adapter:                      73 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       101,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:    61,391,768 bp\n",
      "  Read 1:    30,695,884 bp\n",
      "  Read 2:    30,695,884 bp\n",
      "Total written (filtered):     61,387,540 bp (100.0%)\n",
      "  Read 1:    30,695,775 bp\n",
      "  Read 2:    30,691,765 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 35 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t1588.2\t0\t33\n",
      "5\t2\t99.3\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 73 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t1588.2\t0\t47\n",
      "4\t6\t397.0\t0\t6\n",
      "5\t1\t99.3\t0\t1\n",
      "7\t1\t6.2\t0\t1\n",
      "219\t18\t0.0\t2\t16 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29cexposed071717M_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29cexposed071717M_140_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29cexposed071717M_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29cexposed071717M_140_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29cexposed071717M_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29cexposed071717M_140_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29cexposed071717M_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29cexposed071717M_140_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:06       231,495 reads  @     29.1 µs/read;   2.06 M reads/minute\n",
      "Finished in 6.77 s (29 us/read; 2.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            231,495\n",
      "  Read 1 with adapter:                     104 (0.0%)\n",
      "  Read 2 with adapter:                     160 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       231,495 (100.0%)\n",
      "\n",
      "Total basepairs processed:   139,822,980 bp\n",
      "  Read 1:    69,911,490 bp\n",
      "  Read 2:    69,911,490 bp\n",
      "Total written (filtered):    139,805,956 bp (100.0%)\n",
      "  Read 1:    69,911,163 bp\n",
      "  Read 2:    69,894,793 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 104 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t100\t3617.1\t0\t100\n",
      "4\t1\t904.3\t0\t1\n",
      "5\t2\t226.1\t0\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 160 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t3617.1\t0\t72\n",
      "4\t8\t904.3\t0\t8\n",
      "5\t4\t226.1\t0\t4\n",
      "6\t1\t56.5\t0\t1\n",
      "218\t2\t0.0\t2\t0 2\n",
      "219\t73\t0.0\t2\t50 23\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29dexposed071717M_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29dexposed071717M_141_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29dexposed071717M_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29dexposed071717M_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29dexposed071717M_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29dexposed071717M_141_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29dexposed071717M_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29dexposed071717M_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        87,295 reads  @     32.8 µs/read;   1.83 M reads/minute\n",
      "Finished in 2.89 s (33 us/read; 1.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             87,295\n",
      "  Read 1 with adapter:                      42 (0.0%)\n",
      "  Read 2 with adapter:                      39 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        87,295 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,726,180 bp\n",
      "  Read 1:    26,363,090 bp\n",
      "  Read 2:    26,363,090 bp\n",
      "Total written (filtered):     52,722,044 bp (100.0%)\n",
      "  Read 1:    26,362,962 bp\n",
      "  Read 2:    26,359,082 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 42 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t1364.0\t0\t40\n",
      "4\t2\t341.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 39 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t1364.0\t0\t18\n",
      "4\t2\t341.0\t0\t2\n",
      "5\t1\t85.2\t0\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t17\t0.0\t2\t12 5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29ebaseline070717M_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29ebaseline070717M_142_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29ebaseline070717M_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29ebaseline070717M_142_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29ebaseline070717M_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29ebaseline070717M_142_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29ebaseline070717M_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29ebaseline070717M_142_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        68,309 reads  @     35.0 µs/read;   1.71 M reads/minute\n",
      "Finished in 2.42 s (35 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,309\n",
      "  Read 1 with adapter:                      30 (0.0%)\n",
      "  Read 2 with adapter:                      60 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        68,309 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,258,636 bp\n",
      "  Read 1:    20,629,318 bp\n",
      "  Read 2:    20,629,318 bp\n",
      "Total written (filtered):     41,253,152 bp (100.0%)\n",
      "  Read 1:    20,629,212 bp\n",
      "  Read 2:    20,623,940 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 30 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t1067.3\t0\t25\n",
      "4\t3\t266.8\t0\t3\n",
      "7\t1\t4.2\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 60 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t1067.3\t0\t33\n",
      "4\t1\t266.8\t0\t1\n",
      "5\t1\t66.7\t0\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "219\t24\t0.0\t2\t20 4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29eexposed071717M_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29eexposed071717M_143_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29eexposed071717M_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29eexposed071717M_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29eexposed071717M_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29eexposed071717M_143_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29eexposed071717M_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29eexposed071717M_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[       8=---] 00:00:16       615,605 reads  @     27.2 µs/read;   2.21 M reads/minute\n",
      "Finished in 16.77 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            615,605\n",
      "  Read 1 with adapter:                     287 (0.0%)\n",
      "  Read 2 with adapter:                     361 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       615,605 (100.0%)\n",
      "\n",
      "Total basepairs processed:   371,825,420 bp\n",
      "  Read 1:   185,912,710 bp\n",
      "  Read 2:   185,912,710 bp\n",
      "Total written (filtered):    371,793,360 bp (100.0%)\n",
      "  Read 1:   185,911,811 bp\n",
      "  Read 2:   185,881,549 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 287 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t269\t9618.8\t0\t269\n",
      "4\t8\t2404.7\t0\t8\n",
      "5\t6\t601.2\t0\t6\n",
      "6\t3\t150.3\t0\t3\n",
      "12\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 361 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t192\t9618.8\t0\t192\n",
      "4\t21\t2404.7\t0\t21\n",
      "5\t6\t601.2\t0\t6\n",
      "6\t1\t150.3\t0\t1\n",
      "8\t1\t9.4\t0\t1\n",
      "20\t1\t0.0\t2\t1\n",
      "218\t4\t0.0\t2\t1 3\n",
      "219\t135\t0.0\t2\t92 43\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29fexposed071717M_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29fexposed071717M_144_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29fexposed071717M_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29fexposed071717M_144_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29fexposed071717M_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29fexposed071717M_144_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29fexposed071717M_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29fexposed071717M_144_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       123,704 reads  @     32.1 µs/read;   1.87 M reads/minute\n",
      "Finished in 4.01 s (32 us/read; 1.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            123,704\n",
      "  Read 1 with adapter:                      53 (0.0%)\n",
      "  Read 2 with adapter:                      77 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       123,704 (100.0%)\n",
      "\n",
      "Total basepairs processed:    74,717,216 bp\n",
      "  Read 1:    37,358,608 bp\n",
      "  Read 2:    37,358,608 bp\n",
      "Total written (filtered):     74,711,191 bp (100.0%)\n",
      "  Read 1:    37,358,437 bp\n",
      "  Read 2:    37,352,754 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 53 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t1932.9\t0\t50\n",
      "4\t1\t483.2\t0\t1\n",
      "5\t1\t120.8\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 77 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t1932.9\t0\t46\n",
      "4\t4\t483.2\t0\t4\n",
      "6\t1\t30.2\t0\t1\n",
      "219\t26\t0.0\t2\t20 6\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29hbaseline070717M_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29hbaseline070717M_145_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29hbaseline070717M_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29hbaseline070717M_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29hbaseline070717M_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29hbaseline070717M_145_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29hbaseline070717M_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29hbaseline070717M_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       166,230 reads  @     30.2 µs/read;   1.98 M reads/minute\n",
      "Finished in 5.06 s (30 us/read; 1.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            166,230\n",
      "  Read 1 with adapter:                     105 (0.1%)\n",
      "  Read 2 with adapter:                     128 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       166,230 (100.0%)\n",
      "\n",
      "Total basepairs processed:   100,402,920 bp\n",
      "  Read 1:    50,201,460 bp\n",
      "  Read 2:    50,201,460 bp\n",
      "Total written (filtered):    100,392,832 bp (100.0%)\n",
      "  Read 1:    50,201,086 bp\n",
      "  Read 2:    50,191,746 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 105 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t92\t2597.3\t0\t92\n",
      "4\t6\t649.3\t0\t6\n",
      "5\t1\t162.3\t0\t1\n",
      "6\t2\t40.6\t0\t2\n",
      "9\t1\t0.6\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 128 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t70\t2597.3\t0\t70\n",
      "4\t13\t649.3\t0\t13\n",
      "5\t1\t162.3\t0\t1\n",
      "32\t1\t0.0\t2\t0 1\n",
      "217\t1\t0.0\t2\t1\n",
      "219\t42\t0.0\t2\t35 7\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29iexposed071717M_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29iexposed071717M_146_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29iexposed071717M_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29iexposed071717M_146_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29iexposed071717M_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C29iexposed071717M_146_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29iexposed071717M_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C29iexposed071717M_146_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        99,668 reads  @     33.4 µs/read;   1.80 M reads/minute\n",
      "Finished in 3.37 s (34 us/read; 1.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             99,668\n",
      "  Read 1 with adapter:                      44 (0.0%)\n",
      "  Read 2 with adapter:                      77 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        99,668 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,199,472 bp\n",
      "  Read 1:    30,099,736 bp\n",
      "  Read 2:    30,099,736 bp\n",
      "Total written (filtered):     60,191,538 bp (100.0%)\n",
      "  Read 1:    30,099,602 bp\n",
      "  Read 2:    30,091,936 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 44 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t1557.3\t0\t42\n",
      "4\t2\t389.3\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 77 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t1557.3\t0\t34\n",
      "4\t5\t389.3\t0\t5\n",
      "5\t3\t97.3\t0\t3\n",
      "218\t2\t0.0\t2\t1 1\n",
      "219\t33\t0.0\t2\t20 13\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30aexposed071717L_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30aexposed071717L_147_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30aexposed071717L_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30aexposed071717L_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30aexposed071717L_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30aexposed071717L_147_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30aexposed071717L_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30aexposed071717L_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04       130,994 reads  @     30.9 µs/read;   1.94 M reads/minute\n",
      "Finished in 4.09 s (31 us/read; 1.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,994\n",
      "  Read 1 with adapter:                      93 (0.1%)\n",
      "  Read 2 with adapter:                     149 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       130,994 (100.0%)\n",
      "\n",
      "Total basepairs processed:    79,120,376 bp\n",
      "  Read 1:    39,560,188 bp\n",
      "  Read 2:    39,560,188 bp\n",
      "Total written (filtered):     79,108,576 bp (100.0%)\n",
      "  Read 1:    39,559,889 bp\n",
      "  Read 2:    39,548,687 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 93 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t2046.8\t0\t80\n",
      "4\t9\t511.7\t0\t9\n",
      "5\t2\t127.9\t0\t2\n",
      "6\t1\t32.0\t0\t1\n",
      "7\t1\t8.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 149 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t70\t2046.8\t0\t70\n",
      "4\t22\t511.7\t0\t22\n",
      "5\t3\t127.9\t0\t3\n",
      "6\t2\t32.0\t0\t2\n",
      "8\t1\t2.0\t0\t1\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t50\t0.0\t2\t35 15\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30bexposed071717L_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30bexposed071717L_148_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30bexposed071717L_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30bexposed071717L_148_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30bexposed071717L_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30bexposed071717L_148_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30bexposed071717L_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30bexposed071717L_148_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       135,316 reads  @     31.4 µs/read;   1.91 M reads/minute\n",
      "Finished in 4.29 s (32 us/read; 1.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            135,316\n",
      "  Read 1 with adapter:                      67 (0.0%)\n",
      "  Read 2 with adapter:                      98 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       135,316 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,730,864 bp\n",
      "  Read 1:    40,865,432 bp\n",
      "  Read 2:    40,865,432 bp\n",
      "Total written (filtered):     81,723,211 bp (100.0%)\n",
      "  Read 1:    40,865,216 bp\n",
      "  Read 2:    40,857,995 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 67 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t62\t2114.3\t0\t62\n",
      "4\t3\t528.6\t0\t3\n",
      "6\t1\t33.0\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 98 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t2114.3\t0\t53\n",
      "4\t10\t528.6\t0\t10\n",
      "5\t1\t132.1\t0\t1\n",
      "6\t1\t33.0\t0\t1\n",
      "219\t33\t0.0\t2\t23 10\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30fexposed071717L_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30fexposed071717L_149_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30fexposed071717L_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30fexposed071717L_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30fexposed071717L_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30fexposed071717L_149_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30fexposed071717L_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30fexposed071717L_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       161,055 reads  @     29.0 µs/read;   2.07 M reads/minute\n",
      "Finished in 4.70 s (29 us/read; 2.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,055\n",
      "  Read 1 with adapter:                      68 (0.0%)\n",
      "  Read 2 with adapter:                      89 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       161,055 (100.0%)\n",
      "\n",
      "Total basepairs processed:    97,277,220 bp\n",
      "  Read 1:    48,638,610 bp\n",
      "  Read 2:    48,638,610 bp\n",
      "Total written (filtered):     97,269,758 bp (100.0%)\n",
      "  Read 1:    48,638,391 bp\n",
      "  Read 2:    48,631,367 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 68 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t2516.5\t0\t58\n",
      "4\t5\t629.1\t0\t5\n",
      "5\t5\t157.3\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 89 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t2516.5\t0\t45\n",
      "4\t8\t629.1\t0\t8\n",
      "5\t2\t157.3\t0\t2\n",
      "29\t1\t0.0\t2\t0 1\n",
      "30\t1\t0.0\t2\t0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t31\t0.0\t2\t21 10\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30gbaseline070717_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30gbaseline070717_150_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30gbaseline070717_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30gbaseline070717_150_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30gbaseline070717_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30gbaseline070717_150_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30gbaseline070717_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30gbaseline070717_150_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       102,772 reads  @     32.0 µs/read;   1.88 M reads/minute\n",
      "Finished in 3.33 s (32 us/read; 1.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            102,772\n",
      "  Read 1 with adapter:                      50 (0.0%)\n",
      "  Read 2 with adapter:                      75 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       102,772 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,074,288 bp\n",
      "  Read 1:    31,037,144 bp\n",
      "  Read 2:    31,037,144 bp\n",
      "Total written (filtered):     62,068,699 bp (100.0%)\n",
      "  Read 1:    31,036,977 bp\n",
      "  Read 2:    31,031,722 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 50 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t1605.8\t0\t46\n",
      "4\t2\t401.5\t0\t2\n",
      "5\t1\t100.4\t0\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 75 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t1605.8\t0\t42\n",
      "4\t6\t401.5\t0\t6\n",
      "5\t2\t100.4\t0\t2\n",
      "6\t1\t25.1\t0\t1\n",
      "219\t24\t0.0\t2\t22 2\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30gexposed071717L_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30gexposed071717L_151_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30gexposed071717L_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30gexposed071717L_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30gexposed071717L_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30gexposed071717L_151_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30gexposed071717L_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30gexposed071717L_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        88,494 reads  @     33.1 µs/read;   1.81 M reads/minute\n",
      "Finished in 2.97 s (34 us/read; 1.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             88,494\n",
      "  Read 1 with adapter:                      51 (0.1%)\n",
      "  Read 2 with adapter:                      96 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        88,494 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,450,376 bp\n",
      "  Read 1:    26,725,188 bp\n",
      "  Read 2:    26,725,188 bp\n",
      "Total written (filtered):     53,437,610 bp (100.0%)\n",
      "  Read 1:    26,725,033 bp\n",
      "  Read 2:    26,712,577 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 51 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t1382.7\t0\t49\n",
      "4\t2\t345.7\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 96 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1382.7\t0\t29\n",
      "4\t8\t345.7\t0\t8\n",
      "5\t1\t86.4\t0\t1\n",
      "6\t1\t21.6\t0\t1\n",
      "218\t2\t0.0\t2\t2\n",
      "219\t55\t0.0\t2\t38 17\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30hexposed071717L_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30hexposed071717L_152_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30hexposed071717L_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30hexposed071717L_152_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30hexposed071717L_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30hexposed071717L_152_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30hexposed071717L_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30hexposed071717L_152_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       121,497 reads  @     32.9 µs/read;   1.82 M reads/minute\n",
      "Finished in 4.04 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            121,497\n",
      "  Read 1 with adapter:                      56 (0.0%)\n",
      "  Read 2 with adapter:                      70 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       121,497 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,384,188 bp\n",
      "  Read 1:    36,692,094 bp\n",
      "  Read 2:    36,692,094 bp\n",
      "Total written (filtered):     73,380,560 bp (100.0%)\n",
      "  Read 1:    36,691,923 bp\n",
      "  Read 2:    36,688,637 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 56 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t1898.4\t0\t53\n",
      "4\t3\t474.6\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 70 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t1898.4\t0\t48\n",
      "4\t7\t474.6\t0\t7\n",
      "219\t15\t0.0\t2\t10 5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30ibaseline070717L_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30ibaseline070717L_153_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30ibaseline070717L_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30ibaseline070717L_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30ibaseline070717L_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30ibaseline070717L_153_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30ibaseline070717L_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30ibaseline070717L_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       125,413 reads  @     31.3 µs/read;   1.92 M reads/minute\n",
      "Finished in 3.97 s (32 us/read; 1.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,413\n",
      "  Read 1 with adapter:                      55 (0.0%)\n",
      "  Read 2 with adapter:                      78 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       125,413 (100.0%)\n",
      "\n",
      "Total basepairs processed:    75,749,452 bp\n",
      "  Read 1:    37,874,726 bp\n",
      "  Read 2:    37,874,726 bp\n",
      "Total written (filtered):     75,741,677 bp (100.0%)\n",
      "  Read 1:    37,874,545 bp\n",
      "  Read 2:    37,867,132 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t1959.6\t0\t44\n",
      "4\t7\t489.9\t0\t7\n",
      "5\t3\t122.5\t0\t3\n",
      "6\t1\t30.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 78 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t1959.6\t0\t37\n",
      "4\t4\t489.9\t0\t4\n",
      "5\t2\t122.5\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "219\t34\t0.0\t2\t24 10\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30jbaseline070717_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30jbaseline070717_154_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30jbaseline070717_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30jbaseline070717_154_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30jbaseline070717_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30jbaseline070717_154_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30jbaseline070717_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30jbaseline070717_154_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04       140,293 reads  @     29.1 µs/read;   2.07 M reads/minute\n",
      "Finished in 4.12 s (29 us/read; 2.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            140,293\n",
      "  Read 1 with adapter:                      59 (0.0%)\n",
      "  Read 2 with adapter:                     128 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       140,293 (100.0%)\n",
      "\n",
      "Total basepairs processed:    84,736,972 bp\n",
      "  Read 1:    42,368,486 bp\n",
      "  Read 2:    42,368,486 bp\n",
      "Total written (filtered):     84,720,617 bp (100.0%)\n",
      "  Read 1:    42,368,298 bp\n",
      "  Read 2:    42,352,319 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 59 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t2192.1\t0\t54\n",
      "4\t1\t548.0\t0\t1\n",
      "5\t2\t137.0\t0\t2\n",
      "6\t2\t34.3\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 128 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t2192.1\t0\t48\n",
      "4\t4\t548.0\t0\t4\n",
      "5\t2\t137.0\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "219\t73\t0.0\t2\t51 22\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30jexposed071717L_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30jexposed071717L_155_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30jexposed071717L_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30jexposed071717L_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30jexposed071717L_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/C30jexposed071717L_155_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30jexposed071717L_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/C30jexposed071717L_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:05       212,034 reads  @     28.2 µs/read;   2.13 M reads/minute\n",
      "Finished in 6.02 s (28 us/read; 2.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            212,034\n",
      "  Read 1 with adapter:                     109 (0.1%)\n",
      "  Read 2 with adapter:                     164 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       212,034 (100.0%)\n",
      "\n",
      "Total basepairs processed:   128,068,536 bp\n",
      "  Read 1:    64,034,268 bp\n",
      "  Read 2:    64,034,268 bp\n",
      "Total written (filtered):    128,055,959 bp (100.0%)\n",
      "  Read 1:    64,033,906 bp\n",
      "  Read 2:    64,022,053 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t101\t3313.0\t0\t101\n",
      "4\t5\t828.3\t0\t5\n",
      "5\t1\t207.1\t0\t1\n",
      "10\t1\t0.2\t1\t0 1\n",
      "24\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 164 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t86\t3313.0\t0\t86\n",
      "4\t17\t828.3\t0\t17\n",
      "5\t4\t207.1\t0\t4\n",
      "6\t2\t51.8\t0\t2\n",
      "32\t1\t0.0\t2\t0 1\n",
      "218\t1\t0.0\t2\t1\n",
      "219\t53\t0.0\t2\t37 16\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10dbaseline062217H_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10dbaseline062217H_157_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10dbaseline062217H_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10dbaseline062217H_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10dbaseline062217H_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10dbaseline062217H_157_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10dbaseline062217H_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10dbaseline062217H_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       148,336 reads  @     35.2 µs/read;   1.71 M reads/minute\n",
      "Finished in 5.26 s (35 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            148,336\n",
      "  Read 1 with adapter:                      39 (0.0%)\n",
      "  Read 2 with adapter:                      62 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       148,336 (100.0%)\n",
      "\n",
      "Total basepairs processed:    89,594,944 bp\n",
      "  Read 1:    44,797,472 bp\n",
      "  Read 2:    44,797,472 bp\n",
      "Total written (filtered):     89,594,619 bp (100.0%)\n",
      "  Read 1:    44,797,350 bp\n",
      "  Read 2:    44,797,269 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 39 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t2317.8\t0\t35\n",
      "4\t3\t579.4\t0\t3\n",
      "5\t1\t144.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 62 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t2317.8\t0\t48\n",
      "4\t12\t579.4\t0\t12\n",
      "5\t1\t144.9\t0\t1\n",
      "6\t1\t36.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10dexposed071717H_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10dexposed071717H_158_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10dexposed071717H_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10dexposed071717H_158_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10dexposed071717H_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10dexposed071717H_158_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10dexposed071717H_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10dexposed071717H_158_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:05       170,042 reads  @     32.8 µs/read;   1.83 M reads/minute\n",
      "Finished in 5.62 s (33 us/read; 1.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            170,042\n",
      "  Read 1 with adapter:                      43 (0.0%)\n",
      "  Read 2 with adapter:                      48 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       170,042 (100.0%)\n",
      "\n",
      "Total basepairs processed:   102,705,368 bp\n",
      "  Read 1:    51,352,684 bp\n",
      "  Read 2:    51,352,684 bp\n",
      "Total written (filtered):    102,705,087 bp (100.0%)\n",
      "  Read 1:    51,352,550 bp\n",
      "  Read 2:    51,352,537 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 43 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t2656.9\t0\t39\n",
      "4\t3\t664.2\t0\t3\n",
      "5\t1\t166.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 48 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t2656.9\t0\t45\n",
      "4\t3\t664.2\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10ebaseline062217H_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10ebaseline062217H_159_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10ebaseline062217H_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10ebaseline062217H_159_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10ebaseline062217H_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10ebaseline062217H_159_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10ebaseline062217H_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10ebaseline062217H_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       160,996 reads  @     33.1 µs/read;   1.81 M reads/minute\n",
      "Finished in 5.37 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            160,996\n",
      "  Read 1 with adapter:                      38 (0.0%)\n",
      "  Read 2 with adapter:                      71 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       160,996 (100.0%)\n",
      "\n",
      "Total basepairs processed:    97,241,584 bp\n",
      "  Read 1:    48,620,792 bp\n",
      "  Read 2:    48,620,792 bp\n",
      "Total written (filtered):     97,241,208 bp (100.0%)\n",
      "  Read 1:    48,620,669 bp\n",
      "  Read 2:    48,620,539 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 38 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t2515.6\t0\t31\n",
      "4\t5\t628.9\t0\t5\n",
      "5\t2\t157.2\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 71 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t2515.6\t0\t61\n",
      "4\t7\t628.9\t0\t7\n",
      "5\t2\t157.2\t0\t2\n",
      "32\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10eexposed071717H_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10eexposed071717H_160_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10eexposed071717H_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10eexposed071717H_160_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10eexposed071717H_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10eexposed071717H_160_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10eexposed071717H_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10eexposed071717H_160_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:06       198,953 reads  @     33.5 µs/read;   1.79 M reads/minute\n",
      "Finished in 6.71 s (34 us/read; 1.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            198,953\n",
      "  Read 1 with adapter:                      64 (0.0%)\n",
      "  Read 2 with adapter:                      75 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       198,953 (100.0%)\n",
      "\n",
      "Total basepairs processed:   120,167,612 bp\n",
      "  Read 1:    60,083,806 bp\n",
      "  Read 2:    60,083,806 bp\n",
      "Total written (filtered):    120,167,161 bp (100.0%)\n",
      "  Read 1:    60,083,603 bp\n",
      "  Read 2:    60,083,558 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 64 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t57\t3108.6\t0\t57\n",
      "4\t4\t777.2\t0\t4\n",
      "5\t2\t194.3\t0\t2\n",
      "6\t1\t48.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 75 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t3108.6\t0\t58\n",
      "4\t12\t777.2\t0\t12\n",
      "5\t4\t194.3\t0\t4\n",
      "6\t1\t48.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10hexposed071717_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10hexposed071717_161_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10hexposed071717_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10hexposed071717_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10hexposed071717_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10hexposed071717_161_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10hexposed071717_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10hexposed071717_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        64,587 reads  @     37.3 µs/read;   1.61 M reads/minute\n",
      "Finished in 2.45 s (38 us/read; 1.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,587\n",
      "  Read 1 with adapter:                      21 (0.0%)\n",
      "  Read 2 with adapter:                      17 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,587 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,010,548 bp\n",
      "  Read 1:    19,505,274 bp\n",
      "  Read 2:    19,505,274 bp\n",
      "Total written (filtered):     39,010,428 bp (100.0%)\n",
      "  Read 1:    19,505,208 bp\n",
      "  Read 2:    19,505,220 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t1009.2\t0\t20\n",
      "6\t1\t15.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 17 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t1009.2\t0\t14\n",
      "4\t3\t252.3\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10ibaseline062217H_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10ibaseline062217H_162_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10ibaseline062217H_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10ibaseline062217H_162_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10ibaseline062217H_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P10ibaseline062217H_162_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10ibaseline062217H_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P10ibaseline062217H_162_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:05       182,513 reads  @     32.7 µs/read;   1.83 M reads/minute\n",
      "Finished in 6.01 s (33 us/read; 1.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            182,513\n",
      "  Read 1 with adapter:                      45 (0.0%)\n",
      "  Read 2 with adapter:                      66 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       182,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:   110,237,852 bp\n",
      "  Read 1:    55,118,926 bp\n",
      "  Read 2:    55,118,926 bp\n",
      "Total written (filtered):    110,237,294 bp (100.0%)\n",
      "  Read 1:    55,118,790 bp\n",
      "  Read 2:    55,118,504 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 45 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t2851.8\t0\t44\n",
      "4\t1\t712.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 66 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t57\t2851.8\t0\t57\n",
      "4\t8\t712.9\t0\t8\n",
      "219\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11bbaseline062217H_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11bbaseline062217H_163_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11bbaseline062217H_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11bbaseline062217H_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11bbaseline062217H_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11bbaseline062217H_163_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11bbaseline062217H_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11bbaseline062217H_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[       8<---] 00:00:15       518,680 reads  @     30.4 µs/read;   1.97 M reads/minute\n",
      "Finished in 15.80 s (30 us/read; 1.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            518,680\n",
      "  Read 1 with adapter:                     100 (0.0%)\n",
      "  Read 2 with adapter:                     163 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       518,680 (100.0%)\n",
      "\n",
      "Total basepairs processed:   313,282,720 bp\n",
      "  Read 1:   156,641,360 bp\n",
      "  Read 2:   156,641,360 bp\n",
      "Total written (filtered):    313,281,891 bp (100.0%)\n",
      "  Read 1:   156,641,044 bp\n",
      "  Read 2:   156,640,847 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 100 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t8104.4\t0\t88\n",
      "4\t9\t2026.1\t0\t9\n",
      "5\t2\t506.5\t0\t2\n",
      "6\t1\t126.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 163 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t142\t8104.4\t0\t142\n",
      "4\t19\t2026.1\t0\t19\n",
      "5\t1\t506.5\t0\t1\n",
      "6\t1\t126.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11bexposed071717H_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11bexposed071717H_164_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11bexposed071717H_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11bexposed071717H_164_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11bexposed071717H_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11bexposed071717H_164_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11bexposed071717H_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11bexposed071717H_164_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:06       226,627 reads  @     30.2 µs/read;   1.99 M reads/minute\n",
      "Finished in 6.88 s (30 us/read; 1.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            226,627\n",
      "  Read 1 with adapter:                      70 (0.0%)\n",
      "  Read 2 with adapter:                      86 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       226,627 (100.0%)\n",
      "\n",
      "Total basepairs processed:   136,882,708 bp\n",
      "  Read 1:    68,441,354 bp\n",
      "  Read 2:    68,441,354 bp\n",
      "Total written (filtered):    136,882,218 bp (100.0%)\n",
      "  Read 1:    68,441,131 bp\n",
      "  Read 2:    68,441,087 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 70 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t63\t3541.0\t0\t63\n",
      "4\t5\t885.3\t0\t5\n",
      "5\t1\t221.3\t0\t1\n",
      "9\t1\t0.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 86 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t3541.0\t0\t77\n",
      "4\t9\t885.3\t0\t9\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11eexposed071717_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11eexposed071717_165_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11eexposed071717_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11eexposed071717_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11eexposed071717_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11eexposed071717_165_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11eexposed071717_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11eexposed071717_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8=-------] 00:00:08       274,480 reads  @     30.8 µs/read;   1.95 M reads/minute\n",
      "Finished in 8.49 s (31 us/read; 1.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            274,480\n",
      "  Read 1 with adapter:                      65 (0.0%)\n",
      "  Read 2 with adapter:                      78 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       274,480 (100.0%)\n",
      "\n",
      "Total basepairs processed:   165,785,920 bp\n",
      "  Read 1:    82,892,960 bp\n",
      "  Read 2:    82,892,960 bp\n",
      "Total written (filtered):    165,785,442 bp (100.0%)\n",
      "  Read 1:    82,892,732 bp\n",
      "  Read 2:    82,892,710 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 65 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t4288.8\t0\t58\n",
      "4\t5\t1072.2\t0\t5\n",
      "6\t1\t67.0\t0\t1\n",
      "28\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 78 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t69\t4288.8\t0\t69\n",
      "4\t5\t1072.2\t0\t5\n",
      "5\t2\t268.0\t0\t2\n",
      "6\t1\t67.0\t0\t1\n",
      "7\t1\t16.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11hbaseline062217_70_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11hbaseline062217_166_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11hbaseline062217_70_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11hbaseline062217_166_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11hbaseline062217_70_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11hbaseline062217_166_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11hbaseline062217_70_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11hbaseline062217_166_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:05       175,775 reads  @     33.6 µs/read;   1.79 M reads/minute\n",
      "Finished in 5.95 s (34 us/read; 1.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            175,775\n",
      "  Read 1 with adapter:                      23 (0.0%)\n",
      "  Read 2 with adapter:                      65 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       175,775 (100.0%)\n",
      "\n",
      "Total basepairs processed:   106,168,100 bp\n",
      "  Read 1:    53,084,050 bp\n",
      "  Read 2:    53,084,050 bp\n",
      "Total written (filtered):    106,167,819 bp (100.0%)\n",
      "  Read 1:    53,083,978 bp\n",
      "  Read 2:    53,083,841 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t2746.5\t0\t21\n",
      "4\t1\t686.6\t0\t1\n",
      "5\t1\t171.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 65 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t2746.5\t0\t54\n",
      "4\t9\t686.6\t0\t9\n",
      "5\t1\t171.7\t0\t1\n",
      "6\t1\t42.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11hexposed071717_71_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11hexposed071717_167_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11hexposed071717_71_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11hexposed071717_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11hexposed071717_71_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P11hexposed071717_167_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11hexposed071717_71_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P11hexposed071717_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[    8<------] 00:00:09       322,064 reads  @     30.6 µs/read;   1.96 M reads/minute\n",
      "Finished in 9.89 s (31 us/read; 1.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            322,064\n",
      "  Read 1 with adapter:                      55 (0.0%)\n",
      "  Read 2 with adapter:                      89 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       322,064 (100.0%)\n",
      "\n",
      "Total basepairs processed:   194,526,656 bp\n",
      "  Read 1:    97,263,328 bp\n",
      "  Read 2:    97,263,328 bp\n",
      "Total written (filtered):    194,526,198 bp (100.0%)\n",
      "  Read 1:    97,263,154 bp\n",
      "  Read 2:    97,263,044 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t5032.2\t0\t47\n",
      "4\t7\t1258.1\t0\t7\n",
      "5\t1\t314.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 89 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t5032.2\t0\t79\n",
      "4\t7\t1258.1\t0\t7\n",
      "5\t1\t314.5\t0\t1\n",
      "6\t1\t78.6\t0\t1\n",
      "8\t1\t4.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12fbaseline062217H_72_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12fbaseline062217H_168_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12fbaseline062217H_72_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12fbaseline062217H_168_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12fbaseline062217H_72_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12fbaseline062217H_168_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12fbaseline062217H_72_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12fbaseline062217H_168_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        67,241 reads  @     38.2 µs/read;   1.57 M reads/minute\n",
      "Finished in 2.61 s (39 us/read; 1.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,241\n",
      "  Read 1 with adapter:                      21 (0.0%)\n",
      "  Read 2 with adapter:                      18 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        67,241 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,613,564 bp\n",
      "  Read 1:    20,306,782 bp\n",
      "  Read 2:    20,306,782 bp\n",
      "Total written (filtered):     40,613,426 bp (100.0%)\n",
      "  Read 1:    20,306,702 bp\n",
      "  Read 2:    20,306,724 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1050.6\t0\t16\n",
      "4\t1\t262.7\t0\t1\n",
      "6\t2\t16.4\t0\t2\n",
      "7\t1\t4.1\t0\t1\n",
      "9\t1\t0.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 18 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1050.6\t0\t15\n",
      "4\t2\t262.7\t0\t2\n",
      "5\t1\t65.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12fexposed071717H_73_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12fexposed071717H_169_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12fexposed071717H_73_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12fexposed071717H_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12fexposed071717H_73_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12fexposed071717H_169_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12fexposed071717H_73_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12fexposed071717H_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[   8<-------] 00:00:07       227,725 reads  @     31.3 µs/read;   1.92 M reads/minute\n",
      "Finished in 7.17 s (31 us/read; 1.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            227,725\n",
      "  Read 1 with adapter:                      67 (0.0%)\n",
      "  Read 2 with adapter:                      89 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       227,725 (100.0%)\n",
      "\n",
      "Total basepairs processed:   137,545,900 bp\n",
      "  Read 1:    68,772,950 bp\n",
      "  Read 2:    68,772,950 bp\n",
      "Total written (filtered):    137,545,381 bp (100.0%)\n",
      "  Read 1:    68,772,725 bp\n",
      "  Read 2:    68,772,656 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 67 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t3558.2\t0\t59\n",
      "4\t5\t889.6\t0\t5\n",
      "5\t1\t222.4\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 89 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t74\t3558.2\t0\t74\n",
      "4\t10\t889.6\t0\t10\n",
      "5\t1\t222.4\t0\t1\n",
      "6\t2\t55.6\t0\t2\n",
      "7\t1\t13.9\t0\t1\n",
      "8\t1\t3.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12gbaseline062217_74_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12gbaseline062217_170_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12gbaseline062217_74_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12gbaseline062217_170_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12gbaseline062217_74_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12gbaseline062217_170_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12gbaseline062217_74_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12gbaseline062217_170_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       192,218 reads  @     32.4 µs/read;   1.85 M reads/minute\n",
      "Finished in 6.26 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            192,218\n",
      "  Read 1 with adapter:                      22 (0.0%)\n",
      "  Read 2 with adapter:                      58 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       192,218 (100.0%)\n",
      "\n",
      "Total basepairs processed:   116,099,672 bp\n",
      "  Read 1:    58,049,836 bp\n",
      "  Read 2:    58,049,836 bp\n",
      "Total written (filtered):    116,099,420 bp (100.0%)\n",
      "  Read 1:    58,049,767 bp\n",
      "  Read 2:    58,049,653 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t3003.4\t0\t21\n",
      "6\t1\t46.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 58 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t3003.4\t0\t54\n",
      "4\t2\t750.9\t0\t2\n",
      "6\t1\t46.9\t0\t1\n",
      "7\t1\t11.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12gexposed071717_75_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12gexposed071717_171_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12gexposed071717_75_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12gexposed071717_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12gexposed071717_75_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12gexposed071717_171_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12gexposed071717_75_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12gexposed071717_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       150,092 reads  @     33.8 µs/read;   1.77 M reads/minute\n",
      "Finished in 5.11 s (34 us/read; 1.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            150,092\n",
      "  Read 1 with adapter:                      20 (0.0%)\n",
      "  Read 2 with adapter:                      36 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       150,092 (100.0%)\n",
      "\n",
      "Total basepairs processed:    90,655,568 bp\n",
      "  Read 1:    45,327,784 bp\n",
      "  Read 2:    45,327,784 bp\n",
      "Total written (filtered):     90,655,363 bp (100.0%)\n",
      "  Read 1:    45,327,695 bp\n",
      "  Read 2:    45,327,668 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t2345.2\t0\t15\n",
      "4\t2\t586.3\t0\t2\n",
      "10\t1\t0.1\t1\t0 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 36 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t2345.2\t0\t30\n",
      "4\t5\t586.3\t0\t5\n",
      "6\t1\t36.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12ibaseline062217_76_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12ibaseline062217_172_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12ibaseline062217_76_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12ibaseline062217_172_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12ibaseline062217_76_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12ibaseline062217_172_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12ibaseline062217_76_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12ibaseline062217_172_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:07       215,159 reads  @     34.5 µs/read;   1.74 M reads/minute\n",
      "Finished in 7.48 s (35 us/read; 1.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            215,159\n",
      "  Read 1 with adapter:                      86 (0.0%)\n",
      "  Read 2 with adapter:                     115 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       215,159 (100.0%)\n",
      "\n",
      "Total basepairs processed:   129,956,036 bp\n",
      "  Read 1:    64,978,018 bp\n",
      "  Read 2:    64,978,018 bp\n",
      "Total written (filtered):    129,955,394 bp (100.0%)\n",
      "  Read 1:    64,977,750 bp\n",
      "  Read 2:    64,977,644 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 86 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t3361.9\t0\t77\n",
      "4\t8\t840.5\t0\t8\n",
      "5\t1\t210.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 115 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t98\t3361.9\t0\t98\n",
      "4\t11\t840.5\t0\t11\n",
      "5\t2\t210.1\t0\t2\n",
      "6\t3\t52.5\t0\t3\n",
      "8\t1\t3.3\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12iexposed071717_77_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12iexposed071717_173_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12iexposed071717_77_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12iexposed071717_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12iexposed071717_77_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P12iexposed071717_173_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12iexposed071717_77_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P12iexposed071717_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        53,258 reads  @     38.9 µs/read;   1.54 M reads/minute\n",
      "Finished in 2.11 s (40 us/read; 1.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,258\n",
      "  Read 1 with adapter:                      10 (0.0%)\n",
      "  Read 2 with adapter:                       7 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,258 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,167,832 bp\n",
      "  Read 1:    16,083,916 bp\n",
      "  Read 2:    16,083,916 bp\n",
      "Total written (filtered):     32,167,781 bp (100.0%)\n",
      "  Read 1:    16,083,886 bp\n",
      "  Read 2:    16,083,895 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 10 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t832.2\t0\t10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 7 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t832.2\t0\t7\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7cbaseline062217H_78_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7cbaseline062217H_174_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7cbaseline062217H_78_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7cbaseline062217H_174_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7cbaseline062217H_78_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7cbaseline062217H_174_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7cbaseline062217H_78_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7cbaseline062217H_174_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       183,305 reads  @     35.2 µs/read;   1.70 M reads/minute\n",
      "Finished in 6.50 s (35 us/read; 1.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            183,305\n",
      "  Read 1 with adapter:                      90 (0.0%)\n",
      "  Read 2 with adapter:                      85 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       183,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:   110,716,220 bp\n",
      "  Read 1:    55,358,110 bp\n",
      "  Read 2:    55,358,110 bp\n",
      "Total written (filtered):    110,715,543 bp (100.0%)\n",
      "  Read 1:    55,357,710 bp\n",
      "  Read 2:    55,357,833 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t76\t2864.1\t0\t76\n",
      "4\t7\t716.0\t0\t7\n",
      "5\t4\t179.0\t0\t4\n",
      "6\t1\t44.8\t0\t1\n",
      "59\t2\t0.0\t1\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 85 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t66\t2864.1\t0\t66\n",
      "4\t17\t716.0\t0\t17\n",
      "5\t1\t179.0\t0\t1\n",
      "6\t1\t44.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7cexposed071717H_79_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7cexposed071717H_175_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7cexposed071717H_79_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7cexposed071717H_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7cexposed071717H_79_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7cexposed071717H_175_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7cexposed071717H_79_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7cexposed071717H_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[     8<-----] 00:00:10       363,121 reads  @     30.2 µs/read;   1.99 M reads/minute\n",
      "Finished in 11.00 s (30 us/read; 1.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            363,121\n",
      "  Read 1 with adapter:                     120 (0.0%)\n",
      "  Read 2 with adapter:                     184 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       363,121 (100.0%)\n",
      "\n",
      "Total basepairs processed:   219,325,084 bp\n",
      "  Read 1:   109,662,542 bp\n",
      "  Read 2:   109,662,542 bp\n",
      "Total written (filtered):    219,324,107 bp (100.0%)\n",
      "  Read 1:   109,662,162 bp\n",
      "  Read 2:   109,661,945 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 120 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t104\t5673.8\t0\t104\n",
      "4\t13\t1418.4\t0\t13\n",
      "5\t2\t354.6\t0\t2\n",
      "6\t1\t88.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 184 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t152\t5673.8\t0\t152\n",
      "4\t23\t1418.4\t0\t23\n",
      "5\t6\t354.6\t0\t6\n",
      "6\t2\t88.7\t0\t2\n",
      "7\t1\t22.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7hbaseline062217H_80_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7hbaseline062217H_176_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7hbaseline062217H_80_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7hbaseline062217H_176_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7hbaseline062217H_80_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7hbaseline062217H_176_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7hbaseline062217H_80_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7hbaseline062217H_176_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[   8<-------] 00:00:06       207,483 reads  @     32.4 µs/read;   1.85 M reads/minute\n",
      "Finished in 6.77 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            207,483\n",
      "  Read 1 with adapter:                      47 (0.0%)\n",
      "  Read 2 with adapter:                      83 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       207,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:   125,319,732 bp\n",
      "  Read 1:    62,659,866 bp\n",
      "  Read 2:    62,659,866 bp\n",
      "Total written (filtered):    125,319,099 bp (100.0%)\n",
      "  Read 1:    62,659,723 bp\n",
      "  Read 2:    62,659,376 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 47 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t3241.9\t0\t45\n",
      "4\t2\t810.5\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 83 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t67\t3241.9\t0\t67\n",
      "4\t10\t810.5\t0\t10\n",
      "5\t3\t202.6\t0\t3\n",
      "7\t1\t12.7\t0\t1\n",
      "8\t1\t3.2\t0\t1\n",
      "219\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7hexposed071717H_81_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7hexposed071717H_177_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7hexposed071717H_81_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7hexposed071717H_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7hexposed071717H_81_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7hexposed071717H_177_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7hexposed071717H_81_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7hexposed071717H_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       152,848 reads  @     29.1 µs/read;   2.06 M reads/minute\n",
      "Finished in 4.49 s (29 us/read; 2.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            152,848\n",
      "  Read 1 with adapter:                      55 (0.0%)\n",
      "  Read 2 with adapter:                      79 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       152,848 (100.0%)\n",
      "\n",
      "Total basepairs processed:    92,320,192 bp\n",
      "  Read 1:    46,160,096 bp\n",
      "  Read 2:    46,160,096 bp\n",
      "Total written (filtered):     92,319,766 bp (100.0%)\n",
      "  Read 1:    46,159,916 bp\n",
      "  Read 2:    46,159,850 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t2388.2\t0\t50\n",
      "4\t2\t597.1\t0\t2\n",
      "6\t1\t37.3\t0\t1\n",
      "7\t1\t9.3\t0\t1\n",
      "9\t1\t0.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 79 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t71\t2388.2\t0\t71\n",
      "4\t7\t597.1\t0\t7\n",
      "5\t1\t149.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7ibaseline062217_82_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7ibaseline062217_178_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7ibaseline062217_82_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7ibaseline062217_178_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7ibaseline062217_82_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7ibaseline062217_178_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7ibaseline062217_82_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7ibaseline062217_178_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[     8<-----] 00:00:11       408,058 reads  @     29.1 µs/read;   2.06 M reads/minute\n",
      "Finished in 11.91 s (29 us/read; 2.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            408,058\n",
      "  Read 1 with adapter:                      95 (0.0%)\n",
      "  Read 2 with adapter:                     151 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       408,058 (100.0%)\n",
      "\n",
      "Total basepairs processed:   246,467,032 bp\n",
      "  Read 1:   123,233,516 bp\n",
      "  Read 2:   123,233,516 bp\n",
      "Total written (filtered):    246,466,241 bp (100.0%)\n",
      "  Read 1:   123,233,204 bp\n",
      "  Read 2:   123,233,037 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 95 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t82\t6375.9\t0\t82\n",
      "4\t9\t1594.0\t0\t9\n",
      "5\t2\t398.5\t0\t2\n",
      "7\t1\t24.9\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 151 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t128\t6375.9\t0\t128\n",
      "4\t20\t1594.0\t0\t20\n",
      "5\t3\t398.5\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7iexposed071717_83_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7iexposed071717_179_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7iexposed071717_83_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7iexposed071717_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7iexposed071717_83_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P7iexposed071717_179_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7iexposed071717_83_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P7iexposed071717_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       192,900 reads  @     32.8 µs/read;   1.83 M reads/minute\n",
      "Finished in 6.38 s (33 us/read; 1.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            192,900\n",
      "  Read 1 with adapter:                      55 (0.0%)\n",
      "  Read 2 with adapter:                      90 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       192,900 (100.0%)\n",
      "\n",
      "Total basepairs processed:   116,511,600 bp\n",
      "  Read 1:    58,255,800 bp\n",
      "  Read 2:    58,255,800 bp\n",
      "Total written (filtered):    116,511,108 bp (100.0%)\n",
      "  Read 1:    58,255,623 bp\n",
      "  Read 2:    58,255,485 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t3014.1\t0\t49\n",
      "4\t3\t753.5\t0\t3\n",
      "5\t1\t188.4\t0\t1\n",
      "6\t1\t47.1\t0\t1\n",
      "7\t1\t11.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 90 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t3014.1\t0\t77\n",
      "4\t9\t753.5\t0\t9\n",
      "5\t2\t188.4\t0\t2\n",
      "6\t1\t47.1\t0\t1\n",
      "32\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8bbaseline062217H_84_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8bbaseline062217H_180_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8bbaseline062217H_84_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8bbaseline062217H_180_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8bbaseline062217H_84_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8bbaseline062217H_180_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8bbaseline062217H_84_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8bbaseline062217H_180_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       159,113 reads  @     32.4 µs/read;   1.85 M reads/minute\n",
      "Finished in 5.20 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            159,113\n",
      "  Read 1 with adapter:                      33 (0.0%)\n",
      "  Read 2 with adapter:                      59 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       159,113 (100.0%)\n",
      "\n",
      "Total basepairs processed:    96,104,252 bp\n",
      "  Read 1:    48,052,126 bp\n",
      "  Read 2:    48,052,126 bp\n",
      "Total written (filtered):     96,103,914 bp (100.0%)\n",
      "  Read 1:    48,052,004 bp\n",
      "  Read 2:    48,051,910 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 33 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t2486.1\t0\t28\n",
      "4\t1\t621.5\t0\t1\n",
      "5\t1\t155.4\t0\t1\n",
      "6\t1\t38.8\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 59 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t2486.1\t0\t50\n",
      "4\t5\t621.5\t0\t5\n",
      "5\t1\t155.4\t0\t1\n",
      "6\t2\t38.8\t0\t2\n",
      "29\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8bexposed071717H_85_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8bexposed071717H_181_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8bexposed071717H_85_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8bexposed071717H_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8bexposed071717H_85_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8bexposed071717H_181_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8bexposed071717H_85_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8bexposed071717H_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       198,536 reads  @     31.2 µs/read;   1.92 M reads/minute\n",
      "Finished in 6.24 s (31 us/read; 1.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            198,536\n",
      "  Read 1 with adapter:                      71 (0.0%)\n",
      "  Read 2 with adapter:                      90 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       198,536 (100.0%)\n",
      "\n",
      "Total basepairs processed:   119,915,744 bp\n",
      "  Read 1:    59,957,872 bp\n",
      "  Read 2:    59,957,872 bp\n",
      "Total written (filtered):    119,914,984 bp (100.0%)\n",
      "  Read 1:    59,957,626 bp\n",
      "  Read 2:    59,957,358 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 71 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t3102.1\t0\t59\n",
      "4\t7\t775.5\t0\t7\n",
      "5\t2\t193.9\t0\t2\n",
      "6\t1\t48.5\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 90 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t3102.1\t0\t72\n",
      "4\t11\t775.5\t0\t11\n",
      "5\t3\t193.9\t0\t3\n",
      "6\t2\t48.5\t0\t2\n",
      "8\t1\t3.0\t0\t1\n",
      "219\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8dbaseline062217H_86_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8dbaseline062217H_182_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8dbaseline062217H_86_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8dbaseline062217H_182_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8dbaseline062217H_86_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8dbaseline062217H_182_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8dbaseline062217H_86_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8dbaseline062217H_182_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       104,577 reads  @     34.3 µs/read;   1.75 M reads/minute\n",
      "Finished in 3.62 s (35 us/read; 1.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,577\n",
      "  Read 1 with adapter:                      20 (0.0%)\n",
      "  Read 2 with adapter:                      40 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       104,577 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,164,508 bp\n",
      "  Read 1:    31,582,254 bp\n",
      "  Read 2:    31,582,254 bp\n",
      "Total written (filtered):     63,164,308 bp (100.0%)\n",
      "  Read 1:    31,582,183 bp\n",
      "  Read 2:    31,582,125 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t1634.0\t0\t18\n",
      "4\t1\t408.5\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 40 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t1634.0\t0\t32\n",
      "4\t7\t408.5\t0\t7\n",
      "5\t1\t102.1\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8dexposed071717H_87_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8dexposed071717H_183_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8dexposed071717H_87_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8dexposed071717H_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8dexposed071717H_87_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8dexposed071717H_183_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8dexposed071717H_87_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8dexposed071717H_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04       122,365 reads  @     34.1 µs/read;   1.76 M reads/minute\n",
      "Finished in 4.22 s (35 us/read; 1.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            122,365\n",
      "  Read 1 with adapter:                      41 (0.0%)\n",
      "  Read 2 with adapter:                      32 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       122,365 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,908,460 bp\n",
      "  Read 1:    36,954,230 bp\n",
      "  Read 2:    36,954,230 bp\n",
      "Total written (filtered):     73,908,223 bp (100.0%)\n",
      "  Read 1:    36,954,105 bp\n",
      "  Read 2:    36,954,118 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 41 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t1912.0\t0\t39\n",
      "4\t2\t478.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 32 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t1912.0\t0\t22\n",
      "4\t6\t478.0\t0\t6\n",
      "5\t2\t119.5\t0\t2\n",
      "6\t2\t29.9\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8gbaseline062217_88_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8gbaseline062217_184_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8gbaseline062217_88_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8gbaseline062217_184_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8gbaseline062217_88_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8gbaseline062217_184_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8gbaseline062217_88_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8gbaseline062217_184_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       150,163 reads  @     33.1 µs/read;   1.81 M reads/minute\n",
      "Finished in 5.00 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            150,163\n",
      "  Read 1 with adapter:                      45 (0.0%)\n",
      "  Read 2 with adapter:                      50 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       150,163 (100.0%)\n",
      "\n",
      "Total basepairs processed:    90,698,452 bp\n",
      "  Read 1:    45,349,226 bp\n",
      "  Read 2:    45,349,226 bp\n",
      "Total written (filtered):     90,698,127 bp (100.0%)\n",
      "  Read 1:    45,349,065 bp\n",
      "  Read 2:    45,349,062 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 45 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t38\t2346.3\t0\t38\n",
      "4\t3\t586.6\t0\t3\n",
      "5\t2\t146.6\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 50 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t41\t2346.3\t0\t41\n",
      "4\t6\t586.6\t0\t6\n",
      "5\t1\t146.6\t0\t1\n",
      "6\t2\t36.7\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8gexposed071717_89_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8gexposed071717_185_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8gexposed071717_89_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8gexposed071717_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8gexposed071717_89_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P8gexposed071717_185_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8gexposed071717_89_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P8gexposed071717_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       154,818 reads  @     33.1 µs/read;   1.81 M reads/minute\n",
      "Finished in 5.17 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            154,818\n",
      "  Read 1 with adapter:                      75 (0.0%)\n",
      "  Read 2 with adapter:                      44 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       154,818 (100.0%)\n",
      "\n",
      "Total basepairs processed:    93,510,072 bp\n",
      "  Read 1:    46,755,036 bp\n",
      "  Read 2:    46,755,036 bp\n",
      "Total written (filtered):     93,509,672 bp (100.0%)\n",
      "  Read 1:    46,754,781 bp\n",
      "  Read 2:    46,754,891 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 75 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t71\t2419.0\t0\t71\n",
      "4\t2\t604.8\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "24\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 44 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t2419.0\t0\t34\n",
      "4\t8\t604.8\t0\t8\n",
      "5\t1\t151.2\t0\t1\n",
      "6\t1\t37.8\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9ebaseline062217_90_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9ebaseline062217_186_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9ebaseline062217_90_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9ebaseline062217_186_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9ebaseline062217_90_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9ebaseline062217_186_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9ebaseline062217_90_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9ebaseline062217_186_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       173,337 reads  @     29.5 µs/read;   2.04 M reads/minute\n",
      "Finished in 5.14 s (30 us/read; 2.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            173,337\n",
      "  Read 1 with adapter:                      33 (0.0%)\n",
      "  Read 2 with adapter:                      63 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       173,337 (100.0%)\n",
      "\n",
      "Total basepairs processed:   104,695,548 bp\n",
      "  Read 1:    52,347,774 bp\n",
      "  Read 2:    52,347,774 bp\n",
      "Total written (filtered):    104,695,244 bp (100.0%)\n",
      "  Read 1:    52,347,668 bp\n",
      "  Read 2:    52,347,576 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 33 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t2708.4\t0\t29\n",
      "4\t3\t677.1\t0\t3\n",
      "7\t1\t10.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 63 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t2708.4\t0\t54\n",
      "4\t9\t677.1\t0\t9\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9eexposed071727_91_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9eexposed071727_187_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9eexposed071727_91_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9eexposed071727_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9eexposed071727_91_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9eexposed071727_187_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9eexposed071727_91_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9eexposed071727_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       152,517 reads  @     33.0 µs/read;   1.82 M reads/minute\n",
      "Finished in 5.07 s (33 us/read; 1.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            152,517\n",
      "  Read 1 with adapter:                      65 (0.0%)\n",
      "  Read 2 with adapter:                      54 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       152,517 (100.0%)\n",
      "\n",
      "Total basepairs processed:    92,120,268 bp\n",
      "  Read 1:    46,060,134 bp\n",
      "  Read 2:    46,060,134 bp\n",
      "Total written (filtered):     92,119,898 bp (100.0%)\n",
      "  Read 1:    46,059,937 bp\n",
      "  Read 2:    46,059,961 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 65 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t64\t2383.1\t0\t64\n",
      "5\t1\t148.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 54 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t2383.1\t0\t47\n",
      "4\t5\t595.8\t0\t5\n",
      "5\t1\t148.9\t0\t1\n",
      "7\t1\t9.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9gbaseline062217_92_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9gbaseline062217_188_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9gbaseline062217_92_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9gbaseline062217_188_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9gbaseline062217_92_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9gbaseline062217_188_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9gbaseline062217_92_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9gbaseline062217_188_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        50,141 reads  @     39.1 µs/read;   1.54 M reads/minute\n",
      "Finished in 2.00 s (40 us/read; 1.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,141\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                      20 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,141 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,285,164 bp\n",
      "  Read 1:    15,142,582 bp\n",
      "  Read 2:    15,142,582 bp\n",
      "Total written (filtered):     30,285,079 bp (100.0%)\n",
      "  Read 1:    15,142,561 bp\n",
      "  Read 2:    15,142,518 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 6 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t783.5\t0\t4\n",
      "4\t1\t195.9\t0\t1\n",
      "5\t1\t49.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 20 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t783.5\t0\t18\n",
      "5\t2\t49.0\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9gexposed071717_93_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9gexposed071717_189_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9gexposed071717_93_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9gexposed071717_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9gexposed071717_93_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9gexposed071717_189_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9gexposed071717_93_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9gexposed071717_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:06       235,115 reads  @     29.7 µs/read;   2.02 M reads/minute\n",
      "Finished in 7.01 s (30 us/read; 2.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            235,115\n",
      "  Read 1 with adapter:                      42 (0.0%)\n",
      "  Read 2 with adapter:                      75 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       235,115 (100.0%)\n",
      "\n",
      "Total basepairs processed:   142,009,460 bp\n",
      "  Read 1:    71,004,730 bp\n",
      "  Read 2:    71,004,730 bp\n",
      "Total written (filtered):    142,009,059 bp (100.0%)\n",
      "  Read 1:    71,004,572 bp\n",
      "  Read 2:    71,004,487 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 42 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t3673.7\t0\t32\n",
      "4\t4\t918.4\t0\t4\n",
      "5\t2\t229.6\t0\t2\n",
      "6\t1\t57.4\t0\t1\n",
      "7\t1\t14.4\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 75 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t63\t3673.7\t0\t63\n",
      "4\t8\t918.4\t0\t8\n",
      "5\t3\t229.6\t0\t3\n",
      "7\t1\t14.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9ibaseline062217_94_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9ibaseline062217_190_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9ibaseline062217_94_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9ibaseline062217_190_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9ibaseline062217_94_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9ibaseline062217_190_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9ibaseline062217_94_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9ibaseline062217_190_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        82,092 reads  @     36.7 µs/read;   1.63 M reads/minute\n",
      "Finished in 3.05 s (37 us/read; 1.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,092\n",
      "  Read 1 with adapter:                      23 (0.0%)\n",
      "  Read 2 with adapter:                      30 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        82,092 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,583,568 bp\n",
      "  Read 1:    24,791,784 bp\n",
      "  Read 2:    24,791,784 bp\n",
      "Total written (filtered):     49,583,400 bp (100.0%)\n",
      "  Read 1:    24,791,711 bp\n",
      "  Read 2:    24,791,689 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t1282.7\t0\t21\n",
      "4\t1\t320.7\t0\t1\n",
      "6\t1\t20.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 30 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t1282.7\t0\t25\n",
      "4\t5\t320.7\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9iexposed071717_95_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9iexposed071717_191_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9iexposed071717_95_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9iexposed071717_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9iexposed071717_95_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/P9iexposed071717_191_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9iexposed071717_95_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/P9iexposed071717_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/neg_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/neg_156_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/neg_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/neg_156_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/neg_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-zc7dl2ke/neg_156_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/neg_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-uqoa63re/6c6b568f-37df-4655-86d7-37cbcef0065f/data/neg_156_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00         2,125 reads  @    196.1 µs/read;   0.31 M reads/minute\n",
      "Finished in 0.48 s (225 us/read; 0.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              2,125\n",
      "  Read 1 with adapter:                       1 (0.0%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         2,125 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,283,500 bp\n",
      "  Read 1:       641,750 bp\n",
      "  Read 2:       641,750 bp\n",
      "Total written (filtered):      1,283,494 bp (100.0%)\n",
      "  Read 1:       641,747 bp\n",
      "  Read 2:       641,747 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 1 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t33.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 1 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t33.2\t0\t1\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: Folder_Rosales_AcropraDis//trim_Rosales_AcropraDis.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#trim PCR primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences Folder_Rosales_AcropraDis/Import_Rosales_AcropraDis.qza \\\n",
    "--p-cores 2 \\\n",
    "--p-front-f GTGYCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACNVGGGTWTCTAAT \\\n",
    "--o-trimmed-sequences Folder_Rosales_AcropraDis//trim_Rosales_AcropraDis.qza \\\n",
    "--verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_Rosales_AcropraDis//trim_Rosales_AcropraDis.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data Folder_Rosales_AcropraDis//trim_Rosales_AcropraDis.qza \\\n",
    "--o-visualization Folder_Rosales_AcropraDis//trim_Rosales_AcropraDis.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_Rosales_AcropraDis/table_Rosales_AcropraDis.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_Rosales_AcropraDis/rep_seqs_Rosales_AcropraDis.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: Folder_Rosales_AcropraDis/dada_stats_Rosales_AcropraDis.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime dada2 denoise-single \\\n",
    "--i-demultiplexed-seqs Folder_Rosales_AcropraDis/trim_Rosales_AcropraDis.qza \\\n",
    "--p-trunc-len 130 \\\n",
    "--p-n-threads 10  \\\n",
    "--p-max-ee 3 \\\n",
    "--o-representative-sequences Folder_Rosales_AcropraDis/rep_seqs_Rosales_AcropraDis.qza \\\n",
    "--o-table Folder_Rosales_AcropraDis/table_Rosales_AcropraDis.qza \\\n",
    "--o-denoising-stats Folder_Rosales_AcropraDis/dada_stats_Rosales_AcropraDis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[31m\u001b[1mThere was an issue with loading the file metadata/ as metadata:\r\n",
      "\r\n",
      "  Metadata file path doesn't exist, or the path points to something other than a file. Please check that the path exists, has read permissions, and points to a regular file (not a directory): metadata/\r\n",
      "\r\n",
      "  There may be more errors present in the metadata file. To get a full report, sample/feature metadata files can be validated with Keemei: https://keemei.qiime2.org\r\n",
      "\r\n",
      "  Find details on QIIME 2 metadata requirements here: https://docs.qiime2.org/2020.11/tutorials/metadata/\u001b[0m\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table summarize \\\n",
    "  --i-table  Folder_Rosales_AcropraDis/table_Rosales_AcropraDis.qza \\\n",
    "  --o-visualization  Folder_Rosales_AcropraDis/table_Rosales_AcropraDis.qzv \\\n",
    "  --m-sample-metadata-file metadata/\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
