{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Meyer_FRT2017/Meyer_FRT2017_fastqfiles\n",
      "/space/home/rosales/projects/2020_SCTLD_meta-analysis\n"
     ]
    }
   ],
   "source": [
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Meyer_FRT2017/Meyer_FRT2017_fastqfiles/\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_Meyer_FRT2017.txt\n",
    "%cd /space/home/rosales/projects/2020_SCTLD_meta-analysis/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as ps\n",
    "import numpy as np\n",
    "\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_Meyer_FRT2017.txt',\n",
    "                                    names=header_list)\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_S\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "dir_path=\"/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Meyer_FRT2017/Meyer_FRT2017_fastqfiles/\"\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Meyer_FRT2017.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir Folder_Meyer_FRT2017"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/anaconda2/envs/qiime2-2020.2/bin/qiime\r\n"
     ]
    }
   ],
   "source": [
    "!which qiime"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Meyer_FRT2017.csv as PairedEndFastqManifestPhred33V2 to Folder_Meyer_FRT2017/Import_Meyer_FRT2017.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime tools import \\\n",
    "--output-path Folder_Meyer_FRT2017/Import_Meyer_FRT2017.qza \\\n",
    "--type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "--input-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Meyer_FRT2017.csv \\\n",
    "--input-format PairedEndFastqManifestPhred33V2\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Import_Meyer_FRT2017.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data Folder_Meyer_FRT2017/Import_Meyer_FRT2017.qza \\\n",
    "--o-visualization Import_Meyer_FRT2017.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/10FTL_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/10FTL_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/10FTL_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/10FTL_62_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/10FTL_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/10FTL_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/10FTL_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/10FTL_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       163,206 reads  @     24.2 µs/read;   2.48 M reads/minute\n",
      "Finished in 4.00 s (25 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            163,206\n",
      "  Read 1 with adapter:                     229 (0.1%)\n",
      "  Read 2 with adapter:                     347 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       163,206 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,288,212 bp\n",
      "  Read 1:    24,644,106 bp\n",
      "  Read 2:    24,644,106 bp\n",
      "Total written (filtered):     49,286,358 bp (100.0%)\n",
      "  Read 1:    24,643,397 bp\n",
      "  Read 2:    24,642,961 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 229 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t214\t2550.1\t0\t214\n",
      "4\t11\t637.5\t0\t11\n",
      "5\t1\t159.4\t0\t1\n",
      "6\t3\t39.8\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 347 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t279\t2550.1\t0\t279\n",
      "4\t45\t637.5\t0\t45\n",
      "5\t13\t159.4\t0\t13\n",
      "6\t8\t39.8\t0\t8\n",
      "7\t1\t10.0\t0\t1\n",
      "8\t1\t2.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/11FTL_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/11FTL_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/11FTL_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/11FTL_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/11FTL_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/11FTL_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/11FTL_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/11FTL_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        16,564 reads  @     59.3 µs/read;   1.01 M reads/minute\n",
      "Finished in 1.03 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,564\n",
      "  Read 1 with adapter:                      70 (0.4%)\n",
      "  Read 2 with adapter:                     102 (0.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        16,564 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,002,328 bp\n",
      "  Read 1:     2,501,164 bp\n",
      "  Read 2:     2,501,164 bp\n",
      "Total written (filtered):      5,001,772 bp (100.0%)\n",
      "  Read 1:     2,500,938 bp\n",
      "  Read 2:     2,500,834 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 70 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t258.8\t0\t56\n",
      "4\t12\t64.7\t0\t12\n",
      "5\t2\t16.2\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 102 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t258.8\t0\t80\n",
      "4\t21\t64.7\t0\t21\n",
      "6\t1\t4.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/12FTL_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/12FTL_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/12FTL_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/12FTL_64_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/12FTL_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/12FTL_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/12FTL_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/12FTL_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,531 reads  @     40.3 µs/read;   1.49 M reads/minute\n",
      "Finished in 1.36 s (42 us/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,531\n",
      "  Read 1 with adapter:                     232 (0.7%)\n",
      "  Read 2 with adapter:                     290 (0.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,531 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,824,362 bp\n",
      "  Read 1:     4,912,181 bp\n",
      "  Read 2:     4,912,181 bp\n",
      "Total written (filtered):      9,822,464 bp (100.0%)\n",
      "  Read 1:     4,911,415 bp\n",
      "  Read 2:     4,911,049 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 232 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t191\t508.3\t0\t191\n",
      "4\t32\t127.1\t0\t32\n",
      "5\t5\t31.8\t0\t5\n",
      "6\t3\t7.9\t0\t3\n",
      "22\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 290 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t205\t508.3\t0\t205\n",
      "4\t59\t127.1\t0\t59\n",
      "5\t16\t31.8\t0\t16\n",
      "6\t9\t7.9\t0\t9\n",
      "147\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/13FTL_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/13FTL_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/13FTL_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/13FTL_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/13FTL_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/13FTL_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/13FTL_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/13FTL_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        77,087 reads  @     28.7 µs/read;   2.09 M reads/minute\n",
      "Finished in 2.26 s (29 us/read; 2.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,087\n",
      "  Read 1 with adapter:                     192 (0.2%)\n",
      "  Read 2 with adapter:                     242 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        77,087 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,280,274 bp\n",
      "  Read 1:    11,640,137 bp\n",
      "  Read 2:    11,640,137 bp\n",
      "Total written (filtered):     23,278,855 bp (100.0%)\n",
      "  Read 1:    11,639,530 bp\n",
      "  Read 2:    11,639,325 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 192 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t166\t1204.5\t0\t166\n",
      "4\t21\t301.1\t0\t21\n",
      "5\t5\t75.3\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 242 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t183\t1204.5\t0\t183\n",
      "4\t40\t301.1\t0\t40\n",
      "5\t12\t75.3\t0\t12\n",
      "6\t6\t18.8\t0\t6\n",
      "7\t1\t4.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/14FTL_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/14FTL_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/14FTL_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/14FTL_66_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/14FTL_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/14FTL_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/14FTL_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/14FTL_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        52,715 reads  @     30.2 µs/read;   1.99 M reads/minute\n",
      "Finished in 1.64 s (31 us/read; 1.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,715\n",
      "  Read 1 with adapter:                     165 (0.3%)\n",
      "  Read 2 with adapter:                     251 (0.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,715 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,919,930 bp\n",
      "  Read 1:     7,959,965 bp\n",
      "  Read 2:     7,959,965 bp\n",
      "Total written (filtered):     15,918,524 bp (100.0%)\n",
      "  Read 1:     7,959,412 bp\n",
      "  Read 2:     7,959,112 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 165 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t132\t823.7\t0\t132\n",
      "4\t17\t205.9\t0\t17\n",
      "5\t11\t51.5\t0\t11\n",
      "6\t1\t12.9\t0\t1\n",
      "7\t4\t3.2\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 251 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t193\t823.7\t0\t193\n",
      "4\t45\t205.9\t0\t45\n",
      "5\t8\t51.5\t0\t8\n",
      "6\t4\t12.9\t0\t4\n",
      "30\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/15FTL_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/15FTL_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/15FTL_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/15FTL_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/15FTL_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/15FTL_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/15FTL_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/15FTL_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        59,267 reads  @     27.0 µs/read;   2.22 M reads/minute\n",
      "Finished in 1.65 s (28 us/read; 2.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,267\n",
      "  Read 1 with adapter:                     103 (0.2%)\n",
      "  Read 2 with adapter:                     210 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        59,267 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,898,634 bp\n",
      "  Read 1:     8,949,317 bp\n",
      "  Read 2:     8,949,317 bp\n",
      "Total written (filtered):     17,897,495 bp (100.0%)\n",
      "  Read 1:     8,948,985 bp\n",
      "  Read 2:     8,948,510 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t86\t926.0\t0\t86\n",
      "4\t12\t231.5\t0\t12\n",
      "5\t4\t57.9\t0\t4\n",
      "6\t1\t14.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 210 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t155\t926.0\t0\t155\n",
      "4\t38\t231.5\t0\t38\n",
      "5\t6\t57.9\t0\t6\n",
      "6\t5\t14.5\t0\t5\n",
      "7\t1\t3.6\t0\t1\n",
      "8\t1\t0.9\t0\t1\n",
      "27\t2\t0.0\t2\t0 2\n",
      "29\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/1FTL_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/1FTL_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/1FTL_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/1FTL_68_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/1FTL_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/1FTL_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/1FTL_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/1FTL_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       138,377 reads  @     23.9 µs/read;   2.51 M reads/minute\n",
      "Finished in 3.36 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            138,377\n",
      "  Read 1 with adapter:                     461 (0.3%)\n",
      "  Read 2 with adapter:                     614 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       138,377 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,789,854 bp\n",
      "  Read 1:    20,894,927 bp\n",
      "  Read 2:    20,894,927 bp\n",
      "Total written (filtered):     41,786,209 bp (100.0%)\n",
      "  Read 1:    20,893,453 bp\n",
      "  Read 2:    20,892,756 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 461 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t401\t2162.1\t0\t401\n",
      "4\t41\t540.5\t0\t41\n",
      "5\t12\t135.1\t0\t12\n",
      "6\t2\t33.8\t0\t2\n",
      "7\t5\t8.4\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 614 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t449\t2162.1\t0\t449\n",
      "4\t119\t540.5\t0\t119\n",
      "5\t29\t135.1\t0\t29\n",
      "6\t10\t33.8\t0\t10\n",
      "8\t3\t2.1\t0\t3\n",
      "29\t1\t0.0\t2\t0 1\n",
      "30\t3\t0.0\t2\t3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/2FTL_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/2FTL_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/2FTL_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/2FTL_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/2FTL_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/2FTL_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/2FTL_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/2FTL_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       126,210 reads  @     24.9 µs/read;   2.40 M reads/minute\n",
      "Finished in 3.21 s (25 us/read; 2.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            126,210\n",
      "  Read 1 with adapter:                     200 (0.2%)\n",
      "  Read 2 with adapter:                     274 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       126,210 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,115,420 bp\n",
      "  Read 1:    19,057,710 bp\n",
      "  Read 2:    19,057,710 bp\n",
      "Total written (filtered):     38,113,856 bp (100.0%)\n",
      "  Read 1:    19,057,067 bp\n",
      "  Read 2:    19,056,789 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 200 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t170\t1972.0\t0\t170\n",
      "4\t20\t493.0\t0\t20\n",
      "5\t8\t123.3\t0\t8\n",
      "6\t1\t30.8\t0\t1\n",
      "7\t1\t7.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 274 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t224\t1972.0\t0\t224\n",
      "4\t34\t493.0\t0\t34\n",
      "5\t8\t123.3\t0\t8\n",
      "6\t6\t30.8\t0\t6\n",
      "7\t1\t7.7\t0\t1\n",
      "30\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/3FTL_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/3FTL_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/3FTL_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/3FTL_70_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/3FTL_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/3FTL_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/3FTL_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/3FTL_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       100,050 reads  @     23.4 µs/read;   2.57 M reads/minute\n",
      "Finished in 2.38 s (24 us/read; 2.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,050\n",
      "  Read 1 with adapter:                     333 (0.3%)\n",
      "  Read 2 with adapter:                     469 (0.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,050 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,215,100 bp\n",
      "  Read 1:    15,107,550 bp\n",
      "  Read 2:    15,107,550 bp\n",
      "Total written (filtered):     30,212,393 bp (100.0%)\n",
      "  Read 1:    15,106,469 bp\n",
      "  Read 2:    15,105,924 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 333 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t276\t1563.3\t0\t276\n",
      "4\t39\t390.8\t0\t39\n",
      "5\t14\t97.7\t0\t14\n",
      "6\t1\t24.4\t0\t1\n",
      "7\t3\t6.1\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 469 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t370\t1563.3\t0\t370\n",
      "4\t74\t390.8\t0\t74\n",
      "5\t10\t97.7\t0\t10\n",
      "6\t10\t24.4\t0\t10\n",
      "8\t2\t1.5\t0\t2\n",
      "30\t1\t0.0\t2\t0 1\n",
      "32\t2\t0.0\t2\t0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/4FTL_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/4FTL_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/4FTL_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/4FTL_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/4FTL_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/4FTL_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/4FTL_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/4FTL_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00         9,678 reads  @     80.7 µs/read;   0.74 M reads/minute\n",
      "Finished in 0.83 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              9,678\n",
      "  Read 1 with adapter:                      55 (0.6%)\n",
      "  Read 2 with adapter:                      80 (0.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         9,678 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,922,756 bp\n",
      "  Read 1:     1,461,378 bp\n",
      "  Read 2:     1,461,378 bp\n",
      "Total written (filtered):      2,922,313 bp (100.0%)\n",
      "  Read 1:     1,461,194 bp\n",
      "  Read 2:     1,461,119 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t151.2\t0\t43\n",
      "4\t6\t37.8\t0\t6\n",
      "5\t5\t9.5\t0\t5\n",
      "6\t1\t2.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 80 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t65\t151.2\t0\t65\n",
      "4\t12\t37.8\t0\t12\n",
      "5\t2\t9.5\t0\t2\n",
      "6\t1\t2.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/5FTL_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/5FTL_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/5FTL_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/5FTL_72_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/5FTL_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/5FTL_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/5FTL_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/5FTL_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        94,544 reads  @     24.5 µs/read;   2.44 M reads/minute\n",
      "Finished in 2.36 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,544\n",
      "  Read 1 with adapter:                     223 (0.2%)\n",
      "  Read 2 with adapter:                     368 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        94,544 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,552,288 bp\n",
      "  Read 1:    14,276,144 bp\n",
      "  Read 2:    14,276,144 bp\n",
      "Total written (filtered):     28,550,315 bp (100.0%)\n",
      "  Read 1:    14,275,434 bp\n",
      "  Read 2:    14,274,881 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 223 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t193\t1477.2\t0\t193\n",
      "4\t23\t369.3\t0\t23\n",
      "5\t4\t92.3\t0\t4\n",
      "6\t2\t23.1\t0\t2\n",
      "7\t1\t5.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 368 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t294\t1477.2\t0\t294\n",
      "4\t47\t369.3\t0\t47\n",
      "5\t12\t92.3\t0\t12\n",
      "6\t12\t23.1\t0\t12\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t1\n",
      "30\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/6FTL_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/6FTL_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/6FTL_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/6FTL_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/6FTL_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/6FTL_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/6FTL_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/6FTL_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        37,793 reads  @     34.4 µs/read;   1.74 M reads/minute\n",
      "Finished in 1.35 s (36 us/read; 1.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,793\n",
      "  Read 1 with adapter:                     164 (0.4%)\n",
      "  Read 2 with adapter:                     254 (0.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        37,793 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,413,486 bp\n",
      "  Read 1:     5,706,743 bp\n",
      "  Read 2:     5,706,743 bp\n",
      "Total written (filtered):     11,412,104 bp (100.0%)\n",
      "  Read 1:     5,706,219 bp\n",
      "  Read 2:     5,705,885 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 164 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t141\t590.5\t0\t141\n",
      "4\t17\t147.6\t0\t17\n",
      "5\t4\t36.9\t0\t4\n",
      "6\t1\t9.2\t0\t1\n",
      "7\t1\t2.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 254 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t190\t590.5\t0\t190\n",
      "4\t45\t147.6\t0\t45\n",
      "5\t6\t36.9\t0\t6\n",
      "6\t13\t9.2\t0\t13\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/7FTL_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/7FTL_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/7FTL_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/7FTL_74_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/7FTL_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/7FTL_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/7FTL_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/7FTL_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       122,318 reads  @     25.3 µs/read;   2.37 M reads/minute\n",
      "Finished in 3.14 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            122,318\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                     111 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       122,318 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,940,036 bp\n",
      "  Read 1:    18,470,018 bp\n",
      "  Read 2:    18,470,018 bp\n",
      "Total written (filtered):     36,939,598 bp (100.0%)\n",
      "  Read 1:    18,469,990 bp\n",
      "  Read 2:    18,469,608 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1911.2\t0\t8\n",
      "4\t1\t477.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 111 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t101\t1911.2\t0\t101\n",
      "4\t3\t477.8\t0\t3\n",
      "5\t2\t119.5\t0\t2\n",
      "6\t1\t29.9\t0\t1\n",
      "7\t1\t7.5\t0\t1\n",
      "18\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/8FTL_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/8FTL_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/8FTL_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/8FTL_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/8FTL_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/8FTL_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/8FTL_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/8FTL_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       151,987 reads  @     24.8 µs/read;   2.42 M reads/minute\n",
      "Finished in 3.83 s (25 us/read; 2.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            151,987\n",
      "  Read 1 with adapter:                     445 (0.3%)\n",
      "  Read 2 with adapter:                     664 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       151,987 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,900,074 bp\n",
      "  Read 1:    22,950,037 bp\n",
      "  Read 2:    22,950,037 bp\n",
      "Total written (filtered):     45,896,363 bp (100.0%)\n",
      "  Read 1:    22,948,588 bp\n",
      "  Read 2:    22,947,775 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 445 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t374\t2374.8\t0\t374\n",
      "4\t42\t593.7\t0\t42\n",
      "5\t17\t148.4\t0\t17\n",
      "6\t10\t37.1\t0\t10\n",
      "7\t2\t9.3\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 664 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t507\t2374.8\t0\t507\n",
      "4\t116\t593.7\t0\t116\n",
      "5\t18\t148.4\t0\t18\n",
      "6\t18\t37.1\t0\t18\n",
      "7\t1\t9.3\t0\t1\n",
      "8\t1\t2.3\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t0 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/9FTL_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/9FTL_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/9FTL_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/9FTL_76_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/9FTL_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/9FTL_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/9FTL_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/9FTL_76_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        72,044 reads  @     27.4 µs/read;   2.19 M reads/minute\n",
      "Finished in 2.02 s (28 us/read; 2.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,044\n",
      "  Read 1 with adapter:                     331 (0.5%)\n",
      "  Read 2 with adapter:                     457 (0.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        72,044 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,757,288 bp\n",
      "  Read 1:    10,878,644 bp\n",
      "  Read 2:    10,878,644 bp\n",
      "Total written (filtered):     21,754,536 bp (100.0%)\n",
      "  Read 1:    10,877,566 bp\n",
      "  Read 2:    10,876,970 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 331 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t276\t1125.7\t0\t276\n",
      "4\t36\t281.4\t0\t36\n",
      "5\t11\t70.4\t0\t11\n",
      "6\t5\t17.6\t0\t5\n",
      "7\t3\t4.4\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 457 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t358\t1125.7\t0\t358\n",
      "4\t75\t281.4\t0\t75\n",
      "5\t14\t70.4\t0\t14\n",
      "6\t5\t17.6\t0\t5\n",
      "7\t1\t4.4\t0\t1\n",
      "8\t2\t1.1\t0\t2\n",
      "30\t1\t0.0\t2\t1\n",
      "147\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10A_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10A_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10A_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10A_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10A_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10A_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10A_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10A_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       175,233 reads  @     23.2 µs/read;   2.59 M reads/minute\n",
      "Finished in 4.10 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            175,233\n",
      "  Read 1 with adapter:                     347 (0.2%)\n",
      "  Read 2 with adapter:                     413 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       175,233 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,920,366 bp\n",
      "  Read 1:    26,460,183 bp\n",
      "  Read 2:    26,460,183 bp\n",
      "Total written (filtered):     52,917,811 bp (100.0%)\n",
      "  Read 1:    26,459,025 bp\n",
      "  Read 2:    26,458,786 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 347 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t301\t2738.0\t0\t301\n",
      "4\t24\t684.5\t0\t24\n",
      "5\t16\t171.1\t0\t16\n",
      "6\t3\t42.8\t0\t3\n",
      "7\t1\t10.7\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "43\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 413 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t321\t2738.0\t0\t321\n",
      "4\t59\t684.5\t0\t59\n",
      "5\t17\t171.1\t0\t17\n",
      "6\t15\t42.8\t0\t15\n",
      "23\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10B_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10B_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10B_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10B_78_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10B_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10B_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10B_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10B_78_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       145,483 reads  @     23.9 µs/read;   2.51 M reads/minute\n",
      "Finished in 3.52 s (24 us/read; 2.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            145,483\n",
      "  Read 1 with adapter:                     264 (0.2%)\n",
      "  Read 2 with adapter:                     411 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       145,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,935,866 bp\n",
      "  Read 1:    21,967,933 bp\n",
      "  Read 2:    21,967,933 bp\n",
      "Total written (filtered):     43,933,300 bp (100.0%)\n",
      "  Read 1:    21,967,091 bp\n",
      "  Read 2:    21,966,209 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 264 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t226\t2273.2\t0\t226\n",
      "4\t27\t568.3\t0\t27\n",
      "5\t10\t142.1\t0\t10\n",
      "6\t1\t35.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 411 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t304\t2273.2\t0\t304\n",
      "4\t63\t568.3\t0\t63\n",
      "5\t22\t142.1\t0\t22\n",
      "6\t7\t35.5\t0\t7\n",
      "7\t5\t8.9\t0\t5\n",
      "8\t1\t2.2\t0\t1\n",
      "25\t1\t0.0\t2\t0 1\n",
      "27\t2\t0.0\t2\t1 1\n",
      "28\t2\t0.0\t2\t0 2\n",
      "29\t3\t0.0\t2\t1 2\n",
      "143\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10C_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10C_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10C_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10C_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10C_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl10C_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10C_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl10C_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:03       162,793 reads  @     22.7 µs/read;   2.65 M reads/minute\n",
      "Finished in 3.74 s (23 us/read; 2.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            162,793\n",
      "  Read 1 with adapter:                     325 (0.2%)\n",
      "  Read 2 with adapter:                     408 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       162,793 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,163,486 bp\n",
      "  Read 1:    24,581,743 bp\n",
      "  Read 2:    24,581,743 bp\n",
      "Total written (filtered):     49,160,929 bp (100.0%)\n",
      "  Read 1:    24,580,702 bp\n",
      "  Read 2:    24,580,227 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 325 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t278\t2543.6\t0\t278\n",
      "4\t31\t635.9\t0\t31\n",
      "5\t14\t159.0\t0\t14\n",
      "6\t1\t39.7\t0\t1\n",
      "7\t1\t9.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 408 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t308\t2543.6\t0\t308\n",
      "4\t75\t635.9\t0\t75\n",
      "5\t16\t159.0\t0\t16\n",
      "6\t7\t39.7\t0\t7\n",
      "27\t1\t0.0\t2\t0 1\n",
      "143\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6A_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6A_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6A_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6A_80_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6A_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6A_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6A_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6A_80_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       144,530 reads  @     25.1 µs/read;   2.39 M reads/minute\n",
      "Finished in 3.67 s (25 us/read; 2.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            144,530\n",
      "  Read 1 with adapter:                      43 (0.0%)\n",
      "  Read 2 with adapter:                     112 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       144,530 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,648,060 bp\n",
      "  Read 1:    21,824,030 bp\n",
      "  Read 2:    21,824,030 bp\n",
      "Total written (filtered):     43,647,450 bp (100.0%)\n",
      "  Read 1:    21,823,895 bp\n",
      "  Read 2:    21,823,555 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 43 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t2258.3\t0\t37\n",
      "4\t6\t564.6\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 112 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t97\t2258.3\t0\t97\n",
      "4\t8\t564.6\t0\t8\n",
      "5\t1\t141.1\t0\t1\n",
      "6\t1\t35.3\t0\t1\n",
      "19\t1\t0.0\t1\t0 1\n",
      "29\t1\t0.0\t2\t1\n",
      "30\t1\t0.0\t2\t0 1\n",
      "31\t1\t0.0\t2\t1\n",
      "32\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6B_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6B_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6B_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6B_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6B_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6B_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6B_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6B_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       107,287 reads  @     26.2 µs/read;   2.29 M reads/minute\n",
      "Finished in 2.86 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,287\n",
      "  Read 1 with adapter:                     223 (0.2%)\n",
      "  Read 2 with adapter:                     257 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       107,287 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,400,674 bp\n",
      "  Read 1:    16,200,337 bp\n",
      "  Read 2:    16,200,337 bp\n",
      "Total written (filtered):     32,399,075 bp (100.0%)\n",
      "  Read 1:    16,199,629 bp\n",
      "  Read 2:    16,199,446 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 223 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t192\t1676.4\t0\t192\n",
      "4\t25\t419.1\t0\t25\n",
      "5\t4\t104.8\t0\t4\n",
      "6\t2\t26.2\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 257 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t206\t1676.4\t0\t206\n",
      "4\t35\t419.1\t0\t35\n",
      "5\t7\t104.8\t0\t7\n",
      "6\t5\t26.2\t0\t5\n",
      "7\t2\t6.5\t0\t2\n",
      "26\t1\t0.0\t2\t1\n",
      "28\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6C_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6C_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6C_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6C_82_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6C_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl6C_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6C_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl6C_82_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        91,447 reads  @     24.5 µs/read;   2.45 M reads/minute\n",
      "Finished in 2.28 s (25 us/read; 2.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,447\n",
      "  Read 1 with adapter:                     117 (0.1%)\n",
      "  Read 2 with adapter:                     175 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        91,447 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,616,994 bp\n",
      "  Read 1:    13,808,497 bp\n",
      "  Read 2:    13,808,497 bp\n",
      "Total written (filtered):     27,616,031 bp (100.0%)\n",
      "  Read 1:    13,808,129 bp\n",
      "  Read 2:    13,807,902 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 117 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t106\t1428.9\t0\t106\n",
      "4\t7\t357.2\t0\t7\n",
      "5\t2\t89.3\t0\t2\n",
      "6\t2\t22.3\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 175 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t141\t1428.9\t0\t141\n",
      "4\t24\t357.2\t0\t24\n",
      "5\t6\t89.3\t0\t6\n",
      "6\t3\t22.3\t0\t3\n",
      "28\t1\t0.0\t2\t0 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl7C_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl7C_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl7C_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl7C_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl7C_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl7C_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl7C_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl7C_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       111,669 reads  @     24.5 µs/read;   2.45 M reads/minute\n",
      "Finished in 2.79 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,669\n",
      "  Read 1 with adapter:                     355 (0.3%)\n",
      "  Read 2 with adapter:                     510 (0.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,669 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,724,038 bp\n",
      "  Read 1:    16,862,019 bp\n",
      "  Read 2:    16,862,019 bp\n",
      "Total written (filtered):     33,721,186 bp (100.0%)\n",
      "  Read 1:    16,860,882 bp\n",
      "  Read 2:    16,860,304 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 355 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t310\t1744.8\t0\t310\n",
      "4\t27\t436.2\t0\t27\n",
      "5\t13\t109.1\t0\t13\n",
      "6\t1\t27.3\t0\t1\n",
      "7\t4\t6.8\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 510 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t383\t1744.8\t0\t383\n",
      "4\t86\t436.2\t0\t86\n",
      "5\t26\t109.1\t0\t26\n",
      "6\t14\t27.3\t0\t14\n",
      "8\t1\t1.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8A_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8A_84_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8A_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8A_84_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8A_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8A_84_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8A_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8A_84_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        56,113 reads  @     30.9 µs/read;   1.94 M reads/minute\n",
      "Finished in 1.78 s (32 us/read; 1.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,113\n",
      "  Read 1 with adapter:                     190 (0.3%)\n",
      "  Read 2 with adapter:                     236 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,113 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,946,126 bp\n",
      "  Read 1:     8,473,063 bp\n",
      "  Read 2:     8,473,063 bp\n",
      "Total written (filtered):     16,944,698 bp (100.0%)\n",
      "  Read 1:     8,472,457 bp\n",
      "  Read 2:     8,472,241 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 190 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t164\t876.8\t0\t164\n",
      "4\t18\t219.2\t0\t18\n",
      "5\t6\t54.8\t0\t6\n",
      "6\t2\t13.7\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 236 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t175\t876.8\t0\t175\n",
      "4\t39\t219.2\t0\t39\n",
      "5\t15\t54.8\t0\t15\n",
      "6\t6\t13.7\t0\t6\n",
      "30\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8B_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8B_85_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8B_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8B_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8B_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8B_85_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8B_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8B_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        85,465 reads  @     27.7 µs/read;   2.17 M reads/minute\n",
      "Finished in 2.41 s (28 us/read; 2.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,465\n",
      "  Read 1 with adapter:                     233 (0.3%)\n",
      "  Read 2 with adapter:                     306 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        85,465 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,810,430 bp\n",
      "  Read 1:    12,905,215 bp\n",
      "  Read 2:    12,905,215 bp\n",
      "Total written (filtered):     25,808,677 bp (100.0%)\n",
      "  Read 1:    12,904,469 bp\n",
      "  Read 2:    12,904,208 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 233 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t200\t1335.4\t0\t200\n",
      "4\t22\t333.8\t0\t22\n",
      "5\t9\t83.5\t0\t9\n",
      "6\t1\t20.9\t0\t1\n",
      "7\t1\t5.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 306 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t241\t1335.4\t0\t241\n",
      "4\t46\t333.8\t0\t46\n",
      "5\t14\t83.5\t0\t14\n",
      "6\t5\t20.9\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8C_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8C_86_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8C_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8C_86_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8C_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl8C_86_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8C_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl8C_86_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        56,643 reads  @     31.4 µs/read;   1.91 M reads/minute\n",
      "Finished in 1.83 s (32 us/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,643\n",
      "  Read 1 with adapter:                     371 (0.7%)\n",
      "  Read 2 with adapter:                     490 (0.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,643 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,106,186 bp\n",
      "  Read 1:     8,553,093 bp\n",
      "  Read 2:     8,553,093 bp\n",
      "Total written (filtered):     17,103,249 bp (100.0%)\n",
      "  Read 1:     8,551,905 bp\n",
      "  Read 2:     8,551,344 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 371 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t316\t885.0\t0\t316\n",
      "4\t41\t221.3\t0\t41\n",
      "5\t8\t55.3\t0\t8\n",
      "6\t6\t13.8\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 490 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t352\t885.0\t0\t352\n",
      "4\t96\t221.3\t0\t96\n",
      "5\t20\t55.3\t0\t20\n",
      "6\t17\t13.8\t0\t17\n",
      "8\t1\t0.9\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t2\t0 1\n",
      "30\t2\t0.0\t2\t0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9A_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9A_87_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9A_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9A_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9A_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9A_87_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9A_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9A_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       159,012 reads  @     24.2 µs/read;   2.47 M reads/minute\n",
      "Finished in 3.90 s (25 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            159,012\n",
      "  Read 1 with adapter:                     263 (0.2%)\n",
      "  Read 2 with adapter:                     331 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       159,012 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,021,624 bp\n",
      "  Read 1:    24,010,812 bp\n",
      "  Read 2:    24,010,812 bp\n",
      "Total written (filtered):     48,019,664 bp (100.0%)\n",
      "  Read 1:    24,009,971 bp\n",
      "  Read 2:    24,009,693 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 263 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t227\t2484.6\t0\t227\n",
      "4\t25\t621.1\t0\t25\n",
      "5\t7\t155.3\t0\t7\n",
      "6\t3\t38.8\t0\t3\n",
      "7\t1\t9.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 331 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t262\t2484.6\t0\t262\n",
      "4\t47\t621.1\t0\t47\n",
      "5\t13\t155.3\t0\t13\n",
      "6\t6\t38.8\t0\t6\n",
      "7\t2\t9.7\t0\t2\n",
      "30\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9B_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9B_88_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9B_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9B_88_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9B_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9B_88_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9B_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9B_88_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       189,498 reads  @     23.4 µs/read;   2.56 M reads/minute\n",
      "Finished in 4.49 s (24 us/read; 2.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            189,498\n",
      "  Read 1 with adapter:                     735 (0.4%)\n",
      "  Read 2 with adapter:                     911 (0.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       189,498 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,228,396 bp\n",
      "  Read 1:    28,614,198 bp\n",
      "  Read 2:    28,614,198 bp\n",
      "Total written (filtered):     57,222,758 bp (100.0%)\n",
      "  Read 1:    28,611,802 bp\n",
      "  Read 2:    28,610,956 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 735 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t616\t2960.9\t0\t616\n",
      "4\t74\t740.2\t0\t74\n",
      "5\t30\t185.1\t0\t30\n",
      "6\t6\t46.3\t0\t6\n",
      "7\t8\t11.6\t0\t8\n",
      "10\t1\t0.2\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 911 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t685\t2960.9\t0\t685\n",
      "4\t160\t740.2\t0\t160\n",
      "5\t36\t185.1\t0\t36\n",
      "6\t21\t46.3\t0\t21\n",
      "7\t1\t11.6\t0\t1\n",
      "26\t2\t0.0\t2\t0 2\n",
      "29\t1\t0.0\t2\t0 1\n",
      "30\t3\t0.0\t2\t2 1\n",
      "31\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9C_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9C_89_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9C_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9C_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9C_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Dl9C_89_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9C_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Dl9C_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       244,958 reads  @     22.3 µs/read;   2.69 M reads/minute\n",
      "Finished in 5.51 s (22 us/read; 2.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            244,958\n",
      "  Read 1 with adapter:                     708 (0.3%)\n",
      "  Read 2 with adapter:                     884 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       244,958 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,977,316 bp\n",
      "  Read 1:    36,988,658 bp\n",
      "  Read 2:    36,988,658 bp\n",
      "Total written (filtered):     73,971,993 bp (100.0%)\n",
      "  Read 1:    36,986,356 bp\n",
      "  Read 2:    36,985,637 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 708 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t602\t3827.5\t0\t602\n",
      "4\t67\t956.9\t0\t67\n",
      "5\t26\t239.2\t0\t26\n",
      "6\t8\t59.8\t0\t8\n",
      "7\t4\t15.0\t0\t4\n",
      "22\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 884 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t667\t3827.5\t0\t667\n",
      "4\t149\t956.9\t0\t149\n",
      "5\t42\t239.2\t0\t42\n",
      "6\t20\t59.8\t0\t20\n",
      "7\t2\t15.0\t0\t2\n",
      "8\t1\t3.7\t0\t1\n",
      "19\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t0 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1A_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1A_90_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1A_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1A_90_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1A_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1A_90_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1A_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1A_90_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       159,132 reads  @     24.6 µs/read;   2.43 M reads/minute\n",
      "Finished in 3.96 s (25 us/read; 2.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            159,132\n",
      "  Read 1 with adapter:                     198 (0.1%)\n",
      "  Read 2 with adapter:                     239 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       159,132 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,057,864 bp\n",
      "  Read 1:    24,028,932 bp\n",
      "  Read 2:    24,028,932 bp\n",
      "Total written (filtered):     48,056,435 bp (100.0%)\n",
      "  Read 1:    24,028,305 bp\n",
      "  Read 2:    24,028,130 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 198 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t175\t2486.4\t0\t175\n",
      "4\t16\t621.6\t0\t16\n",
      "5\t5\t155.4\t0\t5\n",
      "6\t1\t38.9\t0\t1\n",
      "7\t1\t9.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 239 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t189\t2486.4\t0\t189\n",
      "4\t29\t621.6\t0\t29\n",
      "5\t11\t155.4\t0\t11\n",
      "6\t7\t38.9\t0\t7\n",
      "7\t2\t9.7\t0\t2\n",
      "8\t1\t2.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1B_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1B_91_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1B_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1B_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1B_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1B_91_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1B_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1B_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        72,383 reads  @     28.8 µs/read;   2.08 M reads/minute\n",
      "Finished in 2.14 s (30 us/read; 2.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,383\n",
      "  Read 1 with adapter:                     156 (0.2%)\n",
      "  Read 2 with adapter:                     186 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        72,383 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,859,666 bp\n",
      "  Read 1:    10,929,833 bp\n",
      "  Read 2:    10,929,833 bp\n",
      "Total written (filtered):     21,858,445 bp (100.0%)\n",
      "  Read 1:    10,929,267 bp\n",
      "  Read 2:    10,929,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 156 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t132\t1131.0\t0\t132\n",
      "4\t19\t282.7\t0\t19\n",
      "5\t3\t70.7\t0\t3\n",
      "7\t1\t4.4\t0\t1\n",
      "72\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 186 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t147\t1131.0\t0\t147\n",
      "4\t29\t282.7\t0\t29\n",
      "5\t6\t70.7\t0\t6\n",
      "6\t2\t17.7\t0\t2\n",
      "27\t1\t0.0\t2\t1\n",
      "29\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1C_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1C_92_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1C_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1C_92_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1C_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds1C_92_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1C_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds1C_92_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       100,124 reads  @     26.6 µs/read;   2.26 M reads/minute\n",
      "Finished in 2.72 s (27 us/read; 2.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,124\n",
      "  Read 1 with adapter:                     265 (0.3%)\n",
      "  Read 2 with adapter:                     346 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,124 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,237,448 bp\n",
      "  Read 1:    15,118,724 bp\n",
      "  Read 2:    15,118,724 bp\n",
      "Total written (filtered):     30,235,437 bp (100.0%)\n",
      "  Read 1:    15,117,870 bp\n",
      "  Read 2:    15,117,567 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 265 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t226\t1564.4\t0\t226\n",
      "4\t26\t391.1\t0\t26\n",
      "5\t7\t97.8\t0\t7\n",
      "6\t5\t24.4\t0\t5\n",
      "7\t1\t6.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 346 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t268\t1564.4\t0\t268\n",
      "4\t66\t391.1\t0\t66\n",
      "5\t6\t97.8\t0\t6\n",
      "6\t5\t24.4\t0\t5\n",
      "29\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2A_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2A_93_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2A_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2A_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2A_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2A_93_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2A_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2A_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        71,721 reads  @     30.2 µs/read;   1.99 M reads/minute\n",
      "Finished in 2.23 s (31 us/read; 1.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             71,721\n",
      "  Read 1 with adapter:                     155 (0.2%)\n",
      "  Read 2 with adapter:                     190 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        71,721 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,659,742 bp\n",
      "  Read 1:    10,829,871 bp\n",
      "  Read 2:    10,829,871 bp\n",
      "Total written (filtered):     21,658,615 bp (100.0%)\n",
      "  Read 1:    10,829,383 bp\n",
      "  Read 2:    10,829,232 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 155 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t134\t1120.6\t0\t134\n",
      "4\t19\t280.2\t0\t19\n",
      "5\t2\t70.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 190 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t140\t1120.6\t0\t140\n",
      "4\t36\t280.2\t0\t36\n",
      "5\t9\t70.0\t0\t9\n",
      "6\t5\t17.5\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2B_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2B_94_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2B_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2B_94_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2B_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2B_94_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2B_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2B_94_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        64,309 reads  @     27.9 µs/read;   2.15 M reads/minute\n",
      "Finished in 1.84 s (29 us/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,309\n",
      "  Read 1 with adapter:                     184 (0.3%)\n",
      "  Read 2 with adapter:                     237 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        64,309 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,421,318 bp\n",
      "  Read 1:     9,710,659 bp\n",
      "  Read 2:     9,710,659 bp\n",
      "Total written (filtered):     19,419,943 bp (100.0%)\n",
      "  Read 1:     9,710,075 bp\n",
      "  Read 2:     9,709,868 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 184 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t163\t1004.8\t0\t163\n",
      "4\t13\t251.2\t0\t13\n",
      "5\t6\t62.8\t0\t6\n",
      "6\t1\t15.7\t0\t1\n",
      "7\t1\t3.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 237 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t195\t1004.8\t0\t195\n",
      "4\t31\t251.2\t0\t31\n",
      "5\t8\t62.8\t0\t8\n",
      "6\t2\t15.7\t0\t2\n",
      "30\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2C_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2C_95_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2C_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2C_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2C_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds2C_95_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2C_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds2C_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        51,361 reads  @     33.2 µs/read;   1.81 M reads/minute\n",
      "Finished in 1.76 s (34 us/read; 1.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,361\n",
      "  Read 1 with adapter:                     148 (0.3%)\n",
      "  Read 2 with adapter:                     213 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,361 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,511,022 bp\n",
      "  Read 1:     7,755,511 bp\n",
      "  Read 2:     7,755,511 bp\n",
      "Total written (filtered):     15,509,847 bp (100.0%)\n",
      "  Read 1:     7,755,043 bp\n",
      "  Read 2:     7,754,804 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 148 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t132\t802.5\t0\t132\n",
      "4\t10\t200.6\t0\t10\n",
      "5\t4\t50.2\t0\t4\n",
      "6\t2\t12.5\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 213 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t163\t802.5\t0\t163\n",
      "4\t36\t200.6\t0\t36\n",
      "5\t10\t50.2\t0\t10\n",
      "6\t4\t12.5\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3A_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3A_96_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3A_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3A_96_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3A_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3A_96_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3A_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3A_96_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       164,114 reads  @     24.0 µs/read;   2.50 M reads/minute\n",
      "Finished in 3.99 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            164,114\n",
      "  Read 1 with adapter:                     186 (0.1%)\n",
      "  Read 2 with adapter:                     278 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       164,114 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,562,428 bp\n",
      "  Read 1:    24,781,214 bp\n",
      "  Read 2:    24,781,214 bp\n",
      "Total written (filtered):     49,560,907 bp (100.0%)\n",
      "  Read 1:    24,780,627 bp\n",
      "  Read 2:    24,780,280 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 186 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t167\t2564.3\t0\t167\n",
      "4\t13\t641.1\t0\t13\n",
      "5\t3\t160.3\t0\t3\n",
      "6\t2\t40.1\t0\t2\n",
      "7\t1\t10.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 278 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t228\t2564.3\t0\t228\n",
      "4\t35\t641.1\t0\t35\n",
      "5\t10\t160.3\t0\t10\n",
      "6\t2\t40.1\t0\t2\n",
      "8\t2\t2.5\t0\t2\n",
      "32\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3B_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3B_97_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3B_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3B_97_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3B_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3B_97_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3B_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3B_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       137,638 reads  @     24.6 µs/read;   2.44 M reads/minute\n",
      "Finished in 3.45 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            137,638\n",
      "  Read 1 with adapter:                     342 (0.2%)\n",
      "  Read 2 with adapter:                     482 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       137,638 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,566,676 bp\n",
      "  Read 1:    20,783,338 bp\n",
      "  Read 2:    20,783,338 bp\n",
      "Total written (filtered):     41,563,766 bp (100.0%)\n",
      "  Read 1:    20,782,231 bp\n",
      "  Read 2:    20,781,535 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 342 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t296\t2150.6\t0\t296\n",
      "4\t31\t537.6\t0\t31\n",
      "5\t12\t134.4\t0\t12\n",
      "6\t1\t33.6\t0\t1\n",
      "7\t1\t8.4\t0\t1\n",
      "22\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 482 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t368\t2150.6\t0\t368\n",
      "4\t78\t537.6\t0\t78\n",
      "5\t20\t134.4\t0\t20\n",
      "6\t12\t33.6\t0\t12\n",
      "8\t1\t2.1\t0\t1\n",
      "28\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t0 1\n",
      "147\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3C_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3C_98_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3C_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3C_98_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3C_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds3C_98_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3C_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds3C_98_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       156,119 reads  @     24.2 µs/read;   2.48 M reads/minute\n",
      "Finished in 3.83 s (25 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            156,119\n",
      "  Read 1 with adapter:                     213 (0.1%)\n",
      "  Read 2 with adapter:                     334 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       156,119 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,147,938 bp\n",
      "  Read 1:    23,573,969 bp\n",
      "  Read 2:    23,573,969 bp\n",
      "Total written (filtered):     47,146,172 bp (100.0%)\n",
      "  Read 1:    23,573,305 bp\n",
      "  Read 2:    23,572,867 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 213 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t192\t2439.4\t0\t192\n",
      "4\t18\t609.8\t0\t18\n",
      "5\t2\t152.5\t0\t2\n",
      "6\t1\t38.1\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 334 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t267\t2439.4\t0\t267\n",
      "4\t45\t609.8\t0\t45\n",
      "5\t16\t152.5\t0\t16\n",
      "6\t3\t38.1\t0\t3\n",
      "7\t1\t9.5\t0\t1\n",
      "8\t2\t2.4\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4A_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4A_99_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4A_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4A_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4A_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4A_99_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4A_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4A_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       111,770 reads  @     25.9 µs/read;   2.31 M reads/minute\n",
      "Finished in 2.94 s (26 us/read; 2.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,770\n",
      "  Read 1 with adapter:                     252 (0.2%)\n",
      "  Read 2 with adapter:                     345 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,770 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,754,540 bp\n",
      "  Read 1:    16,877,270 bp\n",
      "  Read 2:    16,877,270 bp\n",
      "Total written (filtered):     33,752,552 bp (100.0%)\n",
      "  Read 1:    16,876,473 bp\n",
      "  Read 2:    16,876,079 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 252 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t221\t1746.4\t0\t221\n",
      "4\t23\t436.6\t0\t23\n",
      "5\t6\t109.2\t0\t6\n",
      "6\t2\t27.3\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 345 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t255\t1746.4\t0\t255\n",
      "4\t62\t436.6\t0\t62\n",
      "5\t14\t109.2\t0\t14\n",
      "6\t10\t27.3\t0\t10\n",
      "8\t2\t1.7\t0\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "19\t1\t0.0\t1\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4B_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4B_100_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4B_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4B_100_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4B_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4B_100_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4B_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4B_100_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02       102,816 reads  @     26.5 µs/read;   2.26 M reads/minute\n",
      "Finished in 2.79 s (27 us/read; 2.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            102,816\n",
      "  Read 1 with adapter:                     347 (0.3%)\n",
      "  Read 2 with adapter:                     444 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       102,816 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,050,432 bp\n",
      "  Read 1:    15,525,216 bp\n",
      "  Read 2:    15,525,216 bp\n",
      "Total written (filtered):     31,047,785 bp (100.0%)\n",
      "  Read 1:    15,524,081 bp\n",
      "  Read 2:    15,523,704 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 347 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t285\t1606.5\t0\t285\n",
      "4\t40\t401.6\t0\t40\n",
      "5\t14\t100.4\t0\t14\n",
      "6\t6\t25.1\t0\t6\n",
      "7\t2\t6.3\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 444 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t338\t1606.5\t0\t338\n",
      "4\t75\t401.6\t0\t75\n",
      "5\t17\t100.4\t0\t17\n",
      "6\t11\t25.1\t0\t11\n",
      "7\t1\t6.3\t0\t1\n",
      "8\t1\t1.6\t0\t1\n",
      "32\t1\t0.0\t2\t0 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4C_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4C_101_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4C_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4C_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4C_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/Ds4C_101_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4C_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/Ds4C_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       122,982 reads  @     26.5 µs/read;   2.26 M reads/minute\n",
      "Finished in 3.31 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            122,982\n",
      "  Read 1 with adapter:                     309 (0.3%)\n",
      "  Read 2 with adapter:                     440 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       122,982 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,140,564 bp\n",
      "  Read 1:    18,570,282 bp\n",
      "  Read 2:    18,570,282 bp\n",
      "Total written (filtered):     37,138,013 bp (100.0%)\n",
      "  Read 1:    18,569,298 bp\n",
      "  Read 2:    18,568,715 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 309 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t270\t1921.6\t0\t270\n",
      "4\t27\t480.4\t0\t27\n",
      "5\t9\t120.1\t0\t9\n",
      "7\t3\t7.5\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 440 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t336\t1921.6\t0\t336\n",
      "4\t62\t480.4\t0\t62\n",
      "5\t25\t120.1\t0\t25\n",
      "6\t11\t30.0\t0\t11\n",
      "8\t1\t1.9\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "29\t2\t0.0\t2\t0 2\n",
      "31\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA1_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA1_102_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA1_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA1_102_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA1_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA1_102_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA1_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA1_102_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        28,274 reads  @     38.9 µs/read;   1.54 M reads/minute\n",
      "Finished in 1.14 s (40 us/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,274\n",
      "  Read 1 with adapter:                      95 (0.3%)\n",
      "  Read 2 with adapter:                     102 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,274 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,538,748 bp\n",
      "  Read 1:     4,269,374 bp\n",
      "  Read 2:     4,269,374 bp\n",
      "Total written (filtered):      8,538,094 bp (100.0%)\n",
      "  Read 1:     4,269,074 bp\n",
      "  Read 2:     4,269,020 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 95 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t81\t441.8\t0\t81\n",
      "4\t13\t110.4\t0\t13\n",
      "5\t1\t27.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 102 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t75\t441.8\t0\t75\n",
      "4\t17\t110.4\t0\t17\n",
      "5\t6\t27.6\t0\t6\n",
      "6\t3\t6.9\t0\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA2_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA2_103_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA2_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA2_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA2_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA2_103_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA2_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA2_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        35,187 reads  @     36.3 µs/read;   1.65 M reads/minute\n",
      "Finished in 1.32 s (38 us/read; 1.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,187\n",
      "  Read 1 with adapter:                      99 (0.3%)\n",
      "  Read 2 with adapter:                      96 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,187 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,626,474 bp\n",
      "  Read 1:     5,313,237 bp\n",
      "  Read 2:     5,313,237 bp\n",
      "Total written (filtered):     10,625,857 bp (100.0%)\n",
      "  Read 1:     5,312,928 bp\n",
      "  Read 2:     5,312,929 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t89\t549.8\t0\t89\n",
      "4\t8\t137.4\t0\t8\n",
      "5\t2\t34.4\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 96 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t83\t549.8\t0\t83\n",
      "4\t9\t137.4\t0\t9\n",
      "5\t3\t34.4\t0\t3\n",
      "8\t1\t0.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA3_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA3_104_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA3_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA3_104_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA3_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA3_104_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA3_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA3_104_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       185,193 reads  @     24.5 µs/read;   2.45 M reads/minute\n",
      "Finished in 4.58 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            185,193\n",
      "  Read 1 with adapter:                     259 (0.1%)\n",
      "  Read 2 with adapter:                     289 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       185,193 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,928,286 bp\n",
      "  Read 1:    27,964,143 bp\n",
      "  Read 2:    27,964,143 bp\n",
      "Total written (filtered):     55,926,508 bp (100.0%)\n",
      "  Read 1:    27,963,320 bp\n",
      "  Read 2:    27,963,188 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 259 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t233\t2893.6\t0\t233\n",
      "4\t12\t723.4\t0\t12\n",
      "5\t9\t180.9\t0\t9\n",
      "6\t4\t45.2\t0\t4\n",
      "7\t1\t11.3\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 289 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t246\t2893.6\t0\t246\n",
      "4\t26\t723.4\t0\t26\n",
      "5\t13\t180.9\t0\t13\n",
      "6\t2\t45.2\t0\t2\n",
      "8\t1\t2.8\t0\t1\n",
      "28\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA4_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA4_105_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA4_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA4_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA4_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBA4_105_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA4_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBA4_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       128,398 reads  @     23.9 µs/read;   2.51 M reads/minute\n",
      "Finished in 3.12 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            128,398\n",
      "  Read 1 with adapter:                     253 (0.2%)\n",
      "  Read 2 with adapter:                     304 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       128,398 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,776,196 bp\n",
      "  Read 1:    19,388,098 bp\n",
      "  Read 2:    19,388,098 bp\n",
      "Total written (filtered):     38,774,396 bp (100.0%)\n",
      "  Read 1:    19,387,307 bp\n",
      "  Read 2:    19,387,089 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 253 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t229\t2006.2\t0\t229\n",
      "4\t19\t501.6\t0\t19\n",
      "5\t2\t125.4\t0\t2\n",
      "6\t3\t31.3\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 304 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t236\t2006.2\t0\t236\n",
      "4\t46\t501.6\t0\t46\n",
      "5\t15\t125.4\t0\t15\n",
      "6\t7\t31.3\t0\t7\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB1_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB1_107_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB1_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB1_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB1_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB1_107_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB1_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB1_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        68,874 reads  @     26.6 µs/read;   2.26 M reads/minute\n",
      "Finished in 1.87 s (27 us/read; 2.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,874\n",
      "  Read 1 with adapter:                      86 (0.1%)\n",
      "  Read 2 with adapter:                     108 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        68,874 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,799,948 bp\n",
      "  Read 1:    10,399,974 bp\n",
      "  Read 2:    10,399,974 bp\n",
      "Total written (filtered):     20,799,312 bp (100.0%)\n",
      "  Read 1:    10,399,704 bp\n",
      "  Read 2:    10,399,608 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 86 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t78\t1076.2\t0\t78\n",
      "4\t5\t269.0\t0\t5\n",
      "5\t2\t67.3\t0\t2\n",
      "6\t1\t16.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t1076.2\t0\t79\n",
      "4\t19\t269.0\t0\t19\n",
      "5\t7\t67.3\t0\t7\n",
      "6\t3\t16.8\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB2_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB2_108_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB2_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB2_108_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB2_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB2_108_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB2_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB2_108_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        79,613 reads  @     26.2 µs/read;   2.29 M reads/minute\n",
      "Finished in 2.13 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,613\n",
      "  Read 1 with adapter:                     141 (0.2%)\n",
      "  Read 2 with adapter:                     184 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        79,613 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,043,126 bp\n",
      "  Read 1:    12,021,563 bp\n",
      "  Read 2:    12,021,563 bp\n",
      "Total written (filtered):     24,042,055 bp (100.0%)\n",
      "  Read 1:    12,021,112 bp\n",
      "  Read 2:    12,020,943 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 141 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t121\t1244.0\t0\t121\n",
      "4\t15\t311.0\t0\t15\n",
      "5\t3\t77.7\t0\t3\n",
      "6\t1\t19.4\t0\t1\n",
      "7\t1\t4.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 184 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t139\t1244.0\t0\t139\n",
      "4\t29\t311.0\t0\t29\n",
      "5\t11\t77.7\t0\t11\n",
      "6\t3\t19.4\t0\t3\n",
      "7\t2\t4.9\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB3_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB3_109_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB3_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB3_109_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB3_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBB3_109_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB3_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBB3_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:07       338,067 reads  @     21.8 µs/read;   2.75 M reads/minute\n",
      "Finished in 7.42 s (22 us/read; 2.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            338,067\n",
      "  Read 1 with adapter:                     265 (0.1%)\n",
      "  Read 2 with adapter:                     293 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       338,067 (100.0%)\n",
      "\n",
      "Total basepairs processed:   102,096,234 bp\n",
      "  Read 1:    51,048,117 bp\n",
      "  Read 2:    51,048,117 bp\n",
      "Total written (filtered):    102,094,484 bp (100.0%)\n",
      "  Read 1:    51,047,302 bp\n",
      "  Read 2:    51,047,182 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 265 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t249\t5282.3\t0\t249\n",
      "4\t13\t1320.6\t0\t13\n",
      "5\t2\t330.1\t0\t2\n",
      "6\t1\t82.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 293 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t255\t5282.3\t0\t255\n",
      "4\t28\t1320.6\t0\t28\n",
      "5\t4\t330.1\t0\t4\n",
      "6\t4\t82.5\t0\t4\n",
      "7\t2\t20.6\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC1_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC1_111_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC1_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC1_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC1_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC1_111_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC1_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC1_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        41,706 reads  @     33.3 µs/read;   1.80 M reads/minute\n",
      "Finished in 1.43 s (34 us/read; 1.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,706\n",
      "  Read 1 with adapter:                      55 (0.1%)\n",
      "  Read 2 with adapter:                      77 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,706 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,595,212 bp\n",
      "  Read 1:     6,297,606 bp\n",
      "  Read 2:     6,297,606 bp\n",
      "Total written (filtered):     12,594,792 bp (100.0%)\n",
      "  Read 1:     6,297,436 bp\n",
      "  Read 2:     6,297,356 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 55 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t52\t651.7\t0\t52\n",
      "4\t2\t162.9\t0\t2\n",
      "6\t1\t10.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 77 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t651.7\t0\t59\n",
      "4\t17\t162.9\t0\t17\n",
      "5\t1\t40.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC2_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC2_112_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC2_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC2_112_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC2_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC2_112_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC2_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC2_112_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        90,670 reads  @     23.7 µs/read;   2.53 M reads/minute\n",
      "Finished in 2.19 s (24 us/read; 2.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,670\n",
      "  Read 1 with adapter:                     223 (0.2%)\n",
      "  Read 2 with adapter:                     263 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        90,670 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,382,340 bp\n",
      "  Read 1:    13,691,170 bp\n",
      "  Read 2:    13,691,170 bp\n",
      "Total written (filtered):     27,380,729 bp (100.0%)\n",
      "  Read 1:    13,690,457 bp\n",
      "  Read 2:    13,690,272 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 223 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t188\t1416.7\t0\t188\n",
      "4\t30\t354.2\t0\t30\n",
      "5\t2\t88.5\t0\t2\n",
      "6\t2\t22.1\t0\t2\n",
      "7\t1\t5.5\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 263 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t198\t1416.7\t0\t198\n",
      "4\t44\t354.2\t0\t44\n",
      "5\t15\t88.5\t0\t15\n",
      "6\t4\t22.1\t0\t4\n",
      "7\t1\t5.5\t0\t1\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC3_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC3_113_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC3_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC3_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC3_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC3_113_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC3_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC3_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        79,129 reads  @     25.7 µs/read;   2.34 M reads/minute\n",
      "Finished in 2.07 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,129\n",
      "  Read 1 with adapter:                      39 (0.0%)\n",
      "  Read 2 with adapter:                      42 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        79,129 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,896,958 bp\n",
      "  Read 1:    11,948,479 bp\n",
      "  Read 2:    11,948,479 bp\n",
      "Total written (filtered):     23,896,711 bp (100.0%)\n",
      "  Read 1:    11,948,362 bp\n",
      "  Read 2:    11,948,349 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 39 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t1236.4\t0\t39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 42 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t1236.4\t0\t39\n",
      "4\t2\t309.1\t0\t2\n",
      "5\t1\t77.3\t0\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC4_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC4_114_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC4_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC4_114_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC4_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBC4_114_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC4_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBC4_114_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00         3,860 reads  @    106.2 µs/read;   0.57 M reads/minute\n",
      "Finished in 0.47 s (121 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              3,860\n",
      "  Read 1 with adapter:                       3 (0.1%)\n",
      "  Read 2 with adapter:                       6 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         3,860 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,165,720 bp\n",
      "  Read 1:       582,860 bp\n",
      "  Read 2:       582,860 bp\n",
      "Total written (filtered):      1,165,692 bp (100.0%)\n",
      "  Read 1:       582,851 bp\n",
      "  Read 2:       582,841 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 3 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t60.3\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 6 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t60.3\t0\t5\n",
      "4\t1\t15.1\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD1_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD1_116_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD1_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD1_116_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD1_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD1_116_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD1_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD1_116_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        60,338 reads  @     27.5 µs/read;   2.18 M reads/minute\n",
      "Finished in 1.71 s (28 us/read; 2.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,338\n",
      "  Read 1 with adapter:                      61 (0.1%)\n",
      "  Read 2 with adapter:                      90 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        60,338 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,222,076 bp\n",
      "  Read 1:     9,111,038 bp\n",
      "  Read 2:     9,111,038 bp\n",
      "Total written (filtered):     18,221,604 bp (100.0%)\n",
      "  Read 1:     9,110,852 bp\n",
      "  Read 2:     9,110,752 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 61 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t942.8\t0\t58\n",
      "4\t3\t235.7\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 90 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t942.8\t0\t79\n",
      "4\t7\t235.7\t0\t7\n",
      "5\t3\t58.9\t0\t3\n",
      "6\t1\t14.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD2_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD2_117_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD2_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD2_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD2_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD2_117_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD2_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD2_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        30,105 reads  @     38.2 µs/read;   1.57 M reads/minute\n",
      "Finished in 1.19 s (40 us/read; 1.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,105\n",
      "  Read 1 with adapter:                      92 (0.3%)\n",
      "  Read 2 with adapter:                     110 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,105 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,091,710 bp\n",
      "  Read 1:     4,545,855 bp\n",
      "  Read 2:     4,545,855 bp\n",
      "Total written (filtered):      9,091,053 bp (100.0%)\n",
      "  Read 1:     4,545,558 bp\n",
      "  Read 2:     4,545,495 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 92 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t470.4\t0\t80\n",
      "4\t6\t117.6\t0\t6\n",
      "5\t4\t29.4\t0\t4\n",
      "6\t1\t7.3\t0\t1\n",
      "7\t1\t1.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t89\t470.4\t0\t89\n",
      "4\t15\t117.6\t0\t15\n",
      "5\t3\t29.4\t0\t3\n",
      "6\t3\t7.3\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD3_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD3_118_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD3_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD3_118_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD3_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBD3_118_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD3_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBD3_118_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8<----------] 00:00:00        12,906 reads  @     63.8 µs/read;   0.94 M reads/minute\n",
      "Finished in 0.86 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,906\n",
      "  Read 1 with adapter:                      15 (0.1%)\n",
      "  Read 2 with adapter:                      17 (0.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        12,906 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,897,612 bp\n",
      "  Read 1:     1,948,806 bp\n",
      "  Read 2:     1,948,806 bp\n",
      "Total written (filtered):      3,897,507 bp (100.0%)\n",
      "  Read 1:     1,948,758 bp\n",
      "  Read 2:     1,948,749 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 15 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t201.7\t0\t14\n",
      "6\t1\t3.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 17 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t201.7\t0\t13\n",
      "4\t3\t50.4\t0\t3\n",
      "6\t1\t3.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF1_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF1_120_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF1_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF1_120_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF1_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF1_120_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF1_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF1_120_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00         9,843 reads  @     77.5 µs/read;   0.77 M reads/minute\n",
      "Finished in 0.80 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              9,843\n",
      "  Read 1 with adapter:                      24 (0.2%)\n",
      "  Read 2 with adapter:                      37 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         9,843 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,972,586 bp\n",
      "  Read 1:     1,486,293 bp\n",
      "  Read 2:     1,486,293 bp\n",
      "Total written (filtered):      2,972,390 bp (100.0%)\n",
      "  Read 1:     1,486,216 bp\n",
      "  Read 2:     1,486,174 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 24 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t153.8\t0\t20\n",
      "4\t3\t38.4\t0\t3\n",
      "5\t1\t9.6\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 37 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t153.8\t0\t31\n",
      "4\t5\t38.4\t0\t5\n",
      "6\t1\t2.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF2_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF2_121_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF2_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF2_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF2_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF2_121_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF2_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF2_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        10,623 reads  @     74.9 µs/read;   0.80 M reads/minute\n",
      "Finished in 0.84 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,623\n",
      "  Read 1 with adapter:                      24 (0.2%)\n",
      "  Read 2 with adapter:                      16 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        10,623 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,208,146 bp\n",
      "  Read 1:     1,604,073 bp\n",
      "  Read 2:     1,604,073 bp\n",
      "Total written (filtered):      3,208,019 bp (100.0%)\n",
      "  Read 1:     1,603,998 bp\n",
      "  Read 2:     1,604,021 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 24 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t166.0\t0\t22\n",
      "4\t1\t41.5\t0\t1\n",
      "5\t1\t10.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 16 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t166.0\t0\t12\n",
      "4\t4\t41.5\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF3_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF3_122_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF3_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF3_122_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF3_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF3_122_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF3_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF3_122_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        59,953 reads  @     29.6 µs/read;   2.03 M reads/minute\n",
      "Finished in 1.82 s (30 us/read; 1.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,953\n",
      "  Read 1 with adapter:                     170 (0.3%)\n",
      "  Read 2 with adapter:                     185 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        59,953 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,105,806 bp\n",
      "  Read 1:     9,052,903 bp\n",
      "  Read 2:     9,052,903 bp\n",
      "Total written (filtered):     18,104,648 bp (100.0%)\n",
      "  Read 1:     9,052,367 bp\n",
      "  Read 2:     9,052,281 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 170 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t151\t936.8\t0\t151\n",
      "4\t14\t234.2\t0\t14\n",
      "5\t3\t58.5\t0\t3\n",
      "6\t2\t14.6\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 185 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t144\t936.8\t0\t144\n",
      "4\t29\t234.2\t0\t29\n",
      "5\t6\t58.5\t0\t6\n",
      "6\t4\t14.6\t0\t4\n",
      "7\t1\t3.7\t0\t1\n",
      "13\t1\t0.0\t1\t1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF4_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF4_123_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF4_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF4_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF4_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBF4_123_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF4_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBF4_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       175,522 reads  @     24.2 µs/read;   2.48 M reads/minute\n",
      "Finished in 4.29 s (24 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            175,522\n",
      "  Read 1 with adapter:                     236 (0.1%)\n",
      "  Read 2 with adapter:                     303 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       175,522 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,007,644 bp\n",
      "  Read 1:    26,503,822 bp\n",
      "  Read 2:    26,503,822 bp\n",
      "Total written (filtered):     53,005,878 bp (100.0%)\n",
      "  Read 1:    26,503,086 bp\n",
      "  Read 2:    26,502,792 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 236 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t212\t2742.5\t0\t212\n",
      "4\t20\t685.6\t0\t20\n",
      "5\t4\t171.4\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 303 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t234\t2742.5\t0\t234\n",
      "4\t50\t685.6\t0\t50\n",
      "5\t10\t171.4\t0\t10\n",
      "6\t7\t42.9\t0\t7\n",
      "7\t1\t10.7\t0\t1\n",
      "29\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBa_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBa_106_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBa_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBa_106_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBa_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBa_106_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBa_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBa_106_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00         8,605 reads  @     82.4 µs/read;   0.73 M reads/minute\n",
      "Finished in 0.75 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,605\n",
      "  Read 1 with adapter:                      46 (0.5%)\n",
      "  Read 2 with adapter:                      34 (0.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         8,605 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,598,710 bp\n",
      "  Read 1:     1,299,355 bp\n",
      "  Read 2:     1,299,355 bp\n",
      "Total written (filtered):      2,598,452 bp (100.0%)\n",
      "  Read 1:     1,299,209 bp\n",
      "  Read 2:     1,299,243 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 46 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t134.5\t0\t40\n",
      "4\t4\t33.6\t0\t4\n",
      "5\t2\t8.4\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 34 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t134.5\t0\t26\n",
      "4\t6\t33.6\t0\t6\n",
      "5\t2\t8.4\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBb_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBb_110_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBb_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBb_110_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBb_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBb_110_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBb_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBb_110_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00         6,863 reads  @     83.1 µs/read;   0.72 M reads/minute\n",
      "Finished in 0.61 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              6,863\n",
      "  Read 1 with adapter:                      10 (0.1%)\n",
      "  Read 2 with adapter:                      13 (0.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         6,863 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,072,626 bp\n",
      "  Read 1:     1,036,313 bp\n",
      "  Read 2:     1,036,313 bp\n",
      "Total written (filtered):      2,072,551 bp (100.0%)\n",
      "  Read 1:     1,036,283 bp\n",
      "  Read 2:     1,036,268 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 10 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t107.2\t0\t10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t107.2\t0\t10\n",
      "4\t1\t26.8\t0\t1\n",
      "5\t1\t6.7\t0\t1\n",
      "6\t1\t1.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBc_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBc_115_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBc_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBc_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBc_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBc_115_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBc_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBc_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8<----------] 00:00:00        17,609 reads  @     48.9 µs/read;   1.23 M reads/minute\n",
      "Finished in 0.90 s (51 us/read; 1.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,609\n",
      "  Read 1 with adapter:                      50 (0.3%)\n",
      "  Read 2 with adapter:                      47 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        17,609 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,317,918 bp\n",
      "  Read 1:     2,658,959 bp\n",
      "  Read 2:     2,658,959 bp\n",
      "Total written (filtered):      5,317,610 bp (100.0%)\n",
      "  Read 1:     2,658,805 bp\n",
      "  Read 2:     2,658,805 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 50 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t275.1\t0\t46\n",
      "4\t4\t68.8\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 47 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t275.1\t0\t37\n",
      "4\t7\t68.8\t0\t7\n",
      "5\t3\t17.2\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBd_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBd_119_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBd_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBd_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBd_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-k0p45bgs/PBd_119_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBd_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-3rog__mf/e16a6c7e-9b20-4ff1-8286-6c926629224e/data/PBd_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        11,947 reads  @     67.2 µs/read;   0.89 M reads/minute\n",
      "Finished in 0.84 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,947\n",
      "  Read 1 with adapter:                      23 (0.2%)\n",
      "  Read 2 with adapter:                      41 (0.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,947 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,607,994 bp\n",
      "  Read 1:     1,803,997 bp\n",
      "  Read 2:     1,803,997 bp\n",
      "Total written (filtered):      3,607,778 bp (100.0%)\n",
      "  Read 1:     1,803,924 bp\n",
      "  Read 2:     1,803,854 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t186.7\t0\t21\n",
      "5\t2\t11.7\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 41 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t186.7\t0\t26\n",
      "4\t11\t46.7\t0\t11\n",
      "5\t3\t11.7\t0\t3\n",
      "6\t1\t2.9\t0\t1\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: Folder_Meyer_FRT2017/trim_Meyer_FRT2017.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#trim PCR primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences Folder_Meyer_FRT2017/Import_Meyer_FRT2017.qza \\\n",
    "--p-cores 2 \\\n",
    "--p-front-f GTGYCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACNVGGGTWTCTAAT \\\n",
    "--o-trimmed-sequences Folder_Meyer_FRT2017/trim_Meyer_FRT2017.qza \\\n",
    "--verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_Meyer_FRT2017/trim_Meyer_FRT2017.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data Folder_Meyer_FRT2017/trim_Meyer_FRT2017.qza \\\n",
    "--o-visualization Folder_Meyer_FRT2017/trim_Meyer_FRT2017.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_Meyer_FRT2017/table_Meyer_FRT201.qza\u001b[0m\r\n",
      "\u001b[32mSaved FeatureData[Sequence] to: rep_seqs_Meyer_FRT2017.qza\u001b[0m\r\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: Folder_Meyer_FRT2017/dada_stats_Meyer_FRT201.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime dada2 denoise-single \\\n",
    "--i-demultiplexed-seqs Folder_Meyer_FRT2017/trim_Meyer_FRT2017.qza \\\n",
    "--p-trunc-len 130 \\\n",
    "--p-n-threads 10  \\\n",
    "--p-max-ee 3 \\\n",
    "--o-representative-sequences Folder_Meyer_FRT2017/rep_seqs_Meyer_FRT2017.qza \\\n",
    "--o-table Folder_Meyer_FRT2017/table_Meyer_FRT201.qza \\\n",
    "--o-denoising-stats Folder_Meyer_FRT2017/dada_stats_Meyer_FRT201.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_Meyer_FRT2017/table_Meyer_FRT201.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: Folder_Meyer_FRT2017/dada_stats_Meyer_FRT201.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file Folder_Meyer_FRT2017/table_Meyer_FRT201.qza \\\n",
    "  --o-visualization Folder_Meyer_FRT2017/table_Meyer_FRT201.qzv\n",
    "\n",
    "!qiime metadata tabulate \\\n",
    "  --m-input-file Folder_Meyer_FRT2017/dada_stats_Meyer_FRT201.qza \\\n",
    "  --o-visualization Folder_Meyer_FRT2017/dada_stats_Meyer_FRT201.qzv\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
