{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "env: STUDY=Apprill_USVIFeb2020\n"
     ]
    }
   ],
   "source": [
    "%env STUDY Apprill_USVIFeb2020"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "for i in {1..3}\n",
    "do\n",
    "mkdir \"run${i}_folder\"\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "#remove return character from excel file\n",
    "!sed -r 's/\\r//g' run1April_feb.txt > run1April_febV2.txt\n",
    "!sed -r 's/\\r//g' run2April_feb.txt > run2April_febV2.txt\n",
    "!sed -r 's/\\r//g' run3April_feb.txt > run3April_febV2.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash \n",
    "cat run1April_febV2.txt | while read i; do mv $i run1_folder;\n",
    "done\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash \n",
    "cat run2April_febV2.txt | while read i; do mv $i run2_folder;\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash \n",
    "cat run3April_febV2.txt | while read i; do mv $i run3_folder;\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIFeb2020/run2_folder\n",
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIFeb2020/run3_folder\n"
     ]
    }
   ],
   "source": [
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIFeb2020/run1_folder\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_run1_$STUDY.txt\n",
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIFeb2020/run2_folder\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_run2_$STUDY.txt\n",
    "%cd /space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIFeb2020/run3_folder\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_run3_$STUDY.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as ps\n",
    "import numpy as np\n",
    "\n",
    "\n",
    "dir_path='/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIFeb2020/run1_folder/'\n",
    "\n",
    "\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_run1_Apprill_USVIFeb2020.txt',\n",
    "                                    names=header_list)\n",
    "\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_L001\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "\n",
    "\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_USVIFeb2020I_run1.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "dir_path='/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIFeb2020/run2_folder/'\n",
    "\n",
    "\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_run2_Apprill_USVIFeb2020.txt',\n",
    "                                    names=header_list)\n",
    "\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_L001\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "\n",
    "\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_USVIFeb2020I_run2.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "dir_path='/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/Apprill_USVIFeb2020/run3_folder/'\n",
    "\n",
    "\n",
    "header_list = [\"filepath\"]\n",
    "files_df=ps.read_table('/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_run3_Apprill_USVIFeb2020.txt',\n",
    "                                    names=header_list)\n",
    "\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_L001\",expand=True,)\n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "\n",
    "\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('R1','R2')\n",
    "\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_USVIFeb2020I_run3.csv',\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: cannot create directory ‘Folder_Apprill_USVIFeb2020’: File exists\r\n"
     ]
    }
   ],
   "source": [
    "!cd /space/home/rosales/projects/2020_SCTLD_meta-analysis/\n",
    "!mkdir Folder_Apprill_USVIFeb2020"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Imported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_USVIFeb2020I_run1.csv as PairedEndFastqManifestPhred33V2 to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_USVIFeb2020/Import_files_Apprill_USVIFeb2020I_run1\n",
      "Imported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_USVIFeb2020I_run2.csv as PairedEndFastqManifestPhred33V2 to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_USVIFeb2020/Import_files_Apprill_USVIFeb2020I_run2\n",
      "Imported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_USVIFeb2020I_run3.csv as PairedEndFastqManifestPhred33V2 to /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_USVIFeb2020/Import_files_Apprill_USVIFeb2020I_run3\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for f1 in /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_Apprill_USVIFeb2020I_run*\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.csv};\n",
    "\n",
    "qiime tools import \\\n",
    "--output-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_USVIFeb2020/Import_$fn \\\n",
    "--type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "--input-path $f1 \\\n",
    "--input-format PairedEndFastqManifestPhred33V2\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: Folder_Apprill_USVIFeb2020/Import_files_Apprill_USVIFeb2020I_run1.qzv\n",
      "Saved Visualization to: Folder_Apprill_USVIFeb2020/Import_files_Apprill_USVIFeb2020I_run2.qzv\n",
      "Saved Visualization to: Folder_Apprill_USVIFeb2020/Import_files_Apprill_USVIFeb2020I_run3.qzv\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for f1 in /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_USVIFeb2020/Import_files_Apprill_USVIFeb2020I_run*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data $f1 \\\n",
    "--o-visualization Folder_Apprill_USVIFeb2020/$fn.qzv\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/100tiss_S31_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/100tiss_S31_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/100tiss_S31_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/100tiss_S31_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.51 s (23 us/read; 2.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,860\n",
      "  Read 1 with adapter:                 107,151 (98.4%)\n",
      "  Read 2 with adapter:                 108,024 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,860 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,778,360 bp\n",
      "  Read 1:    16,422,300 bp\n",
      "  Read 2:    16,356,060 bp\n",
      "Total written (filtered):     28,583,221 bp (87.2%)\n",
      "  Read 1:    14,386,847 bp\n",
      "  Read 2:    14,196,374 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 107151 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t0 4\n",
      "18\t649\t0.0\t1\t61 588\n",
      "19\t106207\t0.0\t1\t98641 7566\n",
      "20\t279\t0.0\t1\t62 217\n",
      "21\t1\t0.0\t1\t0 1\n",
      "34\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108024 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1700.9\t0\t1\n",
      "4\t1\t425.2\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t14\t0.0\t1\t7 7\n",
      "19\t938\t0.0\t1\t149 789\n",
      "20\t106812\t0.0\t2\t103300 3512\n",
      "21\t246\t0.0\t2\t53 193\n",
      "24\t1\t0.0\t2\t1\n",
      "25\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/102tiss_S3_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/102tiss_S3_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/102tiss_S3_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/102tiss_S3_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.91 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            124,762\n",
      "  Read 1 with adapter:                 122,580 (98.3%)\n",
      "  Read 2 with adapter:                 123,614 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       124,762 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,536,190 bp\n",
      "  Read 1:    18,807,957 bp\n",
      "  Read 2:    18,728,233 bp\n",
      "Total written (filtered):     32,736,364 bp (87.2%)\n",
      "  Read 1:    16,479,444 bp\n",
      "  Read 2:    16,256,920 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 122580 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t10\t0.0\t1\t8 2\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t5\t0.0\t1\t3 2\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t10\t0.0\t1\t4 6\n",
      "18\t724\t0.0\t1\t68 656\n",
      "19\t121449\t0.0\t1\t112682 8767\n",
      "20\t373\t0.0\t1\t65 308\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 123614 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1949.4\t0\t2\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t3\t0.0\t1\t3\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t11\t0.0\t1\t5 6\n",
      "19\t1187\t0.0\t1\t157 1030\n",
      "20\t122081\t0.0\t2\t118252 3829\n",
      "21\t319\t0.0\t2\t60 259\n",
      "22\t1\t0.0\t2\t0 1\n",
      "25\t1\t0.0\t2\t1\n",
      "30\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/104tiss_S32_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/104tiss_S32_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/104tiss_S32_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/104tiss_S32_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.40 s (22 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,646\n",
      "  Read 1 with adapter:                 109,925 (98.5%)\n",
      "  Read 2 with adapter:                 110,933 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,646 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,592,610 bp\n",
      "  Read 1:    16,830,739 bp\n",
      "  Read 2:    16,761,871 bp\n",
      "Total written (filtered):     29,286,440 bp (87.2%)\n",
      "  Read 1:    14,742,524 bp\n",
      "  Read 2:    14,543,916 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109925 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t5\t0.0\t1\t5\n",
      "13\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t599\t0.0\t1\t57 542\n",
      "19\t109016\t0.0\t1\t101064 7952\n",
      "20\t297\t0.0\t1\t47 250\n",
      "22\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110933 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t9\t0.0\t1\t2 7\n",
      "19\t947\t0.0\t1\t121 826\n",
      "20\t109671\t0.0\t2\t106509 3162\n",
      "21\t299\t0.0\t2\t58 241\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/168eco_S8_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/168eco_S8_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/168eco_S8_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/168eco_S8_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.47 s (24 us/read; 2.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            102,779\n",
      "  Read 1 with adapter:                 101,270 (98.5%)\n",
      "  Read 2 with adapter:                 102,061 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       102,779 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,955,653 bp\n",
      "  Read 1:    15,509,508 bp\n",
      "  Read 2:    15,446,145 bp\n",
      "Total written (filtered):     26,991,578 bp (87.2%)\n",
      "  Read 1:    13,585,767 bp\n",
      "  Read 2:    13,405,811 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101270 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t622\t0.0\t1\t45 577\n",
      "19\t100362\t0.0\t1\t93986 6376\n",
      "20\t274\t0.0\t1\t46 228\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 102061 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1605.9\t0\t2\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t1028\t0.0\t1\t129 899\n",
      "20\t100750\t0.0\t2\t98150 2600\n",
      "21\t264\t0.0\t2\t49 215\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/171eco_S9_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/171eco_S9_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/171eco_S9_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/171eco_S9_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.49 s (23 us/read; 2.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,746\n",
      "  Read 1 with adapter:                 107,091 (98.5%)\n",
      "  Read 2 with adapter:                 108,087 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,746 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,760,523 bp\n",
      "  Read 1:    16,414,104 bp\n",
      "  Read 2:    16,346,419 bp\n",
      "Total written (filtered):     28,565,280 bp (87.2%)\n",
      "  Read 1:    14,379,767 bp\n",
      "  Read 2:    14,185,513 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 107091 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t1 3\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t9\t0.0\t1\t3 6\n",
      "18\t575\t0.0\t1\t43 532\n",
      "19\t106244\t0.0\t1\t98896 7348\n",
      "20\t252\t0.0\t1\t36 216\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108087 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1699.2\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t4\t0.0\t1\t1 3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t8\t0.0\t1\t2 6\n",
      "19\t1009\t0.0\t1\t146 863\n",
      "20\t106781\t0.0\t2\t103994 2787\n",
      "21\t276\t0.0\t2\t51 225\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/172eco_S10_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/172eco_S10_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/172eco_S10_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/172eco_S10_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.57 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,517\n",
      "  Read 1 with adapter:                 107,689 (98.3%)\n",
      "  Read 2 with adapter:                 108,616 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,517 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,984,055 bp\n",
      "  Read 1:    16,526,014 bp\n",
      "  Read 2:    16,458,041 bp\n",
      "Total written (filtered):     28,766,917 bp (87.2%)\n",
      "  Read 1:    14,480,271 bp\n",
      "  Read 2:    14,286,646 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 107689 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t9\t0.0\t1\t5 4\n",
      "18\t641\t0.0\t1\t49 592\n",
      "19\t106714\t0.0\t1\t99603 7111\n",
      "20\t317\t0.0\t1\t50 267\n",
      "21\t1\t0.0\t1\t1\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108616 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1711.2\t0\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t15\t0.0\t1\t6 9\n",
      "19\t1090\t0.0\t1\t143 947\n",
      "20\t107210\t0.0\t2\t104334 2876\n",
      "21\t284\t0.0\t2\t63 221\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/173eco_S11_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/173eco_S11_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/173eco_S11_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/173eco_S11_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.61 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            105,235\n",
      "  Read 1 with adapter:                 103,571 (98.4%)\n",
      "  Read 2 with adapter:                 104,446 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       105,235 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,696,743 bp\n",
      "  Read 1:    15,880,494 bp\n",
      "  Read 2:    15,816,249 bp\n",
      "Total written (filtered):     27,641,118 bp (87.2%)\n",
      "  Read 1:    13,913,002 bp\n",
      "  Read 2:    13,728,116 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103571 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t596\t0.0\t1\t49 547\n",
      "19\t102677\t0.0\t1\t95646 7031\n",
      "20\t284\t0.0\t1\t53 231\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 104446 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1644.3\t0\t1\n",
      "4\t1\t411.1\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t8\t0.0\t1\t3 5\n",
      "19\t1009\t0.0\t1\t138 871\n",
      "20\t103141\t0.0\t2\t100267 2874\n",
      "21\t281\t0.0\t2\t66 215\n",
      "39\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/175eco_S12_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/175eco_S12_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/175eco_S12_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/175eco_S12_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.29 s (21 us/read; 2.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,439\n",
      "  Read 1 with adapter:                 109,601 (98.4%)\n",
      "  Read 2 with adapter:                 110,527 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,439 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,573,008 bp\n",
      "  Read 1:    16,819,982 bp\n",
      "  Read 2:    16,753,026 bp\n",
      "Total written (filtered):     29,281,390 bp (87.2%)\n",
      "  Read 1:    14,738,013 bp\n",
      "  Read 2:    14,543,377 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109601 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t8\t0.0\t1\t7 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t6\t0.0\t1\t1 5\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t656\t0.0\t1\t48 608\n",
      "19\t108612\t0.0\t1\t101123 7489\n",
      "20\t304\t0.0\t1\t51 253\n",
      "27\t1\t0.0\t1\t0 1\n",
      "39\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110527 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1741.2\t0\t1\n",
      "4\t2\t435.3\t0\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t12\t0.0\t1\t7 5\n",
      "19\t1088\t0.0\t1\t135 953\n",
      "20\t109123\t0.0\t2\t106026 3097\n",
      "21\t294\t0.0\t2\t74 220\n",
      "24\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/181eco_S14_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/181eco_S14_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/181eco_S14_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/181eco_S14_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.56 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,348\n",
      "  Read 1 with adapter:                 108,506 (98.3%)\n",
      "  Read 2 with adapter:                 109,555 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       110,348 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,213,998 bp\n",
      "  Read 1:    16,641,680 bp\n",
      "  Read 2:    16,572,318 bp\n",
      "Total written (filtered):     28,962,791 bp (87.2%)\n",
      "  Read 1:    14,580,564 bp\n",
      "  Read 2:    14,382,227 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108506 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "7\t1\t6.7\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t6\t0.0\t1\t3 3\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t5\t0.0\t1\t0 5\n",
      "18\t703\t0.0\t1\t50 653\n",
      "19\t107489\t0.0\t1\t99959 7530\n",
      "20\t294\t0.0\t1\t56 238\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 109555 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1724.2\t0\t1\n",
      "4\t2\t431.0\t0\t2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t10\t0.0\t1\t5 5\n",
      "19\t1185\t0.0\t1\t128 1057\n",
      "20\t108056\t0.0\t2\t105029 3027\n",
      "21\t288\t0.0\t2\t75 213\n",
      "22\t1\t0.0\t2\t1\n",
      "24\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/183eco_S15_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/183eco_S15_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/183eco_S15_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/183eco_S15_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.65 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,392\n",
      "  Read 1 with adapter:                  99,925 (98.6%)\n",
      "  Read 2 with adapter:                 100,696 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       101,392 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,518,929 bp\n",
      "  Read 1:    15,291,634 bp\n",
      "  Read 2:    15,227,295 bp\n",
      "Total written (filtered):     26,607,838 bp (87.2%)\n",
      "  Read 1:    13,393,444 bp\n",
      "  Read 2:    13,214,394 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99925 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t658\t0.0\t1\t40 618\n",
      "19\t98965\t0.0\t1\t92663 6302\n",
      "20\t287\t0.0\t1\t52 235\n",
      "22\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t1\t1\n",
      "40\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 100696 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1584.2\t0\t2\n",
      "4\t1\t396.1\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t0 4\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t13\t0.0\t1\t5 8\n",
      "19\t1161\t0.0\t1\t111 1050\n",
      "20\t99234\t0.0\t2\t96503 2731\n",
      "21\t269\t0.0\t2\t47 222\n",
      "22\t1\t0.0\t2\t1\n",
      "27\t1\t0.0\t2\t1\n",
      "28\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/187eco_S16_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/187eco_S16_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/187eco_S16_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/187eco_S16_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.50 s (23 us/read; 2.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,236\n",
      "  Read 1 with adapter:                 106,693 (98.6%)\n",
      "  Read 2 with adapter:                 107,444 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,236 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,551,737 bp\n",
      "  Read 1:    16,310,118 bp\n",
      "  Read 2:    16,241,619 bp\n",
      "Total written (filtered):     28,377,235 bp (87.2%)\n",
      "  Read 1:    14,283,351 bp\n",
      "  Read 2:    14,093,884 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 106693 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t7\t0.0\t1\t5 2\n",
      "18\t665\t0.0\t1\t52 613\n",
      "19\t105714\t0.0\t1\t99182 6532\n",
      "20\t294\t0.0\t1\t54 240\n",
      "21\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 107444 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1691.2\t0\t3\n",
      "4\t2\t422.8\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t7\t0.0\t1\t3 4\n",
      "19\t1261\t0.0\t1\t108 1153\n",
      "20\t105878\t0.0\t2\t102933 2945\n",
      "21\t280\t0.0\t2\t68 212\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/190eco_S17_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/190eco_S17_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/190eco_S17_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/190eco_S17_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.64 s (23 us/read; 2.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,461\n",
      "  Read 1 with adapter:                 113,760 (98.5%)\n",
      "  Read 2 with adapter:                 114,631 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,461 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,740,585 bp\n",
      "  Read 1:    17,407,661 bp\n",
      "  Read 2:    17,332,924 bp\n",
      "Total written (filtered):     30,288,136 bp (87.2%)\n",
      "  Read 1:    15,246,701 bp\n",
      "  Read 2:    15,041,435 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113760 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.4\t0\t3\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t708\t0.0\t1\t45 663\n",
      "19\t112735\t0.0\t1\t105164 7571\n",
      "20\t297\t0.0\t1\t52 245\n",
      "56\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 114631 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1804.1\t0\t6\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t9\t0.0\t1\t7 2\n",
      "19\t1248\t0.0\t1\t132 1116\n",
      "20\t113094\t0.0\t2\t109667 3427\n",
      "21\t266\t0.0\t2\t67 199\n",
      "22\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/191eco_S18_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/191eco_S18_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/191eco_S18_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/191eco_S18_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.69 s (23 us/read; 2.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,684\n",
      "  Read 1 with adapter:                 115,849 (98.4%)\n",
      "  Read 2 with adapter:                 116,660 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       117,684 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,441,261 bp\n",
      "  Read 1:    17,758,554 bp\n",
      "  Read 2:    17,682,707 bp\n",
      "Total written (filtered):     30,908,487 bp (87.2%)\n",
      "  Read 1:    15,557,877 bp\n",
      "  Read 2:    15,350,610 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 115849 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t695\t0.0\t1\t46 649\n",
      "19\t114828\t0.0\t1\t107460 7368\n",
      "20\t307\t0.0\t1\t56 251\n",
      "21\t1\t0.0\t1\t0 1\n",
      "23\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 116660 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1838.8\t0\t4\n",
      "4\t1\t459.7\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t14\t0.0\t1\t10 4\n",
      "19\t1239\t0.0\t1\t114 1125\n",
      "20\t115097\t0.0\t2\t111628 3469\n",
      "21\t294\t0.0\t2\t67 227\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/201eco_S19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/201eco_S19_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/201eco_S19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/201eco_S19_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.59 s (23 us/read; 2.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,303\n",
      "  Read 1 with adapter:                 111,637 (98.5%)\n",
      "  Read 2 with adapter:                 112,372 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,303 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,089,492 bp\n",
      "  Read 1:    17,081,040 bp\n",
      "  Read 2:    17,008,452 bp\n",
      "Total written (filtered):     29,722,493 bp (87.2%)\n",
      "  Read 1:    14,960,395 bp\n",
      "  Read 2:    14,762,098 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 111637 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "14\t7\t0.0\t1\t1 6\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t712\t0.0\t1\t40 672\n",
      "19\t110599\t0.0\t1\t103281 7318\n",
      "20\t312\t0.0\t1\t49 263\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 112372 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1770.4\t0\t1\n",
      "4\t2\t442.6\t0\t2\n",
      "12\t3\t0.0\t1\t2 1\n",
      "14\t4\t0.0\t1\t0 4\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t8\t0.0\t1\t3 5\n",
      "19\t1236\t0.0\t1\t125 1111\n",
      "20\t110835\t0.0\t2\t107268 3567\n",
      "21\t275\t0.0\t2\t55 220\n",
      "33\t1\t0.0\t2\t0 1\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/100tiss_S31_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/100tiss_S31_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/100tiss_S31_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/100tiss_S31_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/102tiss_S3_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/102tiss_S3_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/102tiss_S3_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/102tiss_S3_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/104tiss_S32_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/104tiss_S32_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/104tiss_S32_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/104tiss_S32_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/168eco_S8_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/168eco_S8_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/168eco_S8_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/168eco_S8_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/171eco_S9_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/171eco_S9_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/171eco_S9_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/171eco_S9_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/172eco_S10_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/172eco_S10_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/172eco_S10_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/172eco_S10_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/173eco_S11_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/173eco_S11_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/173eco_S11_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/173eco_S11_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/175eco_S12_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/175eco_S12_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/175eco_S12_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/175eco_S12_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/181eco_S14_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/181eco_S14_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/181eco_S14_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/181eco_S14_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/183eco_S15_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/183eco_S15_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/183eco_S15_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/183eco_S15_50_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/187eco_S16_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/187eco_S16_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/187eco_S16_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/187eco_S16_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/190eco_S17_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/190eco_S17_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/190eco_S17_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/190eco_S17_52_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/191eco_S18_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/191eco_S18_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/191eco_S18_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/191eco_S18_53_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/201eco_S19_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/201eco_S19_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/201eco_S19_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/201eco_S19_54_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/202eco_S20_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/202eco_S20_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/202eco_S20_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/202eco_S20_55_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/202eco_S20_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/202eco_S20_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/202eco_S20_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/202eco_S20_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.88 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            122,719\n",
      "  Read 1 with adapter:                 120,749 (98.4%)\n",
      "  Read 2 with adapter:                 121,553 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       122,719 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,922,645 bp\n",
      "  Read 1:    18,500,540 bp\n",
      "  Read 2:    18,422,105 bp\n",
      "Total written (filtered):     32,199,173 bp (87.2%)\n",
      "  Read 1:    16,206,777 bp\n",
      "  Read 2:    15,992,396 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 120749 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t753\t0.0\t1\t56 697\n",
      "19\t119637\t0.0\t1\t111443 8194\n",
      "20\t344\t0.0\t1\t65 279\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 121553 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1917.5\t0\t7\n",
      "4\t2\t479.4\t0\t2\n",
      "11\t2\t0.0\t1\t1 1\n",
      "13\t4\t0.0\t1\t2 2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t12\t0.0\t1\t4 8\n",
      "19\t1421\t0.0\t1\t132 1289\n",
      "20\t119799\t0.0\t2\t116072 3727\n",
      "21\t297\t0.0\t2\t65 232\n",
      "24\t1\t0.0\t2\t1\n",
      "34\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/203eco_S21_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/203eco_S21_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/203eco_S21_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/203eco_S21_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.62 s (24 us/read; 2.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,522\n",
      "  Read 1 with adapter:                 107,874 (98.5%)\n",
      "  Read 2 with adapter:                 108,735 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,522 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,985,476 bp\n",
      "  Read 1:    16,527,086 bp\n",
      "  Read 2:    16,458,390 bp\n",
      "Total written (filtered):     28,762,617 bp (87.2%)\n",
      "  Read 1:    14,477,959 bp\n",
      "  Read 2:    14,284,658 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 107874 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t684\t0.0\t1\t51 633\n",
      "19\t106909\t0.0\t1\t100124 6785\n",
      "20\t262\t0.0\t1\t46 216\n",
      "37\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108735 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1711.3\t0\t4\n",
      "4\t1\t427.8\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t8\t0.0\t1\t1 7\n",
      "18\t6\t0.0\t1\t5 1\n",
      "19\t1052\t0.0\t1\t104 948\n",
      "20\t107387\t0.0\t2\t104576 2811\n",
      "21\t265\t0.0\t2\t47 218\n",
      "31\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/205eco_S22_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/205eco_S22_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/205eco_S22_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/205eco_S22_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.38 s (20 us/read; 3.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,128\n",
      "  Read 1 with adapter:                 118,305 (98.5%)\n",
      "  Read 2 with adapter:                 119,184 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       120,128 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,131,362 bp\n",
      "  Read 1:    18,104,134 bp\n",
      "  Read 2:    18,027,228 bp\n",
      "Total written (filtered):     31,501,494 bp (87.2%)\n",
      "  Read 1:    15,856,865 bp\n",
      "  Read 2:    15,644,629 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 118305 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t1 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t0 3\n",
      "17\t7\t0.0\t1\t4 3\n",
      "18\t776\t0.0\t1\t61 715\n",
      "19\t117196\t0.0\t1\t109461 7735\n",
      "20\t315\t0.0\t1\t54 261\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 119184 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t8\t0.0\t1\t3 5\n",
      "19\t1322\t0.0\t1\t121 1201\n",
      "20\t117555\t0.0\t2\t114003 3552\n",
      "21\t290\t0.0\t2\t71 219\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/206eco_S23_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/206eco_S23_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/206eco_S23_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/206eco_S23_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.47 s (22 us/read; 2.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,653\n",
      "  Read 1 with adapter:                 109,050 (98.6%)\n",
      "  Read 2 with adapter:                 109,837 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       110,653 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,269,672 bp\n",
      "  Read 1:    16,670,921 bp\n",
      "  Read 2:    16,598,751 bp\n",
      "Total written (filtered):     29,002,575 bp (87.2%)\n",
      "  Read 1:    14,599,331 bp\n",
      "  Read 2:    14,403,244 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109050 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t682\t0.0\t1\t40 642\n",
      "19\t108030\t0.0\t1\t101104 6926\n",
      "20\t321\t0.0\t1\t59 262\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "39\t1\t0.0\t1\t1\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 109837 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1729.0\t0\t1\n",
      "8\t1\t1.7\t0\t1\n",
      "10\t5\t0.1\t1\t5\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t12\t0.0\t1\t8 4\n",
      "19\t1405\t0.0\t1\t130 1275\n",
      "20\t108110\t0.0\t2\t104792 3318\n",
      "21\t298\t0.0\t2\t57 241\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/207eco_S24_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/207eco_S24_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/207eco_S24_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/207eco_S24_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.62 s (24 us/read; 2.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,995\n",
      "  Read 1 with adapter:                 107,380 (98.5%)\n",
      "  Read 2 with adapter:                 108,173 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,995 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,832,352 bp\n",
      "  Read 1:    16,450,428 bp\n",
      "  Read 2:    16,381,924 bp\n",
      "Total written (filtered):     28,630,029 bp (87.2%)\n",
      "  Read 1:    14,410,607 bp\n",
      "  Read 2:    14,219,422 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 107380 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t1 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t9\t0.0\t1\t5 4\n",
      "18\t657\t0.0\t1\t48 609\n",
      "19\t106422\t0.0\t1\t99726 6696\n",
      "20\t276\t0.0\t1\t40 236\n",
      "21\t2\t0.0\t1\t2\n",
      "33\t1\t0.0\t1\t1\n",
      "39\t1\t0.0\t1\t1\n",
      "48\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108173 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1703.0\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t1174\t0.0\t1\t111 1063\n",
      "20\t106682\t0.0\t2\t103815 2867\n",
      "21\t293\t0.0\t2\t53 240\n",
      "28\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t1\n",
      "42\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/220tiss_S33_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/220tiss_S33_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/220tiss_S33_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/220tiss_S33_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.17 s (21 us/read; 2.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,557\n",
      "  Read 1 with adapter:                 102,681 (98.2%)\n",
      "  Read 2 with adapter:                 103,638 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       104,557 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,504,664 bp\n",
      "  Read 1:    15,785,876 bp\n",
      "  Read 2:    15,718,788 bp\n",
      "Total written (filtered):     27,481,890 bp (87.2%)\n",
      "  Read 1:    13,835,135 bp\n",
      "  Read 2:    13,646,755 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 102681 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t417\t0.0\t1\t44 373\n",
      "19\t102030\t0.0\t1\t94462 7568\n",
      "20\t230\t0.0\t1\t41 189\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 103638 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1633.7\t0\t1\n",
      "4\t3\t408.4\t0\t3\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t6\t0.0\t1\t1 5\n",
      "19\t822\t0.0\t1\t105 717\n",
      "20\t102567\t0.0\t2\t98793 3774\n",
      "21\t229\t0.0\t2\t51 178\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/222tiss_S34_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/222tiss_S34_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/222tiss_S34_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/222tiss_S34_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.40 s (22 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,690\n",
      "  Read 1 with adapter:                 109,603 (98.1%)\n",
      "  Read 2 with adapter:                 110,557 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,690 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,651,169 bp\n",
      "  Read 1:    16,860,876 bp\n",
      "  Read 2:    16,790,293 bp\n",
      "Total written (filtered):     29,358,977 bp (87.2%)\n",
      "  Read 1:    14,778,820 bp\n",
      "  Read 2:    14,580,157 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109603 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t8\t0.0\t1\t3 5\n",
      "18\t649\t0.0\t1\t65 584\n",
      "19\t108655\t0.0\t1\t99554 9101\n",
      "20\t286\t0.0\t1\t48 238\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110557 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1745.2\t0\t2\n",
      "4\t3\t436.3\t0\t3\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t10\t0.0\t1\t5 5\n",
      "19\t1170\t0.0\t1\t155 1015\n",
      "20\t109061\t0.0\t2\t105001 4060\n",
      "21\t303\t0.0\t2\t62 241\n",
      "31\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/225tis_S40_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/225tis_S40_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/225tis_S40_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/225tis_S40_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.14 s (22 us/read; 2.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,906\n",
      "  Read 1 with adapter:                  93,107 (98.1%)\n",
      "  Read 2 with adapter:                  93,951 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        94,906 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,591,283 bp\n",
      "  Read 1:    14,323,325 bp\n",
      "  Read 2:    14,267,958 bp\n",
      "Total written (filtered):     24,944,471 bp (87.2%)\n",
      "  Read 1:    12,554,650 bp\n",
      "  Read 2:    12,389,821 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 93107 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.4\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t543\t0.0\t1\t36 507\n",
      "19\t92295\t0.0\t1\t84813 7482\n",
      "20\t253\t0.0\t1\t43 210\n",
      "21\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 93951 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1482.9\t0\t2\n",
      "4\t1\t370.7\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t6\t0.0\t1\t4 2\n",
      "19\t1087\t0.0\t1\t111 976\n",
      "20\t92570\t0.0\t2\t89237 3333\n",
      "21\t274\t0.0\t2\t59 215\n",
      "22\t3\t0.0\t2\t1 2\n",
      "42\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/232tiss_S41_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/232tiss_S41_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/232tiss_S41_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/232tiss_S41_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.66 s (25 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,566\n",
      "  Read 1 with adapter:                 106,603 (98.2%)\n",
      "  Read 2 with adapter:                 107,560 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,566 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,676,730 bp\n",
      "  Read 1:    16,370,923 bp\n",
      "  Read 2:    16,305,807 bp\n",
      "Total written (filtered):     28,501,239 bp (87.2%)\n",
      "  Read 1:    14,345,848 bp\n",
      "  Read 2:    14,155,391 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 106603 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t2 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t615\t0.0\t1\t55 560\n",
      "19\t105676\t0.0\t1\t97434 8242\n",
      "20\t297\t0.0\t1\t44 253\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 107560 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1696.3\t0\t3\n",
      "4\t1\t424.1\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t12\t0.0\t1\t5 7\n",
      "19\t910\t0.0\t1\t137 773\n",
      "20\t106358\t0.0\t2\t102703 3655\n",
      "21\t266\t0.0\t2\t72 194\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/262tiss_S35_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/262tiss_S35_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/262tiss_S35_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/262tiss_S35_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.95 s (20 us/read; 2.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,112\n",
      "  Read 1 with adapter:                  94,018 (97.8%)\n",
      "  Read 2 with adapter:                  95,172 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        96,112 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,957,089 bp\n",
      "  Read 1:    14,510,479 bp\n",
      "  Read 2:    14,446,610 bp\n",
      "Total written (filtered):     25,268,381 bp (87.3%)\n",
      "  Read 1:    12,724,458 bp\n",
      "  Read 2:    12,543,923 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 94018 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t375.4\t0\t1\n",
      "6\t1\t23.5\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t518\t0.0\t1\t49 469\n",
      "19\t93234\t0.0\t1\t84815 8419\n",
      "20\t250\t0.0\t1\t44 206\n",
      "46\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 95172 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t375.4\t0\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t10\t0.0\t1\t3 7\n",
      "19\t931\t0.0\t1\t113 818\n",
      "20\t93997\t0.0\t2\t90810 3187\n",
      "21\t227\t0.0\t2\t46 181\n",
      "26\t1\t0.0\t2\t1\n",
      "31\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/271eco_S25_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/271eco_S25_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/271eco_S25_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/271eco_S25_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.93 s (19 us/read; 3.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,540\n",
      "  Read 1 with adapter:                 101,949 (98.5%)\n",
      "  Read 2 with adapter:                 102,846 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       103,540 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,169,027 bp\n",
      "  Read 1:    15,615,618 bp\n",
      "  Read 2:    15,553,409 bp\n",
      "Total written (filtered):     27,176,446 bp (87.2%)\n",
      "  Read 1:    13,679,027 bp\n",
      "  Read 2:    13,497,419 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101949 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t9\t0.0\t1\t4 5\n",
      "18\t633\t0.0\t1\t45 588\n",
      "19\t101028\t0.0\t1\t93992 7036\n",
      "20\t270\t0.0\t1\t48 222\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 102846 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1617.8\t0\t1\n",
      "4\t1\t404.5\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t9\t0.0\t1\t6 3\n",
      "19\t1107\t0.0\t1\t117 990\n",
      "20\t101421\t0.0\t2\t98539 2882\n",
      "21\t295\t0.0\t2\t66 229\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/75Ltiss_S37_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/75Ltiss_S37_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/75Ltiss_S37_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/75Ltiss_S37_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.48 s (25 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,192\n",
      "  Read 1 with adapter:                  99,505 (98.3%)\n",
      "  Read 2 with adapter:                 100,436 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       101,192 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,495,339 bp\n",
      "  Read 1:    15,277,876 bp\n",
      "  Read 2:    15,217,463 bp\n",
      "Total written (filtered):     26,596,918 bp (87.2%)\n",
      "  Read 1:    13,387,518 bp\n",
      "  Read 2:    13,209,400 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99505 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t450\t0.0\t1\t42 408\n",
      "19\t98797\t0.0\t1\t92161 6636\n",
      "20\t244\t0.0\t1\t47 197\n",
      "21\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 100436 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1581.1\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t6\t0.0\t1\t2 4\n",
      "19\t760\t0.0\t1\t118 642\n",
      "20\t99455\t0.0\t2\t96642 2813\n",
      "21\t202\t0.0\t2\t37 165\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/80Ltiss_S4_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/80Ltiss_S4_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/80Ltiss_S4_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/80Ltiss_S4_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.75 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            118,082\n",
      "  Read 1 with adapter:                 116,144 (98.4%)\n",
      "  Read 2 with adapter:                 117,079 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       118,082 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,503,599 bp\n",
      "  Read 1:    17,789,406 bp\n",
      "  Read 2:    17,714,193 bp\n",
      "Total written (filtered):     30,956,585 bp (87.2%)\n",
      "  Read 1:    15,583,128 bp\n",
      "  Read 2:    15,373,457 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 116144 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t730\t0.0\t1\t71 659\n",
      "19\t115037\t0.0\t1\t106705 8332\n",
      "20\t361\t0.0\t1\t53 308\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 117079 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t16\t0.0\t1\t7 9\n",
      "19\t1080\t0.0\t1\t141 939\n",
      "20\t115666\t0.0\t2\t112065 3601\n",
      "21\t303\t0.0\t2\t72 231\n",
      "43\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/81tiss_S27_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/81tiss_S27_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/81tiss_S27_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/81tiss_S27_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.13 s (18 us/read; 3.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,724\n",
      "  Read 1 with adapter:                 113,754 (98.3%)\n",
      "  Read 2 with adapter:                 114,636 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,724 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,890,138 bp\n",
      "  Read 1:    17,469,994 bp\n",
      "  Read 2:    17,420,144 bp\n",
      "Total written (filtered):     30,437,281 bp (87.2%)\n",
      "  Read 1:    15,309,092 bp\n",
      "  Read 2:    15,128,189 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113754 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t4\t0.4\t0\t4\n",
      "10\t2\t0.1\t1\t0 2\n",
      "12\t6\t0.0\t1\t5 1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t11\t0.0\t1\t5 6\n",
      "18\t604\t0.0\t1\t69 535\n",
      "19\t112798\t0.0\t1\t104844 7954\n",
      "20\t324\t0.0\t1\t72 252\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 114636 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t452.0\t0\t2\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t13\t0.0\t1\t8 5\n",
      "19\t945\t0.0\t1\t132 813\n",
      "20\t113353\t0.0\t2\t110005 3348\n",
      "21\t308\t0.0\t2\t73 235\n",
      "26\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/82Ltiss_S38_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/82Ltiss_S38_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/82Ltiss_S38_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/82Ltiss_S38_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.30 s (24 us/read; 2.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             95,552\n",
      "  Read 1 with adapter:                  94,084 (98.5%)\n",
      "  Read 2 with adapter:                  94,914 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        95,552 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,731,617 bp\n",
      "  Read 1:    14,396,960 bp\n",
      "  Read 2:    14,334,657 bp\n",
      "Total written (filtered):     25,046,865 bp (87.2%)\n",
      "  Read 1:    12,609,734 bp\n",
      "  Read 2:    12,437,131 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 94084 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t4\t0.4\t0\t4\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t6\t0.0\t1\t5 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t7\t0.0\t1\t5 2\n",
      "18\t509\t0.0\t1\t49 460\n",
      "19\t93294\t0.0\t1\t86779 6515\n",
      "20\t259\t0.0\t1\t47 212\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 94914 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1493.0\t0\t2\n",
      "4\t2\t373.2\t0\t2\n",
      "8\t1\t1.5\t0\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t6\t0.0\t1\t6\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t846\t0.0\t1\t121 725\n",
      "20\t93803\t0.0\t2\t91067 2736\n",
      "21\t240\t0.0\t2\t49 191\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/83tiss_S28_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/83tiss_S28_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/83tiss_S28_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/83tiss_S28_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.21 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,811\n",
      "  Read 1 with adapter:                  81,266 (94.7%)\n",
      "  Read 2 with adapter:                  84,456 (98.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        85,811 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,823,477 bp\n",
      "  Read 1:    12,935,648 bp\n",
      "  Read 2:    12,887,829 bp\n",
      "Total written (filtered):     22,591,216 bp (87.5%)\n",
      "  Read 1:    11,391,931 bp\n",
      "  Read 2:    11,199,285 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 81266 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.3\t0\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t3\t0.0\t1\t3\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t9\t0.0\t1\t3 6\n",
      "18\t482\t0.0\t1\t42 440\n",
      "19\t80564\t0.0\t1\t70948 9616\n",
      "20\t203\t0.0\t1\t33 170\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 84456 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1340.8\t0\t1\n",
      "4\t2\t335.2\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "18\t8\t0.0\t1\t4 4\n",
      "19\t711\t0.0\t1\t117 594\n",
      "20\t83504\t0.0\t2\t80927 2577\n",
      "21\t227\t0.0\t2\t33 194\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/203eco_S21_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/203eco_S21_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/203eco_S21_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/203eco_S21_56_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/205eco_S22_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/205eco_S22_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/205eco_S22_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/205eco_S22_57_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/206eco_S23_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/206eco_S23_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/206eco_S23_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/206eco_S23_58_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/207eco_S24_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/207eco_S24_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/207eco_S24_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/207eco_S24_59_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/220tiss_S33_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/220tiss_S33_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/220tiss_S33_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/220tiss_S33_60_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/222tiss_S34_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/222tiss_S34_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/222tiss_S34_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/222tiss_S34_61_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/225tis_S40_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/225tis_S40_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/225tis_S40_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/225tis_S40_62_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/232tiss_S41_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/232tiss_S41_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/232tiss_S41_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/232tiss_S41_63_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/262tiss_S35_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/262tiss_S35_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/262tiss_S35_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/262tiss_S35_64_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/271eco_S25_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/271eco_S25_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/271eco_S25_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/271eco_S25_65_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/75Ltiss_S37_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/75Ltiss_S37_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/75Ltiss_S37_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/75Ltiss_S37_66_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/80Ltiss_S4_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/80Ltiss_S4_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/80Ltiss_S4_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/80Ltiss_S4_67_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/81tiss_S27_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/81tiss_S27_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/81tiss_S27_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/81tiss_S27_68_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/82Ltiss_S38_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/82Ltiss_S38_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/82Ltiss_S38_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/82Ltiss_S38_69_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/83tiss_S28_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/83tiss_S28_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/83tiss_S28_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/83tiss_S28_70_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/84Ltiss_S5_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/84Ltiss_S5_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/84Ltiss_S5_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/84Ltiss_S5_71_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/84Ltiss_S5_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/84Ltiss_S5_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/84Ltiss_S5_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/84Ltiss_S5_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.61 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,328\n",
      "  Read 1 with adapter:                 113,212 (98.2%)\n",
      "  Read 2 with adapter:                 114,227 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,328 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,706,995 bp\n",
      "  Read 1:    17,389,832 bp\n",
      "  Read 2:    17,317,163 bp\n",
      "Total written (filtered):     30,272,911 bp (87.2%)\n",
      "  Read 1:    15,239,312 bp\n",
      "  Read 2:    15,033,599 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113212 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t7\t0.0\t1\t7\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t727\t0.0\t1\t68 659\n",
      "19\t112118\t0.0\t1\t103915 8203\n",
      "20\t339\t0.0\t1\t71 268\n",
      "31\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 114227 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1802.0\t0\t4\n",
      "4\t4\t450.5\t0\t4\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t14\t0.0\t1\t6 8\n",
      "19\t1037\t0.0\t1\t139 898\n",
      "20\t112869\t0.0\t2\t109490 3379\n",
      "21\t286\t0.0\t2\t54 232\n",
      "24\t1\t0.0\t2\t1\n",
      "25\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/86tiss_S39_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/86tiss_S39_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/86tiss_S39_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/86tiss_S39_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.14 s (22 us/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             98,868\n",
      "  Read 1 with adapter:                  97,161 (98.3%)\n",
      "  Read 2 with adapter:                  97,977 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        98,868 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,766,118 bp\n",
      "  Read 1:    14,911,266 bp\n",
      "  Read 2:    14,854,852 bp\n",
      "Total written (filtered):     25,961,561 bp (87.2%)\n",
      "  Read 1:    13,065,577 bp\n",
      "  Read 2:    12,895,984 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 97161 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t580\t0.0\t1\t45 535\n",
      "19\t96313\t0.0\t1\t89708 6605\n",
      "20\t256\t0.0\t1\t46 210\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 97977 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t4\t0.0\t1\t4\n",
      "19\t868\t0.0\t1\t117 751\n",
      "20\t96848\t0.0\t2\t93933 2915\n",
      "21\t249\t0.0\t2\t43 206\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/87tiss_S7_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/87tiss_S7_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/87tiss_S7_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/87tiss_S7_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.63 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,946\n",
      "  Read 1 with adapter:                 113,971 (98.3%)\n",
      "  Read 2 with adapter:                 115,087 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,946 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,923,155 bp\n",
      "  Read 1:    17,497,144 bp\n",
      "  Read 2:    17,426,011 bp\n",
      "Total written (filtered):     30,456,938 bp (87.2%)\n",
      "  Read 1:    15,331,964 bp\n",
      "  Read 2:    15,124,974 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113971 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t5\t0.0\t1\t1 4\n",
      "18\t557\t0.0\t1\t35 522\n",
      "19\t113068\t0.0\t1\t104807 8261\n",
      "20\t332\t0.0\t1\t50 282\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 115087 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t7\t0.0\t1\t5 2\n",
      "18\t15\t0.0\t1\t6 9\n",
      "19\t875\t0.0\t1\t120 755\n",
      "20\t113879\t0.0\t2\t110409 3470\n",
      "21\t298\t0.0\t2\t67 231\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/88tiss_S1_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/88tiss_S1_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/88tiss_S1_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/88tiss_S1_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.44 s (22 us/read; 2.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,577\n",
      "  Read 1 with adapter:                 106,893 (98.4%)\n",
      "  Read 2 with adapter:                 107,835 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,577 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,709,569 bp\n",
      "  Read 1:    16,388,500 bp\n",
      "  Read 2:    16,321,069 bp\n",
      "Total written (filtered):     28,522,858 bp (87.2%)\n",
      "  Read 1:    14,357,805 bp\n",
      "  Read 2:    14,165,053 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 106893 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t470\t0.0\t1\t52 418\n",
      "19\t106153\t0.0\t1\t98987 7166\n",
      "20\t255\t0.0\t1\t47 208\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 107835 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1696.5\t0\t2\n",
      "4\t3\t424.1\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t7\t0.0\t1\t1 6\n",
      "19\t777\t0.0\t1\t107 670\n",
      "20\t106786\t0.0\t2\t103687 3099\n",
      "21\t250\t0.0\t2\t65 185\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/90tiss_S29_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/90tiss_S29_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/90tiss_S29_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/90tiss_S29_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.32 s (21 us/read; 2.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,853\n",
      "  Read 1 with adapter:                 108,023 (98.3%)\n",
      "  Read 2 with adapter:                 108,990 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,853 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,078,710 bp\n",
      "  Read 1:    16,571,449 bp\n",
      "  Read 2:    16,507,261 bp\n",
      "Total written (filtered):     28,847,432 bp (87.2%)\n",
      "  Read 1:    14,519,330 bp\n",
      "  Read 2:    14,328,102 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108023 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t535\t0.0\t1\t57 478\n",
      "19\t107195\t0.0\t1\t99785 7410\n",
      "20\t277\t0.0\t1\t56 221\n",
      "28\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108990 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1716.5\t0\t2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t3\t0.0\t1\t2 1\n",
      "19\t857\t0.0\t1\t164 693\n",
      "20\t107836\t0.0\t2\t104995 2841\n",
      "21\t284\t0.0\t2\t54 230\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/92tiss_S2_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/92tiss_S2_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/92tiss_S2_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/92tiss_S2_76_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.18 s (21 us/read; 2.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,807\n",
      "  Read 1 with adapter:                 102,046 (98.3%)\n",
      "  Read 2 with adapter:                 103,068 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       103,807 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,289,120 bp\n",
      "  Read 1:    15,664,871 bp\n",
      "  Read 2:    15,624,249 bp\n",
      "Total written (filtered):     27,290,251 bp (87.2%)\n",
      "  Read 1:    13,726,417 bp\n",
      "  Read 2:    13,563,834 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 102046 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t6\t0.0\t1\t1 5\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t642\t0.0\t1\t43 599\n",
      "19\t101081\t0.0\t1\t93048 8033\n",
      "20\t304\t0.0\t1\t57 247\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 103068 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1622.0\t0\t4\n",
      "13\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t13\t0.0\t1\t3 10\n",
      "19\t1107\t0.0\t1\t123 984\n",
      "20\t101655\t0.0\t2\t98085 3570\n",
      "21\t278\t0.0\t2\t64 214\n",
      "28\t1\t0.0\t2\t0 1\n",
      "29\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/98tiss_S30_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/98tiss_S30_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/98tiss_S30_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/98tiss_S30_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.56 s (23 us/read; 2.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,705\n",
      "  Read 1 with adapter:                 109,875 (98.4%)\n",
      "  Read 2 with adapter:                 110,956 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,705 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,631,012 bp\n",
      "  Read 1:    16,849,909 bp\n",
      "  Read 2:    16,781,103 bp\n",
      "Total written (filtered):     29,325,296 bp (87.2%)\n",
      "  Read 1:    14,762,627 bp\n",
      "  Read 2:    14,562,669 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109875 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t2\t0.1\t1\t1 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t8\t0.0\t1\t4 4\n",
      "18\t576\t0.0\t1\t53 523\n",
      "19\t108997\t0.0\t1\t100964 8033\n",
      "20\t287\t0.0\t1\t66 221\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110956 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t436.3\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t15\t0.0\t1\t6 9\n",
      "19\t923\t0.0\t1\t122 801\n",
      "20\t109725\t0.0\t2\t106131 3594\n",
      "21\t287\t0.0\t2\t65 222\n",
      "36\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/D1_S6_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/D1_S6_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/D1_S6_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/D1_S6_78_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.83 s (51 us/read; 1.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,309\n",
      "  Read 1 with adapter:                  15,776 (96.7%)\n",
      "  Read 2 with adapter:                  16,171 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        16,309 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,827,055 bp\n",
      "  Read 1:     2,419,508 bp\n",
      "  Read 2:     2,407,547 bp\n",
      "Total written (filtered):      4,204,134 bp (87.1%)\n",
      "  Read 1:     2,119,830 bp\n",
      "  Read 2:     2,084,304 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 15776 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t115\t0.0\t1\t13 102\n",
      "19\t15612\t0.0\t1\t14114 1498\n",
      "20\t49\t0.0\t1\t2 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 16171 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t63.7\t0\t1\n",
      "18\t3\t0.0\t1\t0 3\n",
      "19\t192\t0.0\t1\t15 177\n",
      "20\t15938\t0.0\t2\t15430 508\n",
      "21\t37\t0.0\t2\t6 31\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/D4_S26_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/D4_S26_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/D4_S26_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/D4_S26_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.77 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,354\n",
      "  Read 1 with adapter:                  11,944 (96.7%)\n",
      "  Read 2 with adapter:                  12,025 (97.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        12,354 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,836,761 bp\n",
      "  Read 1:       939,877 bp\n",
      "  Read 2:       896,884 bp\n",
      "Total written (filtered):      1,369,563 bp (74.6%)\n",
      "  Read 1:       712,997 bp\n",
      "  Read 2:       656,566 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11944 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t1 1\n",
      "18\t75\t0.0\t1\t9 66\n",
      "19\t11834\t0.0\t1\t11191 643\n",
      "20\t33\t0.0\t1\t4 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 12025 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t190\t0.0\t1\t16 174\n",
      "20\t11791\t0.0\t2\t11498 293\n",
      "21\t39\t0.0\t2\t10 29\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/H2Ocontrol-run1_S43_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/H2Ocontrol-run1_S43_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/H2Ocontrol-run1_S43_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/H2Ocontrol-run1_S43_80_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.90 s (41 us/read; 1.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,782\n",
      "  Read 1 with adapter:                  21,448 (98.5%)\n",
      "  Read 2 with adapter:                  21,544 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        21,782 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,409,249 bp\n",
      "  Read 1:     2,229,895 bp\n",
      "  Read 2:     2,179,354 bp\n",
      "Total written (filtered):      3,571,189 bp (81.0%)\n",
      "  Read 1:     1,822,490 bp\n",
      "  Read 2:     1,748,699 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21448 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t149\t0.0\t1\t13 136\n",
      "19\t21249\t0.0\t1\t20033 1216\n",
      "20\t48\t0.0\t1\t7 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 21544 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t85.1\t0\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "19\t250\t0.0\t1\t31 219\n",
      "20\t21237\t0.0\t2\t20647 590\n",
      "21\t54\t0.0\t2\t10 44\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/MockE-run1_S42_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/MockE-run1_S42_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/MockE-run1_S42_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/MockE-run1_S42_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.70 s (23 us/read; 2.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,857\n",
      "  Read 1 with adapter:                  73,019 (97.5%)\n",
      "  Read 2 with adapter:                  74,297 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        74,857 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,529,101 bp\n",
      "  Read 1:    11,293,168 bp\n",
      "  Read 2:    11,235,933 bp\n",
      "Total written (filtered):     19,656,656 bp (87.3%)\n",
      "  Read 1:     9,906,019 bp\n",
      "  Read 2:     9,750,637 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 73019 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.3\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t376\t0.0\t1\t36 340\n",
      "19\t72438\t0.0\t1\t66724 5714\n",
      "20\t196\t0.0\t1\t34 162\n",
      "34\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 74297 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1169.6\t0\t2\n",
      "4\t2\t292.4\t0\t2\n",
      "9\t2\t0.3\t0\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t8\t0.0\t1\t3 5\n",
      "19\t704\t0.0\t1\t88 616\n",
      "20\t73389\t0.0\t2\t71724 1665\n",
      "21\t184\t0.0\t2\t36 148\n",
      "22\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/86tiss_S39_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/86tiss_S39_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/86tiss_S39_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/86tiss_S39_72_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/87tiss_S7_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/87tiss_S7_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/87tiss_S7_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/87tiss_S7_73_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/88tiss_S1_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/88tiss_S1_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/88tiss_S1_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/88tiss_S1_74_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/90tiss_S29_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/90tiss_S29_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/90tiss_S29_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/90tiss_S29_75_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/92tiss_S2_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/92tiss_S2_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/92tiss_S2_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/92tiss_S2_76_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/98tiss_S30_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/98tiss_S30_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/98tiss_S30_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/98tiss_S30_77_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/D1_S6_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/D1_S6_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/D1_S6_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/D1_S6_78_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/D4_S26_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/D4_S26_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/D4_S26_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/D4_S26_79_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/H2Ocontrol-run1_S43_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/H2Ocontrol-run1_S43_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/H2Ocontrol-run1_S43_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/H2Ocontrol-run1_S43_80_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/MockE-run1_S42_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i9_zvo3a/MockE-run1_S42_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/MockE-run1_S42_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-gf2ycjw5/2d4ec927-9a1a-4ec6-baf8-223657f6fd01/data/MockE-run1_S42_81_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: Folder_Apprill_USVIFeb2020/trim_Import_files_Apprill_USVIFeb2020I_run1.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/103tiss_S26_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/103tiss_S26_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/103tiss_S26_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/103tiss_S26_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.32 s (20 us/read; 2.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,751\n",
      "  Read 1 with adapter:                 115,356 (99.7%)\n",
      "  Read 2 with adapter:                 115,147 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       115,751 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,882,619 bp\n",
      "  Read 1:    17,477,269 bp\n",
      "  Read 2:    17,405,350 bp\n",
      "Total written (filtered):     30,389,469 bp (87.1%)\n",
      "  Read 1:    15,285,978 bp\n",
      "  Read 2:    15,103,491 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 115356 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t9\t0.0\t1\t5 4\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t767\t0.0\t1\t75 692\n",
      "19\t114204\t0.0\t1\t109578 4626\n",
      "20\t365\t0.0\t1\t64 301\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 115147 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t452.2\t0\t2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t14\t0.0\t1\t4 10\n",
      "19\t1287\t0.0\t1\t154 1133\n",
      "20\t113555\t0.0\t2\t110508 3047\n",
      "21\t285\t0.0\t2\t61 224\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/163eco_S8_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/163eco_S8_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/163eco_S8_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/163eco_S8_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.76 s (21 us/read; 2.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,955\n",
      "  Read 1 with adapter:                  82,623 (99.6%)\n",
      "  Read 2 with adapter:                  82,594 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        82,955 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,974,490 bp\n",
      "  Read 1:    12,523,728 bp\n",
      "  Read 2:    12,450,762 bp\n",
      "Total written (filtered):     21,753,748 bp (87.1%)\n",
      "  Read 1:    10,954,271 bp\n",
      "  Read 2:    10,799,477 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 82623 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.3\t0\t2\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t518\t0.0\t1\t40 478\n",
      "19\t81848\t0.0\t1\t79968 1880\n",
      "20\t239\t0.0\t1\t45 194\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 82594 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t7\t0.0\t1\t4 3\n",
      "19\t748\t0.0\t1\t72 676\n",
      "20\t81631\t0.0\t2\t80478 1153\n",
      "21\t202\t0.0\t2\t44 158\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/164eco_S4_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/164eco_S4_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/164eco_S4_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/164eco_S4_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.48 s (23 us/read; 2.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,855\n",
      "  Read 1 with adapter:                 109,602 (99.8%)\n",
      "  Read 2 with adapter:                 109,375 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,855 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,075,980 bp\n",
      "  Read 1:    16,585,255 bp\n",
      "  Read 2:    16,490,725 bp\n",
      "Total written (filtered):     28,807,431 bp (87.1%)\n",
      "  Read 1:    14,503,280 bp\n",
      "  Read 2:    14,304,151 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109602 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t8\t0.0\t1\t6 2\n",
      "18\t695\t0.0\t1\t59 636\n",
      "19\t108554\t0.0\t1\t106125 2429\n",
      "20\t329\t0.0\t1\t71 258\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 109375 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t9\t0.0\t1\t5 4\n",
      "19\t1119\t0.0\t1\t150 969\n",
      "20\t107924\t0.0\t2\t106223 1701\n",
      "21\t304\t0.0\t2\t72 232\n",
      "23\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/166eco_S9_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/166eco_S9_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/166eco_S9_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/166eco_S9_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.73 s (23 us/read; 2.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,213\n",
      "  Read 1 with adapter:                  74,983 (99.7%)\n",
      "  Read 2 with adapter:                  74,858 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,613,246 bp\n",
      "  Read 1:    11,339,235 bp\n",
      "  Read 2:    11,274,011 bp\n",
      "Total written (filtered):     19,692,530 bp (87.1%)\n",
      "  Read 1:     9,914,945 bp\n",
      "  Read 2:     9,777,585 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 74983 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.3\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t0 3\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t545\t0.0\t1\t38 507\n",
      "19\t74207\t0.0\t1\t72312 1895\n",
      "20\t214\t0.0\t1\t50 164\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 74858 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t9\t0.0\t1\t5 4\n",
      "19\t908\t0.0\t1\t69 839\n",
      "20\t73723\t0.0\t2\t72284 1439\n",
      "21\t214\t0.0\t2\t48 166\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/169eco_S5_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/169eco_S5_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/169eco_S5_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/169eco_S5_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.48 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,515\n",
      "  Read 1 with adapter:                 109,161 (99.7%)\n",
      "  Read 2 with adapter:                 108,942 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,515 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,961,822 bp\n",
      "  Read 1:    16,528,687 bp\n",
      "  Read 2:    16,433,135 bp\n",
      "Total written (filtered):     28,710,405 bp (87.1%)\n",
      "  Read 1:    14,455,062 bp\n",
      "  Read 2:    14,255,343 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109161 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t2\t0.1\t1\t1 1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t668\t0.0\t1\t66 602\n",
      "19\t108160\t0.0\t1\t105803 2357\n",
      "20\t310\t0.0\t1\t67 243\n",
      "31\t1\t0.0\t1\t0 1\n",
      "36\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108942 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1711.2\t0\t3\n",
      "4\t1\t427.8\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t0 4\n",
      "18\t11\t0.0\t1\t6 5\n",
      "19\t1154\t0.0\t1\t155 999\n",
      "20\t107492\t0.0\t2\t105844 1648\n",
      "21\t264\t0.0\t2\t55 209\n",
      "22\t1\t0.0\t2\t1\n",
      "24\t3\t0.0\t2\t1 2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/170eco_S6_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/170eco_S6_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/170eco_S6_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/170eco_S6_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.34 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,631\n",
      "  Read 1 with adapter:                 110,299 (99.7%)\n",
      "  Read 2 with adapter:                 110,139 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       110,631 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,284,409 bp\n",
      "  Read 1:    16,690,141 bp\n",
      "  Read 2:    16,594,268 bp\n",
      "Total written (filtered):     28,987,370 bp (87.1%)\n",
      "  Read 1:    14,594,894 bp\n",
      "  Read 2:    14,392,476 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110299 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t746\t0.0\t1\t71 675\n",
      "19\t109208\t0.0\t1\t106615 2593\n",
      "20\t333\t0.0\t1\t57 276\n",
      "32\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110139 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t432.2\t0\t1\n",
      "9\t2\t0.4\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t8\t0.0\t1\t6 2\n",
      "19\t1171\t0.0\t1\t117 1054\n",
      "20\t108692\t0.0\t2\t106936 1756\n",
      "21\t259\t0.0\t2\t55 204\n",
      "25\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/174eco_S10_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/174eco_S10_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/174eco_S10_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/174eco_S10_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.14 s (21 us/read; 2.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,011\n",
      "  Read 1 with adapter:                 102,770 (99.8%)\n",
      "  Read 2 with adapter:                 102,497 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       103,011 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,010,216 bp\n",
      "  Read 1:    15,550,337 bp\n",
      "  Read 2:    15,459,879 bp\n",
      "Total written (filtered):     27,008,977 bp (87.1%)\n",
      "  Read 1:    13,598,062 bp\n",
      "  Read 2:    13,410,915 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 102770 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t5\t0.0\t1\t4 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t6\t0.0\t1\t1 5\n",
      "18\t612\t0.0\t1\t51 561\n",
      "19\t101851\t0.0\t1\t99774 2077\n",
      "20\t287\t0.0\t1\t47 240\n",
      "21\t1\t0.0\t1\t1\n",
      "36\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 102497 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t402.4\t0\t1\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t2\t0.0\t1\t0 2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t8\t0.0\t1\t2 6\n",
      "19\t1181\t0.0\t1\t133 1048\n",
      "20\t101006\t0.0\t2\t99380 1626\n",
      "21\t289\t0.0\t2\t58 231\n",
      "22\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/177eco_S11_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/177eco_S11_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/177eco_S11_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/177eco_S11_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.48 s (24 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,165\n",
      "  Read 1 with adapter:                 100,935 (99.8%)\n",
      "  Read 2 with adapter:                 100,670 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       101,165 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,451,842 bp\n",
      "  Read 1:    15,269,445 bp\n",
      "  Read 2:    15,182,397 bp\n",
      "Total written (filtered):     26,522,056 bp (87.1%)\n",
      "  Read 1:    13,352,139 bp\n",
      "  Read 2:    13,169,917 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 100935 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t657\t0.0\t1\t49 608\n",
      "19\t100015\t0.0\t1\t97894 2121\n",
      "20\t251\t0.0\t1\t47 204\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 100670 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1580.7\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t16\t0.0\t1\t6 10\n",
      "19\t1090\t0.0\t1\t112 978\n",
      "20\t99297\t0.0\t2\t97734 1563\n",
      "21\t257\t0.0\t2\t50 207\n",
      "32\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/178eco_S12_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/178eco_S12_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/178eco_S12_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/178eco_S12_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.64 s (22 us/read; 2.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,609\n",
      "  Read 1 with adapter:                 117,303 (99.7%)\n",
      "  Read 2 with adapter:                 117,047 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       117,609 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,410,378 bp\n",
      "  Read 1:    17,755,013 bp\n",
      "  Read 2:    17,655,365 bp\n",
      "Total written (filtered):     30,842,384 bp (87.1%)\n",
      "  Read 1:    15,526,768 bp\n",
      "  Read 2:    15,315,616 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 117303 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t778\t0.0\t1\t61 717\n",
      "19\t116198\t0.0\t1\t113623 2575\n",
      "20\t312\t0.0\t1\t50 262\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 117047 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t3\t0.0\t1\t3\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t8\t0.0\t1\t6 2\n",
      "18\t14\t0.0\t1\t6 8\n",
      "19\t1347\t0.0\t1\t125 1222\n",
      "20\t115375\t0.0\t2\t113451 1924\n",
      "21\t292\t0.0\t2\t69 223\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/179eco_S7_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/179eco_S7_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/179eco_S7_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/179eco_S7_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.22 s (20 us/read; 3.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,898\n",
      "  Read 1 with adapter:                 112,629 (99.8%)\n",
      "  Read 2 with adapter:                 112,400 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       112,898 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,957,033 bp\n",
      "  Read 1:    17,029,027 bp\n",
      "  Read 2:    16,928,006 bp\n",
      "Total written (filtered):     29,570,629 bp (87.1%)\n",
      "  Read 1:    14,889,545 bp\n",
      "  Read 2:    14,681,084 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 112629 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t4\t0.4\t0\t4\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t769\t0.0\t1\t42 727\n",
      "19\t111495\t0.0\t1\t108964 2531\n",
      "20\t346\t0.0\t1\t57 289\n",
      "23\t1\t0.0\t1\t1\n",
      "54\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 112400 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t441.0\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t4\t0.0\t1\t2 2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t4\t0.0\t1\t3 1\n",
      "19\t1262\t0.0\t1\t136 1126\n",
      "20\t110831\t0.0\t2\t109025 1806\n",
      "21\t285\t0.0\t2\t59 226\n",
      "31\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/185eco_S13_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/185eco_S13_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/185eco_S13_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/185eco_S13_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.40 s (22 us/read; 2.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,537\n",
      "  Read 1 with adapter:                 107,158 (99.6%)\n",
      "  Read 2 with adapter:                 107,022 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       107,537 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,344,738 bp\n",
      "  Read 1:    16,220,008 bp\n",
      "  Read 2:    16,124,730 bp\n",
      "Total written (filtered):     28,169,891 bp (87.1%)\n",
      "  Read 1:    14,184,533 bp\n",
      "  Read 2:    13,985,358 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 107158 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.6\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t9\t0.0\t1\t5 4\n",
      "18\t718\t0.0\t1\t38 680\n",
      "19\t106125\t0.0\t1\t103633 2492\n",
      "20\t290\t0.0\t1\t49 241\n",
      "21\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 107022 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t12\t0.0\t1\t6 6\n",
      "19\t1270\t0.0\t1\t118 1152\n",
      "20\t105458\t0.0\t2\t103778 1680\n",
      "21\t272\t0.0\t2\t52 220\n",
      "23\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/188eco_S14_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/188eco_S14_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/188eco_S14_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/188eco_S14_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.48 s (24 us/read; 2.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            102,230\n",
      "  Read 1 with adapter:                 101,923 (99.7%)\n",
      "  Read 2 with adapter:                 101,752 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       102,230 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,772,811 bp\n",
      "  Read 1:    15,431,791 bp\n",
      "  Read 2:    15,341,020 bp\n",
      "Total written (filtered):     26,802,521 bp (87.1%)\n",
      "  Read 1:    13,495,610 bp\n",
      "  Read 2:    13,306,911 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101923 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t642\t0.0\t1\t52 590\n",
      "19\t100944\t0.0\t1\t98719 2225\n",
      "20\t321\t0.0\t1\t57 264\n",
      "21\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 101752 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1597.3\t0\t2\n",
      "4\t1\t399.3\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t4\t0.0\t1\t1 3\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "18\t6\t0.0\t1\t3 3\n",
      "19\t1097\t0.0\t1\t138 959\n",
      "20\t100366\t0.0\t2\t98822 1544\n",
      "21\t269\t0.0\t2\t55 214\n",
      "23\t1\t0.0\t2\t0 1\n",
      "28\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/189eco_S15_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/189eco_S15_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/189eco_S15_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/189eco_S15_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.52 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,406\n",
      "  Read 1 with adapter:                 108,101 (99.7%)\n",
      "  Read 2 with adapter:                 107,888 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       108,406 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,633,591 bp\n",
      "  Read 1:    16,365,443 bp\n",
      "  Read 2:    16,268,148 bp\n",
      "Total written (filtered):     28,423,504 bp (87.1%)\n",
      "  Read 1:    14,311,970 bp\n",
      "  Read 2:    14,111,534 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108101 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t711\t0.0\t1\t64 647\n",
      "19\t107056\t0.0\t1\t104717 2339\n",
      "20\t319\t0.0\t1\t63 256\n",
      "26\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 107888 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t423.5\t0\t2\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t5\t0.0\t1\t1 4\n",
      "16\t6\t0.0\t1\t4 2\n",
      "18\t12\t0.0\t1\t5 7\n",
      "19\t1292\t0.0\t1\t125 1167\n",
      "20\t106283\t0.0\t2\t104548 1735\n",
      "21\t284\t0.0\t2\t68 216\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/194eco_S16_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/194eco_S16_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/194eco_S16_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/194eco_S16_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.60 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,806\n",
      "  Read 1 with adapter:                 114,431 (99.7%)\n",
      "  Read 2 with adapter:                 114,286 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,806 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,542,447 bp\n",
      "  Read 1:    17,322,461 bp\n",
      "  Read 2:    17,219,986 bp\n",
      "Total written (filtered):     30,084,211 bp (87.1%)\n",
      "  Read 1:    15,148,791 bp\n",
      "  Read 2:    14,935,420 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 114431 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t3\t0.0\t1\t1 2\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t853\t0.0\t1\t69 784\n",
      "19\t113220\t0.0\t1\t110424 2796\n",
      "20\t341\t0.0\t1\t73 268\n",
      "21\t1\t0.0\t1\t0 1\n",
      "91\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 114286 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t6\t0.0\t1\t5 1\n",
      "19\t1387\t0.0\t1\t149 1238\n",
      "20\t112598\t0.0\t2\t110609 1989\n",
      "21\t287\t0.0\t2\t46 241\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/103tiss_S26_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/103tiss_S26_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/103tiss_S26_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/103tiss_S26_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/163eco_S8_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/163eco_S8_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/163eco_S8_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/163eco_S8_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/164eco_S4_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/164eco_S4_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/164eco_S4_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/164eco_S4_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/166eco_S9_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/166eco_S9_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/166eco_S9_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/166eco_S9_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/169eco_S5_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/169eco_S5_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/169eco_S5_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/169eco_S5_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/170eco_S6_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/170eco_S6_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/170eco_S6_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/170eco_S6_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/174eco_S10_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/174eco_S10_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/174eco_S10_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/174eco_S10_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/177eco_S11_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/177eco_S11_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/177eco_S11_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/177eco_S11_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/178eco_S12_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/178eco_S12_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/178eco_S12_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/178eco_S12_50_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/179eco_S7_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/179eco_S7_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/179eco_S7_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/179eco_S7_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/185eco_S13_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/185eco_S13_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/185eco_S13_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/185eco_S13_52_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/188eco_S14_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/188eco_S14_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/188eco_S14_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/188eco_S14_53_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/189eco_S15_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/189eco_S15_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/189eco_S15_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/189eco_S15_54_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/194eco_S16_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/194eco_S16_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/194eco_S16_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/194eco_S16_55_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/196eco_S17_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/196eco_S17_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/196eco_S17_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/196eco_S17_56_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/196eco_S17_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/196eco_S17_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/196eco_S17_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/196eco_S17_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.45 s (20 us/read; 3.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,318\n",
      "  Read 1 with adapter:                 125,001 (99.7%)\n",
      "  Read 2 with adapter:                 124,771 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       125,318 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,668,473 bp\n",
      "  Read 1:    18,890,611 bp\n",
      "  Read 2:    18,777,862 bp\n",
      "Total written (filtered):     32,799,825 bp (87.1%)\n",
      "  Read 1:    16,516,190 bp\n",
      "  Read 2:    16,283,635 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 125001 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t900\t0.0\t1\t55 845\n",
      "19\t123710\t0.0\t1\t120862 2848\n",
      "20\t373\t0.0\t1\t72 301\n",
      "21\t2\t0.0\t1\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 124771 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1958.1\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t4\t0.0\t1\t2 2\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t10\t0.0\t1\t8 2\n",
      "19\t1422\t0.0\t1\t149 1273\n",
      "20\t122983\t0.0\t2\t120829 2154\n",
      "21\t336\t0.0\t2\t75 261\n",
      "28\t1\t0.0\t2\t1\n",
      "38\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/199eco_S18_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/199eco_S18_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/199eco_S18_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/199eco_S18_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.85 s (23 us/read; 2.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,326\n",
      "  Read 1 with adapter:                 125,006 (99.7%)\n",
      "  Read 2 with adapter:                 124,778 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       125,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,720,134 bp\n",
      "  Read 1:    18,916,683 bp\n",
      "  Read 2:    18,803,451 bp\n",
      "Total written (filtered):     32,851,291 bp (87.1%)\n",
      "  Read 1:    16,542,193 bp\n",
      "  Read 2:    16,309,098 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 125006 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t8\t0.0\t1\t6 2\n",
      "17\t8\t0.0\t1\t6 2\n",
      "18\t911\t0.0\t1\t77 834\n",
      "19\t123707\t0.0\t1\t120809 2898\n",
      "20\t362\t0.0\t1\t82 280\n",
      "27\t1\t0.0\t1\t1\n",
      "35\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 124778 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t489.6\t0\t2\n",
      "10\t3\t0.1\t1\t3\n",
      "13\t1\t0.0\t1\t0 1\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t7\t0.0\t1\t3 4\n",
      "19\t1425\t0.0\t1\t153 1272\n",
      "20\t123008\t0.0\t2\t120917 2091\n",
      "21\t323\t0.0\t2\t65 258\n",
      "22\t2\t0.0\t2\t1 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/200eco_S19_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/200eco_S19_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/200eco_S19_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/200eco_S19_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.87 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            127,863\n",
      "  Read 1 with adapter:                 127,565 (99.8%)\n",
      "  Read 2 with adapter:                 127,346 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       127,863 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,440,785 bp\n",
      "  Read 1:    19,278,672 bp\n",
      "  Read 2:    19,162,113 bp\n",
      "Total written (filtered):     33,471,813 bp (87.1%)\n",
      "  Read 1:    16,855,377 bp\n",
      "  Read 2:    16,616,436 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 127565 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t779\t0.0\t1\t63 716\n",
      "19\t126409\t0.0\t1\t123611 2798\n",
      "20\t361\t0.0\t1\t59 302\n",
      "21\t4\t0.0\t1\t1 3\n",
      "35\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 127346 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t15\t0.0\t1\t8 7\n",
      "19\t1458\t0.0\t1\t140 1318\n",
      "20\t125561\t0.0\t2\t123524 2037\n",
      "21\t301\t0.0\t2\t75 226\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/212tiss_S27_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/212tiss_S27_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/212tiss_S27_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/212tiss_S27_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.88 s (21 us/read; 2.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            137,898\n",
      "  Read 1 with adapter:                 137,545 (99.7%)\n",
      "  Read 2 with adapter:                 137,319 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       137,898 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,414,149 bp\n",
      "  Read 1:    20,768,838 bp\n",
      "  Read 2:    20,645,311 bp\n",
      "Total written (filtered):     36,055,827 bp (87.1%)\n",
      "  Read 1:    18,155,976 bp\n",
      "  Read 2:    17,899,851 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 137545 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t4\t0.0\t1\t4\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t7\t0.0\t1\t6 1\n",
      "18\t794\t0.0\t1\t91 703\n",
      "19\t136297\t0.0\t1\t132397 3900\n",
      "20\t425\t0.0\t1\t72 353\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 137319 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2154.7\t0\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t9\t0.0\t1\t5 4\n",
      "19\t1172\t0.0\t1\t163 1009\n",
      "20\t135775\t0.0\t2\t133141 2634\n",
      "21\t350\t0.0\t2\t87 263\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/214tiss_S28_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/214tiss_S28_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/214tiss_S28_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/214tiss_S28_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.69 s (21 us/read; 2.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            128,867\n",
      "  Read 1 with adapter:                 128,539 (99.7%)\n",
      "  Read 2 with adapter:                 128,360 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       128,867 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,806,080 bp\n",
      "  Read 1:    19,456,955 bp\n",
      "  Read 2:    19,349,125 bp\n",
      "Total written (filtered):     33,797,596 bp (87.1%)\n",
      "  Read 1:    17,015,064 bp\n",
      "  Read 2:    16,782,532 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 128539 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t593\t0.0\t1\t57 536\n",
      "19\t127619\t0.0\t1\t123927 3692\n",
      "20\t312\t0.0\t1\t85 227\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 128360 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t5\t0.0\t1\t4 1\n",
      "19\t877\t0.0\t1\t132 745\n",
      "20\t127128\t0.0\t2\t124593 2535\n",
      "21\t336\t0.0\t2\t84 252\n",
      "22\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/215tiss_S29_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/215tiss_S29_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/215tiss_S29_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/215tiss_S29_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.54 s (22 us/read; 2.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,611\n",
      "  Read 1 with adapter:                 117,250 (99.7%)\n",
      "  Read 2 with adapter:                 116,978 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       117,611 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,400,216 bp\n",
      "  Read 1:    17,747,456 bp\n",
      "  Read 2:    17,652,760 bp\n",
      "Total written (filtered):     30,834,113 bp (87.1%)\n",
      "  Read 1:    15,520,055 bp\n",
      "  Read 2:    15,314,058 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 117250 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "16\t7\t0.0\t1\t3 4\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t648\t0.0\t1\t62 586\n",
      "19\t116235\t0.0\t1\t112615 3620\n",
      "20\t348\t0.0\t1\t67 281\n",
      "21\t1\t0.0\t1\t0 1\n",
      "31\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 116978 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t459.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t5\t0.0\t1\t3 2\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t7\t0.0\t1\t4 3\n",
      "19\t1075\t0.0\t1\t172 903\n",
      "20\t115569\t0.0\t2\t113295 2274\n",
      "21\t314\t0.0\t2\t80 234\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/217tiss_S30_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/217tiss_S30_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/217tiss_S30_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/217tiss_S30_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.60 s (20 us/read; 2.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            127,010\n",
      "  Read 1 with adapter:                 126,562 (99.6%)\n",
      "  Read 2 with adapter:                 126,374 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       127,010 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,175,079 bp\n",
      "  Read 1:    19,135,129 bp\n",
      "  Read 2:    19,039,950 bp\n",
      "Total written (filtered):     33,244,465 bp (87.1%)\n",
      "  Read 1:    16,730,978 bp\n",
      "  Read 2:    16,513,487 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 126562 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.9\t0\t1\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t3\t0.1\t1\t2 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t6\t0.0\t1\t4 2\n",
      "13\t3\t0.0\t1\t3\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t6\t0.0\t1\t2 4\n",
      "17\t8\t0.0\t1\t6 2\n",
      "18\t833\t0.0\t1\t88 745\n",
      "19\t125303\t0.0\t1\t121272 4031\n",
      "20\t392\t0.0\t1\t81 311\n",
      "29\t1\t0.0\t1\t1\n",
      "50\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 126374 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t4\t0.0\t1\t0 4\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t17\t0.0\t1\t11 6\n",
      "19\t1236\t0.0\t1\t144 1092\n",
      "20\t124776\t0.0\t2\t122127 2649\n",
      "21\t330\t0.0\t2\t63 267\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/218tiss_S31_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/218tiss_S31_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/218tiss_S31_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/218tiss_S31_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.38 s (24 us/read; 2.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             98,385\n",
      "  Read 1 with adapter:                  97,685 (99.3%)\n",
      "  Read 2 with adapter:                  97,903 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        98,385 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,602,700 bp\n",
      "  Read 1:    14,843,979 bp\n",
      "  Read 2:    14,758,721 bp\n",
      "Total written (filtered):     25,789,780 bp (87.1%)\n",
      "  Read 1:    12,988,328 bp\n",
      "  Read 2:    12,801,452 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 97685 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t604\t0.0\t1\t63 541\n",
      "19\t96763\t0.0\t1\t93515 3248\n",
      "20\t303\t0.0\t1\t53 250\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 97903 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t384.3\t0\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t6\t0.0\t1\t5 1\n",
      "18\t9\t0.0\t1\t3 6\n",
      "19\t910\t0.0\t1\t118 792\n",
      "20\t96735\t0.0\t2\t94986 1749\n",
      "21\t233\t0.0\t2\t46 187\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/224tiss_S32_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/224tiss_S32_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/224tiss_S32_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/224tiss_S32_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.38 s (23 us/read; 2.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,999\n",
      "  Read 1 with adapter:                 104,759 (99.8%)\n",
      "  Read 2 with adapter:                 104,565 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       104,999 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,563,958 bp\n",
      "  Read 1:    15,826,467 bp\n",
      "  Read 2:    15,737,491 bp\n",
      "Total written (filtered):     27,483,429 bp (87.1%)\n",
      "  Read 1:    13,836,463 bp\n",
      "  Read 2:    13,646,966 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 104759 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t615\t0.0\t1\t52 563\n",
      "19\t103850\t0.0\t1\t101474 2376\n",
      "20\t277\t0.0\t1\t49 228\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 104565 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t410.2\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t6\t0.0\t1\t3 3\n",
      "19\t982\t0.0\t1\t124 858\n",
      "20\t103302\t0.0\t2\t101589 1713\n",
      "21\t264\t0.0\t2\t54 210\n",
      "45\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/227tiss_S33_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/227tiss_S33_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/227tiss_S33_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/227tiss_S33_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.55 s (21 us/read; 2.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            123,006\n",
      "  Read 1 with adapter:                 122,686 (99.7%)\n",
      "  Read 2 with adapter:                 122,444 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       123,006 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,991,176 bp\n",
      "  Read 1:    18,548,670 bp\n",
      "  Read 2:    18,442,506 bp\n",
      "Total written (filtered):     32,212,749 bp (87.1%)\n",
      "  Read 1:    16,218,099 bp\n",
      "  Read 2:    15,994,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 122686 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t9\t0.0\t1\t8 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t728\t0.0\t1\t56 672\n",
      "19\t121581\t0.0\t1\t118537 3044\n",
      "20\t358\t0.0\t1\t66 292\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 122444 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t480.5\t0\t1\n",
      "8\t1\t1.9\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t6\t0.0\t1\t5 1\n",
      "18\t14\t0.0\t1\t9 5\n",
      "19\t1215\t0.0\t1\t145 1070\n",
      "20\t120858\t0.0\t2\t118872 1986\n",
      "21\t336\t0.0\t2\t76 260\n",
      "22\t1\t0.0\t2\t0 1\n",
      "24\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/228tiss_S34_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/228tiss_S34_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/228tiss_S34_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/228tiss_S34_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.46 s (21 us/read; 2.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,936\n",
      "  Read 1 with adapter:                 117,688 (99.8%)\n",
      "  Read 2 with adapter:                 117,434 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       117,936 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,495,973 bp\n",
      "  Read 1:    17,799,798 bp\n",
      "  Read 2:    17,696,175 bp\n",
      "Total written (filtered):     30,912,468 bp (87.1%)\n",
      "  Read 1:    15,564,155 bp\n",
      "  Read 2:    15,348,313 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 117688 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t658\t0.0\t1\t65 593\n",
      "19\t116714\t0.0\t1\t114001 2713\n",
      "20\t299\t0.0\t1\t64 235\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 117434 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1842.8\t0\t1\n",
      "4\t2\t460.7\t0\t2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t3\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t10\t0.0\t1\t8 2\n",
      "19\t994\t0.0\t1\t121 873\n",
      "20\t116158\t0.0\t2\t114151 2007\n",
      "21\t261\t0.0\t2\t65 196\n",
      "44\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/229tiss_S35_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/229tiss_S35_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/229tiss_S35_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/229tiss_S35_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.27 s (22 us/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,765\n",
      "  Read 1 with adapter:                 104,456 (99.7%)\n",
      "  Read 2 with adapter:                 104,255 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       104,765 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,505,706 bp\n",
      "  Read 1:    15,800,735 bp\n",
      "  Read 2:    15,704,971 bp\n",
      "Total written (filtered):     27,437,396 bp (87.1%)\n",
      "  Read 1:    13,816,509 bp\n",
      "  Read 2:    13,620,887 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 104456 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "16\t8\t0.0\t1\t2 6\n",
      "17\t5\t0.0\t1\t0 5\n",
      "18\t702\t0.0\t1\t68 634\n",
      "19\t103420\t0.0\t1\t100619 2801\n",
      "20\t318\t0.0\t1\t63 255\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 104255 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t409.2\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t2 2\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t15\t0.0\t1\t11 4\n",
      "19\t1170\t0.0\t1\t126 1044\n",
      "20\t102810\t0.0\t2\t100997 1813\n",
      "21\t249\t0.0\t2\t31 218\n",
      "22\t1\t0.0\t2\t1\n",
      "23\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/234tiss_S36_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/234tiss_S36_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/234tiss_S36_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/234tiss_S36_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.21 s (17 us/read; 3.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,689\n",
      "  Read 1 with adapter:                 129,372 (99.8%)\n",
      "  Read 2 with adapter:                 129,069 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       129,689 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,040,452 bp\n",
      "  Read 1:    19,577,214 bp\n",
      "  Read 2:    19,463,238 bp\n",
      "Total written (filtered):     34,002,171 bp (87.1%)\n",
      "  Read 1:    17,119,483 bp\n",
      "  Read 2:    16,882,688 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 129372 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "10\t3\t0.1\t1\t1 2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t7\t0.0\t1\t2 5\n",
      "17\t10\t0.0\t1\t6 4\n",
      "18\t642\t0.0\t1\t84 558\n",
      "19\t128330\t0.0\t1\t124777 3553\n",
      "20\t370\t0.0\t1\t72 298\n",
      "21\t2\t0.0\t1\t0 2\n",
      "33\t1\t0.0\t1\t1\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 129069 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2026.4\t0\t1\n",
      "4\t1\t506.6\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t5\n",
      "18\t9\t0.0\t1\t2 7\n",
      "19\t983\t0.0\t1\t148 835\n",
      "20\t127782\t0.0\t2\t125513 2269\n",
      "21\t279\t0.0\t2\t71 208\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/250tiss_S38_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/250tiss_S38_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/250tiss_S38_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/250tiss_S38_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.68 s (22 us/read; 2.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            119,744\n",
      "  Read 1 with adapter:                 119,380 (99.7%)\n",
      "  Read 2 with adapter:                 119,213 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       119,744 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,025,141 bp\n",
      "  Read 1:    18,067,046 bp\n",
      "  Read 2:    17,958,095 bp\n",
      "Total written (filtered):     31,373,765 bp (87.1%)\n",
      "  Read 1:    15,799,201 bp\n",
      "  Read 2:    15,574,564 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 119380 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t3\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t5\t0.0\t1\t1 4\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t595\t0.0\t1\t53 542\n",
      "19\t118458\t0.0\t1\t115132 3326\n",
      "20\t305\t0.0\t1\t55 250\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 119213 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t467.8\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t16\t0.0\t1\t6 10\n",
      "19\t901\t0.0\t1\t140 761\n",
      "20\t118002\t0.0\t2\t116064 1938\n",
      "21\t279\t0.0\t2\t59 220\n",
      "22\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/251tiss_S39_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/251tiss_S39_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/251tiss_S39_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/251tiss_S39_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.78 s (21 us/read; 2.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,216\n",
      "  Read 1 with adapter:                 129,900 (99.8%)\n",
      "  Read 2 with adapter:                 129,703 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       130,216 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,203,573 bp\n",
      "  Read 1:    19,656,773 bp\n",
      "  Read 2:    19,546,800 bp\n",
      "Total written (filtered):     34,142,599 bp (87.1%)\n",
      "  Read 1:    17,189,011 bp\n",
      "  Read 2:    16,953,588 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 129900 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t8\t0.0\t1\t8\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t595\t0.0\t1\t50 545\n",
      "19\t128959\t0.0\t1\t125230 3729\n",
      "20\t324\t0.0\t1\t49 275\n",
      "21\t1\t0.0\t1\t0 1\n",
      "30\t1\t0.0\t1\t1\n",
      "44\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 129703 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t508.7\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t4\t0.0\t1\t0 4\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t13\t0.0\t1\t8 5\n",
      "19\t1038\t0.0\t1\t149 889\n",
      "20\t128321\t0.0\t2\t125754 2567\n",
      "21\t313\t0.0\t2\t65 248\n",
      "24\t1\t0.0\t2\t1\n",
      "36\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/264tiss_S40_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/264tiss_S40_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/264tiss_S40_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/264tiss_S40_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.08 s (18 us/read; 3.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            116,626\n",
      "  Read 1 with adapter:                 116,318 (99.7%)\n",
      "  Read 2 with adapter:                 116,064 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       116,626 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,110,431 bp\n",
      "  Read 1:    17,609,513 bp\n",
      "  Read 2:    17,500,918 bp\n",
      "Total written (filtered):     30,580,718 bp (87.1%)\n",
      "  Read 1:    15,399,954 bp\n",
      "  Read 2:    15,180,764 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 116318 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t2\t0.0\t1\t0 2\n",
      "16\t8\t0.0\t1\t2 6\n",
      "17\t8\t0.0\t1\t5 3\n",
      "18\t750\t0.0\t1\t61 689\n",
      "19\t115177\t0.0\t1\t112300 2877\n",
      "20\t359\t0.0\t1\t75 284\n",
      "24\t1\t0.0\t1\t0 1\n",
      "26\t1\t0.0\t1\t1\n",
      "34\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 116064 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t19\t0.0\t1\t12 7\n",
      "19\t1303\t0.0\t1\t152 1151\n",
      "20\t114436\t0.0\t2\t112489 1947\n",
      "21\t293\t0.0\t2\t55 238\n",
      "24\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/199eco_S18_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/199eco_S18_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/199eco_S18_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/199eco_S18_57_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/200eco_S19_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/200eco_S19_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/200eco_S19_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/200eco_S19_58_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/212tiss_S27_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/212tiss_S27_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/212tiss_S27_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/212tiss_S27_59_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/214tiss_S28_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/214tiss_S28_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/214tiss_S28_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/214tiss_S28_60_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/215tiss_S29_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/215tiss_S29_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/215tiss_S29_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/215tiss_S29_61_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/217tiss_S30_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/217tiss_S30_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/217tiss_S30_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/217tiss_S30_62_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/218tiss_S31_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/218tiss_S31_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/218tiss_S31_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/218tiss_S31_63_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/224tiss_S32_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/224tiss_S32_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/224tiss_S32_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/224tiss_S32_64_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/227tiss_S33_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/227tiss_S33_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/227tiss_S33_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/227tiss_S33_65_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/228tiss_S34_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/228tiss_S34_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/228tiss_S34_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/228tiss_S34_66_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/229tiss_S35_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/229tiss_S35_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/229tiss_S35_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/229tiss_S35_67_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/234tiss_S36_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/234tiss_S36_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/234tiss_S36_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/234tiss_S36_68_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/250tiss_S38_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/250tiss_S38_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/250tiss_S38_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/250tiss_S38_69_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/251tiss_S39_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/251tiss_S39_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/251tiss_S39_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/251tiss_S39_70_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/264tiss_S40_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/264tiss_S40_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/264tiss_S40_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/264tiss_S40_71_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command:This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/268eco_S20_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/268eco_S20_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/268eco_S20_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/268eco_S20_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.56 s (20 us/read; 2.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,436\n",
      "  Read 1 with adapter:                 125,078 (99.7%)\n",
      "  Read 2 with adapter:                 124,834 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       125,436 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,718,008 bp\n",
      "  Read 1:    18,912,488 bp\n",
      "  Read 2:    18,805,520 bp\n",
      "Total written (filtered):     32,846,452 bp (87.1%)\n",
      "  Read 1:    16,536,502 bp\n",
      "  Read 2:    16,309,950 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 125078 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.9\t0\t1\n",
      "10\t2\t0.1\t1\t1 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t793\t0.0\t1\t75 718\n",
      "19\t123876\t0.0\t1\t121189 2687\n",
      "20\t389\t0.0\t1\t87 302\n",
      "21\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 124834 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t490.0\t0\t1\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t14\t0.0\t1\t11 3\n",
      "19\t1341\t0.0\t1\t144 1197\n",
      "20\t123150\t0.0\t2\t121080 2070\n",
      "21\t319\t0.0\t2\t61 258\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/272eco_S22_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/272eco_S22_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/272eco_S22_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/272eco_S22_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.36 s (21 us/read; 2.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,530\n",
      "  Read 1 with adapter:                 110,276 (99.8%)\n",
      "  Read 2 with adapter:                 110,063 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       110,530 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,210,449 bp\n",
      "  Read 1:    16,653,079 bp\n",
      "  Read 2:    16,557,370 bp\n",
      "Total written (filtered):     28,915,245 bp (87.1%)\n",
      "  Read 1:    14,558,244 bp\n",
      "  Read 2:    14,357,001 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110276 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t3\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t13\t0.0\t1\t7 6\n",
      "18\t695\t0.0\t1\t61 634\n",
      "19\t109203\t0.0\t1\t106849 2354\n",
      "20\t355\t0.0\t1\t78 277\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110063 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t431.8\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t14\t0.0\t1\t6 8\n",
      "19\t1087\t0.0\t1\t113 974\n",
      "20\t108662\t0.0\t2\t107018 1644\n",
      "21\t290\t0.0\t2\t61 229\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/76Ltiss_S2_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/76Ltiss_S2_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/76Ltiss_S2_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/76Ltiss_S2_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.70 s (21 us/read; 2.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            126,961\n",
      "  Read 1 with adapter:                 126,625 (99.7%)\n",
      "  Read 2 with adapter:                 126,417 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       126,961 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,154,269 bp\n",
      "  Read 1:    19,137,008 bp\n",
      "  Read 2:    19,017,261 bp\n",
      "Total written (filtered):     33,221,653 bp (87.1%)\n",
      "  Read 1:    16,731,631 bp\n",
      "  Read 2:    16,490,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 126625 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t4\t0.5\t0\t4\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t6\t0.0\t1\t0 6\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t7\t0.0\t1\t2 5\n",
      "18\t814\t0.0\t1\t74 740\n",
      "19\t125353\t0.0\t1\t121837 3516\n",
      "20\t432\t0.0\t1\t74 358\n",
      "30\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 126417 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1983.8\t0\t2\n",
      "4\t1\t495.9\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t23\t0.0\t1\t13 10\n",
      "19\t1255\t0.0\t1\t166 1089\n",
      "20\t124791\t0.0\t2\t122512 2279\n",
      "21\t327\t0.0\t2\t86 241\n",
      "22\t1\t0.0\t2\t0 1\n",
      "23\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/78Ltiss_S41_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/78Ltiss_S41_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/78Ltiss_S41_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/78Ltiss_S41_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.45 s (21 us/read; 2.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            116,545\n",
      "  Read 1 with adapter:                 116,254 (99.8%)\n",
      "  Read 2 with adapter:                 116,056 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       116,545 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,066,041 bp\n",
      "  Read 1:    17,589,235 bp\n",
      "  Read 2:    17,476,806 bp\n",
      "Total written (filtered):     30,537,102 bp (87.1%)\n",
      "  Read 1:    15,380,669 bp\n",
      "  Read 2:    15,156,433 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 116254 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t550\t0.0\t1\t57 493\n",
      "19\t115308\t0.0\t1\t112620 2688\n",
      "20\t375\t0.0\t1\t75 300\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 116056 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1821.0\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t2 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t15\t0.0\t1\t6 9\n",
      "19\t968\t0.0\t1\t158 810\n",
      "20\t114756\t0.0\t2\t112969 1787\n",
      "21\t301\t0.0\t2\t70 231\n",
      "22\t1\t0.0\t2\t1\n",
      "32\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/79Ltiss_S42_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/79Ltiss_S42_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/79Ltiss_S42_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/79Ltiss_S42_76_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.42 s (21 us/read; 2.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,688\n",
      "  Read 1 with adapter:                 113,430 (99.8%)\n",
      "  Read 2 with adapter:                 113,228 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,688 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,230,354 bp\n",
      "  Read 1:    17,166,646 bp\n",
      "  Read 2:    17,063,708 bp\n",
      "Total written (filtered):     29,811,649 bp (87.1%)\n",
      "  Read 1:    15,011,880 bp\n",
      "  Read 2:    14,799,769 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113430 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t8\t0.0\t1\t7 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t5\t0.0\t1\t3 2\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t579\t0.0\t1\t56 523\n",
      "19\t112566\t0.0\t1\t109449 3117\n",
      "20\t265\t0.0\t1\t64 201\n",
      "26\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 113228 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1776.4\t0\t1\n",
      "4\t1\t444.1\t0\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t821\t0.0\t1\t133 688\n",
      "20\t112138\t0.0\t2\t109776 2362\n",
      "21\t253\t0.0\t2\t61 192\n",
      "22\t1\t0.0\t2\t0 1\n",
      "28\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/85tiss_S24_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/85tiss_S24_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/85tiss_S24_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/85tiss_S24_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.67 s (26 us/read; 2.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,030\n",
      "  Read 1 with adapter:                  64,823 (99.7%)\n",
      "  Read 2 with adapter:                  64,684 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        65,030 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,584,939 bp\n",
      "  Read 1:     9,814,887 bp\n",
      "  Read 2:     9,770,052 bp\n",
      "Total written (filtered):     17,060,543 bp (87.1%)\n",
      "  Read 1:     8,583,546 bp\n",
      "  Read 2:     8,476,997 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 64823 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t450\t0.0\t1\t31 419\n",
      "19\t64157\t0.0\t1\t62027 2130\n",
      "20\t205\t0.0\t1\t40 165\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 64684 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t784\t0.0\t1\t78 706\n",
      "20\t63686\t0.0\t2\t62277 1409\n",
      "21\t199\t0.0\t2\t35 164\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/91tiss_S25_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/91tiss_S25_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/91tiss_S25_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/91tiss_S25_78_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 3.00 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            134,078\n",
      "  Read 1 with adapter:                 133,607 (99.6%)\n",
      "  Read 2 with adapter:                 133,488 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       134,078 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,362,881 bp\n",
      "  Read 1:    20,237,914 bp\n",
      "  Read 2:    20,124,967 bp\n",
      "Total written (filtered):     35,155,928 bp (87.1%)\n",
      "  Read 1:    17,699,812 bp\n",
      "  Read 2:    17,456,116 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 133607 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.5\t0\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t6\t0.0\t1\t0 6\n",
      "17\t7\t0.0\t1\t1 6\n",
      "18\t724\t0.0\t1\t67 657\n",
      "19\t132491\t0.0\t1\t128144 4347\n",
      "20\t371\t0.0\t1\t66 305\n",
      "21\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 133488 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t6\t0.0\t1\t1 5\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t14\t0.0\t1\t7 7\n",
      "19\t1123\t0.0\t1\t167 956\n",
      "20\t132032\t0.0\t2\t129290 2742\n",
      "21\t301\t0.0\t2\t75 226\n",
      "45\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/97tiss_S1_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/97tiss_S1_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/97tiss_S1_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/97tiss_S1_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.35 s (19 us/read; 3.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,469\n",
      "  Read 1 with adapter:                 125,130 (99.7%)\n",
      "  Read 2 with adapter:                 124,964 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       125,469 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,777,340 bp\n",
      "  Read 1:    18,944,220 bp\n",
      "  Read 2:    18,833,120 bp\n",
      "Total written (filtered):     32,901,646 bp (87.1%)\n",
      "  Read 1:    16,567,134 bp\n",
      "  Read 2:    16,334,512 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 125130 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t8\t0.0\t1\t6 2\n",
      "18\t628\t0.0\t1\t58 570\n",
      "19\t124150\t0.0\t1\t121244 2906\n",
      "20\t331\t0.0\t1\t63 268\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 124964 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t8\t0.0\t1\t3 5\n",
      "19\t968\t0.0\t1\t142 826\n",
      "20\t123634\t0.0\t2\t121564 2070\n",
      "21\t346\t0.0\t2\t74 272\n",
      "26\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/D2_S3_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/D2_S3_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/D2_S3_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/D2_S3_80_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.78 s (50 us/read; 1.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,787\n",
      "  Read 1 with adapter:                  15,740 (99.7%)\n",
      "  Read 2 with adapter:                  15,705 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        15,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,720,812 bp\n",
      "  Read 1:     2,367,239 bp\n",
      "  Read 2:     2,353,573 bp\n",
      "Total written (filtered):      4,107,907 bp (87.0%)\n",
      "  Read 1:     2,068,272 bp\n",
      "  Read 2:     2,039,635 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 15740 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.1\t0\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t113\t0.0\t1\t7 106\n",
      "19\t15573\t0.0\t1\t15034 539\n",
      "20\t49\t0.0\t1\t8 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 15705 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t205\t0.0\t1\t22 183\n",
      "20\t15454\t0.0\t2\t15046 408\n",
      "21\t45\t0.0\t2\t8 37\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/D5_S23_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/D5_S23_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/D5_S23_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/D5_S23_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.27 s (26 us/read; 2.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,688\n",
      "  Read 1 with adapter:                  49,553 (99.7%)\n",
      "  Read 2 with adapter:                  49,367 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,688 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,362,640 bp\n",
      "  Read 1:     6,205,862 bp\n",
      "  Read 2:     6,156,778 bp\n",
      "Total written (filtered):     10,434,579 bp (84.4%)\n",
      "  Read 1:     5,264,590 bp\n",
      "  Read 2:     5,169,989 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 49553 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t355\t0.0\t1\t16 339\n",
      "19\t49054\t0.0\t1\t47909 1145\n",
      "20\t139\t0.0\t1\t22 117\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 49367 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t194.1\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t4\t0.0\t1\t3 1\n",
      "19\t635\t0.0\t1\t60 575\n",
      "20\t48578\t0.0\t2\t47546 1032\n",
      "21\t140\t0.0\t2\t31 109\n",
      "22\t1\t0.0\t2\t1\n",
      "23\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/H2Ocontrol_S44_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/H2Ocontrol_S44_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/H2Ocontrol_S44_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/H2Ocontrol_S44_82_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.89 s (40 us/read; 1.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,086\n",
      "  Read 1 with adapter:                  21,985 (99.5%)\n",
      "  Read 2 with adapter:                  21,888 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,086 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,574,962 bp\n",
      "  Read 1:     2,294,525 bp\n",
      "  Read 2:     2,280,437 bp\n",
      "Total written (filtered):      3,719,868 bp (81.3%)\n",
      "  Read 1:     1,876,928 bp\n",
      "  Read 2:     1,842,940 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 21985 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t183\t0.0\t1\t13 170\n",
      "19\t21726\t0.0\t1\t21237 489\n",
      "20\t72\t0.0\t1\t12 60\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 21888 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t86.3\t0\t1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t1\t0.0\t1\t0 1\n",
      "19\t300\t0.0\t1\t18 282\n",
      "20\t21521\t0.0\t2\t21060 461\n",
      "21\t62\t0.0\t2\t11 51\n",
      "22\t1\t0.0\t2\t0 1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/MockE_S43_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/MockE_S43_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/MockE_S43_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/MockE_S43_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.15 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,099\n",
      "  Read 1 with adapter:                  95,591 (99.5%)\n",
      "  Read 2 with adapter:                  95,683 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        96,099 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,911,566 bp\n",
      "  Read 1:    14,502,925 bp\n",
      "  Read 2:    14,408,641 bp\n",
      "Total written (filtered):     25,182,672 bp (87.1%)\n",
      "  Read 1:    12,686,956 bp\n",
      "  Read 2:    12,495,716 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 95591 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t2 1\n",
      "14\t3\t0.0\t1\t2 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t11\t0.0\t1\t6 5\n",
      "18\t491\t0.0\t1\t48 443\n",
      "19\t94812\t0.0\t1\t92488 2324\n",
      "20\t264\t0.0\t1\t52 212\n",
      "21\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t0 1\n",
      "40\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 95683 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1501.5\t0\t1\n",
      "4\t1\t375.4\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t9\t0.0\t1\t7 2\n",
      "18\t9\t0.0\t1\t6 3\n",
      "19\t892\t0.0\t1\t104 788\n",
      "20\t94505\t0.0\t2\t93280 1225\n",
      "21\t261\t0.0\t2\t58 203\n",
      " cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/268eco_S20_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/268eco_S20_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/268eco_S20_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/268eco_S20_72_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/272eco_S22_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/272eco_S22_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/272eco_S22_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/272eco_S22_73_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/76Ltiss_S2_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/76Ltiss_S2_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/76Ltiss_S2_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/76Ltiss_S2_74_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/78Ltiss_S41_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/78Ltiss_S41_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/78Ltiss_S41_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/78Ltiss_S41_75_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/79Ltiss_S42_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/79Ltiss_S42_76_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/79Ltiss_S42_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/79Ltiss_S42_76_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/85tiss_S24_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/85tiss_S24_77_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/85tiss_S24_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/85tiss_S24_77_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/91tiss_S25_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/91tiss_S25_78_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/91tiss_S25_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/91tiss_S25_78_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/97tiss_S1_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/97tiss_S1_79_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/97tiss_S1_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/97tiss_S1_79_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/D2_S3_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/D2_S3_80_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/D2_S3_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/D2_S3_80_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/D5_S23_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/D5_S23_81_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/D5_S23_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/D5_S23_81_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/H2Ocontrol_S44_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/H2Ocontrol_S44_82_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/H2Ocontrol_S44_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/H2Ocontrol_S44_82_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/MockE_S43_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-npsvhg3c/MockE_S43_83_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/MockE_S43_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-ymxkfn0h/e691fb1d-ca99-4c50-aa21-4e32226a2eb0/data/MockE_S43_83_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: Folder_Apprill_USVIFeb2020/trim_Import_files_Apprill_USVIFeb2020I_run2.qza\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/105tiss_S25_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/105tiss_S25_38_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/105tiss_S25_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/105tiss_S25_38_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.33 s (24 us/read; 2.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             99,156\n",
      "  Read 1 with adapter:                  98,160 (99.0%)\n",
      "  Read 2 with adapter:                  97,925 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        99,156 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,887,136 bp\n",
      "  Read 1:    14,954,565 bp\n",
      "  Read 2:    14,932,571 bp\n",
      "Total written (filtered):     26,064,863 bp (87.2%)\n",
      "  Read 1:    13,089,877 bp\n",
      "  Read 2:    12,974,986 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 98160 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t6\t0.0\t1\t6\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t5\t0.0\t1\t1 4\n",
      "18\t507\t0.0\t1\t36 471\n",
      "19\t97422\t0.0\t1\t94036 3386\n",
      "20\t214\t0.0\t1\t37 177\n",
      "21\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 97925 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1549.3\t0\t7\n",
      "4\t3\t387.3\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t2\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t8\t0.0\t1\t6 2\n",
      "19\t929\t0.0\t1\t113 816\n",
      "20\t96738\t0.0\t2\t93895 2843\n",
      "21\t234\t0.0\t2\t43 191\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/165eco_S1_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/165eco_S1_39_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/165eco_S1_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/165eco_S1_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.16 s (23 us/read; 2.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,685\n",
      "  Read 1 with adapter:                  93,943 (99.2%)\n",
      "  Read 2 with adapter:                  93,715 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        94,685 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,562,466 bp\n",
      "  Read 1:    14,292,123 bp\n",
      "  Read 2:    14,270,343 bp\n",
      "Total written (filtered):     24,904,476 bp (87.2%)\n",
      "  Read 1:    12,507,488 bp\n",
      "  Read 2:    12,396,988 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 93943 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t369.9\t0\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t455\t0.0\t1\t37 418\n",
      "19\t93228\t0.0\t1\t90768 2460\n",
      "20\t249\t0.0\t1\t26 223\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 93715 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1479.5\t0\t9\n",
      "4\t3\t369.9\t0\t3\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t12\t0.0\t1\t2 10\n",
      "19\t897\t0.0\t1\t112 785\n",
      "20\t92566\t0.0\t2\t90504 2062\n",
      "21\t217\t0.0\t2\t47 170\n",
      "31\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/167eco_S2_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/167eco_S2_40_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/167eco_S2_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/167eco_S2_40_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.23 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             99,519\n",
      "  Read 1 with adapter:                  98,696 (99.2%)\n",
      "  Read 2 with adapter:                  98,530 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        99,519 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,001,296 bp\n",
      "  Read 1:    15,013,198 bp\n",
      "  Read 2:    14,988,098 bp\n",
      "Total written (filtered):     26,156,926 bp (87.2%)\n",
      "  Read 1:    13,138,303 bp\n",
      "  Read 2:    13,018,623 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 98696 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t388.7\t0\t1\n",
      "12\t5\t0.0\t1\t5\n",
      "16\t6\t0.0\t1\t5 1\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t482\t0.0\t1\t37 445\n",
      "19\t97968\t0.0\t1\t95221 2747\n",
      "20\t227\t0.0\t1\t41 186\n",
      "25\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 98530 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1555.0\t0\t15\n",
      "4\t3\t388.7\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t10\t0.0\t1\t3 7\n",
      "19\t1017\t0.0\t1\t94 923\n",
      "20\t97241\t0.0\t2\t95055 2186\n",
      "21\t232\t0.0\t2\t58 174\n",
      "40\t1\t0.0\t2\t1\n",
      "41\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/180eco_S3_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/180eco_S3_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/180eco_S3_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/180eco_S3_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.62 s (23 us/read; 2.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,754\n",
      "  Read 1 with adapter:                 112,758 (99.1%)\n",
      "  Read 2 with adapter:                 112,463 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,754 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,305,262 bp\n",
      "  Read 1:    17,166,014 bp\n",
      "  Read 2:    17,139,248 bp\n",
      "Total written (filtered):     29,915,272 bp (87.2%)\n",
      "  Read 1:    15,024,066 bp\n",
      "  Read 2:    14,891,206 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 112758 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t5\n",
      "18\t643\t0.0\t1\t53 590\n",
      "19\t111840\t0.0\t1\t108631 3209\n",
      "20\t260\t0.0\t1\t39 221\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 112463 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1777.4\t0\t10\n",
      "4\t2\t444.4\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t10\t0.0\t1\t4 6\n",
      "19\t1187\t0.0\t1\t114 1073\n",
      "20\t110976\t0.0\t2\t108237 2739\n",
      "21\t265\t0.0\t2\t42 223\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/182eco_S4_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/182eco_S4_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/182eco_S4_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/182eco_S4_42_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.61 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,364\n",
      "  Read 1 with adapter:                 108,514 (99.2%)\n",
      "  Read 2 with adapter:                 108,153 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,364 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,905,518 bp\n",
      "  Read 1:    16,464,959 bp\n",
      "  Read 2:    16,440,559 bp\n",
      "Total written (filtered):     28,682,191 bp (87.2%)\n",
      "  Read 1:    14,403,507 bp\n",
      "  Read 2:    14,278,684 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108514 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t4\t0.0\t1\t2 2\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t490\t0.0\t1\t33 457\n",
      "19\t107756\t0.0\t1\t104923 2833\n",
      "20\t251\t0.0\t1\t51 200\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108153 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1708.8\t0\t9\n",
      "4\t3\t427.2\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t4\t0.0\t1\t0 4\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t4\t0.0\t1\t2 2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t6\t0.0\t1\t2 4\n",
      "19\t1110\t0.0\t1\t121 989\n",
      "20\t106770\t0.0\t2\t104258 2512\n",
      "21\t239\t0.0\t2\t58 181\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/184eco_S5_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/184eco_S5_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/184eco_S5_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/184eco_S5_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.62 s (23 us/read; 2.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,492\n",
      "  Read 1 with adapter:                 112,537 (99.2%)\n",
      "  Read 2 with adapter:                 112,161 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,492 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,173,192 bp\n",
      "  Read 1:    17,099,462 bp\n",
      "  Read 2:    17,073,730 bp\n",
      "Total written (filtered):     29,793,251 bp (87.2%)\n",
      "  Read 1:    14,961,667 bp\n",
      "  Read 2:    14,831,584 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 112537 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t7\t0.0\t1\t6 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t5\t0.0\t1\t3 2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t1 3\n",
      "18\t538\t0.0\t1\t40 498\n",
      "19\t111729\t0.0\t1\t108599 3130\n",
      "20\t248\t0.0\t1\t39 209\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 112161 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1773.3\t0\t11\n",
      "4\t3\t443.3\t0\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t10\t0.0\t1\t5 5\n",
      "19\t1085\t0.0\t1\t120 965\n",
      "20\t110789\t0.0\t2\t108169 2620\n",
      "21\t252\t0.0\t2\t36 216\n",
      "22\t1\t0.0\t2\t1\n",
      "24\t1\t0.0\t2\t1\n",
      "37\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/186eco_S6_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/186eco_S6_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/186eco_S6_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/186eco_S6_44_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.59 s (23 us/read; 2.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,787\n",
      "  Read 1 with adapter:                 110,765 (99.1%)\n",
      "  Read 2 with adapter:                 110,797 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,694,169 bp\n",
      "  Read 1:    16,859,630 bp\n",
      "  Read 2:    16,834,539 bp\n",
      "Total written (filtered):     29,375,143 bp (87.2%)\n",
      "  Read 1:    14,755,488 bp\n",
      "  Read 2:    14,619,655 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110765 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t11\t0.0\t1\t7 4\n",
      "18\t560\t0.0\t1\t35 525\n",
      "19\t109983\t0.0\t1\t106985 2998\n",
      "20\t207\t0.0\t1\t33 174\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110797 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1746.7\t0\t9\n",
      "4\t3\t436.7\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t8\t0.0\t1\t6 2\n",
      "19\t1010\t0.0\t1\t95 915\n",
      "20\t109559\t0.0\t2\t107090 2469\n",
      "21\t201\t0.0\t2\t45 156\n",
      "25\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/192eco_S7_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/192eco_S7_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/192eco_S7_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/192eco_S7_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.45 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,932\n",
      "  Read 1 with adapter:                 103,032 (99.1%)\n",
      "  Read 2 with adapter:                 102,770 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       103,932 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,280,133 bp\n",
      "  Read 1:    15,652,087 bp\n",
      "  Read 2:    15,628,046 bp\n",
      "Total written (filtered):     27,268,747 bp (87.2%)\n",
      "  Read 1:    13,694,904 bp\n",
      "  Read 2:    13,573,843 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103032 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t6\t0.0\t1\t1 5\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t593\t0.0\t1\t31 562\n",
      "19\t102204\t0.0\t1\t99198 3006\n",
      "20\t222\t0.0\t1\t29 193\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 102770 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1623.9\t0\t9\n",
      "4\t2\t406.0\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t1 2\n",
      "18\t10\t0.0\t1\t4 6\n",
      "19\t1209\t0.0\t1\t98 1111\n",
      "20\t101297\t0.0\t2\t98692 2605\n",
      "21\t235\t0.0\t2\t48 187\n",
      "41\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/193eco_S8_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/193eco_S8_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/193eco_S8_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/193eco_S8_46_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.37 s (22 us/read; 2.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,513\n",
      "  Read 1 with adapter:                 105,718 (99.3%)\n",
      "  Read 2 with adapter:                 105,463 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       106,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,124,258 bp\n",
      "  Read 1:    16,074,345 bp\n",
      "  Read 2:    16,049,913 bp\n",
      "Total written (filtered):     28,008,076 bp (87.2%)\n",
      "  Read 1:    14,066,140 bp\n",
      "  Read 2:    13,941,936 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 105718 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t2\t0.0\t1\t2\n",
      "15\t4\t0.0\t1\t1 3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t12\t0.0\t1\t7 5\n",
      "18\t571\t0.0\t1\t26 545\n",
      "19\t104898\t0.0\t1\t102223 2675\n",
      "20\t225\t0.0\t1\t43 182\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 105463 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1664.3\t0\t15\n",
      "4\t5\t416.1\t0\t5\n",
      "10\t3\t0.1\t1\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t10\t0.0\t1\t5 5\n",
      "19\t1082\t0.0\t1\t102 980\n",
      "20\t104076\t0.0\t2\t101740 2336\n",
      "21\t260\t0.0\t2\t41 219\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/195eco_S9_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/195eco_S9_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/195eco_S9_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/195eco_S9_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.69 s (23 us/read; 2.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,157\n",
      "  Read 1 with adapter:                 116,155 (99.1%)\n",
      "  Read 2 with adapter:                 116,013 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       117,157 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,331,482 bp\n",
      "  Read 1:    17,679,481 bp\n",
      "  Read 2:    17,652,001 bp\n",
      "Total written (filtered):     30,806,169 bp (87.2%)\n",
      "  Read 1:    15,473,006 bp\n",
      "  Read 2:    15,333,163 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 116155 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.4\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t6\t0.0\t1\t4 2\n",
      "18\t672\t0.0\t1\t37 635\n",
      "19\t115201\t0.0\t1\t112090 3111\n",
      "20\t265\t0.0\t1\t56 209\n",
      "21\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 116013 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1830.6\t0\t16\n",
      "4\t1\t457.6\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t7\t0.0\t1\t3 4\n",
      "19\t1317\t0.0\t1\t119 1198\n",
      "20\t114406\t0.0\t2\t111896 2510\n",
      "21\t253\t0.0\t2\t44 209\n",
      "35\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/197eco_S10_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/197eco_S10_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/197eco_S10_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/197eco_S10_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.62 s (23 us/read; 2.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,823\n",
      "  Read 1 with adapter:                 110,777 (99.1%)\n",
      "  Read 2 with adapter:                 110,715 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,823 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,698,302 bp\n",
      "  Read 1:    16,861,620 bp\n",
      "  Read 2:    16,836,682 bp\n",
      "Total written (filtered):     29,381,027 bp (87.2%)\n",
      "  Read 1:    14,757,299 bp\n",
      "  Read 2:    14,623,728 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 110777 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t436.8\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t3\t0.0\t1\t1 2\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t565\t0.0\t1\t41 524\n",
      "19\t109969\t0.0\t1\t106957 3012\n",
      "20\t224\t0.0\t1\t42 182\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110715 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1747.2\t0\t15\n",
      "4\t6\t436.8\t0\t6\n",
      "9\t3\t0.4\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t6\t0.0\t1\t2 4\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t12\t0.0\t1\t4 8\n",
      "19\t1090\t0.0\t1\t87 1003\n",
      "20\t109352\t0.0\t2\t107143 2209\n",
      "21\t219\t0.0\t2\t38 181\n",
      "24\t1\t0.0\t2\t1\n",
      "25\t1\t0.0\t2\t1\n",
      "31\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/198eco_S11_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/198eco_S11_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/198eco_S11_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/198eco_S11_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.51 s (25 us/read; 2.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,335\n",
      "  Read 1 with adapter:                  97,687 (97.4%)\n",
      "  Read 2 with adapter:                  99,294 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,335 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,246,250 bp\n",
      "  Read 1:    15,135,959 bp\n",
      "  Read 2:    15,110,291 bp\n",
      "Total written (filtered):     26,405,808 bp (87.3%)\n",
      "  Read 1:    13,280,270 bp\n",
      "  Read 2:    13,125,538 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 97687 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t391.9\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t534\t0.0\t1\t37 497\n",
      "19\t96917\t0.0\t1\t93551 3366\n",
      "20\t222\t0.0\t1\t46 176\n",
      "24\t1\t0.0\t1\t1\n",
      "31\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 99294 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t1567.7\t0\t14\n",
      "4\t1\t391.9\t0\t1\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t8\t0.0\t1\t2 6\n",
      "19\t1065\t0.0\t1\t99 966\n",
      "20\t97932\t0.0\t2\t95968 1964\n",
      "21\t265\t0.0\t2\t53 212\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/204eco_S12_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/204eco_S12_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/204eco_S12_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/204eco_S12_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.54 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,357\n",
      "  Read 1 with adapter:                 108,415 (99.1%)\n",
      "  Read 2 with adapter:                 108,204 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,357 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,921,928 bp\n",
      "  Read 1:    16,473,352 bp\n",
      "  Read 2:    16,448,576 bp\n",
      "Total written (filtered):     28,699,536 bp (87.2%)\n",
      "  Read 1:    14,413,886 bp\n",
      "  Read 2:    14,285,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108415 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t637\t0.0\t1\t38 599\n",
      "19\t107529\t0.0\t1\t104635 2894\n",
      "20\t241\t0.0\t1\t36 205\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108204 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1708.7\t0\t11\n",
      "4\t4\t427.2\t0\t4\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "18\t7\t0.0\t1\t2 5\n",
      "19\t1097\t0.0\t1\t103 994\n",
      "20\t106838\t0.0\t2\t104552 2286\n",
      "21\t242\t0.0\t2\t35 207\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/208eco_S13_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/208eco_S13_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/208eco_S13_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/208eco_S13_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.21 s (27 us/read; 2.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             83,308\n",
      "  Read 1 with adapter:                  76,290 (91.6%)\n",
      "  Read 2 with adapter:                  82,406 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        83,308 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,127,372 bp\n",
      "  Read 1:    12,573,267 bp\n",
      "  Read 2:    12,554,105 bp\n",
      "Total written (filtered):     22,030,954 bp (87.7%)\n",
      "  Read 1:    11,123,973 bp\n",
      "  Read 2:    10,906,981 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 76290 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t370\t0.0\t1\t26 344\n",
      "19\t75754\t0.0\t1\t70798 4956\n",
      "20\t158\t0.0\t1\t28 130\n",
      "21\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 82406 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t1301.7\t0\t19\n",
      "4\t3\t325.4\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t7\t0.0\t1\t4 3\n",
      "19\t716\t0.0\t1\t72 644\n",
      "20\t81487\t0.0\t2\t79909 1578\n",
      "21\t166\t0.0\t2\t36 130\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/105tiss_S25_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/105tiss_S25_38_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/105tiss_S25_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/105tiss_S25_38_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/165eco_S1_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/165eco_S1_39_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/165eco_S1_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/165eco_S1_39_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/167eco_S2_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/167eco_S2_40_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/167eco_S2_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/167eco_S2_40_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/180eco_S3_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/180eco_S3_41_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/180eco_S3_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/180eco_S3_41_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/182eco_S4_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/182eco_S4_42_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/182eco_S4_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/182eco_S4_42_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/184eco_S5_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/184eco_S5_43_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/184eco_S5_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/184eco_S5_43_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/186eco_S6_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/186eco_S6_44_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/186eco_S6_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/186eco_S6_44_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/192eco_S7_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/192eco_S7_45_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/192eco_S7_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/192eco_S7_45_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/193eco_S8_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/193eco_S8_46_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/193eco_S8_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/193eco_S8_46_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/195eco_S9_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/195eco_S9_47_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/195eco_S9_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/195eco_S9_47_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/197eco_S10_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/197eco_S10_48_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/197eco_S10_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/197eco_S10_48_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/198eco_S11_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/198eco_S11_49_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/198eco_S11_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/198eco_S11_49_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/204eco_S12_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/204eco_S12_50_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/204eco_S12_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/204eco_S12_50_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/208eco_S13_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/208eco_S13_51_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/208eco_S13_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/208eco_S13_51_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/219tiss_S26_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/219tiss_S26_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/219tiss_S26_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/219tiss_S26_52_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/219tiss_S26_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/219tiss_S26_52_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/219tiss_S26_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/219tiss_S26_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.54 s (23 us/read; 2.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,568\n",
      "  Read 1 with adapter:                 108,399 (98.9%)\n",
      "  Read 2 with adapter:                 108,504 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       109,568 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,059,757 bp\n",
      "  Read 1:    16,539,394 bp\n",
      "  Read 2:    16,520,363 bp\n",
      "Total written (filtered):     28,831,473 bp (87.2%)\n",
      "  Read 1:    14,480,100 bp\n",
      "  Read 2:    14,351,373 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 108399 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t7\t0.0\t1\t3 4\n",
      "18\t448\t0.0\t1\t43 405\n",
      "19\t107712\t0.0\t1\t103831 3881\n",
      "20\t221\t0.0\t1\t43 178\n",
      "21\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108504 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t1712.0\t0\t20\n",
      "4\t6\t428.0\t0\t6\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t11\t0.0\t1\t6 5\n",
      "19\t839\t0.0\t1\t109 730\n",
      "20\t107371\t0.0\t2\t104418 2953\n",
      "21\t245\t0.0\t2\t41 204\n",
      "22\t1\t0.0\t2\t0 1\n",
      "28\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/230tiss_S27_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/230tiss_S27_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/230tiss_S27_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/230tiss_S27_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.67 s (24 us/read; 2.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,228\n",
      "  Read 1 with adapter:                 109,945 (98.8%)\n",
      "  Read 2 with adapter:                 110,049 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       111,228 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,544,236 bp\n",
      "  Read 1:    16,788,394 bp\n",
      "  Read 2:    16,755,842 bp\n",
      "Total written (filtered):     29,255,625 bp (87.2%)\n",
      "  Read 1:    14,699,725 bp\n",
      "  Read 2:    14,555,900 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 109945 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t434.5\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t7\t0.0\t1\t4 3\n",
      "18\t505\t0.0\t1\t42 463\n",
      "19\t109107\t0.0\t1\t104777 4330\n",
      "20\t313\t0.0\t1\t40 273\n",
      "25\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 110049 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t1737.9\t0\t17\n",
      "11\t2\t0.0\t1\t2\n",
      "14\t4\t0.0\t1\t2 2\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t13\t0.0\t1\t10 3\n",
      "19\t933\t0.0\t1\t99 834\n",
      "20\t108818\t0.0\t2\t105547 3271\n",
      "21\t257\t0.0\t2\t51 206\n",
      "23\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/235tiss_S28_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/235tiss_S28_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/235tiss_S28_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/235tiss_S28_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.16 s (22 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,249\n",
      "  Read 1 with adapter:                  99,015 (98.8%)\n",
      "  Read 2 with adapter:                  99,120 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       100,249 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,243,319 bp\n",
      "  Read 1:    15,133,174 bp\n",
      "  Read 2:    15,110,145 bp\n",
      "Total written (filtered):     26,380,782 bp (87.2%)\n",
      "  Read 1:    13,252,217 bp\n",
      "  Read 2:    13,128,565 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 99015 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t391.6\t0\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t3\t0.0\t1\t1 2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t495\t0.0\t1\t43 452\n",
      "19\t98280\t0.0\t1\t94681 3599\n",
      "20\t220\t0.0\t1\t38 182\n",
      "37\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 99120 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1566.4\t0\t8\n",
      "4\t4\t391.6\t0\t4\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t10\t0.0\t1\t5 5\n",
      "19\t781\t0.0\t1\t95 686\n",
      "20\t98083\t0.0\t2\t95315 2768\n",
      "21\t225\t0.0\t2\t47 178\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/248tiss_S29_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/248tiss_S29_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/248tiss_S29_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/248tiss_S29_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.49 s (23 us/read; 2.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,497\n",
      "  Read 1 with adapter:                 106,243 (98.8%)\n",
      "  Read 2 with adapter:                 106,360 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       107,497 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,388,463 bp\n",
      "  Read 1:    16,205,963 bp\n",
      "  Read 2:    16,182,500 bp\n",
      "Total written (filtered):     28,244,119 bp (87.2%)\n",
      "  Read 1:    14,187,630 bp\n",
      "  Read 2:    14,056,489 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 106243 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t488\t0.0\t1\t35 453\n",
      "19\t105501\t0.0\t1\t101490 4011\n",
      "20\t242\t0.0\t1\t48 194\n",
      "28\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 106360 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t1679.6\t0\t22\n",
      "4\t3\t419.9\t0\t3\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "17\t4\t0.0\t1\t0 4\n",
      "18\t10\t0.0\t1\t4 6\n",
      "19\t934\t0.0\t1\t123 811\n",
      "20\t105152\t0.0\t2\t102000 3152\n",
      "21\t231\t0.0\t2\t52 179\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/253tiss_S30_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/253tiss_S30_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/253tiss_S30_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/253tiss_S30_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.67 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,166\n",
      "  Read 1 with adapter:                 111,711 (98.7%)\n",
      "  Read 2 with adapter:                 111,859 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       113,166 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,133,423 bp\n",
      "  Read 1:    17,079,703 bp\n",
      "  Read 2:    17,053,720 bp\n",
      "Total written (filtered):     29,775,424 bp (87.2%)\n",
      "  Read 1:    14,957,604 bp\n",
      "  Read 2:    14,817,820 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 111711 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t7\t0.0\t1\t2 5\n",
      "18\t628\t0.0\t1\t57 571\n",
      "19\t110806\t0.0\t1\t106824 3982\n",
      "20\t264\t0.0\t1\t30 234\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 111859 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t1768.2\t0\t19\n",
      "4\t5\t442.1\t0\t5\n",
      "9\t2\t0.4\t0\t2\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t3\t0.0\t1\t1 2\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t5\t0.0\t1\t2 3\n",
      "19\t1055\t0.0\t1\t119 936\n",
      "20\t110500\t0.0\t2\t107397 3103\n",
      "21\t261\t0.0\t2\t50 211\n",
      "22\t2\t0.0\t2\t1 1\n",
      "24\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/256tiss_S31_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/256tiss_S31_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/256tiss_S31_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/256tiss_S31_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.93 s (20 us/read; 2.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,115\n",
      "  Read 1 with adapter:                  94,448 (98.3%)\n",
      "  Read 2 with adapter:                  95,050 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        96,115 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,991,867 bp\n",
      "  Read 1:    14,506,349 bp\n",
      "  Read 2:    14,485,518 bp\n",
      "Total written (filtered):     25,297,771 bp (87.3%)\n",
      "  Read 1:    12,712,248 bp\n",
      "  Read 2:    12,585,523 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 94448 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t375.4\t0\t3\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t2\n",
      "17\t5\t0.0\t1\t1 4\n",
      "18\t512\t0.0\t1\t44 468\n",
      "19\t93716\t0.0\t1\t89659 4057\n",
      "20\t203\t0.0\t1\t45 158\n",
      "21\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 95050 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t1501.8\t0\t13\n",
      "4\t3\t375.4\t0\t3\n",
      "9\t2\t0.4\t0\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t3\t0.0\t1\t3\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t6\t0.0\t1\t2 4\n",
      "19\t893\t0.0\t1\t95 798\n",
      "20\t93863\t0.0\t2\t91005 2858\n",
      "21\t257\t0.0\t2\t47 210\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/259tiss_S32_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/259tiss_S32_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/259tiss_S32_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/259tiss_S32_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.45 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             98,891\n",
      "  Read 1 with adapter:                  97,481 (98.6%)\n",
      "  Read 2 with adapter:                  97,821 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        98,891 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,805,505 bp\n",
      "  Read 1:    14,913,944 bp\n",
      "  Read 2:    14,891,561 bp\n",
      "Total written (filtered):     25,998,255 bp (87.2%)\n",
      "  Read 1:    13,062,167 bp\n",
      "  Read 2:    12,936,088 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 97481 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t4\t0.0\t1\t2 2\n",
      "16\t4\t0.0\t1\t1 3\n",
      "17\t8\t0.0\t1\t3 5\n",
      "18\t519\t0.0\t1\t44 475\n",
      "19\t96729\t0.0\t1\t93406 3323\n",
      "20\t216\t0.0\t1\t28 188\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 97821 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1545.2\t0\t9\n",
      "4\t3\t386.3\t0\t3\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t1 4\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t966\t0.0\t1\t136 830\n",
      "20\t96610\t0.0\t2\t94224 2386\n",
      "21\t212\t0.0\t2\t39 173\n",
      "23\t1\t0.0\t2\t1\n",
      "33\t1\t0.0\t2\t1\n",
      "42\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/270eco_S14_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/270eco_S14_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/270eco_S14_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/270eco_S14_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.23 s (22 us/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            102,753\n",
      "  Read 1 with adapter:                 101,484 (98.8%)\n",
      "  Read 2 with adapter:                 101,905 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       102,753 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,904,903 bp\n",
      "  Read 1:    15,463,606 bp\n",
      "  Read 2:    15,441,297 bp\n",
      "Total written (filtered):     26,939,441 bp (87.2%)\n",
      "  Read 1:    13,535,585 bp\n",
      "  Read 2:    13,403,856 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 101484 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "18\t323\t0.0\t1\t36 287\n",
      "19\t100984\t0.0\t1\t98299 2685\n",
      "20\t172\t0.0\t1\t33 139\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 101905 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1605.5\t0\t6\n",
      "4\t1\t401.4\t0\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "18\t3\t0.0\t1\t3\n",
      "19\t714\t0.0\t1\t67 647\n",
      "20\t100996\t0.0\t2\t99114 1882\n",
      "21\t184\t0.0\t2\t34 150\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/273eco_S15_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/273eco_S15_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/273eco_S15_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/273eco_S15_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.09 s (21 us/read; 2.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,462\n",
      "  Read 1 with adapter:                 100,359 (98.9%)\n",
      "  Read 2 with adapter:                 100,518 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       101,462 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,598,682 bp\n",
      "  Read 1:    15,311,001 bp\n",
      "  Read 2:    15,287,681 bp\n",
      "Total written (filtered):     26,682,890 bp (87.2%)\n",
      "  Read 1:    13,404,477 bp\n",
      "  Read 2:    13,278,413 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 100359 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t396.3\t0\t1\n",
      "12\t4\t0.0\t1\t3 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t492\t0.0\t1\t37 455\n",
      "19\t99597\t0.0\t1\t96767 2830\n",
      "20\t253\t0.0\t1\t38 215\n",
      "41\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 100518 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1585.3\t0\t7\n",
      "4\t8\t396.3\t0\t8\n",
      "12\t3\t0.0\t1\t2 1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t3\t0.0\t1\t3\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t974\t0.0\t1\t118 856\n",
      "20\t99294\t0.0\t2\t97263 2031\n",
      "21\t215\t0.0\t2\t37 178\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/274eco_S16_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/274eco_S16_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/274eco_S16_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/274eco_S16_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.72 s (23 us/read; 2.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,126\n",
      "  Read 1 with adapter:                 116,014 (99.1%)\n",
      "  Read 2 with adapter:                 115,876 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       117,126 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,335,147 bp\n",
      "  Read 1:    17,681,338 bp\n",
      "  Read 2:    17,653,809 bp\n",
      "Total written (filtered):     30,815,027 bp (87.2%)\n",
      "  Read 1:    15,477,432 bp\n",
      "  Read 2:    15,337,595 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 116014 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t457.5\t0\t3\n",
      "9\t1\t0.4\t0\t1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t7\t0.0\t1\t6 1\n",
      "18\t525\t0.0\t1\t37 488\n",
      "19\t115223\t0.0\t1\t112040 3183\n",
      "20\t251\t0.0\t1\t49 202\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 115876 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t1830.1\t0\t27\n",
      "4\t3\t457.5\t0\t3\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t9\t0.0\t1\t5 4\n",
      "19\t1019\t0.0\t1\t98 921\n",
      "20\t114568\t0.0\t2\t111901 2667\n",
      "21\t237\t0.0\t2\t53 184\n",
      "22\t1\t0.0\t2\t1\n",
      "29\t1\t0.0\t2\t1\n",
      "36\t1\t0.0\t2\t0 1\n",
      "42\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/77Ltiss_S33_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/77Ltiss_S33_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/77Ltiss_S33_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/77Ltiss_S33_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.53 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,206\n",
      "  Read 1 with adapter:                 106,152 (99.0%)\n",
      "  Read 2 with adapter:                 106,114 (99.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       107,206 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,339,148 bp\n",
      "  Read 1:    16,182,190 bp\n",
      "  Read 2:    16,156,958 bp\n",
      "Total written (filtered):     28,201,330 bp (87.2%)\n",
      "  Read 1:    14,165,639 bp\n",
      "  Read 2:    14,035,691 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 106152 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t418.8\t0\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t0 3\n",
      "16\t5\t0.0\t1\t2 3\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t506\t0.0\t1\t45 461\n",
      "19\t105392\t0.0\t1\t102002 3390\n",
      "20\t238\t0.0\t1\t52 186\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 106114 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1675.1\t0\t11\n",
      "4\t3\t418.8\t0\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t3\t0.0\t1\t3\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t9\t0.0\t1\t4 5\n",
      "19\t972\t0.0\t1\t107 865\n",
      "20\t104880\t0.0\t2\t102135 2745\n",
      "21\t232\t0.0\t2\t43 189\n",
      "28\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/81Ltiss_S34_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/81Ltiss_S34_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/81Ltiss_S34_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/81Ltiss_S34_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.95 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,649\n",
      "  Read 1 with adapter:                 119,584 (99.1%)\n",
      "  Read 2 with adapter:                 119,519 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       120,649 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,400,093 bp\n",
      "  Read 1:    18,212,455 bp\n",
      "  Read 2:    18,187,638 bp\n",
      "Total written (filtered):     31,738,952 bp (87.2%)\n",
      "  Read 1:    15,940,689 bp\n",
      "  Read 2:    15,798,263 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 119584 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t471.3\t0\t2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t0 3\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t6\t0.0\t1\t6\n",
      "18\t495\t0.0\t1\t44 451\n",
      "19\t118829\t0.0\t1\t114858 3971\n",
      "20\t244\t0.0\t1\t30 214\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 119519 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t1885.1\t0\t15\n",
      "4\t6\t471.3\t0\t6\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t2\t0.0\t1\t0 2\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t8\t0.0\t1\t5 3\n",
      "19\t864\t0.0\t1\t116 748\n",
      "20\t118343\t0.0\t2\t115237 3106\n",
      "21\t271\t0.0\t2\t62 209\n",
      "22\t1\t0.0\t2\t0 1\n",
      "24\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/82tiss_S18_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/82tiss_S18_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/82tiss_S18_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/82tiss_S18_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.12 s (20 us/read; 2.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,301\n",
      "  Read 1 with adapter:                 103,225 (99.0%)\n",
      "  Read 2 with adapter:                 103,146 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       104,301 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,482,987 bp\n",
      "  Read 1:    15,749,045 bp\n",
      "  Read 2:    15,733,942 bp\n",
      "Total written (filtered):     27,460,299 bp (87.2%)\n",
      "  Read 1:    13,788,196 bp\n",
      "  Read 2:    13,672,103 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 103225 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t407.4\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t6\t0.0\t1\t5 1\n",
      "13\t5\t0.0\t1\t5\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t6\t0.0\t1\t1 5\n",
      "17\t7\t0.0\t1\t4 3\n",
      "18\t563\t0.0\t1\t39 524\n",
      "19\t102379\t0.0\t1\t98870 3509\n",
      "20\t255\t0.0\t1\t41 214\n",
      "33\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 103146 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t1629.7\t0\t9\n",
      "4\t2\t407.4\t0\t2\n",
      "10\t2\t0.1\t1\t2\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t5\t0.0\t1\t1 4\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t9\t0.0\t1\t5 4\n",
      "19\t1064\t0.0\t1\t120 944\n",
      "20\t101774\t0.0\t2\t99113 2661\n",
      "21\t270\t0.0\t2\t54 216\n",
      "24\t1\t0.0\t2\t1\n",
      "29\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/83Ltiss_S35_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/83Ltiss_S35_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/83Ltiss_S35_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/83Ltiss_S35_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.72 s (21 us/read; 2.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            128,453\n",
      "  Read 1 with adapter:                 127,049 (98.9%)\n",
      "  Read 2 with adapter:                 127,066 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       128,453 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,762,568 bp\n",
      "  Read 1:    19,393,545 bp\n",
      "  Read 2:    19,369,023 bp\n",
      "Total written (filtered):     33,809,257 bp (87.2%)\n",
      "  Read 1:    16,980,143 bp\n",
      "  Read 2:    16,829,114 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 127049 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t2.0\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t3\t0.0\t1\t3\n",
      "16\t9\t0.0\t1\t3 6\n",
      "17\t8\t0.0\t1\t4 4\n",
      "18\t700\t0.0\t1\t59 641\n",
      "19\t126050\t0.0\t1\t121570 4480\n",
      "20\t268\t0.0\t1\t45 223\n",
      "29\t1\t0.0\t1\t1\n",
      "30\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 127066 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t2007.1\t0\t20\n",
      "4\t6\t501.8\t0\t6\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t3\t0.0\t1\t2 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t15\t0.0\t1\t9 6\n",
      "19\t1183\t0.0\t1\t138 1045\n",
      "20\t125531\t0.0\t2\t122266 3265\n",
      "21\t299\t0.0\t2\t61 238\n",
      "31\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/84tiss_S19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/84tiss_S19_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/84tiss_S19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/84tiss_S19_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.87 s (23 us/read; 2.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,121\n",
      "  Read 1 with adapter:                 123,810 (99.0%)\n",
      "  Read 2 with adapter:                 123,729 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       125,121 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,753,515 bp\n",
      "  Read 1:    18,890,920 bp\n",
      "  Read 2:    18,862,595 bp\n",
      "Total written (filtered):     32,927,969 bp (87.2%)\n",
      "  Read 1:    16,538,876 bp\n",
      "  Read 2:    16,389,093 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 123810 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1955.0\t0\t1\n",
      "4\t1\t488.8\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t3\t0.0\t1\t1 2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "16\t8\t0.0\t1\t4 4\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t506\t0.0\t1\t46 460\n",
      "19\t123030\t0.0\t1\t118821 4209\n",
      "20\t253\t0.0\t1\t46 207\n",
      "27\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 123729 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1955.0\t0\t12\n",
      "4\t1\t488.8\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t5\t0.0\t1\t4 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t14\t0.0\t1\t5 9\n",
      "19\t1016\t0.0\t1\t122 894\n",
      "20\t122424\t0.0\t2\t119151 3273\n",
      "21\t250\t0.0\t2\t54 196\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/89tiss_S20_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/89tiss_S20_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/89tiss_S20_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/89tiss_S20_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.70 s (21 us/read; 2.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,105\n",
      "  Read 1 with adapter:                 127,680 (98.9%)\n",
      "  Read 2 with adapter:                 127,505 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       129,105 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,854,614 bp\n",
      "  Read 1:    19,446,958 bp\n",
      "  Read 2:    19,407,656 bp\n",
      "Total written (filtered):     33,880,379 bp (87.2%)\n",
      "  Read 1:    17,021,513 bp\n",
      "  Read 2:    16,858,866 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 127680 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t504.3\t0\t2\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t4\t0.0\t1\t3 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t0 3\n",
      "17\t14\t0.0\t1\t10 4\n",
      "18\t760\t0.0\t1\t57 703\n",
      "19\t126521\t0.0\t1\t122169 4352\n",
      "20\t369\t0.0\t1\t61 308\n",
      "21\t1\t0.0\t1\t0 1\n",
      "56\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 127505 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t2017.3\t0\t18\n",
      "4\t6\t504.3\t0\t6\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t3\t0.0\t1\t1 2\n",
      "14\t4\t0.0\t1\t1 3\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t16\t0.0\t1\t6 10\n",
      "19\t1159\t0.0\t1\t143 1016\n",
      "20\t125944\t0.0\t2\t122438 3506\n",
      "21\t345\t0.0\t2\t56 289\n",
      "50\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/230tiss_S27_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/230tiss_S27_53_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/230tiss_S27_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/230tiss_S27_53_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/235tiss_S28_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/235tiss_S28_54_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/235tiss_S28_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/235tiss_S28_54_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/248tiss_S29_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/248tiss_S29_55_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/248tiss_S29_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/248tiss_S29_55_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/253tiss_S30_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/253tiss_S30_56_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/253tiss_S30_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/253tiss_S30_56_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/256tiss_S31_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/256tiss_S31_57_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/256tiss_S31_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/256tiss_S31_57_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/259tiss_S32_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/259tiss_S32_58_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/259tiss_S32_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/259tiss_S32_58_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/270eco_S14_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/270eco_S14_59_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/270eco_S14_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/270eco_S14_59_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/273eco_S15_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/273eco_S15_60_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/273eco_S15_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/273eco_S15_60_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/274eco_S16_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/274eco_S16_61_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/274eco_S16_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/274eco_S16_61_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/77Ltiss_S33_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/77Ltiss_S33_62_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/77Ltiss_S33_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/77Ltiss_S33_62_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/81Ltiss_S34_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/81Ltiss_S34_63_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/81Ltiss_S34_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/81Ltiss_S34_63_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/82tiss_S18_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/82tiss_S18_64_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/82tiss_S18_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/82tiss_S18_64_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/83Ltiss_S35_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/83Ltiss_S35_65_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/83Ltiss_S35_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/83Ltiss_S35_65_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/84tiss_S19_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/84tiss_S19_66_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/84tiss_S19_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/84tiss_S19_66_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/89tiss_S20_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/89tiss_S20_67_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/89tiss_S20_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/89tiss_S20_67_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/93tiss_S21_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/93tiss_S21_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/93tiss_S21_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/93tiss_S21_68_L001_R2_001.fastq.gzThis is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/93tiss_S21_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/93tiss_S21_68_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/93tiss_S21_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/93tiss_S21_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.88 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            123,438\n",
      "  Read 1 with adapter:                 122,245 (99.0%)\n",
      "  Read 2 with adapter:                 122,103 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       123,438 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,239,947 bp\n",
      "  Read 1:    18,631,746 bp\n",
      "  Read 2:    18,608,201 bp\n",
      "Total written (filtered):     32,476,718 bp (87.2%)\n",
      "  Read 1:    16,309,504 bp\n",
      "  Read 2:    16,167,214 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 122245 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t4\t0.0\t1\t2 2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t599\t0.0\t1\t37 562\n",
      "19\t121399\t0.0\t1\t117160 4239\n",
      "20\t231\t0.0\t1\t40 191\n",
      "25\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 122103 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1928.7\t0\t11\n",
      "4\t3\t482.2\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t0 2\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t5\t0.0\t1\t3 2\n",
      "19\t1045\t0.0\t1\t119 926\n",
      "20\t120756\t0.0\t2\t117355 3401\n",
      "21\t272\t0.0\t2\t41 231\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/94tiss_S22_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/94tiss_S22_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/94tiss_S22_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/94tiss_S22_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.60 s (23 us/read; 2.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,721\n",
      "  Read 1 with adapter:                 113,378 (98.8%)\n",
      "  Read 2 with adapter:                 113,238 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       114,721 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,600,926 bp\n",
      "  Read 1:    17,312,608 bp\n",
      "  Read 2:    17,288,318 bp\n",
      "Total written (filtered):     30,183,631 bp (87.2%)\n",
      "  Read 1:    15,158,848 bp\n",
      "  Read 2:    15,024,783 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 113378 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "12\t4\t0.0\t1\t2 2\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t3 2\n",
      "18\t612\t0.0\t1\t37 575\n",
      "19\t112475\t0.0\t1\t108461 4014\n",
      "20\t274\t0.0\t1\t44 230\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 113238 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t1792.5\t0\t16\n",
      "4\t4\t448.1\t0\t4\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t4\t0.0\t1\t0 4\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t19\t0.0\t1\t8 11\n",
      "19\t1100\t0.0\t1\t138 962\n",
      "20\t111796\t0.0\t2\t108744 3052\n",
      "21\t289\t0.0\t2\t60 229\n",
      "22\t1\t0.0\t2\t0 1\n",
      "25\t1\t0.0\t2\t1\n",
      "27\t1\t0.0\t2\t1\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/95tiss_S23_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/95tiss_S23_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/95tiss_S23_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/95tiss_S23_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.06 s (23 us/read; 2.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,554\n",
      "  Read 1 with adapter:                  87,911 (97.1%)\n",
      "  Read 2 with adapter:                  89,447 (98.8%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        90,554 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,312,141 bp\n",
      "  Read 1:    13,664,744 bp\n",
      "  Read 2:    13,647,397 bp\n",
      "Total written (filtered):     23,854,316 bp (87.3%)\n",
      "  Read 1:    11,994,762 bp\n",
      "  Read 2:    11,859,554 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 87911 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t353.7\t0\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t3\t0.0\t1\t1 2\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t438\t0.0\t1\t44 394\n",
      "19\t87266\t0.0\t1\t83005 4261\n",
      "20\t191\t0.0\t1\t34 157\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 89447 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t1414.9\t0\t18\n",
      "4\t3\t353.7\t0\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t6\t0.0\t1\t2 4\n",
      "18\t9\t0.0\t1\t5 4\n",
      "19\t901\t0.0\t1\t92 809\n",
      "20\t88289\t0.0\t2\t85783 2506\n",
      "21\t217\t0.0\t2\t35 182\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/99tiss_S24_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/99tiss_S24_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/99tiss_S24_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/99tiss_S24_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 2.21 s (22 us/read; 2.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             98,759\n",
      "  Read 1 with adapter:                  97,690 (98.9%)\n",
      "  Read 2 with adapter:                  97,696 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        98,759 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,727,815 bp\n",
      "  Read 1:    14,875,687 bp\n",
      "  Read 2:    14,852,128 bp\n",
      "Total written (filtered):     25,919,103 bp (87.2%)\n",
      "  Read 1:    13,019,938 bp\n",
      "  Read 2:    12,899,165 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 97690 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t0 4\n",
      "18\t523\t0.0\t1\t34 489\n",
      "19\t96929\t0.0\t1\t93644 3285\n",
      "20\t226\t0.0\t1\t37 189\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 97696 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1543.1\t0\t10\n",
      "4\t4\t385.8\t0\t4\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t6\t0.0\t1\t3 3\n",
      "18\t12\t0.0\t1\t4 8\n",
      "19\t898\t0.0\t1\t117 781\n",
      "20\t96520\t0.0\t2\t94159 2361\n",
      "21\t242\t0.0\t2\t60 182\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/D3_S36_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/D3_S36_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/D3_S36_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/D3_S36_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.08 s (36 us/read; 1.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,746\n",
      "  Read 1 with adapter:                  29,408 (98.9%)\n",
      "  Read 2 with adapter:                  29,352 (98.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,746 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,949,944 bp\n",
      "  Read 1:     4,482,918 bp\n",
      "  Read 2:     4,467,026 bp\n",
      "Total written (filtered):      7,804,645 bp (87.2%)\n",
      "  Read 1:     3,924,284 bp\n",
      "  Read 2:     3,880,361 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29408 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t192\t0.0\t1\t10 182\n",
      "19\t29139\t0.0\t1\t28127 1012\n",
      "20\t76\t0.0\t1\t5 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 29352 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t464.8\t0\t5\n",
      "4\t2\t116.2\t0\t2\n",
      "18\t1\t0.0\t1\t0 1\n",
      "19\t324\t0.0\t1\t20 304\n",
      "20\t28952\t0.0\t2\t28024 928\n",
      "21\t68\t0.0\t2\t6 62\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/D6_S17_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/D6_S17_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/D6_S17_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/D6_S17_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.30 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,959\n",
      "  Read 1 with adapter:                  39,576 (99.0%)\n",
      "  Read 2 with adapter:                  39,132 (97.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        39,959 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,685,117 bp\n",
      "  Read 1:     5,357,011 bp\n",
      "  Read 2:     5,328,106 bp\n",
      "Total written (filtered):      9,151,133 bp (85.6%)\n",
      "  Read 1:     4,605,224 bp\n",
      "  Read 2:     4,545,909 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 39576 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t243\t0.0\t1\t20 223\n",
      "19\t39241\t0.0\t1\t38212 1029\n",
      "20\t90\t0.0\t1\t16 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 39132 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t624.4\t0\t2\n",
      "10\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t3\t0.0\t1\t1 2\n",
      "19\t477\t0.0\t1\t30 447\n",
      "20\t38538\t0.0\t2\t37450 1088\n",
      "21\t106\t0.0\t2\t21 85\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/H2Ocontrol-run3_S38_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/H2Ocontrol-run3_S38_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/H2Ocontrol-run3_S38_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/H2Ocontrol-run3_S38_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 0.96 s (40 us/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,791\n",
      "  Read 1 with adapter:                  23,531 (98.9%)\n",
      "  Read 2 with adapter:                  22,897 (96.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,791 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,941,685 bp\n",
      "  Read 1:     2,494,010 bp\n",
      "  Read 2:     2,447,675 bp\n",
      "Total written (filtered):      4,037,028 bp (81.7%)\n",
      "  Read 1:     2,046,983 bp\n",
      "  Read 2:     1,990,045 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23531 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t169\t0.0\t1\t10 159\n",
      "19\t23292\t0.0\t1\t22607 685\n",
      "20\t64\t0.0\t1\t9 55\n",
      "47\t2\t0.0\t1\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 22897 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t371.7\t0\t3\n",
      "4\t1\t92.9\t0\t1\n",
      "13\t3\t0.0\t1\t3\n",
      "18\t1\t0.0\t1\t0 1\n",
      "19\t301\t0.0\t1\t27 274\n",
      "20\t22530\t0.0\t2\t21703 827\n",
      "21\t55\t0.0\t2\t10 45\n",
      "28\t1\t0.0\t2\t1\n",
      "29\t2\t0.0\t2\t2\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/MockE-run3_S37_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/MockE-run3_S37_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/MockE-run3_S37_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/MockE-run3_S37_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "Finished in 1.96 s (21 us/read; 2.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,865\n",
      "  Read 1 with adapter:                  90,696 (97.7%)\n",
      "  Read 2 with adapter:                  91,820 (98.9%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        92,865 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,981,339 bp\n",
      "  Read 1:    14,011,733 bp\n",
      "  Read 2:    13,969,606 bp\n",
      "Total written (filtered):     24,423,085 bp (87.3%)\n",
      "  Read 1:    12,288,836 bp\n",
      "  Read 2:    12,134,249 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 90696 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t362.8\t0\t1\n",
      "10\t1\t0.1\t1\t1\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t6\t0.0\t1\t1 5\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t437\t0.0\t1\t40 397\n",
      "19\t90031\t0.0\t1\t86896 3135\n",
      "20\t209\t0.0\t1\t28 181\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 91820 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1451.0\t0\t11\n",
      "4\t7\t362.8\t0\t7\n",
      "10\t1\t0.1\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t11\t0.0\t1\t3 8\n",
      "19\t888\t0.0\t1\t96 792\n",
      "20\t90687\t0.0\t2\t88853 1834\n",
      "21\t208\t0.0\t2\t37 171\n",
      "\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/94tiss_S22_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/94tiss_S22_69_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/94tiss_S22_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/94tiss_S22_69_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/95tiss_S23_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/95tiss_S23_70_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/95tiss_S23_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/95tiss_S23_70_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/99tiss_S24_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/99tiss_S24_71_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/99tiss_S24_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/99tiss_S24_71_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/D3_S36_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/D3_S36_72_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/D3_S36_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/D3_S36_72_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/D6_S17_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/D6_S17_73_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/D6_S17_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/D6_S17_73_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/H2Ocontrol-run3_S38_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/H2Ocontrol-run3_S38_74_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/H2Ocontrol-run3_S38_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/H2Ocontrol-run3_S38_74_L001_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/MockE-run3_S37_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1d06t6e2/MockE-run3_S37_75_L001_R2_001.fastq.gz --front GTGYCAGCMGCCGCGGTAA -G GGACTACNVGGGTWTCTAAT /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/MockE-run3_S37_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-4e8u7skh/2396bebf-7f13-4257-8699-c438834f9d4f/data/MockE-run3_S37_75_L001_R2_001.fastq.gz\n",
      "\n",
      "Saved SampleData[PairedEndSequencesWithQuality] to: Folder_Apprill_USVIFeb2020/trim_Import_files_Apprill_USVIFeb2020I_run3.qza\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "#trim PCR primers\n",
    "for f1 in /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_USVIFeb2020/Import_files_Apprill_USVIFeb2020I_run*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%*.csv};\n",
    "\n",
    "qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences $f1 \\\n",
    "--p-cores 2 \\\n",
    "--p-front-f GTGYCAGCMGCCGCGGTAA \\\n",
    "--p-front-r GGACTACNVGGGTWTCTAAT \\\n",
    "--o-trimmed-sequences Folder_Apprill_USVIFeb2020/trim_$fn \\\n",
    "--verbose\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: Folder_Apprill_USVIFeb2020/trim_Import_files_Apprill_USVIFeb2020I_run1.qzv\n",
      "Saved Visualization to: Folder_Apprill_USVIFeb2020/trim_Import_files_Apprill_USVIFeb2020I_run2.qzv\n",
      "Saved Visualization to: Folder_Apprill_USVIFeb2020/trim_Import_files_Apprill_USVIFeb2020I_run3.qzv\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for f1 in /space/home/rosales/projects/2020_SCTLD_meta-analysis/Folder_Apprill_USVIFeb2020/trim*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data $f1 \\\n",
    "--o-visualization Folder_Apprill_USVIFeb2020/$fn.qzv\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "#%%bash\n",
    "\n",
    "#for f1 in Folder_Apprill_USVIFeb2020/trim_Import_files_Apprill_USVIFeb2020I_run*.qza\n",
    "#do\n",
    "#newfile=\"$(basename $f1)\"\n",
    "#fn=${newfile%.qza};\n",
    "\n",
    "#qiime dada2 denoise-paired --i-demultiplexed-seqs $f1 --p-trunc-len-f 130 --p-trunc-len-r  130 --p-n-threads 10 --o-representative-sequences Folder_Apprill_USVIFeb2020/rep_$fn.qza --o-table Folder_Apprill_USVIFeb2020/table_$fn.qza --o-denoising-stats Folder_Apprill_USVIFeb2020/dada_$fn.qza \n",
    "\n",
    "#done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: Folder_Apprill_USVIFeb2020/table_trim_Import_files_Apprill_USVIFeb2020I_run1.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Apprill_USVIFeb2020/rep_trim_Import_files_Apprill_USVIFeb2020I_run1.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Apprill_USVIFeb2020/dada_trim_Import_files_Apprill_USVIFeb2020I_run1.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Apprill_USVIFeb2020/table_trim_Import_files_Apprill_USVIFeb2020I_run2.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Apprill_USVIFeb2020/rep_trim_Import_files_Apprill_USVIFeb2020I_run2.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Apprill_USVIFeb2020/dada_trim_Import_files_Apprill_USVIFeb2020I_run2.qza\n",
      "Saved FeatureTable[Frequency] to: Folder_Apprill_USVIFeb2020/table_trim_Import_files_Apprill_USVIFeb2020I_run3.qza\n",
      "Saved FeatureData[Sequence] to: Folder_Apprill_USVIFeb2020/rep_trim_Import_files_Apprill_USVIFeb2020I_run3.qza\n",
      "Saved SampleData[DADA2Stats] to: Folder_Apprill_USVIFeb2020/dada_trim_Import_files_Apprill_USVIFeb2020I_run3.qza\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for f1 in Folder_Apprill_USVIFeb2020/trim_Import_files_Apprill_USVIFeb2020I_run*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime dada2 denoise-single \\\n",
    "--i-demultiplexed-seqs $f1 \\\n",
    "--p-trunc-len 0 \\\n",
    "--p-n-threads 10 \\\n",
    "--p-max-ee 3 \\\n",
    "--o-representative-sequences Folder_Apprill_USVIFeb2020/rep_$fn.qza \\\n",
    "--o-table Folder_Apprill_USVIFeb2020/table_$fn.qza \\\n",
    "--o-denoising-stats Folder_Apprill_USVIFeb2020/dada_$fn.qza \n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_Apprill_USVIFeb2020/rep_trim_Import_files_Apprill_USVIFeb2020.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table merge-seqs \\\n",
    "--i-data Folder_Apprill_USVIFeb2020/rep_trim_Import_files_Apprill_USVIFeb2020I_run1.qza \\\n",
    "--i-data Folder_Apprill_USVIFeb2020/rep_trim_Import_files_Apprill_USVIFeb2020I_run2.qza \\\n",
    "--i-data Folder_Apprill_USVIFeb2020/rep_trim_Import_files_Apprill_USVIFeb2020I_run3.qza \\\n",
    "--o-merged-data Folder_Apprill_USVIFeb2020/rep_trim_Import_files_Apprill_USVIFeb2020.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_Apprill_USVIFeb2020/table_Apprill_USVIFeb2020I.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table merge \\\n",
    " --i-tables Folder_Apprill_USVIFeb2020/table_trim_Import_files_Apprill_USVIFeb2020I_run1.qza \\\n",
    " --i-tables Folder_Apprill_USVIFeb2020/table_trim_Import_files_Apprill_USVIFeb2020I_run2.qza \\\n",
    " --i-tables Folder_Apprill_USVIFeb2020/table_trim_Import_files_Apprill_USVIFeb2020I_run3.qza \\\n",
    " --o-merged-table Folder_Apprill_USVIFeb2020/table_Apprill_USVIFeb2020I.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: Folder_Apprill_USVIFeb2020/dada_trim_Import_files_Apprill_USVIFeb2020I_run1.qzv\n",
      "Saved Visualization to: Folder_Apprill_USVIFeb2020/dada_trim_Import_files_Apprill_USVIFeb2020I_run2.qzv\n",
      "Saved Visualization to: Folder_Apprill_USVIFeb2020/dada_trim_Import_files_Apprill_USVIFeb2020I_run3.qzv\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for f1 in Folder_Apprill_USVIFeb2020/dada_trim_Import_files_Apprill_USVIFeb2020I_run*.qza\n",
    "do\n",
    "newfile=\"$(basename $f1)\"\n",
    "fn=${newfile%.qza};\n",
    "\n",
    "qiime metadata tabulate \\\n",
    "  --m-input-file $f1 \\\n",
    "  --o-visualization Folder_Apprill_USVIFeb2020/$fn\n",
    "\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Taxonomy] to: Folder_Apprill_USVIFeb2020/taxonomy_Apprill_USVIFeb2020.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier /space/home/rosales/databases/SILVA_DBs/silva-138-99-nb-classifier.qza \\\n",
    "  --i-reads Folder_Apprill_USVIFeb2020/rep_trim_Import_files_Apprill_USVIFeb2020.qza \\\n",
    "  --p-n-jobs -7 \\\n",
    "  --o-classification Folder_Apprill_USVIFeb2020/taxonomy_Apprill_USVIFeb2020.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_Apprill_USVIFeb2020/rep-BacArc_Apprill_USVIFeb2020.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime taxa filter-seqs \\\n",
    "--i-sequences Folder_Apprill_USVIFeb2020/rep_trim_Import_files_Apprill_USVIFeb2020.qza \\\n",
    "--i-taxonomy Folder_Apprill_USVIFeb2020/taxonomy_Apprill_USVIFeb2020.qza \\\n",
    "--p-exclude mitochondria,chloroplast \\\n",
    "--p-include d__Bacteria,d__Archaea \\\n",
    "--o-filtered-sequences Folder_Apprill_USVIFeb2020/rep-BacArc_Apprill_USVIFeb2020.qza \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_Apprill_USVIFeb2020/table-BacArc_Apprill_USVIFeb2020.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime taxa filter-table \\\n",
    "--i-table Folder_Apprill_USVIFeb2020/table_Apprill_USVIFeb2020I.qza \\\n",
    "--i-taxonomy Folder_Apprill_USVIFeb2020/taxonomy_Apprill_USVIFeb2020.qza \\\n",
    "--p-exclude mitochondria,chloroplast \\\n",
    "--p-include d__Bacteria,d__Archaea \\\n",
    "--o-filtered-table Folder_Apprill_USVIFeb2020/table-BacArc_Apprill_USVIFeb2020.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_Apprill_USVIFeb2020/taxonomy_Apprill_USVIFeb2020.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: Folder_Apprill_USVIFeb2020/taxa-bar-Apprill_USVIFeb2020.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file Folder_Apprill_USVIFeb2020/taxonomy_Apprill_USVIFeb2020.qza \\\n",
    "  --o-visualization Folder_Apprill_USVIFeb2020/taxonomy_Apprill_USVIFeb2020.qzv\n",
    "\n",
    "\n",
    "!qiime taxa barplot \\\n",
    "  --i-table Folder_Apprill_USVIFeb2020/table-BacArc_Apprill_USVIFeb2020.qza \\\n",
    "  --i-taxonomy Folder_Apprill_USVIFeb2020/taxonomy_Apprill_USVIFeb2020.qza \\\n",
    "  --m-metadata-file  metadata/metadata_Apprill_USVIFeb2020.txt \\\n",
    "  --o-visualization  Folder_Apprill_USVIFeb2020/taxa-bar-Apprill_USVIFeb2020.qzv"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
