{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "env: STUDY=StewartMedina_FL2019\n"
     ]
    }
   ],
   "source": [
    "%env STUDY StewartMedina_FL2019\n",
    "dir_path='/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/StewartMedina_FL2019/Seq_Sweet16S/'\n",
    "wdir= \"/space/home/rosales/projects/2020_SCTLD_meta-analysis/\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/rawdata/2020_SCTLD_16S_meta_analysis/StewartMedina_FL2019/Seq_Sweet16S\n",
      "/space/home/rosales/projects/2020_SCTLD_meta-analysis\n"
     ]
    }
   ],
   "source": [
    "%cd $dir_path\n",
    "!ls *.gz > /space/home/rosales/projects/2020_SCTLD_meta-analysis/files_$STUDY.txt\n",
    "%cd $wdir"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import re\n",
    "# import file with header \"filepath\"\n",
    "header_list = [\"filepath\"]\n",
    "files_df=pd.read_table(\"/space/home/rosales/projects/2020_SCTLD_meta-analysis/files_StewartMedina_FL2019.txt\",\n",
    "                                    names=header_list)\n",
    "# change from F and R to R1 and R2 to parse on \"_R\"\n",
    "files_df['filepath']=files_df['filepath'].str.replace('_F_','_R1_')\n",
    "files_df['filepath']=files_df['filepath'].str.replace('_R_','_R2_')\n",
    "#files_df[files_df['filepath'].str.match('PSTR2c-719')]\n",
    "# parse column filepath on '_R' which creates two columns a sample ID anf the rest of the filename\n",
    "files_df[['sample-id', \"other\"]]= files_df.filepath.str.split(\"_R\",expand=True,)\n",
    "# if string contains R1 in filename make a new coulumn \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'forward-absolute-filepath'] = files_df['filepath'] \n",
    "files_df.loc[files_df['filepath'].str.contains('_R1_'), 'reverse-absolute-filepath'] = files_df['filepath']\n",
    "#change Reverse and Forward back to the original IDs\n",
    "files_df['forward-absolute-filepath']=files_df['forward-absolute-filepath'].str.replace('_R1_','_F_')\n",
    "files_df['reverse-absolute-filepath']=files_df['reverse-absolute-filepath'].str.replace('_R1_','_R_')\n",
    "\n",
    "files_df['forward-absolute-filepath'] = dir_path + files_df['forward-absolute-filepath'].astype(str)\n",
    "files_df['reverse-absolute-filepath'] = dir_path + files_df['reverse-absolute-filepath'].astype(str)\n",
    "# only print three columns needed slected\n",
    "files_df =files_df[[\"sample-id\",\"forward-absolute-filepath\", \"reverse-absolute-filepath\"]]\n",
    "# remove duplicates samples\n",
    "files_df=files_df.drop_duplicates(\"sample-id\")\n",
    "name ='StewartMedina_FL2019'\n",
    "files_df.to_csv('/space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_{0}.csv'.format(name),\n",
    "index=False, header=True, sep =\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir Folder_StewartMedina_FL2019"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/space/home/rosales/anaconda2/envs/qiime2-2020.2/bin/qiime\r\n"
     ]
    }
   ],
   "source": [
    "!which qiime"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_StewartMedina_FL2019.csv as PairedEndFastqManifestPhred33V2 to Folder_StewartMedina_FL2019/Import_StewartMedina_FL2019.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime tools import \\\n",
    "    --output-path Folder_StewartMedina_FL2019/Import_StewartMedina_FL2019.qza \\\n",
    "    --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "    --input-path /space/home/rosales/projects/2020_SCTLD_meta-analysis/manifest_files/files_StewartMedina_FL2019.csv \\\n",
    "    --input-format PairedEndFastqManifestPhred33V2\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_StewartMedina_FL2019/Import_StewartMedina_FL2019.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data Folder_StewartMedina_FL2019/Import_StewartMedina_FL2019.qza \\\n",
    "--o-visualization Folder_StewartMedina_FL2019/Import_StewartMedina_FL2019.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1c-719-9_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1c-719-9_79_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1c-719-9_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1c-719-9_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1c-719-9_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1c-719-9_79_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1c-719-9_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1c-719-9_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        30,935 reads  @     46.0 µs/read;   1.30 M reads/minute\n",
      "Finished in 1.46 s (47 us/read; 1.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,935\n",
      "  Read 1 with adapter:                  30,874 (99.8%)\n",
      "  Read 2 with adapter:                  30,794 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,935 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,280,500 bp\n",
      "  Read 1:     4,640,250 bp\n",
      "  Read 2:     4,640,250 bp\n",
      "Total written (filtered):      8,078,271 bp (87.0%)\n",
      "  Read 1:     4,053,726 bp\n",
      "  Read 2:     4,024,545 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 30874 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t148\t0.0\t1\t18 130\n",
      "19\t30667\t0.0\t1\t30070 597\n",
      "20\t54\t0.0\t2\t1 53\n",
      "43\t1\t0.0\t4\t0 0 0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 30794 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t224\t0.0\t1\t32 188 4\n",
      "20\t30524\t0.0\t2\t29800 717 7\n",
      "21\t43\t0.0\t2\t4 39\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1c-917-253_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1c-917-253_80_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1c-917-253_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1c-917-253_80_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1c-917-253_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1c-917-253_80_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1c-917-253_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1c-917-253_80_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        16,257 reads  @     49.1 µs/read;   1.22 M reads/minute\n",
      "Finished in 0.83 s (51 us/read; 1.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,257\n",
      "  Read 1 with adapter:                  16,237 (99.9%)\n",
      "  Read 2 with adapter:                  16,169 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        16,257 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,877,100 bp\n",
      "  Read 1:     2,438,550 bp\n",
      "  Read 2:     2,438,550 bp\n",
      "Total written (filtered):      4,245,393 bp (87.0%)\n",
      "  Read 1:     2,130,104 bp\n",
      "  Read 2:     2,115,289 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 16237 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t72\t0.0\t1\t10 62\n",
      "19\t16144\t0.0\t1\t15858 286\n",
      "20\t19\t0.0\t2\t0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 16169 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 1 2\n",
      "19\t132\t0.0\t1\t16 112 4\n",
      "20\t16016\t0.0\t2\t15626 381 9\n",
      "21\t17\t0.0\t2\t1 16\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1d-716-85_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1d-716-85_81_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1d-716-85_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1d-716-85_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1d-716-85_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT1d-716-85_81_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1d-716-85_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT1d-716-85_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        28,971 reads  @     44.9 µs/read;   1.34 M reads/minute\n",
      "Finished in 1.33 s (46 us/read; 1.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,971\n",
      "  Read 1 with adapter:                  28,922 (99.8%)\n",
      "  Read 2 with adapter:                  28,845 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,971 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,691,300 bp\n",
      "  Read 1:     4,345,650 bp\n",
      "  Read 2:     4,345,650 bp\n",
      "Total written (filtered):      7,565,175 bp (87.0%)\n",
      "  Read 1:     3,796,212 bp\n",
      "  Read 2:     3,768,963 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 28922 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t132\t0.0\t1\t20 112\n",
      "19\t28735\t0.0\t1\t28151 584\n",
      "20\t54\t0.0\t2\t3 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 28845 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t4\t0.0\t1\t0 1 3\n",
      "19\t241\t0.0\t1\t37 196 8\n",
      "20\t28560\t0.0\t2\t27800 748 12\n",
      "21\t39\t0.0\t2\t0 38 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2c-719-115_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2c-719-115_82_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2c-719-115_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2c-719-115_82_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2c-719-115_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2c-719-115_82_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2c-719-115_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2c-719-115_82_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        24,037 reads  @     47.7 µs/read;   1.26 M reads/minute\n",
      "Finished in 1.18 s (49 us/read; 1.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,037\n",
      "  Read 1 with adapter:                  24,008 (99.9%)\n",
      "  Read 2 with adapter:                  23,934 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        24,037 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,211,100 bp\n",
      "  Read 1:     3,605,550 bp\n",
      "  Read 2:     3,605,550 bp\n",
      "Total written (filtered):      6,276,572 bp (87.0%)\n",
      "  Read 1:     3,149,504 bp\n",
      "  Read 2:     3,127,068 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 24008 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t128\t0.0\t1\t14 114\n",
      "19\t23849\t0.0\t1\t23393 456\n",
      "20\t28\t0.0\t2\t1 26 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 23934 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 1 8\n",
      "19\t211\t0.0\t1\t23 184 4\n",
      "20\t23685\t0.0\t2\t23095 582 8\n",
      "21\t27\t0.0\t2\t1 26\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2c-917-227_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2c-917-227_83_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2c-917-227_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2c-917-227_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2c-917-227_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2c-917-227_83_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2c-917-227_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2c-917-227_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,510 reads  @     42.1 µs/read;   1.43 M reads/minute\n",
      "Finished in 1.40 s (43 us/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,510\n",
      "  Read 1 with adapter:                  32,457 (99.8%)\n",
      "  Read 2 with adapter:                  32,301 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,510 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,753,000 bp\n",
      "  Read 1:     4,876,500 bp\n",
      "  Read 2:     4,876,500 bp\n",
      "Total written (filtered):      8,490,684 bp (87.1%)\n",
      "  Read 1:     4,259,920 bp\n",
      "  Read 2:     4,230,764 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 32457 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t1 4\n",
      "18\t153\t0.0\t1\t14 139\n",
      "19\t32226\t0.0\t1\t31588 638\n",
      "20\t71\t0.0\t2\t2 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 32301 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t7\t0.0\t1\t0 2 5\n",
      "19\t296\t0.0\t1\t30 260 6\n",
      "20\t31947\t0.0\t2\t30958 981 8\n",
      "21\t46\t0.0\t2\t1 45\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2d-719-92_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2d-719-92_84_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2d-719-92_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2d-719-92_84_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2d-719-92_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT2d-719-92_84_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2d-719-92_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT2d-719-92_84_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        25,336 reads  @     50.4 µs/read;   1.19 M reads/minute\n",
      "Finished in 1.32 s (52 us/read; 1.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,336\n",
      "  Read 1 with adapter:                  25,294 (99.8%)\n",
      "  Read 2 with adapter:                  25,253 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,336 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,600,800 bp\n",
      "  Read 1:     3,800,400 bp\n",
      "  Read 2:     3,800,400 bp\n",
      "Total written (filtered):      6,615,475 bp (87.0%)\n",
      "  Read 1:     3,319,933 bp\n",
      "  Read 2:     3,295,542 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 25294 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t138\t0.0\t1\t17 121\n",
      "19\t25127\t0.0\t1\t24637 490\n",
      "20\t27\t0.0\t2\t2 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 25253 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t5\t0.0\t1\t0 2 3\n",
      "19\t226\t0.0\t1\t26 193 7\n",
      "20\t24985\t0.0\t2\t24286 694 5\n",
      "21\t35\t0.0\t2\t2 32 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT3c-719-117_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT3c-719-117_85_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT3c-719-117_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT3c-719-117_85_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT3c-719-117_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT3c-719-117_85_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT3c-719-117_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT3c-719-117_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        11,814 reads  @     90.9 µs/read;   0.66 M reads/minute\n",
      "Finished in 1.11 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,814\n",
      "  Read 1 with adapter:                  11,791 (99.8%)\n",
      "  Read 2 with adapter:                  11,742 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,814 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,544,200 bp\n",
      "  Read 1:     1,772,100 bp\n",
      "  Read 2:     1,772,100 bp\n",
      "Total written (filtered):      3,085,454 bp (87.1%)\n",
      "  Read 1:     1,548,112 bp\n",
      "  Read 2:     1,537,342 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 11791 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t62\t0.0\t1\t9 53\n",
      "19\t11708\t0.0\t1\t11490 218\n",
      "20\t21\t0.0\t2\t4 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 11742 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t92\t0.0\t1\t8 83 1\n",
      "20\t11637\t0.0\t2\t11207 422 8\n",
      "21\t12\t0.0\t2\t1 11\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT3d-719-96_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT3d-719-96_86_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT3d-719-96_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT3d-719-96_86_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT3d-719-96_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT3d-719-96_86_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT3d-719-96_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT3d-719-96_86_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        35,111 reads  @     48.0 µs/read;   1.25 M reads/minute\n",
      "Finished in 1.72 s (49 us/read; 1.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,111\n",
      "  Read 1 with adapter:                  35,019 (99.7%)\n",
      "  Read 2 with adapter:                  34,934 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,111 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,533,300 bp\n",
      "  Read 1:     5,266,650 bp\n",
      "  Read 2:     5,266,650 bp\n",
      "Total written (filtered):      9,169,644 bp (87.1%)\n",
      "  Read 1:     4,601,432 bp\n",
      "  Read 2:     4,568,212 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 35019 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t214\t0.0\t1\t33 181\n",
      "19\t34751\t0.0\t1\t34016 735\n",
      "20\t50\t0.0\t2\t4 46\n",
      "46\t1\t0.0\t4\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 34934 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 1 8\n",
      "19\t275\t0.0\t1\t31 233 11\n",
      "20\t34600\t0.0\t2\t33195 1380 25\n",
      "21\t49\t0.0\t2\t1 46 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT4d-719-95_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT4d-719-95_87_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT4d-719-95_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT4d-719-95_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT4d-719-95_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT4d-719-95_87_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT4d-719-95_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT4d-719-95_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        19,207 reads  @     66.4 µs/read;   0.90 M reads/minute\n",
      "Finished in 1.31 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,207\n",
      "  Read 1 with adapter:                  19,163 (99.8%)\n",
      "  Read 2 with adapter:                  19,134 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        19,207 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,762,100 bp\n",
      "  Read 1:     2,881,050 bp\n",
      "  Read 2:     2,881,050 bp\n",
      "Total written (filtered):      5,015,572 bp (87.0%)\n",
      "  Read 1:     2,517,048 bp\n",
      "  Read 2:     2,498,524 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 19163 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t102\t0.0\t1\t11 91\n",
      "19\t19029\t0.0\t1\t18690 339\n",
      "20\t27\t0.0\t2\t2 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 19134 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 1 7\n",
      "19\t167\t0.0\t1\t22 142 3\n",
      "20\t18933\t0.0\t2\t18295 626 12\n",
      "21\t23\t0.0\t2\t0 23\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT5d-719-94_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT5d-719-94_88_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT5d-719-94_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT5d-719-94_88_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT5d-719-94_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/CNAT5d-719-94_88_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT5d-719-94_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/CNAT5d-719-94_88_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        20,435 reads  @     56.0 µs/read;   1.07 M reads/minute\n",
      "Finished in 1.18 s (58 us/read; 1.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,435\n",
      "  Read 1 with adapter:                  20,392 (99.8%)\n",
      "  Read 2 with adapter:                  20,075 (98.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        20,435 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,130,500 bp\n",
      "  Read 1:     3,065,250 bp\n",
      "  Read 2:     3,065,250 bp\n",
      "Total written (filtered):      5,341,709 bp (87.1%)\n",
      "  Read 1:     2,677,901 bp\n",
      "  Read 2:     2,663,808 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 20392 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t124\t0.0\t1\t14 110\n",
      "19\t20233\t0.0\t1\t19823 410\n",
      "20\t33\t0.0\t2\t2 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 20075 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t1 0 4\n",
      "19\t180\t0.0\t1\t25 151 4\n",
      "20\t19864\t0.0\t2\t19172 679 13\n",
      "21\t22\t0.0\t2\t0 22\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "56\t3\t0.0\t4\t0 0 0 3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1c-719-73_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1c-719-73_89_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1c-719-73_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1c-719-73_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1c-719-73_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1c-719-73_89_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1c-719-73_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1c-719-73_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        19,294 reads  @     62.1 µs/read;   0.97 M reads/minute\n",
      "Finished in 1.23 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,294\n",
      "  Read 1 with adapter:                  19,255 (99.8%)\n",
      "  Read 2 with adapter:                  19,196 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        19,294 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,788,200 bp\n",
      "  Read 1:     2,894,100 bp\n",
      "  Read 2:     2,894,100 bp\n",
      "Total written (filtered):      5,038,677 bp (87.1%)\n",
      "  Read 1:     2,528,352 bp\n",
      "  Read 2:     2,510,325 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 19255 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t1\n",
      "18\t116\t0.0\t1\t20 96\n",
      "19\t19110\t0.0\t1\t18732 378\n",
      "20\t28\t0.0\t2\t2 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 19196 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "18\t6\t0.0\t1\t0 2 4\n",
      "19\t137\t0.0\t1\t19 115 3\n",
      "20\t19040\t0.0\t2\t18416 610 14\n",
      "21\t12\t0.0\t2\t0 12\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1c-917-277_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1c-917-277_90_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1c-917-277_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1c-917-277_90_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1c-917-277_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1c-917-277_90_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1c-917-277_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1c-917-277_90_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        31,889 reads  @     53.0 µs/read;   1.13 M reads/minute\n",
      "Finished in 1.72 s (54 us/read; 1.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,889\n",
      "  Read 1 with adapter:                  31,822 (99.8%)\n",
      "  Read 2 with adapter:                  31,716 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        31,889 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,566,700 bp\n",
      "  Read 1:     4,783,350 bp\n",
      "  Read 2:     4,783,350 bp\n",
      "Total written (filtered):      8,328,191 bp (87.1%)\n",
      "  Read 1:     4,178,884 bp\n",
      "  Read 2:     4,149,307 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 31822 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t183\t0.0\t1\t28 155\n",
      "19\t31595\t0.0\t1\t30905 690\n",
      "20\t41\t0.0\t2\t4 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 31716 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t2\t0.0\t1\t2\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t11\t0.0\t1\t0 4 7\n",
      "19\t287\t0.0\t1\t35 240 12\n",
      "20\t31373\t0.0\t2\t30072 1278 23\n",
      "21\t41\t0.0\t2\t3 38\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1d-716-86_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1d-716-86_91_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1d-716-86_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1d-716-86_91_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1d-716-86_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1d-716-86_91_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1d-716-86_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1d-716-86_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        13,856 reads  @     78.0 µs/read;   0.77 M reads/minute\n",
      "Finished in 1.11 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,856\n",
      "  Read 1 with adapter:                  13,829 (99.8%)\n",
      "  Read 2 with adapter:                  13,753 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        13,856 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,156,800 bp\n",
      "  Read 1:     2,078,400 bp\n",
      "  Read 2:     2,078,400 bp\n",
      "Total written (filtered):      3,619,161 bp (87.1%)\n",
      "  Read 1:     1,815,720 bp\n",
      "  Read 2:     1,803,441 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 13829 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t72\t0.0\t1\t11 61\n",
      "19\t13740\t0.0\t1\t13500 240\n",
      "20\t14\t0.0\t2\t1 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 13753 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t0 1 1\n",
      "19\t113\t0.0\t1\t12 99 2\n",
      "20\t13618\t0.0\t2\t13175 439 4\n",
      "21\t19\t0.0\t2\t0 18 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1d-918-251_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1d-918-251_92_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1d-918-251_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1d-918-251_92_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1d-918-251_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB1d-918-251_92_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1d-918-251_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB1d-918-251_92_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        11,680 reads  @     96.5 µs/read;   0.62 M reads/minute\n",
      "Finished in 1.16 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,680\n",
      "  Read 1 with adapter:                  11,655 (99.8%)\n",
      "  Read 2 with adapter:                  11,624 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,680 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,504,000 bp\n",
      "  Read 1:     1,752,000 bp\n",
      "  Read 2:     1,752,000 bp\n",
      "Total written (filtered):      3,050,204 bp (87.0%)\n",
      "  Read 1:     1,530,607 bp\n",
      "  Read 2:     1,519,597 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 11655 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t67\t0.0\t1\t13 54\n",
      "19\t11573\t0.0\t1\t11380 193\n",
      "20\t15\t0.0\t2\t1 12 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 11624 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t90\t0.0\t1\t15 75\n",
      "20\t11513\t0.0\t2\t11164 344 5\n",
      "21\t17\t0.0\t2\t0 17\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2c-719-50_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2c-719-50_93_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2c-719-50_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2c-719-50_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2c-719-50_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2c-719-50_93_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2c-719-50_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2c-719-50_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        17,730 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 1.30 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,730\n",
      "  Read 1 with adapter:                  17,693 (99.8%)\n",
      "  Read 2 with adapter:                  17,678 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        17,730 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,319,000 bp\n",
      "  Read 1:     2,659,500 bp\n",
      "  Read 2:     2,659,500 bp\n",
      "Total written (filtered):      4,629,520 bp (87.0%)\n",
      "  Read 1:     2,323,428 bp\n",
      "  Read 2:     2,306,092 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 17693 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t105\t0.0\t1\t16 89\n",
      "19\t17560\t0.0\t1\t17188 372\n",
      "20\t23\t0.0\t2\t0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 17678 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t4\t0.0\t1\t0 0 4\n",
      "19\t154\t0.0\t1\t18 133 3\n",
      "20\t17499\t0.0\t2\t16828 660 11\n",
      "21\t19\t0.0\t2\t3 16\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2c-917-212_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2c-917-212_94_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2c-917-212_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2c-917-212_94_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2c-917-212_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2c-917-212_94_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2c-917-212_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2c-917-212_94_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        11,903 reads  @     94.7 µs/read;   0.63 M reads/minute\n",
      "Finished in 1.16 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,903\n",
      "  Read 1 with adapter:                  11,873 (99.7%)\n",
      "  Read 2 with adapter:                  11,834 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,903 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,570,900 bp\n",
      "  Read 1:     1,785,450 bp\n",
      "  Read 2:     1,785,450 bp\n",
      "Total written (filtered):      3,108,787 bp (87.1%)\n",
      "  Read 1:     1,559,942 bp\n",
      "  Read 2:     1,548,845 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 11873 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t86\t0.0\t1\t11 75\n",
      "19\t11777\t0.0\t1\t11532 245\n",
      "20\t9\t0.0\t2\t0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 11834 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 1 2\n",
      "19\t80\t0.0\t1\t10 69 1\n",
      "20\t11741\t0.0\t2\t11238 492 11\n",
      "21\t9\t0.0\t2\t0 9\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2d-716-84_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2d-716-84_95_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2d-716-84_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2d-716-84_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2d-716-84_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2d-716-84_95_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2d-716-84_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2d-716-84_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        16,769 reads  @     68.3 µs/read;   0.88 M reads/minute\n",
      "Finished in 1.18 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,769\n",
      "  Read 1 with adapter:                  16,735 (99.8%)\n",
      "  Read 2 with adapter:                  16,699 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        16,769 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,030,700 bp\n",
      "  Read 1:     2,515,350 bp\n",
      "  Read 2:     2,515,350 bp\n",
      "Total written (filtered):      4,378,972 bp (87.0%)\n",
      "  Read 1:     2,197,472 bp\n",
      "  Read 2:     2,181,500 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 16735 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t101\t0.0\t1\t13 88\n",
      "19\t16606\t0.0\t1\t16260 346\n",
      "20\t24\t0.0\t2\t2 21 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 16699 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t0 1\n",
      "18\t5\t0.0\t1\t0 1 4\n",
      "19\t136\t0.0\t1\t14 119 3\n",
      "20\t16534\t0.0\t2\t15900 618 16\n",
      "21\t23\t0.0\t2\t0 23\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2d-918-247_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2d-918-247_96_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2d-918-247_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2d-918-247_96_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2d-918-247_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB2d-918-247_96_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2d-918-247_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB2d-918-247_96_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        24,227 reads  @     52.3 µs/read;   1.15 M reads/minute\n",
      "Finished in 1.30 s (54 us/read; 1.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,227\n",
      "  Read 1 with adapter:                  24,191 (99.9%)\n",
      "  Read 2 with adapter:                  24,063 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        24,227 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,268,100 bp\n",
      "  Read 1:     3,634,050 bp\n",
      "  Read 2:     3,634,050 bp\n",
      "Total written (filtered):      6,327,486 bp (87.1%)\n",
      "  Read 1:     3,174,512 bp\n",
      "  Read 2:     3,152,974 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 24191 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t130\t0.0\t1\t21 109\n",
      "19\t24016\t0.0\t1\t23525 491\n",
      "20\t43\t0.0\t2\t1 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 24063 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t4\t0.0\t1\t0 1 3\n",
      "19\t209\t0.0\t1\t35 169 5\n",
      "20\t23818\t0.0\t2\t23031 782 5\n",
      "21\t31\t0.0\t2\t1 30\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3c-719-119_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3c-719-119_97_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3c-719-119_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3c-719-119_97_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3c-719-119_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3c-719-119_97_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3c-719-119_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3c-719-119_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        10,710 reads  @    101.5 µs/read;   0.59 M reads/minute\n",
      "Finished in 1.12 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,710\n",
      "  Read 1 with adapter:                  10,679 (99.7%)\n",
      "  Read 2 with adapter:                  10,628 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        10,710 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,213,000 bp\n",
      "  Read 1:     1,606,500 bp\n",
      "  Read 2:     1,606,500 bp\n",
      "Total written (filtered):      2,797,297 bp (87.1%)\n",
      "  Read 1:     1,403,665 bp\n",
      "  Read 2:     1,393,632 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 10679 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t78\t0.0\t1\t9 69\n",
      "19\t10583\t0.0\t1\t10374 209\n",
      "20\t16\t0.0\t2\t1 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 10628 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 0 2\n",
      "19\t79\t0.0\t1\t5 74\n",
      "20\t10526\t0.0\t2\t10194 328 4\n",
      "21\t11\t0.0\t2\t0 11\n",
      "58\t10\t0.0\t4\t0 0 8 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3c-917-233_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3c-917-233_98_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3c-917-233_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3c-917-233_98_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3c-917-233_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3c-917-233_98_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3c-917-233_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3c-917-233_98_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        13,227 reads  @     73.8 µs/read;   0.81 M reads/minute\n",
      "Finished in 1.01 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,227\n",
      "  Read 1 with adapter:                  13,209 (99.9%)\n",
      "  Read 2 with adapter:                  13,190 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        13,227 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,968,100 bp\n",
      "  Read 1:     1,984,050 bp\n",
      "  Read 2:     1,984,050 bp\n",
      "Total written (filtered):      3,453,444 bp (87.0%)\n",
      "  Read 1:     1,733,100 bp\n",
      "  Read 2:     1,720,344 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 13209 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t76\t0.0\t1\t13 63\n",
      "19\t13105\t0.0\t1\t12852 253\n",
      "20\t26\t0.0\t2\t3 23\n",
      "50\t1\t0.0\t4\t0 0 0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 13190 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t4\t0.0\t1\t0 0 4\n",
      "19\t107\t0.0\t1\t11 91 5\n",
      "20\t13060\t0.0\t2\t12587 469 4\n",
      "21\t17\t0.0\t2\t1 16\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3d-716-80_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3d-716-80_99_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3d-716-80_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3d-716-80_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3d-716-80_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3d-716-80_99_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3d-716-80_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3d-716-80_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        23,974 reads  @     51.1 µs/read;   1.17 M reads/minute\n",
      "Finished in 1.26 s (52 us/read; 1.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,974\n",
      "  Read 1 with adapter:                  23,924 (99.8%)\n",
      "  Read 2 with adapter:                  23,860 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,974 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,192,200 bp\n",
      "  Read 1:     3,596,100 bp\n",
      "  Read 2:     3,596,100 bp\n",
      "Total written (filtered):      6,260,748 bp (87.0%)\n",
      "  Read 1:     3,141,657 bp\n",
      "  Read 2:     3,119,091 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 23924 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t131\t0.0\t1\t15 116\n",
      "19\t23766\t0.0\t1\t23326 440\n",
      "20\t25\t0.0\t2\t3 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 23860 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t1 2 5\n",
      "19\t200\t0.0\t1\t23 169 8\n",
      "20\t23628\t0.0\t2\t22765 839 24\n",
      "21\t23\t0.0\t2\t0 20 3\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3d-918-232_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3d-918-232_100_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3d-918-232_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3d-918-232_100_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3d-918-232_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB3d-918-232_100_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3d-918-232_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB3d-918-232_100_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,262 reads  @     52.8 µs/read;   1.14 M reads/minute\n",
      "Finished in 1.73 s (54 us/read; 1.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,262\n",
      "  Read 1 with adapter:                  32,199 (99.8%)\n",
      "  Read 2 with adapter:                  32,117 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,262 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,678,600 bp\n",
      "  Read 1:     4,839,300 bp\n",
      "  Read 2:     4,839,300 bp\n",
      "Total written (filtered):      8,424,834 bp (87.0%)\n",
      "  Read 1:     4,227,655 bp\n",
      "  Read 2:     4,197,179 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 32199 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t169\t0.0\t1\t17 152\n",
      "19\t31974\t0.0\t1\t31381 593\n",
      "20\t52\t0.0\t2\t4 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 32117 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 1 6\n",
      "19\t254\t0.0\t1\t29 220 5\n",
      "20\t31810\t0.0\t2\t30909 885 16\n",
      "21\t43\t0.0\t2\t0 42 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB4d-918-220_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB4d-918-220_101_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB4d-918-220_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB4d-918-220_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB4d-918-220_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DLAB4d-918-220_101_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB4d-918-220_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DLAB4d-918-220_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        34,528 reads  @     45.9 µs/read;   1.31 M reads/minute\n",
      "Finished in 1.62 s (47 us/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,528\n",
      "  Read 1 with adapter:                  34,452 (99.8%)\n",
      "  Read 2 with adapter:                  34,308 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,528 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,358,400 bp\n",
      "  Read 1:     5,179,200 bp\n",
      "  Read 2:     5,179,200 bp\n",
      "Total written (filtered):      9,018,009 bp (87.1%)\n",
      "  Read 1:     4,524,778 bp\n",
      "  Read 2:     4,493,231 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 34452 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t2 3\n",
      "18\t209\t0.0\t1\t27 182\n",
      "19\t34181\t0.0\t1\t33470 711\n",
      "20\t56\t0.0\t2\t7 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 34308 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t1 0 9\n",
      "19\t265\t0.0\t1\t36 223 6\n",
      "20\t33983\t0.0\t2\t32742 1221 20\n",
      "21\t46\t0.0\t2\t3 42 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t4\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO1c-719-23_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO1c-719-23_102_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO1c-719-23_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO1c-719-23_102_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO1c-719-23_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO1c-719-23_102_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO1c-719-23_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO1c-719-23_102_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        17,907 reads  @     66.7 µs/read;   0.90 M reads/minute\n",
      "Finished in 1.23 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,907\n",
      "  Read 1 with adapter:                  17,873 (99.8%)\n",
      "  Read 2 with adapter:                  17,846 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        17,907 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,372,100 bp\n",
      "  Read 1:     2,686,050 bp\n",
      "  Read 2:     2,686,050 bp\n",
      "Total written (filtered):      4,675,802 bp (87.0%)\n",
      "  Read 1:     2,346,556 bp\n",
      "  Read 2:     2,329,246 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 17873 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t114\t0.0\t1\t8 106\n",
      "19\t17729\t0.0\t1\t17382 347\n",
      "20\t27\t0.0\t2\t1 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 17846 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t128\t0.0\t1\t14 110 4\n",
      "20\t17693\t0.0\t2\t17024 655 14\n",
      "21\t21\t0.0\t2\t0 21\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2c-719-14_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2c-719-14_103_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2c-719-14_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2c-719-14_103_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2c-719-14_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2c-719-14_103_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2c-719-14_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2c-719-14_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        28,369 reads  @     48.1 µs/read;   1.25 M reads/minute\n",
      "Finished in 1.39 s (49 us/read; 1.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,369\n",
      "  Read 1 with adapter:                  28,304 (99.8%)\n",
      "  Read 2 with adapter:                  28,265 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,369 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,510,700 bp\n",
      "  Read 1:     4,255,350 bp\n",
      "  Read 2:     4,255,350 bp\n",
      "Total written (filtered):      7,407,997 bp (87.0%)\n",
      "  Read 1:     3,717,718 bp\n",
      "  Read 2:     3,690,279 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 28304 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t190\t0.0\t1\t31 159\n",
      "19\t28055\t0.0\t1\t27423 632\n",
      "20\t55\t0.0\t2\t2 52 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 28265 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t261\t0.0\t1\t34 224 3\n",
      "20\t27964\t0.0\t2\t26988 968 8\n",
      "21\t36\t0.0\t2\t0 36\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2d-716-10_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2d-716-10_104_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2d-716-10_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2d-716-10_104_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2d-716-10_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2d-716-10_104_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2d-716-10_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2d-716-10_104_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        22,659 reads  @     49.9 µs/read;   1.20 M reads/minute\n",
      "Finished in 1.16 s (51 us/read; 1.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,659\n",
      "  Read 1 with adapter:                  22,616 (99.8%)\n",
      "  Read 2 with adapter:                  22,519 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,659 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,797,700 bp\n",
      "  Read 1:     3,398,850 bp\n",
      "  Read 2:     3,398,850 bp\n",
      "Total written (filtered):      5,917,849 bp (87.1%)\n",
      "  Read 1:     2,969,220 bp\n",
      "  Read 2:     2,948,629 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 22616 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t114\t0.0\t1\t26 88\n",
      "19\t22459\t0.0\t1\t21955 504\n",
      "20\t42\t0.0\t2\t4 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 22519 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t1 0 1\n",
      "19\t174\t0.0\t1\t27 143 4\n",
      "20\t22324\t0.0\t2\t21583 729 12\n",
      "21\t19\t0.0\t2\t0 19\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2d-918-206_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2d-918-206_105_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2d-918-206_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2d-918-206_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2d-918-206_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO2d-918-206_105_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2d-918-206_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO2d-918-206_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        12,369 reads  @     89.4 µs/read;   0.67 M reads/minute\n",
      "Finished in 1.14 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,369\n",
      "  Read 1 with adapter:                  12,343 (99.8%)\n",
      "  Read 2 with adapter:                  12,262 (99.1%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        12,369 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,710,700 bp\n",
      "  Read 1:     1,855,350 bp\n",
      "  Read 2:     1,855,350 bp\n",
      "Total written (filtered):      3,230,734 bp (87.1%)\n",
      "  Read 1:     1,620,885 bp\n",
      "  Read 2:     1,609,849 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 12343 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t69\t0.0\t1\t12 57\n",
      "19\t12257\t0.0\t1\t12014 243\n",
      "20\t17\t0.0\t2\t1 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 12262 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t90\t0.0\t1\t10 78 2\n",
      "20\t12148\t0.0\t2\t11732 410 6\n",
      "21\t14\t0.0\t2\t0 14\n",
      "56\t1\t0.0\t4\t0 0 0 1\n",
      "57\t1\t0.0\t4\t0 0 1\n",
      "58\t7\t0.0\t4\t0 0 6 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO3c-719-24_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO3c-719-24_106_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO3c-719-24_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO3c-719-24_106_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO3c-719-24_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO3c-719-24_106_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO3c-719-24_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO3c-719-24_106_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        11,140 reads  @     96.9 µs/read;   0.62 M reads/minute\n",
      "Finished in 1.11 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,140\n",
      "  Read 1 with adapter:                  11,116 (99.8%)\n",
      "  Read 2 with adapter:                  11,099 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,140 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,342,000 bp\n",
      "  Read 1:     1,671,000 bp\n",
      "  Read 2:     1,671,000 bp\n",
      "Total written (filtered):      2,908,927 bp (87.0%)\n",
      "  Read 1:     1,459,842 bp\n",
      "  Read 2:     1,449,085 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 11116 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t0 1\n",
      "18\t59\t0.0\t1\t10 49\n",
      "19\t11038\t0.0\t1\t10835 203\n",
      "20\t17\t0.0\t2\t2 15\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 11099 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t75\t0.0\t1\t5 69 1\n",
      "20\t11006\t0.0\t2\t10673 327 6\n",
      "21\t14\t0.0\t2\t0 13 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO3d-716-34_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO3d-716-34_107_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO3d-716-34_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO3d-716-34_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO3d-716-34_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO3d-716-34_107_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO3d-716-34_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO3d-716-34_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        15,347 reads  @     63.1 µs/read;   0.95 M reads/minute\n",
      "Finished in 1.00 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,347\n",
      "  Read 1 with adapter:                  15,319 (99.8%)\n",
      "  Read 2 with adapter:                  15,289 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        15,347 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,604,100 bp\n",
      "  Read 1:     2,302,050 bp\n",
      "  Read 2:     2,302,050 bp\n",
      "Total written (filtered):      4,007,455 bp (87.0%)\n",
      "  Read 1:     2,011,070 bp\n",
      "  Read 2:     1,996,385 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 15319 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t97\t0.0\t1\t14 83\n",
      "19\t15200\t0.0\t1\t14890 310\n",
      "20\t20\t0.0\t2\t0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 15289 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 0 2\n",
      "19\t136\t0.0\t1\t16 116 4\n",
      "20\t15126\t0.0\t2\t14690 433 3\n",
      "21\t25\t0.0\t2\t0 24 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO4c-719-97_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO4c-719-97_108_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO4c-719-97_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO4c-719-97_108_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO4c-719-97_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO4c-719-97_108_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO4c-719-97_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO4c-719-97_108_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        10,930 reads  @     95.7 µs/read;   0.63 M reads/minute\n",
      "Finished in 1.08 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,930\n",
      "  Read 1 with adapter:                  10,908 (99.8%)\n",
      "  Read 2 with adapter:                  10,884 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        10,930 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,279,000 bp\n",
      "  Read 1:     1,639,500 bp\n",
      "  Read 2:     1,639,500 bp\n",
      "Total written (filtered):      2,854,188 bp (87.0%)\n",
      "  Read 1:     1,432,305 bp\n",
      "  Read 2:     1,421,883 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 10908 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t72\t0.0\t1\t7 65\n",
      "19\t10818\t0.0\t1\t10590 228\n",
      "20\t17\t0.0\t2\t3 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 10884 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t79\t0.0\t1\t13 65 1\n",
      "20\t10779\t0.0\t2\t10434 337 8\n",
      "21\t21\t0.0\t2\t0 21\n",
      "25\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO4d-716-125_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO4d-716-125_109_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO4d-716-125_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO4d-716-125_109_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO4d-716-125_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO4d-716-125_109_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO4d-716-125_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO4d-716-125_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        23,927 reads  @     46.3 µs/read;   1.30 M reads/minute\n",
      "Finished in 1.14 s (48 us/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,927\n",
      "  Read 1 with adapter:                  23,887 (99.8%)\n",
      "  Read 2 with adapter:                  23,816 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,927 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,178,100 bp\n",
      "  Read 1:     3,589,050 bp\n",
      "  Read 2:     3,589,050 bp\n",
      "Total written (filtered):      6,248,161 bp (87.0%)\n",
      "  Read 1:     3,135,258 bp\n",
      "  Read 2:     3,112,903 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 23887 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t113\t0.0\t1\t14 99\n",
      "19\t23722\t0.0\t1\t23262 460\n",
      "20\t52\t0.0\t2\t2 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 23816 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t2 0 3\n",
      "19\t196\t0.0\t1\t20 169 7\n",
      "20\t23582\t0.0\t2\t22999 567 16\n",
      "21\t33\t0.0\t2\t3 30\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO5c-719-90_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO5c-719-90_110_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO5c-719-90_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO5c-719-90_110_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO5c-719-90_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO5c-719-90_110_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO5c-719-90_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO5c-719-90_110_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        13,265 reads  @     78.0 µs/read;   0.77 M reads/minute\n",
      "Finished in 1.07 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,265\n",
      "  Read 1 with adapter:                  13,234 (99.8%)\n",
      "  Read 2 with adapter:                  13,215 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        13,265 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,979,500 bp\n",
      "  Read 1:     1,989,750 bp\n",
      "  Read 2:     1,989,750 bp\n",
      "Total written (filtered):      3,463,904 bp (87.0%)\n",
      "  Read 1:     1,738,369 bp\n",
      "  Read 2:     1,725,535 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 13234 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t78\t0.0\t1\t7 71\n",
      "19\t13140\t0.0\t1\t12874 266\n",
      "20\t15\t0.0\t2\t3 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 13215 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t92\t0.0\t1\t14 76 2\n",
      "20\t13107\t0.0\t2\t12668 438 1\n",
      "21\t13\t0.0\t2\t0 12 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO5d-716-30_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO5d-716-30_111_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO5d-716-30_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO5d-716-30_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO5d-716-30_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/DSTO5d-716-30_111_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO5d-716-30_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/DSTO5d-716-30_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        25,451 reads  @     47.1 µs/read;   1.27 M reads/minute\n",
      "Finished in 1.24 s (49 us/read; 1.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,451\n",
      "  Read 1 with adapter:                  25,401 (99.8%)\n",
      "  Read 2 with adapter:                  25,345 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,451 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,635,300 bp\n",
      "  Read 1:     3,817,650 bp\n",
      "  Read 2:     3,817,650 bp\n",
      "Total written (filtered):      6,646,069 bp (87.0%)\n",
      "  Read 1:     3,335,117 bp\n",
      "  Read 2:     3,310,952 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 25401 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t115\t0.0\t1\t13 102\n",
      "19\t25248\t0.0\t1\t24736 512\n",
      "20\t35\t0.0\t2\t2 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 25345 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t2\t0.0\t1\t2\n",
      "18\t7\t0.0\t1\t0 1 6\n",
      "19\t219\t0.0\t1\t29 186 4\n",
      "20\t25076\t0.0\t2\t24374 695 7\n",
      "21\t41\t0.0\t2\t1 39 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1c-917-237_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1c-917-237_112_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1c-917-237_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1c-917-237_112_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1c-917-237_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1c-917-237_112_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1c-917-237_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1c-917-237_112_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        35,320 reads  @     40.0 µs/read;   1.50 M reads/minute\n",
      "Finished in 1.44 s (41 us/read; 1.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,320\n",
      "  Read 1 with adapter:                  35,258 (99.8%)\n",
      "  Read 2 with adapter:                  35,139 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,320 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,596,000 bp\n",
      "  Read 1:     5,298,000 bp\n",
      "  Read 2:     5,298,000 bp\n",
      "Total written (filtered):      9,223,774 bp (87.0%)\n",
      "  Read 1:     4,628,224 bp\n",
      "  Read 2:     4,595,550 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 35258 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t182\t0.0\t1\t28 154\n",
      "19\t35014\t0.0\t1\t34277 737\n",
      "20\t60\t0.0\t2\t5 54 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 35139 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t11\t0.0\t1\t1 1 9\n",
      "19\t349\t0.0\t1\t38 303 8\n",
      "20\t34720\t0.0\t2\t33522 1177 21\n",
      "21\t50\t0.0\t2\t2 48\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "23\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1c-918-249_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1c-918-249_113_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1c-918-249_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1c-918-249_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1c-918-249_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1c-918-249_113_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1c-918-249_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1c-918-249_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        16,830 reads  @     65.1 µs/read;   0.92 M reads/minute\n",
      "Finished in 1.13 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,830\n",
      "  Read 1 with adapter:                  16,803 (99.8%)\n",
      "  Read 2 with adapter:                  16,766 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        16,830 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,049,000 bp\n",
      "  Read 1:     2,524,500 bp\n",
      "  Read 2:     2,524,500 bp\n",
      "Total written (filtered):      4,394,584 bp (87.0%)\n",
      "  Read 1:     2,205,303 bp\n",
      "  Read 2:     2,189,281 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 16803 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t88\t0.0\t1\t9 79\n",
      "19\t16678\t0.0\t1\t16364 314\n",
      "20\t35\t0.0\t2\t0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 16766 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t128\t0.0\t1\t12 113 3\n",
      "20\t16603\t0.0\t2\t16114 482 7\n",
      "21\t31\t0.0\t2\t1 27 3\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1d-716-33_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1d-716-33_114_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1d-716-33_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1d-716-33_114_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1d-716-33_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA1d-716-33_114_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1d-716-33_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA1d-716-33_114_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        21,318 reads  @     54.9 µs/read;   1.09 M reads/minute\n",
      "Finished in 1.20 s (56 us/read; 1.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,318\n",
      "  Read 1 with adapter:                  21,279 (99.8%)\n",
      "  Read 2 with adapter:                  21,237 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        21,318 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,395,400 bp\n",
      "  Read 1:     3,197,700 bp\n",
      "  Read 2:     3,197,700 bp\n",
      "Total written (filtered):      5,566,581 bp (87.0%)\n",
      "  Read 1:     2,793,485 bp\n",
      "  Read 2:     2,773,096 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 21279 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t110\t0.0\t1\t8 102\n",
      "19\t21129\t0.0\t1\t20746 383\n",
      "20\t35\t0.0\t2\t3 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 21237 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t1 0 5\n",
      "19\t152\t0.0\t1\t18 131 3\n",
      "20\t21051\t0.0\t2\t20483 552 16\n",
      "21\t28\t0.0\t2\t1 27\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA2c-918-204_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA2c-918-204_115_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA2c-918-204_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA2c-918-204_115_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA2c-918-204_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA2c-918-204_115_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA2c-918-204_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA2c-918-204_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        26,755 reads  @     45.6 µs/read;   1.31 M reads/minute\n",
      "Finished in 1.25 s (47 us/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,755\n",
      "  Read 1 with adapter:                  26,717 (99.9%)\n",
      "  Read 2 with adapter:                  26,639 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        26,755 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,026,500 bp\n",
      "  Read 1:     4,013,250 bp\n",
      "  Read 2:     4,013,250 bp\n",
      "Total written (filtered):      6,986,404 bp (87.0%)\n",
      "  Read 1:     3,505,729 bp\n",
      "  Read 2:     3,480,675 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 26717 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t138\t0.0\t1\t21 117\n",
      "19\t26537\t0.0\t1\t26048 489\n",
      "20\t40\t0.0\t2\t1 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 26639 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t2 1 6\n",
      "19\t229\t0.0\t1\t24 201 4\n",
      "20\t26360\t0.0\t2\t25603 744 13\n",
      "21\t40\t0.0\t2\t1 39\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA2d-716-124_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA2d-716-124_116_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA2d-716-124_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA2d-716-124_116_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA2d-716-124_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA2d-716-124_116_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA2d-716-124_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA2d-716-124_116_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        14,198 reads  @     77.0 µs/read;   0.78 M reads/minute\n",
      "Finished in 1.12 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,198\n",
      "  Read 1 with adapter:                  14,176 (99.8%)\n",
      "  Read 2 with adapter:                  14,116 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        14,198 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,259,400 bp\n",
      "  Read 1:     2,129,700 bp\n",
      "  Read 2:     2,129,700 bp\n",
      "Total written (filtered):      3,707,925 bp (87.1%)\n",
      "  Read 1:     1,860,440 bp\n",
      "  Read 2:     1,847,485 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 14176 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t2 1\n",
      "18\t92\t0.0\t1\t11 81\n",
      "19\t14063\t0.0\t1\t13799 264\n",
      "20\t17\t0.0\t2\t0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 14116 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t115\t0.0\t1\t17 96 2\n",
      "20\t13977\t0.0\t2\t13615 358 4\n",
      "21\t18\t0.0\t2\t2 16\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA3c-917-274_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA3c-917-274_117_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA3c-917-274_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA3c-917-274_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA3c-917-274_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA3c-917-274_117_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA3c-917-274_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA3c-917-274_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        18,123 reads  @     61.7 µs/read;   0.97 M reads/minute\n",
      "Finished in 1.15 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,123\n",
      "  Read 1 with adapter:                  18,096 (99.9%)\n",
      "  Read 2 with adapter:                  18,058 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        18,123 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,436,900 bp\n",
      "  Read 1:     2,718,450 bp\n",
      "  Read 2:     2,718,450 bp\n",
      "Total written (filtered):      4,732,085 bp (87.0%)\n",
      "  Read 1:     2,374,692 bp\n",
      "  Read 2:     2,357,393 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 18096 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t100\t0.0\t1\t19 81\n",
      "19\t17956\t0.0\t1\t17614 342\n",
      "20\t38\t0.0\t2\t4 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 18058 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 0 2\n",
      "19\t130\t0.0\t1\t25 104 1\n",
      "20\t17896\t0.0\t2\t17466 423 7\n",
      "21\t29\t0.0\t2\t2 26 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA3d-716-123_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA3d-716-123_118_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA3d-716-123_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA3d-716-123_118_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA3d-716-123_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA3d-716-123_118_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA3d-716-123_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA3d-716-123_118_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        19,538 reads  @     63.5 µs/read;   0.95 M reads/minute\n",
      "Finished in 1.27 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,538\n",
      "  Read 1 with adapter:                  19,511 (99.9%)\n",
      "  Read 2 with adapter:                  19,373 (99.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        19,538 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,861,400 bp\n",
      "  Read 1:     2,930,700 bp\n",
      "  Read 2:     2,930,700 bp\n",
      "Total written (filtered):      5,103,448 bp (87.1%)\n",
      "  Read 1:     2,560,060 bp\n",
      "  Read 2:     2,543,388 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 19511 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t100\t0.0\t1\t14 86\n",
      "19\t19374\t0.0\t1\t18977 397\n",
      "20\t35\t0.0\t2\t2 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 19373 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t178\t0.0\t1\t23 148 7\n",
      "20\t19162\t0.0\t2\t18639 514 9\n",
      "21\t32\t0.0\t2\t1 31\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4Ad-716-82_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4Ad-716-82_119_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4Ad-716-82_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4Ad-716-82_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4Ad-716-82_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4Ad-716-82_119_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4Ad-716-82_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4Ad-716-82_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        11,904 reads  @     82.4 µs/read;   0.73 M reads/minute\n",
      "Finished in 1.01 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,904\n",
      "  Read 1 with adapter:                  11,878 (99.8%)\n",
      "  Read 2 with adapter:                  11,834 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,904 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,571,200 bp\n",
      "  Read 1:     1,785,600 bp\n",
      "  Read 2:     1,785,600 bp\n",
      "Total written (filtered):      3,108,951 bp (87.1%)\n",
      "  Read 1:     1,559,962 bp\n",
      "  Read 2:     1,548,989 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 11878 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t60\t0.0\t1\t8 52\n",
      "19\t11799\t0.0\t1\t11551 248\n",
      "20\t18\t0.0\t2\t2 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 11834 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t4\t0.0\t1\t0 1 3\n",
      "19\t80\t0.0\t1\t11 67 2\n",
      "20\t11731\t0.0\t2\t11369 356 6\n",
      "21\t19\t0.0\t2\t1 18\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4Bd-716-87_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4Bd-716-87_120_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4Bd-716-87_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4Bd-716-87_120_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4Bd-716-87_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4Bd-716-87_120_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4Bd-716-87_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4Bd-716-87_120_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        11,391 reads  @     75.2 µs/read;   0.80 M reads/minute\n",
      "Finished in 0.89 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,391\n",
      "  Read 1 with adapter:                  11,367 (99.8%)\n",
      "  Read 2 with adapter:                  11,351 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,391 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,417,300 bp\n",
      "  Read 1:     1,708,650 bp\n",
      "  Read 2:     1,708,650 bp\n",
      "Total written (filtered):      2,974,428 bp (87.0%)\n",
      "  Read 1:     1,492,717 bp\n",
      "  Read 2:     1,481,711 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 11367 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t58\t0.0\t1\t12 46\n",
      "19\t11285\t0.0\t1\t11057 227 1\n",
      "20\t22\t0.0\t2\t3 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 11351 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t4\t0.0\t1\t0 0 4\n",
      "19\t98\t0.0\t1\t6 90 2\n",
      "20\t11226\t0.0\t2\t10861 359 6\n",
      "21\t21\t0.0\t2\t2 18 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4c-917-264_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4c-917-264_121_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4c-917-264_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4c-917-264_121_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4c-917-264_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA4c-917-264_121_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4c-917-264_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA4c-917-264_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        25,281 reads  @     39.5 µs/read;   1.52 M reads/minute\n",
      "Finished in 1.03 s (41 us/read; 1.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,281\n",
      "  Read 1 with adapter:                  25,232 (99.8%)\n",
      "  Read 2 with adapter:                  25,197 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,281 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,584,300 bp\n",
      "  Read 1:     3,792,150 bp\n",
      "  Read 2:     3,792,150 bp\n",
      "Total written (filtered):      6,601,238 bp (87.0%)\n",
      "  Read 1:     3,312,852 bp\n",
      "  Read 2:     3,288,386 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 25232 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t140\t0.0\t1\t21 119\n",
      "19\t25053\t0.0\t1\t24590 463\n",
      "20\t36\t0.0\t2\t1 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 25197 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t0 1\n",
      "18\t7\t0.0\t1\t1 1 5\n",
      "19\t188\t0.0\t1\t24 159 5\n",
      "20\t24970\t0.0\t2\t24316 649 5\n",
      "21\t31\t0.0\t2\t1 29 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA5d-719-93_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA5d-719-93_122_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA5d-719-93_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA5d-719-93_122_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA5d-719-93_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/MMEA5d-719-93_122_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA5d-719-93_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/MMEA5d-719-93_122_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        13,210 reads  @     65.1 µs/read;   0.92 M reads/minute\n",
      "Finished in 0.89 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,210\n",
      "  Read 1 with adapter:                  13,188 (99.8%)\n",
      "  Read 2 with adapter:                  13,157 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        13,210 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,963,000 bp\n",
      "  Read 1:     1,981,500 bp\n",
      "  Read 2:     1,981,500 bp\n",
      "Total written (filtered):      3,449,425 bp (87.0%)\n",
      "  Read 1:     1,730,982 bp\n",
      "  Read 2:     1,718,443 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 13188 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t71\t0.0\t1\t11 60\n",
      "19\t13097\t0.0\t1\t12836 261\n",
      "20\t19\t0.0\t2\t4 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 13157 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t96\t0.0\t1\t8 86 2\n",
      "20\t13039\t0.0\t2\t12677 357 5\n",
      "21\t19\t0.0\t2\t1 18\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1c-719-110_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1c-719-110_123_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1c-719-110_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1c-719-110_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1c-719-110_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1c-719-110_123_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1c-719-110_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1c-719-110_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        37,287 reads  @     51.2 µs/read;   1.17 M reads/minute\n",
      "Finished in 1.94 s (52 us/read; 1.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,287\n",
      "  Read 1 with adapter:                  37,222 (99.8%)\n",
      "  Read 2 with adapter:                  37,056 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        37,287 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,186,100 bp\n",
      "  Read 1:     5,593,050 bp\n",
      "  Read 2:     5,593,050 bp\n",
      "Total written (filtered):      9,738,194 bp (87.1%)\n",
      "  Read 1:     4,885,993 bp\n",
      "  Read 2:     4,852,201 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 37222 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t201\t0.0\t1\t29 172\n",
      "19\t36960\t0.0\t1\t36210 749 1\n",
      "20\t56\t0.0\t2\t6 49 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 37056 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t1 1 8\n",
      "19\t313\t0.0\t1\t33 278 2\n",
      "20\t36678\t0.0\t2\t35310 1352 16\n",
      "21\t50\t0.0\t2\t0 49 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "24\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1c-917-271_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1c-917-271_124_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1c-917-271_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1c-917-271_124_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1c-917-271_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1c-917-271_124_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1c-917-271_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1c-917-271_124_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        15,270 reads  @     67.9 µs/read;   0.88 M reads/minute\n",
      "Finished in 1.07 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,270\n",
      "  Read 1 with adapter:                  15,245 (99.8%)\n",
      "  Read 2 with adapter:                  15,204 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        15,270 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,581,000 bp\n",
      "  Read 1:     2,290,500 bp\n",
      "  Read 2:     2,290,500 bp\n",
      "Total written (filtered):      3,987,488 bp (87.0%)\n",
      "  Read 1:     2,000,939 bp\n",
      "  Read 2:     1,986,549 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 15245 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t113\t0.0\t1\t16 97\n",
      "19\t15107\t0.0\t1\t14794 313\n",
      "20\t23\t0.0\t2\t1 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 15204 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t4\t0.0\t1\t0 0 4\n",
      "19\t144\t0.0\t1\t16 126 2\n",
      "20\t15033\t0.0\t2\t14479 544 10\n",
      "21\t23\t0.0\t2\t3 20\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1d-716-78_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1d-716-78_125_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1d-716-78_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1d-716-78_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1d-716-78_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1d-716-78_125_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1d-716-78_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1d-716-78_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        11,337 reads  @     93.0 µs/read;   0.65 M reads/minute\n",
      "Finished in 1.09 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,337\n",
      "  Read 1 with adapter:                  11,324 (99.9%)\n",
      "  Read 2 with adapter:                  11,064 (97.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,337 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,401,100 bp\n",
      "  Read 1:     1,700,550 bp\n",
      "  Read 2:     1,700,550 bp\n",
      "Total written (filtered):      2,964,180 bp (87.2%)\n",
      "  Read 1:     1,485,438 bp\n",
      "  Read 2:     1,478,742 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 11324 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t60\t0.0\t1\t3 57\n",
      "19\t11248\t0.0\t1\t11043 205\n",
      "20\t16\t0.0\t2\t3 12 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 11064 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t4\t0.0\t1\t0 0 4\n",
      "19\t82\t0.0\t1\t13 67 2\n",
      "20\t10950\t0.0\t2\t10597 346 7\n",
      "21\t10\t0.0\t2\t1 9\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "55\t6\t0.0\t4\t0 0 0 0 6\n",
      "56\t11\t0.0\t4\t0 0 0 2 9\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1d-918-266_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1d-918-266_126_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1d-918-266_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1d-918-266_126_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1d-918-266_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN1d-918-266_126_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1d-918-266_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN1d-918-266_126_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        11,629 reads  @     95.9 µs/read;   0.63 M reads/minute\n",
      "Finished in 1.15 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,629\n",
      "  Read 1 with adapter:                  11,604 (99.8%)\n",
      "  Read 2 with adapter:                  11,575 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,629 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,488,700 bp\n",
      "  Read 1:     1,744,350 bp\n",
      "  Read 2:     1,744,350 bp\n",
      "Total written (filtered):      3,036,845 bp (87.0%)\n",
      "  Read 1:     1,523,919 bp\n",
      "  Read 2:     1,512,926 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 11604 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t59\t0.0\t1\t6 53\n",
      "19\t11528\t0.0\t1\t11279 249\n",
      "20\t16\t0.0\t2\t1 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 11575 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 1 4\n",
      "19\t91\t0.0\t1\t14 74 3\n",
      "20\t11456\t0.0\t2\t10997 449 10\n",
      "21\t21\t0.0\t2\t0 20 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN2c-917-280_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN2c-917-280_127_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN2c-917-280_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN2c-917-280_127_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN2c-917-280_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN2c-917-280_127_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN2c-917-280_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN2c-917-280_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        23,544 reads  @     52.8 µs/read;   1.14 M reads/minute\n",
      "Finished in 1.28 s (54 us/read; 1.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,544\n",
      "  Read 1 with adapter:                  23,500 (99.8%)\n",
      "  Read 2 with adapter:                  23,437 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,544 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,063,200 bp\n",
      "  Read 1:     3,531,600 bp\n",
      "  Read 2:     3,531,600 bp\n",
      "Total written (filtered):      6,148,224 bp (87.0%)\n",
      "  Read 1:     3,085,199 bp\n",
      "  Read 2:     3,063,025 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 23500 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t0 2\n",
      "18\t123\t0.0\t1\t16 107\n",
      "19\t23344\t0.0\t1\t22874 470\n",
      "20\t29\t0.0\t2\t1 28\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 23437 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t4\t0.0\t1\t0 0 4\n",
      "19\t174\t0.0\t1\t19 147 8\n",
      "20\t23238\t0.0\t2\t22364 855 19\n",
      "21\t20\t0.0\t2\t5 15\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN2d-918-241_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN2d-918-241_128_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN2d-918-241_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN2d-918-241_128_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN2d-918-241_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN2d-918-241_128_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN2d-918-241_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN2d-918-241_128_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        11,482 reads  @     83.5 µs/read;   0.72 M reads/minute\n",
      "Finished in 1.00 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,482\n",
      "  Read 1 with adapter:                  11,462 (99.8%)\n",
      "  Read 2 with adapter:                  11,440 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,482 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,444,600 bp\n",
      "  Read 1:     1,722,300 bp\n",
      "  Read 2:     1,722,300 bp\n",
      "Total written (filtered):      2,998,177 bp (87.0%)\n",
      "  Read 1:     1,504,575 bp\n",
      "  Read 2:     1,493,602 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 11462 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t1\n",
      "18\t64\t0.0\t1\t10 54\n",
      "19\t11380\t0.0\t1\t11150 230\n",
      "20\t17\t0.0\t2\t2 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 11440 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t1 1 3\n",
      "19\t103\t0.0\t1\t10 88 5\n",
      "20\t11321\t0.0\t2\t10961 352 8\n",
      "21\t11\t0.0\t2\t0 11\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3c-719-120_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3c-719-120_129_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3c-719-120_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3c-719-120_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3c-719-120_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3c-719-120_129_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3c-719-120_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3c-719-120_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        30,315 reads  @     46.0 µs/read;   1.31 M reads/minute\n",
      "Finished in 1.43 s (47 us/read; 1.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,315\n",
      "  Read 1 with adapter:                  30,256 (99.8%)\n",
      "  Read 2 with adapter:                  30,112 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,315 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,094,500 bp\n",
      "  Read 1:     4,547,250 bp\n",
      "  Read 2:     4,547,250 bp\n",
      "Total written (filtered):      7,917,683 bp (87.1%)\n",
      "  Read 1:     3,972,533 bp\n",
      "  Read 2:     3,945,150 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 30256 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t200\t0.0\t1\t28 172\n",
      "19\t30003\t0.0\t1\t29402 601\n",
      "20\t53\t0.0\t2\t4 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 30112 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t252\t0.0\t1\t34 210 8\n",
      "20\t29813\t0.0\t2\t28788 1008 17\n",
      "21\t37\t0.0\t2\t2 34 1\n",
      "22\t2\t0.0\t2\t1 0 1\n",
      "28\t1\t0.0\t2\t1\n",
      "56\t1\t0.0\t4\t0 0 0 1\n",
      "57\t1\t0.0\t4\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3c-917-254_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3c-917-254_130_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3c-917-254_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3c-917-254_130_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3c-917-254_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3c-917-254_130_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3c-917-254_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3c-917-254_130_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        14,309 reads  @     65.3 µs/read;   0.92 M reads/minute\n",
      "Finished in 0.97 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,309\n",
      "  Read 1 with adapter:                  14,281 (99.8%)\n",
      "  Read 2 with adapter:                  14,236 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        14,309 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,292,700 bp\n",
      "  Read 1:     2,146,350 bp\n",
      "  Read 2:     2,146,350 bp\n",
      "Total written (filtered):      3,736,809 bp (87.1%)\n",
      "  Read 1:     1,875,078 bp\n",
      "  Read 2:     1,861,731 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 14281 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t76\t0.0\t1\t8 68\n",
      "19\t14190\t0.0\t1\t13928 262\n",
      "20\t13\t0.0\t2\t0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 14236 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t4\t0.0\t1\t0 0 4\n",
      "19\t117\t0.0\t1\t19 96 2\n",
      "20\t14092\t0.0\t2\t13504 576 12\n",
      "21\t22\t0.0\t2\t1 21\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3d-918-235_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3d-918-235_131_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3d-918-235_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3d-918-235_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3d-918-235_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN3d-918-235_131_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3d-918-235_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN3d-918-235_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        34,096 reads  @     55.3 µs/read;   1.08 M reads/minute\n",
      "Finished in 1.92 s (56 us/read; 1.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,096\n",
      "  Read 1 with adapter:                  34,037 (99.8%)\n",
      "  Read 2 with adapter:                  33,886 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,096 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,228,800 bp\n",
      "  Read 1:     5,114,400 bp\n",
      "  Read 2:     5,114,400 bp\n",
      "Total written (filtered):      8,904,784 bp (87.1%)\n",
      "  Read 1:     4,467,836 bp\n",
      "  Read 2:     4,436,948 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 34037 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t194\t0.0\t1\t27 167\n",
      "19\t33778\t0.0\t1\t33107 671\n",
      "20\t63\t0.0\t2\t4 58 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 33886 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t9\t0.0\t1\t0 1 8\n",
      "19\t308\t0.0\t1\t31 265 12\n",
      "20\t33507\t0.0\t2\t32379 1118 10\n",
      "21\t61\t0.0\t2\t7 54\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN4d-918-260_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN4d-918-260_132_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN4d-918-260_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN4d-918-260_132_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN4d-918-260_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OANN4d-918-260_132_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN4d-918-260_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OANN4d-918-260_132_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:02        44,564 reads  @     45.1 µs/read;   1.33 M reads/minute\n",
      "Finished in 2.04 s (46 us/read; 1.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,564\n",
      "  Read 1 with adapter:                  44,475 (99.8%)\n",
      "  Read 2 with adapter:                  44,340 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,564 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,369,200 bp\n",
      "  Read 1:     6,684,600 bp\n",
      "  Read 2:     6,684,600 bp\n",
      "Total written (filtered):     11,637,859 bp (87.0%)\n",
      "  Read 1:     5,839,769 bp\n",
      "  Read 2:     5,798,090 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 44475 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t5\t0.0\t1\t1 4\n",
      "18\t256\t0.0\t1\t35 221\n",
      "19\t44142\t0.0\t1\t43129 1013\n",
      "20\t72\t0.0\t2\t9 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 44340 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t1 1 7\n",
      "19\t337\t0.0\t1\t42 287 8\n",
      "20\t43931\t0.0\t2\t42260 1651 20\n",
      "21\t61\t0.0\t2\t1 59 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1c-719-11_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1c-719-11_133_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1c-719-11_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1c-719-11_133_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1c-719-11_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1c-719-11_133_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1c-719-11_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1c-719-11_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        17,285 reads  @     65.6 µs/read;   0.91 M reads/minute\n",
      "Finished in 1.17 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,285\n",
      "  Read 1 with adapter:                  17,250 (99.8%)\n",
      "  Read 2 with adapter:                  17,204 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        17,285 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,185,500 bp\n",
      "  Read 1:     2,592,750 bp\n",
      "  Read 2:     2,592,750 bp\n",
      "Total written (filtered):      4,513,885 bp (87.0%)\n",
      "  Read 1:     2,265,074 bp\n",
      "  Read 2:     2,248,811 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 17250 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t95\t0.0\t1\t12 83\n",
      "19\t17134\t0.0\t1\t16819 315\n",
      "20\t21\t0.0\t2\t2 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 17204 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t155\t0.0\t1\t21 129 5\n",
      "20\t17021\t0.0\t2\t16529 479 13\n",
      "21\t22\t0.0\t2\t0 21 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1d-716-83_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1d-716-83_134_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1d-716-83_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1d-716-83_134_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1d-716-83_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1d-716-83_134_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1d-716-83_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1d-716-83_134_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        29,279 reads  @     44.3 µs/read;   1.35 M reads/minute\n",
      "Finished in 1.33 s (46 us/read; 1.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,279\n",
      "  Read 1 with adapter:                  29,232 (99.8%)\n",
      "  Read 2 with adapter:                  29,154 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,279 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,783,700 bp\n",
      "  Read 1:     4,391,850 bp\n",
      "  Read 2:     4,391,850 bp\n",
      "Total written (filtered):      7,645,552 bp (87.0%)\n",
      "  Read 1:     3,836,558 bp\n",
      "  Read 2:     3,808,994 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 29232 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t5\t0.0\t1\t5\n",
      "18\t151\t0.0\t1\t20 131\n",
      "19\t29026\t0.0\t1\t28429 597\n",
      "20\t46\t0.0\t2\t5 41\n",
      "28\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 29154 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t1 2 3\n",
      "19\t250\t0.0\t1\t23 221 6\n",
      "20\t28863\t0.0\t2\t27922 924 17\n",
      "21\t32\t0.0\t2\t0 32\n",
      "22\t3\t0.0\t2\t0 1 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1d-918-261_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1d-918-261_135_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1d-918-261_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1d-918-261_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1d-918-261_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV1d-918-261_135_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1d-918-261_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV1d-918-261_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        14,132 reads  @     73.2 µs/read;   0.82 M reads/minute\n",
      "Finished in 1.07 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,132\n",
      "  Read 1 with adapter:                  14,118 (99.9%)\n",
      "  Read 2 with adapter:                  14,072 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        14,132 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,239,600 bp\n",
      "  Read 1:     2,119,800 bp\n",
      "  Read 2:     2,119,800 bp\n",
      "Total written (filtered):      3,690,087 bp (87.0%)\n",
      "  Read 1:     1,851,625 bp\n",
      "  Read 2:     1,838,462 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 14118 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t86\t0.0\t1\t10 76\n",
      "19\t14013\t0.0\t1\t13733 280\n",
      "20\t19\t0.0\t2\t1 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 14072 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t4\t0.0\t1\t0 1 3\n",
      "19\t116\t0.0\t1\t15 100 1\n",
      "20\t13930\t0.0\t2\t13472 450 8\n",
      "21\t22\t0.0\t2\t1 21\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV2c-719-6_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV2c-719-6_136_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV2c-719-6_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV2c-719-6_136_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV2c-719-6_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV2c-719-6_136_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV2c-719-6_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV2c-719-6_136_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        25,543 reads  @     48.5 µs/read;   1.24 M reads/minute\n",
      "Finished in 1.28 s (50 us/read; 1.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,543\n",
      "  Read 1 with adapter:                  25,502 (99.8%)\n",
      "  Read 2 with adapter:                  25,358 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,543 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,662,900 bp\n",
      "  Read 1:     3,831,450 bp\n",
      "  Read 2:     3,831,450 bp\n",
      "Total written (filtered):      6,671,522 bp (87.1%)\n",
      "  Read 1:     3,346,989 bp\n",
      "  Read 2:     3,324,533 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 25502 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t119\t0.0\t1\t22 97\n",
      "19\t25337\t0.0\t1\t24837 499 1\n",
      "20\t43\t0.0\t2\t5 38\n",
      "22\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 25358 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "18\t8\t0.0\t1\t0 1 7\n",
      "19\t267\t0.0\t1\t34 229 4\n",
      "20\t25037\t0.0\t2\t24392 629 16\n",
      "21\t45\t0.0\t2\t2 41 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV3c-719-12_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV3c-719-12_137_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV3c-719-12_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV3c-719-12_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV3c-719-12_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV3c-719-12_137_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV3c-719-12_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV3c-719-12_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,235 reads  @     46.4 µs/read;   1.29 M reads/minute\n",
      "Finished in 1.53 s (47 us/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,235\n",
      "  Read 1 with adapter:                  32,186 (99.8%)\n",
      "  Read 2 with adapter:                  32,022 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,235 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,670,500 bp\n",
      "  Read 1:     4,835,250 bp\n",
      "  Read 2:     4,835,250 bp\n",
      "Total written (filtered):      8,418,880 bp (87.1%)\n",
      "  Read 1:     4,223,823 bp\n",
      "  Read 2:     4,195,057 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 32186 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t153\t0.0\t1\t23 130\n",
      "19\t31979\t0.0\t1\t31341 638\n",
      "20\t52\t0.0\t2\t3 48 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 32022 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "18\t7\t0.0\t1\t1 0 6\n",
      "19\t291\t0.0\t1\t37 242 12\n",
      "20\t31670\t0.0\t2\t30728 930 12\n",
      "21\t48\t0.0\t2\t2 46\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV4c-719-35_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV4c-719-35_138_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV4c-719-35_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV4c-719-35_138_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV4c-719-35_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV4c-719-35_138_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV4c-719-35_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV4c-719-35_138_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        21,549 reads  @     52.3 µs/read;   1.15 M reads/minute\n",
      "Finished in 1.16 s (54 us/read; 1.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,549\n",
      "  Read 1 with adapter:                  21,515 (99.8%)\n",
      "  Read 2 with adapter:                  21,454 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        21,549 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,464,700 bp\n",
      "  Read 1:     3,232,350 bp\n",
      "  Read 2:     3,232,350 bp\n",
      "Total written (filtered):      5,627,022 bp (87.0%)\n",
      "  Read 1:     2,823,636 bp\n",
      "  Read 2:     2,803,386 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 21515 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "18\t114\t0.0\t1\t6 108\n",
      "19\t21353\t0.0\t1\t20922 431\n",
      "20\t47\t0.0\t2\t2 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 21454 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t1 0 1\n",
      "19\t143\t0.0\t1\t14 125 4\n",
      "20\t21279\t0.0\t2\t20727 540 12\n",
      "21\t29\t0.0\t2\t0 28 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV5c-719-113_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV5c-719-113_139_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV5c-719-113_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV5c-719-113_139_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV5c-719-113_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFAV5c-719-113_139_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV5c-719-113_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFAV5c-719-113_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        39,247 reads  @     47.7 µs/read;   1.26 M reads/minute\n",
      "Finished in 1.91 s (49 us/read; 1.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,247\n",
      "  Read 1 with adapter:                  39,150 (99.8%)\n",
      "  Read 2 with adapter:                  39,059 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        39,247 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,774,100 bp\n",
      "  Read 1:     5,887,050 bp\n",
      "  Read 2:     5,887,050 bp\n",
      "Total written (filtered):     10,249,685 bp (87.1%)\n",
      "  Read 1:     5,143,463 bp\n",
      "  Read 2:     5,106,222 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 39150 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t0 3\n",
      "18\t299\t0.0\t1\t45 254\n",
      "19\t38785\t0.0\t1\t37925 860\n",
      "20\t60\t0.0\t2\t6 54\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 39059 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t1 6 12\n",
      "19\t366\t0.0\t1\t40 314 12\n",
      "20\t38625\t0.0\t2\t36929 1667 29\n",
      "21\t46\t0.0\t2\t2 42 2\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFRA3d-918-221_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFRA3d-918-221_140_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFRA3d-918-221_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFRA3d-918-221_140_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFRA3d-918-221_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/OFRA3d-918-221_140_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFRA3d-918-221_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/OFRA3d-918-221_140_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        16,226 reads  @     69.2 µs/read;   0.87 M reads/minute\n",
      "Finished in 1.16 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,226\n",
      "  Read 1 with adapter:                  16,199 (99.8%)\n",
      "  Read 2 with adapter:                  16,174 (99.7%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        16,226 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,867,800 bp\n",
      "  Read 1:     2,433,900 bp\n",
      "  Read 2:     2,433,900 bp\n",
      "Total written (filtered):      4,236,738 bp (87.0%)\n",
      "  Read 1:     2,126,178 bp\n",
      "  Read 2:     2,110,560 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 16199 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t80\t0.0\t1\t9 71\n",
      "19\t16095\t0.0\t1\t15754 341\n",
      "20\t23\t0.0\t2\t1 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 16174 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t144\t0.0\t1\t10 129 5\n",
      "20\t16009\t0.0\t2\t15500 504 5\n",
      "21\t16\t0.0\t2\t1 14 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI1c-719-20_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI1c-719-20_141_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI1c-719-20_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI1c-719-20_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI1c-719-20_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI1c-719-20_141_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI1c-719-20_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI1c-719-20_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        10,396 reads  @    104.2 µs/read;   0.58 M reads/minute\n",
      "Finished in 1.12 s (108 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,396\n",
      "  Read 1 with adapter:                  10,362 (99.7%)\n",
      "  Read 2 with adapter:                  10,352 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        10,396 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,118,800 bp\n",
      "  Read 1:     1,559,400 bp\n",
      "  Read 2:     1,559,400 bp\n",
      "Total written (filtered):      2,715,007 bp (87.1%)\n",
      "  Read 1:     1,362,570 bp\n",
      "  Read 2:     1,352,437 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 10362 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t55\t0.0\t1\t7 48\n",
      "19\t10295\t0.0\t1\t10107 188\n",
      "20\t9\t0.0\t2\t0 9\n",
      "21\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 10352 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 0 2\n",
      "19\t82\t0.0\t1\t9 70 3\n",
      "20\t10259\t0.0\t2\t9821 428 10\n",
      "21\t9\t0.0\t2\t0 8 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI1d-716-118_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI1d-716-118_142_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI1d-716-118_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI1d-716-118_142_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI1d-716-118_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI1d-716-118_142_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI1d-716-118_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI1d-716-118_142_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        36,087 reads  @     44.4 µs/read;   1.35 M reads/minute\n",
      "Finished in 1.64 s (45 us/read; 1.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,087\n",
      "  Read 1 with adapter:                  36,009 (99.8%)\n",
      "  Read 2 with adapter:                  35,875 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        36,087 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,826,100 bp\n",
      "  Read 1:     5,413,050 bp\n",
      "  Read 2:     5,413,050 bp\n",
      "Total written (filtered):      9,424,878 bp (87.1%)\n",
      "  Read 1:     4,729,056 bp\n",
      "  Read 2:     4,695,822 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 36009 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t6\t0.0\t1\t6\n",
      "18\t209\t0.0\t1\t38 171\n",
      "19\t35738\t0.0\t1\t34973 765\n",
      "20\t53\t0.0\t2\t4 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 35875 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t292\t0.0\t1\t37 249 6\n",
      "20\t35530\t0.0\t2\t34089 1413 28\n",
      "21\t44\t0.0\t2\t0 44\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI2c-719-27_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI2c-719-27_143_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI2c-719-27_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI2c-719-27_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI2c-719-27_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI2c-719-27_143_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI2c-719-27_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI2c-719-27_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        34,615 reads  @     55.9 µs/read;   1.07 M reads/minute\n",
      "Finished in 1.97 s (57 us/read; 1.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,615\n",
      "  Read 1 with adapter:                  34,545 (99.8%)\n",
      "  Read 2 with adapter:                  34,436 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,615 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,384,500 bp\n",
      "  Read 1:     5,192,250 bp\n",
      "  Read 2:     5,192,250 bp\n",
      "Total written (filtered):      9,039,831 bp (87.1%)\n",
      "  Read 1:     4,536,045 bp\n",
      "  Read 2:     4,503,786 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 34545 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t0 1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t193\t0.0\t1\t24 169\n",
      "19\t34294\t0.0\t1\t33543 751\n",
      "20\t54\t0.0\t2\t2 49 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 34436 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "18\t6\t0.0\t1\t1 0 5\n",
      "19\t292\t0.0\t1\t28 250 14\n",
      "20\t34087\t0.0\t2\t32785 1275 27\n",
      "21\t47\t0.0\t2\t4 42 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI3c-719-22_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI3c-719-22_144_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI3c-719-22_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI3c-719-22_144_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI3c-719-22_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI3c-719-22_144_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI3c-719-22_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI3c-719-22_144_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        28,456 reads  @     45.6 µs/read;   1.32 M reads/minute\n",
      "Finished in 1.33 s (47 us/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,456\n",
      "  Read 1 with adapter:                  28,409 (99.8%)\n",
      "  Read 2 with adapter:                  28,323 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,456 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,536,800 bp\n",
      "  Read 1:     4,268,400 bp\n",
      "  Read 2:     4,268,400 bp\n",
      "Total written (filtered):      7,430,927 bp (87.0%)\n",
      "  Read 1:     3,728,719 bp\n",
      "  Read 2:     3,702,208 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 28409 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t151\t0.0\t1\t24 127\n",
      "19\t28194\t0.0\t1\t27559 635\n",
      "20\t63\t0.0\t2\t1 61 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 28323 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t6\t0.0\t1\t2 0 4\n",
      "19\t293\t0.0\t1\t30 255 8\n",
      "20\t27984\t0.0\t2\t27069 899 16\n",
      "21\t38\t0.0\t2\t1 36 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI4c-719-28_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI4c-719-28_145_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI4c-719-28_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI4c-719-28_145_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI4c-719-28_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI4c-719-28_145_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI4c-719-28_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI4c-719-28_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        13,204 reads  @     61.7 µs/read;   0.97 M reads/minute\n",
      "Finished in 0.85 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,204\n",
      "  Read 1 with adapter:                  13,185 (99.9%)\n",
      "  Read 2 with adapter:                  13,152 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        13,204 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,961,200 bp\n",
      "  Read 1:     1,980,600 bp\n",
      "  Read 2:     1,980,600 bp\n",
      "Total written (filtered):      3,447,784 bp (87.0%)\n",
      "  Read 1:     1,730,134 bp\n",
      "  Read 2:     1,717,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 13185 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t70\t0.0\t1\t10 60\n",
      "19\t13091\t0.0\t1\t12831 260\n",
      "20\t23\t0.0\t2\t2 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 13152 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t109\t0.0\t1\t13 94 2\n",
      "20\t13021\t0.0\t2\t12625 389 7\n",
      "21\t21\t0.0\t2\t0 20 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI5c-719-98_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI5c-719-98_146_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI5c-719-98_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI5c-719-98_146_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI5c-719-98_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PCLI5c-719-98_146_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI5c-719-98_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PCLI5c-719-98_146_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        10,776 reads  @     96.5 µs/read;   0.62 M reads/minute\n",
      "Finished in 1.07 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,776\n",
      "  Read 1 with adapter:                  10,750 (99.8%)\n",
      "  Read 2 with adapter:                  10,716 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        10,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,232,800 bp\n",
      "  Read 1:     1,616,400 bp\n",
      "  Read 2:     1,616,400 bp\n",
      "Total written (filtered):      2,814,388 bp (87.1%)\n",
      "  Read 1:     1,412,203 bp\n",
      "  Read 2:     1,402,185 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 10750 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t69\t0.0\t1\t11 58\n",
      "19\t10665\t0.0\t1\t10442 223\n",
      "20\t16\t0.0\t2\t0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 10716 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t4\t0.0\t1\t0 1 3\n",
      "19\t111\t0.0\t1\t9 97 5\n",
      "20\t10588\t0.0\t2\t10166 417 5\n",
      "21\t12\t0.0\t2\t0 12\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1c-719-91_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1c-719-91_147_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1c-719-91_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1c-719-91_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1c-719-91_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1c-719-91_147_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1c-719-91_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1c-719-91_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        23,365 reads  @     50.5 µs/read;   1.19 M reads/minute\n",
      "Finished in 1.21 s (52 us/read; 1.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,365\n",
      "  Read 1 with adapter:                  23,303 (99.7%)\n",
      "  Read 2 with adapter:                  23,282 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        23,365 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,009,500 bp\n",
      "  Read 1:     3,504,750 bp\n",
      "  Read 2:     3,504,750 bp\n",
      "Total written (filtered):      6,101,489 bp (87.0%)\n",
      "  Read 1:     3,062,137 bp\n",
      "  Read 2:     3,039,352 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 23303 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t2 2\n",
      "18\t187\t0.0\t1\t26 161\n",
      "19\t23057\t0.0\t1\t22502 555\n",
      "20\t54\t0.0\t2\t5 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 23282 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t1\n",
      "18\t8\t0.0\t1\t1 1 6\n",
      "19\t256\t0.0\t1\t29 219 8\n",
      "20\t22980\t0.0\t2\t22091 870 19\n",
      "21\t36\t0.0\t2\t1 34 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1d-716-32_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1d-716-32_148_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1d-716-32_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1d-716-32_148_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1d-716-32_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1d-716-32_148_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1d-716-32_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1d-716-32_148_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        18,880 reads  @     60.9 µs/read;   0.99 M reads/minute\n",
      "Finished in 1.17 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,880\n",
      "  Read 1 with adapter:                  18,834 (99.8%)\n",
      "  Read 2 with adapter:                  18,795 (99.5%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        18,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,664,000 bp\n",
      "  Read 1:     2,832,000 bp\n",
      "  Read 2:     2,832,000 bp\n",
      "Total written (filtered):      4,930,530 bp (87.1%)\n",
      "  Read 1:     2,474,245 bp\n",
      "  Read 2:     2,456,285 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 18834 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t126\t0.0\t1\t16 110\n",
      "19\t18665\t0.0\t1\t18213 452\n",
      "20\t41\t0.0\t2\t3 37 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 18795 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t4\t0.0\t1\t1 0 3\n",
      "19\t205\t0.0\t1\t32 170 3\n",
      "20\t18558\t0.0\t2\t17867 676 15\n",
      "21\t28\t0.0\t2\t1 27\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1d-918-246_70_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1d-918-246_149_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1d-918-246_70_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1d-918-246_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1d-918-246_70_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR1d-918-246_149_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1d-918-246_70_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR1d-918-246_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        15,130 reads  @     76.4 µs/read;   0.79 M reads/minute\n",
      "Finished in 1.19 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,130\n",
      "  Read 1 with adapter:                  15,095 (99.8%)\n",
      "  Read 2 with adapter:                  15,032 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        15,130 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,539,000 bp\n",
      "  Read 1:     2,269,500 bp\n",
      "  Read 2:     2,269,500 bp\n",
      "Total written (filtered):      3,951,711 bp (87.1%)\n",
      "  Read 1:     1,982,744 bp\n",
      "  Read 2:     1,968,967 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 15095 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t1\t0.0\t1\t1\n",
      "18\t69\t0.0\t1\t8 61\n",
      "19\t15003\t0.0\t1\t14689 314\n",
      "20\t21\t0.0\t2\t1 20\n",
      "21\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 15032 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t121\t0.0\t1\t15 97 9\n",
      "20\t14888\t0.0\t2\t14387 493 8\n",
      "21\t20\t0.0\t2\t1 19\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2c-719-114_71_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2c-719-114_150_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2c-719-114_71_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2c-719-114_150_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2c-719-114_71_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2c-719-114_150_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2c-719-114_71_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2c-719-114_150_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        22,137 reads  @     52.8 µs/read;   1.14 M reads/minute\n",
      "Finished in 1.20 s (54 us/read; 1.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,137\n",
      "  Read 1 with adapter:                  22,100 (99.8%)\n",
      "  Read 2 with adapter:                  22,002 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,137 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,641,100 bp\n",
      "  Read 1:     3,320,550 bp\n",
      "  Read 2:     3,320,550 bp\n",
      "Total written (filtered):      5,781,498 bp (87.1%)\n",
      "  Read 1:     2,900,785 bp\n",
      "  Read 2:     2,880,713 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 22100 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t2\t0.0\t1\t2\n",
      "17\t4\t0.0\t1\t3 1\n",
      "18\t159\t0.0\t1\t12 147\n",
      "19\t21895\t0.0\t1\t21448 447\n",
      "20\t40\t0.0\t2\t0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 22002 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 1 5\n",
      "19\t232\t0.0\t1\t27 199 6\n",
      "20\t21725\t0.0\t2\t20947 755 23\n",
      "21\t37\t0.0\t2\t2 35\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2c-917-276_72_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2c-917-276_151_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2c-917-276_72_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2c-917-276_151_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2c-917-276_72_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2c-917-276_151_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2c-917-276_72_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2c-917-276_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        16,154 reads  @     68.8 µs/read;   0.87 M reads/minute\n",
      "Finished in 1.14 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,154\n",
      "  Read 1 with adapter:                  16,131 (99.9%)\n",
      "  Read 2 with adapter:                  16,087 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        16,154 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,846,200 bp\n",
      "  Read 1:     2,423,100 bp\n",
      "  Read 2:     2,423,100 bp\n",
      "Total written (filtered):      4,218,123 bp (87.0%)\n",
      "  Read 1:     2,116,663 bp\n",
      "  Read 2:     2,101,460 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 16131 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t80\t0.0\t1\t10 70\n",
      "19\t16020\t0.0\t1\t15726 294\n",
      "20\t30\t0.0\t2\t2 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 16087 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t126\t0.0\t1\t8 114 4\n",
      "20\t15934\t0.0\t2\t15503 423 8\n",
      "21\t24\t0.0\t2\t2 22\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2d-716-122_73_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2d-716-122_152_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2d-716-122_73_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2d-716-122_152_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2d-716-122_73_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR2d-716-122_152_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2d-716-122_73_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR2d-716-122_152_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:01        12,123 reads  @     89.8 µs/read;   0.67 M reads/minute\n",
      "Finished in 1.12 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,123\n",
      "  Read 1 with adapter:                  12,109 (99.9%)\n",
      "  Read 2 with adapter:                  12,047 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        12,123 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,636,900 bp\n",
      "  Read 1:     1,818,450 bp\n",
      "  Read 2:     1,818,450 bp\n",
      "Total written (filtered):      3,166,036 bp (87.1%)\n",
      "  Read 1:     1,588,426 bp\n",
      "  Read 2:     1,577,610 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 12109 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t78\t0.0\t1\t15 63\n",
      "19\t12007\t0.0\t1\t11763 244\n",
      "20\t22\t0.0\t2\t2 20\n",
      "30\t1\t0.0\t3\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 12047 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t108\t0.0\t1\t14 93 1\n",
      "20\t11922\t0.0\t2\t11559 357 6\n",
      "21\t14\t0.0\t2\t0 14\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3c-719-116_74_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3c-719-116_153_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3c-719-116_74_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3c-719-116_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3c-719-116_74_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3c-719-116_153_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3c-719-116_74_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3c-719-116_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        30,986 reads  @     53.8 µs/read;   1.12 M reads/minute\n",
      "Finished in 1.70 s (55 us/read; 1.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,986\n",
      "  Read 1 with adapter:                  30,932 (99.8%)\n",
      "  Read 2 with adapter:                  30,849 (99.6%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,986 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,295,800 bp\n",
      "  Read 1:     4,647,900 bp\n",
      "  Read 2:     4,647,900 bp\n",
      "Total written (filtered):      8,091,421 bp (87.0%)\n",
      "  Read 1:     4,060,305 bp\n",
      "  Read 2:     4,031,116 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 30932 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t172\t0.0\t1\t20 152\n",
      "19\t30695\t0.0\t1\t30121 574\n",
      "20\t63\t0.0\t2\t3 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 30849 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t1 0 1\n",
      "19\t239\t0.0\t1\t32 198 9\n",
      "20\t30554\t0.0\t2\t29731 804 19\n",
      "21\t51\t0.0\t2\t2 48 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3c-917-269_75_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3c-917-269_154_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3c-917-269_75_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3c-917-269_154_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3c-917-269_75_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3c-917-269_154_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3c-917-269_75_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3c-917-269_154_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,647 reads  @     48.6 µs/read;   1.24 M reads/minute\n",
      "Finished in 1.62 s (50 us/read; 1.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,647\n",
      "  Read 1 with adapter:                  32,589 (99.8%)\n",
      "  Read 2 with adapter:                  32,453 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,647 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,794,100 bp\n",
      "  Read 1:     4,897,050 bp\n",
      "  Read 2:     4,897,050 bp\n",
      "Total written (filtered):      8,526,217 bp (87.1%)\n",
      "  Read 1:     4,277,990 bp\n",
      "  Read 2:     4,248,227 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 32589 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "18\t176\t0.0\t1\t20 156\n",
      "19\t32350\t0.0\t1\t31737 613\n",
      "20\t58\t0.0\t2\t5 53\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 32453 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "18\t9\t0.0\t1\t0 1 8\n",
      "19\t284\t0.0\t1\t41 242 1\n",
      "20\t32111\t0.0\t2\t31326 774 11\n",
      "21\t46\t0.0\t2\t2 44\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t4\t0 0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3d-716-121_76_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3d-716-121_155_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3d-716-121_76_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3d-716-121_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3d-716-121_76_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR3d-716-121_155_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3d-716-121_76_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR3d-716-121_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        33,651 reads  @     55.9 µs/read;   1.07 M reads/minute\n",
      "Finished in 1.92 s (57 us/read; 1.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,651\n",
      "  Read 1 with adapter:                  33,583 (99.8%)\n",
      "  Read 2 with adapter:                  33,410 (99.3%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,651 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,095,300 bp\n",
      "  Read 1:     5,047,650 bp\n",
      "  Read 2:     5,047,650 bp\n",
      "Total written (filtered):      8,789,384 bp (87.1%)\n",
      "  Read 1:     4,409,703 bp\n",
      "  Read 2:     4,379,681 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 33583 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t3\t0.0\t1\t1 2\n",
      "18\t172\t0.0\t1\t31 141\n",
      "19\t33360\t0.0\t1\t32705 655\n",
      "20\t48\t0.0\t2\t5 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 33410 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t270\t0.0\t1\t32 224 14\n",
      "20\t33087\t0.0\t2\t32096 975 16\n",
      "21\t45\t0.0\t2\t0 45\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR4d-716-79_77_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR4d-716-79_156_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR4d-716-79_77_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR4d-716-79_156_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR4d-716-79_77_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR4d-716-79_156_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR4d-716-79_77_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR4d-716-79_156_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8<----------] 00:00:00        12,563 reads  @     76.3 µs/read;   0.79 M reads/minute\n",
      "Finished in 1.00 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,563\n",
      "  Read 1 with adapter:                  12,541 (99.8%)\n",
      "  Read 2 with adapter:                  12,341 (98.2%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        12,563 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,768,900 bp\n",
      "  Read 1:     1,884,450 bp\n",
      "  Read 2:     1,884,450 bp\n",
      "Total written (filtered):      3,283,431 bp (87.1%)\n",
      "  Read 1:     1,646,218 bp\n",
      "  Read 2:     1,637,213 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 12541 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "18\t70\t0.0\t1\t4 66\n",
      "19\t12444\t0.0\t1\t12207 237\n",
      "20\t25\t0.0\t2\t3 21 1\n",
      "21\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 12341 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t1 0 5\n",
      "19\t85\t0.0\t1\t4 79 2\n",
      "20\t12216\t0.0\t2\t11862 346 8\n",
      "21\t21\t0.0\t2\t0 21\n",
      "56\t1\t0.0\t4\t0 0 0 1\n",
      "58\t11\t0.0\t4\t0 0 10 1\n",
      "59\t1\t0.0\t4\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR5d-716-81_78_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR5d-716-81_157_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR5d-716-81_78_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR5d-716-81_157_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.8 with Python 3.6.7\n",
      "Command line parameters: --cores 2 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR5d-716-81_78_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-jvt8tb1l/PSTR5d-716-81_157_L001_R2_001.fastq.gz --front ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA -G TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR5d-716-81_78_L001_R1_001.fastq.gz /tmp/qiime2-archive-mz4ani11/99814272-8b25-49ee-aa69-d636b968623f/data/PSTR5d-716-81_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 2 cores in paired-end mode ...\n",
      "[8=----------] 00:00:01        35,593 reads  @     44.9 µs/read;   1.34 M reads/minute\n",
      "Finished in 1.64 s (46 us/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,593\n",
      "  Read 1 with adapter:                  35,522 (99.8%)\n",
      "  Read 2 with adapter:                  35,382 (99.4%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,593 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,677,900 bp\n",
      "  Read 1:     5,338,950 bp\n",
      "  Read 2:     5,338,950 bp\n",
      "Total written (filtered):      9,295,739 bp (87.1%)\n",
      "  Read 1:     4,664,153 bp\n",
      "  Read 2:     4,631,586 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 41; Trimmed: 35522 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-41 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t200\t0.0\t1\t20 180\n",
      "19\t35266\t0.0\t1\t34522 744\n",
      "20\t47\t0.0\t2\t4 43\n",
      "22\t1\t0.0\t2\t0 1\n",
      "33\t1\t0.0\t3\t1\n",
      "48\t1\t0.0\t4\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 42; Trimmed: 35382 times; Reverse-complemented: 0 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-42 bp: 4\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t309\t0.0\t1\t32 269 8\n",
      "20\t35007\t0.0\t2\t33729 1252 26\n",
      "21\t55\t0.0\t2\t3 52\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: Folder_StewartMedina_FL2019/trim_StewartMedina_FL2019.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#trim PCR primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences Folder_StewartMedina_FL2019/Import_StewartMedina_FL2019.qza \\\n",
    "--p-cores 2 \\\n",
    "--p-front-f ACACTGACGACATGGTTCTACAGTGCCAGCMGCCGCGGTAA \\\n",
    "--p-front-r TACGGTAGCAGAGACTTGGTCTGGACTACHVGGGTWTCTAAT \\\n",
    "--o-trimmed-sequences Folder_StewartMedina_FL2019/trim_StewartMedina_FL2019.qza \\\n",
    "--verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_StewartMedina_FL2019/trim_StewartMedina_FL2019.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "--verbose \\\n",
    "--i-data Folder_StewartMedina_FL2019/trim_StewartMedina_FL2019.qza \\\n",
    "--o-visualization Folder_StewartMedina_FL2019/trim_StewartMedina_FL2019.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_StewartMedina_FL2019/table_StewartMedina_FL2019.qza\u001b[0m\r\n",
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_StewartMedina_FL2019/rep_StewartMedina_FL2019.qza\u001b[0m\r\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: Folder_StewartMedina_FL2019/dada_stats_StewartMedina_FL2019.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime dada2 denoise-paired \\\n",
    "--i-demultiplexed-seqs Folder_StewartMedina_FL2019/trim_StewartMedina_FL2019.qza \\\n",
    "--p-trunc-len-f 0 \\\n",
    "--p-trunc-len-r  0 \\\n",
    "--p-n-threads 10  \\\n",
    "--o-representative-sequences Folder_StewartMedina_FL2019/rep_StewartMedina_FL2019.qza \\\n",
    "--o-table Folder_StewartMedina_FL2019/table_StewartMedina_FL2019.qza \\\n",
    "--o-denoising-stats Folder_StewartMedina_FL2019/dada_stats_StewartMedina_FL2019"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_StewartMedina_FL2019/table_StewartMedina_FL2019.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_StewartMedina_FL2019/rep_StewartMedina_FL2019.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: Folder_StewartMedina_FL2019/dada_stats_StewartMedina_FL2019.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime dada2 denoise-single \\\n",
    "--i-demultiplexed-seqs Folder_StewartMedina_FL2019/trim_StewartMedina_FL2019.qza \\\n",
    "--p-trunc-len 130 \\\n",
    "--p-n-threads 10  \\\n",
    "--p-max-ee 3 \\\n",
    "--o-representative-sequences Folder_StewartMedina_FL2019/rep_StewartMedina_FL2019.qza \\\n",
    "--o-table Folder_StewartMedina_FL2019/table_StewartMedina_FL2019.qza \\\n",
    "--o-denoising-stats Folder_StewartMedina_FL2019/dada_stats_StewartMedina_FL2019"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Taxonomy] to: Folder_StewartMedina_FL2019/taxonomy_StewartMedina_FL2019.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier /space/home/rosales/databases/SILVA_DBs/silva-138-99-nb-classifier.qza \\\n",
    "  --i-reads Folder_StewartMedina_FL2019/rep_StewartMedina_FL2019.qza \\\n",
    "  --p-n-jobs -7 \\\n",
    "  --o-classification Folder_StewartMedina_FL2019/taxonomy_StewartMedina_FL2019.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Folder_StewartMedina_FL2019/table-BacArc_StewartMedina_FL2019.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: Folder_StewartMedina_FL2019/rep_seqs_BacArc_StewartMedina_FL2019.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime taxa filter-table \\\n",
    "--i-table Folder_StewartMedina_FL2019/table_StewartMedina_FL2019.qza \\\n",
    "--i-taxonomy Folder_StewartMedina_FL2019/taxonomy_StewartMedina_FL2019.qza \\\n",
    "--p-exclude mitochondria,chloroplast \\\n",
    "--p-include d__Bacteria,d__Archaea \\\n",
    "--o-filtered-table Folder_StewartMedina_FL2019/table-BacArc_StewartMedina_FL2019.qza\n",
    "\n",
    "!qiime taxa filter-seqs \\\n",
    "--i-sequences Folder_StewartMedina_FL2019/rep_StewartMedina_FL2019.qza \\\n",
    "--i-taxonomy Folder_StewartMedina_FL2019/taxonomy_StewartMedina_FL2019.qza \\\n",
    "--p-exclude mitochondria,chloroplast \\\n",
    "--p-include d__Bacteria,d__Archaea \\\n",
    "--o-filtered-sequences  Folder_StewartMedina_FL2019/rep_seqs_BacArc_StewartMedina_FL2019.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Folder_StewartMedina_FL2019/taxonomy_StewartMedina_FL2019.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: Folder_StewartMedina_FL2019/taxa-bar-Stewart_Medina_KeyWest16S2019.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file Folder_StewartMedina_FL2019/taxonomy_StewartMedina_FL2019.qza \\\n",
    "  --o-visualization Folder_StewartMedina_FL2019/taxonomy_StewartMedina_FL2019.qzv\n",
    "\n",
    "\n",
    "!qiime taxa barplot \\\n",
    "  --i-table Folder_StewartMedina_FL2019/table-BacArc_StewartMedina_FL2019.qza \\\n",
    "  --i-taxonomy Folder_StewartMedina_FL2019/taxonomy_StewartMedina_FL2019.qza \\\n",
    "  --m-metadata-file  metadata/metadata_Stewart_Medina_KeyWest16S2019.txt \\\n",
    "  --o-visualization Folder_StewartMedina_FL2019/taxa-bar-Stewart_Medina_KeyWest16S2019.qzv"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
